# MAIN TEXT

## Spectrum and characterisation of BRCA1 and BRCA2 deleterious mutations in high-risk Czech patients with breast and/or ovarian cancer

### Abstract

BackgroundThe incidence of breast cancer has doubled over the past 20 years in the Czech Republic. Hereditary factors may be a cause of young onset, bilateral breast or ovarian cancer, and familial accumulation of the disease. BRCA1 and BRCA2 mutations account for an important fraction of hereditary breast and ovarian cancer cases. One thousand and ten unrelated high-risk probands with breast and/or ovarian cancer were analysed for the presence of a BRCA1 or BRCA2 gene mutation at the Masaryk Memorial Cancer Institute (Czech Republic) during 1999–2006.MethodsThe complete coding sequences and splice sites of both genes were screened, and the presence of large intragenic rearrangements in BRCA1 was verified. Putative splice-site variants were analysed at the cDNA level for their potential to alter mRNA splicing.ResultsIn 294 unrelated families (29.1% of the 1,010 probands) pathogenic mutations were identified, with 44 different BRCA1 mutations and 41 different BRCA2 mutations being detected in 204 and 90 unrelated families, respectively. In total, three BRCA1 founder mutations (c.5266dupC; c.3700_3704del5; p.Cys61Gly) and two BRCA2 founder mutations (c.7913_7917del5; c.8537_8538del2) represent 52% of all detected mutations in Czech high-risk probands. Nine putative splice-site variants were evaluated at the cDNA level. Three splice-site variants in BRCA1 (c.302-3C>G; c.4185G>A and c.4675+1G>A) and six splice-site variants in BRCA2 (c.475G>A; c.476-2>G; c.7007G>A; c.8755-1G>A; c.9117+2T>A and c.9118-2A>G) were demonstrated to result in aberrant transcripts and are considered as deleterious mutations.ConclusionThis study represents an evaluation of deleterious genetic variants in the BRCA1 and 2 genes in the Czech population. The classification of several splice-site variants as true pathogenic mutations may prove useful for genetic counselling of families with high risk of breast and ovarian cancer.

### Background

The incidence of breast cancer has doubled over the past 20 years in the Czech Republic [1]. A family history of breast cancer is one of the most significant risk factors for development of the disease. Hereditary factors may cause early onset, bilateral breast or ovarian cancer, and familial accumulation of the disease. Approximately 5 to 10% of all breast cancer cases, and 25 to 40% of cases diagnosed under the age of 35 years, have hereditary origins [2].

Mutations in BRCA1 (breast cancer 1, early onset; MIM#113705) and BRCA2 (Breast Cancer Type 2 susceptibility protein; MIM#600185) account for an autosomal dominant transmission of susceptibility to breast and ovarian cancer. Among female BRCA1 mutation carriers, the lifetime risk of breast cancer has been estimated at between 65 and 85%, and the risk of ovarian cancer between 39 and 63% [3-5]. For BRCA2 mutation carriers, the lifetime risk for breast cancer ranges from 45 to 85%, and between 11 and 27% for ovarian cancer [3,5]. The large variations suggest that cancer risk associated with BRCA mutations may be modified by other genetic factors in addition to lifestyle and environmental conditions [6].

With the exception of alterations in the extreme carboxy-terminal of BRCA2 (3' of the polymorphic stop codon K3326X), all BRCA1 and BRCA2 truncation mutations are considered to be deleterious [7,8]. Experimental data have revealed the presence of two nuclear localisation signals (NLS) within the final 156 amino acids of the BRCA2 protein. These data indicate that all disease-linked truncating mutations localised upstream of the NLS are likely to be non-functional, losing their ability to translocate to the nucleus [9].

Aside from the many unequivocal pathogenic mutations, several variants of unknown clinical significance have been reported in the BRCA1 and BRCA2 genes. Messenger ribonucleic acid (mRNA) analysis has only been performed, and the pathogenic effect clearly demonstrated, for a limited number of splice-site alterations [10,11]. In addition, only a few missense mutations have been accepted as deleterious disease-associated genetic alterations that can be reliably used for clinical management. When assessing the clinical significance of a detected variant, several approaches of further analysis are recommended to classify variants as deleterious, suspected deleterious, of unknown (uncertain) clinical significance or polymorphic/neutral [12].

The aim of our study was to demonstrate the pathogenity of all suspected deleterious variants detected in the high-risk Czech population.

### Methods

Patients were referred to the Masaryk Memorial Cancer Institute (MMCI) in Brno for genetic counselling by physicians from various specialisations or were sent for testing by other medical geneticists from various parts of the Czech Republic between 1999 and 2005. All tested individuals provided a signed informed consent following appropriate genetic counselling. Genetic testing was offered to high-risk individuals meeting any of the following criteria:

I. At least 3 diagnoses of breast and/or ovarian cancer in first and/or second degree relatives, diagnosed at any age (320 families).

II. Two affected first degree relatives (or second degree in the case of paternal transmission) diagnosed with breast and/or ovarian cancer, at least one of them diagnosed before the age of 50 (252 families).

III. Any patient with a) bilateral breast, first diagnosed before 50 (29 patients), or b) bilateral ovarian cancer, diagnosed before 50 (7 patients), or c) both breast and ovarian cancer, without an apparent family history of these cancers (19 patients).

IV. Early-onset sporadic unilateral breast (121 patients) or ovarian (19 patients) cancer cases diagnosed before the age of 40.

V. Any male patient with breast cancer, independent of age at diagnosis and family history (16 males who do not fulfil criteria I or II).

VI. In several families with hereditary breast and/or ovarian cancer predisposition, none of the affected relatives was available for testing. Healthy high-risk individuals, who had a 50% chance of being a carrier of the familial mutation, were therefore analysed (77 families).

VII. In addition, we analysed 150 affected probands on the borderline of our inclusion criteria (such as probands from small families with 2 affected relatives diagnosed between 51 to 60 years of age, or patients diagnosed before the age of 50 with a third degree relative with breast/ovarian cancer) referred by clinical geneticists.

Inclusion criteria I, II and VI were considered as familial cases, whilst criteria III, IV and V were considered as severe sporadic cases.

Genomic DNA was isolated from blood samples with the QIAamp DNA blood purification kit (Qiagen). The non-radioactive protein truncation test (PTT) (Promega) was used for analysis of exon 11 of the BRCA1 gene, and for exons 10 and 11 of the BRCA2 gene. The remaining exons and their splice sites were screened by Heteroduplex analysis (HA) using Mutation Detection Enhancement (MDE) gel solution (Cambrex), essentially as described by Claes et al. [13]. Fragments with aberrant mobility, detected on PTT or HA gels, were sequenced on an independently amplified polymerase chain reaction (PCR) product by direct sequencing using the Thermo Sequenase Primer Cycle Sequencing Kit, or the Thermo Sequenase Cy5 Dye Terminator Cycle Sequencing Kit (Amersham Biosciences), and analysed on the ALF express™ DNA sequencer (Pharmacia). Each newly found sequence alteration was confirmed by sequencing from both forward and reverse directions. Each detected mutation was confirmed on an independently drawn blood sample. In the case of a deceased patient, a germline mutation was confirmed on DNA isolated from non-tumour paraffin embedded tissue.

In addition, we evaluated the frequency of large genomic rearrangements in the BRCA1 gene with Multiplex Ligation-dependent Probe Amplification, or MLPA (MRC-Holland). More details about this study are reported elsewhere [14].

All putative splice-site variants were tested by the Splice Site Prediction Programs for their potential to alter splicing [11]. Subsequently, complementary DNA (cDNA) analysis was performed to verify the in silico results.

Total ribonucleic acid (RNA), treated with a nonsense-mediated mRNA decay inhibitor (Puromycin), was extracted from short-term PHA-stimulated lymphocyte cultures [15] of the patient and from controls not carrying the studied splice-site alteration.

The region of the BRCA1/BRCA2 cDNA encompassing the relevant fragment was amplified by PCR. If more than two aberrant transcripts were obtained, the PCR products derived from cDNA templates were cloned into the vector PCR 2.1-TOPO and transfected into Escherichia coli strain Top10F (Invitrogen). After amplification with M13 primers, the products were resolved on agarose gels and sequenced. The cDNA analysis approach has previously been described in more detail by Claes et al. [11].

Detected mutations and other sequence alterations are described at the cDNA level according to the GenBank: U14680 BRCA1 reference sequence and GenBank: NM_000059 BRCA2 reference sequence, using the recommended nomenclature system for human gene mutations [16].

The cDNA numbering for the traditional mutation nomenclature used in the BIC (Breast Cancer Information Core) Database is based on reference sequences as stated above, where the A of the ATG translation initiation codon is at position 120 of the BRCA1 mRNA and at position 229 of the BRCA2 mRNA, respectively. The HUGO-approved systematic nomenclature follows the rule where the nucleotide +1 is the A of the ATG translation initiation codon. In the text of this article, we use the systematic nomenclature. In the tables, both BIC and systematic cDNA numbering are used.

### Patients and criteria for testing

Patients were referred to the Masaryk Memorial Cancer Institute (MMCI) in Brno for genetic counselling by physicians from various specialisations or were sent for testing by other medical geneticists from various parts of the Czech Republic between 1999 and 2005. All tested individuals provided a signed informed consent following appropriate genetic counselling. Genetic testing was offered to high-risk individuals meeting any of the following criteria:

I. At least 3 diagnoses of breast and/or ovarian cancer in first and/or second degree relatives, diagnosed at any age (320 families).

II. Two affected first degree relatives (or second degree in the case of paternal transmission) diagnosed with breast and/or ovarian cancer, at least one of them diagnosed before the age of 50 (252 families).

III. Any patient with a) bilateral breast, first diagnosed before 50 (29 patients), or b) bilateral ovarian cancer, diagnosed before 50 (7 patients), or c) both breast and ovarian cancer, without an apparent family history of these cancers (19 patients).

IV. Early-onset sporadic unilateral breast (121 patients) or ovarian (19 patients) cancer cases diagnosed before the age of 40.

V. Any male patient with breast cancer, independent of age at diagnosis and family history (16 males who do not fulfil criteria I or II).

VI. In several families with hereditary breast and/or ovarian cancer predisposition, none of the affected relatives was available for testing. Healthy high-risk individuals, who had a 50% chance of being a carrier of the familial mutation, were therefore analysed (77 families).

VII. In addition, we analysed 150 affected probands on the borderline of our inclusion criteria (such as probands from small families with 2 affected relatives diagnosed between 51 to 60 years of age, or patients diagnosed before the age of 50 with a third degree relative with breast/ovarian cancer) referred by clinical geneticists.

Inclusion criteria I, II and VI were considered as familial cases, whilst criteria III, IV and V were considered as severe sporadic cases.

### Mutation screening

Genomic DNA was isolated from blood samples with the QIAamp DNA blood purification kit (Qiagen). The non-radioactive protein truncation test (PTT) (Promega) was used for analysis of exon 11 of the BRCA1 gene, and for exons 10 and 11 of the BRCA2 gene. The remaining exons and their splice sites were screened by Heteroduplex analysis (HA) using Mutation Detection Enhancement (MDE) gel solution (Cambrex), essentially as described by Claes et al. [13]. Fragments with aberrant mobility, detected on PTT or HA gels, were sequenced on an independently amplified polymerase chain reaction (PCR) product by direct sequencing using the Thermo Sequenase Primer Cycle Sequencing Kit, or the Thermo Sequenase Cy5 Dye Terminator Cycle Sequencing Kit (Amersham Biosciences), and analysed on the ALF express™ DNA sequencer (Pharmacia). Each newly found sequence alteration was confirmed by sequencing from both forward and reverse directions. Each detected mutation was confirmed on an independently drawn blood sample. In the case of a deceased patient, a germline mutation was confirmed on DNA isolated from non-tumour paraffin embedded tissue.

In addition, we evaluated the frequency of large genomic rearrangements in the BRCA1 gene with Multiplex Ligation-dependent Probe Amplification, or MLPA (MRC-Holland). More details about this study are reported elsewhere [14].

### cDNA analysis

All putative splice-site variants were tested by the Splice Site Prediction Programs for their potential to alter splicing [11]. Subsequently, complementary DNA (cDNA) analysis was performed to verify the in silico results.

Total ribonucleic acid (RNA), treated with a nonsense-mediated mRNA decay inhibitor (Puromycin), was extracted from short-term PHA-stimulated lymphocyte cultures [15] of the patient and from controls not carrying the studied splice-site alteration.

The region of the BRCA1/BRCA2 cDNA encompassing the relevant fragment was amplified by PCR. If more than two aberrant transcripts were obtained, the PCR products derived from cDNA templates were cloned into the vector PCR 2.1-TOPO and transfected into Escherichia coli strain Top10F (Invitrogen). After amplification with M13 primers, the products were resolved on agarose gels and sequenced. The cDNA analysis approach has previously been described in more detail by Claes et al. [11].

### Nomenclature

Detected mutations and other sequence alterations are described at the cDNA level according to the GenBank: U14680 BRCA1 reference sequence and GenBank: NM_000059 BRCA2 reference sequence, using the recommended nomenclature system for human gene mutations [16].

The cDNA numbering for the traditional mutation nomenclature used in the BIC (Breast Cancer Information Core) Database is based on reference sequences as stated above, where the A of the ATG translation initiation codon is at position 120 of the BRCA1 mRNA and at position 229 of the BRCA2 mRNA, respectively. The HUGO-approved systematic nomenclature follows the rule where the nucleotide +1 is the A of the ATG translation initiation codon. In the text of this article, we use the systematic nomenclature. In the tables, both BIC and systematic cDNA numbering are used.

### Results

In this study, we describe molecular genetic analyses of the BRCA1 and BRCA2 genes in 1,010 unrelated high-risk probands. Through PCR based screening methods, 188 families were identified as carrying deleterious BRCA1 mutations (34 different mutations – Table 1) and 80 families as harbouring deleterious BRCA2 mutations (35 different mutations – Table 2). Furthermore, we identified 10 different alterations in highly conserved splice sites in 15 families (Table 3), which were analysed at the mRNA level. In 11 additional families, 6 different large intragenic rearrangements in the BRCA1 gene were detected by MLPA. These were further characterised and are described in detail elsewhere [14]. Altogether, a pathogenic mutation was found in 294 of 1,010 (29.1%) unrelated families (Tables 1 to 3).

The spectrum of the BRCA1 mutations is summarised in Tables 1 and 3. As generally reported, the majority of the BRCA1 mutations were mutations leading to a premature stop codon (21 different frame-shift, 9 different nonsense, and 4 different splice-site mutations (described below). Furthermore, we identified 4 different deleterious missense mutations localised in the 100% conserved cysteine residues of the BRCA1 C3HC4 RING domain: p.Cys24Tyr; p.Cys39Arg; p.Cys61Gly and p.Cys64Tyr.

Spectrum of frame-shift, missense and nonsense BRCA1 germline mutations detected in Czech patients.

BRCA1 reference sequence GenBank: U14680. Mutations marked in bold had not been submitted to the BIC database from any other population prior to November 2007 [21].

Spectrum of frame-shift and nonsense BRCA2 germline mutations detected in Czech patients.

BRCA2 reference sequence GenBank: U43746. Mutations marked in bold had not been submitted to the BIC database from any other population prior to November 2007 [21].

Splice-site alterations detected in Czech high-risk breast/ovarian cancer patients.

* Predicted consensus values by the Splice Site Prediction Program: Neural Networks [23].

All deleterious mutations identified in the BRCA2 gene were truncating mutations (30 different frame-shift mutations, 5 different nonsense mutations, and 6 different splice-site mutations (described below)) and are summarised in Tables 2 and 3.

For all putative splice-site variants, we performed further analyses to unequivocally determine their pathogenic effect. We have previously reported that BRCA1 c.4185G>A and BRCA2 c.9118A-2>G lead to aberrant transcripts [11]. In this publication, we provide results of the effects at the cDNA level for 7 additional putative splice-site variants that have not been described previously, i.e. two variants in the BRCA1 gene (c.302-3C>G and c.4675+1G>A) and five in the BRCA2 gene (c.475G>A; c.476-2>G; c.7007G>A; c.8755-1G>A and c.9117+2T>A).

To set aside nonsense-mediated mRNA decay (NMD), an inhibitor of the NMD pathway was used before RNA isolation from lymphocyte cultures. As we had previously found that splice-site mutations not only lead to transcripts affecting the exon following or preceding the mutation but may also affect other neighbouring exons, we chose RT-PCR primers that were not in the immediately adjacent exons.

Results of the RT-PCR experiments are shown in Figure 1. The predicted consensus values, the experimentally determined effect on mRNA, and deduced results at the protein level are summarised in Table 3. For 4 of the putative splice-site mutations analysed (BRCA1 c.4675+1G>A, BRCA2 c.476-2A>G, c.8755-1G>A and c.9117+2T>A), aberrant splicing was anticipated as these variants affect the highly conserved nucleotides at positions +1+2/-1-2 of the splice donor/acceptor sites.

RT-PCR results for the splice-site mutations identified in our patient population. Below the picture we indicate the position of the RT primers used. C = control. A detailed description of the relevant fragments obtained can be found in Table 3. However, we omitted in this Table details about several novel alternative transcripts containing intronic sequences with high acceptor/donor scores at the 5'/3' ends, complicating the interpretation of the results and of which the clinical relevance is currently unknown. For instance, cloning RT-PCR products spanning exons 21 to 26 revealed several novel alternative transcripts containing insertions from sequences of introns 24 and 25 (cfr. extra bands above the full length fragment in control persons and bands around 920–930 bp in the patient with BRCA2 c.9117+2T>A, representing transcripts lacking exon 23 but containing these intronic sequences. Similar observations were obtained for the RT-PCR analyses for BRCA2 c.8755-1G>A.

Furthermore, we validated 3 variants outside the highly conserved +1+2/-1–2 nucleotides. BRCA1 c.302-3C>G activates a cryptic splice site in intron 6, leading to the inclusion of two extra base pairs from this intron. This results in a frame-shift and a premature termination of translation.

The G to A transition in the last nucleotide of BRCA2 exon 5 (c.475G>A) results in a transcript lacking in frame exon 5, which was absent in control individuals. Unfortunately, no patient with both this mutation and an SNP in exons 2–10 of BRCA2 was available to allow investigation of whether the mutant allele was still able to transcribe a full- length product. This mutation was identified in two unrelated families that were too small to evaluate segregation.

RT-PCR with a forward primer located in exon 11 and a reverse primer in exon 15 revealed that the BRCA2 c.7007G>A mutation, in the last nucleotide of exon 13, is not a missense mutation (p.Arg2336His). We found a transcript lacking exon 13, a transcript lacking exons 12 and 13, and a transcript lacking exon 12. The transcript lacking exon 12 is present at about the same level in blood lymphocytes of both the patient with the mutation and a control person (data of the cloning experiments not shown). The BRCA2 c.7007G>A mutation was found in 2 unrelated families: in an affected female with a severe family history of breast cancer, fulfilling criterion I, and in an affected male with a maternal aunt and cousin diagnosed with breast cancer.

BRCA1 mutations were found to be about 2 times more frequent than BRCA2 mutations in our study population. Mutation detection frequencies (deleterious and splice-site mutations and large intragenic rearrangements) in relation to the inclusion criteria are summarised in Table 4. Surprisingly, the highest mutation frequency (14 out of 19 cases, or 73.7%) was obtained in apparently sporadic patients diagnosed with both breast and ovarian cancers. By re-evaluation of the pedigrees, we found that the apparent lack of a family history might have been due to a small number of relatives and/or paternal transmission in these families. As expected, a higher mutation detection rate was obtained in breast and ovarian (HBOC) cancer families fulfilling criteria I and II (96/160, or 60%) as compared to breast cancer only (HBC) families fulfilling the same criteria (114/412, or 27.7%).

Summary of deleterious mutations in BRCA genes identified in relation to inclusion criteria and phenotype.

Abbreviations: inclusion criteria – see Methods section; mutation – only deleterious mutations summarised in Table 1 and Table 2 are considered; CI – Confidence Interval; HOC – hereditary ovarian cancer syndrome; HBOC – hereditary breast and ovarian cancer syndrome; HBC – hereditary breast cancer only syndrome.

Six out of 16 male breast cancer patients (37.5%) carried a deleterious mutation. Unexpectedly, the same number of mutations was identified in the BRCA2 gene (c.7913_7917del5; c.9403delC, and the splice-site mutation c.7007G>A) as in the BRCA1 gene (2 males carrying c.5266dupC and one patient with c.3700_3704del5). The average age of male breast cancer diagnosis in mutation carriers was 72.5 years (ranging from 58 to 80 years). Four of the mutation carriers reported a family history of breast or prostate cancer but did not meet inclusion criteria II. The average age at diagnosis of breast cancer of the 10 males without a BRCA mutation was 57.2 years, ranging from 37 to 80 years. In addition, we identified rare variants of unknown significance in four male patients: 2 in BRCA1 (c.5277+48_5277+59dup12; c. 5277+78G>A) and 2 in the BRCA2 gene (p.Ala75Pro; c.10095delCins11).

A deleterious mutation was detected in 20 out of 77 healthy, unaffected probands (26%; 15 in BRCA1 and 5 in the BRCA2 gene) with a severe family history of breast and/or ovarian cancer but no living affected relative. Furthermore, we identified a deleterious mutation in 20 out of 150 affected probands (13.3%; 12 in BRCA1 and 8 in the BRCA2 gene) who did not fully meet our inclusion criteria for testing but who were referred by genetic counsellors.

### Deleterious BRCA mutations

In this study, we describe molecular genetic analyses of the BRCA1 and BRCA2 genes in 1,010 unrelated high-risk probands. Through PCR based screening methods, 188 families were identified as carrying deleterious BRCA1 mutations (34 different mutations – Table 1) and 80 families as harbouring deleterious BRCA2 mutations (35 different mutations – Table 2). Furthermore, we identified 10 different alterations in highly conserved splice sites in 15 families (Table 3), which were analysed at the mRNA level. In 11 additional families, 6 different large intragenic rearrangements in the BRCA1 gene were detected by MLPA. These were further characterised and are described in detail elsewhere [14]. Altogether, a pathogenic mutation was found in 294 of 1,010 (29.1%) unrelated families (Tables 1 to 3).

The spectrum of the BRCA1 mutations is summarised in Tables 1 and 3. As generally reported, the majority of the BRCA1 mutations were mutations leading to a premature stop codon (21 different frame-shift, 9 different nonsense, and 4 different splice-site mutations (described below). Furthermore, we identified 4 different deleterious missense mutations localised in the 100% conserved cysteine residues of the BRCA1 C3HC4 RING domain: p.Cys24Tyr; p.Cys39Arg; p.Cys61Gly and p.Cys64Tyr.

Spectrum of frame-shift, missense and nonsense BRCA1 germline mutations detected in Czech patients.

BRCA1 reference sequence GenBank: U14680. Mutations marked in bold had not been submitted to the BIC database from any other population prior to November 2007 [21].

Spectrum of frame-shift and nonsense BRCA2 germline mutations detected in Czech patients.

BRCA2 reference sequence GenBank: U43746. Mutations marked in bold had not been submitted to the BIC database from any other population prior to November 2007 [21].

Splice-site alterations detected in Czech high-risk breast/ovarian cancer patients.

* Predicted consensus values by the Splice Site Prediction Program: Neural Networks [23].

All deleterious mutations identified in the BRCA2 gene were truncating mutations (30 different frame-shift mutations, 5 different nonsense mutations, and 6 different splice-site mutations (described below)) and are summarised in Tables 2 and 3.

### Splice-site alterations

For all putative splice-site variants, we performed further analyses to unequivocally determine their pathogenic effect. We have previously reported that BRCA1 c.4185G>A and BRCA2 c.9118A-2>G lead to aberrant transcripts [11]. In this publication, we provide results of the effects at the cDNA level for 7 additional putative splice-site variants that have not been described previously, i.e. two variants in the BRCA1 gene (c.302-3C>G and c.4675+1G>A) and five in the BRCA2 gene (c.475G>A; c.476-2>G; c.7007G>A; c.8755-1G>A and c.9117+2T>A).

To set aside nonsense-mediated mRNA decay (NMD), an inhibitor of the NMD pathway was used before RNA isolation from lymphocyte cultures. As we had previously found that splice-site mutations not only lead to transcripts affecting the exon following or preceding the mutation but may also affect other neighbouring exons, we chose RT-PCR primers that were not in the immediately adjacent exons.

Results of the RT-PCR experiments are shown in Figure 1. The predicted consensus values, the experimentally determined effect on mRNA, and deduced results at the protein level are summarised in Table 3. For 4 of the putative splice-site mutations analysed (BRCA1 c.4675+1G>A, BRCA2 c.476-2A>G, c.8755-1G>A and c.9117+2T>A), aberrant splicing was anticipated as these variants affect the highly conserved nucleotides at positions +1+2/-1-2 of the splice donor/acceptor sites.

RT-PCR results for the splice-site mutations identified in our patient population. Below the picture we indicate the position of the RT primers used. C = control. A detailed description of the relevant fragments obtained can be found in Table 3. However, we omitted in this Table details about several novel alternative transcripts containing intronic sequences with high acceptor/donor scores at the 5'/3' ends, complicating the interpretation of the results and of which the clinical relevance is currently unknown. For instance, cloning RT-PCR products spanning exons 21 to 26 revealed several novel alternative transcripts containing insertions from sequences of introns 24 and 25 (cfr. extra bands above the full length fragment in control persons and bands around 920–930 bp in the patient with BRCA2 c.9117+2T>A, representing transcripts lacking exon 23 but containing these intronic sequences. Similar observations were obtained for the RT-PCR analyses for BRCA2 c.8755-1G>A.

Furthermore, we validated 3 variants outside the highly conserved +1+2/-1–2 nucleotides. BRCA1 c.302-3C>G activates a cryptic splice site in intron 6, leading to the inclusion of two extra base pairs from this intron. This results in a frame-shift and a premature termination of translation.

The G to A transition in the last nucleotide of BRCA2 exon 5 (c.475G>A) results in a transcript lacking in frame exon 5, which was absent in control individuals. Unfortunately, no patient with both this mutation and an SNP in exons 2–10 of BRCA2 was available to allow investigation of whether the mutant allele was still able to transcribe a full- length product. This mutation was identified in two unrelated families that were too small to evaluate segregation.

RT-PCR with a forward primer located in exon 11 and a reverse primer in exon 15 revealed that the BRCA2 c.7007G>A mutation, in the last nucleotide of exon 13, is not a missense mutation (p.Arg2336His). We found a transcript lacking exon 13, a transcript lacking exons 12 and 13, and a transcript lacking exon 12. The transcript lacking exon 12 is present at about the same level in blood lymphocytes of both the patient with the mutation and a control person (data of the cloning experiments not shown). The BRCA2 c.7007G>A mutation was found in 2 unrelated families: in an affected female with a severe family history of breast cancer, fulfilling criterion I, and in an affected male with a maternal aunt and cousin diagnosed with breast cancer.

### Mutation detection rates

BRCA1 mutations were found to be about 2 times more frequent than BRCA2 mutations in our study population. Mutation detection frequencies (deleterious and splice-site mutations and large intragenic rearrangements) in relation to the inclusion criteria are summarised in Table 4. Surprisingly, the highest mutation frequency (14 out of 19 cases, or 73.7%) was obtained in apparently sporadic patients diagnosed with both breast and ovarian cancers. By re-evaluation of the pedigrees, we found that the apparent lack of a family history might have been due to a small number of relatives and/or paternal transmission in these families. As expected, a higher mutation detection rate was obtained in breast and ovarian (HBOC) cancer families fulfilling criteria I and II (96/160, or 60%) as compared to breast cancer only (HBC) families fulfilling the same criteria (114/412, or 27.7%).

Summary of deleterious mutations in BRCA genes identified in relation to inclusion criteria and phenotype.

Abbreviations: inclusion criteria – see Methods section; mutation – only deleterious mutations summarised in Table 1 and Table 2 are considered; CI – Confidence Interval; HOC – hereditary ovarian cancer syndrome; HBOC – hereditary breast and ovarian cancer syndrome; HBC – hereditary breast cancer only syndrome.

Six out of 16 male breast cancer patients (37.5%) carried a deleterious mutation. Unexpectedly, the same number of mutations was identified in the BRCA2 gene (c.7913_7917del5; c.9403delC, and the splice-site mutation c.7007G>A) as in the BRCA1 gene (2 males carrying c.5266dupC and one patient with c.3700_3704del5). The average age of male breast cancer diagnosis in mutation carriers was 72.5 years (ranging from 58 to 80 years). Four of the mutation carriers reported a family history of breast or prostate cancer but did not meet inclusion criteria II. The average age at diagnosis of breast cancer of the 10 males without a BRCA mutation was 57.2 years, ranging from 37 to 80 years. In addition, we identified rare variants of unknown significance in four male patients: 2 in BRCA1 (c.5277+48_5277+59dup12; c. 5277+78G>A) and 2 in the BRCA2 gene (p.Ala75Pro; c.10095delCins11).

A deleterious mutation was detected in 20 out of 77 healthy, unaffected probands (26%; 15 in BRCA1 and 5 in the BRCA2 gene) with a severe family history of breast and/or ovarian cancer but no living affected relative. Furthermore, we identified a deleterious mutation in 20 out of 150 affected probands (13.3%; 12 in BRCA1 and 8 in the BRCA2 gene) who did not fully meet our inclusion criteria for testing but who were referred by genetic counsellors.

### Discussion

Molecular genetic analysis for the BRCA1 and BRCA2 genes has been available at the MMCI since 1999. By the end of 2006, an unequivocal deleterious mutation had been identified in 294 high-risk Czech probands out of 1,010 probands selected for genetic testing (29.1%). Similar inclusion criteria are used in other centres [17].

In familial cases, we obtained a mutation detection rate of 36.7% (210/572) in sporadic bilateral breast, bilateral ovarian or breast, and ovarian cancer patients 43.6% (24/55) but only 10.7% (13/121) in early onset unilateral breast cancer patients. However, the latter is slightly higher than generally reported in Europe (< 5–9%) in early onset sporadic cases [18,17]. BRCA1 mutations were twice as frequent as BRCA2 mutations in our study population. The presence of an ovarian cancer case turned out to be a good indicator for the presence of a causative BRCA mutation: the highest mutation detection rates were obtained in apparently sporadic patients diagnosed with both breast and ovarian cancer (73,7%) and in breast-ovarian cancer families (about 60%). This detection rate seems to be rather high when compared with ones reported for example in Germany (43%) or Belgium using similar inclusion criteria [18,17]. Detection rate highly depends on severity of family history and this might be result of accumulating of very severe affected families seeking genetic advice in our population.

The majority of the BRCA1 and 2 deleterious mutations lead to a premature termination of translation. As the protein truncation test was used for screening BRCA1 exon 11 and BRCA2 exons 10 and 11, missense mutations were not detectable in these exons. To date, however, no undoubtedly deleterious missense mutations have been reported for these exons. Truncating mutations may be assumed to cause disease, but the pathogenic effect of missense and intronic variants is more equivocal. Only the p.Cys24Tyr; p.Cys39Arg; p.Cys61Gly and p.Cys64Tyr missense mutations in the BRCA1 gene are classified as deleterious mutations. These substitutions take place in the 100% conserved cysteine residues of the BRCA1 C3HC4 RING domain. All these missense mutations in the Zn2+-binding residues of the C3HC4 RING domain have previously been experimentally shown to abolish dimerization of BRCA1 with the BARD1 protein, resulting in loss of ubiquitin protein ligase activity and causing γ-radiation hypersensitivity [19,20]. The p.Cys61Gly is one of the most frequent founder BRCA1 mutations found in the Czech Republic, as well as in many other European countries [21].

Five mutations (BRCA1: p.Cys61Gly, c.5266dupC, c.3700_3704del5, and BRCA2: c.7913_7917del5 and c.8537_8538delAG) represent 52% of all mutations detected in our study population. Although there was a diversity of mutations in both genes, there are strong Slavic founder effects, particularly for two BRCA1 mutations (p.Cys61Gly, and c.5266dupC) found as founder mutations in Poland [21,22]. The BRCA1 c.3700_3704del5 mutation is a frequent mutation in Germany [17,21]. Three BRCA1 Czech founder mutations (p.Cys61Gly, c.3700_3704del5, and c.5266dupC) accounted for 9.8%, 14.2% and 36.8% of the BRCA1 mutations identified, respectively. A Czech founder effect is evident for two BRCA2 mutations (c.7913_7917del5 and c.8537_8538del2) that are reported elsewhere only from Canada and the USA [21]. These BRCA2 mutations accounted for 15.6% and 16.7% of the BRCA2 mutations identified, respectively.

A broad spectrum of other mutations demonstrates the genetic diversity of the Czech population and reflects the spectrum of mutations found in other European countries [21]. The Czech population is mainly of Slavic origin. Due to its location in Central Europe, however, and historical influences during past centuries, when the Czech region was a part of the Austro-Hungarian Empire, an influx of other nations is evident.

All putative splice-site mutations were tested computationally for their ability to change the donor/acceptor score in silico, as well as to elucidate the creation or activation of cryptic splice sites. For all except one (BRCA1 c.4675+1G>A), the novel splice sites were correctly identified by the Neural Networks prediction program, thereby demonstrating its usefulness [23].

For several variants (BRCA1: c.302-3C>G; c.4675+1G>A and BRCA2 c.475G>A, c.476-2>G; c.7007G>A; c.8755-1G>A; c.9117+2T>A), reliable RT-PCR data were not available from the literature or the BIC database. For genetic counselling purposes, it is extremely important to differentiate between deleterious and polymorphic splice-site alterations. Analysis at the mRNA level allowed us to identify alterations leading to aberrant splicing. Especially for genetic alterations outside the almost 100% conserved +1/+2 and -1/-2 nucleotides, RT-PCR analyses can contribute to a correct and definitive clinical interpretation. We clearly demonstrate here that BRCA1 c.302-3C>G and BRCA2 c.475G>A, c.7007G>A lead to aberrant splicing.

In general, splice-site alterations lead to skipping of the complete exon following the acceptor or preceding the donor site in which the mutation is located. This was indeed what we found for BRCA1 c.4185G>A and BRCA2 c.475G>A; c.9117+2T>A and c.8755-1G>A. Other mutations result in the creation of a novel splice site or activation of a cryptic splice site, such as was found for BRCA1 c.302-3C>G and BRCA2 c.9118-2A>G.

Mutations in splice sites of some exons lead to multiple exon skipping. For example, mutations in the splice sites of BRCA2 exon 6 seem to also affect exon 5, as observed from RT-PCR analyses for c.476-2A>G and the previously reported c.516+1G>A mutation [[11], Figure 1]. A further example is the BRCA1 c.4675+1G>A mutation in intron 15. mRNA analysis revealed several different transcripts, including some with a skip of exon 14. Multiple exon skipping has previously been shown to be associated with mutations in 5' as well as 3' splice sites in several genes [24]. By the identification and further functional analyses of such splice mutations, new regulatory elements and mechanisms involved in splicing may be revealed.

The BRCA2 c.7007G>A mutation, in the last nucleotide of exon 13, was previously studied by RT-PCR with primers located in exons 12 and 14, revealing only a transcript lacking exon 13 [25]. As we chose RT-PCR primers that were not in the immediately adjacent exons, we discovered a more complex splicing pattern, i.e. a transcript lacking exon 13, a transcript lacking exons 12 and 13, and a transcript lacking exon 12. The latter transcript was present at the same level in control persons. An exon 12 alternatively spliced BRCA2 transcript has previously been described as a major alternative BRCA2 transcript with an expression level of about 10% in normal breast tissue relative to the wild type BRCA2 transcript. The expression level of the delta12-BRCA2 transcript has been shown to be higher in some tumour tissues compared with their normal breast tissues [26].

In addition, our cDNA analyses revealed several novel alternative transcripts containing intronic sequences with high acceptor/donor scores at the 5'/3' ends. Further investigations are required to determine the prevalence of these transcripts in different tissues and to define their possible role. A limitation of our studies is that they were performed on transcriptional products of peripheral lymphocytes, rather than on the breast or ovarian tissues that are considered to be the sites of initial abnormalities in BRCA1/2-related tumors. Further studies are needed to assess whether different quantitative and/or qualitative effects of the alterations investigated are observed in these tissues.

Numerous BRCA1 and BRCA2 sequence alterations identified during screening are kept in the laboratory database. Nonsynonymous single nucleotide polymorphisms still may have the potential to represent modifiers of inherited susceptibility. As deleterious germline mutations in BRCA1 and BRCA2 confer a greatly increased risk of breast cancer, some sequence variants may be only moderate or low penetrant alleles.

### Conclusion

We detected 204 families harbouring deleterious BRCA1 mutations (44 different), and 90 families harbouring BRCA2 mutations (41 different). BRCA1 mutations were found to be about 2 times more frequent than BRCA2 mutations in high-risk members of the Czech population, based on our study population. Mutation detection frequencies (deleterious and splice-site mutations and large intragenic rearrangements) in relation to the inclusion criteria were highest in the group of patients diagnosed with both breast and ovarian cancers (73.7%) and in familial cases with clustering of breast and ovarian cancers in first and second degree relatives (60%).

Five founder mutations, three BRCA1 (c.5266dupC; c.3700_3704del5; p.Cys61Gly) and two BRCA2 mutations (c.7913_7917del5; c.8537_8538del2), account for 52% of all detected mutations. These mutations could be used as a first-step screen. However, the screening of the whole region of both genes is highly indicated for high-risk Czech breast/ovarian cancer patients.

We evaluated 9 putative splice-site variants at the cDNA level. Three splice-site variants in the BRCA1 gene (c.302-3C>G; 4185G>A and c.4675+1G>A) and six splice-site variants in the BRCA2 gene (c.475G>A; c.476-2>G; c.7007G>A; c.8755-1G>A; c.9117+2T>A and c.9118-2A>G) were demonstrated to result in aberrant transcripts and are considered as deleterious mutations. As deleterious germline mutations in BRCA1 and BRCA2 confer a greatly increased risk of breast cancer, some sequence variants in both genes might be candidates for moderate or low penetrance alleles. Each sequence variant detected in high-risk probands, therefore, has to be evaluated through a complex approach to establish the clinical consequence. A result obtained through genetic testing for hereditary breast and ovarian cancer has great impact on the medical management of a patient and accurate interpretation is, therefore, imperative.

### Abbreviations

CI – Confidence Interval; HA – Heteroduplex Analysis; HBC – Hereditary Breast Cancer; HBOC – Hereditary Breast and Ovarian Cancer; HOC – Hereditary Ovarian Cancer; MMCI – Masaryk Memorial Cancer Institute; NLS – Nuclear Localization Signal; NMD – Nonsense-mediated mRNA Decay; PCR – Polymerase Chain Reaction; PTT – Protein Truncation Test.

### Competing interests

The authors declare that they have no competing interests.

### Authors' contributions

EM, ML and LF participated equally in the design of this study. ML, EM and PV carried out the molecular analysis and interpreted the results. LF and MN participated in genetic counselling and the selection of participating patients and their family members. KC and IC provided the mRNA analysis of all putative splice-site variants and their interpretation. HP, VK and JK participated in mutation screening. EM drafted the paper and finalised the manuscript, with the help of KC. All authors read and approved the final manuscript.

### Pre-publication history

The pre-publication history for this paper can be accessed here:



# SUPPLEMENTAL FILE 1: 1471-2407-8-140.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)