# MAIN TEXT

## Efficiency of BRCAPRO and Myriad II mutation probability thresholds versus cancer history criteria alone for BRCA1/2 mutation detection

### Abstract

Considerable differences exist amongst countries in the mutation probability methods and thresholds used to select patients for BRCA1/2 genetic screening. In order to assess the added value of mutation probability methods, we have retrospectively calculated the BRCAPRO and Myriad II probabilities in 306 probands who had previously been selected for DNA-analysis according to criteria based on familial history of cancer. DNA-analysis identified 52 mutations (16.9%) and 11 unclassified variants (UVs, 3.6%). Compared to cancer history, a threshold ≥10% with BRCAPRO or with Myriad II excluded about 40% of the patients from analysis, including four with a mutation and probabilities <10% with both programs. All four probands had a BRCA2 mutation. BRCAPRO and Myriad II showed similar specificity at 10% threshold, overall BRCAPRO was more sensitive than Myriad II for the detection of mutations. Only two of the probands with an UV had probabilities >20% with BRCAPRO and Myriad II. In summary, BRCAPRO and Myriad II are more efficient than cancer history alone to exclude patients without a mutation. BRCAPRO performs better for the detection of BRCA1 mutations than of BRCA2 mutations. The Myriad II scores provided no additional information than the BRCAPRO scores alone for the detection of patients with a mutation. The use of thresholds excluded from analysis the majority of patients carrying an UV.

### Introduction

The two major susceptibility genes for breast cancer (BC) and ovarian cancer (OC), BRCA1 (MIM: 113705) and BRCA2 (MIM: 600185) were discovered in 1994 and 1995, respectively [1, 2]. Initially, the Protein Truncation Test (PTT) was the screening method widely used as a quick and inexpensive test to detect pathogenic truncating mutations in these genes. At a later stage, the screening was optimised using denaturing high performance liquid chromatography (DHPLC) [3], direct sequencing and Multiplex Ligation-dependent Probe Amplification (MLPA) [4], in which all types of mutations or deletions could be detected. A drawback of this optimised screening methodology is that, as compared to PTT, it is laborious and more expensive. Furthermore, inherited mutations in these two genes only account for a small fraction of the familial clustering of BC and OC in the total group of index patients [5–9]. Therefore, accurate selection criteria for patients eligible for DNA-analysis are mandatory.

Various risk prediction algorithms and models have been developed to identify putative BRCA1/2 mutation carriers. The first published model was that of the University of Pennsylvania (U-Penn), also called the Couch model [10]; followed by Shattuck-Eidens model or Myriad I [11]; Myriad II, an extension of the previous model [12]; the Myriad tables by Frank et al. [13]; the BRCAPRO model [14]; the Manchester scoring system of Evans et al. [15] and the recent BOADICEA model of Antoniou et al. [16].

In addition to the different types of models, also the thresholds used in these models to establish the indication to perform DNA-analysis, vary substantially among countries. In 1996, the American Society of Clinical Oncology recommended that consideration of BRCA1/2 testing should be offered to patients with strong familial features, such as BC in the family or very early age at onset of the disease, c.q. corresponding to those patients whose prior probability of carrying a mutation exceeds 10% [17]. However, the updated American guidelines in 2003 did not recommend a numerical threshold [18]. Currently, the majority of Dutch Cancer Genetics Services apply a 10% pre-test probability as threshold to perform DNA analysis of the BRCA1 and BRCA2 genes [19], whereas in the UK most centres offer mutation analysis to families with a 20% or greater pre-test probability of carrying a mutation [20].

The goal of our study was to retrospectively analyze the added value of predictive mutation probability scores and optimal thresholds in terms of sensitivity and specificity of two currently used mutation probability models BRCAPRO and Myriad II, in a group of patients prior selected on the basis of family history of cancer.

### Materials and methods

Our study includes 306 probands who had been selected for DNA-analysis based on established criteria about personal and family cancer history (shown in Table 1).Table 1Indications for DNA-analysis based on cancer history (classified by number of affected relatives)Number of affected relativesIndication for performing DNA-analysis1aBC <35 yearsbBilateral BC; first time < 50 yearscMale BCdOC <50 years2aTwo-first degree relatives with BC/OC, one case <50 yearsbTwo-first degree relatives with OCcTwo-second degree relatives with BC/OC, both <50 years≥3Three or more affected relatives in the family
BC Breast cancer, OC Ovarian cancer

Indications for DNA-analysis based on cancer history (classified by number of affected relatives)

BC Breast cancer, OC Ovarian cancer

In the period 1996–2000 partial mutation screening of the BRCA1 and BRCA2 genes was performed using the protein truncation test (PTT). The PTT was used only for exon 11 of BRCA1 and exons 10 and 11 of BRCA2. From 2000 on, complete genetic screening was performed using DHPLC [3], direct sequencing and for BRCA1 also MLPA [4].

Clinical data of all the probands and their first and second degree relatives were retrieved from their medical records, which included cancer site: BC and/or OC, bilateral BC, BC in male relatives, age at diagnosis and number of affected relatives (data not shown). All probands were affected with breast cancer and/or ovarian cancer. There were no probands with known Jewish ancestry.

BRCAPRO [14] is a Mendelian model that incorporates mutated allele frequencies and cancer specific penetrances, in addition to the following clinical data about the probands and first and second degree relatives: number of women affected with BC; OC only; discrimination between paternal and maternal inheritance pattern; BC under age 50 and OC; bilateral BC; relative with both OC and BC; affected and unaffected individuals; Ashkenazi Jewish ancestry and male BC.

The Myriad II prevalence tables [12] are based on proband and family history accompanying results of BRCA1/2 deleterious variant samples tested by the Myriad company. The calculation takes into account if there are relatives who have been diagnosed with OC at any age, if relatives have been diagnosed with BC, grouped by those over and under age 50 years at diagnosis of BC, and the presence of male BC. It does not take into account bilateral BC and specific age at diagnosis. In addition, it only allows inclusion of a maximum of three relatives, including the patient; and it does not calculate BRCA1 and BRCA2 mutation probabilities separately like BRCAPRO. A more detailed description of both models has been published [12–14].

Carrier probabilities with BRCAPRO and Myriad II were calculated with the CancerGene software package from the University of Texas Southwestern Medical Center (CaGene version 4.0). Additional information about CancerGene is available at http://www.utsouthwestern.edu.

We calculated the carrier probabilities using the BRCAPRO and Myriad II models and compared the clinical characteristics between the group of probands with and those without a pathogenic mutation. Carriers of UVs were reported separately and not included in either the mutation positive or mutation negative groups for the purpose of the analysis of clinical features. Genetic alterations reported as polymorphisms were scored as mutation negatives. All data were analyzed using SPSS statistical software version 15.0 for Windows. The sensitivity, specificity, positive and negative predictive values, and positive and negative likelihood ratios were calculated for both BRCAPRO and Myriad II, and each of them for 10 and 20% thresholds.

Receiver operating characteristic (ROC) curves were drawn and the areas under the curve (AUC) were calculated in the group of 295 probands with either a mutation or a wild-type genotype. Calculations were made for BRCAPRO and for Myriad II and within each of them at 10 and 20% thresholds.

### Patients and DNA-analysis

Our study includes 306 probands who had been selected for DNA-analysis based on established criteria about personal and family cancer history (shown in Table 1).Table 1Indications for DNA-analysis based on cancer history (classified by number of affected relatives)Number of affected relativesIndication for performing DNA-analysis1aBC <35 yearsbBilateral BC; first time < 50 yearscMale BCdOC <50 years2aTwo-first degree relatives with BC/OC, one case <50 yearsbTwo-first degree relatives with OCcTwo-second degree relatives with BC/OC, both <50 years≥3Three or more affected relatives in the family
BC Breast cancer, OC Ovarian cancer

Indications for DNA-analysis based on cancer history (classified by number of affected relatives)

BC Breast cancer, OC Ovarian cancer

In the period 1996–2000 partial mutation screening of the BRCA1 and BRCA2 genes was performed using the protein truncation test (PTT). The PTT was used only for exon 11 of BRCA1 and exons 10 and 11 of BRCA2. From 2000 on, complete genetic screening was performed using DHPLC [3], direct sequencing and for BRCA1 also MLPA [4].

Clinical data of all the probands and their first and second degree relatives were retrieved from their medical records, which included cancer site: BC and/or OC, bilateral BC, BC in male relatives, age at diagnosis and number of affected relatives (data not shown). All probands were affected with breast cancer and/or ovarian cancer. There were no probands with known Jewish ancestry.

### Mutation probability methods

BRCAPRO [14] is a Mendelian model that incorporates mutated allele frequencies and cancer specific penetrances, in addition to the following clinical data about the probands and first and second degree relatives: number of women affected with BC; OC only; discrimination between paternal and maternal inheritance pattern; BC under age 50 and OC; bilateral BC; relative with both OC and BC; affected and unaffected individuals; Ashkenazi Jewish ancestry and male BC.

The Myriad II prevalence tables [12] are based on proband and family history accompanying results of BRCA1/2 deleterious variant samples tested by the Myriad company. The calculation takes into account if there are relatives who have been diagnosed with OC at any age, if relatives have been diagnosed with BC, grouped by those over and under age 50 years at diagnosis of BC, and the presence of male BC. It does not take into account bilateral BC and specific age at diagnosis. In addition, it only allows inclusion of a maximum of three relatives, including the patient; and it does not calculate BRCA1 and BRCA2 mutation probabilities separately like BRCAPRO. A more detailed description of both models has been published [12–14].

Carrier probabilities with BRCAPRO and Myriad II were calculated with the CancerGene software package from the University of Texas Southwestern Medical Center (CaGene version 4.0). Additional information about CancerGene is available at http://www.utsouthwestern.edu.

### Statistical analysis

We calculated the carrier probabilities using the BRCAPRO and Myriad II models and compared the clinical characteristics between the group of probands with and those without a pathogenic mutation. Carriers of UVs were reported separately and not included in either the mutation positive or mutation negative groups for the purpose of the analysis of clinical features. Genetic alterations reported as polymorphisms were scored as mutation negatives. All data were analyzed using SPSS statistical software version 15.0 for Windows. The sensitivity, specificity, positive and negative predictive values, and positive and negative likelihood ratios were calculated for both BRCAPRO and Myriad II, and each of them for 10 and 20% thresholds.

Receiver operating characteristic (ROC) curves were drawn and the areas under the curve (AUC) were calculated in the group of 295 probands with either a mutation or a wild-type genotype. Calculations were made for BRCAPRO and for Myriad II and within each of them at 10 and 20% thresholds.

### Results

Our study included a total of 306 probands who were eligible for BRCA1 and BRCA2 mutation detection, based on established personal and family cancer history criteria, as shown in Table 1. With the PTT we found 20 pathogenic mutations (6.8%). In the remaining 286 probands, complete BRCA1 and BRCA2 gene sequencing showed 43 additional genetic variations (14%). Thirty-two of them (10.4%) could be classified as a pathogenic mutation (BRCA1
n = 13, BRCA2 n = 19). The remaining 11 variants (3.6%) were classified as unclassified variants (UVs, BRCA1
n = 7, BRCA2
n = 4). Figure 1 summarizes the chronological sequence of diagnostic methods and selection criteria applied in the studied population. An individual description of the genetic variants identified is presented in Table 2. Instructions for the nomenclature for the sequence variations can be seen at: http://www.genomic.unimelb.edu.au/mdi/mutnomen/recs.html.Fig. 1Chronological sequence of events in the study population
Table 2Genetic variants identified
BRCA1

BRCA2
ExonNucleotide changeNumber of familiesExonNucleotide changeNumber of families
Pathogenic mutations found with PTT
11c.2193_2197del5511c.5213_5216del4111c.2722G>T411c.5645C>G111c.1291_1292insT111c.5682C>G111c.2646_2648delTGC111c.6275_6576delTT111c.2685_2686delAA111c.6396_6397insA111c.2989_2990insAA111c.6486_6489del4111c.6643delA1Total613Total77
Pathogenic mutations found with DHPLC, MLPA, and sequencing
1del ex1a/b-213c.115delG12c.68_69delAG25c.469_470delAA25c.191G>A16c.516 + 1G>T18c.514C>T18c.658_659delGT217c.4987- ?_2074 + ?del110c.1773-1776del4118c.5080G>T110c.1813insA120c.5277 + 1G>A411c.4314delC122c.5346G>A115c.7436_7437-2delAGAT124c.5503C>T116c.7734_7740del6ins9218c.8058_8059delT319c.8374_8384del11ins3123c.9063_9078del16123c.9117G>A2Total913Total1319
Unclassified variants
11c.692C>T110c.1385A>G111c.1258G>T115c.7469T>C111c.1486C>T118c.8111C>T117c.4987-3C>G121c.8723T>G119(I)c.5194-25A>C120c.5216A>G123c.5435C>G1Total77Total44

Chronological sequence of events in the study population

Genetic variants identified

Classification of the genetic variants (mutations or UVs) was done according to the Breast Cancer Information Core (BIC) database: [http://research.nhgri.nih.gov/bic/].

The mean carrier probabilities with BRCAPRO and Myriad II for the groups with a mutation and without a mutation are shown in Table 3. The probability scores are also shown for the subgroup of patients with mutations detected with PTT and for those with mutations detected by complete analysis. The carrier probabilities of the probands with a mutation (n = 53) were significantly higher than those without a mutation, in both models (BRCAPRO: P < 0.0001, Myriad II: P < 0.0001). Apart from BRCAPRO and Myriad II, the occurrence of OC in the family (P = 0.001) was the only clinical parameter which was significantly associated with the presence of a BRCA1/2 mutation (data not shown).Table 3BRCAPRO and Myriad II scores of BRCA mutation positive versus negative populationsMutations with PTT
N = 21Mutations with complete analysis
N = 32Total mutations
N = 53No mutation
N = 243
P values (*)BRCAPROM
±SDRange0.699(± 0.248)0.186–0.9970.586(± 0.339)0.012–1.000.629(± 0.310)0.012–1.000.240(± 0.273)0.001–0.994<0.0001Myriad IIM±SDRange0.293(± 0.124)0.122–0.4650.202(± 0.126)0.019–0.4210.237(± 0.131)0.019–0.4650.119(± 0.102)0.007–0.603<0.0001Results expressed as: M ± SD mean (± standard deviation)(*) P values: total mutations versus no mutations: t test

BRCAPRO and Myriad II scores of BRCA mutation positive versus negative populations

M

±SD

Range

0.699

(± 0.248)

0.186–0.997

0.586

(± 0.339)

0.012–1.00

0.629

(± 0.310)

0.012–1.00

0.240

(± 0.273)

0.001–0.994

M

±SD

Range

0.293

(± 0.124)

0.122–0.465

0.202

(± 0.126)

0.019–0.421

0.237

(± 0.131)

0.019–0.465

0.119

(± 0.102)

0.007–0.603

Results expressed as: M ± SD mean (± standard deviation)

(*) P values: total mutations versus no mutations: t test

The performance of the models, at 10 and 20% thresholds, is presented in Table 4 and Fig. 2 shows the corresponding ROC curves. Using 10% threshold with BRCAPRO 110 probands had a probability <10% of whom four had a mutation. With Myriad, nine out of 126 probands with a probability, <10% had a mutation. The four probands missed with BRCAPRO were also missed with Myriad. These four probands carried a BRCA2 mutation, none of them had a family history of male BC, OC or bilateral BC. In one case the indication for DNA analysis was three BC cases in the family (indication 3, Table 1). The other three cases have a family history of two BC cases with one case below age 50 (indication 2a, Table 1).Table 4Performance parameters of BRCAPRO and Myriad IIThreshold 10%Threshold 20%BRCAPROMyriad IIBRCAPROMyriad IIMutations/probands below threshold (*)4/1109/1267/16018/249Sensitivity (95% CI)0.875 (0.760–0.990)0.719 (0.563–0.875)0.781 (0.638–0.925)0.438 (0.266–0.609)Specificity (95% CI)0.433 (0.372–0.494)0.476 (0.415–0.538)0.602 (0.542–0.663)0.909 (0.874–0.945)Positive predictive value (95% CI)0.163 (0.108–0.218)0.147 (0.092–0.203)0.198 (0.129–0.268)0.378 (0.222–0.535)Negative predictive value (95% CI)0.965 (0.931–0.999)0.931 (0.887–0.974)0.956 (0.925–0.988)0.928 (0.896–0.960)(*) Number of probands with a mutation/total number of probands below the thresholds (i.e., <10% or <20%)
Fig. 2ROC curves for BRCAPRO and Myriad II, at 10 and 20% thresholds

Performance parameters of BRCAPRO and Myriad II

(*) Number of probands with a mutation/total number of probands below the thresholds (i.e., <10% or <20%)

ROC curves for BRCAPRO and Myriad II, at 10 and 20% thresholds

Results using the 20% threshold, 7 out of 160 probands with BRCAPRO and 18 out of 249 probands with Myriad II had a mutation with a probability below the threshold. At both thresholds BRCAPRO is more sensitive than Myriad II, at 10% they both have similar specificity while at 20% is Myriad II more specific (Table 4). As shown by the four AUC calculations from Fig. 2: (a) BRCAPRO performed better than Myriad II at both thresholds and (b) the AUC for the 20% thresholds were better than 10% thresholds.

The group of 11 UVs had median mutation probabilities with BRCAPRO of 13.6% (range 0.3–97.3%) and with Myriad II of 17.4% (range 7.3–37.6%). When compared with the group with a mutation, the UVs had lower probabilities than the group with a mutation: with BRCAPRO were statistically significant (P = 0.004), with Myriad II did not achieve statistical significance (P = 0.093) (data not shown).

As shown in Table 5, with Myriad II nine UVs and with BRCAPRO seven of the 11 UVs had mutation probabilities below 20%. The UVs: c.4987-3C>G (putative splice variant), and c.5216A>G in BRCA1 were the only two UVs with probability values >20% both with BRCAPRO and Myriad II.Table 5Biochemical and epidemiological features of the unclassified variants with corresponding BRCAPRO and Myriad II scoresExonNucleotide changeEffect on proteinNumber of familiesBRCAPRO (%)Myriad II (%)Polarity changeConserved mammals/otherTimes reported (BIC)Co-segregation (literature)Co-occurrence (literature)Classification (literature)References
BRCA1
11c.692C>Tp.T231 M159.917.8YN/N2NDNUnknown[31]11c.1258G>Tp.D420Y16.510.5YN/Y3NDNUnknown[31]11c.1486C>Tp.R496C10.33.4YN/N43NDYNeutral[31, 32]17c.4987-3C>GNA96.937.6––0NDND20c.5194-25A>CNA114.017.4––0NDND20c.5216A>Gp.D1739G197.332.5YY/Y2NDNDCancer-associated[30]23c.5435C>Gp.P1812R14.67.5YY/Y0NDNDTotal: 77
BRCA2
10c.1385A>Gp.E462G137.817.8YY/N35NYNeutral[29, 33, 34]15c.7469T>Cp.I2490T113.619.9YN/N238NDND18c.8111C>Tp.S2704F111.57.3YN/N1NDND21c.8723T>Gp.V2908G11.29.8NY/N8NNNeutral[34]Total: 44Alignments were based on following species and NCBI reference sequences: BRCA1 Human (NP_009225), Chimpanzee (NP_001038958), Gorilla (AAT44835), Orang (AAT44834), Macaque (NP_001108421), Dog (NP_001013434), Rat (NP_036646), Mouse (NP_033894), Cow (NP_848668), Opossum (NP_001029141), Chicken (NP_989500), Frog (AAI70141); BRCA2 Human (CAA64484), Chimp (XP_509619), Dog (BAB91245), Cat (NP_001009858), Mouse (NP_033895), Opossum (ABP48762), Chicken (NP_989607). Y Yes, N No, ND Not done, NA Non Applicable

Biochemical and epidemiological features of the unclassified variants with corresponding BRCAPRO and Myriad II scores

Alignments were based on following species and NCBI reference sequences: BRCA1 Human (NP_009225), Chimpanzee (NP_001038958), Gorilla (AAT44835), Orang (AAT44834), Macaque (NP_001108421), Dog (NP_001013434), Rat (NP_036646), Mouse (NP_033894), Cow (NP_848668), Opossum (NP_001029141), Chicken (NP_989500), Frog (AAI70141); BRCA2 Human (CAA64484), Chimp (XP_509619), Dog (BAB91245), Cat (NP_001009858), Mouse (NP_033895), Opossum (ABP48762), Chicken (NP_989607). Y Yes, N No, ND Not done, NA Non Applicable

### DNA-analysis

Our study included a total of 306 probands who were eligible for BRCA1 and BRCA2 mutation detection, based on established personal and family cancer history criteria, as shown in Table 1. With the PTT we found 20 pathogenic mutations (6.8%). In the remaining 286 probands, complete BRCA1 and BRCA2 gene sequencing showed 43 additional genetic variations (14%). Thirty-two of them (10.4%) could be classified as a pathogenic mutation (BRCA1
n = 13, BRCA2 n = 19). The remaining 11 variants (3.6%) were classified as unclassified variants (UVs, BRCA1
n = 7, BRCA2
n = 4). Figure 1 summarizes the chronological sequence of diagnostic methods and selection criteria applied in the studied population. An individual description of the genetic variants identified is presented in Table 2. Instructions for the nomenclature for the sequence variations can be seen at: http://www.genomic.unimelb.edu.au/mdi/mutnomen/recs.html.Fig. 1Chronological sequence of events in the study population
Table 2Genetic variants identified
BRCA1

BRCA2
ExonNucleotide changeNumber of familiesExonNucleotide changeNumber of families
Pathogenic mutations found with PTT
11c.2193_2197del5511c.5213_5216del4111c.2722G>T411c.5645C>G111c.1291_1292insT111c.5682C>G111c.2646_2648delTGC111c.6275_6576delTT111c.2685_2686delAA111c.6396_6397insA111c.2989_2990insAA111c.6486_6489del4111c.6643delA1Total613Total77
Pathogenic mutations found with DHPLC, MLPA, and sequencing
1del ex1a/b-213c.115delG12c.68_69delAG25c.469_470delAA25c.191G>A16c.516 + 1G>T18c.514C>T18c.658_659delGT217c.4987- ?_2074 + ?del110c.1773-1776del4118c.5080G>T110c.1813insA120c.5277 + 1G>A411c.4314delC122c.5346G>A115c.7436_7437-2delAGAT124c.5503C>T116c.7734_7740del6ins9218c.8058_8059delT319c.8374_8384del11ins3123c.9063_9078del16123c.9117G>A2Total913Total1319
Unclassified variants
11c.692C>T110c.1385A>G111c.1258G>T115c.7469T>C111c.1486C>T118c.8111C>T117c.4987-3C>G121c.8723T>G119(I)c.5194-25A>C120c.5216A>G123c.5435C>G1Total77Total44

Chronological sequence of events in the study population

Genetic variants identified

Classification of the genetic variants (mutations or UVs) was done according to the Breast Cancer Information Core (BIC) database: [http://research.nhgri.nih.gov/bic/].

### Mutation probabilities

The mean carrier probabilities with BRCAPRO and Myriad II for the groups with a mutation and without a mutation are shown in Table 3. The probability scores are also shown for the subgroup of patients with mutations detected with PTT and for those with mutations detected by complete analysis. The carrier probabilities of the probands with a mutation (n = 53) were significantly higher than those without a mutation, in both models (BRCAPRO: P < 0.0001, Myriad II: P < 0.0001). Apart from BRCAPRO and Myriad II, the occurrence of OC in the family (P = 0.001) was the only clinical parameter which was significantly associated with the presence of a BRCA1/2 mutation (data not shown).Table 3BRCAPRO and Myriad II scores of BRCA mutation positive versus negative populationsMutations with PTT
N = 21Mutations with complete analysis
N = 32Total mutations
N = 53No mutation
N = 243
P values (*)BRCAPROM
±SDRange0.699(± 0.248)0.186–0.9970.586(± 0.339)0.012–1.000.629(± 0.310)0.012–1.000.240(± 0.273)0.001–0.994<0.0001Myriad IIM±SDRange0.293(± 0.124)0.122–0.4650.202(± 0.126)0.019–0.4210.237(± 0.131)0.019–0.4650.119(± 0.102)0.007–0.603<0.0001Results expressed as: M ± SD mean (± standard deviation)(*) P values: total mutations versus no mutations: t test

BRCAPRO and Myriad II scores of BRCA mutation positive versus negative populations

M

±SD

Range

0.699

(± 0.248)

0.186–0.997

0.586

(± 0.339)

0.012–1.00

0.629

(± 0.310)

0.012–1.00

0.240

(± 0.273)

0.001–0.994

M

±SD

Range

0.293

(± 0.124)

0.122–0.465

0.202

(± 0.126)

0.019–0.421

0.237

(± 0.131)

0.019–0.465

0.119

(± 0.102)

0.007–0.603

Results expressed as: M ± SD mean (± standard deviation)

(*) P values: total mutations versus no mutations: t test

The performance of the models, at 10 and 20% thresholds, is presented in Table 4 and Fig. 2 shows the corresponding ROC curves. Using 10% threshold with BRCAPRO 110 probands had a probability <10% of whom four had a mutation. With Myriad, nine out of 126 probands with a probability, <10% had a mutation. The four probands missed with BRCAPRO were also missed with Myriad. These four probands carried a BRCA2 mutation, none of them had a family history of male BC, OC or bilateral BC. In one case the indication for DNA analysis was three BC cases in the family (indication 3, Table 1). The other three cases have a family history of two BC cases with one case below age 50 (indication 2a, Table 1).Table 4Performance parameters of BRCAPRO and Myriad IIThreshold 10%Threshold 20%BRCAPROMyriad IIBRCAPROMyriad IIMutations/probands below threshold (*)4/1109/1267/16018/249Sensitivity (95% CI)0.875 (0.760–0.990)0.719 (0.563–0.875)0.781 (0.638–0.925)0.438 (0.266–0.609)Specificity (95% CI)0.433 (0.372–0.494)0.476 (0.415–0.538)0.602 (0.542–0.663)0.909 (0.874–0.945)Positive predictive value (95% CI)0.163 (0.108–0.218)0.147 (0.092–0.203)0.198 (0.129–0.268)0.378 (0.222–0.535)Negative predictive value (95% CI)0.965 (0.931–0.999)0.931 (0.887–0.974)0.956 (0.925–0.988)0.928 (0.896–0.960)(*) Number of probands with a mutation/total number of probands below the thresholds (i.e., <10% or <20%)
Fig. 2ROC curves for BRCAPRO and Myriad II, at 10 and 20% thresholds

Performance parameters of BRCAPRO and Myriad II

(*) Number of probands with a mutation/total number of probands below the thresholds (i.e., <10% or <20%)

ROC curves for BRCAPRO and Myriad II, at 10 and 20% thresholds

Results using the 20% threshold, 7 out of 160 probands with BRCAPRO and 18 out of 249 probands with Myriad II had a mutation with a probability below the threshold. At both thresholds BRCAPRO is more sensitive than Myriad II, at 10% they both have similar specificity while at 20% is Myriad II more specific (Table 4). As shown by the four AUC calculations from Fig. 2: (a) BRCAPRO performed better than Myriad II at both thresholds and (b) the AUC for the 20% thresholds were better than 10% thresholds.

The group of 11 UVs had median mutation probabilities with BRCAPRO of 13.6% (range 0.3–97.3%) and with Myriad II of 17.4% (range 7.3–37.6%). When compared with the group with a mutation, the UVs had lower probabilities than the group with a mutation: with BRCAPRO were statistically significant (P = 0.004), with Myriad II did not achieve statistical significance (P = 0.093) (data not shown).

As shown in Table 5, with Myriad II nine UVs and with BRCAPRO seven of the 11 UVs had mutation probabilities below 20%. The UVs: c.4987-3C>G (putative splice variant), and c.5216A>G in BRCA1 were the only two UVs with probability values >20% both with BRCAPRO and Myriad II.Table 5Biochemical and epidemiological features of the unclassified variants with corresponding BRCAPRO and Myriad II scoresExonNucleotide changeEffect on proteinNumber of familiesBRCAPRO (%)Myriad II (%)Polarity changeConserved mammals/otherTimes reported (BIC)Co-segregation (literature)Co-occurrence (literature)Classification (literature)References
BRCA1
11c.692C>Tp.T231 M159.917.8YN/N2NDNUnknown[31]11c.1258G>Tp.D420Y16.510.5YN/Y3NDNUnknown[31]11c.1486C>Tp.R496C10.33.4YN/N43NDYNeutral[31, 32]17c.4987-3C>GNA96.937.6––0NDND20c.5194-25A>CNA114.017.4––0NDND20c.5216A>Gp.D1739G197.332.5YY/Y2NDNDCancer-associated[30]23c.5435C>Gp.P1812R14.67.5YY/Y0NDNDTotal: 77
BRCA2
10c.1385A>Gp.E462G137.817.8YY/N35NYNeutral[29, 33, 34]15c.7469T>Cp.I2490T113.619.9YN/N238NDND18c.8111C>Tp.S2704F111.57.3YN/N1NDND21c.8723T>Gp.V2908G11.29.8NY/N8NNNeutral[34]Total: 44Alignments were based on following species and NCBI reference sequences: BRCA1 Human (NP_009225), Chimpanzee (NP_001038958), Gorilla (AAT44835), Orang (AAT44834), Macaque (NP_001108421), Dog (NP_001013434), Rat (NP_036646), Mouse (NP_033894), Cow (NP_848668), Opossum (NP_001029141), Chicken (NP_989500), Frog (AAI70141); BRCA2 Human (CAA64484), Chimp (XP_509619), Dog (BAB91245), Cat (NP_001009858), Mouse (NP_033895), Opossum (ABP48762), Chicken (NP_989607). Y Yes, N No, ND Not done, NA Non Applicable

Biochemical and epidemiological features of the unclassified variants with corresponding BRCAPRO and Myriad II scores

Alignments were based on following species and NCBI reference sequences: BRCA1 Human (NP_009225), Chimpanzee (NP_001038958), Gorilla (AAT44835), Orang (AAT44834), Macaque (NP_001108421), Dog (NP_001013434), Rat (NP_036646), Mouse (NP_033894), Cow (NP_848668), Opossum (NP_001029141), Chicken (NP_989500), Frog (AAI70141); BRCA2 Human (CAA64484), Chimp (XP_509619), Dog (BAB91245), Cat (NP_001009858), Mouse (NP_033895), Opossum (ABP48762), Chicken (NP_989607). Y Yes, N No, ND Not done, NA Non Applicable

### Pathogenic mutations

The mean carrier probabilities with BRCAPRO and Myriad II for the groups with a mutation and without a mutation are shown in Table 3. The probability scores are also shown for the subgroup of patients with mutations detected with PTT and for those with mutations detected by complete analysis. The carrier probabilities of the probands with a mutation (n = 53) were significantly higher than those without a mutation, in both models (BRCAPRO: P < 0.0001, Myriad II: P < 0.0001). Apart from BRCAPRO and Myriad II, the occurrence of OC in the family (P = 0.001) was the only clinical parameter which was significantly associated with the presence of a BRCA1/2 mutation (data not shown).Table 3BRCAPRO and Myriad II scores of BRCA mutation positive versus negative populationsMutations with PTT
N = 21Mutations with complete analysis
N = 32Total mutations
N = 53No mutation
N = 243
P values (*)BRCAPROM
±SDRange0.699(± 0.248)0.186–0.9970.586(± 0.339)0.012–1.000.629(± 0.310)0.012–1.000.240(± 0.273)0.001–0.994<0.0001Myriad IIM±SDRange0.293(± 0.124)0.122–0.4650.202(± 0.126)0.019–0.4210.237(± 0.131)0.019–0.4650.119(± 0.102)0.007–0.603<0.0001Results expressed as: M ± SD mean (± standard deviation)(*) P values: total mutations versus no mutations: t test

BRCAPRO and Myriad II scores of BRCA mutation positive versus negative populations

M

±SD

Range

0.699

(± 0.248)

0.186–0.997

0.586

(± 0.339)

0.012–1.00

0.629

(± 0.310)

0.012–1.00

0.240

(± 0.273)

0.001–0.994

M

±SD

Range

0.293

(± 0.124)

0.122–0.465

0.202

(± 0.126)

0.019–0.421

0.237

(± 0.131)

0.019–0.465

0.119

(± 0.102)

0.007–0.603

Results expressed as: M ± SD mean (± standard deviation)

(*) P values: total mutations versus no mutations: t test

The performance of the models, at 10 and 20% thresholds, is presented in Table 4 and Fig. 2 shows the corresponding ROC curves. Using 10% threshold with BRCAPRO 110 probands had a probability <10% of whom four had a mutation. With Myriad, nine out of 126 probands with a probability, <10% had a mutation. The four probands missed with BRCAPRO were also missed with Myriad. These four probands carried a BRCA2 mutation, none of them had a family history of male BC, OC or bilateral BC. In one case the indication for DNA analysis was three BC cases in the family (indication 3, Table 1). The other three cases have a family history of two BC cases with one case below age 50 (indication 2a, Table 1).Table 4Performance parameters of BRCAPRO and Myriad IIThreshold 10%Threshold 20%BRCAPROMyriad IIBRCAPROMyriad IIMutations/probands below threshold (*)4/1109/1267/16018/249Sensitivity (95% CI)0.875 (0.760–0.990)0.719 (0.563–0.875)0.781 (0.638–0.925)0.438 (0.266–0.609)Specificity (95% CI)0.433 (0.372–0.494)0.476 (0.415–0.538)0.602 (0.542–0.663)0.909 (0.874–0.945)Positive predictive value (95% CI)0.163 (0.108–0.218)0.147 (0.092–0.203)0.198 (0.129–0.268)0.378 (0.222–0.535)Negative predictive value (95% CI)0.965 (0.931–0.999)0.931 (0.887–0.974)0.956 (0.925–0.988)0.928 (0.896–0.960)(*) Number of probands with a mutation/total number of probands below the thresholds (i.e., <10% or <20%)
Fig. 2ROC curves for BRCAPRO and Myriad II, at 10 and 20% thresholds

Performance parameters of BRCAPRO and Myriad II

(*) Number of probands with a mutation/total number of probands below the thresholds (i.e., <10% or <20%)

ROC curves for BRCAPRO and Myriad II, at 10 and 20% thresholds

Results using the 20% threshold, 7 out of 160 probands with BRCAPRO and 18 out of 249 probands with Myriad II had a mutation with a probability below the threshold. At both thresholds BRCAPRO is more sensitive than Myriad II, at 10% they both have similar specificity while at 20% is Myriad II more specific (Table 4). As shown by the four AUC calculations from Fig. 2: (a) BRCAPRO performed better than Myriad II at both thresholds and (b) the AUC for the 20% thresholds were better than 10% thresholds.

### Unclassified variants

The group of 11 UVs had median mutation probabilities with BRCAPRO of 13.6% (range 0.3–97.3%) and with Myriad II of 17.4% (range 7.3–37.6%). When compared with the group with a mutation, the UVs had lower probabilities than the group with a mutation: with BRCAPRO were statistically significant (P = 0.004), with Myriad II did not achieve statistical significance (P = 0.093) (data not shown).

As shown in Table 5, with Myriad II nine UVs and with BRCAPRO seven of the 11 UVs had mutation probabilities below 20%. The UVs: c.4987-3C>G (putative splice variant), and c.5216A>G in BRCA1 were the only two UVs with probability values >20% both with BRCAPRO and Myriad II.Table 5Biochemical and epidemiological features of the unclassified variants with corresponding BRCAPRO and Myriad II scoresExonNucleotide changeEffect on proteinNumber of familiesBRCAPRO (%)Myriad II (%)Polarity changeConserved mammals/otherTimes reported (BIC)Co-segregation (literature)Co-occurrence (literature)Classification (literature)References
BRCA1
11c.692C>Tp.T231 M159.917.8YN/N2NDNUnknown[31]11c.1258G>Tp.D420Y16.510.5YN/Y3NDNUnknown[31]11c.1486C>Tp.R496C10.33.4YN/N43NDYNeutral[31, 32]17c.4987-3C>GNA96.937.6––0NDND20c.5194-25A>CNA114.017.4––0NDND20c.5216A>Gp.D1739G197.332.5YY/Y2NDNDCancer-associated[30]23c.5435C>Gp.P1812R14.67.5YY/Y0NDNDTotal: 77
BRCA2
10c.1385A>Gp.E462G137.817.8YY/N35NYNeutral[29, 33, 34]15c.7469T>Cp.I2490T113.619.9YN/N238NDND18c.8111C>Tp.S2704F111.57.3YN/N1NDND21c.8723T>Gp.V2908G11.29.8NY/N8NNNeutral[34]Total: 44Alignments were based on following species and NCBI reference sequences: BRCA1 Human (NP_009225), Chimpanzee (NP_001038958), Gorilla (AAT44835), Orang (AAT44834), Macaque (NP_001108421), Dog (NP_001013434), Rat (NP_036646), Mouse (NP_033894), Cow (NP_848668), Opossum (NP_001029141), Chicken (NP_989500), Frog (AAI70141); BRCA2 Human (CAA64484), Chimp (XP_509619), Dog (BAB91245), Cat (NP_001009858), Mouse (NP_033895), Opossum (ABP48762), Chicken (NP_989607). Y Yes, N No, ND Not done, NA Non Applicable

Biochemical and epidemiological features of the unclassified variants with corresponding BRCAPRO and Myriad II scores

Alignments were based on following species and NCBI reference sequences: BRCA1 Human (NP_009225), Chimpanzee (NP_001038958), Gorilla (AAT44835), Orang (AAT44834), Macaque (NP_001108421), Dog (NP_001013434), Rat (NP_036646), Mouse (NP_033894), Cow (NP_848668), Opossum (NP_001029141), Chicken (NP_989500), Frog (AAI70141); BRCA2 Human (CAA64484), Chimp (XP_509619), Dog (BAB91245), Cat (NP_001009858), Mouse (NP_033895), Opossum (ABP48762), Chicken (NP_989607). Y Yes, N No, ND Not done, NA Non Applicable

### Discussion

This study included 306 probands who had prior been selected for DNA-analysis of BRCA1 and BRCA2 mutations based on cancer history criteria. From 1995 until 2000, the PTT was the widely used quick, inexpensive detection technique, which at that time resulted in the detection of a mutation in 20 unrelated patients. At a later stage, complete sequencing analysis of the coding regions including intron–exon boundaries was performed, which resulted in the identification of an additional 43 genetic variations in individual probands of which 32 were pathogenic and 11 unclassified (UV). We then performed a retrospective study to estimate the mutation probabilities using BRCAPRO and Myriad II in all cases.

Concerning mutation detection rate, our results are in line with previous reports ([21] which show that complete BRCA1/2 analysis substantially increases the number of variations detected as compared to PTT alone. However, it is less obvious which methods and which thresholds to use for selecting the patients for DNA-analysis. In this respect, the right balance between costs and mutation coverage needs to be determined: obviously, the ideal method has a high sensitivity, a high specificity and a high negative predictive value.

We found in our population study that BRCAPRO and Myriad II are more specific than cancer history, since they both excluded from analysis about 40% of the patients who did not have a mutation. Looking at sensitivity, BRCAPRO was more sensitive than Myriad II which is in line with previous publications by Berry et al. [18, 22]. Overall BRCAPRO performed better than Myriad II, as shown by their ROC curves.

Several groups have compared risk prediction models among each other [22–26]. Both Antoniou et al. [25] and Lindor et al. [26] compared performance of various carrier prediction algorithms, including BRCAPRO and Myriad II. In the study of Lindor et al. (26) no a priori selection criteria were required to perform DNA-testing. All models tested performed reasonably well. In the study of Antoniou et al. [25], with 1,934 families the largest published so far, families in whom DNA-testing had been performed were supplied by the clinical genetics centres and from research studies. The indication criteria used to perform DNA-analysis in those settings are not specified. The authors show that BOADICEA provided the best discrimination of the five algorithms analyzed, though no significant differences were observed with this model and BRCAPRO. Both studies [25, 26] concluded that these prediction models under-predicted the numbers of carriers in the low estimated risk category. However, none of these studies have compared the added value of the use of quantitative methods, i.e., scores obtained with mutation probability detection methods to qualitative criteria based on cancer family history as indication for DNA analysis.

About the thresholds, the AUCs were better at 20% than at 10%, which also indicates the preference from an economic point. However, from a clinical point of view sensitivity is more important than specificity, which means that a choice has to be made between sparing 16% of the total of tests performed (i.e., 50 probands with mutation probabilities between 10 and 20% with BRCAPRO) at the price of missing three probands with a mutation.

Noteworthy is that the four pathogenic mutations that were missed with BRCAPRO, were also four of the nine which were missed with Myriad II using the 10% threshold and that all four are located in the BRCA2 gene. This suggests that for detection of BRCA1 mutations, BRCAPRO at a threshold of 10% is the method of choice: i.e., it has the same sensitivity as the cancer history criteria and reduces the number of tests by 40%. The fact that these probability models have a better performance for BRCA1 than BRCA2 has been reported previously by James et al. [23] and Bodmer et al. [24], and is related to the lower penetrance of the mutations in BRCA2. Indeed, as explained in the results section, none of the families of these patients included any of the most characteristic features of the BRCA families, such as OC, bilateral BC or male BC.

An additional advantage of the use of models with thresholds is the exclusion of many patients carrying an UV. Indeed, since the majority of the UVs that have been detected and reported in the literature are later reclassified as neutral variants [27], a specific method is also desirable which can selectively exclude those UVs which are not clinically relevant. We have previously reported that patients with a pathogenic mutation have significantly higher a priori probabilities with BRCAPRO and Myriad II than those with a UV [28], which also suggests that most of the UVs are neutral variants. In addition, BRCAPRO and Myriad II have been shown to be useful parameters also to predict whether a UV is deleterious when used in multiparametric regression models [29].

Interestingly, from the two UVs with mutation probabilities above 20% with BRCAPRO and Myriad II, the p.D1739G variant in BRCA1 has been classified as cancer-associated in the literature according to criteria, such as polarity change, conservation amongst species and in silico models [30]. Conversely, for most of those UVs with a mutation probability below 20%, evidence from other sources also argue against them being deleterious. This is the case for: p.R496C in BRCA1 [31, 32] and p.E462G [29, 33, 34] and p.V2908G [34] in BRCA2.

In conclusion, BRCAPRO and Myriad II probability scores allow exclusion of a large proportion of mutation negative probands for BRCA1/2 diagnosis, as compared to using familial cancer history alone. When comparing the two models, BRCAPRO is more sensitive in detecting known pathogenic mutations than Myriad II. Myriad II scores provide no added value to BRCAPRO alone. BRCAPRO performs better for the identification of patients with a BRCA1 than with a BRCA2 mutation. The use of quantitative scores exclude from study a considerable number of patients carrying an UV. Information about the probability scores may be also valuable for the classification of UVs.



# SUPPLEMENTAL FILE 1: 10689_2009_Article_9305.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)