# MAIN TEXT

## A naturally occurring human RPA subunit homolog does not support DNA replication or cell-cycle progression

### Abstract

Replication Protein A (RPA) is a single-stranded DNA-binding protein essential for DNA replication, repair, recombination and cell-cycle regulation. A human homolog of the RPA2 subunit, called RPA4, was previously identified and shown to be expressed in colon mucosal and placental cells; however, the function of RPA4 was not determined. To examine the function of RPA4 in human cells, we carried out knockdown and replacement studies to determine whether RPA4 can substitute for RPA2 in the cell. Unlike RPA2, exogenous RPA4 expression did not support chromosomal DNA replication and lead to cell-cycle arrest in G2/M. In addition, RPA4 localized to sites of DNA repair and reduced γ-H2AX caused by RPA2 depletion. These studies suggest that RPA4 cannot support cell proliferation but can support processes that maintain the genomic integrity of the cell.

### INTRODUCTION

In order to proliferate and maintain genomic integrity, a cell must recognize and repair DNA damage and replicate DNA with high fidelity. One protein intricately involved with these processes is the single-stranded DNA (ssDNA)-binding protein, Replication Protein A (RPA) (1–3). RPA was originally isolated as a factor essential for simian virus 40 (SV40) DNA replication (1) and has since been shown to have integral roles in DNA repair, recombination, chromosome stability and cell-cycle regulation (1–3).

In addition to ssDNA binding, RPA interacts with a number of proteins involved in genome maintenance and cell-cycle control (e.g. XPA, ATR-ATRIP, p53) (4), and mutations in RPA or in proteins that interact with RPA are often correlated with human disease, particularly cancer. It has been shown that a cancer predisposing mutation in BRCA2 (Y42C) disrupts the interaction between BRCA2 and RPA (5). Furthermore, a single amino acid substitution (L221P) in RPA1 results in a high rate of lymphoid tumor development and shortened lifespan when heterozygous in mice (6). This mutation is analogous to a mutation, rfa1-t48, originally isolated in the budding yeast Saccharomyces cerevisiae that displays multiple DNA damage sensitivities (7). It has also been demonstrated that increased expression of RPA1 and RPA2 correlates with increased severity of colon cancer (8). This is not surprising, since RPA is essential for cells to proliferate (1).

Although ‘canonical’ RPA is composed of the subunits RPA1, RPA2 and RPA3, some organisms, such as seed plants (e.g. rice, Arabidopsis thaliana) and some protists, contain multiple RPA subunit genes that form multiple RPA complexes (9,10). In the case of plants, the different RPA complexes appear to have different functions (11). In human cells, a single homolog of the RPA2 subunit has been identified, called RPA4. RPA4 was isolated through a HeLa cell library interaction-trap/yeast two-hybrid screen as a factor that interacts with RPA1 (12). Initial studies showed that RPA4 also interacts with RPA3, but not with RPA2, and immunoprecipitation of RPA1 coprecipitates RPA4 after expression in rabbit reticulocyte lysate or in 293 cells (12). RPA4 transcribed/translated in vitro with RPA1 and RPA3 is retained on ssDNA cellulose, suggesting that this ‘alternative’ complex has ssDNA-binding capability (12). More recently, it has been shown that RPA4 forms a stable complex with RPA1 and RPA3 and has solution properties indistinguishable from canonical RPA (13).

The studies presented here focus on understanding the function of RPA4 in human cells. We present a genomic analysis of the RPA4 gene that indicates that RPA4 is mammalian-specific. We show that expression of RPA4 in cells does not support chromosomal DNA replication or cell-cycle progression. However, we present evidence that RPA4 functions in cellular DNA metabolism. RPA4 can localize to DNA repair foci and appears to participate in the cellular DNA damage response. We identify the region of RPA4 responsible for the observed phenotypes. These findings suggest that RPA4 expression may be involved in maintaining cell quiescence.

### MATERIALS AND METHODS

To identify exogenous expression of RPA in HeLa cells, enhanced green fluorescent protein (EGFP)-tagged RPA1, RPA2, RPA3 and RPA4 constructs were generated; EGFP-tagged RPA1 (pEGFP-hsRPA70), RPA2 (pEGFP-hsRPA32) and RPA3 (pEGFP-hsRPA14) were generated previously (14). EGFP-RPA4 (pEGFP-hsRPA4) was generated by PCR amplification of the RPA4 coding region from pBABE-puro-RPA4 (12) using primers O-606 (5′-CAGATCTCGAGGTGGAGGCATGAGTAAGAGTGGGTTTGGG-3′) and O-607 (5′-CCCGCGGTACCTCAATCAGCAGACTTAAAATG-3′) and inserted into the XhoI-KpnI sites of pEGFP-C1 (Clontech). EGFP-scRPA32 (pEGFP-scRPA32) was generated by PCR amplification of the RFA2 coding region from p11d-tscRPA (15) using primers O-587 (5′-GACTCAGATCTGGTGGAGGCATGGCAAGTTATCAACCATATAACGAATATTCATC-3′) and O-588 (5′-CCCGCGGTACCTCATAGGGCAAAGAAGTTATTGTCATCAAAAG-3′) and inserted into the BglII-KpnI sites of pEGFP-C1 (Clontech). All constructs were confirmed by sequencing.

RPA2/RPA4 ‘hybrid’ constructs were generated by first creating a ‘cassettized’ EGFP-RPA2 construct (pEGFP-hsRPA32-AS). The QuikChange XL Site-Directed Mutagenesis Kit was used to generate a unique SpeI site between the DNA-binding domain (DBD) and the C-terminal region containing the winged-helix domain (WHD) on plasmid pEGFP-hsRPA32 with primers O-635 (5′-GGTACTAAGCAAAGCCACTAGTCAGCCCTCAGCAGGGAG-3′) and O-636 (5′-CTCCCTGCTGAGGGCTGACTAGTGGCTTTGCTTAGTACC-3′). This plasmid (pEGFP-hsRPA32-S) was then used to generate a unique AflII site between the phosphorylation domain (PD) and the DBD with primers O-644 (5′-GCCGAAAAGAAATCAAGACTTAAGGCCCAGCACATTGTG-3′) and O-645 (5′-CACAATGTGCTGGGCCTTAAGTCTTGATTTCTTTTCGGC-3′). PCR was used to amplify the putative PD, the putative DBD, or the C-terminus of RPA4, and these fragments were used to replace the corresponding domain of RPA2. Primers used to amplify the RPA4 PD were O-606 and O-678 (5′-TGGGCCTTAAGCTTAGGTCTTTGGGTCTTAATAGCAG-3′), to amplify the RPA4 DBD were O-676 (5′-CAAGACTTAAGGCCCAGGACGTTGTACCGTGTAACGTGAACC-3′) and O-677 (5′-GGCTGACTAGTGGCTTTATCCAGCATCATGTGTGC-3′), and to amplify the RPA4 C-terminus were O-679 (5′-AAGCCACTAGTCGTCGTGATACCACTGTAGAAAGTG-3′) and O-607.

RPA2 or RPA4 ‘core’ plasmids, containing only the DBD of either gene, were constructed using standard PCR and cloning techniques. Briefly, DBD-D was amplified with primers O-672 (5′-CAGATCTCGAGGTGGAGGCCAGCACATTGTGCCCTGTACTATATCTC-3′) and O-673 (5′-CCCGCGGTACCTCAGGCTTTGCTTAGTACCATGTGTGC-3′) and cloned into the XhoI-KpnI sites of pEGFP-C1. DBD-G was amplified with primers O-674 (5′-GCGGCAGATCTGGTGGAGGCCAGGACGTTGTACCGTGTAACGTGAACC-3′) and O-675 (5′-CCCGCGGTACCTCAGGCTTTATCCAGCATCATGTGTGC-3′) and cloned into the BglII-KpnI sites of pEGFP-C1.

HeLa cells were grown in Dubellco’s Modified Eagle’s Medium (DMEM) supplemented with 10% bovine calf serum (BCS) at 37°C and 5% CO2. Small interfering RNA (siRNA) targeting the 3′ untranslated region (UTR) of RPA2 mRNA was generated (Dharmacon). The sequence of the siRNA for RPA2 was identical to previously published siRNA (16). Silencer negative control #2 RNA (Ambion) was used as a control to examine the specificity of RPA2 knockdown.

HeLa cells were seeded in six-well tissue culture plates at 2 × 105 cells/well for 18–24 h. Cells were then transfected with 200 pmol RPA2 siRNA using Lipofectamine 2000 (Invitrogen) at time zero (t = 0). At 24 h post-transfection of siRNA (t = 24), the media was removed from the cells and fresh DMEM/10% BCS was added to each well. The cells were then transfected with 250 ng of the appropriate plasmid DNA using Lipofectamine 2000. At 48 h post-transfection of siRNA (t = 48), the media was removed and fresh DMEM/10% BCS was added to the cells. The cells were then grown until collected for protein, immunofluorescence (IF), or flow cytometry.

Cells were trypsinized and collected at various times post-transfection and pelleted at 1.5 rcf for 5 min. The cells were washed once with phosphate buffered saline (137 mM NaCl, 2.7 mM KCl, 4.3 mM Na2HPO4·7H2O, 1.4 mM KH2PO4) and pelleted at 1.5 rcf for 5 min. Cells were then lysed in RIPA buffer [1% (w/w) NP-40, 1% (w/v) sodium deoxycholate, 0.1% (w/v) SDS, 150 mM NaCl, 10 mM sodium phosphate (pH 7.2), 2 mM EDTA, 50 mM sodium fluoride, 0.2 mM sodium vanadate, 1 µg/ml aprotinin] and placed at −80°C. Cell lysates were thawed, sonicated with a microtip at setting four using a Sonic Dismembrator 550 (Fisher) by pulsing for 3 s four times, and the protein was quantitated using the DC assay (Bio-Rad). Equal amounts (100 µg) of protein were loaded on an 8–14% gradient SDS–PAGE gel and run at 40 W for 1.5–2 h. Gels were electroblotted onto Bio-Rad nitrocellulose membrane at 0.2 mA for 16–20 h at 4°C.

The membrane was blocked with 10% non-fat dry milk/1 × TBS/0.1% Tween 20 for 30 min. The blocking solution was removed and primary antibody in 10% non-fat dry milk/1 × TBS/0.1% Tween 20 was added to the membrane and incubated at room temperature for 2 h to overnight. Primary antibodies used were N2.2 (rabbit polyclonal to RPA1 and RPA2) or JL-8 (monoclonal to GFP; Clontech) at 1:500 and 1:6000, respectively. The membrane was then washed three times with 1 × TBS/0.1% Tween 20. The secondary antibody goat anti-mouse IgG horseradish peroxidase (HRP) (Sigma) or goat anti-rabbit IgG HRP (Sigma) was diluted 1:20 000 in 1 × TBS/0.1% Tween 20, added to the membrane, and incubated for 1–2 h at room temperature. The membrane was washed four times with 1 × TBS/0.1% Tween 20. SuperSingal West Pico Chemiluminescent Substrate (Pierce) was used to detect HRP.

To examine the localization of RPA to DNA, HeLa cells were treated as described for RNAi above and grown for 90–96 h post-transfection of RPA2 siRNA. Cells either had no additional treatments (un-stressed) or were incubated for the final 4 h with indicated amount camptothecin or etoposide (DNA damaged). The cells were then fixed, extracted to leave only chromatin bound protein, and prepared for fluorescence microscopy as described (17).

Cells were treated, assayed, and designations for flow cytometry are as described (17). Where indicated, DNA damage was induced as described above. Synchronized cells were generated with a 27 h treatment with 5 µg/ml aphidicolin (Sigma) starting at 57 h (t = 57) post-transfection of RPA2 siRNA. At 84 h (t = 84) post-transfection, cells were released from aphidicolin block by rinsing the cells once with 2 ml PBS and adding 2 ml fresh medium. Cells were collected at indicated time after release and examined by flow cytometry.

Geno3D (http://geno3d-pbil.ibcp.fr) was used to obtain predicted protein structures for the DBDs of RPA4 (aa 44–171), RPA2-basic (aa 44–171) and RPA4-acidic (aa 44–172) or the C-terminus of RPA4 (aa 196–263). Briefly, RPA4, RPA2-basic and RPA4-acidic were modeled against known structures of RPA2 DBD-D [2PQA-chain C (18), 2PI2-chain A (18), 1L10-chain B (19) and 1QUQ-chain A (20)]. All structures (known and predicted) were examined using Swiss-PDB Viewer and displayed using Mac MegaPOV.

To determine the statistical significance of differences in the percentage of cells in S- or G2/M-phase, cell-cycle analysis software integrated into FlowJo (TreeStar) was first used to measure these values for cells expressing each construct. These values were then entered into InStat (GraphPad) as raw data, and the mean, standard deviation, sample size, standard error of the mean, and upper and lower confidence limits were determined. A one-way analysis of variance (ANOVA) was performed using the Tukey–Kramer multiple comparisons test. InStat also tested the data for normality using the Kolmogorov–Smirnov test. Where statistical significance is mentioned, the type of test, q-value and corresponding P-value are shown. For S- or G2/M-phase statistics, a q-value >4.732 is considered significant (P < 0.05).

### Exogenous RPA expression constructs

To identify exogenous expression of RPA in HeLa cells, enhanced green fluorescent protein (EGFP)-tagged RPA1, RPA2, RPA3 and RPA4 constructs were generated; EGFP-tagged RPA1 (pEGFP-hsRPA70), RPA2 (pEGFP-hsRPA32) and RPA3 (pEGFP-hsRPA14) were generated previously (14). EGFP-RPA4 (pEGFP-hsRPA4) was generated by PCR amplification of the RPA4 coding region from pBABE-puro-RPA4 (12) using primers O-606 (5′-CAGATCTCGAGGTGGAGGCATGAGTAAGAGTGGGTTTGGG-3′) and O-607 (5′-CCCGCGGTACCTCAATCAGCAGACTTAAAATG-3′) and inserted into the XhoI-KpnI sites of pEGFP-C1 (Clontech). EGFP-scRPA32 (pEGFP-scRPA32) was generated by PCR amplification of the RFA2 coding region from p11d-tscRPA (15) using primers O-587 (5′-GACTCAGATCTGGTGGAGGCATGGCAAGTTATCAACCATATAACGAATATTCATC-3′) and O-588 (5′-CCCGCGGTACCTCATAGGGCAAAGAAGTTATTGTCATCAAAAG-3′) and inserted into the BglII-KpnI sites of pEGFP-C1 (Clontech). All constructs were confirmed by sequencing.

RPA2/RPA4 ‘hybrid’ constructs were generated by first creating a ‘cassettized’ EGFP-RPA2 construct (pEGFP-hsRPA32-AS). The QuikChange XL Site-Directed Mutagenesis Kit was used to generate a unique SpeI site between the DNA-binding domain (DBD) and the C-terminal region containing the winged-helix domain (WHD) on plasmid pEGFP-hsRPA32 with primers O-635 (5′-GGTACTAAGCAAAGCCACTAGTCAGCCCTCAGCAGGGAG-3′) and O-636 (5′-CTCCCTGCTGAGGGCTGACTAGTGGCTTTGCTTAGTACC-3′). This plasmid (pEGFP-hsRPA32-S) was then used to generate a unique AflII site between the phosphorylation domain (PD) and the DBD with primers O-644 (5′-GCCGAAAAGAAATCAAGACTTAAGGCCCAGCACATTGTG-3′) and O-645 (5′-CACAATGTGCTGGGCCTTAAGTCTTGATTTCTTTTCGGC-3′). PCR was used to amplify the putative PD, the putative DBD, or the C-terminus of RPA4, and these fragments were used to replace the corresponding domain of RPA2. Primers used to amplify the RPA4 PD were O-606 and O-678 (5′-TGGGCCTTAAGCTTAGGTCTTTGGGTCTTAATAGCAG-3′), to amplify the RPA4 DBD were O-676 (5′-CAAGACTTAAGGCCCAGGACGTTGTACCGTGTAACGTGAACC-3′) and O-677 (5′-GGCTGACTAGTGGCTTTATCCAGCATCATGTGTGC-3′), and to amplify the RPA4 C-terminus were O-679 (5′-AAGCCACTAGTCGTCGTGATACCACTGTAGAAAGTG-3′) and O-607.

RPA2 or RPA4 ‘core’ plasmids, containing only the DBD of either gene, were constructed using standard PCR and cloning techniques. Briefly, DBD-D was amplified with primers O-672 (5′-CAGATCTCGAGGTGGAGGCCAGCACATTGTGCCCTGTACTATATCTC-3′) and O-673 (5′-CCCGCGGTACCTCAGGCTTTGCTTAGTACCATGTGTGC-3′) and cloned into the XhoI-KpnI sites of pEGFP-C1. DBD-G was amplified with primers O-674 (5′-GCGGCAGATCTGGTGGAGGCCAGGACGTTGTACCGTGTAACGTGAACC-3′) and O-675 (5′-CCCGCGGTACCTCAGGCTTTATCCAGCATCATGTGTGC-3′) and cloned into the BglII-KpnI sites of pEGFP-C1.

### Tissue culture, RNA interference (RNAi) and exogenous RPA expression

HeLa cells were grown in Dubellco’s Modified Eagle’s Medium (DMEM) supplemented with 10% bovine calf serum (BCS) at 37°C and 5% CO2. Small interfering RNA (siRNA) targeting the 3′ untranslated region (UTR) of RPA2 mRNA was generated (Dharmacon). The sequence of the siRNA for RPA2 was identical to previously published siRNA (16). Silencer negative control #2 RNA (Ambion) was used as a control to examine the specificity of RPA2 knockdown.

HeLa cells were seeded in six-well tissue culture plates at 2 × 105 cells/well for 18–24 h. Cells were then transfected with 200 pmol RPA2 siRNA using Lipofectamine 2000 (Invitrogen) at time zero (t = 0). At 24 h post-transfection of siRNA (t = 24), the media was removed from the cells and fresh DMEM/10% BCS was added to each well. The cells were then transfected with 250 ng of the appropriate plasmid DNA using Lipofectamine 2000. At 48 h post-transfection of siRNA (t = 48), the media was removed and fresh DMEM/10% BCS was added to the cells. The cells were then grown until collected for protein, immunofluorescence (IF), or flow cytometry.

### Cell lysates and protein detection

Cells were trypsinized and collected at various times post-transfection and pelleted at 1.5 rcf for 5 min. The cells were washed once with phosphate buffered saline (137 mM NaCl, 2.7 mM KCl, 4.3 mM Na2HPO4·7H2O, 1.4 mM KH2PO4) and pelleted at 1.5 rcf for 5 min. Cells were then lysed in RIPA buffer [1% (w/w) NP-40, 1% (w/v) sodium deoxycholate, 0.1% (w/v) SDS, 150 mM NaCl, 10 mM sodium phosphate (pH 7.2), 2 mM EDTA, 50 mM sodium fluoride, 0.2 mM sodium vanadate, 1 µg/ml aprotinin] and placed at −80°C. Cell lysates were thawed, sonicated with a microtip at setting four using a Sonic Dismembrator 550 (Fisher) by pulsing for 3 s four times, and the protein was quantitated using the DC assay (Bio-Rad). Equal amounts (100 µg) of protein were loaded on an 8–14% gradient SDS–PAGE gel and run at 40 W for 1.5–2 h. Gels were electroblotted onto Bio-Rad nitrocellulose membrane at 0.2 mA for 16–20 h at 4°C.

The membrane was blocked with 10% non-fat dry milk/1 × TBS/0.1% Tween 20 for 30 min. The blocking solution was removed and primary antibody in 10% non-fat dry milk/1 × TBS/0.1% Tween 20 was added to the membrane and incubated at room temperature for 2 h to overnight. Primary antibodies used were N2.2 (rabbit polyclonal to RPA1 and RPA2) or JL-8 (monoclonal to GFP; Clontech) at 1:500 and 1:6000, respectively. The membrane was then washed three times with 1 × TBS/0.1% Tween 20. The secondary antibody goat anti-mouse IgG horseradish peroxidase (HRP) (Sigma) or goat anti-rabbit IgG HRP (Sigma) was diluted 1:20 000 in 1 × TBS/0.1% Tween 20, added to the membrane, and incubated for 1–2 h at room temperature. The membrane was washed four times with 1 × TBS/0.1% Tween 20. SuperSingal West Pico Chemiluminescent Substrate (Pierce) was used to detect HRP.

### Induction of DNA damage and fluorescence microscopy

To examine the localization of RPA to DNA, HeLa cells were treated as described for RNAi above and grown for 90–96 h post-transfection of RPA2 siRNA. Cells either had no additional treatments (un-stressed) or were incubated for the final 4 h with indicated amount camptothecin or etoposide (DNA damaged). The cells were then fixed, extracted to leave only chromatin bound protein, and prepared for fluorescence microscopy as described (17).

### Flow cytometry and cell synchronization

Cells were treated, assayed, and designations for flow cytometry are as described (17). Where indicated, DNA damage was induced as described above. Synchronized cells were generated with a 27 h treatment with 5 µg/ml aphidicolin (Sigma) starting at 57 h (t = 57) post-transfection of RPA2 siRNA. At 84 h (t = 84) post-transfection, cells were released from aphidicolin block by rinsing the cells once with 2 ml PBS and adding 2 ml fresh medium. Cells were collected at indicated time after release and examined by flow cytometry.

### Molecular modeling of RPA4 and derivatives

Geno3D (http://geno3d-pbil.ibcp.fr) was used to obtain predicted protein structures for the DBDs of RPA4 (aa 44–171), RPA2-basic (aa 44–171) and RPA4-acidic (aa 44–172) or the C-terminus of RPA4 (aa 196–263). Briefly, RPA4, RPA2-basic and RPA4-acidic were modeled against known structures of RPA2 DBD-D [2PQA-chain C (18), 2PI2-chain A (18), 1L10-chain B (19) and 1QUQ-chain A (20)]. All structures (known and predicted) were examined using Swiss-PDB Viewer and displayed using Mac MegaPOV.

### Statistical analysis

To determine the statistical significance of differences in the percentage of cells in S- or G2/M-phase, cell-cycle analysis software integrated into FlowJo (TreeStar) was first used to measure these values for cells expressing each construct. These values were then entered into InStat (GraphPad) as raw data, and the mean, standard deviation, sample size, standard error of the mean, and upper and lower confidence limits were determined. A one-way analysis of variance (ANOVA) was performed using the Tukey–Kramer multiple comparisons test. InStat also tested the data for normality using the Kolmogorov–Smirnov test. Where statistical significance is mentioned, the type of test, q-value and corresponding P-value are shown. For S- or G2/M-phase statistics, a q-value >4.732 is considered significant (P < 0.05).

### RESULTS

RPA4 was identified prior to the sequencing of the human genome. With sequencing and annotation of the human and other genomes, it was possible to analyze the RPA4 gene and examine its evolutionary conservation. This analysis identified three interesting features of the RPA4 gene. First, RPA4 is intronless, suggesting that RPA4 arose from a viral- or retrotransposon-mediated gene duplication event. Second, RPA4 resides on the X-chromosome at position q21.33. Third, RPA4 lies in the intron of a known coding gene, diaphanous 2 (DIAPH2). DIAPH2 encodes a formin-related actin-binding protein (21,22). Expression of RPA4 is not well characterized, but available public data indicates that RPA4 is expressed in different tissues than DIAPH2 (http://genome.ucsc.edu/), indicating that it is independently regulated.

The RPA4-related sequences are only found in mammals. Primates (human, chimpanzee, orangutan, monkey and marmoset) and horse contain complete coding sequences for RPA4 region (Figure 1). Marmoset RPA4 is the only gene not located on the X-chromosome. Other mammals examined (e.g. cow, elephant, dog, cat, rabbit, mouse, rat and armadillo) have partial RPA4-related sequences or a RPA4 pseudogene at the equivalent position of the X-chromosome. We are unable to identify RPA4-related sequences in any non-mammalian genome, and conclude that functional RPA4 genes are found only in primates and a few other mammals.
Figure 1.Alignment of known and putative RPA4 and RPA2 from six mammalian species. RPA4 and RPA2 protein sequences are from Homo sapiens (Hosa) (NP037479, NP002937) and the predicted sequences from Pan troglodytes (Patr) (XP001136507, XP513251), Pongo abelii (Poab) (CAH90667), Callithrix jacchus (Caja) (RPA4 protein derived from DNA Contig10151:1273-53091; RPA2 protein derived from DNA Contig150:1594-1135700), Macaca mullata (Mamu) (XP001088106) and Equus caballus (Eqca) (XP001491922, XP001500549). Amino acid positions are denoted at the beginning and end of each line of sequence. Alignment was performed and identical (*), conserved (:), and semiconserved (.) residues are denoted below each alignment by TCOFFEE. Grey and black shaded residues denote regions of least conservation or major charge difference between RPA2 and RPA4 noted in Keshav et al. (12). Black shaded residues also denote amino acids swapped between RPA2 and RPA4 in our studies. The N-termini containing the putative phosphorylation domain are denoted with lowercase letters, the OB-fold DBDs are identified by UPPERCASE letters, and the C-termini are denoted with lowercase italics.

Alignment of known and putative RPA4 and RPA2 from six mammalian species. RPA4 and RPA2 protein sequences are from Homo sapiens (Hosa) (NP037479, NP002937) and the predicted sequences from Pan troglodytes (Patr) (XP001136507, XP513251), Pongo abelii (Poab) (CAH90667), Callithrix jacchus (Caja) (RPA4 protein derived from DNA Contig10151:1273-53091; RPA2 protein derived from DNA Contig150:1594-1135700), Macaca mullata (Mamu) (XP001088106) and Equus caballus (Eqca) (XP001491922, XP001500549). Amino acid positions are denoted at the beginning and end of each line of sequence. Alignment was performed and identical (*), conserved (:), and semiconserved (.) residues are denoted below each alignment by TCOFFEE. Grey and black shaded residues denote regions of least conservation or major charge difference between RPA2 and RPA4 noted in Keshav et al. (12). Black shaded residues also denote amino acids swapped between RPA2 and RPA4 in our studies. The N-termini containing the putative phosphorylation domain are denoted with lowercase letters, the OB-fold DBDs are identified by UPPERCASE letters, and the C-termini are denoted with lowercase italics.

To determine the function of RPA4 in cells, we used a knockdown-replacement strategy. A short interfering RNA (siRNA) was used to target the 3′ untranslated region of RPA2 mRNA for degradation in HeLa cells. A decrease in RPA2 protein was observed at 48 h post-transfection of siRNA; only ∼30% of RPA2 remained in the cells (Figure 2a). No change in RPA2 levels was observed in mock treated cells and in cells transfected with a non-specific control siRNA (Figure 2a and data not shown). Maximal knockdown of RPA2 occurred between 72 and 96 h post-transfection and resulted in coordinate depletion of RPA1 protein [Figure 2a; as previously observed in ref. (23)].
Figure 2.Knockdown of endogenous RPA2. (a) Examination of mock and RPA2 siRNA knockdown by western blot showing RPA2 and RPA1 depletion. Number of hours post-transfection of siRNA are denoted above each lane. R, 200 ng purified recombinant human RPA. (b) Cell-cycle analysis of mock or RPA2 siRNA knockdown. DNA content was analyzed by flow cytometry and plotted as number of cells versus DNA content (FL2-A fluorescence). Cells with unreplicated DNA content are labeled as G1 phase, cells with replicated DNA content are labeled G2/M phase and cells with intermediate DNA content are labeled S-phase. The number of hours post-transfection of mock or RPA2 siRNA is designated in the upper right corner of each histogram. (c) Microscopic examination of nuclei following mock or RPA2 depletion. Hours post-transfection are designated at the upper left of each image. Arrows designate fragmented and micronuclei formation.

Knockdown of endogenous RPA2. (a) Examination of mock and RPA2 siRNA knockdown by western blot showing RPA2 and RPA1 depletion. Number of hours post-transfection of siRNA are denoted above each lane. R, 200 ng purified recombinant human RPA. (b) Cell-cycle analysis of mock or RPA2 siRNA knockdown. DNA content was analyzed by flow cytometry and plotted as number of cells versus DNA content (FL2-A fluorescence). Cells with unreplicated DNA content are labeled as G1 phase, cells with replicated DNA content are labeled G2/M phase and cells with intermediate DNA content are labeled S-phase. The number of hours post-transfection of mock or RPA2 siRNA is designated in the upper right corner of each histogram. (c) Microscopic examination of nuclei following mock or RPA2 depletion. Hours post-transfection are designated at the upper left of each image. Arrows designate fragmented and micronuclei formation.

Although RPA2 protein levels decreased by 70% at 48 h post-transfection, a definitive cell-cycle phenotype was not observed until 72–96 h (Figure 2b). RPA2 depletion resulted in a decrease in G1-phase cells and an increase in cells accumulating in early S-phase (Figure 2b), characterized by an unequal (asymmetric) distribution of S-phase cells. This indicates that depletion of RPA2 results in a replication defect. Another striking phenotype was an increase in cells with sub-G1 DNA content and reduced forward and side light-scattering properties after 96 h (Supplementary Figure S1a). These features are indicative of cell death in RPA2-depleted cells and consistent with the observation of the formation of fragmented and micronuclei at these times (Figure 2c). This suggests that depleted cells are undergoing apoptosis, and that RPA2 is essential for proliferating human cells.

We generated N-terminally tagged GFP-RPA2 and GFP-RPA4 constructs and transfected them into RPA2-depleted cells to examine RPA4 function in the cell. GFP-expressing cells were identified using flow cytometry (Figure 3a, left column; GFP-RPA2) and showed a cell-cycle distribution similar to mock-transfected cells (Figure 3a, right column). This confirmed that exogenous GFP-tagged RPA2 was expressed and was functional in the cell. We also expressed GFP-tagged RPA1 or RPA3 in RPA2-depleted cells and found that GFP-RPA1 expression was undetectable, while GFP-RPA3 was expressed at a slightly lower level (Supplementary Figure S1b). This suggests that RPA2 is necessary to stabilize RPA complex formation. These findings are consistent with previous studies showing that RPA2 and RPA3 are stable in the cell when RPA1 is depleted (14) and form a subcomplex in solution in the absence of RPA1 (24).
Figure 3.Rescue of RPA2 knockdown by exogenous RPA2 or RPA4 expression. (a) Cells were either mock- or RPA2 siRNA-transfected (designated above the dot-plots and histograms). After 24 h, the cells were transfected with either empty vector (EV) or with an RPA subunit-containing vector (designated in the upper right corner of each dot-plot and histogram). At 90 h post-transfection, cells were stained for flow cytometry, and exogenous RPA2- or RPA4-positive cells were identified based on their GFP expression (FL1-H fluorescence). The DNA content of GFP-positive (RPA2- or RPA4-expressing) cells (right box of each dot-plot) was plotted as a histogram. The DNA content of GFP-negative cells (left box of each dot-plot) was plotted for all samples transfected with EV. (b) Quantitation of the percentage of cells in G1, S and G2/M phases (denoted below bar graph) of the cell cycle. Error bars represent standard deviation. (c) Response to DNA damage. After 24 h transfection (where indicated) with siRNA, cells were either mock-transfected or transfected with indicated RPA subunit-containing vector (designated in upper right). Cells were treated as described in (a). Top row: no additional treatments, bottom row: cells incubated for final 24 h with 0.2 µM camptothecin (Cpt). DNA content of GPF-negative (mock) or GFP-positive cells (RPA2, RPA4) shown with percent of cells in S-phase indicated. (d) Localization of RPA2 or RPA4 to repair foci. Cells were treated as in (a) and DNA damage was induced by treatment with 2 µM camptothecin for 4 hr. Lower panels show RPA2 or RPA4 localization. For mock-transfected cells, RPA2 antibody (71-9A) was used to detect endogenous RPA2. For RPA2-depleted cells, GFP was detected for RPA2 or RPA4. Upper panels show corresponding nuclei stained with DAPI.

Rescue of RPA2 knockdown by exogenous RPA2 or RPA4 expression. (a) Cells were either mock- or RPA2 siRNA-transfected (designated above the dot-plots and histograms). After 24 h, the cells were transfected with either empty vector (EV) or with an RPA subunit-containing vector (designated in the upper right corner of each dot-plot and histogram). At 90 h post-transfection, cells were stained for flow cytometry, and exogenous RPA2- or RPA4-positive cells were identified based on their GFP expression (FL1-H fluorescence). The DNA content of GFP-positive (RPA2- or RPA4-expressing) cells (right box of each dot-plot) was plotted as a histogram. The DNA content of GFP-negative cells (left box of each dot-plot) was plotted for all samples transfected with EV. (b) Quantitation of the percentage of cells in G1, S and G2/M phases (denoted below bar graph) of the cell cycle. Error bars represent standard deviation. (c) Response to DNA damage. After 24 h transfection (where indicated) with siRNA, cells were either mock-transfected or transfected with indicated RPA subunit-containing vector (designated in upper right). Cells were treated as described in (a). Top row: no additional treatments, bottom row: cells incubated for final 24 h with 0.2 µM camptothecin (Cpt). DNA content of GPF-negative (mock) or GFP-positive cells (RPA2, RPA4) shown with percent of cells in S-phase indicated. (d) Localization of RPA2 or RPA4 to repair foci. Cells were treated as in (a) and DNA damage was induced by treatment with 2 µM camptothecin for 4 hr. Lower panels show RPA2 or RPA4 localization. For mock-transfected cells, RPA2 antibody (71-9A) was used to detect endogenous RPA2. For RPA2-depleted cells, GFP was detected for RPA2 or RPA4. Upper panels show corresponding nuclei stained with DAPI.

In contrast to RPA1, GFP-RPA4 was abundantly expressed in the absence of RPA2 (Figure 3a, left column, see also Figure 5b). RPA4-positive cells had a more symmetric distribution of S-phase cells (Figure 3a) than RPA2-depleted cells (asymmetric S-phase distribution); however, there was a detectable increase in S-phase cells compared to cells rescued with exogenous RPA2 (Figure 3b). The percentage of RPA4-expressing cells in S-phase was statistically higher than mock-transfected (RPA2-expressing) cells (Tukey–Kramer; q = 6.823; P < 0.001) and not statistically different from RPA2-depleted cells (Tukey–Kramer; q = 1.909; P > 0.05). In addition, RPA4-positive cells had a cell-cycle defect; they accumulated in G2/M (Figure 3a and b). This G2/M arrest was statistically significant compared to mock-transfected cells (Tukey–Kramer; q = 6.048; P < 0.01), and was alleviated upon addition of caffeine (not shown), a potent checkpoint inhibitor (25). DNA damaging agents caused cells expressing exogenous RPA2 or RPA4 to cell-cycle arrest (Figure 3c). Cell-cycle arrest was observed in either S- or G2/M-phase depending on the concentration of DNA damaging agent and length of exposure (Figure 3c and data not shown). The damage-dependent arrest was alleviated upon the addition of caffeine (not shown), demonstrating that it was the result of checkpoint activation. Cells depleted for RPA2 and treated with DNA damage showed the same early S-phase arrest observed in un-damaged cells. We conclude that the early S-phase arrest is a terminal phenotype of RPA2 depletion.
Figure 4.Co-localization of RPA forms with γ-H2AX or phosphorylated Chk2. For all panels, cells were treated with RPA2 siRNA and transfected with indicated GFP-RPA fusion expression plasmid [RPA2, RPA4, or yeast RPA2 (sc2)] as described in Figure 3. Cells were then either grown untreated or in the presence of 20 µM camptothecin (a and b) or 34 µM etoposide (c and d) for last 4 h. Cells were prepared for immuno-fluorescence as described in ‘Materials and Methods’ section and stained with (a) rabbit anti-γ-H2AX (phospho-Ser139) or (b) anti-phospho-Chk2 (Thr68; P-CHK2) antibodies (Cell Signaling Technology). Arrows indicated representative foci. Antibodies were used according to the manufacturer’s recommendations and detected using Texas Red-X goat anti-rabbit IgG (Invitrogen). Chromatin-associated RPA was visualized using intrinsic green fluorescence and nuclei (DNA) were visualized with DAPI staining. Flow cytometry samples examining total γ-H2AX (b) and phospho-Chk2 checkpoint activation (d) were prepared as described in ‘Materials and Methods’ section. RPA2-depleted cells expressing non-functional yeast RPA2 (green histogram), RPA2 (blue histogram), or RPA4 (red histogram) were stained with (b) anti-γ-H2AX Alexa Fluor 647 (Cell Signaling Technology) or (d) anti-phospho-Chk2 (as above) and R-phycoerythrin goat anti-rabbit IgG (Invitrogen). Fluorescence intensity shown on a log scale.
Figure 5.‘Cassettized’ RPA2 and complementation of RPA2 knockdown by exogenous RPA2/RPA4 ‘hybrid’ and ‘core’ constructs. (a) Release from aphidicolin block. Cells were depleted for RPA2, treated with aphidicolin as described in ‘Materials and Methods’ section. After either 0 or 8 h after release from aphidicolin cells were collected and examined by flow cytometry as described in Figure 2b. Black histograms represent RPA2-depleted cells, and red histograms represent exogenous RPA-expresssing cells. Construct expressed is designated above each set of histograms. Sc2, yeast RPA2. (b) Schematic of constructs. RPA2 is divided into three domains: phosphorylation domain (PD), DBD and the C-terminus containing the WHD. XhoI, AflII, SpeI and KpnI sites flank the domains (designated above the RPA2 schematic). RPA2 domains are blue and RPA4 domains are orange. Names of construct represent which PD, DBD, and WHD they contain and are designated to the right of each schematic. DBD-D, RPA2 DBD; DBD-G, RPA4 DBD; Native, natural forms; Hybrid, RPA2 and RPA4 combinations; Core, DBDs only. (c) Native and hybrid expression. GFP antibody was used to detect exogenous expression. Construct expressed is denoted above each lane; NV, no plasmid vector; position of GFP-RPA4 (RPA4), GFP-RPA2 (RPA2) are indicated by arrows; NS, non-specific band that migrates close to position of RPA4-note NV lane. The non-specific band overlaps with RPA4 and makes this band appear more intense. (d) Expression and rescue by hybrid constructs. Procedure and designations are as in Figure 3a. The percentage of cells in S-phase for each construct is shown in each histogram as the mean ± standard deviation. (e) Quantitation of cells in G2/M-phase from experiment shown in (d). Mock- or RPA2 siRNA-treatment is designated below graph. Construct is designated below each bar of graph. Error bars represent standard deviation.

Co-localization of RPA forms with γ-H2AX or phosphorylated Chk2. For all panels, cells were treated with RPA2 siRNA and transfected with indicated GFP-RPA fusion expression plasmid [RPA2, RPA4, or yeast RPA2 (sc2)] as described in Figure 3. Cells were then either grown untreated or in the presence of 20 µM camptothecin (a and b) or 34 µM etoposide (c and d) for last 4 h. Cells were prepared for immuno-fluorescence as described in ‘Materials and Methods’ section and stained with (a) rabbit anti-γ-H2AX (phospho-Ser139) or (b) anti-phospho-Chk2 (Thr68; P-CHK2) antibodies (Cell Signaling Technology). Arrows indicated representative foci. Antibodies were used according to the manufacturer’s recommendations and detected using Texas Red-X goat anti-rabbit IgG (Invitrogen). Chromatin-associated RPA was visualized using intrinsic green fluorescence and nuclei (DNA) were visualized with DAPI staining. Flow cytometry samples examining total γ-H2AX (b) and phospho-Chk2 checkpoint activation (d) were prepared as described in ‘Materials and Methods’ section. RPA2-depleted cells expressing non-functional yeast RPA2 (green histogram), RPA2 (blue histogram), or RPA4 (red histogram) were stained with (b) anti-γ-H2AX Alexa Fluor 647 (Cell Signaling Technology) or (d) anti-phospho-Chk2 (as above) and R-phycoerythrin goat anti-rabbit IgG (Invitrogen). Fluorescence intensity shown on a log scale.

‘Cassettized’ RPA2 and complementation of RPA2 knockdown by exogenous RPA2/RPA4 ‘hybrid’ and ‘core’ constructs. (a) Release from aphidicolin block. Cells were depleted for RPA2, treated with aphidicolin as described in ‘Materials and Methods’ section. After either 0 or 8 h after release from aphidicolin cells were collected and examined by flow cytometry as described in Figure 2b. Black histograms represent RPA2-depleted cells, and red histograms represent exogenous RPA-expresssing cells. Construct expressed is designated above each set of histograms. Sc2, yeast RPA2. (b) Schematic of constructs. RPA2 is divided into three domains: phosphorylation domain (PD), DBD and the C-terminus containing the WHD. XhoI, AflII, SpeI and KpnI sites flank the domains (designated above the RPA2 schematic). RPA2 domains are blue and RPA4 domains are orange. Names of construct represent which PD, DBD, and WHD they contain and are designated to the right of each schematic. DBD-D, RPA2 DBD; DBD-G, RPA4 DBD; Native, natural forms; Hybrid, RPA2 and RPA4 combinations; Core, DBDs only. (c) Native and hybrid expression. GFP antibody was used to detect exogenous expression. Construct expressed is denoted above each lane; NV, no plasmid vector; position of GFP-RPA4 (RPA4), GFP-RPA2 (RPA2) are indicated by arrows; NS, non-specific band that migrates close to position of RPA4-note NV lane. The non-specific band overlaps with RPA4 and makes this band appear more intense. (d) Expression and rescue by hybrid constructs. Procedure and designations are as in Figure 3a. The percentage of cells in S-phase for each construct is shown in each histogram as the mean ± standard deviation. (e) Quantitation of cells in G2/M-phase from experiment shown in (d). Mock- or RPA2 siRNA-treatment is designated below graph. Construct is designated below each bar of graph. Error bars represent standard deviation.

It has been shown that RPA2 localizes to sites of repair after DNA damage (16). This localization is observed in cells that have been exposed to DNA damage and extracted to visualize tightly associated, chromatin bound protein. To determine whether RPA4 has similar localization, cells expressing endogenous RPA2 or RPA2-depleted cells expressing GFP-RPA2 or GFP-RPA4 were treated with camptothecin. Figure 3d shows that like both endogenous and exogenous RPA2, RPA4 localized to foci after DNA damage. To confirm that the foci observed are sites of DNA repair, we examined co-localization with the phosphorylated forms of two proteins known to localize to sites of DNA in the DNA damage response: phosphorylated H2AX (γ-H2AX) and phosphorylated Chk2 (P-Chk2) (26,27). After DNA damage, both RPA2 and RPA4 co-localize to foci with γ-H2AX (Figure 4a) and with P-Chk2 (Figure 4c). The RPA2 homolog from yeast (S. cerevisae) was used as a control for these experiments. Yeast RPA2 (sc2) does not form a complex with human RPA subunits and was not observed to form foci (Figure 4). This demonstrates that localization is dependent on a human RPA2 isoform and is not an artifact of the expression system. In the absence of DNA damage, foci were not observed in cells expressing RPA2 or sc2. However, RPA4 and γ-H2AX foci but not P-Chk2 foci were observed in cells expressing RPA4 in the absence of DNA damage. This suggests RPA2-depleted cells expressing RPA4 contain abnormal DNA structures, but that these structures do not cause Chk2 activation.

The G2/M arrest and foci formation observed in RPA4-expressing cells in the absence of DNA damage could be caused by a deficiency in either replication or DNA repair leading to checkpoint activation (or both). To try to determine the checkpoint status in these cells, total γ-H2AX and P-Chk2 staining was examined by flow cytometry. Unstressed RPA2 expressing cells have low levels of γ-H2AX staining (left peak, Figure 4b). Unstressed cells expressing yeast RPA2 also mostly have low levels γ-H2AX staining cells; however, a small population of cells was observed that stain very strongly for γ-H2AX (Figure 4b). Since the cells examined by flow cytometry have not been extracted, these two populations cannot be directly compared to extracted cells observed by immunofluorescence. However, given that RPA2 (Figure 2c, Supplementary Figure S1a) or RPA1 depletion (14,23) appear to cause apoptosis, we propose that the high γ-H2AX staining cells indicate apoptotic DNA fragmentation with robust γ-H2AX staining (28). When RPA4 was expressed in unstressed cells, no high staining γ-H2AX cells were observed. This argues that expression of RPA4 prevents the formation of this population of cells. Most of the unstressed cells expressing RPA4 had low γ-H2AX staining but some cells were observed with slightly elevated γ-H2AX staining (Figure 4b). The later population is consistent with some γ-H2AX foci being observed in unstressed cells expressing RPA4.

After DNA damage, a population with high γ-H2AX staining was observed with all forms of exogenous protein consistent with a normal cellular DNA damage response (Figure 4b). With all three forms of exogenous RPA, we observed low levels of P-Chk2 in the absence of DNA damage and uniform, high levels of P-Chk2 after DNA damage (Figure 4d). Taken together with the γ-H2AX data, these results indicate that cells expressing RPA4 can support checkpoint activation and that expression of RPA4 can suppress cell death caused by RPA2 depletion. Further they indicate that RPA4-expressing cells contain some type of abnormal DNA structure that is sufficient to cause elevated γ-H2AX but not Chk2 activation (low P-Chk2). It is most likely that the abnormal DNA structure is triggering G2/M checkpoint activation observed in RPA4-expressing cells.

The RPA4-expressing cells had a similar percentage of cells in S-phase as RPA2 depleted cells (Figure 3b). This suggests that RPA4 may have a defect in DNA replication. To confirm these findings, we examined the ability of RPA4 to support DNA replication in synchronized cells. Cells were treated with the DNA polymerase inhibitor aphidicolin (APH) (29) to synchronize cells at the G1/S boundary. The cells were subsequently released into APH-free medium, and progression through S-phase was monitored. As expected, RPA2- and RPA2-depleted cells expressing a control, yeast RPA2 (Sc2), were unable to progress through S-phase (Figure 5a). Exogenous RPA2-expressing cells were proficient for DNA replication as indicated by a population of cells progressing through S-phase by 8 h after APH release (Figure 5a). In contrast, the RPA4-expressing cells were unable to progress through S-phase (Figure 5a). These results are consistent with the finding that an RPA complex containing RPA4 in place of RPA2 is unable to support SV40 DNA replication in vitro (13). We conclude that RPA4 is unable to support chromosomal DNA replication. We hypothesize that the γ-H2AX/RPA foci and the G2/M arrest observed in proliferating cells containing RPA4 is caused by incomplete chromosomal DNA replication.

RPA2 can be divided into three regions (Figure 5b): (i) the N-terminal phosphorylation domain (PD, aa 1–44), (ii) the central DBD-D (aa 45–172) and (iii) the C-terminal region (aa 173–270) containing a linker and a WHD. Amino acid sequence analysis revealed that RPA4 shares 47% amino acid sequence identity and 63% amino acid similarity to RPA2 and appears to have a similar domain organization. The similarity of the putative phosphorylation domain, putative DBD (DBD-G), and the putative WHD (without the linker) of RPA4 are 67, 71, 52%, respectively. To identify the region(s) of RPA4 responsible for the differences in activity, we generated a ‘cassettized’ GFP-RPA2 by inserting a unique restriction site immediately upstream and downstream of each region (Figure 5b). Each region of RPA2 was then substituted with the corresponding region of RPA4 producing three hybrid forms, called 422, 242 and 224, where numerals indicate the source of each of the three domains (Figure 5b).

The GFP-RPA2/RPA4 ‘hybrid’ constructs were expressed (Figure 5c) and examined in cells as described above. All hybrid constructs reduced the percentage of cells in S-phase compared to RPA2-depleted or RPA2-depleted/RPA4-expressing cells (Figure 5d). However, only the 242 hybrid construct containing DBD-G displayed the G2/M arrest phenotype (Figure 5d and e). We conclude that DBD-G is not able to substitute for DBD-D in RPA2 (i.e. DBD-D is essential for chromosomal DNA replication).

These studies using asynchronous cell cultures indicated that the hybrid constructs can rescue, to varying degrees, the replication defect due to RPA2 depletion, whereas RPA4 expression cannot. We next tested each hybrid in cells synchronized with aphidicolin. The hybrid constructs supported S-phase progression to different extents: 422-expressing cells entered S-phase as efficiently as RPA2-expressing cells, 224-expressing cells entered S-phase (but less efficiently) and very few 242-expressing cells were able enter S-phase (Figure 5a). We conclude that the putative phosphorylation domain of RPA4 has no effect on replication, the C-terminus of RPA4 affects the efficiency of replication, and that the DBD of RPA2 is essential for cellular replication, since only hybrid 242, which contains the putative DBD of RPA4, completely prevented replication. These results are consistent with the previous finding that the DBD of RPA2 is the only essential domain required for cell viability in unstressed yeast cells (30,31). The replication defect cannot be directly attributed to differences in DNA-binding affinities, because Mason et al. (13) demonstrated that the alternative RPA complex (containing RPA4) binds ssDNA similarly to wild-type RPA (containing RPA2).

Only hybrid 242 caused G2/M arrest similar to RPA4 (Figure 5d and e). To explore a possible cause for this G2/M arrest, we examined the ability of the hybrid constructs to participate in genome maintenance by examining γ-H2AX staining. All hybrid constructs had γ-H2AX staining similar to RPA4: no high γ-H2AX staining cells were observed. In addition, all hybrids were localized to foci after DNA damage (Supplementary Figure S2). This suggests that all of the domains of RPA4 can substitute for the comparable domain of RPA2 in these cellular processes.

We also measured whether RPA2 DBD-D is sufficient for replication in human cells and if RPA4 DBD-G is sufficient for G2/M arrest using ‘core’ constructs (Figure 5b). Following RPA2 depletion, cells expressing DBD-D progressed through S-phase, although the efficiency of progression was reduced compared to exogenous RPA2 (Supplemental Figure S3). In contrast, cells expressing DBD-G showed a dramatic reduction in progression through S-phase, similar to RPA4 (Supplemental Figure S3). These studies do not rule out DBD-G supporting a low level of DNA synthesis but demonstrate that DBD-G activity is similar to RPA4 while DBD-D is closer to RPA2. The DBD is the domain most highly conserved between RPA2 and RPA4 (Figure 1), yet one, DBD-D, is functional for replication and the other, DBD-G, is defective for replication and causes unstressed cells to arrest in G2/M.

There is a short region, noted previously (12), where RPA2 contains an acidic stretch of amino acids not found in RPA4 (Figure 6a). The corresponding stretch of amino acids (aa 108–123) in RPA4 contains a number of basic residues (Figure 6a). In our studies of ‘core’ DBD constructs (Figure 5a and d), we observed ∼5 kDa size difference in the mobility of RPA2 DBD-D and RPA4 DBD-G polypeptides, consistent with substantial differences in charge in this region (Figure 6b). This region lies in the 3–4 loop (L34) of the known DBD-D structure and is postulated to be flexible, as it only appears in one of nine known structures of RPA2 (18). Protein modeling revealed that although predicted structures of DBD-G are similar to DBD-D, the predicted electrostatic surface potentials of the L34 region of these two domains are very different (Figure 6b).
Figure 6.Examination of basic/acidic constructs. (a) Alignment of loop3-4 (L34) regions of RPA2 and RPA4. The L34 region lies in the DBD of each protein. Amino acid position is designated to the left and right of the sequence. Identical residues are denoted by asterisk, and the positively and negatively charged residues that differ between RPA2 and RPA4 are denoted with − or +. (b) Protein modeling and predicted electrostatic surface potential for RPA2 and RPA4 DBDs. The known structure of DBD-D (18) is shown to the left, and one model of DBD-G (Geno3D) is shown to the right. Important features are denoted as follows: L12, loop between β-sheets 1 and 2; L34, loop between β-sheets 3 and 4; L45, loop between β-sheets 4 and 5; cleft, putative DNA-binding cleft. For electrostatic surface potential: red, acidic; blue, basic. Western blot of expression is shown in the middle and constructs are denoted above each lane and identified to the left and right of the blot (G-RPA or G-DBD indicates GFP-tagged protein). NS, non-specific bands. (c) Cell-cycle phenotypes for exogenous basic/acidic constructs. Cells were transfected and examined at 96 h. post-transfection as in Figure 3a. The percentage of cell in G2/M phase is shown in each histogram as the mean ± standard deviation. (d) Aphidicolin block and release of cells containing 2Basic and 4Acidic. Data generated in same experiment shown in Figure 5a: cells were treated and designated as described in Figure 5a. (e) Dominant negative analysis. In the presence of endogenous RPA2, exogenous RPA2, RPA4 and 2Basic were expressed. Overlays of the DNA histograms are shown. Black histograms represent cells with no vector and red histograms represent cells expressing indicted RPA form.

Examination of basic/acidic constructs. (a) Alignment of loop3-4 (L34) regions of RPA2 and RPA4. The L34 region lies in the DBD of each protein. Amino acid position is designated to the left and right of the sequence. Identical residues are denoted by asterisk, and the positively and negatively charged residues that differ between RPA2 and RPA4 are denoted with − or +. (b) Protein modeling and predicted electrostatic surface potential for RPA2 and RPA4 DBDs. The known structure of DBD-D (18) is shown to the left, and one model of DBD-G (Geno3D) is shown to the right. Important features are denoted as follows: L12, loop between β-sheets 1 and 2; L34, loop between β-sheets 3 and 4; L45, loop between β-sheets 4 and 5; cleft, putative DNA-binding cleft. For electrostatic surface potential: red, acidic; blue, basic. Western blot of expression is shown in the middle and constructs are denoted above each lane and identified to the left and right of the blot (G-RPA or G-DBD indicates GFP-tagged protein). NS, non-specific bands. (c) Cell-cycle phenotypes for exogenous basic/acidic constructs. Cells were transfected and examined at 96 h. post-transfection as in Figure 3a. The percentage of cell in G2/M phase is shown in each histogram as the mean ± standard deviation. (d) Aphidicolin block and release of cells containing 2Basic and 4Acidic. Data generated in same experiment shown in Figure 5a: cells were treated and designated as described in Figure 5a. (e) Dominant negative analysis. In the presence of endogenous RPA2, exogenous RPA2, RPA4 and 2Basic were expressed. Overlays of the DNA histograms are shown. Black histograms represent cells with no vector and red histograms represent cells expressing indicted RPA form.

Using site-directed mutagenesis, we generated a form of GFP-RPA2 where the acidic L34 loop was replaced with the basic L34 loop from RPA4 (called 2Basic) and a form of GFP-RPA4 where the basic L34 was replaced with the acidic L34 loop from RPA2 (called 4Acidic). Exogenous expression of 2Basic in RPA2-depleted HeLa cells resulted in G2/M arrest, similar to, but more pronounced than, exogenous expression of RPA4 (Figure 6c, see also Figure 5e). Conversely, 4Acidic expression resulted in percentages of G2/M cells more similar to RPA2-expressing cells (Figure 6c). This suggests that L34 is critical for RPA2 function, and that the RPA4 basic loop is both necessary and sufficient for the arrest phenotype observed for constructs containing DBD-G. S-phase progression was also examined using synchronized cells. Neither 2Basic nor 4Acidic could support S-phase progression (Figure 6d) indicating that L34 from RPA4 is sufficient to inactivate RPA2, but that the corresponding loop from RPA2 does not restore function to RPA4.

Since RPA4 is deficient for DNA replication and proficient for G2/M arrest, we tested whether or not exogenous RPA4 expression could inhibit endogenous RPA2 function. Cells were transfected with GFP-RPA2 or GFP-RPA4 without depletion of endogenous RPA2. One would predict a dominant negative phenotype to result in an increase in S-phase cells (replication deficiency), a G2/M checkpoint (arrest proficiency), or both. We were unable to detect an appreciable increase in S- or G2/M-phase after exogenous RPA4 expression (Figure 6e). However, expression of 2Basic, which contains the L34 region (only 16 aa) of RPA4, caused a detectable increase in G2/M arrested cells (Figure 6e). This is consistent with 2Basic having a stronger phenotype in RPA2-depleted cells than RPA4. The absence of a dominant negative phenotype with RPA4 could be the result of expression levels, protein stability, or complex preference. Recently, RPA complexes containing 2Basic or RPA4 (but not 4Acidic) have been shown to inhibit wild-type RPA function in SV40 replication in vitro (13). Together these findings suggest that 2Basic, and probably RPA4, can inhibit normal RPA2 functions in cells when expressed at high enough levels, and that the basic L34 region of RPA4 is sufficient for this inhibition.

### Genomic analysis of RPA4 sequences

RPA4 was identified prior to the sequencing of the human genome. With sequencing and annotation of the human and other genomes, it was possible to analyze the RPA4 gene and examine its evolutionary conservation. This analysis identified three interesting features of the RPA4 gene. First, RPA4 is intronless, suggesting that RPA4 arose from a viral- or retrotransposon-mediated gene duplication event. Second, RPA4 resides on the X-chromosome at position q21.33. Third, RPA4 lies in the intron of a known coding gene, diaphanous 2 (DIAPH2). DIAPH2 encodes a formin-related actin-binding protein (21,22). Expression of RPA4 is not well characterized, but available public data indicates that RPA4 is expressed in different tissues than DIAPH2 (http://genome.ucsc.edu/), indicating that it is independently regulated.

The RPA4-related sequences are only found in mammals. Primates (human, chimpanzee, orangutan, monkey and marmoset) and horse contain complete coding sequences for RPA4 region (Figure 1). Marmoset RPA4 is the only gene not located on the X-chromosome. Other mammals examined (e.g. cow, elephant, dog, cat, rabbit, mouse, rat and armadillo) have partial RPA4-related sequences or a RPA4 pseudogene at the equivalent position of the X-chromosome. We are unable to identify RPA4-related sequences in any non-mammalian genome, and conclude that functional RPA4 genes are found only in primates and a few other mammals.
Figure 1.Alignment of known and putative RPA4 and RPA2 from six mammalian species. RPA4 and RPA2 protein sequences are from Homo sapiens (Hosa) (NP037479, NP002937) and the predicted sequences from Pan troglodytes (Patr) (XP001136507, XP513251), Pongo abelii (Poab) (CAH90667), Callithrix jacchus (Caja) (RPA4 protein derived from DNA Contig10151:1273-53091; RPA2 protein derived from DNA Contig150:1594-1135700), Macaca mullata (Mamu) (XP001088106) and Equus caballus (Eqca) (XP001491922, XP001500549). Amino acid positions are denoted at the beginning and end of each line of sequence. Alignment was performed and identical (*), conserved (:), and semiconserved (.) residues are denoted below each alignment by TCOFFEE. Grey and black shaded residues denote regions of least conservation or major charge difference between RPA2 and RPA4 noted in Keshav et al. (12). Black shaded residues also denote amino acids swapped between RPA2 and RPA4 in our studies. The N-termini containing the putative phosphorylation domain are denoted with lowercase letters, the OB-fold DBDs are identified by UPPERCASE letters, and the C-termini are denoted with lowercase italics.

Alignment of known and putative RPA4 and RPA2 from six mammalian species. RPA4 and RPA2 protein sequences are from Homo sapiens (Hosa) (NP037479, NP002937) and the predicted sequences from Pan troglodytes (Patr) (XP001136507, XP513251), Pongo abelii (Poab) (CAH90667), Callithrix jacchus (Caja) (RPA4 protein derived from DNA Contig10151:1273-53091; RPA2 protein derived from DNA Contig150:1594-1135700), Macaca mullata (Mamu) (XP001088106) and Equus caballus (Eqca) (XP001491922, XP001500549). Amino acid positions are denoted at the beginning and end of each line of sequence. Alignment was performed and identical (*), conserved (:), and semiconserved (.) residues are denoted below each alignment by TCOFFEE. Grey and black shaded residues denote regions of least conservation or major charge difference between RPA2 and RPA4 noted in Keshav et al. (12). Black shaded residues also denote amino acids swapped between RPA2 and RPA4 in our studies. The N-termini containing the putative phosphorylation domain are denoted with lowercase letters, the OB-fold DBDs are identified by UPPERCASE letters, and the C-termini are denoted with lowercase italics.

### Rescue of RPA2 depleted cells by exogenous RPA subunit expression

To determine the function of RPA4 in cells, we used a knockdown-replacement strategy. A short interfering RNA (siRNA) was used to target the 3′ untranslated region of RPA2 mRNA for degradation in HeLa cells. A decrease in RPA2 protein was observed at 48 h post-transfection of siRNA; only ∼30% of RPA2 remained in the cells (Figure 2a). No change in RPA2 levels was observed in mock treated cells and in cells transfected with a non-specific control siRNA (Figure 2a and data not shown). Maximal knockdown of RPA2 occurred between 72 and 96 h post-transfection and resulted in coordinate depletion of RPA1 protein [Figure 2a; as previously observed in ref. (23)].
Figure 2.Knockdown of endogenous RPA2. (a) Examination of mock and RPA2 siRNA knockdown by western blot showing RPA2 and RPA1 depletion. Number of hours post-transfection of siRNA are denoted above each lane. R, 200 ng purified recombinant human RPA. (b) Cell-cycle analysis of mock or RPA2 siRNA knockdown. DNA content was analyzed by flow cytometry and plotted as number of cells versus DNA content (FL2-A fluorescence). Cells with unreplicated DNA content are labeled as G1 phase, cells with replicated DNA content are labeled G2/M phase and cells with intermediate DNA content are labeled S-phase. The number of hours post-transfection of mock or RPA2 siRNA is designated in the upper right corner of each histogram. (c) Microscopic examination of nuclei following mock or RPA2 depletion. Hours post-transfection are designated at the upper left of each image. Arrows designate fragmented and micronuclei formation.

Knockdown of endogenous RPA2. (a) Examination of mock and RPA2 siRNA knockdown by western blot showing RPA2 and RPA1 depletion. Number of hours post-transfection of siRNA are denoted above each lane. R, 200 ng purified recombinant human RPA. (b) Cell-cycle analysis of mock or RPA2 siRNA knockdown. DNA content was analyzed by flow cytometry and plotted as number of cells versus DNA content (FL2-A fluorescence). Cells with unreplicated DNA content are labeled as G1 phase, cells with replicated DNA content are labeled G2/M phase and cells with intermediate DNA content are labeled S-phase. The number of hours post-transfection of mock or RPA2 siRNA is designated in the upper right corner of each histogram. (c) Microscopic examination of nuclei following mock or RPA2 depletion. Hours post-transfection are designated at the upper left of each image. Arrows designate fragmented and micronuclei formation.

Although RPA2 protein levels decreased by 70% at 48 h post-transfection, a definitive cell-cycle phenotype was not observed until 72–96 h (Figure 2b). RPA2 depletion resulted in a decrease in G1-phase cells and an increase in cells accumulating in early S-phase (Figure 2b), characterized by an unequal (asymmetric) distribution of S-phase cells. This indicates that depletion of RPA2 results in a replication defect. Another striking phenotype was an increase in cells with sub-G1 DNA content and reduced forward and side light-scattering properties after 96 h (Supplementary Figure S1a). These features are indicative of cell death in RPA2-depleted cells and consistent with the observation of the formation of fragmented and micronuclei at these times (Figure 2c). This suggests that depleted cells are undergoing apoptosis, and that RPA2 is essential for proliferating human cells.

We generated N-terminally tagged GFP-RPA2 and GFP-RPA4 constructs and transfected them into RPA2-depleted cells to examine RPA4 function in the cell. GFP-expressing cells were identified using flow cytometry (Figure 3a, left column; GFP-RPA2) and showed a cell-cycle distribution similar to mock-transfected cells (Figure 3a, right column). This confirmed that exogenous GFP-tagged RPA2 was expressed and was functional in the cell. We also expressed GFP-tagged RPA1 or RPA3 in RPA2-depleted cells and found that GFP-RPA1 expression was undetectable, while GFP-RPA3 was expressed at a slightly lower level (Supplementary Figure S1b). This suggests that RPA2 is necessary to stabilize RPA complex formation. These findings are consistent with previous studies showing that RPA2 and RPA3 are stable in the cell when RPA1 is depleted (14) and form a subcomplex in solution in the absence of RPA1 (24).
Figure 3.Rescue of RPA2 knockdown by exogenous RPA2 or RPA4 expression. (a) Cells were either mock- or RPA2 siRNA-transfected (designated above the dot-plots and histograms). After 24 h, the cells were transfected with either empty vector (EV) or with an RPA subunit-containing vector (designated in the upper right corner of each dot-plot and histogram). At 90 h post-transfection, cells were stained for flow cytometry, and exogenous RPA2- or RPA4-positive cells were identified based on their GFP expression (FL1-H fluorescence). The DNA content of GFP-positive (RPA2- or RPA4-expressing) cells (right box of each dot-plot) was plotted as a histogram. The DNA content of GFP-negative cells (left box of each dot-plot) was plotted for all samples transfected with EV. (b) Quantitation of the percentage of cells in G1, S and G2/M phases (denoted below bar graph) of the cell cycle. Error bars represent standard deviation. (c) Response to DNA damage. After 24 h transfection (where indicated) with siRNA, cells were either mock-transfected or transfected with indicated RPA subunit-containing vector (designated in upper right). Cells were treated as described in (a). Top row: no additional treatments, bottom row: cells incubated for final 24 h with 0.2 µM camptothecin (Cpt). DNA content of GPF-negative (mock) or GFP-positive cells (RPA2, RPA4) shown with percent of cells in S-phase indicated. (d) Localization of RPA2 or RPA4 to repair foci. Cells were treated as in (a) and DNA damage was induced by treatment with 2 µM camptothecin for 4 hr. Lower panels show RPA2 or RPA4 localization. For mock-transfected cells, RPA2 antibody (71-9A) was used to detect endogenous RPA2. For RPA2-depleted cells, GFP was detected for RPA2 or RPA4. Upper panels show corresponding nuclei stained with DAPI.

Rescue of RPA2 knockdown by exogenous RPA2 or RPA4 expression. (a) Cells were either mock- or RPA2 siRNA-transfected (designated above the dot-plots and histograms). After 24 h, the cells were transfected with either empty vector (EV) or with an RPA subunit-containing vector (designated in the upper right corner of each dot-plot and histogram). At 90 h post-transfection, cells were stained for flow cytometry, and exogenous RPA2- or RPA4-positive cells were identified based on their GFP expression (FL1-H fluorescence). The DNA content of GFP-positive (RPA2- or RPA4-expressing) cells (right box of each dot-plot) was plotted as a histogram. The DNA content of GFP-negative cells (left box of each dot-plot) was plotted for all samples transfected with EV. (b) Quantitation of the percentage of cells in G1, S and G2/M phases (denoted below bar graph) of the cell cycle. Error bars represent standard deviation. (c) Response to DNA damage. After 24 h transfection (where indicated) with siRNA, cells were either mock-transfected or transfected with indicated RPA subunit-containing vector (designated in upper right). Cells were treated as described in (a). Top row: no additional treatments, bottom row: cells incubated for final 24 h with 0.2 µM camptothecin (Cpt). DNA content of GPF-negative (mock) or GFP-positive cells (RPA2, RPA4) shown with percent of cells in S-phase indicated. (d) Localization of RPA2 or RPA4 to repair foci. Cells were treated as in (a) and DNA damage was induced by treatment with 2 µM camptothecin for 4 hr. Lower panels show RPA2 or RPA4 localization. For mock-transfected cells, RPA2 antibody (71-9A) was used to detect endogenous RPA2. For RPA2-depleted cells, GFP was detected for RPA2 or RPA4. Upper panels show corresponding nuclei stained with DAPI.

In contrast to RPA1, GFP-RPA4 was abundantly expressed in the absence of RPA2 (Figure 3a, left column, see also Figure 5b). RPA4-positive cells had a more symmetric distribution of S-phase cells (Figure 3a) than RPA2-depleted cells (asymmetric S-phase distribution); however, there was a detectable increase in S-phase cells compared to cells rescued with exogenous RPA2 (Figure 3b). The percentage of RPA4-expressing cells in S-phase was statistically higher than mock-transfected (RPA2-expressing) cells (Tukey–Kramer; q = 6.823; P < 0.001) and not statistically different from RPA2-depleted cells (Tukey–Kramer; q = 1.909; P > 0.05). In addition, RPA4-positive cells had a cell-cycle defect; they accumulated in G2/M (Figure 3a and b). This G2/M arrest was statistically significant compared to mock-transfected cells (Tukey–Kramer; q = 6.048; P < 0.01), and was alleviated upon addition of caffeine (not shown), a potent checkpoint inhibitor (25). DNA damaging agents caused cells expressing exogenous RPA2 or RPA4 to cell-cycle arrest (Figure 3c). Cell-cycle arrest was observed in either S- or G2/M-phase depending on the concentration of DNA damaging agent and length of exposure (Figure 3c and data not shown). The damage-dependent arrest was alleviated upon the addition of caffeine (not shown), demonstrating that it was the result of checkpoint activation. Cells depleted for RPA2 and treated with DNA damage showed the same early S-phase arrest observed in un-damaged cells. We conclude that the early S-phase arrest is a terminal phenotype of RPA2 depletion.
Figure 4.Co-localization of RPA forms with γ-H2AX or phosphorylated Chk2. For all panels, cells were treated with RPA2 siRNA and transfected with indicated GFP-RPA fusion expression plasmid [RPA2, RPA4, or yeast RPA2 (sc2)] as described in Figure 3. Cells were then either grown untreated or in the presence of 20 µM camptothecin (a and b) or 34 µM etoposide (c and d) for last 4 h. Cells were prepared for immuno-fluorescence as described in ‘Materials and Methods’ section and stained with (a) rabbit anti-γ-H2AX (phospho-Ser139) or (b) anti-phospho-Chk2 (Thr68; P-CHK2) antibodies (Cell Signaling Technology). Arrows indicated representative foci. Antibodies were used according to the manufacturer’s recommendations and detected using Texas Red-X goat anti-rabbit IgG (Invitrogen). Chromatin-associated RPA was visualized using intrinsic green fluorescence and nuclei (DNA) were visualized with DAPI staining. Flow cytometry samples examining total γ-H2AX (b) and phospho-Chk2 checkpoint activation (d) were prepared as described in ‘Materials and Methods’ section. RPA2-depleted cells expressing non-functional yeast RPA2 (green histogram), RPA2 (blue histogram), or RPA4 (red histogram) were stained with (b) anti-γ-H2AX Alexa Fluor 647 (Cell Signaling Technology) or (d) anti-phospho-Chk2 (as above) and R-phycoerythrin goat anti-rabbit IgG (Invitrogen). Fluorescence intensity shown on a log scale.
Figure 5.‘Cassettized’ RPA2 and complementation of RPA2 knockdown by exogenous RPA2/RPA4 ‘hybrid’ and ‘core’ constructs. (a) Release from aphidicolin block. Cells were depleted for RPA2, treated with aphidicolin as described in ‘Materials and Methods’ section. After either 0 or 8 h after release from aphidicolin cells were collected and examined by flow cytometry as described in Figure 2b. Black histograms represent RPA2-depleted cells, and red histograms represent exogenous RPA-expresssing cells. Construct expressed is designated above each set of histograms. Sc2, yeast RPA2. (b) Schematic of constructs. RPA2 is divided into three domains: phosphorylation domain (PD), DBD and the C-terminus containing the WHD. XhoI, AflII, SpeI and KpnI sites flank the domains (designated above the RPA2 schematic). RPA2 domains are blue and RPA4 domains are orange. Names of construct represent which PD, DBD, and WHD they contain and are designated to the right of each schematic. DBD-D, RPA2 DBD; DBD-G, RPA4 DBD; Native, natural forms; Hybrid, RPA2 and RPA4 combinations; Core, DBDs only. (c) Native and hybrid expression. GFP antibody was used to detect exogenous expression. Construct expressed is denoted above each lane; NV, no plasmid vector; position of GFP-RPA4 (RPA4), GFP-RPA2 (RPA2) are indicated by arrows; NS, non-specific band that migrates close to position of RPA4-note NV lane. The non-specific band overlaps with RPA4 and makes this band appear more intense. (d) Expression and rescue by hybrid constructs. Procedure and designations are as in Figure 3a. The percentage of cells in S-phase for each construct is shown in each histogram as the mean ± standard deviation. (e) Quantitation of cells in G2/M-phase from experiment shown in (d). Mock- or RPA2 siRNA-treatment is designated below graph. Construct is designated below each bar of graph. Error bars represent standard deviation.

Co-localization of RPA forms with γ-H2AX or phosphorylated Chk2. For all panels, cells were treated with RPA2 siRNA and transfected with indicated GFP-RPA fusion expression plasmid [RPA2, RPA4, or yeast RPA2 (sc2)] as described in Figure 3. Cells were then either grown untreated or in the presence of 20 µM camptothecin (a and b) or 34 µM etoposide (c and d) for last 4 h. Cells were prepared for immuno-fluorescence as described in ‘Materials and Methods’ section and stained with (a) rabbit anti-γ-H2AX (phospho-Ser139) or (b) anti-phospho-Chk2 (Thr68; P-CHK2) antibodies (Cell Signaling Technology). Arrows indicated representative foci. Antibodies were used according to the manufacturer’s recommendations and detected using Texas Red-X goat anti-rabbit IgG (Invitrogen). Chromatin-associated RPA was visualized using intrinsic green fluorescence and nuclei (DNA) were visualized with DAPI staining. Flow cytometry samples examining total γ-H2AX (b) and phospho-Chk2 checkpoint activation (d) were prepared as described in ‘Materials and Methods’ section. RPA2-depleted cells expressing non-functional yeast RPA2 (green histogram), RPA2 (blue histogram), or RPA4 (red histogram) were stained with (b) anti-γ-H2AX Alexa Fluor 647 (Cell Signaling Technology) or (d) anti-phospho-Chk2 (as above) and R-phycoerythrin goat anti-rabbit IgG (Invitrogen). Fluorescence intensity shown on a log scale.

‘Cassettized’ RPA2 and complementation of RPA2 knockdown by exogenous RPA2/RPA4 ‘hybrid’ and ‘core’ constructs. (a) Release from aphidicolin block. Cells were depleted for RPA2, treated with aphidicolin as described in ‘Materials and Methods’ section. After either 0 or 8 h after release from aphidicolin cells were collected and examined by flow cytometry as described in Figure 2b. Black histograms represent RPA2-depleted cells, and red histograms represent exogenous RPA-expresssing cells. Construct expressed is designated above each set of histograms. Sc2, yeast RPA2. (b) Schematic of constructs. RPA2 is divided into three domains: phosphorylation domain (PD), DBD and the C-terminus containing the WHD. XhoI, AflII, SpeI and KpnI sites flank the domains (designated above the RPA2 schematic). RPA2 domains are blue and RPA4 domains are orange. Names of construct represent which PD, DBD, and WHD they contain and are designated to the right of each schematic. DBD-D, RPA2 DBD; DBD-G, RPA4 DBD; Native, natural forms; Hybrid, RPA2 and RPA4 combinations; Core, DBDs only. (c) Native and hybrid expression. GFP antibody was used to detect exogenous expression. Construct expressed is denoted above each lane; NV, no plasmid vector; position of GFP-RPA4 (RPA4), GFP-RPA2 (RPA2) are indicated by arrows; NS, non-specific band that migrates close to position of RPA4-note NV lane. The non-specific band overlaps with RPA4 and makes this band appear more intense. (d) Expression and rescue by hybrid constructs. Procedure and designations are as in Figure 3a. The percentage of cells in S-phase for each construct is shown in each histogram as the mean ± standard deviation. (e) Quantitation of cells in G2/M-phase from experiment shown in (d). Mock- or RPA2 siRNA-treatment is designated below graph. Construct is designated below each bar of graph. Error bars represent standard deviation.

### RPA4 function in DNA repair and apoptosis

It has been shown that RPA2 localizes to sites of repair after DNA damage (16). This localization is observed in cells that have been exposed to DNA damage and extracted to visualize tightly associated, chromatin bound protein. To determine whether RPA4 has similar localization, cells expressing endogenous RPA2 or RPA2-depleted cells expressing GFP-RPA2 or GFP-RPA4 were treated with camptothecin. Figure 3d shows that like both endogenous and exogenous RPA2, RPA4 localized to foci after DNA damage. To confirm that the foci observed are sites of DNA repair, we examined co-localization with the phosphorylated forms of two proteins known to localize to sites of DNA in the DNA damage response: phosphorylated H2AX (γ-H2AX) and phosphorylated Chk2 (P-Chk2) (26,27). After DNA damage, both RPA2 and RPA4 co-localize to foci with γ-H2AX (Figure 4a) and with P-Chk2 (Figure 4c). The RPA2 homolog from yeast (S. cerevisae) was used as a control for these experiments. Yeast RPA2 (sc2) does not form a complex with human RPA subunits and was not observed to form foci (Figure 4). This demonstrates that localization is dependent on a human RPA2 isoform and is not an artifact of the expression system. In the absence of DNA damage, foci were not observed in cells expressing RPA2 or sc2. However, RPA4 and γ-H2AX foci but not P-Chk2 foci were observed in cells expressing RPA4 in the absence of DNA damage. This suggests RPA2-depleted cells expressing RPA4 contain abnormal DNA structures, but that these structures do not cause Chk2 activation.

The G2/M arrest and foci formation observed in RPA4-expressing cells in the absence of DNA damage could be caused by a deficiency in either replication or DNA repair leading to checkpoint activation (or both). To try to determine the checkpoint status in these cells, total γ-H2AX and P-Chk2 staining was examined by flow cytometry. Unstressed RPA2 expressing cells have low levels of γ-H2AX staining (left peak, Figure 4b). Unstressed cells expressing yeast RPA2 also mostly have low levels γ-H2AX staining cells; however, a small population of cells was observed that stain very strongly for γ-H2AX (Figure 4b). Since the cells examined by flow cytometry have not been extracted, these two populations cannot be directly compared to extracted cells observed by immunofluorescence. However, given that RPA2 (Figure 2c, Supplementary Figure S1a) or RPA1 depletion (14,23) appear to cause apoptosis, we propose that the high γ-H2AX staining cells indicate apoptotic DNA fragmentation with robust γ-H2AX staining (28). When RPA4 was expressed in unstressed cells, no high staining γ-H2AX cells were observed. This argues that expression of RPA4 prevents the formation of this population of cells. Most of the unstressed cells expressing RPA4 had low γ-H2AX staining but some cells were observed with slightly elevated γ-H2AX staining (Figure 4b). The later population is consistent with some γ-H2AX foci being observed in unstressed cells expressing RPA4.

After DNA damage, a population with high γ-H2AX staining was observed with all forms of exogenous protein consistent with a normal cellular DNA damage response (Figure 4b). With all three forms of exogenous RPA, we observed low levels of P-Chk2 in the absence of DNA damage and uniform, high levels of P-Chk2 after DNA damage (Figure 4d). Taken together with the γ-H2AX data, these results indicate that cells expressing RPA4 can support checkpoint activation and that expression of RPA4 can suppress cell death caused by RPA2 depletion. Further they indicate that RPA4-expressing cells contain some type of abnormal DNA structure that is sufficient to cause elevated γ-H2AX but not Chk2 activation (low P-Chk2). It is most likely that the abnormal DNA structure is triggering G2/M checkpoint activation observed in RPA4-expressing cells.

### RPA4 function DNA replication

The RPA4-expressing cells had a similar percentage of cells in S-phase as RPA2 depleted cells (Figure 3b). This suggests that RPA4 may have a defect in DNA replication. To confirm these findings, we examined the ability of RPA4 to support DNA replication in synchronized cells. Cells were treated with the DNA polymerase inhibitor aphidicolin (APH) (29) to synchronize cells at the G1/S boundary. The cells were subsequently released into APH-free medium, and progression through S-phase was monitored. As expected, RPA2- and RPA2-depleted cells expressing a control, yeast RPA2 (Sc2), were unable to progress through S-phase (Figure 5a). Exogenous RPA2-expressing cells were proficient for DNA replication as indicated by a population of cells progressing through S-phase by 8 h after APH release (Figure 5a). In contrast, the RPA4-expressing cells were unable to progress through S-phase (Figure 5a). These results are consistent with the finding that an RPA complex containing RPA4 in place of RPA2 is unable to support SV40 DNA replication in vitro (13). We conclude that RPA4 is unable to support chromosomal DNA replication. We hypothesize that the γ-H2AX/RPA foci and the G2/M arrest observed in proliferating cells containing RPA4 is caused by incomplete chromosomal DNA replication.

### Identifying the region responsible for RPA4 phenotypes

RPA2 can be divided into three regions (Figure 5b): (i) the N-terminal phosphorylation domain (PD, aa 1–44), (ii) the central DBD-D (aa 45–172) and (iii) the C-terminal region (aa 173–270) containing a linker and a WHD. Amino acid sequence analysis revealed that RPA4 shares 47% amino acid sequence identity and 63% amino acid similarity to RPA2 and appears to have a similar domain organization. The similarity of the putative phosphorylation domain, putative DBD (DBD-G), and the putative WHD (without the linker) of RPA4 are 67, 71, 52%, respectively. To identify the region(s) of RPA4 responsible for the differences in activity, we generated a ‘cassettized’ GFP-RPA2 by inserting a unique restriction site immediately upstream and downstream of each region (Figure 5b). Each region of RPA2 was then substituted with the corresponding region of RPA4 producing three hybrid forms, called 422, 242 and 224, where numerals indicate the source of each of the three domains (Figure 5b).

The GFP-RPA2/RPA4 ‘hybrid’ constructs were expressed (Figure 5c) and examined in cells as described above. All hybrid constructs reduced the percentage of cells in S-phase compared to RPA2-depleted or RPA2-depleted/RPA4-expressing cells (Figure 5d). However, only the 242 hybrid construct containing DBD-G displayed the G2/M arrest phenotype (Figure 5d and e). We conclude that DBD-G is not able to substitute for DBD-D in RPA2 (i.e. DBD-D is essential for chromosomal DNA replication).

These studies using asynchronous cell cultures indicated that the hybrid constructs can rescue, to varying degrees, the replication defect due to RPA2 depletion, whereas RPA4 expression cannot. We next tested each hybrid in cells synchronized with aphidicolin. The hybrid constructs supported S-phase progression to different extents: 422-expressing cells entered S-phase as efficiently as RPA2-expressing cells, 224-expressing cells entered S-phase (but less efficiently) and very few 242-expressing cells were able enter S-phase (Figure 5a). We conclude that the putative phosphorylation domain of RPA4 has no effect on replication, the C-terminus of RPA4 affects the efficiency of replication, and that the DBD of RPA2 is essential for cellular replication, since only hybrid 242, which contains the putative DBD of RPA4, completely prevented replication. These results are consistent with the previous finding that the DBD of RPA2 is the only essential domain required for cell viability in unstressed yeast cells (30,31). The replication defect cannot be directly attributed to differences in DNA-binding affinities, because Mason et al. (13) demonstrated that the alternative RPA complex (containing RPA4) binds ssDNA similarly to wild-type RPA (containing RPA2).

Only hybrid 242 caused G2/M arrest similar to RPA4 (Figure 5d and e). To explore a possible cause for this G2/M arrest, we examined the ability of the hybrid constructs to participate in genome maintenance by examining γ-H2AX staining. All hybrid constructs had γ-H2AX staining similar to RPA4: no high γ-H2AX staining cells were observed. In addition, all hybrids were localized to foci after DNA damage (Supplementary Figure S2). This suggests that all of the domains of RPA4 can substitute for the comparable domain of RPA2 in these cellular processes.

We also measured whether RPA2 DBD-D is sufficient for replication in human cells and if RPA4 DBD-G is sufficient for G2/M arrest using ‘core’ constructs (Figure 5b). Following RPA2 depletion, cells expressing DBD-D progressed through S-phase, although the efficiency of progression was reduced compared to exogenous RPA2 (Supplemental Figure S3). In contrast, cells expressing DBD-G showed a dramatic reduction in progression through S-phase, similar to RPA4 (Supplemental Figure S3). These studies do not rule out DBD-G supporting a low level of DNA synthesis but demonstrate that DBD-G activity is similar to RPA4 while DBD-D is closer to RPA2. The DBD is the domain most highly conserved between RPA2 and RPA4 (Figure 1), yet one, DBD-D, is functional for replication and the other, DBD-G, is defective for replication and causes unstressed cells to arrest in G2/M.

### Function of the L34 loop region of RPA4

There is a short region, noted previously (12), where RPA2 contains an acidic stretch of amino acids not found in RPA4 (Figure 6a). The corresponding stretch of amino acids (aa 108–123) in RPA4 contains a number of basic residues (Figure 6a). In our studies of ‘core’ DBD constructs (Figure 5a and d), we observed ∼5 kDa size difference in the mobility of RPA2 DBD-D and RPA4 DBD-G polypeptides, consistent with substantial differences in charge in this region (Figure 6b). This region lies in the 3–4 loop (L34) of the known DBD-D structure and is postulated to be flexible, as it only appears in one of nine known structures of RPA2 (18). Protein modeling revealed that although predicted structures of DBD-G are similar to DBD-D, the predicted electrostatic surface potentials of the L34 region of these two domains are very different (Figure 6b).
Figure 6.Examination of basic/acidic constructs. (a) Alignment of loop3-4 (L34) regions of RPA2 and RPA4. The L34 region lies in the DBD of each protein. Amino acid position is designated to the left and right of the sequence. Identical residues are denoted by asterisk, and the positively and negatively charged residues that differ between RPA2 and RPA4 are denoted with − or +. (b) Protein modeling and predicted electrostatic surface potential for RPA2 and RPA4 DBDs. The known structure of DBD-D (18) is shown to the left, and one model of DBD-G (Geno3D) is shown to the right. Important features are denoted as follows: L12, loop between β-sheets 1 and 2; L34, loop between β-sheets 3 and 4; L45, loop between β-sheets 4 and 5; cleft, putative DNA-binding cleft. For electrostatic surface potential: red, acidic; blue, basic. Western blot of expression is shown in the middle and constructs are denoted above each lane and identified to the left and right of the blot (G-RPA or G-DBD indicates GFP-tagged protein). NS, non-specific bands. (c) Cell-cycle phenotypes for exogenous basic/acidic constructs. Cells were transfected and examined at 96 h. post-transfection as in Figure 3a. The percentage of cell in G2/M phase is shown in each histogram as the mean ± standard deviation. (d) Aphidicolin block and release of cells containing 2Basic and 4Acidic. Data generated in same experiment shown in Figure 5a: cells were treated and designated as described in Figure 5a. (e) Dominant negative analysis. In the presence of endogenous RPA2, exogenous RPA2, RPA4 and 2Basic were expressed. Overlays of the DNA histograms are shown. Black histograms represent cells with no vector and red histograms represent cells expressing indicted RPA form.

Examination of basic/acidic constructs. (a) Alignment of loop3-4 (L34) regions of RPA2 and RPA4. The L34 region lies in the DBD of each protein. Amino acid position is designated to the left and right of the sequence. Identical residues are denoted by asterisk, and the positively and negatively charged residues that differ between RPA2 and RPA4 are denoted with − or +. (b) Protein modeling and predicted electrostatic surface potential for RPA2 and RPA4 DBDs. The known structure of DBD-D (18) is shown to the left, and one model of DBD-G (Geno3D) is shown to the right. Important features are denoted as follows: L12, loop between β-sheets 1 and 2; L34, loop between β-sheets 3 and 4; L45, loop between β-sheets 4 and 5; cleft, putative DNA-binding cleft. For electrostatic surface potential: red, acidic; blue, basic. Western blot of expression is shown in the middle and constructs are denoted above each lane and identified to the left and right of the blot (G-RPA or G-DBD indicates GFP-tagged protein). NS, non-specific bands. (c) Cell-cycle phenotypes for exogenous basic/acidic constructs. Cells were transfected and examined at 96 h. post-transfection as in Figure 3a. The percentage of cell in G2/M phase is shown in each histogram as the mean ± standard deviation. (d) Aphidicolin block and release of cells containing 2Basic and 4Acidic. Data generated in same experiment shown in Figure 5a: cells were treated and designated as described in Figure 5a. (e) Dominant negative analysis. In the presence of endogenous RPA2, exogenous RPA2, RPA4 and 2Basic were expressed. Overlays of the DNA histograms are shown. Black histograms represent cells with no vector and red histograms represent cells expressing indicted RPA form.

Using site-directed mutagenesis, we generated a form of GFP-RPA2 where the acidic L34 loop was replaced with the basic L34 loop from RPA4 (called 2Basic) and a form of GFP-RPA4 where the basic L34 was replaced with the acidic L34 loop from RPA2 (called 4Acidic). Exogenous expression of 2Basic in RPA2-depleted HeLa cells resulted in G2/M arrest, similar to, but more pronounced than, exogenous expression of RPA4 (Figure 6c, see also Figure 5e). Conversely, 4Acidic expression resulted in percentages of G2/M cells more similar to RPA2-expressing cells (Figure 6c). This suggests that L34 is critical for RPA2 function, and that the RPA4 basic loop is both necessary and sufficient for the arrest phenotype observed for constructs containing DBD-G. S-phase progression was also examined using synchronized cells. Neither 2Basic nor 4Acidic could support S-phase progression (Figure 6d) indicating that L34 from RPA4 is sufficient to inactivate RPA2, but that the corresponding loop from RPA2 does not restore function to RPA4.

Since RPA4 is deficient for DNA replication and proficient for G2/M arrest, we tested whether or not exogenous RPA4 expression could inhibit endogenous RPA2 function. Cells were transfected with GFP-RPA2 or GFP-RPA4 without depletion of endogenous RPA2. One would predict a dominant negative phenotype to result in an increase in S-phase cells (replication deficiency), a G2/M checkpoint (arrest proficiency), or both. We were unable to detect an appreciable increase in S- or G2/M-phase after exogenous RPA4 expression (Figure 6e). However, expression of 2Basic, which contains the L34 region (only 16 aa) of RPA4, caused a detectable increase in G2/M arrested cells (Figure 6e). This is consistent with 2Basic having a stronger phenotype in RPA2-depleted cells than RPA4. The absence of a dominant negative phenotype with RPA4 could be the result of expression levels, protein stability, or complex preference. Recently, RPA complexes containing 2Basic or RPA4 (but not 4Acidic) have been shown to inhibit wild-type RPA function in SV40 replication in vitro (13). Together these findings suggest that 2Basic, and probably RPA4, can inhibit normal RPA2 functions in cells when expressed at high enough levels, and that the basic L34 region of RPA4 is sufficient for this inhibition.

### DISCUSSION

All cells require DNA repair to maintain the integrity of their genome, and canonical RPA is required for multiple DNA repair pathways. We have shown that RPA4 may be able to participate in DNA repair processes: (i) RPA4 is able to localize to sites of DNA damage, (ii) expression of RPA4 supports checkpoint activation after DNA damage and (iii) reduces high γ-H2AX levels that occur after depletion of RPA2. These studies also show that, unlike RPA2, RPA4 does not support chromosomal DNA replication. We believe that incomplete replication causes the appearance of γ-H2AX foci and leads to cell-cycle arrest in unstressed, proliferating cells expressing only RPA4. We also have recently shown that RPA4 can support some repair processes in vitro (Mason, unpublished data). Thus, both forms appear to support DNA repair, but only RPA2 supports DNA replication and cell-cycle progression.

Hybrid subunits were used to identify the region of RPA4 is responsible for its phenotypes. From these studies, we were able to demonstrate: (i) the putative phosphorylation domain of RPA4 has no effect on DNA replication, (ii) the C-terminus of RPA4 can support a substantial amount of cellular DNA replication and (iii) the DBD G, the putative DBD of RPA4, is necessary and sufficient to prevent cellular DNA replication.

The C-terminus of RPA2 has been shown to be essential for SV40 DNA replication in vitro (32). This finding was confirmed by a recent study by Mason et al. who showed that an RPA complex containing the RPA4 C-terminus substituted for the normal RPA2 C-terminus (hybrid 224) is defective in SV40 DNA replication in vitro (13). However, we show that in human cells, a substantial amount of chromosomal DNA replication was observed in cells containing only hybrid 224. This indicates that the inability of the hybrid 224 to support in vitro DNA replication is specific to the SV40 system and not to cellular replication. These conclusions are also consistent with the observation in yeast that the C-terminus of RPA2 is dispensable for cell viability (30).

The putative DBD of RPA4 is responsible for the observed phenotypes (replication defect and G2/M arrest). This domain shares the highest sequence identity between RPA2 and RPA4, especially in regions that have been shown to be important for the global fold (18). This domain of RPA2 is required for formation of the RPA complex, has weak ssDNA-binding activity, and participates in specific protein–protein interactions (4). Recent biochemical characterization of an alternative RPA complex (containing RPA4) demonstrated that it has properties indistinguishable from the canonical RPA (13). This suggests that the phenotypes caused by RPA4 are most likely due to changes in protein–protein interactions. In particular, the loop (L34) region shows a large charge difference between RPA4 and RPA2. By manipulating this region in RPA2 and RPA4, we were able to show that RPA2 containing the L34 loop region of RPA4 behaved similarly to RPA4. Given that this loop is flexible, we propose that this loop region may be responsible for altering specific protein–protein interactions in the cell that result in the inability to replicate DNA and trigger cell-cycle arrest. We are currently investigating this possibility.

Initial observations found that RPA4 protein was detected predominantly in quiescent cells, and RPA2 was detected predominantly in proliferating cells (12). For example, RPA4 was originally found to be expressed in colon mucosal cells, which are mostly non-proliferative (12). More recently it has been noted that RPA2 expression is increased and correlates with the severity of colon cancer (8). Keshav et al. (12) suggested that preferential expression of RPA4 versus RPA2 may be indicative of tissue differentiation and cell quiescence. We have shown that RPA4 is unable to support DNA replication, and that DBD-G (specifically, L34 of RPA4) also causes accumulation of cells in G2/M. Our data are consistent with Keshav’s hypothesis. RPA4 cannot support DNA replication but does appear to participate in DNA recognition and DNA repair. This suggests a model in which cells expressing predominantly RPA4 would be able to maintain their genome but not carry out DNA replication or cell proliferation. This model also suggests either that expression of RPA4 contributes to modulation of cell proliferation or that RPA4 expression must be down-regulated for proliferation to occur.

A prediction of this model is that if both RPA4 and RPA2 are present in cells, the RPA4 should compete with endogenous levels of RPA2 and inhibit DNA replication and cell-cycle progression. This type of inhibition was observed in vitro: a RPA complex containing RPA4 inhibits the function of canonical RPA2 containing complex in SV40 DNA replication (13). We did not observe a G2/M arrest in cells expressing both RPA2 and RPA4; however, we did observe a dominant negative phenotype when both RPA2 and RPA2Basic were expressed (Figure 6e). One explanation for these results is that exogenous RPA4 expression in this system may simply not be high enough to observe an effect in the presence of endogenous RPA2, but that a similar level of expression of the 2Basic subunit competes more efficiently with wild-type RPA2 and results in a phenotype. We are currently examining whether increasing the expression of RPA4 using other promoters or different cells results in modulation of cell growth.

The different distributions of cells observed in asynchronous RPA2-depleted cells also suggest an inhibitory function for RPA4 in the cell. In asynchronous cells, we observed an asymmetric distribution of cells in S-phase when RPA2 is depleted (e.g. Figure 6c). This distribution is most easily explained by the fact that RPA2 depletion is gradual, and some percentage of cells will exit G1 and enter into S-phase before RPA2 is depleted below a critical level. This asymmetric distribution is not observed in cells expressing exogenous RPA4 (e.g. Figure 6c). In these cells, RPA4 is being expressed at high levels as RPA2 depletion is occurring. If RPA4 did not inhibit RPA2 function, an asymmetric distribution similar to the depleted cells should have been observed. Since it was not, this suggests that the level of RPA4 present was sufficient to inhibit cells with residual (lower) amounts of RPA2 from exiting G1 phase and entering S-phase. This supports the conclusion that RPA4 can inhibit RPA2 function.

Together, these analyses support a maintenance function for RPA4 in non-proliferating cells. These studies suggest that RPA4 expression levels could have dramatic effects during development and could help modulate cell proliferation status.

### RPA4 and RPA2 have different cellular functions

All cells require DNA repair to maintain the integrity of their genome, and canonical RPA is required for multiple DNA repair pathways. We have shown that RPA4 may be able to participate in DNA repair processes: (i) RPA4 is able to localize to sites of DNA damage, (ii) expression of RPA4 supports checkpoint activation after DNA damage and (iii) reduces high γ-H2AX levels that occur after depletion of RPA2. These studies also show that, unlike RPA2, RPA4 does not support chromosomal DNA replication. We believe that incomplete replication causes the appearance of γ-H2AX foci and leads to cell-cycle arrest in unstressed, proliferating cells expressing only RPA4. We also have recently shown that RPA4 can support some repair processes in vitro (Mason, unpublished data). Thus, both forms appear to support DNA repair, but only RPA2 supports DNA replication and cell-cycle progression.

### The contribution of the individual regions of RPA2 and RPA4 to cellular replication

Hybrid subunits were used to identify the region of RPA4 is responsible for its phenotypes. From these studies, we were able to demonstrate: (i) the putative phosphorylation domain of RPA4 has no effect on DNA replication, (ii) the C-terminus of RPA4 can support a substantial amount of cellular DNA replication and (iii) the DBD G, the putative DBD of RPA4, is necessary and sufficient to prevent cellular DNA replication.

The C-terminus of RPA2 has been shown to be essential for SV40 DNA replication in vitro (32). This finding was confirmed by a recent study by Mason et al. who showed that an RPA complex containing the RPA4 C-terminus substituted for the normal RPA2 C-terminus (hybrid 224) is defective in SV40 DNA replication in vitro (13). However, we show that in human cells, a substantial amount of chromosomal DNA replication was observed in cells containing only hybrid 224. This indicates that the inability of the hybrid 224 to support in vitro DNA replication is specific to the SV40 system and not to cellular replication. These conclusions are also consistent with the observation in yeast that the C-terminus of RPA2 is dispensable for cell viability (30).

The putative DBD of RPA4 is responsible for the observed phenotypes (replication defect and G2/M arrest). This domain shares the highest sequence identity between RPA2 and RPA4, especially in regions that have been shown to be important for the global fold (18). This domain of RPA2 is required for formation of the RPA complex, has weak ssDNA-binding activity, and participates in specific protein–protein interactions (4). Recent biochemical characterization of an alternative RPA complex (containing RPA4) demonstrated that it has properties indistinguishable from the canonical RPA (13). This suggests that the phenotypes caused by RPA4 are most likely due to changes in protein–protein interactions. In particular, the loop (L34) region shows a large charge difference between RPA4 and RPA2. By manipulating this region in RPA2 and RPA4, we were able to show that RPA2 containing the L34 loop region of RPA4 behaved similarly to RPA4. Given that this loop is flexible, we propose that this loop region may be responsible for altering specific protein–protein interactions in the cell that result in the inability to replicate DNA and trigger cell-cycle arrest. We are currently investigating this possibility.

### RPA4 and cell-cycle regulation

Initial observations found that RPA4 protein was detected predominantly in quiescent cells, and RPA2 was detected predominantly in proliferating cells (12). For example, RPA4 was originally found to be expressed in colon mucosal cells, which are mostly non-proliferative (12). More recently it has been noted that RPA2 expression is increased and correlates with the severity of colon cancer (8). Keshav et al. (12) suggested that preferential expression of RPA4 versus RPA2 may be indicative of tissue differentiation and cell quiescence. We have shown that RPA4 is unable to support DNA replication, and that DBD-G (specifically, L34 of RPA4) also causes accumulation of cells in G2/M. Our data are consistent with Keshav’s hypothesis. RPA4 cannot support DNA replication but does appear to participate in DNA recognition and DNA repair. This suggests a model in which cells expressing predominantly RPA4 would be able to maintain their genome but not carry out DNA replication or cell proliferation. This model also suggests either that expression of RPA4 contributes to modulation of cell proliferation or that RPA4 expression must be down-regulated for proliferation to occur.

A prediction of this model is that if both RPA4 and RPA2 are present in cells, the RPA4 should compete with endogenous levels of RPA2 and inhibit DNA replication and cell-cycle progression. This type of inhibition was observed in vitro: a RPA complex containing RPA4 inhibits the function of canonical RPA2 containing complex in SV40 DNA replication (13). We did not observe a G2/M arrest in cells expressing both RPA2 and RPA4; however, we did observe a dominant negative phenotype when both RPA2 and RPA2Basic were expressed (Figure 6e). One explanation for these results is that exogenous RPA4 expression in this system may simply not be high enough to observe an effect in the presence of endogenous RPA2, but that a similar level of expression of the 2Basic subunit competes more efficiently with wild-type RPA2 and results in a phenotype. We are currently examining whether increasing the expression of RPA4 using other promoters or different cells results in modulation of cell growth.

The different distributions of cells observed in asynchronous RPA2-depleted cells also suggest an inhibitory function for RPA4 in the cell. In asynchronous cells, we observed an asymmetric distribution of cells in S-phase when RPA2 is depleted (e.g. Figure 6c). This distribution is most easily explained by the fact that RPA2 depletion is gradual, and some percentage of cells will exit G1 and enter into S-phase before RPA2 is depleted below a critical level. This asymmetric distribution is not observed in cells expressing exogenous RPA4 (e.g. Figure 6c). In these cells, RPA4 is being expressed at high levels as RPA2 depletion is occurring. If RPA4 did not inhibit RPA2 function, an asymmetric distribution similar to the depleted cells should have been observed. Since it was not, this suggests that the level of RPA4 present was sufficient to inhibit cells with residual (lower) amounts of RPA2 from exiting G1 phase and entering S-phase. This supports the conclusion that RPA4 can inhibit RPA2 function.

Together, these analyses support a maintenance function for RPA4 in non-proliferating cells. These studies suggest that RPA4 expression levels could have dramatic effects during development and could help modulate cell proliferation status.

### SUPPLEMENTARY DATA

Supplementary Data are available at NAR Online.

### FUNDING

American Heart Association Postdoctoral Fellowship (0525735Z to S.J.H.); National Institute of Health Research Grant (GM44721 to M.S.W.); Center for Aging Program at the University of Iowa (CA103672 to M.S.W.). Funding for open access charge: National Institutes of Health.

Conflict of interest statement. None declared.

### Supplementary Material



# SUPPLEMENTAL FILE 1: gkp1062.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)