# MAIN TEXT

## Bayes Analysis Provides Evidence of Pathogenicity for the BRCA1 c.135-1G>T (IVS3-1) and BRCA2 c.7977-1G>C (IVS17-1) Variants Displaying In Vitro Splicing Results of Equivocal Clinical Significance

### Abstract

Although in vitro splicing assays can provide useful information about the clinical interpretation of sequence variants in high-risk cancer genes such as BRCA1 and BRCA2, results can sometimes be difficult to interpret. The BRCA1 c.135-1G>T (IVS3-1G>T) variant has been shown to give rise to an in-frame deletion of exon 5 (BRCA1 c.135_212del) that is predicted to encode 26 amino acids. BRCA2 c.7977-1G>C (IVS17-1G>C) was shown to increase the expression of two naturally occurring transcripts that contain frameshifts (BRCA2, c.7977_8311del (exon 18 deletion); BRCA2, c.7806_8331del (exon 17&18 deletion)). In this study we conducted multifactorial likelihood analysis to evaluate the clinical significance of these two variants, including assessing variant segregation in families by Bayes analysis, and breast tumor pathology features suggestive of positive mutation status. Multifactorial analysis provided strong evidence for causality for both of these variants. The Bayes scores from a single family with BRCA1 c.135-1G>T was 9528:1, and incorporation of pathology features gave an overall likelihood of causality of 28108:1. The Bayes scores from five informative families with BRCA2 c.7977-1G>C was 47401:1, and the combined Bayes-pathology odds of causality was 29389:1. Multifactorial likelihood analysis indicates that the BRCA1 c.135-1G>T and BRCA2 c.7977-1G>C variants are disease-associated mutations which should be managed clinically in the same fashion as classical truncating mutations.



# SUPPLEMENTAL FILE 1: nihms163486.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)