# MAIN TEXT

## Identification of a Danish breast/ovarian cancer family double heterozygote for BRCA1 and BRCA2 mutations

### Abstract

Mutations in the two breast cancer susceptibility genes BRCA1 and BRCA2 are associated with increased risk of breast and ovarian cancer. Patients with mutations in both genes are rarely reported and often involve Ashkenazi founder mutations. Here we report the first identification of a Danish breast and ovarian cancer family heterozygote for mutations in the BRCA1 and BRCA2 genes. The BRCA1 nucleotide 5215G > A/c.5096G > A mutation results in the missense mutation Arg1699Gln, while the BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G is novel. Exon trapping experiments and reverse transcriptase (RT)–PCR analysis revealed that the BRCA2 mutation results in skipping of exon 7, thereby introducing a frameshift and a premature stop codon. We therefore classify the mutation as disease causing. Since the BRCA1 Arg1699Gln mutation is also suggested to be disease-causing, we consider this family double heterozygote for BRCA1 and BRCA2 mutations.

### Introduction

Germ-line mutations in BRCA1 (MIM# 113705) and BRCA2 (MIM# 600185) confer a high risk of breast and ovarian cancer. A disease causing mutation will result in 60–80% risk of developing breast cancer for a female carrier, while the risk of getting ovarian cancer is 20–40% or 10–20% with a BRCA1 or BRCA2 mutation, respectively [reviewed in [1]]. Most mutations in BRCA1 and BRCA2 have been listed in Breast Cancer Information Core (BIC) [2], and includes pathogenic mutations such as frameshift, nonsense, and splicing mutations, of which several have been identified in Danish breast and/or ovarian cancer patients [3–6]. Moreover, several founder mutations have been identified in a variety of different populations, including Ashkenazi Jews, Icelanders and Greenlandic Inuits [7–9]. A large number of BRCA1/BRCA2 missense, silent and intron mutations are of unknown significance. To enable the clinicians to give the affected patients satisfactory advice about preventive care, assessment of these mutations of unknown significance by functional studies are advised. Intron variants can be analysed by RT–PCR or by examining the splicing pattern in vitro using a mini gene system (reviewed in [10]). Here we report the first identification of a Danish family with germ-line mutations in both the BRCA1 and BRCA2 genes. The BRCA1 nucleotide 5215G > A/c.5096G > A mutation results in the missense mutation Arg1699Gln, while the BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G mutation is novel. Functional characterization of the BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G mutation by exon trapping experiments and RT–PCR analysis, revealed that the mutation results in skipping of exon 7 and leads to incorporation of 18 out of frame amino acids followed by a stop codon. We therefore classify the mutation as disease causing. Since the BRCA1 Arg1699Gln mutation is also suggested to be disease-causing, we consider this family double heterozygote for BRCA1 and BRCA2 mutations.

### Materials and methods

A 59 year-old woman with breast cancer (53) and ovarian cancer (59) was referred to genetic counselling. The family had two other cases of breast cancer (Fig. 1). Following verbal and written consent, blood samples were collected from the proband for mutation screening and after renewed consent a third blood sample was collected for RNA analysis. The family history was verified using the registry of the DBCG (Danish Breast Cancer Cooperative Group), hospital medical records and pathology reports, and genetic counselling was provided for family members.Fig. 1Family pedigree. Breast and ovarian cancer are indicated as well as the age at diagnosis. Diagonal slash indicates deceased, while the proband is indicated with an arrow. Mutation positive individuals are indicated with +. +/+ indicates individuals with both mutations

Family pedigree. Breast and ovarian cancer are indicated as well as the age at diagnosis. Diagonal slash indicates deceased, while the proband is indicated with an arrow. Mutation positive individuals are indicated with +. +/+ indicates individuals with both mutations

Genomic DNA was purified from whole blood using the QIAamp DNA mini kit (Qiagen), and from paraffin-embedded tissue using the Chemagic DNA tissue kit (Chemagen) according to the manufacturer’s instructions. BRCA1 and BRCA2 screening was performed as recently described [4]. Sequence variations were verified in a new blood sample. The BRCA1 mutation is numbered according to GenBank accession number U14680, in which A in the AUG start codon has number 120, while the BRCA2 mutation is numbered according to GenBank accession number NM_000059, in which A in the AUG start codon has number 229. Moreover, the BRCA1 and BRCA2 mutations are numbered according to GenBank accession number NC_00017.10 and NC_000013.9 using the guidelines from the Human Genetic Variation Society.

In silico prediction of the functional consequence of the missense variants was performed using SIFT (Sorting Intolerant From Tolerant): http://blocks.fhcrc.org/sift/SIFT.html [11] Polyphen: http://coot.embl.de/PolyPhen/ [12] and PMut: http://mmb2.pcb.ub.es:8080/PMut/ [13]. The following three splice site prediction programmes were used to predict the effect of the BRCA2 IVS7 + 4A > G mutation on the efficiency of splicing: www.fruitfly.org/seq_tools/splice.html [14], www.cbs.dtu.dk/services/NetGene2 [15] and www.umd.be/SSF [16]. The genomic sequence spanning the BRCA2 nucleotide 859 + 4 A > G/c.631 + 4 A > G mutation was submitted according to the guidelines of each programme and default settings were used in all predictions.

pSPL3-BRCA2-wild type plasmid and pSPL3-BRCA2-mutant plasmid, containing BRCA2 exon 7 and flanking intron sequences, was constructed by cloning genomic DNA from the proband using the following primers BRCA2-F, 5′-GATCACCTCGAGCGTTATACCTTTGCCCTGAGATTTA -3′ and BRCA2-R 5′-GATCACCTGCAGTTTAATGCCCCAATTACCAC -3′ into the pSPL3 splicing vector as recently described [5]. All constructs were verified by sequencing. COS7 cells were transfected using Fugene 6 (Roche) with either pSPL3-BRCA2-wild-type or pSPL3-BRCA2-mutant plasmids, RNA was purified and RT–PCR was performed as recently described [5].

A fresh blood sample was obtained from the patient. Total cellular RNA was isolated with Trizol (Invitrogen) according to the manufacturer’s instructions. For reverse transcription–PCR (RT–PCR), cDNA was synthesized using random hexamer primers and M-MuLV reverse transcriptase (New England Biolabs) as described by the supplier. The cDNA was amplified with the BRCA2 specific primers 5′-CAAATTAGACTTAGGAAGGAATGTTCC-3′ and 5′-CATGAGGAAATACAGTTTCAGATGC-3′. The samples were separated by agarose gel electrophoresis and visualized by ethidium bromide staining. Finally, the bands were purified, cloned into pCR-Blunt II-TOPO (Invitrogen) and sequenced using an ABI3730 DNA analyzer (Applied Biosystems).

### Patients

A 59 year-old woman with breast cancer (53) and ovarian cancer (59) was referred to genetic counselling. The family had two other cases of breast cancer (Fig. 1). Following verbal and written consent, blood samples were collected from the proband for mutation screening and after renewed consent a third blood sample was collected for RNA analysis. The family history was verified using the registry of the DBCG (Danish Breast Cancer Cooperative Group), hospital medical records and pathology reports, and genetic counselling was provided for family members.Fig. 1Family pedigree. Breast and ovarian cancer are indicated as well as the age at diagnosis. Diagonal slash indicates deceased, while the proband is indicated with an arrow. Mutation positive individuals are indicated with +. +/+ indicates individuals with both mutations

Family pedigree. Breast and ovarian cancer are indicated as well as the age at diagnosis. Diagonal slash indicates deceased, while the proband is indicated with an arrow. Mutation positive individuals are indicated with +. +/+ indicates individuals with both mutations

### BRCA1 and BRCA2 screening

Genomic DNA was purified from whole blood using the QIAamp DNA mini kit (Qiagen), and from paraffin-embedded tissue using the Chemagic DNA tissue kit (Chemagen) according to the manufacturer’s instructions. BRCA1 and BRCA2 screening was performed as recently described [4]. Sequence variations were verified in a new blood sample. The BRCA1 mutation is numbered according to GenBank accession number U14680, in which A in the AUG start codon has number 120, while the BRCA2 mutation is numbered according to GenBank accession number NM_000059, in which A in the AUG start codon has number 229. Moreover, the BRCA1 and BRCA2 mutations are numbered according to GenBank accession number NC_00017.10 and NC_000013.9 using the guidelines from the Human Genetic Variation Society.

### In silico analyse

In silico prediction of the functional consequence of the missense variants was performed using SIFT (Sorting Intolerant From Tolerant): http://blocks.fhcrc.org/sift/SIFT.html [11] Polyphen: http://coot.embl.de/PolyPhen/ [12] and PMut: http://mmb2.pcb.ub.es:8080/PMut/ [13]. The following three splice site prediction programmes were used to predict the effect of the BRCA2 IVS7 + 4A > G mutation on the efficiency of splicing: www.fruitfly.org/seq_tools/splice.html [14], www.cbs.dtu.dk/services/NetGene2 [15] and www.umd.be/SSF [16]. The genomic sequence spanning the BRCA2 nucleotide 859 + 4 A > G/c.631 + 4 A > G mutation was submitted according to the guidelines of each programme and default settings were used in all predictions.

### Exon trapping analysis

pSPL3-BRCA2-wild type plasmid and pSPL3-BRCA2-mutant plasmid, containing BRCA2 exon 7 and flanking intron sequences, was constructed by cloning genomic DNA from the proband using the following primers BRCA2-F, 5′-GATCACCTCGAGCGTTATACCTTTGCCCTGAGATTTA -3′ and BRCA2-R 5′-GATCACCTGCAGTTTAATGCCCCAATTACCAC -3′ into the pSPL3 splicing vector as recently described [5]. All constructs were verified by sequencing. COS7 cells were transfected using Fugene 6 (Roche) with either pSPL3-BRCA2-wild-type or pSPL3-BRCA2-mutant plasmids, RNA was purified and RT–PCR was performed as recently described [5].

### RNA analysis

A fresh blood sample was obtained from the patient. Total cellular RNA was isolated with Trizol (Invitrogen) according to the manufacturer’s instructions. For reverse transcription–PCR (RT–PCR), cDNA was synthesized using random hexamer primers and M-MuLV reverse transcriptase (New England Biolabs) as described by the supplier. The cDNA was amplified with the BRCA2 specific primers 5′-CAAATTAGACTTAGGAAGGAATGTTCC-3′ and 5′-CATGAGGAAATACAGTTTCAGATGC-3′. The samples were separated by agarose gel electrophoresis and visualized by ethidium bromide staining. Finally, the bands were purified, cloned into pCR-Blunt II-TOPO (Invitrogen) and sequenced using an ABI3730 DNA analyzer (Applied Biosystems).

### Results

The proband was referred to genetic counselling, since she was diagnosed with breast cancer at the age of 53 and ovarian cancer at the age of 59. Her deceased father and deceased sister were diagnosed with breast cancer at the age of 76 and 43, respectively (Fig. 1). Blood samples were collected, genomic DNA was purified and the entire coding region and the exon–intron boundaries of BRCA1 and BRCA2 were screened by dHPLC and sequencing. Moreover, the DNA was examined for large genomic rearrangements by MLPA analysis. The analysis identified a BRCA1 nucleotide 5215G > A/c.5096G > A mutation (Fig. 2a), which introduces a missense mutation changing amino acid 1699 from an arginine to a glutamine. Moreover, a novel BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G mutation of unknown significance were identified (Fig. 2b). Both mutations were also identified in the proband’s two children. Moreover, analysis of genomic DNA from paraffin-embedded cancer tissue from the probands deceased father revealed that both mutations were inherited from him (Fig. 1). To indicate whether the mutations could be disease-causing, three different protein prediction programmes were used to predict the functional consequence of the BRCA1 Arg1699Gln mutation [11–13], while three different splice site prediction programmes were used to predict the effect of the BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G mutation on splicing of exon 7 [14–16]. All three protein prediction programmes estimated the BRCA1 Arg1699Gln mutation to be pathogenic. Moreover, two of the splice site programmes predicted the BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G mutation to have an effect on splicing of exon 7, while the third one (NetGene2) was unable to find both the wild type and the mutated splice site. It was therefore decided to analyse the BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G mutation by functional analysis using exon trapping. A fragment including BRCA2 exon 7 containing either the wild-type or the nucleotide 859 + 4A > G/c.631 + 4A > G mutation, respectively, 148 bp of intron 6 (IVS6), and 344 bp of intron 7 (IVS7) was cloned into the minigene vector pSPL3. The wild-type and mutant constructs was then transfected into COS-7 cells (in duplicates) and after 48 h mRNA was purified and examined by RT–PCR in duplicate on a 2% agarose gel (Fig. 3a). The normal wild-type BRCA2 exon 7 generated one transcript comprising the expected 292 bp and a very faint band of 177 bp, while the BRCA2 exon 7 nucleotide 859 + 4 A > G/c.631 + 4 A > G mutant yielded one faint band of 292 bp and a very strong band of 177 bp, respectively. Sequencing revealed that the 292 bp band contained exon 7, while the 177 bp band lacked exon 7. Finally, RT–PCR analysis using mRNA purified from whole blood was performed. The PCR products were analysed on a 1% agarose gel (Fig. 3b). Two PCR products were amplified from the patient—a 368 bp band and a smaller 253 bp band. Cloning and sequence analysis revealed that the 368 bp band contained exon 7, while the 253 bp band lacked exon 7 (data not shown).Fig. 2Identification of the BRCA1 nucleotide 5215G > A/c.5096G > A mutation, and the BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G mutation. DNA was purified from the patient and the BRCA1 and BRCA2 genes were amplified using intronic primer pairs flanking each exon. The PCR products were pre-screened by dHPLC (denaturing high performance liquid chromatography) and sequenced. The analysis revealed a a
BRCA1 nucleotide 5215G > A/c.5096G > A mutation and a b
BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G mutation
Fig. 3Exon trapping and RT–PCR analysis. a COS-7 cells were transfected with pSPL3-BRCA2-exon 7 wild-type or pSPL3-BRCA2-exon 7 mutant plasmids in duplicates. Total RNA was isolated, RT–PCR analysis was performed and the PCR products were resolved on a 2% agarose gel. The 292 bp product corresponds to inclusion of exon 7 (unaltered splicing), while the 172 bp product corresponds to the exclusion of exon 7. The sizes of the DNA marker (M) are indicated to the right. b RT–PCR was performed on RNA purified from whole blood. The cDNA was amplified with specific BRCA2 primers. The sample was separated by agarose gel electrophoresis and visualized by ethidium bromide staining. Two RT–PCR products (368 and 253 bp) were obtained from the patient (Lane 2). The PCR products were cloned and sequence analysis revealed that the 253 bp band lacked exon 7 (data not shown)

Identification of the BRCA1 nucleotide 5215G > A/c.5096G > A mutation, and the BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G mutation. DNA was purified from the patient and the BRCA1 and BRCA2 genes were amplified using intronic primer pairs flanking each exon. The PCR products were pre-screened by dHPLC (denaturing high performance liquid chromatography) and sequenced. The analysis revealed a a
BRCA1 nucleotide 5215G > A/c.5096G > A mutation and a b
BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G mutation

Exon trapping and RT–PCR analysis. a COS-7 cells were transfected with pSPL3-BRCA2-exon 7 wild-type or pSPL3-BRCA2-exon 7 mutant plasmids in duplicates. Total RNA was isolated, RT–PCR analysis was performed and the PCR products were resolved on a 2% agarose gel. The 292 bp product corresponds to inclusion of exon 7 (unaltered splicing), while the 172 bp product corresponds to the exclusion of exon 7. The sizes of the DNA marker (M) are indicated to the right. b RT–PCR was performed on RNA purified from whole blood. The cDNA was amplified with specific BRCA2 primers. The sample was separated by agarose gel electrophoresis and visualized by ethidium bromide staining. Two RT–PCR products (368 and 253 bp) were obtained from the patient (Lane 2). The PCR products were cloned and sequence analysis revealed that the 253 bp band lacked exon 7 (data not shown)

### Discussion

Patients double heterozygote for pathogenic mutations in BRCA1 and BRCA2 are rarely observed, and most reports involve the three Ashkenazi Jewish founder mutations BRCA1 nucleotide 185delAG and nucleotide 5382insC and BRCA2 nucleotide 6174delT (reviewed in [17]) [18–20]. However, four reports describe other BRCA1 and BRCA2 mutations, including a Dutch study describing two patients with a Dutch BRCA1 founder mutation and two different BRCA2 frameshift mutations [17], an Australian study describing one patient with a BRCA1 frameshift mutation and a BRCA2 splice site mutation [21], while a Korean study reported two cases of double heterozygosity for frameshift or nonsense mutations in the BRCA1 and BRCA2 genes [22]. Finally, a Spanish family with a pathogenic BRCA1 Ala1708Glu missense mutation and a BRCA2 frameshift mutation has been identified [23].

In this study, we have identified a Danish breast and ovarian cancer family with germ-line mutations in the BRCA1 and BRCA2 genes. Interestingly, the proband, the probands father, and the probands two children have inherited both mutations. The BRCA1 mutation changes the amino acid residue argenine on position 1699 to a glutamine. Arg1699 is highly conserved from man to plants [24]. It resides in the BRCT domain of BRCA1, which is involved in protein–protein interaction (reviewed in [25]). Crystal structure of the BRCA1 BRCT domain has shown that Arg1699 participates in the salt bridge with Asp1840 and Glu1836 [26]. Moreover, the Arg1699Gln mutation changes the amino acid residue from basic to polar, and in silico analysis using the SIFT, Polyphen or PMut software all regarded the BRCA1 Arg1699Gln mutation as disease-causing. Finally, we have only observed the mutation in breast and/or ovarian cancer families, and not in 200 healthy controls. Still, there have previously been inconclusive reports regarding the Arg1699Gln. Some in silico based studies have not been able to classify the mutation and group it as a variant of unknown significance [27, 28]. In contrast, other studies consider it to be deleterious using in silico analysis [29, 30]. Functional studies have shown that the BRCA1 Arg1699Gln mutation reduces the transcriptional activity of BRCA1 [31]. Moreover a loss in phosphospecificity of BRCA1 Gln1699 in its binding to BACH1—compared to BRCA1 Arg1699—has been observed [32]. Finally, the Arg1699Gln mutation appears to be defective in nuclear foci formation [33]. Moreover it should be noted that another substitution on this position (Arg1699Trp) is considered deleterious based on both functional and co-segregation studies [27]. Together, these data indicate that the BRCA1 Arg1699Gln mutation could be disease-causing, although it may have a lower penetrance.

The BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G mutation has not previously been reported in the BIC database or in the literature. Several other mutations have been identified in this area, including nucleotide 859 + 2T > G/c.631 + 2T > G, which disrupt the splice site consensus sequence and induce skipping of exon 7 [34], nucleotide 859G > A/c.631G > A, which also has been shown to cause skipping of exon 7 [35], as well as nucleotide 859 + 1G > A/c.631 + 1G > A [2]. Analysis of the BRCA2 nucleotide 859 + 4A > G/c.631 + 4A > G mutation using splice site prediction programmes suggested that the mutation affected the exon 7 splice doner site. Therefore we performed exon trapping experiments and RT–PCR analysis on RNA purified from whole blood. The analysis revealed that the mutation resulted in skipping of exon 7, which introduces a premature stop codon after incorporation of 18 out of frame amino acids. We therefore classify this mutation as disease-causing. Interestingly, even though the family is double heterozygote for mutations in BRCA1 and BRCA2, the affected family members do not have breast and/or ovarian cancer at a younger age than individuals with single BRCA1 or BRCA2 deleterious mutations. This is in agreement with other studies showing that double heterozygosity does not seem to lead to a more severe phenotype [17].

In summary, we here report the first Danish case of a patient who is double heterozygote for BRCA1 and BRCA2 mutations. Genetic screening is now offered to all family members.



# SUPPLEMENTAL FILE 1: 10689_2010_Article_9345.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)