# MAIN TEXT

## Contribution of bioinformatics predictions and functional splicing assays to the interpretation of unclassified variants of the BRCA genes

### Abstract

A large fraction of sequence variants of unknown significance (VUS) of the breast and ovarian cancer susceptibility genes BRCA1 and BRCA2 may induce splicing defects. We analyzed 53 VUSs of BRCA1 or BRCA2, detected in consecutive molecular screenings, by using five splicing prediction programs, and we classified them into two groups according to the strength of the predictions. In parallel, we tested them by using functional splicing assays. A total of 10 VUSs were predicted by two or more programs to induce a significant reduction of splice site strength or activation of cryptic splice sites or generation of new splice sites. Minigene-based splicing assays confirmed four of these predictions. Five additional VUSs, all at internal exon positions, were not predicted to induce alterations of splice sites, but revealed variable levels of exon skipping, most likely induced by the modification of exonic splicing regulatory elements. We provide new data in favor of the pathogenic nature of the variants BRCA1 c.212+3A>G and BRCA1 c.5194−12G>A, which induced aberrant out-of-frame mRNA forms. Moreover, the novel variant BRCA2 c.7977−7C>G induced in frame inclusion of 6 nt from the 3′ end of intron 17. The novel variants BRCA2 c.520C>T and BRCA2 c.7992T>A induced incomplete skipping of exons 7 and 18, respectively. This work highlights the contribution of splicing minigene assays to the assessment of pathogenicity, not only when patient RNA is not available, but also as a tool to improve the accuracy of bioinformatics predictions.



# SUPPLEMENTAL FILE 1: ejhg2011100a.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)