# MAIN TEXT

## Establishment of a PCR analysis method for canine BRCA2

### Abstract

BackgroundMammary tumors are the most common tumor type in both human and canine females. In women, carriers of mutations in BRCA2, a tumor suppressor gene product, have a higher risk of breast cancer. Canine BRCA2 has also been suggested to have a relationship with mammary tumors. However, clearly deleterious BRCA2 mutations have not been identified in any canine mammary tumors, as appropriate methods to detect mutations or a consensus BRCA2 sequence have not been reported.FindingsFor amplification and sequencing of BRCA2, we designed 14 and 20 PCR primer sets corresponding to the BRCA2 open reading frame (ORF) and all 27 exons, respectively, including exon-intron boundaries of the canine BRCA2 regions, respectively. To define the consensus canine BRCA2 ORF sequence, we used established methods to sequence the full-length canine BRCA2 ORF sequence from two ovaries and a testis obtained from individual healthy mongrel dogs and partially sequence BRCA2 genomic sequences in 20-56 tumor-free dogs, each aged over 6 years. Subsequently, we compared these sequences and seven previously reported sequences, and defined the most common base sequences as the consensus canine BRCA2 ORF sequence. Moreover, we established a detection method for identifying splicing variants. Unexpectedly, we also identified novel splicing variants in normal testes during establishment of these methods.ConclusionsThe present analysis methods for determining the BRCA2 base sequence and for detecting BRCA2 splicing variants and the BRCA2 ORF consensus sequence are useful for better understanding the relationship between canine BRCA2 mutation status and cancer risk.

### Findings

Mammary tumors are the most common tumor type in both human and canine females, constituting about half of all tumors in female dogs [1-4]. Furthermore, approximately half of canine mammary tumors are malignant [5,6]. In humans, heritable breast cancers have been linked with mutations in the breast cancer susceptibility gene BRCA2. Genetic analysis, including detection of deleterious mutations and splicing variants, to identify BRCA2 mutation carriers is strongly advocated, as the lifetime risk of breast cancer is high (81-88%) for females carrying a BRCA2 mutation [7,8].

In a recent study, it was suggested that the canine BRCA2 gene locus is associated with mammary tumors based on single nucleotide polymorphism analysis of an intronic marker [9,10]. Consistent with this notion, we previously showed that loss of heterozygosity, which is one of the mechanisms of BRCA2 inactivation, was present in a mammary tumor [11]. Canine BRCA2 missense mutations have also been reported in mammary tumors [11-13]. However, clearly deleterious mutations in the canine BRCA2 sequence have not been identified in mammary tumors due to the lack of appropriate methods to detect such mutations. Furthermore, a full-length consensus canine BRCA2 open reading frame (ORF) sequence has not been defined, as full-length canine BRCA2 has only been identified in a single sample [14].

Determination of the base sequence of BRCA2 in a tumor sample and of this sequence comparison with the BRCA2 consensus sequence is the most standard method for detecting mutations in tumor samples in humans. During the course of our present study, one study reported the mutation analysis of full-length of canine BRCA2, but they used many primer sets (about 50 sets) and amplified sequence only from genomic DNA [15]. To establish a more efficient mutation analysis method for cDNA and genomic DNA that requires fewer primer sets, we designed 14 and 20 primer sets in order to sequence the BRCA2 ORF and all 27 exons, respectively, including the exon-intron boundaries of the canine BRCA2 regions. All PCR targets were successfully amplified, and were sufficient to determine DNA base sequences (Figure 1A and 1B).

PCR products amplified by each primer set. (A) cDNA samples prepared from total RNA of each mammary gland were amplified. (B) Genomic DNA from each mammary gland was amplified. (C) Splicing variants of the cDNA from total RNA of the mammary gland and testis were amplified. Primer sets for each lane are shown in Table 1. The "M" indicates the molecular size marker (1-kbp DNA ladder; New England Biolabs). Arrowhead and "*" indicate novel BRCA2 transcript and non-specific PCR products, respectively.

Some splicing variants of tumor suppressor genes (e.g., BRCA2) in tumor tissue have been associated with tumorigenesis because these splicing variants often lead to frameshift mutations [16,17]. Thus, we next designed five primer sets for detecting splicing variants from cDNA (Figure 1C). All PCR targets were successfully amplified, and the predicted sizes of PCR products were confirmed. During the establishment of this method, we unexpectedly identified splicing variants between exon 10 and exon 14 in normal testes (Figure 1C and 2). These transcripts skipped most of exon 11, leading to frameshift mutations (Figure 2).

Identification of splicing variants within exon 11 in normal testes. The splicing variants identified in normal testes lacked a large portion of exon 11. (A) To confirm the presence of a novel BRCA2 transcript, splicing variants of the cDNA from total RNA of the testes were amplified using nested PCR. (B-E) Electropherogram showing the direct sequencing data and overview of the novel BRCA2 transcript that lacked nucleotides 2390 to 6429 (B), 2390 to 6429 and 6588 to 6908 (C), 2390 to 3380, and 4649 to 6429 (D) and 2085 to 2321 and 2390 to 6429 (E) from Testis-1, -3, -4, and -5, respectively. The Testis-1 was the same sample used to generate the data in Figure 1C. Primer sets for each lane are shown in Table 1. The "M" indicates the molecular size marker (1-kbp DNA ladder). Arrowheads indicate novel BRCA2 transcripts.

To define the consensus canine BRCA2 ORF, we sequenced the full-length canine BRCA2 ORF in two ovaries and a testis obtained from individual healthy mongrel dogs using the method described here. We identified six single nucleotide variations (516 T > C, 1366 T > G, 2428 T > G, 2609A > C, 4481A > C and 8257 T > C) and two insertion/deletion polymorphisms (7126ins/delGTT and 10204ins/delAAA) (Accession numbers: AB622997, AB622998 and AB622999). None of these variations resulted in nonsense or frameshift mutations. To determine the most common base sequences and generate a consensus canine BRCA2 ORF sequence, we compared these three new sequences (six alleles) and the seven previously reported sequences (Accession numbers: AB043895.5, NC_006607.2, Z75664 and FJ464397-FJ464400) (Table 2). The four variations (516 T (103I), 2428 T (740 G), 4481A (1425 T), and 8257 T (2683I)) could be defined as consensus base sequences, but the other four variations (1366 T > G, 2609A > C (K801Q), 7126ins/delGTT, and 10204ins/delAAA) could not be defined as such because the frequencies between the major and minor alleles in each variation were nearly identical. We therefore sequenced these four variations in genomic DNA from 20-23 normal blood samples from tumor-free dogs aged over 6 years; the methods described here were used (Tables 3 and 4). We finally defined the most common base sequences as the consensus canine BRCA2 ORF sequence (Table 2). The 10204insAAA variation was consensus sequence in dogs, but in four miniature Dachshunds this variation was determined to be a minor variation (allele ratio; del:ins = 6:2, Table 4). To confirm the consensus sequence in miniature Dachshunds, we sequenced BRCA2 DNA from an additional 32 blood samples, and the assembled allele ratio was del:ins = 30:42 (Table 4).

We established a PCR analysis method for canine BRCA2 in order to determine the base sequence from cDNA and genomic DNA, and to detect splicing variants. We identified novel splicing variants in normal canine testes. The functions of these splicing variants were not assessed in this study; nevertheless, these results indicated that the established method was a useful tool for detecting splicing variants.

We also defined the consensus sequence using methods established and described here. During the definition of the consensus BRCA2 ORF, we identified three novel (516 T > C, 2428 T > G, and 8257 T > C) and three reported (1366 T > G, 2609A > C and 4481A > C) single nucleotide variations and two reported insertion/deletion polymorphisms (7126ins/delGTT and 10204ins/delAAA) (Accession numbers: AB622997, AB622998 and AB622999) [11,12,15,18]. The variations 1366 T > G (C386W), 2609A > C (K801Q), 4481A > C (T1425P), and 10204ins/delAAA (M3332IV) are located in the histone acetyltransferase domain, the FANCG binding domain, BRC repeat 3, and nuclear localization signal 2, respectively [13,19-21]. The effects of these variations on BRCA2 function were not understood, with the exception of 10204insAAA; nuclear localization signal 2 harboring the 10204insAAA variation showed enhanced nuclear localization [13]. The other nonsynonymous variations were not located in previously known functional domains.

We identified four variations (1366 T > G, 2609A > C, 7126ins/delGTT, and 10204ins/delAAA), in which the allele frequency of minor variations in genomic DNA from normal blood samples was very high (28.5-37.5%). Such frequent variations in the BRCA2 gene have not been reported in other species. These highly frequent variations thus appear to be a canine BRCA2-specific feature, and should be considered when studying canine BRCA2. These four variations were found in the homozygous state in some blood samples from elderly tumor-free dogs. Homozygous mutations in BRCA2 are assumed to be embryonic-lethal mutations or responsible for Fanconi anemia, which is characterized by bone marrow failure, developmental abnormalities, and predisposition to cancer [22,23]. Thus, these four variations were probably neutral variations, although the 10204insAAA variation is reportedly a candidate malignant mutation in dogs [11].

In this study, we established a PCR analysis method and defined the consensus sequence of BRCA2 ORF to identify canine BRCA2 mutations. Using these methods, we are now able to perform BRCA2 mutation analysis and search for abnormal BRCA2 splicing variants from mammary tumors in dogs, as is done in human cases.

### Methods

Two ovaries (from two mongrel dogs), six testes (from a mongrel dog and five Beagles), a mammary gland (from a female Beagle) and 56 blood samples (Table 4) from tumor-free dogs were kindly provided by Dr. Takashi Kubo and Dr. Go Honda. All experimental procedures were approved by and conducted in accordance with the Guidelines for Institutional Laboratory Animal Care and Use of the School of Veterinary Medicine at Kitasato University, Japan (Approval Number: 11-065).

Total RNA was isolated from ovaries, six testes, and one mammary gland, which each were stored in RNAlater solution (Life Technologies, Grand Island, NY), using a TRIzol and PureLink RNA micro kit (Life Technologies). First-strand cDNA was synthesized from 1-5 μg of total RNA using SuperScript III (Life Technologies). Genomic DNA samples were extracted using a Gentra Puregene tissue kit (Qiagen, Hilden, Germany).

For PCR amplification of the full-length canine BRCA2 ORF from cDNA and all 27 exons from genomic DNA, we designed 14 and 20 primer sets, respectively (Table 1). We also designed five primer sets to detect splicing variants and a primer sets to confirm a novel BRCA2 transcript that lacked most of exon 11 using nested PCR (Table 1). Each reaction mixture contained 0.1 μL of first-strand cDNA reaction products or 10-50 ng of genomic DNA as a template, each forward and reverse primer at 300 nM, 200 μM dNTPs, 0.02 U of KOD FX DNA polymerase (Toyobo, Japan), and 1× PCR buffer, which was supplied with the enzyme, in a total volume of 10 μL. PCR included one cycle of 2 min at 94°C, followed by 35 cycles of 10 s at 98°C, 30 s at the optimal temperature shown in Table 1 the optimal time shown in Table 1 at 68°C, and a final extension step of 7 min at 68°C. PCR products were treated with shrimp alkaline phosphatase (Affymetrix, Santa Clara, CA) and Exonuclease I (New England BioLabs, Beverley, MA) before sequencing, which was performed with the BigDye Terminator Cycle Sequencing kit Version 3.1 and a ABI PRISM 3100-Avant DNA sequencer (Life Technologies). Direct DNA sequencing was performed at least twice for each amplicon. When we attempted to define the consensus canine BRCA2 ORF sequence, two or three amplicons from each sample were sequenced. Because we detected only three electropherogram patterns among the PCR products with the insertion/deletion mutation sites, we were able to determine the heterozygous insertion/deletion mutations by direct sequencing (Additional file 1: Figure S1).

Nucleotide base sequences of primers

Comparison between our sequences from the canine BRCA2 open reading frame with registered sequences

N. D., not determined.

aNucleotide and amino acid location is based on AB043895.5

bFull-length sequence was determined by cDNA sequencing (Accession number; AB622997, AB622998 and AB622999). When frequencies of major and minor alleles were nearly equal or were inconsistent with reported sequences, alleles were further analyzed by partial sequencing of blood genome DNA from 20-23 dogs (Table 3 and 4).

cSequence from one dog was regarded as one allele because allele type analyses have not been described in these reports. The study by Hsu et al. examined three dogs, while others studied only one dog. Accession numbers for sequences reported by Ochiai et al., the Genome Project, Bignell et al. and Hsu et al. are AB043895.5, NC_006607.2, Z75664 and FJ464397-FJ464400, respectively.

Genotype frequency of four variations in normal blood samples

aNucleotide and amino acid locations are based on AB043895.5

Information of blood samples and allele type of the four frequently found variations

aNucleotide and amino acid locations are based on AB043895.5

"-" indicates not determined.

### Specimens

Two ovaries (from two mongrel dogs), six testes (from a mongrel dog and five Beagles), a mammary gland (from a female Beagle) and 56 blood samples (Table 4) from tumor-free dogs were kindly provided by Dr. Takashi Kubo and Dr. Go Honda. All experimental procedures were approved by and conducted in accordance with the Guidelines for Institutional Laboratory Animal Care and Use of the School of Veterinary Medicine at Kitasato University, Japan (Approval Number: 11-065).

### Total RNA and genomic DNA extraction, and preparation of cDNA

Total RNA was isolated from ovaries, six testes, and one mammary gland, which each were stored in RNAlater solution (Life Technologies, Grand Island, NY), using a TRIzol and PureLink RNA micro kit (Life Technologies). First-strand cDNA was synthesized from 1-5 μg of total RNA using SuperScript III (Life Technologies). Genomic DNA samples were extracted using a Gentra Puregene tissue kit (Qiagen, Hilden, Germany).

### PCR and sequencing

For PCR amplification of the full-length canine BRCA2 ORF from cDNA and all 27 exons from genomic DNA, we designed 14 and 20 primer sets, respectively (Table 1). We also designed five primer sets to detect splicing variants and a primer sets to confirm a novel BRCA2 transcript that lacked most of exon 11 using nested PCR (Table 1). Each reaction mixture contained 0.1 μL of first-strand cDNA reaction products or 10-50 ng of genomic DNA as a template, each forward and reverse primer at 300 nM, 200 μM dNTPs, 0.02 U of KOD FX DNA polymerase (Toyobo, Japan), and 1× PCR buffer, which was supplied with the enzyme, in a total volume of 10 μL. PCR included one cycle of 2 min at 94°C, followed by 35 cycles of 10 s at 98°C, 30 s at the optimal temperature shown in Table 1 the optimal time shown in Table 1 at 68°C, and a final extension step of 7 min at 68°C. PCR products were treated with shrimp alkaline phosphatase (Affymetrix, Santa Clara, CA) and Exonuclease I (New England BioLabs, Beverley, MA) before sequencing, which was performed with the BigDye Terminator Cycle Sequencing kit Version 3.1 and a ABI PRISM 3100-Avant DNA sequencer (Life Technologies). Direct DNA sequencing was performed at least twice for each amplicon. When we attempted to define the consensus canine BRCA2 ORF sequence, two or three amplicons from each sample were sequenced. Because we detected only three electropherogram patterns among the PCR products with the insertion/deletion mutation sites, we were able to determine the heterozygous insertion/deletion mutations by direct sequencing (Additional file 1: Figure S1).

Nucleotide base sequences of primers

Comparison between our sequences from the canine BRCA2 open reading frame with registered sequences

N. D., not determined.

aNucleotide and amino acid location is based on AB043895.5

bFull-length sequence was determined by cDNA sequencing (Accession number; AB622997, AB622998 and AB622999). When frequencies of major and minor alleles were nearly equal or were inconsistent with reported sequences, alleles were further analyzed by partial sequencing of blood genome DNA from 20-23 dogs (Table 3 and 4).

cSequence from one dog was regarded as one allele because allele type analyses have not been described in these reports. The study by Hsu et al. examined three dogs, while others studied only one dog. Accession numbers for sequences reported by Ochiai et al., the Genome Project, Bignell et al. and Hsu et al. are AB043895.5, NC_006607.2, Z75664 and FJ464397-FJ464400, respectively.

Genotype frequency of four variations in normal blood samples

aNucleotide and amino acid locations are based on AB043895.5

Information of blood samples and allele type of the four frequently found variations

aNucleotide and amino acid locations are based on AB043895.5

"-" indicates not determined.

### Competing interests

The authors declare that they have no competing interests.

### Authors' contributions

YY outlined the design of and coordinated the study, performed the experiments, and drafted the manuscript. MM, K. Ochiai, K. Orino, and WK participated in the design of the study and interpretation of the data and helped to draft the manuscript. K. Okuda, TT, SC, SA, TO, and MB performed several experiments.

### Supplementary Material

Figure S1. Example of an electropherogram by direct sequencing from PCR products having the insertion/deletion mutation (7126ins/delGTT).

Click here for file



# SUPPLEMENTAL FILE 1: 1756-0500-5-173.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)