# MAIN TEXT

## BRCA1, BRCA2, PALB2, and CDKN2A Mutations in Familial Pancreatic Cancer (FPC): A PACGENE Study

### Abstract

PurposeFamilial Pancreatic Cancer (FPC) kindreds contain at least two affected first-degree relatives (FDR). Comprehensive data are needed to assist clinical risk assessment and genetic testing.MethodsGermline DNA samples from 727 unrelated probands with positive family history (521 met criteria for FPC) were CLIA-tested for mutations in BRCA1 and BRCA2 (including analysis of deletions and rearrangements), PALB2, and CDKN2A. We compared prevalence of deleterious mutations between FPC probands and non-FPC probands (kindreds containing at least two affected biologic relatives, but not FDR). We also examined the impact of family history of breast and ovarian cancer and melanoma.ResultsPrevalence of deleterious mutations (excluding variants of unknown significance) among FPC probands was: BRCA1, 1.2%; BRCA2, 3.7%; PALB2, 0.6%; CDKN2A, 2.5%. Four novel deleterious mutations were detected. FPC probands carry more mutations in the four genes (8.0%) than non-FPC probands (3.5%) (odds ratio=2.40, 95% CI=(1.06, 5.44), p=0.03). The probability of testing positive for deleterious mutations in any of the four genes ranges up to 10.4%, depending upon family history of cancers. BRCA2 and CDKN2A account for the majority of mutations in FPC.ConclusionGenetic testing of multiple relevant genes in probands with a positive family history is warranted, particularly for FPC.

### Introduction

Pancreatic adenocarcinoma is currently the fourth leading cause of cancer death in the United States1 and is anticipated to be the second leading cause by the year 2020.2 Despite medical advances, overall five-year survival rates have not significantly changed, with the vast majority of diagnoses made at advanced stages of disease.1 The relatively lower incidence of pancreatic cancer compared to other malignancies makes it challenging to conduct the large-scale studies that are needed to determine appropriate early screening measures.3 It is critical to identify populations at high risk who may potentially benefit from earlier detection, with concomitant implications for intervention or therapy.4

Family history studies suggest that approximately 5–10% of pancreatic adenocarcinoma has a strong hereditary basis, and familial pancreatic cancer (FPC) is thought to be genetically heterogeneous. FPC, defined as a kindred with at least two affected first degree relatives (FDR), describes an established entity of inherited pancreatic cancer.5 Our knowledge of the genetic basis of FPC largely arises from observed increased pancreatic cancer risk in those with hereditary malignant syndromes. A number of candidate susceptibility genes have been proposed to date, and four genes, BRCA1, BRCA2, PALB2, and CKDN2A, appear to account for the majority of known genetic causes of FPC.5

Individuals carrying germline mutations in BRCA1 and BRCA2 demonstrate increased risk for developing other malignancies, including pancreatic cancer.6–8 Although germline mutations in BRCA1 and BRCA2 are associated with Hereditary and Ovarian Breast Cancer (HBOC) syndrome, this increased risk of pancreatic malignancy can also manifest in families who do not meet criteria for HBOC.9,10 In the initial studies by the Breast Cancer Linkage Consortium, the relative risk of developing pancreatic cancer was increased by a mean of 2.26-fold for BRCA16 and 3.5-fold for BRCA27 carriers. However, risk ascertainment was obtained in hereditary breast cancer families rather than families ascertained through pancreatic cancer, thus likely underestimating the actual risk for pancreatic cancer in BRCA1 and BRCA2 mutation carriers. While initial investigations cite the presence of pancreatic cancer in HBOC families with deleterious BRCA1 mutations8,11,12, no germline BRCA1 mutations were identified in a series of pancreatic cancer families.13 Thus, available evidence indicates that when individuals are ascertained through FPC kindreds, the risk of pancreatic cancer in BRCA1 mutation carriers is less than it is for BRCA2 mutation carriers. Murphy et al. reported 17% prevalence of BRCA2 mutations among affected individuals from 26 European FPC kindreds containing three or more affected members with pancreatic cancer.14 Subsequent studies of individuals with pancreatic cancer from families meeting FPC criteria (two or more affected first-degree relatives) estimated BRCA2 prevalence ranging between 6–10%.15 Furthermore, the ethnic variation of the population influences mutation prevalence rates of BRCA1 and BRCA2 and should be recognized when interpreting the literature. For example, among Ashkenazi Jews, similar mutation prevalences were observed for both BRCA1 and BRCA2.16,17 The role of BRCA1 and BRCA2 mutations in larger samples of FPC kindreds remains to be elucidated. Determination of BRCA mutation status has potential therapeutic implications, as those carrying such mutations have been shown to benefit from therapies that inhibit poly[ADP ribose]polymerase (PARP inhibitors).18,19

PALB2, a co-localizer and partner gene to BRCA2, is also proposed to be involved in FPC. PALB2 was originally identified as a novel protein that complexes with BRCA2, leading to its stability and facilitating DNA repair.20 Bi-allelic germline mutations in PALB2 lead to the development of Fanconi anemia,21 while mono-allelic mutations increase breast cancer susceptibility. While searching for candidate pancreatic cancer susceptibility genes, Jones et al. discovered an inherited deleterious PALB2 mutation coupled with a second inactivating hit in a patient with pancreatic cancer.22 Further PALB2 sequencing in a cohort of 96 FPC patients showed that 3–4% carried deleterious mutations. With the exception of one European study,23 subsequent studies have reported a lower prevalence of PALB2 mutations in FPC.24,25 Studies with large sample sizes and unbiased selection criteria are needed to provide a more complete understanding of the role of BRCA1, BRCA2, and PALB2 in pancreatic cancer susceptibility.

The CDKN2A gene located on chromosome 9p21 encodes the p16 protein, an important cell cycle regulator that inhibits cyclins, thus preventing premature transition from G1 to the S phase and serving as an important tumor suppressor. Germline mutations in CDKN2A are responsible for early-onset melanomas often associated with the development of Familial Atypical Multiple Mole Melanoma (FAMMM) syndrome. Increased risk for pancreatic cancer development was observed in cases of CDKN2A-associated familial melanoma.26 Examining CDKN2A in German FPC patients, Bartsch et al. found that mutations were rare, unless patients had concurrent melanoma.27 Studies performed in other regions of Europe ultimately demonstrated the occurrence of CDKN2A mutations in FPC kindreds without melanoma, with prevalences ranging from 20–30%.28 Such elevated rates, however, were likely influenced by specific founder mutations; one study also included patients of other familial cancer syndromes. In a large study in the United States of CDKN2A germline mutations among 1,537 unselected mostly sporadic pancreatic cancer cases, McWilliams et al.29 found a much lower overall prevalence of CDKN2A mutations (0.6%), with higher rates in the subset of cases with affected first degree relatives (FDR); the limited family history data in this study left open the question of germline CDKN2A mutations in patients with FPC, particularly families without evidence of FAMMM.

In order to better inform genetic counseling of patients and families through more precise prevalence estimates, we comprehensively analyzed BRCA1, BRCA2, PALB2, and CDKN2A in a large cohort of FPC kindreds ascertained via the multicenter Pancreatic Cancer Genetic Epidemiology (PACGENE) Consortium.

### Material and Methods

Institutional review board approval was obtained at all participating sites, and written consent was obtained from all probands in order to be included in the study. PACGENE Consortium sites had assembled 2,853 unrelated kindreds containing at least two family members affected with pancreatic cancer from which subjects for this study sample were drawn (i.e., not all probands had available biospecimens). Ascertainment and recruitment methods were previously described.30 Probands were biopsy-proven or clinically documented to have a diagnosis of pancreatic adenocarcinoma. We identified 727 unrelated kindreds that contained at least two biologically related family members affected with pancreatic cancer and for which a proband DNA sample was available. PACGENE sites include Mayo Clinic (Rochester, Minnesota) (n=341), Johns Hopkins University (Baltimore, Maryland) (n=107), Barbara Ann Karmanos Cancer Institute (Detroit, Michigan) (n=45), University of Toronto (Ontario, Canada) (n=131), and Dana Farber Cancer Institute (Boston, Massachusetts) (n=58). Subjects from kindreds similar to the PACGENE sites were contributed from the University of Texas MD Anderson Cancer Center (Houston, TX) (n=38) and University of Utah (Salt Lake City, UT) (n=7). In general, probands were unselected for hereditary cancer syndrome patterns or whether genetic mutation status in one of the four tested genes may have been previously known. Some potential probands with known mutations in one of the genes being tested may have been excluded by some sites, but this was not systematic. DNA was extracted at each contributing site from peripheral blood or buccal cell samples. Baseline demographic and family history information were available, typically self-reported.

Of the 727 kindreds in this study, a subset of 521 met criteria for FPC (having two FDRs with pancreatic cancer), and the remaining 206 were familial non-FPC cases (these kindreds contained at least two affected biologic relatives, but not FDR). A small proportion (1.2%) of the total sample also had a personal history of melanoma; among females, personal history of breast and ovarian cancers occurred in 6.4% and 0.6%, respectively. All subjects were assigned a unique identifier, and all samples were de-identified during analysis by Myriad Genetic Laboratories, Inc.

Re-sequencing analysis for germline mutations in BRCA1, BRCA2, PALB2, and CDKN2A and large rearrangement analysis for BRCA1 and BRCA2 was conducted by Myriad Genetic Laboratories, Inc. Full-sequence DNA analysis of these four genes and breakpoint analysis for five large genomic rearrangements in BRCA1 (exon13del3835bp, exon13ins6kb, exon14-20del26kb, exon22del510bp, and exon8-9del7.1kb) were performed using previously described methods.31,32 All testing adhered to Clinical Laboratory Improvement Amendments (CLIA) requirements.

Briefly, for each of the four genes, full-gene sequencing was performed in both forward and reverse directions. The non-coding intronic regions of each gene that are analyzed do not extend more than 20 base pairs proximal to the 5′ end and ten base pairs distal to the 3′ end of each exon. Aliquots of subjects’ DNA are each subjected to polymerase chain reaction (PCR) amplification to generate exon-specific amplicons that can be directly sequenced. The amplified products are each sequenced in forward and reverse directions using fluorescent dye-labeled sequencing primers. Electropherogram tracings of each amplicon are analyzed by a proprietary computer-based review system followed by visual inspection and confirmation of all clinically significant variants. Genetic variants are detected by comparison with a consensus wild-type sequence constructed for each gene. All potential clinically significant variants are independently confirmed by repeated PCR amplification of the indicated gene region(s) and sequence determination as above. In addition, large-rearrangement analysis of BRCA1 and BRCA2 was performed for each sample using the BRACAnalysis Rearrangement Test (BART), a quantitative multiplex endpoint PCR assay that detects all large deletions and duplications across the coding regions and promoters of BRCA1 and BRCA2 using a quantitative endpoint multiplex PCR assay. BART uses a set of 12 reactions comprising 11 multiplex PCR reactions containing 9 to 14 amplicons per multiplex, and one contamination detection reaction. These amplicons cover coding exons, promoters, and flanking regions for BRCA1 and BRCA2.33

Deleterious (including suspected deleterious) mutations, variants of uncertain significance (VUS), and single nucleotide polymorphisms (SNPs) in the four genes were detected and distinguished for the data analysis. Deleterious (including suspected deleterious) mutations, VUS, and SNPs were defined as those established in the current published literature as well as those previously catalogued in Myriad’s established genetic mutation database for these genes. Novel, previously unreported mutations discovered in this study were defined as those not present in the Myriad gene mutation database. All variants were classified in accordance with the recommendations of the American College of Medical Genetics and Genomics (ACMG) for standards in the interpretation and reporting of sequence variations.34

Prevalence of deleterious mutations and VUS of the four genes studied were compared between individuals of FPC and familial non-FPC kindreds. We focused on the sub-analysis of the probands who had complete results on all four genes to better gain insight for genetic counseling for BRCA1, BRCA2, PALB2, and CDKN2A. Descriptive statistics and mutation rates were calculated. Comparisons of the mutation prevalence between groups were measured using either chi-square or Fisher’s exact tests, depending on sample sizes. All statistical analyses were conducted using SAS 9.3 (SAS Institute, Cary, NC).

### Subjects

Institutional review board approval was obtained at all participating sites, and written consent was obtained from all probands in order to be included in the study. PACGENE Consortium sites had assembled 2,853 unrelated kindreds containing at least two family members affected with pancreatic cancer from which subjects for this study sample were drawn (i.e., not all probands had available biospecimens). Ascertainment and recruitment methods were previously described.30 Probands were biopsy-proven or clinically documented to have a diagnosis of pancreatic adenocarcinoma. We identified 727 unrelated kindreds that contained at least two biologically related family members affected with pancreatic cancer and for which a proband DNA sample was available. PACGENE sites include Mayo Clinic (Rochester, Minnesota) (n=341), Johns Hopkins University (Baltimore, Maryland) (n=107), Barbara Ann Karmanos Cancer Institute (Detroit, Michigan) (n=45), University of Toronto (Ontario, Canada) (n=131), and Dana Farber Cancer Institute (Boston, Massachusetts) (n=58). Subjects from kindreds similar to the PACGENE sites were contributed from the University of Texas MD Anderson Cancer Center (Houston, TX) (n=38) and University of Utah (Salt Lake City, UT) (n=7). In general, probands were unselected for hereditary cancer syndrome patterns or whether genetic mutation status in one of the four tested genes may have been previously known. Some potential probands with known mutations in one of the genes being tested may have been excluded by some sites, but this was not systematic. DNA was extracted at each contributing site from peripheral blood or buccal cell samples. Baseline demographic and family history information were available, typically self-reported.

Of the 727 kindreds in this study, a subset of 521 met criteria for FPC (having two FDRs with pancreatic cancer), and the remaining 206 were familial non-FPC cases (these kindreds contained at least two affected biologic relatives, but not FDR). A small proportion (1.2%) of the total sample also had a personal history of melanoma; among females, personal history of breast and ovarian cancers occurred in 6.4% and 0.6%, respectively. All subjects were assigned a unique identifier, and all samples were de-identified during analysis by Myriad Genetic Laboratories, Inc.

### Mutation Analysis

Re-sequencing analysis for germline mutations in BRCA1, BRCA2, PALB2, and CDKN2A and large rearrangement analysis for BRCA1 and BRCA2 was conducted by Myriad Genetic Laboratories, Inc. Full-sequence DNA analysis of these four genes and breakpoint analysis for five large genomic rearrangements in BRCA1 (exon13del3835bp, exon13ins6kb, exon14-20del26kb, exon22del510bp, and exon8-9del7.1kb) were performed using previously described methods.31,32 All testing adhered to Clinical Laboratory Improvement Amendments (CLIA) requirements.

Briefly, for each of the four genes, full-gene sequencing was performed in both forward and reverse directions. The non-coding intronic regions of each gene that are analyzed do not extend more than 20 base pairs proximal to the 5′ end and ten base pairs distal to the 3′ end of each exon. Aliquots of subjects’ DNA are each subjected to polymerase chain reaction (PCR) amplification to generate exon-specific amplicons that can be directly sequenced. The amplified products are each sequenced in forward and reverse directions using fluorescent dye-labeled sequencing primers. Electropherogram tracings of each amplicon are analyzed by a proprietary computer-based review system followed by visual inspection and confirmation of all clinically significant variants. Genetic variants are detected by comparison with a consensus wild-type sequence constructed for each gene. All potential clinically significant variants are independently confirmed by repeated PCR amplification of the indicated gene region(s) and sequence determination as above. In addition, large-rearrangement analysis of BRCA1 and BRCA2 was performed for each sample using the BRACAnalysis Rearrangement Test (BART), a quantitative multiplex endpoint PCR assay that detects all large deletions and duplications across the coding regions and promoters of BRCA1 and BRCA2 using a quantitative endpoint multiplex PCR assay. BART uses a set of 12 reactions comprising 11 multiplex PCR reactions containing 9 to 14 amplicons per multiplex, and one contamination detection reaction. These amplicons cover coding exons, promoters, and flanking regions for BRCA1 and BRCA2.33

Deleterious (including suspected deleterious) mutations, variants of uncertain significance (VUS), and single nucleotide polymorphisms (SNPs) in the four genes were detected and distinguished for the data analysis. Deleterious (including suspected deleterious) mutations, VUS, and SNPs were defined as those established in the current published literature as well as those previously catalogued in Myriad’s established genetic mutation database for these genes. Novel, previously unreported mutations discovered in this study were defined as those not present in the Myriad gene mutation database. All variants were classified in accordance with the recommendations of the American College of Medical Genetics and Genomics (ACMG) for standards in the interpretation and reporting of sequence variations.34

### Data Analysis

Prevalence of deleterious mutations and VUS of the four genes studied were compared between individuals of FPC and familial non-FPC kindreds. We focused on the sub-analysis of the probands who had complete results on all four genes to better gain insight for genetic counseling for BRCA1, BRCA2, PALB2, and CDKN2A. Descriptive statistics and mutation rates were calculated. Comparisons of the mutation prevalence between groups were measured using either chi-square or Fisher’s exact tests, depending on sample sizes. All statistical analyses were conducted using SAS 9.3 (SAS Institute, Cary, NC).

### Results

Baseline demographic characteristics of the 727 probands included in this study are shown in Table 1. Among these, 521 met criteria for FPC while the remaining 206 were classified as familial non-FPC. A slight majority of probands were males (50.9%), and median age of diagnosis was 65 years (range 20–95 years). The sample was largely White/Caucasian (87.3%), and 43 (8.0%) were of Ashkenazi Jewish descent among the 538 who self-reported this information. A majority of kindreds (70.4%) contained two affected members with pancreatic cancer, 19.3% reported 3 affected individuals, and 10.3% contained 4 or more affected individuals.

Table 2 summarizes the deleterious mutations, VUS, and SNPs in the four genes for this sample, highlighting those which were not previously established in the current literature. The four novel deleterious mutations detected were: BRCA2 6224insT, PALB2 E837X (2509G>T), PALB2 W1038X (3113G>A), and CDKN2A 286delG. Novel VUS detected were: BRCA1 H1860Q (5699C>A); BRCA2 S538N (1841G>A), T1586I (4985C>T), and dup exon 1; PALB2 E1018D (3054G>T), E892K (2674G>A), I887S (2660T>G), P1009S (3025C>T), P65L (194C>T), P806L (2417C>T), S578G (1732A>G), Y334D (1000T>G); and CDKN2A G101R (301G>A), L65P (194T>C), and T18P (52A>C). These novel VUS, particularly in PALB2, have been classified as incidental findings and not likely related to the pathogenesis of FPC at this time.35

Table 3 summarizes germline mutation prevalences in the subset of 716 probands who had results for all four genes tested (Tables of all results in the full sample and by gene are provided in Supplemental Data: Table S1). Results in Table 3 are stratified by deleterious mutations and VUS among probands from FPC kindreds and probands from kindreds that did not strictly meet the FPC criterion of containing at least two affected FDR (familial non-FPC). Gene for gene, probands from FPC kindreds carry more deleterious mutations than those from familial non-FPC kindreds. The probability that a proband carried a deleterious mutation in any of these genes was 8.0% and 3.5% in FPC versus familial non-FPC probands, respectively (odds ratio=2.40, 95% confidence interval (1.06, 5.44), p=0.03). The aggregate prevalence is 48/716 (6.7%) for all cases with any positive family history. Overall, deleterious mutations in BRCA2 and CDKN2A were more prevalent compared to either BRCA1 or PALB2.

The vast majority of probands who did test positive for a mutation carried a mutation in only one of the four genes. Only two individuals whose only personal cancer history was that of pancreatic cancer had multiple mutations: one proband carried two novel mutations in PALB2: E837X (2509G>T) (classified as deleterious) and P806L (2417C>T) (classified as VUS); another proband from a familial non-FPC kindred carried a mutation in both BRCA1 (187delAG) and BRCA2 (6174delT). Conversely, two probands who had malignancies in addition to pancreatic cancer tested positive for one mutation each: one proband had breast and ovarian in addition to pancreatic cancer and was found to carry 816delGT in BRCA1; another proband had breast cancer and melanoma in addition to pancreatic cancer and was found to carry V932M (2794G>A) in PALB2. The number of individuals affected with pancreatic cancer in a kindred did not correlate with the prevalence of deleterious mutations in FPC kindreds (ANOVA p=.97) (Shown in Supplemental Data: Table S2).

Table 4 displays germline mutation prevalences in the subset of 716 probands who had results for all four genes tested, stratified by deleterious mutations and VUS and by whether they also had family history of breast cancer, ovarian cancer, or melanoma. As expected, probands with a family history of breast cancer were more likely to test positive for deleterious mutations in BRCA1 (1.9%) or BRCA2 (4.2%), probands with a family history of ovarian cancer were more likely to test positive for deleterious mutations in BRCA1 (5.2%) and BRCA2 (5.2%), and probands with a family history of melanoma were more likely to test positive for deleterious mutations in CDKN2A (7.8%). Overall, the probability that a proband with a family history of any of these three cancers would test positive for a deleterious mutation in any of the four genes is 8.7%. Similar results for probands when family history is restricted to FDR are shown in Supplemental Data (Table S3): those data show that a proband with a family history in a FDR of any of these three cancers has an overall 9.5% probability of testing positive for a deleterious mutation in any of the four genes. To facilitate genetic counseling, we have aggregated a summary of our data showing the distributions of the probabilities of deleterious mutations by various cancer family histories in Figure 1. As can be seen, BRCA2 and CDKN2A constitute the majority of deleterious mutations across cancer family histories. Probands with pancreatic cancer who have a family member with ovarian cancer have 10.4% probability of testing positive for a deleterious mutation in BRCA1 or BRCA2. Probands with melanoma in their family history have a 10.4% probability of testing positive for CDKN2A or BRCA2 mutations. Interestingly, 7/14 (50%) of patients who carried CDKN2A mutations did not have a personal or family history of melanoma. Of the six BRCA1 mutation carriers, 1 (16.7%) had no personal or family history of breast cancer, and 2/6 (33.3%) had no personal or family history of ovarian cancer. Similarly, the numbers among the 25 BRCA2 mutation carriers were 14 (56%) and 21 (84%), respectively. For PALB2, one of four (25%) and none of the mutation carriers had a personal or family history of breast or ovarian cancer, respectively.

We also examined age at onset differences by mutation carrier status among 710 probands who had all four gene tests and available age data. Forty-five carried deleterious mutations and were younger than the others (p=0.03); median ages were 60 (range 42–93) and 65 (20–95), respectively.

### Discussion

In this large study, we provide a comprehensive analysis of germline mutations occurring in the four genes, BRCA1, BRCA2, PALB2, and CDKN2A, among familial pancreatic cancer probands. With the exception of one FPC proband and one familial non-FPC proband, the vast majority of tested individuals carry only one germline mutation in these four genes. We found that 8% of probands who have a FDR with pancreatic cancer (and therefore meet the definition of FPC) harbor a deleterious mutation in one of these four genes and that even probands who have a biological relative other than FDR with pancreatic cancer may carry a deletious mutation, although with significantly less probability. We demonstrated that these genes together account in total for approximately 5–10% of deleterious mutations in FPC. Overall, any proband with a positive family history of pancreatic cancer has a 6.7% probability of carrying a deleterious mutation in one of the genes. Mutations in BRCA2 and CDKN2A were detected more often than those in BRCA1 and PALB2, consistent with the published literature. We also found a younger age of onset among probands who carried a mutation in one of the four genes. Our study confirms and highlights the genetic heterogeneity of FPC. Thus, when genetic testing of probands is considered, multiple genes will need to be evaluated.

When family history of breast or ovarian cancer, or melanoma is considered, there are varying ranges of probabilities; it is of interest that a proband with a family history of pancreatic cancer and any of the three other cancers has an 8.7% probability of carrying a mutation. As familial pancreatic cancer probands are increasingly referred for genetic risk assessment, we aggregated in Figure 1 selected family history scenarios from our data that will help inform the probability of genetic test outcomes.

With respect to genetic testing of probands, our data can inform the strategy to identify particular FPC individuals as candidates for genetic testing and whose families could potentially benefit from genetic risk assessment. We found that, gene for gene, significantly more deleterious mutations were found in FPC kindreds than in those of familial non-FPC kindreds. As such, the yield of identifying a mutation would be greatest among those whose kindreds meet the criteria for FPC. Interestingly, the number of family members affected with pancreatic cancer in a kindred did not correlate with the probability of detecting deleterious mutations (Supplemental Data: Table S2). We could not confidently explore this relationship in the familial non-FPC cases due to the smaller number of mutations detected.

Previous studies have emphasized the importance of family history in pancreatic cancer risk12,36 and increased incidence of early pancreatic lesions detected via early screening measures.37 While our findings could lend promise towards use of genetic testing in early pancreatic cancer screening,38 many questions remain on how to appropriately translate this into the clinical setting for genetically high risk individuals.3

In addition to informing genetic counseling, this report provides perhaps the most comprehensive mutation analysis of PALB2 and CDKN2A in the familial setting. We utilized conventional methods and available databases from Myriad Genetic Laboratories as well as the research community at large to determine the classification of deleterious mutations, VUS, and SNPs. We identified four novel deleterious mutations and 15 VUS among these four genes. It is of interest that half of the novel VUS were seen in PALB2, and that three-fourths of all VUS detected were seen in CDKN2A and PALB2. It may be that our classification criteria are more conservative, as there is limited knowledge of PALB2 mutations and pancreatic cancer due to it being the least characterized of the four genes. Similarly, the experience of CDKN2A has been focused on probands with melanoma ascertained through FAMMM or familial melanoma kindreds. As can be expected, studies of familial melanoma contributed to the classification of deleterious mutations in CDKN2A; our study focused on CDKN2A mutations ascertained through familial pancreatic cancer. It is worth noting that half of the probands who carried CDKN2A deleterious mutations did not have a personal or family history of melanoma. Our data also provide a contrast to what is seen among sporadic patients with pancreatic cancer: among FPC probands, the prevalence of deleterious mutations is nearly 5-fold higher (2.5% vs 0.6%).29 The relatively large numbers of mutations and VUS detected, in both genes, warrant further research to determine if the VUS should be reclassified as deleterious mutation. However, in our study, we noted no significant difference of BRCA2 mutation prevalence between probands from FPC versus familial non-FPC kindreds. BRCA2 germline mutations have also been detected in sporadic pancreatic cancers where family pedigrees were not suggestive of an inherited predisposition.39 Taken together, varying penetrance may potentially explain the noted increased prevalence of deleterious CDKN2A mutations. Until further studies clarify these aspects, recognition of our limited knowledge of PALB2 and CDKN2A is important when counseling families presenting through familial pancreatic cancer and who may harbor mutations in these two genes for which the significance has yet to be elucidated.

We ascertained probands for this study through their diagnosis of pancreatic cancer and having a family history of pancreatic cancer. Analysis of family histories could qualify some of the families to meet criteria for HBOC and FAMMM, but a number of probands who tested positive for the mutations in genes associated with these cancer syndromes would not be considered to have these syndromes by cancer family history; this finding opens an opportunity to broaden the scope of these classic syndromes and for further characterization of the spectrum of cancer risk and penetrance estimates, or alternatively redefine pleiotropic manifestations of the genes.

The large number of probands with a family history of pancreatic cancer from multiple sites is a significant strength in estimating the prevalence of mutations in these four genes. Other strengths include the detailed information on personal and family history of breast and ovarian cancer and melanoma. The DNA samples were all tested under CLIA-standard conditions at the Myriad Genetics laboratory, which assured consistent quality processing protocols, clear criteria for mutation and variant assessment, and utilization of several mutation databases.

There are also several limitations. First, although BART was used, not all deletions and duplications were comprehensively tested for in BRCA1 and BRCA2. As well, duplications and deletions were not tested for in PALB2 and CDKN2A. Second, a number of probands had missing demographic information on Ashkenazi Jewish heritage, potentially important for further stratifying risk of pancreatic cancer development in this ethnic group. We did not have the data to adequately interrogate cancer risk in BRCA1 or BRCA2 mutation carriers of Ashkenazi Jewish heritage. Third, we did not test for mutations in mismatch repair genes associated with Lynch Syndrome. Although mutations in these genes confer increased risk for pancreatic cancer, the risk is more moderate compared to the four genes we did report5. Fourth, some of the sites may have excluded potential probands with already-known gene mutations. The estimated prevalences we present here are therefore underestimates. At this time, we cannot firmly fix the degree of underestimation because the exclusions were not systematic. Future studies should further identify other subject characteristics or risk factors that may assist in selecting appropriate affected individuals for genetic testing with the ultimate hope that this will enhance ongoing efforts toward an effective clinical strategy for screening high risk individuals for pancreatic cancer.40

In this comprehensive study of germline mutations in BRCA1, BRCA2, PALB2, and CDKN2A in a sample of probands with familial pancreatic cancer, we have confirmed genetic heterogeneity and that a greater proportion of mutations occur in BRCA2 and CDKN2A. Our data suggest there is a role for genetic testing in high risk FPC families, especially those containing at least two FDRs, supporting proposals made in previous pancreatic cancer screening guidelines.3 Further studies will elucidate the functional relevance of FPC genes as well as their potential interplay with complex intracellular pathways in the pathogenesis of pancreatic cancer.

### Supplementary Material



# SUPPLEMENTAL FILE 1: nihms632415.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: PMC4439391

[File available at: automated_output/BRCA2/20251125_175029/pmc_fulltext/25356972_supplements/PMC4439391]

