# MAIN TEXT

## Clinically Significant Unclassified Variants in BRCA1 and BRCA2 genes among Korean Breast Cancer Patients

### Abstract

PurposeUnclassified variants (UVs) of BRCA1 and BRCA2 genes are not defined as pathogenic for breast cancer, and their clinical significance currently remains undefined. Therefore, this study was conducted to identify potentially pathogenic UVs by comparing their prevalence between breast cancer patients and controls.Materials and MethodsA total of 328 breast cancer patients underwent BRCA1/2 genetic screening at the National Cancer Center of Korea. Genetic variants of BRCA genes that were categorized as unclassified according to the Breast Cancer Information Core database were selected based on allelic frequency, after which candidate variants were genotyped in 421 healthy controls. We also examined family members of the study participants. Finally, the effects of amino acid substitutions on protein structure and function were predicted in silico.ResultsGenetic tests revealed 33 UVs in BRCA1 and 47 in BRCA2. Among 15 candidates genotyped in healthy controls, c.5339T>C in BRCA1 and c.6029T>G, c.7522G>A in BRCA2 were not detected. Moreover, the c.5339T>C variant in the BRCA1 gene was detected in four patients with a family history of breast cancer. This nonsynonymous variant (Leu1780Pro) in the BRCA1 C-terminal domain was predicted to have an effect on BRCA1 protein structure/function.ConclusionThis study showed that comparison of genotype frequency between cases and controls could help identify UVs of BRCA genes that are potentially pathogenic. Moreover, ourfindings suggest that c.5339T>C in BRCA1 might be a pathogenic variant for patients and their families.

### Introduction

Breast cancer is the second-most common cancer among women in Korea, with an estimated incidence of 65.7 per 100,000 women per year [1]. Moreover, the incidence of breast cancer in Korea has been increasing annually, with relatively younger-aged women increasingly being affected.

Germline mutations of the BRCA1 and BRCA2 genes that encode truncated proteins are associated with a significantly increased risk of cancer in carriers [2-4]. The Korean Hereditary Breast Cancer (KOHBRA) study reported that 15.7% of patients with breast cancer who were tested for genetic mutation carried pathogenic mutations in BRCA genes. Additionally, breast cancer patients with a family history of breast or ovarian cancers showed a prevalence of BRCA mutations as high as 22.3% [5,6].

Mutation screening for BRCA genes has become a widely applied genetic test for cancer predisposition. Currently, the clinical significance of BRCA1/2 sequence variations can be interpreted according to several databases of genetic mutation, including the Breast Cancer Information Core (BIC; http://research.nhgri.nih.gov/bic/) and ClinVar (http://www.ncbi.nlm.nih.gov/clinvar/) [7-9]. However, a large portion of genetic variants of BRCA1 and BRCA2 genes are nontruncating, such as missense or potential splice site changes. Nevertheless, the contribution of these variants to cancer risk currently remains undefined. These unclassified variants (UVs) of BRCA genes have become a clinical issue for carriers because of their unknown clinical significance [10,11]. Some UVs that are found in highly conserved domains or splice sites have been predicted to be deleterious by in silico analyses. Moreover, many UVs are classified as neutral polymorphisms, and some are considered potentially deleterious. The effects of variants on biological function are difficult to assign based on functional assays of BRCA genes. To define the clinical significance of UVs, researchers have suggested various approaches and algorithms to determine whether UVs are deleterious or neutral for the biological function of proteins encoded by BRCA genes [12,13]. Many models based on statistical methods that combine clinical features or predicted gene function with informatics tools, such as Polymorphism Phenotyping (PolyPhen, http://genetics.bwh.harvard.edu/pph/
) or Sorting Intolerant from Tolerant (SIFT, http://blocks.fhcrc.org/sift/SIFT.html), have been suggested [14,15].

In this study, we investigated the prevalence of UVs in BRCA genes in a Korean population. To address the clinical significance of unclassified BRCA gene variants, we collected BRCA gene sequencing data from 328 breast cancer patients. Additionally, six selected UVs of the BRCA1 gene and nine of the BRCA2 gene were genotyped in 421 controls. We also examined the family history of variant carriers and tested BRCA genes of family members. This is the first report comparing the frequency of BRCA UVs in Korean breast cancer patients and healthy controls.

### Materials and Methods

Patients with histologically confirmed breast cancer were enrolled in this study from the genetic counseling clinic and underwent BRCA1/2 mutation testing between April 2008 and June 2015 at the National Cancer Center in Korea. A total of 328 patients who underwent genetic testing for BRCA1 and BRCA2 genes voluntarily participated in this study and agreed to provide the results of genetic testing. As control group, 421 healthy controls were recruited from individuals who visited the National Cancer Center as part of a cancer-screening program. All individuals who participated in this study signed an informed consent form that was approved by the Institutional Review Board of National Cancer Center Korea (IRB No. NCCNCS 13717).

Genomic DNA was extracted from the peripheral blood of participants using a QIAamp DNA Blood Mini kit (Qiagen, Valencia, CA) or a Chemagic DNA Blood 200 Kit (Chemagen, Baesweiler, Germany) according to the manufacturers’ instructions. Genetic testing of BRCA1 and BRCA2 genes was conducted by the Green Cross Company (Yongin, Korea) using a direct sequencing method. Briefly, amplified products were sequenced on an ABI 3500xl Analyzer (Applied Biosystems, Foster City, CA) using Bigdye Terminator v3.1 Cycle Sequencing Kits, and sequences were analyzed using the Sequencer v5.0 software. All genetic variants in BRCA1 and BRCA2 genes were categorized as pathogenic, unclassified, or polymorphic according to the BIC database. All mutations are described according to HUGO-approved systematic nomenclature (http://www.hgvs.org/mutnomen/). GenBank accession numbers NM_007294.3 for BRCA1 and NM_000059.3 for BRCA2 were used as reference sequences.

Large genomic rearrangements of BRCA1/2 genes were also tested using a multiplex ligation-dependent probe amplification (MLPA) assay for patients without pathogenic mutations of the BRCA genes.

Candidate variants were selected for further genotyping in healthy controls based on the frequency of variants of BRCA1/2 genes among cases. Variants were identified by TaqMan probe genotyping (Applied Biosystems) using a QuantStudio 7 Flex real-time PCR system. The reproducibility of genotyping results was confirmed by genotyping 10% of the samples in duplicate.

The effects of amino acid substitutions on protein structure and function were predicted using PolyPhen [16] and SIFT [17]. The tolerance score from SIFT and damaging score from PolyPhen-2 were used to predict the potential effects of UVs on the function of proteins encoded by BRCA genes. The structure of variant proteins was predicted using SWISSMODEL (http://swissmodel.expasy.org/) [18]. The BRCA1 C-terminal (BRCT) domain structure of BRCA1 (PDB entry: 4U4A) was used as a template for modeling.

### 1. Study population

Patients with histologically confirmed breast cancer were enrolled in this study from the genetic counseling clinic and underwent BRCA1/2 mutation testing between April 2008 and June 2015 at the National Cancer Center in Korea. A total of 328 patients who underwent genetic testing for BRCA1 and BRCA2 genes voluntarily participated in this study and agreed to provide the results of genetic testing. As control group, 421 healthy controls were recruited from individuals who visited the National Cancer Center as part of a cancer-screening program. All individuals who participated in this study signed an informed consent form that was approved by the Institutional Review Board of National Cancer Center Korea (IRB No. NCCNCS 13717).

### 2. Sequencing and genotyping of variants

Genomic DNA was extracted from the peripheral blood of participants using a QIAamp DNA Blood Mini kit (Qiagen, Valencia, CA) or a Chemagic DNA Blood 200 Kit (Chemagen, Baesweiler, Germany) according to the manufacturers’ instructions. Genetic testing of BRCA1 and BRCA2 genes was conducted by the Green Cross Company (Yongin, Korea) using a direct sequencing method. Briefly, amplified products were sequenced on an ABI 3500xl Analyzer (Applied Biosystems, Foster City, CA) using Bigdye Terminator v3.1 Cycle Sequencing Kits, and sequences were analyzed using the Sequencer v5.0 software. All genetic variants in BRCA1 and BRCA2 genes were categorized as pathogenic, unclassified, or polymorphic according to the BIC database. All mutations are described according to HUGO-approved systematic nomenclature (http://www.hgvs.org/mutnomen/). GenBank accession numbers NM_007294.3 for BRCA1 and NM_000059.3 for BRCA2 were used as reference sequences.

Large genomic rearrangements of BRCA1/2 genes were also tested using a multiplex ligation-dependent probe amplification (MLPA) assay for patients without pathogenic mutations of the BRCA genes.

Candidate variants were selected for further genotyping in healthy controls based on the frequency of variants of BRCA1/2 genes among cases. Variants were identified by TaqMan probe genotyping (Applied Biosystems) using a QuantStudio 7 Flex real-time PCR system. The reproducibility of genotyping results was confirmed by genotyping 10% of the samples in duplicate.

### 3. In silico analysis of UVs

The effects of amino acid substitutions on protein structure and function were predicted using PolyPhen [16] and SIFT [17]. The tolerance score from SIFT and damaging score from PolyPhen-2 were used to predict the potential effects of UVs on the function of proteins encoded by BRCA genes. The structure of variant proteins was predicted using SWISSMODEL (http://swissmodel.expasy.org/) [18]. The BRCA1 C-terminal (BRCT) domain structure of BRCA1 (PDB entry: 4U4A) was used as a template for modeling.

### Results

Table 1 shows the demographic features of breast cancer patients and controls. All patients were female and were diagnosed with histologically confirmed breast cancer, with the exception of four patients who were diagnosed with ovarian cancer. Patients in our study population were predominantly stage I, with only 11 patients categorized as stage IV. Additionally, more than 76% of patients had a family history of breast or ovarian cancer, including 11 with a family history of both breast and ovarian cancer.

All patients underwent genetic testing of BRCA1 and BRCA2 genes by the direct sequencing method. As shown in Tables 1 and 2, a total of 47 patients (14.3%) harbored 35 different deleterious mutations (frameshift or nonsense) of BRCA genes. To examine large genomic rearrangement of the BRCA genes, we performed MLPA assays in 196 patients. Only four patients showed deletions of a large genomic region.

Sequencing results showed that a total of 181 patients (55.2%) harbored UVs of BRCA genes. Among these, 33 kinds of UVs in the BRCA1 gene were detected in 127 patients, while 47 UVs in the BRCA2 gene were identified from 113 patients. Although these variants were already classified as having uncertain clinical importance, such a high frequency of variants among the population could weaken the significance of the association with cancer. Therefore, we excluded 14 UVs that were present at a frequency of more than 2% among the East-Asian population in the 1000 Genomes Phase 3 database (http://www.1000genomes.org/data). Seven UVs in BRCA1, including rs1799949, rs799912, rs16940, rs799916, rs1060915, rs3092994, and rs8176140, showed a minor allele frequency (MAF) of 37% in the East-Asian population. Furthermore, rs1801406, rs9534262, and rs4942486 in the BRCA2 gene showed reduced significance because their MAF was than 25%. Ultimately, 19 UVs that were detected in at least two patients among our cases were selected for further analysis; however, four UVs could not be genotyped owing to difficulties in probe design. Finally, six UVs in BRCA1 and nine in BRCA2 were further genotyped in 421 age-matched female controls. Among these, c.5339T>C in BRCA1 and c.6029T>G, c.7522G>A in BRCA2 were not detected in healthy controls (Table 3). The c.5339T>C were detected in four patients and c.6029T>G, c.7522G>A were detected in three patients with breast cancer. All three variants caused a substitution of amino acid sequence and c.5339T>C (Leu1780Pro) in BRCA1, and c.7522G>A (Gly2508Ser) in BRCA2 were predicted as damaging variants. In contrast, c.4883T>C and c.2566T>C in BRCA1 and c.2350A>G and c.8187G>T in BRCA2 showed a genotype frequency greater than 2% in the control group.

We examined the potential risk of three UVs that were not detected in 421 healthy controls. Fortunately, we were able to recruit family members of the proband harboring the c.5339T>C variant in the BRCA1 gene. As shown in the pedigree in Fig. 1A, two breast cancer patients in this family and the proband were also diagnosed with ovarian cancer 2 years after being diagnosed with breast cancer. The father of the proband also carried the same UV, and his sister died of breast cancer at the age of 46. Another patient who harbored the same variant was diagnosed with breast cancer at the age of 33, as shown in Fig. 1B. Her mother suffered from ovarian cancer and could not participate in this study. The c.5339T>C variant results in an amino acid change from leucine to proline at position 1780. The predicted structure shows that the mutation site is in the middle of a helix in the BRCT domain of BRCA1, forming a hydrophobic patch with its surrounding residues (Fig. 1C). The BRCT domain is known to recognize and bind phosphorylated pSXXF motifs of FAM175A/Abraxas to recruit BRCA1 to regions of DNA damage [19-21].

### 1. Patient characteristics

Table 1 shows the demographic features of breast cancer patients and controls. All patients were female and were diagnosed with histologically confirmed breast cancer, with the exception of four patients who were diagnosed with ovarian cancer. Patients in our study population were predominantly stage I, with only 11 patients categorized as stage IV. Additionally, more than 76% of patients had a family history of breast or ovarian cancer, including 11 with a family history of both breast and ovarian cancer.

All patients underwent genetic testing of BRCA1 and BRCA2 genes by the direct sequencing method. As shown in Tables 1 and 2, a total of 47 patients (14.3%) harbored 35 different deleterious mutations (frameshift or nonsense) of BRCA genes. To examine large genomic rearrangement of the BRCA genes, we performed MLPA assays in 196 patients. Only four patients showed deletions of a large genomic region.

### 2. UVs of BRCA1/2 genes

Sequencing results showed that a total of 181 patients (55.2%) harbored UVs of BRCA genes. Among these, 33 kinds of UVs in the BRCA1 gene were detected in 127 patients, while 47 UVs in the BRCA2 gene were identified from 113 patients. Although these variants were already classified as having uncertain clinical importance, such a high frequency of variants among the population could weaken the significance of the association with cancer. Therefore, we excluded 14 UVs that were present at a frequency of more than 2% among the East-Asian population in the 1000 Genomes Phase 3 database (http://www.1000genomes.org/data). Seven UVs in BRCA1, including rs1799949, rs799912, rs16940, rs799916, rs1060915, rs3092994, and rs8176140, showed a minor allele frequency (MAF) of 37% in the East-Asian population. Furthermore, rs1801406, rs9534262, and rs4942486 in the BRCA2 gene showed reduced significance because their MAF was than 25%. Ultimately, 19 UVs that were detected in at least two patients among our cases were selected for further analysis; however, four UVs could not be genotyped owing to difficulties in probe design. Finally, six UVs in BRCA1 and nine in BRCA2 were further genotyped in 421 age-matched female controls. Among these, c.5339T>C in BRCA1 and c.6029T>G, c.7522G>A in BRCA2 were not detected in healthy controls (Table 3). The c.5339T>C were detected in four patients and c.6029T>G, c.7522G>A were detected in three patients with breast cancer. All three variants caused a substitution of amino acid sequence and c.5339T>C (Leu1780Pro) in BRCA1, and c.7522G>A (Gly2508Ser) in BRCA2 were predicted as damaging variants. In contrast, c.4883T>C and c.2566T>C in BRCA1 and c.2350A>G and c.8187G>T in BRCA2 showed a genotype frequency greater than 2% in the control group.

### 3. Potential risk of c.5339T>C variant in the BRCA1 gene

We examined the potential risk of three UVs that were not detected in 421 healthy controls. Fortunately, we were able to recruit family members of the proband harboring the c.5339T>C variant in the BRCA1 gene. As shown in the pedigree in Fig. 1A, two breast cancer patients in this family and the proband were also diagnosed with ovarian cancer 2 years after being diagnosed with breast cancer. The father of the proband also carried the same UV, and his sister died of breast cancer at the age of 46. Another patient who harbored the same variant was diagnosed with breast cancer at the age of 33, as shown in Fig. 1B. Her mother suffered from ovarian cancer and could not participate in this study. The c.5339T>C variant results in an amino acid change from leucine to proline at position 1780. The predicted structure shows that the mutation site is in the middle of a helix in the BRCT domain of BRCA1, forming a hydrophobic patch with its surrounding residues (Fig. 1C). The BRCT domain is known to recognize and bind phosphorylated pSXXF motifs of FAM175A/Abraxas to recruit BRCA1 to regions of DNA damage [19-21].

### Discussion

Interpreting UVs in BRCA1 and BRCA2 genes has become a particularly important issue for genetic counseling of cancer patients because of the clinical importance of germline mutations in BRCA genes. Here, we sought to define potentially pathogenic variants by comparing the prevalence of BRCA UVs in 421 healthy controls and 328 breast cancer patients in a Korean population by genotyping. Among the 80 UVs that were found in our patients, 15 were identified in controls while three were detected only in patients with breast cancer, not in controls. Some of these latter variants were predicted to be “probably damaging” based on a high score in PolyPhen-2, and were classified as “intolerant” variants by the SIFT tool. Additionally, the nonsynonymous variant c.5339T>C, which causes an amino acid substitution of proline for leucine (Leu1780Pro) in the BRCT domain, was detected in the BRCA1 gene of four patients with breast cancer. The BRCT domain in the C-terminal, which is known to be essential for BRCA1 to function as a tumor suppressor [19], contributes to binding to target proteins with specificity for phosphorylated pSer-X-X-Phe motifs [20,21]. The substitution of proline for leucine may weaken the hydrophobic patch structure of the BRCT domain, potentially influencing the protein-protein interactions needed for the proper function of BRCA1.

The average age at diagnosis of four patients harboring the c.5339T>C variant was 34, and the youngest patient was diagnosed at the age of 25. One breast cancer patient harboring the same variant was subsequently diagnosed with ovarian cancer following breast cancer, and the mother of one patient suffered from ovarian cancer. Based on these findings, it is plausible to suggest c.5339T>C as a potentially pathogenic variant.

Interestingly, one candidate UV in the BRCA2 gene, c.7522G>A, has been reported as a risk factor for breast cancer in a case-control study. This variant is a nonsynonymous single-nucleotide polymorphism known as rs80359878 that causes an amino acid substitution (Gly2508Ser) in BRCA2. Zhang et al. [22] showed that this missense variant was associated with a 16.5-fold increase in the risk of breast cancer among Chinese women, with an allele frequency of this variant of 0.0023 in cases and 0.0001 in controls.

To define rare variants with potential pathogenicity, we compared the frequency of UVs of BRCA genes among healthy controls with that in breast cancer patients. Our results suggest the potentially deleterious variants, c.5339T>C (Leu1780Pro) in BRCA1 and c.6029T>G (Val2010Gly), c.7522G>A (Gly2508Ser) in BRCA2, which were detected only in cases. This strategy could be strengthened using a large number of cases-controls to select significant variants among previously UVs. Biological experiments should be performed to validate the effects of the variants.

### Conclusion

In conclusion, the c.5339T>C variant in BRCA1 that was detected in four patients may be involved in breast cancer pathogenicity by affecting the function of the BRCT domain of BRCA1. The information provided herein will be useful for individuals carrying these variants, who should be carefully monitored for potential cancer risk.



# SUPPLEMENTAL FILE 1: crt-2016-292.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)