# MAIN TEXT

## BRCA1 and BRCA2 founder mutations account for 78% of germline carriers among hereditary breast cancer families in Chile

### Abstract

Identifying founder mutations in BRCA1 and BRCA2 in specific populations constitute a valuable opportunity for genetic screening. Several studies from different populations have reported recurrent and/or founder mutations representing a relevant proportion of BRCA mutation carriers. In Latin America, only few founder mutations have been described. We screened 453 Chilean patients with hereditary breast cancer for mutations in BRCA1 and BRCA2. For recurrent mutations, we genotyped 11 microsatellite markers in BRCA1 and BRCA2 in order to determine a founder effect through haplotype analysis. We found a total of 25 mutations (6 novel) in 71 index patients among which, nine are present exclusively in Chilean patients. Our analysis revealed the presence of nine founder mutations, 4 in BRCA1 and 5 in BRCA2, shared by 2 to 10 unrelated families and spread in different regions of Chile. Our panel contains the highest amount of founder mutations until today and represents the highest percentage (78%) of BRCA1 and BRCA2 mutation carriers. We suggest that the dramatic reduction of Amerindian population due to smallpox and wars with Spanish conquerors, a scarce population increase during 300 years, and the geographic position of Chile constituted a favorable scenario to establish founder genetic markers in our population.

### INTRODUCTION

It is well known that an early detection of breast cancer is highly relevant in relation to patient survival, for this reason a genetic analysis revealing a high risk to breast cancer for healthy women constitutes valuable information. Twenty years ago two genes, BRCA1 (MIM: 113705) and BRCA2 (MIM: 600185), which mutations confer a high risk to breast cancer were identified in families having four or more relatives with breast or ovarian cancer [1, 2]. Since then, the screening of BRCA1 and BRCA2 has been performed in multiple populations across the world revealing that 10 to 50% of breast cancer patients with family history of breast and/or ovarian cancer carry a BRCA1 or BRCA2 mutation [3]. In Chile, we and others have carried out mutation screening of BRCA1 and BRCA2 in hereditary breast cancer patients, leading into mutation frequencies ranging from 15% [4] to 20% [5]. Among Latin American countries, eight similar studies have been published up today: México [6], Colombia [7], Brazil [8], Argentina [9], Uruguay [10], Venezuela [11] and Chile [4, 5]. Among those studies the frequency of mutation carriers in Latin American women with hereditary breast cancer varies from 15% to 25%.

The presence of founder mutations in BRCA1 and BRCA2 in specific ethnic groups or populations constitutes a valuable opportunity for a genetic screening. The most well-known example is the Ashkenazi Jewish population [12], in which three founder mutations combined have a population frequency of 2% and represent 60% of breast cancer families with a BRCA gene mutation. A second well-known example is the Icelandic population in which one founder mutation in BRCA2 999del5 has a population carrier frequency of 0.4%, and account for 64% of breast cancer families [13, 14, 15]. In the last 15 years several studies have described recurrent mutations in breast cancer patients, some of which have documented a single origin (founder effect) of such recurrent mutation through haplotype analyses. In this relation, it is necessary to highlight that a recurrent mutation is not necessary founder, since some recurrent mutations may correspond to hotspots, arising independently in a population. To be founder, a minimum common haplotype must be identified among carriers of a certain recurrent mutation. In Poland [16] three founder mutations account for up to 86% of all families with a BRCA1 or BRCA2 mutation. In Slovenia five founder mutations account for 69% of all BRCA families [17]. Several other studies from Europe, Asia and North America have reported recurrent and/or founder mutations that represent a relevant proportion of BRCA1 and BRCA2 mutation carriers [18]. In Latin America, there are few different mutations with a demonstrated founder effect described in different populations: Mexico (BRCA1 ex9–12del), Brazil (BRCA1 5382insC), Colombia (BRCA1 3450del4, A1708E, and BRCA2 3034del4) and US Hispanics (BRCA1 185delAG, IVS5+1G>A, S955X, and R1443X) [19, 20]. In addition to these founder mutations, other recurrent mutations have been described in Latin American populations which founder effect through haplotype analyses still has not been tested.

In Chile, our group has screened BRCA1 and BRCA2 genes in 336 patients with a family history of breast cancer and 117 patients unselected for family history, recruited throughout the country. The results of this study led us to define a panel of 9 founder mutations among Chilean patients representing 77.5% of total BRCA mutation carriers.

### RESULTS

We identified 25 different mutations (pathogenic variants, class 5), of which 6 are novel and 19 have been previously reported (Table 1). These mutations are present in 71 index patients, of which 32 carried a mutation in BRCA1 (7.06%) and 39 (8.61%) in BRCA2 leading to a combined mutation prevalence of 15.7% (71/453). It is noteworthy that considering only cases with family history this percentage is higher, being 17.9 %. Nine of the 25 mutations described in this work have been found exclusively in Chilean patients [4, 5, 21] (and this study), two of them being recurrent in our samples (c.1504_1507delTTAA and c.3817C>T, both in BRCA1) (Table 1). We have to remark that mutation p.Asp2723Gly has been previously described (BIC) as a variant of uncertain significance in several populations and in 2008, a functional analysis made by Farrugia et al. [22] defined that this aminoacid change was detrimental for BRCA2 function in DNA repair and centrosome duplication. Since then this mutation has been defined as pathogenic. Mutation c.211A>G in BRCA1 (previously named as R71G) is located in the before last nucleotide of exon 5, eliminating the donor splice site in this exon and revealing a cryptic splice site 22 nucleotides upstream. The deletion of 22 nucleotides in the mRNA creates a stop codon in position 64 [23]. We corrected the protein nomination for this mutation from the previous p.Arg71Gly to p.Cys64Ter.

1Gallardo et al. 2006, 2González-Hormazabal et al. 2011, 3Solano et al., 2016. NO: not reported previously, first mentioned in this manuscript. GeneBank Accession Numbers: BRCA1: L78833 and BRCA2: AY436640.

Considering all mutations (n = 25) 9 were recurrent in our sample set (Table 1, highlighted in bold). This group of nine mutations is present in 77.5% (55/71) of total BRCA carriers, becoming an excellent panel for cost-effective BRCA1 and BRCA2 mutational screening in Chilean patients. The most frequent of these nine recurrent mutation is c.4740_4741dupTG in BRCA2 (Table 1). This mutation was first described in Chilean patients by our group [5] and later found by two groups in unrelated Chilean [21], and Argentinian patients [24]. Two other frequent recurrent mutations (BRCA1 c.3331_3334delCAAG and BRCA2 c.5146_5149delTATG) were found in 9 families each.

In order to determine a probable founder effect for the 9 recurrent mutations we analyzed 11 STR markers (6 for BRCA1 and 5 for BRCA2) in 53 index patients and established the corresponding haplotypes. Patients sharing the same mutation belong to apparently unrelated families residing in different regions along Chile as it is shown in Figure 1.

A map of the Chilean territory indicating cities where patients were recruited is shown. Each mutation is represented with specific color, and each carrier family is indicated by a colored circle.

All microsatellites for BRCA1 and BRCA2 were informative (Figure 2). For three mutations in BRCA1, c.1504_1507delTTAA (2 families), c.3331_3334delCAAG (9 families) and c.3817C>T (4 families), the core haplotype (identical haplotype at loci) was shaped with the six microsatellites markers (Figure 2A) revealing no recombinants in a region of 0.68 Mb. For the fourth mutation, c.3759dupT, the core haplotype was shaped with four markers (Figure 2A) covering a region of 0.46 Mb. In this case one sample is a recombinant for D17S1326 and two samples for markers D17S1327 and D17S1326. On the other hand, for three mutations in BRCA2, c.145G>T (3 families), c.4740_4741dupTG (6 families) and c.8987T>A (6 families), the core haplotype was shaped with the five microsatellites markers (D13S260, D13S1698, D13S171, D13S310 and D13S267) revealing no recombinants in a region of 1.82 Mb (Figure 2B). Conversely, for mutations c.5146_5149delTATG (8 families) and c.9382T>A (3 families) we found one recombinant for D13S260 in our samples. For both mutations the minimum common haplotype covers 1.56Mb (Figure 2B). We were able to determine a common haplotype for all the analyzed mutations.

Minimum common haplotypes identified for each recurrent mutation in BRCA1 (A) and BRCA2 (B). Values under each STR marker indicate the size of the shared alleles. Scheme (bottom image) represents the localization of all markers and mutations in relation to each gene.

We found a total of 110 coding and non-coding variants, 15 not previously reported in the literature or public data bases. Among the 95 reported variants, 62 correspond to polymorphisms (allelic frequencies ≥ 1%, not pathogenic, class 1) reported in at least one population in the 1000 Genomes database. In addition we detected 33 rare variants (Supplementary Table 1), either with allelic frequencies < 1% in all populations reported (1000 Genomes or dbSNP) or allelic frequencies not reported. The classification of the 33 rare variants (Supplementary Table 1) was concluded after revising three databases (BIC, ClinVar, BRCA Share) and functional assays in published papers. Nineteen of these variants have been classified as “benign” or “likely benign” (also neutral/likely neutral) in ClinVar and BRCA Share databases (Supplementary Table 1). Seven variants are reported as “unclassified variant” or “variant with uncertain significance” (Supplementary Table 1: ClinVar, BRCA Share). The remaining 7 variants have a controversial classification, being reported as benign/likely benign (neutral/likely neutral) by one database and uncertain by another. Among the 14 variants reported previously, at least once in data bases as “uncertain”, three have been tested in functional assays concluding a neutral or non-pathogenic effect, indicated under conclusion in Supplementary Table 1. In summary, according to ENIGMA recommendations and considering reported in silico analyses, functional assays, and co-occurrence with a BRCA1/2 pathogenic variant, there are 17 not pathogenic and 6 likely not pathogenic variants. For 10 rare variants there was insufficient information so were interpreted as “uncertain” (Supplementary Table 1).

In relation to novel variants in this study a total of 11 are described in Table 2. Among these, 6 are coding variants and 5 non-coding. In the coding region only two novel synonymous variants were found in our study. These variants are predicted to produce a weak or null effect in splicing by the three utilized tools. On the other hand among these, 2 variants, p.Pro34 = co-occur with a mutation in BRCA1, c. 211A>G. These results suggest that these variants should be classified as neutral or non-pathogenic.

a) HCI, Breast Cancer Genes Prior Probabilities, b) VEP, Variant Effect Predictor, c) Align-GVGD, d) SIFT, e) PROVEAN, f) PolyPhen-2, g) HSF, Human Splice Finder.

Three variants are novel non-synonymous (Table 2), and none of them co-occur with a BRCA mutation. Determining the functional effect of a missense variant is a relevant step to interpret their possible pathogenic role in breast cancer. The analysis tools give contradictory predictions for 2/3 variants, so these should be classified as uncertain, until a functional effect can be assessed. Variants c.4746C>G has a null effect predicted by all tools.

Finally, we found one novel truncating variant in exon 27 of BRCA2 (Table 2: p.Gln3409Ter) downstream of p.Lys3326Ter, which is frequently described as neutral (ClinVar). For this reason we assumed that the new variant should also be neutral as recommended by ENIGMA. Non-coding novel variants are deep in the introns and according to ENIGMA recommendations and ACMG guidelines they should be considered as non-pathogenic. None of these novel variants co-occur with a pathogenic mutation.

### Novel and recurrent mutations in BRCA1 and BRCA2

We identified 25 different mutations (pathogenic variants, class 5), of which 6 are novel and 19 have been previously reported (Table 1). These mutations are present in 71 index patients, of which 32 carried a mutation in BRCA1 (7.06%) and 39 (8.61%) in BRCA2 leading to a combined mutation prevalence of 15.7% (71/453). It is noteworthy that considering only cases with family history this percentage is higher, being 17.9 %. Nine of the 25 mutations described in this work have been found exclusively in Chilean patients [4, 5, 21] (and this study), two of them being recurrent in our samples (c.1504_1507delTTAA and c.3817C>T, both in BRCA1) (Table 1). We have to remark that mutation p.Asp2723Gly has been previously described (BIC) as a variant of uncertain significance in several populations and in 2008, a functional analysis made by Farrugia et al. [22] defined that this aminoacid change was detrimental for BRCA2 function in DNA repair and centrosome duplication. Since then this mutation has been defined as pathogenic. Mutation c.211A>G in BRCA1 (previously named as R71G) is located in the before last nucleotide of exon 5, eliminating the donor splice site in this exon and revealing a cryptic splice site 22 nucleotides upstream. The deletion of 22 nucleotides in the mRNA creates a stop codon in position 64 [23]. We corrected the protein nomination for this mutation from the previous p.Arg71Gly to p.Cys64Ter.

1Gallardo et al. 2006, 2González-Hormazabal et al. 2011, 3Solano et al., 2016. NO: not reported previously, first mentioned in this manuscript. GeneBank Accession Numbers: BRCA1: L78833 and BRCA2: AY436640.

Considering all mutations (n = 25) 9 were recurrent in our sample set (Table 1, highlighted in bold). This group of nine mutations is present in 77.5% (55/71) of total BRCA carriers, becoming an excellent panel for cost-effective BRCA1 and BRCA2 mutational screening in Chilean patients. The most frequent of these nine recurrent mutation is c.4740_4741dupTG in BRCA2 (Table 1). This mutation was first described in Chilean patients by our group [5] and later found by two groups in unrelated Chilean [21], and Argentinian patients [24]. Two other frequent recurrent mutations (BRCA1 c.3331_3334delCAAG and BRCA2 c.5146_5149delTATG) were found in 9 families each.

### Common haplotypes found for recurrent BRCA1 and BRCA2 mutations

In order to determine a probable founder effect for the 9 recurrent mutations we analyzed 11 STR markers (6 for BRCA1 and 5 for BRCA2) in 53 index patients and established the corresponding haplotypes. Patients sharing the same mutation belong to apparently unrelated families residing in different regions along Chile as it is shown in Figure 1.

A map of the Chilean territory indicating cities where patients were recruited is shown. Each mutation is represented with specific color, and each carrier family is indicated by a colored circle.

All microsatellites for BRCA1 and BRCA2 were informative (Figure 2). For three mutations in BRCA1, c.1504_1507delTTAA (2 families), c.3331_3334delCAAG (9 families) and c.3817C>T (4 families), the core haplotype (identical haplotype at loci) was shaped with the six microsatellites markers (Figure 2A) revealing no recombinants in a region of 0.68 Mb. For the fourth mutation, c.3759dupT, the core haplotype was shaped with four markers (Figure 2A) covering a region of 0.46 Mb. In this case one sample is a recombinant for D17S1326 and two samples for markers D17S1327 and D17S1326. On the other hand, for three mutations in BRCA2, c.145G>T (3 families), c.4740_4741dupTG (6 families) and c.8987T>A (6 families), the core haplotype was shaped with the five microsatellites markers (D13S260, D13S1698, D13S171, D13S310 and D13S267) revealing no recombinants in a region of 1.82 Mb (Figure 2B). Conversely, for mutations c.5146_5149delTATG (8 families) and c.9382T>A (3 families) we found one recombinant for D13S260 in our samples. For both mutations the minimum common haplotype covers 1.56Mb (Figure 2B). We were able to determine a common haplotype for all the analyzed mutations.

Minimum common haplotypes identified for each recurrent mutation in BRCA1 (A) and BRCA2 (B). Values under each STR marker indicate the size of the shared alleles. Scheme (bottom image) represents the localization of all markers and mutations in relation to each gene.

### Polymorphisms and unclassified variants in BRCA1 and BRCA2

We found a total of 110 coding and non-coding variants, 15 not previously reported in the literature or public data bases. Among the 95 reported variants, 62 correspond to polymorphisms (allelic frequencies ≥ 1%, not pathogenic, class 1) reported in at least one population in the 1000 Genomes database. In addition we detected 33 rare variants (Supplementary Table 1), either with allelic frequencies < 1% in all populations reported (1000 Genomes or dbSNP) or allelic frequencies not reported. The classification of the 33 rare variants (Supplementary Table 1) was concluded after revising three databases (BIC, ClinVar, BRCA Share) and functional assays in published papers. Nineteen of these variants have been classified as “benign” or “likely benign” (also neutral/likely neutral) in ClinVar and BRCA Share databases (Supplementary Table 1). Seven variants are reported as “unclassified variant” or “variant with uncertain significance” (Supplementary Table 1: ClinVar, BRCA Share). The remaining 7 variants have a controversial classification, being reported as benign/likely benign (neutral/likely neutral) by one database and uncertain by another. Among the 14 variants reported previously, at least once in data bases as “uncertain”, three have been tested in functional assays concluding a neutral or non-pathogenic effect, indicated under conclusion in Supplementary Table 1. In summary, according to ENIGMA recommendations and considering reported in silico analyses, functional assays, and co-occurrence with a BRCA1/2 pathogenic variant, there are 17 not pathogenic and 6 likely not pathogenic variants. For 10 rare variants there was insufficient information so were interpreted as “uncertain” (Supplementary Table 1).

In relation to novel variants in this study a total of 11 are described in Table 2. Among these, 6 are coding variants and 5 non-coding. In the coding region only two novel synonymous variants were found in our study. These variants are predicted to produce a weak or null effect in splicing by the three utilized tools. On the other hand among these, 2 variants, p.Pro34 = co-occur with a mutation in BRCA1, c. 211A>G. These results suggest that these variants should be classified as neutral or non-pathogenic.

a) HCI, Breast Cancer Genes Prior Probabilities, b) VEP, Variant Effect Predictor, c) Align-GVGD, d) SIFT, e) PROVEAN, f) PolyPhen-2, g) HSF, Human Splice Finder.

Three variants are novel non-synonymous (Table 2), and none of them co-occur with a BRCA mutation. Determining the functional effect of a missense variant is a relevant step to interpret their possible pathogenic role in breast cancer. The analysis tools give contradictory predictions for 2/3 variants, so these should be classified as uncertain, until a functional effect can be assessed. Variants c.4746C>G has a null effect predicted by all tools.

Finally, we found one novel truncating variant in exon 27 of BRCA2 (Table 2: p.Gln3409Ter) downstream of p.Lys3326Ter, which is frequently described as neutral (ClinVar). For this reason we assumed that the new variant should also be neutral as recommended by ENIGMA. Non-coding novel variants are deep in the introns and according to ENIGMA recommendations and ACMG guidelines they should be considered as non-pathogenic. None of these novel variants co-occur with a pathogenic mutation.

### DISCUSSION

We described the screening of BRCA1 and BRCA2 in a Chilean cohort of 453 patients. As shown in Table 3 the higher percentage of patients with a mutation in BRCA1/2 is among families with a male breast cancer (40%, only in BRCA2), followed by the group having breast and ovarian cancer cases (37.7%) being higher for BRCA1. In relation to patients with no family history the highest percentage of mutation carriers is among bilateral breast cancer patients. It is noteworthy that in the group of younger patients < 40 y 6/7 patients have a mutation in BRCA1. In summary the distribution of BRCA mutation carriers is not remarkably different to what has been already described in other breast cancer cohorts. In relation to clinico-pathological characteristics the only relevant data is that among BRCA1 mutation carriers a 58% are triple negative (ER-, PR- and HER2-), as it has been also described by other groups. Analyzing the group of triple negative patients 70% has a mutation in BRCA1.

The screening of 453 high risk breast cancer patients from different regions of Chile revealed the presence of nine founder mutations, 4 in BRCA1 and 5 in BRCA2. These mutations are shared by 2 to 10 unrelated families spread in different regions of Chile, and correspond to 77.5% (55/71) of mutation carriers. All unrelated patients having the same mutation share a minimum haplotype revealing a common ancestor for each mutation. In several populations, as it is mentioned in the introduction, founder or recurrent mutations have been described; however the Chilean panel is until today the one containing the highest amount of founder mutations and also representing the highest percentage (77.5%) of BRCA1 and BRCA2 mutation carriers, in a single population. These results reveal an excellent low cost panel to screen BRCA1 and BRCA2 carriers in Chile, among breast cancer patients including cases apparently sporadic.

It is important to highlight that the three Ashkenazi-Jewish founder mutations are not frequently observed in our population as occurs in Argentina and Brazil. Considering the two large studies in Chile that include almost 800 patients [21] (and this study) three families carry the BRCA1 c.68_69delAG (formerly known as 185delAG), two carry BRCA2 c.5946delT (also known as 6174delT), and one present BRCA1 c.5266dupC (formerly known as 5382insC), representing less than 7% of Chilean mutation carriers, while in Argentina is close to 35% [24]. In the case of our group of families among 13 Ashkenazi Jewish only 2 present a mutation. Family with c.5266dupC is not from Ashkenazi Jewish origin. These results suggest that the Jewish community in Chile have a different ethnic origin compared to those in Brazil and Argentina, or that the prevalence of these mutations among the founders of this community in our country was lower than that in other countries from the region.

In Latin America, there are a few mutations with a demonstrated founder effect described in different populations: Mexico (BRCA1 ex9–12del), Brazil (BRCA1 5382insC), Colombia (BRCA1 3450del4, A1708E, and BRCA2 3034del4) and US Hispanics (BRCA1 185delAG, IVS5+1G>A, S955X, 2552delC and R1443X) [20, 25, 26]. Other recurrent mutations have been described in Latin American populations which founder effect has not yet been tested. The majority of mutations in BRCA1 or BRCA2 found in Latin-American breast cancer patients are private for each population, with very few mutations shared between countries [27]. Recently a panel of BRCA1 and BRCA2, HISPANEL [28] has been constructed with diverse mutations from Hispanic breast cancer women from USA, based on the information in manuscripts describing mutations in BRCA1 and BRCA2 from Latin American countries and data bases. The HISPANEL, including close to 100 recurrent mutations, has been applied in breast cancer patients from Mexico, Colombia, Peru and Brazil, with diverse results [29–32], due to the diversity of BRCA1 and BRCA2 mutations among different Latin American populations, and the variable criteria to select the patients to be analyzed. Indeed our study as well as another study in Chilean patients [21], have shown that only 6 Chilean mutations are present in the HISPANEL, among which only three are Chilean founder mutations, representing only 25% of mutation carriers. This analysis confirms that the HISPANEL is not reliable for screening BRCA1/2 mutations in our breast cancer patients. In addition we found very few mutations shared with other Latin-American populations revealing that most mutations where introduced in the continent by independent events.

In relation to the 9 founder mutations described in this study c.1504_1507delTTAA and c.3817C>T in BRCA1 are not described elsewhere consisting in private mutations from Chile. In addition c.4740_4741dupTG in BRCA2 was also detected in two families from Argentina, and c.145G>T in BRCA2, was described in one Argentinian family. Due to the frequent migration events between these two countries this finding is not rare. It is noteworthy that mutation c.145G>T in BRCA2 was detected exclusively in patients recruited at Valdivia (South of Chile), probably representing a specific settlement of colonists (Figure 1). In the contrary the other 8 founder mutations are present in different cities along the Chilean territory (Figure 1).

An interesting case is the mutation in BRCA1, c.3331_3334delCAAG, that is also founder in Colombia, Spain and Portugal (Tuazon A, manuscript in preparation), and also recurrent in 4 Brazilian families [33]. This mutation has been globally described among Caucasian population from Europe, Australia and America (BIC). In concordance with our history of colonization BRCA1 founder mutations c.3331_3334delCAAG and c.3759dupT, as well as BRCA2 founder mutations c.145G>T and c.5146_5149delTATG have been described in Spain. This latter mutation, c.5146_5149delTATG in BRCA2, was first reported in families from the regions of Castilla-León [34] and Aragon [35] in Spain and later stablished as a Spanish founder mutation [36] originated at Castilla-León.

The finding of 9 founder mutations among Chilean breast cancer families is a consequence of history of colonization, and how the population of Chile was settled. Several historical records mention that the indigenous population in Chile at the time of the arrival of Spaniards in 1541 was around 1 million, but it was dramatically reduced as a consequence of battles with Conquerors, dying mainly male indigenous, as well as diseases (Instituto Nacional de Estadísticas, INE 2009). In addition mostly male Spaniards came to Chile during the first three centuries, with a very scarce population increase; therefore the genetic bases of the Chilean population come from mating between Amerindian women and Spanish men [37] (and INE, 2009). These historic issues have been revealed in a previous study from our group, demonstrating that the Chilean population is composed in 84% by Amerindian mitochondrial DNA (maternal lineage) and 32% of Amerindian Y chromosome (paternal lineage) [38]. Even though at the end of the XIX century considerable German and French migrations occurred, and others thereafter, the genetic composition of the Chilean population today still reflects the history of the first three centuries [39]. We could suggest that the dramatic reduction of the Amerindian population in conjunction with the arrival of Spanish conquerors (highly predominant in males), for close to 300 years, was a favorable scenario to establish founder genetic markers in the population. In addition the geographic position of Chile is also beneficial to establish founder genetic markers or mutations since no immigrations came from the Pacific Ocean, few from the north of Chile and very scarce from Europe through Argentina.

### MATERIALS AND METHODS

A total of 336 patients with familial breast/ovarian cancer were recruited from 1999 to 2015 using either of the following family history criteria: 1) three or more relatives with breast cancer diagnosed after 45 years old, 2) two relatives with breast cancer at least one diagnosed before age 45, 3) one breast cancer and one ovarian cancer at any age, or 4) one male breast cancer and one female breast cancer. The 117 patients with breast cancer and no family history were selected by 5) being diagnosed at 40 years old or younger, 6) presenting bilateral breast cancer or 7) presenting breast and ovarian cancer. The selected patients were recruited from three Public Hospitals (Antofagasta, Valdivia and Santiago) and one private Institution (Santiago). Two of the public Hospitals assist patients from Regions XV, I, II, and III (Antofagasta Hospital, North of Chile) and IX, XIV, X and XI (Valdivia Hospital, South of Chile), covering 4.5 million Chilean inhabitants (28% of total Chile. Based on Cense 2012). The third Public Hospital (Santiago) assists close to 25% of total patients from the Metropolitan Region. Low and middle income patients attend to these three hospitals.

After informed consent, blood samples were obtained from affected individuals and relatives. This protocol, which adheres to the Helsinki declaration and local regulations, was approved by the Ethics Committees belonging to all the Hospitals where the patients came from. All patients received genetic counselling before and after the genetic test.

DNA was isolated using the method described by Lahiri and Nurnberger [39]. PCR amplification covering all coding sequences and intron-exon boundaries, of exons 2 to 24 (BRCA1) and 2 to 27 (BRCA2) was performed by standard methods. In the first 147 patients, BRCA1 and BRCA2 mutations were screened through heteroduplex analysis, Protein Truncation Test (PTT) Single Strand Conformational Polymorphism (SSCP), and Sanger sequencing of selected exons. In 306 patients mutation screening was performed by next-generation sequencing (NGS). Enrichment of BRCA1 and BRCA2 sequences for NGS was performed with BRCA MASTR v2.1 (Multiplicom, Belgium) as follows. Briefly, 5 multiplex reactions were used to amplify all coding exons and intron/exon boundaries of BRCA1 and BRCA2 (93 amplicons), followed by sample barcoding. Sequencing was performed in a Junior Roche 454. Analysis of sequencing data was performed with Amplicon Variant Analyzer (AVA) software (Roche) using references sequences for BRCA1 and BRCA2 provided by Multiplicom, and filtered to allow the call of all possible variants. The obtained variants were filtered manually using a 30× depth cutoff for each forward and reverse sequences, and allelic variant frequencies over 30%. Allelic frequencies for heterozygous variants were determined between 30 and 70%, and for homozygous variants over 70%. All detected variants were confirmed by Sanger sequencing, and named according to the Human Genome Variation Society (http://www.hgvs.org/mutnomen/) and relative to GenBank reference sequences (BRCA1: L78833 and BRCA2: AY436640). In addition, all new mutations identified in the 306 patients analyzed by NGS were re-screened in the first group of patients.

Novel variants were defined as pathogenic (class 5), likely pathogenic (class 4), uncertain significance (VUS; class 3), likely not pathogenic (class 2) and not pathogenic (class 1) following recommendations of the American College of Medical Genetics and Genomics [40] and ENIGMA (ENIGMA consortium, https://enigmaconsortium.org/library/general-documents). The already described variants (pathogenic, non-pathogenic and uncertain) were assigned after reviewing information available in BIC (Breast Cancer Information Core, https://research.nhgri.nih.gov/projects/bic/), ClinVar (http://www.ncbi.nlm.nih.gov/clinvar/), BRCA Share (former Universal Mutation Database, http://umd.be/) 1000 genomes (http://www.internationalgenome.org/), Leiden Open Variation Database (http://www.lovd.nl/3.0/home) and published manuscripts. Novel variants not classified under pathogenic, were analyzed with two integrative platforms a) HCI Breast Cancer Genes Prior Probabilities (http://priors.hci.utah.edu/PRIORS/references.php); b) Variant Effect Predictor [41] (VEP, http://www.ensembl.org/info/docs/tools/vep/index.html); and the recommended in silico tools [40] (ACMG and ENIGMA), c) Align-GVGD [42] (http://agvgd.hci.utah.edu), d) SIFT [43] (http://sift.jcvi.org), e) PROVEAN [44] (http://provean.jcvi.org/index.php), f) PolyPhen-2 [45] (http://genetics.bwh.harvard.edu/pph2). In addition, synonymous variants were also assessed for splicing effect using Human Splice Finder [46] (HSP, http://www.umd.be/HSF3/). Default parameters were applied for each tool.

We analyzed a total of 11 Short Tandem Repeats (STR) markers. For mutations in BRCA1: D17S1323, D17S1322, D17S855, D17S1320, D17S1326 and D17S1327, covering 0.68 Mb, and for mutations in BRCA2: D13S260, D13S1698, D13S267, D13S171 and D13S310 covering 1.82 Mb. All loci were amplified by PCR using primers, end-labeled with different fluorophores (VIC, 6-FAM, PET and NED). PCR products were analyzed by capillary electrophoresis in an ABI PRISM 3130x fragment analyzer (Applied Technologies) using Peak Scanner Software 2 to confirm product amplification and size. The STR alleles associated to each recurrent mutation were grouped in haplotypes for all samples carrying the same mutation, and phases were confirmed by PHASE v2.1 [47, 48] (stephenslab.ucicago.edu/phase). A minimum common haplotype was determined to define a common ancestor.

### Patients

A total of 336 patients with familial breast/ovarian cancer were recruited from 1999 to 2015 using either of the following family history criteria: 1) three or more relatives with breast cancer diagnosed after 45 years old, 2) two relatives with breast cancer at least one diagnosed before age 45, 3) one breast cancer and one ovarian cancer at any age, or 4) one male breast cancer and one female breast cancer. The 117 patients with breast cancer and no family history were selected by 5) being diagnosed at 40 years old or younger, 6) presenting bilateral breast cancer or 7) presenting breast and ovarian cancer. The selected patients were recruited from three Public Hospitals (Antofagasta, Valdivia and Santiago) and one private Institution (Santiago). Two of the public Hospitals assist patients from Regions XV, I, II, and III (Antofagasta Hospital, North of Chile) and IX, XIV, X and XI (Valdivia Hospital, South of Chile), covering 4.5 million Chilean inhabitants (28% of total Chile. Based on Cense 2012). The third Public Hospital (Santiago) assists close to 25% of total patients from the Metropolitan Region. Low and middle income patients attend to these three hospitals.

After informed consent, blood samples were obtained from affected individuals and relatives. This protocol, which adheres to the Helsinki declaration and local regulations, was approved by the Ethics Committees belonging to all the Hospitals where the patients came from. All patients received genetic counselling before and after the genetic test.

### Mutation screening

DNA was isolated using the method described by Lahiri and Nurnberger [39]. PCR amplification covering all coding sequences and intron-exon boundaries, of exons 2 to 24 (BRCA1) and 2 to 27 (BRCA2) was performed by standard methods. In the first 147 patients, BRCA1 and BRCA2 mutations were screened through heteroduplex analysis, Protein Truncation Test (PTT) Single Strand Conformational Polymorphism (SSCP), and Sanger sequencing of selected exons. In 306 patients mutation screening was performed by next-generation sequencing (NGS). Enrichment of BRCA1 and BRCA2 sequences for NGS was performed with BRCA MASTR v2.1 (Multiplicom, Belgium) as follows. Briefly, 5 multiplex reactions were used to amplify all coding exons and intron/exon boundaries of BRCA1 and BRCA2 (93 amplicons), followed by sample barcoding. Sequencing was performed in a Junior Roche 454. Analysis of sequencing data was performed with Amplicon Variant Analyzer (AVA) software (Roche) using references sequences for BRCA1 and BRCA2 provided by Multiplicom, and filtered to allow the call of all possible variants. The obtained variants were filtered manually using a 30× depth cutoff for each forward and reverse sequences, and allelic variant frequencies over 30%. Allelic frequencies for heterozygous variants were determined between 30 and 70%, and for homozygous variants over 70%. All detected variants were confirmed by Sanger sequencing, and named according to the Human Genome Variation Society (http://www.hgvs.org/mutnomen/) and relative to GenBank reference sequences (BRCA1: L78833 and BRCA2: AY436640). In addition, all new mutations identified in the 306 patients analyzed by NGS were re-screened in the first group of patients.

### Clinical significance determination for sequence variants

Novel variants were defined as pathogenic (class 5), likely pathogenic (class 4), uncertain significance (VUS; class 3), likely not pathogenic (class 2) and not pathogenic (class 1) following recommendations of the American College of Medical Genetics and Genomics [40] and ENIGMA (ENIGMA consortium, https://enigmaconsortium.org/library/general-documents). The already described variants (pathogenic, non-pathogenic and uncertain) were assigned after reviewing information available in BIC (Breast Cancer Information Core, https://research.nhgri.nih.gov/projects/bic/), ClinVar (http://www.ncbi.nlm.nih.gov/clinvar/), BRCA Share (former Universal Mutation Database, http://umd.be/) 1000 genomes (http://www.internationalgenome.org/), Leiden Open Variation Database (http://www.lovd.nl/3.0/home) and published manuscripts. Novel variants not classified under pathogenic, were analyzed with two integrative platforms a) HCI Breast Cancer Genes Prior Probabilities (http://priors.hci.utah.edu/PRIORS/references.php); b) Variant Effect Predictor [41] (VEP, http://www.ensembl.org/info/docs/tools/vep/index.html); and the recommended in silico tools [40] (ACMG and ENIGMA), c) Align-GVGD [42] (http://agvgd.hci.utah.edu), d) SIFT [43] (http://sift.jcvi.org), e) PROVEAN [44] (http://provean.jcvi.org/index.php), f) PolyPhen-2 [45] (http://genetics.bwh.harvard.edu/pph2). In addition, synonymous variants were also assessed for splicing effect using Human Splice Finder [46] (HSP, http://www.umd.be/HSF3/). Default parameters were applied for each tool.

### Founder haplotype determination for recurrent mutations

We analyzed a total of 11 Short Tandem Repeats (STR) markers. For mutations in BRCA1: D17S1323, D17S1322, D17S855, D17S1320, D17S1326 and D17S1327, covering 0.68 Mb, and for mutations in BRCA2: D13S260, D13S1698, D13S267, D13S171 and D13S310 covering 1.82 Mb. All loci were amplified by PCR using primers, end-labeled with different fluorophores (VIC, 6-FAM, PET and NED). PCR products were analyzed by capillary electrophoresis in an ABI PRISM 3130x fragment analyzer (Applied Technologies) using Peak Scanner Software 2 to confirm product amplification and size. The STR alleles associated to each recurrent mutation were grouped in haplotypes for all samples carrying the same mutation, and phases were confirmed by PHASE v2.1 [47, 48] (stephenslab.ucicago.edu/phase). A minimum common haplotype was determined to define a common ancestor.

### SUPPLEMENTARY MATERIALS TABLES



# SUPPLEMENTAL FILE 1: oncotarget-08-74233.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: PMC5650336

[File available at: automated_output/BRCA2/20251125_175029/pmc_fulltext/29088781_supplements/PMC5650336]

