# MAIN TEXT

## Combined annotation‐dependent depletion score for BRCA1/2 variants in patients with breast and/or ovarian cancer

### Abstract

Utility of combined annotation‐dependent depletion (CADD) score was recently reported to rank pathogenicity as C‐scores ranging 1‐99 for both confirmed deleterious mutation. Using C‐scores for BRCA1/2 variants, we tried to constitute the classification system for variant of uncertain significance (VUS), which had been a major problem of genetic testing for hereditary breast and/or ovarian cancer. We analyzed BRCA1/2 genes for 283 patients with breast and/or ovarian cancer. The deleterious mutation and missesne mutations, minor variant, and wild type of BRCA1 and ‐2 were 5, 27, 251 and 15, 85, 183, respectively. Meanwhile, the variants with C‐score ≥10 were involved in 19/283 (6.7%) in BRCA1 and 34/283 (12%) in BRCA2. All deleterious mutations were included in this group. Frequency of personal history and family history of ovarian cancer were significantly high, and frequency of serous adenocarcinoma of ovary and triple negative breast cancer was relatively high in the group with deleterious mutations. Similar findings were seen in patients with variants of C‐score ≥10. According to the C‐score and population frequency, we could define VUS for 11 patients out of 283 patients (3.9 CADD is useful to classify the variant of BRCA1/2 and selecting the patient who needs further segregation studies.

### INTRODUCTION

Hereditary breast and/or ovarian cancer (HBOC) syndrome is caused by germline deleterious mutations in BRCA1 and BRCA2.1 Indication for germline genetic testing for BRCA is increasing as a result of directed cancer chemotherapy,2 novel targeted therapies,3, 4 and selection of therapeutic surgery.5 However, a major problem with genetic testing and counseling for the HBOC patient is the finding of variant of uncertain significance (VUS).6 VUS represents a particular challenge as the clinical significance cannot be inferred from sequence information alone. Misinterpretations of VUS can lead to real clinical harm for both patients and families. The number of novel variants without confirmed information in the databases seems to increase in accordance with increased genetic testing.7

The effort of scientific team in Myriad Genetcs Inc. (Salt Lake City, UT, USA) for their accumulated data decreased the ratio of VUS to 2.1%.8 They have a variant classification program that consists of several factors. They reported that most of the variants were classified as benign mutations which had no effect on developing cancer.

In contrast, combined annotation‐dependent depletion (CADD) score was recently reported by Kircher et al9 CADD can rank the pathogenicity as C‐scores ranging 1‐99 for both convinced deleterious mutations (frameshift and nonsense) and missense mutations. This ranking is meaningful for classifying the variants.

Using C‐scores for BRCA1/2 variants, we tried to constitute the classification system for VUS, which is important for genetic counseling to clarify the pathogenesis of variants.

### MATERIALS AND METHODS

We carried out next‐generation analysis for BRCA1/2 genes for 283 patients with breast and/or ovarian cancer from September 2013 to December 2016. Patients were selected according to National Comprehensive Cancer Network (NCCN) criteria for future genetic evaluation.10 This cohort consisted of 177 patients with breast cancer, 12 with breast and ovarian cancer, and 94 with ovarian cancer, in which only 15 patients (5.3%) did not meet the criteria of the NCCN criteria. Our cohort consisted of breast and/or ovarian cancer patients with a high risk for HBOC. The deleterious mutation, minor variant, and wild type of BRCA1 and 2 were 5, 27, 251 and 15, 85, 183, respectively. Next‐generation sequencing for BRCA 1/2 germline gene was done using Ion AmpliSeq™ BRCA1 and BRCA2 Panel (Thermo Fishier Scientific, Waltham, MA, USA) containing 167 primer pairs. Detailed methods were described in our previous reports.11, 12

Classification of deleterious mutation was applied for frameshift, nonsense, and splice site mutations that lead to premature truncation of the protein. Most of the cases were analyzed for their large deletion by the Multiplex Ligation‐dependent Probe Amplification (MLPA) method which was carried out by FALCO Biosystems (Kyoto, Japan). Missense variants with minor allele frequency (MAF) <0.01 were selected as rare variants according to 1000 Genomes Project data,13 in which deleterious mutations were determined by available databases and reports.

Combined annotation‐dependent depletion (CADD) was applied for these rare variants as well as for deleterious mutations. C‐scores were obtained by non‐commercial applications developed by Kircher et al9 CADD annotations were made using a wide range of data types including conservation matrix such as Genomic Evolutionary Rate Profiling (GERP),14 phastCons,15 and phyloP16; functional genomic data such as DNAase hypersensitivity and transcription factor binding; transcript information such as distance to exon‐intron boundaries or expression levels in commonly studied cell lines17; and protein‐level scores such as that of Grantham,18 SIFT,19 and PolyPhen.20 In the CADD system, a support vector machine (SVM) is trained and Phred‐like scores (scaled C‐scores) are defined ranging from 1‐99, based on the rank of each variant relative to all possible 8.6 billion substitutions in the human reference genome.9 The results of this transformation are “scaled” C‐scores. Top 10% in the ranking of CADD scores are assigned C‐score 10, top 1% to C‐score 20 and top 0.1% to C‐score 30 etc.

Patients with variants were divided into 4 groups: deleterious mutations; variants with C‐score ≥10; variants with 10 >  C‐score ≥1; and control group including both variants with C‐score <1 and wild type (because we found no difference in clinical features between variants with C‐score <1 and wild type). If the patient had several minor variants, she was divided into a group according to the higher value of the C‐score. Clinical features associated with HBOC were compared between each group and control group. Personal and family history of breast and/or ovarian cancer, histology of ovarian cancer (serous or non‐serous) and breast cancer (triple negative breast cancer [TNBC] or non‐TNBC), age of developing breast cancer (≤45 years or older), and whether bilateral or unilateral breast cancer were selected as the clinical features.

Finally, we showed our classification for the minor variants using C‐score and segregation studies compared with the classification of the other annotation systems (SFIT and PolyPhen2) and representative database of ClinVar.

Frequency of variants were statistically analyzed using Fisher's exact test and χ2‐test as appropriate (StatMate by ATMS, Tokyo, Japan).

### Patients and methods

We carried out next‐generation analysis for BRCA1/2 genes for 283 patients with breast and/or ovarian cancer from September 2013 to December 2016. Patients were selected according to National Comprehensive Cancer Network (NCCN) criteria for future genetic evaluation.10 This cohort consisted of 177 patients with breast cancer, 12 with breast and ovarian cancer, and 94 with ovarian cancer, in which only 15 patients (5.3%) did not meet the criteria of the NCCN criteria. Our cohort consisted of breast and/or ovarian cancer patients with a high risk for HBOC. The deleterious mutation, minor variant, and wild type of BRCA1 and 2 were 5, 27, 251 and 15, 85, 183, respectively. Next‐generation sequencing for BRCA 1/2 germline gene was done using Ion AmpliSeq™ BRCA1 and BRCA2 Panel (Thermo Fishier Scientific, Waltham, MA, USA) containing 167 primer pairs. Detailed methods were described in our previous reports.11, 12

Classification of deleterious mutation was applied for frameshift, nonsense, and splice site mutations that lead to premature truncation of the protein. Most of the cases were analyzed for their large deletion by the Multiplex Ligation‐dependent Probe Amplification (MLPA) method which was carried out by FALCO Biosystems (Kyoto, Japan). Missense variants with minor allele frequency (MAF) <0.01 were selected as rare variants according to 1000 Genomes Project data,13 in which deleterious mutations were determined by available databases and reports.

Combined annotation‐dependent depletion (CADD) was applied for these rare variants as well as for deleterious mutations. C‐scores were obtained by non‐commercial applications developed by Kircher et al9 CADD annotations were made using a wide range of data types including conservation matrix such as Genomic Evolutionary Rate Profiling (GERP),14 phastCons,15 and phyloP16; functional genomic data such as DNAase hypersensitivity and transcription factor binding; transcript information such as distance to exon‐intron boundaries or expression levels in commonly studied cell lines17; and protein‐level scores such as that of Grantham,18 SIFT,19 and PolyPhen.20 In the CADD system, a support vector machine (SVM) is trained and Phred‐like scores (scaled C‐scores) are defined ranging from 1‐99, based on the rank of each variant relative to all possible 8.6 billion substitutions in the human reference genome.9 The results of this transformation are “scaled” C‐scores. Top 10% in the ranking of CADD scores are assigned C‐score 10, top 1% to C‐score 20 and top 0.1% to C‐score 30 etc.

Patients with variants were divided into 4 groups: deleterious mutations; variants with C‐score ≥10; variants with 10 >  C‐score ≥1; and control group including both variants with C‐score <1 and wild type (because we found no difference in clinical features between variants with C‐score <1 and wild type). If the patient had several minor variants, she was divided into a group according to the higher value of the C‐score. Clinical features associated with HBOC were compared between each group and control group. Personal and family history of breast and/or ovarian cancer, histology of ovarian cancer (serous or non‐serous) and breast cancer (triple negative breast cancer [TNBC] or non‐TNBC), age of developing breast cancer (≤45 years or older), and whether bilateral or unilateral breast cancer were selected as the clinical features.

Finally, we showed our classification for the minor variants using C‐score and segregation studies compared with the classification of the other annotation systems (SFIT and PolyPhen2) and representative database of ClinVar.

### Statistical analysis

Frequency of variants were statistically analyzed using Fisher's exact test and χ2‐test as appropriate (StatMate by ATMS, Tokyo, Japan).

### RESULTS

Variants with C‐score ≥10 were involved in 19/283 (6.7%) in BRCA1 (Figure 1A) and in 34/283 (12%) in BRCA2 (Figure 1B). All deleterious mutations scored over 10 by CADD, except for one patient with a large deletion of BRCA2 which was excluded from this study.

C‐scores of (A) BRCA1 and (B) BRCA2 variants in 283 breast and/or ovarian cancers. Deleterious mutations are indicated by black column, minor variants are indicated by gray. B, A patient with a large deletion detected by Multiplex Ligation‐dependent Probe Amplification method has no C‐score which is indicated by a star

Number of variants including deleterious mutations and rare variants of BRCA1 was 33. Among the 33 variants, 19 cases (57%) were C‐score ≥10, none were 10 >  C‐score ≥1, and 14 cases were C‐score <1. Consequently, 5 cases out of the 19 cases with C‐score ≥10 (26%) were defined as deleterious (2 frameshift and 3 nonsense mutations).

Among the 119 variants in BRCA2, C‐score ≥10 was seen in 34/119 (28%) (Figure 1B). Deleterious mutation was defined in 14/34 (41%) variants with C‐score ≥10. However, C‐score was not obtained for 1 patient with a large deletion detected by MLPA that is indicated by a star in Figure 1B.

Frequency of BRCA1 C‐score ≥10 was significantly high (P = .01) in patients with ovarian cancer (including patients with breast and ovarian cancer) with 14.1% (15/106) compared with breast cancer (excluding patients with breast and ovarian cancer) with 2.8% (5/177). However this difference was not observed in BRCA2; 11% (19/177) in breast cancer, and 14% (15/106) in ovarian cancer (Table 1). Furthermore, the frequency of C‐score ≥10 of BRCA1 and BRCA2 is higher in serous adenocarcinoma compared with non‐serous adenocarcinoma of the ovary. Significant difference (P = .05) was seen in BRCA2 (Table 1). In 177 patients with breast cancer, frequency of C‐score ≥10 was not elevated in younger patients (≤45 years) and in bilateral breast cancer. Meanwhile, the frequency of BRCA1 C‐score ≥10 was elevated in TNBC with 12% (2/17) compared with non‐TNBC with 1.9% (3/160) but statistical significance was not seen.

Frequency of BRCA variation with CADD score ≧10 according to the clinical factors

BO; Breast and Ovarian Cancer; BC; Breast Cancer; OC; Ovarian Cancer; NS, not significant; TNBC; Triple Negative Breast Cancer

P‐value was calculated by Fisher's exact test.

() P‐value by χ2 test.

Regarding the family histories of 283 patients, the number of families with breast cancer did not correlate with the frequency of C‐score ≥10. Also, the frequency of BRCA1 C‐score ≥10 was 21% (4/19) in patients with ovarian cancer family history which was higher than in the patients without ovarian cancer family history at 5.7% (15/264) (P = .03 by χ2‐test, and not significant [NS] by Fisher's exact test).

We compared clinical features of HBOC among 4 groups: deleterious mutations; variants with C‐score ≥10; 10 >  C‐score ≥1; and control group (including variants with C‐score <1 and wild type). Frequencies of personal history and family history of ovarian cancer were significantly high in the group with deleterious mutations. Also, frequency of serous adenocarcinoma of ovary and TNBC was relatively high (Table 2). Similar findings were seen in the variants with C‐score ≥10 in which personal history of ovarian cancer was significantly high and frequency of serous adenocarcinoma was relatively high. According to clinical features of breast cancer, significant differences were not seen in both the deleterious mutations group and in variants with C‐score ≥10.

Cilinical features of variants with deleterious mutation, variants with C‐score≧10, 10>C‐score≧1

C, C‐score; OC, ovarian Cancer; BC, Breast Cancer; BO, Beast and Ovarian Cancer; TNBC, Triple Negtive Breast Cancer.

Significant P value is written by Bold.

Classifications for deleterious mutations and rare variants of BRCA1 and BRCA2 are listed in Tables 3 and 4, respectively. According to the C‐score and population frequency that was estimated by the number of patients carrying the same variants, we excluded variants as polymorphism and classified VUS for 7 patients with BRCA1 and 4 patients with BRCA2. In total, 11 patients out of 283 patients (3.9%) were classified as VUS.

Classifications for Variations of BRCA1 according to C‐score

MAF, Minor Allele Frequency; NA, Not Applicable; nd, not documented; VUS, Vaiant of Uncertain Significance.

Classifications for Variants of BRCA2 according to C‐score

MAF, Minor Allele Frequency; NA, Not Applicable; nd, not documented; VUS, Variant of Uncertain Significance.

The classifications for the other variants are shown in Table 3. Variants of p.Tyr856His and p.Ser1577Pro were seen in 5 and 3 patients, respectively. The population frequency was estimated as relatively high and classified as benign polymorphism. They were classified as conflicting (likely benign) in the ClinVar database. Meanwhile, variants with C‐score <10 in BRCA1 were observed in 13 cases. The variant of p.Met1628Thr was seen in 9 patients, estimated high population frequency to be classified as benign, and the remaining 4 variants were also classified as benign by other in silico analysis (SIFT and PolyPhen 2). These missense mutations were estimated as benign polymorphism.

The classifications for BRCA2 variants are shown in Table 4. The variant of p.Lys2729Asn with C‐score 23.1 was seen in 10 patients, in whom one patient had another BRCA2 deleterious mutation, so that we classified this variant as benign. Also, p.Lys322Gln with C‐score 16.8 was seen in 6 patients, in whom one patient was offered genetic counseling and her segregation study suggested this mutation was not pathogenic (see Case 2 in the following segregation studies). The remaining 3 variants (3/283 [1%]) were classified as VUS, and they were not documented in the ClinVar database.

We classified 11 variants with C‐score <10 as benign because of the following reasons. The variants of p.Gly2044Val, p.Val2109Ile, p.Met784Val, and p.Met784Val were seen in 12, 7, 12, and 38 patients to be estimated as benign by high population frequency. The other variants were seen individually. All of these variants were not documented in the ClinVar database, but were estimated as benign and/or tolerated in SIFT and PolyPhen2.

We obtained important information by genetic counseling and segregation studies for patients with rare variants as follows.

A 73‐year‐old woman with breast cancer was found to have BRCA1 p.Val271Met with C‐score of 24. This variant was judged benign by Myriad (we had validation data with FALCO Biosystems)11 and uncertain by ClinVar. She had an aunt with breast cancer, a brother with pancreatic cancer and a daughter with ovarian cancer (Figure 2A). Her daughter with ovarian cancer had genetic counseling and testing showed the same variant. She suffers from recurrent serous adenocarcinoma of the ovary. Correlation of the variant and HBOC was suspected in this case. We need further follow up for this family.

Family trees of 3 patients with minor variants. (A) A 73‐year‐old woman with breast cancer was found to have BRCA1 p.Val271Met with C‐score of 24. Her daughter with ovarian cancer had genetic counseling and testing showed the same variant. She suffered from recurrent serous adenocarcinoma of the ovary. B, A 45‐year‐old woman with breast cancer was found to have BRCA2 p.Lys322Gln with C‐score of 16.89. This variant was identified in 6 patients with breast cancer. Her sister with breast cancer was referred to genetic counseling and testing showed her not to have the same variant. C, A 71‐year‐old woman with breast cancer had a variant of BRCA1 p.Met1628Thr with C‐score of 0.023. This variant was not documented in the ClinVar database. Her daughter was referred to genetic counseling and testing. This variant was not found in the daughter, so the pathogenicity was defined to be low

A 45‐year‐old woman with breast cancer was found to have BRCA2 p.Lys322Gln with C‐score of 16.89. This variant was identified in 6 patients with breast cancer. She has a sister suffering from breast cancer since 40 years of age but no other family members with cancer (Figure 2B). Her sister was referred to genetic counseling and testing showed her not to have the same variant. The frequency of this mutation was estimated to be high. According to these findings, we recognized the pathogenicity of this variant as low.

A 71‐year‐old woman with breast cancer had a variant of BRCA1 p.Met1628Thr with C‐score of 0.023. This variant was not documented in the ClinVar database. Her mother had colon cancer, a brother had liver and gastric cancer, and a daughter suffered from breast cancer at 46 years of age (Figure 2C). Her daughter was referred to genetic counseling and testing. This variant was not found in the daughter, so the pathogenicity was defined to be low.

### Variants with C‐score ≥10 in BRCA1 and BRCA2

Variants with C‐score ≥10 were involved in 19/283 (6.7%) in BRCA1 (Figure 1A) and in 34/283 (12%) in BRCA2 (Figure 1B). All deleterious mutations scored over 10 by CADD, except for one patient with a large deletion of BRCA2 which was excluded from this study.

C‐scores of (A) BRCA1 and (B) BRCA2 variants in 283 breast and/or ovarian cancers. Deleterious mutations are indicated by black column, minor variants are indicated by gray. B, A patient with a large deletion detected by Multiplex Ligation‐dependent Probe Amplification method has no C‐score which is indicated by a star

Number of variants including deleterious mutations and rare variants of BRCA1 was 33. Among the 33 variants, 19 cases (57%) were C‐score ≥10, none were 10 >  C‐score ≥1, and 14 cases were C‐score <1. Consequently, 5 cases out of the 19 cases with C‐score ≥10 (26%) were defined as deleterious (2 frameshift and 3 nonsense mutations).

Among the 119 variants in BRCA2, C‐score ≥10 was seen in 34/119 (28%) (Figure 1B). Deleterious mutation was defined in 14/34 (41%) variants with C‐score ≥10. However, C‐score was not obtained for 1 patient with a large deletion detected by MLPA that is indicated by a star in Figure 1B.

Frequency of BRCA1 C‐score ≥10 was significantly high (P = .01) in patients with ovarian cancer (including patients with breast and ovarian cancer) with 14.1% (15/106) compared with breast cancer (excluding patients with breast and ovarian cancer) with 2.8% (5/177). However this difference was not observed in BRCA2; 11% (19/177) in breast cancer, and 14% (15/106) in ovarian cancer (Table 1). Furthermore, the frequency of C‐score ≥10 of BRCA1 and BRCA2 is higher in serous adenocarcinoma compared with non‐serous adenocarcinoma of the ovary. Significant difference (P = .05) was seen in BRCA2 (Table 1). In 177 patients with breast cancer, frequency of C‐score ≥10 was not elevated in younger patients (≤45 years) and in bilateral breast cancer. Meanwhile, the frequency of BRCA1 C‐score ≥10 was elevated in TNBC with 12% (2/17) compared with non‐TNBC with 1.9% (3/160) but statistical significance was not seen.

Frequency of BRCA variation with CADD score ≧10 according to the clinical factors

BO; Breast and Ovarian Cancer; BC; Breast Cancer; OC; Ovarian Cancer; NS, not significant; TNBC; Triple Negative Breast Cancer

P‐value was calculated by Fisher's exact test.

() P‐value by χ2 test.

Regarding the family histories of 283 patients, the number of families with breast cancer did not correlate with the frequency of C‐score ≥10. Also, the frequency of BRCA1 C‐score ≥10 was 21% (4/19) in patients with ovarian cancer family history which was higher than in the patients without ovarian cancer family history at 5.7% (15/264) (P = .03 by χ2‐test, and not significant [NS] by Fisher's exact test).

### Clinical features of variations with deleterious mutations, variants with C‐score ≥10, and 10 > C‐score ≥1

We compared clinical features of HBOC among 4 groups: deleterious mutations; variants with C‐score ≥10; 10 >  C‐score ≥1; and control group (including variants with C‐score <1 and wild type). Frequencies of personal history and family history of ovarian cancer were significantly high in the group with deleterious mutations. Also, frequency of serous adenocarcinoma of ovary and TNBC was relatively high (Table 2). Similar findings were seen in the variants with C‐score ≥10 in which personal history of ovarian cancer was significantly high and frequency of serous adenocarcinoma was relatively high. According to clinical features of breast cancer, significant differences were not seen in both the deleterious mutations group and in variants with C‐score ≥10.

Cilinical features of variants with deleterious mutation, variants with C‐score≧10, 10>C‐score≧1

C, C‐score; OC, ovarian Cancer; BC, Breast Cancer; BO, Beast and Ovarian Cancer; TNBC, Triple Negtive Breast Cancer.

Significant P value is written by Bold.

### Classifications for minor variants using C‐score and population frequencies

Classifications for deleterious mutations and rare variants of BRCA1 and BRCA2 are listed in Tables 3 and 4, respectively. According to the C‐score and population frequency that was estimated by the number of patients carrying the same variants, we excluded variants as polymorphism and classified VUS for 7 patients with BRCA1 and 4 patients with BRCA2. In total, 11 patients out of 283 patients (3.9%) were classified as VUS.

Classifications for Variations of BRCA1 according to C‐score

MAF, Minor Allele Frequency; NA, Not Applicable; nd, not documented; VUS, Vaiant of Uncertain Significance.

Classifications for Variants of BRCA2 according to C‐score

MAF, Minor Allele Frequency; NA, Not Applicable; nd, not documented; VUS, Variant of Uncertain Significance.

The classifications for the other variants are shown in Table 3. Variants of p.Tyr856His and p.Ser1577Pro were seen in 5 and 3 patients, respectively. The population frequency was estimated as relatively high and classified as benign polymorphism. They were classified as conflicting (likely benign) in the ClinVar database. Meanwhile, variants with C‐score <10 in BRCA1 were observed in 13 cases. The variant of p.Met1628Thr was seen in 9 patients, estimated high population frequency to be classified as benign, and the remaining 4 variants were also classified as benign by other in silico analysis (SIFT and PolyPhen 2). These missense mutations were estimated as benign polymorphism.

The classifications for BRCA2 variants are shown in Table 4. The variant of p.Lys2729Asn with C‐score 23.1 was seen in 10 patients, in whom one patient had another BRCA2 deleterious mutation, so that we classified this variant as benign. Also, p.Lys322Gln with C‐score 16.8 was seen in 6 patients, in whom one patient was offered genetic counseling and her segregation study suggested this mutation was not pathogenic (see Case 2 in the following segregation studies). The remaining 3 variants (3/283 [1%]) were classified as VUS, and they were not documented in the ClinVar database.

We classified 11 variants with C‐score <10 as benign because of the following reasons. The variants of p.Gly2044Val, p.Val2109Ile, p.Met784Val, and p.Met784Val were seen in 12, 7, 12, and 38 patients to be estimated as benign by high population frequency. The other variants were seen individually. All of these variants were not documented in the ClinVar database, but were estimated as benign and/or tolerated in SIFT and PolyPhen2.

### Segregation studies for 3 patients with rare variants

We obtained important information by genetic counseling and segregation studies for patients with rare variants as follows.

A 73‐year‐old woman with breast cancer was found to have BRCA1 p.Val271Met with C‐score of 24. This variant was judged benign by Myriad (we had validation data with FALCO Biosystems)11 and uncertain by ClinVar. She had an aunt with breast cancer, a brother with pancreatic cancer and a daughter with ovarian cancer (Figure 2A). Her daughter with ovarian cancer had genetic counseling and testing showed the same variant. She suffers from recurrent serous adenocarcinoma of the ovary. Correlation of the variant and HBOC was suspected in this case. We need further follow up for this family.

Family trees of 3 patients with minor variants. (A) A 73‐year‐old woman with breast cancer was found to have BRCA1 p.Val271Met with C‐score of 24. Her daughter with ovarian cancer had genetic counseling and testing showed the same variant. She suffered from recurrent serous adenocarcinoma of the ovary. B, A 45‐year‐old woman with breast cancer was found to have BRCA2 p.Lys322Gln with C‐score of 16.89. This variant was identified in 6 patients with breast cancer. Her sister with breast cancer was referred to genetic counseling and testing showed her not to have the same variant. C, A 71‐year‐old woman with breast cancer had a variant of BRCA1 p.Met1628Thr with C‐score of 0.023. This variant was not documented in the ClinVar database. Her daughter was referred to genetic counseling and testing. This variant was not found in the daughter, so the pathogenicity was defined to be low

A 45‐year‐old woman with breast cancer was found to have BRCA2 p.Lys322Gln with C‐score of 16.89. This variant was identified in 6 patients with breast cancer. She has a sister suffering from breast cancer since 40 years of age but no other family members with cancer (Figure 2B). Her sister was referred to genetic counseling and testing showed her not to have the same variant. The frequency of this mutation was estimated to be high. According to these findings, we recognized the pathogenicity of this variant as low.

A 71‐year‐old woman with breast cancer had a variant of BRCA1 p.Met1628Thr with C‐score of 0.023. This variant was not documented in the ClinVar database. Her mother had colon cancer, a brother had liver and gastric cancer, and a daughter suffered from breast cancer at 46 years of age (Figure 2C). Her daughter was referred to genetic counseling and testing. This variant was not found in the daughter, so the pathogenicity was defined to be low.

### Case 1

A 73‐year‐old woman with breast cancer was found to have BRCA1 p.Val271Met with C‐score of 24. This variant was judged benign by Myriad (we had validation data with FALCO Biosystems)11 and uncertain by ClinVar. She had an aunt with breast cancer, a brother with pancreatic cancer and a daughter with ovarian cancer (Figure 2A). Her daughter with ovarian cancer had genetic counseling and testing showed the same variant. She suffers from recurrent serous adenocarcinoma of the ovary. Correlation of the variant and HBOC was suspected in this case. We need further follow up for this family.

Family trees of 3 patients with minor variants. (A) A 73‐year‐old woman with breast cancer was found to have BRCA1 p.Val271Met with C‐score of 24. Her daughter with ovarian cancer had genetic counseling and testing showed the same variant. She suffered from recurrent serous adenocarcinoma of the ovary. B, A 45‐year‐old woman with breast cancer was found to have BRCA2 p.Lys322Gln with C‐score of 16.89. This variant was identified in 6 patients with breast cancer. Her sister with breast cancer was referred to genetic counseling and testing showed her not to have the same variant. C, A 71‐year‐old woman with breast cancer had a variant of BRCA1 p.Met1628Thr with C‐score of 0.023. This variant was not documented in the ClinVar database. Her daughter was referred to genetic counseling and testing. This variant was not found in the daughter, so the pathogenicity was defined to be low

### Case 2

A 45‐year‐old woman with breast cancer was found to have BRCA2 p.Lys322Gln with C‐score of 16.89. This variant was identified in 6 patients with breast cancer. She has a sister suffering from breast cancer since 40 years of age but no other family members with cancer (Figure 2B). Her sister was referred to genetic counseling and testing showed her not to have the same variant. The frequency of this mutation was estimated to be high. According to these findings, we recognized the pathogenicity of this variant as low.

### Case 3

A 71‐year‐old woman with breast cancer had a variant of BRCA1 p.Met1628Thr with C‐score of 0.023. This variant was not documented in the ClinVar database. Her mother had colon cancer, a brother had liver and gastric cancer, and a daughter suffered from breast cancer at 46 years of age (Figure 2C). Her daughter was referred to genetic counseling and testing. This variant was not found in the daughter, so the pathogenicity was defined to be low.

### DISCUSSION

It is important in genetic counseling and testing to provide an appropriate explanation for variants. Most variants may not be associated with a high risk of cancer but a misinterpreted variant has the potential to lead to mismanagement of patients and their relatives. IARC Unclassified Genetic Variants Working Group21 and other researchers22 recommended the following classification for variants. The classification consists of the following 5 categories: (i) ‘deleterious’ (pathogenic); (ii) ‘suspected deleterious’ (likely pathogenic); (iii) ‘VUS’; (iv) ‘genetic variant, favor polymorphism’ (likely not pathogenic); and (v) ‘polymorphism’ (not pathogenic). However, these 5 categories unfortunately confound the clinical direction of the patients.

There are several online database resources that provide some interpretation of BRCA sequence variants, such as The Breast Cancer Information Core (BIC),23 Human Variome Project,24 or ClinVar.25 We referred each variant to ClinVar as a standard reporting database recommended by The American College of Medical Genetics.7 As mentioned earlier, some variants were not documented in the ClinVar database, so that we needed another system to classify the variants.

There are many existing annotations useful for prioritizing causal variants (e.g. PolyPhen,26 SIFT,27 and GERP28), but they have several limitations. First, factors of annotations vary widely, from constitutions to functions. Second, each annotation has its own metric being rarely comparable. Third, each annotation was subject to major ascertainment biases and might not be generalized. Fourth, combined annotations might have only overlapping significance. These limitations have caused many potentially relevant annotations to be ignored.

Combined annotation‐dependent depletion is a framework for estimating the relative pathogenicity of human genetic variants by integrating many diverse annotations into a single, quantitative score. CADD has been implemented as a support vector machine trained to differentiate 14.7 million high‐frequency human‐derived alleles from 14.7 million simulated variants.9 Also, we can compute a “C‐score” for all 8.6 billion possible human single nucleotide variants and short insertion/deletions. C‐score correlates with allelic diversity, annotation of functionality, pathogenicity, disease severity, experimentally measured regulatory effect and complex trait associations, and highly ranks known pathogenic variants within individual genomes.

There are several studies of the power of CADD to classify the variants of familial cancer panels29, 30, 31 in which superiority of the CADD score rather than other in silico analysis is reported. Although limited clinical validity for the identification of pathogenic variants in non‐coding regions has been reported,30 as well as the original report by Kircher,9 the validity of the C‐score ≥10 in clinically relevant genes was also suggested in a dataset of mismatch repair gene variants.31 We still have no reports concerning the validation of CADD in BRCA mutations.

The classification of Myriad (Myriad's New Mutations Committee [NMC]) was made using 8 parameters8: 1, literature review; 2, population frequency; 3, mRNA splice‐site assay; 4, functional assays; 5, evolutionary conservation; 6, segregation analysis; 7, identification of homozygous and compound heterozygous individual (intrans); and 8, mutation co‐occurrence. Re‐classification was made by history‐weighing algorithm and the rate of VUS was reduced at 2.1% for all tested patients.32, 33 Although the classification by Myriad is valuable to determine the pathogenicity of minor variants, not every researcher can access the Myriad database. Furthermore, their algorithm is not designed for low penetrance mutations and newly discovered variants. In addition, it needs a huge clinical database. We need a new tool to classify minor variants despite no definite information being available. It is clear that there is currently no internationally accepted standard for BRCA testing report, and no agreed consistent classification system; some laboratories report variants without interpretation, some use a narrative approach and some use locally developed guidelines or published schemes.34, 35, 36

In our experience, CADD could rank all variants of BRCA by C‐scores in which all deleterious mutations were included except for the large deletion detected by the MLPA method. If we defined C‐Score ≥10 which means top 10% in ranking for pathogenicity, we could reduce the frequency of VUS at 3.9%. This value is satisfactory compared with the VUS rate reported by Myriad at 2.1%.8 Variants with C‐score <10 were estimated benign according to other factors, population frequency, other in silico analysis (SIFT and PolyPhen2), and mutation co‐occurrence. Furthermore, clinical features of patients with C‐score ≥10 supported the utility of CADD. Although significant differences were not observed in family history of ovarian cancer and frequency of serous adenocarcinoma, similar findings with deleterious mutations were seen. Patients with ovarian cancer, TNBC and family history of ovarian cancer were more frequent in variants with C‐score ≥10.

In practice, we could not define pathogenicity as “deleterious” by C‐score individually, as clinical evidence is necessary to determine the pathogenicity. CADD might be useful to select the VUS which is referred for genetic counseling and for segregation studies. Information gained by segregation studies is significant for both patients and clinicians. We can avoid an ambiguous explanation for the patient about variants using C‐score and segregation studies. Although several strategies to classify VUS were reported,37, 38 we established the counseling system using CADD in which targeted VUS were easily selected and we can propose significant segregation studies for the patients. We could reduce the frequency of VUS at 3.9% using CADD and population frequency. We tried further segregation studies for the patients with variants classified as VUS, and determined a benefit of using C‐score for genetic counseling.

### ACKNOWLEDGMENTS

We thank Hidetoshi Shigetomo, Yumi Kubota, Rituko Yokouchi, and Nao Sato for their assistance.

### CONFLICTS OF INTEREST

Authors declare no conflicts of interest for this article.



# SUPPLEMENTAL FILE 1: CAS-109-453.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: genetics_screening.pdf

None