# MAIN TEXT

## BRCA1 and BRCA2 Gene Mutations Screening In Sporadic Breast Cancer Patients In Kazakhstan

### Abstract

BackgroundA large number of distinct mutations in the BRCA1 and BRCA2 genes have been reported worldwide, but little is known regarding the role of these inherited susceptibility genes in breast cancer risk among Kazakhstan women.AimTo evaluate the role of BRCA1/2 mutations in Kazakhstan women presenting with sporadic breast cancer.MethodsWe investigated the distribution and nature of polymorphisms in BRCA1 and BRCA2 entire coding regions in 156 Kazakhstan sporadic breast cancer cases and 112 age-matched controls using automatic direct sequencing.ResultsWe identified 22 distinct variants, including 16 missense mutations and 6 polymorphisms in BRCA1/2 genes. In BRCA1, 9 missense mutations and 3 synonymous polymorphisms were observed. In BRCA2, 7 missense mutations and 3 polymorphisms were detected. There was a higher prevalence of observed mutations in Caucasian breast cancer cases compared to Asian cases (p<0.05); higher frequencies of sequence variants were observed in Asian controls. No recurrent or founder mutations were observed in BRCA1/2 genes. There were no statistically significant differences in age at diagnosis, tumor histology, size of tumor, and lymph node involvement between women with breast cancer with or without the BRCA sequence alterations.ConclusionsConsidering the majority of breast cancer cases are sporadic, the present study will be helpful in the evaluation of the need for the genetic screening of BRCA1/2 mutations and reliable genetic counseling for Kazakhstan sporadic breast cancer patients. Evaluation of common polymorphisms and mutations and breast cancer risk in families with genetic predisposition to breast cancer is ongoing in another current investigation.

### Introduction

Breast cancer is the most common malignancy in females and one of the leading causes of death from cancer in women worldwide.1 It accounts for 23% of all cancers among women and is the second most common cancer overall when both sexes are considered. Breast cancer showed the biggest proportional increase in the number of new cases in women from 2006–2011 and remains the leading form of cancer in women in Kazakhstan. 2

Germline mutations in BRCA1 and BRCA2 genes account for genetic predisposition and increased risk of breast and ovarian cancers.3 Two major breast cancer susceptibility genes are BRCA1 (MIM 113705, Genbank accession no. U14680) and BRCA2 (MIM 600185, Genbank accession no. U43746), located on the long arms of chromosomes 174 and 13,5 respectively, and both apparently function as tumor suppressor genes. BRCA1 is a large protein of 1863 amino acids and BRCA2, with 3418 amino acids, is even larger. Both proteins are involved in the control of homologous recombination (HR) and double-strand break repair in response to DNA damage.6–11
BRCA1/2 have been shown to serve as important central components in multiple biological pathways that regulate cell-cycle progression, centrosome duplication, DNA damage repair, cell growth, and apoptosis.12 Evidence that the loss of BRCA1 alleles or low expression of BRCA1 in a large proportion of sporadic breast cancer cases supports the role of BRCA1 in the development of sporadic breast cancer.13–15

Genetic linkage analysis4 and refine mapping16,17 provided the evidence of the location of penetrance as having a germline mutation; it was found that the most commonly encountered sporadic forms of breast cancer vary among different populations.18–20 This contributory variation may be attributed to their different gene pool make and also due to low penetrance gene involvement. However, some studies have pointed out that germline mutations in BRCA1/2 contribute little to the induction of breast cancer in some countries.21,22

Mutations in the BRCA1 and BRCA2 genes were first reported in conjunction with their identification in 199423 and 1995.24,25 During last decade BRCA1 and BRCA2 have been extensively screened for mutations; numerous mutations have been reported to be clearly associated with cancer susceptibility and have been registered in the Breast Cancer Information Core Database (BIC).26 Most of these are frameshift or nonsense mutations leading to truncated and therefore, inactive BRCA1/2 proteins. These mutations have a pathogenetic role and are characterized by a high penetrance. Furthermore, they are thought to increase the lifetime risk of developing breast cancer 50% by age 50 and 85% by age 70.27–29

Hundreds of alterations have been reported for BRCA1/2 genes, but not all are able to confer a higher risk of developing breast cancer with age27,28 and, indeed, the pathogenetic effect of a significant number of single amino acid changes is still unknown. This is particularly true for polymorphisms, naturally occurring gene sequence variations, often affecting only a single nucleotide, that have recently been associated with altered cancer risk.30–32 There are also a certain number of single amino acid changes that have been identified and classified as non-characterized variants. It is not known whether these variants may affect BRCA1 function and thus bring about an increased risk for breast and/or ovarian cancer.

Disease-associated mutations are distributed over the entire coding regions of these genes, and these demonstrate considerable ethnogeographic variation.33 For example, in the Ashkenazi Jewish34,35 or in the Icelanders36 breast cancer predisposition has been demonstrated to be due to recurrent mutations (founder mutations) originating from a single ancestor.

Molecular analysis of BRCA1/2 in different populations has demonstrated a very large mutational spectrum and variable mutation prevalence related to the different techniques employed, selection criteria, and ethnic origin of the patients. In families with no prior history of breast cancer, frequency of BRCA mutation was found significantly low from 0.02% to 10%.33 In Asia, the prevalence of BRCA1/2 mutations among unselected breast cancer cases was reported at 5.1% in Philippines,37 and 2.5%–3.1% in Korea.38,39 The frequency of BRCA1 mutations in Chinese women with breast cancer without family history was 5.4%.40 On the other hand, 2% of breast cancer cases in the largest breast cancer population-based study in a UK population showed association with BRCA genes with 0.7% attribution of BRCA1 (Anglian Breast Cancer Study Group, 2000).41 Several hospital-based series of unselected breast cancers implicate BRCA1 and BRCA2 in 2–5% and 0–2% of all cases, respectively.41 This variation may be attributed to differences on the genome level among various ethnic and population heterogeneity. The reason of marginally low penetrance of BRCA1/2 germline mutations may be attributed to the polygenic involvement and heterogeneity of samples origin too. As in Asia, the overall prevalence of germline mutation varies from 0.8% in Japanese42 to 8.0% in Singapore region43, indicating involvement of other genes and population response with respect to various types and origin of cancers. Moreover, inter-individual variation does exist among the ethnic groups in association with various risk factors as reported by Peto et al.44 showing mutation prevalence as 3.5% before age 35 declining to 0.49% in ≥ 50 years.

Studies on BRCA gene mutations have been mainly performed in western populations and the majority of these have involved hereditary breast and ovarian cancer families. Thus, studies focusing on sporadic breast cancer and data collection in Asians, especially in the Kazakh population, remain relatively sparse. To our knowledge, this is the first study to evaluate the frequency and type of sequence alterations of BRCA1/2 genes in Kazakhstan breast cancer patients. In order to evaluate the role of BRCA1 and BRCA2 germline mutations in the Kazakhstan population, 156 sporadic breast cancer patients were analyzed for mutations throughout the entire coding regions of the BRCA1 and BRCA2 genes by using direct sequencing.

### Materials and methods

Prior to this study, ethical approval was obtained from the Ethical committees of Semey State Medical University and in National Center for Biotechnology. A total of 156 Kazakhstan women with pathologically confirmed breast cancer; 83 women operated in Oncological Center in Semey, Semey, Kazakhstan from 1984–2005 year and 73 women operated in Astana Oncological Center, Astana, Kazakhstan from 2008–2011; year were enrolled in this study (breast cancer group). Family histories were obtained through individual interviews and only cases without family history of affected first- or second-degree relatives with breast and/or ovarian cancer were included in this study.

Clinical and pathological characteristics such as: age at diagnosis (operation), histological subtype (WHO histological classification)45 histological grade, T stage (TNM clinical classification, and lymph node involvement were obtained from medical records. Control subjects, matched to cases based on age and ethnicity were randomly selected from the community in Semey and Astana (n=112). None of the controls had a personal history of malignancy at the time of ascertainment. Before the study, written informed consent was obtained from all participants. Both cases and controls were divided in to two groups by race (by first name of study participants and their parents): Caucasian (n=88 and 40 cases and controls, respectively) and Asian (n=68 and 72 cases and controls, respectively).

Peripheral blood samples (ca.10 ml) were collected into vacutainers with K2EDTA.

Genomic DNA was extracted from whole blood samples from cases with breast cancer as well as controls using a Master Pure DNA purification Kit (EPICENTRE Biotechnologies, USA) and DNA extraction kit (Promega, USA) in accordance with the manufacturer’s protocols.

For PCR amplification of the 22 coding regions of BRCA1 and the 26 coding regions of BRCA2, primers using Primer 3 v. 0.4.0 program were designed (available from authors on request). Since exon 11 of BRCA1 varies by 3426 base pairs, we amplified 10 overlapping regions of this exon; for exon 10 of BRCA2, we designed 2 pairs of primers and for exon 11 of BRCA2 - 3 pairs of primers.

Amplification of DNA fragments was performed in Tetrad BioRad Thermal Cycler (BIO RAD, USA) in 25μl of solution containing 150 mM Tris–HCl (pH 8.0), 500 mM KCl, 25 mM MgCl2, 10 mM each dNTP, 10 pmol of primers, 25–125 ng of genomic DNA, and 2 units of AmpliTaq Gold DNA polymerase. The PCR was performed according to the following conditions: initial denaturation at 95°C for 10 minutes, followed by 35 cycles of 95°C for 15 seconds, 58°C for 30 seconds, and 72°C for 30 seconds. The quality of amplification was determined by separation of the PCR products on a 1.5% agarose gel. The PCR products were purified using ExoSAP-IT (USB, USA) and incubated at 37°C for 40 min, 80°C for 20 min, and stored at 4°C. Purified PCR products were further used in the sequencing reaction process.

All amplified products were sequenced in forward and reverse directions using the BigDye Terminator v3.1 Cycle sequencing Kit (Applied Biosystems, USA) on an ABI 3130xL DNA Analyzer (Applied Biosystems, Foster city, CA, USA). The PCR products were sequenced using the same primers as the ones used for PCR amplification. Sequence PCR products were cleaned using Sefadex 50 and MultiScreen filtration colons (Millipore Corporation, USA). A chromatographic tracing of each amplicon was analyzed by proprietary sequence analysis software (Sequence Analysis 5.3.1, SeqScape v.2.6, Finch TV v1.3.1) followed by visual inspection and confirmation. The sequence was compared with the Breast Cancer information Core, BICdatabase; http://research.nhgri.nih.gov/bic), Human Genome Mutation Database (http://www.hgmd.cf.ac.uk/ac/index.php) and the National center for Biotechnology Information database (http://www.ncbi.nlm.nih.gov).

Approved recommendations of nomenclature for the description of sequence variants were adopted (http://www.hgvs.org/mutnomen/). Numbering according to GenBank Accession no. NM_007294.1 for BRCA1 and NM_000059.1 for BRCA2, the A of the ATG translation initiation codon is +1, according to approved guidelines were used.

Also, traditional mutation nomenclature used in BIC database26 where nucleotide numbers refer to the wild type cDNA sequence of BRCA1 (RefSeq accession number U14680) with numbering starting at the A of the first ATG at the position 120 and to the wild type cDNA sequence of BRCA2 (RefSeq accession number U43746) with numbering starting at the A of the first ATG at the position 229 were used.

We used the term “sequence variation” and “sequence alteration” to prevent confusion with the terms “mutation” and “polymorphism”, mutation meaning “change” or “disease-causing change” and polymorphism meaning “non disease-causing change” or “change found at a frequency of 1% or higher in the population”.

Single-nucleotide polymorphisms may fall within coding sequences of genes, non-coding regions of genes, or in the intergenic regions (regions between genes). SNPs within a coding sequence do not necessarily change the amino acid sequence of the protein that is produced, due to degeneracy of the genetic code. A SNP in which both alleles produce the same polypeptide sequence is called a synonymous polymorphism (sometimes called a silent mutation). If a different polypeptide sequence is produced, the polymorphism is a replacement polymorphism. A replacement polymorphism change may be either missense, which results in a different amino acid, or nonsense, which results in a premature stop codon. Over half of all known disease mutations come from replacement polymorphisms.24

A variation in a genetic sequence whose association with disease risk is unknown is also called a variant of uncertain significance, unclassified variant, and VUS (an alteration in the normal sequence of a gene, the significance of which is unclear until further study of the genotype and corresponding phenotype in a sufficiently large population).

Clinical and pathological characteristics and BRCA mutation results were analyzed using SPSS 19.0 (SPSS, Tokyo, Japan). Differences in categorical variables between mutation-positive and mutation-negative group were compared using chi-square analysis, cross tables or Fisher’s exact test. A probability value of less than 0.05 was considered to indicate significance.

### Patients

Prior to this study, ethical approval was obtained from the Ethical committees of Semey State Medical University and in National Center for Biotechnology. A total of 156 Kazakhstan women with pathologically confirmed breast cancer; 83 women operated in Oncological Center in Semey, Semey, Kazakhstan from 1984–2005 year and 73 women operated in Astana Oncological Center, Astana, Kazakhstan from 2008–2011; year were enrolled in this study (breast cancer group). Family histories were obtained through individual interviews and only cases without family history of affected first- or second-degree relatives with breast and/or ovarian cancer were included in this study.

Clinical and pathological characteristics such as: age at diagnosis (operation), histological subtype (WHO histological classification)45 histological grade, T stage (TNM clinical classification, and lymph node involvement were obtained from medical records. Control subjects, matched to cases based on age and ethnicity were randomly selected from the community in Semey and Astana (n=112). None of the controls had a personal history of malignancy at the time of ascertainment. Before the study, written informed consent was obtained from all participants. Both cases and controls were divided in to two groups by race (by first name of study participants and their parents): Caucasian (n=88 and 40 cases and controls, respectively) and Asian (n=68 and 72 cases and controls, respectively).

### Blood sample collection

Peripheral blood samples (ca.10 ml) were collected into vacutainers with K2EDTA.

### DNA extraction

Genomic DNA was extracted from whole blood samples from cases with breast cancer as well as controls using a Master Pure DNA purification Kit (EPICENTRE Biotechnologies, USA) and DNA extraction kit (Promega, USA) in accordance with the manufacturer’s protocols.

### Polymerase chain reaction

For PCR amplification of the 22 coding regions of BRCA1 and the 26 coding regions of BRCA2, primers using Primer 3 v. 0.4.0 program were designed (available from authors on request). Since exon 11 of BRCA1 varies by 3426 base pairs, we amplified 10 overlapping regions of this exon; for exon 10 of BRCA2, we designed 2 pairs of primers and for exon 11 of BRCA2 - 3 pairs of primers.

Amplification of DNA fragments was performed in Tetrad BioRad Thermal Cycler (BIO RAD, USA) in 25μl of solution containing 150 mM Tris–HCl (pH 8.0), 500 mM KCl, 25 mM MgCl2, 10 mM each dNTP, 10 pmol of primers, 25–125 ng of genomic DNA, and 2 units of AmpliTaq Gold DNA polymerase. The PCR was performed according to the following conditions: initial denaturation at 95°C for 10 minutes, followed by 35 cycles of 95°C for 15 seconds, 58°C for 30 seconds, and 72°C for 30 seconds. The quality of amplification was determined by separation of the PCR products on a 1.5% agarose gel. The PCR products were purified using ExoSAP-IT (USB, USA) and incubated at 37°C for 40 min, 80°C for 20 min, and stored at 4°C. Purified PCR products were further used in the sequencing reaction process.

### DNA sequencing

All amplified products were sequenced in forward and reverse directions using the BigDye Terminator v3.1 Cycle sequencing Kit (Applied Biosystems, USA) on an ABI 3130xL DNA Analyzer (Applied Biosystems, Foster city, CA, USA). The PCR products were sequenced using the same primers as the ones used for PCR amplification. Sequence PCR products were cleaned using Sefadex 50 and MultiScreen filtration colons (Millipore Corporation, USA). A chromatographic tracing of each amplicon was analyzed by proprietary sequence analysis software (Sequence Analysis 5.3.1, SeqScape v.2.6, Finch TV v1.3.1) followed by visual inspection and confirmation. The sequence was compared with the Breast Cancer information Core, BICdatabase; http://research.nhgri.nih.gov/bic), Human Genome Mutation Database (http://www.hgmd.cf.ac.uk/ac/index.php) and the National center for Biotechnology Information database (http://www.ncbi.nlm.nih.gov).

### Mutation Nomenclature

Approved recommendations of nomenclature for the description of sequence variants were adopted (http://www.hgvs.org/mutnomen/). Numbering according to GenBank Accession no. NM_007294.1 for BRCA1 and NM_000059.1 for BRCA2, the A of the ATG translation initiation codon is +1, according to approved guidelines were used.

Also, traditional mutation nomenclature used in BIC database26 where nucleotide numbers refer to the wild type cDNA sequence of BRCA1 (RefSeq accession number U14680) with numbering starting at the A of the first ATG at the position 120 and to the wild type cDNA sequence of BRCA2 (RefSeq accession number U43746) with numbering starting at the A of the first ATG at the position 229 were used.

We used the term “sequence variation” and “sequence alteration” to prevent confusion with the terms “mutation” and “polymorphism”, mutation meaning “change” or “disease-causing change” and polymorphism meaning “non disease-causing change” or “change found at a frequency of 1% or higher in the population”.

Single-nucleotide polymorphisms may fall within coding sequences of genes, non-coding regions of genes, or in the intergenic regions (regions between genes). SNPs within a coding sequence do not necessarily change the amino acid sequence of the protein that is produced, due to degeneracy of the genetic code. A SNP in which both alleles produce the same polypeptide sequence is called a synonymous polymorphism (sometimes called a silent mutation). If a different polypeptide sequence is produced, the polymorphism is a replacement polymorphism. A replacement polymorphism change may be either missense, which results in a different amino acid, or nonsense, which results in a premature stop codon. Over half of all known disease mutations come from replacement polymorphisms.24

A variation in a genetic sequence whose association with disease risk is unknown is also called a variant of uncertain significance, unclassified variant, and VUS (an alteration in the normal sequence of a gene, the significance of which is unclear until further study of the genotype and corresponding phenotype in a sufficiently large population).

### Statistical analysis

Clinical and pathological characteristics and BRCA mutation results were analyzed using SPSS 19.0 (SPSS, Tokyo, Japan). Differences in categorical variables between mutation-positive and mutation-negative group were compared using chi-square analysis, cross tables or Fisher’s exact test. A probability value of less than 0.05 was considered to indicate significance.

### Results

Mean age of breast cancer patients and controls was 51.2±9.5 years and 57.8±9.1 years, respectively. Sixty-eight women with breast cancer were Asian and eighty eight – Caucasian. Among healthy women, 72 were Asian and 40 - Caucasian. All breast cancer cases were distributed by size of primary breast carcinoma (TNM clinical classification45): T1, T2, T3, T4 – 15, 96, 39, 6 cases respectively, 14 cases with N1 lymph node metastasis, 15 women with N1 lymph node metastasis, and 5 women had N3 lymph node involvement.

A summary of patient characteristics can be found in:

Characteristics of Study Participants

T – primary tumor: T1 - ≤2cm, T2 – >2 to 5cm, T3 - >5cm, T4 – tumor of any size with direct extension to chest wall or skin, N – regional lymph nodes, M – distant metastasis

p<0.05 vs controls

Mutational screening of BRCA1 and BRCA2 coding regions was performed for 156 sporadic breast cancer cases and 112 controls. Mutation analysis of the BRCA1 and BRCA2 genes revealed the presence of 22 distinct variants, including 16 missense mutations and 6 polymorphisms (Tables 2 and 3).

The sequence variants identified in BRCA1 gene 9 missense mutations – c.95G>T (G32V), c.254A>G (N85S), c.1067A>G (Q356R), c.2612C>T (P871L), c.3113A>G (E1038G), c.3348A>G (K1183R), c.4744C>G (S1542C), c.5397A>G (A1627G) and c.5585T>G (T1862G), three synonymous polymorphisms - Ser694Ser, Leu771Leu and Ser1436Ser. Most of these polymorphisms were found in exon 11 of BRCA1, which is 60% of all BRCA1 coding region.

Detection of the sequence variants in 11 exon of BRCA1.

(A) DNA sequence of 11 exon of the forward strand of BRCA1 from patient reveals a heterozygous missense mutation c.1067A>G (Q356R), result in amino acid change Gln→Arg (right), and wild type (left). (B) A homozygous missense mutation c.2612C>T (P871L), result in amino acid change Pro→Leu (reverse strand). (C) A heterozygous missense mutation c.3113A>G (E1038G), result in amino acid change Glu→Gly (forward strand). (D) A homozygous missense mutation c.3348A>G (K1183R), result in amino acid change Lys→Arg (forward strand). (E) A synonymous polymorphism c.2082C>T Ser694Ser (S694S) (forward strand). (F) A synonymous polymorphism c.2311T>C Leu771Leu (L771L) (forward strand)

Details and the frequency of the variants detected in the BRCA1 in breast cancer and control groups.

Numbering according to GenBank Accession no. NM_007294.1, the A of the ATG translation initiation codon is +1,according to approved guidelines (http://www.hgvs.org/mutnomen/)

The nomenclature as used in the BIC database is given in parentheses. Mutation nomenclature is according to RefSeq accession number U14680 (BRCA1) with numbering starting at the A of the first ATG at the position 120

p<0.05,

p<0.001 between cases and controls

p<0.05,

p<0.001 between Caucasians and Asians in group

Missense mutations c.95G>T (G32V), c.254A>G (N85S), c.3348A>G (K1183R) were detected with frequency in 54.1%, 53.0%, 59.0% of cases and 43.7%, 37.5%, 45.5% of controls, respectively, showing significantly higher prevalence in cases (p<0.05). There was no significant prevalence in frequency of missense mutations c.1067A>G (Q356R), c.2612C>T (P871L), c.3113A>G (E1038G), c.4744C>G (S1542C), c.5397A>G (A1627G) in cases comparing to controls. Synonymous polymorphisms S694S and L771L were detected in 45 (54.2%) of cases and 48 (42.8%) controls. S1436S was found in 76 (49.0%) cases and 54 (48.2%) controls.

In BRCA2, gene missense mutations c.865A>C (N289H), c.10234A>G (I3412V) were detected with higher frequency in 15.2% and 30.8% of cases compared to controls (10.1%, 18.7%, respectively, p<0.05).

Details and the frequency of the variants detected in the BRCA2 in breast cancer and control groups.

Numbering according to GenBank Accession no. NM_000059.1, the A of the ATG translation initiation codon is +1, according to approved guidelines (http://www.hgvs.org/mutnomen/).

The nomenclature as used in the BIC database is given in parentheses. Mutation nomenclature is according to GenBank accession number U43746 (BRCA2) with numbering starting at the A of the first ATG at the posotion 229.

p<0.05,

p<0.001 between cases and controls

p<0.05,

p<0.001 between Caucasians and Asians in group

There was no significant difference in frequency of missense mutations c.2350A>G (M784V), c.2410G>A (D804N), c.3422T>C (I1141T), c.3572C>T (S1191F) in cases compared to controls. Synonymous polymorphisms S455S, H473H, were detected in 81 (52.2%) cases and S2114S was detected in 61 (39.0%) cases, p>0.05 (Table 3).

All identified polymorphisms were previously reported in the BIC database26 (Table 2,3).

Interestingly, frequency of nearly all sequence variants was significantly different in Caucasian and Asian groups of study participants (Table 2,3). There was higher prevalence of observed mutations in Caucasian cases comparing to Asian breast cancer cases (p<0.05) and higher frequency of sequence variants was observed in the control group in Asians.

The clinical and pathological characteristics of women with breast cancer are summarized in:

Clinical-pathological profile of breast cancer cases with/or without BRCA1/2 polymorphisms.

T – primary tumor: T1 - ≤2cm, T2 – >2 to 5cm, T3 - >5cm, T4 – tumor of any size with direct extension to chest wall or skin, N – regional lymph nodes, M – distant metastasis

There was no significant difference between both groups of cases in mean age at diagnosis (50.6±8.8 years vs. 52.7±10.7 years, respectively, p=0.36). Assuming that the absolute numbers of co-existing BRCA1/2 alterations in each case may also play a relevant biological role, the cases in the mutation-positive group were divided in three subgroups: (a) with 1 alteration; (b) with 2–3 alterations; (c) with 4–6 alterations. We found that 79 of 156 women (71.2%) carried 4–6 alterations. This may be due to polymorphisms located in one haplogroup and inherited together. Also amongst controls, in 48 (73.8%) of 65 alteration carriers we detected more than 3 mutations simultaneously (data not shown).

Invasive ductal carcinoma was the predominant histological subtype in both groups (61.0% and 70.8%, respectively). Primary tumors larger than 5.0 cm were more frequently found in women carrying sequence alterations. However, the size of tumor (T stage) and lymph node involvement did not show a statistically significant difference between these two groups (p=0.92 and p=1.0, respectively).

### Discussion

156 Kazakhstan patients with sporadic breast cancer were analyzed for mutations throughout the entire coding regions of the BRCA1 and BRCA2 genes, using direct sequencing. Whereas the majority of studies on BRCA gene mutations have focused on western populations with a family history of breast or ovarian cancer, only a relatively small number of investigations on the role of the BRCA genes have been undertaken in Asian sporadic breast cancer populations. A large number of distinct mutations in the BRCA1 and BRCA2 genes have been reported worldwide, but little is known regarding the role of these inherited susceptibility genes in breast cancer risk among Kazakhstan women. So far there was no information about the role of the BRCA1/2 gene in breast cancer risk among Kazakhstan women.

The incidence of detectable BRCA1 sequence alterations was estimated for the first time in females with sporadic breast cancer as well as in healthy women from Semipalatinsk (East Kazakhstan) region of Kazakhstan in our studies for the first time beginning from 2006.46 We showed a higher prevalence of BRCA1 sequence alterations in exon 11 in 59 (71.1%) from 83 women with breast cancer and in 65 (58.0%) from 112 healthy women (controls).

In the present study, the entire coding regions of BRCA1 and BRCA2 were analyzed, and patients with sporadic breast cancer were selected by excluding patients at high risk of being mutation-carriers. This included those with a family history of breast or ovarian cancer, those diagnosed at less than 35 years of age, and those with bilateral or multifocal breast cancer. Given that the great majority of breast cancer cases are sporadic, further extensive studies are needed to precisely identify the roles of BRCA genes in sporadic breast cancer. In our present study we sequenced all coding regions of both BRCA1 and BRCA2 genes in larger group of cases including an Astana cohort, where women gathered from different Kazakhstan regions.

The sequence variants identified in the BRCA1/2 genes include 16 missense mutations of unknown clinical significance and 6 synonymous polymorphisms by mutation type (Table 2, 3 and Figure 1). All cases of the single nucleotide changes in BRCA1 and BRCA2 detected in the study were recorded according to the Breast Cancer Information Core.26

It seems likely that five mutations in BRCA1 (c.95G>T, c.254 A>G, c.2612C>T, c.3113A>G, c.3348A>G) and three mutations in BRCA2 (c.2127T>C, c.2410G>A, c.10234A>G) are neutral polymorphisms, in view of the relatively high allele frequencies (>30%) of these variants. Because of the unavailability of a functional BRCA protein assay system, the disease associations of other mutations remain uncertain. However, the possibility cannot be ruled out that some of these unverified variants are pathogenically relevant.

No deleterious mutations were detected in either gene among studied groups. The lack in the germline of clearly deleterious alterations might be unsurprising. In fact, these are extremely rare in patients not selected due to a family history and/or early disease onset.41 However, the true contribution made by the BRCA genes to sporadic breast cancer remains controversial for a number of reasons. First, missense mutations with an unknown significance could have a pathogenic effect. Secondly, in addition to missense mutations, silent polymorphisms may affect the splicing mechanism. However, these variants cannot be classified as disease associated in the absence of a good functional assay system for BRCA1 and BRCA2. When a functional assay becomes available, it will be important to elucidate the relevance of such variations with unknown clinical significances.

To our knowledge, this report is the first to include information on the prevalence of missense mutations of unknown significance, and to provide information on polymorphisms in the Kazakhstan population in both BRCA1 and BRCA2 genes. Moreover, these Kazakhstan population-based polymorphisms could be used as potential markers.

Missense mutation Q356R in BRCA1 was detected in 11 (7.2%) breast cancer cases and 10 (8.9%) control subjects. A study on the BRCA1 polymorphisms reported that the Arg356 allele had a higher genotype distribution in healthy controls than in breast cancer patients30 and may thus play a protective role against breast cancer. In this study, the polymorphism at codon 356 in the BRCA1 gene had previously been described as being inversely associated with breast cancer risk (Gln356→ Arg, OR 0.88, 95% confidence interval [CI] 0.63–1.23; Arg356→ Arg, OR 0.00, 95% CI 0.00–0.56).30 Another study showed that Q356R polymorphism was significantly associated with family history of ovarian cancer, suggesting that this sequence variant may increase ovarian cancer risk.31

In contrast, Tommasi et al.47 analyzed BRCA1 mutational risk using Myriad II software and showed that K1183R, the polymorphism in exon 11 as reported in BIC26, resulted inversely related with BRCA1 mutation carrier status. Also they showed that BRCA1 sequence alterations such as P871L and E1038G were not significantly related with higher BRCA1 mutational risk.47 This data leads us to suggest further investigation of the effects of these sequence variants on BRCA1 activity to understand whether these variations have any pathological role.

We found that 79 of 156 women (71.2%) carried 4–6 alterations. This may be due to polymorphisms located in one haplogroup that are inherited together. Dunning et al examined the frequency of four polymorphisms: Gln356Arg, Pro871Leu, Glu1038Gly and Ser1613Gly in large series of breast and ovarian cancer cases and matched controls.30. Due to strong linkage disequilibrium, these four sites generate only three haplotypes with a frequency >1.3%. The two most common haplotypes, defined by the alleles Gln356Pro871Glu1038Ser1613 and Gln356Leu871Gly1038Gly1613, have frequencies of 0.57 and 0.32, respectively, and these frequencies do not differ significantly between patient and control groups, indicating that the most common polymorphisms of the BRCA1 gene do not make a significant contribution to breast or ovarian cancer risk.30

Common polymorphisms in BRCA1/2 genes appear to be highly prevalent in Kazakhstan breast cancer cases and in healthy controls which is in concordance with previously reported findings in some Asian and European populations.37–43, 47–49 Frequencies of these polymorphisms were higher in breast cancer cases vs controls, particularly in the BRCA2 gene, p<0.05. To date, there is little evidence that highly penetrant, germline mutations in BRCA1/2 are observed in sporadic cases, but whether common polymorphisms play a role in disease risk is still controversial.

The roles of common missense SNPs, as well as variation in noncoding regions (that may influence risk through expression levels and alternative splicing), have yet to be thoroughly explored (at single loci as well as throughout the genome) as markers of breast cancer susceptibility. Ongoing efforts to systematically characterize genetic polymorphisms, such as the International HapMap Project,50 provide the foundation for conducting comprehensive association studies of common variation.

In a large study, Freedman ML et al tested common variation across the BRCA1 locus in African American, Native Hawaiian, Japanese, Latino, and White women in the Multiethnic Cohort Study.51 28 single nucleotide polymorphisms (SNPs) spanning the BRCA1 gene were used to define patterns of common variation in these populations. The majority of SNPs were in strong linkage disequilibrium with one another. Nine tagging SNPs, including five missense SNPs, were selected to predict the common BRCA1 variants and haplotypes among the non–African American groups (five additional SNPs were required for African Americans) and genotyped in a breast cancer case-control study nested in the Multiethnic Cohort Study (cases, n = 1,715; controls, n = 2,502). In their another study52 Freedman ML et al observed most of the common BRCA2 haplotypes to be shared among Native Hawaiians, Japanese, Latinos, and Whites; four of the eight common haplotypes were found in at least three ethnic populations and six of the eight were found in at least two groups. Authors found no evidence for significant associations between common variation in BRCA1/2 and risk of breast cancer.51,52

In another study, Cox DG et al reported a modestly positive association between a BRCA1 haplotype and breast cancer among White women in the Nurses’ Health Study (OR, 1.18, 95% CI, 1.02–1.37).53

Interestingly, the frequencies of nearly all sequence variants were significantly different in Caucasian and Asian groups among study participants (Table 2, 3). There was higher prevalence of observed mutations in Caucasian cases comparing to Asian breast cancer cases, (p<0.05) and higher frequency of sequence variants was observed in control group in Asians. Further studies of large numbers of cases may give a more accurate estimation of prevalence and variations between Asian and Caucasian populations in Kazakhstan. The vast majority of common variation is shared between populations; however, allele frequencies are known to vary across populations54 and studies conducted in a multiethnic population may lend insight into better understanding ethnic differences in breast cancer risk.55

Although the role of common variation in BRCA1 and sporadic breast cancer risk has been thoroughly addressed, it remains a possibility that this locus may still prove to be involved in breast cancer risk. Specifically, rare (<5%) variants may contribute to disease; to address this hypothesis, however, large-scale resequencing efforts (to discover the rare variants) and testing of these variants in larger cohorts, such as the National Cancer Institute Consortium of Cohorts,56 will be required. Another possibility is that a sporadic breast cancer is actually a collection of genetically distinct subclasses of breast cancer. In this scenario, it would be unlikely that the same set of underlying susceptibility alleles occur in all breast cancer cases (i.e., the genetic architecture of disease is not genetically homogeneous). If these subgroups are not recognized and analyzed separately, then the power to detect them will be diminished. At the histologic and molecular levels, breast tumors have different characteristics; subsets of breast tumors as defined by immunohistochemistry [e.g., estrogen-receptor (+/−) and HER2/neu(+/−)] often display different biological behaviors, such as time to disease progression and response to therapy, which may reflect different genetic origins. Studies have shown that breast tumors of women with hereditary breast cancer with mutations in BRCA1 and a subset of women with sporadic disease (f25%) share similar traits, including a basal cell histology, higher grade tumors, cytokeratins 5/6, and estrogen receptor negativity, suggesting they may have a similar etiology.57–59 The ability to stratify breast cancer cases by expression profiling, immunohistochemistry, methylation patterns, and/or clinical variables may facilitate the identification of more genetically homogeneous subsets of cancer cases, and therefore may help to identify causal variants underlying specific breast cancer phenotypes.

There are several limitations to this study. We did not include results of screening entire exons of BRCA1 and BRCA2 genes in families with breast/ovarian cancer members. This study is still ongoing because family BC cases in Kazakhstan are sparse. Further evaluation is needed to clarify the relationship between frequency of sequence alterations in BRCA1/2 genes and breast cancer risks in Kazakhstan women in family-based and genetically homogeneous cases.

In conclusion, 156 Kazakhstan patients with sporadic breast cancer were analyzed for mutations throughout the entire coding regions of the BRCA1 and BRCA2 genes, using direct sequencing. The present investigation revealed 22 different sequence variants. Although we found none of pathological deleterious mutations in BRCA1/2 genes, we believe that the present study allows a better evaluation of the need for the genetic screening of BRCA mutations in sporadic breast cancer patients in Kazakhstan. However, large population-based screening studies are needed to establish the frequency, penetrance, and significance of the broad spectrum of variations in the sequence of BRCA1/2 genes in Kazakhstan population. It is hoped that similar mutation surveys in other Central Asian countries will be completed so that information can be compared and the most common mutations identified.



# SUPPLEMENTAL FILE 1: cajgh-02-29.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)