# MAIN TEXT

## Breast cancer patients suggestive of Li-Fraumeni syndrome: mutational spectrum, candidate genes, and unexplained heredity

### Abstract

BackgroundBreast cancer is the most prevalent tumor entity in Li-Fraumeni syndrome. Up to 80% of individuals with a Li-Fraumeni-like phenotype do not harbor detectable causative germline TP53 variants. Yet, no systematic panel analyses for a wide range of cancer predisposition genes have been conducted on cohorts of women with breast cancer fulfilling Li-Fraumeni(-like) clinical diagnostic criteria.MethodsTo specifically help explain the diagnostic gap of TP53 wild-type Li-Fraumeni(-like) breast cancer cases, we performed array-based CGH (comparative genomic hybridization) and panel-based sequencing of 94 cancer predisposition genes on 83 breast cancer patients suggestive of Li-Fraumeni syndrome who had previously had negative test results for causative BRCA1, BRCA2, and TP53 germline variants.ResultsWe identified 13 pathogenic or likely pathogenic germline variants in ten patients and in nine genes, including four copy number aberrations and nine single-nucleotide variants or small indels. Three patients presented as double-mutation carriers involving two different genes each. In five patients (5 of 83; 6% of cohort), we detected causative pathogenic variants in established hereditary breast cancer susceptibility genes (i.e., PALB2, CHEK2, ATM). Five further patients (5 of 83; 6% of cohort) were found to harbor pathogenic variants in genes lacking a firm association with breast cancer susceptibility to date (i.e., Fanconi pathway genes, RECQ family genes, CDKN2A/p14ARF, and RUNX1).ConclusionsOur study details the mutational spectrum in breast cancer patients suggestive of Li-Fraumeni syndrome and indicates the need for intensified research on monoallelic variants in Fanconi pathway and RECQ family genes. Notably, this study further reveals a large portion of still unexplained Li-Fraumeni(-like) cases, warranting comprehensive investigation of recently described candidate genes as well as noncoding regions of the TP53 gene in patients with Li-Fraumeni(-like) syndrome lacking TP53 variants in coding regions.Electronic supplementary materialThe online version of this article (10.1186/s13058-018-1011-1) contains supplementary material, which is available to authorized users.

### Background

Li-Fraumeni syndrome (LFS) is a rare but highly penetrant cancer predisposition syndrome characterized by the early onset and familial aggregation of a variety of malignant neoplasms [1]. Germline pathogenic variants (PVs) in the tumor suppressor gene TP53 are primarily responsible for this autosomal dominantly inherited disease [2]; however, because TP53 PVs can be confirmed in only about 70% of suspected families [3], diagnosis of LFS is usually based on clinical evaluation and conformance to stringent criteria independent of mutational status. Different diagnostic criteria with varying stringency in terms of tumor abundance, age of onset, and spectrum of malignancies are in use, including classic LFS criteria, Birch’s and Eeles’ Li-Fraumeni-like syndrome (LFL) criteria, and several versions of the Chompret criteria [1, 4–9]. Although in principle any type of neoplasm may occur, a set of core cancers, namely breast cancer (BC), sarcomas, brain tumors, adrenocortical carcinomas, and leukemia, are expected to account for up to 77% of all tumor types occurring in patients with LFS [10].

Penetrance is remarkably high in carriers of germline TP53 PVs, with 84% of female carriers and 41% of male carriers developing a tumor by age 45 years [11]. The gender difference is due to BC being one of the most predominating factors in this syndrome, representing up to 80% of all cancer cases in the age class of 16–45 years in females with LFS [10, 11]. However, because the syndrome itself is rare, the contribution of TP53 germline alterations to hereditary BC overall is estimated to be less than 1% [12]. Because patients with BC harboring germline TP53 PVs typically present with very early age of onset, routine TP53 testing has been suggested for women who develop BC before the age of 30 years, independent of family history, and TP53 detection rates within cohorts of patients with early-onset BC have been reported to be between 4% and 8% [8, 13, 14]. The 2008 and 2015 versions of the revised Chompret criteria [7, 9] are the sole criteria incorporating this important factor into their diagnostics, allowing for patients with early-onset BC (< 36 years) or very early-onset BC (< 31 years), respectively, to be included in LFS/LFL diagnostic procedures. Yet, two recent studies focused on cohorts of women meeting hereditary breast and ovarian cancer (HBOC) criteria clearly illustrate that a large percentage of germline TP53 mutation carriers may still be missed by current criteria; the two studies reported TP53 PVs in 13 patients overall, half of whom did not clinically meet either classic LFS or Chompret criteria, nor did they present with very early-onset disease [15, 16].

While not all germline TP53 mutation carriers may be covered by LFS/LFL criteria, many families who do clinically conform to LFS/LFL criteria lack detectable germline TP53 PVs, which is demonstrated by TP53 mutation detection rates ranging from ~ 55% to 70% in classic LFS criteria, ~ 25% to 30% in LFL criteria, and ~ 20% to 35% in Chompret criteria [3, 6, 13]. This means that up to 45% of patients meeting classical LFS criteria and up to 80% of patients meeting Chompret or LFL criteria are left unexplained in a genetic sense. Few other candidate genes in the TP53 pathway have been investigated in this context, one of them being CDKN2A, which was recently found to be mutated on a germline level in several LFL families in which the index case had a sarcoma [17]. Moreover, germline PVs in CHEK2 have continuously but controversially been implicated in LFL phenotypes.

To the best of our knowledge, no systematic panel analyses for a wide range of known and suspected cancer predisposition genes have been conducted on cohorts of women with BC fulfilling LFL clinical diagnostic criteria. To further address this issue, as well as to elucidate the mutational spectrum in German population subjects with BC suggestive of LFS/LFL, we conducted both massive parallel sequencing and copy number analyses for a set of 94 cancer predisposition genes in a cohort of 83 TP53-negative and BRCA1/2-negative BC patients from the German population who met at least one of the hitherto suggested Li-Fraumeni-related criteria (LFS/LFL/Chompret criteria).

### Methods

Eighty-three unrelated study subjects were selected from among a pool of female patients with BC who conformed to the German consortium criteria for HBOC [18] and whose pedigree had been established during genetic counseling in the tumor genetics outpatient clinic at Hannover Medical School, Germany, between 2002 and 2015. Prior to study inclusion, all patients had a negative test result for pathogenic single-nucleotide variants or small indels (Sanger sequencing) as well as gross genomic rearrangements (multiplex ligation-dependent probe amplification [MLPA], SALSA MLPA kits [MRC Holland, Amsterdam, The Netherlands], P002B/P087 for BRCA1, P045-B3/P077 for BRCA2, and P056-C1 for TP53) within the genes BRCA1, BRCA2, and TP53. Final inclusion of study participants was based on personal medical or family history characteristics suggestive of Li-Fraumeni-like traits and conformance to at least one of the hitherto existing Li-Fraumeni-related criteria (LFS/LFL/Chompret criteria). Classification into LFS/LFL and Chompret criteria was performed either on the particular proband undergoing mutation analysis or on another index case within the same family. The majority of patients were assumed to be of European-Caucasian ancestry. All patients signed informed consent forms, and the project was approved by the research ethics committee of Hannover Medical School (approval number 3528).

Among our cohort of 83 patients with BC, 50 unrelated women had early-onset BC (age at diagnosis ranging from 19 to 34 years, median age 29 years), 9 index cases were diagnosed with bilateral BC or two primary independent breast carcinomas, 4 individuals harbored at least one independent neoplasm besides BC, in 10 families either the index patient or one of her relatives had a sarcoma, and in 16 families either the index patient or a relative was diagnosed with a brain tumor (multiple entries being possible).

Germline DNA was isolated from peripheral blood leukocytes according to standard procedures. For library preparation, the Illumina TruSight Cancer Panel (Illumina, San Diego, CA, USA) was used (see Additional file 1 for the entire list of the 94 included genes and SNPs). All samples were processed according to the manufacturer’s recommendations and sequenced on an Illumina NextSeq 500 platform. Mean sequence depth was at least 100 × for each sample. A minimum sequence depth of 20 × for at least 97% of the region of interest (ROI) (i.e., coding regions and the first two base pairs [bp] of flanking intronic regions) could be obtained for 81 of 83 samples; 2 samples failed to confer to these quality parameters. The majority of samples (68 of 83; 82%) reached 20 × coverage for at least 99% of the target region. All 94 panel genes were analyzed using the megSAP analysis pipeline [19], filtering for variants with a minimum variant allele frequency (VAF) of 15%, a relatively low VAF that was chosen for the eventuality of constitutional TP53 mosaicism [20]. Yet, all detected PVs exhibited a VAF of > 40% (range, 43–56%). Reported nucleotide positions refer to GRCh37/hg19.

Integrative Genomics Viewer (IGV) [21, 22] was employed to visualize sequencing results. Variants were further filtered with the software GSvar (part of ngs-bits [23]) to identify frameshift, nonsense, splice site, missense, inframe insertion/deletion, and 3′/5′ untranslated region (UTR) variants that had a minor allele frequency (MAF) at or below 0.1% in the 1000 Genomes Project [24], ExAC [25], and Kaviar [26] databases and were located in the ROI as defined above. As a second measure, via GSvar analysis, we filtered all variants that were predicted to be pathogenic by at least 2 of 4 in silico prediction tools (i.e., MetaLR [27], Sift [28], PolyPhen-2 HVAR, and PolyPhen-2 HDIV [29]) or that were documented as pathogenic in either the ClinVar [30] or HGMD [31] database, as long as they were neither listed n > 2 in the FLOSSIES database of healthy older women [32], nor classified as class 1–2 variants according to a consented expert decision within the German HBOC Consortium, nor reported as predominantly benign or likely benign in ClinVar [30]. Variants were classified according to American College of Medical Genetics and Genomics (ACMG) guidelines [33]. For splicing prediction, in silico splicing tools (i.e., SSF, MaxEnt, NNSplice, GeneSplicer, and HSF) included in Alamut software version 2.8 rev. 1 (interactive biosoftware, Rouen, France) were used. In addition, and with particular relevance to potential TP53 variants of reduced penetrance or hypomorphic alleles, all variants detected within the TP53 gene (including synonymous and intronic variants) were analyzed. Regarding TP53’s UTRs, the Illumina gene panel used captures approximately 70 bp of both 5′ and 3′ UTRs with adequate depth, resulting in only a small fraction of the 3′ UTR being covered.

For detection of copy number changes, a custom-made eArray covering the identical set of genes targeted by Illumina’s TruSight Cancer Panel was used (SureDesign 069100, 8x60K; Agilent Technologies, Santa Clara, CA, USA) [34]. All samples were processed according to the manufacturer’s instructions. Microarray slides were scanned using an Agilent microarray scanner system, and standard settings of the Feature Extraction Software (version 11.0.1.1) were applied for data normalization. Data analysis was subsequently performed via Agilent’s Genomic Workbench (version 7.0.4.0). Nucleotide positions refer to GRCh37/hg19. Annotation of alterations was computed under different settings dependent on data quality. Verification of detected copy number variations with a second independent method was performed either via MLPA, if probe sets were commercially available for the relevant region (SALSA MLPA kits [MRC Holland], P042-B1 for ATM; P056 for CHEK2; P008-C1 for PMS2), and/or via next-generation sequencing (NGS)-based copy number evaluation using CnvHunter, which is part of ngs-bits [23].

### Study cohort

Eighty-three unrelated study subjects were selected from among a pool of female patients with BC who conformed to the German consortium criteria for HBOC [18] and whose pedigree had been established during genetic counseling in the tumor genetics outpatient clinic at Hannover Medical School, Germany, between 2002 and 2015. Prior to study inclusion, all patients had a negative test result for pathogenic single-nucleotide variants or small indels (Sanger sequencing) as well as gross genomic rearrangements (multiplex ligation-dependent probe amplification [MLPA], SALSA MLPA kits [MRC Holland, Amsterdam, The Netherlands], P002B/P087 for BRCA1, P045-B3/P077 for BRCA2, and P056-C1 for TP53) within the genes BRCA1, BRCA2, and TP53. Final inclusion of study participants was based on personal medical or family history characteristics suggestive of Li-Fraumeni-like traits and conformance to at least one of the hitherto existing Li-Fraumeni-related criteria (LFS/LFL/Chompret criteria). Classification into LFS/LFL and Chompret criteria was performed either on the particular proband undergoing mutation analysis or on another index case within the same family. The majority of patients were assumed to be of European-Caucasian ancestry. All patients signed informed consent forms, and the project was approved by the research ethics committee of Hannover Medical School (approval number 3528).

Among our cohort of 83 patients with BC, 50 unrelated women had early-onset BC (age at diagnosis ranging from 19 to 34 years, median age 29 years), 9 index cases were diagnosed with bilateral BC or two primary independent breast carcinomas, 4 individuals harbored at least one independent neoplasm besides BC, in 10 families either the index patient or one of her relatives had a sarcoma, and in 16 families either the index patient or a relative was diagnosed with a brain tumor (multiple entries being possible).

### Patient selection

Eighty-three unrelated study subjects were selected from among a pool of female patients with BC who conformed to the German consortium criteria for HBOC [18] and whose pedigree had been established during genetic counseling in the tumor genetics outpatient clinic at Hannover Medical School, Germany, between 2002 and 2015. Prior to study inclusion, all patients had a negative test result for pathogenic single-nucleotide variants or small indels (Sanger sequencing) as well as gross genomic rearrangements (multiplex ligation-dependent probe amplification [MLPA], SALSA MLPA kits [MRC Holland, Amsterdam, The Netherlands], P002B/P087 for BRCA1, P045-B3/P077 for BRCA2, and P056-C1 for TP53) within the genes BRCA1, BRCA2, and TP53. Final inclusion of study participants was based on personal medical or family history characteristics suggestive of Li-Fraumeni-like traits and conformance to at least one of the hitherto existing Li-Fraumeni-related criteria (LFS/LFL/Chompret criteria). Classification into LFS/LFL and Chompret criteria was performed either on the particular proband undergoing mutation analysis or on another index case within the same family. The majority of patients were assumed to be of European-Caucasian ancestry. All patients signed informed consent forms, and the project was approved by the research ethics committee of Hannover Medical School (approval number 3528).

### Cohort characteristics

Among our cohort of 83 patients with BC, 50 unrelated women had early-onset BC (age at diagnosis ranging from 19 to 34 years, median age 29 years), 9 index cases were diagnosed with bilateral BC or two primary independent breast carcinomas, 4 individuals harbored at least one independent neoplasm besides BC, in 10 families either the index patient or one of her relatives had a sarcoma, and in 16 families either the index patient or a relative was diagnosed with a brain tumor (multiple entries being possible).

### Next-generation sequencing and bioinformatics analysis

Germline DNA was isolated from peripheral blood leukocytes according to standard procedures. For library preparation, the Illumina TruSight Cancer Panel (Illumina, San Diego, CA, USA) was used (see Additional file 1 for the entire list of the 94 included genes and SNPs). All samples were processed according to the manufacturer’s recommendations and sequenced on an Illumina NextSeq 500 platform. Mean sequence depth was at least 100 × for each sample. A minimum sequence depth of 20 × for at least 97% of the region of interest (ROI) (i.e., coding regions and the first two base pairs [bp] of flanking intronic regions) could be obtained for 81 of 83 samples; 2 samples failed to confer to these quality parameters. The majority of samples (68 of 83; 82%) reached 20 × coverage for at least 99% of the target region. All 94 panel genes were analyzed using the megSAP analysis pipeline [19], filtering for variants with a minimum variant allele frequency (VAF) of 15%, a relatively low VAF that was chosen for the eventuality of constitutional TP53 mosaicism [20]. Yet, all detected PVs exhibited a VAF of > 40% (range, 43–56%). Reported nucleotide positions refer to GRCh37/hg19.

Integrative Genomics Viewer (IGV) [21, 22] was employed to visualize sequencing results. Variants were further filtered with the software GSvar (part of ngs-bits [23]) to identify frameshift, nonsense, splice site, missense, inframe insertion/deletion, and 3′/5′ untranslated region (UTR) variants that had a minor allele frequency (MAF) at or below 0.1% in the 1000 Genomes Project [24], ExAC [25], and Kaviar [26] databases and were located in the ROI as defined above. As a second measure, via GSvar analysis, we filtered all variants that were predicted to be pathogenic by at least 2 of 4 in silico prediction tools (i.e., MetaLR [27], Sift [28], PolyPhen-2 HVAR, and PolyPhen-2 HDIV [29]) or that were documented as pathogenic in either the ClinVar [30] or HGMD [31] database, as long as they were neither listed n > 2 in the FLOSSIES database of healthy older women [32], nor classified as class 1–2 variants according to a consented expert decision within the German HBOC Consortium, nor reported as predominantly benign or likely benign in ClinVar [30]. Variants were classified according to American College of Medical Genetics and Genomics (ACMG) guidelines [33]. For splicing prediction, in silico splicing tools (i.e., SSF, MaxEnt, NNSplice, GeneSplicer, and HSF) included in Alamut software version 2.8 rev. 1 (interactive biosoftware, Rouen, France) were used. In addition, and with particular relevance to potential TP53 variants of reduced penetrance or hypomorphic alleles, all variants detected within the TP53 gene (including synonymous and intronic variants) were analyzed. Regarding TP53’s UTRs, the Illumina gene panel used captures approximately 70 bp of both 5′ and 3′ UTRs with adequate depth, resulting in only a small fraction of the 3′ UTR being covered.

### Array CGH and copy number evaluation

For detection of copy number changes, a custom-made eArray covering the identical set of genes targeted by Illumina’s TruSight Cancer Panel was used (SureDesign 069100, 8x60K; Agilent Technologies, Santa Clara, CA, USA) [34]. All samples were processed according to the manufacturer’s instructions. Microarray slides were scanned using an Agilent microarray scanner system, and standard settings of the Feature Extraction Software (version 11.0.1.1) were applied for data normalization. Data analysis was subsequently performed via Agilent’s Genomic Workbench (version 7.0.4.0). Nucleotide positions refer to GRCh37/hg19. Annotation of alterations was computed under different settings dependent on data quality. Verification of detected copy number variations with a second independent method was performed either via MLPA, if probe sets were commercially available for the relevant region (SALSA MLPA kits [MRC Holland], P042-B1 for ATM; P056 for CHEK2; P008-C1 for PMS2), and/or via next-generation sequencing (NGS)-based copy number evaluation using CnvHunter, which is part of ngs-bits [23].

### Results

Of the 83 families in our cohort, 48 met Eeles’ LFL criteria [5]; 23 met the original stringent Chompret criteria [6]; 75 were consistent with the temporarily suggested 2008 version of Chompret criteria, which loosened age restrictions and included all BRCA1/2-negative patients with BC before age 36 years, regardless of family history [7]; 43 met the 2009 version of Chompret criteria, which again reversed the latter point [8]; 53 met the current Chompret 2015 version criteria, which by definition include all BRCA1/2-negative patients with BC before age 31 years, regardless of family history [9]; 12 families met Birch’s LFL criteria [4]; and 1 family met classic LFS criteria [1] (multiple entries being possible). Considering mutation carriers’ families only (10 of 83 families), all 10 families were consistent with the 2008 version of Chompret criteria, whereas 6 of 10 could be classified into Eeles’ LFL and Chompret’s 2009 version criteria. When we examined the mutation detection rate for each classification system separately, we found that the majority of the applied criteria performed similarly, with the highest detection rate observed within members of both Eeles’ LFL criteria and Chompret’s original/2008/2009 criteria (13–14%).

For an overview of the classification results and phenotypic characteristics of the cohort, see Table 1. Information about LFS and LFL criteria definitions is given in Additional file 2.Table 1Classification and cohort characteristicsTotal sample, n (%)Mutation carriers, n (%)Mutation carriers per group, n (%)Eeles’ LFL criteria48/83 (58%)6/10 (60%)6/48 (13%)Birch’s LFL criteria12/83 (15%)1/10 (10%)1/12 (8%)Original Chompret criteria23/83 (28%)3/10 (30%)3/23 (13%)Chompret 2008 version criteria75/83 (90%)10/10 (100%)10/75 (13%)Chompret 2009 version criteria43/83 (52%)6/10 (60%)6/43 (14%)Chompret 2015 version criteria53/83 (64%)5/10 (50%)5/53 (9%)Classic LFS criteria1/83 (1%)0/100/1Early-onset BC (i.e., ≤ 34 years)50/83 (60%)7/10 (70%)7/50 (14%)Bilateral BC/two primary BCs9/83 (11%)2/10 (20%)2/9 (22%)Additional neoplasms besides BC4/83 (5%)0/100/4Sarcoma in family or self10/83 (12%)0/100/10Brain tumor in family or self16/83 (20%)2/10 (20%)2/16 (13%)Abbreviations: BC Breast cancer, LFL Li-Fraumeni-like syndrome, LFS Li-Fraumeni syndromeSee Additional file 2 regarding LFS/LFL criteria

Classification and cohort characteristics

Abbreviations: BC Breast cancer, LFL Li-Fraumeni-like syndrome, LFS Li-Fraumeni syndrome

See Additional file 2 regarding LFS/LFL criteria

Within the cohort of 83 subjects with BC, we detected 13 pathogenic or likely pathogenic heterozygous germline variants (ACMG class 4–5; i.e., nonsense, frameshift, missense, [consensus] splice site variants, or copy number aberrations) in 10 unrelated patients and 9 genes [ATM (n = 3), CDKN2A (n = 1), CHEK2 (n = 1), FANCI (n = 1), PALB2 (n = 2), PMS2 (n = 1), RECQL4 (n = 2), RUNX1 (n = 1), and WRN (n = 1)]. These 13 deleterious variants include 4 gross genomic deletions/insertions and 9 truncating NGS-detected variants. Altogether, 3 patients presented as double-mutation carriers, harboring 2 (likely) PVs each (i.e., PMS2 and FANCI, CDKN2A and RECQL4, as well as ATM and CHEK2). Half of the 10 mutation carriers (5 of 83; 6% of the entire cohort) were found to harbor PVs in widely accepted BC susceptibility genes (i.e., PALB2, ATM, and CHEK2), whereas the remaining half (5 of 83; 6% of the entire cohort) carried PVs in candidate genes for which no firm association with BC incidence has yet been established in a heterozygous germline setting (CDKN2A, RUNX1, FANCI, WRN, and RECQL4). Four variants detected via NGS and 2 of the 4 copy number aberrations detected via array-based CGH (comparative genomic hybridization) have not previously been described in the literature. For an overview of all identified classes 4 and 5 variants, see Table 2. An overview of all index patients carrying (likely) pathogenic germline variants, including their personal and family histories as well as available clinical data and LFS/LFL classification, is given in Table 3, and respective pedigrees can be accessed in Additional file 3.Table 2Detected pathogenic and likely pathogenic single-nucleotide variants, small indels, and copy number alterationsIndex patient identifierGene (transcript GRCh37/hg19)Variant typedbSNPExons/total no. of exonscDNA changePredicted amino acid changeAberration array CGHACMG classPreviously reporteda7PALB2 (NM_024675.3)Frameshiftrs5157261244/13c.509_510delGAp.(Arg170Ilefs*14)5Yes30RUNX1 (NM_001754.4)Splice donorrs3751313723/8c.97+1G>A4No32ATM (NM_000051.3)Deletion62–63/63arr [GRCh37]11q22.3 (108233779_108240057)×15Yes40ATM (NM_000051.3)Nonsense61/63c.8793T>Ap.(Cys2931*)5YesCHEK2 (NM_007194.3)Deletion9–10/15arr [GRCh37]22q12.1 (29092709_29097723)×15Yes58CDKN2A (NM_058195.3)Nonsense2/3c.292C>Tp.(Arg98*)4NoRECQL4 (NM_004260.3)Splice donor7/21c.1390+1G>C4No59WRN (NM_000553.4)Deletion15–16/35arr [GRCh37]8p12 (30948138_30949422)×14No60ATM (NM_000051.3)Nonsensers58777985240/63c.5932G>Tp.(Glu1978*)5Yes65FANCI (NM_001113378.1)Nonsensers12191816437/38c.3853C>Tp.(Arg1285*)5YesPMS2 (NM_000535.5)Deletion3–8/15arr [GRCh37]7p22.1 (6035238_6042593)×15No76PALB2 (NM_024675.3)Frameshiftrs1801771433/13c.172_175delTTGTp.(Gln60Argfs*7)5Yes79RECQL4 (NM_004260.3)Missense and splice regionrs18673907216/22c.2755G>Ap.(Ala919Thr)4NoAbbreviations: ACMG American College of Medical Genetics and Genomics, cDNA Complementary DNA, CGH Comparative genomic hybridization, dbSNP Single Nucleotide Polymorphism databaseaBased on reports in the literature, ClinVar, Decipher, and gene-specific databasesTable 3Clinical information and summary of personal and family histories of mutation carriersIndex patient identifierVariantPersonal history (age at diagnosis in years)Immunohistochemistry (if available)Family history of cancer (age at diagnosis in years)Conformance to LFS/LFL criteriaa7PALB2:p.(Arg170Ilefs*14)BC (40)1. TNBCMat. – M: BC (40)EelesPat. – F: leukemia (< 50), GM: esophagus (60)Chompret 2008Chompret 200930RUNX1:c.97+1G>ABC (33)N/AMat. – M: BC (56), GM: BC (60)Chompret 200832ATM exon 62–63 delBC bilateral (30 + 40)1. Triple-positive2. Triple-positiveMat. – M: OvCa (51), half-S: BC (41) + lung (46), U: leukemia (45), GM: cancer, GF: cancerEelesorig. ChompretChompret 2008Chompret 2009Chompret 201540ATM:p.(Cys2931*)BC (39)N/AMat. – M: OvCa (40), A: leukemia (20)EelesCHEK2 exon 9–10 delChompret 2008Chompret 200958CDKN2A:p.(Arg98*)BC (32)HER2+Mat. – GM: ureter (64)Chompret 2008RECQL4:c.1390+1G>CPat. – GF: hypopharynx (62)59WRN exon 15–16 delBC (32)N/AMat. – half-S: melanoma (46), GF: lung (64), GM: CRC (50), GGM: CRC (59), U: kidney (60), this U’s sons: melanoma (45), basalioma (36)Eelesorig. ChompretChompret 2008Pat. – U: brain (25), 10 Us/As: all died of cancer at a young ageChompret 2009Chompret 2015Birch60ATM:p.(Glu1978*)BC (40)HR+Mat. – M: BC (51), GM: BC (73)EelesPat. – F: glioblastoma (42)Chompret 2008Chompret 2009Chompret 201565FANCI:p.(Arg1285*)BC (30)HER2+Mat. – U: CRC (37), GF: CRC (70), GGM: cancerChompret 2008PMS2 exon 3–8 delPat. – GF: esophagus (74)Chompret 201576PALB2:p.(Gln60Argfs*7)BC bilateral (33 + 39)1. HR+ lobular2. HR+ (HER2+ in metastases)B: lung (43)EelesMat. – M: pancreas (58), A: melanoma (67)orig. ChompretChompret 2008Chompret 200979RECQL4:p.(Ala919Thr)BC (27)HR+Mat. – M: NHL (42), GF: bladder (54)Chompret 2008Pat. – GM: BC (53), GU: lymphoma (52), GA: BC (57), this GA’s daughter: BC (47)Chompret 2015Abbreviations: A Aunt, B Brother, BC Breast cancer, CRC Colorectal cancer, del Deletion, F Father, GA Grand aunt, GF Grandfather, GGM Great grandmother, GM Grandmother, GU Grand uncle, HER2+ HER2 (ERBB2) overexpression/amplification, HR+ Hormone receptor-positive, M Mother, Mat. Maternal, N/A Not accessible, Ne Nephew, NHL Non-Hodgkin lymphoma, Ni Niece, OvCa Ovarian cancer, Pat. Paternal, S Sister, TNBC Triple-negative breast cancer, U UncleaInformation about Li-Fraumeni and Li-Fraumeni-like criteria definition are given in Additional file 2

Detected pathogenic and likely pathogenic single-nucleotide variants, small indels, and copy number alterations

Abbreviations: ACMG American College of Medical Genetics and Genomics, cDNA Complementary DNA, CGH Comparative genomic hybridization, dbSNP Single Nucleotide Polymorphism database

aBased on reports in the literature, ClinVar, Decipher, and gene-specific databases

Clinical information and summary of personal and family histories of mutation carriers

Abbreviations: A Aunt, B Brother, BC Breast cancer, CRC Colorectal cancer, del Deletion, F Father, GA Grand aunt, GF Grandfather, GGM Great grandmother, GM Grandmother, GU Grand uncle, HER2+ HER2 (ERBB2) overexpression/amplification, HR+ Hormone receptor-positive, M Mother, Mat. Maternal, N/A Not accessible, Ne Nephew, NHL Non-Hodgkin lymphoma, Ni Niece, OvCa Ovarian cancer, Pat. Paternal, S Sister, TNBC Triple-negative breast cancer, U Uncle

aInformation about Li-Fraumeni and Li-Fraumeni-like criteria definition are given in Additional file 2

Besides classes 4–5 variants, we detected 49 variants of unknown significance (VUS, ACMG class 3) within our collective for the parameters detailed above. Via array-based CGH, we identified a duplication of exons 1–21 of the BLM gene, which currently ranks as a class 3 variant owing to a lack of more precise breakpoint information. Via NGS-based sequencing, we detected 46 missense or disruptive inframe deletion and 2 splice region VUS with an MAF ≤ 0.1% occurring in 35 of the 94 investigated genes. Among this list of VUS, we observed an unexpected frequency of very rare FANCA missense variants that were linked to either very early-onset BC or exceptionally Li-Fraumeni-suggestive family/personal history. For a summary of all VUS with an MAF ≤ 0.1% detected via NGS-based sequencing, see Table 4. Neither of the identified VUS was included in the statistical analysis of this work.Table 4Missense and splice variants of unknown significance detected by NGS-based sequencingGene (transcript GRCh37/hg19)Index patient identifierVariant typeExons/total no. of exonscDNA changePredicted amino acid changedbSNPTotal allele frequency, gnomAD; population with highest AFaIn silico predictionbSplicing predictions at nearest natural junctionALK (NM_004304.4)52Splice region and synonymous17/29c.2817C>Tp.(Gly939Gly)rs1120224660.0001840; SA: 0.0006173Native acceptor site:MaxEnt: + 23.7%NNSPLICE: + 9.5%HSF: + 0.9%GeneSplicer: + 32.0%APC (NM_001127510.2)83Missense17/17c.4918C>Tp.(Arg1640Trp)rs3734406140.00007223;CO: 0.0001548ATM (NM_000051.3)20Missense10/63c.1271C>Ap.(Pro424His)rs1474726130.00002188;DA: 0.00004189ATM (NM_000051.3)20Missense50/63c.7357C>Tp.(Arg2453Cys)rs7554185710.00001219;CEA: 0.00005807BRCA2 (NM_000059.3)36Missense10/27c.831T>Gp.(Asn277Lys)rs288977050.00006632;BNFE: 0.0001440BRCA2 (NM_000059.3)30Missense11/27c.6101G>Ap.(Arg2034His)rs803588490.000004069;BNFE: 0.000008973BRCA2 (NM_000059.3)80Missense14/27c.7021C>Tp.(Arg2341Cys)rs412935050.00002439;DEA: 0.00005798BRIP1 (NM_032043.2)68Missense15/20c.2220G>Tp.(Gln740His)rs455896370.0005198;BL: 0.001395CHEK2 (NM_007194.379Missense4/15c.539G>Ap.(Arg180His)rs1378530090.00006494;CL: 0.0002615CHEK2 (NM_007194.3)83Missense6/15c.688G>Cp.(Ala230Pro)rs7486362160.000004063;DNFE: 0.000008956DDB2 (NM_000107.2)23Missense7/10c.947C>Tp.(Ser316Phe)rs3757889660.00001218;CSA: 0.00003249DICER1 (NM_030621.4)8Missense25/29c.4228A>Tp.(Asn1410Tyr)not foundDDIS3L2 (NM_152383.4)65Missense20/21c.2450C>Tp.(Thr817Met)rs3768168580.00006697;CSA: 0.0003505EGFR (NM_005228.3)54Missense17/28c.2039G>Ap.(Arg680Gln)rs3733362510.00009084;DFE: 0.0003912ERCC2 (NM_000400.3)24Disruptive inframe deletion20/23c.1857_1859delCATp.(Ile619del)0.000008127;EA: 0.00005800ERCC2 (NM_000400.3)55Missense22/23c.2083C>Tp.(Arg695Cys)rs2013929110.0001372;DA: 0.0003331ERCC4 (NM_005236.2)23Missense11/11c.2395C>Tp.(Arg799Trp)rs1219130490.0004476;DNFE: 0.0008138ERCC5 (NM_000123.3)23Missense1/15c.56C>Tp.(Pro19Leu)rs342913970.0005702;CNFE: 0.001092ERCC5 (NM_000123.3)76Missense13/15c.2818G>Ap.(Val940Met)rs1463448550.0009378;CAJ: 0.005122EXT2 (NM_000401.3)13Missense6/14c.1064G>Ap.(Arg355His)rs1497275180.0006422;DAJ: 0.003448EXT2 (NM_000401.3)54Missense7/14c.1186G>Ap.(Val396Met)rs1389430910.0004148;DNFE: 0.0007261FANCA (NM_000135.2)41, 73Missense1/43c.64T>Gp.(Trp22Gly)not foundDFANCA (NM_000135.2)8Missense16/43c.1489C>Gp.(Pro497Ala)not foundDFANCA (NM_000135.2)27Missense22/43c.2000C > Gp.(Pro667Arg)rs7552935960.00002230;DSA: 0.00004382FANCA (NM_000135.2)49Missense35/43c.3430C>Tp.(Arg1144Trp)rs1436718720.0005269;DNFE: 0.0009237FANCA (NM_000135.2)54Missense37/43c.3688C>Gp.(Leu1230Val)rs5764014590.00002030;DA: 0.00006535FANCD2 (NM_033084.3)57Missense21/43c.1933G>Tp.(Asp645Tyr)rs1464962530.0001371;CNFE: 0.0002288FANCI (NM_001113378.1)35Missense17/38c.1589T>Cp.(Leu530Pro)rs7663461560.000008122;DSA: 0.00006497GPC3 (NM_001164617.1)19Missense8/9c.1630C>Tp.(Arg544Cys)rs7595437030.00009270;DA: 0.0001707HNF1A (NM_000545.6)48Missense5/10c.1061C>Tp.(Thr354Met)rs7570688090.00006495;DL: 0.0001743KIT (NM_000222.2)39Missense3/21c.391G>Ap.(Asp131Asn)0.00001807;CA: 0.00004164MET (NM_001127500.1)13Missense2/21c.1076G>Ap.(Arg359Gln)rs2012740410.0002347;DNFE: 0.0004298MET (NM_001127500.1)75Missense14/21c.3023G>Ap.(Ser1008Asn)not foundDMLH1 (NM_000249.3)51Missense13/19c.1457C>Tp.(Ser486Phe)rs5328731410.000004061;CEA: 0.00005798NF1 (NM_001042492)28Missense5/58c.575G>Ap.(Arg192Gln)rs5877816700.00005294;CEA: 0.0005803NSD1 (NM_022455.4)10Missense23/23c.7352G>Ap.(Arg2451Lys)rs2001156650.0001119;DNFE: 0.0002212PALB2 (NM_024675.3)56Missense8/13c.2792T>Gp.(Leu931Arg)rs7738313040.00001221;DNFE: 0.00002694PMS1 (NM_000534.4)67Missense2/13c.118G>Cp.(Val40Leu)not foundBPMS1 (NM_000534.4)12Missense3/13c.287C>Ap.(Ala96Asp)rs1394146060.000004063;DNFE: 0.000008962PTCH1 (NM_001083602.1)4Missense23/24c.3749A>Gp.(Tyr1250Cys)rs1470671710.0005463;DAJ: 0.001098RAD51C (NM_058216.2)42Splice region8/8c.1026+5_1026+7 delGTArs7473119930.00001219;Native donor site:SSF: − 100.0%MaxEnt: − 100.0%NNSPLICE: − 99.6%HSF: − 14.8%NFE: 0.00002687RAD51D (NM_002878.3)52Missense5/10c.355T>Cp.(Cys119Arg)rs2013138610.00005413;BL: 0.00008716RHBDF2 (NM_024599.5)3, 75Missense8/19c.940G>Ap.(Ala314Thr)rs1404333740.0007812;CFE: 0.001173SDHB (NM_003000.2)81Missense2/8c.178A>Gp.(Thr60Ala)rs345992810.00006095;DNFE: 0.0001165SLX4 (NM_032444.2)14Missense14/15c.4831G>Ap.(Glu1611Lys)rs7661104790.00002847;DEA: 0.00005798TSC1 (NM_000368.4)51Missense12/23c.1178C>Tp.(Thr393Ile)rs2014522380.00002170;DA: 0.00004165TSC2 (NM_000548.4)32Missense36/42c.4582G>Cp.(Glu1528Gln)not foundDWRN (NM_000553.4)55Missense19/35c.2131C>Tp.(Arg711Trp)rs345607880.0002057;DNFE: 0.0003948Abbreviations: HSF Human Splicing Finder, SSF Splice Site FinderaNFE European (non-Finnish), FE European (Finnish), A African, L Latino, EA East Asian, SA South Asian, AJ Ashkenazi Jewish, O OtherbBased on in silico prediction tools Align GVGD, MetaLR, SIFT, and Polymorphism Phenotyping version 2 (PolyPhen-2) (HDIV), as well as phyloP basewise conservation scores; rated as probably damaging (D) with at least 3 of 5 tools predicting damage; rated as probably benign (B) with at least 4 of 5 tools predicting tolerance; or rated as contradictory (C) with 2 of 5 tools predicting damage or conflicting results

Missense and splice variants of unknown significance detected by NGS-based sequencing

Abbreviations: HSF Human Splicing Finder, SSF Splice Site Finder

aNFE European (non-Finnish), FE European (Finnish), A African, L Latino, EA East Asian, SA South Asian, AJ Ashkenazi Jewish, O Other

bBased on in silico prediction tools Align GVGD, MetaLR, SIFT, and Polymorphism Phenotyping version 2 (PolyPhen-2) (HDIV), as well as phyloP basewise conservation scores; rated as probably damaging (D) with at least 3 of 5 tools predicting damage; rated as probably benign (B) with at least 4 of 5 tools predicting tolerance; or rated as contradictory (C) with 2 of 5 tools predicting damage or conflicting results

Regarding the detection of unconventional, potentially harmful aberrations in TP53 itself, we did not identify any variants other than commonly known polymorphisms in our cohort. A list of all detected TP53 variants is accessible in Additional file 4.

### LFS/LFL classification details and performance of classification systems

Of the 83 families in our cohort, 48 met Eeles’ LFL criteria [5]; 23 met the original stringent Chompret criteria [6]; 75 were consistent with the temporarily suggested 2008 version of Chompret criteria, which loosened age restrictions and included all BRCA1/2-negative patients with BC before age 36 years, regardless of family history [7]; 43 met the 2009 version of Chompret criteria, which again reversed the latter point [8]; 53 met the current Chompret 2015 version criteria, which by definition include all BRCA1/2-negative patients with BC before age 31 years, regardless of family history [9]; 12 families met Birch’s LFL criteria [4]; and 1 family met classic LFS criteria [1] (multiple entries being possible). Considering mutation carriers’ families only (10 of 83 families), all 10 families were consistent with the 2008 version of Chompret criteria, whereas 6 of 10 could be classified into Eeles’ LFL and Chompret’s 2009 version criteria. When we examined the mutation detection rate for each classification system separately, we found that the majority of the applied criteria performed similarly, with the highest detection rate observed within members of both Eeles’ LFL criteria and Chompret’s original/2008/2009 criteria (13–14%).

For an overview of the classification results and phenotypic characteristics of the cohort, see Table 1. Information about LFS and LFL criteria definitions is given in Additional file 2.Table 1Classification and cohort characteristicsTotal sample, n (%)Mutation carriers, n (%)Mutation carriers per group, n (%)Eeles’ LFL criteria48/83 (58%)6/10 (60%)6/48 (13%)Birch’s LFL criteria12/83 (15%)1/10 (10%)1/12 (8%)Original Chompret criteria23/83 (28%)3/10 (30%)3/23 (13%)Chompret 2008 version criteria75/83 (90%)10/10 (100%)10/75 (13%)Chompret 2009 version criteria43/83 (52%)6/10 (60%)6/43 (14%)Chompret 2015 version criteria53/83 (64%)5/10 (50%)5/53 (9%)Classic LFS criteria1/83 (1%)0/100/1Early-onset BC (i.e., ≤ 34 years)50/83 (60%)7/10 (70%)7/50 (14%)Bilateral BC/two primary BCs9/83 (11%)2/10 (20%)2/9 (22%)Additional neoplasms besides BC4/83 (5%)0/100/4Sarcoma in family or self10/83 (12%)0/100/10Brain tumor in family or self16/83 (20%)2/10 (20%)2/16 (13%)Abbreviations: BC Breast cancer, LFL Li-Fraumeni-like syndrome, LFS Li-Fraumeni syndromeSee Additional file 2 regarding LFS/LFL criteria

Classification and cohort characteristics

Abbreviations: BC Breast cancer, LFL Li-Fraumeni-like syndrome, LFS Li-Fraumeni syndrome

See Additional file 2 regarding LFS/LFL criteria

### Variant detection in patients with LFL personal or family history

Within the cohort of 83 subjects with BC, we detected 13 pathogenic or likely pathogenic heterozygous germline variants (ACMG class 4–5; i.e., nonsense, frameshift, missense, [consensus] splice site variants, or copy number aberrations) in 10 unrelated patients and 9 genes [ATM (n = 3), CDKN2A (n = 1), CHEK2 (n = 1), FANCI (n = 1), PALB2 (n = 2), PMS2 (n = 1), RECQL4 (n = 2), RUNX1 (n = 1), and WRN (n = 1)]. These 13 deleterious variants include 4 gross genomic deletions/insertions and 9 truncating NGS-detected variants. Altogether, 3 patients presented as double-mutation carriers, harboring 2 (likely) PVs each (i.e., PMS2 and FANCI, CDKN2A and RECQL4, as well as ATM and CHEK2). Half of the 10 mutation carriers (5 of 83; 6% of the entire cohort) were found to harbor PVs in widely accepted BC susceptibility genes (i.e., PALB2, ATM, and CHEK2), whereas the remaining half (5 of 83; 6% of the entire cohort) carried PVs in candidate genes for which no firm association with BC incidence has yet been established in a heterozygous germline setting (CDKN2A, RUNX1, FANCI, WRN, and RECQL4). Four variants detected via NGS and 2 of the 4 copy number aberrations detected via array-based CGH (comparative genomic hybridization) have not previously been described in the literature. For an overview of all identified classes 4 and 5 variants, see Table 2. An overview of all index patients carrying (likely) pathogenic germline variants, including their personal and family histories as well as available clinical data and LFS/LFL classification, is given in Table 3, and respective pedigrees can be accessed in Additional file 3.Table 2Detected pathogenic and likely pathogenic single-nucleotide variants, small indels, and copy number alterationsIndex patient identifierGene (transcript GRCh37/hg19)Variant typedbSNPExons/total no. of exonscDNA changePredicted amino acid changeAberration array CGHACMG classPreviously reporteda7PALB2 (NM_024675.3)Frameshiftrs5157261244/13c.509_510delGAp.(Arg170Ilefs*14)5Yes30RUNX1 (NM_001754.4)Splice donorrs3751313723/8c.97+1G>A4No32ATM (NM_000051.3)Deletion62–63/63arr [GRCh37]11q22.3 (108233779_108240057)×15Yes40ATM (NM_000051.3)Nonsense61/63c.8793T>Ap.(Cys2931*)5YesCHEK2 (NM_007194.3)Deletion9–10/15arr [GRCh37]22q12.1 (29092709_29097723)×15Yes58CDKN2A (NM_058195.3)Nonsense2/3c.292C>Tp.(Arg98*)4NoRECQL4 (NM_004260.3)Splice donor7/21c.1390+1G>C4No59WRN (NM_000553.4)Deletion15–16/35arr [GRCh37]8p12 (30948138_30949422)×14No60ATM (NM_000051.3)Nonsensers58777985240/63c.5932G>Tp.(Glu1978*)5Yes65FANCI (NM_001113378.1)Nonsensers12191816437/38c.3853C>Tp.(Arg1285*)5YesPMS2 (NM_000535.5)Deletion3–8/15arr [GRCh37]7p22.1 (6035238_6042593)×15No76PALB2 (NM_024675.3)Frameshiftrs1801771433/13c.172_175delTTGTp.(Gln60Argfs*7)5Yes79RECQL4 (NM_004260.3)Missense and splice regionrs18673907216/22c.2755G>Ap.(Ala919Thr)4NoAbbreviations: ACMG American College of Medical Genetics and Genomics, cDNA Complementary DNA, CGH Comparative genomic hybridization, dbSNP Single Nucleotide Polymorphism databaseaBased on reports in the literature, ClinVar, Decipher, and gene-specific databasesTable 3Clinical information and summary of personal and family histories of mutation carriersIndex patient identifierVariantPersonal history (age at diagnosis in years)Immunohistochemistry (if available)Family history of cancer (age at diagnosis in years)Conformance to LFS/LFL criteriaa7PALB2:p.(Arg170Ilefs*14)BC (40)1. TNBCMat. – M: BC (40)EelesPat. – F: leukemia (< 50), GM: esophagus (60)Chompret 2008Chompret 200930RUNX1:c.97+1G>ABC (33)N/AMat. – M: BC (56), GM: BC (60)Chompret 200832ATM exon 62–63 delBC bilateral (30 + 40)1. Triple-positive2. Triple-positiveMat. – M: OvCa (51), half-S: BC (41) + lung (46), U: leukemia (45), GM: cancer, GF: cancerEelesorig. ChompretChompret 2008Chompret 2009Chompret 201540ATM:p.(Cys2931*)BC (39)N/AMat. – M: OvCa (40), A: leukemia (20)EelesCHEK2 exon 9–10 delChompret 2008Chompret 200958CDKN2A:p.(Arg98*)BC (32)HER2+Mat. – GM: ureter (64)Chompret 2008RECQL4:c.1390+1G>CPat. – GF: hypopharynx (62)59WRN exon 15–16 delBC (32)N/AMat. – half-S: melanoma (46), GF: lung (64), GM: CRC (50), GGM: CRC (59), U: kidney (60), this U’s sons: melanoma (45), basalioma (36)Eelesorig. ChompretChompret 2008Pat. – U: brain (25), 10 Us/As: all died of cancer at a young ageChompret 2009Chompret 2015Birch60ATM:p.(Glu1978*)BC (40)HR+Mat. – M: BC (51), GM: BC (73)EelesPat. – F: glioblastoma (42)Chompret 2008Chompret 2009Chompret 201565FANCI:p.(Arg1285*)BC (30)HER2+Mat. – U: CRC (37), GF: CRC (70), GGM: cancerChompret 2008PMS2 exon 3–8 delPat. – GF: esophagus (74)Chompret 201576PALB2:p.(Gln60Argfs*7)BC bilateral (33 + 39)1. HR+ lobular2. HR+ (HER2+ in metastases)B: lung (43)EelesMat. – M: pancreas (58), A: melanoma (67)orig. ChompretChompret 2008Chompret 200979RECQL4:p.(Ala919Thr)BC (27)HR+Mat. – M: NHL (42), GF: bladder (54)Chompret 2008Pat. – GM: BC (53), GU: lymphoma (52), GA: BC (57), this GA’s daughter: BC (47)Chompret 2015Abbreviations: A Aunt, B Brother, BC Breast cancer, CRC Colorectal cancer, del Deletion, F Father, GA Grand aunt, GF Grandfather, GGM Great grandmother, GM Grandmother, GU Grand uncle, HER2+ HER2 (ERBB2) overexpression/amplification, HR+ Hormone receptor-positive, M Mother, Mat. Maternal, N/A Not accessible, Ne Nephew, NHL Non-Hodgkin lymphoma, Ni Niece, OvCa Ovarian cancer, Pat. Paternal, S Sister, TNBC Triple-negative breast cancer, U UncleaInformation about Li-Fraumeni and Li-Fraumeni-like criteria definition are given in Additional file 2

Detected pathogenic and likely pathogenic single-nucleotide variants, small indels, and copy number alterations

Abbreviations: ACMG American College of Medical Genetics and Genomics, cDNA Complementary DNA, CGH Comparative genomic hybridization, dbSNP Single Nucleotide Polymorphism database

aBased on reports in the literature, ClinVar, Decipher, and gene-specific databases

Clinical information and summary of personal and family histories of mutation carriers

Abbreviations: A Aunt, B Brother, BC Breast cancer, CRC Colorectal cancer, del Deletion, F Father, GA Grand aunt, GF Grandfather, GGM Great grandmother, GM Grandmother, GU Grand uncle, HER2+ HER2 (ERBB2) overexpression/amplification, HR+ Hormone receptor-positive, M Mother, Mat. Maternal, N/A Not accessible, Ne Nephew, NHL Non-Hodgkin lymphoma, Ni Niece, OvCa Ovarian cancer, Pat. Paternal, S Sister, TNBC Triple-negative breast cancer, U Uncle

aInformation about Li-Fraumeni and Li-Fraumeni-like criteria definition are given in Additional file 2

Besides classes 4–5 variants, we detected 49 variants of unknown significance (VUS, ACMG class 3) within our collective for the parameters detailed above. Via array-based CGH, we identified a duplication of exons 1–21 of the BLM gene, which currently ranks as a class 3 variant owing to a lack of more precise breakpoint information. Via NGS-based sequencing, we detected 46 missense or disruptive inframe deletion and 2 splice region VUS with an MAF ≤ 0.1% occurring in 35 of the 94 investigated genes. Among this list of VUS, we observed an unexpected frequency of very rare FANCA missense variants that were linked to either very early-onset BC or exceptionally Li-Fraumeni-suggestive family/personal history. For a summary of all VUS with an MAF ≤ 0.1% detected via NGS-based sequencing, see Table 4. Neither of the identified VUS was included in the statistical analysis of this work.Table 4Missense and splice variants of unknown significance detected by NGS-based sequencingGene (transcript GRCh37/hg19)Index patient identifierVariant typeExons/total no. of exonscDNA changePredicted amino acid changedbSNPTotal allele frequency, gnomAD; population with highest AFaIn silico predictionbSplicing predictions at nearest natural junctionALK (NM_004304.4)52Splice region and synonymous17/29c.2817C>Tp.(Gly939Gly)rs1120224660.0001840; SA: 0.0006173Native acceptor site:MaxEnt: + 23.7%NNSPLICE: + 9.5%HSF: + 0.9%GeneSplicer: + 32.0%APC (NM_001127510.2)83Missense17/17c.4918C>Tp.(Arg1640Trp)rs3734406140.00007223;CO: 0.0001548ATM (NM_000051.3)20Missense10/63c.1271C>Ap.(Pro424His)rs1474726130.00002188;DA: 0.00004189ATM (NM_000051.3)20Missense50/63c.7357C>Tp.(Arg2453Cys)rs7554185710.00001219;CEA: 0.00005807BRCA2 (NM_000059.3)36Missense10/27c.831T>Gp.(Asn277Lys)rs288977050.00006632;BNFE: 0.0001440BRCA2 (NM_000059.3)30Missense11/27c.6101G>Ap.(Arg2034His)rs803588490.000004069;BNFE: 0.000008973BRCA2 (NM_000059.3)80Missense14/27c.7021C>Tp.(Arg2341Cys)rs412935050.00002439;DEA: 0.00005798BRIP1 (NM_032043.2)68Missense15/20c.2220G>Tp.(Gln740His)rs455896370.0005198;BL: 0.001395CHEK2 (NM_007194.379Missense4/15c.539G>Ap.(Arg180His)rs1378530090.00006494;CL: 0.0002615CHEK2 (NM_007194.3)83Missense6/15c.688G>Cp.(Ala230Pro)rs7486362160.000004063;DNFE: 0.000008956DDB2 (NM_000107.2)23Missense7/10c.947C>Tp.(Ser316Phe)rs3757889660.00001218;CSA: 0.00003249DICER1 (NM_030621.4)8Missense25/29c.4228A>Tp.(Asn1410Tyr)not foundDDIS3L2 (NM_152383.4)65Missense20/21c.2450C>Tp.(Thr817Met)rs3768168580.00006697;CSA: 0.0003505EGFR (NM_005228.3)54Missense17/28c.2039G>Ap.(Arg680Gln)rs3733362510.00009084;DFE: 0.0003912ERCC2 (NM_000400.3)24Disruptive inframe deletion20/23c.1857_1859delCATp.(Ile619del)0.000008127;EA: 0.00005800ERCC2 (NM_000400.3)55Missense22/23c.2083C>Tp.(Arg695Cys)rs2013929110.0001372;DA: 0.0003331ERCC4 (NM_005236.2)23Missense11/11c.2395C>Tp.(Arg799Trp)rs1219130490.0004476;DNFE: 0.0008138ERCC5 (NM_000123.3)23Missense1/15c.56C>Tp.(Pro19Leu)rs342913970.0005702;CNFE: 0.001092ERCC5 (NM_000123.3)76Missense13/15c.2818G>Ap.(Val940Met)rs1463448550.0009378;CAJ: 0.005122EXT2 (NM_000401.3)13Missense6/14c.1064G>Ap.(Arg355His)rs1497275180.0006422;DAJ: 0.003448EXT2 (NM_000401.3)54Missense7/14c.1186G>Ap.(Val396Met)rs1389430910.0004148;DNFE: 0.0007261FANCA (NM_000135.2)41, 73Missense1/43c.64T>Gp.(Trp22Gly)not foundDFANCA (NM_000135.2)8Missense16/43c.1489C>Gp.(Pro497Ala)not foundDFANCA (NM_000135.2)27Missense22/43c.2000C > Gp.(Pro667Arg)rs7552935960.00002230;DSA: 0.00004382FANCA (NM_000135.2)49Missense35/43c.3430C>Tp.(Arg1144Trp)rs1436718720.0005269;DNFE: 0.0009237FANCA (NM_000135.2)54Missense37/43c.3688C>Gp.(Leu1230Val)rs5764014590.00002030;DA: 0.00006535FANCD2 (NM_033084.3)57Missense21/43c.1933G>Tp.(Asp645Tyr)rs1464962530.0001371;CNFE: 0.0002288FANCI (NM_001113378.1)35Missense17/38c.1589T>Cp.(Leu530Pro)rs7663461560.000008122;DSA: 0.00006497GPC3 (NM_001164617.1)19Missense8/9c.1630C>Tp.(Arg544Cys)rs7595437030.00009270;DA: 0.0001707HNF1A (NM_000545.6)48Missense5/10c.1061C>Tp.(Thr354Met)rs7570688090.00006495;DL: 0.0001743KIT (NM_000222.2)39Missense3/21c.391G>Ap.(Asp131Asn)0.00001807;CA: 0.00004164MET (NM_001127500.1)13Missense2/21c.1076G>Ap.(Arg359Gln)rs2012740410.0002347;DNFE: 0.0004298MET (NM_001127500.1)75Missense14/21c.3023G>Ap.(Ser1008Asn)not foundDMLH1 (NM_000249.3)51Missense13/19c.1457C>Tp.(Ser486Phe)rs5328731410.000004061;CEA: 0.00005798NF1 (NM_001042492)28Missense5/58c.575G>Ap.(Arg192Gln)rs5877816700.00005294;CEA: 0.0005803NSD1 (NM_022455.4)10Missense23/23c.7352G>Ap.(Arg2451Lys)rs2001156650.0001119;DNFE: 0.0002212PALB2 (NM_024675.3)56Missense8/13c.2792T>Gp.(Leu931Arg)rs7738313040.00001221;DNFE: 0.00002694PMS1 (NM_000534.4)67Missense2/13c.118G>Cp.(Val40Leu)not foundBPMS1 (NM_000534.4)12Missense3/13c.287C>Ap.(Ala96Asp)rs1394146060.000004063;DNFE: 0.000008962PTCH1 (NM_001083602.1)4Missense23/24c.3749A>Gp.(Tyr1250Cys)rs1470671710.0005463;DAJ: 0.001098RAD51C (NM_058216.2)42Splice region8/8c.1026+5_1026+7 delGTArs7473119930.00001219;Native donor site:SSF: − 100.0%MaxEnt: − 100.0%NNSPLICE: − 99.6%HSF: − 14.8%NFE: 0.00002687RAD51D (NM_002878.3)52Missense5/10c.355T>Cp.(Cys119Arg)rs2013138610.00005413;BL: 0.00008716RHBDF2 (NM_024599.5)3, 75Missense8/19c.940G>Ap.(Ala314Thr)rs1404333740.0007812;CFE: 0.001173SDHB (NM_003000.2)81Missense2/8c.178A>Gp.(Thr60Ala)rs345992810.00006095;DNFE: 0.0001165SLX4 (NM_032444.2)14Missense14/15c.4831G>Ap.(Glu1611Lys)rs7661104790.00002847;DEA: 0.00005798TSC1 (NM_000368.4)51Missense12/23c.1178C>Tp.(Thr393Ile)rs2014522380.00002170;DA: 0.00004165TSC2 (NM_000548.4)32Missense36/42c.4582G>Cp.(Glu1528Gln)not foundDWRN (NM_000553.4)55Missense19/35c.2131C>Tp.(Arg711Trp)rs345607880.0002057;DNFE: 0.0003948Abbreviations: HSF Human Splicing Finder, SSF Splice Site FinderaNFE European (non-Finnish), FE European (Finnish), A African, L Latino, EA East Asian, SA South Asian, AJ Ashkenazi Jewish, O OtherbBased on in silico prediction tools Align GVGD, MetaLR, SIFT, and Polymorphism Phenotyping version 2 (PolyPhen-2) (HDIV), as well as phyloP basewise conservation scores; rated as probably damaging (D) with at least 3 of 5 tools predicting damage; rated as probably benign (B) with at least 4 of 5 tools predicting tolerance; or rated as contradictory (C) with 2 of 5 tools predicting damage or conflicting results

Missense and splice variants of unknown significance detected by NGS-based sequencing

Abbreviations: HSF Human Splicing Finder, SSF Splice Site Finder

aNFE European (non-Finnish), FE European (Finnish), A African, L Latino, EA East Asian, SA South Asian, AJ Ashkenazi Jewish, O Other

bBased on in silico prediction tools Align GVGD, MetaLR, SIFT, and Polymorphism Phenotyping version 2 (PolyPhen-2) (HDIV), as well as phyloP basewise conservation scores; rated as probably damaging (D) with at least 3 of 5 tools predicting damage; rated as probably benign (B) with at least 4 of 5 tools predicting tolerance; or rated as contradictory (C) with 2 of 5 tools predicting damage or conflicting results

Regarding the detection of unconventional, potentially harmful aberrations in TP53 itself, we did not identify any variants other than commonly known polymorphisms in our cohort. A list of all detected TP53 variants is accessible in Additional file 4.

### Discussion

In this study, we approached the question whether breast cancer patients with an LFS/LFL-suggestive phenotype may harbor germline aberrations in known or proposed cancer susceptibility genes beyond TP53 and BRCA1/2. Investigating the spectrum of germline mutations in a cohort of 83 BC patients suggestive of LFS/LFL, we identified 10 patients carrying (likely) PVs in the analyzed genes. 3 of these 10 patients carried 2 (likely) PVs each.

As anticipated beforehand, we detected several PVs in established BC-associated genes. Two women (subjects 7, 76) carry previously described, protein-truncating, heterozygous PALB2 frameshift variants. The variant PALB2:p.(Gln60Argfs*7), occurring in patient 76, has previously been detected in several unrelated patients with BC [35, 36], whereas the variant PALB2:p.(Arg170Ilefs*14) in patient 7 is a known Polish founder mutation [37]. Notably, patient 76 developed hormone receptor-positive BC disease at the age of 33 years and contralateral BC at age 39 years, whereas patient 7 presented with triple-negative BC, which is in line with studies reporting breast tumors of PALB2 mutation carriers to be triple-negative in about 34% of these patients [38].

Furthermore, 3 women of our cohort carry heterozygous PVs in ATM, one of whom additionally harbors a PV in CHEK2. A deletion of the last 2 exons of ATM (62–63) was detected in patient 32. Deletions of exon 63 or exons 62–63 of the ATM gene have previously been reported in patients with ataxia telangiectasia and are considered to be functionally relevant [39, 40]. In two further patients (60, 40), we detected previously described ATM nonsense variants; the variant ATM:p.(Glu1978*) has been reported in BC cases before [41], whereas the variant ATM:p.(Cys2931*) has been described as a class 5 variant in a patient with ataxia telangiectasia [42]. The latter index person (40) additionally harbors a CHEK2 exon 9–10 deletion, which is a known pathogenic Slavic founder mutation [43]. Remarkably, 2 of the 3 ATM mutation carriers of our cohort have first-degree relatives presenting with ovarian cancer, one of whom harbors the additional CHEK2 exon 9–10 deletion. While monoallelic ATM germline PVs have been described in patients with ovarian cancer, heterozygous ATM variants have been associated predominantly with elevated BC incidence rather than other cancer types [44, 45].

One woman (patient 58), who is additionally affected by the class 4 splice donor site variant RECQL4:c.1390+1G>C, was identified to carry a CDKN2A/p14ARF nonsense variant in exon 2 of 3 total exons [CDKN2A:p.(Arg98*)]. The CDKN2A locus encodes two distinct proteins, p14ARF and p16INK4, that are defined by translating the common second exon in alternate reading frames. The variant we detected is not predicted to affect expression or function of p16INK4, a tumor suppressor implicated in the CDK4/CDK6-RB1 pathway; instead, p14ARF expression is expected to be abolished via nonsense-mediated RNA decay (NMD). The protein p14ARF acts upstream of TP53 by binding directly to MDM2, an E3 ubiquitin ligase controlling the activity and stability of TP53. Because p14ARF promotes MDM2’s degradation, TP53 is stabilized and accumulates, and a TP53 response manifests in elevated levels of p21CIP1, inducing cell cycle arrest [46]. PVs in human p14ARF exon 2 have been reported to disrupt its nucleolar localization and impair its ability to block nuclear export of MDM2 and TP53 [47]. The exon 2 variant in our cohort was identified in an index patient whose grandfather was diagnosed with hypopharyngeal carcinoma, which is in line with CDKN2A PVs predisposing to tobacco-related cancers such as orolaryngeal cancer, next to its predominant role in hereditary melanoma, pancreatic cancer, and further tumor entities [MIM:600160]. Of note and with particular relevance to LFS and LFL, germline CDKN2A PVs have recently been demonstrated to account for a subset (8 of 190) of hereditary sarcoma cases negative for germline TP53 PVs [17]. Conversely, however, sarcomas are rare in CDKN2A mutation carriers, and the affected family of our cohort also does not present with sarcoma cases. In a somatic context, CDKN2A PVs have previously been associated with the mutational landscape of BC [48].

In index patient 30, we detected a splice donor site variant in RUNX1 (RUNX1:c.97+1G>A) predicted deleterious by 4 of 5 splice prediction programs owing to complete loss of the native splice site. As of yet, this variant has not been described in the literature and was classified by us as a class 4 variant on the basis of in silico prediction. RUNX1 is a gene well known for its crucial role in the hematopoietic system and for its association with sporadic and familial leukemia [49]. The above-mentioned variant was identified in a family exclusively struck by BC. Of note, no hematologic abnormalities were reported in this family at the time of consultation. Even though the variant was detected in 43% of the sequencing reads (VAF 43%, depth 400 ×), we cannot exclude the possibility that this variant might be due to clonal hematopoiesis (i.e., undiagnosed hematologic disorder). In the case of a confirmed germline event, functional splicing assays would be needed to confirm the disruptive nature of the variant. In regard to BC pathology, RUNX1 has been suggested to be largely understudied [50], and genome-wide sequencing of cohorts of patients with BC have subsequently exposed RUNX1 as one of the most frequently mutated and/or deleted genes in BC in a somatic setting [51, 52]. Its role in BC progression has been related to estrogen signaling but remains elusive, likely including oncogenic rather than tumor-suppressive functions in mammary epithelial cells [53–55]. A conclusive role for RUNX1 germline variations in BC pathogenesis cannot be assessed currently and needs further clarification.

The nonsense FANCI variant FANCI:p.(Arg1285*) in exon 37 of 38 total exons, located within the last 72 bp of the second to last exon and therefore imprecise in regard to NMD, was detected in patient 65, who was additionally found to carry a deletion of exons 3–8 of the PMS2 gene, which was verified by MLPA analysis. Because the maternal lineage includes two colorectal carcinomas in two of the index person’s second-degree relatives, diagnosed at 37 and 70 years of age, the PMS2 variant is primarily suggestive of being responsible for the familial cancer phenotypes. Nonetheless, the FANCI variant concerns a highly conserved arginine, has previously been described in patients with Fanconi anemia (FA), and has been found to impair DNA binding and ubiquitination of the ID2 complex, a dimeric complex formed by FANCI and FANCD2, which is relevant for DNA crosslink repair [56, 57]. Because the paternal lineage also shows incidence of malignant disease, it is conceivable that the FANCI and PMS2 variants may jointly combine in the index person’s genome, leading to a more severe phenotype, such as earlier age of onset of disease.

Furthermore, 6 patients of our cohort were identified to carry 5 very rare FANCA VUS: FANCA:p.(Trp22Gly) twice, FANCA:p.(Pro497Ala), FANCA:p.(Pro667Arg), FANCA:p.(Arg1144Trp), and FANCA:p.(Leu1230Val). All but one of these rare FANCA VUS were absent in a large cohort of 11,000 exome-sequenced individuals of HMGU (Helmholtz Zentrum München – Deutsches Forschungszentrum für Gesundheit und Umwelt) and are either not or very rarely found in datasets from the Genome Aggregation Database [58] (see Table 4 for details). Interestingly, clinical presentation of these heterozygous FANCA missense VUS carriers comprises some of the most striking LFL features, involving either very early-onset BC or exceptionally Li-Fraumeni-suggestive personal and family traits, such as a woman with triple primaries (malignant hemangiopericytoma at age 12 years, BC at age 28 years, and contralateral BC at age 47 years) or a woman whose sister and half-brother were diagnosed with two liposarcomas and a melanoma, respectively. The median age of onset of BC disease for these 6 FANCA VUS carriers was 29.5 years (27, 27, 28, 31, 35, and 44 years), whereas the median age of onset for the entire cohort was 33 years. In a biallelic context, the Fanconi family genes are associated with FA, a condition characterized by congenital abnormalities, bone marrow failure, and cancer predisposition already during childhood. FANCA gene PVs are by far the most common in FA, responsible for at least 60% of all cases of FA [59]. Heterozygous parents and siblings of patients with FA have not been found to exhibit an elevated incidence of malignant disease [60]; however, monoallelic FANCA variants have been investigated only sparsely in a disease context, and with inconsistent results [61–63]. Considering the rarity of each FANCA VUS detected in the present study, and given the possibility of differing FANCA missense variants conferring vastly diverse effects on protein function, the disproportionate occurrence of these rare variants appears to be worthy of further investigation.

Two class 4–5 aberrations (patients 59 and 79) and one class 3 copy number change (patient 63) were identified in WRN, RECQL4, and BLM — 3 of 5 genes belonging to the human family of RecQ helicases, comprising enzymes that drive the unwinding of DNA in an ATP- and Mg2+-dependent manner and which are essential for genome maintenance and stability. In an autosomal recessive setting, each of these genes confers very rare and complex syndromes, namely Werner syndrome (WRN), Bloom syndrome (BLM), and Rothmund-Thomson syndrome or RAPADILINO syndrome (RECQL4). In regard to LFS, in which sarcoma incidence is being considered somewhat of a hallmark within the tumor spectrum, a particularly striking fact about the above-mentioned syndromes is that the spectrum of malignancies conferred by them is dominated by sarcomas as well [64, 65]. Moreover, for all three proteins, a firm association with TP53 has been described [65, 66]. Whereas biallelic impairment is considered mandatory for the full spectrum of the syndromes to develop, WRN and BLM have additionally been suggested as BC susceptibility genes in a monoallelic setting [67–69], and dominant-negative effects or gain-of-function processes have previously been proposed for specific missense mutant WRN or BLM proteins [70].

To our knowledge, this is the first study focusing on a German population-based cohort of familial BC patients suggestive of LFL but negative for causative TP53 as well as BRCA1/2 germline PVs to be systematically panel-tested for aberrations in further cancer susceptibility genes. One hypothesis for the emergence of an LFL phenotype without detectable TP53 PV is the simultaneous and additive occurrence of dual or multiple clearly pathogenic aberrations in more than one cancer susceptibility gene. Our study design was able to confirm double heterozygosity in 3 cases. In regard to patients with BC, double heterozygosity for germline PVs in BC predisposition genes has been detected and discussed before [67]. This type of oligogenic or polygenic model might explain the fact that often the index patients of our cohort happen to be burdened by cancer incidences from both family lineages. The model would also explain why the family mode may not follow a strict autosomal dominant inheritance pattern, which is usually considered a hallmark of classic LFS.

With the current state of knowledge and excluding all VUS, the presently described aberrations, occurring in 10 (12%) of our cohort’s individuals, may in part explain or contribute to some of the corresponding phenotypes. However, the gravity and the spectrum of malignant disease in many of our cohort’s families seem to suggest that additional PVs in modifying genes or as yet unknown cancer predisposition genes may play a role in some of them. Importantly, these results indicate that even after testing a fairly extensive number of cancer susceptibility genes with a panel design covering both sequence alterations and copy number changes, a large number of LFL BC cases remain unexplained. Few studies have approached the quest for further susceptibility genes in individual TP53-negative LFS/LFL cases, either by specifically testing single genes associated with the spectrum of tumors appearing in LFS or by performing whole-exome sequencing. Associations have been suggested for LFS-associated brain tumors with nonsense PVs in CASP9 (caspase-9) [71], for a POT1 (protection of telomeres 1) missense variant with cardiac and breast angiosarcomas [72], and, as mentioned above, for CDKN2A PVs with hereditary sarcoma cases [17].

Apart from the idea of aberrations in additional susceptibility genes being responsible for TP53-negative LFL cases, novel mechanisms for TP53 impairment are also emerging. These include structural variants that cannot easily be detected by conventional approaches such as intron 1 rearrangements [73], variants in the far-off 3′ UTR affecting microRNA binding [74] (a region not covered by the Illumina gene panel used in our study), and novel splicing PVs [75]. Whereas the severe phenotype known for LFS is expected to result predominantly from missense or isoform-specific variants conferring a dominant-negative effect of the mutated over the wild-type proteins, TP53 haploinsufficiency (e.g., loss of function or gene dosage effects due to splice variants or 3′ UTR variants and deletions, respectively) is emerging as a likely mechanism for a more subtle phenotype, one that may in fact be described as LFL, as has previously been suggested [75]. Testing patients without variants in coding TP53 regions, such as the ones in our cohort, for the above-mentioned variants would constitute a crucial future undertaking.

The strengths of this study include a well-characterized cohort of LFL BC patients and the dual-method study design of NGS and array-based CGH to cover a larger number of aberrations. Limitations include the following:

1. Segregation analyses would be mandatory in order to characterize the impact of the detected variants in affected and unaffected family members and to predict the penetrance associated with these variants; unfortunately, DNA was not available for any of the family members of affected germline mutation carriers.

2. Breakpoint analysis and, if applicable, translocation details are urgently needed for the BLM duplication.

3. Functional assays would be helpful in determining the consequences of the detected VUS and novel aberrations.

### Variants in established BC susceptibility genes (PALB2, ATM, CHEK2)

As anticipated beforehand, we detected several PVs in established BC-associated genes. Two women (subjects 7, 76) carry previously described, protein-truncating, heterozygous PALB2 frameshift variants. The variant PALB2:p.(Gln60Argfs*7), occurring in patient 76, has previously been detected in several unrelated patients with BC [35, 36], whereas the variant PALB2:p.(Arg170Ilefs*14) in patient 7 is a known Polish founder mutation [37]. Notably, patient 76 developed hormone receptor-positive BC disease at the age of 33 years and contralateral BC at age 39 years, whereas patient 7 presented with triple-negative BC, which is in line with studies reporting breast tumors of PALB2 mutation carriers to be triple-negative in about 34% of these patients [38].

Furthermore, 3 women of our cohort carry heterozygous PVs in ATM, one of whom additionally harbors a PV in CHEK2. A deletion of the last 2 exons of ATM (62–63) was detected in patient 32. Deletions of exon 63 or exons 62–63 of the ATM gene have previously been reported in patients with ataxia telangiectasia and are considered to be functionally relevant [39, 40]. In two further patients (60, 40), we detected previously described ATM nonsense variants; the variant ATM:p.(Glu1978*) has been reported in BC cases before [41], whereas the variant ATM:p.(Cys2931*) has been described as a class 5 variant in a patient with ataxia telangiectasia [42]. The latter index person (40) additionally harbors a CHEK2 exon 9–10 deletion, which is a known pathogenic Slavic founder mutation [43]. Remarkably, 2 of the 3 ATM mutation carriers of our cohort have first-degree relatives presenting with ovarian cancer, one of whom harbors the additional CHEK2 exon 9–10 deletion. While monoallelic ATM germline PVs have been described in patients with ovarian cancer, heterozygous ATM variants have been associated predominantly with elevated BC incidence rather than other cancer types [44, 45].

### Variants in candidate BC susceptibility genes

One woman (patient 58), who is additionally affected by the class 4 splice donor site variant RECQL4:c.1390+1G>C, was identified to carry a CDKN2A/p14ARF nonsense variant in exon 2 of 3 total exons [CDKN2A:p.(Arg98*)]. The CDKN2A locus encodes two distinct proteins, p14ARF and p16INK4, that are defined by translating the common second exon in alternate reading frames. The variant we detected is not predicted to affect expression or function of p16INK4, a tumor suppressor implicated in the CDK4/CDK6-RB1 pathway; instead, p14ARF expression is expected to be abolished via nonsense-mediated RNA decay (NMD). The protein p14ARF acts upstream of TP53 by binding directly to MDM2, an E3 ubiquitin ligase controlling the activity and stability of TP53. Because p14ARF promotes MDM2’s degradation, TP53 is stabilized and accumulates, and a TP53 response manifests in elevated levels of p21CIP1, inducing cell cycle arrest [46]. PVs in human p14ARF exon 2 have been reported to disrupt its nucleolar localization and impair its ability to block nuclear export of MDM2 and TP53 [47]. The exon 2 variant in our cohort was identified in an index patient whose grandfather was diagnosed with hypopharyngeal carcinoma, which is in line with CDKN2A PVs predisposing to tobacco-related cancers such as orolaryngeal cancer, next to its predominant role in hereditary melanoma, pancreatic cancer, and further tumor entities [MIM:600160]. Of note and with particular relevance to LFS and LFL, germline CDKN2A PVs have recently been demonstrated to account for a subset (8 of 190) of hereditary sarcoma cases negative for germline TP53 PVs [17]. Conversely, however, sarcomas are rare in CDKN2A mutation carriers, and the affected family of our cohort also does not present with sarcoma cases. In a somatic context, CDKN2A PVs have previously been associated with the mutational landscape of BC [48].

In index patient 30, we detected a splice donor site variant in RUNX1 (RUNX1:c.97+1G>A) predicted deleterious by 4 of 5 splice prediction programs owing to complete loss of the native splice site. As of yet, this variant has not been described in the literature and was classified by us as a class 4 variant on the basis of in silico prediction. RUNX1 is a gene well known for its crucial role in the hematopoietic system and for its association with sporadic and familial leukemia [49]. The above-mentioned variant was identified in a family exclusively struck by BC. Of note, no hematologic abnormalities were reported in this family at the time of consultation. Even though the variant was detected in 43% of the sequencing reads (VAF 43%, depth 400 ×), we cannot exclude the possibility that this variant might be due to clonal hematopoiesis (i.e., undiagnosed hematologic disorder). In the case of a confirmed germline event, functional splicing assays would be needed to confirm the disruptive nature of the variant. In regard to BC pathology, RUNX1 has been suggested to be largely understudied [50], and genome-wide sequencing of cohorts of patients with BC have subsequently exposed RUNX1 as one of the most frequently mutated and/or deleted genes in BC in a somatic setting [51, 52]. Its role in BC progression has been related to estrogen signaling but remains elusive, likely including oncogenic rather than tumor-suppressive functions in mammary epithelial cells [53–55]. A conclusive role for RUNX1 germline variations in BC pathogenesis cannot be assessed currently and needs further clarification.

The nonsense FANCI variant FANCI:p.(Arg1285*) in exon 37 of 38 total exons, located within the last 72 bp of the second to last exon and therefore imprecise in regard to NMD, was detected in patient 65, who was additionally found to carry a deletion of exons 3–8 of the PMS2 gene, which was verified by MLPA analysis. Because the maternal lineage includes two colorectal carcinomas in two of the index person’s second-degree relatives, diagnosed at 37 and 70 years of age, the PMS2 variant is primarily suggestive of being responsible for the familial cancer phenotypes. Nonetheless, the FANCI variant concerns a highly conserved arginine, has previously been described in patients with Fanconi anemia (FA), and has been found to impair DNA binding and ubiquitination of the ID2 complex, a dimeric complex formed by FANCI and FANCD2, which is relevant for DNA crosslink repair [56, 57]. Because the paternal lineage also shows incidence of malignant disease, it is conceivable that the FANCI and PMS2 variants may jointly combine in the index person’s genome, leading to a more severe phenotype, such as earlier age of onset of disease.

Furthermore, 6 patients of our cohort were identified to carry 5 very rare FANCA VUS: FANCA:p.(Trp22Gly) twice, FANCA:p.(Pro497Ala), FANCA:p.(Pro667Arg), FANCA:p.(Arg1144Trp), and FANCA:p.(Leu1230Val). All but one of these rare FANCA VUS were absent in a large cohort of 11,000 exome-sequenced individuals of HMGU (Helmholtz Zentrum München – Deutsches Forschungszentrum für Gesundheit und Umwelt) and are either not or very rarely found in datasets from the Genome Aggregation Database [58] (see Table 4 for details). Interestingly, clinical presentation of these heterozygous FANCA missense VUS carriers comprises some of the most striking LFL features, involving either very early-onset BC or exceptionally Li-Fraumeni-suggestive personal and family traits, such as a woman with triple primaries (malignant hemangiopericytoma at age 12 years, BC at age 28 years, and contralateral BC at age 47 years) or a woman whose sister and half-brother were diagnosed with two liposarcomas and a melanoma, respectively. The median age of onset of BC disease for these 6 FANCA VUS carriers was 29.5 years (27, 27, 28, 31, 35, and 44 years), whereas the median age of onset for the entire cohort was 33 years. In a biallelic context, the Fanconi family genes are associated with FA, a condition characterized by congenital abnormalities, bone marrow failure, and cancer predisposition already during childhood. FANCA gene PVs are by far the most common in FA, responsible for at least 60% of all cases of FA [59]. Heterozygous parents and siblings of patients with FA have not been found to exhibit an elevated incidence of malignant disease [60]; however, monoallelic FANCA variants have been investigated only sparsely in a disease context, and with inconsistent results [61–63]. Considering the rarity of each FANCA VUS detected in the present study, and given the possibility of differing FANCA missense variants conferring vastly diverse effects on protein function, the disproportionate occurrence of these rare variants appears to be worthy of further investigation.

Two class 4–5 aberrations (patients 59 and 79) and one class 3 copy number change (patient 63) were identified in WRN, RECQL4, and BLM — 3 of 5 genes belonging to the human family of RecQ helicases, comprising enzymes that drive the unwinding of DNA in an ATP- and Mg2+-dependent manner and which are essential for genome maintenance and stability. In an autosomal recessive setting, each of these genes confers very rare and complex syndromes, namely Werner syndrome (WRN), Bloom syndrome (BLM), and Rothmund-Thomson syndrome or RAPADILINO syndrome (RECQL4). In regard to LFS, in which sarcoma incidence is being considered somewhat of a hallmark within the tumor spectrum, a particularly striking fact about the above-mentioned syndromes is that the spectrum of malignancies conferred by them is dominated by sarcomas as well [64, 65]. Moreover, for all three proteins, a firm association with TP53 has been described [65, 66]. Whereas biallelic impairment is considered mandatory for the full spectrum of the syndromes to develop, WRN and BLM have additionally been suggested as BC susceptibility genes in a monoallelic setting [67–69], and dominant-negative effects or gain-of-function processes have previously been proposed for specific missense mutant WRN or BLM proteins [70].

To our knowledge, this is the first study focusing on a German population-based cohort of familial BC patients suggestive of LFL but negative for causative TP53 as well as BRCA1/2 germline PVs to be systematically panel-tested for aberrations in further cancer susceptibility genes. One hypothesis for the emergence of an LFL phenotype without detectable TP53 PV is the simultaneous and additive occurrence of dual or multiple clearly pathogenic aberrations in more than one cancer susceptibility gene. Our study design was able to confirm double heterozygosity in 3 cases. In regard to patients with BC, double heterozygosity for germline PVs in BC predisposition genes has been detected and discussed before [67]. This type of oligogenic or polygenic model might explain the fact that often the index patients of our cohort happen to be burdened by cancer incidences from both family lineages. The model would also explain why the family mode may not follow a strict autosomal dominant inheritance pattern, which is usually considered a hallmark of classic LFS.

With the current state of knowledge and excluding all VUS, the presently described aberrations, occurring in 10 (12%) of our cohort’s individuals, may in part explain or contribute to some of the corresponding phenotypes. However, the gravity and the spectrum of malignant disease in many of our cohort’s families seem to suggest that additional PVs in modifying genes or as yet unknown cancer predisposition genes may play a role in some of them. Importantly, these results indicate that even after testing a fairly extensive number of cancer susceptibility genes with a panel design covering both sequence alterations and copy number changes, a large number of LFL BC cases remain unexplained. Few studies have approached the quest for further susceptibility genes in individual TP53-negative LFS/LFL cases, either by specifically testing single genes associated with the spectrum of tumors appearing in LFS or by performing whole-exome sequencing. Associations have been suggested for LFS-associated brain tumors with nonsense PVs in CASP9 (caspase-9) [71], for a POT1 (protection of telomeres 1) missense variant with cardiac and breast angiosarcomas [72], and, as mentioned above, for CDKN2A PVs with hereditary sarcoma cases [17].

Apart from the idea of aberrations in additional susceptibility genes being responsible for TP53-negative LFL cases, novel mechanisms for TP53 impairment are also emerging. These include structural variants that cannot easily be detected by conventional approaches such as intron 1 rearrangements [73], variants in the far-off 3′ UTR affecting microRNA binding [74] (a region not covered by the Illumina gene panel used in our study), and novel splicing PVs [75]. Whereas the severe phenotype known for LFS is expected to result predominantly from missense or isoform-specific variants conferring a dominant-negative effect of the mutated over the wild-type proteins, TP53 haploinsufficiency (e.g., loss of function or gene dosage effects due to splice variants or 3′ UTR variants and deletions, respectively) is emerging as a likely mechanism for a more subtle phenotype, one that may in fact be described as LFL, as has previously been suggested [75]. Testing patients without variants in coding TP53 regions, such as the ones in our cohort, for the above-mentioned variants would constitute a crucial future undertaking.

The strengths of this study include a well-characterized cohort of LFL BC patients and the dual-method study design of NGS and array-based CGH to cover a larger number of aberrations. Limitations include the following:

1. Segregation analyses would be mandatory in order to characterize the impact of the detected variants in affected and unaffected family members and to predict the penetrance associated with these variants; unfortunately, DNA was not available for any of the family members of affected germline mutation carriers.

2. Breakpoint analysis and, if applicable, translocation details are urgently needed for the BLM duplication.

3. Functional assays would be helpful in determining the consequences of the detected VUS and novel aberrations.

### Variants in genes associated with BC in a somatic context (CDKN2A, RUNX1)

One woman (patient 58), who is additionally affected by the class 4 splice donor site variant RECQL4:c.1390+1G>C, was identified to carry a CDKN2A/p14ARF nonsense variant in exon 2 of 3 total exons [CDKN2A:p.(Arg98*)]. The CDKN2A locus encodes two distinct proteins, p14ARF and p16INK4, that are defined by translating the common second exon in alternate reading frames. The variant we detected is not predicted to affect expression or function of p16INK4, a tumor suppressor implicated in the CDK4/CDK6-RB1 pathway; instead, p14ARF expression is expected to be abolished via nonsense-mediated RNA decay (NMD). The protein p14ARF acts upstream of TP53 by binding directly to MDM2, an E3 ubiquitin ligase controlling the activity and stability of TP53. Because p14ARF promotes MDM2’s degradation, TP53 is stabilized and accumulates, and a TP53 response manifests in elevated levels of p21CIP1, inducing cell cycle arrest [46]. PVs in human p14ARF exon 2 have been reported to disrupt its nucleolar localization and impair its ability to block nuclear export of MDM2 and TP53 [47]. The exon 2 variant in our cohort was identified in an index patient whose grandfather was diagnosed with hypopharyngeal carcinoma, which is in line with CDKN2A PVs predisposing to tobacco-related cancers such as orolaryngeal cancer, next to its predominant role in hereditary melanoma, pancreatic cancer, and further tumor entities [MIM:600160]. Of note and with particular relevance to LFS and LFL, germline CDKN2A PVs have recently been demonstrated to account for a subset (8 of 190) of hereditary sarcoma cases negative for germline TP53 PVs [17]. Conversely, however, sarcomas are rare in CDKN2A mutation carriers, and the affected family of our cohort also does not present with sarcoma cases. In a somatic context, CDKN2A PVs have previously been associated with the mutational landscape of BC [48].

In index patient 30, we detected a splice donor site variant in RUNX1 (RUNX1:c.97+1G>A) predicted deleterious by 4 of 5 splice prediction programs owing to complete loss of the native splice site. As of yet, this variant has not been described in the literature and was classified by us as a class 4 variant on the basis of in silico prediction. RUNX1 is a gene well known for its crucial role in the hematopoietic system and for its association with sporadic and familial leukemia [49]. The above-mentioned variant was identified in a family exclusively struck by BC. Of note, no hematologic abnormalities were reported in this family at the time of consultation. Even though the variant was detected in 43% of the sequencing reads (VAF 43%, depth 400 ×), we cannot exclude the possibility that this variant might be due to clonal hematopoiesis (i.e., undiagnosed hematologic disorder). In the case of a confirmed germline event, functional splicing assays would be needed to confirm the disruptive nature of the variant. In regard to BC pathology, RUNX1 has been suggested to be largely understudied [50], and genome-wide sequencing of cohorts of patients with BC have subsequently exposed RUNX1 as one of the most frequently mutated and/or deleted genes in BC in a somatic setting [51, 52]. Its role in BC progression has been related to estrogen signaling but remains elusive, likely including oncogenic rather than tumor-suppressive functions in mammary epithelial cells [53–55]. A conclusive role for RUNX1 germline variations in BC pathogenesis cannot be assessed currently and needs further clarification.

### Variants in Fanconi pathway genes (FANCI, FANCA)

The nonsense FANCI variant FANCI:p.(Arg1285*) in exon 37 of 38 total exons, located within the last 72 bp of the second to last exon and therefore imprecise in regard to NMD, was detected in patient 65, who was additionally found to carry a deletion of exons 3–8 of the PMS2 gene, which was verified by MLPA analysis. Because the maternal lineage includes two colorectal carcinomas in two of the index person’s second-degree relatives, diagnosed at 37 and 70 years of age, the PMS2 variant is primarily suggestive of being responsible for the familial cancer phenotypes. Nonetheless, the FANCI variant concerns a highly conserved arginine, has previously been described in patients with Fanconi anemia (FA), and has been found to impair DNA binding and ubiquitination of the ID2 complex, a dimeric complex formed by FANCI and FANCD2, which is relevant for DNA crosslink repair [56, 57]. Because the paternal lineage also shows incidence of malignant disease, it is conceivable that the FANCI and PMS2 variants may jointly combine in the index person’s genome, leading to a more severe phenotype, such as earlier age of onset of disease.

Furthermore, 6 patients of our cohort were identified to carry 5 very rare FANCA VUS: FANCA:p.(Trp22Gly) twice, FANCA:p.(Pro497Ala), FANCA:p.(Pro667Arg), FANCA:p.(Arg1144Trp), and FANCA:p.(Leu1230Val). All but one of these rare FANCA VUS were absent in a large cohort of 11,000 exome-sequenced individuals of HMGU (Helmholtz Zentrum München – Deutsches Forschungszentrum für Gesundheit und Umwelt) and are either not or very rarely found in datasets from the Genome Aggregation Database [58] (see Table 4 for details). Interestingly, clinical presentation of these heterozygous FANCA missense VUS carriers comprises some of the most striking LFL features, involving either very early-onset BC or exceptionally Li-Fraumeni-suggestive personal and family traits, such as a woman with triple primaries (malignant hemangiopericytoma at age 12 years, BC at age 28 years, and contralateral BC at age 47 years) or a woman whose sister and half-brother were diagnosed with two liposarcomas and a melanoma, respectively. The median age of onset of BC disease for these 6 FANCA VUS carriers was 29.5 years (27, 27, 28, 31, 35, and 44 years), whereas the median age of onset for the entire cohort was 33 years. In a biallelic context, the Fanconi family genes are associated with FA, a condition characterized by congenital abnormalities, bone marrow failure, and cancer predisposition already during childhood. FANCA gene PVs are by far the most common in FA, responsible for at least 60% of all cases of FA [59]. Heterozygous parents and siblings of patients with FA have not been found to exhibit an elevated incidence of malignant disease [60]; however, monoallelic FANCA variants have been investigated only sparsely in a disease context, and with inconsistent results [61–63]. Considering the rarity of each FANCA VUS detected in the present study, and given the possibility of differing FANCA missense variants conferring vastly diverse effects on protein function, the disproportionate occurrence of these rare variants appears to be worthy of further investigation.

### Variants in RECQ helicases (WRN, RECQL4)

Two class 4–5 aberrations (patients 59 and 79) and one class 3 copy number change (patient 63) were identified in WRN, RECQL4, and BLM — 3 of 5 genes belonging to the human family of RecQ helicases, comprising enzymes that drive the unwinding of DNA in an ATP- and Mg2+-dependent manner and which are essential for genome maintenance and stability. In an autosomal recessive setting, each of these genes confers very rare and complex syndromes, namely Werner syndrome (WRN), Bloom syndrome (BLM), and Rothmund-Thomson syndrome or RAPADILINO syndrome (RECQL4). In regard to LFS, in which sarcoma incidence is being considered somewhat of a hallmark within the tumor spectrum, a particularly striking fact about the above-mentioned syndromes is that the spectrum of malignancies conferred by them is dominated by sarcomas as well [64, 65]. Moreover, for all three proteins, a firm association with TP53 has been described [65, 66]. Whereas biallelic impairment is considered mandatory for the full spectrum of the syndromes to develop, WRN and BLM have additionally been suggested as BC susceptibility genes in a monoallelic setting [67–69], and dominant-negative effects or gain-of-function processes have previously been proposed for specific missense mutant WRN or BLM proteins [70].

To our knowledge, this is the first study focusing on a German population-based cohort of familial BC patients suggestive of LFL but negative for causative TP53 as well as BRCA1/2 germline PVs to be systematically panel-tested for aberrations in further cancer susceptibility genes. One hypothesis for the emergence of an LFL phenotype without detectable TP53 PV is the simultaneous and additive occurrence of dual or multiple clearly pathogenic aberrations in more than one cancer susceptibility gene. Our study design was able to confirm double heterozygosity in 3 cases. In regard to patients with BC, double heterozygosity for germline PVs in BC predisposition genes has been detected and discussed before [67]. This type of oligogenic or polygenic model might explain the fact that often the index patients of our cohort happen to be burdened by cancer incidences from both family lineages. The model would also explain why the family mode may not follow a strict autosomal dominant inheritance pattern, which is usually considered a hallmark of classic LFS.

With the current state of knowledge and excluding all VUS, the presently described aberrations, occurring in 10 (12%) of our cohort’s individuals, may in part explain or contribute to some of the corresponding phenotypes. However, the gravity and the spectrum of malignant disease in many of our cohort’s families seem to suggest that additional PVs in modifying genes or as yet unknown cancer predisposition genes may play a role in some of them. Importantly, these results indicate that even after testing a fairly extensive number of cancer susceptibility genes with a panel design covering both sequence alterations and copy number changes, a large number of LFL BC cases remain unexplained. Few studies have approached the quest for further susceptibility genes in individual TP53-negative LFS/LFL cases, either by specifically testing single genes associated with the spectrum of tumors appearing in LFS or by performing whole-exome sequencing. Associations have been suggested for LFS-associated brain tumors with nonsense PVs in CASP9 (caspase-9) [71], for a POT1 (protection of telomeres 1) missense variant with cardiac and breast angiosarcomas [72], and, as mentioned above, for CDKN2A PVs with hereditary sarcoma cases [17].

Apart from the idea of aberrations in additional susceptibility genes being responsible for TP53-negative LFL cases, novel mechanisms for TP53 impairment are also emerging. These include structural variants that cannot easily be detected by conventional approaches such as intron 1 rearrangements [73], variants in the far-off 3′ UTR affecting microRNA binding [74] (a region not covered by the Illumina gene panel used in our study), and novel splicing PVs [75]. Whereas the severe phenotype known for LFS is expected to result predominantly from missense or isoform-specific variants conferring a dominant-negative effect of the mutated over the wild-type proteins, TP53 haploinsufficiency (e.g., loss of function or gene dosage effects due to splice variants or 3′ UTR variants and deletions, respectively) is emerging as a likely mechanism for a more subtle phenotype, one that may in fact be described as LFL, as has previously been suggested [75]. Testing patients without variants in coding TP53 regions, such as the ones in our cohort, for the above-mentioned variants would constitute a crucial future undertaking.

The strengths of this study include a well-characterized cohort of LFL BC patients and the dual-method study design of NGS and array-based CGH to cover a larger number of aberrations. Limitations include the following:

1. Segregation analyses would be mandatory in order to characterize the impact of the detected variants in affected and unaffected family members and to predict the penetrance associated with these variants; unfortunately, DNA was not available for any of the family members of affected germline mutation carriers.

2. Breakpoint analysis and, if applicable, translocation details are urgently needed for the BLM duplication.

3. Functional assays would be helpful in determining the consequences of the detected VUS and novel aberrations.

### Conclusions

Our study helps define the mutational spectrum in breast cancer patients suggestive of LFS, contributes to a potential relevance of CDKN2A/p14ARF in LFS/LFL settings, and points out the need for intensified research on monoallelic variants in Fanconi pathway and RECQ family genes. Notably, our study further reveals that there remains a large portion of unexplained LFS/LFL cases and emphasizes the necessity of advanced research on novel susceptibility genes as well as noncoding TP53 variants in patients with negative test results for TP53 variants in coding regions.

### Additional files

Additional file 1:TruSight Cancer Target Genes and SNPs (a list of all 94 investigated genes) (Illumina). (DOCX 15 kb)
Additional file 2:Definitions of Li-Fraumeni criteria (official definitions of available clinical criteria, including classic LFS criteria, LFL criteria of Eeles and Birch, and 3 versions of Chompret criteria). (DOCX 18 kb)
Additional file 3:Pedigrees of investigated families with (likely) pathogenic variants (pedigrees of all families of carriers of pathogenic or likely pathogenic variants in our cohort, including age of onset of malignant disease). (DOCX 692 kb)
Additional file 4:Total list of TP53 (NM_000546.5) variants detected via NGS-based sequencing (entire list of all detected TP53 variants in our collective). (DOCX 35 kb)

TruSight Cancer Target Genes and SNPs (a list of all 94 investigated genes) (Illumina). (DOCX 15 kb)

Definitions of Li-Fraumeni criteria (official definitions of available clinical criteria, including classic LFS criteria, LFL criteria of Eeles and Birch, and 3 versions of Chompret criteria). (DOCX 18 kb)

Pedigrees of investigated families with (likely) pathogenic variants (pedigrees of all families of carriers of pathogenic or likely pathogenic variants in our cohort, including age of onset of malignant disease). (DOCX 692 kb)

Total list of TP53 (NM_000546.5) variants detected via NGS-based sequencing (entire list of all detected TP53 variants in our collective). (DOCX 35 kb)

Additional file 1:TruSight Cancer Target Genes and SNPs (a list of all 94 investigated genes) (Illumina). (DOCX 15 kb)
Additional file 2:Definitions of Li-Fraumeni criteria (official definitions of available clinical criteria, including classic LFS criteria, LFL criteria of Eeles and Birch, and 3 versions of Chompret criteria). (DOCX 18 kb)
Additional file 3:Pedigrees of investigated families with (likely) pathogenic variants (pedigrees of all families of carriers of pathogenic or likely pathogenic variants in our cohort, including age of onset of malignant disease). (DOCX 692 kb)
Additional file 4:Total list of TP53 (NM_000546.5) variants detected via NGS-based sequencing (entire list of all detected TP53 variants in our collective). (DOCX 35 kb)

TruSight Cancer Target Genes and SNPs (a list of all 94 investigated genes) (Illumina). (DOCX 15 kb)

Definitions of Li-Fraumeni criteria (official definitions of available clinical criteria, including classic LFS criteria, LFL criteria of Eeles and Birch, and 3 versions of Chompret criteria). (DOCX 18 kb)

Pedigrees of investigated families with (likely) pathogenic variants (pedigrees of all families of carriers of pathogenic or likely pathogenic variants in our cohort, including age of onset of malignant disease). (DOCX 692 kb)

Total list of TP53 (NM_000546.5) variants detected via NGS-based sequencing (entire list of all detected TP53 variants in our collective). (DOCX 35 kb)



# SUPPLEMENTAL FILE 1: 13058_2018_Article_1011.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)