# MAIN TEXT

## Novel Germline Mutations of BRCA1 and BRCA2 in Korean Familial Breast Cancer Patients

### Abstract

Breast cancer is the second most common cancer in Korean women. Germline mutations in the BRCA1 and BRCA2 genes cause hereditary breast cancer and are detected in 15–20% of hereditary breast cancer. We investigated the BRCA1 and BRCA2 mutations in 114 familial breast cancer patients using next-generation sequencing. We confirmed 20 different mutations of BRCA1 and BRCA2 in 25 subjects (21.9%). Two such mutations in eight patients were novel (not reported in any variant database or previous study). Six mutations have been reported as disease-causing mutations in public databases. Seven mutations were found only in a single nucleotide polymorphism database and one mutation has been reported in Korea. The BRCA1/2 mutation frequency was similar to that of other studies on familial breast cancer patients in the Korean population. Further studies should examine more cases and mutations of whole exons.

### INTRODUCTION

Breast cancer has been the second most common cancer in Korean women since 2004. In 2013, 19,316 patients were newly diagnosed with breast cancer and the crude incidence was 76.2 cases per 100,000 women.1 The median age at diagnosis is 50 years, and more than half of all cases are diagnosed in postmenopausal women. However, the proportion of early breast cancer has consistently increased.2

BRCA1 and BRCA2 are tumor suppressor genes that are inherited in an autosomal dominant pattern. Germline mutations of these two highly penetrant genes cause hereditary breast cancer34 and are detected in 15–20% of hereditary breast cancer cases.5 The reported lifetime risk of breast cancer with mutations in the BRCA1/2 genes is between 60% and 85%.67 The management of breast cancer patients differs depending on whether a mutation is present or not.89

The prevalence of BRCA mutations varies with ethnicity and country.1011 Many studies have examined the prevalence of BRCA1 and BRCA2 mutations in Korean breast cancer patients,121314 including two recent studies1214 that reported that the prevalence of BRCA mutations among familial breast cancer patients was 19.6% and 20.60%, respectively. However, studies of more patients are still needed to evaluate the prevalence of BRCA mutations in Korean breast cancer patients.

In this study, we investigated the prevalence of BRCA1 and BRCA2 mutations and explored the novel variants among the familial breast cancer patients in Korea.

### MATERIALS AND METHODS

All patients were enrolled at Chonnam National University Hwasun Hospital (Hwasun, Korea) between 2005 and 2012. From 3,540 female breast cancer patients, we selected 161 patients (4.6%) with a family history of breast cancer. After excluding 47 patients with low quantity and quality of genomic DNA for library preparation with the polymerase chain reaction (PCR)-based Access Array, 114 breast cancer patients were included in the analyses, including 16 patients younger than 40 years old, 6 smokers, and 11 obese patients with a body mass index ≥30 kg/m2. Information on the family history of breast cancer and other risk factors was collected via review of medical records. Study participants agreed with the informed consent form and this study was approved by the Institutional Review Board of Chonnam National University Hwasun Hospital (HCRI-09-043-3).

Genomic DNA was extracted from peripheral blood using a QIAamp DNA Mini Kit (QIAGEN, Chatsworth, CA, USA), according to the manufacturer's protocol.

The procedure entailed the design of target-specific primers, samples preparation, and then running a PCR-based Access Array (Fluidigm, San Francisco, CA, USA) and qualifying and quantifying harvested PCR products for sequencing. Briefly, 184 amplicons were designed to cover all exons of BRCA1 and BRCA2. Samples were amplified using the 48.48 PCR-based Access Array (Fluidigm) and purified pooled samples were sequenced on a MiSeq sequencer (Illumina, Hayward, CA, USA), according to the manufacturers' protocols.

The data was analyzed on a PCR Amplicon workflow with MiSeq Reporter software ver. 2.4.60 (Illumina, Hayward, CA, USA), which used the Burrows–Wheeler Aligner. Reads were aligned against the GRCh37/hg19 reference genomes of targeted regions in the sample sheet. Integrative Genomics Viewer was used to visualize the quality and variance of the Bam and VCF files.15

Variants with a minor allele frequency >1% in Ensemble and synonymous and intronic variants located outside the exon/intron boundaries were excluded from further analysis. To classify pathogenic mutations, all nonsynonymous variants were compared to a BRCA locus-specific database (http://research.nhgri.nih.gov/bic/, http://databases.lovd.nl/shared/genes/BRCA1, and http://databases.lovd.nl/shared/genes/BRCA2) and The Human Gene Mutation Database (HGMD, http://www.hgmd.cf.ac.uk/ac/index.php) and amino acid variation was predicted using the predictive algorithms SIFT, PolyPhen-2, and Mutation T@ster web servers.161718 All variants in this study were classified as pathogenic mutations or variants of uncertain clinical significance (VUCS), as previously described.19 All rare variants were confirmed by conventional Sanger sequencing and high-resolution melting analysis.

### 1. Subjects

All patients were enrolled at Chonnam National University Hwasun Hospital (Hwasun, Korea) between 2005 and 2012. From 3,540 female breast cancer patients, we selected 161 patients (4.6%) with a family history of breast cancer. After excluding 47 patients with low quantity and quality of genomic DNA for library preparation with the polymerase chain reaction (PCR)-based Access Array, 114 breast cancer patients were included in the analyses, including 16 patients younger than 40 years old, 6 smokers, and 11 obese patients with a body mass index ≥30 kg/m2. Information on the family history of breast cancer and other risk factors was collected via review of medical records. Study participants agreed with the informed consent form and this study was approved by the Institutional Review Board of Chonnam National University Hwasun Hospital (HCRI-09-043-3).

### 2. Mutation detection

Genomic DNA was extracted from peripheral blood using a QIAamp DNA Mini Kit (QIAGEN, Chatsworth, CA, USA), according to the manufacturer's protocol.

The procedure entailed the design of target-specific primers, samples preparation, and then running a PCR-based Access Array (Fluidigm, San Francisco, CA, USA) and qualifying and quantifying harvested PCR products for sequencing. Briefly, 184 amplicons were designed to cover all exons of BRCA1 and BRCA2. Samples were amplified using the 48.48 PCR-based Access Array (Fluidigm) and purified pooled samples were sequenced on a MiSeq sequencer (Illumina, Hayward, CA, USA), according to the manufacturers' protocols.

### 3. Data analysis

The data was analyzed on a PCR Amplicon workflow with MiSeq Reporter software ver. 2.4.60 (Illumina, Hayward, CA, USA), which used the Burrows–Wheeler Aligner. Reads were aligned against the GRCh37/hg19 reference genomes of targeted regions in the sample sheet. Integrative Genomics Viewer was used to visualize the quality and variance of the Bam and VCF files.15

Variants with a minor allele frequency >1% in Ensemble and synonymous and intronic variants located outside the exon/intron boundaries were excluded from further analysis. To classify pathogenic mutations, all nonsynonymous variants were compared to a BRCA locus-specific database (http://research.nhgri.nih.gov/bic/, http://databases.lovd.nl/shared/genes/BRCA1, and http://databases.lovd.nl/shared/genes/BRCA2) and The Human Gene Mutation Database (HGMD, http://www.hgmd.cf.ac.uk/ac/index.php) and amino acid variation was predicted using the predictive algorithms SIFT, PolyPhen-2, and Mutation T@ster web servers.161718 All variants in this study were classified as pathogenic mutations or variants of uncertain clinical significance (VUCS), as previously described.19 All rare variants were confirmed by conventional Sanger sequencing and high-resolution melting analysis.

### RESULTS

Table 1 shows the clinical characteristics of the study participants. The mean age at diagnosis of breast cancer was 51.7 (range 29.7–77.9) years. The proportions of smokers, alcohol drinkers, estrogen-receptor-positive, progesterone-receptor-positive, and menopausal subjects among the all breast cancer subjects and subjects with BRCA mutations are also presented. No subjects had a history of ovarian cancer (Table 1).

Table 2 shows the distribution of mutations in BRCA1 and BRCA2. We confirmed 20 different BRCA1/2 mutations in 25 (21.9%) of the 114 patients. Eight of the mutations were in the BRCA1 gene and twelve were in the BRCA2 gene. Six mutations (in nine patients) have been reported as disease-causing mutations in public databases (BIC and HGMD). Two BRCA1 mutations and five BRCA2 mutations were found only in a single nucleotide polymorphism database and one mutation had been reported in a Korean. Two of the BRCA1 and four of the BRCA2 mutations in eight patients were novel mutations (not reported in any variant database or previous study).

Interestingly, three novel mutations and four reported mutations involved two unique frameshift (Tyr655ValfsTer18 in BRCA1 and c.1286_1287insA in BRCA2), two splicing (c.5074_5074+4delGGTAT in BRCA1 and Arg2336Alafs [7006delC] in BRCA2) and three nonsense (Tyr130Ter in BRCA1, Arg2494Ter, and Glu1113Ter in BRCA2) mutations. All others are missense mutations. Further segregation and functional studies are needed to identify the pathogenicity of these variants (Table 2).

### DISCUSSION

We confirmed 20 different BRCA1/2 mutations in 25 (21.9%) of 114 breast cancer patients with a family history of breast cancer using PCR-based 48.48 Access Array microfluidic technology (Fluidigm). Six have been reported as disease-causing mutations in public databases (BIC and HGMD). Two BRCA1 and four BRCA2 mutations in eight patients were novel mutations (not reported in any variant database or previous study).

Two mutations (Tyr130Ter in BRCA1 and Arg2494Ter in BRCA2) that we identified are frequently reported in Asians. Tyr130Ter (stop gained) is the most common BRCA1 gene mutation and is frequently observed in Koreans, Japanese, and Chinese.2021 Arg2494Ter is the most common BRCA2 mutation found in Koreans including in the United States (Los Angeles and Stanford).20 Interestingly, seven mutations in this study were in the same position as mutations found in previous studies, but were different types. Tyr655ValfsTer18 in BRCA1 is a duplication frameshift found in Chinese patients;22 we also found a deletion frameshift (Lys654SerfsTer47) in Chinese and Korean patientss.2324 Various intervening sequences were found, including c.5074 +1G>A or T and +3A>G and c.5075 −1G>T or −2A>T including c.5074_5074+4delGGTAT in BRCA1.23252627 Val409 in BRCA1 replaced Ter in Japan and Leu in this study.25 In BRCA2 in this study, Ser1632 Asn, Tyr1710Asp, Arg2336Alafs (7006delC), and Gln3026Glu were found as insertion, deletion, missense or deletion, and nonsense mutations in other studies.2325 These mutations indicate that some sites in the gene are frequently mutated.

In our study, the BRCA1/2 mutation frequency (21.9%) was similar to other reports (19.6% and 20.6%) of familial breast cancer patients in the Korean population.1214 These mutation frequencies were much higher in familial breast cancer patients than in patients with no family history of breast or ovarian cancer (8.6%)20 or with sporadic breast cancer (3.1% and 10%)1328 in Koreans. The prevalence of BRCA1/2 mutation among breast cancer patients varies in Western (1.8–3.6%)11 and Asian (0.8–4.4%) countries.2425 The rate of deleterious mutations in BRCA1 and BRCA2 has been reported to be 20–40% in familial breast cancer patients.2930 Among the 20 mutations found in our study, two were frameshift, two were splicing and three were nonsense mutations. These three types of mutations result in a completely different or often nonfunctional protein products. Therefore, these were considered possible pathogenic mutations.

The differences in reported results may be due to the different populations studied, techniques used (e.g., direct sequencing, next-generation sequencing [NGS], or targeting only several mutations and not whole exons), and / or patient selection criteria. As large insertions/deletions could not be detected by the Access Array-based NGS used in this study, the mutation rate in our study population might be higher, although the rate of large insertions/deletions is low. Therefore, further studies should examine more cases and mutations of entire exons. Further segregation and functional studies are needed to clarify the pathogenicity of these variants.



# SUPPLEMENTAL FILE 1: cmj-55-99.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)