# MAIN TEXT

## BRCA1 and BRCA2 Testing through Next Generation Sequencing in a Small Cohort of Italian Breast/Ovarian Cancer Patients: Novel Pathogenic and Unknown Clinical Significance Variants

### Abstract

The aim of this report is to describe results of BRCA1 and BRCA2 Next Generation Sequencing Analysis (NGS) analysis in 132 selected Italian patients with breast/ovarian cancer. A NGS pipeline with a reliable Copy Number Variation (CNV) prediction algorithm was applied. In addition, VarSome and Priors V2.0 Software were employed for in silico analysis of novel missense variants. A total of 37 BRCA1 and BRCA2 pathogenic variants were found in 34 unrelated subjects with a frequency of positive patients of 25.7% (34/132). Twenty-four deleterious variants were detected in BRCA1 (representing the 64.9% of all identified pathogenic defects) and thirteen (35.1% of all identified pathogenic variants) in BRCA2 gene. The percentage of patients carrying a variant of unknown significance (VUS) was 7.5% (10/132). In addition, seven novel variants (five in BRCA2 and two in BRCA1 gene), never previously reported, were identified. Our approach represents a robust and easy-to-use method for full BRCA1/2 screening. However, a consistent number of our high-risk families still remained without a satisfying answer. Necessarily, further collective efforts must be directed to a definitive classification of VUSs. The future auspice is that the use of multi-gene panel and more advanced screenings, such as whole exome sequencing and/or RNA seq, in routine diagnostics increases the detection rate.

### 1. Introduction

Germline pathogenic variants (PVs) in the BRCA1 and BRCA2 genes confer a significantly increased lifetime risk for the development of both breast and ovarian cancer. For BRCA1, the assessed average risk of breast and ovarian cancers ranges from 57 to 65% and 20 to 50%, respectively; for BRCA2, the risk ranges from 35 to 57% and 5 to 23%, respectively [1].

The detection of BRCA1/2 pathogenic variants has important clinical implications for the risk assessment and medical patients’ management. Interventions, such as bilateral mastectomy, salpingo-oophorectomy or annual breast magnetic resonance imaging (MRI) screening, are offered to women carrying BRCA1/2 PVs in order to reduce cancer risk and to consent early detection. The result of molecular analysis can also influence cancer therapy, principally around the use of platinum agents or poly(ADP-ribose) polymerase (PARP) inhibitors [2]. For these reasons, in the last years, the number BRCA1/2 genetic tests has increased dramatically. In addition, the development of high-throughput sequencing tools has led many diagnostics centers to use Next Generation Sequencing (NGS)-based platforms for clinical molecular analysis [3,4]. As a consequence, using of these new generation technologies has determined a significant increase in the detection of novel variants. Unfortunately, many of these variants do not provide the evidence of a pathogenic effect (in particular the missense variants or those involved in splicing process) and are classified as variants of uncertain significance (VUSs). The misinterpretation of a VUS has the potential to lead to mismanagement of both the patients and their relatives. For this reason, many efforts should be spent to classify correctly a VUS as deleterious variant rather than as a benign polymorphism [5,6,7]. Currently, to aid genetic counseling of subjects carrying a BRCA1/2 VUSs, both genetic and functional molecular methods (in-silico and through experimental procedures) are used for the classification of new variants. Genetic analysis is based on the integration of collected family data (co-segregation with disease, co-occurrence with known pathogenic mutations and family history of cancer) into computational models to calculate the likelihood that a VUS is disease-causing [8]. In addition, in silico analysis, predicting the impact of the mutant amino acid on protein folding, functional assays, testing the effect of a VUS on known functions of encoded protein, and gene expression studies are carefully evaluated [9,10,11].

The present study used next-generation sequencing (NGS) technology to determine the prevalence of germline BRCA1/2 variants in a small cohort of breast/ovarian Italian cancer patients. The obtained results are extensively discussed remarking the novel findings.

### 2. Results

We screened BRCA1 and BRCA2 genes in 132 Italian patients with early onset or family history of breast and/or ovarian cancers. In particular, 11 patients were women, without any family history of cancer, diagnosed with early onset hormone dependent unilateral breast cancer (median age at diagnosis was 33 years, range 27 to 39). Among the 121 probands with a positive family history of breast and/or ovarian cancers: 86 patients (82 women and four men) were diagnosed with unilateral breast cancer, 14 with bilateral breast cancer (13 women and one man), nine with breast and ovarian cancer and 12 were affected from ovarian cancer (Figure 1). All breast cancers were hormone-receptor positive except 19 triple negative (with negative estrogen receptors, negative progesterone receptors and negative Her2/neu status). All ovarian cancers were serous carcinomas, 17 high-grade and four low-grade. The median age at diagnosis of breast and ovarian cancer was 43 (range 29 to 65) and 52 (range 41 to 66) years, respectively. To better show our findings, Figure 2 reports the distribution and the frequency of all detected variants (with assessed or unknown clinical significant) in BRCA1 and BRCA2 genes.

A total of 37 BRCA1 and BRCA2 PVs were found in 34 unrelated subjects with a frequency of positive patients of 25.7% (34/132) (Table 1): 24 deleterious variants were present in BRCA1 (representing the 64.9% of all identified pathogenic defects) and 13 (35.1% of all identified PVs) in the BRCA2 gene. The percentage of patients carrying a variant of unknown significance (VUS) was 7.5% (10/132). Overall, we identified seven novel variants never previously reported (five in BRCA2 and two in BRCA1 gene) (Table 1).

Among the eleven patients with early onset breast cancer, we identified three women (SA22-18, MV55-17 and LA89-17 in Table 1) carrying known PVs (the c.3607C>T and the c.1016delA mutations in BRCA1 and the c.1670T>G in BRCA2) and two (AM11-17 and TC22-17 in Table 1) with VUS (the c.10040T>C in BRCA2 and the c.3344_3346delAAG in BRCA1).

Fourteen PVs and six VUS were identified among the 67 patients with unilateral hormone-dependent breast cancer and positive family history. In particular, eight PVs (all previously reported including two large rearrangements), two VUS not previously described (the c.5308G>T and the c.1268C>T) and one known VUS, the c.548G>T, were detected in BRCA1 gene (Table 1). In BRCA2 gene we identified six deleterious variants (one of these was a novel small insertion, the c.6590_6591insA) and three novel VUS (the c.7225C>T, the c.5971G>A and the c.9219C>T) (Table 1). Two patients (PV86-17 and DPF54-18 in Table 1) carried one BRCA2 allele with two PVs in cis.

Seven PVs and one VUS (all known variants) were detected in eight patients affected from triple negative breast cancer (Table 1).

Among the patients with bilateral breast cancer, we identified a woman (patient DP97-18 in Table 1) carrying one PV in BRCA1(the c.5095C>T) and a small deleterious deletion in BRCA2 gene (the c.1238delT). In addition, four PVs (one of these corresponding to a novel small deletion, the c.4899_4902delCTTT, causing a premature stop codon at amino acid 1634) and one VUS were identified in BRCA2 gene while only a deleterious variant was detected in BRCA1 (Table 1).

Finally, four patients affected from ovarian cancer carried BRCA1 PVs, while a woman was positive for a deleterious variant in BRCA2 gene (Table 1). The BRCA1 c.4964_4982del19, a founder mutation from the southern region of Calabria [12], was detected in a patient (VF31-17 in Table 1) suffering from breast and ovarian cancer and belonging to a family of south Italy origins.

Two novel frameshift variants were identified in BRCA2 gene. The small frameshift deletion c.4899_4902delCTTT, leading to premature protein termination at codon 1634 (p.Phe1634Ter) within exon 11, was detected in a patient with bilateral breast cancer (patient CG98-18 in Table 1; Figure 3a). The family history was strongly suggestive: the dead father was diagnosed with lung cancer at the age of 66 years, while the paternal aunt died of breast cancer developed at the age of 52 years. Two paternal cousins were suffering from early onset breast cancer and the youngest resulted positive to BRCA2 genetic screening.

The pathogenicity of the c.4899_4902delCTTT variant is due to the disruption of crucial functional domains (in particular the DNA Binding Domain (DBD), that includes the amino acids 2481-3186) producing a premature truncated non-functional protein. For this reason, according to ENIGMA guidelines [13], it can be classified as Pathogenic (Class 5).

Similarly, the c.6590_6591insA variant (never reported in queried databases or in literature) showed a clear deleterious effect and it can also be considered as Class 5 variant. This alteration was identified in an early onset breast cancer patient (patient FG65-19 in Table 1; Figure 3b) with three affected sisters, two of these resulting as p.Glu2198Ter carriers, while the third was not tested.

We identified five novel single nucleotide variants, three in BRCA2 gene and two in BRCA1. The BRCA2 c.7225C>T (p.Pro2409Ser) variant was detected in a young woman (patient OA78-18 in Table 1) with breast cancer and in two affected relatives (Figure 4a,b). VarSome software classified this substitution as variant of “Uncertain Significance”. However, some evidences supported its potential pathogenicity: in the hot-spot region of the BRCA2 DBD (amino acids 2481-3186), a proved clinical importance was assigned to the most reported missense variants. In addition, the allele c.7225T frequency resulted unknown (no data in GnomAD exomes and GnomAD genomes) and the verdict of computational analysis was “Pathogenic” (pathogenic predictions by DANN, GERP, dbNSFP. FATHMM, LRT, MetaLR, MetaSVM, MutationAssessor and MutationTaster while only PROVEAN in silico analysis resulted as benign).

Otherwise, the BRCA2 c.5971G>A (p.Ala1991Thr) variant, detected in a man with breast cancer at the age of 50 years (patient PM55-16 in Table 1; Figure 5a), was considered by VarSome software as “Likely Benign”. This rare variant (no frequency data on allele c.5971A were available) is located in BRC-7 (1971-2005) motif, where no amino acidic alterations with clinical importance were reported. Noteworthy, the computational verdict was “Benign” because seven benign predictions were provided (coming from dbNSFP, FATHMM, LRT, MetaLR, MetaSVM, MutationAssessor, MutationTaster and PROVEAN) vs one prediction of pathogenicity (from DANN). Finally, the p.Ala1991 position was not reported as conserved (GERP++ rejected substitutions = 0.346 is less than 4).

The last novel variant detected in BRCA2 gene was the c.9219C>T (p.Asp3073Asp) (patient MR52-18 in Table 1; Figure 5b) mapping in DBD of BRCA2 protein. This synonymous variant was not predicted as altering the splicing process (in transcript ENST00000544455.1 and NM_000059.3) by Priors analysis (score: 0.02). However, no population frequency data were available for this variant that was considered as “VUS” by VarSome software.

The first novel missense variant found in BRCA1 gene was the c.5308G>T (p.Gly1770Trp) (patient RR56-18 in Table 1; Figure 6b) located in the hot spot region of the protein (BRTC Domains 1650-1863). Based on the obtained results, VarSome indicated the c.5308G>T as “Likely Pathogenic”. In fact, no frequency data were available on allele c.5308T and the computational analysis verdict was “Pathogenic” because seven pathogenic results (by DANN, GERP, dbNSFP.FATHMM, MetaLR, MetaSVM, MutationAssessor and MutationTaster) vs two benign predictions (from LRT and PROVEAN) were obtained. Noteworthy, a further pathogenic evidence was revealed: at the same aminoacidic position, the c.5309G>T (p.Gly1770Val; rs863224765) variant has been previously reported and classified as Likely Pathogenic in both Clin Var and LOVD database.

Genetic testing was offered to proband’s father, diagnosed with breast cancer at the age of 55, and this screening detected the novel variant in heterozygosis. The same result was obtained analyzing the sequencing data from a paternal cousin with the breast cancer (Figure 6a).

In addition, the BRCA1 c.1268C>T (p.Ser423Phe) substitution was detected in a woman with breast cancer (patient GH09-19 in Table 1; Figure 5c) and was considered as “VUS” by VarSome software. There were not available frequency data on allele c.1268T and 6 pathogenic predictions (by DANN, dbNSFP.FATHMM, MetaLR, MetaSVM, MutationAssessor and PROVEAN) vs two benign predictions (from LRT and MutationTaster) were reported. Unfortunately, it was not possible to perform genetic testing on the other affected family members.

Finally, Figure 7 describes the pedigree of DP97-18 patient (Table 1) carrying one pathogenic variant in BRCA1 (the c.5095C>T) and a small deleterious deletion in BRCA2 gene (the c.1238delT). Genetic testing was also performed in some consenting relatives (III.1, III.5, III.6 and IV.2) and the segregation analysis revealed that the BRCA2 mutation belonged to paternal familial branch, while the BRCA1 PV was inherited from the mother. In fact, the subject III.6, a maternal proband’s cousin diagnosed with a metastatic ovarian cancer at the age of 53 and for whom the analysis was performed on formalin-fixed-paraffin-embedded (FFPE) tissue, carried the BRCA1 c.5095C>T variant in heterozygous status while the III.1 subject, a paternal cousin with a diagnosis of breast cancer at the age of 47, resulted positive for the BRCA2 small deleterious variant. In this regard, Table 2 reports data regarding all Italian cases related to patients carrying double BRCA1 and BRCA2 mutations.

### 2.1. Molecular Analysis of BRCA1/2 Gene

We screened BRCA1 and BRCA2 genes in 132 Italian patients with early onset or family history of breast and/or ovarian cancers. In particular, 11 patients were women, without any family history of cancer, diagnosed with early onset hormone dependent unilateral breast cancer (median age at diagnosis was 33 years, range 27 to 39). Among the 121 probands with a positive family history of breast and/or ovarian cancers: 86 patients (82 women and four men) were diagnosed with unilateral breast cancer, 14 with bilateral breast cancer (13 women and one man), nine with breast and ovarian cancer and 12 were affected from ovarian cancer (Figure 1). All breast cancers were hormone-receptor positive except 19 triple negative (with negative estrogen receptors, negative progesterone receptors and negative Her2/neu status). All ovarian cancers were serous carcinomas, 17 high-grade and four low-grade. The median age at diagnosis of breast and ovarian cancer was 43 (range 29 to 65) and 52 (range 41 to 66) years, respectively. To better show our findings, Figure 2 reports the distribution and the frequency of all detected variants (with assessed or unknown clinical significant) in BRCA1 and BRCA2 genes.

A total of 37 BRCA1 and BRCA2 PVs were found in 34 unrelated subjects with a frequency of positive patients of 25.7% (34/132) (Table 1): 24 deleterious variants were present in BRCA1 (representing the 64.9% of all identified pathogenic defects) and 13 (35.1% of all identified PVs) in the BRCA2 gene. The percentage of patients carrying a variant of unknown significance (VUS) was 7.5% (10/132). Overall, we identified seven novel variants never previously reported (five in BRCA2 and two in BRCA1 gene) (Table 1).

Among the eleven patients with early onset breast cancer, we identified three women (SA22-18, MV55-17 and LA89-17 in Table 1) carrying known PVs (the c.3607C>T and the c.1016delA mutations in BRCA1 and the c.1670T>G in BRCA2) and two (AM11-17 and TC22-17 in Table 1) with VUS (the c.10040T>C in BRCA2 and the c.3344_3346delAAG in BRCA1).

Fourteen PVs and six VUS were identified among the 67 patients with unilateral hormone-dependent breast cancer and positive family history. In particular, eight PVs (all previously reported including two large rearrangements), two VUS not previously described (the c.5308G>T and the c.1268C>T) and one known VUS, the c.548G>T, were detected in BRCA1 gene (Table 1). In BRCA2 gene we identified six deleterious variants (one of these was a novel small insertion, the c.6590_6591insA) and three novel VUS (the c.7225C>T, the c.5971G>A and the c.9219C>T) (Table 1). Two patients (PV86-17 and DPF54-18 in Table 1) carried one BRCA2 allele with two PVs in cis.

Seven PVs and one VUS (all known variants) were detected in eight patients affected from triple negative breast cancer (Table 1).

Among the patients with bilateral breast cancer, we identified a woman (patient DP97-18 in Table 1) carrying one PV in BRCA1(the c.5095C>T) and a small deleterious deletion in BRCA2 gene (the c.1238delT). In addition, four PVs (one of these corresponding to a novel small deletion, the c.4899_4902delCTTT, causing a premature stop codon at amino acid 1634) and one VUS were identified in BRCA2 gene while only a deleterious variant was detected in BRCA1 (Table 1).

Finally, four patients affected from ovarian cancer carried BRCA1 PVs, while a woman was positive for a deleterious variant in BRCA2 gene (Table 1). The BRCA1 c.4964_4982del19, a founder mutation from the southern region of Calabria [12], was detected in a patient (VF31-17 in Table 1) suffering from breast and ovarian cancer and belonging to a family of south Italy origins.

### 2.2. Novel Variants and In Silico Prediction Analysis

Two novel frameshift variants were identified in BRCA2 gene. The small frameshift deletion c.4899_4902delCTTT, leading to premature protein termination at codon 1634 (p.Phe1634Ter) within exon 11, was detected in a patient with bilateral breast cancer (patient CG98-18 in Table 1; Figure 3a). The family history was strongly suggestive: the dead father was diagnosed with lung cancer at the age of 66 years, while the paternal aunt died of breast cancer developed at the age of 52 years. Two paternal cousins were suffering from early onset breast cancer and the youngest resulted positive to BRCA2 genetic screening.

The pathogenicity of the c.4899_4902delCTTT variant is due to the disruption of crucial functional domains (in particular the DNA Binding Domain (DBD), that includes the amino acids 2481-3186) producing a premature truncated non-functional protein. For this reason, according to ENIGMA guidelines [13], it can be classified as Pathogenic (Class 5).

Similarly, the c.6590_6591insA variant (never reported in queried databases or in literature) showed a clear deleterious effect and it can also be considered as Class 5 variant. This alteration was identified in an early onset breast cancer patient (patient FG65-19 in Table 1; Figure 3b) with three affected sisters, two of these resulting as p.Glu2198Ter carriers, while the third was not tested.

We identified five novel single nucleotide variants, three in BRCA2 gene and two in BRCA1. The BRCA2 c.7225C>T (p.Pro2409Ser) variant was detected in a young woman (patient OA78-18 in Table 1) with breast cancer and in two affected relatives (Figure 4a,b). VarSome software classified this substitution as variant of “Uncertain Significance”. However, some evidences supported its potential pathogenicity: in the hot-spot region of the BRCA2 DBD (amino acids 2481-3186), a proved clinical importance was assigned to the most reported missense variants. In addition, the allele c.7225T frequency resulted unknown (no data in GnomAD exomes and GnomAD genomes) and the verdict of computational analysis was “Pathogenic” (pathogenic predictions by DANN, GERP, dbNSFP. FATHMM, LRT, MetaLR, MetaSVM, MutationAssessor and MutationTaster while only PROVEAN in silico analysis resulted as benign).

Otherwise, the BRCA2 c.5971G>A (p.Ala1991Thr) variant, detected in a man with breast cancer at the age of 50 years (patient PM55-16 in Table 1; Figure 5a), was considered by VarSome software as “Likely Benign”. This rare variant (no frequency data on allele c.5971A were available) is located in BRC-7 (1971-2005) motif, where no amino acidic alterations with clinical importance were reported. Noteworthy, the computational verdict was “Benign” because seven benign predictions were provided (coming from dbNSFP, FATHMM, LRT, MetaLR, MetaSVM, MutationAssessor, MutationTaster and PROVEAN) vs one prediction of pathogenicity (from DANN). Finally, the p.Ala1991 position was not reported as conserved (GERP++ rejected substitutions = 0.346 is less than 4).

The last novel variant detected in BRCA2 gene was the c.9219C>T (p.Asp3073Asp) (patient MR52-18 in Table 1; Figure 5b) mapping in DBD of BRCA2 protein. This synonymous variant was not predicted as altering the splicing process (in transcript ENST00000544455.1 and NM_000059.3) by Priors analysis (score: 0.02). However, no population frequency data were available for this variant that was considered as “VUS” by VarSome software.

The first novel missense variant found in BRCA1 gene was the c.5308G>T (p.Gly1770Trp) (patient RR56-18 in Table 1; Figure 6b) located in the hot spot region of the protein (BRTC Domains 1650-1863). Based on the obtained results, VarSome indicated the c.5308G>T as “Likely Pathogenic”. In fact, no frequency data were available on allele c.5308T and the computational analysis verdict was “Pathogenic” because seven pathogenic results (by DANN, GERP, dbNSFP.FATHMM, MetaLR, MetaSVM, MutationAssessor and MutationTaster) vs two benign predictions (from LRT and PROVEAN) were obtained. Noteworthy, a further pathogenic evidence was revealed: at the same aminoacidic position, the c.5309G>T (p.Gly1770Val; rs863224765) variant has been previously reported and classified as Likely Pathogenic in both Clin Var and LOVD database.

Genetic testing was offered to proband’s father, diagnosed with breast cancer at the age of 55, and this screening detected the novel variant in heterozygosis. The same result was obtained analyzing the sequencing data from a paternal cousin with the breast cancer (Figure 6a).

In addition, the BRCA1 c.1268C>T (p.Ser423Phe) substitution was detected in a woman with breast cancer (patient GH09-19 in Table 1; Figure 5c) and was considered as “VUS” by VarSome software. There were not available frequency data on allele c.1268T and 6 pathogenic predictions (by DANN, dbNSFP.FATHMM, MetaLR, MetaSVM, MutationAssessor and PROVEAN) vs two benign predictions (from LRT and MutationTaster) were reported. Unfortunately, it was not possible to perform genetic testing on the other affected family members.

Finally, Figure 7 describes the pedigree of DP97-18 patient (Table 1) carrying one pathogenic variant in BRCA1 (the c.5095C>T) and a small deleterious deletion in BRCA2 gene (the c.1238delT). Genetic testing was also performed in some consenting relatives (III.1, III.5, III.6 and IV.2) and the segregation analysis revealed that the BRCA2 mutation belonged to paternal familial branch, while the BRCA1 PV was inherited from the mother. In fact, the subject III.6, a maternal proband’s cousin diagnosed with a metastatic ovarian cancer at the age of 53 and for whom the analysis was performed on formalin-fixed-paraffin-embedded (FFPE) tissue, carried the BRCA1 c.5095C>T variant in heterozygous status while the III.1 subject, a paternal cousin with a diagnosis of breast cancer at the age of 47, resulted positive for the BRCA2 small deleterious variant. In this regard, Table 2 reports data regarding all Italian cases related to patients carrying double BRCA1 and BRCA2 mutations.

### 3. Discussion

The frequency of BRCA1/2 PVs in this clinical report was 25.7%, in agreement with previously published studies [4,19,20]. In particular, the c.4117G>T (p.Glu1373Ter) resulted the most frequent PV, representing the 17% (4/24) of all disease-causing alterations identified in BRCA1 gene. However, this result was not unexpected. In fact, since this variant is often detected in our patients, we are investigating the possible role of this mutation as founder in the Lazio region (data not yet published).

To follow, we discuss the relevant points of our study emphasizing the new findings. As reported in the section results, five novel single nucleotide variants, never previously described in literature or within the main reference databases, have been identified. In order to obtain information about their potential pathogenicity, an accurate evaluation of the patient’s family history was performed (offering genetic testing to all consenting family members) and VarSome software was interrogated. The collected data suggest to very carefully consider two of these variants: the c.5308G>T (p.Gly1770Trp) in BRCA1 gene and the c.7225C>T (p.Pro2409Ser) in BRCA2. The BRCA1 c.5308G>T (p.Gly1770Trp) was detected in a young woman with breast cancer and in two affected relatives (Figure 6a,b). Three evidences supporting its pathogenicity (results section) emerged from VarSome analysis in addition to the fact that at the same aminoacidic position the p.(Gly1770Trp) variant was previously described with a deleterious significance. Quiles et al. performed the structural analysis of three variants located in the BRCA1 BRCT domain (p.Tyr1703Ser, p.Trp1718Leu and p.Gly1770Val) revealing significant alterations of BRCT structure. The transcription activation (TA) assay showed that these variants dramatically compromise the transcriptional activity of BRCA1 and were classified as likely pathogenic BRCA1 mutations [21]. At the same time, Bouwman et al. developed a cDNA-based functional assay to classify BRCA1 unknown variants showing the deleterious effects of the p.Gly1770Val mutation [11]. Overall, these events support the suspect that the p.(Gly1770Trp) may be the disease causing in our family.

The c.7225C>T (p.Pro2409Ser) variant was detected in a young proband diagnosed with breast cancer at the age of 36 and showing a strong family history of cancer. We were able to extend the genetic testing to two affected relatives: the mother with a diagnosis of ovarian cancer and a maternal cousin with early onset breast cancer. Both these women carried the c.7225C>T (p.Pro2409Ser) variant in heterozygous status (Figure 4a,b). In addition, our proband was enrolled in a different study where a multi-gene panel testing for hereditary breast/ovarian cancer predisposition (including the following genes: ATM, BARD1,CDH1,CHEK2, PALB2, PTEN, RAD50, TP53, PIK3CA, RAD51D, RAD51C, PMS2, MLH1, MSH2, MSH6, XRCC2, NBN, NF1 and STK-11) was used to assess the mutational status of selected patients. Also in this case, the BRCA2 c.7225C>T (p.Pro2409Ser) was the only relevant identified alteration. Functional studies are needed in order to investigate the exact role of this novel variant and waiting for further data, we recommend to carefully consider it during the clinical and genetic counselling of carried patients.

Regarding the others novel identified variants, we believe that: a benign significant could be attributed to the BRCA2 c.5971G>A (p.Ala1991Thr) and the BRCA2 c.9219C>T (p.Asp3073Asp) variants, while it is reasonable to consider the BRCA1 c.1268C>T (p.Ser423Phe) as a VUS. In fact, the collected information on this last variant do not consent to adduce further predictions.

Interestingly, among the patients with bilateral breast cancer, we found a woman (patient DP97-18 in Table 1) carrying one PV in BRCA1 (the c.5095C>T) and a small deleterious deletion in BRCA2 gene (the c.1238delT). The presence of double mutations of BRCA1 and BRCA2 seems to be an extremely rare event in the general population, although not exceptional in Ashkenazi breast-cancer patients [22]. After a careful examination of the literature, we have identified ten cases of Italian patients with double heterozygosity (Table 2). Most of these studies consists in the description of a single case or of a small number of families [23,24,25,26,27]. A more comprehensive case-control study was published in 2016 using the Consortium of Investigators of Modifiers of BRCA1/2 (CIMBA) dataset of 32,295 female patients carrying pathogenic BRCA1/2 variants [27]. From this dataset, 93 trans-heterozygotes (TH) (0.3%) who had inherited deleterious mutations in both BRCA1 and BRCA2 genes were identified. The conclusions of this study suggested that BRCA1 mutation in TH patients may drive the clinical TH phenotype (based on elevated ovarian cancer risk and earlier age of breast cancer diagnosis vs subjects carrying a single pathogenic variant in BRCA2) [27]. We think that, although TH subjects would not appear to show a more severe phenotype considering the age of disease onset, cumulative lifetime risk and multiple primary tumors, further studies are needed to individualize the counseling and the clinical management of these patients.

### 4. Material and Methods

The present study included 132 Italian early onset and familial breast and/or ovarian cancer cases selected by genetic counselling and referred for BRCA1/2 testing to the Laboratory of Molecular Diagnostics and Genomics of Fondazione Policlinico Universitario A. Gemelli, IRCCS of Rome between 2016 and 2018.

Single cases of early onset cancer were patients diagnosed with breast cancer before age of 40. For family cases, the likelihood of carrying a BRCA1/2 pathogenic variants was calculated using BRCAPRO risk assessment tool that is a part of the Cancer Genes software program, version 5.1 (University of Texas Southwestern Medical Center; available at http://www4.utsouthwestern.edu/ breasthealth/cagene) [28]. All patients, after subscription of the consent form, were asked to provide detailed information regarding personal and family history of cancer by an interview. Clinical and pathological characteristics were collected from medical records and pathology reports. The study was conducted in accordance with the Declaration of Helsinki, and the protocol (Protocol ID: 0007205/16) was approved by the Ethics Committee of Università Cattolica del Sacro Cuore, Fondazione Policlinico Universitario Agostino Gemelli (Project ID: ESR14-10185, Approval date: 24 February 2016).

Genomic DNA was obtained using an automatic extractor (MagCore HF16 Plus, Diatech Lab Line, Jesi, Italy); DNA concentration and quality were evaluated by NanoPhotometer™ (Implen, Munchen, Germany) and samples were stored at −20 °C until use. Only DNA meeting following requirements: OD260/280 ratio ≥ 1.7, concentration ≥ 15 ng/μL, no degradation signals visible on agarose gel, were processed.

Library preparation was performed using Devyser BRCA NGS kit (DEVYSER, Hägersten, Sweden), according to the manufacturer’s instructions. The reagent kit V2, 500 Cycles PE, on the Illumina MiSeq System (Illumina, San Diego, CA, USA), and Amplicon Suite Software (SmartSeq, Novara, Italy) were used for sequencing reaction and NGS results interpretation, respectively. The BRCA1 and BRCA2 genes reference were: NG_005905.2, NM_007294.3 and NG_012772.3, NM_000059.3. A cut off of 500× coverage was applied to all analyses.

In order to confirm NGS results, PCR-amplified fragments were directly sequenced using BigDye Terminator Cycle Sequencing kit v3.1(Thermo Fisher Scientific) and 3500 Genetic Analyzer (Thermo Fisher Scientific), according to the manufacturer’s instructions. The SeqScape v3.0 software package (Thermo Fisher Scientific) was employed for electropherograms analysis. Variants were named according to Human Genome Variation Society nomenclature (http://www.hgvs.org/mutnomen/). ClinVar (https://www.ncbi.nlm.nih.gov/clinvar/), Leiden Open Variation Database (LOVD; https://www.lovd.nl/) and Evidence-based Network for the Interpretation of Germline Mutant Alleles (ENIGMA; https://enigmaconsortium.org/) Database were used as main reference. Finally, in order to confirm the Copy Number Variation (CNV) prediction results provided by Amplicon Suite Software (SmartSeq, V2, Novara, Italy), the positive samples were investigated using MAQ (Multiplex Amplicon Quantification) assay, as previously reported [29].

We used VarSome Software (https://varsome.com, V6.7, Lausanne, Switzerland) to investigate the pathogenicity of novel BRCA1/2 point variants. VarSome is both a powerful annotation tool and search engine for human genomic variants, and a platform enabling the sharing of knowledge on specific variants [30]. If the query is an unknown variant, VarSome provides several evidences supporting the pathogenicity and/or benignity. The verdict is reported according to the American College of Medical Genetics (ACMG) guidelines [31], classifying the variant as: ‘Pathogenic’, ‘Likely Pathogenic’, ‘Likely Benign’, ‘Benign’ or ‘Uncertain Significance’. Population frequency data are taken from gnomAD [14], Kaviar3 [15] and ICGC Somatic [16], while pathogenicity predictions from dbNSFP [17], which compiles prediction scores from 20 different algorithms, and DANN [18]. Finally, Priors V2.0 Software (http://priors.hci.utah.edu/PRIORS/help.php, Salt Lake City, UT, USA) was employed to evaluate the effect of novel synonymous variants on the splicing process.

### 4.1. Patients

The present study included 132 Italian early onset and familial breast and/or ovarian cancer cases selected by genetic counselling and referred for BRCA1/2 testing to the Laboratory of Molecular Diagnostics and Genomics of Fondazione Policlinico Universitario A. Gemelli, IRCCS of Rome between 2016 and 2018.

Single cases of early onset cancer were patients diagnosed with breast cancer before age of 40. For family cases, the likelihood of carrying a BRCA1/2 pathogenic variants was calculated using BRCAPRO risk assessment tool that is a part of the Cancer Genes software program, version 5.1 (University of Texas Southwestern Medical Center; available at http://www4.utsouthwestern.edu/ breasthealth/cagene) [28]. All patients, after subscription of the consent form, were asked to provide detailed information regarding personal and family history of cancer by an interview. Clinical and pathological characteristics were collected from medical records and pathology reports. The study was conducted in accordance with the Declaration of Helsinki, and the protocol (Protocol ID: 0007205/16) was approved by the Ethics Committee of Università Cattolica del Sacro Cuore, Fondazione Policlinico Universitario Agostino Gemelli (Project ID: ESR14-10185, Approval date: 24 February 2016).

### 4.2. Molecular Analysis

Genomic DNA was obtained using an automatic extractor (MagCore HF16 Plus, Diatech Lab Line, Jesi, Italy); DNA concentration and quality were evaluated by NanoPhotometer™ (Implen, Munchen, Germany) and samples were stored at −20 °C until use. Only DNA meeting following requirements: OD260/280 ratio ≥ 1.7, concentration ≥ 15 ng/μL, no degradation signals visible on agarose gel, were processed.

Library preparation was performed using Devyser BRCA NGS kit (DEVYSER, Hägersten, Sweden), according to the manufacturer’s instructions. The reagent kit V2, 500 Cycles PE, on the Illumina MiSeq System (Illumina, San Diego, CA, USA), and Amplicon Suite Software (SmartSeq, Novara, Italy) were used for sequencing reaction and NGS results interpretation, respectively. The BRCA1 and BRCA2 genes reference were: NG_005905.2, NM_007294.3 and NG_012772.3, NM_000059.3. A cut off of 500× coverage was applied to all analyses.

In order to confirm NGS results, PCR-amplified fragments were directly sequenced using BigDye Terminator Cycle Sequencing kit v3.1(Thermo Fisher Scientific) and 3500 Genetic Analyzer (Thermo Fisher Scientific), according to the manufacturer’s instructions. The SeqScape v3.0 software package (Thermo Fisher Scientific) was employed for electropherograms analysis. Variants were named according to Human Genome Variation Society nomenclature (http://www.hgvs.org/mutnomen/). ClinVar (https://www.ncbi.nlm.nih.gov/clinvar/), Leiden Open Variation Database (LOVD; https://www.lovd.nl/) and Evidence-based Network for the Interpretation of Germline Mutant Alleles (ENIGMA; https://enigmaconsortium.org/) Database were used as main reference. Finally, in order to confirm the Copy Number Variation (CNV) prediction results provided by Amplicon Suite Software (SmartSeq, V2, Novara, Italy), the positive samples were investigated using MAQ (Multiplex Amplicon Quantification) assay, as previously reported [29].

### 4.3. In Silico Prediction Analysis

We used VarSome Software (https://varsome.com, V6.7, Lausanne, Switzerland) to investigate the pathogenicity of novel BRCA1/2 point variants. VarSome is both a powerful annotation tool and search engine for human genomic variants, and a platform enabling the sharing of knowledge on specific variants [30]. If the query is an unknown variant, VarSome provides several evidences supporting the pathogenicity and/or benignity. The verdict is reported according to the American College of Medical Genetics (ACMG) guidelines [31], classifying the variant as: ‘Pathogenic’, ‘Likely Pathogenic’, ‘Likely Benign’, ‘Benign’ or ‘Uncertain Significance’. Population frequency data are taken from gnomAD [14], Kaviar3 [15] and ICGC Somatic [16], while pathogenicity predictions from dbNSFP [17], which compiles prediction scores from 20 different algorithms, and DANN [18]. Finally, Priors V2.0 Software (http://priors.hci.utah.edu/PRIORS/help.php, Salt Lake City, UT, USA) was employed to evaluate the effect of novel synonymous variants on the splicing process.

### 5. Conclusions

In this study, we reported that association of NGS screening with the employment of prediction tools and co-segregation analysis improves the identification of disease causing variants in BRCA1 and BRCA2 genes. Our approach identified risk-relevant mutations in 25.7% of analyzed patients. However, we cannot overlook that a consistent number of our high-risk families still remained without a satisfying answer. Unfortunately, the percentage of VUSs in BRCA1/2 genes is still high and further collective efforts are necessary to reach a definitive classification.

Finally, the auspice is that the use of multi-gene panel in routine diagnostics increases the detection rate, even if the first results from literature data look at more advanced screenings such as whole exome sequencing and/or RNA seq.



# SUPPLEMENTAL FILE 1: ijms-20-03442.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)