# MAIN TEXT

## Spectrum and prevalence of BRCA1/2 germline mutations in Pakistani breast cancer patients: results from a large comprehensive study

### Abstract

BackgroundPathogenic germline mutations in BRCA1 and BRCA2 (BRCA1/2) account for the majority of hereditary breast and/or ovarian cancers worldwide. To refine the spectrum of BRCA1/2 mutations and to accurately estimate the prevalence of mutation in the Pakistani population, we studied 539 breast cancer patients selected for family history and age of diagnosis.MethodsComprehensive screening for BRCA1/2 germline mutations was performed using state-of-the-art technologies.ResultsA total of 133 deleterious mutations were identified in 539 families (24.7%), comprising 110 in BRCA1 and 23 in BRCA2. The prevalence of BRCA1/2 small-range mutations and large genomic rearrangements was 55.4% (36/65) for families with breast and ovarian cancer, 27.4% (67/244) for families with two or more cases of breast cancer, 18.5% (5/27) for families with male breast cancer, and 12.3% (25/203) for families with a single case of early-onset breast cancer. Nine mutations were specific to the Pakistani population. Eighteen mutations in BRCA1 and three in BRCA2 were recurrent and accounted for 68.2% (75/110) and 34.8% (8/23) of all identified mutations in BRCA1 and BRCA2, respectively. Most of these mutations were exclusive to a specific ethnic group and may result from founder effects.ConclusionsOur findings show that BRCA1/2 mutations account for one in four cases of hereditary breast/ovarian cancer, one in five cases of male breast cancer, and one in eight cases of early-onset breast cancer in Pakistan. Our study suggests genetic testing of an extended panel of 21 recurrent BRCA1/2 mutations for appropriately selected patients and their families in Pakistan.

### Background

Individuals harboring BRCA1/2 germline mutations have high lifetime risks of breast and ovarian cancer. The identification of individuals harboring BRCA1/2 mutations is crucial to assess their cancer risk, consider preventive measures and tailor cancer management strategies.

Several studies have investigated the prevalence of BRCA1/2 small-range mutations and/or large genomic rearrangements (LGRs) with frequencies varying from 17.6% to 29.8% in white populations from Europe and Australia [1–5] and 9.4% to 21.7% in non-whites from Asia [6–8]. The prevalence and distribution of BRCA1/2 mutations vary across populations, mainly due to population-specific recurrent or founder mutations. Accurate identification of the population-specific mutation spectrum is therefore the first step towards incorporating appropriate genetic BRCA1/2 testing into clinical practice in a particular population. This information is not fully elucidated in Pakistan, a country with one of the highest rates of breast cancer in Asia.

To date, no large comprehensive studies evaluating the BRCA1/2 mutations have been reported in the Pakistani population and mutations in males have not been identified so far. Small-range mutations were previously reported in 341 unselected breast and 120 ovarian cancer patients, in which the analysis was restricted to a few exons only [9]. We conducted two studies in early-onset and familial breast/ovarian cancer patients from Pakistan. In the initial study the complete coding regions and exon-intron boundaries of BRCA1/2 were screened for small-range mutations in 176 patients [10]. In the other study 120 BRCA1/2 small-range mutations negative patients were screened for LGRs [11]. Other Asian studies also had small sample sizes [12, 13], reported small-range mutations only [14, 15], and/or restricted LGR analyses to a small number of study participants [6, 16, 17].

Here, we refined the spectrum of BRCA1/2 mutations and more precisely estimated the mutation frequencies including small-range mutations and LGRs in 539 early-onset and familial breast cancer patients from Pakistan.

### Methods

Five hundred and ninety-three breast cancer only or breast and ovarian cancer families were enrolled through index breast and/or ovarian cancer patients who presented at the Shaukat Khanum Memorial Cancer Hospital and Research Centre (SKMCH&RC) in Lahore, Pakistan, from September 2004 to August 2015. The recruited families were classified into five risk groups based on family history of breast/ovarian cancer or age at diagnosis (Table 1) as described previously [19]. After enrollment, 54 families were excluded (Fig. 1), leaving 539 families in the study.
Table 1BRCA1/2 mutation frequencies according to family structureRisk groupPhenotype of familiesNo. of familiesNo. of families with mutations (%) in
BRCA1

BRCA2

BRCA1/2
Small-rangeLGRsAllSmall-rangeLGRsAllAll families539101 (18.7)9 (1.7)110 (20.4)23 (4.3)0 (0)23 (4.3)133 (24.7)aFemale breast cancer families44767 (15.0)7 (1.6)74 (16.6)18 (4.0)0 (0)18 (4.0)92 (20.6)A11 case ≤ 30 years20320 (9.8)2 (1.0)22 (10.8)3 (1.5)0 (0)3 (1.5)25 (12.3)A22 cases, >1 diagnosed ≤50 years13120 (15.3)4 (3.0)24 (18.3)6 (4.6)0 (0)6 (4.6)30 (22.9)A3>3 cases, >1 diagnosed ≤50 years11327 (23.9)1 (0.9)28 (24.8)9 (8.0)0 (0)9 (8.0)37 (32.7)A4Male breast cancer families>1 case of male breast cancer271 (3.7)0 (0)1 (3.7)4 (14.8)0 (0)4 (14.8)5 (18.5)BBreast-ovarian cancer families>1 breast cancer and >1 ovarian cancer6533 (50.8)2 (3.0)35 (53.8)1 (1.5)0 (0)1 (1.5)36 (55.4)LGRs large genomic rearrangementsaIncluding 57 previously reported families [11, 18]
Fig. 1Flow diagram for the study participant’s enrollment, screening methods used and BRCA1/2 mutations detected

BRCA1/2 mutation frequencies according to family structure

LGRs large genomic rearrangements

aIncluding 57 previously reported families [11, 18]

Flow diagram for the study participant’s enrollment, screening methods used and BRCA1/2 mutations detected

Clinical and histopathological data and comprehensive information on personal and family history of cancer(s), and ethnicity were obtained from all study participants. The Institutional Review Board of the SKMCH&RC approved the study. All study participants signed an informed written consent before providing a blood sample.

Genomic DNA was extracted from 9 to 18 ml of whole blood samples, as described previously [20]. The entire coding regions of the BRCA1 (Genbank accession number U14680) and BRCA2 (Genbank accession number U43746) genes including exon-intron boundaries were screened in 139 patient DNA samples for small-range mutations using denaturing high-performance liquid chromatography (DHPLC) analysis as previously described [21, 22]. Each sample revealing variants was sequenced as described elsewhere [18]. Four-hundred female patients selected based on triple-negative vs. non triple-negative breast cancer phenotype had been screened for small-range BRCA1/2 mutations and 33 mutations were described [18]. All small-range mutation-negative patients had been screened for LGRs using multiplex ligation-dependent probe amplification and three LGRs were described [11]. For the current study, families were selected on the basis of family history of breast/ovarian cancer, male breast cancer or age at diagnosis.

All BRCA1/2 alterations identified in the current study were classified into pathogenic mutations, variants of unknown significance, or polymorphisms. Pathogenic mutations were defined as (i) small-range mutations which affect one or a few nucleotides including frameshift, nonsense, or splice-site mutations and generate a premature termination codon, except BRCA2 exon 27 variants generating a premature termination codon after codon 3010 [23] and (ii) LGRs that span one or more exons. Mutations were designated using the Human Genome Variation Society (HGVS) and the Breast Cancer Information Core (BIC) committee nomenclature.

All identified mutations were searched in various mutation databases including BIC (https://research.nhgri.nih.gov/bic/), ClinVar (http://www.ncbi.nim.nih.gov/clinvar/), LOVD (http://databases.lovd.nl/shared/genes/BRCA2), ARUP (http://arup.utah.edu/database/BRCA/), and BRCA Exchange (http://brcaexchange.org/). Mutations not reported in these databases were considered as novel and specific to Pakistani population.

Distribution of clinical and histopathological characteristics between BRCA1/2 carriers and non-carriers were estimated using Fisher’s exact test for categorical variables and the Wilcoxon rank-sum test for quantitative variables. All statistical tests were two-sided. Results were considered significant at a p value of <0.05. All statistical analyses were done using StatXact 4 for Windows (Cytel, Cambridge, USA) and R, version 2.1.

### Enrollment of families

Five hundred and ninety-three breast cancer only or breast and ovarian cancer families were enrolled through index breast and/or ovarian cancer patients who presented at the Shaukat Khanum Memorial Cancer Hospital and Research Centre (SKMCH&RC) in Lahore, Pakistan, from September 2004 to August 2015. The recruited families were classified into five risk groups based on family history of breast/ovarian cancer or age at diagnosis (Table 1) as described previously [19]. After enrollment, 54 families were excluded (Fig. 1), leaving 539 families in the study.
Table 1BRCA1/2 mutation frequencies according to family structureRisk groupPhenotype of familiesNo. of familiesNo. of families with mutations (%) in
BRCA1

BRCA2

BRCA1/2
Small-rangeLGRsAllSmall-rangeLGRsAllAll families539101 (18.7)9 (1.7)110 (20.4)23 (4.3)0 (0)23 (4.3)133 (24.7)aFemale breast cancer families44767 (15.0)7 (1.6)74 (16.6)18 (4.0)0 (0)18 (4.0)92 (20.6)A11 case ≤ 30 years20320 (9.8)2 (1.0)22 (10.8)3 (1.5)0 (0)3 (1.5)25 (12.3)A22 cases, >1 diagnosed ≤50 years13120 (15.3)4 (3.0)24 (18.3)6 (4.6)0 (0)6 (4.6)30 (22.9)A3>3 cases, >1 diagnosed ≤50 years11327 (23.9)1 (0.9)28 (24.8)9 (8.0)0 (0)9 (8.0)37 (32.7)A4Male breast cancer families>1 case of male breast cancer271 (3.7)0 (0)1 (3.7)4 (14.8)0 (0)4 (14.8)5 (18.5)BBreast-ovarian cancer families>1 breast cancer and >1 ovarian cancer6533 (50.8)2 (3.0)35 (53.8)1 (1.5)0 (0)1 (1.5)36 (55.4)LGRs large genomic rearrangementsaIncluding 57 previously reported families [11, 18]
Fig. 1Flow diagram for the study participant’s enrollment, screening methods used and BRCA1/2 mutations detected

BRCA1/2 mutation frequencies according to family structure

LGRs large genomic rearrangements

aIncluding 57 previously reported families [11, 18]

Flow diagram for the study participant’s enrollment, screening methods used and BRCA1/2 mutations detected

Clinical and histopathological data and comprehensive information on personal and family history of cancer(s), and ethnicity were obtained from all study participants. The Institutional Review Board of the SKMCH&RC approved the study. All study participants signed an informed written consent before providing a blood sample.

### BRCA1/2 mutation screening

Genomic DNA was extracted from 9 to 18 ml of whole blood samples, as described previously [20]. The entire coding regions of the BRCA1 (Genbank accession number U14680) and BRCA2 (Genbank accession number U43746) genes including exon-intron boundaries were screened in 139 patient DNA samples for small-range mutations using denaturing high-performance liquid chromatography (DHPLC) analysis as previously described [21, 22]. Each sample revealing variants was sequenced as described elsewhere [18]. Four-hundred female patients selected based on triple-negative vs. non triple-negative breast cancer phenotype had been screened for small-range BRCA1/2 mutations and 33 mutations were described [18]. All small-range mutation-negative patients had been screened for LGRs using multiplex ligation-dependent probe amplification and three LGRs were described [11]. For the current study, families were selected on the basis of family history of breast/ovarian cancer, male breast cancer or age at diagnosis.

### Mutation classification

All BRCA1/2 alterations identified in the current study were classified into pathogenic mutations, variants of unknown significance, or polymorphisms. Pathogenic mutations were defined as (i) small-range mutations which affect one or a few nucleotides including frameshift, nonsense, or splice-site mutations and generate a premature termination codon, except BRCA2 exon 27 variants generating a premature termination codon after codon 3010 [23] and (ii) LGRs that span one or more exons. Mutations were designated using the Human Genome Variation Society (HGVS) and the Breast Cancer Information Core (BIC) committee nomenclature.

All identified mutations were searched in various mutation databases including BIC (https://research.nhgri.nih.gov/bic/), ClinVar (http://www.ncbi.nim.nih.gov/clinvar/), LOVD (http://databases.lovd.nl/shared/genes/BRCA2), ARUP (http://arup.utah.edu/database/BRCA/), and BRCA Exchange (http://brcaexchange.org/). Mutations not reported in these databases were considered as novel and specific to Pakistani population.

### Statistical analyses

Distribution of clinical and histopathological characteristics between BRCA1/2 carriers and non-carriers were estimated using Fisher’s exact test for categorical variables and the Wilcoxon rank-sum test for quantitative variables. All statistical tests were two-sided. Results were considered significant at a p value of <0.05. All statistical analyses were done using StatXact 4 for Windows (Cytel, Cambridge, USA) and R, version 2.1.

### Results

A total of 539 index patients from unrelated families were enrolled and stratified into five risk groups (Table 1). The mean age of disease onset was 35.4 years (range 18-78) for female breast cancer (n=502), 45.4 years (range 23-66) for ovarian cancer (n=30) and 54.5 years (range 27-76) for male breast cancer (n=27) patients.

Evaluation of pooled data from 539 patients yielded 71 distinct pathogenic mutations in 133 families (24.7%) (Table 1). Fifty-three BRCA1 mutations were detected in 110 families (20.4%) and 18 BRCA2 mutations in 23 families (4.3%). Five mutations in BRCA1 (9.4%) and four mutations in BRCA2 (22.2%) were novel (Table 2). The phenotypes of all families carrying BRCA1/2 mutations are presented in Table 3.
Table 2Deleterious BRCA1/2 germline mutations in Pakistani breast/ovarian cancer familiesFamilyExonBIC designationHGVS designationMutation typebReported in databases (No. of entries)cNucleotideCodonDesignationNucleotide changeaEffect on proteinBRCA1-small-range mutations 432218523185insAc.66dupp.(Glu23Argfs*18)FSBIC (32)d 723218523185delAGc.68_69delp.(Glu23Valfs*17)FSBIC (2036)d 372Intron 4IVS4-2-IVS4-2A>Gc.135-2A>GSplice siteSPBIC (1) 254e7454112454delAc.335delp.(Asn112Ilefs*7)FSClinVar (2) 4497509130Y130Xc.390C>Gp.(Tyr130*)NSLOVD (3) 296, 317, 340e, 511, 521, 626e, 74711804229804delTc.685delp.(Ser229Leufs*5)FSBIC (2) 470e11903262Q262Xc.784C>Tp.(Gln262*)NSClinVar (3) 7111110142991014delGTc.895_896delp.(Val299Argfs*4)FSBIC (2) 669e1111273361127delAc.1008delp.(Glu337Lysfs*4)FSNod 7481113073961307delTc.1188delp.(Asp396Glufs*14)FSLOVD (1) 241e1113093971309delAc.1190delp.(Asp397Alafs*13)FSClinVar (3) 7221115184851518_1572dup55c.1399_1453dupp.(Ala485Glufs*13)FSNo 336e111590491Q491Xc.1471C>Tp.(Gln491*)NSBIC (4)d N121118985931898delTATGGAAc.1779_1785delp.(Met594Serfs*3)FSLOVD (2) N28, 328e, 557e111912598L598Xc.1793T>Gp.(Leu598*)NSBIC (1)d 574e1120806552080insAc.1961dupp.(Tyr655Valfs*18)FSBIC (13)d 488e112268717E717Xc.2149G>Tp.(Glu717*)NSClinVar (2) 236e, 283e, 489e, 493e, 5931123887572388delGc.2269delp.(Val757Phefs*8)FSBIC (10)d 3631124337722433delGTc.2314_2315delp.(Val772Thrfs*4)FSLOVD (2) 550h112457780Q780Xc.2338C>Tp.(Gln780*)NSBIC (36) 362, 4691124597802459delGGAAc.2340_2343delp.(Glu781Valfs*10)FSLOVD (2) 421e, 442, 510e, 542, 619e1125248022524delTGc.2405_2406delp.(Val802Glufs*7)FSBIC (5)d N341126578462657delAAT-insGc.2538_2540delinsGp.(Met847Glyfs*4)FSLOVD (2) 415e, 660e112722868S868Xc.2603C>Gp.(Ser868*)NSBIC (11)d 411e113090991K991Xc.2971A>Tp.(Lys991*)NSClinVar (2) 247e1131009943100delGTc.2981_2982delp.(Cys994*)FSClinVar (5) 29911324810433248delTATTAATGAAc.3129_3138delp.(Asn1043Lysfs*2)FSClinVar (3) N13e, 399e11345811133458delTGAc.3339_3341delp.(Tyr1113*)FSClinVar (2) 6101135311138G1138Xc.3412G>Tp.(Gly1138*)NSClinVar (3) N2511359611593596delAAAGc.3477_3480delp.(Ile1159Metfs*50)FSBIC (3) 279e, 445e1137171200Q1200Xc.3598C>Tp.(Gln1200*)NSBIC (21)d 3821137261203R1203Xc.3607C>Tp.(Arg1203*)NSBIC (36) 64611381912343819delGTAAAc.3700_3704delp.(Val1234Glnfs*8)FSBIC (61) 210e, 211e, 313e, 316, 332e, 463, 520, 624, 652e, 65311388912573889delAGc.3770_3771delp.(Glu1257Glyfs*9)FSBIC (23)d N4e, 687, 724, 74311418413554184delTCAAc.4065_4068delp.(Asn1355Lysfs*10)FSBIC (144)d 318e1243021395Q1395Xc.4183C>Tp.(Gln1395*)NSBIC (28)d 408e1344461443R1443Xc.4327C>Tp.(Arg1443*)NSBIC (128) 523e, 555, N18, 598e, 612, 621Intron 14IVS14-1-IVS14-1G>Ac.4485-1G>ASplice siteSPBIC (2)d 220e, 275e, 512e1546271503S1503Xc.4508C>Ap.(Ser1503*)NSBIC (1)d 609e15478415584784delGc.4665delp.(Arg1555Serfs*4)FSNo 611e16498116214981delAc.4862delp.(Asp1621Valfs*12)FSNo 249e, 65817515416795154delCc.5035delp.(Leu1679*)FSBIC (2) 276e, 679Intron 17IVS17+1-IVS17+1G>Ac.5074+1G>ASplice siteSPBIC (3) 68520535817475358delCc.5239delp.(Gln1747Lysfs*18)FSLOVD (2) 73420538517565385dupCc.5266dupp.(Gln1756Profs*74)FSLOVD (376) 706Intron 20IVS20-1-IVS20-1G>Cc.5278-1G>CSplice siteSPLOVD (5)d 67821542917715429dupGc.5310dupp.(Pro1771Alafs*59)FSLOVD (1) 278, 338e22548017875480delTGc.5361_5362delp.(Cys1787Trpfs*42)FSClinVar (3) 6822254961793K1793Xc.5377A>Tp.(Lys1793*)NSClinVar (1) 248eIntron 23IVS23-2-IVS23-2A>Tc.5468-2A>TSplice siteSPClinVar (1) 260, 264, 329e, 377e, 389, 439, 481, 501, 5222456221835R1835Xc.5503C>Tp.(Arg1835*)NSBIC (74)dBRCA1-large genomic rearrangementse 229, 291, 314, 379, 406, 498, 5491-2--del exon 1-2g.41271967_41308900delfLGR(42)g 261, 71921-24--del exon 21-24g.41172653_41205744delfLGRNoBRCA2-small-range mutations 497, 700332031W31Xc.92G>Ap.(Trp31*)NSClinVar (4) N26Intron 4IVS4-2-IVS4-2A>Gc.426-2A>GSplice siteSPClinVar (4) 5459993255993delCACAAc.765_769delp.(Asn255Lysfs*19)FSNo 3301015284341528delAAAAc.1300_1303delp.(Lys434Glufs*25)FSClinVar (2) 6021130489413048delAc.2820delp.(Val941Cysfs*19)FSNo 2061130639453063delAc.2835delp.(Asp946Ilefs*14)FSClinVar (2) 50511408812874088delAc.3860delp.(Asn1287Ilefs*6)FSBIC (2) 222, 407h, 525, 540h11545017415450delGTAAc.5222_5225delp.(Ser1741Thrfs*35)FSBIC (1) 627, 6841159101894Y1894Xc.5682C>Ap.(Tyr1894*)NSBIC (3) 295e11595019085950delCTc.5722_5723delp.(Leu1908Argfs*2)FSBIC (43)d 44711669621566696delTCc.6468_6469delp.(Gln2157Ilefs*18)FSBIC (24)d 548h11704422747044delAAGAGc.6816_6820delp.(Gly2274Alafs*17)FSClinVar (6) 57915780325267803delAc.7575delp.(Ala2526Glnfs*2)FSLOVD (2) 492Intron 17IVS17+2-IVS17+2C>Ac.7976+2C>ASplice siteSPClinVar (1) 71320877328498773delAAc.8545_8546delp.(Lys2849Glyfs*19)FSNo 70220877928608779_8798dup20c.8551_8570dupp.(Lys2860Asnfs*10)FSNo 207h21889728908897insTc.8669dupp.(Thr2891Asnfs*16)FSClinVar (1) 538Intron 21IVS21+4-IVS21+4A>Gc.8754+4A>GSplice siteSPBIC (7)aNumbering starts at the first A of the first coding ATG (located in exon 2) of NCBI GenBank accession number U14680 (BRCA1) and U43746 (BRCA2)bFS frameshift mutation, LGR large genomic rearrangement, MS missense mutation, NS nonsense mutation, SP splice-site mutationcBIC Breast Cancer Information Core database (https://research.nhgri.nih.gov/projects/bic/), LOVD Leiden Open Variation Database (http://databases.lovd.nl/shared/genes/BRCA2); ClinVar (https://www.ncbi.nlm.nih.gov/clinvar/), date last accessed June 26, 2018dPreviously reported in Pakistani breast/ovarian cancer cases [9, 10]eFamilies and mutation description have been previously reported [11, 18]fGenomic locale for chromosome 17, from the UCSC genome browser, Feb 2009 assemblygNot available in databases; reported in various studies [1, 11]hFamilies with male breast cancer
Table 3Characteristics of the 133 families with deleterious BRCA1/2 mutationsFamilyNo. of cancersAge at onset (years)Other cancer(s)c(age at onset in years)EthnicityFemale BC (Bilateral)OC (OC+BC)BCOCFamilies carrying BRCA1-small-range mutations 236a1-22b--Pathan 3161-25b--Punjabi 2641-26b--Punjabi 7061-26b-Uterus (67)Punjabi N121-26b--Punjabi 6241-27b--Punjabi N251-28b--Punjabi 276a1-28b--Punjabi 6101-28b--Punjabi 6781-28b--Punjabi 411a1-29b-Stomach (70)Punjabi 7241-29b-Renal (48), lung (65), throat (65), unknownPunjabi N281-30b--Punjabi 279a1(1)-27/36b--Punjabi 2782-25b,32--Kashmiri 332a2-26b,51-Leukemia (45)Punjabi 6822-28b,40-Uterus (<62,65), throat (<72)Punjabi N182-29b,<50--Punjabi 421a2-30b,33--Punjabi 4322-30b,53-Skin (12), oral (54)Punjabi 5202-30b,47-Uterus (32)Punjabi 4492-32b,55--Punjabi 557a2-32b,45-Unknown (<55), renal (70)Punjabi 7472-33b,38--Unknown 7222-20,34b-Unknown (<18,<40)Punjabi 6872-37b,45--Punjabi 470a2-40b,40-Stomach (46), colon (59), lungPunjabi 510a2-40b,55--Punjabi N13a2-40b,>50--Punjabi 5932-43,44b-Leukemia (22)Pathan 2992(1)-24/27b,55--Punjabi 660a2(1)-25/26b,70-BladderPunjabi 2602(1)-25/26b,28--Punjabi 5112(1)-30/33b,<32-Brain (75)Punjabi N343-24b,<30,31--Punjabi 669a3-25b,<40,<50-Brain (<78), oral (<80)Punjabi 6853-26b,26,?-Blood (2x)Mohajir 7233-28b,40,?--Pathan 6123-29b,<30,40-Throat (45), uterus (48)Punjabi 313a3-30b, 48,?--Punjabi 336a3-23,30b,38-Prostate (29)Punjabi 493a3-35b,55,>55--Pathan 3823-36b,50,?--Punjabi 489a3-25,42b,45--Punjabi 7433-40,44b,62-Bone (60), leukemia (60)Pathan 6583-26,44b,50--Punjabi 377a3-31,50b,85-Thyroid (59), intestine (70), bladder (75), liverPunjabi 550d3-50b,55,>50-Lung, unknownPunjabi 3723(1)-21/21b,29,45-Squamous cell carcinoma scalp (22)bPathan 626a3(1)-35b,36/37, 42--Balochi 3893(1)-22/32,42,48b-Brain (36), uterus (70)Punjabi 247a4-27,28b,40,42-Uterus (31, 55)Siriaki 652a4-31b,33,42,50--Punjabi 3624-31,32,35b,45-Liver (>40), abdomenPathan 399a4-43, 44b,50,?-Abdomen (45), lung (45), prostate (53)Punjabi 338a4-30,40,44,48b-Stomach (73)Punjabi 408a4(1)-24b,31/31,33,50-Abdomen (54), esophagus (74)Punjabi 5214(1)-25/38b,27,33,70-Stomach (60, 65), lung, unknownPunjabi 6535-24,32,35b,37,50-Colon (42), throat (66)Punjabi 7345-37,38b,55,?,?--Punjabi 2967(1)-21b,<30,34,43,44/44,51,52--Punjabi 4398-30,<35,40b,40,40,40,>50,>65-Uterus (40), prostate, unknownPunjabi 249a8(1)-30/31b,37,45,?,?,?,?,?--Punjabi 619a1130b>50-Punjabi 64611(1)34b36b-Punjabi 74811(1)47b52bSkin (45), liver (50)Punjabi 54212(1)40b46b,70Leukemia (74)Punjabi 210a1445b?,?,?,?Brain (32), abdomen, lung, leukemiaPunjabi 241a2129b,6435LymphomaPunjabi 598a2130b,56>50Stomach (>50, >50), tongue (>50)Punjabi 4632135b,5848-Punjabi 48121<25,47b36Lung (55), uterusPunjabi 6212150,>6050bProstate (65)Punjabi 211a21(1)26b,5050-Punjabi 415a21(1)34,35b35bLeukemia (45), unknownPunjabi 67921(1)28b,<3349b-Punjabi N4a21 (1)41b,4545Uterus (38)Punjabi 488a22(1)40b,5542b,45-Punjabi 55522(1)29b,3636b,>50Leukemia (10), vocal cord (45)Punjabi 31723(1)41,4647b,52,55Fallopian tube (47)bPunjabi 318a24(2)40,46b40,42b,44,58Bladder (50, 50)Pathan 4422(1)234,50/50b28,52Leukemia (15)Punjabi 283a2(1)334/38b,5654,55,65-Pathan 254a3127,32b,4341BrainPunjabi 7113140b,?,??Gall bladderSindhi 445a31(1)44b,>60,7374Gall bladderPunjabi 3633232,35,7047b, ?Lung (65), oral (70), liverKashmiri 329a3334b,39,?39,<50,?-Punjabi 609a4(1)129b,31,48/55,6530-Mohajir 328a4129,30b,31,3955-Punjabi 5014134b, 35, >50,??Brain (42)Punjabi 52242(1)30b,<40,45,4535,60Uterus (41)Punjabi 611a42(1)31b,36,37,4250,55Blood (30)Punjabi 523a3(1)439/46,40,53b30,45,51,60-Punjabi 275a4(1)1(1)34/40b,40,42,5040-Punjabi 46951(1)29b,29,<35,>35,>5532b-Mohajir 340a6134b,42,<50,<50,<50,<5054-Balochi 574a6232,32,35,45b,48,>5048,55-Mohajir 512a6(1)1<25,<30,<40,46/55b,<50,>50<50Uterus (<50)Kashmiri 248a72(2)23b,<25,34,<40,<46,<60,?23b,<60-Punjabi 220a81(1)25,27b,<30,53,58,63,77<30-PunjabiFamilies carrying BRCA1-LGRsa 2291-28b--Punjabi 3792-29,31b-Liver (38)Punjabi 2612-33,34b--Punjabi 4062-39b,40-Abdomen (65)Punjabi 4982-40,41b--Siriaki 5492-38,72b-UnknownPunjabi 3146-32b,42,56,70,?,?-Uterus (54), pharynx (59), brain (63), abdomenPunjabi 2913139,42,4848bStomach, brainPunjabi 7193(1)1>40,42b,??-PunjabiFamilies carrying BRCA2-small-range mutations 3301-29b-Lung (48, 58, 66), tongue (55)Punjabi 2061-30b--Pathan 540d1-67b--Mohajir 207d1-76b-Intestine (60)Punjabi 295a1(1)-23/23b-Leukemia (49), esophagus (50)Punjabi N262-26b,35--Pathan 6022-31b,43--Punjabi 4922-38b,39--Mohajir 5052-43,46b--Pathan 7132-35,56b--Kashmiri 7002(1)-35/43b,46-Throat (72)Punjabi 6272(1)-42b,51/51--Pathan 5453-35,36b,47-Brain (50), uterus (50), bone (54)Punjabi 4973-51b,55,50-BrainSiriaki 548d3-45,50,69b--Pathan 7024-26,30b,<33,70-Throat (72)Punjabi 5794-35,49,<50,51b-Oral (35), gall bladder (42)Kashmiri 407d4-31,45(male),45,54b-Esophagus (39,59), leukemia (64)Mohajir 5385-34b,38,45,50,58-Retinoblastoma (3), pancreas (75), liver (83)Pathan 6845-40b,45,48,50,<57-Throat (<48, <82), stomach (53), intestine (60)Pathan 4473(1)231b,<50,55/6550,>50Abdomen (>50), colon (62), brain (65)Punjabi 5254(1)132,33/35b,<50,60<45Bladder (>60)Mohajir 22272(1)35,42,43,50,54b,60,?47,53bLung (60), prostate (70), lungKashmiriBC breast cancer, OC ovarian cancer, Unknown cancer phenotype is not knownaMutations previously described [11, 18]; bProband; cAge at cancer diagnosis is mentioned along with cancer phenotype. For relatives with unknown age at cancer onset, only cancer phenotype is mentioned; dFamilies with male breast cancer

Deleterious BRCA1/2 germline mutations in Pakistani breast/ovarian cancer families

aNumbering starts at the first A of the first coding ATG (located in exon 2) of NCBI GenBank accession number U14680 (BRCA1) and U43746 (BRCA2)

bFS frameshift mutation, LGR large genomic rearrangement, MS missense mutation, NS nonsense mutation, SP splice-site mutation

cBIC Breast Cancer Information Core database (https://research.nhgri.nih.gov/projects/bic/), LOVD Leiden Open Variation Database (http://databases.lovd.nl/shared/genes/BRCA2); ClinVar (https://www.ncbi.nlm.nih.gov/clinvar/), date last accessed June 26, 2018

dPreviously reported in Pakistani breast/ovarian cancer cases [9, 10]

eFamilies and mutation description have been previously reported [11, 18]

fGenomic locale for chromosome 17, from the UCSC genome browser, Feb 2009 assembly

gNot available in databases; reported in various studies [1, 11]

hFamilies with male breast cancer

Characteristics of the 133 families with deleterious BRCA1/2 mutations

BC breast cancer, OC ovarian cancer, Unknown cancer phenotype is not known

aMutations previously described [11, 18]; bProband; cAge at cancer diagnosis is mentioned along with cancer phenotype. For relatives with unknown age at cancer onset, only cancer phenotype is mentioned; dFamilies with male breast cancer

Twenty-one (21/71; 29.6%) mutations including 18 in BRCA1 and three in BRCA2 occurred more than once (Fig. 2a, b). These mutations were identified in 83 unrelated families and accounted for 62.4% (83/133) of all families with mutations. The most common BRCA1 mutation was c.3770_3771del (ten Punjabi families), followed by c.5503C>T (nine Punjabi families), exon 1-2 deletion (seven Punjabi families), c.685del (five Punjabi and two Balochi families), c.4485-1G>A (six Punjabi families), c.2269del (one Punjabi and four Pathan families), c.2405_2406del (five Punjabi families), c.4065_4068del (three Punjabi and one Pathan families), c.1793T>G and c.4508C>A (three Punjabi families each), exon 21-24 deletion, c.2603C>G, c.3339_3341del, c.3598C>T, c.5035del, c.5074+1G>A, and c.5361_5362del (two Punjabi families each) and c.2340_2343del (one Pathan and one Mohajir families). The most common BRCA2 mutation was c.5222_5225del (one Punjabi and three Mohajir families), followed by c.92G>A (two Punjabi families) and c.5682C>A (two Pathan families).
Fig. 2Distribution of deleterious germline mutations identified in Pakistani breast/ovarian cancer families across the BRCA1 and BRCA2 genes. Recurrent mutations are marked with (RM). The distribution of breast cancer (BC) and ovarian cancer (OC) in families according to the position of the mutations in BRCA1 (a) and BRCA2 (b) is shown. Regions inferred to be breast cancer cluster regions (BCCRs) and ovarian cancer cluster regions (OCCRs) according to Rebbeck and colleagues [24] are shown at the bottom

Distribution of deleterious germline mutations identified in Pakistani breast/ovarian cancer families across the BRCA1 and BRCA2 genes. Recurrent mutations are marked with (RM). The distribution of breast cancer (BC) and ovarian cancer (OC) in families according to the position of the mutations in BRCA1 (a) and BRCA2 (b) is shown. Regions inferred to be breast cancer cluster regions (BCCRs) and ovarian cancer cluster regions (OCCRs) according to Rebbeck and colleagues [24] are shown at the bottom

In addition to the deleterious mutations, 153 (28.4%) distinct BRCA1/2 sequence variants were detected: 79 missense variants, 48 non-coding variants, 24 synonymous variants, one in-frame deletion, and one polymorphic nonsense variant in exon 27 of BRCA2 (data not shown).

The frequencies of BRCA1/2 mutations by risk group are provided in Table 1. For BRCA1, the highest mutation frequency was noted in families with breast and ovarian cancer (53.8%), followed by families with at least three breast cancer cases (24.8%), families with two breast cancer cases (18.3%), or families with one early-onset breast cancer case (<30 years) (10.8%). For BRCA2, the highest frequency was observed in families with male breast cancer (14.8%).

BRCA1 carriers (n=110) were more often identified among female patients (99.1% vs. 94.6%, p=0.039) and belonged to the Punjabi ethnic group (81.8% vs. 68.7%, p=0.030) compared to non-carriers (n=406). In contrast, BRCA2 carriers (n=23) were more common among male patients (17.4% vs. 5.4%, p=0.043) and more often belonged to Pathan ethnic group (34.8% vs. 15.5%, p=0.009).

Female breast cancer patients with mutations in BRCA1 (n=106) or BRCA2 (n=19) had a similar mean age of diagnosis (34.0 years (range 21–72) and 37.7 years (range 23-56), respectively, p=0.073, Wilcoxon rank-sum test), which did not differ to that of non-carriers (n=377) (35.7 years (range 18-78). In contrast, male breast cancer patients harboring BRCA2 mutations (n=4) had an older mean age of diagnosis than non-carriers (n=22) (66.5 years (range 54-76) and 52.5 years (range 27-69) years, respectively, p=0.039, Wilcoxon rank-sum test).

BRCA1-associated breast tumors more often were invasive ductal carcinomas (99.0% vs. 91.4%, p=0.004), triple-negative (60.8% vs. 22.6%, p=<0.0001), and of higher tumor grade (grade 3: 94.9% vs. 63.2%, p=<0.0001) compared to tumors of non-carriers. BRCA2-associated breast tumors more often were PR positive compared to tumors of non-carriers (81.8% vs. 57.2%, p=0.025) (data not shown).

### Spectrum of BRCA1/2 mutations

Evaluation of pooled data from 539 patients yielded 71 distinct pathogenic mutations in 133 families (24.7%) (Table 1). Fifty-three BRCA1 mutations were detected in 110 families (20.4%) and 18 BRCA2 mutations in 23 families (4.3%). Five mutations in BRCA1 (9.4%) and four mutations in BRCA2 (22.2%) were novel (Table 2). The phenotypes of all families carrying BRCA1/2 mutations are presented in Table 3.
Table 2Deleterious BRCA1/2 germline mutations in Pakistani breast/ovarian cancer familiesFamilyExonBIC designationHGVS designationMutation typebReported in databases (No. of entries)cNucleotideCodonDesignationNucleotide changeaEffect on proteinBRCA1-small-range mutations 432218523185insAc.66dupp.(Glu23Argfs*18)FSBIC (32)d 723218523185delAGc.68_69delp.(Glu23Valfs*17)FSBIC (2036)d 372Intron 4IVS4-2-IVS4-2A>Gc.135-2A>GSplice siteSPBIC (1) 254e7454112454delAc.335delp.(Asn112Ilefs*7)FSClinVar (2) 4497509130Y130Xc.390C>Gp.(Tyr130*)NSLOVD (3) 296, 317, 340e, 511, 521, 626e, 74711804229804delTc.685delp.(Ser229Leufs*5)FSBIC (2) 470e11903262Q262Xc.784C>Tp.(Gln262*)NSClinVar (3) 7111110142991014delGTc.895_896delp.(Val299Argfs*4)FSBIC (2) 669e1111273361127delAc.1008delp.(Glu337Lysfs*4)FSNod 7481113073961307delTc.1188delp.(Asp396Glufs*14)FSLOVD (1) 241e1113093971309delAc.1190delp.(Asp397Alafs*13)FSClinVar (3) 7221115184851518_1572dup55c.1399_1453dupp.(Ala485Glufs*13)FSNo 336e111590491Q491Xc.1471C>Tp.(Gln491*)NSBIC (4)d N121118985931898delTATGGAAc.1779_1785delp.(Met594Serfs*3)FSLOVD (2) N28, 328e, 557e111912598L598Xc.1793T>Gp.(Leu598*)NSBIC (1)d 574e1120806552080insAc.1961dupp.(Tyr655Valfs*18)FSBIC (13)d 488e112268717E717Xc.2149G>Tp.(Glu717*)NSClinVar (2) 236e, 283e, 489e, 493e, 5931123887572388delGc.2269delp.(Val757Phefs*8)FSBIC (10)d 3631124337722433delGTc.2314_2315delp.(Val772Thrfs*4)FSLOVD (2) 550h112457780Q780Xc.2338C>Tp.(Gln780*)NSBIC (36) 362, 4691124597802459delGGAAc.2340_2343delp.(Glu781Valfs*10)FSLOVD (2) 421e, 442, 510e, 542, 619e1125248022524delTGc.2405_2406delp.(Val802Glufs*7)FSBIC (5)d N341126578462657delAAT-insGc.2538_2540delinsGp.(Met847Glyfs*4)FSLOVD (2) 415e, 660e112722868S868Xc.2603C>Gp.(Ser868*)NSBIC (11)d 411e113090991K991Xc.2971A>Tp.(Lys991*)NSClinVar (2) 247e1131009943100delGTc.2981_2982delp.(Cys994*)FSClinVar (5) 29911324810433248delTATTAATGAAc.3129_3138delp.(Asn1043Lysfs*2)FSClinVar (3) N13e, 399e11345811133458delTGAc.3339_3341delp.(Tyr1113*)FSClinVar (2) 6101135311138G1138Xc.3412G>Tp.(Gly1138*)NSClinVar (3) N2511359611593596delAAAGc.3477_3480delp.(Ile1159Metfs*50)FSBIC (3) 279e, 445e1137171200Q1200Xc.3598C>Tp.(Gln1200*)NSBIC (21)d 3821137261203R1203Xc.3607C>Tp.(Arg1203*)NSBIC (36) 64611381912343819delGTAAAc.3700_3704delp.(Val1234Glnfs*8)FSBIC (61) 210e, 211e, 313e, 316, 332e, 463, 520, 624, 652e, 65311388912573889delAGc.3770_3771delp.(Glu1257Glyfs*9)FSBIC (23)d N4e, 687, 724, 74311418413554184delTCAAc.4065_4068delp.(Asn1355Lysfs*10)FSBIC (144)d 318e1243021395Q1395Xc.4183C>Tp.(Gln1395*)NSBIC (28)d 408e1344461443R1443Xc.4327C>Tp.(Arg1443*)NSBIC (128) 523e, 555, N18, 598e, 612, 621Intron 14IVS14-1-IVS14-1G>Ac.4485-1G>ASplice siteSPBIC (2)d 220e, 275e, 512e1546271503S1503Xc.4508C>Ap.(Ser1503*)NSBIC (1)d 609e15478415584784delGc.4665delp.(Arg1555Serfs*4)FSNo 611e16498116214981delAc.4862delp.(Asp1621Valfs*12)FSNo 249e, 65817515416795154delCc.5035delp.(Leu1679*)FSBIC (2) 276e, 679Intron 17IVS17+1-IVS17+1G>Ac.5074+1G>ASplice siteSPBIC (3) 68520535817475358delCc.5239delp.(Gln1747Lysfs*18)FSLOVD (2) 73420538517565385dupCc.5266dupp.(Gln1756Profs*74)FSLOVD (376) 706Intron 20IVS20-1-IVS20-1G>Cc.5278-1G>CSplice siteSPLOVD (5)d 67821542917715429dupGc.5310dupp.(Pro1771Alafs*59)FSLOVD (1) 278, 338e22548017875480delTGc.5361_5362delp.(Cys1787Trpfs*42)FSClinVar (3) 6822254961793K1793Xc.5377A>Tp.(Lys1793*)NSClinVar (1) 248eIntron 23IVS23-2-IVS23-2A>Tc.5468-2A>TSplice siteSPClinVar (1) 260, 264, 329e, 377e, 389, 439, 481, 501, 5222456221835R1835Xc.5503C>Tp.(Arg1835*)NSBIC (74)dBRCA1-large genomic rearrangementse 229, 291, 314, 379, 406, 498, 5491-2--del exon 1-2g.41271967_41308900delfLGR(42)g 261, 71921-24--del exon 21-24g.41172653_41205744delfLGRNoBRCA2-small-range mutations 497, 700332031W31Xc.92G>Ap.(Trp31*)NSClinVar (4) N26Intron 4IVS4-2-IVS4-2A>Gc.426-2A>GSplice siteSPClinVar (4) 5459993255993delCACAAc.765_769delp.(Asn255Lysfs*19)FSNo 3301015284341528delAAAAc.1300_1303delp.(Lys434Glufs*25)FSClinVar (2) 6021130489413048delAc.2820delp.(Val941Cysfs*19)FSNo 2061130639453063delAc.2835delp.(Asp946Ilefs*14)FSClinVar (2) 50511408812874088delAc.3860delp.(Asn1287Ilefs*6)FSBIC (2) 222, 407h, 525, 540h11545017415450delGTAAc.5222_5225delp.(Ser1741Thrfs*35)FSBIC (1) 627, 6841159101894Y1894Xc.5682C>Ap.(Tyr1894*)NSBIC (3) 295e11595019085950delCTc.5722_5723delp.(Leu1908Argfs*2)FSBIC (43)d 44711669621566696delTCc.6468_6469delp.(Gln2157Ilefs*18)FSBIC (24)d 548h11704422747044delAAGAGc.6816_6820delp.(Gly2274Alafs*17)FSClinVar (6) 57915780325267803delAc.7575delp.(Ala2526Glnfs*2)FSLOVD (2) 492Intron 17IVS17+2-IVS17+2C>Ac.7976+2C>ASplice siteSPClinVar (1) 71320877328498773delAAc.8545_8546delp.(Lys2849Glyfs*19)FSNo 70220877928608779_8798dup20c.8551_8570dupp.(Lys2860Asnfs*10)FSNo 207h21889728908897insTc.8669dupp.(Thr2891Asnfs*16)FSClinVar (1) 538Intron 21IVS21+4-IVS21+4A>Gc.8754+4A>GSplice siteSPBIC (7)aNumbering starts at the first A of the first coding ATG (located in exon 2) of NCBI GenBank accession number U14680 (BRCA1) and U43746 (BRCA2)bFS frameshift mutation, LGR large genomic rearrangement, MS missense mutation, NS nonsense mutation, SP splice-site mutationcBIC Breast Cancer Information Core database (https://research.nhgri.nih.gov/projects/bic/), LOVD Leiden Open Variation Database (http://databases.lovd.nl/shared/genes/BRCA2); ClinVar (https://www.ncbi.nlm.nih.gov/clinvar/), date last accessed June 26, 2018dPreviously reported in Pakistani breast/ovarian cancer cases [9, 10]eFamilies and mutation description have been previously reported [11, 18]fGenomic locale for chromosome 17, from the UCSC genome browser, Feb 2009 assemblygNot available in databases; reported in various studies [1, 11]hFamilies with male breast cancer
Table 3Characteristics of the 133 families with deleterious BRCA1/2 mutationsFamilyNo. of cancersAge at onset (years)Other cancer(s)c(age at onset in years)EthnicityFemale BC (Bilateral)OC (OC+BC)BCOCFamilies carrying BRCA1-small-range mutations 236a1-22b--Pathan 3161-25b--Punjabi 2641-26b--Punjabi 7061-26b-Uterus (67)Punjabi N121-26b--Punjabi 6241-27b--Punjabi N251-28b--Punjabi 276a1-28b--Punjabi 6101-28b--Punjabi 6781-28b--Punjabi 411a1-29b-Stomach (70)Punjabi 7241-29b-Renal (48), lung (65), throat (65), unknownPunjabi N281-30b--Punjabi 279a1(1)-27/36b--Punjabi 2782-25b,32--Kashmiri 332a2-26b,51-Leukemia (45)Punjabi 6822-28b,40-Uterus (<62,65), throat (<72)Punjabi N182-29b,<50--Punjabi 421a2-30b,33--Punjabi 4322-30b,53-Skin (12), oral (54)Punjabi 5202-30b,47-Uterus (32)Punjabi 4492-32b,55--Punjabi 557a2-32b,45-Unknown (<55), renal (70)Punjabi 7472-33b,38--Unknown 7222-20,34b-Unknown (<18,<40)Punjabi 6872-37b,45--Punjabi 470a2-40b,40-Stomach (46), colon (59), lungPunjabi 510a2-40b,55--Punjabi N13a2-40b,>50--Punjabi 5932-43,44b-Leukemia (22)Pathan 2992(1)-24/27b,55--Punjabi 660a2(1)-25/26b,70-BladderPunjabi 2602(1)-25/26b,28--Punjabi 5112(1)-30/33b,<32-Brain (75)Punjabi N343-24b,<30,31--Punjabi 669a3-25b,<40,<50-Brain (<78), oral (<80)Punjabi 6853-26b,26,?-Blood (2x)Mohajir 7233-28b,40,?--Pathan 6123-29b,<30,40-Throat (45), uterus (48)Punjabi 313a3-30b, 48,?--Punjabi 336a3-23,30b,38-Prostate (29)Punjabi 493a3-35b,55,>55--Pathan 3823-36b,50,?--Punjabi 489a3-25,42b,45--Punjabi 7433-40,44b,62-Bone (60), leukemia (60)Pathan 6583-26,44b,50--Punjabi 377a3-31,50b,85-Thyroid (59), intestine (70), bladder (75), liverPunjabi 550d3-50b,55,>50-Lung, unknownPunjabi 3723(1)-21/21b,29,45-Squamous cell carcinoma scalp (22)bPathan 626a3(1)-35b,36/37, 42--Balochi 3893(1)-22/32,42,48b-Brain (36), uterus (70)Punjabi 247a4-27,28b,40,42-Uterus (31, 55)Siriaki 652a4-31b,33,42,50--Punjabi 3624-31,32,35b,45-Liver (>40), abdomenPathan 399a4-43, 44b,50,?-Abdomen (45), lung (45), prostate (53)Punjabi 338a4-30,40,44,48b-Stomach (73)Punjabi 408a4(1)-24b,31/31,33,50-Abdomen (54), esophagus (74)Punjabi 5214(1)-25/38b,27,33,70-Stomach (60, 65), lung, unknownPunjabi 6535-24,32,35b,37,50-Colon (42), throat (66)Punjabi 7345-37,38b,55,?,?--Punjabi 2967(1)-21b,<30,34,43,44/44,51,52--Punjabi 4398-30,<35,40b,40,40,40,>50,>65-Uterus (40), prostate, unknownPunjabi 249a8(1)-30/31b,37,45,?,?,?,?,?--Punjabi 619a1130b>50-Punjabi 64611(1)34b36b-Punjabi 74811(1)47b52bSkin (45), liver (50)Punjabi 54212(1)40b46b,70Leukemia (74)Punjabi 210a1445b?,?,?,?Brain (32), abdomen, lung, leukemiaPunjabi 241a2129b,6435LymphomaPunjabi 598a2130b,56>50Stomach (>50, >50), tongue (>50)Punjabi 4632135b,5848-Punjabi 48121<25,47b36Lung (55), uterusPunjabi 6212150,>6050bProstate (65)Punjabi 211a21(1)26b,5050-Punjabi 415a21(1)34,35b35bLeukemia (45), unknownPunjabi 67921(1)28b,<3349b-Punjabi N4a21 (1)41b,4545Uterus (38)Punjabi 488a22(1)40b,5542b,45-Punjabi 55522(1)29b,3636b,>50Leukemia (10), vocal cord (45)Punjabi 31723(1)41,4647b,52,55Fallopian tube (47)bPunjabi 318a24(2)40,46b40,42b,44,58Bladder (50, 50)Pathan 4422(1)234,50/50b28,52Leukemia (15)Punjabi 283a2(1)334/38b,5654,55,65-Pathan 254a3127,32b,4341BrainPunjabi 7113140b,?,??Gall bladderSindhi 445a31(1)44b,>60,7374Gall bladderPunjabi 3633232,35,7047b, ?Lung (65), oral (70), liverKashmiri 329a3334b,39,?39,<50,?-Punjabi 609a4(1)129b,31,48/55,6530-Mohajir 328a4129,30b,31,3955-Punjabi 5014134b, 35, >50,??Brain (42)Punjabi 52242(1)30b,<40,45,4535,60Uterus (41)Punjabi 611a42(1)31b,36,37,4250,55Blood (30)Punjabi 523a3(1)439/46,40,53b30,45,51,60-Punjabi 275a4(1)1(1)34/40b,40,42,5040-Punjabi 46951(1)29b,29,<35,>35,>5532b-Mohajir 340a6134b,42,<50,<50,<50,<5054-Balochi 574a6232,32,35,45b,48,>5048,55-Mohajir 512a6(1)1<25,<30,<40,46/55b,<50,>50<50Uterus (<50)Kashmiri 248a72(2)23b,<25,34,<40,<46,<60,?23b,<60-Punjabi 220a81(1)25,27b,<30,53,58,63,77<30-PunjabiFamilies carrying BRCA1-LGRsa 2291-28b--Punjabi 3792-29,31b-Liver (38)Punjabi 2612-33,34b--Punjabi 4062-39b,40-Abdomen (65)Punjabi 4982-40,41b--Siriaki 5492-38,72b-UnknownPunjabi 3146-32b,42,56,70,?,?-Uterus (54), pharynx (59), brain (63), abdomenPunjabi 2913139,42,4848bStomach, brainPunjabi 7193(1)1>40,42b,??-PunjabiFamilies carrying BRCA2-small-range mutations 3301-29b-Lung (48, 58, 66), tongue (55)Punjabi 2061-30b--Pathan 540d1-67b--Mohajir 207d1-76b-Intestine (60)Punjabi 295a1(1)-23/23b-Leukemia (49), esophagus (50)Punjabi N262-26b,35--Pathan 6022-31b,43--Punjabi 4922-38b,39--Mohajir 5052-43,46b--Pathan 7132-35,56b--Kashmiri 7002(1)-35/43b,46-Throat (72)Punjabi 6272(1)-42b,51/51--Pathan 5453-35,36b,47-Brain (50), uterus (50), bone (54)Punjabi 4973-51b,55,50-BrainSiriaki 548d3-45,50,69b--Pathan 7024-26,30b,<33,70-Throat (72)Punjabi 5794-35,49,<50,51b-Oral (35), gall bladder (42)Kashmiri 407d4-31,45(male),45,54b-Esophagus (39,59), leukemia (64)Mohajir 5385-34b,38,45,50,58-Retinoblastoma (3), pancreas (75), liver (83)Pathan 6845-40b,45,48,50,<57-Throat (<48, <82), stomach (53), intestine (60)Pathan 4473(1)231b,<50,55/6550,>50Abdomen (>50), colon (62), brain (65)Punjabi 5254(1)132,33/35b,<50,60<45Bladder (>60)Mohajir 22272(1)35,42,43,50,54b,60,?47,53bLung (60), prostate (70), lungKashmiriBC breast cancer, OC ovarian cancer, Unknown cancer phenotype is not knownaMutations previously described [11, 18]; bProband; cAge at cancer diagnosis is mentioned along with cancer phenotype. For relatives with unknown age at cancer onset, only cancer phenotype is mentioned; dFamilies with male breast cancer

Deleterious BRCA1/2 germline mutations in Pakistani breast/ovarian cancer families

aNumbering starts at the first A of the first coding ATG (located in exon 2) of NCBI GenBank accession number U14680 (BRCA1) and U43746 (BRCA2)

bFS frameshift mutation, LGR large genomic rearrangement, MS missense mutation, NS nonsense mutation, SP splice-site mutation

cBIC Breast Cancer Information Core database (https://research.nhgri.nih.gov/projects/bic/), LOVD Leiden Open Variation Database (http://databases.lovd.nl/shared/genes/BRCA2); ClinVar (https://www.ncbi.nlm.nih.gov/clinvar/), date last accessed June 26, 2018

dPreviously reported in Pakistani breast/ovarian cancer cases [9, 10]

eFamilies and mutation description have been previously reported [11, 18]

fGenomic locale for chromosome 17, from the UCSC genome browser, Feb 2009 assembly

gNot available in databases; reported in various studies [1, 11]

hFamilies with male breast cancer

Characteristics of the 133 families with deleterious BRCA1/2 mutations

BC breast cancer, OC ovarian cancer, Unknown cancer phenotype is not known

aMutations previously described [11, 18]; bProband; cAge at cancer diagnosis is mentioned along with cancer phenotype. For relatives with unknown age at cancer onset, only cancer phenotype is mentioned; dFamilies with male breast cancer

Twenty-one (21/71; 29.6%) mutations including 18 in BRCA1 and three in BRCA2 occurred more than once (Fig. 2a, b). These mutations were identified in 83 unrelated families and accounted for 62.4% (83/133) of all families with mutations. The most common BRCA1 mutation was c.3770_3771del (ten Punjabi families), followed by c.5503C>T (nine Punjabi families), exon 1-2 deletion (seven Punjabi families), c.685del (five Punjabi and two Balochi families), c.4485-1G>A (six Punjabi families), c.2269del (one Punjabi and four Pathan families), c.2405_2406del (five Punjabi families), c.4065_4068del (three Punjabi and one Pathan families), c.1793T>G and c.4508C>A (three Punjabi families each), exon 21-24 deletion, c.2603C>G, c.3339_3341del, c.3598C>T, c.5035del, c.5074+1G>A, and c.5361_5362del (two Punjabi families each) and c.2340_2343del (one Pathan and one Mohajir families). The most common BRCA2 mutation was c.5222_5225del (one Punjabi and three Mohajir families), followed by c.92G>A (two Punjabi families) and c.5682C>A (two Pathan families).
Fig. 2Distribution of deleterious germline mutations identified in Pakistani breast/ovarian cancer families across the BRCA1 and BRCA2 genes. Recurrent mutations are marked with (RM). The distribution of breast cancer (BC) and ovarian cancer (OC) in families according to the position of the mutations in BRCA1 (a) and BRCA2 (b) is shown. Regions inferred to be breast cancer cluster regions (BCCRs) and ovarian cancer cluster regions (OCCRs) according to Rebbeck and colleagues [24] are shown at the bottom

Distribution of deleterious germline mutations identified in Pakistani breast/ovarian cancer families across the BRCA1 and BRCA2 genes. Recurrent mutations are marked with (RM). The distribution of breast cancer (BC) and ovarian cancer (OC) in families according to the position of the mutations in BRCA1 (a) and BRCA2 (b) is shown. Regions inferred to be breast cancer cluster regions (BCCRs) and ovarian cancer cluster regions (OCCRs) according to Rebbeck and colleagues [24] are shown at the bottom

In addition to the deleterious mutations, 153 (28.4%) distinct BRCA1/2 sequence variants were detected: 79 missense variants, 48 non-coding variants, 24 synonymous variants, one in-frame deletion, and one polymorphic nonsense variant in exon 27 of BRCA2 (data not shown).

### BRCA1/2 mutation frequencies

The frequencies of BRCA1/2 mutations by risk group are provided in Table 1. For BRCA1, the highest mutation frequency was noted in families with breast and ovarian cancer (53.8%), followed by families with at least three breast cancer cases (24.8%), families with two breast cancer cases (18.3%), or families with one early-onset breast cancer case (<30 years) (10.8%). For BRCA2, the highest frequency was observed in families with male breast cancer (14.8%).

### Patient and tumors characteristics by BRCA1/2 status

BRCA1 carriers (n=110) were more often identified among female patients (99.1% vs. 94.6%, p=0.039) and belonged to the Punjabi ethnic group (81.8% vs. 68.7%, p=0.030) compared to non-carriers (n=406). In contrast, BRCA2 carriers (n=23) were more common among male patients (17.4% vs. 5.4%, p=0.043) and more often belonged to Pathan ethnic group (34.8% vs. 15.5%, p=0.009).

Female breast cancer patients with mutations in BRCA1 (n=106) or BRCA2 (n=19) had a similar mean age of diagnosis (34.0 years (range 21–72) and 37.7 years (range 23-56), respectively, p=0.073, Wilcoxon rank-sum test), which did not differ to that of non-carriers (n=377) (35.7 years (range 18-78). In contrast, male breast cancer patients harboring BRCA2 mutations (n=4) had an older mean age of diagnosis than non-carriers (n=22) (66.5 years (range 54-76) and 52.5 years (range 27-69) years, respectively, p=0.039, Wilcoxon rank-sum test).

BRCA1-associated breast tumors more often were invasive ductal carcinomas (99.0% vs. 91.4%, p=0.004), triple-negative (60.8% vs. 22.6%, p=<0.0001), and of higher tumor grade (grade 3: 94.9% vs. 63.2%, p=<0.0001) compared to tumors of non-carriers. BRCA2-associated breast tumors more often were PR positive compared to tumors of non-carriers (81.8% vs. 57.2%, p=0.025) (data not shown).

### Discussion

To our knowledge, this is the largest Pakistani study that comprehensively investigated the spectrum of BRCA1/2 small-range mutations and LGRs and prevalence of mutations in 539 high-risk families. Mutations were identified in 24.7% (133/539) of families. Eighteen BRCA1 and three BRCA2 mutations were recurrent and accounted for 68.2% and 34.8% of all mutations in BRCA1 and BRCA2, respectively. Nine mutations were specific to the Pakistani population, whereas other mutations had been reported elsewhere.

The most common type of identified mutations were frameshift mutations (60.6%) followed by nonsense mutations (25.4%). These data are consistent with our previous report [10] and a recent worldwide study [25]. In Pakistani patients, BRCA1 mutations were about 5-times more frequent than BRCA2 mutations. A similar distribution was observed in two Asian studies from South India [26] and Saudi Arabia [27] and most studies among white populations [3–5, 28]. This is likely due to the predominance of recurrent BRCA1 mutations in these populations. Contradictory results were reported in other Asian studies from China, Hong Kong, Korea, and Indonesia, where BRCA2 mutations were observed at an equal or a higher frequency than BRCA1 mutations [6, 12, 15–17].

Among the 133 mutations identified in our study, 18 BRCA1 and three BRCA2 mutations were recurrent, accounting for 68.2% and 34.8% of all mutations in BRCA1 and BRCA2, respectively. The proportion of recurrent BRCA1 mutations to the total number of identified BRCA1 mutations is higher than our previous report [10], which is likely due to the larger size of the present study. Of the identified recurrent mutations, the majority was also reported as recurrent mutations in other populations [1, 4, 25], while few were exclusively identified in a specific ethnic group of Pakistan. Fourteen BRCA1 mutations (c.3770_3771del, c.5503C>T, c.4485-1G>A, c.2405_2406del, c.1793T>G, c.4508C>A, c.2603C>G, c.3339_3341del, c.3598C>T, c.5035del, c.5074+1G>A, c.5361_5362del, exon 1-2 deletion, and exon 21-24 deletion) and one BRCA2 mutation (c.92G>A) were identified only in the Punjabi ethnic group. One BRCA2 mutation (c.5682C>A) was found only in the Pathan ethnic group. Five other recurrent mutations were identified in more than one ethnic group. Our findings imply that a panel of ethnic specific recurrent mutations may be useful for initial screening of high-risk patients from these ethnic groups. Founder effects were previously suggested for six of the 18 recurrent BRCA1 mutations (c.3770_3771del, c.4065_4068del, c.4485-1G>A, c.4508C>A, c.5503C>T, exon 1-2 deletion) [9–11], while haplotype analyses of the remaining recurrent mutations have not been performed so far. The high percentage of recurrent BRCA1 mutations facilitates the development of a local, economical, and efficient ethnic-specific genetic testing strategy in Pakistan.

BRCA1/2 mutations were identified in 24.7% of Pakistani breast cancer families. This frequency is higher than that from our initial report (17%) [10], probably due to the larger study size and comprehensive mutation analyses of both genes. This frequency is also higher than those from other Asian reports from Hong Kong, Malaysia, and Korea, ranging from 9.4% to 21.7% [6–8, 16, 17]. These findings further support the notion that the BRCA1/2 mutation frequencies vary among different populations. Our data are similar to those reported in white populations, ranging from 17.6% to 29.8% [1, 2, 4]. We found the highest mutation frequency in breast and ovarian cancer families (55.4%), in agreement with previous studies from Pakistan [10], Korea [16], and studies in white populations [4, 28]. We observed a 2.52 fold (53.8% vs. 21.3%) increased occurrence of BRCA1 mutations in breast and ovarian cancer families compared to breast cancer only families, in line with previous reports [1, 4, 6, 28]. Our findings support the notion that the presence of ovarian cancer in Pakistani breast cancer families increases the likelihood for the occurrence of BRCA1 mutation.

In the present study on 27 families with male breast cancer, a BRCA1/2 mutation frequency of approximately 19% was observed, with BRCA2 mutations being about 4-times more common than BRCA1 mutations. Our data are in line with previous studies [4, 14]. This observed frequency is higher than that reported in our initial much smaller study, in which no mutations were identified [10]. In agreement with the National Comprehensive Cancer Network (NCCN) guidelines, our data also warrant BRCA1/2 testing in families with male breast cancer (NCCN Guidelines Version 2.2019).

The main strength of this study is its large size of 539 high-risk families, the comprehensive screening of both genes for small-range mutations and LGRs using highly sensitive methods (allowing the identification of recurrent BRCA1/2 mutations in the Pakistani population and the more accurate estimation of BRCA1/2 mutation frequencies among high-risk families), and the confirmations of mutations in an independent patient’s sample. However, our study also has some limitations. Participants were recruited at one tertiary care cancer center in Lahore, which may have introduced selection bias. Families belonging to Punjabi and Pathan ethnic groups are over-represented and, therefore, mutations in these groups may be over-represented. Nevertheless, Punjabi (44.7%) and Pathan (15.4%) are the most common ethnic groups reported in Pakistan (The World Factbook). Further, our data are based on self-reported ethnicity of study participants, which may lead to a misclassification of the ethnic origin of some of them.

### Conclusions

In summary, our study showed that BRCA1/2 mutations account for 24.7% of high-risk breast cancer patients in Pakistan. Our results have important clinical implications, such as personalized treatment with platinum-based or PARP-inhibitor therapy for breast/ovarian cancer patients carrying a pathogenic BRCA1/2 mutation and early detection, surgical prevention, and chemoprevention strategies for their unaffected BRCA1/2 mutation positive relatives. Overall, BRCA1/2 mutations account for one in four patients with a family history of breast cancer/breast and ovarian cancer, one in five patients with male breast cancer, and one in eight patients with early-onset breast cancer. Eighteen mutations in BRCA1 and three in BRCA2 were recurrent and accounted for 68.2% and 34.8% of all identified mutations in BRCA1 and BRCA2, respectively. Our data suggest that BRCA1 testing may be justified for families with multiple female breast cancers, breast and ovarian cancer or early-onset breast cancer and BRCA2 testing for families with male breast cancer from Pakistan. Our findings will help in tailoring cost-effective genetic testing approach for the high-risk Pakistani population or for individuals of Pakistani origin residing in other countries.



# SUPPLEMENTAL FILE 1: 13053_2019_Article_125.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)