# MAIN TEXT

## Circulating DNA as prognostic biomarker in patients with advanced hepatocellular carcinoma: a translational exploratory study from the SORAMIC trial

### Abstract

BackgroundLiquid biopsy based on cell-free DNA circulating in plasma has shown solid results as a non-invasive biomarker. In the present study we evaluated the utility of circulating free DNA (cfDNA) and the sub-type tumor DNA (ctDNA) in hepatocellular cancer (HCC) patients to assess therapy response and clinical outcome.MethodsA cohort of 13 patients recruited in the context of the SORAMIC trial with unresectable, advanced HCC and different etiological and clinicopathological characteristics was included in this exploratory study. Plasma samples were collected between liver micro-intervention and beginning of sorafenib-based systemic therapy and then in correspondence of three additional follow-ups. DNA was isolated from plasma and next generation sequencing (NGS) was performed on a panel of 597 selected cancer-relevant genes.ResultscfDNA levels showed a significant correlation with the presence of metastases and survival. In addition cfDNA kinetic over time revealed a trend with the clinical history of the patients, supporting its use as a biomarker to monitor therapy. NGS-based analysis on ctDNA identified 28 variants, detectable in different combinations at the different time points. Among the variants, HNF1A, BAX and CYP2B6 genes showed the highest mutation frequency and a significant association with the patients’ clinicopathological characteristics, suggesting a possible role as driver genes in this specific clinical setting.ConclusionsTaken together, the results support the prognostic value of cfDNA/ctDNA in advanced HCC patients with the potential to predict therapy response. These findings support the clinical utility of liquid biopsy in advanced HCC improving individualized therapy and possible earlier identification of treatment responders.

### Background

Hepatocellular carcinoma (HCC) is the sixth most common cancer and the second leading cause of cancer-related death globally, with a median survival of only 7 months, if untreated [1–3]. Although cirrhosis, generally resulting from chronic inflammation and oxidative stress [4], is recognized as the most frequent risk factor, patients are often diagnosed with HCC in an already advanced phase, when treatment is limited and resection or transplantation are not any longer an option. This is in part due to the limited sensitivity and specificity of the standard diagnostic modalities of imaging and biomarkers, such as serum alpha-fetoprotein (AFP) [5], in the detection of small tumors during surveillance of high risk populations. Advanced HCC often shows a poor outcome, with very limited benefit from cytotoxic agents. Targeted therapy with the multi-tyrosine kinase inhibitor sorafenib has been the mainstay of systemic treatment, nevertheless resulting in modest improvements in overall survival over placebo and often only in subgroups of patients [6]. Identification of early predictors of therapy response would be highly desirable in order to move quickly patients to more effective treatments.

Circulating cell-free DNA (cfDNA) is detectable in the plasma and serum of healthy individuals as consequence of cellular necrosis or apoptosis. Cancer patients show a higher amount of cfDNA since tumor cells divide faster than normal cells, and cfDNA is released in higher proportion [7–9]. The fraction of cfDNA derived from the tumor is indicated as circulating tumor DNA (ctDNA) [10, 11]. In the last years, analysis of both cfDNA and ctDNA has gained considerable attention as novel blood biomarkers, since quantification and kinetic analysis of cfDNA [12, 13] and molecular profiling of ctDNA [14] have demonstrated both a predictive and prognostic value. Venipuncture, with capacity for serial sampling over time, may offer advantages over standard biopsy, avoiding complications from incisions and reflecting the genetic heterogeneity of the whole tumor. In addition, due to the short half-life of circulating DNA [15], genetic analysis may offer “real time” insights into the kinetic mutations arising during therapy. The prognostic value of cfDNA/ctDNA in early HCC has been previously shown [16, 17]. The aim of the present study was to evaluate if cfDNA quantification over time may be a valid monitoring strategy to predict clinical outcome in advanced HCC patients in the context of the SORAMIC trial [18, 19], and whether molecular profiling of ctDNA may track the genetic evolution of the tumor during the combined systemic and liver-directed micro-interventional treatment.

### Methods

A total of 13 male patients (median age (years ± SD): 68 ± 8.91, range 46–82) from the prospective randomized multicenter phase II SORAMIC trial (EudraCT 2009-012576-27, NCT01126645) were included in this exploratory study [18, 19]. Participants were recruited between 2011 and 2016 at different European study centers. Advanced, unresectable HCC patients received radioembolization with Yttrium-90 (90Y-RE) in combination with systemic treatment with sorafenib (n = 10) or radiofrequency ablation in combination with sorafenib (n = 2) or placebo (n = 1). Treatment efficacy was evaluated based on imaging and alfa-fetoprotein (AFP) [20, 21]. The median overall survival was 23.8 months (range 11.9 – 41.4) and only two patients showed recurrence with a progression free survival of 22.3 and 26.8 months. Blood samples were collected at four different time points: after micro-interventional therapy and before starting the systemic therapy with sorafenib (T1), approximately 8 weeks after beginning of systemic therapy (T2, median numbers of days between T1 and T2: 112), at first follow up (FU) (T3, median numbers of days between T1 and T3: 224) and at second FU (T4, median numbers of days between T1 and T4: 297). The SORAMIC trial design is given in Fig. 1, where the time points relative to blood collection are indicated. The study was approved by all the involved ethical boards and conducted in accordance with the Declaration of Helsinki [22]. Before entering the study, all patients gave their written informed consent.Fig. 1SORAMIC trial design. Indicated in red are the time points chosen for discovery of the variants. SNV, single nucleotide variant; InDel, insertion and deletion; R, randomization; TX, therapy; RFA, radiofrequency ablation; 90Y-RE, 90Y-radioembolization

SORAMIC trial design. Indicated in red are the time points chosen for discovery of the variants. SNV, single nucleotide variant; InDel, insertion and deletion; R, randomization; TX, therapy; RFA, radiofrequency ablation; 90Y-RE, 90Y-radioembolization

Peripheral blood (5 ml) was drawn in EDTA tubes (Becton–Dickinson, Heidelberg, Germany) and processed immediately (centrifugation 3000 rpm, 5 min, 4 °C) to collect plasma and buffy coats, which were aliquoted and stored at − 80 °C until further use. Plasma was used for extraction of cell-free DNA (cfDNA) and buffy coats were used for extraction of genomic DNA (gDNA). DNA extraction, quality control and NGS were performed at Eurofins Genomics GmbH (Konstanz, Germany). The quality and quantity of the extracted DNA was assessed via electropherograms and fluorometer concentration determinations (for details, see Additional file 1: Figure S1). The extracted DNA was used for library preparation and subsequent sequencing. A summary of the depth of coverage is shown in Additional file 2: Table S1.

The genes were selected according to their importance and in accordance with clinical guidelines such as the European Society for Medical Oncology (ESMO), the National Comprehensive Cancer Network (NCCN), and the College of American Pathologists (CAP).

NGS was performed on Illumina platform using the 150 paired-end mode. The sequencing reads were quality checked and mapped to the human genome using BWA with default parameters [23]. After target processing and alignment refinement, variants (single nucleotide variants, SNV; Insertions and deletions, InDel; and gene fusions according to Chimer DB 2.0 [24]) were discovered and annotated. To distinguish somatic from germline mutations, genomic DNA extracted from buffy coats was analyzed in parallel (Additional file 3: Table S2). The whole experimental workflow is schematically described in Fig. 2.Fig. 2Schematic description of the experimental workflow. Peripheral blood was collected and processed to separate plasma from buffy coat containing white blood cells (WBC). Whole circulating DNA was extracted and used for library preparation. Next generation sequencing was performed on Illumina platform using the 150 paired-end (PE) mode. To distinguish somatic from germline mutations, genomic DNA extracted from WBC was analyzed in parallel

Schematic description of the experimental workflow. Peripheral blood was collected and processed to separate plasma from buffy coat containing white blood cells (WBC). Whole circulating DNA was extracted and used for library preparation. Next generation sequencing was performed on Illumina platform using the 150 paired-end (PE) mode. To distinguish somatic from germline mutations, genomic DNA extracted from WBC was analyzed in parallel

All statistical analyses were performed using IBM SPSS Statistics 21.0.0 (IBM Corporation, New York, N.Y.). Results for numerical data are given as median together with minimum and maximum of the sample (i.e. range). The correlation between tumor size and cfDNA concentrations at different time points were analyzed using the Pearson’s correlation coefficient and corresponding significance test. cfDNA at different time points were compared between patient groups featuring different clinicopathological characteristics using the non-parametric Mann–Whitney U test. The association between variants at different time points and patients’ clinical characteristics was evaluated using the Fisher’s exact test. Overall survival (OS) was analyzed using the Kaplan–Meier method, and survival estimates in different groups were compared using the log-rank test. Patients were clustered in two groups corresponding to high and low cfDNA concentration (high cfDNA = over the median, low cfDNA = below the median). OS was calculated from the date of primary tumor diagnosis to the date of death or the date of the last clinical follow-up. All tests were carried out two-sided. Due to the low sample size of n = 13 patients, no alpha adjustment was made. All statistical tests are interpreted at a significance level of alpha = 5% and according results are considered exploratory.

### Patients and therapy

A total of 13 male patients (median age (years ± SD): 68 ± 8.91, range 46–82) from the prospective randomized multicenter phase II SORAMIC trial (EudraCT 2009-012576-27, NCT01126645) were included in this exploratory study [18, 19]. Participants were recruited between 2011 and 2016 at different European study centers. Advanced, unresectable HCC patients received radioembolization with Yttrium-90 (90Y-RE) in combination with systemic treatment with sorafenib (n = 10) or radiofrequency ablation in combination with sorafenib (n = 2) or placebo (n = 1). Treatment efficacy was evaluated based on imaging and alfa-fetoprotein (AFP) [20, 21]. The median overall survival was 23.8 months (range 11.9 – 41.4) and only two patients showed recurrence with a progression free survival of 22.3 and 26.8 months. Blood samples were collected at four different time points: after micro-interventional therapy and before starting the systemic therapy with sorafenib (T1), approximately 8 weeks after beginning of systemic therapy (T2, median numbers of days between T1 and T2: 112), at first follow up (FU) (T3, median numbers of days between T1 and T3: 224) and at second FU (T4, median numbers of days between T1 and T4: 297). The SORAMIC trial design is given in Fig. 1, where the time points relative to blood collection are indicated. The study was approved by all the involved ethical boards and conducted in accordance with the Declaration of Helsinki [22]. Before entering the study, all patients gave their written informed consent.Fig. 1SORAMIC trial design. Indicated in red are the time points chosen for discovery of the variants. SNV, single nucleotide variant; InDel, insertion and deletion; R, randomization; TX, therapy; RFA, radiofrequency ablation; 90Y-RE, 90Y-radioembolization

SORAMIC trial design. Indicated in red are the time points chosen for discovery of the variants. SNV, single nucleotide variant; InDel, insertion and deletion; R, randomization; TX, therapy; RFA, radiofrequency ablation; 90Y-RE, 90Y-radioembolization

### DNA isolation, library preparation and next-generation sequencing (NGS)

Peripheral blood (5 ml) was drawn in EDTA tubes (Becton–Dickinson, Heidelberg, Germany) and processed immediately (centrifugation 3000 rpm, 5 min, 4 °C) to collect plasma and buffy coats, which were aliquoted and stored at − 80 °C until further use. Plasma was used for extraction of cell-free DNA (cfDNA) and buffy coats were used for extraction of genomic DNA (gDNA). DNA extraction, quality control and NGS were performed at Eurofins Genomics GmbH (Konstanz, Germany). The quality and quantity of the extracted DNA was assessed via electropherograms and fluorometer concentration determinations (for details, see Additional file 1: Figure S1). The extracted DNA was used for library preparation and subsequent sequencing. A summary of the depth of coverage is shown in Additional file 2: Table S1.

The genes were selected according to their importance and in accordance with clinical guidelines such as the European Society for Medical Oncology (ESMO), the National Comprehensive Cancer Network (NCCN), and the College of American Pathologists (CAP).

NGS was performed on Illumina platform using the 150 paired-end mode. The sequencing reads were quality checked and mapped to the human genome using BWA with default parameters [23]. After target processing and alignment refinement, variants (single nucleotide variants, SNV; Insertions and deletions, InDel; and gene fusions according to Chimer DB 2.0 [24]) were discovered and annotated. To distinguish somatic from germline mutations, genomic DNA extracted from buffy coats was analyzed in parallel (Additional file 3: Table S2). The whole experimental workflow is schematically described in Fig. 2.Fig. 2Schematic description of the experimental workflow. Peripheral blood was collected and processed to separate plasma from buffy coat containing white blood cells (WBC). Whole circulating DNA was extracted and used for library preparation. Next generation sequencing was performed on Illumina platform using the 150 paired-end (PE) mode. To distinguish somatic from germline mutations, genomic DNA extracted from WBC was analyzed in parallel

Schematic description of the experimental workflow. Peripheral blood was collected and processed to separate plasma from buffy coat containing white blood cells (WBC). Whole circulating DNA was extracted and used for library preparation. Next generation sequencing was performed on Illumina platform using the 150 paired-end (PE) mode. To distinguish somatic from germline mutations, genomic DNA extracted from WBC was analyzed in parallel

### Statistical analysis

All statistical analyses were performed using IBM SPSS Statistics 21.0.0 (IBM Corporation, New York, N.Y.). Results for numerical data are given as median together with minimum and maximum of the sample (i.e. range). The correlation between tumor size and cfDNA concentrations at different time points were analyzed using the Pearson’s correlation coefficient and corresponding significance test. cfDNA at different time points were compared between patient groups featuring different clinicopathological characteristics using the non-parametric Mann–Whitney U test. The association between variants at different time points and patients’ clinical characteristics was evaluated using the Fisher’s exact test. Overall survival (OS) was analyzed using the Kaplan–Meier method, and survival estimates in different groups were compared using the log-rank test. Patients were clustered in two groups corresponding to high and low cfDNA concentration (high cfDNA = over the median, low cfDNA = below the median). OS was calculated from the date of primary tumor diagnosis to the date of death or the date of the last clinical follow-up. All tests were carried out two-sided. Due to the low sample size of n = 13 patients, no alpha adjustment was made. All statistical tests are interpreted at a significance level of alpha = 5% and according results are considered exploratory.

### Results

Within the cohort chosen for this study, patients were divided in two subgroups: patients with an age below 65, who did not present with liver cirrhosis at time of recruitment, did not report any past alcohol abuse and, based on the SORAMIC trial [19] results, responded better to therapies; and patients over 65, who presented with liver cirrhosis at time of recruitment, reported past alcohol abuse and responded worse to therapies. None of the patients were positive for Hepatitis B, only 1 was positive for Hepatitis C, and 4 presented non-alcoholic steatohepatitis (NASH). Three patients (33.3%, for n = 4 the number of lesions was not available) presented with diffuse disease (> 20 lesions), while among the others the median number of lesions was 2 (range 0–28) and the median size of the lesions was 49 mm (range 8–160 mm). Three patients (23.0%) showed portal vein infiltration, all of them presented with a Child–Pugh A liver function, 46.1% (n = 6) of the patients were Barcelona Clinic Liver Cancer Staging (BCLC) C, while the remaining were A (n = 3, 23.0%) or B (n = 4, 30.7%). Four patients (30.7%) presented lymph node metastases and one of them (7.7%) also bone metastases. Two patients (66.6%) showed recurrence at 2 years FU. A summary of the clinicopathological characteristics of the patients is given in Table 1.Table 1Clinicopathological characteristics of the patientsIDAgeLiver cirrhosisHBVHCVAlcohol abuseOtherNumber of lesions (> 20 = DD)Max. diameter largest lesion (mm)PVICPCP pointsBCLC stageTherapyMRdeathOS (months)A46NoNegNegNoNash2834NoA5CRESRFBNoNo040.7B60NoNegNegNo–99160NoA6CRESRFBYesNo138.6C61NoNegNegNoNash998NoA5BRESRFBNoNo023.4D61NoNegNegNoNashna88NoA5BRESRFBNoNo016.1E59NoNegposNo–na55YesA5CRESRFBYesNo111.7F68YesNegNegYesNash391YesA6CRESRFBYesNo126.8G75YesNegNegYes–nanaNoA6BRESRFBNoNo113.7H68YesNegNegYes–315YesA5CRESRFBNoNo116.8I66YesNegNegYes–na65NoA5CRESRFBYesNo113.0J69YesNegNegYes–275NoA5BRESRFBNoNo117.5K70YesNegNegYes–143NoA5ARFAPLCNoYes127.1L82YesNegNegYes–140NoA6ARFASRFBNoNo126.3M73YesNegNegYes–133NoA5ARFASRFBNoYes132.5HBV, hepatitis B virus; HCV, hepatitis C virus; DD, diffuse disease; Nash, non-alcoholic steatohepatitis; PVI, portal vein infiltration; CP, Child–Pugh, BCLC, Barcelona Clinic Liver Cancer; RE, 90Y-Radioembolization; SRFB, sorafenib; PLCB, placebo; M, metastases; R, recurrence; OS, overall survival, in months; na, not available

Clinicopathological characteristics of the patients

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RFA

PLC

RFA

SRFB

RFA

SRFB

HBV, hepatitis B virus; HCV, hepatitis C virus; DD, diffuse disease; Nash, non-alcoholic steatohepatitis; PVI, portal vein infiltration; CP, Child–Pugh, BCLC, Barcelona Clinic Liver Cancer; RE, 90Y-Radioembolization; SRFB, sorafenib; PLCB, placebo; M, metastases; R, recurrence; OS, overall survival, in months; na, not available

cfDNA was isolated from plasma collected at the 4 time points (T1, T2, T3, and T4) during the SORAMIC trial. Nine of the 52 cfDNA samples (17.3%) were excluded from the subsequent analysis due to high genomic DNA contamination or because the concentration was below the threshold (Additional file 1: Figure S1). In the remaining samples, the amount of cfDNA at the different time points was ranging between 2.04 ng/ml to 160.75 ng/ml with a median of 10.2 ng/ml at T1, 15.3 ng/ml at T2, 9.3 ng/ml at T3 and 13.3 ng/ml at T4 (Table 2). We did not observe a significant correlation between cfDNA concentration at T1 and tumor volume or between cfDNA concentrations at any time points and patient’s clinical characteristics (all p > 0.1) (data not shown). On the contrary, we did observe a difference between the concentrations of cfDNA in the presence or absence of metastases, which was significant (p = 0.012) or borderline significant (p = 0.073) at T1 and T2, respectively (Fig. 3). The relative amount of cfDNA at the different time points was also evaluated with respect to overall survival (OS). Patients were grouped according to high or low cfDNA levels with respect to the median values at the different time points. We found a borderline significance at the latest time points (T3, p = 0.057; T4, p = 0.095), suggesting that patients keeping high levels of plasmatic cfDNA while receiving systemic therapy might have a worse outcome (Fig. 4). On the contrary we did not find any association between cfDNA levels at earlier time points and OS (all p > 0.1). The fact the we found a borderline significance might be due to the low number of patients included in the analysis, therefore it will be necessary to validate these preliminary results with a bigger cohort of patients.Table 2Amount of cfDNA (ng/ml) and AFP (ng/ml) in plasma samples at the different time pointsPatientsT1T2T3T4Median [cfDNA]cfDNAcfDNAcfDNAcfDNAAFPAFPAFPAFPAna39.448.3910.1110.185.0011.0013.006.00B12.7817.68na15.3015.3373.0075.0033.0045.00C4.357.342.723.403.92.003.003.003.00D2.8114.964.189.076.64.003.009.0016.00E30.6315.3017.2038.6223.93.003.0010.0042.10F28.3357.1230.6014.7329.52.004.00na4.00G10.2011.9011.5620.9411.74.006.008.004.00H1.815.649.3811.027.513.0019.0030.0072.00I30.3724.287.4113.3318.81681.001685.00991.00967.00J7.96na12.60160.7512.66154.002410.008732.003855.00K2.044.427.289.255.84.003.003.003.00L20.2616.6816.6816.6816.87.005.006.007.00Mnanana8.988.989.004.004.006.00Median [cfDNA]10.215.39.313.3cfDNA median values are given for each patient at the four time points (last column) or for each time point for all patients (last row)na, not available
Fig. 3cfDNA concentrations at different time points and presence of metastases. The analysis shows that patients presenting metastases when recruited in the trial, had still a significant higher amount of plasmatic cfDNA in the timeframe between micro-interventional therapy and the beginning of sorafenib-based systemic therapy (T1, p = 0.012). Borderline significance (p = 0.073) was found after the beginning of systemic therapy (T2), while no significant difference was found at the two later time points. Comparison was performed using the Mann–Whitney U test (* ≤ 0.05)
Fig. 4Survival plots for HCC patients grouped according to the amount of cfDNA at the different time points. Patients’ total population was grouped according to the corresponding cfDNA concentration with respect to the median value found at each time point. Higher cfDNA concentration at later time points (T3 and T4) was showing a trend with a shorter OS (p = 0.057 and p = 0.095) while no association (p > 0.1) was found and earlier time points (T1 and T2). High and low cfDNA levels were defined as being above or below the median values at the different time points. OS was analyzed using the Kaplan–Meier method and survival estimates in the different groups were compared using the log-rank test

Amount of cfDNA (ng/ml) and AFP (ng/ml) in plasma samples at the different time points

cfDNA median values are given for each patient at the four time points (last column) or for each time point for all patients (last row)

na, not available

cfDNA concentrations at different time points and presence of metastases. The analysis shows that patients presenting metastases when recruited in the trial, had still a significant higher amount of plasmatic cfDNA in the timeframe between micro-interventional therapy and the beginning of sorafenib-based systemic therapy (T1, p = 0.012). Borderline significance (p = 0.073) was found after the beginning of systemic therapy (T2), while no significant difference was found at the two later time points. Comparison was performed using the Mann–Whitney U test (* ≤ 0.05)

Survival plots for HCC patients grouped according to the amount of cfDNA at the different time points. Patients’ total population was grouped according to the corresponding cfDNA concentration with respect to the median value found at each time point. Higher cfDNA concentration at later time points (T3 and T4) was showing a trend with a shorter OS (p = 0.057 and p = 0.095) while no association (p > 0.1) was found and earlier time points (T1 and T2). High and low cfDNA levels were defined as being above or below the median values at the different time points. OS was analyzed using the Kaplan–Meier method and survival estimates in the different groups were compared using the log-rank test

AFP is detectable in as many as 30% of HCC patients at time of diagnosis and usually remains low during the course of the disease, even with advanced HCC [20]. AFP > 400–500 ng/ml supports diagnosis of HCC [21]. At the considered time points, only 2 (15.4%) of the patients showed high AFP values (> 400 ng/ml), 4 (30.8%) showed values between 20 and 400 ng/ml and 7 (53.8%) showed values between 2 and 20 ng/ml (Table 2). It has been already shown that cfDNA has clinical diagnostic relevance, reflecting therapy response [25]. We report here in more detail three vignettes of patients with evidence of disease progression, for which cfDNA offered earlier monitoring value than conventional biomarkers such as AFP (Fig. 5). A concentration of 400 ng/ml was chosen as cut-off value. The first patient (patient B) entered in the trial with a diffuse disease in the liver and lymph nodes and bone metastases; he died after 38.6 months from enrolment in the SORAMIC trial. AFP concentration was high (373 ng/ml) before starting sorafenib treatment; however it decreased very rapidly below 50 ng/ml and remained low while assuming sorafenib, suggesting efficacy of treatment. On the contrary, we did not find a corresponding reduction in the cfDNA concentration which instead increased during time, suggesting the presence of progressive disease, as confirmed by CT imaging at follow up (Fig. 5a). The second patient (patient E) entered in the study with lymph node metastases, portal vein infiltration and disease progression on treatment with an OS of 11.7 months from the beginning of therapy. At time points considered, AFP values were measured always below 50 ng/ml; in contrast, cfDNA level, while showing a drastic decrease immediately after the beginning of the therapy, increased rapidly between 6 and 8 months from the beginning of the therapy, reflecting the rapid progression of the disease and indicating a change in the response to sorafenib (Fig. 5b). The third patient (patient K) entered in the study presenting only one hepatic lesion and no evidence of metastases. AFP was always measured below 5 ng/ml at all time points, however cfDNA showed a steadily increase, predicting the progression of the disease. The patient finally relapsed and died shortly after (Fig. 5c). In conclusion, our data, in line with other reports, showed improved diagnostic sensitivity of cfDNA over AFP [24, 26], suggesting also in advanced HCC cfDNA as a biomarker for monitoring therapy efficacy.Fig. 5Comparison between cfDNA and AFP levels in three cases corresponding to patients showing different clinical characteristics. Panels a, b and c represent the described clinical cases of patients B, E and K, respectively

Comparison between cfDNA and AFP levels in three cases corresponding to patients showing different clinical characteristics. Panels a, b and c represent the described clinical cases of patients B, E and K, respectively

Genomic profiling of ctDNA was undertaken at the different time points, with the screening of 597 cancer-relevant genes. Only those mutations proven to have clinical impact according to the ClinVar database (National Center for Biotechnology Information-NCBI) were further analyzed. In total 28 variants, including single nucleotide variations (SNV) or insertions and deletions (InDel) were discovered in our dataset, while no gene fusion was identified (Fig. 6). All patients showed at least one variant (median = 6 variants per patient, range 3–16), however patterns varied between patients. Variants were found in genes involved in DNA repair, multidrug resistance, cell cycle control, signal transduction, transcription control, chromatin structure, apoptosis and DNA methylation (Table 3). We detected somatic mutations in recognized HCC driver genes such as CTNNB1 and TP53, however at very low frequency. Over the 28 variants, three genes were showing the highest mutation allele frequency (MAF): HNF1A (n = 12, 92.3%), BAX (n = 9, 69.2%) and CYB2B6 (n = 6, 46.1%). We observed that in a high percentage of genes (n = 19, 68%) variants were found mainly after the beginning of sorafenib treatment, suggesting a possible clonal selection induced by the therapy itself. Nevertheless, a different kinetic was also observed. For example, in the case of HNF1A, 50% of the patients showed the variant already before starting to assume sorafenib. At T2, only one patient was still positive, suggesting a good therapy response. However at T3 the variant was showing a MAF of 82% (n = 9), a finding which could suggest a clonal expansion of cells not responsive any longer to sorafenib.Fig. 6Heatmaps showing the variants discovered in the two subgroups of patients (A–E: below 65, no liver cirrhosis, no past alcohol abuse and better response to therapies; F–M: over 65, with liver cirrhosis, past alcohol abuse and worse response to therapies) at the different time points. In (a) and in (b) SNV and InDel with relative mutation frequencies are reported, respectively
Table 3List of variants grouped according to gene functionGene groupLocationAACodonPatients (%)DNA repair ATMchr11:108121426–c.1236-2A>T2 (15.4) BLMchr15:91304138p.N515fsnac.1544delAc.*468delA1 (7.7) BRCA2chr13:32954022p.T18fsp.T3033fsc.52insAc.9097insA1 (7.7) ERCC5chr13:103524611p.K150fsp.K917fsc.450delAc.2751delA5 (38.4) MSH6chr2:48030639p.F56fsp.F1088fs–p.F958fsp.F786fsc.165delCc.3261delCc.*2608delCc.2871delCc.2355delC6 (46.1) RAD50chr5:131931451p.K722fs––p.?661fsc.2165delAc.*1791delAc.*351delAc.1982delA4 (30.8) WRNchr8:30915970p.K5fsc.15delA2 (15.4)Multidrug resistance/xenobiotic metabolism ABCB1chr7:87160618p.S893Ap.S829Ac.2677T>Gc.2485T>G3 (23.0) CYP2B6chr19:41515263p.K262Rc.785A>G6 (46.1) CYP2C9chr10:96702047p.R144Cc.430C>T1 (7.7) CYP2D6chr22:42524994p.P34Sp.P12Sc.100C>Tc.34C>T2 (15.4) GSTP1chr11:67352689–p.I105Vc.*137A>Gc.313A>G2 (15.4) SLCO1B1chr12:21331549p.V174Ac.521T>C2 (15.4)Cell cycle control/proliferation CTNNB1chr3:41266098p.D32Vp.D25Vc.95A>Tc.74A>T1 (7.75) FGFR2chr10:123274794p.Y261Cp.Y147Cp.Y287Cp.Y263Cp.Y286Cp.Y375Cp.Y376Cp.Y377Cp.Y260C–c.782A>Gc.440A>Gc.860A>Gc.788A>Gc.857A>Gc.1124A>Gc.1127A>Gc.1130A>Gc.779A>Gc.*171A>G1 (7.75) FGFR4chr5:176520243p.G23Rp.G388Rc.67G>Ac.1162G>A1 (7.75) TP53chr17:7579447p.G254Dp.G113Dp.G206Dp.G152Dp.G86Dp.G234Dc.374G>Ac.338G>Ac.617G>Ac.455G>Ac.257G>Ac.701G>A1 (7.75) TP53*chr17:7579407p.A86fsp.A47fsc.257del279c.140del1621 (7.75)Signal transduction signalling PIK3CAchr3:178952085p.H1047Rc.3140A>G1 (7.75) PIK3CA*chr3:178952085p.H1047Lc.3140A>T1 (7.75) SOS1chr2:39249927p.S548Rc.1642A>C1 (7.75)Chromatin structure ASXL1chr20:31022441p.G641fsp.G646fsc.1919insGc.1934insG3 (23.0)Liver transcription factor HNF1Achr12:121432114p.P291fsp.G226fs–c.864delGc.677delGc.*304delG12 (92.3)Apoptosis BAXchr19:49458970p.E41fsp.R24fsp.E24fsc.121insGc.69insGc.70insG9 (69.2)DNA methylation DNMT3Achr2:25457242–p.R693Hp.R659Hp.R882Hc.*498G>Ac.2078G>Ac.1976G>Ac.2645G>A2 (15.4)Chromosome location, amino acid (AA) change and codon change are indicated(na, not available)

Heatmaps showing the variants discovered in the two subgroups of patients (A–E: below 65, no liver cirrhosis, no past alcohol abuse and better response to therapies; F–M: over 65, with liver cirrhosis, past alcohol abuse and worse response to therapies) at the different time points. In (a) and in (b) SNV and InDel with relative mutation frequencies are reported, respectively

List of variants grouped according to gene function

p.N515fs

na

c.1544delA

c.*468delA

p.T18fs

p.T3033fs

c.52insA

c.9097insA

p.K150fs

p.K917fs

c.450delA

c.2751delA

p.F56fs

p.F1088fs

–

p.F958fs

p.F786fs

c.165delC

c.3261delC

c.*2608delC

c.2871delC

c.2355delC

p.K722fs

–

–

p.?661fs

c.2165delA

c.*1791delA

c.*351delA

c.1982delA

p.S893A

p.S829A

c.2677T>G

c.2485T>G

p.P34S

p.P12S

c.100C>T

c.34C>T

–

p.I105V

c.*137A>G

c.313A>G

p.D32V

p.D25V

c.95A>T

c.74A>T

p.Y261C

p.Y147C

p.Y287C

p.Y263C

p.Y286C

p.Y375C

p.Y376C

p.Y377C

p.Y260C

–

c.782A>G

c.440A>G

c.860A>G

c.788A>G

c.857A>G

c.1124A>G

c.1127A>G

c.1130A>G

c.779A>G

c.*171A>G

p.G23R

p.G388R

c.67G>A

c.1162G>A

p.G254D

p.G113D

p.G206D

p.G152D

p.G86D

p.G234D

c.374G>A

c.338G>A

c.617G>A

c.455G>A

c.257G>A

c.701G>A

p.A86fs

p.A47fs

c.257del279

c.140del162

p.G641fs

p.G646fs

c.1919insG

c.1934insG

p.P291fs

p.G226fs

–

c.864delG

c.677delG

c.*304delG

p.E41fs

p.R24fs

p.E24fs

c.121insG

c.69insG

c.70insG

–

p.R693H

p.R659H

p.R882H

c.*498G>A

c.2078G>A

c.1976G>A

c.2645G>A

Chromosome location, amino acid (AA) change and codon change are indicated

(na, not available)

A possible association between the genetic variants and the patient’s clinicopathological characteristics was also evaluated. Due to the little size of the cohort, patients were clustered into those with variants identified only at early time points (T1 and T2) and those with variants identified only at later time points (T3 and T4). We measured a significant association between the BAX variant and portal vein invasion (p = 0.014) and between the HNF1A variant and liver cirrhosis (p = 0.032). For both genes, the significance was found only at the earlier time points. On the contrary, CYP2B showed to be significantly (p = 0.005) associated to BCLC grading only at the later time points. No further correlations were found (all p > 1.0) (Table 4).Table 4Association between variant detection at different time points and patient’s clinicopathological characteristicsVariablePatientsCYP2B6 (T1–T2)CYP2B6 (FU)BAX (T1–T2)BAX (FU)HNF1A (T1–T2)HNF1A (FU)n8846512Portal vein invasion Yes3313113 No10571549 p0.2310.51
0.014*
111BCLC A + B7471227 C6413435 p1
0.005*
0.2660.2860.5921Metastases Yes4312323 No9572339 p0.6080.2170.530.26610.538Cirrhosis Yes8663417 No5221245 p0.2930.2930.6081
0.032*
1Tumor size (cm) < 56242226 > 56532326 p0.24211111The analysis shows that there is a significant association between variants in CYP2B6, BAX and HNF1A and BCLS status (p = 0.005) portal vein invasion (p = 0.014) and absence of liver cirrhosis (p = 0.032), respectively. No association was found between any variant and presence of metastases or tumor size (all p > 1.0). Comparison was performed (for each time group separately) using the Fischer exact test

Association between variant detection at different time points and patient’s clinicopathological characteristics

The analysis shows that there is a significant association between variants in CYP2B6, BAX and HNF1A and BCLS status (p = 0.005) portal vein invasion (p = 0.014) and absence of liver cirrhosis (p = 0.032), respectively. No association was found between any variant and presence of metastases or tumor size (all p > 1.0). Comparison was performed (for each time group separately) using the Fischer exact test

We further evaluated the prognostic value of the genetic variants with respect to OS. Patients carrying the CYP2B6 variant in the time frame T1–T2 showed a worse OS (p = 0.013). On the contrary, when the same variant was detected at later time points, no significant correlation was found (p = 0.360) (Fig. 7). No other genetic variant showed any correlation with the clinical outcome at any of the time points considered (all p > 0.1).Fig. 7Survival plots for HCC patients carrying the CP2B6 variant. Patients were grouped according to the time points (T1–T2, a; T3–T4, b). Patients carrying the variant at T1–T2, showed a worse survival with respect to those patients carrying the variant at T3–T4. OS was analyzed using the Kaplan–Meier method and survival estimates in the different groups were compared using the log-rank test

Survival plots for HCC patients carrying the CP2B6 variant. Patients were grouped according to the time points (T1–T2, a; T3–T4, b). Patients carrying the variant at T1–T2, showed a worse survival with respect to those patients carrying the variant at T3–T4. OS was analyzed using the Kaplan–Meier method and survival estimates in the different groups were compared using the log-rank test

### Clinicopathological characteristics of the patients

Within the cohort chosen for this study, patients were divided in two subgroups: patients with an age below 65, who did not present with liver cirrhosis at time of recruitment, did not report any past alcohol abuse and, based on the SORAMIC trial [19] results, responded better to therapies; and patients over 65, who presented with liver cirrhosis at time of recruitment, reported past alcohol abuse and responded worse to therapies. None of the patients were positive for Hepatitis B, only 1 was positive for Hepatitis C, and 4 presented non-alcoholic steatohepatitis (NASH). Three patients (33.3%, for n = 4 the number of lesions was not available) presented with diffuse disease (> 20 lesions), while among the others the median number of lesions was 2 (range 0–28) and the median size of the lesions was 49 mm (range 8–160 mm). Three patients (23.0%) showed portal vein infiltration, all of them presented with a Child–Pugh A liver function, 46.1% (n = 6) of the patients were Barcelona Clinic Liver Cancer Staging (BCLC) C, while the remaining were A (n = 3, 23.0%) or B (n = 4, 30.7%). Four patients (30.7%) presented lymph node metastases and one of them (7.7%) also bone metastases. Two patients (66.6%) showed recurrence at 2 years FU. A summary of the clinicopathological characteristics of the patients is given in Table 1.Table 1Clinicopathological characteristics of the patientsIDAgeLiver cirrhosisHBVHCVAlcohol abuseOtherNumber of lesions (> 20 = DD)Max. diameter largest lesion (mm)PVICPCP pointsBCLC stageTherapyMRdeathOS (months)A46NoNegNegNoNash2834NoA5CRESRFBNoNo040.7B60NoNegNegNo–99160NoA6CRESRFBYesNo138.6C61NoNegNegNoNash998NoA5BRESRFBNoNo023.4D61NoNegNegNoNashna88NoA5BRESRFBNoNo016.1E59NoNegposNo–na55YesA5CRESRFBYesNo111.7F68YesNegNegYesNash391YesA6CRESRFBYesNo126.8G75YesNegNegYes–nanaNoA6BRESRFBNoNo113.7H68YesNegNegYes–315YesA5CRESRFBNoNo116.8I66YesNegNegYes–na65NoA5CRESRFBYesNo113.0J69YesNegNegYes–275NoA5BRESRFBNoNo117.5K70YesNegNegYes–143NoA5ARFAPLCNoYes127.1L82YesNegNegYes–140NoA6ARFASRFBNoNo126.3M73YesNegNegYes–133NoA5ARFASRFBNoYes132.5HBV, hepatitis B virus; HCV, hepatitis C virus; DD, diffuse disease; Nash, non-alcoholic steatohepatitis; PVI, portal vein infiltration; CP, Child–Pugh, BCLC, Barcelona Clinic Liver Cancer; RE, 90Y-Radioembolization; SRFB, sorafenib; PLCB, placebo; M, metastases; R, recurrence; OS, overall survival, in months; na, not available

Clinicopathological characteristics of the patients

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RE

SRFB

RFA

PLC

RFA

SRFB

RFA

SRFB

HBV, hepatitis B virus; HCV, hepatitis C virus; DD, diffuse disease; Nash, non-alcoholic steatohepatitis; PVI, portal vein infiltration; CP, Child–Pugh, BCLC, Barcelona Clinic Liver Cancer; RE, 90Y-Radioembolization; SRFB, sorafenib; PLCB, placebo; M, metastases; R, recurrence; OS, overall survival, in months; na, not available

### Correlation between cfDNA levels, patient’s clinicopathological characteristics and clinical outcome

cfDNA was isolated from plasma collected at the 4 time points (T1, T2, T3, and T4) during the SORAMIC trial. Nine of the 52 cfDNA samples (17.3%) were excluded from the subsequent analysis due to high genomic DNA contamination or because the concentration was below the threshold (Additional file 1: Figure S1). In the remaining samples, the amount of cfDNA at the different time points was ranging between 2.04 ng/ml to 160.75 ng/ml with a median of 10.2 ng/ml at T1, 15.3 ng/ml at T2, 9.3 ng/ml at T3 and 13.3 ng/ml at T4 (Table 2). We did not observe a significant correlation between cfDNA concentration at T1 and tumor volume or between cfDNA concentrations at any time points and patient’s clinical characteristics (all p > 0.1) (data not shown). On the contrary, we did observe a difference between the concentrations of cfDNA in the presence or absence of metastases, which was significant (p = 0.012) or borderline significant (p = 0.073) at T1 and T2, respectively (Fig. 3). The relative amount of cfDNA at the different time points was also evaluated with respect to overall survival (OS). Patients were grouped according to high or low cfDNA levels with respect to the median values at the different time points. We found a borderline significance at the latest time points (T3, p = 0.057; T4, p = 0.095), suggesting that patients keeping high levels of plasmatic cfDNA while receiving systemic therapy might have a worse outcome (Fig. 4). On the contrary we did not find any association between cfDNA levels at earlier time points and OS (all p > 0.1). The fact the we found a borderline significance might be due to the low number of patients included in the analysis, therefore it will be necessary to validate these preliminary results with a bigger cohort of patients.Table 2Amount of cfDNA (ng/ml) and AFP (ng/ml) in plasma samples at the different time pointsPatientsT1T2T3T4Median [cfDNA]cfDNAcfDNAcfDNAcfDNAAFPAFPAFPAFPAna39.448.3910.1110.185.0011.0013.006.00B12.7817.68na15.3015.3373.0075.0033.0045.00C4.357.342.723.403.92.003.003.003.00D2.8114.964.189.076.64.003.009.0016.00E30.6315.3017.2038.6223.93.003.0010.0042.10F28.3357.1230.6014.7329.52.004.00na4.00G10.2011.9011.5620.9411.74.006.008.004.00H1.815.649.3811.027.513.0019.0030.0072.00I30.3724.287.4113.3318.81681.001685.00991.00967.00J7.96na12.60160.7512.66154.002410.008732.003855.00K2.044.427.289.255.84.003.003.003.00L20.2616.6816.6816.6816.87.005.006.007.00Mnanana8.988.989.004.004.006.00Median [cfDNA]10.215.39.313.3cfDNA median values are given for each patient at the four time points (last column) or for each time point for all patients (last row)na, not available
Fig. 3cfDNA concentrations at different time points and presence of metastases. The analysis shows that patients presenting metastases when recruited in the trial, had still a significant higher amount of plasmatic cfDNA in the timeframe between micro-interventional therapy and the beginning of sorafenib-based systemic therapy (T1, p = 0.012). Borderline significance (p = 0.073) was found after the beginning of systemic therapy (T2), while no significant difference was found at the two later time points. Comparison was performed using the Mann–Whitney U test (* ≤ 0.05)
Fig. 4Survival plots for HCC patients grouped according to the amount of cfDNA at the different time points. Patients’ total population was grouped according to the corresponding cfDNA concentration with respect to the median value found at each time point. Higher cfDNA concentration at later time points (T3 and T4) was showing a trend with a shorter OS (p = 0.057 and p = 0.095) while no association (p > 0.1) was found and earlier time points (T1 and T2). High and low cfDNA levels were defined as being above or below the median values at the different time points. OS was analyzed using the Kaplan–Meier method and survival estimates in the different groups were compared using the log-rank test

Amount of cfDNA (ng/ml) and AFP (ng/ml) in plasma samples at the different time points

cfDNA median values are given for each patient at the four time points (last column) or for each time point for all patients (last row)

na, not available

cfDNA concentrations at different time points and presence of metastases. The analysis shows that patients presenting metastases when recruited in the trial, had still a significant higher amount of plasmatic cfDNA in the timeframe between micro-interventional therapy and the beginning of sorafenib-based systemic therapy (T1, p = 0.012). Borderline significance (p = 0.073) was found after the beginning of systemic therapy (T2), while no significant difference was found at the two later time points. Comparison was performed using the Mann–Whitney U test (* ≤ 0.05)

Survival plots for HCC patients grouped according to the amount of cfDNA at the different time points. Patients’ total population was grouped according to the corresponding cfDNA concentration with respect to the median value found at each time point. Higher cfDNA concentration at later time points (T3 and T4) was showing a trend with a shorter OS (p = 0.057 and p = 0.095) while no association (p > 0.1) was found and earlier time points (T1 and T2). High and low cfDNA levels were defined as being above or below the median values at the different time points. OS was analyzed using the Kaplan–Meier method and survival estimates in the different groups were compared using the log-rank test

### cfDNA in comparison to the conventional biomarker AFP

AFP is detectable in as many as 30% of HCC patients at time of diagnosis and usually remains low during the course of the disease, even with advanced HCC [20]. AFP > 400–500 ng/ml supports diagnosis of HCC [21]. At the considered time points, only 2 (15.4%) of the patients showed high AFP values (> 400 ng/ml), 4 (30.8%) showed values between 20 and 400 ng/ml and 7 (53.8%) showed values between 2 and 20 ng/ml (Table 2). It has been already shown that cfDNA has clinical diagnostic relevance, reflecting therapy response [25]. We report here in more detail three vignettes of patients with evidence of disease progression, for which cfDNA offered earlier monitoring value than conventional biomarkers such as AFP (Fig. 5). A concentration of 400 ng/ml was chosen as cut-off value. The first patient (patient B) entered in the trial with a diffuse disease in the liver and lymph nodes and bone metastases; he died after 38.6 months from enrolment in the SORAMIC trial. AFP concentration was high (373 ng/ml) before starting sorafenib treatment; however it decreased very rapidly below 50 ng/ml and remained low while assuming sorafenib, suggesting efficacy of treatment. On the contrary, we did not find a corresponding reduction in the cfDNA concentration which instead increased during time, suggesting the presence of progressive disease, as confirmed by CT imaging at follow up (Fig. 5a). The second patient (patient E) entered in the study with lymph node metastases, portal vein infiltration and disease progression on treatment with an OS of 11.7 months from the beginning of therapy. At time points considered, AFP values were measured always below 50 ng/ml; in contrast, cfDNA level, while showing a drastic decrease immediately after the beginning of the therapy, increased rapidly between 6 and 8 months from the beginning of the therapy, reflecting the rapid progression of the disease and indicating a change in the response to sorafenib (Fig. 5b). The third patient (patient K) entered in the study presenting only one hepatic lesion and no evidence of metastases. AFP was always measured below 5 ng/ml at all time points, however cfDNA showed a steadily increase, predicting the progression of the disease. The patient finally relapsed and died shortly after (Fig. 5c). In conclusion, our data, in line with other reports, showed improved diagnostic sensitivity of cfDNA over AFP [24, 26], suggesting also in advanced HCC cfDNA as a biomarker for monitoring therapy efficacy.Fig. 5Comparison between cfDNA and AFP levels in three cases corresponding to patients showing different clinical characteristics. Panels a, b and c represent the described clinical cases of patients B, E and K, respectively

Comparison between cfDNA and AFP levels in three cases corresponding to patients showing different clinical characteristics. Panels a, b and c represent the described clinical cases of patients B, E and K, respectively

### Somatic mutation analysis

Genomic profiling of ctDNA was undertaken at the different time points, with the screening of 597 cancer-relevant genes. Only those mutations proven to have clinical impact according to the ClinVar database (National Center for Biotechnology Information-NCBI) were further analyzed. In total 28 variants, including single nucleotide variations (SNV) or insertions and deletions (InDel) were discovered in our dataset, while no gene fusion was identified (Fig. 6). All patients showed at least one variant (median = 6 variants per patient, range 3–16), however patterns varied between patients. Variants were found in genes involved in DNA repair, multidrug resistance, cell cycle control, signal transduction, transcription control, chromatin structure, apoptosis and DNA methylation (Table 3). We detected somatic mutations in recognized HCC driver genes such as CTNNB1 and TP53, however at very low frequency. Over the 28 variants, three genes were showing the highest mutation allele frequency (MAF): HNF1A (n = 12, 92.3%), BAX (n = 9, 69.2%) and CYB2B6 (n = 6, 46.1%). We observed that in a high percentage of genes (n = 19, 68%) variants were found mainly after the beginning of sorafenib treatment, suggesting a possible clonal selection induced by the therapy itself. Nevertheless, a different kinetic was also observed. For example, in the case of HNF1A, 50% of the patients showed the variant already before starting to assume sorafenib. At T2, only one patient was still positive, suggesting a good therapy response. However at T3 the variant was showing a MAF of 82% (n = 9), a finding which could suggest a clonal expansion of cells not responsive any longer to sorafenib.Fig. 6Heatmaps showing the variants discovered in the two subgroups of patients (A–E: below 65, no liver cirrhosis, no past alcohol abuse and better response to therapies; F–M: over 65, with liver cirrhosis, past alcohol abuse and worse response to therapies) at the different time points. In (a) and in (b) SNV and InDel with relative mutation frequencies are reported, respectively
Table 3List of variants grouped according to gene functionGene groupLocationAACodonPatients (%)DNA repair ATMchr11:108121426–c.1236-2A>T2 (15.4) BLMchr15:91304138p.N515fsnac.1544delAc.*468delA1 (7.7) BRCA2chr13:32954022p.T18fsp.T3033fsc.52insAc.9097insA1 (7.7) ERCC5chr13:103524611p.K150fsp.K917fsc.450delAc.2751delA5 (38.4) MSH6chr2:48030639p.F56fsp.F1088fs–p.F958fsp.F786fsc.165delCc.3261delCc.*2608delCc.2871delCc.2355delC6 (46.1) RAD50chr5:131931451p.K722fs––p.?661fsc.2165delAc.*1791delAc.*351delAc.1982delA4 (30.8) WRNchr8:30915970p.K5fsc.15delA2 (15.4)Multidrug resistance/xenobiotic metabolism ABCB1chr7:87160618p.S893Ap.S829Ac.2677T>Gc.2485T>G3 (23.0) CYP2B6chr19:41515263p.K262Rc.785A>G6 (46.1) CYP2C9chr10:96702047p.R144Cc.430C>T1 (7.7) CYP2D6chr22:42524994p.P34Sp.P12Sc.100C>Tc.34C>T2 (15.4) GSTP1chr11:67352689–p.I105Vc.*137A>Gc.313A>G2 (15.4) SLCO1B1chr12:21331549p.V174Ac.521T>C2 (15.4)Cell cycle control/proliferation CTNNB1chr3:41266098p.D32Vp.D25Vc.95A>Tc.74A>T1 (7.75) FGFR2chr10:123274794p.Y261Cp.Y147Cp.Y287Cp.Y263Cp.Y286Cp.Y375Cp.Y376Cp.Y377Cp.Y260C–c.782A>Gc.440A>Gc.860A>Gc.788A>Gc.857A>Gc.1124A>Gc.1127A>Gc.1130A>Gc.779A>Gc.*171A>G1 (7.75) FGFR4chr5:176520243p.G23Rp.G388Rc.67G>Ac.1162G>A1 (7.75) TP53chr17:7579447p.G254Dp.G113Dp.G206Dp.G152Dp.G86Dp.G234Dc.374G>Ac.338G>Ac.617G>Ac.455G>Ac.257G>Ac.701G>A1 (7.75) TP53*chr17:7579407p.A86fsp.A47fsc.257del279c.140del1621 (7.75)Signal transduction signalling PIK3CAchr3:178952085p.H1047Rc.3140A>G1 (7.75) PIK3CA*chr3:178952085p.H1047Lc.3140A>T1 (7.75) SOS1chr2:39249927p.S548Rc.1642A>C1 (7.75)Chromatin structure ASXL1chr20:31022441p.G641fsp.G646fsc.1919insGc.1934insG3 (23.0)Liver transcription factor HNF1Achr12:121432114p.P291fsp.G226fs–c.864delGc.677delGc.*304delG12 (92.3)Apoptosis BAXchr19:49458970p.E41fsp.R24fsp.E24fsc.121insGc.69insGc.70insG9 (69.2)DNA methylation DNMT3Achr2:25457242–p.R693Hp.R659Hp.R882Hc.*498G>Ac.2078G>Ac.1976G>Ac.2645G>A2 (15.4)Chromosome location, amino acid (AA) change and codon change are indicated(na, not available)

Heatmaps showing the variants discovered in the two subgroups of patients (A–E: below 65, no liver cirrhosis, no past alcohol abuse and better response to therapies; F–M: over 65, with liver cirrhosis, past alcohol abuse and worse response to therapies) at the different time points. In (a) and in (b) SNV and InDel with relative mutation frequencies are reported, respectively

List of variants grouped according to gene function

p.N515fs

na

c.1544delA

c.*468delA

p.T18fs

p.T3033fs

c.52insA

c.9097insA

p.K150fs

p.K917fs

c.450delA

c.2751delA

p.F56fs

p.F1088fs

–

p.F958fs

p.F786fs

c.165delC

c.3261delC

c.*2608delC

c.2871delC

c.2355delC

p.K722fs

–

–

p.?661fs

c.2165delA

c.*1791delA

c.*351delA

c.1982delA

p.S893A

p.S829A

c.2677T>G

c.2485T>G

p.P34S

p.P12S

c.100C>T

c.34C>T

–

p.I105V

c.*137A>G

c.313A>G

p.D32V

p.D25V

c.95A>T

c.74A>T

p.Y261C

p.Y147C

p.Y287C

p.Y263C

p.Y286C

p.Y375C

p.Y376C

p.Y377C

p.Y260C

–

c.782A>G

c.440A>G

c.860A>G

c.788A>G

c.857A>G

c.1124A>G

c.1127A>G

c.1130A>G

c.779A>G

c.*171A>G

p.G23R

p.G388R

c.67G>A

c.1162G>A

p.G254D

p.G113D

p.G206D

p.G152D

p.G86D

p.G234D

c.374G>A

c.338G>A

c.617G>A

c.455G>A

c.257G>A

c.701G>A

p.A86fs

p.A47fs

c.257del279

c.140del162

p.G641fs

p.G646fs

c.1919insG

c.1934insG

p.P291fs

p.G226fs

–

c.864delG

c.677delG

c.*304delG

p.E41fs

p.R24fs

p.E24fs

c.121insG

c.69insG

c.70insG

–

p.R693H

p.R659H

p.R882H

c.*498G>A

c.2078G>A

c.1976G>A

c.2645G>A

Chromosome location, amino acid (AA) change and codon change are indicated

(na, not available)

### Association between variants, patient’s clinicopathological characteristics and overall survival

A possible association between the genetic variants and the patient’s clinicopathological characteristics was also evaluated. Due to the little size of the cohort, patients were clustered into those with variants identified only at early time points (T1 and T2) and those with variants identified only at later time points (T3 and T4). We measured a significant association between the BAX variant and portal vein invasion (p = 0.014) and between the HNF1A variant and liver cirrhosis (p = 0.032). For both genes, the significance was found only at the earlier time points. On the contrary, CYP2B showed to be significantly (p = 0.005) associated to BCLC grading only at the later time points. No further correlations were found (all p > 1.0) (Table 4).Table 4Association between variant detection at different time points and patient’s clinicopathological characteristicsVariablePatientsCYP2B6 (T1–T2)CYP2B6 (FU)BAX (T1–T2)BAX (FU)HNF1A (T1–T2)HNF1A (FU)n8846512Portal vein invasion Yes3313113 No10571549 p0.2310.51
0.014*
111BCLC A + B7471227 C6413435 p1
0.005*
0.2660.2860.5921Metastases Yes4312323 No9572339 p0.6080.2170.530.26610.538Cirrhosis Yes8663417 No5221245 p0.2930.2930.6081
0.032*
1Tumor size (cm) < 56242226 > 56532326 p0.24211111The analysis shows that there is a significant association between variants in CYP2B6, BAX and HNF1A and BCLS status (p = 0.005) portal vein invasion (p = 0.014) and absence of liver cirrhosis (p = 0.032), respectively. No association was found between any variant and presence of metastases or tumor size (all p > 1.0). Comparison was performed (for each time group separately) using the Fischer exact test

Association between variant detection at different time points and patient’s clinicopathological characteristics

The analysis shows that there is a significant association between variants in CYP2B6, BAX and HNF1A and BCLS status (p = 0.005) portal vein invasion (p = 0.014) and absence of liver cirrhosis (p = 0.032), respectively. No association was found between any variant and presence of metastases or tumor size (all p > 1.0). Comparison was performed (for each time group separately) using the Fischer exact test

We further evaluated the prognostic value of the genetic variants with respect to OS. Patients carrying the CYP2B6 variant in the time frame T1–T2 showed a worse OS (p = 0.013). On the contrary, when the same variant was detected at later time points, no significant correlation was found (p = 0.360) (Fig. 7). No other genetic variant showed any correlation with the clinical outcome at any of the time points considered (all p > 0.1).Fig. 7Survival plots for HCC patients carrying the CP2B6 variant. Patients were grouped according to the time points (T1–T2, a; T3–T4, b). Patients carrying the variant at T1–T2, showed a worse survival with respect to those patients carrying the variant at T3–T4. OS was analyzed using the Kaplan–Meier method and survival estimates in the different groups were compared using the log-rank test

Survival plots for HCC patients carrying the CP2B6 variant. Patients were grouped according to the time points (T1–T2, a; T3–T4, b). Patients carrying the variant at T1–T2, showed a worse survival with respect to those patients carrying the variant at T3–T4. OS was analyzed using the Kaplan–Meier method and survival estimates in the different groups were compared using the log-rank test

### Discussion

HCC is a heterogeneous pathology, involving multiple somatic genomic alterations responsible for different degrees of tumor differentiation and related clinical outcomes.

TP53 and CTNNB1 are the two most frequent mutations although other driver mutations have been identified [27]. Genomic analysis, based on liver biopsy, allows the molecular subtyping of HCC and opens the door to select and adapt therapies to the individual patient’s needs. Apart from being associated to a higher risk for the patient, tissue biopsy presents nevertheless some limitations: first, the genetic heterogeneity in tumor differentiation cannot be assessed by a single site sampling; second, the histopathological assessment of vascular invasion requires large biopsies; third, being highly invasive, biopsy is usually performed only once during the whole therapy. Liquid biopsy is independent of these limitations and shows predictive and prognostic value in several types of cancer including HCC [12, 13, 28]. cfDNA quantitative changes have been mainly linked to tumor’s burden and therapeutic efficacy [26], while ctDNA qualitative changes can detect genetic variations in the tumor as a whole [16, 29].

In this exploratory study we investigated the prognostic value of cfDNA/ctDNA isolated from advanced HCC patients over multiple time points in the context of the SORAMIC trial. We observed that the amount of cfDNA measured immediately after micro-intervention significantly correlated with the presence of distant metastases, supporting also in this clinical context a diagnostic value of cfDNA to unravel secondary lesions, as showed in other tumor types [25, 30, 31]. In addition, patients with post-operative high levels of cfDNA had a shorter OS. Interestingly, this holds true only at the later time points, after several weeks of sorafenib treatment, suggesting that also in advanced HCC cfDNA can support in monitoring therapy response. This finding is relevant when considering the use of other biomarkers such as AFP [32, 33] whose prognostic value is still under debate [34, 35]. Among the patients enrolled in this study, only two patients had, over the considered time points, AFP concentrations higher than the assigned cut-off value of 400 ng/ml. On the contrary, quantitative analysis of post-operative cfDNA was always feasible and showed a trend with the clinical history of the patients, supporting the use of cfDNA to monitor therapy response also in advanced HCC.

NGS analysis showed that all 13 patients presented SNV and/or InDel variants in a total of 28 genes, with 6 as the median number of variants per patient. We never found two patients with an identical profile, supporting the concept that HCC is a highly heterogeneous disease. Interestingly, only in few cases we detected the hotspot variants normally associated to HCC as those in TP53 or CTNNB1. We cannot exclude that this result is due to the low number of patients included in the study, therefore a validation study with a larger cohort will be necessary. However, these driver mutations are usually associated to specific etiological characteristics, not present in our patients’ group, such as HBV infection, dietary exposure to aflatoxin B1, or are per se very rare events [36]. The absence of these variants could be therefore associated to the clinical characteristics of the patients. In addition, in a multimodal meta-analysis run over 1494 HCC samples, it has been shown that other driver mutations, less frequent nevertheless still clinically significant, can be identified [37]. From the 28 variants, we identified in particular 3 genes displaying high MAFs. CYP2B6 (mutated in 46.1% of the patients at least at one time point) belongs to the Cytochrome P450 class of enzymes which are important catalysts among all metabolizing enzymes. The isoform CYP2B6, expressed mainly in the liver, is highly polymorphic [38] (with the haplotype CYP2B6*6 found in the Caucasian population in up to 25% of the cases) and shows clinical relevance [38, 39]. However, there are contradictory results on the effect of the haplotype *6 (h*6). On one side it has been reported that patients carrying h*6 have a reduced amount of mRNA and protein [40], with an inferior response to specific treatments. On the other side, studies demonstrated that h*6 increase the enzymatic activity of CYP2B6 [41]. In our study, we found that 46% of the patients presented a CYP2B6 variant corresponding to h*6 and that the detection of this variant at earlier time points was significantly associated with a worse prognosis and a shorter OS. Since variants in enzymes involved in the metabolism can affect the response to specific drugs [42] and CYP450 and its haplotypes are involved in the metabolism of several drugs [39, 43], we can hypothesize that those patients presenting the variant h*6 responded worse to the therapy with sorafenib and therefore had a short OS.

BAX (mutated in 69.2% of the patients at least at one time point) belongs to the Bcl-2 gene family of pro-apoptotic proteins [44]. Inactivation of BAX might be a mechanism to escape cell death, providing a survival advantage to the tumor. In colorectal cancer this particular frameshift mutation is indeed conferring an advantage to tumor progression, with p53-independent induction of apoptosis [45]. Sorafenib is a multikinase inhibitor with a strong anti-proliferative and anti-angiogenic effect on different cell types [46], displaying also a pro-apoptotic effect [47]. BAX might play a role in cellular death induced by sorafenib, and the presence of this frameshift variant might predict a low therapy efficacy due to the inactivation of the pathway inducing apoptosis. Although we did not find any significant correlation between the presence of variants of BAX and OS, we unraveled a significant correlation with portal vein invasion. Tumor cells able to overcome apoptosis display higher invasive characteristics. The fact that 3 patients carrying this mutation are positive for portal vein infiltration might support the role of BAX in tumor invasion and the possible inefficacy of sorafenib in these patients. However at the moment this is just a hypothesis and further analysis will be necessary to confirm it.

Finally, HNF1A showed the highest MAF, reaching 92.3% over the whole cohort. HNF1A codes for a transcription factor (HNF1α) which regulates liver differentiation and development [48]. This transcription factor functions as a tumor suppressor [49] and its inactivation might play an important role in tumor development [50]. A recent work of Takashima et al., demonstrated that the ectopic expression of HNF1α combined to other two transcription factors (HNF4a and FOXA3) successfully inhibited proliferation in a cellular model system [51]. If confirmed in other model system, HNF1A could be a novel target for combined therapy, especially when drug resistance is developed.

Moreover, while HNF1A inactivation is known to be frequent in hepatocellular adenoma, somatic mutations in this gene are relatively a rare event in HCC and very often bound to specific etiological characteristics such as absence of cirrhosis and viral infection [52]. In our cohort, none of the patients but one presented with viral infection and the presence of the variant showed a significant negative association with cirrhosis. Although TP53 and CTBNN1 are the two most frequently mutated genes in HCC, there are other low frequency driver genes, such as HNF1A, which nevertheless are significantly associated to patient’ survival. We can hypothesize that in the patient cohort selected for this study, HNF1A was a driver mutation. If confirmed in a larger number of patients, HNF1A could represent a novel target for personalized therapy.

A remarkable result we observed was the dynamic change of the mutation status over time. With respect to each gene taken individually, in the time frame between T1 and T4 we identified 6 categories of patients, showing: always the wildtype gene (WT); always the mutated gene (M); a single shift WT → M; a single shift M → WT; a double shift WT → M → WT; a double shift M → WT → M. With the small number of patients included in the study, for the moment we can only speculate that these changes are mirroring clonal heterogeneity found in the primary tumors [53, 54], evidenced because the analysis was done by liquid biopsy. It is reasonable to speculate that the shift WT → M might take place because the mutated clones are more resistant to therapy; therefore post-treatment samples are enriched with ctDNA carrying the mutation. These clones possibly develop spontaneously and the mutation they carry confers a selective advantage to cells. On the contrary, the shift M → WT might be observed because the mutated clones are more sensitive to therapy, which possibly could slow down their proliferation, while WT clones divide and expand faster. In the latter case we cannot exclude that ctDNA carrying the mutation is still present, however in this case the corresponding MAF is under the NGS detection threshold. Dynamic changes in clonal expansion during therapy are a known event [55], also strongly suggested by our data. Due to the limited number of samples, we cannot provide here anything more than a hypothesis. However it will be surely interesting to pursue in this direction, enlarging the number of cases and validating these preliminary results.

### Conclusions

A validation study with a larger cohort and the inclusion of appropriate control groups will be necessary to confirm these preliminary results and to support their clinical value. Also a more detailed comparison between patients receiving ablation or radio-embolization should be considered to understand if different procedures possible with the micro-interventional therapy might induce the accumulation of alternative genetic mutations. However, from the results obtained from this exploratory study we can conclude that also in advanced HCC after micro-intervention cfDNA quantification is feasible, can support standard imaging analysis for the early detection of metastatic lesions and is showing a prognostic value with respect to OS. In addition, interrogating ctDNA for genetic variants can reduce understaging of clonal selection as a consequence of systemic therapy, in the potential to improve longitudinal monitoring of therapy efficacy. HCC is a highly heterogeneous tumor type; it would be therefore an advantage to tailor therapy according to the intrinsic genetic characteristics. None of the patients included in this study showed an identical genomic profile, supporting the concept that also in advanced HCC personalized therapy is necessary to improve clinical outcome. A rapid adaption of therapy in concomitance to variant appearance and therefore a more efficient treatment might be particularly important for patients already in an advanced stage of the disease and therefore with a relative short overall survival expectancy.

### Supplementary information

Additional file 1: Figure S1. Size distribution of extracted cfDNA after capillary electrophoresis (CE). DNA extracted from all the plasma samples were fractionated by CE and analyzed. cfDNA displays a size of approximately 150 base pairs (bp), corresponding to one nucleosome, and multiple of this size, while gDNA has a higher molecular weight of several kilobases (kb). Samples displaying a unique peak around 150 bp were further analyzed, while samples showing a peak several kb were no further processed. Panels A and B report examples of cfDNA and gDNA as determined by CE, respectively (RFU, relative fluorescence unit; LM, lower marker; UM, upper marker).
Additional file 2: Table S1. Depth of coverage. An average coverage of at least 1000 times or more was reached for all the samples (average 1743.63, range 1182.17–2870.64), except one (sample F4) which showed a lower depth (840.41).
Additional file 3: Table S2. List of variants as detected in genomic DNA extracted from buffy coats for each patient. Chromosome location, gene name and mutation frequency are indicated. The analysis was done for all the patients except for patient E, whose corresponding cellular sample was not available. For this patient we set arbitrarily a mutation frequency threshold corresponding to the average of the other samples. Genes presenting a mutation frequency of 50 (± 5) and 100 (± 5) % were considered carrying a germline mutation and therefore were excluded from the following analysis.

Additional file 1: Figure S1. Size distribution of extracted cfDNA after capillary electrophoresis (CE). DNA extracted from all the plasma samples were fractionated by CE and analyzed. cfDNA displays a size of approximately 150 base pairs (bp), corresponding to one nucleosome, and multiple of this size, while gDNA has a higher molecular weight of several kilobases (kb). Samples displaying a unique peak around 150 bp were further analyzed, while samples showing a peak several kb were no further processed. Panels A and B report examples of cfDNA and gDNA as determined by CE, respectively (RFU, relative fluorescence unit; LM, lower marker; UM, upper marker).

Additional file 2: Table S1. Depth of coverage. An average coverage of at least 1000 times or more was reached for all the samples (average 1743.63, range 1182.17–2870.64), except one (sample F4) which showed a lower depth (840.41).

Additional file 3: Table S2. List of variants as detected in genomic DNA extracted from buffy coats for each patient. Chromosome location, gene name and mutation frequency are indicated. The analysis was done for all the patients except for patient E, whose corresponding cellular sample was not available. For this patient we set arbitrarily a mutation frequency threshold corresponding to the average of the other samples. Genes presenting a mutation frequency of 50 (± 5) and 100 (± 5) % were considered carrying a germline mutation and therefore were excluded from the following analysis.

Additional file 1: Figure S1. Size distribution of extracted cfDNA after capillary electrophoresis (CE). DNA extracted from all the plasma samples were fractionated by CE and analyzed. cfDNA displays a size of approximately 150 base pairs (bp), corresponding to one nucleosome, and multiple of this size, while gDNA has a higher molecular weight of several kilobases (kb). Samples displaying a unique peak around 150 bp were further analyzed, while samples showing a peak several kb were no further processed. Panels A and B report examples of cfDNA and gDNA as determined by CE, respectively (RFU, relative fluorescence unit; LM, lower marker; UM, upper marker).
Additional file 2: Table S1. Depth of coverage. An average coverage of at least 1000 times or more was reached for all the samples (average 1743.63, range 1182.17–2870.64), except one (sample F4) which showed a lower depth (840.41).
Additional file 3: Table S2. List of variants as detected in genomic DNA extracted from buffy coats for each patient. Chromosome location, gene name and mutation frequency are indicated. The analysis was done for all the patients except for patient E, whose corresponding cellular sample was not available. For this patient we set arbitrarily a mutation frequency threshold corresponding to the average of the other samples. Genes presenting a mutation frequency of 50 (± 5) and 100 (± 5) % were considered carrying a germline mutation and therefore were excluded from the following analysis.

Additional file 1: Figure S1. Size distribution of extracted cfDNA after capillary electrophoresis (CE). DNA extracted from all the plasma samples were fractionated by CE and analyzed. cfDNA displays a size of approximately 150 base pairs (bp), corresponding to one nucleosome, and multiple of this size, while gDNA has a higher molecular weight of several kilobases (kb). Samples displaying a unique peak around 150 bp were further analyzed, while samples showing a peak several kb were no further processed. Panels A and B report examples of cfDNA and gDNA as determined by CE, respectively (RFU, relative fluorescence unit; LM, lower marker; UM, upper marker).

Additional file 2: Table S1. Depth of coverage. An average coverage of at least 1000 times or more was reached for all the samples (average 1743.63, range 1182.17–2870.64), except one (sample F4) which showed a lower depth (840.41).

Additional file 3: Table S2. List of variants as detected in genomic DNA extracted from buffy coats for each patient. Chromosome location, gene name and mutation frequency are indicated. The analysis was done for all the patients except for patient E, whose corresponding cellular sample was not available. For this patient we set arbitrarily a mutation frequency threshold corresponding to the average of the other samples. Genes presenting a mutation frequency of 50 (± 5) and 100 (± 5) % were considered carrying a germline mutation and therefore were excluded from the following analysis.



# SUPPLEMENTAL FILE 1: 12967_2019_Article_2079.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)