# MAIN TEXT

## BRCA1 and BRCA2 genes mutations among high risk breast cancer patients in Jordan

### Abstract

Familial breast cancer is estimated to account for 15–20% of all cases of breast cancer. Surveillance for familial breast cancer is well-established world-wide. However, this service does not exist in Jordan, due to the scarcity of information with regard to the genetic profiling of these patients, and therefore lack of recommendations for policy-makers. As such, patients with very strong family history of breast or ovarian cancers are not screened routinely; leading to preventable delay in diagnosis. Whole coding sequencing for BCRA1/BCRA2 using next-generation sequencing (NGS)/Ion PGM System was performed. Sanger sequencing were then used to confirm the pathogenic variants detected by NGS. In this study, 192 breast cancer patients (and 8 ovarian cancer cases) were included. The prevalence of recurrent pathogenic mutations was 14.5%, while the prevalence of newly detected mutations was 3.5%. Two novel pathogenic mutations were identified in BRCA2 genes. The common mutations in the Ashkenazi population used for screening may not apply in the Jordanian population, as previously reported mutations were not prevalent, and other new mutations were identified. These data will aid to establish a specific screening test for BRCA 1/BRCA2 in the Jordanian population.

### Introduction

Breast cancer is the most common cancer in females worldwide contributing to 25.4% of the total number of new cancer cases among women in 2018. Worldwide, more than two million new cases are diagnosed each year and nearly 600,000 women die from breast cancer each year1. According to the Jordanian National Cancer Registry report of the year 2015, there were 1145 female breast cancer cases accounting for 20.6% of all newly-diagnosed cancer cases and 39.4% of the cancers in females2.

Breast cancer at an early age is more likely to be associated with an increased familial risk as hereditary predisposition is estimated to cause about 5–10% of all breast cancers3. Nearly 50% of all hereditary breast cancer cases are due to germline mutations in the BRCA1/BRCA2 genes and are associated with early-onset breast cancer4. The cumulative risk estimates for developing breast cancer by age 80 are 72% for BRCA1 carriers and 69% for BRCA2 carriers4.

Nearly 1650 mutations and sequence variants have been reported in the BRCA1 gene and 1730 in BRCA25. BRCA1 and BRCA2 are both tumor suppressor genes that play a role in DNA repair. The BRCA1 gene plays also a role in checkpoint control. Mutations in either gene are associated with an autosomal dominant inherited form of breast cancer6,7.

Several studies in the Arab world were done to identify the mutations in BRCA1/BRCA2 genes and to assess prevalence of known mutations8–12. The main limitations for the above studies have been the small number of participants and the uncontrolled selection criteria. A recent study conducted in Jordan showed that BRCA1/BRCA2 variants are common among Jordanian breast cancer patients as 27% of high-risk breast cancer patients had pathogenic or likely pathogenic variants in BRCA1 or BRCA2 genes9.

Recently, a comprehensive assessment of the mutational spectrum of BRCA1/BRCA2 in the Middle East, North Africa, and Southern Europe was published13. Laitamn et al. reported recurring pathogenic sequence variants (PSVs) identifying 232 PSVs and 239 PSVs in BRCA1 and BRCA2, respectively. This comprehensive data will lead to directed testing of high-risk subpopulations, and therefore earlier diagnosis and superior management leading to better outcomes. However, there is no founder mutation(s) in BRCA1/BRCA2 in the Jordanian population that could be used for early genetic screening.

In this study, the authors evaluated the potential contribution and the frequency of mutations in the BCRA1/BCRA2 genes in selected Jordanian patients with breast and ovarian cancers, identified based on NICE criteria for high risk of familial cancer, through performing whole coding sequence for BCRA1/ BCRA2 with the purpose of providing baseline data for establishment of familial cancer services in Jordan.

### Materials and methods

The study was approved by the Institutional Review Boards and Ethics Committees at Mutah University and the Central Ethics Committee at the Ministry of Health, Jordan. All included patients were informed about the objective and protocol of the study and gave written informed consent in accordance with the Declaration of Helsinki prior to undergoing any procedures.

This prospective cross-sectional study was conducted et al. Bashir Hospital, Amman, Jordan. All Jordanian breast cancer patients referred to this hospital during the study period were under surveillance for familial breast or ovarian cancer according to a uniform selection criterion (see below).The U.K. National Institute for Health and Care Excellence (NICE), November 2013 Clinical guidelines for identification of patients with high risk for familial breast/ovarian cancer who required referral for testing in a specialist genetics clinic were used for identifying study participants and labelling them as ‘high risk for familial cancer14.

Patients files were reviewed for clinical and pathology data such as stage, grade, site, laterality, oestrogen receptor status, HER2 (or HER2 neu) positive status, etc.

Peripheral blood samples were drawn from study participants (192 breast cancer patients and 8 ovarian cancer cases) and sent for analysis at the Cell Therapy Centre at Jordan University. Genomic DNA was extracted from 5 ml EDTA tubes using QIAprep Spin Miniprep Kit according to manufacturer's recommendations, and quantified using Nano Drop Spectrophotometer (Thermo Fisher Scientific).

The coding exons and flanking intronic regions of the BRCA1/BRCA2 genes were amplified using Ion AmpliSeq Library Kit 2.0 (Life Technologies) and Ion AmpliSeq BRCA1/BRCA2 Panel (Life Technologies). The whole process was performed following the manufacturer’s instructions with minor modifications as previously described15,16. Each library was barcoded using Ion Xpress Barcode Adapters kit (Life Technologies) and its concentration was determined with Ion Library TaqMan Quantitation Kit (Life Technologies). For template preparation, emulsion PCR and enrichment were carried out using Ion OneTouch 2 system and the Ion OneTouch ES Instrument (Life Technologies) with Ion PGM Hi-Q View OT2 Kit (Life Technologies). An average of six prepared barcoded samples were loaded into a single Ion 316 Chip v2 (Life Technologies) per sequencing run. The sequencing reactions were performed in the Ion torrent PGM System (Life Technologies) using sequencing Ion PGM Hi_Q view sequencing kit (Life Technologies). The generated data on Ion PGM were processed through on Torrent Suite, version 5.0.4, running on a local server (Life Technologies). The VCF files were uploaded to Ion Reporter (Life Technologies) and annotated for single-nucleotide variants (SNV), insertions and deletions (indels), and splice site changes. Reads were visualized with the Integrative Genomics Viewer (IGV) and Alamut (Interactive Biosoftware, Rouen, France).

Variants that are clearly classified in the Breast Cancer Information Core (BIC) database, Human Gene Mutation Database Pro (HGMD pro) and ClinVar databases, and not previously described as pathogenic in the literature were evaluated based on the guidelines and standards for interpretation of sequence variants recommended by the American College of Medical Genetics and Genomics (ACMG)17.

Sanger sequencing was performed for validating selected candidate variants identified by NGS. Primers flanking the candidate loci were designed using Primer 3.0 and synthesized by IDT, CA, USA (Supplementary table 1). PCR amplification was performed using Platinum PCR SuperMix (Invitrogen, CA, USA). Direct automated sequencing for the purified PCR products was performed on an ABI 3500 Capillary DNA Sequencer using the BigDye Terminator Cycle Sequencing kit (Applied Biosystems, CA, USA). The sequence chromatograms were visualized by Sequencing Analysis SeqA software (Applied Biosystems) and analyzed by Chromas Pro software (Technolysium Ltd., South Brisbane, Australia).

Collected Data was analyzed using R Statistical Computing Software version 3.4.3 (R Foundation for Statistical Computing, Vienna, Austria). Descriptive statistics including means and standard deviations are reported for numerical patient characteristics while frequencies and percentages are reported for categorical characteristics. Predictors of BRCA1 and BRCA2 mutations were identified through multiple linear regression with stepwise selection. A significance level of 0.05 is used throughout the analysis.

### Ethical approval and consent to participate

The study was approved by the Institutional Review Boards and Ethics Committees at Mutah University and the Central Ethics Committee at the Ministry of Health, Jordan. All included patients were informed about the objective and protocol of the study and gave written informed consent in accordance with the Declaration of Helsinki prior to undergoing any procedures.

### Study population

This prospective cross-sectional study was conducted et al. Bashir Hospital, Amman, Jordan. All Jordanian breast cancer patients referred to this hospital during the study period were under surveillance for familial breast or ovarian cancer according to a uniform selection criterion (see below).The U.K. National Institute for Health and Care Excellence (NICE), November 2013 Clinical guidelines for identification of patients with high risk for familial breast/ovarian cancer who required referral for testing in a specialist genetics clinic were used for identifying study participants and labelling them as ‘high risk for familial cancer14.

Patients files were reviewed for clinical and pathology data such as stage, grade, site, laterality, oestrogen receptor status, HER2 (or HER2 neu) positive status, etc.

### Sample collection and genomic DNA isolation

Peripheral blood samples were drawn from study participants (192 breast cancer patients and 8 ovarian cancer cases) and sent for analysis at the Cell Therapy Centre at Jordan University. Genomic DNA was extracted from 5 ml EDTA tubes using QIAprep Spin Miniprep Kit according to manufacturer's recommendations, and quantified using Nano Drop Spectrophotometer (Thermo Fisher Scientific).

### Next-generation sequencing and variant analysis

The coding exons and flanking intronic regions of the BRCA1/BRCA2 genes were amplified using Ion AmpliSeq Library Kit 2.0 (Life Technologies) and Ion AmpliSeq BRCA1/BRCA2 Panel (Life Technologies). The whole process was performed following the manufacturer’s instructions with minor modifications as previously described15,16. Each library was barcoded using Ion Xpress Barcode Adapters kit (Life Technologies) and its concentration was determined with Ion Library TaqMan Quantitation Kit (Life Technologies). For template preparation, emulsion PCR and enrichment were carried out using Ion OneTouch 2 system and the Ion OneTouch ES Instrument (Life Technologies) with Ion PGM Hi-Q View OT2 Kit (Life Technologies). An average of six prepared barcoded samples were loaded into a single Ion 316 Chip v2 (Life Technologies) per sequencing run. The sequencing reactions were performed in the Ion torrent PGM System (Life Technologies) using sequencing Ion PGM Hi_Q view sequencing kit (Life Technologies). The generated data on Ion PGM were processed through on Torrent Suite, version 5.0.4, running on a local server (Life Technologies). The VCF files were uploaded to Ion Reporter (Life Technologies) and annotated for single-nucleotide variants (SNV), insertions and deletions (indels), and splice site changes. Reads were visualized with the Integrative Genomics Viewer (IGV) and Alamut (Interactive Biosoftware, Rouen, France).

### BRCA1/BRCA2 library preparation and Ion PGM next-generation sequencing

The coding exons and flanking intronic regions of the BRCA1/BRCA2 genes were amplified using Ion AmpliSeq Library Kit 2.0 (Life Technologies) and Ion AmpliSeq BRCA1/BRCA2 Panel (Life Technologies). The whole process was performed following the manufacturer’s instructions with minor modifications as previously described15,16. Each library was barcoded using Ion Xpress Barcode Adapters kit (Life Technologies) and its concentration was determined with Ion Library TaqMan Quantitation Kit (Life Technologies). For template preparation, emulsion PCR and enrichment were carried out using Ion OneTouch 2 system and the Ion OneTouch ES Instrument (Life Technologies) with Ion PGM Hi-Q View OT2 Kit (Life Technologies). An average of six prepared barcoded samples were loaded into a single Ion 316 Chip v2 (Life Technologies) per sequencing run. The sequencing reactions were performed in the Ion torrent PGM System (Life Technologies) using sequencing Ion PGM Hi_Q view sequencing kit (Life Technologies). The generated data on Ion PGM were processed through on Torrent Suite, version 5.0.4, running on a local server (Life Technologies). The VCF files were uploaded to Ion Reporter (Life Technologies) and annotated for single-nucleotide variants (SNV), insertions and deletions (indels), and splice site changes. Reads were visualized with the Integrative Genomics Viewer (IGV) and Alamut (Interactive Biosoftware, Rouen, France).

### Evaluation of the variants clinical significance

Variants that are clearly classified in the Breast Cancer Information Core (BIC) database, Human Gene Mutation Database Pro (HGMD pro) and ClinVar databases, and not previously described as pathogenic in the literature were evaluated based on the guidelines and standards for interpretation of sequence variants recommended by the American College of Medical Genetics and Genomics (ACMG)17.

### DNA Sanger sequencing

Sanger sequencing was performed for validating selected candidate variants identified by NGS. Primers flanking the candidate loci were designed using Primer 3.0 and synthesized by IDT, CA, USA (Supplementary table 1). PCR amplification was performed using Platinum PCR SuperMix (Invitrogen, CA, USA). Direct automated sequencing for the purified PCR products was performed on an ABI 3500 Capillary DNA Sequencer using the BigDye Terminator Cycle Sequencing kit (Applied Biosystems, CA, USA). The sequence chromatograms were visualized by Sequencing Analysis SeqA software (Applied Biosystems) and analyzed by Chromas Pro software (Technolysium Ltd., South Brisbane, Australia).

### Statistical analysis

Collected Data was analyzed using R Statistical Computing Software version 3.4.3 (R Foundation for Statistical Computing, Vienna, Austria). Descriptive statistics including means and standard deviations are reported for numerical patient characteristics while frequencies and percentages are reported for categorical characteristics. Predictors of BRCA1 and BRCA2 mutations were identified through multiple linear regression with stepwise selection. A significance level of 0.05 is used throughout the analysis.

### Results

Of the 200 patients included in the study; 192 of them had breast cancer as the primary cancer site, while the remaining eight cases ovarian cancer as the primary site. 7.5% of them (15 patients) had bilateral breast cancer and 3% (6 patients) had recurrent cancer. The mean age of the study population was 43.6 ± 2.7 with an age interval of 22 to 70. Regarding clinical characteristics of participants (Table 1), 45% of patients were diagnosed at stage III or IV, while 32% of them were diagnosed at stage II. 26.5% had distant and 52.5% had regional metastasis at diagnosis. Large proportion (83%) of them had tumor size of 2 cm or more at time of diagnosis. HER2 receptor status was positive in 24.5% and negative in 64.5% of study participants, while the status was not reported for 3.5% of the cases (Table 1).Table 1Clinical characteristics of study participants.ValueNRaw %Valid %Cumulative %Primary siteBreast192969696Ovaries844100Site of cancerBilateral157.57.57.5Left92464653.5Right9346.546.5100Stage at diagnosis for breast cancerI46232323II64323255III34171772IV562828100MetastasisNo148747474Yes522626100Recurrence since baselineNo194979797Yes633100Laterality for breast cancerBi16888Uni1849292100Tumor size at histological examination(< 2 cm)34171717(> = 2 cm)1668383100Extent of diseaseDistant5326.526.626.6Local4120.520.647.2Regional10552.552.8100Oestrogen Receptors statusNegative73.53.63.6Positive2914.514.918.6Unknown1587981.4100ERNegative3718.519.119.1Positive13969.571.690.7Unknown1899.3100PRNegative462323.723.7Positive13366.568.692.3Unknown157.57.7100Hert-NeuNegative12964.566.866.8Positive4924.525.492.2Unknown157.57.8100E cadherinNegative1899.39.3Positive1688.217.5Unknown1608082.5100

Clinical characteristics of study participants.

The overall prevalence of recurrent and novel mutations (pathogenic and variants of uncertain significance (VUS)) in BRCA1/ BRCA2 genes was 18% (Table 2). The prevalence of recurrent pathogenic mutations was 14.5%, while the prevalence of novel mutations (pathogenic and VUS) was 3.5%. Two confirmed novel pathogenic mutations were identified in BRCA2 genes. Regarding site of mutation, 7.5% (15 patients) had pathogenic mutations in BRCA1 gene and 7% (14 patients) had pathogenic mutations in BRCA2 gene. Five VUS novel variants in BRCA2 gene were identified in this study. Regarding previously reported VUS, this study detected 6 in BRCA1 genes and 9 in BRCA2 genes (Table 3, Supplementary table 2).Table 2Prevalence of mutations pathogenic and VUS mutations.CategoryNumber of patientsPrevalenceRecurrent mutationsBRCA1 Positive157.50%BRCA2 Positive147.00%BRCA1 or BRCA2 Positive2914.50%Possible (recurrent and novel) mutationsBRCA1 Positive73.50%BRCA2 Positive147.00%BRCA1 or BRCA2 Positive2110.50%Recurrent and novel (VUS and pathogenic) mutationsBRCA1 Positive157.50%BRCA2 Positive2110.50%BRCA1 or BRCA2 Positive3618.00%Table 3Pathogenic BRCA1/BRCA2 variants found in breast/ovarian cancer patients (n = 200).VariantCase Freq /ZygosityType of cancerMutation DatabasedbSNP IDProtein PredictionMAFgnomAD (%)ReferenceExon /IntronHGVS cDNAHGVS aaVariant EffectClinVarBICHGMD(Accession #)SIFTPolyPhen-2Mutation TasterCONDELBRCA1: pathogenic variantsE18c.5186C>Ap.Ala1729GluMissense1 (0.5%) HetBreastPathogenicCIDMCM950153rs28897696DeleteriousProbably damagingDisease causingDeleterious0.000421E18c.5158C>Tp.Arg1720TrpMissense1 (0.5%) HetBreastPathogenicCIDMCM041706rs55770810DeleteriousProbably damagingDisease causingDeleterious0.00221E10c.4065_4068delTCAAp.Asn1355Lysfs*10Frameshift2 (1%) HetBreastPathogenicCIDMCD941619rs80357508––––0.000822E10c.1224delAp.Val409*Nonsense1 (0.5%) HetOvarianPathogenicN/ADMCD159317rs879255320––––N/A23E19c.5224C>Tp.Gln1742*Nonsense1 (0.5%) HetOvarianPathogenicN/ADMCM1211123N/A––––N/A24E3c.121C>Tp.His41TyrMissense2 (1%)HetBreastN/AN/ADMCM1712699N/ADeleteriousProbably damagingDisease causingDeleteriousN/A25BRCA2: pathogenic variantsE10c.2254_2257delGACTp.Asp752Phefs*19Frameshift2 (1%) HetBreastPathogenicCIDM CD023496rs80359326––––N/A28E11c.5042_5043delTGp.Val1681Glufs*7Frameshift1 (0.5%) HetBreastPathogenicCIDMCD082063rs80359478––––N/A30E11c.5351dupAp.Asn1784Lysfs*3Frameshift1 (0.5%) HetBreastPathogenicCIDMCI004026rs80359508––––N/A23E11c.6634_6637delTGTTp.Cys2212Leufs*16Frameshift2 (1%) HetBreastPathogenicN/ADMCD077812rs397507871––––N/A24E11c.6224_6225delAAp.Lys2075Serfs*2Frameshift1 (0.5%) HetOvarianN/AN/AN/AN/A––––N/ANovelE21c.8696A>Gp.Gln2899ArgMissense1 (0.5%) HetBreastN/AN/AN/AN/ADeleteriousProbably damagingDisease causingDeleteriousN/ANovelBIC Breast Cancer Information Core, HGMD Human Gene Mutation Database, MAF Minor allele frequency, E Exon, Het Heterozygous, Hom Homozygous, CI Clinically Important, DM Disease causing mutation, N/A Not available.

Prevalence of mutations pathogenic and VUS mutations.

Pathogenic BRCA1/BRCA2 variants found in breast/ovarian cancer patients (n = 200).

DM

CM950153

DM

CM041706

DM

CD941619

DM

CD159317

DM

CM1211123

DM

CM1712699

DM

CD082063

DM

CI004026

DM

CD077812

BIC Breast Cancer Information Core, HGMD Human Gene Mutation Database, MAF Minor allele frequency, E Exon, Het Heterozygous, Hom Homozygous, CI Clinically Important, DM Disease causing mutation, N/A Not available.

Figure 1 shows mutations in BRCA1/BRCA2 according to patients’ selection criteria. Participants with relatives with ovarian cancer had higher prevalence of mutations compared with other groups. Regression analysis results showed that E cadherin, PR, ER, oestrogen receptors status, laterality and recurrence since baseline were the statistically significant predictors of the presence of BRCA1 mutations. On the other hand, E cadherin, laterality and recurrence since baseline were statistically significant predictors of BRCA2 mutations.Figure 1BRCA1 and BRCA2 variants versus inclusion criteria. Top BRCA1 patients, middle BRCA2 patients and bottom BRCA1 and BRCA2 patients. Criteria code combined: 1: Female breast cancer patients younger than the age of 35, 2: At least the following female breast cancers only in the family, 3: Families containing one relative with ovarian cancer at any age and, on the same side of the family, 4: Families affected by bilateral cancer (each breast cancer has the same count value as one relative, 5: Families containing male breast cancer at any age and, on the same side of the family.

BRCA1 and BRCA2 variants versus inclusion criteria. Top BRCA1 patients, middle BRCA2 patients and bottom BRCA1 and BRCA2 patients. Criteria code combined: 1: Female breast cancer patients younger than the age of 35, 2: At least the following female breast cancers only in the family, 3: Families containing one relative with ovarian cancer at any age and, on the same side of the family, 4: Families affected by bilateral cancer (each breast cancer has the same count value as one relative, 5: Families containing male breast cancer at any age and, on the same side of the family.

BRCA1 and BRCA2 coding regions and intronic boundaries were amplified. The PCR was performed using the 167 primer pairs. Ion Torrent PGM generated 150 bp average sequence reads length from the sequencer, with 455,768 bp mean target read, 97.5% Target base coverage at 100X, 96.36% uniformity of base coverage, 90.18% on targeted region with a 2,738X average base coverage depth of both genes.

BRCA1/BRCA2 sequencing results revealed 37 different patients with pathogenic and VUS. Seven of those participants had novel variants that do not exist in the literature, diseases and population databases (Table 3, Supplementary table 2). As listed in Table 3, six BRCA1 variants were pathogenic, two of them were nonsense, one frameshift and three missense variants. In addition, six BRCA1 variants were VUS; four of them were missense variants, one VUS was synonymous, and another was a new splice site change (c.5396-6 T > C) that was present for the first time in our breast cancer patient (Fig. 2, Supplementary table 2). BRCA2 germline variants are presented in Table 3. Two pathogenic BRCA2 variants (p.K2075S fs*2; c.6224_6225delAA and p.Q2899R; c.8696A > G) were novel, they are neither registered in the disease nor the population databases. Those two novel pathogenic variants were further validated by Sanger sequencing (Supplementary Fig. 1 a-b). Four others recurrent BRCA2 pathogenic variants were identified. They were registered in diseases databases but absent from the population databases gnomAD indicating they are rare variants (Fig. 2 and Table 3). The duplication variant in BRCA2 (p.N1784K fs*3; c.5351dupA) was in a homopolymer region and therefore we validated it through Sanger sequencing (Supplementary Fig. 1c).We also found five distinct novel variant in BRCA2 and classified them as VUS (p.S958R; c.2874 T > G, p.I1000L; c.2998A > C, p.H1525L; c.4574A > T, p. = (p.P2883P), c.8649A > T, p.T3204I; c.9611C > T). In addition, our cohort had nine variants that were classified as VUS (Supplementary table 2). However, the founder variants of BRCA1 (c.185delAG, BRCA1 c.5382insC), and BRCA2 (c.6174delT) in the Ashkenazi Jewish population, were absent in the current cohort, indicating that BRCA variants vary among different populations18.Figure 2Schematic diagram of BRCA1 and BRCA2 proteins with the position of all identified pathogenic variants. Novel variants are marked with a caret (^).

Schematic diagram of BRCA1 and BRCA2 proteins with the position of all identified pathogenic variants. Novel variants are marked with a caret (^).

### Participants

Of the 200 patients included in the study; 192 of them had breast cancer as the primary cancer site, while the remaining eight cases ovarian cancer as the primary site. 7.5% of them (15 patients) had bilateral breast cancer and 3% (6 patients) had recurrent cancer. The mean age of the study population was 43.6 ± 2.7 with an age interval of 22 to 70. Regarding clinical characteristics of participants (Table 1), 45% of patients were diagnosed at stage III or IV, while 32% of them were diagnosed at stage II. 26.5% had distant and 52.5% had regional metastasis at diagnosis. Large proportion (83%) of them had tumor size of 2 cm or more at time of diagnosis. HER2 receptor status was positive in 24.5% and negative in 64.5% of study participants, while the status was not reported for 3.5% of the cases (Table 1).Table 1Clinical characteristics of study participants.ValueNRaw %Valid %Cumulative %Primary siteBreast192969696Ovaries844100Site of cancerBilateral157.57.57.5Left92464653.5Right9346.546.5100Stage at diagnosis for breast cancerI46232323II64323255III34171772IV562828100MetastasisNo148747474Yes522626100Recurrence since baselineNo194979797Yes633100Laterality for breast cancerBi16888Uni1849292100Tumor size at histological examination(< 2 cm)34171717(> = 2 cm)1668383100Extent of diseaseDistant5326.526.626.6Local4120.520.647.2Regional10552.552.8100Oestrogen Receptors statusNegative73.53.63.6Positive2914.514.918.6Unknown1587981.4100ERNegative3718.519.119.1Positive13969.571.690.7Unknown1899.3100PRNegative462323.723.7Positive13366.568.692.3Unknown157.57.7100Hert-NeuNegative12964.566.866.8Positive4924.525.492.2Unknown157.57.8100E cadherinNegative1899.39.3Positive1688.217.5Unknown1608082.5100

Clinical characteristics of study participants.

### Overall BRCA1/BRCA2 mutations

The overall prevalence of recurrent and novel mutations (pathogenic and variants of uncertain significance (VUS)) in BRCA1/ BRCA2 genes was 18% (Table 2). The prevalence of recurrent pathogenic mutations was 14.5%, while the prevalence of novel mutations (pathogenic and VUS) was 3.5%. Two confirmed novel pathogenic mutations were identified in BRCA2 genes. Regarding site of mutation, 7.5% (15 patients) had pathogenic mutations in BRCA1 gene and 7% (14 patients) had pathogenic mutations in BRCA2 gene. Five VUS novel variants in BRCA2 gene were identified in this study. Regarding previously reported VUS, this study detected 6 in BRCA1 genes and 9 in BRCA2 genes (Table 3, Supplementary table 2).Table 2Prevalence of mutations pathogenic and VUS mutations.CategoryNumber of patientsPrevalenceRecurrent mutationsBRCA1 Positive157.50%BRCA2 Positive147.00%BRCA1 or BRCA2 Positive2914.50%Possible (recurrent and novel) mutationsBRCA1 Positive73.50%BRCA2 Positive147.00%BRCA1 or BRCA2 Positive2110.50%Recurrent and novel (VUS and pathogenic) mutationsBRCA1 Positive157.50%BRCA2 Positive2110.50%BRCA1 or BRCA2 Positive3618.00%Table 3Pathogenic BRCA1/BRCA2 variants found in breast/ovarian cancer patients (n = 200).VariantCase Freq /ZygosityType of cancerMutation DatabasedbSNP IDProtein PredictionMAFgnomAD (%)ReferenceExon /IntronHGVS cDNAHGVS aaVariant EffectClinVarBICHGMD(Accession #)SIFTPolyPhen-2Mutation TasterCONDELBRCA1: pathogenic variantsE18c.5186C>Ap.Ala1729GluMissense1 (0.5%) HetBreastPathogenicCIDMCM950153rs28897696DeleteriousProbably damagingDisease causingDeleterious0.000421E18c.5158C>Tp.Arg1720TrpMissense1 (0.5%) HetBreastPathogenicCIDMCM041706rs55770810DeleteriousProbably damagingDisease causingDeleterious0.00221E10c.4065_4068delTCAAp.Asn1355Lysfs*10Frameshift2 (1%) HetBreastPathogenicCIDMCD941619rs80357508––––0.000822E10c.1224delAp.Val409*Nonsense1 (0.5%) HetOvarianPathogenicN/ADMCD159317rs879255320––––N/A23E19c.5224C>Tp.Gln1742*Nonsense1 (0.5%) HetOvarianPathogenicN/ADMCM1211123N/A––––N/A24E3c.121C>Tp.His41TyrMissense2 (1%)HetBreastN/AN/ADMCM1712699N/ADeleteriousProbably damagingDisease causingDeleteriousN/A25BRCA2: pathogenic variantsE10c.2254_2257delGACTp.Asp752Phefs*19Frameshift2 (1%) HetBreastPathogenicCIDM CD023496rs80359326––––N/A28E11c.5042_5043delTGp.Val1681Glufs*7Frameshift1 (0.5%) HetBreastPathogenicCIDMCD082063rs80359478––––N/A30E11c.5351dupAp.Asn1784Lysfs*3Frameshift1 (0.5%) HetBreastPathogenicCIDMCI004026rs80359508––––N/A23E11c.6634_6637delTGTTp.Cys2212Leufs*16Frameshift2 (1%) HetBreastPathogenicN/ADMCD077812rs397507871––––N/A24E11c.6224_6225delAAp.Lys2075Serfs*2Frameshift1 (0.5%) HetOvarianN/AN/AN/AN/A––––N/ANovelE21c.8696A>Gp.Gln2899ArgMissense1 (0.5%) HetBreastN/AN/AN/AN/ADeleteriousProbably damagingDisease causingDeleteriousN/ANovelBIC Breast Cancer Information Core, HGMD Human Gene Mutation Database, MAF Minor allele frequency, E Exon, Het Heterozygous, Hom Homozygous, CI Clinically Important, DM Disease causing mutation, N/A Not available.

Prevalence of mutations pathogenic and VUS mutations.

Pathogenic BRCA1/BRCA2 variants found in breast/ovarian cancer patients (n = 200).

DM

CM950153

DM

CM041706

DM

CD941619

DM

CD159317

DM

CM1211123

DM

CM1712699

DM

CD082063

DM

CI004026

DM

CD077812

BIC Breast Cancer Information Core, HGMD Human Gene Mutation Database, MAF Minor allele frequency, E Exon, Het Heterozygous, Hom Homozygous, CI Clinically Important, DM Disease causing mutation, N/A Not available.

Figure 1 shows mutations in BRCA1/BRCA2 according to patients’ selection criteria. Participants with relatives with ovarian cancer had higher prevalence of mutations compared with other groups. Regression analysis results showed that E cadherin, PR, ER, oestrogen receptors status, laterality and recurrence since baseline were the statistically significant predictors of the presence of BRCA1 mutations. On the other hand, E cadherin, laterality and recurrence since baseline were statistically significant predictors of BRCA2 mutations.Figure 1BRCA1 and BRCA2 variants versus inclusion criteria. Top BRCA1 patients, middle BRCA2 patients and bottom BRCA1 and BRCA2 patients. Criteria code combined: 1: Female breast cancer patients younger than the age of 35, 2: At least the following female breast cancers only in the family, 3: Families containing one relative with ovarian cancer at any age and, on the same side of the family, 4: Families affected by bilateral cancer (each breast cancer has the same count value as one relative, 5: Families containing male breast cancer at any age and, on the same side of the family.

BRCA1 and BRCA2 variants versus inclusion criteria. Top BRCA1 patients, middle BRCA2 patients and bottom BRCA1 and BRCA2 patients. Criteria code combined: 1: Female breast cancer patients younger than the age of 35, 2: At least the following female breast cancers only in the family, 3: Families containing one relative with ovarian cancer at any age and, on the same side of the family, 4: Families affected by bilateral cancer (each breast cancer has the same count value as one relative, 5: Families containing male breast cancer at any age and, on the same side of the family.

### DNA sequencing and variant analysis

BRCA1 and BRCA2 coding regions and intronic boundaries were amplified. The PCR was performed using the 167 primer pairs. Ion Torrent PGM generated 150 bp average sequence reads length from the sequencer, with 455,768 bp mean target read, 97.5% Target base coverage at 100X, 96.36% uniformity of base coverage, 90.18% on targeted region with a 2,738X average base coverage depth of both genes.

BRCA1/BRCA2 sequencing results revealed 37 different patients with pathogenic and VUS. Seven of those participants had novel variants that do not exist in the literature, diseases and population databases (Table 3, Supplementary table 2). As listed in Table 3, six BRCA1 variants were pathogenic, two of them were nonsense, one frameshift and three missense variants. In addition, six BRCA1 variants were VUS; four of them were missense variants, one VUS was synonymous, and another was a new splice site change (c.5396-6 T > C) that was present for the first time in our breast cancer patient (Fig. 2, Supplementary table 2). BRCA2 germline variants are presented in Table 3. Two pathogenic BRCA2 variants (p.K2075S fs*2; c.6224_6225delAA and p.Q2899R; c.8696A > G) were novel, they are neither registered in the disease nor the population databases. Those two novel pathogenic variants were further validated by Sanger sequencing (Supplementary Fig. 1 a-b). Four others recurrent BRCA2 pathogenic variants were identified. They were registered in diseases databases but absent from the population databases gnomAD indicating they are rare variants (Fig. 2 and Table 3). The duplication variant in BRCA2 (p.N1784K fs*3; c.5351dupA) was in a homopolymer region and therefore we validated it through Sanger sequencing (Supplementary Fig. 1c).We also found five distinct novel variant in BRCA2 and classified them as VUS (p.S958R; c.2874 T > G, p.I1000L; c.2998A > C, p.H1525L; c.4574A > T, p. = (p.P2883P), c.8649A > T, p.T3204I; c.9611C > T). In addition, our cohort had nine variants that were classified as VUS (Supplementary table 2). However, the founder variants of BRCA1 (c.185delAG, BRCA1 c.5382insC), and BRCA2 (c.6174delT) in the Ashkenazi Jewish population, were absent in the current cohort, indicating that BRCA variants vary among different populations18.Figure 2Schematic diagram of BRCA1 and BRCA2 proteins with the position of all identified pathogenic variants. Novel variants are marked with a caret (^).

Schematic diagram of BRCA1 and BRCA2 proteins with the position of all identified pathogenic variants. Novel variants are marked with a caret (^).

### Discussion

This study was set out to identify BRCA1/BRCA2 mutations in patients at high risk of familial breast and ovarian cancer using a systematic approach. Our study showed a high prevalence of 18.0% for positive mutations in BRCA1 (7.5%) or BRCA2 (10.5%) genes. Two novel pathogenic mutations and five novel VUS variants were identified in this study, in addition to identifications of patients with recurrent pathogenic and VUS mutations. These figures are higher than recent data from neighboring Arab Peninsula of 16% or from other Asian countries like Japan of 7%15–19. This could be related to our inclusion criteria based on NICE guidelines for being at high risk of familial breast or ovarian cancers.

Out of 12 pathogenic variants detected in this study in both BRCA1/BRCA2, five pathogenic variants were reported previously among Caucasian, Western populations and Latin American. These variants including c.5186C > A,, c.4065_4068delTCAA in BRCA1 and c.5042_5043delTG, c.5351dupA in BRCA220–24.Additionally, three pathogenic variants were previously reported among Asia population. Both c.1224delA and c.5224C > T in BRCA1 and c.6634_6637delTGTT in BRCA2 were originally reported in Japan and China populations25,26. However, different recurrent pathogenic variants were previously reported among Arab population. The c.121C > T pathogenic variant in BRCA1and the c.2254_2257delGACT pathogenic variant in BRCA2 were recently reported in Palestinian patients27,28. Additionally, the c.5158C > T pathogenic variant in BRCA1 was previously reported in Morocco and Saudi Arabia13. Both c.6224_6225delAA and c.8696A > G variants in BRCA2 have not been described in the BIC database and other databases including HGMD professional and ClinVar. Therefore, they are considered to be novel. It’s worth to mention that four variants in both BRCA1 and BRCA2 were common in more than one patient (Table 3)29–32.

Positive patients in our study (recurrent or novel mutations) were diagnosed at advanced stages of breast cancer and ovarian cancers, although they are at high risk of familial cancer. This preventable delay is expected to have a major negative impact on morbidity, morbidity, and cost of management. Such patients could have received preventive cancer treatment giving their very strong family history, should they have been tested properly in due time. Establishing the mutational profile in our region will allow directed testing of high-risk families, therefore increasing the benefit while reducing costs. This is an agreement with Laitman et al. who identified several highly-recurring mutations in the region13.

Our participants presented with breast cancer at an early age with a mean of 43.7 ± 2.7. Breast cancer at an early age is more likely to be associated with an increased familial risk, particularly in women with a germline BRCA1 mutation33. In a study of women with breast cancer diagnosed before age 30, BRCA1/BRCA2 and TP53 mutations were found in about half who had strong family histories of breast cancer and in less than 10% of women with non-familial breast cancer34. A young age at diagnosis for breast cancer patients in Jordan and the Arab world suggests a high probability of familial predisposition. A systematic review on age at diagnosis for breast cancer in the Arab world revealed that average age was 48 (SD = 2.8), range 43–52, median 48.5 and mode 45. These figures are based on data from 28 articles that included years amongst the 7455 patients. The percentage of patients that were younger than 50 years old was reported in 11 articles from 8 countries and included 5144 patients; 65.5% (SD = 11) who were less than 50 years old (range 49–78%, median = 66%). It is recommended that further research to be conducted among young breast and/or ovarian cancer patients in the Arab world to examine the prevalence of known mutations and to search for new potential pathogenic mutations.

Identification of mutations in BRCA1/BRCA2 genes leading to familial breast cancer has had a major impact on prevention of breast and ovarian cancer through development of clinical practice guidelines for management and referral systems for women with a personal or family history of breast or ovarian cancer35. Results from some countries (e.g. U.K., France and Canada) and specific cities (Manchester, Marseilles, and Montreal) provided excellent examples of the clinical and economic effectiveness of these preventive services36. Management of patients start as early as the age of 25 with clear guidelines for referral and management37. In many developing countries, including Jordan, this approach has not been put in place38. It has been well-reported in previous studies that positive carriers for BRCA1/BRCA2 have high cumulative risk estimates for developing breast cancer reaching 72% for BRCA1 carriers and 69% for BRCA2 carrier by age of 805. These patients could undergo preventive measures following timely identification of mutations, reducing morbidity and mortality arising from breast and ovarian cancers.

### Conclusion

This study assessed the prevalence of reported mutations and identified novel mutations for BRCA1/BRCA2 genes. Giving the high prevalence of positive mutations in BRCA1/BRCA2 genes, it is essential to establish a service for familial breast and ovarian cancer service in Jordan and the MENA region, that includes genetic counselling and early genetic screening.

### Supplementary information

Supplementary Table S2Supplementary Table S1Supplementary LegendSupplementary Figure S1

Supplementary Table S2

Supplementary Table S1

Supplementary Legend

Supplementary Figure S1

Supplementary Table S2Supplementary Table S1Supplementary LegendSupplementary Figure S1

Supplementary Table S2

Supplementary Table S1

Supplementary Legend

Supplementary Figure S1



# SUPPLEMENTAL FILE 1: 41598_2020_Article_74250.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)