# MAIN TEXT

## Reclassification of BRCA1 and BRCA2 variants found in ovarian epithelial, fallopian tube, and primary peritoneal cancers

### Abstract

ObjectiveWe investigated the proportions of and reclassified BRCA1/2 variants of unknown significance (VUS) in Korean patients with epithelial ovarian, tubal, and primary peritoneal cancers.MethodsData from 805 patients who underwent genetic testing for BRCA1/2 from January 1, 2006 to August 31, 2018 were included. The VUS in BRCA1/2 were reclassified using the 2015 American College of Medical Genetics and Genomics and the Association for Molecular Pathology standards and guidelines.ResultsA BRCA1 pathogenic variant was found in 17.0% (137/805) of the patients, and BRCA1 VUS were found in 15.9% (128/805) of the patients. Further, 8.7% (69/805) of the patients possessed a BRCA2 pathogenic variant and 18.4% (148/805) of the patients possessed BRCA2 VUS. Fifty-three specific BRCA1 VUS were found and 20 were further reclassified as benign (n=11), likely benign (n=5), likely pathogenic (n=3), and pathogenic (n=1). The remaining 33 remained classified as VUS. For BRCA2, 55 specific VUS were detected; among these, 14 were reclassified as benign or likely benign, and 2 were reclassified as likely pathogenic. Among the 805 patients, 195 were found to have only VUS and no pathogenic variants (PV), and 41.5% (81/195) were reclassified as benign or likely benign, and 10.3% (20/195) as pathogenic or likely pathogenic variants.ConclusionsApproximately 33.3% (36/108) of the specific BRCA1/2 variants analyzed in this study that were initially classified as VUS over a 13-year period were reclassified. Among these, 5.6% (6/108) were reclassified as pathogenic or likely pathogenic variants.

### INTRODUCTION

Ovarian cancer is often deadly and is ranked second in terms of mortality and third in terms of incidence among gynecological malignancies worldwide [1]. Particularly in Korea, where the prevalence of ovarian cancer is increasing, it is already the second most common gynecological malignancy [2]. Ovarian carcinoma is the most common hereditary gynecological cancer, and BRCA1 and BRCA2 are associated with most hereditary ovarian cancers. An inherited disposition is found in 10% of ovarian cancers [3]. The average cumulative risk for developing ovarian cancer for BRCA1-mutation carriers by the age of 70 years is 39% (95% confidence interval=18%–54%), and the corresponding estimate for BRCA2 is 11% (2.4%–19%) [4].

Accurate risk assessment for potential ovarian cancer cases is vital for cancer prevention, and a suitable management program for the proband in each family is important. Thus, detection of BRCA1/2 mutation is an indispensable part of ovarian cancer management. Therefore, some study groups, such as the National Comprehensive Cancer Network, recommend that patients with ovarian cancer in their family should have a genetic risk evaluation and BRCA1/2 testing [5].

Genetic testing categorizes ovarian cancer into three main types: benign, pathologic, or uncertain clinical significance; each type has various clinical implications. BRCA1/2 variants which have not been established or ruled out as a disease risk are referred to as “variants of unknown significance” (VUS) [6]. The field of genetic testing is rapidly expanding, and interpretation of results is a greater challenge now that a larger number of genetic VUS have been isolated [7]. In, South Korea, which is a unitary country, approximately 3.9%–24.6% of patients tested for BRCA1/2 possessed one or more VUS, primarily in breast cancer patients [89101112]. Although BRCA1 and BRCA2 gene testing plays a pivotal role in selecting treatment options for patients with hereditary gynecological cancer, VUS therapeutic decision-making faces constraints. Moreover, the clinical significance of each VUS has not yet been reported.

Although a substantial number of studies have been conducted for VUS re-classification and multifarious tools, such as in silico tests, have been utilized to confirm the pathogenicity of VUS, standardized methods have not yet been established. In this study, we aimed to elucidate BRCA1/2 VUS in gynecological cancer patients and reclassify them using the 2015 American College of Medical Genetics and Genomics and the Association for Molecular Pathology standards and guidelines (ACMG/AMP 2015 guidelines).

### MATERIALS and METHODS

Medical records, including genetic testing of patients with gynecological cancer, were retrospectively reviewed between January 2006 and August 2018. A total of 870 patients were tested, and their BRCA1 or BRCA2 status was recorded.

All patients with epithelial ovarian cancer, fallopian tube cancer, and primary peritoneal carcinoma were included, while patients with borderline malignancy of ovarian tumor, non-epithelial ovarian cancer, and breast cancer without gynecological malignancy were excluded (Fig. 1). However, a familial mutational test for the member proband with gynecological cancer was not incorporated into this analysis.

ACMG/AMP, American College of Medical Genetics and Genomics and the Association for Molecular Pathology; POFT, peritoneal, ovarian, and fallopian tube; PV, pathogenic variants; VUS, variants of unknown significance.

This study adhered to the ethical tenets of the Declaration of Helsinki and was approved by the Institutional Review Board (IRB) of the National Cancer Center in Goyang, Korea (IRB number: 2018-0259). The need for informed consent was waived because of the low risk designated by the IRB.

Germline or somatic mutations of BRCA1/2 were tested by Sanger sequencing or by next generation sequencing (NGS). Briefly, genomic DNA was extracted from the peripheral blood of participants using a Chemagic DNA Blood 200 Kit (Chemagen, Baesweiler, Germany) or a QIAamp DNA Blood Mini kit (Qiagen, Valencia, CA, USA). Amplified PCR products were sequenced on an ABI 3500xl analyzer (Applied Biosystems, Foster City, CA, USA), using the BigDye Terminator v3.1 Cycle Sequencing Kit. Sequences were analyzed using Sequencer v5.0 software. NGS was done on the Illumina MiSeqDX (Illumina Inc., San Diego, CA, USA) generating 2×150 bp reads. Alignment was conducted with BWA-MEM (version 0.7.10), duplicated reads were marked with Picard (version 1.138) and local alignment, base quality recalibration, and variant calling were performed with the Genome Analysis Tool kit (GATK, version 3.5), SAMtools (version 0.1.19), FreeBayes (v0.9.21-26-gbfd9832) and Scalpel (version 0.5.3). Variant annotation was conducted with a Variant Effect Predictor and dbNSFP. All genetic variants were determined according to the Breast Cancer Information Core database and both BIC database and ClinVar (https://www.ncbi.nlm.nih.gov/clinvar/) database were used from the 2014 onwards. All variants were described according to HUGO-approved systematic nomenclature.

According to the ACMG/AMP 2015 guidelines, variant pathogenicity was reviewed (Fig. 2). The detailed methods used in this study were described in Supplementary Tables 1 and 2.

BA, benign stand-alone evidence; BP, benign supporting evidence; BS, benign strong evidence; PM, pathogenic moderate evidence; PP, pathogenic supporting evidence; PS, pathogenic strong evidence; PV, pathogenic variants evidence; PVS, pathogenic very strong evidence; VUS, variants of unknown significance.

The variant pathogenicity in the absence of controls (or at an extremely low frequency, if recessive) was determined using the Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium was assigned pathogenic moderate (PM) evidence 2. Databases were used for benign strong evidence 1 (BS1) when the allele frequency was greater than 1% and less than 5%.

For pathogenic strong evidence 4 (PS4), we conducted a literature survey and looked up information from well-established previous case-control studies. If association analysis showed OR >5 and its confidential interval >1, according to the ACMG/AMP guidelines, we regarded the results as PS4 (Supplementary Table 1).

For benign supporting (BP) evidence 6 or pathogenic supporting (PP) evidence 5, we used the ClinVar database as a source. The database uses a five-tier system, “pathogenic,” “likely pathogenic,” “uncertain significance,” “conflicting,” “likely benign,” and “benign.” We decided to classify “pathogenic” and “likely pathogenic” as evidence of PP5, and “likely benign” and “benign” as evidence of BP6.

BP4 and PP3 required multiple lines of computational evidence to support any effect on the gene or gene product; thus, we used MutationTaster, Polyphen-2, SIFT, Align-GVGD, and PANTHER. If more than 4 out of 5 of the results matched, we regarded the results as BP4 or PP3. Human Splicing Finder was used for splicing variants of BP7. We conducted a literature survey for information from previous functional studies on results regarding PP3 and BP4.

Finally, we reclassified the patients with reclassified VUS. If they have multiple VUS, we assigned the status which is more pathogenic. The detailed methods used in this study were described in Supplementary Tables 1 and 2.

### 1. Subjects

Medical records, including genetic testing of patients with gynecological cancer, were retrospectively reviewed between January 2006 and August 2018. A total of 870 patients were tested, and their BRCA1 or BRCA2 status was recorded.

All patients with epithelial ovarian cancer, fallopian tube cancer, and primary peritoneal carcinoma were included, while patients with borderline malignancy of ovarian tumor, non-epithelial ovarian cancer, and breast cancer without gynecological malignancy were excluded (Fig. 1). However, a familial mutational test for the member proband with gynecological cancer was not incorporated into this analysis.

ACMG/AMP, American College of Medical Genetics and Genomics and the Association for Molecular Pathology; POFT, peritoneal, ovarian, and fallopian tube; PV, pathogenic variants; VUS, variants of unknown significance.

This study adhered to the ethical tenets of the Declaration of Helsinki and was approved by the Institutional Review Board (IRB) of the National Cancer Center in Goyang, Korea (IRB number: 2018-0259). The need for informed consent was waived because of the low risk designated by the IRB.

### 2. BRCA1/2 sequencing

Germline or somatic mutations of BRCA1/2 were tested by Sanger sequencing or by next generation sequencing (NGS). Briefly, genomic DNA was extracted from the peripheral blood of participants using a Chemagic DNA Blood 200 Kit (Chemagen, Baesweiler, Germany) or a QIAamp DNA Blood Mini kit (Qiagen, Valencia, CA, USA). Amplified PCR products were sequenced on an ABI 3500xl analyzer (Applied Biosystems, Foster City, CA, USA), using the BigDye Terminator v3.1 Cycle Sequencing Kit. Sequences were analyzed using Sequencer v5.0 software. NGS was done on the Illumina MiSeqDX (Illumina Inc., San Diego, CA, USA) generating 2×150 bp reads. Alignment was conducted with BWA-MEM (version 0.7.10), duplicated reads were marked with Picard (version 1.138) and local alignment, base quality recalibration, and variant calling were performed with the Genome Analysis Tool kit (GATK, version 3.5), SAMtools (version 0.1.19), FreeBayes (v0.9.21-26-gbfd9832) and Scalpel (version 0.5.3). Variant annotation was conducted with a Variant Effect Predictor and dbNSFP. All genetic variants were determined according to the Breast Cancer Information Core database and both BIC database and ClinVar (https://www.ncbi.nlm.nih.gov/clinvar/) database were used from the 2014 onwards. All variants were described according to HUGO-approved systematic nomenclature.

### 3. Reclassification of VUS

According to the ACMG/AMP 2015 guidelines, variant pathogenicity was reviewed (Fig. 2). The detailed methods used in this study were described in Supplementary Tables 1 and 2.

BA, benign stand-alone evidence; BP, benign supporting evidence; BS, benign strong evidence; PM, pathogenic moderate evidence; PP, pathogenic supporting evidence; PS, pathogenic strong evidence; PV, pathogenic variants evidence; PVS, pathogenic very strong evidence; VUS, variants of unknown significance.

The variant pathogenicity in the absence of controls (or at an extremely low frequency, if recessive) was determined using the Exome Sequencing Project, 1000 Genomes Project, or Exome Aggregation Consortium was assigned pathogenic moderate (PM) evidence 2. Databases were used for benign strong evidence 1 (BS1) when the allele frequency was greater than 1% and less than 5%.

For pathogenic strong evidence 4 (PS4), we conducted a literature survey and looked up information from well-established previous case-control studies. If association analysis showed OR >5 and its confidential interval >1, according to the ACMG/AMP guidelines, we regarded the results as PS4 (Supplementary Table 1).

For benign supporting (BP) evidence 6 or pathogenic supporting (PP) evidence 5, we used the ClinVar database as a source. The database uses a five-tier system, “pathogenic,” “likely pathogenic,” “uncertain significance,” “conflicting,” “likely benign,” and “benign.” We decided to classify “pathogenic” and “likely pathogenic” as evidence of PP5, and “likely benign” and “benign” as evidence of BP6.

BP4 and PP3 required multiple lines of computational evidence to support any effect on the gene or gene product; thus, we used MutationTaster, Polyphen-2, SIFT, Align-GVGD, and PANTHER. If more than 4 out of 5 of the results matched, we regarded the results as BP4 or PP3. Human Splicing Finder was used for splicing variants of BP7. We conducted a literature survey for information from previous functional studies on results regarding PP3 and BP4.

Finally, we reclassified the patients with reclassified VUS. If they have multiple VUS, we assigned the status which is more pathogenic. The detailed methods used in this study were described in Supplementary Tables 1 and 2.

### RESULTS

A total of 870 patients were tested for BRCA1/BRCA2 variants, regardless of the type of cancer. A total of 805 patients were included in this study, which was conducted between January 1, 2006 and August 31, 2018. Among the 805 patients, the Sanger method was used for 647 patients. In addition, next-generation sequencing (NGS) was used for 190 patients: 137 germline, 54 somatic, and 1 both germline and somatic NGS. Thirty-two patients were tested utilizing the Sanger method and NGS simultaneously. Five among these were tested via germline NGS and 27 via somatic NGS (Supplementary Table 3).

Among the 805 patients, 751 patients exhibited epithelial ovarian cancer, 12 primary peritoneal cancer, and 42 fallopian tube cancer. Patient characteristics, Histologic subtype, and surgical stages are depicted in Supplementary Tables 4, 7, 8, 9.

Total 97 BRCA1 or BRCA2 specific pathogenic variants (59 BRCA1 PV and 38 BRCA2 PV) were detected and 108 BRCA1 or BRCA2 specific VUS (53 BRCA1 VUS and 55 BRCA2 VUS) from the included 805 patients after removing duplicates (Supplementary Table 5). Among the 805 patients, 50.2% (404/805) exhibited only neutral variants that were neither pathogenic nor VUS. 25.6% (206/805) of patients were found to possess a pathogenic variant, 30.3% (244/805) VUS, and 6.1% (49/805) both PV and VUS simultaneously. In the BRCA1 gene, 17.0% (137/805) of patients exhibited PV, 15.9% (128/805) VUS, and 1.7% (14/805) both variants. Further, in the BRCA2 gene, 8.6% (69/805) of patients exhibited PV, 18.4% (148/805) VUS, and 1.4% (11/805) both variants. No patients were found to have BRCA1 PV, BRCA1 VUS, BRCA2 PV, and BRCA2 VUS simultaneously (Fig. 1).

In total, of the specific BRCA1 or BRCA2 VUS that were reclassified, 5.6% (6/108) were reclassified as pathogenic/likely pathogenic and 27.8% (30/108) were reclassified as benign/likely benign. 66.7% (72/108) of the specific BRCA1/2 VUS were not reclassified.

The BRCA1 VUS consisted of 25 missense, 10 synonymous, 17 intronic, and 1 in-frame deletion variants. The frequencies of the specific BRCA1 VUS ranged from 0.12% (1/805) to 2.9% (23/805). Using the ACMG/AMP 2015 guidelines, 11 specific VUS were reclassified as benign, 5 as likely benign, 3 as likely pathogenic (c.5215G>C, c.5363G>T, and c.5017_5019delCAC), and 1 as pathogenic (c.5339T>C). Thirty-three specific VUS were not reclassified (Table 1, Supplementary Fig. 1).

1000G (EAS), minor allele frequency from East-Asian population in the 1000 Genome Phase 3 database; ACMG/AMP, American College of Medical Genetics and Genomics and the Association for Molecular Pathology; B, benign; BA, benign stand-alone evidence; BP, benign supporting evidence; BS, benign strong evidence; c, coding sequence; CIP, conflicting interpretations of pathogenicity; ESP, Exome Sequencing Project; ExAC, Exome Aggregation Consortium; HGVS, Human Genome Variation Society; IVS, intervening sequence; LB, likely benign; LP, likely pathogenic; OV, observed variant(s) without frequency or population information; P, pathogenic; p, protein sequence; PM, pathogenic moderate evidence; PP, pathogenic supporting evidence; PS, pathogenic strong evidence; PVS, pathogenic very strong evidence; VUS, variants of unknown significance.

The 55 specific BRCA2 VUS consisted of 39 missense, 7 synonymous, 8 intronic, and one in-frame deletion variants. The frequencies of these specific variants ranged from 0.12% (1/805) to 3.4% (27/805). Six specific VUS were reclassified as benign, 8 as likely benign, 2 as likely pathogenic (c.9154C>T, c.8488-1G>A). Thirty-nine specific VUS were not reclassified. An overview of these results is found in (Table 2, Supplementary Fig. 1).

1000G (EAS), minor allele frequency from East-Asian population in the 1000 Genome Phase 3 database; ACMG/AMP, American College of Medical Genetics and Genomics and the Association for Molecular Pathology; B, benign; BA, benign stand-alone evidence; BP, benign supporting evidence; BS, benign strong evidence; c, coding sequence; CIP, conflicting interpretations of pathogenicity; ESP, Exome Sequencing Project; ExAC, Exome Aggregation Consortium; HGVS, Human Genome Variation Society; IVS, intervening sequence; LB, likely benign; LP, likely pathogenic; OV, observed variant(s) without frequency or population information; P, pathogenic; p, protein sequence; PM, pathogenic moderate evidence; PP, pathogenic supporting evidence; PS, pathogenic strong evidence; PVS, pathogenic very strong evidence; VUS, variants of unknown significance.

Excluding patients with both PV and VUS, there were 195 patients with only VUS. 82 were with BRCA1 VUS only, 90 were with BRCA2 VUS only and 23 were with BRCA1 and BRCA2 simultaneously (Fig. 1). Among the 195 patients, the status for 25.1% (49/195) changed to benign, that for 16.4% (32/195) changed to likely benign, that for 5.1% (10/195) changed to likely pathogenic, and that for 5.1% (10/195) changed to pathogenic. Each reclassification rate for BRCA1 VUS, BRCA2 VUS, and BRCA1 plus BRCA2 subset was reported in Table 3.

VUS, variants of unknown.

### 1. Reclassification of specific BRCA1 and BRCA2 “VUS”

In total, of the specific BRCA1 or BRCA2 VUS that were reclassified, 5.6% (6/108) were reclassified as pathogenic/likely pathogenic and 27.8% (30/108) were reclassified as benign/likely benign. 66.7% (72/108) of the specific BRCA1/2 VUS were not reclassified.

The BRCA1 VUS consisted of 25 missense, 10 synonymous, 17 intronic, and 1 in-frame deletion variants. The frequencies of the specific BRCA1 VUS ranged from 0.12% (1/805) to 2.9% (23/805). Using the ACMG/AMP 2015 guidelines, 11 specific VUS were reclassified as benign, 5 as likely benign, 3 as likely pathogenic (c.5215G>C, c.5363G>T, and c.5017_5019delCAC), and 1 as pathogenic (c.5339T>C). Thirty-three specific VUS were not reclassified (Table 1, Supplementary Fig. 1).

1000G (EAS), minor allele frequency from East-Asian population in the 1000 Genome Phase 3 database; ACMG/AMP, American College of Medical Genetics and Genomics and the Association for Molecular Pathology; B, benign; BA, benign stand-alone evidence; BP, benign supporting evidence; BS, benign strong evidence; c, coding sequence; CIP, conflicting interpretations of pathogenicity; ESP, Exome Sequencing Project; ExAC, Exome Aggregation Consortium; HGVS, Human Genome Variation Society; IVS, intervening sequence; LB, likely benign; LP, likely pathogenic; OV, observed variant(s) without frequency or population information; P, pathogenic; p, protein sequence; PM, pathogenic moderate evidence; PP, pathogenic supporting evidence; PS, pathogenic strong evidence; PVS, pathogenic very strong evidence; VUS, variants of unknown significance.

The 55 specific BRCA2 VUS consisted of 39 missense, 7 synonymous, 8 intronic, and one in-frame deletion variants. The frequencies of these specific variants ranged from 0.12% (1/805) to 3.4% (27/805). Six specific VUS were reclassified as benign, 8 as likely benign, 2 as likely pathogenic (c.9154C>T, c.8488-1G>A). Thirty-nine specific VUS were not reclassified. An overview of these results is found in (Table 2, Supplementary Fig. 1).

1000G (EAS), minor allele frequency from East-Asian population in the 1000 Genome Phase 3 database; ACMG/AMP, American College of Medical Genetics and Genomics and the Association for Molecular Pathology; B, benign; BA, benign stand-alone evidence; BP, benign supporting evidence; BS, benign strong evidence; c, coding sequence; CIP, conflicting interpretations of pathogenicity; ESP, Exome Sequencing Project; ExAC, Exome Aggregation Consortium; HGVS, Human Genome Variation Society; IVS, intervening sequence; LB, likely benign; LP, likely pathogenic; OV, observed variant(s) without frequency or population information; P, pathogenic; p, protein sequence; PM, pathogenic moderate evidence; PP, pathogenic supporting evidence; PS, pathogenic strong evidence; PVS, pathogenic very strong evidence; VUS, variants of unknown significance.

### 2. Reclassified “patients” due to the reclassification of BRCA1 or BRCA2 VUS

Excluding patients with both PV and VUS, there were 195 patients with only VUS. 82 were with BRCA1 VUS only, 90 were with BRCA2 VUS only and 23 were with BRCA1 and BRCA2 simultaneously (Fig. 1). Among the 195 patients, the status for 25.1% (49/195) changed to benign, that for 16.4% (32/195) changed to likely benign, that for 5.1% (10/195) changed to likely pathogenic, and that for 5.1% (10/195) changed to pathogenic. Each reclassification rate for BRCA1 VUS, BRCA2 VUS, and BRCA1 plus BRCA2 subset was reported in Table 3.

VUS, variants of unknown.

### DISCUSSION

In the current study, 108 specific BRCA1/2 VUS were detected. There were 195 people with only VUS, and the 51.8% (101/195) patients were reclassified. The status of 41.5% (81/195) patients was changed to benign or likely benign, and that of 10.3% (20/195) to pathogenic or likely pathogenic.

Several studies have shown that about 10%–15% of the cases tested for BRCA exhibited a pathogenic variant [20]. Further, previous studies have reported a range of prevalence for BRCA VUS (3.9%–22.5%) [78910112122]. However, these results are primarily from breast cancer studies, as there are very few studies that have solely focused on gynecological oncology. Moreover, in Korea, genetic studies of patients with breast cancer have been conducted on a large scale through research conducted by the Korean Hereditary Breast Cancer (Korean Hereditary Breast Cancer) Study [2324], which has reported a pathogenic variant prevalence of 16.5% (418/2,526). However, in the case of gynecological cancers, there is no such large-scale study. Several studies have reported that the frequency of BRCA1/2 germline mutations among patients with epithelial ovarian cancer ranges from 31.9% (37/116) [25] to 33% (13/40) [26]. However, in each of those studies, only a small fraction of ovarian cancer patients was subject to genetic testing (116 out of 711 and 40 out of 337, respectively). Thus, selection biases might have occurred due to lack of these genetic tests. In the present study, we detected a BRCA1 PV prevalence of 17.0% (137/805) and a BRCA2 PV prevalence of 8.6% (69/805). In addition, this study found BRCA1 VUS and BRCA2 VUS prevalence of 15.9% (128/805), and 18.4% (148/805), which was slightly lower than the previously reported incidence.

The ACMG/AMP 2015 guidelines were created to standardize the classification of genetic variants, and they proposed multiple criteria which can be combined to assign pathogenicity status. However, from their disclaimer, adherence to these standards and guidelines is voluntary and does not necessarily assure a successful medical outcome [12]. They suggested 16 criteria as evidence of pathogenicity, and 12 criteria as evidence for the benign effect, even with different degrees of evidence available. Thus, the determination of pathogenicity based on these criteria could vary depending on the researcher. For instance, for in silico tests, which were classified as criteria for PP3 or BP4, the use of “multiple lines of computational evidence” could result in arbitrary classification. The current study used tools such as “Mutation taster,” “Align-GVGD,” “PANTHER,” “Polyphen,” and “SIFT,” and 4 out of 5 matches were required for certainty. However, in a previous study [9], “nsSNPAnalyzer” was used in addition to the tools used in the current study and considered up to 80% as evidence of BP4 or PP3.

In another study [10], Human Splicing Finder and a criteria of 5 concordances out of 6 was used for the criteria for PP5 “Reputable source recently reported variants as pathogenic, but the evidence was not available to the laboratory to perform an independent evaluation,” or BP6 “Reputable source recently reported a variant as benign, but the evidence was not available to the laboratory to perform an independent evaluation.” A previous study [9] utilized 6 population databases and used information with 50% agreement between databases. However, the present study and a previous study [10] only used the ClinVar database.

In the case of BRCA1/2, differences regarding BP1 (missense variant in a gene for which primarily truncating variants are known to cause disease) could lead to inaccurate results for all missense variants [24]. A previous study [10] and the present study do not utilized this criterion because several single amino acid changes caused by single nucleotide variations could cause a pathogenic result. Conversely, another study [9] used BP1 as benign supporting evidence.

There were 4 reclassified BRCA1 variants (c.5215G>C [p.Asp1739His], c.5363G>T [p.Gly1788Val], c.5339T>C [p.Leu1780Pro], and c.5017_5019delCAC [p.His1673del]). Only a few studies have reported VUS found in patients with malignant disease rather than in a healthy cohort, regardless of organ, and reclassified the BRCA1/2 VUS. The rate at which specific VUS were reclassified as pathogenic or likely pathogenic variants was 5.6% (6/108) and that specific VUS reclassified as benign or likely benign variants was 27.8% (30/108). These rates were comparable to those observed in previous studies (Table 4) [21].

ACMG/AMP, American College of Medical Genetics and Genomics and the Association for Molecular Pathology; B, benign; EOC, epithelial ovarian cancer; FTC, fallopian tube cancer; HBOC, hereditary breast and ovarian cancer syndrome; LB, likely benign; LP, likely pathogenic; P, pathogenic; PPC, primary peritoneal cancer; VUS, variants of unknown significance.

c.5215G>C is a missense change at an amino acid residue where a different missense change that was determined to be pathogenic has been seen before (c.5216A>T [p.Asp1739Val]). This could be used for evidence of PM5 pathogenicity using ACMG/AMP 2015 criteria. c.5017_5019delCAC (p.His1673del) is an in-frame deletion and causes protein length change. Its allele frequency could not be found in the 1000 Genomes Project, Exome Aggregation Consortium, and Exome Variant Server databases; however, 0.37% (3/805) of patients had this variant in the present study.

Further, 2 BRCA2 VUS were reclassified as likely pathogenic (c.9154C>T [p.Arg3052Trp], c.9110_9112delAAC [p.Gln3037del] and c.8488-1G>A). In the case of c.8488-1G>A, several functional studies have reported it as deleterious [1927], and its allele frequency could not be found in the 1000 Genomes Project, Exome Aggregation Consortium, and Exome Variant Server databases. Discordant with other reports, in silico testing using the Human Splicing Finder, detected no significant splicing motif alteration and showed that this mutation potentially had no impact on splicing.

In the present study, 17 synonymous variants (10 in BRCA1 and 7 in BRCA2) were detected, and 4 of these variants were reclassified as benign while the others remained VUS. The degeneracy of the genetic code means that many mutations in coding sequences, particularly at the third base codon, do not affect the protein sequence and are, therefore, considered silent. However, there is increasing evidence that they could still have effects on transcription, splicing, mRNA transport or translation, any of which could alter the phenotype [28]. For example, c.9117G>C (p.Pro3089=) is a known pathogenic variant. However, since it is a simple synonymous variant, it could easily be misinterpreted as a benign variant. Synonymous variants should, therefore, not be judged as benign without definite evidence.

Current evidence suggests that several candidate pathogenic/likely pathogenic variants could not be classified as such due to lack of evidence to fulfill the ACMG/AMP guidelines. For example, the BRCA1 variant c.81-9C>G in the present study showed activation of the intron cryptic acceptor site and potential alteration of splicing (PP3) as the result of in silico testing by Human Splicing Finder. Moreover, the frequency of this variant could not be found in the 1000 Genomes Project, Exome Sequencing Project, or the Exome Aggregation Consortium; therefore, we assigned it to PM2 [29]. c.81-9C>G is an intronic variant which had the effect of creation and use of a novel acceptor site for frameshift mutations. Therefore, a co-segregation study, pedigree analysis, and functional study are required to determine pathogenicity.

Literally, the variants which have not been established or ruled out as a disease risk are referred to as VUS. Under the present circumstances, VUS is treated the same as benign. For example, there may be patients who have a strong family history and consistently present novel variants from the family. However, pathogenicity cannot be easily determined unless the variant is a frame-shift or nonsense mutation. If the pathogenicity of the variant has not been demonstrated without uncertainty, we cannot use poly (ADP-ribose) polymerase-1 inhibitor, cannot perform preventive procedure such as risk-reducing salpingo-oophorectomy. The best way to determine the pathogenicity of a new variant is to conduct a functional study. However, because the number of variants is infinite in theory, it is currently unreasonable to conduct a functional study on all variants. Therefore, by consistently conducting the same attempts as in this study, we selected VUS as a candidate for a pathogenic variant. After that, if the functional study of the VUSs is done, it would be a good challenge to effectively prevent and treat hereditary gynecological cancer.

The limitations of the present study are that this was a study performed in a single institution and may have selection bias due to the population undergoing genetic testing at this institution. The female relatives of the ovarian cancer patients tend to visit the same hospital; therefore, the subjects in the present data could be closely related each other, and the frequency of a certain variant could be exaggerated. In addition, we used a retrospective chart review for data collection, and data for the pedigree analysis or co-segregation test was partly insufficient.

In conclusion, the clinical significance of VUS in BRCA1/2 is changing over time, and in this study, we reclassified 5.6% (6/108) of specific VUS identified over a 13-year period as pathogenic or likely pathogenic. Among the 195 patients with VUS, the BRCA1/2 variants of 13.8% (27/195) were reclassified as benign, those of 27.7% (54/194) to likely benign, and those of 10.3% (20/195) to pathogenic or likely pathogenic. Therefore, continued monitoring of patients with mutations, and further investigation into the pathogenicity of VUS is required in the future.



# SUPPLEMENTAL FILE 1: jgo-31-e83.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: PMC7593220

[File available at: automated_output/BRCA2/20251125_175029/pmc_fulltext/33078592_supplements/PMC7593220]



# SUPPLEMENTAL FILE 3: ovarian.pdf

[PDF file available at: ovarian.pdf]

