# MAIN TEXT

## Germline mutational spectrum in Armenian breast cancer patients suspected of hereditary breast and ovarian cancer

### Abstract

Hereditary breast and ovarian cancer (HBOC) can be identified by genetic testing of cancer-causing genes. In this study, we identified a spectrum of genetic variations among 76 individuals of Armenian descent either with a family history of cancer or breast cancer before the age of 40. We screened 76 suspected HBOC patients and family members as well as four healthy controls using a targeted and hereditary comprehensive cancer panel (127 genes). We found 26 pathogenic (path) and 6 likely pathogenic (LPath)variants in 6 genes in 44 patients (58%); these variants were found in BRCA1 (17), BRCA2 (19), CHEK2 (4), PALB2 (2), and NBN (1). A few different variants were found in unrelated individuals; most notably, variant p.Trp1815Ter in the BRCA1 gene occurred in four unrelated patients. We did not find any known significant variants in five patients. Comprehensive cancer panel testing revealed pathogenic variants in cancer genes other than BRCA1 and BRCA2, suggesting that testing only BRCA1 and BRCA2 would have missed 8 out of 44 suspected HBOC patients (18%). These data also confirm that a comprehensive cancer panel testing approach could be an appropriate way to identify most of the variants associated with hereditary breast cancer.

### Introduction

Hereditary breast and ovarian cancer (HBOC) and their causative variants have become one of the most studied hereditary cancers. Although the BRCA1 (NM_007294.3) and BRCA2 (NM_000059.3) gene variants were the primary focus of such studies in the past, hereditary cancer panel testing has recently replaced the approach involving only the BRCA1 and BRCA2 genes. In fact, it has been shown that close to 10% of hereditary breast cancer can be caused by variants in genes such as CHEK2 (NM_007194.3), PALB2 (NM_024675.3), ATM (NM_000051.3), and MUTYH (NM_001128425.1)1. There are hundreds of genes that could be involved in tumorigenesis and cancer, yet not all are involved in every cancer. Thus, there are specific hereditary cancer panels for breast, ovarian, colorectal, endometrial, prostate, and other cancers. These panels contain some of the most prevalently mutated genes in each specific cancer, yet there could be genes that may cause hereditary cancer in a specific population that may not be included in general cancer gene panels. Therefore, ethnic-specific population studies could be important in the identification and inclusion of genes that may cause hereditary cancers in a specific population.

Additionally, there are several cancer-causing variants that could be more prevalent in an ethnic or specific population. The most famous of these variants are the three BRCA1 and BRCA2 variants (i.e., 5382insC and 185delAG in BRCA1 and 6174delT in BRCA2) that are prevalent in the Ashkenazi Jewish population2. Ethnic-specific studies have shown that there may be variants classified as founder mutations, as well as unique variants in suspected HBOC patients. In fact, the number of founder mutations compared to unique variants is quite small, indicating that some populations will not benefit from targeted gene testing. Recently, gene panels have been shown to be the better approach, expanding HBOC variant discovery to several genes from different pathways (Breast Cancer Information Core, BIC). This approach has resulted in the identification of many pathogenic (Path) and likely pathogenic (LPath) variants, and variants of unknown significance (VUSs) from genes other than BRCA1 and BRCA2. Path and LPath variants are important in drug discovery and treatment options, while VUSs could be significant in the initiation of more studies to identify their cancer-causing roles. Some deleterious variants could introduce premature termination codons through frameshift deletions or insertions, nonsense or splice junction mutations, or large deletions or duplications. Some splice site mutations and large rearrangements do not change the reading frame but result in a loss or gain of one to several exons, which could potentially have an impact on gene function. Deleterious missense mutations are typically confined within specific residues of functional motifs. However, the risk contribution of numerous other sequence variants remains unclear. Some VUSs include missense changes and small in-frame deletions and insertions that mostly lead to one amino acid change without a frameshift as well as alterations in noncoding intervening sequences or in untranslated exonic regions3.

There have been two small-scale studies on breast cancer genetics in the Armenian population. One study attempted to find known BRCA1 and BRCA2 founder mutations in 46 suspected HBOC patients, but no founder mutations were found4. Another study attempted to identify BRCA1 and BRCA2 mutations in six suspected HBOC patients of Armenian descent using targeted panel testing. No pathogenic variants were found in these patients5.

In this study, we analyzed the mutation spectrum of 76 patients of Armenian descent with suspicion of hereditary breast cancer (family history of cancer) or breast cancer under the age of 40, selected according to the National Comprehensive Cancer Network (NCCN) guidelines, using a comprehensive 127-gene hereditary cancer panel. The purpose of this study was not only to identify the mutational spectrum of HBOC in the Armenian population but also to identify the genes that could have breast cancer-causing variants in the Armenian population. We intended to identify these genes to provide insights for an Armenian-specific HBOC-testing gene panel and a comparison with genes in panels for other ethnic groups. We also anticipated finding novel variants that could potentially cause breast cancer due to potential different distributions of disease-causing variants in the Armenian population. This is the first large-scale study of breast cancer-causing variants in the Armenian population.

### Materials and methods

We tested 76 suspected HBOC patients or family members and 4 healthy controls using a targeted test (24 patients) and a hereditary comprehensive cancer panel (127 genes). Overall, 34 patients were <40 years of age (22 with a family history of cancer, 63%), 16 patients were between 40 and 50 years old (10 with a family history of cancer, 63%), and 26 patients were >50 years of age (25 with a family history of cancer, 96%). All patients were selected according to the criteria provided in the NCCN Clinical Practice Guidelines in Oncology (Version 2.2019-July 30.2018). Only patients with a confirmed histopathologic diagnosis of invasive breast cancer (BC) were included in the study. Following genetic counseling, the probability for each female patient to be a mutation carrier in one or both of the BRCA genes was estimated using the BRCAPRO model of the University of Texas Southwestern Medical Center at Dallas CancerGene software, version 5.1 (http://www4.utsouthwestern.edu/breasthealth/cagene/). The BRCAPRO risk is derived through the Bayesian probability model and takes into account the first and second degree relatives of a patient, age at the time of diagnosis of BC and/or ovarian cancer (OC), and ages of unaffected family members6. The male patients with a diagnosis of BC were not subject to BRCAPRO risk calculation given that the model values for male BC patients were significantly higher than those for female BC patients. Patients were all ethnically Armenian. This study was approved by the Institutional Review Board of the Center of Medical Genetics and Primary Health Care. Informed consent was obtained from all human subjects participating in this research.

Variant analysis was performed using an advanced bioinformatics pipeline and manual curation.

A 127-gene comprehensive cancer panel (AIP, ALK, APC, ATM, ATR, AXIN2, BAP1, BARD1, BLM, BMPR1A, BRCA1, BRCA1, BRIP1, BUB1B, CASR, CDC73, CDH1, CDK4, CDKN1B, CDKN1C, CDKN2A, CEBPA, CHEK2, CTC1, CTNNA1, CYLD, DDB2, DICER1, DIS3L2, DKC1, EGLN1, EPCAM, ERCC1, ERCC2, ERCC3, ERCC4, ERCC5, EXT1, EXT2, EZH2, FAN1, FANCA, FANCB, FANCC, FANCD2, FANCE, FANCF, FANCG, FANCI, FANCL, FANCM, FH, FLCN, GALNT12, GATA2, GPC3, GREM1, HOXB13, HRAS, KIF1B, KIT, LZTR1, MAX, MC1R, MEN1, MET, MITF, MLH1, MLH3, MRE11, MSH2, MSH6, MUTYH, NBN, NF1, NF2, NHP2, NOP10, NTHL1, PALB2, PDGFRA, PHOX2B, PMS2, POLD1, POLE, POLH, POT1, PRKAR1A, PRSS1, PTCH1, PTCH2, PTEN, RAD50, RAD51C, RAD51D, RB1, RECQL4, RET, RUNX1, SDHA, SDHAF2, SDHB, SDHC, SDHD, SLC45A2, SLX4, SMAD4, SMARCA4, SMARCB1, SMARCE1, STK11, SUFU, TERC, TERT, TINF2, TMEM127, TP53, TSC1, TSC2, TYR, VHL, WRAP53, WRN, WT1, XPA, XPC, and XRCC2) offered by Fulgent Diagnostics (Temple City, CA) was used to sequence 76 patients and 4 healthy controls. Next-generation sequencing (NGS) was performed on a HiSeq instrument from Illumina, La Jolla, CA. The average depth of sequencing depth/mean coverage was between 1100x and 1300x. The assay mainly covers the exonic coding regions 10–20 bp into the flanking introns. The promoter region of the PTEN gene was sequenced, and the promoter regions of the BRCA1 and BRCA2 genes were not included in the panel. PMS2 pseudogene detection was performed by Fulgent’s proprietary “Misalignment Tool”, which analyzed both reads from the real gene and the pseudogene. Some patients were also confirmed by long-range PCR for mutations in exon 13 to exon 15 of the PMS2 gene. After sequencing, the Fulgent bioinformatics pipeline was used to obtain a list of variants with data from standards and guidelines for the interpretation of sequence variants recommended by the American College of Medical Genetics and Genomics (ACMG)7,8. Each variant for every sample was analyzed in detail before any classification was made.

The program Functional Analysis Through Hidden Markov Models (FATHMM) predicts whether single nucleotide variants (SNVs) in the human genome are likely to be functional or nonfunctional in inherited diseases. FATHMM uses distinct models for coding and noncoding regions to improve overall accuracy. The coding predictor is based on six groups of features representing sequence conservation, nucleotide sequence characteristics, genomic features (codons, splice sites, etc.), amino acid features and expression levels in different tissues. We used a threshold of 0, which generated a sensitivity of 0.949.

The program sorting intolerant from tolerant (SIFT) predicts whether an amino acid substitution is likely to affect protein function based on sequence homology and the physicochemical similarity between the alternate amino acids10. The data we provide for each amino acid substitution are a score and a qualitative prediction (either “tolerated” or “deleterious”). The score is the normalized probability that the amino acid change is tolerated, so scores closer to zero are more likely to be deleterious. The qualitative prediction is derived from this score such that substitutions with a score <0.05 are called “deleterious”, and all others are called “tolerated”. We used the criteria of <0.05 as damaging and between 0.05–0.07 as probably damaging.

The program PolyPhen-2 predicts the effects of an amino acid substitution on the structure and function of a protein using sequence homology, Pfam annotations, 3D structures from PDB where available, and a number of other databases and tools (including DSSP and ncoils). The PolyPhen score represents the probability that a substitution is damaging, so values closer to one are more confidently predicted to be deleterious11. The qualitative prediction is based on the false- positive rate of the classifier model used to make the predictions. We used the following criteria: scores of 0.446–0.908 as probably damaging and scores of 0.908 or more as damaging.

The combined annotation-dependent depletion (CADD) tool scores the predicted deleteriousness of single nucleotide variants and insertion/deletion variants in the human genome by integrating multiple annotations, including conservation and functional information, into one metric. Phred-style CADD raw scores are displayed, and variants with higher scores are more likely to be deleterious. CADD provides genome wide scores12. CADD provides a ranking rather than a prediction or default cutoff, with higher scores more likely to be damaging. We used the following criteria: scores of 10–20 as probably damaging and scores of >20 as damaging.

All variants reported in this study were described according to current HGVS mutation nomenclature guidelines and were verified using Variant Validator13. All variants identified in this study were submitted to the ClinVar database.

### Patient selection and BRCAPRO score

We tested 76 suspected HBOC patients or family members and 4 healthy controls using a targeted test (24 patients) and a hereditary comprehensive cancer panel (127 genes). Overall, 34 patients were <40 years of age (22 with a family history of cancer, 63%), 16 patients were between 40 and 50 years old (10 with a family history of cancer, 63%), and 26 patients were >50 years of age (25 with a family history of cancer, 96%). All patients were selected according to the criteria provided in the NCCN Clinical Practice Guidelines in Oncology (Version 2.2019-July 30.2018). Only patients with a confirmed histopathologic diagnosis of invasive breast cancer (BC) were included in the study. Following genetic counseling, the probability for each female patient to be a mutation carrier in one or both of the BRCA genes was estimated using the BRCAPRO model of the University of Texas Southwestern Medical Center at Dallas CancerGene software, version 5.1 (http://www4.utsouthwestern.edu/breasthealth/cagene/). The BRCAPRO risk is derived through the Bayesian probability model and takes into account the first and second degree relatives of a patient, age at the time of diagnosis of BC and/or ovarian cancer (OC), and ages of unaffected family members6. The male patients with a diagnosis of BC were not subject to BRCAPRO risk calculation given that the model values for male BC patients were significantly higher than those for female BC patients. Patients were all ethnically Armenian. This study was approved by the Institutional Review Board of the Center of Medical Genetics and Primary Health Care. Informed consent was obtained from all human subjects participating in this research.

### Gene panel sequencing and bioinformatics

Variant analysis was performed using an advanced bioinformatics pipeline and manual curation.

A 127-gene comprehensive cancer panel (AIP, ALK, APC, ATM, ATR, AXIN2, BAP1, BARD1, BLM, BMPR1A, BRCA1, BRCA1, BRIP1, BUB1B, CASR, CDC73, CDH1, CDK4, CDKN1B, CDKN1C, CDKN2A, CEBPA, CHEK2, CTC1, CTNNA1, CYLD, DDB2, DICER1, DIS3L2, DKC1, EGLN1, EPCAM, ERCC1, ERCC2, ERCC3, ERCC4, ERCC5, EXT1, EXT2, EZH2, FAN1, FANCA, FANCB, FANCC, FANCD2, FANCE, FANCF, FANCG, FANCI, FANCL, FANCM, FH, FLCN, GALNT12, GATA2, GPC3, GREM1, HOXB13, HRAS, KIF1B, KIT, LZTR1, MAX, MC1R, MEN1, MET, MITF, MLH1, MLH3, MRE11, MSH2, MSH6, MUTYH, NBN, NF1, NF2, NHP2, NOP10, NTHL1, PALB2, PDGFRA, PHOX2B, PMS2, POLD1, POLE, POLH, POT1, PRKAR1A, PRSS1, PTCH1, PTCH2, PTEN, RAD50, RAD51C, RAD51D, RB1, RECQL4, RET, RUNX1, SDHA, SDHAF2, SDHB, SDHC, SDHD, SLC45A2, SLX4, SMAD4, SMARCA4, SMARCB1, SMARCE1, STK11, SUFU, TERC, TERT, TINF2, TMEM127, TP53, TSC1, TSC2, TYR, VHL, WRAP53, WRN, WT1, XPA, XPC, and XRCC2) offered by Fulgent Diagnostics (Temple City, CA) was used to sequence 76 patients and 4 healthy controls. Next-generation sequencing (NGS) was performed on a HiSeq instrument from Illumina, La Jolla, CA. The average depth of sequencing depth/mean coverage was between 1100x and 1300x. The assay mainly covers the exonic coding regions 10–20 bp into the flanking introns. The promoter region of the PTEN gene was sequenced, and the promoter regions of the BRCA1 and BRCA2 genes were not included in the panel. PMS2 pseudogene detection was performed by Fulgent’s proprietary “Misalignment Tool”, which analyzed both reads from the real gene and the pseudogene. Some patients were also confirmed by long-range PCR for mutations in exon 13 to exon 15 of the PMS2 gene. After sequencing, the Fulgent bioinformatics pipeline was used to obtain a list of variants with data from standards and guidelines for the interpretation of sequence variants recommended by the American College of Medical Genetics and Genomics (ACMG)7,8. Each variant for every sample was analyzed in detail before any classification was made.

### FATHMM, SIFT, PolyPhen-2, and CADD

The program Functional Analysis Through Hidden Markov Models (FATHMM) predicts whether single nucleotide variants (SNVs) in the human genome are likely to be functional or nonfunctional in inherited diseases. FATHMM uses distinct models for coding and noncoding regions to improve overall accuracy. The coding predictor is based on six groups of features representing sequence conservation, nucleotide sequence characteristics, genomic features (codons, splice sites, etc.), amino acid features and expression levels in different tissues. We used a threshold of 0, which generated a sensitivity of 0.949.

The program sorting intolerant from tolerant (SIFT) predicts whether an amino acid substitution is likely to affect protein function based on sequence homology and the physicochemical similarity between the alternate amino acids10. The data we provide for each amino acid substitution are a score and a qualitative prediction (either “tolerated” or “deleterious”). The score is the normalized probability that the amino acid change is tolerated, so scores closer to zero are more likely to be deleterious. The qualitative prediction is derived from this score such that substitutions with a score <0.05 are called “deleterious”, and all others are called “tolerated”. We used the criteria of <0.05 as damaging and between 0.05–0.07 as probably damaging.

The program PolyPhen-2 predicts the effects of an amino acid substitution on the structure and function of a protein using sequence homology, Pfam annotations, 3D structures from PDB where available, and a number of other databases and tools (including DSSP and ncoils). The PolyPhen score represents the probability that a substitution is damaging, so values closer to one are more confidently predicted to be deleterious11. The qualitative prediction is based on the false- positive rate of the classifier model used to make the predictions. We used the following criteria: scores of 0.446–0.908 as probably damaging and scores of 0.908 or more as damaging.

The combined annotation-dependent depletion (CADD) tool scores the predicted deleteriousness of single nucleotide variants and insertion/deletion variants in the human genome by integrating multiple annotations, including conservation and functional information, into one metric. Phred-style CADD raw scores are displayed, and variants with higher scores are more likely to be deleterious. CADD provides genome wide scores12. CADD provides a ranking rather than a prediction or default cutoff, with higher scores more likely to be damaging. We used the following criteria: scores of 10–20 as probably damaging and scores of >20 as damaging.

### FATHMM

The program Functional Analysis Through Hidden Markov Models (FATHMM) predicts whether single nucleotide variants (SNVs) in the human genome are likely to be functional or nonfunctional in inherited diseases. FATHMM uses distinct models for coding and noncoding regions to improve overall accuracy. The coding predictor is based on six groups of features representing sequence conservation, nucleotide sequence characteristics, genomic features (codons, splice sites, etc.), amino acid features and expression levels in different tissues. We used a threshold of 0, which generated a sensitivity of 0.949.

### Sorting intolerant from tolerant (SIFT)

The program sorting intolerant from tolerant (SIFT) predicts whether an amino acid substitution is likely to affect protein function based on sequence homology and the physicochemical similarity between the alternate amino acids10. The data we provide for each amino acid substitution are a score and a qualitative prediction (either “tolerated” or “deleterious”). The score is the normalized probability that the amino acid change is tolerated, so scores closer to zero are more likely to be deleterious. The qualitative prediction is derived from this score such that substitutions with a score <0.05 are called “deleterious”, and all others are called “tolerated”. We used the criteria of <0.05 as damaging and between 0.05–0.07 as probably damaging.

### PolyPhen-2

The program PolyPhen-2 predicts the effects of an amino acid substitution on the structure and function of a protein using sequence homology, Pfam annotations, 3D structures from PDB where available, and a number of other databases and tools (including DSSP and ncoils). The PolyPhen score represents the probability that a substitution is damaging, so values closer to one are more confidently predicted to be deleterious11. The qualitative prediction is based on the false- positive rate of the classifier model used to make the predictions. We used the following criteria: scores of 0.446–0.908 as probably damaging and scores of 0.908 or more as damaging.

### Combined annotation-dependent depletion (CADD)

The combined annotation-dependent depletion (CADD) tool scores the predicted deleteriousness of single nucleotide variants and insertion/deletion variants in the human genome by integrating multiple annotations, including conservation and functional information, into one metric. Phred-style CADD raw scores are displayed, and variants with higher scores are more likely to be deleterious. CADD provides genome wide scores12. CADD provides a ranking rather than a prediction or default cutoff, with higher scores more likely to be damaging. We used the following criteria: scores of 10–20 as probably damaging and scores of >20 as damaging.

### Variants

All variants reported in this study were described according to current HGVS mutation nomenclature guidelines and were verified using Variant Validator13. All variants identified in this study were submitted to the ClinVar database.

### Results

We found 44 patients with pathogenic or likely pathogenic variants (32 unique variants) in 76 tested patients or family members (58%). The number of patients with BRCA1 and BRCA2 variants was 36, ~82% of this group. These variants were closely split, with 19 patients with BRCA2 (14 unique) and 17 patients with BRCA1 (10 unique) variants. The remaining variants (i.e., 9) were identified in the following genes: BRIP1 (1), CHEK2 (4), NBN (1), and PALB2 (2). The majority of the BRCA1 variants were either frameshift or nonsense mutations with 6 frameshift, 1 nonsense, 1 missense, and 2 intronic mutations. The BRCA2 gene had a slightly different variant comprising 8 frameshift, 4 nonsense, and 2 missense mutations. Variants in the rest of the genes were a mix of frameshift, nonsense, and missense mutations (Table 1). Variant p.Trp1815Ter in the BRCA1 gene was seen in four unrelated patients, all with a family history of cancer, indicating a possible founder mutation in the Armenian population. Variant p.Ile1159Metfs in BRCA1 was identified in the same family members from two different families, and p.Leu1669Ter in BRCA2 was identified in two sisters and one unrelated individual. Additionally, each one of the variants p.Cys1146Leufs in BRCA1 and p.Ala938Profs in BRCA2 were present in two unrelated patients. These data suggest that these variants may also be founder mutations. BRCA1 gene variants were found in three different exons: exon 5 (1), exon 11 (6), and exon 23 (1). The BRCA2 gene variants were found in seven different exons: exon 7 (1), exon 10 (2), exon 11 (7), exon 16 (1), exon 22 (1), exon 23 (1), and exon 25 (1). Two of the pathogenic variants in the BRCA1 gene were splice acceptor site variants (NM_007294.3: c.302-1G>A) and (NM_007294.3: c.4358-2A>G). These splice site mutations could result in a truncated or altered protein, potentially interfering with its function in DNA repair. Other mutations have been reported in the splice site at positions BRCA1 c.302-1 or c.302-2 (source, ClinVar).Table 1Patients with only pathogenic and likely pathogenic variants.Patient IDGenderExonGene nameVariant (cDNA)Variant (protein)AODCancer typeFamily history of cancerBRCAPRO score (%)Variant classificationP-01F5BRCA1c.211A>Gp.Arg71Gly36UBCBRC in 1st degree relative34.20%PATHP-02F–BRCA1c.302-1G>A-32UBCNo family history reported6.20%PATHP-03F11BRCA1c.798_799delp.Ser267Lysfs56UBCBRC in 1st degree relative8.00%PATHP-04F11BRCA1c.1504_1508delTTAAAp.Leu502Alafs40BBCNo family history reported5.90%PATHP-05F11BRCA1c.2649_2650insGGCAp.Thr884Glyfs36UBCMoth LUC (42); Mat G.Moth OC (57)24.00%PATHP-06F11BRCA1c.3436_3439delTGTTp.Cys1146Leufs32UBCBRC in 1st degree relative41.50%PATHP-07F11BRCA1c.3436_3439delTGTTp.Cys1146Leufs45UBCBRC in 1st degree relative12.80%PATHP-08aF11BRCA1c.3477_3480delAAAGp.Ile1159Metfs31UBCSis BRC50.40%PATHP-09aF11BRCA1c.3477_3480delAAAGp.Ile1159Metfs32UBCSis BRC50.40%PATHP-10aF11BRCA1c.3477_3480delAAAGp.Ile1159Metfs54UBCBro BRC3.50%PATHP-11aM11BRCA1c.3477_3480delAAAGp.Ile1159Metfs57UBCSis BRCN/APATHP-12F11BRCA1c.3485delAp.Asp1162Valfs51UBCSis OC (42); Daughter OC (32)85.30%PATHP-13bF–BRCA1c.4358-2A>G-40OC&BCMoth OC (50); Pat G.Moth LUC; Sis OC99.70%LPATHP-14F23BRCA1c.5444G>Ap.Trp1815Ter26UBCMoth BRC (32); Mat G.Moth BRC (34); Mat G.G.Moth BRC (45); Moth Mat Aunts BRC (36&37)92.30%PATHP-15F23BRCA1c.5444G>Ap.Trp1815Ter29UBCPat G.Moth BRC (60); Pat Aunt: BRC (46)33.30%PATHP-16F23BRCA1c.5444G>Ap.Trp1815Ter38UBCSis BRC & UterC; Bro StomC36.40%PATHP-17M23BRCA1c.5444G>Ap.Trp1815Ter65UBCMoth BRC (50); Pat Aunt: BRC (55)N/APATHP-18F7BRCA2c.574dupAp.Met192Asnfs36BBCNo family history reported15.10%PATHP-19F10BRCA2c.1414C>Tp.Gln472Ter53UBCMoth BRC (59); Sis BRC (42); Sist BRC (50); Mat Aunt: BRC (68 & 72); Mat G.fath ProsC (65); Mat G.Moth OC (43)81.30%PATHP-20F10BRCA2c.1528G>Tp.Glu510Ter55UBCBRC in 1st degree relative10.10%PATHP-21F11BRCA2c.2095C>Tp.Gln699Ter30UBCPat G.Moth UterC6.90%PATHP-22a,bF11BRCA2c.2623G>Cp.Val875Leu51UBCSis BRC (44); Mat G.Moth BRC (87)9.20%LPATHP-23a,bF11BRCA2c.2623G>Cp.Val875Leu44UBCSis BRC (51); Mat G.Moth BRC (87)9.70%LPATHP-24F11BRCA2c.2808_2811delACAAp.Ala938Profs29UBCBRC in 1st degree relative71.60%PATHP-25F11BRCA2c.2808_2811delACAAp.Ala938Profs38UBCMoth BRC (39); Mat Aunt: BRC (40)75.00%PATHP-26F11BRCA2c.4037_4038delCTp.Thr1346Serfs53UBCBRC in 1st degree relative11.10%PATHP-27F11BRCA2c.4548_4549delCAp.Lys1517Argfs62UBCBRC in 1st degree relative7.00%PATHP-28F11BRCA2c.5006T>Gp.Leu1669Ter81UBCBRC in 1st degree relative1.60%PATHP-29aF11BRCA2c.5006T>Gp.Leu1669Ter26UBCSis BRC (56)20.60%PATHP-30aF11BRCA2c.5006T>Gp.Leu1669Ter56UBCSis BRC (26)17.30%PATHP-31F11BRCA2c.6302delAp.Asn2101Metfs31UBCMat Aunt: BRC (36); Sec Cous BRC (43)29.00%PATHP-32F16BRCA2c.7689delCp.His2563Glnfs38UBCBRC in 1st degree relative28.70%PATHP-33M16BRCA2c.7689delCp.His2563Glnfs68UBCBRC in 1st degree relativeN/APATHP-34bF22BRCA2c.8851G>Tp.Ala2951Ser49UBCSis BRC (38); Pat Aunt BRC (70); Mat G.Moth BRC34.20%LPATHP-35F23BRCA2c.9097dupAp.Thr3033Asnfs48UBCBRC in 1st degree relative10.10%PATHP-36F25BRCA2c.9253delAp.Thr3085Glnfs41UBCMoth BRC (46)17.70%PATHP-37F7BRIP1c.917dupp.Asn306Lysfs59UBCBRC in 1st degree relative7.00%PATHP-38F3CHEK2c.409C>Tp.Arg137Ter54UBCBRC in 1st degree relative10.30%PATHP-39bF3CHEK2c.422A>Cp.Lys141Thr45UBCMoth BRC (57); Sis BRC (34); Mat Aunt UterC (52)42.00%LPATHP-40F4CHEK2c.470T>Cp.Ile157Thr46UBCBRC in 1st degree relative18.20%LPATHP-41F4CHEK2c.499G>Ap.Gly167Arg54UBCBRC in 1st degree relative10.30%LPATHP-42F11NBNc.1502G>Ap.Trp501Ter51UBCBRC in 1st degree relative12.90%PATHP-43F4PALB2c.932_933insCp.Lys311Asnfs68UBCBro ProsC (62)3.30%PATHP-44F12PALB2c.3299_3306dupp.Val1103Leufs51UBCBRC in 1st degree relative12.90%PATHComplete patient and variant information and family history for identified pathogenic and likely pathogenic mutations.P patient, PATH pathogenic, LPATH likely pathogenic, AOD age of diagnosis, BRC breast cancer, OC ovarian cancer, LUC lung cancer, UterC uterine cancer, StomC stomach cancer, ProsC prostate cancer, Moth mother, Fath father, Sis sister, Bro brother, Mat maternal, Pat paternal, G grand, UBC unilateral breast cancer, BBC bilateral breast cancer, N/A not applicable.aSibling pairs.bPatient also had a VUS variant.

Patients with only pathogenic and likely pathogenic variants.

Complete patient and variant information and family history for identified pathogenic and likely pathogenic mutations.

P patient, PATH pathogenic, LPATH likely pathogenic, AOD age of diagnosis, BRC breast cancer, OC ovarian cancer, LUC lung cancer, UterC uterine cancer, StomC stomach cancer, ProsC prostate cancer, Moth mother, Fath father, Sis sister, Bro brother, Mat maternal, Pat paternal, G grand, UBC unilateral breast cancer, BBC bilateral breast cancer, N/A not applicable.

aSibling pairs.

bPatient also had a VUS variant.

We found four patients in this patient cohort that did not report any family history of cancer yet had pathogenic or likely pathogenic variants identified. Three of them had bilateral breast cancer, and one had unilateral breast cancer before the age of 35. Additionally, to the best of our knowledge, one of the identified variants, p.Val875Leu, in BRCA2 has not been reported in any patient suspected of HBOC. A detailed explanation of every pathogenic and likely pathogenic variant annotation is presented in the supplementary material.

We found 37 variants of unknown significance (VUSs) in 32 patients, and based on our bioinformatics pipeline designation, these variants had indications of being significant but are not well studied or reported (Table 2). We reported a 45% VUS rate, which is higher than the industry-reported 30–40% VUS rate. A detailed explanation of every VUS annotation is presented in the supplementary material. We did not analyze the following VUSs: (MUTYH p.Leu420Met; CHEK2 p.Asp438Tyr; SLX4 p.Cys1805Arg; and MLH1 p.His318Gln) in patients who also carried likely pathogenic variants; these variants are only reported in the Supplemental Table.Table 2Patients with variants of unknown significance (VUS).VariantGenderExonGene nameVariant (cDNA)Variant (protein)AODCancer typeFamily history of cancerBRCAPRO score (%)VUS-01bP-45F18ATMc.3371A>Tp.Tyr1124Phe26UBCPat 2 cousins BRC; Moth’s Pat Uncle ProstC8.40%VUS-02bP-45F3XRCC2c.268C>Tp.Leu90Phe26UBCPat 2 cousins BRC; Moth’s Pat Uncle ProstC8.40%VUS-03P-46F15SLX4c.5413T>Cp.Cys1805Arg26UBCNo family history reported8.50%VUS-04bP-47F11RECQL4c.2704C>Tp.Arg902Trp27UBCNo family history reported8.10%VUS-05bP-47F10POLEc.5653G>Ap.Ala1885Thr27UBCNo family history reported8.10%VUS-06bP-48F8AXIN2c.2083G>Tp.Ala695Ser28UBCNo family history reported7.70%VUS-07bP-48F2TSHRc.202C>Tp.Pro68Ser28UBCNo family history reported7.70%VUS-08bP-49F11SLX4c.2320G>Tp.Ala774Ser28UBCMoth BRC (44); Mat G.Moth EsophC (75); G.Fath GastC (62); Mat Aunt GastC (50); Pat Aunt ColorecC (60)21.40%VUS-09bP-49F10WRAP53c.1564delGp.Ala522Argfs28UBCMoth BRC (44); Mat G.Moth EsophC (75); G.Fath GastC (62); Mat Aunt GastC (50); Pat Aunt ColorecC (60)21.40%VUS-10bP-50F21BRCA2c.8699A>Tp.Asp2900Val29UBCMoth Melanoma5.70%VUS-11bP-50F2SLX4c.421G>Tp.Gly141Trp29UBCMoth Melanoma5.70%VUS-12P-51F-BRCA1c.-86C>T-30UBCNo family history reported6.90%VUS-13P-52F21BRCA1c.5360G>Ap.Cys1787Tyr31UBCNo family history reported5.90%VUS-14P-53F6WRAP53c.838G>Ap.Ala280Thr31UBCNo family history reported6.50%VUS-15P-54F20FANCD2c.1777C>Tp.Pro593Ser33UBCNo family history reported5.80%VUS-16P-55F36FANCIc.3812C>Tp.Ser1271Phe34UBCFath ProstC24.00%VUS-17P-56F50ATMc.7503T>Ap.Asn2501Lys34UBC2 Pat Aunts BRC35.20%VUS-18P-57F12SLX4c.4423A>Gp.Thr1475Ala35UBCPat Uncle LUC (62)5.20%VUS-19P-58F7FANCBc.1480A>Gp.Thr494Ala36UBCNo family history reported5.00%VUS-20P-59F4CHEK2c.480A>Gp.Ile160Met36UBCMoth BRC (57)12.10%VUS-21P-60F7ATRc.1602G>Cp.Trp534Cys39UBCMoth BRC (48)17.70%VUS-22P-61F4SMARCA4c.403C>Gp.Pro135Ala40BBCPat Aunt ThroatC (63); Pat uncle LiverC (63)5.00%VUS-23cP-13F13MUTYHc.1258C>Ap.Leu420Met40OC&BCMoth OVC (50); Pat G.Moth LUC; Sis OVC99.70%VUS-24P-62F4PALB2c.833_834delTAinsATp.Leu278His41UBCSis BRC (57); Mat Aunt BRC (60)13.40%VUS-25P-63F1MC1Rc.104G>Ap.Cys35Tyr43BBCNo family history reported4.70%VUS-26a,cP-22F12CHEK2c.1312G>Tp.Asp438Tyr44UBCSis BRC (51); Mat G.Moth BRC (87)9.70%VUS-27a,cP-23F12CHEK2c.1312G>Tp.Asp438Tyr51UBCSis BRC (44); Mat G.Moth BRC (87)9.20%VUS-28cP-39F15SLX4c.5413T>Cp.Cys1805Arg45UBCMoth BRC (57); Sis BRC (34); Mat Aunt UterC (52)42.00%VUS-29bP-64F3SDHBc.269G>Ap.Arg90Gln46BBCNo family history reported2.50%VUS-30bP-64F13PALB2c.3428T>Ap.Leu1143His46BBCNo family history reported2.50%VUS-31cP-34F11MLH1c.954C>Ap.His318Gln49UBCSis BRC (38); Pat Aunt BRC (70); Mat G. Moth BRC34.20%VUS-32P-65F8MSH6c.3727A>Tp.Thr1243Ser51BBCNo family history reported1.70%VUS-33P-66F4ATRc.992A>Gp.Asp331Gly53UBCSis BRC (45 & 55)20.80%VUS-34P-67F7MUTYHc.553C>Tp.Arg185Trp54UBCBRC in 1st degree relative10.30%VUS-35P-68F8PALB2c.2821A>Gp.Ile941Val55BBCMat Cous BRC; Pat Cous BRC1.90%VUS-36P-69F19BRCA1c.5191G>Ap.Glu1731Lys55UBCBRC in 1st degree relative8.40%VUS-37P-70F13MUTYHc.1258C>Ap.Leu420Met55BBCMoth BRC (55)5.50%Complete patient and variant information and family history for identified variants of unknown significance (VUS).AOD age of diagnosis, BRC breast cancer, OVC ovarian cancer, UterC uterine cancer, StomC stomach cancer, ProsC prostate cancer, ThroatC throat cancer, LUC lung cancer, LiverC liver cancer, EosaphC esophagus cancer, GastC gastric cancer, ColorecC colorectal cancer Moth mother, Fath father, Sis sister, Bro brother, Mat maternal, Pat paternal, G grand, P patient.aSibling pairs.bTwo VUS were identified in the same patient.cPatient also had a PATH/LPATH variant.

Patients with variants of unknown significance (VUS).

VUS-01b

P-45

VUS-02b

P-45

VUS-03

P-46

VUS-04b

P-47

VUS-05b

P-47

VUS-06b

P-48

VUS-07b

P-48

VUS-08b

P-49

VUS-09b

P-49

VUS-10b

P-50

VUS-11b

P-50

VUS-12

P-51

VUS-13

P-52

VUS-14

P-53

VUS-15

P-54

VUS-16

P-55

VUS-17

P-56

VUS-18

P-57

VUS-19

P-58

VUS-20

P-59

VUS-21

P-60

VUS-22

P-61

VUS-23c

P-13

VUS-24

P-62

VUS-25

P-63

VUS-26a,c

P-22

VUS-27a,c

P-23

VUS-28c

P-39

VUS-29b

P-64

VUS-30b

P-64

VUS-31c

P-34

VUS-32

P-65

VUS-33

P-66

VUS-34

P-67

VUS-35

P-68

VUS-36

P-69

VUS-37

P-70

Complete patient and variant information and family history for identified variants of unknown significance (VUS).

AOD age of diagnosis, BRC breast cancer, OVC ovarian cancer, UterC uterine cancer, StomC stomach cancer, ProsC prostate cancer, ThroatC throat cancer, LUC lung cancer, LiverC liver cancer, EosaphC esophagus cancer, GastC gastric cancer, ColorecC colorectal cancer Moth mother, Fath father, Sis sister, Bro brother, Mat maternal, Pat paternal, G grand, P patient.

aSibling pairs.

bTwo VUS were identified in the same patient.

cPatient also had a PATH/LPATH variant.

We also found ten patients with two variants; five patients had one likely pathogenic variant and one VUS, and another five had two VUSs. Four of the patients reported cancer in both maternal and paternal relatives, and two did not report a family history. The other four patients reported cancer only in their maternal relatives, but two of these patients were sisters (Supplemental Table 1).

We did not find any VUSs, pathogenic variants, or likely pathogenic variants in six breast cancer patients, and they only had benign/likely benign variants (Supplemental Table 1). Breast cancer in these patients could have been due to environmental factors or possibly genomic structural changes such as large deletions, duplications, and inversions, which were not investigated in this study. Additionally, none of the 76 patients had a VUS, pathogenic, or likely pathogenic variant in any of the following 37 genes: AIP, BAP1, CDC73, CDK4, CDKN1B, CDKN2A, CTC1, CYLD, EPCAM, ERCC1, ERCC3, FAN1, FANCG, FLCN, GALNT12, GATA2, HOXB13, HRAS, MAX, NHP2, PDGFRA, POT1, PRKAR1A, RAD51C, RAD51D, RUNX1, SDHA, SDHAF2, SMAD4, SMARCB1, SMARCE1, SUFU, TERC, TMEM127, TP53, VHL, and XPA.

Three of the four healthy controls were a 92-year-old grandmother and her 72-year-old daughter with no family history of cancer and her 50-year-old grandson with two paternal aunts with breast cancer at ages 75 and 76. The other healthy control was an unrelated 50-year-old female had no family history of cancer. The testing of the 92-year-old grandmother and her 72-year-old daughter showed no pathogenic variants or VUSs, yet the tested 50-year-old grandson had CHEK2 (p.Arg145Trp) likely pathogenic variant, evidently from his paternal side as two of his paternal aunts had breast cancer. The other healthy female control, who was unrelated to the rest of the healthy controls, had no pathogenic variants or VUSs.

We analyzed the age distribution of patients with path/LPath variants. The average age for the Path/LPath group was almost 46 years old. This group had more BRCA1 variants in <40-year-old patients than BRCA2 variants, and these variants were present more often in older populations (up to 70 years old). Interestingly, the non-BRCA variants present in this group were found mostly in the 40- and 50-year-old patients. The average age for BRCA1 variants was 41 years (range: 26–65 years), and the average age for BRCA2 variants was 48 years (range: 29–81 years). The average age for other variants was 50 years (range: 34–68 years) (Fig. 1).Fig. 1Age distribution of pathogenic and likely pathogenic variants identified in BRCA1, BRCA2, and the other genes.

Age distribution of pathogenic and likely pathogenic variants identified in BRCA1, BRCA2, and the other genes.

We used FATHMM, SIFT, PolyPhen2, and CADD prediction programs to assess the functional effects of Path/LPath classifications (Supplementary Table 2) and the VUS classifications (Table 3). One variant (FANCD2 p.Pro593Ser) had a tolerated classification from all four prediction programs, and two other variants (ATR p.Asp331Gly and AXIN2 p.Ala695Ser) were classified as tolerated by the 3 out of 4 programs. Nine of the VUSs had a FATHMM prediction as tolerant, while they had damaging predictions from the other three programs, with the exception of three, had another tolerant prediction (Table 3). These programs use different algorithms; thus, we consider all of the designations significant. All predictions for the Path/LPath variants, except variant p.Val875Leu in BRCA2, were damaging (D) or probably damaging (PD) with significant scores (Supplementary Table 2). Notably, one of the VUS mutations was SLX4 c.5413T>C (p.Cys1805Arg) variant, which has been shown to disrupt SLX4-SLX1 complex formation in functional studies10. The FATHMM classification of this variant was tolerant, yet it had a significant score from the other three programs and was consequently predicted to be damaging. We observed the same pattern in three variants classified as a VUS (ATR c.1602G>C, p.Trp534Cys; PALB2, c.2821A>G, p.Ile941Val; and PALB2 c.3428T>A, p.Leu1143His) (Table 3). Such a comparison could be used for further assessments of variants classified as a VUS. All the variants with a FATHMM classification of a negative score had a risk estimate of being damaging, which indicates that these substitutions are likely to interfere with the function of the protein. Almost all the variants that we classified as a VUS had scores as low as the known pathogenic variants for the SIFT, PolyPhen2, and CADD programs. The variant c.838G>A (p.Ala280Thr) in the WRAP53 gene, which we classified as a VUS, had a damaging or probably damaging score from all the programs. This variant has not been reported in breast cancer, but according to this analysis, it could interfere with the function of the protein.Table 3Comparison of FATHMM, SIFT, PolyPhen2, and CADD predications and scores for VUS.GenesDNA changeProtein changeFATHMMFATHMM scoreSIFTSIFT scorePolyPhenPolyPhen scoreCADDCADD scoreATMc.3371A>Tp.Tyr1124PhePD−0.58D0.03D0.912D22ATMc.7503T>Ap.Asn2501LysD−1.98D0.01D0.974D25.2ATRc.992A>Gp.Asp331GlyT−0.87T0.43T0PD16.87ATRc.1602G>Cp.Trp534CysT2.82T0.16D0.986D30AXIN2c.2083G>Tp.Ala695SerT−0.1T0.2T0.2D22.3BRCA1c.5191G>Ap.Glu1731LysD0.62D0.01PD0.767D32BRCA1c.5360G>Ap.Cys1787TyrD−1.25D0PD0.995D25.8BRCA2c.8699A>Tp.Asp2900ValPD−0.57D0D0.999D27.9CHEK2c.1312G>Tp.Asp438TyrPD−0.17D0D0.965D27CHEK2c.480A>Gp.Ile160MetD−3.02D0.01D0.979D22.6FANCBc.1480A>Gp.Thr494AlaT0.9078D0.04PD0.894D22.6FANCD2c.1777C>Tp.Pro593SerT0.97T1T0T0.036FANCIc.3812C>Tp.Ser1271PheD−1.77D0PD0.843D29.5MC1Rc.104G>Ap.Cys35TyrT0.04PD0.07D0.998D23.8MLH1c.954C>Ap.His318GlnPD−1.37D0D0.985PD14.82MSH6c.3727A>Tp.Thr1243SerD−2.04PD0.06D0.946D21.3MUTYHc.553C>Tp.Arg185TrpT2.56D0.03T0.321D21.9MUTYHc.1258C>Ap.Leu420MetD−2.8D0.02T0.261D22.6PALB2c.2821A>Gp.Ile941ValT1.53D0.01D0.999D25.6PALB2c.3428T>Ap.Leu1143HisT1.59D0D0.995D26.7POLEc.5653G>Ap.Ala1885ThrD0.9671T0.26B0.058D21.9RECQL4c.2704C>Tp.Arg902TrpD0.7973D0D0.995D24.8SDHBc.269G>Ap.Arg90GlnD−5.35D0D1.0D32SLX4c.421G>Tp.Gly141TrpT5.2D0PD0.76D20.3SLX4c.2320G>Tp.Ala774SerT0.42D0D0.999D27.7SLX4c.4423A>Gp.Thr1475AlaT5.22PD0.06T0.024PD11.85SLX4c.5413T>Cp.Cys1805ArgT3.62D0D0.999D23.4SMARCA4c.403C>Gp.Pro135AlaD−2.16PD0.07T0.001PD12.31TSHRc.202C>Tp.Pro68SerD−3.28T0.1D0.973D23.1WRAP53c.838G>Ap.Ala280ThrPD−0.21D0D0.988D28XRCC2c.268C>Tp.Leu90PheT1.07D0.05T0.224D22.5Scores from FATHMM, SIFT, PolyPhen, and CADD predications are compared for variants of unknown significance (VUS).D damaging, T tolerated, PD probably damaging.

Comparison of FATHMM, SIFT, PolyPhen2, and CADD predications and scores for VUS.

Scores from FATHMM, SIFT, PolyPhen, and CADD predications are compared for variants of unknown significance (VUS).

D damaging, T tolerated, PD probably damaging.

### Pathogenic or likely pathogenic variants

We found 44 patients with pathogenic or likely pathogenic variants (32 unique variants) in 76 tested patients or family members (58%). The number of patients with BRCA1 and BRCA2 variants was 36, ~82% of this group. These variants were closely split, with 19 patients with BRCA2 (14 unique) and 17 patients with BRCA1 (10 unique) variants. The remaining variants (i.e., 9) were identified in the following genes: BRIP1 (1), CHEK2 (4), NBN (1), and PALB2 (2). The majority of the BRCA1 variants were either frameshift or nonsense mutations with 6 frameshift, 1 nonsense, 1 missense, and 2 intronic mutations. The BRCA2 gene had a slightly different variant comprising 8 frameshift, 4 nonsense, and 2 missense mutations. Variants in the rest of the genes were a mix of frameshift, nonsense, and missense mutations (Table 1). Variant p.Trp1815Ter in the BRCA1 gene was seen in four unrelated patients, all with a family history of cancer, indicating a possible founder mutation in the Armenian population. Variant p.Ile1159Metfs in BRCA1 was identified in the same family members from two different families, and p.Leu1669Ter in BRCA2 was identified in two sisters and one unrelated individual. Additionally, each one of the variants p.Cys1146Leufs in BRCA1 and p.Ala938Profs in BRCA2 were present in two unrelated patients. These data suggest that these variants may also be founder mutations. BRCA1 gene variants were found in three different exons: exon 5 (1), exon 11 (6), and exon 23 (1). The BRCA2 gene variants were found in seven different exons: exon 7 (1), exon 10 (2), exon 11 (7), exon 16 (1), exon 22 (1), exon 23 (1), and exon 25 (1). Two of the pathogenic variants in the BRCA1 gene were splice acceptor site variants (NM_007294.3: c.302-1G>A) and (NM_007294.3: c.4358-2A>G). These splice site mutations could result in a truncated or altered protein, potentially interfering with its function in DNA repair. Other mutations have been reported in the splice site at positions BRCA1 c.302-1 or c.302-2 (source, ClinVar).Table 1Patients with only pathogenic and likely pathogenic variants.Patient IDGenderExonGene nameVariant (cDNA)Variant (protein)AODCancer typeFamily history of cancerBRCAPRO score (%)Variant classificationP-01F5BRCA1c.211A>Gp.Arg71Gly36UBCBRC in 1st degree relative34.20%PATHP-02F–BRCA1c.302-1G>A-32UBCNo family history reported6.20%PATHP-03F11BRCA1c.798_799delp.Ser267Lysfs56UBCBRC in 1st degree relative8.00%PATHP-04F11BRCA1c.1504_1508delTTAAAp.Leu502Alafs40BBCNo family history reported5.90%PATHP-05F11BRCA1c.2649_2650insGGCAp.Thr884Glyfs36UBCMoth LUC (42); Mat G.Moth OC (57)24.00%PATHP-06F11BRCA1c.3436_3439delTGTTp.Cys1146Leufs32UBCBRC in 1st degree relative41.50%PATHP-07F11BRCA1c.3436_3439delTGTTp.Cys1146Leufs45UBCBRC in 1st degree relative12.80%PATHP-08aF11BRCA1c.3477_3480delAAAGp.Ile1159Metfs31UBCSis BRC50.40%PATHP-09aF11BRCA1c.3477_3480delAAAGp.Ile1159Metfs32UBCSis BRC50.40%PATHP-10aF11BRCA1c.3477_3480delAAAGp.Ile1159Metfs54UBCBro BRC3.50%PATHP-11aM11BRCA1c.3477_3480delAAAGp.Ile1159Metfs57UBCSis BRCN/APATHP-12F11BRCA1c.3485delAp.Asp1162Valfs51UBCSis OC (42); Daughter OC (32)85.30%PATHP-13bF–BRCA1c.4358-2A>G-40OC&BCMoth OC (50); Pat G.Moth LUC; Sis OC99.70%LPATHP-14F23BRCA1c.5444G>Ap.Trp1815Ter26UBCMoth BRC (32); Mat G.Moth BRC (34); Mat G.G.Moth BRC (45); Moth Mat Aunts BRC (36&37)92.30%PATHP-15F23BRCA1c.5444G>Ap.Trp1815Ter29UBCPat G.Moth BRC (60); Pat Aunt: BRC (46)33.30%PATHP-16F23BRCA1c.5444G>Ap.Trp1815Ter38UBCSis BRC & UterC; Bro StomC36.40%PATHP-17M23BRCA1c.5444G>Ap.Trp1815Ter65UBCMoth BRC (50); Pat Aunt: BRC (55)N/APATHP-18F7BRCA2c.574dupAp.Met192Asnfs36BBCNo family history reported15.10%PATHP-19F10BRCA2c.1414C>Tp.Gln472Ter53UBCMoth BRC (59); Sis BRC (42); Sist BRC (50); Mat Aunt: BRC (68 & 72); Mat G.fath ProsC (65); Mat G.Moth OC (43)81.30%PATHP-20F10BRCA2c.1528G>Tp.Glu510Ter55UBCBRC in 1st degree relative10.10%PATHP-21F11BRCA2c.2095C>Tp.Gln699Ter30UBCPat G.Moth UterC6.90%PATHP-22a,bF11BRCA2c.2623G>Cp.Val875Leu51UBCSis BRC (44); Mat G.Moth BRC (87)9.20%LPATHP-23a,bF11BRCA2c.2623G>Cp.Val875Leu44UBCSis BRC (51); Mat G.Moth BRC (87)9.70%LPATHP-24F11BRCA2c.2808_2811delACAAp.Ala938Profs29UBCBRC in 1st degree relative71.60%PATHP-25F11BRCA2c.2808_2811delACAAp.Ala938Profs38UBCMoth BRC (39); Mat Aunt: BRC (40)75.00%PATHP-26F11BRCA2c.4037_4038delCTp.Thr1346Serfs53UBCBRC in 1st degree relative11.10%PATHP-27F11BRCA2c.4548_4549delCAp.Lys1517Argfs62UBCBRC in 1st degree relative7.00%PATHP-28F11BRCA2c.5006T>Gp.Leu1669Ter81UBCBRC in 1st degree relative1.60%PATHP-29aF11BRCA2c.5006T>Gp.Leu1669Ter26UBCSis BRC (56)20.60%PATHP-30aF11BRCA2c.5006T>Gp.Leu1669Ter56UBCSis BRC (26)17.30%PATHP-31F11BRCA2c.6302delAp.Asn2101Metfs31UBCMat Aunt: BRC (36); Sec Cous BRC (43)29.00%PATHP-32F16BRCA2c.7689delCp.His2563Glnfs38UBCBRC in 1st degree relative28.70%PATHP-33M16BRCA2c.7689delCp.His2563Glnfs68UBCBRC in 1st degree relativeN/APATHP-34bF22BRCA2c.8851G>Tp.Ala2951Ser49UBCSis BRC (38); Pat Aunt BRC (70); Mat G.Moth BRC34.20%LPATHP-35F23BRCA2c.9097dupAp.Thr3033Asnfs48UBCBRC in 1st degree relative10.10%PATHP-36F25BRCA2c.9253delAp.Thr3085Glnfs41UBCMoth BRC (46)17.70%PATHP-37F7BRIP1c.917dupp.Asn306Lysfs59UBCBRC in 1st degree relative7.00%PATHP-38F3CHEK2c.409C>Tp.Arg137Ter54UBCBRC in 1st degree relative10.30%PATHP-39bF3CHEK2c.422A>Cp.Lys141Thr45UBCMoth BRC (57); Sis BRC (34); Mat Aunt UterC (52)42.00%LPATHP-40F4CHEK2c.470T>Cp.Ile157Thr46UBCBRC in 1st degree relative18.20%LPATHP-41F4CHEK2c.499G>Ap.Gly167Arg54UBCBRC in 1st degree relative10.30%LPATHP-42F11NBNc.1502G>Ap.Trp501Ter51UBCBRC in 1st degree relative12.90%PATHP-43F4PALB2c.932_933insCp.Lys311Asnfs68UBCBro ProsC (62)3.30%PATHP-44F12PALB2c.3299_3306dupp.Val1103Leufs51UBCBRC in 1st degree relative12.90%PATHComplete patient and variant information and family history for identified pathogenic and likely pathogenic mutations.P patient, PATH pathogenic, LPATH likely pathogenic, AOD age of diagnosis, BRC breast cancer, OC ovarian cancer, LUC lung cancer, UterC uterine cancer, StomC stomach cancer, ProsC prostate cancer, Moth mother, Fath father, Sis sister, Bro brother, Mat maternal, Pat paternal, G grand, UBC unilateral breast cancer, BBC bilateral breast cancer, N/A not applicable.aSibling pairs.bPatient also had a VUS variant.

Patients with only pathogenic and likely pathogenic variants.

Complete patient and variant information and family history for identified pathogenic and likely pathogenic mutations.

P patient, PATH pathogenic, LPATH likely pathogenic, AOD age of diagnosis, BRC breast cancer, OC ovarian cancer, LUC lung cancer, UterC uterine cancer, StomC stomach cancer, ProsC prostate cancer, Moth mother, Fath father, Sis sister, Bro brother, Mat maternal, Pat paternal, G grand, UBC unilateral breast cancer, BBC bilateral breast cancer, N/A not applicable.

aSibling pairs.

bPatient also had a VUS variant.

We found four patients in this patient cohort that did not report any family history of cancer yet had pathogenic or likely pathogenic variants identified. Three of them had bilateral breast cancer, and one had unilateral breast cancer before the age of 35. Additionally, to the best of our knowledge, one of the identified variants, p.Val875Leu, in BRCA2 has not been reported in any patient suspected of HBOC. A detailed explanation of every pathogenic and likely pathogenic variant annotation is presented in the supplementary material.

### Variants of unknown significance (VUSs)

We found 37 variants of unknown significance (VUSs) in 32 patients, and based on our bioinformatics pipeline designation, these variants had indications of being significant but are not well studied or reported (Table 2). We reported a 45% VUS rate, which is higher than the industry-reported 30–40% VUS rate. A detailed explanation of every VUS annotation is presented in the supplementary material. We did not analyze the following VUSs: (MUTYH p.Leu420Met; CHEK2 p.Asp438Tyr; SLX4 p.Cys1805Arg; and MLH1 p.His318Gln) in patients who also carried likely pathogenic variants; these variants are only reported in the Supplemental Table.Table 2Patients with variants of unknown significance (VUS).VariantGenderExonGene nameVariant (cDNA)Variant (protein)AODCancer typeFamily history of cancerBRCAPRO score (%)VUS-01bP-45F18ATMc.3371A>Tp.Tyr1124Phe26UBCPat 2 cousins BRC; Moth’s Pat Uncle ProstC8.40%VUS-02bP-45F3XRCC2c.268C>Tp.Leu90Phe26UBCPat 2 cousins BRC; Moth’s Pat Uncle ProstC8.40%VUS-03P-46F15SLX4c.5413T>Cp.Cys1805Arg26UBCNo family history reported8.50%VUS-04bP-47F11RECQL4c.2704C>Tp.Arg902Trp27UBCNo family history reported8.10%VUS-05bP-47F10POLEc.5653G>Ap.Ala1885Thr27UBCNo family history reported8.10%VUS-06bP-48F8AXIN2c.2083G>Tp.Ala695Ser28UBCNo family history reported7.70%VUS-07bP-48F2TSHRc.202C>Tp.Pro68Ser28UBCNo family history reported7.70%VUS-08bP-49F11SLX4c.2320G>Tp.Ala774Ser28UBCMoth BRC (44); Mat G.Moth EsophC (75); G.Fath GastC (62); Mat Aunt GastC (50); Pat Aunt ColorecC (60)21.40%VUS-09bP-49F10WRAP53c.1564delGp.Ala522Argfs28UBCMoth BRC (44); Mat G.Moth EsophC (75); G.Fath GastC (62); Mat Aunt GastC (50); Pat Aunt ColorecC (60)21.40%VUS-10bP-50F21BRCA2c.8699A>Tp.Asp2900Val29UBCMoth Melanoma5.70%VUS-11bP-50F2SLX4c.421G>Tp.Gly141Trp29UBCMoth Melanoma5.70%VUS-12P-51F-BRCA1c.-86C>T-30UBCNo family history reported6.90%VUS-13P-52F21BRCA1c.5360G>Ap.Cys1787Tyr31UBCNo family history reported5.90%VUS-14P-53F6WRAP53c.838G>Ap.Ala280Thr31UBCNo family history reported6.50%VUS-15P-54F20FANCD2c.1777C>Tp.Pro593Ser33UBCNo family history reported5.80%VUS-16P-55F36FANCIc.3812C>Tp.Ser1271Phe34UBCFath ProstC24.00%VUS-17P-56F50ATMc.7503T>Ap.Asn2501Lys34UBC2 Pat Aunts BRC35.20%VUS-18P-57F12SLX4c.4423A>Gp.Thr1475Ala35UBCPat Uncle LUC (62)5.20%VUS-19P-58F7FANCBc.1480A>Gp.Thr494Ala36UBCNo family history reported5.00%VUS-20P-59F4CHEK2c.480A>Gp.Ile160Met36UBCMoth BRC (57)12.10%VUS-21P-60F7ATRc.1602G>Cp.Trp534Cys39UBCMoth BRC (48)17.70%VUS-22P-61F4SMARCA4c.403C>Gp.Pro135Ala40BBCPat Aunt ThroatC (63); Pat uncle LiverC (63)5.00%VUS-23cP-13F13MUTYHc.1258C>Ap.Leu420Met40OC&BCMoth OVC (50); Pat G.Moth LUC; Sis OVC99.70%VUS-24P-62F4PALB2c.833_834delTAinsATp.Leu278His41UBCSis BRC (57); Mat Aunt BRC (60)13.40%VUS-25P-63F1MC1Rc.104G>Ap.Cys35Tyr43BBCNo family history reported4.70%VUS-26a,cP-22F12CHEK2c.1312G>Tp.Asp438Tyr44UBCSis BRC (51); Mat G.Moth BRC (87)9.70%VUS-27a,cP-23F12CHEK2c.1312G>Tp.Asp438Tyr51UBCSis BRC (44); Mat G.Moth BRC (87)9.20%VUS-28cP-39F15SLX4c.5413T>Cp.Cys1805Arg45UBCMoth BRC (57); Sis BRC (34); Mat Aunt UterC (52)42.00%VUS-29bP-64F3SDHBc.269G>Ap.Arg90Gln46BBCNo family history reported2.50%VUS-30bP-64F13PALB2c.3428T>Ap.Leu1143His46BBCNo family history reported2.50%VUS-31cP-34F11MLH1c.954C>Ap.His318Gln49UBCSis BRC (38); Pat Aunt BRC (70); Mat G. Moth BRC34.20%VUS-32P-65F8MSH6c.3727A>Tp.Thr1243Ser51BBCNo family history reported1.70%VUS-33P-66F4ATRc.992A>Gp.Asp331Gly53UBCSis BRC (45 & 55)20.80%VUS-34P-67F7MUTYHc.553C>Tp.Arg185Trp54UBCBRC in 1st degree relative10.30%VUS-35P-68F8PALB2c.2821A>Gp.Ile941Val55BBCMat Cous BRC; Pat Cous BRC1.90%VUS-36P-69F19BRCA1c.5191G>Ap.Glu1731Lys55UBCBRC in 1st degree relative8.40%VUS-37P-70F13MUTYHc.1258C>Ap.Leu420Met55BBCMoth BRC (55)5.50%Complete patient and variant information and family history for identified variants of unknown significance (VUS).AOD age of diagnosis, BRC breast cancer, OVC ovarian cancer, UterC uterine cancer, StomC stomach cancer, ProsC prostate cancer, ThroatC throat cancer, LUC lung cancer, LiverC liver cancer, EosaphC esophagus cancer, GastC gastric cancer, ColorecC colorectal cancer Moth mother, Fath father, Sis sister, Bro brother, Mat maternal, Pat paternal, G grand, P patient.aSibling pairs.bTwo VUS were identified in the same patient.cPatient also had a PATH/LPATH variant.

Patients with variants of unknown significance (VUS).

VUS-01b

P-45

VUS-02b

P-45

VUS-03

P-46

VUS-04b

P-47

VUS-05b

P-47

VUS-06b

P-48

VUS-07b

P-48

VUS-08b

P-49

VUS-09b

P-49

VUS-10b

P-50

VUS-11b

P-50

VUS-12

P-51

VUS-13

P-52

VUS-14

P-53

VUS-15

P-54

VUS-16

P-55

VUS-17

P-56

VUS-18

P-57

VUS-19

P-58

VUS-20

P-59

VUS-21

P-60

VUS-22

P-61

VUS-23c

P-13

VUS-24

P-62

VUS-25

P-63

VUS-26a,c

P-22

VUS-27a,c

P-23

VUS-28c

P-39

VUS-29b

P-64

VUS-30b

P-64

VUS-31c

P-34

VUS-32

P-65

VUS-33

P-66

VUS-34

P-67

VUS-35

P-68

VUS-36

P-69

VUS-37

P-70

Complete patient and variant information and family history for identified variants of unknown significance (VUS).

AOD age of diagnosis, BRC breast cancer, OVC ovarian cancer, UterC uterine cancer, StomC stomach cancer, ProsC prostate cancer, ThroatC throat cancer, LUC lung cancer, LiverC liver cancer, EosaphC esophagus cancer, GastC gastric cancer, ColorecC colorectal cancer Moth mother, Fath father, Sis sister, Bro brother, Mat maternal, Pat paternal, G grand, P patient.

aSibling pairs.

bTwo VUS were identified in the same patient.

cPatient also had a PATH/LPATH variant.

### Patients with two variants

We also found ten patients with two variants; five patients had one likely pathogenic variant and one VUS, and another five had two VUSs. Four of the patients reported cancer in both maternal and paternal relatives, and two did not report a family history. The other four patients reported cancer only in their maternal relatives, but two of these patients were sisters (Supplemental Table 1).

### Patients and genes with no variant and healthy controls

We did not find any VUSs, pathogenic variants, or likely pathogenic variants in six breast cancer patients, and they only had benign/likely benign variants (Supplemental Table 1). Breast cancer in these patients could have been due to environmental factors or possibly genomic structural changes such as large deletions, duplications, and inversions, which were not investigated in this study. Additionally, none of the 76 patients had a VUS, pathogenic, or likely pathogenic variant in any of the following 37 genes: AIP, BAP1, CDC73, CDK4, CDKN1B, CDKN2A, CTC1, CYLD, EPCAM, ERCC1, ERCC3, FAN1, FANCG, FLCN, GALNT12, GATA2, HOXB13, HRAS, MAX, NHP2, PDGFRA, POT1, PRKAR1A, RAD51C, RAD51D, RUNX1, SDHA, SDHAF2, SMAD4, SMARCB1, SMARCE1, SUFU, TERC, TMEM127, TP53, VHL, and XPA.

Three of the four healthy controls were a 92-year-old grandmother and her 72-year-old daughter with no family history of cancer and her 50-year-old grandson with two paternal aunts with breast cancer at ages 75 and 76. The other healthy control was an unrelated 50-year-old female had no family history of cancer. The testing of the 92-year-old grandmother and her 72-year-old daughter showed no pathogenic variants or VUSs, yet the tested 50-year-old grandson had CHEK2 (p.Arg145Trp) likely pathogenic variant, evidently from his paternal side as two of his paternal aunts had breast cancer. The other healthy female control, who was unrelated to the rest of the healthy controls, had no pathogenic variants or VUSs.

### Age distribution of variants

We analyzed the age distribution of patients with path/LPath variants. The average age for the Path/LPath group was almost 46 years old. This group had more BRCA1 variants in <40-year-old patients than BRCA2 variants, and these variants were present more often in older populations (up to 70 years old). Interestingly, the non-BRCA variants present in this group were found mostly in the 40- and 50-year-old patients. The average age for BRCA1 variants was 41 years (range: 26–65 years), and the average age for BRCA2 variants was 48 years (range: 29–81 years). The average age for other variants was 50 years (range: 34–68 years) (Fig. 1).Fig. 1Age distribution of pathogenic and likely pathogenic variants identified in BRCA1, BRCA2, and the other genes.

Age distribution of pathogenic and likely pathogenic variants identified in BRCA1, BRCA2, and the other genes.

### FATHMM, SIFT, PolyPhen2, and CADD

We used FATHMM, SIFT, PolyPhen2, and CADD prediction programs to assess the functional effects of Path/LPath classifications (Supplementary Table 2) and the VUS classifications (Table 3). One variant (FANCD2 p.Pro593Ser) had a tolerated classification from all four prediction programs, and two other variants (ATR p.Asp331Gly and AXIN2 p.Ala695Ser) were classified as tolerated by the 3 out of 4 programs. Nine of the VUSs had a FATHMM prediction as tolerant, while they had damaging predictions from the other three programs, with the exception of three, had another tolerant prediction (Table 3). These programs use different algorithms; thus, we consider all of the designations significant. All predictions for the Path/LPath variants, except variant p.Val875Leu in BRCA2, were damaging (D) or probably damaging (PD) with significant scores (Supplementary Table 2). Notably, one of the VUS mutations was SLX4 c.5413T>C (p.Cys1805Arg) variant, which has been shown to disrupt SLX4-SLX1 complex formation in functional studies10. The FATHMM classification of this variant was tolerant, yet it had a significant score from the other three programs and was consequently predicted to be damaging. We observed the same pattern in three variants classified as a VUS (ATR c.1602G>C, p.Trp534Cys; PALB2, c.2821A>G, p.Ile941Val; and PALB2 c.3428T>A, p.Leu1143His) (Table 3). Such a comparison could be used for further assessments of variants classified as a VUS. All the variants with a FATHMM classification of a negative score had a risk estimate of being damaging, which indicates that these substitutions are likely to interfere with the function of the protein. Almost all the variants that we classified as a VUS had scores as low as the known pathogenic variants for the SIFT, PolyPhen2, and CADD programs. The variant c.838G>A (p.Ala280Thr) in the WRAP53 gene, which we classified as a VUS, had a damaging or probably damaging score from all the programs. This variant has not been reported in breast cancer, but according to this analysis, it could interfere with the function of the protein.Table 3Comparison of FATHMM, SIFT, PolyPhen2, and CADD predications and scores for VUS.GenesDNA changeProtein changeFATHMMFATHMM scoreSIFTSIFT scorePolyPhenPolyPhen scoreCADDCADD scoreATMc.3371A>Tp.Tyr1124PhePD−0.58D0.03D0.912D22ATMc.7503T>Ap.Asn2501LysD−1.98D0.01D0.974D25.2ATRc.992A>Gp.Asp331GlyT−0.87T0.43T0PD16.87ATRc.1602G>Cp.Trp534CysT2.82T0.16D0.986D30AXIN2c.2083G>Tp.Ala695SerT−0.1T0.2T0.2D22.3BRCA1c.5191G>Ap.Glu1731LysD0.62D0.01PD0.767D32BRCA1c.5360G>Ap.Cys1787TyrD−1.25D0PD0.995D25.8BRCA2c.8699A>Tp.Asp2900ValPD−0.57D0D0.999D27.9CHEK2c.1312G>Tp.Asp438TyrPD−0.17D0D0.965D27CHEK2c.480A>Gp.Ile160MetD−3.02D0.01D0.979D22.6FANCBc.1480A>Gp.Thr494AlaT0.9078D0.04PD0.894D22.6FANCD2c.1777C>Tp.Pro593SerT0.97T1T0T0.036FANCIc.3812C>Tp.Ser1271PheD−1.77D0PD0.843D29.5MC1Rc.104G>Ap.Cys35TyrT0.04PD0.07D0.998D23.8MLH1c.954C>Ap.His318GlnPD−1.37D0D0.985PD14.82MSH6c.3727A>Tp.Thr1243SerD−2.04PD0.06D0.946D21.3MUTYHc.553C>Tp.Arg185TrpT2.56D0.03T0.321D21.9MUTYHc.1258C>Ap.Leu420MetD−2.8D0.02T0.261D22.6PALB2c.2821A>Gp.Ile941ValT1.53D0.01D0.999D25.6PALB2c.3428T>Ap.Leu1143HisT1.59D0D0.995D26.7POLEc.5653G>Ap.Ala1885ThrD0.9671T0.26B0.058D21.9RECQL4c.2704C>Tp.Arg902TrpD0.7973D0D0.995D24.8SDHBc.269G>Ap.Arg90GlnD−5.35D0D1.0D32SLX4c.421G>Tp.Gly141TrpT5.2D0PD0.76D20.3SLX4c.2320G>Tp.Ala774SerT0.42D0D0.999D27.7SLX4c.4423A>Gp.Thr1475AlaT5.22PD0.06T0.024PD11.85SLX4c.5413T>Cp.Cys1805ArgT3.62D0D0.999D23.4SMARCA4c.403C>Gp.Pro135AlaD−2.16PD0.07T0.001PD12.31TSHRc.202C>Tp.Pro68SerD−3.28T0.1D0.973D23.1WRAP53c.838G>Ap.Ala280ThrPD−0.21D0D0.988D28XRCC2c.268C>Tp.Leu90PheT1.07D0.05T0.224D22.5Scores from FATHMM, SIFT, PolyPhen, and CADD predications are compared for variants of unknown significance (VUS).D damaging, T tolerated, PD probably damaging.

Comparison of FATHMM, SIFT, PolyPhen2, and CADD predications and scores for VUS.

Scores from FATHMM, SIFT, PolyPhen, and CADD predications are compared for variants of unknown significance (VUS).

D damaging, T tolerated, PD probably damaging.

### Discussion

In this study, we reported 44 variants that are responsible for hereditary breast cancer and 37 VUSs that may be involved in hereditary and/or early onset of breast cancer in the Armenian population. The pathogenic/likely pathogenic group had 45 variants, 33 of which were unique, indicating that focusing on so-called founder mutations will miss the majority of the pathogenic/likely pathogenic variants causing breast cancer. We also reported 36 VUSs (31 unique), which were found in functional domains and mostly in mutational hotspots with the potential to interfere with protein function. These variants were found in known cancer-causing genes, such as ATR, BRCA1, CHEK2, MLH1, and MUTYH, and in two less prominent genes, SLX4 and WRAP53. The SLX4 complex is required for the efficient repair of DNA interstrand crosslinks (ICLs)14. The importance of SLX4 for ICL repair was underscored by the findings that biallelic mutations in SLX4 in humans cause Fanconi anemia (FA)15. The SLX1-SLX4 complex has a preference for 5’-flap structures and promotes symmetrical cleavage of static and migrating Holliday junctions (HJs). Finally, Wilson et al. reported that SLX1 foci could not be detected when they overexpressed a mutant form of SLX4 (p.Cys1805Arg) that is incapable of interacting with SLX116. The Variant SLX4 p.Cys1805Arg allele frequency in GnomAD exomes is 0.000008, which does not exceed the estimated maximal expected allele frequency for a pathogenic SLX4 variant of 0.0001, and the variant was not found in GnomAD genomes (PM2 Pathogenic Moderate). Pathogenic predictions from several programs support its deleterious effect (PP3 Pathogenic Supporting). Variant CHEK2 p.Asp438Tyr is in a hotspot region of 7 pathogenic nonsense and frameshift variants (PM1 Pathogenic Moderate). It has been shown to exhibit a 70% reduction in kinase activity in a cell-based assay when compared to that of wild-type CHEK217, suggesting that the variant may have a functional impact. However, it is unclear whether such a reduction in kinase activity would be sufficient to contribute to cancer risk. The allele frequencies of this variant in European and Finnish subpopulations are slightly higher than the estimated maximal expected allele frequency of a pathogenic CHEK2 variant (0.0000284), suggesting that this variant is likely a benign polymorphism. However, population data in this region of CHEK2 are not considered reliable due to high pseudogene homology18. The variant has been reported in breast, ovarian, and prostate cancer patients in the literature, without strong evidence for causality. However, one case–control study reported a significant association with prostate cancer19. It was reported as a VUS in male breast cancer20. In our study, this variant was found in two sisters with breast cancer at ages 44 and 51, and both carry the BRCA2 p.Val875Leu likely pathogenic variant. Variant MLH1 p.His318Gln is in a mutation hotspot of 22 pathogenic nonsense and frameshift variants (PM1 Pathogenic Moderate). This variant is not reported in either GnomAD exomes or in GnomAD genomes (PM2 Pathogenic Moderate). A total of 207 out of 273 non-VUS missense mutations in the MLH1 gene are pathogenic (PP2 Pathogenic Supporting). Pathogenicity prediction software does not agree on the potential impact of this missense change. The variant is reported in ClinVar as a VUS (Variation ID: 187486). This variant has not been reported in the literature in individuals with MLH1-related disease. The MLH1 p.His318Gln variant was identified in a patient (i.e., P-34) who also had the LPath variant p.Ala2951Ser in the BRCA2 gene. Since the evidence on the contribution of pathogenic variants in mismatch repair genes for breast cancer is not very strong, it is more likely that the LPath variant in BRCA2 plays a crucial role in the development of familial breast cancer, and the involvement of the variant p.His318Gln in the MLH1 gene in the patient’s breast cancer may be marginal or limited.

Variant MUTYH p.Leu420Met is in a mutation hotspot along with 12 pathogenic variants (PM1 Pathogenic Moderate). Additionally, 61 of 104 non-VUS missense mutations in the MUTYH gene were pathogenic (PP2 Pathogenic Supporting). The variant was observed in ExAC with an allele frequency of 71/120676 (1/1699), which does not exceed the estimated maximal expected allele frequency for a pathogenic MUTYH variant of 1/219. Pathogenic predictions from multiple programs support its deleterious effect (PP3 Pathogenic Supporting). This variant has been reported in a family with HNPCC and breast, pancreatic, and colorectal cancers. A functional study indicated that the variant acted comparable to wild-type21. Meanwhile, MUTYH heterozygous or homozygous mutations among breast cancer patients with or without a history of the disease evidenced an association of MUTYH with an increased risk of BC22. The MUTYH p.Leu420Met variant was identified in two patients (i.e., P-13 and P-70). Patient 13 also carried the LPath variant c.4358-2A>G in the BRCA1 gene, which could play a more crucial role in the development of this patient’s breast cancer. Therefore, the role of the MUTYH p.Leu420Met variant may also be marginal.

Variant p.Trp1815Ter in the BRCA1 gene was found in three unrelated female patients who were 26, 29, and 38 years old. In contrast, the male patient who also had this variant was 65 years old, a late onset for this variant in male patients. Variant p.Ile1159Metfs was found in two sets of siblings, two sisters at ages 31 and 32, and a brother/sister pair at ages 54 and 57. This difference in the age of onset for these two sibling pairs suggests a variable penetrance for this mutation. Perhaps the twin sisters had another genetic or environmental contributing factor for their early onset of breast cancer. Variant p.Leu1669Ter in BRCA2 was also seen in two sisters with breast cancer at ages 26 and 56; it was also seen in an 81-year-old female. This large age gap between these three patients is another example of deleterious mutations in a BRCA gene that manifest variable penetrance.

During analysis of the age distribution for variants, we realized that more BRCA1 and BRCA2 variants were classified in the pathogenic group. This was expected since these two genes are studied incomparably more than others and were the first genes to be tested for HBOC. This study and similar studies emphasize the significance of comprehensive gene panels to identify non-BRCA variants that could be involved in hereditary breast cancer. The percentage of found VUSs using a comprehensive cancer panel with 127 genes did increase the number of VUSs, and it allowed us to find several variants implicated in hereditary cancers, 8 of which have not been reported in suspected HBOC patients. VUS rates have been recently reported as 34.8% in a combined study conducted in Greece, Turkey, and Romania using a 36-gene panel23. Testing of 127 genes resulted in the identification of a 45% VUS rate, which is higher than the reported rate in the recent study where a 36-gene panel was used. However, this rate is quite reasonable, considering a threefold increase in the number of genes tested compared to the currently used panels.

In conclusion, our study identified variants involved in breast cancer in the Armenian population. We also reported nine novel variants (Tables 1 and 2) that, to the best of our knowledge, had not been reported previously in patients with breast cancer. We realized that variants with a higher frequency or possible founder mutations represented only 10% of the variants, thereby missing the rest. Thus, we concluded that testing with comprehensive cancer panels increases the chances of finding cancer-causing variants in genes that are not routinely tested for in breast cancer patients. These patients and perhaps their family members would need genetic counseling before and after testing to help them understand their treatment and prevention measures, such as surgical intervention, targeted therapy, and surveillance strategies.

### Supplementary information

Supplementary Table 2
Supplementary Table for all variants
Supplementary Materials for Variant Classificaiton

Supplementary Table 2

Supplementary Table for all variants

Supplementary Materials for Variant Classificaiton

Supplementary Table 2
Supplementary Table for all variants
Supplementary Materials for Variant Classificaiton

Supplementary Table 2

Supplementary Table for all variants

Supplementary Materials for Variant Classificaiton



# SUPPLEMENTAL FILE 1: 41439_2021_Article_140.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: PMC7870655

[File available at: automated_output/BRCA2/20251125_175029/pmc_fulltext/33558524_supplements/PMC7870655]

