# MAIN TEXT

## Mutational landscape and potential therapeutic targets for sporadic pancreatic neuroendocrine tumors based on target next-generation sequencing

### Abstract

Pancreatic neuroendocrine tumor (PNET), a heterogenous type of neoplasm with limited treatment options, is relatively rare and to date, the genetic background has remained to be fully elucidated. The present study aimed to determine the mutational landscape of PNET with and without liver metastasis, as well as its clinical application value for treatment. Fresh tumor tissues were collected from 14 patients with PNET following surgery, 4 of whom had developed liver metastasis. Subsequently, targeted next-generation sequencing of 612 cancer-associated genes and comprehensive analysis were performed on the tumor tissues. The results identified 63 somatic mutations in 53 genes in the 14 patients with PNET, amongst which menin 1 was identified as the most recurrently mutated gene. The analysis also identified several novel recurrently mutated genes, including adrenoceptor alpha 2B, ARVCF delta catenin family member, carbamoyl-phosphate synthetase 2, aspartate transcarbamylase, and dihydroorotase and neuregulin 1. Among the 53 mutated genes, 11 were enriched in the PI3K/AKT signaling pathway (adjusted P=7.12x10-5). In addition, 4 patients with PNET with liver metastasis had distinctly different mutational profiles compared with those without liver metastasis; 13 genes were discovered to be exclusively mutated in the liver metastasis group of the patients with PNET, including ATRX chromatin remodeler, thioredoxin reductase 2, anus kinase 3, ARVCF delta catenin family member, integrin subunit alpha V and RAD50 double strand break repair protein. In addition, two potentially actionable alterations in BRCA2 DNA repair-associated (p.Q548Q) and neurofibromin 1 (p.Q1188X) were identified using the OncoKB database. In conclusion, the present study generated a comprehensive mutational profile of 14 patients with PNET and further described the features of patients with liver metastasis, which highlights potential targets for drug development of PNET.

### Introduction

Pancreatic neuroendocrine tumor (PNET) is a relatively rare type of neuroendocrine malignancy, originating from the islets of Langerhans in the pancreas. PNETs comprise ~1-2% of all pancreatic neoplasms, demonstrating an annual incidence of 1 per 100,000 and a mortality rate of 60% (1-4). PNETs currently represent the second most common epithelial neoplasm after pancreatic ductal adenocarcinoma (PDAC) and are exhibiting a rising incidence and prevalence, particularly among Caucasians and Asians (4-6).

In total, ~85% of PNETs are non-functional, presenting with a worse prognosis compared with hormone-secreting functional PNETs (7,8). PNETs represent a diverse group of heterogeneous neoplasms with limited treatment options, which are morphologically and genetically different from PDACs and have far fewer mutations (6,9). Although biological classifications may help define the various clinical behaviors and thus guide treatment, to the best of our knowledge, the genetic background of PNETs remains to be fully characterized.

Menin 1 (MEN1), death domain associated protein (DAXX), ATRX chromatin remodeler (ATRX) and mTOR signaling pathway genes, including phosphatase and tensin homolog (PTEN), TSC complex subunit 1, TSC complex subunit 2, DEP domain containing 5, GATOR1 subcomplex subunit and phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit alpha (PIK3CA), were previously identified to be commonly mutated in PNETs (8,10). Additional novel germline mutations in the DNA repair genes, mutY DNA glycosylase, checkpoint kinase 2 and BRCA2 DNA repair-associated (BRCA2), and recurrent inactivating mutations or chromosomal rearrangements in chromatin-remodeling genes, including SET domain containing 2, histone lysine methyltransferase (SETD2), AT-rich interaction domain 1A (ARID1A) and lysine methyltransferase 2C (KMT2C), have also been reported (4). At the time of diagnosis, ~60% of patients with nonfunctional PNETs have developed liver metastases, which results in a worse clinical outcome (11). Therefore, an improved understanding of the genetic background in PNET liver metastases may pave the way for the development of potential novel treatment strategies.

In the present study, 14 patients with PNETs were recruited and target next-generation sequencing (NGS) was performed on the primary fresh tumor tissues. The hybridization capture-based NGS panel, Biotecan PanCancer Panoramic Detection (12), which was designed according to the cancer-related database, clinical guidelines and high-quality references of solid tumors, covered the exons and promoter areas of the 612 cancer-associated genes (2.75M). Based on the sequencing data, a mutational profile of 14 patients with PNET was constructed and differences in the mutational profile between patients with and without liver metastasis were further investigated. The study also evaluated the utility of the NGS panel to guide the clinical management of patients with PNET.

### Materials and methods

Fresh primary tumors were collected from 14 pathologically confirmed sporadic patients with PNETs following surgery at the Changhai Hospital (Shanghai, China) between February 2016 and September 2017. Images of H&E-stained tumor tissues of two patients are exhibited in Fig. S1A and B. The percentage of Ki-67 nuclear staining (Ki-67 index; Fig. S1C-F) (13) was used to determine the grade of PNET (14). The histological diagnosis of tumors was performed and confirmed by two pathologists. None of the patients had received any therapeutic procedures, including chemotherapy or radiotherapy, prior to surgery. Samples were immediately frozen in liquid nitrogen and stored at -80˚C until further analysis. The clinicopathological features of the 14 patients are presented in Table I.

Genomic (g)DNA from the fresh frozen tumor tissues was extracted using a QIAamp DNA Mini kit (Qiagen GmbH). The quantity and purity of the gDNA were assessed using a Qubit® 3.0 fluorometer (Invitrogen; Thermo Fisher Scientific, Inc.) and a NanoDrop ND-1000 (Thermo Fisher Scientific, Inc.). The fragmentation status was evaluated using the Agilent 2200 TapeStation system using the Genomic DNA ScreenTape assay (Agilent Technologies, Inc.) to produce a DNA integrity number. An additional QC step to determine the fresh frozen tissue DNA integrity was performed using a multiplex PCR approach (15). In brief, 30 ng gDNA was amplified using three different size sets of primers for the GAPDH gene (200-400 bp), and the concentration of the PCR products was determined using an Agilent 2100 Bioanalyzer instrument (Agilent Technologies, Inc.). Then, to estimate the fresh frozen tissue gDNA fragmentation, the average yield ratio value was determined by calculating the yield ratio of each amplicon compared with the Standard human genomic DNA (Promega Corproation; cat. no. G3041).

A total of 300 ng of each gDNA sample, which was based on Qubit quantification, was mechanically fragmented (duty factor 10%; peak incident power 175 W; cycles per burst 200; treatment time 240 sec; bath temperature 2-8˚C) on an E220 focused ultrasonicator (Covaris, Inc.). The target DNA fragment size is 150-200 bp. Subsequently, 200 ng sheared gDNA was used to perform end repair, A-tailing and adapter ligation with a KAPA library preparation kit (Kapa Biosystems Inc.), according to the manufacturer's protocol. Subsequently, the libraries were captured using Agilent SureSelect XT custom 0.5-2.9M probes (Agilent Technologies, Inc.) and amplified.

Following QC and quantification using the Agilent 2100 bioanalyzer (Agilent Technologies, Inc.) and a Qubit 2.0 fluorometer (Invitrogen; Thermo Fisher Scientific, Inc.), the libraries were sequenced on the Illumina Next 500 platform (Illumina, Inc.) in high-output mode, with 2x75 cycles.

Clean data were obtained after filtering out the adapters and reads with a proportion of N>10%, with N being the unidentified bases in the sequencing process, using fastp (fastp 0.19.4) (16). Low-quality bases (Phred score <15) were excised from the 3' ends of reads. Reads with length of <36 bp after excision were removed. The clean data were mapped to the reference human genome (University of California Santa Cruz ID: hg19; GenBank accession no. GCA_000001405.1) using the BWA alignment algorithm (BWA. 0.7.17) (17). The alignment in the SAM format was converted to BAM files using SAMtools (Samtools 1.9.0) (18). Next, the genome analysis toolkit (GATK; v4.0.2.1) (19) was used for sorting, duplicate marking and base recalibration. The final BAM files were analyzed using QualiMap v.2.2.1(20) to provide an overall overview of the data, including mapped reads, mean mapping quality and mean coverage. The variants [single nucleotide variants (SNV) and insertion (Ins)-deletion (Del) mutations (InDels)] were called for unpaired tumor sequences with 40 pooled blood samples (from healthy individuals) using the GATK mutect2 tumor-only mode with the parameter af-alleles-not-in-resource 0.00025%, and the germline mutations and contaminations were filtered out using GATK FilterMutectCalls with parameters (max-germline-posterior 0.995). Somatic variants were annotated using the ANNOVAR software tool.

The following filter conditions were used to identify the candidate somatic alterations: i) all variations with COSMIC evidence (http://cancer.sanger.ac.uk/cosmic) were retained; ii) the variants with a mutational allele frequency (maf) >0.001 in the 1,000 Genomes databases (1,000 Genomes Project Consortium; https://www.internationalgenome.org/) (21) were removed; iii) the functional benign variant sites predicted by PolyPhen-2(22), SIFT (23), MutationTaster (24) and Combined Annotation-Dependent Depletion (25) were removed; and iv) only the following variant classifications were retained: Missense_Mutation, Nonsense_Mutation, Splice_Site, Frame_Shift_Ins, Frame_Shift_Del, In_Frame_Ins, In_Frame_Del.

The mutational landscape across the cohort was created using the maftools package in R software (R 3.5.1, R Core Team; https://www.R-Project.org). Kyoto Encyclopedia of Genes and Genomes (KEGG) signaling pathway enrichment analysis (https://www.kegg.jp/) was performed to investigate the biological importance of the altered genes in all samples using the clusterProfiler package (26) in R software (R 3.5.1). A cut-off value of the adjusted P-value (p.adjust) of <0.05 was used to identify significantly enriched terms.

### 

Fresh primary tumors were collected from 14 pathologically confirmed sporadic patients with PNETs following surgery at the Changhai Hospital (Shanghai, China) between February 2016 and September 2017. Images of H&E-stained tumor tissues of two patients are exhibited in Fig. S1A and B. The percentage of Ki-67 nuclear staining (Ki-67 index; Fig. S1C-F) (13) was used to determine the grade of PNET (14). The histological diagnosis of tumors was performed and confirmed by two pathologists. None of the patients had received any therapeutic procedures, including chemotherapy or radiotherapy, prior to surgery. Samples were immediately frozen in liquid nitrogen and stored at -80˚C until further analysis. The clinicopathological features of the 14 patients are presented in Table I.

Genomic (g)DNA from the fresh frozen tumor tissues was extracted using a QIAamp DNA Mini kit (Qiagen GmbH). The quantity and purity of the gDNA were assessed using a Qubit® 3.0 fluorometer (Invitrogen; Thermo Fisher Scientific, Inc.) and a NanoDrop ND-1000 (Thermo Fisher Scientific, Inc.). The fragmentation status was evaluated using the Agilent 2200 TapeStation system using the Genomic DNA ScreenTape assay (Agilent Technologies, Inc.) to produce a DNA integrity number. An additional QC step to determine the fresh frozen tissue DNA integrity was performed using a multiplex PCR approach (15). In brief, 30 ng gDNA was amplified using three different size sets of primers for the GAPDH gene (200-400 bp), and the concentration of the PCR products was determined using an Agilent 2100 Bioanalyzer instrument (Agilent Technologies, Inc.). Then, to estimate the fresh frozen tissue gDNA fragmentation, the average yield ratio value was determined by calculating the yield ratio of each amplicon compared with the Standard human genomic DNA (Promega Corproation; cat. no. G3041).

A total of 300 ng of each gDNA sample, which was based on Qubit quantification, was mechanically fragmented (duty factor 10%; peak incident power 175 W; cycles per burst 200; treatment time 240 sec; bath temperature 2-8˚C) on an E220 focused ultrasonicator (Covaris, Inc.). The target DNA fragment size is 150-200 bp. Subsequently, 200 ng sheared gDNA was used to perform end repair, A-tailing and adapter ligation with a KAPA library preparation kit (Kapa Biosystems Inc.), according to the manufacturer's protocol. Subsequently, the libraries were captured using Agilent SureSelect XT custom 0.5-2.9M probes (Agilent Technologies, Inc.) and amplified.

Following QC and quantification using the Agilent 2100 bioanalyzer (Agilent Technologies, Inc.) and a Qubit 2.0 fluorometer (Invitrogen; Thermo Fisher Scientific, Inc.), the libraries were sequenced on the Illumina Next 500 platform (Illumina, Inc.) in high-output mode, with 2x75 cycles.

Clean data were obtained after filtering out the adapters and reads with a proportion of N>10%, with N being the unidentified bases in the sequencing process, using fastp (fastp 0.19.4) (16). Low-quality bases (Phred score <15) were excised from the 3' ends of reads. Reads with length of <36 bp after excision were removed. The clean data were mapped to the reference human genome (University of California Santa Cruz ID: hg19; GenBank accession no. GCA_000001405.1) using the BWA alignment algorithm (BWA. 0.7.17) (17). The alignment in the SAM format was converted to BAM files using SAMtools (Samtools 1.9.0) (18). Next, the genome analysis toolkit (GATK; v4.0.2.1) (19) was used for sorting, duplicate marking and base recalibration. The final BAM files were analyzed using QualiMap v.2.2.1(20) to provide an overall overview of the data, including mapped reads, mean mapping quality and mean coverage. The variants [single nucleotide variants (SNV) and insertion (Ins)-deletion (Del) mutations (InDels)] were called for unpaired tumor sequences with 40 pooled blood samples (from healthy individuals) using the GATK mutect2 tumor-only mode with the parameter af-alleles-not-in-resource 0.00025%, and the germline mutations and contaminations were filtered out using GATK FilterMutectCalls with parameters (max-germline-posterior 0.995). Somatic variants were annotated using the ANNOVAR software tool.

The following filter conditions were used to identify the candidate somatic alterations: i) all variations with COSMIC evidence (http://cancer.sanger.ac.uk/cosmic) were retained; ii) the variants with a mutational allele frequency (maf) >0.001 in the 1,000 Genomes databases (1,000 Genomes Project Consortium; https://www.internationalgenome.org/) (21) were removed; iii) the functional benign variant sites predicted by PolyPhen-2(22), SIFT (23), MutationTaster (24) and Combined Annotation-Dependent Depletion (25) were removed; and iv) only the following variant classifications were retained: Missense_Mutation, Nonsense_Mutation, Splice_Site, Frame_Shift_Ins, Frame_Shift_Del, In_Frame_Ins, In_Frame_Del.

The mutational landscape across the cohort was created using the maftools package in R software (R 3.5.1, R Core Team; https://www.R-Project.org). Kyoto Encyclopedia of Genes and Genomes (KEGG) signaling pathway enrichment analysis (https://www.kegg.jp/) was performed to investigate the biological importance of the altered genes in all samples using the clusterProfiler package (26) in R software (R 3.5.1). A cut-off value of the adjusted P-value (p.adjust) of <0.05 was used to identify significantly enriched terms.

### Patient samples

Fresh primary tumors were collected from 14 pathologically confirmed sporadic patients with PNETs following surgery at the Changhai Hospital (Shanghai, China) between February 2016 and September 2017. Images of H&E-stained tumor tissues of two patients are exhibited in Fig. S1A and B. The percentage of Ki-67 nuclear staining (Ki-67 index; Fig. S1C-F) (13) was used to determine the grade of PNET (14). The histological diagnosis of tumors was performed and confirmed by two pathologists. None of the patients had received any therapeutic procedures, including chemotherapy or radiotherapy, prior to surgery. Samples were immediately frozen in liquid nitrogen and stored at -80˚C until further analysis. The clinicopathological features of the 14 patients are presented in Table I.

### DNA extraction and quality control (QC)

Genomic (g)DNA from the fresh frozen tumor tissues was extracted using a QIAamp DNA Mini kit (Qiagen GmbH). The quantity and purity of the gDNA were assessed using a Qubit® 3.0 fluorometer (Invitrogen; Thermo Fisher Scientific, Inc.) and a NanoDrop ND-1000 (Thermo Fisher Scientific, Inc.). The fragmentation status was evaluated using the Agilent 2200 TapeStation system using the Genomic DNA ScreenTape assay (Agilent Technologies, Inc.) to produce a DNA integrity number. An additional QC step to determine the fresh frozen tissue DNA integrity was performed using a multiplex PCR approach (15). In brief, 30 ng gDNA was amplified using three different size sets of primers for the GAPDH gene (200-400 bp), and the concentration of the PCR products was determined using an Agilent 2100 Bioanalyzer instrument (Agilent Technologies, Inc.). Then, to estimate the fresh frozen tissue gDNA fragmentation, the average yield ratio value was determined by calculating the yield ratio of each amplicon compared with the Standard human genomic DNA (Promega Corproation; cat. no. G3041).

### Library preparation, hybridization capture and amplification

A total of 300 ng of each gDNA sample, which was based on Qubit quantification, was mechanically fragmented (duty factor 10%; peak incident power 175 W; cycles per burst 200; treatment time 240 sec; bath temperature 2-8˚C) on an E220 focused ultrasonicator (Covaris, Inc.). The target DNA fragment size is 150-200 bp. Subsequently, 200 ng sheared gDNA was used to perform end repair, A-tailing and adapter ligation with a KAPA library preparation kit (Kapa Biosystems Inc.), according to the manufacturer's protocol. Subsequently, the libraries were captured using Agilent SureSelect XT custom 0.5-2.9M probes (Agilent Technologies, Inc.) and amplified.

### Illumina sequencing

Following QC and quantification using the Agilent 2100 bioanalyzer (Agilent Technologies, Inc.) and a Qubit 2.0 fluorometer (Invitrogen; Thermo Fisher Scientific, Inc.), the libraries were sequenced on the Illumina Next 500 platform (Illumina, Inc.) in high-output mode, with 2x75 cycles.

### Bioinformatics analysis

Clean data were obtained after filtering out the adapters and reads with a proportion of N>10%, with N being the unidentified bases in the sequencing process, using fastp (fastp 0.19.4) (16). Low-quality bases (Phred score <15) were excised from the 3' ends of reads. Reads with length of <36 bp after excision were removed. The clean data were mapped to the reference human genome (University of California Santa Cruz ID: hg19; GenBank accession no. GCA_000001405.1) using the BWA alignment algorithm (BWA. 0.7.17) (17). The alignment in the SAM format was converted to BAM files using SAMtools (Samtools 1.9.0) (18). Next, the genome analysis toolkit (GATK; v4.0.2.1) (19) was used for sorting, duplicate marking and base recalibration. The final BAM files were analyzed using QualiMap v.2.2.1(20) to provide an overall overview of the data, including mapped reads, mean mapping quality and mean coverage. The variants [single nucleotide variants (SNV) and insertion (Ins)-deletion (Del) mutations (InDels)] were called for unpaired tumor sequences with 40 pooled blood samples (from healthy individuals) using the GATK mutect2 tumor-only mode with the parameter af-alleles-not-in-resource 0.00025%, and the germline mutations and contaminations were filtered out using GATK FilterMutectCalls with parameters (max-germline-posterior 0.995). Somatic variants were annotated using the ANNOVAR software tool.

The following filter conditions were used to identify the candidate somatic alterations: i) all variations with COSMIC evidence (http://cancer.sanger.ac.uk/cosmic) were retained; ii) the variants with a mutational allele frequency (maf) >0.001 in the 1,000 Genomes databases (1,000 Genomes Project Consortium; https://www.internationalgenome.org/) (21) were removed; iii) the functional benign variant sites predicted by PolyPhen-2(22), SIFT (23), MutationTaster (24) and Combined Annotation-Dependent Depletion (25) were removed; and iv) only the following variant classifications were retained: Missense_Mutation, Nonsense_Mutation, Splice_Site, Frame_Shift_Ins, Frame_Shift_Del, In_Frame_Ins, In_Frame_Del.

### Statistical analysis

The mutational landscape across the cohort was created using the maftools package in R software (R 3.5.1, R Core Team; https://www.R-Project.org). Kyoto Encyclopedia of Genes and Genomes (KEGG) signaling pathway enrichment analysis (https://www.kegg.jp/) was performed to investigate the biological importance of the altered genes in all samples using the clusterProfiler package (26) in R software (R 3.5.1). A cut-off value of the adjusted P-value (p.adjust) of <0.05 was used to identify significantly enriched terms.

### Results

The demographic and baseline characteristics of the study participants are summarized in Table I. A total of 14 patients with PNET were analyzed in the present study, including 7 males and 7 females, with a median age of 50 years (age range, 28-65 years). In the study cohort, all tumors of patients were non-functional PNETs, which do not make hormones, or make hormones that do not cause a set of symptoms. Among them, 4 patients had tumors located in the pancreas head and the other 10 patients had tumors located in the pancreas tail. Regarding the World Health Organization grade (27), 1 patient had a grade (G)1 classified NET, 11 patients had G2 classified NETs and 2 patients had G3 classified NETs.

In 13 patients, the median follow-up time was 20 months (range, 5-38 months; Table SI). One patient (case no. 11) was lost to follow-up. Among the 14 patients who were examined for lymph node status, 7 patients were identified to have lymph node metastasis, while nerve invasion and vascular tumor thrombus were present in 4 patients, and 4 patients demonstrated distant metastasis to the liver. Furthermore, 2 patients also presented with type II diabetes mellitus (T2DM). The majority of the patients remained alive during the follow-up, except for 1 patient (case no. 8), who died at 15 months following surgery. In addition, 1 patient (case no. 13) developed liver metastasis at 7 months following surgery. The percentage of Ki-67 of 14 patients varied between 1 and 40% (Table SI).

Fig. S2 presents a summary of the maf files (Table SII) in the 14 patients with PNET. In total, 63 variations in 53 genes were identified, involving 5 variant classifications. For the SNVs, C>T was the most frequent SNV class. The median number of variants identified in the 14 samples was 5 (range, 1-8).

Fig. 1 presents the mutational profile of 14 patients with PNET, including the top 20 recurrently mutated genes, according to the presence or absence of liver metastasis. The mutated gene with the highest frequency was MEN1 (4/14; 29%), which was consistent with the previously reported mutation rate of MEN1 (23.1-56%) in PNET (5,6). The other recurrent altered genes were revealed to be adrenoceptor alpha 2B (ADRA2B; 3/14; 21%), ARVCF delta catenin family member (ARVCF; 2/14; 14%), carbamoyl-phosphate synthetase 2, aspartate transcarbamylase, and dihydroorotase (CAD; 2/14; 14%), neuregulin 1 (NRG1; 2/14; 14%), tumor protein p53 (TP53; 2/14; 14%) and thioredoxin reductase 2 (TXNRD2; 2/14; 14%) (Fig. 1). ATRX was only identified to be mutated in one case (1/14; 7%), which was lower than the reported rate of somatic mutations (13-25%) (5,28). Fig. 2 illustrates the frequency of mutations and the resulting protein structure of MEN1 and ADRA2B, while that of other genes (mutation frequency ≥2/14), including ARVCF, CAD, NRG1, TP53 and TXNRD2, are presented in Fig. S3.

To further investigate the biological functions of the mutated genes, KEGG signaling pathway enrichment analyses were performed. Table SIII presents all of the significantly enriched signaling pathways associated with the mutated genes. The top 15 significantly altered signaling pathways based on p.adjust are presented in Fig. 3A. In total, 11 mutated genes were identified to be enriched in the PI3K/AKT signaling pathway (p.adjust=7.12x10-5), including fibroblast growth factor receptor 3 (FGFR3), Janus kinase 3 (JAK3), integrin subunit alpha V (ITGAV), EPH receptor A2 (EPHA2), phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit delta (PIK3CD), integrin subunit beta 1 (ITGB1), fibroblast growth factor receptor 2 (FGFR2), phosphoinositide-3-kinase regulatory subunit 3 (PIK3R3), erb-b2 receptor tyrosine kinase 4 (ERBB4), TP53 and PTEN. The PI3K/AKT signaling pathway is a well-established driver of carcinogenesis and has been reported in various types of cancer, such as gastric (29), breast (30) and colorectal cancer (31). Other well-known cancer-associated signaling pathways were also identified, including the focal adhesion, ErbB signaling pathway, Ras signaling pathway and MAPK signaling pathway. Endocrine-related signaling pathways, including the insulin resistance pathway, were also identified.

Three genes were discovered to be altered in the two groups, including MEN1, CAD and NRG1 (Fig. 3B). In addition, 13 genes were exclusively mutated in those patients with PNET who developed liver metastasis, including ATRX, TXNRD2, JAK3, ARVCF, ITGAV, RAD50 double strand break repair protein (RAD50), protein kinase C delta (PRKCD), tubulin alpha 3c (TUBA3C), DNA methyltransferase 3 alpha (DNMT3A), PIK3R3, integrin subunit alpha L (ITGAL), partner and localizer of BRCA2 (PALB2) and pyruvate dehydrogenase kinase 2 (PDK2). The results of the KEGG signaling pathway enrichment analysis of these 13 genes is presented in Table SIV, and it was discovered that the genes were involved in ‘Homologous recombination’, ‘Chemokine signaling pathway’ and ‘Type II diabetes mellitus’.

To determine the clinical value of these genomic alterations, the OncoKB-annotator (https://github.com/oncokb/oncokb-annotator) was used to group all alterations, without filtering the conditions, into various levels according to the evidence of clinical actionability. Altogether, two potentially actionable genomic alterations in BRCA2 (p.Q548Q) and neurofibromin 1 (NF1; p.Q1188X) were detected in patient nos. 14 and 8, respectively (Fig. 4). The evidence level for BRCA2 (p.Q548Q) for patients with PNET was LEVEL_2B, which was defined by OncoKB as a standard care biomarker recommended by the National Comprehensive Cancer Network (NCCN, https://education.nccn.org/) or other expert panels as predictive of a response to a Food and Drug Administration-approved drug; the corresponding drugs were niraparib, olaparib, rucaparib and talazoparib. The evidence level for NF1 (p.Q1188X) was LEVEL_4, which was defined as compelling biological evidence supporting the biomarker as being predictive of a response to a drug, and the corresponding drugs were cobimetinib and trametinib.

### 

The demographic and baseline characteristics of the study participants are summarized in Table I. A total of 14 patients with PNET were analyzed in the present study, including 7 males and 7 females, with a median age of 50 years (age range, 28-65 years). In the study cohort, all tumors of patients were non-functional PNETs, which do not make hormones, or make hormones that do not cause a set of symptoms. Among them, 4 patients had tumors located in the pancreas head and the other 10 patients had tumors located in the pancreas tail. Regarding the World Health Organization grade (27), 1 patient had a grade (G)1 classified NET, 11 patients had G2 classified NETs and 2 patients had G3 classified NETs.

In 13 patients, the median follow-up time was 20 months (range, 5-38 months; Table SI). One patient (case no. 11) was lost to follow-up. Among the 14 patients who were examined for lymph node status, 7 patients were identified to have lymph node metastasis, while nerve invasion and vascular tumor thrombus were present in 4 patients, and 4 patients demonstrated distant metastasis to the liver. Furthermore, 2 patients also presented with type II diabetes mellitus (T2DM). The majority of the patients remained alive during the follow-up, except for 1 patient (case no. 8), who died at 15 months following surgery. In addition, 1 patient (case no. 13) developed liver metastasis at 7 months following surgery. The percentage of Ki-67 of 14 patients varied between 1 and 40% (Table SI).

Fig. S2 presents a summary of the maf files (Table SII) in the 14 patients with PNET. In total, 63 variations in 53 genes were identified, involving 5 variant classifications. For the SNVs, C>T was the most frequent SNV class. The median number of variants identified in the 14 samples was 5 (range, 1-8).

Fig. 1 presents the mutational profile of 14 patients with PNET, including the top 20 recurrently mutated genes, according to the presence or absence of liver metastasis. The mutated gene with the highest frequency was MEN1 (4/14; 29%), which was consistent with the previously reported mutation rate of MEN1 (23.1-56%) in PNET (5,6). The other recurrent altered genes were revealed to be adrenoceptor alpha 2B (ADRA2B; 3/14; 21%), ARVCF delta catenin family member (ARVCF; 2/14; 14%), carbamoyl-phosphate synthetase 2, aspartate transcarbamylase, and dihydroorotase (CAD; 2/14; 14%), neuregulin 1 (NRG1; 2/14; 14%), tumor protein p53 (TP53; 2/14; 14%) and thioredoxin reductase 2 (TXNRD2; 2/14; 14%) (Fig. 1). ATRX was only identified to be mutated in one case (1/14; 7%), which was lower than the reported rate of somatic mutations (13-25%) (5,28). Fig. 2 illustrates the frequency of mutations and the resulting protein structure of MEN1 and ADRA2B, while that of other genes (mutation frequency ≥2/14), including ARVCF, CAD, NRG1, TP53 and TXNRD2, are presented in Fig. S3.

To further investigate the biological functions of the mutated genes, KEGG signaling pathway enrichment analyses were performed. Table SIII presents all of the significantly enriched signaling pathways associated with the mutated genes. The top 15 significantly altered signaling pathways based on p.adjust are presented in Fig. 3A. In total, 11 mutated genes were identified to be enriched in the PI3K/AKT signaling pathway (p.adjust=7.12x10-5), including fibroblast growth factor receptor 3 (FGFR3), Janus kinase 3 (JAK3), integrin subunit alpha V (ITGAV), EPH receptor A2 (EPHA2), phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit delta (PIK3CD), integrin subunit beta 1 (ITGB1), fibroblast growth factor receptor 2 (FGFR2), phosphoinositide-3-kinase regulatory subunit 3 (PIK3R3), erb-b2 receptor tyrosine kinase 4 (ERBB4), TP53 and PTEN. The PI3K/AKT signaling pathway is a well-established driver of carcinogenesis and has been reported in various types of cancer, such as gastric (29), breast (30) and colorectal cancer (31). Other well-known cancer-associated signaling pathways were also identified, including the focal adhesion, ErbB signaling pathway, Ras signaling pathway and MAPK signaling pathway. Endocrine-related signaling pathways, including the insulin resistance pathway, were also identified.

Three genes were discovered to be altered in the two groups, including MEN1, CAD and NRG1 (Fig. 3B). In addition, 13 genes were exclusively mutated in those patients with PNET who developed liver metastasis, including ATRX, TXNRD2, JAK3, ARVCF, ITGAV, RAD50 double strand break repair protein (RAD50), protein kinase C delta (PRKCD), tubulin alpha 3c (TUBA3C), DNA methyltransferase 3 alpha (DNMT3A), PIK3R3, integrin subunit alpha L (ITGAL), partner and localizer of BRCA2 (PALB2) and pyruvate dehydrogenase kinase 2 (PDK2). The results of the KEGG signaling pathway enrichment analysis of these 13 genes is presented in Table SIV, and it was discovered that the genes were involved in ‘Homologous recombination’, ‘Chemokine signaling pathway’ and ‘Type II diabetes mellitus’.

To determine the clinical value of these genomic alterations, the OncoKB-annotator (https://github.com/oncokb/oncokb-annotator) was used to group all alterations, without filtering the conditions, into various levels according to the evidence of clinical actionability. Altogether, two potentially actionable genomic alterations in BRCA2 (p.Q548Q) and neurofibromin 1 (NF1; p.Q1188X) were detected in patient nos. 14 and 8, respectively (Fig. 4). The evidence level for BRCA2 (p.Q548Q) for patients with PNET was LEVEL_2B, which was defined by OncoKB as a standard care biomarker recommended by the National Comprehensive Cancer Network (NCCN, https://education.nccn.org/) or other expert panels as predictive of a response to a Food and Drug Administration-approved drug; the corresponding drugs were niraparib, olaparib, rucaparib and talazoparib. The evidence level for NF1 (p.Q1188X) was LEVEL_4, which was defined as compelling biological evidence supporting the biomarker as being predictive of a response to a drug, and the corresponding drugs were cobimetinib and trametinib.

### Demographic and baseline characteristics of the patients

The demographic and baseline characteristics of the study participants are summarized in Table I. A total of 14 patients with PNET were analyzed in the present study, including 7 males and 7 females, with a median age of 50 years (age range, 28-65 years). In the study cohort, all tumors of patients were non-functional PNETs, which do not make hormones, or make hormones that do not cause a set of symptoms. Among them, 4 patients had tumors located in the pancreas head and the other 10 patients had tumors located in the pancreas tail. Regarding the World Health Organization grade (27), 1 patient had a grade (G)1 classified NET, 11 patients had G2 classified NETs and 2 patients had G3 classified NETs.

In 13 patients, the median follow-up time was 20 months (range, 5-38 months; Table SI). One patient (case no. 11) was lost to follow-up. Among the 14 patients who were examined for lymph node status, 7 patients were identified to have lymph node metastasis, while nerve invasion and vascular tumor thrombus were present in 4 patients, and 4 patients demonstrated distant metastasis to the liver. Furthermore, 2 patients also presented with type II diabetes mellitus (T2DM). The majority of the patients remained alive during the follow-up, except for 1 patient (case no. 8), who died at 15 months following surgery. In addition, 1 patient (case no. 13) developed liver metastasis at 7 months following surgery. The percentage of Ki-67 of 14 patients varied between 1 and 40% (Table SI).

### Recurrent mutated genes in PNET with and without liver metastasis

Fig. S2 presents a summary of the maf files (Table SII) in the 14 patients with PNET. In total, 63 variations in 53 genes were identified, involving 5 variant classifications. For the SNVs, C>T was the most frequent SNV class. The median number of variants identified in the 14 samples was 5 (range, 1-8).

Fig. 1 presents the mutational profile of 14 patients with PNET, including the top 20 recurrently mutated genes, according to the presence or absence of liver metastasis. The mutated gene with the highest frequency was MEN1 (4/14; 29%), which was consistent with the previously reported mutation rate of MEN1 (23.1-56%) in PNET (5,6). The other recurrent altered genes were revealed to be adrenoceptor alpha 2B (ADRA2B; 3/14; 21%), ARVCF delta catenin family member (ARVCF; 2/14; 14%), carbamoyl-phosphate synthetase 2, aspartate transcarbamylase, and dihydroorotase (CAD; 2/14; 14%), neuregulin 1 (NRG1; 2/14; 14%), tumor protein p53 (TP53; 2/14; 14%) and thioredoxin reductase 2 (TXNRD2; 2/14; 14%) (Fig. 1). ATRX was only identified to be mutated in one case (1/14; 7%), which was lower than the reported rate of somatic mutations (13-25%) (5,28). Fig. 2 illustrates the frequency of mutations and the resulting protein structure of MEN1 and ADRA2B, while that of other genes (mutation frequency ≥2/14), including ARVCF, CAD, NRG1, TP53 and TXNRD2, are presented in Fig. S3.

### KEGG signaling pathway enrichment analysis of all somatically mutated genes

To further investigate the biological functions of the mutated genes, KEGG signaling pathway enrichment analyses were performed. Table SIII presents all of the significantly enriched signaling pathways associated with the mutated genes. The top 15 significantly altered signaling pathways based on p.adjust are presented in Fig. 3A. In total, 11 mutated genes were identified to be enriched in the PI3K/AKT signaling pathway (p.adjust=7.12x10-5), including fibroblast growth factor receptor 3 (FGFR3), Janus kinase 3 (JAK3), integrin subunit alpha V (ITGAV), EPH receptor A2 (EPHA2), phosphatidylinositol-4,5-bisphosphate 3-kinase catalytic subunit delta (PIK3CD), integrin subunit beta 1 (ITGB1), fibroblast growth factor receptor 2 (FGFR2), phosphoinositide-3-kinase regulatory subunit 3 (PIK3R3), erb-b2 receptor tyrosine kinase 4 (ERBB4), TP53 and PTEN. The PI3K/AKT signaling pathway is a well-established driver of carcinogenesis and has been reported in various types of cancer, such as gastric (29), breast (30) and colorectal cancer (31). Other well-known cancer-associated signaling pathways were also identified, including the focal adhesion, ErbB signaling pathway, Ras signaling pathway and MAPK signaling pathway. Endocrine-related signaling pathways, including the insulin resistance pathway, were also identified.

### Differences in the mutational profile between two groups with/without liver metastasis

Three genes were discovered to be altered in the two groups, including MEN1, CAD and NRG1 (Fig. 3B). In addition, 13 genes were exclusively mutated in those patients with PNET who developed liver metastasis, including ATRX, TXNRD2, JAK3, ARVCF, ITGAV, RAD50 double strand break repair protein (RAD50), protein kinase C delta (PRKCD), tubulin alpha 3c (TUBA3C), DNA methyltransferase 3 alpha (DNMT3A), PIK3R3, integrin subunit alpha L (ITGAL), partner and localizer of BRCA2 (PALB2) and pyruvate dehydrogenase kinase 2 (PDK2). The results of the KEGG signaling pathway enrichment analysis of these 13 genes is presented in Table SIV, and it was discovered that the genes were involved in ‘Homologous recombination’, ‘Chemokine signaling pathway’ and ‘Type II diabetes mellitus’.

### Actionable genomic alterations in PNETs

To determine the clinical value of these genomic alterations, the OncoKB-annotator (https://github.com/oncokb/oncokb-annotator) was used to group all alterations, without filtering the conditions, into various levels according to the evidence of clinical actionability. Altogether, two potentially actionable genomic alterations in BRCA2 (p.Q548Q) and neurofibromin 1 (NF1; p.Q1188X) were detected in patient nos. 14 and 8, respectively (Fig. 4). The evidence level for BRCA2 (p.Q548Q) for patients with PNET was LEVEL_2B, which was defined by OncoKB as a standard care biomarker recommended by the National Comprehensive Cancer Network (NCCN, https://education.nccn.org/) or other expert panels as predictive of a response to a Food and Drug Administration-approved drug; the corresponding drugs were niraparib, olaparib, rucaparib and talazoparib. The evidence level for NF1 (p.Q1188X) was LEVEL_4, which was defined as compelling biological evidence supporting the biomarker as being predictive of a response to a drug, and the corresponding drugs were cobimetinib and trametinib.

### Discussion

The current treatments for PNET include medical therapy, surgery and radiotherapy, which are frequently unsuccessful, as the median survival time for patients with PNET is as low as ~3.6 years (2). Targeted therapies may provide a potential benefit for the patients; however, in order to develop these therapies, an improved understanding of the genetic landscape of PNET is required. In the present study, 14 PNET samples were analyzed in-depth by NGS-based gene panel sequencing; the study identified several novel recurrent genomic alterations in ADRA2B, ARVCF, CAD and NRG1, as well as the enrichment of altered genes in the PI3K/AKT signaling pathway.

Mutations in MEN1, DAXX/ATRX and mTOR signaling pathway genes have been frequently observed in ~55-65% of patients with non-functional PNET (28). In the present study, a hybridization capture-based NGS panel was constructed to detect the genomic alterations in 612 cancer-associated genes. With this method, 14 cases of nonfunctional PNET with or without liver metastasis were sequenced. The highest frequency of the altered driver gene, MEN1, was also identified in the present study cohort, which was altered in four patients (4/14; 29%). MEN1 encodes menin, a tumor suppressor associated with a syndrome known as multiple endocrine neoplasia type 1(32). Its inactivation drives various phenotypes (4) which involves widespread transcriptional dysregulation via histone modifications (33), the activation of mTOR through AKT expression (34), the suppression of homologous recombination DNA damage response genes (35) and the dysregulation of telomerase reverse transcriptase (36). In addition, the altered genes were discovered to be highly enriched in the PI3K/AKT signaling pathway; this is consistent with other previous studies, which reported that mTOR pathway genes were frequently altered in PNETs and suggested the potential benefits of mTOR inhibitors, including everolimus, for Chinese patients with PNET.

In addition, the results of the present study revealed that the patients with PNETs that developed liver metastasis had distinct mutational profiles in the primary tumors compared with cases without liver metastasis. Lawrence et al (37) reported that there was a high consistency in the genome sequence, structure and expression between the primary tumors and hepatic metastases in one patient with PNET; however, the metastatic tumor lost certain variations or expression features observed in the primary tumor, whilst gaining certain de novo changes. In the present study, 13 genes were identified to be exclusively mutated in the patients who had developed liver metastasis, which were significantly enriched in 7 signaling pathways. The present study paid close attention to T2DM (p.adjust=0.030), which was enriched for 2 mutated genes, PRKCD and PIK3R3. Diabetes mellitus is an endocrine disease, which frequently occurs in patients with PNET. Long-standing diabetes mellitus has been discovered as a risk factor for PNET development (38,39), and has been suggested to be associated with poor prognosis (40). Fan et al (41) determined that patients with PNET with T2DM were at an increased risk for tumor metastasis (odds ratio=2.81; P=0.001) through investigating the clinicopathological characteristics of 299 patients with PNET. However, in the present study, the two patients with PNET and T2DM did not demonstrate liver metastasis. These results may reflect the substantial research in a large group of patients, which is further required to determine the relationship between T2DM and liver metastasis in PNETs.

In the present study, two potentially actionable genomic alterations in BRCA2 (p.Q548Q) and NF1 (p.Q1188X) were identified. BRCA2 is one of the most well-described cancer susceptibility genes, which serves critical roles in homologous DNA repair. High-risk germline or somatic mutations in BRCA2 usually lead to the defective repair of double-strand DNA breaks. BRCA2 mutations have been associated with multiple types of cancer, including breast, ovarian, pancreatic and prostate cancer (42). Several studies or case reports have reported the existence of BRCA2 germline mutations in PNET (4,6,43), which is consistent with the results of the present study and highlights the germline contribution to clinically sporadic PNETs. NF1 resides on chromosome 17 and encodes neurofibromin, a 220 kDa cytoplasmic protein, which is both a negative regulator of Ras and a positive regulator of adenylyl cyclase, the enzyme responsible for the generation of intracellular cyclic AMP (cAMP). Therefore, the loss of NF1 promotes the hyperactivation of downstream effector proteins of Ras signaling, including mTOR and MAPK kinase (MEK), and decreased intracellular cAMP levels. Germline mutations in NF1 were discovered to result in neurofibromatosis type 1, a complex autosomal-dominant disorder that affects multiple organ systems (44,45). PNET is an uncommon clinical manifestation of the NF1 syndrome and is reported to be present in <10% of cases (46). Previously, in an anecdotal report, the heterozygous germline mutation c.499 del TGTT was identified in a patient with NF1 syndrome and a well-differentiated pancreatic endocrine carcinoma (47). To the best of our knowledge, the presence of the NF1 mutant p.Q1188X has not been previously reported in nonfunctional PNET, which may be due to the difficulty to identify causative mutations in NF1 due to the large size of the gene (~60 exons) and the diversity of clinical manifestations. However, whether this novel alteration of NF1 identified in the present study is targetable remains elusive. Further functional annotation is required to guide the clinical application of the MEK inhibitors, namely cobimetinib and trametinib.

Although the present study provided novel results, there are several limitations worthy of consideration. One limitation was the small sample size used in the study; only 14 patients with PNET were included and the present study was a retrospective study performed at a single center, which limits the generalizability of the results to the overall population of patients with PNET. Gene Ontology enrichment also should be used to investigate the biological functions of the mutated genes in the future large cohort study. Furthermore, functional studies in vitro were not performed to determine the biological effects of these mutated genes, which may also limit the significance of the results of the present study.

In conclusion, the present study identified several novel recurrent genomic alterations and confirmed the enrichment of gene alterations in the PI3K/AKT signaling pathway in a small cohort of Chinese patients with PNET. These results may shed light on opportunities for the personalized treatment of sporadic PNETs. Of note, patients with PNET who developed liver metastasis revealed distinct mutational profiles compared with cases without metastasis. Furthermore, two potentially clinically actionable genomic alterations were identified in BRCA2 and NF1, suggesting further clinical options of treatment targets in the future.

### Supplementary Material



# SUPPLEMENTAL FILE 1: etm-21-05-09859.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)