# MAIN TEXT

## Analysis of BRCA1/2 variants of unknown significance in the prospective Korean Hereditary Breast Cancer study

### Abstract

Genetic testing for BRCA1 and BRCA2 is crucial in diagnosing hereditary breast and ovarian cancer syndromes and has increased with the development of multigene panel tests. However, results classified as variants of uncertain significance (VUS) present challenges to clinicians in attempting to choose an appropriate management plans. We reviewed a total of 676 breast cancer patients included in the Korean Hereditary Breast Cancer (KOHBRA) study with a VUS on BRCA mutation tests between November 2007 and April 2013. These results were compared to the ClinVar database. We calculated the incidence and odds ratios for these variants using the Korean Reference Genome Database. A total of 58 and 91 distinct VUS in BRCA1 and BRCA2 were identified in the KOHBRA study (comprising 278 and 453 patients, respectively). A total of 27 variants in the KOHBRA study were not registered in the Single Nucleotide Polymorphism database. Among BRCA1 VUSs, 20 were reclassified as benign or likely benign, four were reclassified as pathogenic or likely pathogenic, and eight remained as VUSs according to the ClinVar database. Of the BRCA2 VUSs, 25 were reclassified as benign or likely benign, two were reclassified as pathogenic or likely pathogenic, and 33 remained as VUS according to the ClinVar database. There were 12 variants with conflicting interpretations of pathogenicity for BRCA1 and 18 for BRCA2. Among them, p.Leu1780Pro showed a particularly high odds ratio. Six pathogenic variants and one conflicting variant identified using ClinVar could be reclassified as pathogenic variants in this study. Using updated ClinVar information and calculating odds ratios can be helpful when reclassifying VUSs in BRCA1/2.

### Introduction

Hereditary breast and ovarian cancer syndrome (HBOC) has been shown to be associated with germline mutations in BRCA1 and BRCA21, spurring demands for genetic testing to identify pathogenic variations in these genes2. The identification of a pathogenic BRCA mutation in a patient diagnosed with breast cancer not only affects their treatment and prognosis, but also enables the prevention of other cancers3. Guidelines for the management of pathogenic variants in BRCA1 and BRCA2 recommend consideration of risk-reducing medications or surgeries4,5.

A genetic test for BRCA has four possible results: no mutation detected, pathogenic mutation, benign mutation, or variant of uncertain significance (VUS). A VUS is an alteration in the gene sequence that has an unknown effect on the function of the gene product. This leaves patients and their physicians with uncertainty due to the inability to interpret the result in a clinical context and a lack of specific guidelines regarding genetic counseling or prophylactic management in mutation carriers and their relatives6.

While an overall VUS rate of 7–15% in women who have received BRCA testing has been reported7, the frequency of VUS varies worldwide depending on the testing prevalence and population ancestry7,8. Researchers reported a frequency of VUS of 21% in African-Americans, 5–6% in people of European ancestry in the United States, and 15% in European laboratories9,10. Myriad Genetic Inc. (Salt Lake City, UT, USA) reported that they decreased the proportion of VUSs to 2.1% using accumulated data11. However, these databases are not public or accessible.

In this study, we aimed to explore the prevalence of VUS in the Korean population and to reclassify these variants using the ClinVar database and the Korean Reference Genome Database (KRGDB).

### Results

Among 2,403 breast cancer patients in the Korean Hereditary Breast Cancer (KOHBRA) study, more than a quarter, 676 (28.13%) patients, had mutations that were classified as VUS. Simultaneous mutations of BRCA1 and BRCA2 were observed in 55 (55/676, 8.14%) patients. Of the 676 subjects, 278 had a VUS in BRCA1, and 453 patients had a VUS in BRCA2 (Fig. 1). refSNP (RS) numbers were reviewed for 262 and 440 subjects with BRCA1 and BRCA2 mutations, respectively.Figure 1Schematic diagram of patient selection: BRCA1 and BRCA 2 (n = 55).

Schematic diagram of patient selection: BRCA1 and BRCA 2 (n = 55).

Table 1 shows the reclassification results for VUS according to the ClinVar database. We classified the results into four groups: benign/likely benign, VUS, conflicting interpretations of pathogenicity, and pathogenic/likely pathogenic. Benign/likely benign was the most common reclassification for both BRCA1 and BRCA2 VUSs. Table 2 shows the rank of VUS genes based on the number of patients.Table 1Reclassification of patients diagnosed with variants of uncertain significance based on ClinVar data.BRCA158 mutations (n = 278)Benign/ likely benign20 mutations (n = 193)VUS8 mutations (n = 8)Conflicting in interpretations of pathogenicity12 mutations (n = 55)Pathogenic/ likely pathogenic4 mutations (n = 6)Variants not registered in SNP database14 mutations (n = 16)BRCA291 mutations (n = 453)Benign/ likely benign25 mutations (n = 328)VUS33 mutations (n = 58)Conflicting in interpretations of pathogenicity18 mutations (n = 51)Pathogenic/ likely pathogenic2 mutations (n = 3)Variants not registered in SNP database13 mutations (n = 13)Table 2Top 10 high-frequency mutations based on the number of patients.BRCA1RS_numberClinVarNo. of patientsORCIc.4883T>Crs4986854Benign570.86490.5865–1.2754c.4484 + 14A>Grs80358022Benign410.58230.3845–0.8818c.2566T>Crs80356892Benign400.36590.2491–0.5374c.3113A>Grs16941Benign170.00790.0049–0.0128c.154C>Trs80357084Conflicting interpretations of pathogenicity150.46460.2421–0.8914c.5339T>Crs80357474Conflicting interpretations of pathogenicity128.51811.1192–64.8277c.4729T>Crs80356909Conflicting interpretations of pathogenicity91.28740.4314–3.8417c.3448C>Trs80357272Benign80.63490.2448–1.6464c.3548A>Grs16942Benign80.00460.0025–0.0086c.547 + 14delGrs273902771Conflicting interpretations of pathogenicity6BRCA2RS_numberClinVarNo. of patientsORCIc.10234A>Grs1801426Benign670.64430.4627–0.8972c.8187G>Trs80359065Benign570.79620.5445–1.1643c.9649-19G>Ars11571830Benign400.54620.3609–0.8266c.5785A>Grs79538375Benign330.81180.4922–1.339c.2971A>Grs1799944Benign290.04320.0299–0.0623c.2350A>Grs11571653Benign280.46240.2868–0.7456c.1744A>Crs80358457Benign240.71350.4046–1.2581c.7052C>Grs80358932Benign/likely benign120.61150.2826–1.3231c.6325G>Ars79456940Conflicting interpretations of pathogenicity100.64880.2754–1.5285c.623T>Grs80358865Uncertain significance91.28670.4313–3.8389

Reclassification of patients diagnosed with variants of uncertain significance based on ClinVar data.

Top 10 high-frequency mutations based on the number of patients.

Of the 278 patients with a BRCA1 VUS, 58 VUSs were identified, and 44 had RS numbers. Twenty of these variants, found in 193 patients, were classified as benign/likely benign. The least common VUSs were classified as pathogenic/likely pathogenic and comprised four variants in six patients (Table 1).

Of the 453 patients with BRCA2 VUSs, 91 VUSs were identified, and 78 had RS numbers. The most common VUSs were benign/likely benign, comprising 25 variants in 328 patients. Meanwhile, pathogenic/likely pathogenic variants were the least common and included two variants in three patients.

Calculating minor allele frequency and plotting graphs thereof, we noted that minor allele frequencies for all variants, except for BRCA1 c.4987-92A>G(rs8176233), BRCA1 c.3113A>G(rs16941), BRCA1 c.3548A>G(rs16942), and BRCA2 c.2971A>G(rs1799944), had P values of 0.05 or less: all four of the variants lacking statistical significance were classified as benign (Supplementary Fig. S1).

In this study, six gene mutations previously classified as VUS were reclassified as likely pathogenic based on ClinVar review. The variants were BRCA1 c.5089T>C (p.Cys1697Arg), BRCA1 c.5509T>C (p. Trp1837Arg), BRCA1 c.5516T>C (p.Leu1839Ser), BRCA1 c.81-9C>G, BRCA2 c.8023A>G (p.Ile2675Val), and BRCA2 c.9004G>A (p.Glu3002Lys).

We calculated the odds ratios (ORs) for each variant using the KRGDB, as shown in Fig. 2. The OR of BRCA1 c.5339T>C (p.Leu1780Pro) was significantly high in analysis with the Wald method with 95% confidence intervals (CI). The corresponding P value was 0.0127 before and 0.889 after Bonferroni correction.Figure 2Odds ratios (ORs) using Korean population data from the Korean Reference Genome Database (KRGDB). Variants that could be identified in the KRGDB were classified according to the ClinVar database (vertical axis). Round dots indicate ORs, and the continuous line through each dot indicates a 95% confidence interval. Variants with orange, green, and blue dots indicate conflicting interpretations of pathogenicity, uncertain significance, and benign/likely benign variants according to the ClinVar database, respectively.

Odds ratios (ORs) using Korean population data from the Korean Reference Genome Database (KRGDB). Variants that could be identified in the KRGDB were classified according to the ClinVar database (vertical axis). Round dots indicate ORs, and the continuous line through each dot indicates a 95% confidence interval. Variants with orange, green, and blue dots indicate conflicting interpretations of pathogenicity, uncertain significance, and benign/likely benign variants according to the ClinVar database, respectively.

### Baseline characteristics

Among 2,403 breast cancer patients in the Korean Hereditary Breast Cancer (KOHBRA) study, more than a quarter, 676 (28.13%) patients, had mutations that were classified as VUS. Simultaneous mutations of BRCA1 and BRCA2 were observed in 55 (55/676, 8.14%) patients. Of the 676 subjects, 278 had a VUS in BRCA1, and 453 patients had a VUS in BRCA2 (Fig. 1). refSNP (RS) numbers were reviewed for 262 and 440 subjects with BRCA1 and BRCA2 mutations, respectively.Figure 1Schematic diagram of patient selection: BRCA1 and BRCA 2 (n = 55).

Schematic diagram of patient selection: BRCA1 and BRCA 2 (n = 55).

### Reclassification using public databases

Table 1 shows the reclassification results for VUS according to the ClinVar database. We classified the results into four groups: benign/likely benign, VUS, conflicting interpretations of pathogenicity, and pathogenic/likely pathogenic. Benign/likely benign was the most common reclassification for both BRCA1 and BRCA2 VUSs. Table 2 shows the rank of VUS genes based on the number of patients.Table 1Reclassification of patients diagnosed with variants of uncertain significance based on ClinVar data.BRCA158 mutations (n = 278)Benign/ likely benign20 mutations (n = 193)VUS8 mutations (n = 8)Conflicting in interpretations of pathogenicity12 mutations (n = 55)Pathogenic/ likely pathogenic4 mutations (n = 6)Variants not registered in SNP database14 mutations (n = 16)BRCA291 mutations (n = 453)Benign/ likely benign25 mutations (n = 328)VUS33 mutations (n = 58)Conflicting in interpretations of pathogenicity18 mutations (n = 51)Pathogenic/ likely pathogenic2 mutations (n = 3)Variants not registered in SNP database13 mutations (n = 13)Table 2Top 10 high-frequency mutations based on the number of patients.BRCA1RS_numberClinVarNo. of patientsORCIc.4883T>Crs4986854Benign570.86490.5865–1.2754c.4484 + 14A>Grs80358022Benign410.58230.3845–0.8818c.2566T>Crs80356892Benign400.36590.2491–0.5374c.3113A>Grs16941Benign170.00790.0049–0.0128c.154C>Trs80357084Conflicting interpretations of pathogenicity150.46460.2421–0.8914c.5339T>Crs80357474Conflicting interpretations of pathogenicity128.51811.1192–64.8277c.4729T>Crs80356909Conflicting interpretations of pathogenicity91.28740.4314–3.8417c.3448C>Trs80357272Benign80.63490.2448–1.6464c.3548A>Grs16942Benign80.00460.0025–0.0086c.547 + 14delGrs273902771Conflicting interpretations of pathogenicity6BRCA2RS_numberClinVarNo. of patientsORCIc.10234A>Grs1801426Benign670.64430.4627–0.8972c.8187G>Trs80359065Benign570.79620.5445–1.1643c.9649-19G>Ars11571830Benign400.54620.3609–0.8266c.5785A>Grs79538375Benign330.81180.4922–1.339c.2971A>Grs1799944Benign290.04320.0299–0.0623c.2350A>Grs11571653Benign280.46240.2868–0.7456c.1744A>Crs80358457Benign240.71350.4046–1.2581c.7052C>Grs80358932Benign/likely benign120.61150.2826–1.3231c.6325G>Ars79456940Conflicting interpretations of pathogenicity100.64880.2754–1.5285c.623T>Grs80358865Uncertain significance91.28670.4313–3.8389

Reclassification of patients diagnosed with variants of uncertain significance based on ClinVar data.

Top 10 high-frequency mutations based on the number of patients.

Of the 278 patients with a BRCA1 VUS, 58 VUSs were identified, and 44 had RS numbers. Twenty of these variants, found in 193 patients, were classified as benign/likely benign. The least common VUSs were classified as pathogenic/likely pathogenic and comprised four variants in six patients (Table 1).

Of the 453 patients with BRCA2 VUSs, 91 VUSs were identified, and 78 had RS numbers. The most common VUSs were benign/likely benign, comprising 25 variants in 328 patients. Meanwhile, pathogenic/likely pathogenic variants were the least common and included two variants in three patients.

Calculating minor allele frequency and plotting graphs thereof, we noted that minor allele frequencies for all variants, except for BRCA1 c.4987-92A>G(rs8176233), BRCA1 c.3113A>G(rs16941), BRCA1 c.3548A>G(rs16942), and BRCA2 c.2971A>G(rs1799944), had P values of 0.05 or less: all four of the variants lacking statistical significance were classified as benign (Supplementary Fig. S1).

In this study, six gene mutations previously classified as VUS were reclassified as likely pathogenic based on ClinVar review. The variants were BRCA1 c.5089T>C (p.Cys1697Arg), BRCA1 c.5509T>C (p. Trp1837Arg), BRCA1 c.5516T>C (p.Leu1839Ser), BRCA1 c.81-9C>G, BRCA2 c.8023A>G (p.Ile2675Val), and BRCA2 c.9004G>A (p.Glu3002Lys).

### Odds ratio estimation

We calculated the odds ratios (ORs) for each variant using the KRGDB, as shown in Fig. 2. The OR of BRCA1 c.5339T>C (p.Leu1780Pro) was significantly high in analysis with the Wald method with 95% confidence intervals (CI). The corresponding P value was 0.0127 before and 0.889 after Bonferroni correction.Figure 2Odds ratios (ORs) using Korean population data from the Korean Reference Genome Database (KRGDB). Variants that could be identified in the KRGDB were classified according to the ClinVar database (vertical axis). Round dots indicate ORs, and the continuous line through each dot indicates a 95% confidence interval. Variants with orange, green, and blue dots indicate conflicting interpretations of pathogenicity, uncertain significance, and benign/likely benign variants according to the ClinVar database, respectively.

Odds ratios (ORs) using Korean population data from the Korean Reference Genome Database (KRGDB). Variants that could be identified in the KRGDB were classified according to the ClinVar database (vertical axis). Round dots indicate ORs, and the continuous line through each dot indicates a 95% confidence interval. Variants with orange, green, and blue dots indicate conflicting interpretations of pathogenicity, uncertain significance, and benign/likely benign variants according to the ClinVar database, respectively.

### Discussion

We re-evaluated genetic results in patients with VUSs in BRCA1 and BRCA2 using the ClinVar database. Since genetic characteristics can vary by ethnicity7,9,10, this study aimed to identify the prevalence of VUS in the Korean population and to re-classify the results of initial genetic tests. Although initial genetic testing results in the KOHBRA study revealed a VUS in 676 patients (278 patients for BRCA1 and 453 patients for BRCA2) out a total of 2,403 breast cancer patients (28.13%, 676/2,403), re-evaluation revealed a lower frequency of VUSs (8.03%, 193/2,403). About a third of the variants that were originally classified as VUS in the KOHBRA study were reclassified as benign/likely benign or pathogenic/likely pathogenic, accounting for two-thirds of all VUS patients (71.45%, 483/676). This result suggests that a re-classification approach using the ClinVar database can reduce the frequency of VUSs in the Korean population.

About two-thirds of the VUSs in the KOHBRA study were reclassified as benign or likely benign [193/278 (69.42%) of BRCA1 patients, 328/453 (72.41%) of BRCA2 patients, and 471/676 (69.67%) of all patients]. A third of the mutation types classified as VUS in the KOHBRA study were downgraded to benign or likely benign (20/58 mutations in BRCA1 and 25/91 in BRCA2). These results were consistent with previous studies1,12,13: So et al. reported that 30/75 (40%) of VUS patients were reclassified as benign or likely benign14. In our study, six patients with BRCA1 VUS (four mutations) and three patients with BRCA2 VUS (two mutations) were reclassified as pathogenic or likely pathogenic.

In this study, six mutations in nine patients were reclassified from VUSs to pathogenic or likely pathogenic variants. BRCA1 c.5089T>C (p.Cys1697Arg), BRCA1 c.5509T>C (p. Trp1837Arg), BRCA1 c.5516T>C (p.Leu1839Ser), and BRCA1 c.81-9C>G were interpreted as likely pathogenic among variants in BRCA1. BRCA2 c.8023A>G (p.Ile2675Val) and BRCA2 c.9004G>A (p.Glu3002Lys) were interpreted as pathogenic/likely pathogenic among variants in BRCA2 based on the ClinVar database. Since all six mutations are supported by sufficient evidence in functional studies (PS3)15–17, have not been reported in a genomic database for the general Korean population (PM2), are classified as pathogenic in ClinVar (PP5), and are deleterious mutations (PP3) according to an in silico study18, they should be reclassified as likely pathogenic or pathogenic mutations. For patients with these mutations, additional genetic counseling and proper management, such as familial genetic testing, risk-reducing medications, or risk-reducing surgery, are needed for the prevention of cancer.

Interestingly, we identified several cases with conflicting interpretations of pathogenicity, for which reviewed the current status of available evidence in the literature (Table 3). In such instances, calculating ORs can help to reclassify them. The KRGDB, which is a large-scale, single-race database collected from 1,722 Koreans, is built for precision medicine research. We analyzed the KRGDB data using the Wald method to obtain ORs and 95% CIs for all VUS variants (Fig. 2). Several mutations (7 BRCA1 and 9 BRCA2 mutations) were evaluated. Most of them showed no significance. However, BRCA1 c.5339T>C (p.Leu1780Pro) showed possible pathogenicity, while BRCA1 c.154C>T (p.Leu52Phe) and BRCA2 c.964A>C (p.Lys322Gln) were potentially benign. However, with the most conservative multiple test correction, Bonferroni correction, the statistically significance of c.5339T>C disappeared. This non-significant level should be carefully interpreted, because several previous studies indicated that this c.5339.T>C variant is pathogenic or likely pathogenic according to clinicopathologic features and American College of Medical Genetics (ACMG) guidelines13,19–21. Therefore, even though the P value after multiple correction indicated a lack of statistical significance, c.5339T>C should be interpreted as a pathogenic or likely pathogenic variant in light of other functional evaluations, including co-segregation, in vitro, and in silico analyses. In verifying additional pathogenic mutation candidates, we merely referred to OR and CI values, but do not insist that OR values alone should be applied. With further accumulation of data in the future, we may be able to recalculate these ORs of P values. Meanwhile, variants BRCA 1 c.5014_5016delCAC, BRCA 1 c.5332G>A, BRCA2 c.182T>C, BRCA 2 c.1909 + 22delT, BRCA 2 c.8486A>G, and BRCA2 c.8954-5A>G have not been reported in the general population and should remain classified as VUS, since there are no reports on their deleterious function.Table 3Conflicting interpretations of pathogenicity.BRCA1RS_number(Likely) PathogenicUncertain significance(Likely) BenignNo. of patientsORCIc.1357G>C,p.Glu453Glnrs768054411311c.154C>Tp.Leu52Phers8035708453150.46460.2421–0.8914c.2481A>Cp.Glu827Asprs3975089702110.70820.0446–11.2359c.2726A>Tp.Asn909Ilers80357127812c.4729T>Cp.Ser1577Prors803569091791.28740.4314–3.8417c.5014_5016delCACp.His1673delrs80358343341c.5068A>Cp.Lys1690Glnrs3975072398110.23580.0246–2.2601c.5332G>Ap.Asp1778Asnrs80357112231c.5339T>Cp.Leu1780Prors8035747441128.51811.1192–64.8277c.547 + 14delGc.547 + 14delGrs273902771246c.671-8A>Gc.671-8A>Grs803581441450.88520.2385–3.2849c.811G>Ap.Val271Metrs8035724411120.35370.065–1.9254BRCA2RS_number(Likely) PathogenicUncertain significance(Likely) BenignNo. of patientsORCIc.10150C>Tp.Arg3384Terrs3975075681520.47170.0791–2.8122c.1568A>Gp.His523Argrs803584432920.28550.0554–1.4714c.1817C>Tp.Pro606Leurs803584695110.71450.0448–11.4046c.182T>Cp.Leu61Prors1555280374211c.1909 + 22delTrs276174816171c.317-10A>Grs81002824111c.3256A>Gp.Ile1086Valrs80358571523c.4599A>Cp.Lys1533Asnrs80358694351c.5554G>Ap.Val1852Ilers803587774210.70780.0446–11.225c.5969A>Cp.Asp1990Alars1486185426171.23910.3642–4.216c.6101G>Ap.Arg2034Hisrs80358849211c.6131G>Tp.Gly2044Valrs561915791840.40780.1194–1.3933c.6325G>Ap.Val2109Ilers79456940210100.64880.2754–1.5285c.7522G>Ap.Gly2508Serrs803589787690.53530.2254–1.2713c.8092G>Ap.Ala2698Thrrs80359052641c.8486A>Gp.Gln2829Argrs80359100211c.8954-5A>Grs886040949511c.964A>Cp.Lys322Glnrs115716403940.23750.0766–0.7367

Conflicting interpretations of pathogenicity.

c.1357G>C,

p.Glu453Gln

c.154C>T

p.Leu52Phe

c.2481A>C

p.Glu827Asp

c.2726A>T

p.Asn909Ile

c.4729T>C

p.Ser1577Pro

c.5014_5016delCAC

p.His1673del

c.5068A>C

p.Lys1690Gln

c.5332G>A

p.Asp1778Asn

c.5339T>C

p.Leu1780Pro

c.547 + 14delG

c.547 + 14delG

c.671-8A>G

c.671-8A>G

c.811G>A

p.Val271Met

c.10150C>T

p.Arg3384Ter

c.1568A>G

p.His523Arg

c.1817C>T

p.Pro606Leu

c.182T>C

p.Leu61Pro

c.3256A>G

p.Ile1086Val

c.4599A>C

p.Lys1533Asn

c.5554G>A

p.Val1852Ile

c.5969A>C

p.Asp1990Ala

c.6101G>A

p.Arg2034His

c.6131G>T

p.Gly2044Val

c.6325G>A

p.Val2109Ile

c.7522G>A

p.Gly2508Ser

c.8092G>A

p.Ala2698Thr

c.8486A>G

p.Gln2829Arg

c.964A>C

p.Lys322Gln

Interpretations of VUSs are complex. Functional studies for reclassifying VUS could be a promising approach. Traditionally, VUS interpretation has depended on inductive conclusions based on information of individual patients22. However, many potential variants in BRCA are present at low variant allele frequencies, with phenotypes that are incompletely penetrant. Findlay et al. reported the application of saturation genome editing (SGE) to measure the functional outcomes of all possible single nucleotide variants (SNVs) in key areas of BRCA115. Functional effects were almost concordant with the established assessments of pathogenicity. Function scores using SGE could help with interpreting the significance of VUSs by providing functional classification and assessment of ambiguous or newly-discovered variants.

The four BRCA1 mutations (c.5089T>C (p.Cys1697Arg), c.5509T>C (p.Trp1837Arg), c.5516T>C (p.Leu1839Ser), and c.81-9C>G) identified as likely pathogenic using the ClinVar database were identified as non-functional in Findlay’s study15. The c.5339T>C (p.Leu1780Pro) variant identified as having conflicting interpretations of pathogenicity in the ClinVar database was also identified as “non-functional” in the functional study results, suggesting this variant as pathogenic15. On the other hand, other variants with conflicting interpretations of pathogenicity in the ClinVar database (including c.154C>T (p.Leu52Phe), c.5068A>C (p.Lys1690Gln), and c.5332G>A (p.Asp1778Asn)) have been categorized as functional or intermediate15. The other BRCA1 variants with conflicting interpretations of pathogenicity in the ClinVar database could not be evaluated according to Findlay’s study, as it analyzed only RING and BRCT domains as targets15.

Cosegregation analysis may also be helpful in re-evaluating VUSs. Zuntini et al. performed cosegregation analysis for 13 VUSs in 11 kindreds to improve VUS evaluation, and two variants were found to have additional supporting evidence of pathogenicity23. Among the variants that were reclassified as pathogenic variants in our study, BRCA1 c.5509T>C (p. Trp1837Arg) was discussed in Zuntini's study as well; however, due to the distinct nature of our data, cosegregation analysis could not be conducted. Cosegregation analysis may help improve understanding of VUSs and provide genetic counseling for specific families, sufficient for pedigree analysis.

When ORs were calculated using the KRGDB for all KOHBRA data, the OR of BRCA1 c.5339T>C (p.Leu1780Pro) was found to be significantly elevated (Fig. 2). This variant was also identified as non-functional in Findlay’s study. In addition, several studies have suggested that this mutation is pathogenic based on other evidence, including a strong family history of breast and ovarian cancer, absence in general population data, impaired function demonstrated by in silico studies, and triple negativity in clinicopathologic features13,14,21. Previous studies have used a similar approach to reclassify some variants24,25.

One limitation of the study is that we reviewed the VUSs by assigning them to a database based on a mostly Caucasian population. Researchers contributing to the Single Nucleotide Polymorphism (SNP) database or ClinVar tend to be concentrated in Western countries, leading to a lack of registration of major variants in Asian populations or a lack of interpretation of variants such as L1780P. Nevertheless, this study was meaningful in that it confirmed VUS status in Koreans using a prospective study and lays the groundwork for broadening our understanding of VUSs and conducting further research. Another limitation of the study is that there is missing information in the KOHBRA data, which are necessary to reclassify VUSs (e.g., 27 variants were not submitted to the SNP database). However, the lacking data comprised only 4.29% of the cohort and would unlikely weaken the power of the current study.

Taken together, most of the mutations that were classified as VUS in the KOHBRA study were reclassified as benign. Four VUSs in BRCA1 and two in BRCA2 VUS were reclassified as pathogenic or likely pathogenic. When ORs were calculated using the KRGDB for all KOHBRA data, the OR of BRCA1 c.5339T>C (p.Leu1780Pro) was significantly high, although ClinVar considered BRCA1 c.5339T>C to have conflicting interpretations of pathogenicity. These seven mutations could be reclassified as likely pathogenic or pathogenic mutations, according to ACMG guidelines. Since the mutations classified as benign in ClinVar have a high normal frequency, it is desirable to judge them as benign.

However, some VUSs remained as having conflicting interpretations of pathogenicity, rather than being re-assessed as benign or pathogenic. Their characteristics will likely be more discernable with the accumulation of more information. When a VUS is reclassified as pathogenic/likely pathogenic, appropriate management, including risk-reducing medication and surgery, should be discussed with patients and their families. In addition to collecting individual data, functional studies using genetic techniques, such as SGE, could help contribute to the functional classification and assessment of VUSs.

### Methods

The study population was obtained from the Korean Hereditary Breast Cancer (KOHBRA) study26. The KOHBRA study is a multicenter prospective cohort study designed to investigate the prevalence and causes of hereditary breast cancer in the Korean population. Through the study, 3,015 subjects were recruited between May 2007 and December 2013 from 36 institutions27. The eligibility criteria were as follows: (1) breast cancer patients with a family history of breast or ovarian cancer; (2) breast cancer patients without a family history of breast or ovarian cancer (non-familial) who were aged 40 years or younger at diagnosis and were diagnosed with bilateral breast cancer or another primary malignancy; (3) male breast cancer patients; and (4) family members of BRCA 1/2 mutation carriers. After excluding several subjects, a total of 2953 subjects (1228 familial breast cancer patients, 1175 non-familial breast cancer patients, and 550 family members of affected carriers) were evaluated. We identified 676 breast cancer patients with VUS on BRCA mutation tests.

These results were reclassified using the ClinVar database (http://www.ncbi.nlm.nih.gov/clinvar/) based on refSNP (RS) numbers. Odds ratios (ORs) for each variant were calculated using Korean population data from the KRGDB, which was established by conducting whole genome sequencing of 1,722 Koreans28. Variants that were not registered in the Single Nucleotide Polymorphism (SNP) database are shown in Table 1. In this study, variations without RS numbers were also included in the denominator when checking the overall frequency of VUS.

BRCA 1/2 genetic testing was performed using genomic DNA from the peripheral blood. Of the 2403 patients, clinicians used fluorescence confirmation sensitive capillary (gel) electrophoresis on 1183 patients, direct sequencing on 1101 patients, and denaturing high-performance liquid chromatography on 119 patients27. Each testing method was selected according to the laboratory linked to the institution. All BRCA 1/2 mutations were classified as pathogenic, VUS, or polymorphic.

Chi-square or Fisher’s exact tests were used for categorical variables. ORs and 95% confidence intervals were obtained using the Wald method. We retrieved allele frequencies from 1722 Koreans (KRGDB). For each variant, an OR was calculated based on its occurrence in 2403 patient cases and in the KRGDB. All analyses were performed using SPSS version 23 (SPSS; Chicago, IL, USA), and statistical significance was defined as P < 0.05.

All sequence variations are described according to the HUGO-approved systematic nomenclature (http://www.hgvs.org/mutnomen/) using GenBank reference sequences (NM_007294.2 for BRCA 1 and NM_000059.3 for BRCA 2). The breast cancer information core nomenclature is also presented for convenience.

This study was approved by the Institutional Review Board of Severance Hospital, Yonsei University. (IRB# 4-2017-0255).

### Subjects

The study population was obtained from the Korean Hereditary Breast Cancer (KOHBRA) study26. The KOHBRA study is a multicenter prospective cohort study designed to investigate the prevalence and causes of hereditary breast cancer in the Korean population. Through the study, 3,015 subjects were recruited between May 2007 and December 2013 from 36 institutions27. The eligibility criteria were as follows: (1) breast cancer patients with a family history of breast or ovarian cancer; (2) breast cancer patients without a family history of breast or ovarian cancer (non-familial) who were aged 40 years or younger at diagnosis and were diagnosed with bilateral breast cancer or another primary malignancy; (3) male breast cancer patients; and (4) family members of BRCA 1/2 mutation carriers. After excluding several subjects, a total of 2953 subjects (1228 familial breast cancer patients, 1175 non-familial breast cancer patients, and 550 family members of affected carriers) were evaluated. We identified 676 breast cancer patients with VUS on BRCA mutation tests.

These results were reclassified using the ClinVar database (http://www.ncbi.nlm.nih.gov/clinvar/) based on refSNP (RS) numbers. Odds ratios (ORs) for each variant were calculated using Korean population data from the KRGDB, which was established by conducting whole genome sequencing of 1,722 Koreans28. Variants that were not registered in the Single Nucleotide Polymorphism (SNP) database are shown in Table 1. In this study, variations without RS numbers were also included in the denominator when checking the overall frequency of VUS.

### BRCA 1/2 mutation analysis

BRCA 1/2 genetic testing was performed using genomic DNA from the peripheral blood. Of the 2403 patients, clinicians used fluorescence confirmation sensitive capillary (gel) electrophoresis on 1183 patients, direct sequencing on 1101 patients, and denaturing high-performance liquid chromatography on 119 patients27. Each testing method was selected according to the laboratory linked to the institution. All BRCA 1/2 mutations were classified as pathogenic, VUS, or polymorphic.

### Statistical analysis

Chi-square or Fisher’s exact tests were used for categorical variables. ORs and 95% confidence intervals were obtained using the Wald method. We retrieved allele frequencies from 1722 Koreans (KRGDB). For each variant, an OR was calculated based on its occurrence in 2403 patient cases and in the KRGDB. All analyses were performed using SPSS version 23 (SPSS; Chicago, IL, USA), and statistical significance was defined as P < 0.05.

### Mutation nomenclature

All sequence variations are described according to the HUGO-approved systematic nomenclature (http://www.hgvs.org/mutnomen/) using GenBank reference sequences (NM_007294.2 for BRCA 1 and NM_000059.3 for BRCA 2). The breast cancer information core nomenclature is also presented for convenience.

This study was approved by the Institutional Review Board of Severance Hospital, Yonsei University. (IRB# 4-2017-0255).

### Supplementary information

Supplementary information

Supplementary information

Supplementary information

Supplementary information



# SUPPLEMENTAL FILE 1: 41598_2021_Article_87792.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)