# MAIN TEXT

## Co-occurrence of thyroid and breast cancer is associated with an increased oncogenic SNP burden

### Abstract

BackgroundEpidemiological evidence suggests that synchronous or metachronous presentation of breast and thyroid cancers exceeds that predicted by chance alone. The following potential explanations have been hypothesized: common environmental or hormonal factors, oncogenic effect of the treatment for the first cancer, closer follow-up of cancer survivors, shared underlying genetic risk factors. While some cases were found to be related to monogenic disorders with autosomal inheritance, the genetic background of most cases of co-occurring breast and thyroid cancer is thought to be polygenic.MethodsIn this retrospective case-control study we compared the genetic profile of patients with a history of breast cancer (n = 15) to patients with co-occurring breast and thyroid cancer (n = 19) using next generation sequencing of 112 hereditary cancer risk genes. Identified variants were categorized based on their known association with breast cancer and oncogenesis in general.ResultsNo difference between patients with breast and double cancers was observed in clinical and pathological characteristics or the number of neutral SNPs. The unweighted and weighted number of SNPs with an established or potential association with breast cancer was significantly lower in the group with breast cancer only (mean difference − 0.58, BCa 95% CI [− 1.09, − 0.06], p = 0.029, and mean difference − 0.36, BCa 95% CI [− 0.70, − 0.02], p = 0.039, respectively). The difference was also significant when we compared the number of SNPs with potential or known association with any malignancy (mean difference − 1.19, BCa 95% CI [− 2.27, − 0.11], p = 0.032 for unweighted, and mean difference − 0.73, BCa 95% CI [− 1.32, − 0.14], p = 0.017 for weighted scores).ConclusionOur findings are compatible with the hypothesis of genetic predisposition in the co-occurrence of breast and thyroid cancer. Further exploration of the underlying genetic mechanisms may help in the identification of patients with an elevated risk for a second cancer at the diagnosis of the first cancer.

### Background

An increased co-occurrence of breast cancer (BC) and differentiated thyroid cancer (TC) compared to chance alone has been found repeatedly in recent decades [1, 2]. Numerous studies have shown a bidirectional association between these cancers: a more pronounced risk of thyroid cancer in breast cancer survivors and a less pronounced increase in the incidence of breast cancer after thyroid cancer [2–4].

A number of hypotheses have been suggested to explain these findings [3, 4]. The one, most frequently cited, describes a common (yet unknown) hormonal etiologic factor. This hypothesis is supported by the observation of a strong female predominance of both tumor types. Estrogen has been implicated in the pathogenesis of both breast and, to a lesser degree, thyroid cancer [5]. Furthermore, thyroid dysfunction has also been tentatively linked to carcinogenesis [6–8]. It is also possible that the hormonal milieu of pregnancy (hCG, increased TRH and prolactin) would increase the thyroid and the estrogen-like effects of thyroid hormones, hence the risk of both cancer types [9]. Endocrine disrupting chemicals affecting estrogen receptors have also been theoretically implicated [10].

Another set of hypotheses emphasize carcinogenic effects of previous cancer treatment. Radio- and hormone therapy for breast cancer and I131 treatment for thyroid tumors have both been implicated in carcinogenesis [11–13]. It is also possible that the increased surveillance of cancer survivors could lead to earlier recognition or even overdiagnosis of a second primary malignancy.

A shared genetic background of these cancers is also a potential explanation for the observed association. Cowden and Cowden-like syndromes are characterized by the presence of hamartomas and an increased risk of, among others, both thyroid and breast cancer. These conditions are mostly monogenic disorders with an autosomal dominant inheritance. Mutations in the PTEN, SDHB, SDHD, MTHFR and PARP4 genes, and hypermethylation of KLLN have all been implicated [14–16]. Much less is known about the genetic background of sporadic cases of metachronous or synchronous breast and thyroid cancer. However, an increased familial risk of other primary malignancies of affected patients points towards underlying germline mutations [3, 17].

In the present paper, we investigated the potential for a shared genetic background of thyroid and breast cancers. The recognition and elucidation of such genetic risk factors could aid in the identification, follow-up, and potential preventive treatment of high-risk individuals with a first primary malignancy.

### Methods

We analyzed the genetic polymorphisms in 112 hereditary cancer risk genes in a case-control study of patients with a history of either breast or both breast and thyroid cancer.

Patients were selected from a pool of n = 274 thyroid cancer patients who had received I131 treatment at our institution between January 2014 and October 2018. Twenty-one of these individuals received treatment for breast cancer before or after the diagnosis of a thyroid cancer. Two patients could not have been reached for consent, resulting in a final sample of 19. Given that the risk of thyroid cancer development after breast cancer is larger than vice versa, the control group consisted of 15 subjects who received treatment for breast cancer at our institution at least 12 years before the present study. Patients with a personal or a family history of thyroid cancer were excluded from this group. These criteria were specified to minimize contamination of the control group with patients with an increased genetic risk for thyroid cancer development. No other inclusion or exclusion criteria were set.

We collected the following clinical characteristics from hospital and outpatient records: age at cancer diagnosis, tumor size, histology, staging and grading, presence or absence of lymph node and distant metastases, details of surgical-, radiation- chemotherapeutic-, hormonal and I131 treatment. Cancer diagnosis for all cases was based on histological data using the WHO classification of breast and endocrine tumors (4th and 3/4th edition respectively). Staging was based on the TNM system (7th edition). Informed consent was obtained from all subjects before any study related procedures were performed. The study was approved by Semmelweis University Regional and Institutional Committee of Science and Research Ethics.

We have aimed to cover all the well-established hereditary cancer risk loci. The final list of the 112 investigated genes was compiled based on literature data [18–20], and is shown in Table 1.
Table 1List of cancer risk genes assessed. BC associated genes are underlinedACDCDKN1BFANCAJAGN1NFIXPTENTERCAIPCDKN2AFANCBKIF1BNHP2RAD50TERTAPCCHEK2FANCCKITNOP10RAD51TINF2ATMDDB2FANCD2MAXNSD1RAD51CTMEM127ATRDICER1FANCEMDH2NTHL1RAD51DTP53AXIN2DIS3L2FANCFMEN1PALB2RB1TSC1BAP1DKC1FANCGMETPARNRECQLTSC2BARD1ELANEFANCIMITFPDGFRARECQL4UBE2TBLMEPAS1FANCLMLH1PMS1RETVHLBMPR1AEPCAMFANCMMNX1PMS2RTEL1VPS45BRCA1ERCC1FHMSH2POLD1SCG5WASBRCA2ERCC2FLCNMSH6POLESLX4WRNBRIP1ERCC3G6PC3MSR1POLHSMAD4WT1BUB1ERCC4GFI1MUTYHPOT1SMARCA4XPACDH1ERCC5GREM1NBNPRKAR1ASTK11XPCCDK4ERCC6HOXB13NF2PTCH1SUFUXRCC2

List of cancer risk genes assessed. BC associated genes are underlined

Exome amplicon library was prepared using the Ion AmpliSeq Library Kit Plus combined with the Ion AmpliSeq Exome RDY kit (ThermoFisher, MA, USA). Briefly, 100 ng of genomic DNA was added to dried down, ultra-high multiplexed primer pairs (12 pools) in a 96-well plate and amplified with the following PCR conditions: at 99 °C for 2 min; at 99 °C for 15 s and at 60 °C for 16 min (10 cycles) and holding at 10 °C. Primers were partially digested using a FuPa reagent, and then sequencing adapters and barcodes were ligated to the amplicons. The library was purified using the Agencourt AMPure XP Reagent (Beckmann Coulter, CA, USA). The concentration of the final library was determined by Ion Library TaqMan Quantitation Kit (ThermoFisher, MA, USA) on an ABI 7500 qPCR instrument with absolute quantification method.

Template preparation was performed with Ion 540 OT2 Kit (ThermoFisher, MA, USA) on semi-automated Ion OneTouch 2 instrument using emPCR method. After breaking the emulsion, the non-templated beads were removed from the solution during the semi-automated enrichment process on Ion OneTouch ES (ThermoFisher, MA, USA) machine. Later adding the sequencing primer and polymerase, the fully prepared Ion Sphere Particles (ISPs) were loaded into an Ion 540 chip, and the sequencing runs were performed using the Ion S5 Sequencing kit (ThermoFisher, MA, USA) with 500 flows.

Sequence data from the Ion Torrent run were analyzed using the platform-specific pipeline software Torrent Suite v5.10 for base calling, trim adapter and primer sequences, filtering out poor quality reads, and de-multiplex the reads according to the barcode sequences. Briefly, TMAP algorithm was used to align the reads to the hg19 human reference genome, and then, the variant caller plug-in was executed to search for germline variants in the targeted regions. Integrative genomics viewer (IGV) was used for visualization of the mapped reads. Variants were annotated using the Ion Reporter (ThermoFisher, MA, USA) software. Variant interpretation was restricted to the selected genes with known connection to oncology diseases. Variants were stratified by gene association with breast cancer and by variant association with cancer (no/potential/known) based on available data from the ClinVar database at the time of writing [21]. Synonymous mutations were not excluded from the analysis.

Descriptive data are given as means ± standard deviation (SD) for continuous data, and frequencies (percentages) for categorical variables. Between-group differences in clinical parameters were assessed using 2-sample t tests for continuous variables, and χ2 tests or Fisher exact tests for categorical variables.

When comparing the number of different variants across groups, the assumptions of normality and homogeneity of variances were assessed using the Shapiro–Wilk test and Levene’s test respectively. Given that in some cases these assumptions were not met, Welch’s t-test was used to compare the number of variants with known, potential or no association with breast cancer and with any cancer. We also used the bootstrap procedure to get robust confidence intervals and p-values. To increase statistical power, we lumped together variants with potential and known associations using both weighted and unweighted scores. To generate the weighted scores, we gave double weight for variants with known association compared to those variants with potential association. To correct for multiple testing we applied the Benjamini-Hochberg procedure targeting a false discovery rate < 15%. A corrected two-tailed p-value less than 0.05 was considered statistically significant. Effect sizes were computed as adjusted standardized mean differences (Hedge’s g). Analyses were conducted using IBM SPSS Statistics for Windows, Version 23.0 (IBM Corp. Released 2015 Armonk, New York, U.S.A.).

### Patients and setting

We analyzed the genetic polymorphisms in 112 hereditary cancer risk genes in a case-control study of patients with a history of either breast or both breast and thyroid cancer.

Patients were selected from a pool of n = 274 thyroid cancer patients who had received I131 treatment at our institution between January 2014 and October 2018. Twenty-one of these individuals received treatment for breast cancer before or after the diagnosis of a thyroid cancer. Two patients could not have been reached for consent, resulting in a final sample of 19. Given that the risk of thyroid cancer development after breast cancer is larger than vice versa, the control group consisted of 15 subjects who received treatment for breast cancer at our institution at least 12 years before the present study. Patients with a personal or a family history of thyroid cancer were excluded from this group. These criteria were specified to minimize contamination of the control group with patients with an increased genetic risk for thyroid cancer development. No other inclusion or exclusion criteria were set.

We collected the following clinical characteristics from hospital and outpatient records: age at cancer diagnosis, tumor size, histology, staging and grading, presence or absence of lymph node and distant metastases, details of surgical-, radiation- chemotherapeutic-, hormonal and I131 treatment. Cancer diagnosis for all cases was based on histological data using the WHO classification of breast and endocrine tumors (4th and 3/4th edition respectively). Staging was based on the TNM system (7th edition). Informed consent was obtained from all subjects before any study related procedures were performed. The study was approved by Semmelweis University Regional and Institutional Committee of Science and Research Ethics.

### Genetic analysis

We have aimed to cover all the well-established hereditary cancer risk loci. The final list of the 112 investigated genes was compiled based on literature data [18–20], and is shown in Table 1.
Table 1List of cancer risk genes assessed. BC associated genes are underlinedACDCDKN1BFANCAJAGN1NFIXPTENTERCAIPCDKN2AFANCBKIF1BNHP2RAD50TERTAPCCHEK2FANCCKITNOP10RAD51TINF2ATMDDB2FANCD2MAXNSD1RAD51CTMEM127ATRDICER1FANCEMDH2NTHL1RAD51DTP53AXIN2DIS3L2FANCFMEN1PALB2RB1TSC1BAP1DKC1FANCGMETPARNRECQLTSC2BARD1ELANEFANCIMITFPDGFRARECQL4UBE2TBLMEPAS1FANCLMLH1PMS1RETVHLBMPR1AEPCAMFANCMMNX1PMS2RTEL1VPS45BRCA1ERCC1FHMSH2POLD1SCG5WASBRCA2ERCC2FLCNMSH6POLESLX4WRNBRIP1ERCC3G6PC3MSR1POLHSMAD4WT1BUB1ERCC4GFI1MUTYHPOT1SMARCA4XPACDH1ERCC5GREM1NBNPRKAR1ASTK11XPCCDK4ERCC6HOXB13NF2PTCH1SUFUXRCC2

List of cancer risk genes assessed. BC associated genes are underlined

Exome amplicon library was prepared using the Ion AmpliSeq Library Kit Plus combined with the Ion AmpliSeq Exome RDY kit (ThermoFisher, MA, USA). Briefly, 100 ng of genomic DNA was added to dried down, ultra-high multiplexed primer pairs (12 pools) in a 96-well plate and amplified with the following PCR conditions: at 99 °C for 2 min; at 99 °C for 15 s and at 60 °C for 16 min (10 cycles) and holding at 10 °C. Primers were partially digested using a FuPa reagent, and then sequencing adapters and barcodes were ligated to the amplicons. The library was purified using the Agencourt AMPure XP Reagent (Beckmann Coulter, CA, USA). The concentration of the final library was determined by Ion Library TaqMan Quantitation Kit (ThermoFisher, MA, USA) on an ABI 7500 qPCR instrument with absolute quantification method.

Template preparation was performed with Ion 540 OT2 Kit (ThermoFisher, MA, USA) on semi-automated Ion OneTouch 2 instrument using emPCR method. After breaking the emulsion, the non-templated beads were removed from the solution during the semi-automated enrichment process on Ion OneTouch ES (ThermoFisher, MA, USA) machine. Later adding the sequencing primer and polymerase, the fully prepared Ion Sphere Particles (ISPs) were loaded into an Ion 540 chip, and the sequencing runs were performed using the Ion S5 Sequencing kit (ThermoFisher, MA, USA) with 500 flows.

Sequence data from the Ion Torrent run were analyzed using the platform-specific pipeline software Torrent Suite v5.10 for base calling, trim adapter and primer sequences, filtering out poor quality reads, and de-multiplex the reads according to the barcode sequences. Briefly, TMAP algorithm was used to align the reads to the hg19 human reference genome, and then, the variant caller plug-in was executed to search for germline variants in the targeted regions. Integrative genomics viewer (IGV) was used for visualization of the mapped reads. Variants were annotated using the Ion Reporter (ThermoFisher, MA, USA) software. Variant interpretation was restricted to the selected genes with known connection to oncology diseases. Variants were stratified by gene association with breast cancer and by variant association with cancer (no/potential/known) based on available data from the ClinVar database at the time of writing [21]. Synonymous mutations were not excluded from the analysis.

### Statistical analysis

Descriptive data are given as means ± standard deviation (SD) for continuous data, and frequencies (percentages) for categorical variables. Between-group differences in clinical parameters were assessed using 2-sample t tests for continuous variables, and χ2 tests or Fisher exact tests for categorical variables.

When comparing the number of different variants across groups, the assumptions of normality and homogeneity of variances were assessed using the Shapiro–Wilk test and Levene’s test respectively. Given that in some cases these assumptions were not met, Welch’s t-test was used to compare the number of variants with known, potential or no association with breast cancer and with any cancer. We also used the bootstrap procedure to get robust confidence intervals and p-values. To increase statistical power, we lumped together variants with potential and known associations using both weighted and unweighted scores. To generate the weighted scores, we gave double weight for variants with known association compared to those variants with potential association. To correct for multiple testing we applied the Benjamini-Hochberg procedure targeting a false discovery rate < 15%. A corrected two-tailed p-value less than 0.05 was considered statistically significant. Effect sizes were computed as adjusted standardized mean differences (Hedge’s g). Analyses were conducted using IBM SPSS Statistics for Windows, Version 23.0 (IBM Corp. Released 2015 Armonk, New York, U.S.A.).

### Results

Table 2 displays patients’ clinical characteristics. BC only patients had a slightly younger age at the time of breast cancer diagnosis compared to the synchronous/metachronous cancer group (47.7 vs 54.4, p = 0.079). All other clinical parameters evaluated were similar in the BC and BC-TC groups. It is worth noting that no difference was found in the frequency of the different BC treatment modalities (irradiation, hormone or chemotherapy).
Table 2Patient characteristics at BC diagnosis and at follow-upBC onlyBC and TCp-valueMean / No of casesSD / %Mean / No of casesSD / %No. of patients1519Age at follow-up67.39.262.310.50.16BC clinical characteristics Age at BC diagnosis47.710.254.411.20.079 BC grade0.363  1337.5%320%1  2112.5%640%0.104  3450%640%1 BC T stage0.393  1436.4%758.3%0.715  2654.5%541.7%0.475  419.1%00%0.441 Lymph node metastasis533.3%842.1%0.728 Vascular invasion222.2%215.4%1 HER2-positivity640%421%0.276 ER-positivity1173%1052%0.296 PR-positivity853%842%0.730 Invasive ductal carcinoma1386.7%1684.2%1 Distant metastasis320%210.5%0.634 BC radiation therapy1493.2%1789.5%1 BC chemotherapy1066.7%960%1 BC hormone therapy1173.3%1062.5%0.704 BC targeted therapy16.7%212.5%1 BC relapse426.7%315.8%0.672TC clinical characteristics Age at TC diagnosis––53.515– Papillary histology––1894.%– Follicular histology––15.3%– Lymph node metastasis––421%–

Patient characteristics at BC diagnosis and at follow-up

The results of the genetic analyses for the BC-TC patients and the control group are presented in Tables 3 and 4 respectively. Details of the statistical analysis are reported in Table 5.
Table 3List of any cancer (and breast cancer) related genes and unique variants identified by individual patients in the BC-TC group. The strength of known association of each variant with oncogenesis is marked as known (**), potential (*) or no (no marker)TC-BC patients (no. of SNPs)Gene (known breast cancer gene)SNPEffect of mutation *- potential **- known associationT + 1 (4)POLEc.1738C > Ap.His580Asn*BUB1c.677C > Tp.Ala226Val*WRNc.355 + 4G > Cintronic*ERCC2c.545C > Tp.Ala182Val*T + 2 (4)SMARCA4c.918G > Cp.Gln306His*TSC1c.3109_3110insGCAp.Gly1037_Ser1038insSer*BIVM-ERCC5,ERCC5c.1954C > G, c.592C > Gp.Pro652Ala, p.Pro198Ala*NBNc.511A > Gp.Ile171Val*T + 3 (5)ATRc.2924 T > Cp.Leu975Ser*KITc.2695A > Gp.Met899Val*BMPR1Ac.563G > Ap.Arg188His*PALB2c.522A > Gp.(=)*HOXB13c.251G > Ap.Gly84Glu**T + 4 (3)BRCA1c.3925A > Cp.Asn1309His*CDKN2Ac.-15_8delGGCGGCGGGGAGCAGCATGGAGCCp.Glu2_Pro3del**CDK4c.625C > Tp.Arg209CysT + 5 (5)SMAD4c.845A > Cp.His282Pro*XPCc.155C > Tp.Ser52Leu*VPS45c.566A > Gp.Glu189Gly*FANCMc.527C > Tp.Thr176IleATMc.4388 T > Gp.Phe1463CysT + 6 (4)ATRc.4357A > Gp.Ile1453Val*ATMc.8983C > Ap.Leu2995Ile*BRCA2c.8755-1G > Aintronic**BRCA1c.692C > Tp.Thr231Met*T + 7 (6)MUTYHc.1276C > Tp.Arg426CysATRc.4912C > Tp.Gln1638Ter**ATRc.1546A > Gp.Thr516Ala*NSD1c.3805 T > Cp.Ser1269Pro*BRCA2c.6968A > Gp.His2323Arg*CHEK2c.614C > Tp.Thr205Ile*T + 8 (4)ATRc.3424A > Gp.Ser1142Gly*PDGFRAc.842C > Tp.Thr281Met*TINF2c.488C > Gp.Pro163Arg*FLCNc.592G > Ap.Asp198AsnT + 9 (1)TERTc.1234C > Tp.His412TyrT + 10 (6)TSC2c.3820 T > Cp.Ser1274Pro*FANCIc.2011A > Gp.Ile671ValFANCIc.2604A > Cp.Glu868AspMFSD3c.1033C > Tp.Arg345Cys*BRCA2c.1483G > Cp.Ala495Pro*BRCA2c.4409_4410delTAp.Ile1470fs**T + 11 (2)PTCH1c.4324C > Tp.Arg1442TrpCDH1c.32 T > Cp.Leu11Pro*T + 12 (5)PMS1c.2783 T > Cp.Leu928Pro*ATMc.1273G > Ap.Ala425Thr*ATMc.1300C > Tp.Pro434Ser*BRCA2c.9038C > Tp.Thr3013IleSTK11c.413A > Gp.Glu138Gly*T + 13 (2)MUTYHc.536A > Gp.Tyr179Cys**SLX4c.2359G > Ap.Glu787LysT + 14 (3)FANCGc.634G > Ap.Ala212Thr*CHEK2c.1556G > Tp.Arg519Leu*KIF1Bc.2680G > Ap.Val894Met*T + 15 (2)RECQLc.1360C > Tp.Arg454Cys*CHEK2c.444 + 1G > Aintronic**T + 16 (4)TSC1c.2418G > Ap.Met806Ile*POLD1c.835_837delGAGp.Glu279del*PDGFRAc.1099G > Ap.Val367Met*DIS3L2c.1447C > Gp.Arg483Gly*T + 17 (3)RECQL4c.3062G > Ap.Arg1021GlnSLX4c.3890G > Ap.Gly1297GluSLX4c.179A > Cp.Gln60ProT + 18 (4)MSH6c.3226C > Tp.Arg1076Cys**ATMc.7290 T > Gp.His2430Gln*MAXc.25G > Tp.Val9Leu*BRCA1c.181 T > Gp.Cys61Gly**T + 19 (0)––Table 4List of any cancer (and breast cancer) related genes and unique variants identified by individual patients in the control group. The strength of known association of each variant with oncogenesis is marked as known (**), potential (*) or no (no marker)BC only patients (no. of SNPs)Gene (known breast cancer gene)SNPEffect of mutation *- potential **- known associationT-1 (3)ATMc.8965C > Gp.Gln2989Glu*MUTYHc.1435G > Ap.Glu479Lys*FANCBc.2435A > Gp.Tyr812Cys*T-2 (2)MSR1c.919G > Tp.Asp307Tyr*ATMc.6067G > Ap.Gly2023ArgT-3 (4)RAD50c.1741C > Tp.His581Tyr*MITFc.1255G > Tp.Glu419Ter**WRNc.95A > Gp.Lys32ArgAPCc.7490C > Tp.Ser2497LeuT-4 (6)ELANEc.341 T > Cp.Leu114Ser*RB1c.10A > Cp.Lys4Gln*WRNc.1149G > Tp.Leu383PheERCC4c.1135C > Tp.Pro379SerMSR1c.667 T > Ap.Ser223Thr*WRNc.2983G > Ap.Ala995ThrT-5 (1)TSC1c.1390G > Ap.Gly464Ser*T-6 (1)CHEK2c.190G > Ap.Glu64Lys**T-7 (3)MSH2c.2187G > Ap.Met729Ile*RAD50c.734A > Gp.Glu245Gly*FANCAc.2658G > Cp.Glu886Asp*T-8 (2)CDH1c.344C > Tp.Thr115Met*METc.2962C > Tp.Arg988CysT-9 (6)MDH2c.415G > Ap.Val139Ile*FANCAc.1874G > Cp.Cys625SerATRc.7303A > Gp.Ile2435Val*TERTc.2726 T > Cp.Val909Ala*PALB2c.2483G > Ap.Cys828Tyr*FANCBc.454 T > Cp.Phe152Leu*T-10 (3)RECQL4c.616A > Cp.Lys206Gln*BLMc.3536C > Tp.Thr1179Ile*FANCCc.632C > Gp.Pro211ArgT-11 (3)MITFc.1334C > Ap.Thr445LysWRNc.1211 T > Cp.Ile404ThrSLX4c.179A > Cp.Gln60ProT-12 (1)FANCIc.824 T > Cp.Ile275ThrT-13 (3)BUB1c.307A > Gp.Ile103ValFANCAc.1874G > Cp.Cys625SerAXIN2c.1994delGp.Gly665fs*T-14 (3)RECQL4c.941C > Tp.Pro314LeuSLX4c.4963A > Gp.Arg1655GlyRAD51Cc.130 T > Cp.Ser44ProT-15 (1)PMS1c.1609G > Ap.Glu537Lys*Table 5Comparison of the two patient groups by the number of genetic variantsBC onlyBC and TCp-valueMeanStd. errorMeanStd. errorGenes associated with BC No. of SNPs with potential/ known association with BC0.270.120.840.220.029 No. of SNPs with potential/ known association with BC (weighted scores)0.170.080.530.150.039112 cancer risk genes No. of SNPs with no known association with oncogenesis1.200.300.740.200.206 No. of SNPs with a potential association with oncogenesis1.400.392.320.370.096 No. of SNPs with a known association with oncogenesis0.200.110.470.140.131 No. of SNPs with potential/ known association with oncogenesis1.600.362.790.390.032 No. of SNPs with potential and known association with oncogenesis (weighted scores)0.900.181.630.230.017

List of any cancer (and breast cancer) related genes and unique variants identified by individual patients in the BC-TC group. The strength of known association of each variant with oncogenesis is marked as known (**), potential (*) or no (no marker)

List of any cancer (and breast cancer) related genes and unique variants identified by individual patients in the control group. The strength of known association of each variant with oncogenesis is marked as known (**), potential (*) or no (no marker)

Comparison of the two patient groups by the number of genetic variants

We found a significantly lower number of SNPs with potential or known association with breast cancer in the BC only group (mean difference − 0.58, BCa 95% CI [− 1.09, − 0.06], t (27) = − 2.30, p = 0.029, g = 0.74 for unweighted and mean difference − 0.36, BCa 95% CI [− 0.70, − 0.02], t(27.26) = − 2.17, p = 0.039, g = 0.70 for weighted scores).

The individual number of SNPs with no known genetic association with cancer was nominally higher in the BC only group (mean difference 0.46, BCa 95% CI [− 0.27, 1.20], t(25.56) = 1.30, p = 0.206, g = 0.39), while the number of potentially associated SNP or those with a known association was nominally lower in the BC only group (mean difference − 0.92, BCa 95% CI [− 2.01, 0.17], t(30.98) = − 1.72, p = 0.096, g = 0.59 and mean difference − 0.27, BCa 95% CI [− 0.63, 0.09], t(31.37) = − 1.55, p = 0.131, g = 0.51, respectively).

When we lumped together SNPs with potential or known association with carcinogenesis to increase statistical power, we found significantly higher values in the synchronous/metachronous cancer group compared to the BC only group using both unweighted (mean difference − 1.19, BCa 95% CI [− 2.27, − 0.11], t(31.88) = − 2.24, p = 0.032, g = 0.76) and weighted (mean difference − 0.73, BCa 95% CI [− 1.32, − 0.14], t(31.80) = − 2.51, p = 0.017, g = 0.83) scores.

### Discussion

In our study, we found a significantly higher burden of established and potential germline cancer risk variants among subjects with a history of metachronous thyroid and breast cancer compared to patients with a history of breast cancer only. The trend was similar and effect sizes were still considerable when we assessed the number of SNPs with known and potential cancer risk separately. However, in these cases, probably due to lack of power, statistical significance was not reached. Our results are compatible with the hypothesis that the sporadic co-occurrence of thyroid and breast cancer is of multigenetic origin and probably related to the burden of the carcinogenic SNPs rather than an individual gene alteration.

Comparison of clinical features of breast cancer, such as onset, stage, histology and treatment also yielded no significant differences. There was a non-significant tendency for later age of breast cancer onset in the study group which is contrary to previous findings [22].

The metachronous and synchronous occurrence of breast and thyroid cancer has been first described nearly four decades ago [1]. The relationship between the two tumor types has since been found to be bidirectional. In recent meta-analyses, the odds ratio for thyroid cancer following breast cancer treatment was 1.55 while the odds ratio of developing BC after TC was reported between 1.18–1.32 [3, 4]. Studies exploring the cause behind this association emphasize either the role of common underlying hormonal and environmental factors, or the role of the first malignancy in the development of the second. The potential role of cancer treatment and the increased surveillance of cancer survivors fall in this latter category. With breast cancer being the most common type of malignancy in women [23] and with the increasing incidence of thyroid cancer worldwide [24, 25], the importance of this topic is clear.

One potential hypothesis explaining this bidirectional relationship implicates common hormonal factors and is rooted in the overwhelming predominance of both cancer types in reproductive-aged women [4]. Estrogen, progesterone and androgen receptors have been shown to be overexpressed not only in breast cancer but also in thyroid neoplasms [26]. In vitro and animal studies also point towards the potential oncogenic effect of estrogens in thyroid cells [26, 27]. An increased TSH secretion in response to estrogens has also been suggested as a potential pathomechanism for thyroid cancer development [3], as serum TSH levels have been shown to correlate with thyroid cancer risk [28]. TSH and thyroid hormones also have been implicated in breast carcinogenesis by in vitro animal and observational studies. T3 seems to stimulate proliferation in breast cancer cells in vitro at least in part through interactions with the estrogen signalling system [29, 30]. Tumor suppressor pathways downstream of the TRβ nuclear thyroid receptor and oncogenic pathways downstream of the membrane receptor αVβ3 have been suggested to mediate the role of thyroid hormones in carcinogenesis [31–33]. While some observational studies in humans found a decreased breast cancer incidence in hypothyroid and an increased incidence in hyperthyroid patients [7, 8], other studies have failed to replicate these findings [34, 35].

Endocrine disrupting chemicals (EDC) have been linked to numerous types of hormone dependent malignancies including both thyroid and breast cancer [36, 37]. In addition to EDCs, obesity, a shared risk factor for both tumor types has also been postulated to increase the risk of synchronous/metachronous cancer development [38, 39]. Selective estrogen receptor modulators (SERMs) are widely used in breast cancer treatment. While there is some evidence that these drugs due to their partial estrogen effect increase TSH secretion and thyroid cell proliferation, their potential role in thyroid cancer development has not yet been studied [13]. The similar frequency of hormone therapy in the BC only, and the BC-TC groups in our study point toward a limited role of sex hormones in the development of co-occurring breast and thyroid cancer.

Other theories suggest the role of prior cancer treatment in the development of the second primary malignancy. Despite several improvements minimizing radiation scatter to surrounding tissues, there is a definitely increased risk for certain malignancies following external beam radiation for breast cancer [40]. While some reports are conflicting [41], most available data do not substantiate such a connection between thyroid cancer and previous adjuvant breast irradiation [42–45]. Similarly, radioactive iodine treatment given for thyroid cancer does not seem to play a role in subsequent breast cancer development [46–48]. Our data with similar frequency of radiation therapy for breast cancer in the investigated groups also argue against the role of external or internal radiation in the development of synchronous/metachronous breast and thyroid cancer.

The increased co-occurrence of these tumor types could also be related to surveillance bias. The higher compliance with, and higher rate of screening efforts among cancer survivors could also affect the time of diagnosis and could lead the substantial overdiagnosis of clinically irrelevant second primary malignancy [49]. This seems to be especially true for differentiated thyroid cancer [50]. However, as there are no screening programs for breast cancer among thyroid cancer survivors or vice versa, surveillance bias is unlikely to account for the preferential association of these tumor types.

Cowden syndrome (CS) and Cowden-like syndromes (CLS) are characterised by hamartomas and an extremely increased risk for several types of malignancies including breast cancer, thyroid cancer, endometrial cancer, colorectal cancer, and melanoma with standardized incidence ratios for breast and thyroid cancer in the range of 6 to 9 [14]. CS and most forms of CLS have an autosomal dominant inheritance. However, these conditions, referred to as PTEN hamartoma tumor syndromes, make up only a small fraction of synchronous/metachronous thyroid and breast cancer cases. This is consistent with our finding that no patient in our samples had a mutation in the PTEN gene. Thus, the existence of other shared genetic risk factors is highly probable. To the best of our knowledge, genetic analyses exploring this notion have not been conducted and genetic factors underlying sporadic metachronous cancer cases are yet to be elucidated.

The main strengths of this study were the state-of-the-art genetic analysis using NGS technology, and the fact that both cases and controls came from the same source population. The main limitation of our research was the relatively low number of patients limiting statistical power even in analyses of genetic scores. Furthermore, our study was unable to identify or investigate any single genetic risk locus behind synchronous/metachronous cancer development. Two types of bias could also have affected our findings. First, participants were recruited years after diagnosis potentially leading to survivor bias. Second, the exclusion of controls with less than 12 years of follow-up lead to an age difference between our study groups that is contrary to previous findings [22], and probably reflects selection bias. However, both of these selections were deemed to be necessary to minimize contamination of the control group with thyroid cancer.

### Conclusions

In conclusion, we reported an increased burden of carcinogenic SNPs in people with both thyroid and breast cancer compared to individuals with breast cancer only, based on whole exome sequencing of 112 known hereditary cancer risk genes. We found no differences in clinicopathologic parameters between the groups suggesting that both groups have similar presentation at the time of diagnosis of breast cancer. While our study was not powered to identify specific risk loci for metachronous cancer development, our findings further support the multigenetic etiology of co-occurring breast and thyroid cancer. Our findings do not directly contradict any of the other, previously detailed theories explaining the association between these tumor types. Nevertheless, our results do underline the need for further genetic research in this field, which are lacking at the moment.



# SUPPLEMENTAL FILE 1: 12885_2021_Article_8377.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)