# MAIN TEXT

## Next-generation sequencing of BRCA1 and BRCA2 genes in Moroccan prostate cancer patients with positive family history

### Abstract

Prostate cancer is the most common male cancer in Morocco. Although sporadic forms account for a large proportion of patients, familial forms of prostate cancer are observed in 20% of cases and about 5% are due to hereditary transmission. Indeed, germline mutations in BRCA1/2 genes have been associated with prostate cancer risk. However, the spectrum of these mutations was not investigated in Moroccan Prostate cancer patients. Thereby, the aim of this study was to characterize and to estimate the prevalence of germline BRCA1/2 mutations and large rearrangements in Moroccan patients with familial prostate cancer. The entire coding regions and intron/exon boundaries of BRCA1 and BRCA2 genes have been analyzed by next generation sequencing (NGS) in a total of 30 familial prostate cancer patients. Three pathogenic mutations were detected in four unrelated patients (13.3%). One BRCA1 mutation (c.1953_1956delGAAA) and two BRCA2 mutations (c.7234_7235insG and BRCA2ΔE12). In addition, sixty-three distinct polymorphisms and unclassified variants have been found. Early identification of germline BRCA1/2 mutations may be relevant for the management of Moroccan prostate cancer patients.

### Introduction

Cancer is a serious public health problem in the world. About 18.1 million new cases and 9.6 million deaths have been reported by the International Agency for Research on Cancer (IARC) in 2018 [1]. Prostate cancer (PrCa) is one of the most commonly diagnosed cancers in men, especially in those aged over 50 [2]. Globally, there are 1,276,106 new cases and 358,989 deaths from PrCa per year [1]. In Morocco, about 3,990 new cases of PrCa have been diagnosed in 2018. Otherwise, it is the most common cancer of the genitourinary system and the most common cause of urological cancer death with 1,861 deaths [1].

Inherited mutations play a key role in the occurrence of PrCa. Epidemiological studies and segregation analysis have suggested a strong genetic origin of PrCa [3, 4]. The first linkage analysis in a series of prostate cancer patients have reported that 9% of familial prostate cancer cases are associated with alleles, located in a dominant susceptibility locus (HPC1), conferring high risk for prostate cancer with a penetrance of 88% at age 85 [5]. Moreover, differences in the incidence and outcome of PrCa observed among men of different race or ethnicity may confirm that some cases are partially attributed to genetic factors [6, 7]. Indeed, about 5 to 15% of PrCa cases are due to high-risk hereditary factors [8, 9]. Genome Wide Association Studies (GWAS) have revealed the association of a number of gene mutations with an increased PrCa risk such as HOXB13, BRCA1, BRCA2, ATM, CHEK2, RAD51D, PALB2 and mismatch repair (MMR) genes [10, 11]. BRCA1 and BRCA2 are involved in maintaining of genome integrity [12]. BRCA1 is a large gene located on chromosome 17q and composed of 22 exons which encode 1683 amino acids [13]. BRCA2 gene has been located on chromosome 13q12-13 in 1995 and presents no homology with BRCA1 gene. Currently, more than 2000 and 2400 distinct germline mutations have been described in BRCA1 and BRCA2 respectively [14]. Previous studies have observed that BRCA1 and BRCA2 pathogenic mutations carriers have 1.8 to 3.8-fold and 2.5 to 8.6-fold increased relative risk of developing PrCa by the age of ≤65 years old, respectively [15–19].

The relevance of BRCA1/2 mutations in patients with PrCa was not yet studied in Morocco. In fact, this work is the first Moroccan study investigating the spectrum of BRCA1 and BRCA2 germline mutations among Moroccan patients with a family history of PrCa using Next-generation sequencing (NGS) approach.

### Materials and methods

This study involved 30 PrCa patients admitted to Mohammed VI center for cancer treatment in Casablanca, and selected according to the following criteria:

At least two cases of PrCa among first (father, brother) or second-degree relatives (grandfather, uncle).

Three cases of PrCa among first (father, son or brothers) or second-degree relatives (nephews, uncles on the maternal or paternal side).

Two cases of PrCa, diagnosed before age 55, in first-degree relatives (father, son or brothers) or second-degree relatives (nephews, uncles on the maternal or paternal side).

Each patient was asked to complete a questionnaire in order to obtain a complete family cancer history. Pathological features and medical data were collected from medical records. The study was performed in accordance with the Declaration of Helsinki protocols and was approved by the institutional ethical committee of BioMedical Research in Casablanca (CERBC) of the Faculty of Medicine and Pharmacy, Casablanca (Morocco), and written informed consent was obtained from each subject.

Genomic DNA was extracted from peripheral blood using a commercially available kit (Isolate II Genomic DNA Kit, Bioline). DNA concentration and purity were measured by NanoDrop 2000 Spectrophotometer and Qubit 3.0 Fluorometer (Thermo Fischer Scientific, Waltham, MA, USA).

10 ng of DNA per sample was used to generate the sequencing library with the Ion PGM™ sequencing system and Oncomine™ BRCA Research Assay (Thermo Fisher Scientific, Waltham, MA, USA). This panel consists of two pools with 265 primer pairs covering complete coding sequence of BRCA1 and BRCA2 genes and splice site sequences at intron/exon junctions. PCR amplicons were partially digested by FuPa enzyme and then ligated to barcoded adapter. The generated amplicons were purified with AMPure™ XP Reagent (Beckman Coulter, Brea, CA, USA). After purification, libraries were quantified, diluted to 100 pM, and amplified through emulsion PCR on Ion OneTouch™ 2 System using Ion PGM™ Hi-Q™ View OT2 Kit (Thermo Fisher Scientific Waltham, MA, USA). Finally, NGS sequencing was performed on the Ion PGM™ sequencer using Ion PGM™ Hi-Q™ View Sequencing Kit (Thermo Fisher Scientific, Waltham, MA, USA).

Quality control of the sequencing data and their alignment to the HG19 human genome were conducted using the Ion Torrent Suite™ Software 5.0.5 (Thermo Fisher Scientific). The generated data were then analyzed by Torrent Variant Caller plugin version 5.0 (Thermo Fisher Scientific) in order to identify genetic variants and Ion Reporter™ software (Thermo Fisher Scientific) for variant annotation. The coverage depth was ≥ 250X.

All mutations were reported following the Human Genome Variation Society (HGVS) nomenclature (http://www.HGVS.org/varnomen) based on the coding sequences NM_007294.3 and NM_000059.3 for BRCA1 and BRCA2, respectively. The variants were categorized as pathogenic or common polymorphisms or variant of uncertain significance (VUS) according to ClinVar database (https://www.ncbi.nlm.nih.gov/clinvar), Breast Cancer Information Core BIC (https://research.nhgri.nih.gov/bic/), the BRCA Exchange (https://brcaexchange.org), Universal Mutation Database (http://www.umd.be/BRCA1/, http://www.umd.be/BRCA2/), and Leiden Open (source) Variation Database (LOVD) (http://www.lovd.nl/3.0/home).

Unclassified variants were analyzed using in silico prediction tools: Polyphen (http://genetics.bwh.harvard.edu/pph2/) and Mutation taster (http://www.mutationtaster.org/).

Population frequency data are taken from various projects GnomAD (h ttps://gnomad.broadinstitute.org), TopMed (https://www.nhlbiwgs.org/topmed-whole-genome-sequencing-project-freeze-5b-phases-1-and-2) and ALFA (https://www.ncbi.nlm.nih.gov/snp/docs/gsr/alfa).

### Patients

This study involved 30 PrCa patients admitted to Mohammed VI center for cancer treatment in Casablanca, and selected according to the following criteria:

At least two cases of PrCa among first (father, brother) or second-degree relatives (grandfather, uncle).

Three cases of PrCa among first (father, son or brothers) or second-degree relatives (nephews, uncles on the maternal or paternal side).

Two cases of PrCa, diagnosed before age 55, in first-degree relatives (father, son or brothers) or second-degree relatives (nephews, uncles on the maternal or paternal side).

Each patient was asked to complete a questionnaire in order to obtain a complete family cancer history. Pathological features and medical data were collected from medical records. The study was performed in accordance with the Declaration of Helsinki protocols and was approved by the institutional ethical committee of BioMedical Research in Casablanca (CERBC) of the Faculty of Medicine and Pharmacy, Casablanca (Morocco), and written informed consent was obtained from each subject.

### Molecular analysis

Genomic DNA was extracted from peripheral blood using a commercially available kit (Isolate II Genomic DNA Kit, Bioline). DNA concentration and purity were measured by NanoDrop 2000 Spectrophotometer and Qubit 3.0 Fluorometer (Thermo Fischer Scientific, Waltham, MA, USA).

10 ng of DNA per sample was used to generate the sequencing library with the Ion PGM™ sequencing system and Oncomine™ BRCA Research Assay (Thermo Fisher Scientific, Waltham, MA, USA). This panel consists of two pools with 265 primer pairs covering complete coding sequence of BRCA1 and BRCA2 genes and splice site sequences at intron/exon junctions. PCR amplicons were partially digested by FuPa enzyme and then ligated to barcoded adapter. The generated amplicons were purified with AMPure™ XP Reagent (Beckman Coulter, Brea, CA, USA). After purification, libraries were quantified, diluted to 100 pM, and amplified through emulsion PCR on Ion OneTouch™ 2 System using Ion PGM™ Hi-Q™ View OT2 Kit (Thermo Fisher Scientific Waltham, MA, USA). Finally, NGS sequencing was performed on the Ion PGM™ sequencer using Ion PGM™ Hi-Q™ View Sequencing Kit (Thermo Fisher Scientific, Waltham, MA, USA).

Quality control of the sequencing data and their alignment to the HG19 human genome were conducted using the Ion Torrent Suite™ Software 5.0.5 (Thermo Fisher Scientific). The generated data were then analyzed by Torrent Variant Caller plugin version 5.0 (Thermo Fisher Scientific) in order to identify genetic variants and Ion Reporter™ software (Thermo Fisher Scientific) for variant annotation. The coverage depth was ≥ 250X.

All mutations were reported following the Human Genome Variation Society (HGVS) nomenclature (http://www.HGVS.org/varnomen) based on the coding sequences NM_007294.3 and NM_000059.3 for BRCA1 and BRCA2, respectively. The variants were categorized as pathogenic or common polymorphisms or variant of uncertain significance (VUS) according to ClinVar database (https://www.ncbi.nlm.nih.gov/clinvar), Breast Cancer Information Core BIC (https://research.nhgri.nih.gov/bic/), the BRCA Exchange (https://brcaexchange.org), Universal Mutation Database (http://www.umd.be/BRCA1/, http://www.umd.be/BRCA2/), and Leiden Open (source) Variation Database (LOVD) (http://www.lovd.nl/3.0/home).

Unclassified variants were analyzed using in silico prediction tools: Polyphen (http://genetics.bwh.harvard.edu/pph2/) and Mutation taster (http://www.mutationtaster.org/).

Population frequency data are taken from various projects GnomAD (h ttps://gnomad.broadinstitute.org), TopMed (https://www.nhlbiwgs.org/topmed-whole-genome-sequencing-project-freeze-5b-phases-1-and-2) and ALFA (https://www.ncbi.nlm.nih.gov/snp/docs/gsr/alfa).

### DNA isolation

Genomic DNA was extracted from peripheral blood using a commercially available kit (Isolate II Genomic DNA Kit, Bioline). DNA concentration and purity were measured by NanoDrop 2000 Spectrophotometer and Qubit 3.0 Fluorometer (Thermo Fischer Scientific, Waltham, MA, USA).

### Next generation sequencing

10 ng of DNA per sample was used to generate the sequencing library with the Ion PGM™ sequencing system and Oncomine™ BRCA Research Assay (Thermo Fisher Scientific, Waltham, MA, USA). This panel consists of two pools with 265 primer pairs covering complete coding sequence of BRCA1 and BRCA2 genes and splice site sequences at intron/exon junctions. PCR amplicons were partially digested by FuPa enzyme and then ligated to barcoded adapter. The generated amplicons were purified with AMPure™ XP Reagent (Beckman Coulter, Brea, CA, USA). After purification, libraries were quantified, diluted to 100 pM, and amplified through emulsion PCR on Ion OneTouch™ 2 System using Ion PGM™ Hi-Q™ View OT2 Kit (Thermo Fisher Scientific Waltham, MA, USA). Finally, NGS sequencing was performed on the Ion PGM™ sequencer using Ion PGM™ Hi-Q™ View Sequencing Kit (Thermo Fisher Scientific, Waltham, MA, USA).

### Data analysis

Quality control of the sequencing data and their alignment to the HG19 human genome were conducted using the Ion Torrent Suite™ Software 5.0.5 (Thermo Fisher Scientific). The generated data were then analyzed by Torrent Variant Caller plugin version 5.0 (Thermo Fisher Scientific) in order to identify genetic variants and Ion Reporter™ software (Thermo Fisher Scientific) for variant annotation. The coverage depth was ≥ 250X.

All mutations were reported following the Human Genome Variation Society (HGVS) nomenclature (http://www.HGVS.org/varnomen) based on the coding sequences NM_007294.3 and NM_000059.3 for BRCA1 and BRCA2, respectively. The variants were categorized as pathogenic or common polymorphisms or variant of uncertain significance (VUS) according to ClinVar database (https://www.ncbi.nlm.nih.gov/clinvar), Breast Cancer Information Core BIC (https://research.nhgri.nih.gov/bic/), the BRCA Exchange (https://brcaexchange.org), Universal Mutation Database (http://www.umd.be/BRCA1/, http://www.umd.be/BRCA2/), and Leiden Open (source) Variation Database (LOVD) (http://www.lovd.nl/3.0/home).

Unclassified variants were analyzed using in silico prediction tools: Polyphen (http://genetics.bwh.harvard.edu/pph2/) and Mutation taster (http://www.mutationtaster.org/).

Population frequency data are taken from various projects GnomAD (h ttps://gnomad.broadinstitute.org), TopMed (https://www.nhlbiwgs.org/topmed-whole-genome-sequencing-project-freeze-5b-phases-1-and-2) and ALFA (https://www.ncbi.nlm.nih.gov/snp/docs/gsr/alfa).

### Results

In this study, we have screened 30 familial prostate cancer patients for BRCA1 and BRCA2 mutations. The median age at PrCa diagnosis was 67.43 years (range 54–80). All patients present with high grade prostatic adenocarcinoma (grade IV), larger tumor size (≥ T2) and distant metastasis (see Table 1).

Age D: Age at diagnostic, ADK: Adenocarcinoma, T: Size of the tumor, N: Node involvement, M: Metastasis, PSA: Protein Specific Antigen, a: At least two cases of PrCa among first (father, brother) or second-degree relatives (grandfather, uncle), b: Three cases of PrCa among first (father, son or brothers) or second-degree relatives (nephews, uncles on the maternal or paternal side).

Three BRCA1 and BRCA2 pathogenic mutations have been detected in four unrelated patients (see Table 2). One patient was found to carry a mutation in BRCA1 gene and three in BRCA2 gene. Therefore, the combined mutation frequency was 13.3% (4/30). The first mutation was identified in exon 10 of BRCA1 gene. A four nucleotide deletion called c.1953_1956delGAAA at the cDNA level and p.Lys653SerfsX47 (K653SfsX47) at the protein level. The second mutation was a small insertion detected in exon 14 of BRCA2 gene (c.7234_7235insG (7463insG)) inducing a shift in the reading frame. Finally, a whole-exon 12 deletion (BRCA2ΔE12) have been identified in BRCA2 gene. All three pathogenic mutations were found in patients with a strong family history of PrCa. The pedigree analysis revealed at least two affected family members with PrCa over two or three generations (Fig 1).

(A) BRCA1 c.1953_1956delGAAA. (B) BRCA2 c.7234_7235insG. (C and D) BRCA2 delExon12. The numbers indicate age of patients at diagnosis.

Additionally, sixty-three non-pathogenic BRCA1 and BRCA2 mutations were detected in this study (Table 3). Twenty eight mutations (14 missense mutations, 4 synonymous substitutions, 9 intronic variants and one non-framshift deletion) and thirty-five mutations (15 missense mutations, 11 synonymous substitutions and 9 intronic variants) were identified in BRCA1 and BRCA2 genes respectively.

a: TOPMED;

b: GnomAD_exome;

c: ALFA Project

Regarding previously reported VUS, in silico analysis was performed for six detected missense mutations reported as VUS or with conflicting interpretations of pathogenecity in Clinvar database (see Table 4). Two mutations c.7954G>A and c.9364G>A in BRCA2 gene were predicted to be implicated in the disease.

### Discussion

Germline mutations in BRCA1 and BRCA2 genes have been associated with a high risk of ovarian/breast and prostate cancers. Indeed, 13–18% of hereditary ovarian cancer cases [20–22], and around 5% of hereditary breast cancer cases are due to BRCA1 and BRCA2 mutations [23]. As well, the frequencies of both mutations in PrCa patients are 0.9% and 2.2% respectively [24]. In Morocco, BRCA1 and BRCA2 mutations have been extensively investigated for breast cancer but not so for prostate cancer. Thereby, our study is the first to describe inherited BRCA1 and BRCA2 mutation spectrum and prevalence in Moroccan PrCa patients using the high-throughput sequencing technique.

According to our findings, the frequency of deleterious BRCA1 (3.33%) and BRCA2 (10%) mutations was 13.33%. This frequency appears to be high compared to those observed in other populations such as in the UK (4% to 6%) [6, 25, 26], Ashkenazi Jews (1.4% to 5.2%) [27–35], Finns (3.3%) [36], in Israel (3.8%) [37], and Portuguese (5.2%) [38]. Considering only BRCA2 mutations, some studies have reported frequencies ranging from 1% to 7% [18, 39–45]. While for BRCA1 mutations, frequencies of 0.1%, 0.4% and 0.45% have been reported in Spain, Poland and UK respectively [16, 46–48]. The variations in BRCA1/2 mutation frequencies across populations may be due to variation in study sample size and inclusion criteria or patients’ ethnic background.

Diversity of mutations and their distribution throughout both genes complicate the initial mutation screening. Overall, each family has its "private" mutation. However, some recurrent mutations have been identified in Ashkenazi Jews (BRCA1 185delAG, BRCA1 5382insC and BRCA2 6174delT) [41–43]. Other founder mutations have also been reported in Portugal [46], Germany [44], Poland [48], Canada [45], Turkey [43], Iceland [42], and USA [40]. In the present study, we detected three pathogenic mutations in four patients from different regions of Morocco. The first mutation is a frameshift deletion in exon 10 of BRCA1 gene (c.1953_1956delGAAA). The second mutation is a frameshift insertion that is located in exon 14 of BRCA2 gene (c.7234_7235insG) and the last one is a deletion of entire exon 12 of BRCA2 gene. All these pathogenic variants and their impact have been previously reported in BIC and ClinVar databases.

First, the c.1953_1956delGAAA (p.Lys653SerfsTer47) mutation, also known as 2072_2075delGAAA or 2072delGAAA or 2072del4 under other nomenclatures, is a deletion of four nucleotide bases in exon 10 which is reported for the first time in the Moroccan population but previously identified in breast and/or ovarian cancer patients from other populations [20, 49–59]. This deletion changes the reading frame and creates a premature stop codon at position 47.

The second pathogenic variant (c.7234_7235insG) also found in BRCA2 gene is an insertion of guanine between nucleotides 7234 and 7235 in exon 14 which causes a shift in reading frame (Stop2413) and, as a consequence, the production of premature truncated protein. This mutation has been previously found in several breast/ovarian cancer families. It was first reported by Esteban Cardeñosa et al. [53] in a single Eastern Spanish family with breast/ovarian cancers. Next, two unrelated Moroccan patients with familial breast cancer were found to carry it in the study by Tazzite et al. [60]. Later, this mutation was also identified by De Juan Jiménez et al. [55] in a single patient with familial breast and ovarian cancer, and by De Juan et al. [61] in a man with breast cancer who did not have family history. Recently, it was described for a second time in a Moroccan breast cancer family [62]. It is important to note that this mutation is described for the first time in a familial PrCa case.

The last mutation was a complete deletion of BRCA2 exon 12 that was found in two unrelated PrCa patients. This isoform is known as BRCA2ΔE12 or BRCA2 del 12. Characterization of this genomic breakpoint or large genomic rearrangement (LGR) revealed a deletion of 96bp, which is similar to the deletion previously described [63, 64]. This mutation is an in-frame deletion that should result in the production of a 32 amino acid shortened protein. Many BRCA1 and BRCA2 LGRs have been associated with hereditary breast, ovarian and prostate cancers [65–68]. Their frequencies vary across different population. In general, they account for 4 to 28% of all BRCA1/2 mutations [69]. The majority of these variants occur within BRCA1, probably because of the high rate of Alu elements in this gene [70, 71]. To our knowledge, no specific function has been attributed to exon 12 of BRCA2 gene. However, its deletion may affect the functions of the adjacent domains or change the structure of the entire polypeptide. Speculatively, BRCA2Δ12 mutation may alter the ability of BRCA2 protein to repair DNA because it is located downstream of exon 11 which contains domains essential for interaction with RAD51 [63]. The detection of this mutation in two unrelated patients supported by the observation of common shared polymorphisms and unclassified variants namely c.3807T>C, c.6513G>C, c.4563A>G and c.7397T>C suggest that BRCA2Δ12 mutation may be a recurrent mutation in our population. Larger studies are needed to confirm this finding.

Several studies have been interested in familial PrCa specifically for BRCA1 and BRCA2 gene testing. Wilkens et al. [27] have tested three BRCA founder mutations in Ashkenazi Jewish families with PrCa. Their results showed that only one unaffected participant carried BRCA2 6174delT mutation [27]. Similarly, 38 PrCa families from UK have not been found to have pathogenic BRCA1 mutations but two BRCA2 germline mutations (6710delACAA and 5531delTT) were found in young patients [25]. In addition, Sinclair et al. [40] have screened 43 individuals with positive family history of PrCa and found one BRCA2 missense mutation (C1206A). In a large German cohort, mutation screening revealed five BRCA2 mutations (c.1813_14_insA, c.3847delGT, c.4449delA, c.6037A>T, c.7495C>T) [44].

Interestingly, all cases in the present cohort show higher grade prostatic adenocarcinoma with higher T stage and high Gleason score. These results are consistent with previous studies showing that male PrCa patients tend to present with more aggressive tumors [72, 73]. The histopathological analysis of 20 tumor tissue samples from patients with hereditary PrCa revealed that BRCA mutation carriers have higher frequency of grade tumors with a Gleason score ≥8 than non carriers (P = 0.012) [74]. This finding was consistent with those of a large study by Castro et al. [26] who found that PrCa patients with BRCA mutations were more likely to have T3-T4 tumors (P = 0.003), Gleason score of at least 8 (P = 0.00003), lymph node invasion (P = 0.00005) and distant metastases at the initial diagnosis (P = 0.005) compared to BRCA-negative patients. Recently, Petrovics et al. [75] have confirmed these observations. They observed that frequencies of deleterious BRCA mutation, particularly BRCA2 mutations, are higher in patients with advanced PrCa [75].

As well as deleterious mutations, BRCA1 and BRCA2 variants of unknown/uncertain significance (VUS) significantly arouse interest amongst the geneticists. Their clinical implication is still unclear which complicates genetic counseling. Complete gene and genome sequencing by NGS increases the number of discovered VUS. In the present work, we have identified 63 variants including neutral polymorphisms and VUS that have been previously reported. Twenty-nine of these are missense mutations (14 in BRCA1 and 15 in BRCA2 genes). Thirty-two variants have already been reported as polymorphisms. In fact, previous studies have identified some single-nucleotide polymorphisms associated with prostate cancer risk including c.442-34C>T (rs799923), c.1067A>G (rs1799950), c.4837A>T (rs1799966), c.4357+117G>C (rs3737559) for BRCA1 gene and c.1114A>C (rs144848) for BRCA2 gene [76, 77].

This is the first report of the prevalence of BRCA1 and BRCA2 mutations in Moroccan PrCa patients who had a family history. To conclude, although our results are considered preliminary due to the small sample size, they underline the importance of BRCA1 and BRCA2 genetic screening in hereditary PrCa. Hence, early identification of germline mutations in BRCA1 and BRCA2 genes may be relevant for management of patients with PrCa and also for preventing future cancers in their relatives.

### Supporting information

(PDF)

Click here for additional data file.



# SUPPLEMENTAL FILE 1: pone.0254101.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 3: PMC8270444

[File available at: automated_output/BRCA2/20251125_175029/pmc_fulltext/34242281_supplements/PMC8270444]

