# MAIN TEXT

## Genomic and Transcriptomic Characterization of Relapsed SCLC Through Rapid Research Autopsy

### Abstract

IntroductionRelapsed SCLC is characterized by therapeutic resistance and high mortality rate. Despite decades of research, mechanisms responsible for therapeutic resistance have remained elusive owing to limited tissues available for molecular studies. Thus, an unmet need remains for molecular characterization of relapsed SCLC to facilitate development of effective therapies.MethodsWe performed whole-exome and transcriptome sequencing of metastatic tumor samples procured from research autopsies of five patients with relapsed SCLC. We implemented bioinformatics tools to infer subclonal phylogeny and identify recurrent genomic alterations. We implemented immune cell signature and single-sample gene set enrichment analyses on tumor and normal transcriptome data from autopsy and additional primary and relapsed SCLC data sets. Furthermore, we evaluated T cell-inflamed gene expression profiles in neuroendocrine (ASCL1, NEUROD1) and non-neuroendocrine (YAP1, POU2F3) SCLC subtypes.ResultsExome sequencing revealed clonal heterogeneity (intertumor and intratumor) arising from branched evolution and identified resistance-associated truncal and subclonal alterations in relapsed SCLC. Transcriptome analyses further revealed a noninflamed phenotype in neuroendocrine SCLC subtypes (ASCL1, NEUROD1) associated with decreased expression of genes involved in adaptive antitumor immunity whereas non-neuroendocrine subtypes (YAP1, POU2F3) revealed a more inflamed phenotype.ConclusionsOur results reveal substantial tumor heterogeneity and complex clonal evolution in relapsed SCLC. Furthermore, we report that neuroendocrine SCLC subtypes are immunologically cold, thus explaining decreased responsiveness to immune checkpoint blockade. These results suggest that the mechanisms of innate and acquired therapeutic resistances are subtype-specific in SCLC and highlight the need for continued investigation to bolster therapy selection and development for this cancer.

### Introduction

SCLC is a lethal neuroendocrine malignancy accounting for 13% to 15% of new lung cancer cases annually worldwide.1,2 Most patients with SCLC present with initially chemotherapy-sensitive disease, but almost all will experience progression or relapse leading to death. In 2019, combination chemotherapy and immunotherapy was approved by the Food and Drug Administration as first-line treatment for metastatic SCLC,3,4 with modest 2-month improvement in survival compared with chemotherapy alone. For decades, topotecan was the only Food and Drug Administration–approved second-line chemotherapy with objective response rate of 10% to 20%,5 until lurbinectedin was granted accelerated approval in 2020 on the basis of a phase 2 trial revealing 35% objective response rate.6 Despite these recent approvals, there is an urgent need to develop more effective therapies for advanced SCLC.

Key studies in the past decade have profiled the mutational landscape of primary SCLC, driven by TP53 and RB1 inactivation.7, 8, 9 Furthermore, distinct SCLC subgroups have been identified on the basis of the expression of key transcription factors including ASCL1 and NEUROD1.10,11 How each SCLC subtype confers different clinical phenotypes and differential response to anticancer therapies is under active investigation,12 with the goal of delivering precision therapy to patients with SCLC by matching each SCLC subtype to specific treatments.

Molecular characterization of relapsed SCLC has been hampered by tissue scarcity owing to rapid clinical deterioration of patients with relapse. Therefore, unlike for primary SCLC, less is known on the genomic and transcriptomic landscapes of relapsed SCLC and mechanisms that mediate therapeutic resistance, although recent nonautopsy studies on relapsed SCLC have begun to address this knowledge gap. For example, Gardner et al.13 used patient-derived xenografts of paired chemosensitive and chemoresistant SCLC tumors to elegantly reveal that acquired chemoresistance occurred through epigenetic silencing of a DNA damage repair factor, SLFN11. Wagner et al.14 performed genomic profiling on a cohort of patients with relapsed SCLC and identified recurrent Wnt pathway alterations as a mechanism of acquired chemoresistance. Weiss et al.15 performed genome-wide exome and RNA sequencing (RNA-seq) on 12 patients with SCLC who relapsed after platinum-based chemotherapy. Aside from driver mutations in RB1 and TP53, the authors identified few recurrent targetable genomic alterations in this cohort of patients. Finally, an important study by Stewart et al.16 performed single-cell sequencing of circulating tumor cells and circulating tumor cell-derived xenografts from patients with platinum-sensitive and refractory SCLC and revealed an association between increased intratumoral heterogeneity and chemoresistance. The latter study is one of the first nonautopsy studies to directly evaluate intratumor heterogeneity in advanced SCLC. Overall, however, further study of relapsed SCLC is needed to identify additional targetable mechanisms underlying therapeutic resistance, including resistance to immunotherapy, which is now approved for frontline treatment in the metastatic setting.

The use of tumor specimens from rapid research autopsy has accelerated the study of tumor heterogeneity and acquired resistance in advanced cancer.17 To our knowledge, this is the first study to perform whole-exome and transcriptome profiling of advanced SCLC through research autopsy. From exome sequencing, we inferred intertumor and intratumor clonal heterogeneity arising from branched evolution and transcriptome analyses supported the subtype-specific suppression of adaptive antitumor immunity in primary and advanced SCLC. Our results provide new insights into the subclonal architecture of advanced SCLC and identify new potentially targetable pathways involved in antitumor immune responses.

### Materials and Methods

Informed consents were obtained from five patients with advanced SCLC to participate in an institutional review board–approved clinical study for tumor profiling by next-generation sequencing and body donation (NCT02090530).17,18 Deceased patients were transported to The Ohio State University Regional Autopsy Center, where research autopsy for tumor procurement was performed no more than 16 hours after the passing of the patients. Computed tomography imaging when available was used to guide procurement from organs with cancer. After autopsy completion, the deceased were transported to a designated funeral home within 24 hours.

Samples with tumor cell content greater than 60% and without substantial necrosis were selected for next-generation sequencing analyses. Genomic DNA and total RNA were extracted using Qiagen kits per manufacturer’s protocol. Libraries were prepared after established protocols (TruSeq Stranded Total RNA with RiboZero Gold, Illumina),18,19 enriched with the xGEN Exome Research Panel version 1.0 (IDT) and sequenced on an Illumina HiSeq 4000.

Circulating tumor DNA (ctDNA) sequencing was isolated using QIAamp Circulating Nucleic Acid Kit (Qiagen) per manufacturer’s protocol. An input of 300 ng was used to generate libraries for paired-end sequencing on a NextSeq instrument achieving median coverage of approximately ×500.20

These bioinformatics analyses were performed as previously described.18 Briefly, sequencing reads were aligned to human genome build 19 (hg19) using Burrows-Wheeler Aligner (bwa),21 deduplication with Picard (https://github.com/broadinstitute/picard), and base quality score recombination and realignment around insertion and deletion (indels) with GATK.22 Variants were called with VarScan223 and allele-specific copy number variations (CNVs) with FALCON.24 Clonal inference was performed using Canopy,25 and mutational signatures were inferred with deconstructSigs.26 Bradley-Terry modeling was used to estimate relative ordering of mutations in the phylogeny branches of autopsy patients with SCLC as previously described.18

Significantly mutated gene (SMG) analysis was identified using MuSiC version 2.0,27 with default settings. Variants from all tumor samples were merged per patient by taking the highest variant allele fraction. The p value and false discovery rate (FDR) estimates are on the basis of three tests including Fisher's combined p value test, convolution test, and likelihood ratio test methods. For a specific gene, if FDR for at least two of these tests is less than or equal to maximum FDR, then it was called as a SMG. For all five SCLC research autopsy patients, the maximum FDR cutoff was set to 0.05.

CHASM was used to identify statistically likely driver mutations.28 Missense variants were merged per patient and deduplicated before input to CHASM.

RNA-seq reads were aligned to hg19 using HISAT2,29 and fragments per kilobase per million reads and transcript reads per million (TPM) were calculated with StringTie30 as described in the Supplementary Methods.

Single-sample gene set enrichment (ssGSEA) version 2.0 was implemented with default settings in R.31 Gene sets assayed were obtained from PanCancer Immuno-Oncology 360 and Tumor Signaling 360 from NanoString. Immune signatures (ImSigs) were run using TPM values with default settings to identify enrichment of 10 ImSig.32 Linear mixed-effects models and t statistics were used to model ssGSEA and ImSig scores by allowing correlation among multiple samples within each patient. Tukey’s method was used for adjusting p values of multiple comparisons of enrichment scores from ssGSEA using NanoString gene sets and scores from ImSig.

RNA-seq data of primary and relapsed SCLC and normal lung tissue were obtained from four previous publications.8,14,15,33 Of these four data sets, one had paired primary SCLC and normal lung samples,8 whereas the rest contained either exclusively tumor14,15 or normal samples.33 Non-SCLC RNA-seq data were downloaded from the The Cancer Genome Atlas data portal (https://portal.gdc.cancer.gov/). We randomly chose a subset of 60 and 49 tumor-normal paired samples of lung adenocarcinoma and squamous cell carcinoma, respectively, for ssGSEA and ImSig analyses.

SCLC subtype classification was performed after a previously published method.34 Briefly, the samples were classified on the basis of defining thresholds (log2[fragments per kilobase per million + 1]) for transcription factors YAP1 (2.5) and POU2F3 (2.5). Samples were classified as ASCL1 or NEUROD1 subtype on the basis of relative expression of these genes, with minimum required difference of 1. The above-mentioned method was unable to classify all samples included in this study; therefore, remaining samples were assigned to SCLC subtypes on the basis of the gene (ASCL1, NEUROD1, YAP1, or POU2F3) with highest expression (TPM) value. When expression difference was less than twofold between ASCL1 and NEUROD1, samples were classified as dual-positive ASCL1/NEUROD1. ARG2 expression between subtypes was compared using two-tailed unpaired t tests.

TPM values for genes in the housekeeping (n = 11) and predictor (n = 18) sets were converted into count-equivalent values, and gene expression profile (GEP) scores were computed as previously described35 and as detailed in the Supplementary Methods. Heatmaps of log-transformed TPM values of the 18 genes in the T cell-inflamed GEP were generated using Qlucore Omics Explorer (Qlucore AB) version 3.6.

### Rapid Research Autopsy

Informed consents were obtained from five patients with advanced SCLC to participate in an institutional review board–approved clinical study for tumor profiling by next-generation sequencing and body donation (NCT02090530).17,18 Deceased patients were transported to The Ohio State University Regional Autopsy Center, where research autopsy for tumor procurement was performed no more than 16 hours after the passing of the patients. Computed tomography imaging when available was used to guide procurement from organs with cancer. After autopsy completion, the deceased were transported to a designated funeral home within 24 hours.

### Tumor DNA/RNA-Seq

Samples with tumor cell content greater than 60% and without substantial necrosis were selected for next-generation sequencing analyses. Genomic DNA and total RNA were extracted using Qiagen kits per manufacturer’s protocol. Libraries were prepared after established protocols (TruSeq Stranded Total RNA with RiboZero Gold, Illumina),18,19 enriched with the xGEN Exome Research Panel version 1.0 (IDT) and sequenced on an Illumina HiSeq 4000.

### Circulating Tumor Sequencing

Circulating tumor DNA (ctDNA) sequencing was isolated using QIAamp Circulating Nucleic Acid Kit (Qiagen) per manufacturer’s protocol. An input of 300 ng was used to generate libraries for paired-end sequencing on a NextSeq instrument achieving median coverage of approximately ×500.20

### Somatic Mutation and Copy Number Variant Calling, Clonal Inference, and Mutational Signatures

These bioinformatics analyses were performed as previously described.18 Briefly, sequencing reads were aligned to human genome build 19 (hg19) using Burrows-Wheeler Aligner (bwa),21 deduplication with Picard (https://github.com/broadinstitute/picard), and base quality score recombination and realignment around insertion and deletion (indels) with GATK.22 Variants were called with VarScan223 and allele-specific copy number variations (CNVs) with FALCON.24 Clonal inference was performed using Canopy,25 and mutational signatures were inferred with deconstructSigs.26 Bradley-Terry modeling was used to estimate relative ordering of mutations in the phylogeny branches of autopsy patients with SCLC as previously described.18

### Significantly Mutated Gene Analysis

Significantly mutated gene (SMG) analysis was identified using MuSiC version 2.0,27 with default settings. Variants from all tumor samples were merged per patient by taking the highest variant allele fraction. The p value and false discovery rate (FDR) estimates are on the basis of three tests including Fisher's combined p value test, convolution test, and likelihood ratio test methods. For a specific gene, if FDR for at least two of these tests is less than or equal to maximum FDR, then it was called as a SMG. For all five SCLC research autopsy patients, the maximum FDR cutoff was set to 0.05.

### Driver Mutation Prediction

CHASM was used to identify statistically likely driver mutations.28 Missense variants were merged per patient and deduplicated before input to CHASM.

### Transcriptome Analysis

RNA-seq reads were aligned to hg19 using HISAT2,29 and fragments per kilobase per million reads and transcript reads per million (TPM) were calculated with StringTie30 as described in the Supplementary Methods.

### Single-Sample Gene Set Enrichment and Immune Signatures

Single-sample gene set enrichment (ssGSEA) version 2.0 was implemented with default settings in R.31 Gene sets assayed were obtained from PanCancer Immuno-Oncology 360 and Tumor Signaling 360 from NanoString. Immune signatures (ImSigs) were run using TPM values with default settings to identify enrichment of 10 ImSig.32 Linear mixed-effects models and t statistics were used to model ssGSEA and ImSig scores by allowing correlation among multiple samples within each patient. Tukey’s method was used for adjusting p values of multiple comparisons of enrichment scores from ssGSEA using NanoString gene sets and scores from ImSig.

### External SCLC RNA-Seq Data Sets

RNA-seq data of primary and relapsed SCLC and normal lung tissue were obtained from four previous publications.8,14,15,33 Of these four data sets, one had paired primary SCLC and normal lung samples,8 whereas the rest contained either exclusively tumor14,15 or normal samples.33 Non-SCLC RNA-seq data were downloaded from the The Cancer Genome Atlas data portal (https://portal.gdc.cancer.gov/). We randomly chose a subset of 60 and 49 tumor-normal paired samples of lung adenocarcinoma and squamous cell carcinoma, respectively, for ssGSEA and ImSig analyses.

### Molecular Subtyping of SCLC Samples

SCLC subtype classification was performed after a previously published method.34 Briefly, the samples were classified on the basis of defining thresholds (log2[fragments per kilobase per million + 1]) for transcription factors YAP1 (2.5) and POU2F3 (2.5). Samples were classified as ASCL1 or NEUROD1 subtype on the basis of relative expression of these genes, with minimum required difference of 1. The above-mentioned method was unable to classify all samples included in this study; therefore, remaining samples were assigned to SCLC subtypes on the basis of the gene (ASCL1, NEUROD1, YAP1, or POU2F3) with highest expression (TPM) value. When expression difference was less than twofold between ASCL1 and NEUROD1, samples were classified as dual-positive ASCL1/NEUROD1. ARG2 expression between subtypes was compared using two-tailed unpaired t tests.

### T Cell-Inflamed Gene Expression Profile Scores and Heatmaps

TPM values for genes in the housekeeping (n = 11) and predictor (n = 18) sets were converted into count-equivalent values, and gene expression profile (GEP) scores were computed as previously described35 and as detailed in the Supplementary Methods. Heatmaps of log-transformed TPM values of the 18 genes in the T cell-inflamed GEP were generated using Qlucore Omics Explorer (Qlucore AB) version 3.6.

### Results

Demographic and clinical information of the five patients with SCLC who underwent research autopsy is presented in Table 1. All had metastatic SCLC at diagnosis and received multiple lines of treatments, including first-line cisplatin or carboplatin and etoposide as autopsies were performed before the approval of atezolizumab and durvalumab. All patients responded to first-line chemotherapy but relapsed within 1 to 5 months of completing the last treatment cycle. Three patients received immunotherapy subsequently but were nonresponders. The time from diagnosis to death ranged from 12 to 25 months. Numerous metastatic tumors from four to five different organs were procured through autopsy of each patient. A total of 60 metastatic tumor samples were selected for whole-exome sequencing (WES) and 30 tumor samples for RNA-seq (Supplementary Data 1). Pretreatment samples were included for WES and RNA-seq when available. Matched normal lung tissue was available for RNA-seq from two autopsy patients (Supplementary Data 1).Table 1Demographics and Clinical Histories of Relapsed Small Cell Lung Cancer (SCLC) Patients Who Underwent Research AutopsySCLC 1SCLC 2SCLC 3SCLC 4SCLC 5GenderFemaleFemaleMaleMaleFemaleAge5954756264EthnicityCaucasianCaucasianCaucasianCaucasianCaucasianSmoking (pack years)6020604547Stage at diagnosisExtensiveExtensiveExtensiveExtensiveExtensiveRadiationPCIa and palliativePCIPCIPalliativePCI1stline therapyCisplatin & etoposide x6Carboplatin & etoposide x4Cisplatin & etoposide x6Carboplatin & etoposide x4Carboplatin & etoposide x4Best response to platinumPRbMild/mixed responsePRPRPRSubsequent therapiesreceivedTopotecanIrinotecanRova-TcPaclitaxelIrinotecanNivolumabImmunotherapydIrinotecanImmunotherapydIrinotecanRova-TcPaclitaxelGemcitabineTime from diagnosis to death23 months12 months25 months12 months22 months# Metastatic tumors (# organs) collected at autopsy25 (5)17 (5)14 (4)21 (5)16 (4)Note: All patients had extensive smoking history, metastatic disease at time of diagnosis, received standard of care platinum-doublet chemotherapy, radiation therapy, and had partial response except for one patient. All patients received more than one line of therapy, including experimental therapies on clinical trials as indicated in the footnote. Patients who received experimental therapies were non-responders.aProphylactic cranial irradiationbPartial responsecClinical trial NCT02674568: Study of Rovalpituzumab Tesirine (SC16LD6.5) for Third-line and Later Treatment of Subjects With Relapsed or Refractory Delta-Like Protein 3-Expressing Small Cell Lung Cancer (TRINITY)dClinical trial NCT02538666: An Investigational Immuno-therapy Study of Nivolumab, or Nivolumab in Combination with Ipilimumab, or Placebo in Patients with Extensive-Stage Disease Small Cell Lung Cancer (ED-SCLC) After Completion of Platinum-based Chemotherapy (CheckMate 451)

Demographics and Clinical Histories of Relapsed Small Cell Lung Cancer (SCLC) Patients Who Underwent Research Autopsy

Note: All patients had extensive smoking history, metastatic disease at time of diagnosis, received standard of care platinum-doublet chemotherapy, radiation therapy, and had partial response except for one patient. All patients received more than one line of therapy, including experimental therapies on clinical trials as indicated in the footnote. Patients who received experimental therapies were non-responders.

Prophylactic cranial irradiation

Partial response

Clinical trial NCT02674568: Study of Rovalpituzumab Tesirine (SC16LD6.5) for Third-line and Later Treatment of Subjects With Relapsed or Refractory Delta-Like Protein 3-Expressing Small Cell Lung Cancer (TRINITY)

Clinical trial NCT02538666: An Investigational Immuno-therapy Study of Nivolumab, or Nivolumab in Combination with Ipilimumab, or Placebo in Patients with Extensive-Stage Disease Small Cell Lung Cancer (ED-SCLC) After Completion of Platinum-based Chemotherapy (CheckMate 451)

WES revealed high tumor mutational burden (TMB) in our SCLC tumor samples, ranging from 5.7 to 29.8 mutations per megabase of genome (Supplementary Data 2–6). The percentage of private somatic variants, or those present in only one tumor sample, in patients 1 to 5 ranged from 32% to 56%, suggesting high intertumoral heterogeneity (Supplementary Fig. 1A and B). We assessed the identities of SMGs using MuSiC,27 finding TP53 as the top SMG mutated in 100% of the tumor samples (Fig. 1A). Additional SMGs included LRP1B, RYR2, and USH2A (Fig. 1A and Supplementary Data 7). We sought to determine whether these genes may be preferentially mutated in SCLC relative to other cancer types by assessing their alteration frequency in The Cancer Genome Atlas PanCancer Atlas studies (>10,000 samples) and 210 predominantly primary SCLC samples7, 8, 9,13 through cBioPortal for Cancer Genomics.36,37 This analysis revealed a high-alteration frequency of these genes in SCLC but also in other cancer types with high TMB (Supplementary Fig. 2A and B).Figure 1Identification of SMGs and CNVs in SCLC autopsy patients. (A) Oncoplot of SMGs identified in our SCLC cohort using MuSiC (FDR < 0.05). Multiple tumor samples per patient with SCLC were sequenced, resulting in exome data from 63 samples including three pretreatment biopsy samples (marked by ∗). Vertical bar graphs (top) reveal total number of Mut. per corresponding tumor sample below. Horizontal bar graphs and percentages (left) reveal mutational frequency of the corresponding gene (right). Type of somatic variant is defined by colored box key at the bottom, with black boxes indicating multiple variants detected in a specific gene in a given tumor sample. (B) Uncurated CNVs in SCLC autopsy patients detected by FALCON. Data from all tumor samples per patient were pooled into a composite CNV profile as illustrated for each patient. Gains were defined as allele number greater than 2.0 (e.g., SOX2) and loss less than 0.5 (e.g., APC). Red line, major allele. Blue line, minor allele. Green line, no change in one or both alleles. CNV, copy number variation; FDR, false discovery rate; Mut., mutation; SMG, significantly mutated gene. #, number.

Identification of SMGs and CNVs in SCLC autopsy patients. (A) Oncoplot of SMGs identified in our SCLC cohort using MuSiC (FDR < 0.05). Multiple tumor samples per patient with SCLC were sequenced, resulting in exome data from 63 samples including three pretreatment biopsy samples (marked by ∗). Vertical bar graphs (top) reveal total number of Mut. per corresponding tumor sample below. Horizontal bar graphs and percentages (left) reveal mutational frequency of the corresponding gene (right). Type of somatic variant is defined by colored box key at the bottom, with black boxes indicating multiple variants detected in a specific gene in a given tumor sample. (B) Uncurated CNVs in SCLC autopsy patients detected by FALCON. Data from all tumor samples per patient were pooled into a composite CNV profile as illustrated for each patient. Gains were defined as allele number greater than 2.0 (e.g., SOX2) and loss less than 0.5 (e.g., APC). Red line, major allele. Blue line, minor allele. Green line, no change in one or both alleles. CNV, copy number variation; FDR, false discovery rate; Mut., mutation; SMG, significantly mutated gene. #, number.

We detected an RB1 E204X nonsense mutation in a subset of tumor samples in patient 5 (Supplementary Data 6), and PTEN mutations in patients 1 and 3. In patient 1, PTEN C105F (a validated driver variant) was present in all metastatic tumor samples except for a residual primary right lung tumor and three brain metastases (Supplementary Data 2). In patient 3, PTEN Y46N was present in a single metastatic tumor sample (Supplementary Data 4).

We identified a high frequency of CNVs using FALCON24 in all SCLC tumor samples from the five autopsy patients (Fig. 1B, Supplementary Data 8). All patients had monoallelic loss, defined as allele copy less than 0.5, of chromosome 17p regions containing TP53. We also detected monoallelic loss of chromosome 13q regions containing RB1 in patients 2, 3, and 5. Notably, tumor samples from all five patients had monoallelic deletions of a region on chromosome 5q containing APC. Consistent with previous studies,8 we detected allele-specific gains, defined as allele copy greater than 2, of chromosome 3q regions containing SOX2 in patients 1, 3, and 5; and 8q regions containing MYC in patients 1, 3, and 4.

We used the tool CANOPY25 to integrate single-nucleotide variant (SNVs), indels, and curated CNVs (Supplementary Fig. 2C) to infer clonal diversity and architecture in advanced SCLC.18 Between 5 and 8 genetically distinct tumor cell clones were inferred to exist in each patient (Figs. 2A–D and 3A, Supplementary Data 9). TP53 and RB1 mutations were classified as truncal in all patients. Mutations in epigenetic modifiers such as CREBBP and HDAC2 were also truncal, along with APC (5q) deletion in a subset of patients. Subclonal alterations included PTEN deletion and mutations and MYC amplification. We used CHASM28 to identify predicted driver mutations in Wnt pathway genes such as XPO1 and AXIN1 (Supplementary Data 10). We used Bradley-Terry models18 to approximate the temporal occurrence of clonal and subclonal alterations and found that truncal alterations in TP53, RB1, CREBBP, and HDAC2 occurred early, consistent with the critical role of these tumor suppressors in SCLC biology (Supplementary Data 11). Finally, we used deconstructSigs26 to analyze mutational signatures and found truncal signature 4, which is associated with tobacco smoking.38Figure 2Inferred clonal phylogeny and heterogeneity in four SCLC autopsy patients. (A–D) (Left) Tumor samples procured from research autopsy as labeled on the human figure. Three large tumors from SCLC1 and SCLC3 were divided into multiple regions for sequencing. (A–D) (Middle) Phylogenetic trees depicting clonal evolution from a normal cell (gray solid circle) are as illustrated, with branch length corresponding to mutational burden. Numbers in parentheses after genes reveal predicted order of occurrence in a truncal branch (e.g., A, CREBBP, third mut. to occur of 273 truncal alterations). The COSMIC tobacco mut. Sig.4 was detected in the trunk of all five phylogenetic trees. Clonal and subclonal predicted driver mut., indicated by ∗, in Wnt signaling genes include CREBBP and TRRAP in (A) SCLC1, (B) FZD in SCLC2, DVL1, and (C) XPO1 in SCLC3, and (D) AXIN1 in SCLC4. (A–D) (Right) Percentage of different clones that compose each tumor sample are as revealed, demonstrating increased clonal heterogeneity. #T1, residual primary tumor in SCLC1; Adr., adrenal; Bx, pretreatment biopsy; del, deletion; Liv., liver; LN, lymph node; LOH, loss of heterozygosity; Lu, lung; mut., mutation; Sig., signature.

Inferred clonal phylogeny and heterogeneity in four SCLC autopsy patients. (A–D) (Left) Tumor samples procured from research autopsy as labeled on the human figure. Three large tumors from SCLC1 and SCLC3 were divided into multiple regions for sequencing. (A–D) (Middle) Phylogenetic trees depicting clonal evolution from a normal cell (gray solid circle) are as illustrated, with branch length corresponding to mutational burden. Numbers in parentheses after genes reveal predicted order of occurrence in a truncal branch (e.g., A, CREBBP, third mut. to occur of 273 truncal alterations). The COSMIC tobacco mut. Sig.4 was detected in the trunk of all five phylogenetic trees. Clonal and subclonal predicted driver mut., indicated by ∗, in Wnt signaling genes include CREBBP and TRRAP in (A) SCLC1, (B) FZD in SCLC2, DVL1, and (C) XPO1 in SCLC3, and (D) AXIN1 in SCLC4. (A–D) (Right) Percentage of different clones that compose each tumor sample are as revealed, demonstrating increased clonal heterogeneity. #T1, residual primary tumor in SCLC1; Adr., adrenal; Bx, pretreatment biopsy; del, deletion; Liv., liver; LN, lymph node; LOH, loss of heterozygosity; Lu, lung; mut., mutation; Sig., signature.

We next evaluated spatiotemporal clonal heterogeneity in advanced SCLC. In patient 1, we identified a tumor cell clone (clone 4) unique to brain metastases (T3–5) and the remnant primary tumor (∗T1) (Fig. 2A). In patient 2, the biopsy and posttreatment samples had similar clonal compositions (Fig. 2B). In patient 3, clone 5 in the biopsy sample decreased substantially in all but three (T7–9) posttreatment tumors corresponding to the brain and upper lung (Fig. 2C). Conversely, clone 6 in the biopsy sample substantially increased in a subset of posttreatment tumors, particularly in the lymph node samples T10 to 12. In patient 4, clone 3 harboring a BRCA2 G620E mutation decreased considerably in all posttreatment tumor samples (T1–12) (Fig. 2D), consistent with the well-characterized platinum-sensitizing effects of BRCA mutations. Furthermore, posttreatment tumors had increased proportions of clones 4 to 6 that contained predicted driver alterations by CHASM in the Wnt pathway (AXIN1 mutation, APC deletion), which has been associated with chemoresistance.14

Finally, in patient 5, we identified a tumor cell clone (clone 5) present at increased proportions in all six liver metastases (T6–11; Fig. 3A). In addition, ctDNA was isolated from the plasma of this patient shortly before death and subjected to WES (Supplementary Data 12). We determined high concordance (p < 10−5) between ubiquitous tumor variants and ctDNA variants, including SNVs (Fig. 3B, Supplementary Fig. 3A) and indels (Supplementary Fig. 3B and C). To infer the abundance of clones identified from autopsy in ctDNA, we used a maximum likelihood approach that revealed similar clonal composition between ctDNA (ctDNA, Fig. 3A) and liver metastases, thus suggesting this patient’s hepatic tumor burden preferentially contributed to ctDNA.Figure 3Concordance between ctD (ctDNA) and tumor profiling in a fifth SCLC autopsy patient. (A) Inferred clonal phylogeny and clonal composition of metastatic tumors from a fifth SCLC autopsy patient. Truncal predicted driver alterations include point mut. in RB1 and Wnt pathway gene PSMC1 and copy number losses of TP53 and APC. Tumor clonal heterogeneity was recapitulated in a ctD sample collected shortly before death of this patient. The ctD sample revealed a high proportion of clone 5, which was detected in increased proportions only in liver metastases T6 to T11. (B) (Left) Bar graph reveals concordant numbers of SNVs between tumor and ctD. (Right) Average VAF of Ubi. SNVs was compared with VAF of ctD variants. amp, amplification; ctD, circulating tumor DNA; del, deletion; mut., mutation; Sig., signature; SNV, single-nucleotide variant; Ubi., ubiquitous; VAF, variant allele frequency. #: number.

Concordance between ctD (ctDNA) and tumor profiling in a fifth SCLC autopsy patient. (A) Inferred clonal phylogeny and clonal composition of metastatic tumors from a fifth SCLC autopsy patient. Truncal predicted driver alterations include point mut. in RB1 and Wnt pathway gene PSMC1 and copy number losses of TP53 and APC. Tumor clonal heterogeneity was recapitulated in a ctD sample collected shortly before death of this patient. The ctD sample revealed a high proportion of clone 5, which was detected in increased proportions only in liver metastases T6 to T11. (B) (Left) Bar graph reveals concordant numbers of SNVs between tumor and ctD. (Right) Average VAF of Ubi. SNVs was compared with VAF of ctD variants. amp, amplification; ctD, circulating tumor DNA; del, deletion; mut., mutation; Sig., signature; SNV, single-nucleotide variant; Ubi., ubiquitous; VAF, variant allele frequency. #: number.

To further characterize advanced SCLC, we performed transcriptome sequencing on a subset of tumor samples from each SCLC autopsy patient (Supplementary Data 1). Given the limited number of matched lung normal and pretreatment/primary SCLC tumors in our data set, we incorporated additional publicly available SCLC RNA-seq data sets8,14,15,33 into all subsequent analyses. Two external SCLC data sets combined had 31 relapsed and three pretreatment samples,14,15 one data set contained 30 primary SCLC and 25 matched normal samples,8 and one data set contained eight normal lung samples.33 We first performed ssGSEA31 using gene sets from the NanoString Tumor Signaling 360 panel (Supplementary Data 13). Intriguingly, most SCLC samples had decreased enrichment of pathways related to antitumor immune response: “avoiding immune destruction” and “tumor-promoting inflammation” compared with normal samples (Supplementary Fig. 4A).

To delineate how the immune tumor microenvironment (TME) may be altered in SCLC, we next performed ssGSEA using the NanoString PanCancer Immuno-Oncology 360 panel. This revealed decreased enrichment (p < 0.001) of processes annotated with adaptive antitumor immune function and immuno-metabolism in SCLC tumors relative to normal tissue (Fig. 4A, Supplementary Data 13). We also used ImSig32 to interrogate immune cell subsets in the SCLC TME, which revealed low gene expression signatures of T cells and innate immune cells in primary and relapsed SCLC tumors (Fig. 4B, Supplementary Data 13). Finally, we repeated ssGSEA and ImSig on our own research autopsy data set, in addition to the above-mentioned analyses of pooled data sets, and confirmed these results (Supplementary Fig. 4B and C).Figure 4Transcriptome analyses reveal decreased expression of genes involved in antitumor immune responses in SCLC. (A) Box plots of ESs from ssGSEA analysis of gene sets derived from NanoString PanCancer IO 360 panel. Statistically significant differences in ES were detected in pairwise comparisons between normal-primary and normal-relapse. ∗, adjusted p < 0.0001. (B) Box plots of ImSig scores for different immune cell types in the adaptive and innate immune systems. Statistically significant differences in ES were detected in pairwise comparisons between normal-primary and normal-relapse. ∗, adjusted p < 0.0001. ES, enrichment score; ImSig, Immune signature; IO, Immune-Oncology; Mac., macrophage; Mono., monocyte; Neutro., neutrophil; NK, natural killer; Pretx, pretreatment; sig., signature; ssGSEA, single-sample gene set enrichment.

Transcriptome analyses reveal decreased expression of genes involved in antitumor immune responses in SCLC. (A) Box plots of ESs from ssGSEA analysis of gene sets derived from NanoString PanCancer IO 360 panel. Statistically significant differences in ES were detected in pairwise comparisons between normal-primary and normal-relapse. ∗, adjusted p < 0.0001. (B) Box plots of ImSig scores for different immune cell types in the adaptive and innate immune systems. Statistically significant differences in ES were detected in pairwise comparisons between normal-primary and normal-relapse. ∗, adjusted p < 0.0001. ES, enrichment score; ImSig, Immune signature; IO, Immune-Oncology; Mac., macrophage; Mono., monocyte; Neutro., neutrophil; NK, natural killer; Pretx, pretreatment; sig., signature; ssGSEA, single-sample gene set enrichment.

T cells are key mediators of the adaptive antitumor immune response and targets of immune checkpoint inhibitors (ICIs). Given that ssGSEA and ImSig results indicated decreased T cell presence in the SCLC TME, we further interrogated expression of 18 genes in the analytically validated T cell-inflamed GEP, which was developed as a predictor of clinical benefit to ICI in multiple cancer types.39 Responders to ICI were reported to have pretreatment baseline tumors with higher GEP scores (hot), whereas nonresponders had tumors with lower baseline GEP scores (cold).35,39 Hierarchical clustering (Fig. 5A, p = 0.05) and calculation of GEP scores in the pooled (Fig. 5B) and autopsy only (Supplementary Fig. 5A) data sets revealed that most primary and relapsed SCLC are cold relative to either matched tissue normal or non-SCLC subtypes of adenocarcinoma and squamous cell carcinoma (Supplementary Fig. 5B, Supplementary Data 14). Only five SCLC tumors were inflamed and had GEP scores higher than the median score of −0.09 in normal samples (Supplementary Data 15).Figure 5T cell-inflamed GEP evaluation reveals different immune phenotypes in neuroendocrine versus non-neuroendocrine SCLC. (A) Heatmap of log-transformed expression values (TPM) of genes representing the analytically validated T cell-inflamed GEP. Hierarchical clustering was performed using Qlucore Omics Explorer v3.6. Most primary and relapsed SCLC tumor samples revealed low GEP expression relative to normal lung tissue. Sample type and data source indicated as labeled. (B) Violin plot of GEP scores for normal, primary, and relapsed SCLC samples. Higher scores indicate “hot” tumors, whereas lower scores indicate “cold” tumors. Two-tailed unpaired t test ∗p < 0.001; ∗∗p < 0.01. (C) Violin plot of TPM values of selected T cell-inflamed GEP and checkpoint genes in normal and SCLC tumor samples. (D) Heatmap of log-transformed expression values (TPM) of T cell-inflamed GEP genes in SCLC tumor samples annotated by subtype and data source as indicated. (E) Violin plot of GEP scores in different subtypes of SCLC and normal lung. Each subtype was compared against normal using two-tailed unpaired t test ∗, p < 0.01. (F) Violin plot of IDO1/2 and ARG1/2 expression in normal, primary, and relapsed SCLC. Primary/pretx or relapsed samples were compared against normal using two-tailed unpaired t test ∗p < 0.001; ∗∗p < 0.005. (G) Violin plot of ARG2 expression in SCLC tumor samples annotated by subtype. Each subtype was compared against normal using two-tailed unpaired t test. ∗p < 0.001; ∗∗p < 0.005. ARG, arginase; GEP, gene expression profile; IDO, indoleamine 2,3-dioxygenase; Pretx, pretreatment; TPM, transcript reads per million.

T cell-inflamed GEP evaluation reveals different immune phenotypes in neuroendocrine versus non-neuroendocrine SCLC. (A) Heatmap of log-transformed expression values (TPM) of genes representing the analytically validated T cell-inflamed GEP. Hierarchical clustering was performed using Qlucore Omics Explorer v3.6. Most primary and relapsed SCLC tumor samples revealed low GEP expression relative to normal lung tissue. Sample type and data source indicated as labeled. (B) Violin plot of GEP scores for normal, primary, and relapsed SCLC samples. Higher scores indicate “hot” tumors, whereas lower scores indicate “cold” tumors. Two-tailed unpaired t test ∗p < 0.001; ∗∗p < 0.01. (C) Violin plot of TPM values of selected T cell-inflamed GEP and checkpoint genes in normal and SCLC tumor samples. (D) Heatmap of log-transformed expression values (TPM) of T cell-inflamed GEP genes in SCLC tumor samples annotated by subtype and data source as indicated. (E) Violin plot of GEP scores in different subtypes of SCLC and normal lung. Each subtype was compared against normal using two-tailed unpaired t test ∗, p < 0.01. (F) Violin plot of IDO1/2 and ARG1/2 expression in normal, primary, and relapsed SCLC. Primary/pretx or relapsed samples were compared against normal using two-tailed unpaired t test ∗p < 0.001; ∗∗p < 0.005. (G) Violin plot of ARG2 expression in SCLC tumor samples annotated by subtype. Each subtype was compared against normal using two-tailed unpaired t test. ∗p < 0.001; ∗∗p < 0.005. ARG, arginase; GEP, gene expression profile; IDO, indoleamine 2,3-dioxygenase; Pretx, pretreatment; TPM, transcript reads per million.

As atezolizumab and durvalumab are both programmed death-ligand 1 (PD-L1) monoclonal antibodies approved in combination with chemotherapy in advanced SCLC, we evaluated the expression of PD-L1 (CD274). CD274 expression was low in most primary and relapsed SCLC samples (Fig. 5C, Supplementary Fig. 5C), consistent with previous analyses revealing that only a few patients (∼20%) with SCLC had tumors with PD-L1 protein expression greater than 1%.40 In contrast to CD274, the expression of immune checkpoint inhibitory ligands CD276 (B7-H3) and CD200 was increased in most SCLC tumor samples (Fig. 5C, Supplementary Fig. 5C). Consistent with ImSig revealing a lack of T cell signature in their TME, SCLC tumor samples had low expression levels of CTLA4, TIGIT, and other immune checkpoint molecules found on T cells (Fig. 5C, Supplementary Fig. 5C). Fially, interrogation of 50 SCLC cell lines from the Cancer Cell Line Encyclopedia recapitulated these gene expression patterns of CD276, CD200, and CD274 (Supplementary Fig. 5D, Supplementary Data 14).

We further evaluated whether the four SCLC subtypes, as defined by the expression of ASCL1, NEUROD1, YAP1, and POU2F3 using a previously described method10 (Supplementary Fig. 6) and on the basis of highest gene expression evaluation (Supplementary Data 16), may have different immune phenotypes. As expected, most (n = 56 of 94) SCLC samples were ASCL1-high, whereas tumors with high expression of ASCL1/NEUROD1 (n = 11), NEUROD1 (n = 13), YAP1 (n = 6), and POU2F3 (n = 8) were the minority. Hierarchical clustering revealed that the non-neuroendocrine SCLC subtypes, YAP1 and POU2F3, had more tumors with elevated T cell-inflamed GEP whereas the classic and variant neuroendocrine subtypes were associated with predominantly low GEP (Fig. 5D). When GEP scores were calculated for each subtype,35
YAP1 and POU2F3 tumors had GEP scores closer to that of normal lung tissue (Fig. 5E). ImSig analysis further revealed that YAP1 tumors, although limited in sample size, are particularly associated with increased T cell and innate immune cell signatures (Supplementary Fig. 7).

Finally, to extend our analysis of suppressed immune function in SCLC, we interrogated expression of immunometabolism genes (Fig. 4A) encoding indoleamine 2,3-dioxygenase and arginase (ARG)1/2 enzymes (Fig. 5F), whose metabolic function have been reported to inhibit effector T cell and natural killer cell activity.41, 42, 43 Of the four genes, only ARG2 expression was significantly increased in primary and relapsed SCLC tumors (Fig. 5F) and cell lines (Supplementary Fig. 5D). We then evaluated ARG2 expression on the basis of SCLC subtypes (Fig. 5G) and in NSCLC (Supplementary Fig. 8A and B). This analysis revealed high ARG2 expression in the neuroendocrine SCLC subtypes (ASCL1, ASCL1/NEUROD1, and NEUROD1) and low ARG2 expression in the non-neuroendocrine subtypes (YAP1 and POU2F3) (Fig. 5G) and low ARG2 expression in NSCLC (Supplementary Fig. 8B). Taken together, these results support the need to further investigate ARG2 as a potential negative regulator of immune TME in the most common neuroendocrine SCLC subtypes.

### Genomic Alterations in Relapsed SCLC

WES revealed high tumor mutational burden (TMB) in our SCLC tumor samples, ranging from 5.7 to 29.8 mutations per megabase of genome (Supplementary Data 2–6). The percentage of private somatic variants, or those present in only one tumor sample, in patients 1 to 5 ranged from 32% to 56%, suggesting high intertumoral heterogeneity (Supplementary Fig. 1A and B). We assessed the identities of SMGs using MuSiC,27 finding TP53 as the top SMG mutated in 100% of the tumor samples (Fig. 1A). Additional SMGs included LRP1B, RYR2, and USH2A (Fig. 1A and Supplementary Data 7). We sought to determine whether these genes may be preferentially mutated in SCLC relative to other cancer types by assessing their alteration frequency in The Cancer Genome Atlas PanCancer Atlas studies (>10,000 samples) and 210 predominantly primary SCLC samples7, 8, 9,13 through cBioPortal for Cancer Genomics.36,37 This analysis revealed a high-alteration frequency of these genes in SCLC but also in other cancer types with high TMB (Supplementary Fig. 2A and B).Figure 1Identification of SMGs and CNVs in SCLC autopsy patients. (A) Oncoplot of SMGs identified in our SCLC cohort using MuSiC (FDR < 0.05). Multiple tumor samples per patient with SCLC were sequenced, resulting in exome data from 63 samples including three pretreatment biopsy samples (marked by ∗). Vertical bar graphs (top) reveal total number of Mut. per corresponding tumor sample below. Horizontal bar graphs and percentages (left) reveal mutational frequency of the corresponding gene (right). Type of somatic variant is defined by colored box key at the bottom, with black boxes indicating multiple variants detected in a specific gene in a given tumor sample. (B) Uncurated CNVs in SCLC autopsy patients detected by FALCON. Data from all tumor samples per patient were pooled into a composite CNV profile as illustrated for each patient. Gains were defined as allele number greater than 2.0 (e.g., SOX2) and loss less than 0.5 (e.g., APC). Red line, major allele. Blue line, minor allele. Green line, no change in one or both alleles. CNV, copy number variation; FDR, false discovery rate; Mut., mutation; SMG, significantly mutated gene. #, number.

Identification of SMGs and CNVs in SCLC autopsy patients. (A) Oncoplot of SMGs identified in our SCLC cohort using MuSiC (FDR < 0.05). Multiple tumor samples per patient with SCLC were sequenced, resulting in exome data from 63 samples including three pretreatment biopsy samples (marked by ∗). Vertical bar graphs (top) reveal total number of Mut. per corresponding tumor sample below. Horizontal bar graphs and percentages (left) reveal mutational frequency of the corresponding gene (right). Type of somatic variant is defined by colored box key at the bottom, with black boxes indicating multiple variants detected in a specific gene in a given tumor sample. (B) Uncurated CNVs in SCLC autopsy patients detected by FALCON. Data from all tumor samples per patient were pooled into a composite CNV profile as illustrated for each patient. Gains were defined as allele number greater than 2.0 (e.g., SOX2) and loss less than 0.5 (e.g., APC). Red line, major allele. Blue line, minor allele. Green line, no change in one or both alleles. CNV, copy number variation; FDR, false discovery rate; Mut., mutation; SMG, significantly mutated gene. #, number.

We detected an RB1 E204X nonsense mutation in a subset of tumor samples in patient 5 (Supplementary Data 6), and PTEN mutations in patients 1 and 3. In patient 1, PTEN C105F (a validated driver variant) was present in all metastatic tumor samples except for a residual primary right lung tumor and three brain metastases (Supplementary Data 2). In patient 3, PTEN Y46N was present in a single metastatic tumor sample (Supplementary Data 4).

### CNVs in Relapsed SCLC

We identified a high frequency of CNVs using FALCON24 in all SCLC tumor samples from the five autopsy patients (Fig. 1B, Supplementary Data 8). All patients had monoallelic loss, defined as allele copy less than 0.5, of chromosome 17p regions containing TP53. We also detected monoallelic loss of chromosome 13q regions containing RB1 in patients 2, 3, and 5. Notably, tumor samples from all five patients had monoallelic deletions of a region on chromosome 5q containing APC. Consistent with previous studies,8 we detected allele-specific gains, defined as allele copy greater than 2, of chromosome 3q regions containing SOX2 in patients 1, 3, and 5; and 8q regions containing MYC in patients 1, 3, and 4.

### Clonal Heterogeneity and Evolution in Relapsed SCLC

We used the tool CANOPY25 to integrate single-nucleotide variant (SNVs), indels, and curated CNVs (Supplementary Fig. 2C) to infer clonal diversity and architecture in advanced SCLC.18 Between 5 and 8 genetically distinct tumor cell clones were inferred to exist in each patient (Figs. 2A–D and 3A, Supplementary Data 9). TP53 and RB1 mutations were classified as truncal in all patients. Mutations in epigenetic modifiers such as CREBBP and HDAC2 were also truncal, along with APC (5q) deletion in a subset of patients. Subclonal alterations included PTEN deletion and mutations and MYC amplification. We used CHASM28 to identify predicted driver mutations in Wnt pathway genes such as XPO1 and AXIN1 (Supplementary Data 10). We used Bradley-Terry models18 to approximate the temporal occurrence of clonal and subclonal alterations and found that truncal alterations in TP53, RB1, CREBBP, and HDAC2 occurred early, consistent with the critical role of these tumor suppressors in SCLC biology (Supplementary Data 11). Finally, we used deconstructSigs26 to analyze mutational signatures and found truncal signature 4, which is associated with tobacco smoking.38Figure 2Inferred clonal phylogeny and heterogeneity in four SCLC autopsy patients. (A–D) (Left) Tumor samples procured from research autopsy as labeled on the human figure. Three large tumors from SCLC1 and SCLC3 were divided into multiple regions for sequencing. (A–D) (Middle) Phylogenetic trees depicting clonal evolution from a normal cell (gray solid circle) are as illustrated, with branch length corresponding to mutational burden. Numbers in parentheses after genes reveal predicted order of occurrence in a truncal branch (e.g., A, CREBBP, third mut. to occur of 273 truncal alterations). The COSMIC tobacco mut. Sig.4 was detected in the trunk of all five phylogenetic trees. Clonal and subclonal predicted driver mut., indicated by ∗, in Wnt signaling genes include CREBBP and TRRAP in (A) SCLC1, (B) FZD in SCLC2, DVL1, and (C) XPO1 in SCLC3, and (D) AXIN1 in SCLC4. (A–D) (Right) Percentage of different clones that compose each tumor sample are as revealed, demonstrating increased clonal heterogeneity. #T1, residual primary tumor in SCLC1; Adr., adrenal; Bx, pretreatment biopsy; del, deletion; Liv., liver; LN, lymph node; LOH, loss of heterozygosity; Lu, lung; mut., mutation; Sig., signature.

Inferred clonal phylogeny and heterogeneity in four SCLC autopsy patients. (A–D) (Left) Tumor samples procured from research autopsy as labeled on the human figure. Three large tumors from SCLC1 and SCLC3 were divided into multiple regions for sequencing. (A–D) (Middle) Phylogenetic trees depicting clonal evolution from a normal cell (gray solid circle) are as illustrated, with branch length corresponding to mutational burden. Numbers in parentheses after genes reveal predicted order of occurrence in a truncal branch (e.g., A, CREBBP, third mut. to occur of 273 truncal alterations). The COSMIC tobacco mut. Sig.4 was detected in the trunk of all five phylogenetic trees. Clonal and subclonal predicted driver mut., indicated by ∗, in Wnt signaling genes include CREBBP and TRRAP in (A) SCLC1, (B) FZD in SCLC2, DVL1, and (C) XPO1 in SCLC3, and (D) AXIN1 in SCLC4. (A–D) (Right) Percentage of different clones that compose each tumor sample are as revealed, demonstrating increased clonal heterogeneity. #T1, residual primary tumor in SCLC1; Adr., adrenal; Bx, pretreatment biopsy; del, deletion; Liv., liver; LN, lymph node; LOH, loss of heterozygosity; Lu, lung; mut., mutation; Sig., signature.

We next evaluated spatiotemporal clonal heterogeneity in advanced SCLC. In patient 1, we identified a tumor cell clone (clone 4) unique to brain metastases (T3–5) and the remnant primary tumor (∗T1) (Fig. 2A). In patient 2, the biopsy and posttreatment samples had similar clonal compositions (Fig. 2B). In patient 3, clone 5 in the biopsy sample decreased substantially in all but three (T7–9) posttreatment tumors corresponding to the brain and upper lung (Fig. 2C). Conversely, clone 6 in the biopsy sample substantially increased in a subset of posttreatment tumors, particularly in the lymph node samples T10 to 12. In patient 4, clone 3 harboring a BRCA2 G620E mutation decreased considerably in all posttreatment tumor samples (T1–12) (Fig. 2D), consistent with the well-characterized platinum-sensitizing effects of BRCA mutations. Furthermore, posttreatment tumors had increased proportions of clones 4 to 6 that contained predicted driver alterations by CHASM in the Wnt pathway (AXIN1 mutation, APC deletion), which has been associated with chemoresistance.14

Finally, in patient 5, we identified a tumor cell clone (clone 5) present at increased proportions in all six liver metastases (T6–11; Fig. 3A). In addition, ctDNA was isolated from the plasma of this patient shortly before death and subjected to WES (Supplementary Data 12). We determined high concordance (p < 10−5) between ubiquitous tumor variants and ctDNA variants, including SNVs (Fig. 3B, Supplementary Fig. 3A) and indels (Supplementary Fig. 3B and C). To infer the abundance of clones identified from autopsy in ctDNA, we used a maximum likelihood approach that revealed similar clonal composition between ctDNA (ctDNA, Fig. 3A) and liver metastases, thus suggesting this patient’s hepatic tumor burden preferentially contributed to ctDNA.Figure 3Concordance between ctD (ctDNA) and tumor profiling in a fifth SCLC autopsy patient. (A) Inferred clonal phylogeny and clonal composition of metastatic tumors from a fifth SCLC autopsy patient. Truncal predicted driver alterations include point mut. in RB1 and Wnt pathway gene PSMC1 and copy number losses of TP53 and APC. Tumor clonal heterogeneity was recapitulated in a ctD sample collected shortly before death of this patient. The ctD sample revealed a high proportion of clone 5, which was detected in increased proportions only in liver metastases T6 to T11. (B) (Left) Bar graph reveals concordant numbers of SNVs between tumor and ctD. (Right) Average VAF of Ubi. SNVs was compared with VAF of ctD variants. amp, amplification; ctD, circulating tumor DNA; del, deletion; mut., mutation; Sig., signature; SNV, single-nucleotide variant; Ubi., ubiquitous; VAF, variant allele frequency. #: number.

Concordance between ctD (ctDNA) and tumor profiling in a fifth SCLC autopsy patient. (A) Inferred clonal phylogeny and clonal composition of metastatic tumors from a fifth SCLC autopsy patient. Truncal predicted driver alterations include point mut. in RB1 and Wnt pathway gene PSMC1 and copy number losses of TP53 and APC. Tumor clonal heterogeneity was recapitulated in a ctD sample collected shortly before death of this patient. The ctD sample revealed a high proportion of clone 5, which was detected in increased proportions only in liver metastases T6 to T11. (B) (Left) Bar graph reveals concordant numbers of SNVs between tumor and ctD. (Right) Average VAF of Ubi. SNVs was compared with VAF of ctD variants. amp, amplification; ctD, circulating tumor DNA; del, deletion; mut., mutation; Sig., signature; SNV, single-nucleotide variant; Ubi., ubiquitous; VAF, variant allele frequency. #: number.

### RNA-Seq Reveals Decreased Antitumor Immunity in Advanced SCLC

To further characterize advanced SCLC, we performed transcriptome sequencing on a subset of tumor samples from each SCLC autopsy patient (Supplementary Data 1). Given the limited number of matched lung normal and pretreatment/primary SCLC tumors in our data set, we incorporated additional publicly available SCLC RNA-seq data sets8,14,15,33 into all subsequent analyses. Two external SCLC data sets combined had 31 relapsed and three pretreatment samples,14,15 one data set contained 30 primary SCLC and 25 matched normal samples,8 and one data set contained eight normal lung samples.33 We first performed ssGSEA31 using gene sets from the NanoString Tumor Signaling 360 panel (Supplementary Data 13). Intriguingly, most SCLC samples had decreased enrichment of pathways related to antitumor immune response: “avoiding immune destruction” and “tumor-promoting inflammation” compared with normal samples (Supplementary Fig. 4A).

To delineate how the immune tumor microenvironment (TME) may be altered in SCLC, we next performed ssGSEA using the NanoString PanCancer Immuno-Oncology 360 panel. This revealed decreased enrichment (p < 0.001) of processes annotated with adaptive antitumor immune function and immuno-metabolism in SCLC tumors relative to normal tissue (Fig. 4A, Supplementary Data 13). We also used ImSig32 to interrogate immune cell subsets in the SCLC TME, which revealed low gene expression signatures of T cells and innate immune cells in primary and relapsed SCLC tumors (Fig. 4B, Supplementary Data 13). Finally, we repeated ssGSEA and ImSig on our own research autopsy data set, in addition to the above-mentioned analyses of pooled data sets, and confirmed these results (Supplementary Fig. 4B and C).Figure 4Transcriptome analyses reveal decreased expression of genes involved in antitumor immune responses in SCLC. (A) Box plots of ESs from ssGSEA analysis of gene sets derived from NanoString PanCancer IO 360 panel. Statistically significant differences in ES were detected in pairwise comparisons between normal-primary and normal-relapse. ∗, adjusted p < 0.0001. (B) Box plots of ImSig scores for different immune cell types in the adaptive and innate immune systems. Statistically significant differences in ES were detected in pairwise comparisons between normal-primary and normal-relapse. ∗, adjusted p < 0.0001. ES, enrichment score; ImSig, Immune signature; IO, Immune-Oncology; Mac., macrophage; Mono., monocyte; Neutro., neutrophil; NK, natural killer; Pretx, pretreatment; sig., signature; ssGSEA, single-sample gene set enrichment.

Transcriptome analyses reveal decreased expression of genes involved in antitumor immune responses in SCLC. (A) Box plots of ESs from ssGSEA analysis of gene sets derived from NanoString PanCancer IO 360 panel. Statistically significant differences in ES were detected in pairwise comparisons between normal-primary and normal-relapse. ∗, adjusted p < 0.0001. (B) Box plots of ImSig scores for different immune cell types in the adaptive and innate immune systems. Statistically significant differences in ES were detected in pairwise comparisons between normal-primary and normal-relapse. ∗, adjusted p < 0.0001. ES, enrichment score; ImSig, Immune signature; IO, Immune-Oncology; Mac., macrophage; Mono., monocyte; Neutro., neutrophil; NK, natural killer; Pretx, pretreatment; sig., signature; ssGSEA, single-sample gene set enrichment.

T cells are key mediators of the adaptive antitumor immune response and targets of immune checkpoint inhibitors (ICIs). Given that ssGSEA and ImSig results indicated decreased T cell presence in the SCLC TME, we further interrogated expression of 18 genes in the analytically validated T cell-inflamed GEP, which was developed as a predictor of clinical benefit to ICI in multiple cancer types.39 Responders to ICI were reported to have pretreatment baseline tumors with higher GEP scores (hot), whereas nonresponders had tumors with lower baseline GEP scores (cold).35,39 Hierarchical clustering (Fig. 5A, p = 0.05) and calculation of GEP scores in the pooled (Fig. 5B) and autopsy only (Supplementary Fig. 5A) data sets revealed that most primary and relapsed SCLC are cold relative to either matched tissue normal or non-SCLC subtypes of adenocarcinoma and squamous cell carcinoma (Supplementary Fig. 5B, Supplementary Data 14). Only five SCLC tumors were inflamed and had GEP scores higher than the median score of −0.09 in normal samples (Supplementary Data 15).Figure 5T cell-inflamed GEP evaluation reveals different immune phenotypes in neuroendocrine versus non-neuroendocrine SCLC. (A) Heatmap of log-transformed expression values (TPM) of genes representing the analytically validated T cell-inflamed GEP. Hierarchical clustering was performed using Qlucore Omics Explorer v3.6. Most primary and relapsed SCLC tumor samples revealed low GEP expression relative to normal lung tissue. Sample type and data source indicated as labeled. (B) Violin plot of GEP scores for normal, primary, and relapsed SCLC samples. Higher scores indicate “hot” tumors, whereas lower scores indicate “cold” tumors. Two-tailed unpaired t test ∗p < 0.001; ∗∗p < 0.01. (C) Violin plot of TPM values of selected T cell-inflamed GEP and checkpoint genes in normal and SCLC tumor samples. (D) Heatmap of log-transformed expression values (TPM) of T cell-inflamed GEP genes in SCLC tumor samples annotated by subtype and data source as indicated. (E) Violin plot of GEP scores in different subtypes of SCLC and normal lung. Each subtype was compared against normal using two-tailed unpaired t test ∗, p < 0.01. (F) Violin plot of IDO1/2 and ARG1/2 expression in normal, primary, and relapsed SCLC. Primary/pretx or relapsed samples were compared against normal using two-tailed unpaired t test ∗p < 0.001; ∗∗p < 0.005. (G) Violin plot of ARG2 expression in SCLC tumor samples annotated by subtype. Each subtype was compared against normal using two-tailed unpaired t test. ∗p < 0.001; ∗∗p < 0.005. ARG, arginase; GEP, gene expression profile; IDO, indoleamine 2,3-dioxygenase; Pretx, pretreatment; TPM, transcript reads per million.

T cell-inflamed GEP evaluation reveals different immune phenotypes in neuroendocrine versus non-neuroendocrine SCLC. (A) Heatmap of log-transformed expression values (TPM) of genes representing the analytically validated T cell-inflamed GEP. Hierarchical clustering was performed using Qlucore Omics Explorer v3.6. Most primary and relapsed SCLC tumor samples revealed low GEP expression relative to normal lung tissue. Sample type and data source indicated as labeled. (B) Violin plot of GEP scores for normal, primary, and relapsed SCLC samples. Higher scores indicate “hot” tumors, whereas lower scores indicate “cold” tumors. Two-tailed unpaired t test ∗p < 0.001; ∗∗p < 0.01. (C) Violin plot of TPM values of selected T cell-inflamed GEP and checkpoint genes in normal and SCLC tumor samples. (D) Heatmap of log-transformed expression values (TPM) of T cell-inflamed GEP genes in SCLC tumor samples annotated by subtype and data source as indicated. (E) Violin plot of GEP scores in different subtypes of SCLC and normal lung. Each subtype was compared against normal using two-tailed unpaired t test ∗, p < 0.01. (F) Violin plot of IDO1/2 and ARG1/2 expression in normal, primary, and relapsed SCLC. Primary/pretx or relapsed samples were compared against normal using two-tailed unpaired t test ∗p < 0.001; ∗∗p < 0.005. (G) Violin plot of ARG2 expression in SCLC tumor samples annotated by subtype. Each subtype was compared against normal using two-tailed unpaired t test. ∗p < 0.001; ∗∗p < 0.005. ARG, arginase; GEP, gene expression profile; IDO, indoleamine 2,3-dioxygenase; Pretx, pretreatment; TPM, transcript reads per million.

As atezolizumab and durvalumab are both programmed death-ligand 1 (PD-L1) monoclonal antibodies approved in combination with chemotherapy in advanced SCLC, we evaluated the expression of PD-L1 (CD274). CD274 expression was low in most primary and relapsed SCLC samples (Fig. 5C, Supplementary Fig. 5C), consistent with previous analyses revealing that only a few patients (∼20%) with SCLC had tumors with PD-L1 protein expression greater than 1%.40 In contrast to CD274, the expression of immune checkpoint inhibitory ligands CD276 (B7-H3) and CD200 was increased in most SCLC tumor samples (Fig. 5C, Supplementary Fig. 5C). Consistent with ImSig revealing a lack of T cell signature in their TME, SCLC tumor samples had low expression levels of CTLA4, TIGIT, and other immune checkpoint molecules found on T cells (Fig. 5C, Supplementary Fig. 5C). Fially, interrogation of 50 SCLC cell lines from the Cancer Cell Line Encyclopedia recapitulated these gene expression patterns of CD276, CD200, and CD274 (Supplementary Fig. 5D, Supplementary Data 14).

We further evaluated whether the four SCLC subtypes, as defined by the expression of ASCL1, NEUROD1, YAP1, and POU2F3 using a previously described method10 (Supplementary Fig. 6) and on the basis of highest gene expression evaluation (Supplementary Data 16), may have different immune phenotypes. As expected, most (n = 56 of 94) SCLC samples were ASCL1-high, whereas tumors with high expression of ASCL1/NEUROD1 (n = 11), NEUROD1 (n = 13), YAP1 (n = 6), and POU2F3 (n = 8) were the minority. Hierarchical clustering revealed that the non-neuroendocrine SCLC subtypes, YAP1 and POU2F3, had more tumors with elevated T cell-inflamed GEP whereas the classic and variant neuroendocrine subtypes were associated with predominantly low GEP (Fig. 5D). When GEP scores were calculated for each subtype,35
YAP1 and POU2F3 tumors had GEP scores closer to that of normal lung tissue (Fig. 5E). ImSig analysis further revealed that YAP1 tumors, although limited in sample size, are particularly associated with increased T cell and innate immune cell signatures (Supplementary Fig. 7).

Finally, to extend our analysis of suppressed immune function in SCLC, we interrogated expression of immunometabolism genes (Fig. 4A) encoding indoleamine 2,3-dioxygenase and arginase (ARG)1/2 enzymes (Fig. 5F), whose metabolic function have been reported to inhibit effector T cell and natural killer cell activity.41, 42, 43 Of the four genes, only ARG2 expression was significantly increased in primary and relapsed SCLC tumors (Fig. 5F) and cell lines (Supplementary Fig. 5D). We then evaluated ARG2 expression on the basis of SCLC subtypes (Fig. 5G) and in NSCLC (Supplementary Fig. 8A and B). This analysis revealed high ARG2 expression in the neuroendocrine SCLC subtypes (ASCL1, ASCL1/NEUROD1, and NEUROD1) and low ARG2 expression in the non-neuroendocrine subtypes (YAP1 and POU2F3) (Fig. 5G) and low ARG2 expression in NSCLC (Supplementary Fig. 8B). Taken together, these results support the need to further investigate ARG2 as a potential negative regulator of immune TME in the most common neuroendocrine SCLC subtypes.

### Discussion

In this study, we leveraged rapid research autopsy to perform genomic and transcriptomic characterization of advanced SCLC. Our results revealed substantial clonal heterogeneity in relapsed SCLC, arising through branched evolution. We identified numerous subclonal alterations likely underlying treatment sensitivity, thus explaining the expansion or reduction of specific clones in tumors at different metastatic sites in each patient. Although limited in sample size, our results revealed that certain clones are enriched in metastatic sites such as the brain and liver, with the liver as the main ctDNA contributor in one patient. The ability to detect tissue-specific clones may have certain clinical use. For example, the central nervous system (CNS) is a frequent site of metastasis in patients with SCLC. The presence of brain-specific subclonal alterations in the ctDNA may help predict which patients are at highest risk for CNS disease and therefore be considered for prophylactic cranial irradiation or novel therapies to prevent CNS disease. Our study demonstrates the use of autopsy in studying subclone preference for specific metastatic niches, which will require additional autopsies of patients with SCLC and other solid tumors.

In addition to identifying TP53 as the top SMG and inferring TP53 mutations as truncal drivers in all autopsy patients, we also identified multiple mutations in LRP1B, which has putative tumor-suppressor functions.44, 45, 46 These truncal alterations raise the possibility that LRPB1-mutant tumor cells may acquire an early fitness advantage over wild-type cells, which is further supported by an increased frequency of LRP1B mutations in primary SCLC tumor samples (Supplementary Fig. 1A). Thus, we propose that functional validation of variants in this gene and others (e.g., RYR2, USH2A) in appropriate model systems may begin to unravel their roles in the biology of SCLC and potentially other cancers.

Alterations in the Wnt signaling pathway have been linked to acquired chemoresistance in relapsed SCLC.14 In a cohort of 30 relapsed SCLC samples, loss of heterozygosity of either APC or CDH8 was detected in 23 of 30 samples, whereas mutations in either gene were detected in 10 of 30 samples. In our autopsy cohort, we identified copy number loss of APC as truncal or very early subclonal events in patients 1 to 3. We found truncal (PSMC1) and multiple subclonal (TRRAP, FZD2, and XPO1) predicted driver mutations in other Wnt signaling pathway genes. For instance, in patient 3, the autopsy tumor samples were enriched for clones C6 to C8 containing an XPO1 mutation. In patient 4, mutation of another Wnt signaling gene, AXIN1, was assigned to a clone of tumor cells (C4) exclusively detected in posttreatment autopsy tumor samples. Therefore, our results are consistent with and support the established association between Wnt pathway and early chemoresistance in advanced SCLC.

Analysis of ctDNA in patients with advanced SCLC represents a valuable approach to tracking tissue-specific clones or emergence of resistant clones.47 Although most studies have compared ctDNA with either a single primary or metastatic tumor biopsy, rapid autopsy enables the comparison of variants detected in ctDNA against variants in multiple metastatic tumors. In our study, we performed WES of ctDNA from patient 5 and detected nearly all ubiquitous tumor variants from autopsy that likely represented most of this patient’s truncal variants (concordance 97%–100% for SNVs and indels). Not surprisingly, for variants that were classified as shared or private at autopsy (e.g., subclonal variants), the metastatic tumor-ctDNA concordance rates were much lower. Therefore, our results reveal effective detection of truncal variants in ctDNA, but considerable limitations for detecting subclonal variants. These data and future autopsy studies will provide important contributions toward refining the limits of ctDNA detection for clinical applications, including the evaluation of minimal residual disease and cancer recurrence.

Within our autopsy SCLC cohort, we performed transcriptome sequencing to evaluate whether mutations in SMGs may correlate with altered expression, thereby representing potential driver genes. However, this initial analysis did not reveal a significant correlation. In addition, consistent with previous studies revealing a paucity of gene fusions as drivers in SCLC,48 we did not detect any putative fusions that were recurrent in more than one tumor sample within our cohort. Recognizing that these results may be attributed to our limited cohort size, we incorporated external primary and relapsed SCLC RNA-seq data sets in subsequent analyses, which ultimately revealed suppressed immune TME in the neuroendocrine SCLC subtypes (ASCL1, NEUROD1, and ASCL1/NEUROD1).

Given that only approximately 20% of patients with SCLC have tumors expressing PD-L1 (>1% by immunohistochemistry),40 improved characterization of the SCLC immune TME remains an unmet need, particularly given recent regulatory approvals of frontline PD-L1 blockade therapy. Our results confirmed low CD274 (PD-L1) expression in most SCLC tumors and cell lines, supporting the above-mentioned clinical observations. Additional ssGSEA and ImSig analyses corroborated decreased adaptive antitumor immune function in primary and relapsed SCLC. Although we lacked outcomes data to correlate with T cell-inflamed GEP scores, which predicts clinical benefit to programmed cell death protein 1/PD-L1 blockade,39 hierarchical clustering revealed a noninflamed phenotype in most primary and advanced SCLC. Interestingly, T cell-inflamed GEP expression was high, or close to that of normal lung samples, in non-neuroendocrine SCLC subtypes (YAP1 and POU2F3) (Fig. 5D), which had low ARG2 expression (Fig. 5E). In contrast, GEP expression was low in the neuroendocrine SCLC subtypes (ASCL1, ASCL1/NEUROD1, and NEUROD1), which had high ARG2 expression. Given these observations, we hypothesize that increased ARG2 expression and function may represent a cell-autonomous oncogenic metabolic adaptation enabling suppression of adaptive antitumor immunity in the SCLC TME independent of PD-L1 expression. These results will need to be validated in larger, independent data sets but suggest that subtyping patients with SCLC before systemic treatment should aid in the identification of patients most likely to benefit from chemo-ICI. Finally, it was recently reported that the non-neuroendocrine SCLC subtypes were more likely to be admixed SCLC and NSCLC,11 which in addition to low ARG2 expression may further explain the higher GEP scores and inflamed phenotype in these tumors given the responsiveness of NSCLC to ICI.

Continuing the analysis of immune TME, in addition to decreased CD274 expression, we reported decreased expression of CTLA4, LAG3, TIM3, and other well-characterized immune checkpoint proteins in advanced SCLC. Conversely, we detected increased expression of alternate checkpoint molecules, CD276 (B7-H3) and CD200. CD276 is a member of the B7 family, which also includes PD-L1 (B7-H1). Consistent with our finding of increased gene expression, CD276 protein expression was recently detected in 64.9% of a SCLC cohort with 90 patients.49 The same study reported PD-L1 expression at a much lower rate of 7.3%.49 CD276 is expressed at low levels in normal tissues but when aberrantly expressed on various tumor cell types contributes to T cell inhibition, tumor cell immune evasion, and is associated with poor prognosis.50 Therefore, targeting CD276 is being explored as an immune-stimulatory strategy in cancer.51 CD200 is a checkpoint molecule that leads to suppression of secretion of proinflammatory cytokines, including interleukin-2 and I interferon-γ.52 Consistent with increased CD200 expression in the SCLC samples we analyzed, CD200 protein was expressed in two other SCLC cohorts.53,54 Furthermore, inhibiting the CD200 signaling axis has been reported to stimulate antigen-specific immune response against glioblastoma multiforme.52 Therefore, these results support further exploration of targeting CD276 or CD200 in preclinical SCLC models to determine whether inhibiting these checkpoint pathways may yield viable therapeutic strategies for advanced SCLC.

As drug development efforts focus on programmed cell death protein 1/PD-L1 and other immune regulatory checkpoints, characterization of both the immune microenvironment and tumor heterogeneity will be critical to differentiate responders from nonresponders. It will also be important to define the relationship between tumor heterogeneity and neoantigen formation. For example, in NSCLC, cytotoxic tumor-infiltrating T cells preferentially develop against clonal neoantigens (derived from clonal or truncal mutations) in patients with durable clinical benefit to ICI.55 The high smoking-associated TMB of SCLC suggests that there may be a high number of clonal neoantigens serving as therapeutic targets. Therefore, a combination of autopsy and ctDNA samples will be critical resources to help characterize the prevalence of clonal neoantigens and optimize their detection from liquid biopsy. This will enable the development of vaccine or T cell-based therapeutic strategies in SCLC and other solid cancers.

In summary, we have partnered with patients for rapid research autopsy to study relapsed SCLC and performed extensive analyses of clonal heterogeneity, subclonal architecture, and the immune microenvironment. Our results suggest a potential explanation for why SCLC, a cancer with high TMB and thus potential tumor neoantigens, exhibits lower than expected response rates to ICI compared with other solid tumors with similar median TMB. Future studies using single-cell sequencing strategies will shed light on unanswered questions, including which cell types are the source of ARG2 expression and whether CD200/CD276 are coexpressed. Metabolome profiling could identify additional metabolic vulnerabilities in SCLC that may be cotargeted to enhance ICI response. Innovative immunotherapeutic approaches targeting alternate checkpoints, potentially combined with isoform-specific ARG inhibition, may ultimately result in more robust clinical activity against advanced SCLC.



# SUPPLEMENTAL FILE 1: main.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: PMC8474405

[File available at: automated_output/BRCA2/20251125_175029/pmc_fulltext/34590014_supplements/PMC8474405]

