# MAIN TEXT

## Splicing predictions, minigene analyses, and ACMG‐AMP clinical classification of 42 germline 
PALB2
 splice‐site variants

### Abstract

Abstract
PALB2 loss‐of‐function variants confer high risk of developing breast cancer. Here we present a systematic functional analysis of PALB2 splice‐site variants detected in approximately 113,000 women in the large‐scale sequencing project Breast Cancer After Diagnostic Gene Sequencing (BRIDGES; https://bridges-research.eu/). Eighty‐two PALB2 variants at the intron‐exon boundaries were analyzed with MaxEntScan. Forty‐two variants were selected for the subsequent splicing functional assays. For this purpose, three splicing reporter minigenes comprising exons 1–12 were constructed. The 42 potential spliceogenic variants were introduced into the minigenes by site‐directed mutagenesis and assayed in MCF‐7/MDA‐MB‐231 cells. Splicing anomalies were observed in 35 variants, 23 of which showed no traces or minimal amounts of the expected full‐length transcripts of each minigene. More than 30 different variant‐induced transcripts were characterized, 23 of which were predicted to truncate the PALB2 protein. The pathogenicity of all variants was interpreted according to an in‐house adaptation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology (ACMG‐AMP) variant classification scheme. Up to 23 variants were classified as pathogenic/likely pathogenic. Remarkably, three ±1,2 variants (c.49‐2A>T, c.108+2T>C, and c.211+1G>A) were classified as variants of unknown significance, as they produced significant amounts of either in‐frame transcripts of unknown impact on the PALB2 protein function or the minigene full‐length transcripts. In conclusion, we have significantly contributed to the ongoing effort of identifying spliceogenic variants in the clinically relevant PALB2 cancer susceptibility gene. Moreover, we suggest some approaches to classify the findings in accordance with the ACMG‐AMP rationale. © 2021 The Authors. The Journal of Pathology published by John Wiley & Sons, Ltd on behalf of The Pathological Society of Great Britain and Ireland.

### Introduction

PALB2 [MIM #610355] was originally identified as a BRCA2‐interacting protein that plays a key role in DNA repair by homologous recombination [1, 2]. Monoallelic germline loss‐of‐function variants in PALB2 confer high risk of developing breast cancer (BC) [3, 4, 5]. Moreover, germline biallelic loss‐of‐function variants of PALB2 cause Fanconi anemia subtype FA‐N and predispose to childhood malignancies [6]. Pathogenic PALB2 variants have been identified in 0.6–3.9% of families with a history of BC, while the absolute BC risk by 70 years of age for PALB2 carriers was estimated in the 33–58% range, and more recently in the 44–68% range, depending on the familial aggregation of BC [4, 5]. Two large population‐based studies have estimated the BC relative risk in carriers of PALB2 protein truncating variants to be 3.83 (2.68–5.63) and 5.02 (3.73–6.76), respectively [7, 8]. Moreover, the association of PALB2 truncating variants with estrogen receptor negative and triple negative BC is particularly strong (odds ratios = 7.35 and 10.36, respectively) [8]. PALB2 loss‐of‐function variants are also associated with pancreatic cancer, ovarian cancer (OC), and male BC [5, 9, 10, 11, 12]. The association of PALB2 with gastric and colorectal cancer is currently disputed [5, 13, 14].

There are several critical steps for gene expression, which are frequent targets of loss‐of‐function variants [15, 16]. Splicing is a pivotal RNA processing stage whereby introns from eukaryotic genes are removed and exons are sequentially joined. Most human protein‐coding genes (94%) contain introns and produce multiple mRNA isoforms via alternative splicing. This process is finely controlled by small nuclear ribonucleoproteins, protein factors, and a broad array of cis‐acting elements that are responsible for the splicing efficiency [16]. There are fundamental preserved sequences that are recognized by the splicing machinery: the 5' (donor/GT) and the 3' (acceptor/AG) splice sites (5'ss and 3'ss, respectively), the polypyrimidine tract, and the branch point. These sequences are possible targets for spliceogenic variants that can produce anomalous transcripts and modify disease risk [17]. Actually, splicing disruption is a frequent deleterious mechanism in genetic diseases [18]. The identification and understanding of the underlying causative mechanisms are crucial in clinical practice [19] with the aim of improving genetic diagnosis, prevention strategies, and therapy.

We had previously shown that spliceogenic variants make a significant contribution to the overall pool of germ‐line loss of function alleles in the BC/OC genes RAD51C [20], RAD51D [21], and BRCA2 [22, 23, 24]. The present work was carried out in the context of the Breast Cancer After Diagnostic Gene Sequencing (BRIDGES) project (https://bridges-research.eu/), where 34 known or suspected BC/OC genes were sequenced in 60,466 BC cases and 53,461 controls [8]. Our purpose was to experimentally characterize the PALB2 spliceogenic variants detected in BRIDGES subjects. Here, we used three ad hoc developed splicing reporter minigenes to test 42 PALB2 unique variants with a high prior probability of being spliceogenic, assigning a final American College of Medical Genetics and Genomics and the Association for Molecular Pathology (ACMG‐AMP) clinical classification to all of them.

### Materials and methods

Ethical approval for this study was obtained from the Ethics Committee of the Spanish National Research Council‐CSIC (28/05/2018).

We identified in BRIDGES carriers (cases and/or controls) a total of 82 unique variants at PALB2 intron/exon boundaries [8]: 3'ss: intron/exon [IVS‐10_IVS‐1/2nt]; 5'ss: exon/intron [2nt/IVS+1_IVS+10]. Variants, transcripts, and predicted protein products were described according to the Human Genome Variation Society guidelines (https://varnomen.hgvs.org/), using the Ensembl reference transcript ID ENSG00000083093 (Genbank NM_024675.4). We also annotated transcripts according to a former shortened description [25].

We selected 42 of 82 variants at the intron‐exon boundaries for minigene analysis based on: (1) splice‐site disruption at the ±1,2 (AG/GT) positions; (2) important MaxEntScan (MES) score changes (≥15%) [26, 27, 28]; (3) creation of de novo alternative splice sites (cut‐off ≥3.0); (4) regardless of MES predictions, variants at other conserved positions of the acceptor (Y11NCAG|G) and donor (MAG|GTRAGT) consensus sequences, such as pyrimidine to purine changes or deletions at the polypyrimidine tract, substitutions of a conserved nucleotide at the intronic positions −3C, +3R, +4A, +5G, +6T, as well as the first (G) and the last three nucleotides of the exon (M, A, G). Patient RNA was not available for any of the 42 selected variants.

The PALB2 gene is composed of 13 exons, 12 of which were cloned into the pSAD v9.0 vector (patent P201231427‐CSIC) [22, 23]. Exon 13 was excluded given that BRIDGES did not report any potential splice‐site variant in it. Exons 1–12 were cloned into three different minigenes: mgPALB2_ex1‐3, mgPALB2_ex4‐6, and mgPALB2_ex5‐12 (Figure 1, supplementary material, Figure S1). The construction of these minigenes is detailed in Supplementary materials and methods. Basically, the inserts were generated either by PCR with Phusion High Fidelity polymerase (Fisher Scientific, Waltham, MA, USA) and the primers shown in supplementary material, Table S1, or by gene synthesis (Genewiz, South Plainfield, NJ, USA). Then, fragments were cloned into the pSAD splicing vector by two methods, either by classical restriction enzyme digestion and ligation or by overlapping extension PCR [29]. All the insert sequences with their structures are shown in supplementary material, Figure S1 and the final minigenes are outlined in Figure 1. The wild type (wt) minigenes were used as templates to generate 42 DNA variants (supplementary material, Table S1) with the QuikChange Lightning kit (Agilent, Santa Clara, CA, USA). All constructs were confirmed by sequencing (Macrogen, Madrid, Spain).

Schematic representation of the wt PALB2 minigenes. Exons are indicated by boxes; green elbow arrows indicate the expected splicing reactions in eukaryotic cells and black arrows locate specific vector RT‐PCR primers. (A) Minigene with PALB2 exons 1–3 (mgPALB2_ex1‐3). (B) Minigene with PALB2 exons 4–6 (mgPALB2_ex4‐6). (C) Minigene with PALB2 exons 5–12 (mgPALB2_ex5‐12).

Approximately 2 × 105 MCF‐7 (human breast adenocarcinoma cell line) cells were transfected with 1 μg minigene using 2 μl lipofectamine LTX (Life Technologies, Carlsbad, CA, USA), as previously described [20, 21]. To check the reproducibility of the splicing outcomes, MDA‐MB‐231 (triple‐negative BC cell line) cells were transfected with the wt and mutant minigenes carrying the following variants: c.47A>G, c.48+1G>A, c.48+2T>C, c.48+4C>T, c.2749‐1G>T, c.2834+3A>G, c.3201+3_3201+4insTG, and c.3350+4A>G. RNA was purified by means of the Genematrix Universal RNA Purification Kit (EURx, Gdansk, Poland), with on‐column DNAse I (EURx) digestion. Reverse transcription was carried out using 400 ng RNA and the RevertAid First Strand cDNA Synthesis Kit (Life Technologies), using the vector‐specific primer RTPSPL3‐RV (5'‐TGAGGAGTGAATTGGTCGAA‐3'). Then, 40 ng cDNA was amplified with platinum‐Taq DNA polymerase (Life Technologies) and the primers SD6‐PSPL3_RT‐FW (5'‐TCACCTGGACAACCTCAAAG‐3') and RTpSAD‐RV (patent P201231427). Samples were denatured at 94 °C for 2 min, followed by 35 cycles of 94 °C/30 s, 60 °C/30 s, and 72 °C (1 min/kb), and a final extension step at 72 °C for 5 min. RT‐PCR products were sequenced by Macrogen. The expected sizes of the minigene full‐length (mgFL) transcripts were 366‐nt (mgPALB2_ex1‐3), 2,556‐nt (mgPALB2_ex 4‐6), and 1,847‐nt (mgPALB2_ex 5‐12).

To estimate the relative abundance of all transcripts, semi‐quantitative fluorescent RT‐PCRs (26 cycles) were performed with three different primers pairs: PSPL3_RT‐FW and FAM‐RTpSAD‐RV for mgPALB2_ex1‐3 (size: 366‐nt), PSPL3_RT‐FW and FAM‐RTPB2_EX6‐RV (5'‐ GTTGCCTGGGTTTATGCTATC‐3') (size: 2,435‐nt) or RTPB2_EX4‐FW (5'‐CACAAATATCAGCACGAAAA‐3') and FAM‐RTPB2_EX6‐RV (size: 918‐nt) for mgPALB2_ex4‐6; and RTPB2_EX6‐FW (5'‐ GATAGCATAAACCCAGGCA‐3') and FAM‐RTpSAD‐RV for mgPALB2_ex5‐12 (size: 906‐nt) [22]. FAM‐labeled products with ROX‐500 or LIZ‐1200 size standards were run by Macrogen and analyzed with the Peak Scanner software V1.0 (Life Technologies). Alternatively, the quantification of variant c.1684+4A>G of mgPALB2_ex4‐6 was carried out by densitometric analysis of agarose gels using ImageJ software [30], because the mgFL transcript size with exon 4 (1,473‐nt) is out of range of the LIZ‐1200 size standard. Three independent experiments of each variant were carried out to calculate the average relative proportions of each transcript and the corresponding standard deviations.

We classified all variants according to a recently proposed ACMG‐AMP point system Bayesian framework (Supplementary materials and methods, and supplementary material, Table S2, Figures S2 and S3A–C) [31, 32, 33]; mgPALB2 read‐outs have been incorporated into the classification system as PS3/BS3 codes with variable strength depending on the actual outcome. As most tested variants produce two or more different transcripts, we proceeded as follows: (1) we assigned a specific PS3/BS3 code strength to each individual transcript and (2) depending on the relative contribution of pathogenic and benign codes to the overall expression, we assigned an overall PS3/BS3 code strength to each variant. To assess its contribution to the final classification, we classified with and without incorporating mgPALB2 data into the ACMG‐AMP scheme. To assist in the classification process, we developed a PALB2 adaptation of the PVS1 decision tree proposed by the ClinGen sequence variant interpretation (SVI) working group [34]. We considered that some pathogenic (PS2, PM1, PM6, PP2, PP4, PP5) and benign (BS2, BP1, BP3, BP5, BP6) codes are not applicable to the classification of PALB2 variants. Splicing predictive codes PVS1 (variable strength) and PP3/BP4 were assigned according to SpliceAI predictions (https://spliceailookup.broadinstitute.org/) [35]. We selected SpliceAI for ACMG‐AMP classification because: (1) we have previously used MES to select many variants under investigation and (2) SpliceAI scores probabilities associated with specific splicing outcomes, a feature not provided by MES, but critical to assign specific PVS1 strengths.

### Ethics approval

Ethical approval for this study was obtained from the Ethics Committee of the Spanish National Research Council‐CSIC (28/05/2018).

### Variant and transcript annotations

We identified in BRIDGES carriers (cases and/or controls) a total of 82 unique variants at PALB2 intron/exon boundaries [8]: 3'ss: intron/exon [IVS‐10_IVS‐1/2nt]; 5'ss: exon/intron [2nt/IVS+1_IVS+10]. Variants, transcripts, and predicted protein products were described according to the Human Genome Variation Society guidelines (https://varnomen.hgvs.org/), using the Ensembl reference transcript ID ENSG00000083093 (Genbank NM_024675.4). We also annotated transcripts according to a former shortened description [25].

### Bioinformatics selection of 
PALB2
 variants

We selected 42 of 82 variants at the intron‐exon boundaries for minigene analysis based on: (1) splice‐site disruption at the ±1,2 (AG/GT) positions; (2) important MaxEntScan (MES) score changes (≥15%) [26, 27, 28]; (3) creation of de novo alternative splice sites (cut‐off ≥3.0); (4) regardless of MES predictions, variants at other conserved positions of the acceptor (Y11NCAG|G) and donor (MAG|GTRAGT) consensus sequences, such as pyrimidine to purine changes or deletions at the polypyrimidine tract, substitutions of a conserved nucleotide at the intronic positions −3C, +3R, +4A, +5G, +6T, as well as the first (G) and the last three nucleotides of the exon (M, A, G). Patient RNA was not available for any of the 42 selected variants.

### Minigene construction and mutagenesis

The PALB2 gene is composed of 13 exons, 12 of which were cloned into the pSAD v9.0 vector (patent P201231427‐CSIC) [22, 23]. Exon 13 was excluded given that BRIDGES did not report any potential splice‐site variant in it. Exons 1–12 were cloned into three different minigenes: mgPALB2_ex1‐3, mgPALB2_ex4‐6, and mgPALB2_ex5‐12 (Figure 1, supplementary material, Figure S1). The construction of these minigenes is detailed in Supplementary materials and methods. Basically, the inserts were generated either by PCR with Phusion High Fidelity polymerase (Fisher Scientific, Waltham, MA, USA) and the primers shown in supplementary material, Table S1, or by gene synthesis (Genewiz, South Plainfield, NJ, USA). Then, fragments were cloned into the pSAD splicing vector by two methods, either by classical restriction enzyme digestion and ligation or by overlapping extension PCR [29]. All the insert sequences with their structures are shown in supplementary material, Figure S1 and the final minigenes are outlined in Figure 1. The wild type (wt) minigenes were used as templates to generate 42 DNA variants (supplementary material, Table S1) with the QuikChange Lightning kit (Agilent, Santa Clara, CA, USA). All constructs were confirmed by sequencing (Macrogen, Madrid, Spain).

Schematic representation of the wt PALB2 minigenes. Exons are indicated by boxes; green elbow arrows indicate the expected splicing reactions in eukaryotic cells and black arrows locate specific vector RT‐PCR primers. (A) Minigene with PALB2 exons 1–3 (mgPALB2_ex1‐3). (B) Minigene with PALB2 exons 4–6 (mgPALB2_ex4‐6). (C) Minigene with PALB2 exons 5–12 (mgPALB2_ex5‐12).

### Splicing functional assays

Approximately 2 × 105 MCF‐7 (human breast adenocarcinoma cell line) cells were transfected with 1 μg minigene using 2 μl lipofectamine LTX (Life Technologies, Carlsbad, CA, USA), as previously described [20, 21]. To check the reproducibility of the splicing outcomes, MDA‐MB‐231 (triple‐negative BC cell line) cells were transfected with the wt and mutant minigenes carrying the following variants: c.47A>G, c.48+1G>A, c.48+2T>C, c.48+4C>T, c.2749‐1G>T, c.2834+3A>G, c.3201+3_3201+4insTG, and c.3350+4A>G. RNA was purified by means of the Genematrix Universal RNA Purification Kit (EURx, Gdansk, Poland), with on‐column DNAse I (EURx) digestion. Reverse transcription was carried out using 400 ng RNA and the RevertAid First Strand cDNA Synthesis Kit (Life Technologies), using the vector‐specific primer RTPSPL3‐RV (5'‐TGAGGAGTGAATTGGTCGAA‐3'). Then, 40 ng cDNA was amplified with platinum‐Taq DNA polymerase (Life Technologies) and the primers SD6‐PSPL3_RT‐FW (5'‐TCACCTGGACAACCTCAAAG‐3') and RTpSAD‐RV (patent P201231427). Samples were denatured at 94 °C for 2 min, followed by 35 cycles of 94 °C/30 s, 60 °C/30 s, and 72 °C (1 min/kb), and a final extension step at 72 °C for 5 min. RT‐PCR products were sequenced by Macrogen. The expected sizes of the minigene full‐length (mgFL) transcripts were 366‐nt (mgPALB2_ex1‐3), 2,556‐nt (mgPALB2_ex 4‐6), and 1,847‐nt (mgPALB2_ex 5‐12).

To estimate the relative abundance of all transcripts, semi‐quantitative fluorescent RT‐PCRs (26 cycles) were performed with three different primers pairs: PSPL3_RT‐FW and FAM‐RTpSAD‐RV for mgPALB2_ex1‐3 (size: 366‐nt), PSPL3_RT‐FW and FAM‐RTPB2_EX6‐RV (5'‐ GTTGCCTGGGTTTATGCTATC‐3') (size: 2,435‐nt) or RTPB2_EX4‐FW (5'‐CACAAATATCAGCACGAAAA‐3') and FAM‐RTPB2_EX6‐RV (size: 918‐nt) for mgPALB2_ex4‐6; and RTPB2_EX6‐FW (5'‐ GATAGCATAAACCCAGGCA‐3') and FAM‐RTpSAD‐RV for mgPALB2_ex5‐12 (size: 906‐nt) [22]. FAM‐labeled products with ROX‐500 or LIZ‐1200 size standards were run by Macrogen and analyzed with the Peak Scanner software V1.0 (Life Technologies). Alternatively, the quantification of variant c.1684+4A>G of mgPALB2_ex4‐6 was carried out by densitometric analysis of agarose gels using ImageJ software [30], because the mgFL transcript size with exon 4 (1,473‐nt) is out of range of the LIZ‐1200 size standard. Three independent experiments of each variant were carried out to calculate the average relative proportions of each transcript and the corresponding standard deviations.

### ACMG‐AMP clinical classification of 
PALB2
 genetic variants

We classified all variants according to a recently proposed ACMG‐AMP point system Bayesian framework (Supplementary materials and methods, and supplementary material, Table S2, Figures S2 and S3A–C) [31, 32, 33]; mgPALB2 read‐outs have been incorporated into the classification system as PS3/BS3 codes with variable strength depending on the actual outcome. As most tested variants produce two or more different transcripts, we proceeded as follows: (1) we assigned a specific PS3/BS3 code strength to each individual transcript and (2) depending on the relative contribution of pathogenic and benign codes to the overall expression, we assigned an overall PS3/BS3 code strength to each variant. To assess its contribution to the final classification, we classified with and without incorporating mgPALB2 data into the ACMG‐AMP scheme. To assist in the classification process, we developed a PALB2 adaptation of the PVS1 decision tree proposed by the ClinGen sequence variant interpretation (SVI) working group [34]. We considered that some pathogenic (PS2, PM1, PM6, PP2, PP4, PP5) and benign (BS2, BP1, BP3, BP5, BP6) codes are not applicable to the classification of PALB2 variants. Splicing predictive codes PVS1 (variable strength) and PP3/BP4 were assigned according to SpliceAI predictions (https://spliceailookup.broadinstitute.org/) [35]. We selected SpliceAI for ACMG‐AMP classification because: (1) we have previously used MES to select many variants under investigation and (2) SpliceAI scores probabilities associated with specific splicing outcomes, a feature not provided by MES, but critical to assign specific PVS1 strengths.

### Results

Eighty‐two PALB2 variants were analyzed with MES (supplementary material, Table S3). Forty‐four of them were predicted to have a potential damaging effect on splicing. Variants c.2748+1G>A and c.2834+2T>C were excluded from the functional study because their effects on splicing were expected to be similar to selected variants c.2748+1G>T and c.2834+1G>A, respectively. A total of 11 and 30 variants impaired the 3'ss and the 5'ss, respectively, while one variant was predicted to generate a de novo 5'ss (c.48+7G>C). Seven splice‐site disrupting variants (c.49‐2A>T, c.109‐2A>G, c.2587‐2A>G, c.2749‐1G>T, c.2997‐1G>A, c.3348C>T, c.3350+4A>G) were also estimated to create a de novo splice site. Variants c.2748+4A>T, c.2834+6T>C, c.3113+3A>G, and c.3201+6T>A were chosen despite their weak MES score changes (<15%) because they affect highly conserved positions at the 5'ss (consensus sequence EXON/intron: MAG/gtragt).

The three wt minigene constructs (mgPALB2_ex1‐3, mgPALB2_ex4‐6, and mgPALB2_ex5‐12) were validated in MCF‐7 cells, where they produced the expected mgFL transcripts: [V1‐PALB2_ex1‐3‐V2, 366‐nt], [V1‐PALB2_ex4_6‐V2, 2,556‐nt], and [V1‐PALB2_ex5_12‐V2, 1,847‐nt], respectively.

The 42 candidate variants were introduced into the corresponding minigene by site‐directed mutagenesis and functionally assayed in MCF‐7 cells. Thirty‐five variants (83.3%) impaired splicing, 23 of which showed a complete lack or minimal amounts (c.48G>A, 0.9%; c.3113+5G>C, 4.9%) of the mgFL transcripts (Table 1, Figures 2, 3, supplementary material, Figure S4). Overall, the 35 spliceogenic variants each produced one to five anomalous transcripts. Eight variants (see Materials and methods) were also examined in MDA‐MB‐231 cells, where they replicated the splicing outcomes detected in MCF‐7 cells.

Splicing outcomes of PALB2 variants.

HGVS, Human Genome Variation Society.

Bold font: no traces or <5% of the mgFL transcript.

[−] site disruption; [+] new site. [↓] reduction of MES score.

Δ, loss of exonic sequences; ▼ inclusion of intronic sequences; E (exon), p (acceptor shift), q (donor shift). When necessary, the exact number of nucleotides inserted or deleted is indicated. For example, transcript ▼(E12q4) denotes the use of an alternative donor site that is located 4 nucleotides downstream of exon 12, causing the addition of 4‐nt to the mature mRNA. ▼(I1mg) refers to the retention of minigene intron 1 that is a shortened version (416 bp) of the genomic PALB2 intron (2,980 bp).

Three uncharacterized transcripts were found: 888‐nt [4.2 ± 0.9%] (c.2997‐1G>A), 592‐nt [8.9 ± 0.3%], and 710‐nt [7.4 ± 0.2%] (both from c.2748+2dup).

Splicing functional assays of variants in minigenes. (A) mgPALB2_ex1‐3 and (B) mgPALB2_ex4‐6. The maps of variants are shown on the left. At the bottom, fluorescent fragment analysis of transcripts generated by the wt and mutant minigenes. FAM‐labeled products (blue peaks) were run with LIZ‐1200 (orange peaks) as size standard. FL, minigene full‐length transcript. Transcript ▼(E1q416) of the variants c.48G>A and c.48+1G>A is not shown because it is out of the size range displayed in minigene mgPALB2_ex1‐3 electropherograms.

Splicing functional assays of variants in minigenes mgPALB2_ex5‐12. The map of variants is shown above. Fluorescent fragment analysis of transcripts generated by the wt and mutant minigenes. FAM‐labeled products (blue peaks) were run with LIZ‐1200 (orange peaks) as size standard. FL, minigene full‐length transcript. Transcript ∆(E9_10) is not shown because it is out of the size range displayed in minigene mgPALB2_ex5‐12.

Eighteen variants affected the classical ±1,2 positions of the 5' or 3'ss. All these variants, except for c.108+2T>C, induced complete aberrant patterns. Variant c.108+2T>C, as well as c.48+2T>C, transforms a GT splice‐donor into an atypical GC donor [36]. Remarkably, although c.48+2T>C only displayed anomalous transcripts, c.108+2T>C produced 85.5% of the mgFL transcript. In addition, other 17 spliceogenic variants affected other positions of the 5'ss, including the three last exonic nucleotides and intronic nucleotides +3, +4, +5, and +6 (Table 1).

Fluorescent fragment analysis revealed the existence of 39 different transcripts, including six mgFL transcripts (five of them carrying rare variants) (Table 1; supplementary material, Table S4 and Figure S4). Alternative site usage was the most frequent spliceogenic mechanism, explaining up to 22 different aberrant transcripts. Two of them, ∆(E10q41) and ∆(E10q65), derived from the activation of atypical GC donors. The second most prevalent mechanism was exon skipping, which was detected in 10 different transcripts. Finally, one transcript contained a full intron 5 retention (▼(I5)). The identity of three transcripts (592, 710, and 888‐nt long) could not be characterized.

Up to 16 of the abovementioned isoforms had been previously characterized as naturally occurring events [25], including: ▼(E1q9), ∆(E1q17), ∆(E2p6), ∆(E2), ∆(E4), ∆(E5p139), ∆(E7p10), ∆(E7), ∆(E8), ∆(E9), ∆(E10), ∆(E9_10), ∆(E10p2), ∆(E10q31), ∆(E11), and ∆(E12) (i.e. some spliceogenic variants apparently upregulate already expressed transcripts).

Twenty‐four transcripts (including one mgFL transcript carrying a nonsense variant) were predicted to introduce premature termination codons (PTC). Fifteen transcripts kept the reading frame: ▼(E1q9), ∆(E2p6), ∆(E2), ▼(E3q48), ∆(E4), ∆(E7), ∆(E8p15), ∆(E9), ∆(E10), ∆(E9_10) (supplementary material, Table S4), four mgFL transcripts carrying missense or synonymous variants, and the wt mgFL transcripts.

mgPALB2 read‐outs showed a remarkable agreement with previous experimental data in carriers (limited to five variants) and with SpliceAI predictions (supplementary material, Table S5). Taken together, the data support the robustness of the mgPALB2 assay and the accuracy of SpliceAI in predicting the actual outcome of spliceogenic variants.

We evaluated the clinical relevance of each variant per the ACMG‐AMP classification scheme using a Bayesian approach and incorporating the mgPALB2 splicing results into the evaluation of all other existing evidence for the variant (see Materials and methods). As shown in Table 2, this resulted in classifying 23 PALB2 variants as pathogenic/likely pathogenic and, equally relevant, eight variants as likely benign (mostly, intronic variants that do not affect splicing). Eleven variants were classified as variants of uncertain significance (VUSs) (Table 2; Supplementary materials and methods; supplementary material, Tables S6, S7; Figures [Link], [Link]). Only 30 of the 42 PALB2 variants under investigation have been described previously in ClinVar, many of them (61%) as VUSs (or conflicting). Our study reduced this rate to 23% (seven of 30, see Table 2).

ACMG‐AMP clinical classification of 42 PALB2 genetic variants detected in the BRIDGES cohort.

−, not reported; HGVS, Human Genome Variation Society; LB, likely benign; LP, likely pathogenic; N/A, the pSAD read‐out did not support neither a pathogenic nor a benign code strength; P, pathogenic; VUS, variant of uncertain significance.

NM_024675.4.

ClinVar last accessed 14 October 2021.

We used an ACMG‐AMP point system Bayesian framework to combine all pathogenic and benign evidence. For each individual PALB2 variant under investigation, we evaluated all 16 pathogenic and 12 benign ACMG‐AMP codes. The table shows only the pathogenic and benign codes that contributed to the final classification (see Supplementary materials and methods for further details).

PVS1 code strengths are based on SpliceAI predictions and a PALB2 adaptation of the ClinGen‐SVI PVS1 decision tree recommendations.

We deconvoluted pSAD read‐outs into individual transcripts that were assigned a pathogenic or benign) code (variable strength) according to a PALB2 adaptation of the ClinGen‐SVI PVS1 decision tree recommendations. Later, we used an ad hoc algorithm (based on expert judgment) to combine the evidence into an overall PS3/BS3 code strength.

For each variant under investigation, we analyzed BRIDGES BC case–control data. Only c.108+1G>A reached a PS4 evidence level. In addition, we search the scientific literature for additional case–control analyses. One study supported PS4 for c.3113G>A.

We assigned PS2, downgraded to supporting strength (PS2_P), to variants with ≤1 count (in 100,000 gnomADv2.1 global alleles).

We assigned PM3 to variants identified in trans with a pathogenic variant in Fanconi Anemia patients (as reported in the scientific literature).

We assigned PM4 not to specific variants, but to specific transcripts (mgPALB2 read‐outs) produced by those variants. Therefore, we have integrated PM4 into the final PS3/BS3 code strength. Brackets (PM4) reflect this fact.

PP3/BP4 based on SpliceAI predictions.

We assigned PP3/BP4 (based on REVEL scores) not to specific variants, but to specific transcripts (mgPALB2 read‐outs) produced by those variants. Therefore, we have integrated PP3/BP4 into the final PS3/BS3 code strength.

We assigned BP7 to the only synonymous variant under investigation not predicted to affect splicing. Later, mgPALB2 analysis demonstrated that the variant does affect splicing. Based on that, BP7 did not contribute to the final classification. See Supplementary materials and methods and supplementary material, Table S2 for further details.

To directly test the impact of splicing analysis in the final classification, we compared our classification scheme with and without incorporating the mgPALB2 evidence (supplementary material, Tables S2 and S8). The number of VUSs rose from 26% (11 of 42) with mgPALB2 data to 60% without, demonstrating the contribution of mgPALB2 in classifying variants. Interestingly, the major contribution to variant classification is not in the subgroup of 17 GT‐AG variants (35% VUSs versus 18% VUSs), but rather in the subgroup of 18 non‐GT‐AG intron variants (95% VUSs versus 28% VUSs). Although the contribution of mgPALB2 in reducing the proportion of VUSs is evident, splicing data can increase uncertainty in a subset of variants, as c.3348C>T illustrates. This synonymous variant not predicted to affect splicing reaches a likely benign classification (see supplementary material, Table S2, mgPALB2 excluded classification). Remarkably, mgPALB2 analysis shows that c.3348C>T is a leaky spliceogenic variant producing a substantial fraction of non‐functional transcripts that introduce uncertainty into the classification scheme. As a result, the variants are classified as VUS (see supplementary material, Table S2, mgPALB2 incorporated classification). Note that c.3348C>T is one of very few examples in which SpliceAI failed to predict mgPALB2 read‐outs accurately (supplementary material, Table S5).

### Bioinformatics selection of 
PALB2
 variants

Eighty‐two PALB2 variants were analyzed with MES (supplementary material, Table S3). Forty‐four of them were predicted to have a potential damaging effect on splicing. Variants c.2748+1G>A and c.2834+2T>C were excluded from the functional study because their effects on splicing were expected to be similar to selected variants c.2748+1G>T and c.2834+1G>A, respectively. A total of 11 and 30 variants impaired the 3'ss and the 5'ss, respectively, while one variant was predicted to generate a de novo 5'ss (c.48+7G>C). Seven splice‐site disrupting variants (c.49‐2A>T, c.109‐2A>G, c.2587‐2A>G, c.2749‐1G>T, c.2997‐1G>A, c.3348C>T, c.3350+4A>G) were also estimated to create a de novo splice site. Variants c.2748+4A>T, c.2834+6T>C, c.3113+3A>G, and c.3201+6T>A were chosen despite their weak MES score changes (<15%) because they affect highly conserved positions at the 5'ss (consensus sequence EXON/intron: MAG/gtragt).

### Minigene assays

The three wt minigene constructs (mgPALB2_ex1‐3, mgPALB2_ex4‐6, and mgPALB2_ex5‐12) were validated in MCF‐7 cells, where they produced the expected mgFL transcripts: [V1‐PALB2_ex1‐3‐V2, 366‐nt], [V1‐PALB2_ex4_6‐V2, 2,556‐nt], and [V1‐PALB2_ex5_12‐V2, 1,847‐nt], respectively.

The 42 candidate variants were introduced into the corresponding minigene by site‐directed mutagenesis and functionally assayed in MCF‐7 cells. Thirty‐five variants (83.3%) impaired splicing, 23 of which showed a complete lack or minimal amounts (c.48G>A, 0.9%; c.3113+5G>C, 4.9%) of the mgFL transcripts (Table 1, Figures 2, 3, supplementary material, Figure S4). Overall, the 35 spliceogenic variants each produced one to five anomalous transcripts. Eight variants (see Materials and methods) were also examined in MDA‐MB‐231 cells, where they replicated the splicing outcomes detected in MCF‐7 cells.

Splicing outcomes of PALB2 variants.

HGVS, Human Genome Variation Society.

Bold font: no traces or <5% of the mgFL transcript.

[−] site disruption; [+] new site. [↓] reduction of MES score.

Δ, loss of exonic sequences; ▼ inclusion of intronic sequences; E (exon), p (acceptor shift), q (donor shift). When necessary, the exact number of nucleotides inserted or deleted is indicated. For example, transcript ▼(E12q4) denotes the use of an alternative donor site that is located 4 nucleotides downstream of exon 12, causing the addition of 4‐nt to the mature mRNA. ▼(I1mg) refers to the retention of minigene intron 1 that is a shortened version (416 bp) of the genomic PALB2 intron (2,980 bp).

Three uncharacterized transcripts were found: 888‐nt [4.2 ± 0.9%] (c.2997‐1G>A), 592‐nt [8.9 ± 0.3%], and 710‐nt [7.4 ± 0.2%] (both from c.2748+2dup).

Splicing functional assays of variants in minigenes. (A) mgPALB2_ex1‐3 and (B) mgPALB2_ex4‐6. The maps of variants are shown on the left. At the bottom, fluorescent fragment analysis of transcripts generated by the wt and mutant minigenes. FAM‐labeled products (blue peaks) were run with LIZ‐1200 (orange peaks) as size standard. FL, minigene full‐length transcript. Transcript ▼(E1q416) of the variants c.48G>A and c.48+1G>A is not shown because it is out of the size range displayed in minigene mgPALB2_ex1‐3 electropherograms.

Splicing functional assays of variants in minigenes mgPALB2_ex5‐12. The map of variants is shown above. Fluorescent fragment analysis of transcripts generated by the wt and mutant minigenes. FAM‐labeled products (blue peaks) were run with LIZ‐1200 (orange peaks) as size standard. FL, minigene full‐length transcript. Transcript ∆(E9_10) is not shown because it is out of the size range displayed in minigene mgPALB2_ex5‐12.

Eighteen variants affected the classical ±1,2 positions of the 5' or 3'ss. All these variants, except for c.108+2T>C, induced complete aberrant patterns. Variant c.108+2T>C, as well as c.48+2T>C, transforms a GT splice‐donor into an atypical GC donor [36]. Remarkably, although c.48+2T>C only displayed anomalous transcripts, c.108+2T>C produced 85.5% of the mgFL transcript. In addition, other 17 spliceogenic variants affected other positions of the 5'ss, including the three last exonic nucleotides and intronic nucleotides +3, +4, +5, and +6 (Table 1).

### Transcript analysis

Fluorescent fragment analysis revealed the existence of 39 different transcripts, including six mgFL transcripts (five of them carrying rare variants) (Table 1; supplementary material, Table S4 and Figure S4). Alternative site usage was the most frequent spliceogenic mechanism, explaining up to 22 different aberrant transcripts. Two of them, ∆(E10q41) and ∆(E10q65), derived from the activation of atypical GC donors. The second most prevalent mechanism was exon skipping, which was detected in 10 different transcripts. Finally, one transcript contained a full intron 5 retention (▼(I5)). The identity of three transcripts (592, 710, and 888‐nt long) could not be characterized.

Up to 16 of the abovementioned isoforms had been previously characterized as naturally occurring events [25], including: ▼(E1q9), ∆(E1q17), ∆(E2p6), ∆(E2), ∆(E4), ∆(E5p139), ∆(E7p10), ∆(E7), ∆(E8), ∆(E9), ∆(E10), ∆(E9_10), ∆(E10p2), ∆(E10q31), ∆(E11), and ∆(E12) (i.e. some spliceogenic variants apparently upregulate already expressed transcripts).

Twenty‐four transcripts (including one mgFL transcript carrying a nonsense variant) were predicted to introduce premature termination codons (PTC). Fifteen transcripts kept the reading frame: ▼(E1q9), ∆(E2p6), ∆(E2), ▼(E3q48), ∆(E4), ∆(E7), ∆(E8p15), ∆(E9), ∆(E10), ∆(E9_10) (supplementary material, Table S4), four mgFL transcripts carrying missense or synonymous variants, and the wt mgFL transcripts.

mgPALB2 read‐outs showed a remarkable agreement with previous experimental data in carriers (limited to five variants) and with SpliceAI predictions (supplementary material, Table S5). Taken together, the data support the robustness of the mgPALB2 assay and the accuracy of SpliceAI in predicting the actual outcome of spliceogenic variants.

### ACMG‐AMP classification of variants

We evaluated the clinical relevance of each variant per the ACMG‐AMP classification scheme using a Bayesian approach and incorporating the mgPALB2 splicing results into the evaluation of all other existing evidence for the variant (see Materials and methods). As shown in Table 2, this resulted in classifying 23 PALB2 variants as pathogenic/likely pathogenic and, equally relevant, eight variants as likely benign (mostly, intronic variants that do not affect splicing). Eleven variants were classified as variants of uncertain significance (VUSs) (Table 2; Supplementary materials and methods; supplementary material, Tables S6, S7; Figures [Link], [Link]). Only 30 of the 42 PALB2 variants under investigation have been described previously in ClinVar, many of them (61%) as VUSs (or conflicting). Our study reduced this rate to 23% (seven of 30, see Table 2).

ACMG‐AMP clinical classification of 42 PALB2 genetic variants detected in the BRIDGES cohort.

−, not reported; HGVS, Human Genome Variation Society; LB, likely benign; LP, likely pathogenic; N/A, the pSAD read‐out did not support neither a pathogenic nor a benign code strength; P, pathogenic; VUS, variant of uncertain significance.

NM_024675.4.

ClinVar last accessed 14 October 2021.

We used an ACMG‐AMP point system Bayesian framework to combine all pathogenic and benign evidence. For each individual PALB2 variant under investigation, we evaluated all 16 pathogenic and 12 benign ACMG‐AMP codes. The table shows only the pathogenic and benign codes that contributed to the final classification (see Supplementary materials and methods for further details).

PVS1 code strengths are based on SpliceAI predictions and a PALB2 adaptation of the ClinGen‐SVI PVS1 decision tree recommendations.

We deconvoluted pSAD read‐outs into individual transcripts that were assigned a pathogenic or benign) code (variable strength) according to a PALB2 adaptation of the ClinGen‐SVI PVS1 decision tree recommendations. Later, we used an ad hoc algorithm (based on expert judgment) to combine the evidence into an overall PS3/BS3 code strength.

For each variant under investigation, we analyzed BRIDGES BC case–control data. Only c.108+1G>A reached a PS4 evidence level. In addition, we search the scientific literature for additional case–control analyses. One study supported PS4 for c.3113G>A.

We assigned PS2, downgraded to supporting strength (PS2_P), to variants with ≤1 count (in 100,000 gnomADv2.1 global alleles).

We assigned PM3 to variants identified in trans with a pathogenic variant in Fanconi Anemia patients (as reported in the scientific literature).

We assigned PM4 not to specific variants, but to specific transcripts (mgPALB2 read‐outs) produced by those variants. Therefore, we have integrated PM4 into the final PS3/BS3 code strength. Brackets (PM4) reflect this fact.

PP3/BP4 based on SpliceAI predictions.

We assigned PP3/BP4 (based on REVEL scores) not to specific variants, but to specific transcripts (mgPALB2 read‐outs) produced by those variants. Therefore, we have integrated PP3/BP4 into the final PS3/BS3 code strength.

We assigned BP7 to the only synonymous variant under investigation not predicted to affect splicing. Later, mgPALB2 analysis demonstrated that the variant does affect splicing. Based on that, BP7 did not contribute to the final classification. See Supplementary materials and methods and supplementary material, Table S2 for further details.

To directly test the impact of splicing analysis in the final classification, we compared our classification scheme with and without incorporating the mgPALB2 evidence (supplementary material, Tables S2 and S8). The number of VUSs rose from 26% (11 of 42) with mgPALB2 data to 60% without, demonstrating the contribution of mgPALB2 in classifying variants. Interestingly, the major contribution to variant classification is not in the subgroup of 17 GT‐AG variants (35% VUSs versus 18% VUSs), but rather in the subgroup of 18 non‐GT‐AG intron variants (95% VUSs versus 28% VUSs). Although the contribution of mgPALB2 in reducing the proportion of VUSs is evident, splicing data can increase uncertainty in a subset of variants, as c.3348C>T illustrates. This synonymous variant not predicted to affect splicing reaches a likely benign classification (see supplementary material, Table S2, mgPALB2 excluded classification). Remarkably, mgPALB2 analysis shows that c.3348C>T is a leaky spliceogenic variant producing a substantial fraction of non‐functional transcripts that introduce uncertainty into the classification scheme. As a result, the variants are classified as VUS (see supplementary material, Table S2, mgPALB2 incorporated classification). Note that c.3348C>T is one of very few examples in which SpliceAI failed to predict mgPALB2 read‐outs accurately (supplementary material, Table S5).

### Discussion

The genetic landscape of hereditary BC is characterized by a high complexity where loss‐of‐function variants in a minimum of eight genes (BRCA1, BRCA2, PALB2, BARD1, RAD51C, RAD51D, ATM, and CHEK2) show a significant association with BC risk in the general population, as several large‐scale sequencing studies have recently reported [5, 7, 8, 37]. Indeed, PALB2 has been firmly established as a high‐risk BC susceptibility gene by these studies, with overall lifetime female BC risk above 30%. Accordingly, germline pathogenic variants in PALB2 are considered actionable findings in many clinical settings, with proposed actions ranging from intensified surveillance to prophylactic surgery [38], according to NCCN (www.nccn.org) and NICE guidelines (www.nice.org.uk/guidance). The latter highlights the clinical relevance of a robust PALB2 variant classification system. Moreover, PALB2 germline defects are not only clinically actionable, but are relatively prevalent, accounting for approximately 10% of the pathogenic variants of the eight genes mentioned above and 0.5% of all BC cases (422 pathogenic variants in 87,158 patients), underlining the role of this gene in hereditary BC. However, a relevant fraction of BC patients carries a VUS in their BC susceptibility genes (1.65% in PALB2) [7], which poses a challenge in genetic counseling, as risk estimates are based solely on personal and familial cancer history. Functional assays of VUS provide critical information for their clinical interpretation.

Aberrant splicing is a known frequent mechanism of gene inactivation associated with germline variants in BC genes [20, 21, 24, 39]. Herein, we have focused on 82 PALB2 variants located at the splice‐site boundaries detected in the BRIDGES cohort, so that we have accomplished the largest splicing functional study of PALB2 by minigenes to date. As we pointed out in previous reports, simplicity, sensitivity, robustness, or versatility are the main features of the minigene strategy, thus supporting its suitability for the preliminary characterization of potential spliceogenic variants. Furthermore, any other potential spliceogenic variant might be tested in these three constructs. Thus, a preliminary analysis of the 3,627 different ClinVar PALB2 variants (https://www.ncbi.nlm.nih.gov/clinvar/, date last accessed 16 April 2021) with splice‐site predictors would select other additional 65 candidate spliceogenic variants that might be promptly checked in the three PALB2 minigenes.

Remarkably, a large proportion of tested variants (35/42, 83.3%) impaired splicing, supporting the specificity of our pre‐selection approach. Moreover, the high sensitivity and resolution of fluorescent fragment analysis allowed us to detect up to 36 different aberrant transcripts. Furthermore, this methodology is appropriate to successfully distinguish transcripts with just 2‐nt deletions [∆(E10p2)] from the mgFL transcript. In general, the splicing outcomes of these variants or of other ones affecting the same splice sites support the reproducibility of the minigenes when they were compared with patient RNA analysis previously described by other groups [25, 40, 41, 42]. Hence, c.48G>A, c.49‐2A>T, c.2748+1G>T (in this case versus c.2748+2T>G), c.2834+1G>A (versus c.2834+2T>C), c.3113+5G>C, and c.3350+5G>A yielded very similar or identical results. However, some discrepancies were observed between patient and minigene RNA assays, or even among different patient RNA reports, for variants c.3113G>A and c.3113+5G>C [6, 25, 42, 43, 44]. This may be due to the complexity of alternative splicing patterns in certain genomic regions as described for BRCA2 variant c.7976+5G>T [45], to technical issues, like the use of NMD inhibitors in the minigene assays that allow the detection of minor PTC transcripts, to the cell type where the tests are performed or to the methods used for characterizing transcripts, such as RT‐PCR or RNA‐seq, from patient RNA.

Despite that ±1,2 splice‐site variants are usually considered pathogenic, some of them produced in‐frame or the expected mgFL transcripts. Additionally, variants at other conserved nucleotides of the splice sites are potentially deleterious. Thus, although most spliceogenic variants of our study (18/34) affected ±1,2 nucleotides, 16 changes at other positions (the last three exonic and intronic +3, +4, +5, and +6 nucleotides) also disrupted splicing, confirming their relevance in splice‐site recognition and suggesting their possible role in disease pathogenicity. In this regard, splicing impairments of +3 to +6 variants were especially difficult to predict by MES, because weak or no splicing impacts were observed for several variants, such as c.48+4C>T or c.2834+3A>G, among others. Moreover, two spliceogenic variants (c.3113+3A>G and c.3201+6T>A) did not significantly affect the MES score but they were selected because they changed conserved nucleotides of the 5'ss (see Materials and methods and Table 1). Then, we compared MES with other algorithms, such as NNSplice and SpliceAI (supplementary material, Tables S9–S11) in 38 previously assayed [+3_to_+6] variants [20, 22, 23, 24, 46]. The accuracy of MES and SpliceAI in the subset of +3 to +6 variants was similar (78.9 and 76.3%, respectively), outperforming NNSplice.

It is also worth mentioning two +2T>C variants (c.48+2T>C and c.108+2T>C) that convert a canonical GT donor into an atypical GC one (Figure 4). About 1% of human 5'ss are GC and are commonly associated to physiological alternative splicing [47, 48]. Variants c.48+2T>C and c.108+2T>C induce different splicing impacts (0 and 86% of the mgFL transcript, respectively, Table 1). In fact, it has been previously estimated that approximately 15–18% of +2T>C substitutions are capable of generating variable amounts of canonical transcripts (1–84%) [49]. Consequently, all these changes should be carefully interpreted because they are not invariably associated with major splicing disruptions. Likewise, two minor anomalous transcripts (∆(E10q41) and ∆(E10q65)) are generated by using a GC‐5'ss.

Sequences of the canonical GT and the atypical GC sites of c.48+2T>C, c.108+2T>C, PALB2 exon 12, BRCA2 exon 17, and transcripts ∆(E10q41) and ∆(E10q65). The size of each letter represents the nucleotide frequency at each position. Pictograms were obtained with WebLogo (https://weblogo.berkeley.edu/logo.cgi).

The GC‐5'ss are intrinsically weak due to the change of the essential +2 nucleotide, but, in general, the other splice‐site positions seem to be more conserved. The natural GC 5'ss of PALB2‐exon 12 and BRCA2‐exon 17 as well as the GC‐5'ss created by c.108+2T>C in intron 2 (mgFL transcript, 86%) are identical (CAG|gcaagt) (Figure 4). These data suggest that the de novo GC of c.108+2T>C behaves like the natural ones of PALB2‐exon 12 and BRCA2‐exon 17 and might be regulated in a similar way. Indeed, several siRNA experiments of splicing factors suggested that BRCA2‐exon 17 recognition is mediated by SC35, SF2/ASF, and Tra2β [23], while 9G8, Tra2β, and SC35 participate in the recognition of an anomalous GC used in the BTK gene [47]. Finally, the splicing outcome of variant c.3348C>T that affects the 5'ss (GC) of exon 12 should also be highlighted. As calculated by MES, c.3348C>T weakens the natural GC donor (MES score: 3.1→1.9) and concomitantly creates a stronger GT site (MES score: 6.5). However, ∆(E12q4) and the mgFL transcript represent 26.7 and 68%, respectively, of the overall expression, indicating preferential use of the GC site and supporting a specific regulation of PALB2‐exon 12 by splicing factors that enhance GC recognition.

As PALB2 expert panel specifications of the ACMG‐AMP guidelines are not yet available (https://clinicalgenome.org/, last accessed 25 November 2021), we have developed in‐house specifications (Supplementary materials and methods). Incorporating mgPALB2 into the classification system makes a substantial impact on final variant classification, reducing VUSs from 28 to 11 (60% reduction), with a particularly high reclassification impact in the subgroup of 18 non‐GT‐AG intron variants. The latter reflects an intrinsic feature of the ACMG‐AMP classification scheme; predictive codes for GT‐AG and non‐GT‐AG intronic variants with similar splicing predictions have nonetheless very different strength (PVS1 versus PP4). As a result, experimental splicing data contribute very little to GT‐AG variants with accurate splicing predictions (replacing a PVS1 code for a PS3 code with identical strength).

Incorporating mgPALB2 read‐outs into the ACMG‐AMP schema has been challenging. In the end, we propose solutions not necessarily supported by the ClinGen SVI. These include: (1) an approach to transform complex mgPALB2 read‐outs into a PS3/BS3 code (supplementary material, Figure S3A) and (2) the arbitrary definition of a ≥90% overall pathogenic (or benign) expression threshold to support PS3 (or BS3). We think that, lacking experimental data supporting another threshold (we are not aware of experimental data demonstrating the level of PALB2 expression conferring haplosufficiency), 90% has the merit of being conservative. Based on this, mgPALB2 read‐outs in up to nine variants were considered conflicting and, therefore, not contributing to the final ACMG‐AMP classification (Table 2 and supplementary material, Table S2). Not surprisingly, all these variants ended up as VUSs. It is tempting to speculate that some of them (e.g. c.2834+5G>A, producing 59% of PTC‐NMD transcripts and 41% of mgFL transcripts) could be associated with some intermediate risk level for developing BC (or other malignancies).

Our study shows the limited contribution of the ACMG‐AMP evidence code PS4 to classify rare genetic variants in BC susceptibility genes, even if associated with high risk (as it is the case of PALB2). After analyzing the larger PALB2 association study reported so far (>113,000 woman in the BRIDGES cohort), only one of the 42 variants under investigation (c.108+1G>A, odds ratio = 4.57, p = 0.007) qualifies for PS4.

An overall reduction in the proportion of VUSs is a desirable feature of any variant classification system but relocating previously classified variants into the VUS category might be clinically relevant. In this regard, we highlight GT‐AG variants c.49‐2A>T, c.108+2T>C, and c.211+1G>A, all of them considered likely pathogenic by ClinVar submitters. We have classified these variants as VUS. It is not the mgPALB2 data that explain the difference (all three ended up as VUSs, regardless of including/excluding mgPALB2 data from our classification scheme), but it is the accuracy of SpliceAI combined with the PALB2 adaptation of the PVS1 decision tree.

We recommend being cautious when classifying GT‐AG variants in high‐risk BC genes such as PALB2 [5], for which prophylactic surgery might be recommended to healthy carriers. In a previous study [25], we already warned about certain PALB2 GT‐AG variants. Based on the present data, we have reevaluated and update these warnings (supplementary material, Table S7).

In summary, we have tested 42 PALB2 splice‐site variants in minigenes that have proven to be an appropriate and straightforward strategy for the characterization of splicing outcomes of putative spliceogenic variants. The subsequent application of the ACMG‐AMP evidence code PS3/BS3 reduced VUSs by 60%. We ended up classifying 23 variants as pathogenic/likely pathogenic. Remarkably, these 23 variants account for approximately 15% of all presumed pathogenic variants reported in BRIDGES subjects.

### Clinical interpretation

As PALB2 expert panel specifications of the ACMG‐AMP guidelines are not yet available (https://clinicalgenome.org/, last accessed 25 November 2021), we have developed in‐house specifications (Supplementary materials and methods). Incorporating mgPALB2 into the classification system makes a substantial impact on final variant classification, reducing VUSs from 28 to 11 (60% reduction), with a particularly high reclassification impact in the subgroup of 18 non‐GT‐AG intron variants. The latter reflects an intrinsic feature of the ACMG‐AMP classification scheme; predictive codes for GT‐AG and non‐GT‐AG intronic variants with similar splicing predictions have nonetheless very different strength (PVS1 versus PP4). As a result, experimental splicing data contribute very little to GT‐AG variants with accurate splicing predictions (replacing a PVS1 code for a PS3 code with identical strength).

Incorporating mgPALB2 read‐outs into the ACMG‐AMP schema has been challenging. In the end, we propose solutions not necessarily supported by the ClinGen SVI. These include: (1) an approach to transform complex mgPALB2 read‐outs into a PS3/BS3 code (supplementary material, Figure S3A) and (2) the arbitrary definition of a ≥90% overall pathogenic (or benign) expression threshold to support PS3 (or BS3). We think that, lacking experimental data supporting another threshold (we are not aware of experimental data demonstrating the level of PALB2 expression conferring haplosufficiency), 90% has the merit of being conservative. Based on this, mgPALB2 read‐outs in up to nine variants were considered conflicting and, therefore, not contributing to the final ACMG‐AMP classification (Table 2 and supplementary material, Table S2). Not surprisingly, all these variants ended up as VUSs. It is tempting to speculate that some of them (e.g. c.2834+5G>A, producing 59% of PTC‐NMD transcripts and 41% of mgFL transcripts) could be associated with some intermediate risk level for developing BC (or other malignancies).

Our study shows the limited contribution of the ACMG‐AMP evidence code PS4 to classify rare genetic variants in BC susceptibility genes, even if associated with high risk (as it is the case of PALB2). After analyzing the larger PALB2 association study reported so far (>113,000 woman in the BRIDGES cohort), only one of the 42 variants under investigation (c.108+1G>A, odds ratio = 4.57, p = 0.007) qualifies for PS4.

An overall reduction in the proportion of VUSs is a desirable feature of any variant classification system but relocating previously classified variants into the VUS category might be clinically relevant. In this regard, we highlight GT‐AG variants c.49‐2A>T, c.108+2T>C, and c.211+1G>A, all of them considered likely pathogenic by ClinVar submitters. We have classified these variants as VUS. It is not the mgPALB2 data that explain the difference (all three ended up as VUSs, regardless of including/excluding mgPALB2 data from our classification scheme), but it is the accuracy of SpliceAI combined with the PALB2 adaptation of the PVS1 decision tree.

We recommend being cautious when classifying GT‐AG variants in high‐risk BC genes such as PALB2 [5], for which prophylactic surgery might be recommended to healthy carriers. In a previous study [25], we already warned about certain PALB2 GT‐AG variants. Based on the present data, we have reevaluated and update these warnings (supplementary material, Table S7).

In summary, we have tested 42 PALB2 splice‐site variants in minigenes that have proven to be an appropriate and straightforward strategy for the characterization of splicing outcomes of putative spliceogenic variants. The subsequent application of the ACMG‐AMP evidence code PS3/BS3 reduced VUSs by 60%. We ended up classifying 23 variants as pathogenic/likely pathogenic. Remarkably, these 23 variants account for approximately 15% of all presumed pathogenic variants reported in BRIDGES subjects.

### Author contributions statement

MdlH and EAV conceived the study. AV‐P, LS‐M, EB‐M, SC, JA, LD, DFE, PD and MPGV were responsible for data curation. AV‐P, EB‐M, LS‐M, VL, MdlH and EAV carried out the formal analysis. PD, DFE, MPGV, MdlH and EAV acquired funding. AV‐P, LS‐M, EB‐M, VL, EF‐B, MD, PP‐S, AE‐S, AGA, SG‐B, MPGV, MdlH and EAV carried out the investigations. AV‐P, EF‐B, LS‐M, EB‐M, AGA and EAV were responsible for methodology. EAV supervised the study. AV‐P, MdlH and EAV wrote the original draft of the manuscript. SG‐B, MdlH and EAV reviewed and edited the manuscript. All authors approved the final version of the manuscript.

### Supporting information

Supplementary materials and methods

Click here for additional data file.

Figure S1. Insert sequences of minigenes mgPALB2_ex1‐3, _ex4‐6 and _ex5‐12

Click here for additional data file.

Figure S2. Mapping PALB2 regions critical to protein function

Click here for additional data file.

Figure S3. (A) Proposed decision tree assigning a PS3/BS3 code strength to mgPALB2 minigene read‐outs. (B) Pathogenic/Benign annotation of mgPALB2 minigene deconvolute read‐outs (transcripts). (C) Pathogenic/Benign annotation of PALB2 transcripts

Click here for additional data file.

Figure S4. Additional splicing functional assays of PALB2 variants

Click here for additional data file.

Table S1. Cloning and mutagenesis primers of PALB2

Table S2. ACMG‐AMP point‐based classification of 42 PALB2 variants

Table S3. Bioinformatics analysis of PALB2 variants with Max Ent Score

Table S4. Transcript annotations according to PALB2 sequence NM_024675.4

Table S5. Comparative of SpliceAI prediction, mgPALB2 read‐outs, and experimental splicing data in carriers

Table S6. PALB2 regions critical to protein function

Table S7.
PALB2 sites/variants for which we place a warning

Table S8. Impact of mgPALB2 data on the ACMG‐AMP classification of 42 PALB2 variants

Table S9. Bioinformatics predictions of +3 to +6 variants and splicing outcomes

Table S10. Summary of bioinformatics predictions

Table S11. Sensitivity and specificity for +3 to +6 variants

Click here for additional data file.



# SUPPLEMENTAL FILE 1: PATH-256-321.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: PMC9306493

[File available at: automated_output/BRCA2/20251125_175029/pmc_fulltext/34846068_supplements/PMC9306493]

