# MAIN TEXT

## BRCA1/2 mutation spectrum in Chinese early-onset breast cancer

### Abstract

BackgroundBreast cancer is the most commonly diagnosed cancer among women. Although many studies have reported the BRCA mutations among breast cancer patients, few studies have focused among Chinese early-onset breast cancer patients. The purpose of this study is to identify BRCA1 and BRCA2 mutation features and their clinical significance of early-onset Chinese breast cancer patients.MethodsA total of 54 female patients diagnosed with breast cancer were enrolled in this study, of which 27 were younger than 40 (study group, mean age 32 years, range, 23–40 years) and 27 were older than 40 (control group, mean age 52 years, range, 41–68 years). Tumor FFPE samples were collected for somatic mutation test, while blood samples or normal tissue were used for germline mutation by both PGM and Miseq platform. All codon exons and functional introns for BRCA1/2 were covered. The clinical significance of mutation types was cross analyzed in several available database. The novel mutations were confirmed by sanger sequencing.ResultsIn study group, 14.8% (4/27) and 3.7% (1/27) patients had deleterious BRCA1/2 germline and somatic mutations respectively. While in control group, only 3.7% (1/27) and 7.4% (2/27) had deleterious BRCA1/2 germline and somatic mutations respectively. BRCA1 germline mutation c.2623C>T and BRCA2 germline mutation c.5852G>A were found to be novel mutation sites and confirmed by sanger sequencing.ConclusionsOur study found two novel BRCA1/2 mutation sites in early-onset breast cancer, and also showed that early-onset breast cancer patients are more likely to harbor germline mutations with deleterious and uncertain significance.

### Introduction

Breast cancer is the most commonly diagnosed cancer among women and is the second leading cause of cancer death for women in US (1). Early-onset breast cancer, though only accounting for 7% of all breast cancers, is the most common cancer among young females (2), and has been described to be more biologically aggressive than in older women, which has been associated with a worse prognosis (3). Nowadays, with the development of new techniques, an increasing number of susceptibility gene mutations related to early-onset breast cancer has been detected to improve diagnosis and therapy of early-onset breast cancer and predict outcome. Among all these detected mutations, BRCA1/2 are still figured out to play an important role in early-onset breast cancer (4-7).

Among the multitude of markers involved into the breast cancer tumorigenesis as EGFR, RANK (8), BRCA1 and BRCA2 are the first two genes found directly related to hereditary breast cancer. BRCA1 is located on 17q21.31, and the exon count is 24. BRCA2 is located on 13q13.1, and the exon count is 27. Both BRCA1 and BRCA2 are considered tumor suppressor gene which involved in maintenance of genome stability, specifically the homologous recombination pathway for double-strand DNA repair. Inherited mutations in BRCA1 and BRCA2, confer increased lifetime risk of developing breast cancer (9). Due to the length of BRCA1/2, and the randomness of mutation sites, using the traditional sanger sequencing, qPCR, or MLPA to detect the whole gene is not ideal. In contrast, the high throughput next-generation sequencing (NGS) could be more efficient, that can detect all the exons and their adjacent regions of BRCA1/2 at a time.

As is mentioned in several published research (10-13), compared with other countries, Chinese women carry different BRCA mutations rate and types. In this study, we tried to figure out a detailed BRCA1/2 germline and somatic mutation spectrum in young Chinese breast cancer patients.

### Methods

A total of 54 female patients diagnosed with breast cancer were enrolled in this study, of which 27 patients (mean age 32 years, range, 23–40 years) diagnosed at the age younger than 40 and the rest 27 (mean age 52 years, range, 41–68 years) diagnosed at the age older than 40 in West China Hospital from January 2010 to December 2016 consecutively, belonging to study group and control group, respectively. DNA of 54 FFPE samples of cancer tissue were collected to test the somatic BRCA1/2 mutations, while DNA of 31 blood samples and 23 FFPE samples of normal tissue were used to exclude the germline BRCA1/2 mutations by two NGS platforms PGM and Miseq. Clinicopathological characteristics were reviewed including age, estrogen-receptor (ER) status, progesterone-receptor (PR) status, human epidermal growth factor-2 (HER-2) status, Ki-67, molecular phenotypes, TNM staging, etc. Patients’ clinical information is showed in Table 1. Approval for the study was granted by the Ethics Committee of West China Hospital (number: 2013-191).

DNA was extracted using QIAamp DNA FFPE Tissue Kit. For library construction, 30 ng of gDNA (measured using the Qubit fluorometer in combination with the Qubit dsDNA HS assay kit) was amplified using BRCAimPLUS DNA panel (SINGLERA Genomics Inc.) and the Ion Ampliseq™ HiFi Master Mix (Ion Ampliseq™ Library kit 2.0). The amplicons were then digested, barcoded and amplified using the Ion Ampliseq™ Library kit 2.0, Ion Xpress™ barcode adapters kit (Life technologies), and Ion-to-Miseq primers according to the manufacturer’s instructions. The library size was checked using the Agilent High Sensitivity DNA Kit by the Bioanalyzer 2100 instrument (Agilent Technologies), and library concentration was evaluated using the Qubit fluorometer and the Qubit dsDNA HS assay kit (Life technologies).

For PGM platform, 50 pM of each library was multiplexed and clonally amplified on the Chef instrument with the Ion PGM™ Hi-Q™ Chef Solutions Cartridge, Ion PGM™ Hi-Q™ Chef Reagents Cartridge, Ion PGM™ Hi-Q™ Chef Supplies and Ion 318™ Chip v2 breast cancer (Life technologies) according to the manufacturer’s instructions. Finally, the Ion 318™ chips loaded with enriched template ISP were sequenced on a PGM™ sequencer with the Ion PGM™ sequencing 200 kit v2 according to the manufacturer’s instructions.

For Miseq platform, all purified libraries were quantified by real-time PCR using the SYBR Fast Illumina Library Quantification Kit (Kapa Biosystems) and pooled to give equal genome coverage from each library. Each multiplexed library pool was sequenced an Illumina MiSeq for 151 cycles from each end read according to the manufacturer’s instructions.

We then analyzed the sequencing data from both Miseq and PGM platforms using the BRCAimPLUS DNA pipeline-a customized bioinformatic analysis workflow for cancer panel. As other bioinformatic pipelines, it involves processing a series of data transformation steps: alignment, variant calling, annotation, filtering and reporting (14-17).

DNA was extracted using QIAamp DNA FFPE Tissue Kit. PCR reactions were run in final volumes of 25 µL containing 200 ng DNA, 0.25 mM dNTPs, 10pmol of each primer and 1.25 unit of Taq polymerase [TIANGEN BIOTECH (BEIJING) CO., LTD.]. PCR was performed in an T100 thermal cycler (Bio-Rad, Hercules, CA, USA) with initial denaturation at 95 °C for 3 min, followed by 35 cycles of 95 °C for 30 s, 60 °C for 30 s, 72 °C for 30 s. The purified PCR products were sequenced by Sanger’s sequencing according to manufacturer’s instruction.

Variants both detected by two platforms were then to be evaluated. According to the classification system of International Agency for Research on Cancer (IARC), American College of Medical Genetics and Genomics (ACMG), and Evidence-based Network for the Interpretation of Germline Mutant Alleles (ENIGMA), the mutations of BRCA gene were divided into five categories: pathogenic (Class 5, the rate of pathogenicity is higher than 0.99), likely pathogenic (Class 4, the rate of pathogenicity is between 0.95 and 0.99), uncertain significance (Class 3, the rate of pathogenicity is between 0.05 and 0.949), likely benign (Class 2, the rate of pathogenicity is between 0.001 and 0.049), benign (Class 1, the rate of pathogenicity is lower than 0.001).

Also, the BRCA gene variants identified were checked for pathogenicity in 4 databases: Breast Cancer Information Core (BIC) (18), Leiden Open Variation Database (LOVD) (19), the Catalogue of Somatic Mutations in Cancer database (COSMIC) (20) and ClinVar database (https://www.ncbi.nlm.nih.gov/clinvar/).

All the statistical analyses were performed using the statistical program for social sciences (SPSS) software package version 19.0 (Chicago, IL, USA). Two independent sample t tests were applied in comparison between groups. Enumeration data was expressed as cases or percentage. Chi-square test was used in comparison between groups, with P<0.05 represented for the difference was statistically significant.

### Cases and samples

A total of 54 female patients diagnosed with breast cancer were enrolled in this study, of which 27 patients (mean age 32 years, range, 23–40 years) diagnosed at the age younger than 40 and the rest 27 (mean age 52 years, range, 41–68 years) diagnosed at the age older than 40 in West China Hospital from January 2010 to December 2016 consecutively, belonging to study group and control group, respectively. DNA of 54 FFPE samples of cancer tissue were collected to test the somatic BRCA1/2 mutations, while DNA of 31 blood samples and 23 FFPE samples of normal tissue were used to exclude the germline BRCA1/2 mutations by two NGS platforms PGM and Miseq. Clinicopathological characteristics were reviewed including age, estrogen-receptor (ER) status, progesterone-receptor (PR) status, human epidermal growth factor-2 (HER-2) status, Ki-67, molecular phenotypes, TNM staging, etc. Patients’ clinical information is showed in Table 1. Approval for the study was granted by the Ethics Committee of West China Hospital (number: 2013-191).

### Next generation sequencing on PGM platform and Miseq platform

DNA was extracted using QIAamp DNA FFPE Tissue Kit. For library construction, 30 ng of gDNA (measured using the Qubit fluorometer in combination with the Qubit dsDNA HS assay kit) was amplified using BRCAimPLUS DNA panel (SINGLERA Genomics Inc.) and the Ion Ampliseq™ HiFi Master Mix (Ion Ampliseq™ Library kit 2.0). The amplicons were then digested, barcoded and amplified using the Ion Ampliseq™ Library kit 2.0, Ion Xpress™ barcode adapters kit (Life technologies), and Ion-to-Miseq primers according to the manufacturer’s instructions. The library size was checked using the Agilent High Sensitivity DNA Kit by the Bioanalyzer 2100 instrument (Agilent Technologies), and library concentration was evaluated using the Qubit fluorometer and the Qubit dsDNA HS assay kit (Life technologies).

For PGM platform, 50 pM of each library was multiplexed and clonally amplified on the Chef instrument with the Ion PGM™ Hi-Q™ Chef Solutions Cartridge, Ion PGM™ Hi-Q™ Chef Reagents Cartridge, Ion PGM™ Hi-Q™ Chef Supplies and Ion 318™ Chip v2 breast cancer (Life technologies) according to the manufacturer’s instructions. Finally, the Ion 318™ chips loaded with enriched template ISP were sequenced on a PGM™ sequencer with the Ion PGM™ sequencing 200 kit v2 according to the manufacturer’s instructions.

For Miseq platform, all purified libraries were quantified by real-time PCR using the SYBR Fast Illumina Library Quantification Kit (Kapa Biosystems) and pooled to give equal genome coverage from each library. Each multiplexed library pool was sequenced an Illumina MiSeq for 151 cycles from each end read according to the manufacturer’s instructions.

We then analyzed the sequencing data from both Miseq and PGM platforms using the BRCAimPLUS DNA pipeline-a customized bioinformatic analysis workflow for cancer panel. As other bioinformatic pipelines, it involves processing a series of data transformation steps: alignment, variant calling, annotation, filtering and reporting (14-17).

### Variants confirmation

DNA was extracted using QIAamp DNA FFPE Tissue Kit. PCR reactions were run in final volumes of 25 µL containing 200 ng DNA, 0.25 mM dNTPs, 10pmol of each primer and 1.25 unit of Taq polymerase [TIANGEN BIOTECH (BEIJING) CO., LTD.]. PCR was performed in an T100 thermal cycler (Bio-Rad, Hercules, CA, USA) with initial denaturation at 95 °C for 3 min, followed by 35 cycles of 95 °C for 30 s, 60 °C for 30 s, 72 °C for 30 s. The purified PCR products were sequenced by Sanger’s sequencing according to manufacturer’s instruction.

### Variants evaluation

Variants both detected by two platforms were then to be evaluated. According to the classification system of International Agency for Research on Cancer (IARC), American College of Medical Genetics and Genomics (ACMG), and Evidence-based Network for the Interpretation of Germline Mutant Alleles (ENIGMA), the mutations of BRCA gene were divided into five categories: pathogenic (Class 5, the rate of pathogenicity is higher than 0.99), likely pathogenic (Class 4, the rate of pathogenicity is between 0.95 and 0.99), uncertain significance (Class 3, the rate of pathogenicity is between 0.05 and 0.949), likely benign (Class 2, the rate of pathogenicity is between 0.001 and 0.049), benign (Class 1, the rate of pathogenicity is lower than 0.001).

Also, the BRCA gene variants identified were checked for pathogenicity in 4 databases: Breast Cancer Information Core (BIC) (18), Leiden Open Variation Database (LOVD) (19), the Catalogue of Somatic Mutations in Cancer database (COSMIC) (20) and ClinVar database (https://www.ncbi.nlm.nih.gov/clinvar/).

### Statistical analysis

All the statistical analyses were performed using the statistical program for social sciences (SPSS) software package version 19.0 (Chicago, IL, USA). Two independent sample t tests were applied in comparison between groups. Enumeration data was expressed as cases or percentage. Chi-square test was used in comparison between groups, with P<0.05 represented for the difference was statistically significant.

### Results

In the cohort of 54 patients, we obtained an average of 4.1 million reads per sample, with a mean coverage of 90% at a mean X coverage of 2031X on the PGM platform and 5.1 million reads per sample with a mean coverage of 92% at a mean X coverage of 2543X on the Miseq platform. In the early-onset breast cancer patients, 2 had no mutations in BRCA1/2 genes. In the rest 25 patients, a total of 12 mutations of BRCA1 were detected by both PGM and Miseq platform in 19 patients (Table 2). Eleven mutations of BRCA2 gene were detected in 22 patients (Table 3). In control group, two patients had no BRCA mutations, while 7 BRCA1 mutations were detected in 22 patients (Table 2), and 9 BRCA2 mutations were detected in 22 patients (Table 3).

In study group, the 11 BRCA1 germline mutations detected were classified as following: 2 5-pathogenic, 4 3-uncertain, 5 1-benign, in accordance with the data found in ClinVar (Tables S1,S2). c.2623C>T was identified as novel germline mutation site found in a patient diagnosed as breast cancer at the age of 27 (Figure S1A). Among all, 4 mutations were found to be pending, while the rest were not found in BIC. According to the Leiden Open Variation Database, 2 mutations were definitely indicated to affect function. In COSMIC database, 4 mutations were found to be neutral.

As for BRCA2, 10 germline mutations were detected involving 2 5-pathogenic, 3 3-uncertain, 5 1-benign in accordance with the data found in ClinVar (Tables S1,S2). c.5852G>A identified as a novel site found in a patient diagnosed as breast cancer at the age of 28 (Figure S1B). Among all, 1 mutation c.10234A>G was found to be class 1 in BIC database, 2 mutations were found to be pending, while the rest were not found in BIC. According to the Leiden Open Variation Database, 1 mutation was definitely indicated to affect function. In COSMIC database, 4 mutations were found to be neutral, and 1 referring to be pathogenic.

In control group, there were only 6 BRCA1 germline in total 27 patients. c.446A>C was a mutation with uncertain significance. c.2566T>C, c.2612C>T, c.3113A>G, c.3548A>G, c.4837A>G were benign mutations. When considered BRCA2, 1 pathogenic mutation c.9294C>G, 2 uncertain mutations c.10150C>G, c.3445A>G were identified as well as 5 benign mutations.

In study group, there were 4 BRCA1 somatic mutations detected in total, including pathogenic mutation c.213-1G>A and three benign mutation c.2612C>T, c.3113A>G, c.3548A>G only detected in one patient respectively. Also, two BRCA2 somatic benign mutations c.10234A>G and c.1114A>C were detected.

In control group, 5 BRCA1 somatic mutations involving pathogenic mutation c .2398_2401delAAAT detected in one patient, and four benign mutations c.2612C>T, c.3548A>G, c.4837A>G, c.3113A>G. Also, 4 BRCA2 somatic mutations were detected, of which c.7414_7415delAA was pathogenic with the rest c.2971A>G, c.865A>C, and c.1114A>C turned out to be benign.

Mutations defined as pathogenic/likely pathogenic and uncertain were selected for further analysis, as is shown in Table 4. In total, 11 germline (7 3-uncertain, 4 5-pathogenic) and 1 somatic (5-pathogenic) of BRCA1/2 mutations were detected in study group, while 4 germline (3 3-uncertain and 1 5-pathogenic) and 2 somatic (2 5-pathogenic) of BRCA1/2 mutations were detected in control group.

In study group, 14.8% (4/27) and 3.7% (1/27) patients had deleterious BRCA1/2 germline and somatic mutations respectively. While in control group, only 3.7% (1/27) and 7.4% (2/27) had deleterious BRCA1/2 germline and somatic mutations, respectively.

The 4 pathogenic germline mutations were c.2059C>T, c.1299dupC, c.9401delG, c.1399A>T found in study group at the age of 40, 28, 36, 40. And 7 uncertain germline mutations existed in study group were c.988G>A, c.4674A>G, c.1036C>T, c.2623C>T, c.1462A>G, c.461A>G, c.5852G>A. When it turns to control group, there was only one pathogenic germline mutation c.9294C>G found in patient at the age of 57 and 3 uncertain germline mutations c.446A>C, c.10150C>G, c.3445A>G.

### NGS test performance

In the cohort of 54 patients, we obtained an average of 4.1 million reads per sample, with a mean coverage of 90% at a mean X coverage of 2031X on the PGM platform and 5.1 million reads per sample with a mean coverage of 92% at a mean X coverage of 2543X on the Miseq platform. In the early-onset breast cancer patients, 2 had no mutations in BRCA1/2 genes. In the rest 25 patients, a total of 12 mutations of BRCA1 were detected by both PGM and Miseq platform in 19 patients (Table 2). Eleven mutations of BRCA2 gene were detected in 22 patients (Table 3). In control group, two patients had no BRCA mutations, while 7 BRCA1 mutations were detected in 22 patients (Table 2), and 9 BRCA2 mutations were detected in 22 patients (Table 3).

### Germline BRCA1/2 mutations detected in young breast cancer patients

In study group, the 11 BRCA1 germline mutations detected were classified as following: 2 5-pathogenic, 4 3-uncertain, 5 1-benign, in accordance with the data found in ClinVar (Tables S1,S2). c.2623C>T was identified as novel germline mutation site found in a patient diagnosed as breast cancer at the age of 27 (Figure S1A). Among all, 4 mutations were found to be pending, while the rest were not found in BIC. According to the Leiden Open Variation Database, 2 mutations were definitely indicated to affect function. In COSMIC database, 4 mutations were found to be neutral.

As for BRCA2, 10 germline mutations were detected involving 2 5-pathogenic, 3 3-uncertain, 5 1-benign in accordance with the data found in ClinVar (Tables S1,S2). c.5852G>A identified as a novel site found in a patient diagnosed as breast cancer at the age of 28 (Figure S1B). Among all, 1 mutation c.10234A>G was found to be class 1 in BIC database, 2 mutations were found to be pending, while the rest were not found in BIC. According to the Leiden Open Variation Database, 1 mutation was definitely indicated to affect function. In COSMIC database, 4 mutations were found to be neutral, and 1 referring to be pathogenic.

In control group, there were only 6 BRCA1 germline in total 27 patients. c.446A>C was a mutation with uncertain significance. c.2566T>C, c.2612C>T, c.3113A>G, c.3548A>G, c.4837A>G were benign mutations. When considered BRCA2, 1 pathogenic mutation c.9294C>G, 2 uncertain mutations c.10150C>G, c.3445A>G were identified as well as 5 benign mutations.

### Somatic BRCA1/2 mutations detected in young breast cancer patients

In study group, there were 4 BRCA1 somatic mutations detected in total, including pathogenic mutation c.213-1G>A and three benign mutation c.2612C>T, c.3113A>G, c.3548A>G only detected in one patient respectively. Also, two BRCA2 somatic benign mutations c.10234A>G and c.1114A>C were detected.

In control group, 5 BRCA1 somatic mutations involving pathogenic mutation c .2398_2401delAAAT detected in one patient, and four benign mutations c.2612C>T, c.3548A>G, c.4837A>G, c.3113A>G. Also, 4 BRCA2 somatic mutations were detected, of which c.7414_7415delAA was pathogenic with the rest c.2971A>G, c.865A>C, and c.1114A>C turned out to be benign.

### BRCA1/2 mutations of pathogenic and uncertain significance in young breast cancer patients

Mutations defined as pathogenic/likely pathogenic and uncertain were selected for further analysis, as is shown in Table 4. In total, 11 germline (7 3-uncertain, 4 5-pathogenic) and 1 somatic (5-pathogenic) of BRCA1/2 mutations were detected in study group, while 4 germline (3 3-uncertain and 1 5-pathogenic) and 2 somatic (2 5-pathogenic) of BRCA1/2 mutations were detected in control group.

In study group, 14.8% (4/27) and 3.7% (1/27) patients had deleterious BRCA1/2 germline and somatic mutations respectively. While in control group, only 3.7% (1/27) and 7.4% (2/27) had deleterious BRCA1/2 germline and somatic mutations, respectively.

The 4 pathogenic germline mutations were c.2059C>T, c.1299dupC, c.9401delG, c.1399A>T found in study group at the age of 40, 28, 36, 40. And 7 uncertain germline mutations existed in study group were c.988G>A, c.4674A>G, c.1036C>T, c.2623C>T, c.1462A>G, c.461A>G, c.5852G>A. When it turns to control group, there was only one pathogenic germline mutation c.9294C>G found in patient at the age of 57 and 3 uncertain germline mutations c.446A>C, c.10150C>G, c.3445A>G.

### Discussion

BRCA status is not only important for the identification of familial cancer predisposition but also to therapeutic choices for breast cancer patients, e.g., the PARP inhibitor therapy (21-23). BRCA gene mutation is closely related to the early-onset breast cancer occurrence. In most national and international guidelines, testing criteria of BRCA includes patients with breast cancer aged less than 35 or 40 years (24). BRCA1/2 mutation carriers diagnosed with breast cancer before age 50 are prone to a worse survival (25). In our study, the mean age of 27 early-age onset breast cancer patients was 32, with the minimum hospitalized age 23. Only 2 had no BRCA1/2 mutation. Twenty patients had BRCA1 mutations and 22 had BRCA2 mutations, with 3 (11.1%) patients had pathogenic mutations involving c.1299dupC, c.2059C>T, c.213-1G>A in BRCA1/2 (7.4%) had pathogenic mutations involving c.9401delG, c.1399A>T in BRCA2, which were not found in control group. The mutation frequency of the deleterious germline mutation in our study is a little bit higher than in other research (24,26,27) may attribute to insufficient number of analyzed cases.

BRCA1 mutation c.2623C>T was identified as a germline mutation for the first time in this study, which was a SNP detected in EXON10 leading to the protein change p.Pro875Ser. The patient who had c.2623C>T mutation as the only BRCA mutation diagnosed as breast cancer at the age of 27. As for BRCA2, c.5852G>A was also identified as a germline mutation for the first time in this study which was a SNP detected in EXON11 leading to the protein change p.Ser1951Asn. The patient who had c.5852G>A mutation as the only BRCA mutation other than benign mutations diagnosed as breast cancer at the age of 28. Since these two new mutation sites were detected in patients at such young age without other suspicious BRCA mutation sites, we have reason to suspect that these two mutation sites may play a role in the pathogenesis of the early-onset breast cancer which need to be further confirmed.

A large number of literatures show strong correlation between BRCA mutation and familial early-onset breast cancer (28-30). In our study, in total, 11 germline (7 3-uncertain, 4 5-pathogenic) and 1 somatic (5-pathogenic) of BRCA1/2 mutations were detected in study group, while 4 germline (3 3-uncertain and 1 5-pathogenic) and 2 somatic (2 5-pathogenic) of BRCA1/2 mutations were detected in control group. In study group, 14.8% (4/27) patients had deleterious BRCA1/2 germline mutations, and 3.7% (1/27) had deleterious BRCA1/2 somatic mutations, while in control group, 3.7% (1/27) had deleterious BRCA1/2 germline mutations, and 7.4% (2/27) had deleterious BRCA1/2 somatic mutations, displaying a trend that early-onset group is more likely to have germline mutations than elderly counterparts. Therefore, there is a strong recommendation for the early-onset breast cancer patients despite of the family history to get BRCA test for potential benefit for their family members as well as benefit for the patient of PARP inhibitor therapy.

One of the limitations of the study is the absence of triple negative samples in the study groups which is among the breast cancer histotypes the more aggressive with the poor prognosis caused by insufficient cases, but we tried to figure out a detailed spectrum of BRCA1/2 germline and somatic mutations of early-onset breast cancer patients in West China Hospital using NGS. Several deleterious and uncertain mutations were observed in this cohort and it is recommended that a more thorough and functional examination of these mutations should be conducted in the future.



# SUPPLEMENTAL FILE 1: tcr-08-02-483.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)