# MAIN TEXT

## Human BRCA pathogenic variants were originated during recent human history

### Abstract

Evolutionary origin of pathogenic variants in human BRCA1 and BRCA2.

### Introduction

Repairing the damaged DNA by environmental and metabolic factors is vital for all lives on earth. In eukaryotes, this is achieved by the DNA damage repair system composed of multiple pathways to repair different types of DNA damage (Jeggo et al, 2016). The homologous recombination pathway repairs double-strand DNA damage by joint activities of a group of genes to reach error-free repair of DNA double-strand break (Jasin & Rothstein, 2013). BRCA1 and BRCA2 (BRCA) are two of the essential genes in the homologous recombination pathway. BRCA1 was arisen in animal and plant kingdoms 1.2 billion yr ago, and BRCA2 was arisen in fungus, plant, and animal kingdoms 1.6 billion yr ago (Kojic et al, 2002; Pfeffer et al, 2017). Studies revealed that BRCA across wide-range species including most of the primates is evolutionarily highly conserved by negative selection in reflecting its essential roles in maintaining genome stability. In contrast, however, BRCA in humans and its closest living relatives of chimps and bonobos is rapidly evolving by positive selection (Huttley et al, 2000; Fleming et al, 2003; Abkevich et al, 2004; Pavlicek et al, 2004; Burk-Herrick et al, 2006; O’Connell, 2010; Lou et al, 2014). The positive selection likely resulted in the new function of BRCA gained in these three species, such as promoting immunity to counter viral infection (Lou et al, 2014), regulating gene expression and metabolism (Rosen et al, 2006; Chen et al, 2020), enhancing neural development (Pao et al, 2014), and increasing reproduction (Smith et al, 2013).

BRCA is one of the best-known genetic predisposition genes for cancer (Narod & Foulkes, 2004). Efforts made in the past decades have identified nearly 70,000 human BRCA variants (Cline et al, 2018) (https://brcaexchange.org/factsheet). A part of human BRCA variants is determined as “pathogenic” or “likely pathogenic” (BRCA PLP) in causing high cancer risk affecting mostly breast and ovary (Welcsh & King, 2001). BRCA PLP is widely used in clinical practice as the marker for cancer diagnosis, prevention, prognosis, and treatment through synthetic lethal mechanism (Anderson et al, 2008; Plon et al, 2008; Huen et al, 2010; George et al, 2017; Bhaskaran et al, 2019).

Human BRCA PLP provides an ideal system to study evolution origin of human disease susceptibility. While high evolutionary conservation across species suggests the possibility that human BRCA PLP could be originated from evolution conservation, the positive selection imposed in the humans, chimps, and bonobos but not in other species highlights another possibility that human BRCA PLP could be originated from human itself rather than from evolution conservation. However, there is no consent so far in determining which of the two possibilities could be the right origin for human BRCA PLP. We analyzed human BRCA variation data reported by previous evolutionary studies in BRCA (Huttley et al, 2000; Fleming et al, 2003; Burk-Herrick et al, 2006; Lou et al, 2014). The results showed that of the 111 human BRCA variants analyzed in these studies, 108 (94.7%) were non-pathogenic including 42 (37.8%) benign, 47 (42.3%) variants of uncertain significance (VUS), but only 6 (5.4%) were Pathogenic (Table S1). Therefore, the information from the previous studies reflects mainly evolutionary conservation of BRCA variation but not the origin of human BRCA PLP.

Table S1 Human BRCA variants used in previous BRCA evolution studies.

Table S1 Human BRCA variants used in previous BRCA evolution studies.

In the current study, we addressed the evolution origin of human BRCA PLP. The rapid progress of genomics studies provides rich DNA sequence data across a wide range of species for phylogenic study, and the recent anthropological studies have also generated abundant DNA sequence data from ancient humans. These rich resources provide unique opportunities to study the evolutionary origin of the human BRCA PLP on the scale and accuracy unimaginable before. Taking the advantages, we performed a comprehensive phylogenic and archeological study to investigate the evolution origin of human BRCA PLP. Data from our study provide evidence to show that human BRCA PLP was highly unlikely originated from cross-species evolutionary conservation, but most likely arisen during recent human history after the latest out-of-Africa migration and the great expansion of modern human population.

### Results

We identified a total of 6,624 BRCA PLP variants (2,972 in BRCA1 and 3,652 in BRCA2) from the ClinVar database for the study (Tables 1 and S2). We searched evidence for potential conservation of BRCA PLP variants between the humans and the 100 vertebrates distributed in eight clades of Primate, Euarchontoglires, Laurasiatheria, Afrotheria, Mammal, Aves, Sarcopterygii, and Fish. We identified 172 (5.8%) human BRCA1 PLP variants shared with 69 species, and 312 (8.6%) human BRCA2 PLP variants shared with 90 species (Figs 1, 2, and S1A and Tables S3 and S4).

Human BRCA PLP variants in 100 vertebrates.

Table S2 Human BRCA PLP variants used in the study.

Table S2 Human BRCA PLP variants used in the study.

Red cell: same as the human PLPs; empty cell: same base as human wild type; blue cell: different from human wild type and PLPs; gray cell: gaps; “-”: no aligned base; “=”: gap with at least one un-aligned base. The figure was generated using GraphPad Prism (version 9.0.0 for Windows, GraphPad Software). 185delAG, 5382insC: the BRCA1 founder pathogenic variants in Ashkenazi Jews population.

Ibid as in Fig 1. 6174delT: the BRCA2 founder pathogenic variants in Ashkenazi Jews population.

(A) Phylogenetic alignment. Red box shows the alignment of the four human BRCA PLP variants in 14 species in Aves clade. (B) Archeological alignment. It shows that three BRCA PLP variants were shared in three ancient cases. Red box: the BRCA PLP variants.

Table S3 Distribution of the 172 shared human BRCA1 PLP variants in other species.

Table S3 Distribution of the 172 shared human BRCA1 PLP variants in other species.

Table S4 Distribution of the 312 shared human BRCA2 PLP variants in other species.

Table S4 Distribution of the 312 shared human BRCA2 PLP variants in other species.

Of the BRCA1 PLP variants shared in the eight clades, Aves had the highest sharing number of 14 species and Sarcopterygii the second highest of eight species than other clades (P < 0.001) (Fig S2). For example, BRCA1 c.3268C>T (p.Gln1090Ter) was shared with 19 species from Rock pigeon in Aves to Lizard in Sarcopterygii, c.2498T>A (p.Leu833Ter) with 18 species from Saker falcon in Aves to Spiny softshell turtle in Sarcopterygii, and c.2138C>A (p.Ser713Ter) with 18 species from Rock pigeon in Aves to Lizard in Sarcopterygii; all 11 PLP variants located in the BRCA1 BRCT domain were shared with the species within Aves and Sarcopterygii clades. Tasmanian devil, a species with high risk of developing facial cancer, also shared 20 human BRCA1 PLP variants. In the shared BRCA2 PLP variants, the highest one including BRCA2 c.8933C>A (p.Ser2978Ter) shared with 36 species from Lesser Egyptian jerboa in Euarchontoglires to Opossum in mammal, and c.8933C>A (p.Ser2978Ter) shared with 36 species from Lesser Egyptian jerboa in Euarchontoglires to Opossum in Mammal to Spotted gar in Fish, the most distal species sharing this PLP. As the most used animal models in biomedical study, mouse shared eight and rat shared seven human BRCA1 PLP variants, of which only two were located at the BRCT domain. Zebrafish, another important biological model, shared 25 human BRCA2 PLP variants.

It shows that among the eight clades, Aves and Sarcopterygii had high sharing with human BRCA PLP, no sharing of BRCA PLP between human and primates of Chimp, Gorilla, Orangutan, Gibbon, Rhesus, Crab-eating macaque, and no sharing of human BRCA1 PLPs in Fish.

Of the species sharing with human BRCA PLP variants, Spiny softshell turtle in Sarcopterygii had the highest sharing number of 34 BRCA1 PLP variants, and Lizard in Sarcopterygii had the highest sharing number of 57 BRCA2 PLP variants. There were no human BRCA PLP variant shared with primates of Chimp, Gorilla, Orangutan, Gibbon, Rhesus, Crab-eating macaque and Baboon whereas there were 10 PLP variants shared with the distal primates of Marmoset, Squirrel monkey, and Bushbaby: Marmoset shared BRCA1 c.850C>T (p.Gln284Ter) and BRCA2 c.1642C>T (p.Gln548Ter), Squirrel monkey and Bushbaby shared an BRCA1 intronic c.1058G>A (p.Trp353Ter), Squirrel monkey and Bushbaby shared BRCA2 c.4689G>A (p.Trp1563Ter), and Bushbaby shared BRCA2 c.2651C>A (p.Ser884Ter), c.2978G>A (p.Trp993Ter), c.4689G>A (p.Trp1563Ter), c.3504G>T (p.Met1168Ile) and c.5263G>T (p.Glu1755Ter). Marmoset, Squirrel monkey, and Bushbaby had the divergent time of 40, 12.5, 15 million yr from hominins, accordingly. As control, we searched BRCA1 c. 68_69del, BRCA1 c.5266dup and BRCA2 c.5946del, the three BRCA founder PLP variants in Ashkenazi Jews population (Levy-Lahad et al, 1997), in the 100 vertebrates. We found no evidence for their presence in these species: the wildtype AG at the position of 68–69 in BRCA1 was present across 51 species from Chimp to Armadillo, the wild-type C at the position of 5,266 in BRCA1 was present across 83 species from Rhesus to Coelacanth, and the wild-type T at the position of 5,946 of BRCA2 was present across 87 species from Chimp to Tetraodon.

Of the 172 BRCA1 and 312 BRCA2 PLP variants shared with non-human species, the major types were stop-gain/nonsense variants (156 [90.1%] shared BRCA1 PLP variants and 280 [89%] shared BRCA2 PLP variants). Although frameshift indels constituted the majority of human BRCA PLP variants, none of them was present in other species (Table 1). The shared PLP variants did not enrich in specific functional domains, but distributed across coding region in both BRCA1 and BRCA2 (Fig S3A and B, P > 0.1): of the 388 of 2,972 (13%) BRCA1 PLP variants located in functional domains, only 13 (3.4%) were shared with other species (11 in BRCT domain and 2 in Coiled-Coil domain); of the 1,077 of 3,652 (29.5%) BRCA2 PLP variants in functional domains, only 54 (5.0%) were shared with other species (15 in BRC repeats, 38 in DBD, 1 in NLS and 8 in Transactivation domain) (Tables S3 and S4).

It shows that the shared human BRCA PLP variants were not enriched at specific functional domains. (A)
BRCA1. (B)
BRCA2. Different colors within coding region indicate the locations of functional domains. Generated by using the ProteinPaint program (https://proteinpaint.stjude.org/).

The results from our phylogenetic analysis reported above do not support evolution conservation as the major source of human BRCA PLP.

As our phylogenetic analysis did not find evidence to support evolution conservation as the major source of human BRCA PLP, we then tested whether human BRCA PLP could be originated from human itself. We performed an archeological analysis by searching human BRCA PLP in ancient human genome sequence data. We first searched the three Neanderthal and two Denisovan genome sequences (Green et al, 2010; Meyer et al, 2012; Prufer et al, 2014, 2017; Mafessoni et al, 2020), which were diverged from the ancestors of modern humans 400–800 thousand year ago (kya). We did not identify any matched human BRCA PLP variants. Next, we searched human BRCA PLP variants in 2,792 ancient human genome sequences dated from 45,000 to 300 yr ago. We identified 46 BRCA PLP variants in 50 ancient individuals, including 24 BRCA1 PLP variants in 28 individuals and 22 BRCA2 PLP variants in 22 individuals (Table 2 and Fig S1B), lived in Africa, Europe, Asia, Oceania, and Central and North America (Fig 3). Each matched PLP variant had dbSNP ID, further ensured the fidelity of the matched variants. The oldest matched BRCA1 PLP variant was BRCA1 c.181T>G (p.C61G) in an individual in Voronezh Oblast, Russia dated about 37,470 yr ago (Seguin-Orlando et al, 2014), the youngest matched BRCA1 PLP variant was BRCA1 c.2599C>T (Q867X) in an individual in Fujian, China dated 307 yr ago (Yang et al, 2020); the oldest matched BRCA2 PLP variant was BRCA2 c.9573G>A (p.W3191X) in an individual in Bor District, Serbia dated about 7,874 yr ago (Mathieson et al, 2018), the youngest matched BRCA2 PLP variant was BRCA2 c.8009C>T (p.S2670L) in an individual in Shefa, Vanuatu dated 350 yr ago (Lipson et al, 2020). Of the 24 matched BRCA1 PLP variants, 14 (58.3%) were stop-gain and six (25%) were located at the BRCT domain; of the 22 matched BRCA2 PLP variants, 15 (68.2%) were stop-gain and three were located at the nucleic acid binding domain. Four of the 24 BRCA1 PLP variants were detected in two individuals.

BRCA PLP variants identified in ancient humans.

Bold refers to the variants detected in more than one individual.

References listed in Table S6

Circle dots: matched BRCA1 PLPs; triangle dot: matched BRCA2 PLPs. The irregular dots: overlapped BRCA1 and BRCA2 PLPs. Colors in icons: time before present. The red circle dot was BRCA1 c.181T>G, dated 37,470 ± 1,210 yr ago (Table 2).

Many BRCA PLP variants are determined as BRCA founder PLP variants in different human ethnic populations, and their arisen times were determined by haplotyping analysis. For example, the arisen times for the three BRCA PLP founder variants (BRCA1 c.68_69del, BRCA1 c.5266dup, and BRCA2 c.5946del) in Ashkenazi Jews population were determined as 1,720, 1,800, and 580 yr ago, respectively (Table 3). We identified 34 BRCA founder PLP variants including 22 in BRCA1 and 12 in BRCA2. Their arisen times were from 3,225 to 140 yr ago (Table 3). For example, BRCA1 c.3228_3229del (p.G1077fs) was the oldest arisen 3,225 yr ago in Tuscany, Italy, BRCA1 c.1175_1214del (p.L345fs) was the youngest arisen 180 yr ago; BRCA2 c.9026_9030del (p.T3009fs) was the oldest arisen 2,760 yr ago in Spanish, BRCA2 c.9118-2A>G was the youngest arisen 220-144 yr ago in Finnish.

BRCA founder PLP variants dated by haplotype analysis.

references are listed in Table S6.

We also traced the ethnic origins for the BRCA PLP variants used in the study. Of the 1,054 BRCA PLP variants with available ethnic information, 548 (52%) were originated from single ethnic population, 327 (31%) were shared only between two ethnic populations. The rates were consistent in both BRCA1 and BRCA2 PLP variants (Table S5A–C).

Table S5 Ethnic origins of human BRCA PLP variants.

Table S5 Ethnic origins of human BRCA PLP variants.

### Phylogenetic analysis of human BRCA PLPs in non-human vertebrates

We identified a total of 6,624 BRCA PLP variants (2,972 in BRCA1 and 3,652 in BRCA2) from the ClinVar database for the study (Tables 1 and S2). We searched evidence for potential conservation of BRCA PLP variants between the humans and the 100 vertebrates distributed in eight clades of Primate, Euarchontoglires, Laurasiatheria, Afrotheria, Mammal, Aves, Sarcopterygii, and Fish. We identified 172 (5.8%) human BRCA1 PLP variants shared with 69 species, and 312 (8.6%) human BRCA2 PLP variants shared with 90 species (Figs 1, 2, and S1A and Tables S3 and S4).

Human BRCA PLP variants in 100 vertebrates.

Table S2 Human BRCA PLP variants used in the study.

Table S2 Human BRCA PLP variants used in the study.

Red cell: same as the human PLPs; empty cell: same base as human wild type; blue cell: different from human wild type and PLPs; gray cell: gaps; “-”: no aligned base; “=”: gap with at least one un-aligned base. The figure was generated using GraphPad Prism (version 9.0.0 for Windows, GraphPad Software). 185delAG, 5382insC: the BRCA1 founder pathogenic variants in Ashkenazi Jews population.

Ibid as in Fig 1. 6174delT: the BRCA2 founder pathogenic variants in Ashkenazi Jews population.

(A) Phylogenetic alignment. Red box shows the alignment of the four human BRCA PLP variants in 14 species in Aves clade. (B) Archeological alignment. It shows that three BRCA PLP variants were shared in three ancient cases. Red box: the BRCA PLP variants.

Table S3 Distribution of the 172 shared human BRCA1 PLP variants in other species.

Table S3 Distribution of the 172 shared human BRCA1 PLP variants in other species.

Table S4 Distribution of the 312 shared human BRCA2 PLP variants in other species.

Table S4 Distribution of the 312 shared human BRCA2 PLP variants in other species.

Of the BRCA1 PLP variants shared in the eight clades, Aves had the highest sharing number of 14 species and Sarcopterygii the second highest of eight species than other clades (P < 0.001) (Fig S2). For example, BRCA1 c.3268C>T (p.Gln1090Ter) was shared with 19 species from Rock pigeon in Aves to Lizard in Sarcopterygii, c.2498T>A (p.Leu833Ter) with 18 species from Saker falcon in Aves to Spiny softshell turtle in Sarcopterygii, and c.2138C>A (p.Ser713Ter) with 18 species from Rock pigeon in Aves to Lizard in Sarcopterygii; all 11 PLP variants located in the BRCA1 BRCT domain were shared with the species within Aves and Sarcopterygii clades. Tasmanian devil, a species with high risk of developing facial cancer, also shared 20 human BRCA1 PLP variants. In the shared BRCA2 PLP variants, the highest one including BRCA2 c.8933C>A (p.Ser2978Ter) shared with 36 species from Lesser Egyptian jerboa in Euarchontoglires to Opossum in mammal, and c.8933C>A (p.Ser2978Ter) shared with 36 species from Lesser Egyptian jerboa in Euarchontoglires to Opossum in Mammal to Spotted gar in Fish, the most distal species sharing this PLP. As the most used animal models in biomedical study, mouse shared eight and rat shared seven human BRCA1 PLP variants, of which only two were located at the BRCT domain. Zebrafish, another important biological model, shared 25 human BRCA2 PLP variants.

It shows that among the eight clades, Aves and Sarcopterygii had high sharing with human BRCA PLP, no sharing of BRCA PLP between human and primates of Chimp, Gorilla, Orangutan, Gibbon, Rhesus, Crab-eating macaque, and no sharing of human BRCA1 PLPs in Fish.

Of the species sharing with human BRCA PLP variants, Spiny softshell turtle in Sarcopterygii had the highest sharing number of 34 BRCA1 PLP variants, and Lizard in Sarcopterygii had the highest sharing number of 57 BRCA2 PLP variants. There were no human BRCA PLP variant shared with primates of Chimp, Gorilla, Orangutan, Gibbon, Rhesus, Crab-eating macaque and Baboon whereas there were 10 PLP variants shared with the distal primates of Marmoset, Squirrel monkey, and Bushbaby: Marmoset shared BRCA1 c.850C>T (p.Gln284Ter) and BRCA2 c.1642C>T (p.Gln548Ter), Squirrel monkey and Bushbaby shared an BRCA1 intronic c.1058G>A (p.Trp353Ter), Squirrel monkey and Bushbaby shared BRCA2 c.4689G>A (p.Trp1563Ter), and Bushbaby shared BRCA2 c.2651C>A (p.Ser884Ter), c.2978G>A (p.Trp993Ter), c.4689G>A (p.Trp1563Ter), c.3504G>T (p.Met1168Ile) and c.5263G>T (p.Glu1755Ter). Marmoset, Squirrel monkey, and Bushbaby had the divergent time of 40, 12.5, 15 million yr from hominins, accordingly. As control, we searched BRCA1 c. 68_69del, BRCA1 c.5266dup and BRCA2 c.5946del, the three BRCA founder PLP variants in Ashkenazi Jews population (Levy-Lahad et al, 1997), in the 100 vertebrates. We found no evidence for their presence in these species: the wildtype AG at the position of 68–69 in BRCA1 was present across 51 species from Chimp to Armadillo, the wild-type C at the position of 5,266 in BRCA1 was present across 83 species from Rhesus to Coelacanth, and the wild-type T at the position of 5,946 of BRCA2 was present across 87 species from Chimp to Tetraodon.

Of the 172 BRCA1 and 312 BRCA2 PLP variants shared with non-human species, the major types were stop-gain/nonsense variants (156 [90.1%] shared BRCA1 PLP variants and 280 [89%] shared BRCA2 PLP variants). Although frameshift indels constituted the majority of human BRCA PLP variants, none of them was present in other species (Table 1). The shared PLP variants did not enrich in specific functional domains, but distributed across coding region in both BRCA1 and BRCA2 (Fig S3A and B, P > 0.1): of the 388 of 2,972 (13%) BRCA1 PLP variants located in functional domains, only 13 (3.4%) were shared with other species (11 in BRCT domain and 2 in Coiled-Coil domain); of the 1,077 of 3,652 (29.5%) BRCA2 PLP variants in functional domains, only 54 (5.0%) were shared with other species (15 in BRC repeats, 38 in DBD, 1 in NLS and 8 in Transactivation domain) (Tables S3 and S4).

It shows that the shared human BRCA PLP variants were not enriched at specific functional domains. (A)
BRCA1. (B)
BRCA2. Different colors within coding region indicate the locations of functional domains. Generated by using the ProteinPaint program (https://proteinpaint.stjude.org/).

The results from our phylogenetic analysis reported above do not support evolution conservation as the major source of human BRCA PLP.

### Archeological analysis of BRCA PLP in ancient humans

As our phylogenetic analysis did not find evidence to support evolution conservation as the major source of human BRCA PLP, we then tested whether human BRCA PLP could be originated from human itself. We performed an archeological analysis by searching human BRCA PLP in ancient human genome sequence data. We first searched the three Neanderthal and two Denisovan genome sequences (Green et al, 2010; Meyer et al, 2012; Prufer et al, 2014, 2017; Mafessoni et al, 2020), which were diverged from the ancestors of modern humans 400–800 thousand year ago (kya). We did not identify any matched human BRCA PLP variants. Next, we searched human BRCA PLP variants in 2,792 ancient human genome sequences dated from 45,000 to 300 yr ago. We identified 46 BRCA PLP variants in 50 ancient individuals, including 24 BRCA1 PLP variants in 28 individuals and 22 BRCA2 PLP variants in 22 individuals (Table 2 and Fig S1B), lived in Africa, Europe, Asia, Oceania, and Central and North America (Fig 3). Each matched PLP variant had dbSNP ID, further ensured the fidelity of the matched variants. The oldest matched BRCA1 PLP variant was BRCA1 c.181T>G (p.C61G) in an individual in Voronezh Oblast, Russia dated about 37,470 yr ago (Seguin-Orlando et al, 2014), the youngest matched BRCA1 PLP variant was BRCA1 c.2599C>T (Q867X) in an individual in Fujian, China dated 307 yr ago (Yang et al, 2020); the oldest matched BRCA2 PLP variant was BRCA2 c.9573G>A (p.W3191X) in an individual in Bor District, Serbia dated about 7,874 yr ago (Mathieson et al, 2018), the youngest matched BRCA2 PLP variant was BRCA2 c.8009C>T (p.S2670L) in an individual in Shefa, Vanuatu dated 350 yr ago (Lipson et al, 2020). Of the 24 matched BRCA1 PLP variants, 14 (58.3%) were stop-gain and six (25%) were located at the BRCT domain; of the 22 matched BRCA2 PLP variants, 15 (68.2%) were stop-gain and three were located at the nucleic acid binding domain. Four of the 24 BRCA1 PLP variants were detected in two individuals.

BRCA PLP variants identified in ancient humans.

Bold refers to the variants detected in more than one individual.

References listed in Table S6

Circle dots: matched BRCA1 PLPs; triangle dot: matched BRCA2 PLPs. The irregular dots: overlapped BRCA1 and BRCA2 PLPs. Colors in icons: time before present. The red circle dot was BRCA1 c.181T>G, dated 37,470 ± 1,210 yr ago (Table 2).

### Dated BRCA founder PLP and ethnic-specific distribution of BRCA PLP

Many BRCA PLP variants are determined as BRCA founder PLP variants in different human ethnic populations, and their arisen times were determined by haplotyping analysis. For example, the arisen times for the three BRCA PLP founder variants (BRCA1 c.68_69del, BRCA1 c.5266dup, and BRCA2 c.5946del) in Ashkenazi Jews population were determined as 1,720, 1,800, and 580 yr ago, respectively (Table 3). We identified 34 BRCA founder PLP variants including 22 in BRCA1 and 12 in BRCA2. Their arisen times were from 3,225 to 140 yr ago (Table 3). For example, BRCA1 c.3228_3229del (p.G1077fs) was the oldest arisen 3,225 yr ago in Tuscany, Italy, BRCA1 c.1175_1214del (p.L345fs) was the youngest arisen 180 yr ago; BRCA2 c.9026_9030del (p.T3009fs) was the oldest arisen 2,760 yr ago in Spanish, BRCA2 c.9118-2A>G was the youngest arisen 220-144 yr ago in Finnish.

BRCA founder PLP variants dated by haplotype analysis.

references are listed in Table S6.

We also traced the ethnic origins for the BRCA PLP variants used in the study. Of the 1,054 BRCA PLP variants with available ethnic information, 548 (52%) were originated from single ethnic population, 327 (31%) were shared only between two ethnic populations. The rates were consistent in both BRCA1 and BRCA2 PLP variants (Table S5A–C).

Table S5 Ethnic origins of human BRCA PLP variants.

Table S5 Ethnic origins of human BRCA PLP variants.

### Discussion

Our study analyzed the evolutionary origin of BRCA PLP in modern human population. Our phylogenetic analysis across 100 vertebrates found no direct evidence for cross-species evolution conservation as the source for human BRCA PLP. Our archeological analysis in 2,792 ancient human individuals dated back to 45,000 yr ago identified 46 BRCA PLP variants of which 45 were arisen within the last 10,000 yr. Our analysis in the haplotyping-dated human BRCA founder PLP variants observed that nearly all were arisen within the last 3,000 yr. We further traced the ethnic origins of the BRCA PLP variants used in the study and observed that the majority were present in single or few ethnic populations. Based on these observations, we propose that human BRCA PLP was most likely arisen in recent human history, possibly within a few thousand years, after the latest human out-of-Africa migration and settlement at different global destinations. We consider that the positive selection on human BRCA could play a major role, and the population expansion could further increase the spectrum of human BRCA PLP in modern human population (Fig 4).

It shows that the positive selection on human BRCA causes high human BRCA variation and accompanied PLP variation, and the expansion of human population further increases the spectrum of human BRCA PLP. X-axis: the timing of modern human after out-of-Africa migration; Left y-axis: size of human population; Right y-axis: number of human BRCA PLP variants. Curve: the growth of human population in the past 50,000 yr.

The prevalence of BRCA PLP is between 0.2% and 0.5% in modern human population. For example, the prevalence is 0.26% in Japanese population (Momozawa et al, 2018), 0.29% in Macau population (Qin et al, 2021), 0.38% in Chinese population (Dong et al, 2021), 0.38% in Mexican population (Fernandez-Lopez et al, 2019), 0.39% in Malaysian population (Wen et al, 2018), 0.53% in Taiwanese population (Chian et al, 2021), and 0.53% in Caucasian populations (Kurian et al, 2019). That implies that one in several hundreds of human individuals carries a BRCA PLP variant. The prevalence of BRCA PLP can be the highest in disease-causing genetic predisposition genes in human. It is interesting to understand why BRCA PLP can reach such high level in human population regardless their deleterious impact. Possible explanations can be (1). The cancer caused by BRCA PLP occurs mostly at later reproduction stage. Before reaching the stage, the PLP has already been transmitted to the next generation; (2). The loss of tumor suppressing function of BRCA due to BRCA PLP could be developmental stage-dependent. It states that the PLP could be beneficial at the reproductive stage but deleterious at later reproduction stage. Positive selection imposed on the reproductive stage can select the BRCA PLP not deleterious at the stage. However, our current study does not determine whether one or both of the explanations could contribute to the higher prevalence of BRCA PLP in modern human population.

Extensive animal model studies, especially from mouse studies, have provided rich evidence showing the pathogenic consequences by BRCA PLP in non-human species. Brca1 knockout-mice showed abnormal post-implantation development and embryonic proliferation (Liu et al, 1996; Hakem et al, 1996), embryonic lethality (Ludwig et al, 1997), neuroepithelial abnormalities (Gowen et al, 1996), irradiation hypersensitivity and genetic instability (Shen et al, 1998), abnormal T cell development (Mak et al, 2000; Xu et al, 2001), and tumorigenesis (Xu et al, 1999; Brodie et al, 2001). Without the presence of other mutated genes such as the mutated TP53, however, the mutated Brca1 alone is not sufficient to directly cause cancer (Xu et al, 2001). This can further explain that the higher prevalence of BRCA PLP in human may have less deleterious impact in the absence of synergetic effects from other genetic events.

Many human BRCA PLP variants were shared with species in Aves and Sarcopterygii clads. This could happen by chance rather than by cross-species conservation, as it is unlikely that the conservation would allow the presence of the gaps across such wide distances. Alternatively, it could also be related to species-specific pathogenicity of the same BRCA PLP variants that the BRCA PLPs in humans may not be pathogenic in non-human species (Gao & Zhang, 2003). For example, Tasmanian devil shared 20 human BRCA PLP variants. Tasmanian has a high risk of developing facial cancer, which is related with the fusion of chromosome 1 and X (Hawkins et al, 2006; Murchison et al, 2010; Taylor et al, 2017), but no evidence to show that the cancer is related to the BRCA PLP variants shared with human. A study analyzed human deleterious mutations in multiple genes including BRCA shared with mouse (Gao & Zhang, 2003), and evaluated multiple theories to explain the biological correlation including the “funder effect,” “fixations of slightly deleterious mutations,” “relaxed selection on late-onset phenotypes,” and “compensatory changes.” The study considered that the “compensatory changes,” which stated that “compensatory mutations at other sites of the same or a different protein render the deleterious mutations neutral,” can be the best to explain the conservation of human deleterious mutation with the distant species of mouse. We consider that the compensation theory can also be used to explain the sharing of human BRCA PLPs with the species in Aves and Sarcopterygii clads, although direct evidence is not available to validate the explanation.

Only limited numbers of BRCA PLP variants were identified in the ancient humans. This can be related to the limited data available from ancient humans. In addition, genomic sequences in many ancient samples had limited coverage due to the rarity and poor quality of ancient DNA. Furthermore, the size differences between ancient and modern human populations can also be a factor. It is estimated that the number of indivuduals in the latest out-of-Africa migration 65,000–50,000 yr ago was about 1,000–2,500 (Henn et al, 2012), the size of the human population in 1,800s was about 1 billion, and the size of modern human population is close to eight billion (https://ourworldindata.org/world-population-growth, https://data.worldbank.org/indicator/SP.POP.TOTL). All human BRCA PLP variants used in our current study were derived from the current human population. With the positive selection, continually increased human population size, and more powerful DNA sequencing technologies under developing, it is expected that more new human BRCA PLP variants would arise and be identified.

Human BRCA1 has 1,863 residues and BRCA2 has 3,418 residues. Although gene with large size can have more chance to generate more genetic variants, it is unlikely that the size in BRCA is a major factor for the large quantity of human BRCA PLP. For example, our study showed that mouse Brca does not change although the size of mouse Brca is very close to the human BRCA (Brca1 1,812 residues and Brca2 3,329 residues) (Wang & Wang, 2021). Another factor related with ethnic-specific PLPs can be that the same BRCA PLP could arise by chance in different ethnic populations at different times. For example, the BRCA1 68-69del founder variant in Ashkenazi Jews is also present in other ethnic populations although at very low frequency, and BRCA1 3607C>T shared between the Slovakia case dated 4,121 ± 100 ago and Dominica case dated 921 ± 500 ago (Table 2). However, this unlikely played a major role in contributing to human BRCA PLP. Pleiotropy effects may also exist that variation in non-BRCA genes may contribute to the selection of BRCA PLPs, particularly for these selected in the post-reproductive age (Williams, 1957). This is particularly interesting as BRCA PLP mainly cause high cancer risk after reproductive age. Homozygotic BRCA PLP carriers seldom survive or develop Fanconi anemia due to the embryonic lethal effect, as BRCA is essential for development (Seo et al, 2018).

A limitation of our study is that most of the BRCA founder PLP variants and ethnic distribution data used in the study were derived from European and descendants. More data from non-European populations should enhance the significance of our study.

### Materials and Methods

Human BRCA PLP variants were from ClinVar database (https://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh38/weekly/clinvar_20201031.vcf.gz, accessed 1 November 2020). Only the variants classified as “pathogenic,” “likely pathogenic,” and “pathogenic and likely pathogenic” were included in the study. The variants with conflict classifications were excluded to ensure the reliability of the analysis.

Human BRCA PLP variants were divided into single nucleotide variants and indel groups. Their genomic positions were annotated by referring to the following reference sequences: BRCA1: genome hg38 NC_000017.11, cDNA NM_007294, NP_009225; BRCA2: genome hg38 NC_000013.11, cDNA NM_000059, NP_000050. BRCA1, and BRCA2 sequences across 100 vertebrate species were downloaded from UCSC Genome Browser (https://genome.ucsc.edu/cgi-bin/hgGateway?redirect=manual&source=genome.ucsc.edu). Sequence alignment followed the procedures in Multiz Alignments of 100 Vertebrates part in UCSC genome browser (https://genome.ucsc.edu/cgi-bin/hgc?db=hg38&c=chr17&l=43106526&r=43106527&o=43106526&t=43106527&g=multiz100way&i=multiz100way). Tree model of the 100 vertebrate species was generated using the phyloFit program in the PHAST package (Murphy et al, 2001). The phylogenetic tree for the 100 vertebrate species was from UCSC resource (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/multiz100way/), and the distance between species on the tree was adjusted to ensure the readability in the figures. PhastCons and phyloP from the PHAST package were used for evolution conversion measurement. Sequence alignment between repeat-masked human hg38 and non-human genome sequences was made by using Lastz (BLASTZ) and multiz (Blanchette et al, 2004; Hubisz et al, 2011; Armstrong et al, 2019; Ramani et al, 2019). Based on the phylogenetic distance from the references, the scoring matrix and parameters for pairwise were adjusted for each species. The high-score chains were placed along genome sequences, and the low-score chains were used to fill gaps. The results of single base-level alignment were collected from the “Multiz Alignments of 100 Vertebrates” section in UCSC genome browser by entering variant position in hg38 using a Python-based tool (https://github.com/Skylette14/GetBase). For indel alignments, the base number of insertion or deletion between human and other species was the same. Each indel alignment across the matched species were manually checked to ensure reliability of the alignment. The positions corresponding to human PLP variants in the aligned non-human sequences were obtained by using GetBase.

Ancient human DNA sequences and related publications were from “Allen Ancient DNA Resource (version 42.2, https://reich.hms.harvard.edu/allen-ancient-dna-resource-aadr-downloadable-genotypes-present-day-and-ancient-dna-data, accessed 1 March 2020), containing genomic sequences from a total of 2,792 ancient human individuals dated from 37,470 to 300 before present. Bam files of ancient genomic sequences were downloaded. The sequences containing BRCA1 (chr17:41,196,312-41,277,500, hg19 by Ensembl) and BRCA2 (chr13:32,889,611-32,973,805, hg19 by Ensembl) were identified and used for mapping to hg19. Mpileup command in SAMtools was used for variant calling from the mapped sequences with the minimal base quality set as 1 (Li et al, 2009). After generation of ancient human vcf files, the called variants were annotated using wANNOVAR (https://wannovar.wglab.org/) (Chang & Wang, 2012), compared with human BRCA PLP variants and manually checked in ClinVar to obtain the related information for these matched by the ancient BRCA sequences. The locations for the ancient PLP carrier’s fossil excavation and the estimated age were based on the original publications. The geographical distribution map of the ancient BRCA PLP variants were generated by using Matlab (The MathWorks, Inc.).

Kruskal–Wallis test was used for statistical comparison for the BRCA PLP variants shared between clades (GraphPad Software). Chi-Squared test was used to compare the distribution of the shared BRCA PLP variants, P < 0.01 was considered as statistical significance.

### Source of human BRCA PLPs

Human BRCA PLP variants were from ClinVar database (https://ftp.ncbi.nlm.nih.gov/pub/clinvar/vcf_GRCh38/weekly/clinvar_20201031.vcf.gz, accessed 1 November 2020). Only the variants classified as “pathogenic,” “likely pathogenic,” and “pathogenic and likely pathogenic” were included in the study. The variants with conflict classifications were excluded to ensure the reliability of the analysis.

### Vertebrate genomic mapping analysis

Human BRCA PLP variants were divided into single nucleotide variants and indel groups. Their genomic positions were annotated by referring to the following reference sequences: BRCA1: genome hg38 NC_000017.11, cDNA NM_007294, NP_009225; BRCA2: genome hg38 NC_000013.11, cDNA NM_000059, NP_000050. BRCA1, and BRCA2 sequences across 100 vertebrate species were downloaded from UCSC Genome Browser (https://genome.ucsc.edu/cgi-bin/hgGateway?redirect=manual&source=genome.ucsc.edu). Sequence alignment followed the procedures in Multiz Alignments of 100 Vertebrates part in UCSC genome browser (https://genome.ucsc.edu/cgi-bin/hgc?db=hg38&c=chr17&l=43106526&r=43106527&o=43106526&t=43106527&g=multiz100way&i=multiz100way). Tree model of the 100 vertebrate species was generated using the phyloFit program in the PHAST package (Murphy et al, 2001). The phylogenetic tree for the 100 vertebrate species was from UCSC resource (http://hgdownload.soe.ucsc.edu/goldenPath/hg38/multiz100way/), and the distance between species on the tree was adjusted to ensure the readability in the figures. PhastCons and phyloP from the PHAST package were used for evolution conversion measurement. Sequence alignment between repeat-masked human hg38 and non-human genome sequences was made by using Lastz (BLASTZ) and multiz (Blanchette et al, 2004; Hubisz et al, 2011; Armstrong et al, 2019; Ramani et al, 2019). Based on the phylogenetic distance from the references, the scoring matrix and parameters for pairwise were adjusted for each species. The high-score chains were placed along genome sequences, and the low-score chains were used to fill gaps. The results of single base-level alignment were collected from the “Multiz Alignments of 100 Vertebrates” section in UCSC genome browser by entering variant position in hg38 using a Python-based tool (https://github.com/Skylette14/GetBase). For indel alignments, the base number of insertion or deletion between human and other species was the same. Each indel alignment across the matched species were manually checked to ensure reliability of the alignment. The positions corresponding to human PLP variants in the aligned non-human sequences were obtained by using GetBase.

### Ancient human genomic mapping analysis

Ancient human DNA sequences and related publications were from “Allen Ancient DNA Resource (version 42.2, https://reich.hms.harvard.edu/allen-ancient-dna-resource-aadr-downloadable-genotypes-present-day-and-ancient-dna-data, accessed 1 March 2020), containing genomic sequences from a total of 2,792 ancient human individuals dated from 37,470 to 300 before present. Bam files of ancient genomic sequences were downloaded. The sequences containing BRCA1 (chr17:41,196,312-41,277,500, hg19 by Ensembl) and BRCA2 (chr13:32,889,611-32,973,805, hg19 by Ensembl) were identified and used for mapping to hg19. Mpileup command in SAMtools was used for variant calling from the mapped sequences with the minimal base quality set as 1 (Li et al, 2009). After generation of ancient human vcf files, the called variants were annotated using wANNOVAR (https://wannovar.wglab.org/) (Chang & Wang, 2012), compared with human BRCA PLP variants and manually checked in ClinVar to obtain the related information for these matched by the ancient BRCA sequences. The locations for the ancient PLP carrier’s fossil excavation and the estimated age were based on the original publications. The geographical distribution map of the ancient BRCA PLP variants were generated by using Matlab (The MathWorks, Inc.).

### Statistical analysis

Kruskal–Wallis test was used for statistical comparison for the BRCA PLP variants shared between clades (GraphPad Software). Chi-Squared test was used to compare the distribution of the shared BRCA PLP variants, P < 0.01 was considered as statistical significance.

### Data Availability

All data used in the study were from public domains as indicated in the text. The data generated from the study were provided as online Figs S1–S3 and Tables S1–S6.

Table S6 References of ancient human genomes and BRCA founder PLP variants.

Table S6 References of ancient human genomes and BRCA founder PLP variants.

### Supplementary Material



# SUPPLEMENTAL FILE 1: LSA-2021-01263.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 7: clinvar_20201031.vcf.gz

[File available at: automated_output/BRCA2/20251125_175029/pmc_fulltext/35165121_supplements/clinvar_20201031.vcf.gz]

