# MAIN TEXT

## Undiagnosed disease program in South Africa: Results from first 100 exomes

### Abstract

AbstractThe Undiagnosed Disease Program in South Africa (UDP) sought to prospectively evaluate the clinical utility of exome sequencing (ES) in a phenotypically diverse, multi‐ethnic cohort of South African patients with suspected rare genetic disorders. ES was undertaken in 100 sequential patients (93 singletons, 3 duos, and 4 trios) recruited to the UDP at Stellenbosch University. The data were analyzed through two separate bioinformatics pipelines (EVIDENCE from 3 billion and our in‐house pipeline). A definitive diagnosis could be reached in 51% (51/100) patients, with 46% (46/100) patients having either pathogenic or likely pathogenic single‐nucleotide variants/indels (SNVs/indels), and 5 patients with likely‐pathogenic copy number variants (CNVs) (5/100). The CNVs were subsequently confirmed on microarray or MLPA analysis. Detailed phenotyping and HPO terms enabled analysis and variant identification. Twenty‐five novel variants in 22 genes are reported here. We provide data from the first year of this UDP and show that even amongst mainly singletons from an understudied, diverse African population, ES is a valuable diagnostic tool, especially if it includes CNV analysis. The remaining undiagnosed patients present a unique opportunity for further research and novel gene discovery.

### INTRODUCTION

Approximately, 4 million South Africans are affected by a rare disease, the majority of which have a genetic basis. The South African population is diverse, aptly named the “rainbow nation.” Traditionally, genetic research and testing has focused on specific subgroups, in whom we could rely on presumed underlying factors, such as founder effects, to make variant identification more straightforward. The majority of South African patients with rare monogenic disorders remain undiagnosed due to the lack of access to next‐generation sequencing (NGS) technology. This has meant that the majority of our population of patients with rare diseases has been both understudied and under‐represented in databases and the literature. NGS‐based diagnostics have proven very effective in several large studies performed internationally, with diagnostic rates up to 58% for exome sequencing (ES; Stark et al., 2016). While most studies have recruited patients for ES after extensive prior testing, the American College of Medical Genetics and Genomics (ACMG) recently recommended that ES or genome sequencing (GS) should be considered first‐ or second‐tier testing in patients with congenital anomalies, developmental delay, or intellectual disability (Manickam et al., 2021). We established the Undiagnosed Disease Program (UDP) at Stellenbosch University to test the utility of ES in our low‐resourced setting and in our diverse patient population. This is one of few studies evaluating ES in sub‐Saharan Africa, and the first UDP to enroll undiagnosed patients with a broad range of suspected monogenic disorders. Here, we present the results of the first 100 patients tested in our UDP.

### MATERIALS AND METHODS

Patients were recruited to our “Undiagnosed Disease Program” (UDP) and offered whole‐exome sequencing (ES). The majority received singleton ES, while in 7 families, trio‐ES or duo‐ES was performed. Eligibility criteria: suspected rare monogenic disorder amenable to diagnosis by ES, still undiagnosed at time of recruitment, in‐depth clinical information available, and consent to be part of the program. Patients were either seen directly by the medical geneticist, or relevant clinical information and images were reviewed, in cases where patients were referred from other centers. Patients did not necessarily have to have had any previous genetic testing, as even chromosomal microarray may not readily be available in certain parts of South Africa. The study population was unique and a study of this kind had not been undertaken on a similar scale previously and includes participants from a population not extensively included in ES studies. At Tygerberg Hospital (TBH), ES testing was run in parallel with standard clinical testing (this could include microarray and a gene panel, if available). Recruitment started in October 2020. Formal informed consent was obtained from patients/parents/guardians for the study and followed a standard genomic testing process, which included offering the families the option to receive secondary findings. The study was approved by the Health Research Ethics Committee of the Faculty of Medicine and Health Sciences at Stellenbosch University (Ethics ref: N/18/03/031).

Blood, saliva, or buccal swab samples were collected from each patient (and parents, where applicable), and genomic DNA was isolated, according to the standard protocols. All exon regions of all human genes (~22,000) were captured using the IDT xGen Exome Research Panel V2 (Integrated DNA Technologies, Coralville, Iowa, USA). Sequencing was performed on the NovaSeq 6000 (Illumina, San Diego, CA) at 3billion Inc. The mean depth of coverage was 100X (>10X = 99·2%).

Data analysis and variant prioritization was accomplished using two separate pipelines, namely the automated 3billion Inc and our in‐house analysis pipeline. The in‐house pipeline is based on bcbio‐nextgen (https://doi.org/10.5281/zenodo.3564938). This pipeline comprises open‐source software and is semi‐automated. Briefly, the following steps were followed: Quality of the sequencing data were checked using FASTQC (http://www.bioinformatics.babraham.ac.uk/projects/fastqc/). Subsequently, the base and sequence adapters with low base quality were removed using Trimmomatic (Bolger et al., 2014). Pre‐processed FASTQ files were aligned to the reference (GRCh37/19) by BWA‐MEM (v.0.7.17; Li & Durbin, 2009). Aligned BAM files were sorted and extracted using samtools (v.1.9; Li et al., 2009). Duplicates were marked using Picard (v.2.20.8; http://broadinstitute.github.io/picard/). Single nucleotide variants (SNVs) and indel variants were called by HaplotypeCaller of GATK (v.3.8; McKenna et al., 2010). CNV analysis was undertaken using ExomeDepth and Pindel (Plagnol et al., 2012; Ye et al., 2018). Finally, variant call files (VCF) were generated. 3billion Inc's automated variant prioritization software, EVIDENCE, was used to prioritize variants based on the phenotype (HPO terms) of each patient (Seo et al., 2020). Our in‐house strategy was as follows: Variants were filtered using vcftools (Danecek et al., 2011). This included the following parameters: a Minor Allele Frequency (MAF) of <0.01 for the selection of rare variants, a minimum quality (minQ) of >30, and a minimum depth (minDP) of >10. Following filtering, annotation was performed using Annovar's filter‐based annotation (Wang et al., 2010). This step annotates the filtered VCF files with in silico predictive algorithms such as SIFT, Polyphen, LRT, Mutation Taster, Mutation Assessor, FATHMM, MetaSVM, MetaLR predictive scores. Additionally, it used GERP++ and CADD scores. Allele frequency was also indicated using databases such as gnomAD, ExAC, and the 1000 Genome Project. Variant classification followed ACMG/AMP guidelines (Richards et al., 2015; Table 1), using EVIDENCE, VarSome (Kopanos et al., 2019), and manual classification mainly for training purposes.

Results from the 46 patients diagnosed with an SNV/indel variants‐(*) indicate novel variants

Note: (*) = novel variant; † = age at death; ¥ = Published: Coetzer and Moosa, AJMG, 2021; § = submitted to ClinVar by 3billion only; §§ = submitted to ClinVar by 3billion and others.

Abbreviations: AD, autosomal dominant; AR, autosomal recessive; Dysm, dysmorphic features; ES, exome sequencing; F, female; FISH, flourescence in‐situ hybridization; GDD, global developmental delay; LP, likely pathogenic; M, male; MCA, Multiple congenital anomalies; MLPA, Multiplex ligation dependent probe amplification; MPS, Mucopolysaccharidosis; Bold fields with PD: Provisional diagnosis as 1 variant of an AR disorder still classified as VUS; Path, pathogenic; SMA, Spinal muscular atrophy; VUS, variant of uncertain significance; XL, X‐linked.

Turn‐around time for ES was 4 to 6 weeks. Raw data were returned to the UDP for analysis. Time to return of results varied based on whether further testing was needed. All results were returned in the context of genetic counseling and with a formal ES report.

### Recruitment

Patients were recruited to our “Undiagnosed Disease Program” (UDP) and offered whole‐exome sequencing (ES). The majority received singleton ES, while in 7 families, trio‐ES or duo‐ES was performed. Eligibility criteria: suspected rare monogenic disorder amenable to diagnosis by ES, still undiagnosed at time of recruitment, in‐depth clinical information available, and consent to be part of the program. Patients were either seen directly by the medical geneticist, or relevant clinical information and images were reviewed, in cases where patients were referred from other centers. Patients did not necessarily have to have had any previous genetic testing, as even chromosomal microarray may not readily be available in certain parts of South Africa. The study population was unique and a study of this kind had not been undertaken on a similar scale previously and includes participants from a population not extensively included in ES studies. At Tygerberg Hospital (TBH), ES testing was run in parallel with standard clinical testing (this could include microarray and a gene panel, if available). Recruitment started in October 2020. Formal informed consent was obtained from patients/parents/guardians for the study and followed a standard genomic testing process, which included offering the families the option to receive secondary findings. The study was approved by the Health Research Ethics Committee of the Faculty of Medicine and Health Sciences at Stellenbosch University (Ethics ref: N/18/03/031).

### Exome sequencing

Blood, saliva, or buccal swab samples were collected from each patient (and parents, where applicable), and genomic DNA was isolated, according to the standard protocols. All exon regions of all human genes (~22,000) were captured using the IDT xGen Exome Research Panel V2 (Integrated DNA Technologies, Coralville, Iowa, USA). Sequencing was performed on the NovaSeq 6000 (Illumina, San Diego, CA) at 3billion Inc. The mean depth of coverage was 100X (>10X = 99·2%).

### Variant identification

Data analysis and variant prioritization was accomplished using two separate pipelines, namely the automated 3billion Inc and our in‐house analysis pipeline. The in‐house pipeline is based on bcbio‐nextgen (https://doi.org/10.5281/zenodo.3564938). This pipeline comprises open‐source software and is semi‐automated. Briefly, the following steps were followed: Quality of the sequencing data were checked using FASTQC (http://www.bioinformatics.babraham.ac.uk/projects/fastqc/). Subsequently, the base and sequence adapters with low base quality were removed using Trimmomatic (Bolger et al., 2014). Pre‐processed FASTQ files were aligned to the reference (GRCh37/19) by BWA‐MEM (v.0.7.17; Li & Durbin, 2009). Aligned BAM files were sorted and extracted using samtools (v.1.9; Li et al., 2009). Duplicates were marked using Picard (v.2.20.8; http://broadinstitute.github.io/picard/). Single nucleotide variants (SNVs) and indel variants were called by HaplotypeCaller of GATK (v.3.8; McKenna et al., 2010). CNV analysis was undertaken using ExomeDepth and Pindel (Plagnol et al., 2012; Ye et al., 2018). Finally, variant call files (VCF) were generated. 3billion Inc's automated variant prioritization software, EVIDENCE, was used to prioritize variants based on the phenotype (HPO terms) of each patient (Seo et al., 2020). Our in‐house strategy was as follows: Variants were filtered using vcftools (Danecek et al., 2011). This included the following parameters: a Minor Allele Frequency (MAF) of <0.01 for the selection of rare variants, a minimum quality (minQ) of >30, and a minimum depth (minDP) of >10. Following filtering, annotation was performed using Annovar's filter‐based annotation (Wang et al., 2010). This step annotates the filtered VCF files with in silico predictive algorithms such as SIFT, Polyphen, LRT, Mutation Taster, Mutation Assessor, FATHMM, MetaSVM, MetaLR predictive scores. Additionally, it used GERP++ and CADD scores. Allele frequency was also indicated using databases such as gnomAD, ExAC, and the 1000 Genome Project. Variant classification followed ACMG/AMP guidelines (Richards et al., 2015; Table 1), using EVIDENCE, VarSome (Kopanos et al., 2019), and manual classification mainly for training purposes.

Results from the 46 patients diagnosed with an SNV/indel variants‐(*) indicate novel variants

Note: (*) = novel variant; † = age at death; ¥ = Published: Coetzer and Moosa, AJMG, 2021; § = submitted to ClinVar by 3billion only; §§ = submitted to ClinVar by 3billion and others.

Abbreviations: AD, autosomal dominant; AR, autosomal recessive; Dysm, dysmorphic features; ES, exome sequencing; F, female; FISH, flourescence in‐situ hybridization; GDD, global developmental delay; LP, likely pathogenic; M, male; MCA, Multiple congenital anomalies; MLPA, Multiplex ligation dependent probe amplification; MPS, Mucopolysaccharidosis; Bold fields with PD: Provisional diagnosis as 1 variant of an AR disorder still classified as VUS; Path, pathogenic; SMA, Spinal muscular atrophy; VUS, variant of uncertain significance; XL, X‐linked.

Turn‐around time for ES was 4 to 6 weeks. Raw data were returned to the UDP for analysis. Time to return of results varied based on whether further testing was needed. All results were returned in the context of genetic counseling and with a formal ES report.

### RESULTS

Of the 100 patients enrolled in the UDP presented here, 53 were females and 47 males. The average age at testing was 5 years 2 months. The youngest patients were stillborn babies (with lethal skeletal dysplasias, Patients 21 and 71) and the oldest, a 45 year old with familial osteopetrosis (Patient 94). Ten diagnosed patients were under age 1. Previous (negative) testing had been conducted in 51 patients (51%). The study population is diverse, representing understudied ancestral and language groups in South Africa. Patients were mainly recruited from the Western Cape Province: 45% South Africans of mixed ancestry, 44% South Africans of African ancestry (mainly isiXhosa speaking), and 10% South Africans of European ancestry (mainly Afrikaans speaking). Only one family (Patient 100) was known to be consanguineous (family originally from Pakistan).

A cumulative diagnostic rate of 51% (51/100) was achieved (Figure 1). In 46 patients, we identified disease‐causing SNVs/indels (Table 1), and in 5 patients, a likely causative copy number variant (CNV) was identified, and subsequently confirmed on either microarray or MLPA analysis (Table 2). The spectrum of inheritance patterns across variants was as follows: 24 autosomal dominant (AD) (including 8 de novo), 17 autosomal recessive (11 homozygous, 6 compound heterozygous), and 5 X‐linked. The average age at diagnosis was 6 years 4 months. Of those diagnosed, 18/46 (39%) had negative prior testing (including karyotype, microarray, gene panels, other genetic and metabolic testing) versus no testing at all (61%) (Table 1). We identified 25 novel variants in 22 genes (Table 1). A well‐known pathogenic variant in PGAP3 was identified in three unrelated patients (Patients 22, 54, 81; Bezuidenhout et al., 2020). Patients 31 and 58 were siblings with the same COL4A4 homozygous pathogenic variant. The only patient (Patient 100) known to be from a consanguineous family was found to be homozygous for a pathogenic variant in ARSA.

Diagnostic yield in this cohort (%)

Results from the five patients with pathogenic CNVs

Several patients diagnosed here are likely the first and only ones in sub‐Saharan Africa with a confirmed diagnosis of their particular rare disease: notably, Patient 40 with CHD3‐ related Snijders Blok–Campeau syndrome, Patient 41 with CTNNB1‐related neurodevelopmental disorder with spastic diplegia and visual defects (NEDSDV), Patient 46 with SETD5‐related MRD25, and Patient 64 with USP9X‐related MRXS99F. Patient 14 with NONO‐related MRXS34 was recently published as the 12th person worldwide with his ultra‐rare disorder (Coetzer & Moosa, 2022).

Sanger sequencing of parents or information available from in‐house patients proved that the variants initially identified as VUSs were confirmed de novo, or in trans, or present in multiple unrelated patients, allowing re‐classification as likely‐pathogenic/pathogenic in certain cases – Families 3, 4, 11, 33 (de novo) and Families 5, 6, 22, 25, 50, 51, 81 (homozygous/compound heterozygous), respectively (Table 1). Three patients had a gene panel concurrently, confirming the variants and diagnosis they received through the UDP (Families 13, 32, and 77) (Table 1).

Thirty‐seven patients had a negative report on initial analysis (Table S1). In the remaining 12 patients, VUSs were identified, which are awaiting resolution/reverse phenotyping/family studies (Table S1).

Patients were assigned to one of ten diagnostic groups, based on their major phenotypic features (Figure 2). Diagnostic rates differed depending on diagnostic group, with the majority of the skeletal disorders being diagnosed (64%) compared with disorders of growth (22%). Approximately, half of the patients with dysmorphic features or multiple congenital anomalies were diagnosed (Figure 2).

Diagnostic yield by disease group

The parents/guardians were given the option to receive secondary findings. Only four families consented to receiving these results, with one known pathogenic variant in BRCA2 identified in Family 13 (NM_007294.4: c.181 T > C; p.Cys61Gly) (ClinVar VCV000017661.55). Family members have received follow‐up genetic counseling, and further members were offered testing and were/will be enrolled in appropriate cancer surveillance programs.

### Patient demographics

Of the 100 patients enrolled in the UDP presented here, 53 were females and 47 males. The average age at testing was 5 years 2 months. The youngest patients were stillborn babies (with lethal skeletal dysplasias, Patients 21 and 71) and the oldest, a 45 year old with familial osteopetrosis (Patient 94). Ten diagnosed patients were under age 1. Previous (negative) testing had been conducted in 51 patients (51%). The study population is diverse, representing understudied ancestral and language groups in South Africa. Patients were mainly recruited from the Western Cape Province: 45% South Africans of mixed ancestry, 44% South Africans of African ancestry (mainly isiXhosa speaking), and 10% South Africans of European ancestry (mainly Afrikaans speaking). Only one family (Patient 100) was known to be consanguineous (family originally from Pakistan).

### Diagnostic rate

A cumulative diagnostic rate of 51% (51/100) was achieved (Figure 1). In 46 patients, we identified disease‐causing SNVs/indels (Table 1), and in 5 patients, a likely causative copy number variant (CNV) was identified, and subsequently confirmed on either microarray or MLPA analysis (Table 2). The spectrum of inheritance patterns across variants was as follows: 24 autosomal dominant (AD) (including 8 de novo), 17 autosomal recessive (11 homozygous, 6 compound heterozygous), and 5 X‐linked. The average age at diagnosis was 6 years 4 months. Of those diagnosed, 18/46 (39%) had negative prior testing (including karyotype, microarray, gene panels, other genetic and metabolic testing) versus no testing at all (61%) (Table 1). We identified 25 novel variants in 22 genes (Table 1). A well‐known pathogenic variant in PGAP3 was identified in three unrelated patients (Patients 22, 54, 81; Bezuidenhout et al., 2020). Patients 31 and 58 were siblings with the same COL4A4 homozygous pathogenic variant. The only patient (Patient 100) known to be from a consanguineous family was found to be homozygous for a pathogenic variant in ARSA.

Diagnostic yield in this cohort (%)

Results from the five patients with pathogenic CNVs

Several patients diagnosed here are likely the first and only ones in sub‐Saharan Africa with a confirmed diagnosis of their particular rare disease: notably, Patient 40 with CHD3‐ related Snijders Blok–Campeau syndrome, Patient 41 with CTNNB1‐related neurodevelopmental disorder with spastic diplegia and visual defects (NEDSDV), Patient 46 with SETD5‐related MRD25, and Patient 64 with USP9X‐related MRXS99F. Patient 14 with NONO‐related MRXS34 was recently published as the 12th person worldwide with his ultra‐rare disorder (Coetzer & Moosa, 2022).

Sanger sequencing of parents or information available from in‐house patients proved that the variants initially identified as VUSs were confirmed de novo, or in trans, or present in multiple unrelated patients, allowing re‐classification as likely‐pathogenic/pathogenic in certain cases – Families 3, 4, 11, 33 (de novo) and Families 5, 6, 22, 25, 50, 51, 81 (homozygous/compound heterozygous), respectively (Table 1). Three patients had a gene panel concurrently, confirming the variants and diagnosis they received through the UDP (Families 13, 32, and 77) (Table 1).

Thirty‐seven patients had a negative report on initial analysis (Table S1). In the remaining 12 patients, VUSs were identified, which are awaiting resolution/reverse phenotyping/family studies (Table S1).

### Diagnosis based on disease group

Patients were assigned to one of ten diagnostic groups, based on their major phenotypic features (Figure 2). Diagnostic rates differed depending on diagnostic group, with the majority of the skeletal disorders being diagnosed (64%) compared with disorders of growth (22%). Approximately, half of the patients with dysmorphic features or multiple congenital anomalies were diagnosed (Figure 2).

Diagnostic yield by disease group

### Secondary findings

The parents/guardians were given the option to receive secondary findings. Only four families consented to receiving these results, with one known pathogenic variant in BRCA2 identified in Family 13 (NM_007294.4: c.181 T > C; p.Cys61Gly) (ClinVar VCV000017661.55). Family members have received follow‐up genetic counseling, and further members were offered testing and were/will be enrolled in appropriate cancer surveillance programs.

### DISCUSSION

These are the first data published from a broad ES study based in South Africa, which included a diverse study population, comprised of patients from several major South African ancestral sub‐groups. The demographic makeup of the study population reflects the general make‐up of ancestral diversity and socio‐economic status in our hospital and service‐drainage population: almost exclusively patients who have no health insurance, and who can therefore generally not afford genetic testing. This has meant that this study is one of a very few to provide this population with access to ES and thereby the possibility of a definitive diagnosis. As the UDP is expanded to cover more centers in South Africa, patient access to ES will increase. South Africa has nine provinces, six of which have no genetic services, and limited access to genetic testing. With this UDP, we have begun to fill the gap in knowledge related to the genetic causes of disease in our population. We were very interested to understand the genetic basis of diseases in our diverse population, and to explore whether factors such as traditional absence of sub‐Saharan African patient and population data in international databases would be a limiting factor. The data show that this has not limited our ability to identify and classify disease‐causing variants in this study. We now have a better idea of the spectrum of patients/disorders presenting to our clinics.

With a similar diagnostic rate to those from international studies, our results show promise. Clinical utility of ES is shown here: majority singletons (93%) in a study cohort comprised of a phenotypically diverse, non‐consanguineous, multi‐ethnic South African population. We identified 25 novel variants in 22 genes (Table 1), illustrating that this population has historically been understudied. Observed differences in diagnostic rates per disease group are in line with larger studies, both exome‐ and genome‐based (100,000 Genomes Project Pilot Investigators et al., 2021; Seo et al., 2020; Turro et al., 2020).

The low number of families opting to receive secondary findings is notable. Most parents voiced the desire to focus on the child's immediate problems as the reason for not wanting to receive “other” information, which did not necessarily have a bearing on the child's health immediately. Follow‐up studies will formally investigate the reasons behind the low uptake of secondary findings.

Several patients had their clinical diagnosis confirmed molecularly (Table 1) – this was important not only for immediate management, but also to facilitate access to other resources. An example is Patient 36, who was denied a welfare grant based on the fact that the clinical diagnosis of “albinism” was not accepted (he has Oculocutaneous albinism type 3). Only after a copy of his ES result was attached, was his application successful. Importantly, newer therapies and clinical trials will only include patients with confirmed molecular results.

Numerous patients are the first to be diagnosed with their particular disorder in sub‐Saharan Africa. Of the patients who received a diagnosis through this UDP, 39% had had previous negative testing. These data will be used to leverage for support and buy‐in from stakeholders, who have historically not acknowledged the importance of genomics in our context, based on the lack of supportive data regarding applicability to Africa. This study provides first evidence of not only applicability but utility of ES. We aspire to implement ES as a first‐line test to save both costs and time.

In summary, these results have enabled us to conduct genetic counseling, give families more accurate recurrence risks and prognoses, connect patients to relevant support groups, and to establish local diagnostic testing in certain instances (e.g. PGAP3 common South African variant). Importantly, patients have a definitive diagnosis to aid in access to relevant social welfare and other specialist medical management. ES allows us to reanalyze the data in light of new information. EVIDENCE is updated automatically and allows for daily re‐analysis of negative ES. Especially in patients who presented with clear dysmorphology and/or global developmental delay, we see a real potential for novel gene discovery. As the UDP expands to include more “omics,” more patients may be diagnosed using other methods. An important aspect of this study was capacity building, and it has allowed us to build and optimize our in‐house bioinformatics pipeline in preparation for expansion of the UDP, future studies, and the implementation of clinical genomics in southern Africa.

### CONFLICT OF INTEREST

SM and KCC declare no conflict of interest. EL and GHS are employees of 3billion Inc.

### Supporting information

Table S1 Post‐ES undiagnosed patients

Appendix S1 Supporting Information



# SUPPLEMENTAL FILE 1: AJMG-188-2684.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)