# MAIN TEXT

## Spectrum of germline pathogenic variants using a targeted next generation sequencing panel and genotype-phenotype correlations in patients with suspected hereditary breast cancer at an academic medical centre in Pakistan

### Abstract

BackgroundBreast cancer is the most common malignancy in women, affecting over 1.5 million women every year, which accounts for the highest number of cancer-related deaths in women globally.Hereditary breast cancer (HBC), an important subset of breast cancer, accounts for 5–10% of total cases. However, in Low Middle-Income Countries (LMICs), the population-specific risk of HBC in different ethnicities and the correlation with certain clinical characteristics remain unexplored.MethodsRetrospective chart review of patients who visited the HBC clinic and proceeded with multi-gene panel testing from May 2017 to April 2020.Descriptive and inferential statistics were used to analyze clinical characteristics of patients. Fisher’s exact, Pearson’s chi-squared tests and Logistic regression analysis were used for categorical variables and Wilcoxon rank-sum test were used for quantitative variables. For comparison between two independent groups, Mann-Whitney test was performed. Results were considered significant at a p value of < 0.05.ResultsOut of 273 patients, 22% tested positive, 37% had a VUS and 41% had a negative genetic test result. Fifty-five percent of the positive patients had pathogenic variants in either BRCA1 or BRCA2, while the remaining positive results were attributed to other genes. Patients with a positive result had a younger age at diagnosis compared to those having a VUS and a negative result; median age 37.5 years, IQR (Interquartile range) (31.5–48). Additionally, patients with triple negative breast cancer (TNBC) were almost 3 times more likely to have a positive result (OR = 2.79, CI = 1.42–5.48 p = 0.003). Of all patients with positive results, 25% of patients had a negative family history of breast and/or related cancers.ConclusionsIn our HBC clinic, we observed that our rate of positive results is comparable, yet at the higher end of the range which is reported in other populations. The importance of expanded, multi-gene panel testing is highlighted by the fact that almost half of the patients had pathogenic or likely pathogenic variants in genes other than BRCA1/2, and that our test positivity rate would have only been 12.8% if only BRCA1/2 testing was done. As the database expands and protocol-driven referrals are made across the country, our insight about the genetic architecture of HBC in our population will continue to increase.Supplementary InformationThe online version contains supplementary material available at 10.1186/s13053-022-00232-2.

### Introduction

Breast cancer is the most common malignancy worldwide, affecting over 1.5 million women (25% of all women with cancer) every year, accounting for the highest number of cancer-related deaths in women globally [1, 2].

The risk of developing breast cancer is broadly categorized into non-hereditary and hereditary cancer risk. The baseline population risk of having a diagnosis of sporadic breast cancer in a woman ranges between 10 and 12%, which means that roughly one in nine women will develop breast cancer in their lifetime [3]. This risk substantially increases in individuals with a germline disease-causing variant in one of the breast cancers associated genes, confirming a diagnosis of hereditary breast cancer (HBC).

HBC, an important subset of breast cancer, accounts for 5–10% of total cases [1, 4, 5]. These cases are attributed to pathogenic (P) and likely pathogenic (LP) germline variants in genes that cause a predisposition for breast cancer [6, 7]. In addition, approximately 20% of individuals with breast cancer have a close relative who also had breast cancer, suggesting a familial link, but no specific genetic variant is identified [8].

Identifying HBC is crucial for optimizing long-term outcomes in both symptomatic and pre-symptomatic individuals. Patients diagnosed with a hereditary breast cancer syndrome are eligible for tailored management, based on whether they have a high or moderate risk of developing breast or other related cancers in the future. They can, thus be offered a more precise and individualized management plan, that can include optimal surgical intervention and eligibility for targeted therapy, along with the need for a high-risk surveillance plan or/and prophylactic surgery for associated malignancy risk, summarized in Table 1 [9].Table 1NCCN recommendation for gene specific risk assessment, high risk surveillance plan and prophylactic surgery for associated malignancy risk. Table adapted from NCCN Guidelines (version 2.2021) [9]GeneEvidence for increased riskAbsolute risk (%)Other cancer risks and syndromesBreast cancer managementOther cancers managementBRCA1Very strong (with predisposition to triple-negative disease)> 60Ovarian cancerPancreatic cancerProstate cancerBreast awareness starting at 18 monthsClinical breast exam, every 6–12 months, starting at 25 years of ageAnnual breast MRI screeningDiscuss option of RRMAnnual mammogram screening in men with gynecomastia at age 50 or 10 years before the earliest known male breast cancer in the familyRecommend RRBSO typically at 35–40 years of ageIf no RRBSO, then TVUS with serum CA-125 for ovarian cancer screening starting at 30–35 yearsConsider prostate cancer screening in men starting at age 40 yearsPancreatic cancer screening using contrast enhanced MRI/MRCP or EUSBRCA2Very strong (with predisposition to ER+ disease)> 60Ovarian cancerPancreatic cancerProstate cancerMelanomaBreast awareness starting at 18 monthsClinical breast exam, every 6–12 months, starting at 25 years of ageAnnual breast MRI screeningDiscuss option of RRMAnnual mammogram screening in men with gynecomastia at age 50 or 10 years before the earliest known male breast cancer in the familyRecommend RRBSO typically at 40–45 years of ageIf no RRBSO, then TVUS with serum CA-125 for ovarian cancer screening starting at 30–35 yearsRecommend prostate cancer screening in men starting at age 40 yearsPancreatic cancer screening using contrast enhanced MRI/MRCP or EUSATMStrong15–40Ovarian cancerPancreatic cancerAtaxia telangiectasiaScreening with annual mammogram with consideration for tomosynthesis and consider breast MRI with contrast starting at age 40 yearsInsufficient evidence for RRM, manage based on FHInsufficient evidence for RRBSO, manage based on FHScreening mutation carriers with a family history of pancreatic cancerCHECK2Strong15–40Colon cancerScreening with annual mammogram with consideration for tomosynthesis and consider breast MRI with contrast starting at age 40 yearsInsufficient evidence for RRM, manage based on FHColon cancer surveillance is recommended same as APC which is colonoscopy (preferred) or flexible sigmoidoscopy every 12 months starting at age 10–15 yearsNF1Strong15–40Malignant peripheral nerve sheath tumors, GIST, othersScreening with annual mammogram with consideration for tomosynthesis starting at age 30 years and consider breast MRI with contrast from ages 30–50 yearsInsufficient evidence for RRM, manage based on FHRecommend referral to NF1 specialist for evaluation and management of other associated tumorsCDH1Strong41–60Hereditary diffuse gastric cancerScreening with annual mammogram with consideration for tomosynthesis and consider breast MRI with contrast starting at age 30 yearsInsufficient evidence for RRM, manage based on FHProphylactic total gastrectomy for CDH1 mutation carriers is recommended between ages 18 and 40 years and earlier than 18 years if gastric cancer in a family member before 25 years of ageIf patient elects not to undergo gastrectomy, screen every 6–12 months by upper endoscopy with multiple random biopsiesPALB2Strong41–60Ovarian cancerPancreatic cancer (limited increased risk)Screening with annual mammogram with consideration for tomosynthesis and breast MRI with contrast starting at age 30 yearsDiscuss option of RRMFor ovarian cancer management there is insufficient evidence, managed based on FHScreening mutation carriers with a family history of pancreatic cancerPTENStrong41–60Thyroid cancerColon cancerEndometrial cancerCowden syndromeWomen: Screening with annual mammogram with consideration for tomosynthesis and breast MRI with contrast starting at age 30–35 years or 5–10 years before the earliest known breast cancer in the familyDiscuss option for RRMStart endometrial cancer screening by age 35 years, patient education regarding AUB and symptoms is important, consider endometrial biopsy every 1–2 yearsDiscuss option of hysterectomy upon completion of familyMen and Women:Annual thyroid U/S starting at age 7 yearsColonoscopy starting at age 35 years unless symptomatic or if close relative with colon cancer, then start 5–10 years before the earliest known colon cancer in the familyConsider renal U/S starting at age 40 years, then every 1–2 yearAnnual dermatology exams for melanomaSTK11Strong41–60Ovarian (non-epithelial) cancerPancreatic cancerPeutz-Jeghers syndromeEvidence insufficient for RRM, manage based on FHPeutz-Jeghers syndrome management:Clinical breast exam every 6 months, with annual mammogram and breast MRI starting at 30 years of ageColonoscopy and upper endoscopy every 2–3 years starting in late teensSmall bowel visualization (CT or MRI enterography or video capsule endoscopy baseline starting at age 8–10 years, then every 2–3 yearsPancreatic cancer screening using contrast enhanced MRI/MRCP or EUSAnnual physical examination for observation of precocious puberty starting at 8 yearsPelvic examination and pap smear annually starting 18–20 yearsAnnual testicular exam and observation for feminizing changes starting at 10 yearsTP53Strong> 60Pancreatic cancer (limited)Li-Fraumeni syndromeScreening with annual mammogram with consideration for tomosynthesis and breast MRI with contrastDiscuss option of RRMColonoscopy and upper endoscopy every 2–5 years starting at 25 years or 5 years before the earliest known colon cancer in the familyAnnual dermatological examination starting at 18 yearsAnnual whole-body MRI including brainScreening mutation carriers with a family history of pancreatic cancerBARD1Limited (but stronger with triple negative disease)Insufficient dataNoneScreening with annual mammogram with consideration for tomosynthesis and consider breast MRI with contrast starting at age 40Insufficient evidence for RRM, manage based on FHBRIP1Limited; potential increase in female breast cancer (including triple negative disease)Insufficient dataOvarian cancerInsufficient data for breast cancer management, managed based on FHConsider RRSO at 45–50 years of ageNBNNo increased risk except 657del5 mutation which has mixed evidenceInsufficient dataLimited risk for Ovarian cancerInsufficient data for breast cancer management, managed based on FHOvarian cancer management also based on FHMSH2MLH1MSH6PMS2EPCAMLimited< 15Ovarian cancerPancreatic cancer (excluding PMS2)Colon, Uterine and othersLynch syndromeInsufficient data for breast cancer management, managed based on FHScreening mutation carriers with a family history of pancreatic cancerOther management is underLynch syndrome management requires surveillance in accordance with the mutated geneRAD51CLimited; potential increase in female breast cancer (including triple negative disease)15–40Ovarian cancerInsufficient data for breast cancer management, managed based on FHConsider RRSO at 45–50 years of ageRAD51DLimited; potential increase in female breast cancer (including triple negative disease)15–40Ovarian cancerInsufficient data for breast cancer management, managed based on FHConsider RRSO at 45–50 years of ageCDKN2ANone–Pancreatic cancerMelanoma–Pancreatic cancer screening using contrast enhanced MRI/MRCP or EUSRRM Risk reducing mastectomy, RRBSO Risk reducing bilateral salpingo-oophorectomy, GIST Gastro-intestinal stromal tumor, TVUS Transvaginal ultrasound,  MRCP Magnetic resonance cholangiopancreatography, EUS Endoscopic ultrasound, FH Family history

NCCN recommendation for gene specific risk assessment, high risk surveillance plan and prophylactic surgery for associated malignancy risk. Table adapted from NCCN Guidelines (version 2.2021) [9]

Ovarian cancer

Pancreatic cancer

Prostate cancer

Breast awareness starting at 18 months

Clinical breast exam, every 6–12 months, starting at 25 years of age

Annual breast MRI screening

Discuss option of RRM

Annual mammogram screening in men with gynecomastia at age 50 or 10 years before the earliest known male breast cancer in the family

Recommend RRBSO typically at 35–40 years of age

If no RRBSO, then TVUS with serum CA-125 for ovarian cancer screening starting at 30–35 years

Consider prostate cancer screening in men starting at age 40 years

Pancreatic cancer screening using contrast enhanced MRI/MRCP or EUS

Ovarian cancer

Pancreatic cancer

Prostate cancer

Melanoma

Breast awareness starting at 18 months

Clinical breast exam, every 6–12 months, starting at 25 years of age

Annual breast MRI screening

Discuss option of RRM

Annual mammogram screening in men with gynecomastia at age 50 or 10 years before the earliest known male breast cancer in the family

Recommend RRBSO typically at 40–45 years of age

If no RRBSO, then TVUS with serum CA-125 for ovarian cancer screening starting at 30–35 years

Recommend prostate cancer screening in men starting at age 40 years

Pancreatic cancer screening using contrast enhanced MRI/MRCP or EUS

Ovarian cancer

Pancreatic cancer

Ataxia telangiectasia

Screening with annual mammogram with consideration for tomosynthesis and consider breast MRI with contrast starting at age 40 years

Insufficient evidence for RRM, manage based on FH

Insufficient evidence for RRBSO, manage based on FH

Screening mutation carriers with a family history of pancreatic cancer

Screening with annual mammogram with consideration for tomosynthesis and consider breast MRI with contrast starting at age 40 years

Insufficient evidence for RRM, manage based on FH

Screening with annual mammogram with consideration for tomosynthesis starting at age 30 years and consider breast MRI with contrast from ages 30–50 years

Insufficient evidence for RRM, manage based on FH

Screening with annual mammogram with consideration for tomosynthesis and consider breast MRI with contrast starting at age 30 years

Insufficient evidence for RRM, manage based on FH

Prophylactic total gastrectomy for CDH1 mutation carriers is recommended between ages 18 and 40 years and earlier than 18 years if gastric cancer in a family member before 25 years of age

If patient elects not to undergo gastrectomy, screen every 6–12 months by upper endoscopy with multiple random biopsies

Ovarian cancer

Pancreatic cancer (limited increased risk)

Screening with annual mammogram with consideration for tomosynthesis and breast MRI with contrast starting at age 30 years

Discuss option of RRM

For ovarian cancer management there is insufficient evidence, managed based on FH

Screening mutation carriers with a family history of pancreatic cancer

Thyroid cancer

Colon cancer

Endometrial cancer

Cowden syndrome

Women: Screening with annual mammogram with consideration for tomosynthesis and breast MRI with contrast starting at age 30–35 years or 5–10 years before the earliest known breast cancer in the family

Discuss option for RRM

Start endometrial cancer screening by age 35 years, patient education regarding AUB and symptoms is important, consider endometrial biopsy every 1–2 years

Discuss option of hysterectomy upon completion of family

Men and Women:

Annual thyroid U/S starting at age 7 years

Colonoscopy starting at age 35 years unless symptomatic or if close relative with colon cancer, then start 5–10 years before the earliest known colon cancer in the family

Consider renal U/S starting at age 40 years, then every 1–2 year

Annual dermatology exams for melanoma

Ovarian (non-epithelial) cancer

Pancreatic cancer

Peutz-Jeghers syndrome

Evidence insufficient for RRM, manage based on FH

Peutz-Jeghers syndrome management:

Clinical breast exam every 6 months, with annual mammogram and breast MRI starting at 30 years of age

Colonoscopy and upper endoscopy every 2–3 years starting in late teens

Small bowel visualization (CT or MRI enterography or video capsule endoscopy baseline starting at age 8–10 years, then every 2–3 years

Pancreatic cancer screening using contrast enhanced MRI/MRCP or EUS

Annual physical examination for observation of precocious puberty starting at 8 years

Pelvic examination and pap smear annually starting 18–20 years

Annual testicular exam and observation for feminizing changes starting at 10 years

Pancreatic cancer (limited)

Li-Fraumeni syndrome

Screening with annual mammogram with consideration for tomosynthesis and breast MRI with contrast

Discuss option of RRM

Colonoscopy and upper endoscopy every 2–5 years starting at 25 years or 5 years before the earliest known colon cancer in the family

Annual dermatological examination starting at 18 years

Annual whole-body MRI including brain

Screening mutation carriers with a family history of pancreatic cancer

Screening with annual mammogram with consideration for tomosynthesis and consider breast MRI with contrast starting at age 40

Insufficient evidence for RRM, manage based on FH

MSH2

MLH1

MSH6

PMS2

EPCAM

Ovarian cancer

Pancreatic cancer (excluding PMS2)

Colon, Uterine and others

Lynch syndrome

Screening mutation carriers with a family history of pancreatic cancer

Other management is under

Lynch syndrome management requires surveillance in accordance with the mutated gene

Pancreatic cancer

Melanoma

RRM Risk reducing mastectomy, RRBSO Risk reducing bilateral salpingo-oophorectomy, GIST Gastro-intestinal stromal tumor, TVUS Transvaginal ultrasound,  MRCP Magnetic resonance cholangiopancreatography, EUS Endoscopic ultrasound, FH Family history

Data estimating the global incidence and prevalence of HBC comes predominantly from European populations and non-European High-Income Countries [10–20]. In Low Middle-Income Countries (LMICs), the population-specific risk of HBC in different ethnicities and the correlation with clinical characteristics remains largely unexplored.

There is little published data on HBC in the Pakistani population, and no study has reported the use of multi-gene Next Generation Sequencing (NGS) panel testing. A study using a two-tiered targeted sequencing approach on suspected high-risk patients with breast and/or ovarian cancer, found that 6.7% of patients tested positive, for BRCA1/2 variants. Six out of 42 identified variants were found in multiple unrelated patients [21]. Rashid at el. found that 24.7% of high-risk breast cancer patients in their case series from Pakistan harbored a P/LP variant in BRCA1/2 [22]. In BRCA1, there were 18 recurrent variants and in BRCA2 there were three recurrent variants found in unrelated families. In addition to these small insertions and deletions (INDELS) and single nucleotide variants (SNVs), 2.5% (n = 14/565) of high-risk patients had large genomic rearrangements (LGRs) [23, 24]. The same group further investigated the patients who did not harbor BRCA1/2 variants, sequentially for P/LP variants in TP53, CHEK2, RAD51C, and PALB2, using denaturing high-performance liquid chromatography (HPLC) analysis [25–28].

These studies reported pathogenic variants attributed to TP53 in 0.95% (n = 1/105) and, RAD51C in 0.80% (n = 1/119) of the high-risk breast cancer patients. One nonsense pathogenic variant was identified in PALB2, accounting for 0.79% (n = 1/127) patients. Additionally, four in-silico predicted potentially deleterious variants, including three missense and one 5′ untranslated region variant were also identified in the study participants. Another study from the same group, that looked at the contribution of CHEK2 in 374 breast and/or ovarian cancer patients and identified no known pathogenic variants. However, in 0.53% (n = 2/374) of participants, two missense in-silico predicted potentially deleterious variants were reported.

Another study, from Rashid at el. concluded that RECQL, which is a preliminary evidence breast cancer gene, was not a disease contributor in the studied series of cases [29].

However, as sequencing technology has advanced from single gene sequencing to high-throughput NGS platform based parallel gene sequencing, complemented by the progress in bioinformatics and variant classification, identification of P and LP variants in inherited breast cancer syndrome associated genes has become rapid and cost-effective. Thus, as multi-gene panel testing has now replaced sequential single gene testing, more patients are being identified to harbor variants in other breast-cancer risk genes. Studies report that about 4–16% test positive when they undergo multi-gene panel testing, after testing negative for BRCA1/2 variants [25, 30–34].

In a previous paper, we described the challenges in establishing an HBC clinic at our academic medical center [35]. In this study, we report the results of using an expanded, multi-gene NGS panel for the first time in Pakistani patients and describe the spectrum of pathogenic variants found in a consecutive series of high-risk patients. We also explore unique genotype-phenotype correlations in the Pakistani population.

### Previous studies in Pakistani population

There is little published data on HBC in the Pakistani population, and no study has reported the use of multi-gene Next Generation Sequencing (NGS) panel testing. A study using a two-tiered targeted sequencing approach on suspected high-risk patients with breast and/or ovarian cancer, found that 6.7% of patients tested positive, for BRCA1/2 variants. Six out of 42 identified variants were found in multiple unrelated patients [21]. Rashid at el. found that 24.7% of high-risk breast cancer patients in their case series from Pakistan harbored a P/LP variant in BRCA1/2 [22]. In BRCA1, there were 18 recurrent variants and in BRCA2 there were three recurrent variants found in unrelated families. In addition to these small insertions and deletions (INDELS) and single nucleotide variants (SNVs), 2.5% (n = 14/565) of high-risk patients had large genomic rearrangements (LGRs) [23, 24]. The same group further investigated the patients who did not harbor BRCA1/2 variants, sequentially for P/LP variants in TP53, CHEK2, RAD51C, and PALB2, using denaturing high-performance liquid chromatography (HPLC) analysis [25–28].

These studies reported pathogenic variants attributed to TP53 in 0.95% (n = 1/105) and, RAD51C in 0.80% (n = 1/119) of the high-risk breast cancer patients. One nonsense pathogenic variant was identified in PALB2, accounting for 0.79% (n = 1/127) patients. Additionally, four in-silico predicted potentially deleterious variants, including three missense and one 5′ untranslated region variant were also identified in the study participants. Another study from the same group, that looked at the contribution of CHEK2 in 374 breast and/or ovarian cancer patients and identified no known pathogenic variants. However, in 0.53% (n = 2/374) of participants, two missense in-silico predicted potentially deleterious variants were reported.

Another study, from Rashid at el. concluded that RECQL, which is a preliminary evidence breast cancer gene, was not a disease contributor in the studied series of cases [29].

However, as sequencing technology has advanced from single gene sequencing to high-throughput NGS platform based parallel gene sequencing, complemented by the progress in bioinformatics and variant classification, identification of P and LP variants in inherited breast cancer syndrome associated genes has become rapid and cost-effective. Thus, as multi-gene panel testing has now replaced sequential single gene testing, more patients are being identified to harbor variants in other breast-cancer risk genes. Studies report that about 4–16% test positive when they undergo multi-gene panel testing, after testing negative for BRCA1/2 variants [25, 30–34].

In a previous paper, we described the challenges in establishing an HBC clinic at our academic medical center [35]. In this study, we report the results of using an expanded, multi-gene NGS panel for the first time in Pakistani patients and describe the spectrum of pathogenic variants found in a consecutive series of high-risk patients. We also explore unique genotype-phenotype correlations in the Pakistani population.

### Materials and methods

This is a retrospective review performed at a single Academic Medical Center in Karachi, Pakistan (Aga Khan University Hospital).

We selected patients who were seen in the HBC clinic and proceeded with multi-gene panel testing from May 2017 to April 2020. During this period of 36 months, multi-gene panel for breast cancer was sent for 284 individuals, referred from Breast Surgeons or Oncologists, from within AKUH and other hospitals in Karachi, including Liaquat National Hospital and Shaukat Khanum Memorial Cancer Hospital. Those who met the National Comprehensive Cancer Network (NCCN) referral criteria (NCCN 2016 version:1.2016), summarized in Table 2 were offered testing, and those who went ahead with testing were recruited for this study [36]. With regards to age at diagnosis, we divided our patients in age group brackets of 25–34, 35–44, 45–54, 55–64, and ≥ 65 years. Family history was considered positive if the patient had a close family relative (1st, 2nd or 3rd degree relative) with breast cancer or any other malignancies that may be part of a hereditary cancer syndrome, including ovarian, endometrial, small bowel, gastric, colorectal, pancreatic, prostate, brain, and thyroid cancer.Table 2NCCN guidelines for referral criteria (NCCN 2016 version 1.2016), used in the study [35]Diagnosed with breast cancer at< 50 years of ageDiagnosed with breast cancer at age 46–50 years withUnknown or limited family historyA second breast cancer at any age≥1 close blood relative with breast, ovarian, pancreatic, or prostate cancer at any ageDiagnosed with breast cancer at age ≤ 60 years withTriple-negative breast cancerDiagnosed at any age with≥1 close blood relative with breast cancer at age ≤ 50 yearOvarian, pancreatic, metastatic, intraductal/cribriform histology, or high- or very-high risk group prostate cancer at any age≥ 3 total diagnoses of breast cancer in patient and/or close blood relativesMale breast cancerWith epithelial ovarian cancer (including fallopian tube cancer or peritoneal cancer)Exocrine pancreatic cancer

NCCN guidelines for referral criteria (NCCN 2016 version 1.2016), used in the study [35]

Patients with ER or/and PR, score of 0–2 were considered as ER or/and PR negative, and those with ER or/and PR score ≥ 3 were considered as ER or/and PR positive. For HER2, IHC score of 0 or 1+ was considered as negative, a score of 2+ was considered as equivocal and a score of 3+ was considered as HER2 positive. For equivocal results, FISH (Fluorescence In Situ Hybridization) analysis was carried out, and if it was negative, then initial IHC score of + 2 for HER2 was counted as HER2 negative, otherwise FISH positive led to the initial IHC score of + 2 to be considered as HER2 positive.

This testing was outsourced to one of two Clinical Laboratory Improvement Amendments (CLIA) and College of American Pathologists (CAP), certified commercial genetics laboratories based in the US; Invitae Genetics (n = 268) and Prevention Genetics (n = 16).

The NGS panel offered at Prevention Genetics included 27 genes, associated with hereditary breast cancer syndromes, while the panel offered at Invitae Genetics includes 37 genes, including 23 primary evidence and 14 preliminary evidence genes, associated with Hereditary Breast and Gynecologic cancers, summarized in Table 3 [37, 38].Table 3Summary of primary and preliminary evidence genes included in the multi-cancer panel at Invitae GeneticsGenes with guidelines (Primary evidence)Genes without guidelines (Preliminary evidence)ATM, BRCA1, BRCA2, BRIP1, CDH1ABRAXAS1, AKT1, CDC73, FANCCCHECK2, EPCAM, MLH1, MSH2, MSH6FANCM, MRE11, MUTYH, PIK3CANBN, NF1, PALB2, PMS2, PTENPOLD1, RECQL, RINT1RAD51C, RAD51D, STK11, TP53SDHB, SDHD, XRC22

Summary of primary and preliminary evidence genes included in the multi-cancer panel at Invitae Genetics

The NGS platform makes uses of Illumina technology that offers full-gene sequencing and deletion/duplication analysis, with 99% analytical sensitivity and specificity for SNVs and insertions and deletions that are smaller than 15 base pairs as well as exon-level insertions and deletions. The sequence analysis covers the clinically important regions of the genes offered in the panels, including 10–20 base pairs in non-coding (intronic) regions of the selected gene transcript used.

The patient DNA is captured using a hybridization-based protocol, that is sequenced using Illumina’s Reversible Dye Terminator (RDT) platform and the reads are aligned to the reference sequence (GRCh37). The obtained results are confirmed using Sanger Sequencing at both laboratories. Additionally, Invitae Genetics utilizes other orthogonal technologies, including Pacific Biosciences SMRT sequencing, MLPA, MLPA-seq and Array CGH for NGS result validation, as needed. Thus, this testing is designed to detect SNVs, small INDELS, CNVs and LGRs [39].

The variants identified (sequence finding) in the patients, after they are aligned to the reference sequence were then interpretations and assigned one of the five variant classifications as per ACMG guidelines: including Pathogenic (P), Likely Pathogenic (LP), Variants of Uncertain Significance (VUS), Likely Benign (LB) and Benign (B) [40].

Data was obtained from patients who visited the HBC clinic, from May 2017 to April 2020. Descriptive and inferential statistics have been used to analyze clinical and histopathological characteristics of breast cancer patients with positive, VUS and negative genetic test results.

Fisher’s exact and Pearson’s chi-squared tests for categorical variables and Wilcoxon rank-sum test for quantitative variables were used. For comparison between two independent groups, Mann-Whitney test was performed. Results were considered significant at a p value of < 0.05. All statistical analyses were done using STATA 16.

Ethical Review Approval Exemption (ERC) at the Aga Khan University was received for the retrospective chart review of the patients (insert ERC approval ID 2021–1332-18,429).

### Case selection

We selected patients who were seen in the HBC clinic and proceeded with multi-gene panel testing from May 2017 to April 2020. During this period of 36 months, multi-gene panel for breast cancer was sent for 284 individuals, referred from Breast Surgeons or Oncologists, from within AKUH and other hospitals in Karachi, including Liaquat National Hospital and Shaukat Khanum Memorial Cancer Hospital. Those who met the National Comprehensive Cancer Network (NCCN) referral criteria (NCCN 2016 version:1.2016), summarized in Table 2 were offered testing, and those who went ahead with testing were recruited for this study [36]. With regards to age at diagnosis, we divided our patients in age group brackets of 25–34, 35–44, 45–54, 55–64, and ≥ 65 years. Family history was considered positive if the patient had a close family relative (1st, 2nd or 3rd degree relative) with breast cancer or any other malignancies that may be part of a hereditary cancer syndrome, including ovarian, endometrial, small bowel, gastric, colorectal, pancreatic, prostate, brain, and thyroid cancer.Table 2NCCN guidelines for referral criteria (NCCN 2016 version 1.2016), used in the study [35]Diagnosed with breast cancer at< 50 years of ageDiagnosed with breast cancer at age 46–50 years withUnknown or limited family historyA second breast cancer at any age≥1 close blood relative with breast, ovarian, pancreatic, or prostate cancer at any ageDiagnosed with breast cancer at age ≤ 60 years withTriple-negative breast cancerDiagnosed at any age with≥1 close blood relative with breast cancer at age ≤ 50 yearOvarian, pancreatic, metastatic, intraductal/cribriform histology, or high- or very-high risk group prostate cancer at any age≥ 3 total diagnoses of breast cancer in patient and/or close blood relativesMale breast cancerWith epithelial ovarian cancer (including fallopian tube cancer or peritoneal cancer)Exocrine pancreatic cancer

NCCN guidelines for referral criteria (NCCN 2016 version 1.2016), used in the study [35]

Patients with ER or/and PR, score of 0–2 were considered as ER or/and PR negative, and those with ER or/and PR score ≥ 3 were considered as ER or/and PR positive. For HER2, IHC score of 0 or 1+ was considered as negative, a score of 2+ was considered as equivocal and a score of 3+ was considered as HER2 positive. For equivocal results, FISH (Fluorescence In Situ Hybridization) analysis was carried out, and if it was negative, then initial IHC score of + 2 for HER2 was counted as HER2 negative, otherwise FISH positive led to the initial IHC score of + 2 to be considered as HER2 positive.

### NGS assays

This testing was outsourced to one of two Clinical Laboratory Improvement Amendments (CLIA) and College of American Pathologists (CAP), certified commercial genetics laboratories based in the US; Invitae Genetics (n = 268) and Prevention Genetics (n = 16).

The NGS panel offered at Prevention Genetics included 27 genes, associated with hereditary breast cancer syndromes, while the panel offered at Invitae Genetics includes 37 genes, including 23 primary evidence and 14 preliminary evidence genes, associated with Hereditary Breast and Gynecologic cancers, summarized in Table 3 [37, 38].Table 3Summary of primary and preliminary evidence genes included in the multi-cancer panel at Invitae GeneticsGenes with guidelines (Primary evidence)Genes without guidelines (Preliminary evidence)ATM, BRCA1, BRCA2, BRIP1, CDH1ABRAXAS1, AKT1, CDC73, FANCCCHECK2, EPCAM, MLH1, MSH2, MSH6FANCM, MRE11, MUTYH, PIK3CANBN, NF1, PALB2, PMS2, PTENPOLD1, RECQL, RINT1RAD51C, RAD51D, STK11, TP53SDHB, SDHD, XRC22

Summary of primary and preliminary evidence genes included in the multi-cancer panel at Invitae Genetics

The NGS platform makes uses of Illumina technology that offers full-gene sequencing and deletion/duplication analysis, with 99% analytical sensitivity and specificity for SNVs and insertions and deletions that are smaller than 15 base pairs as well as exon-level insertions and deletions. The sequence analysis covers the clinically important regions of the genes offered in the panels, including 10–20 base pairs in non-coding (intronic) regions of the selected gene transcript used.

The patient DNA is captured using a hybridization-based protocol, that is sequenced using Illumina’s Reversible Dye Terminator (RDT) platform and the reads are aligned to the reference sequence (GRCh37). The obtained results are confirmed using Sanger Sequencing at both laboratories. Additionally, Invitae Genetics utilizes other orthogonal technologies, including Pacific Biosciences SMRT sequencing, MLPA, MLPA-seq and Array CGH for NGS result validation, as needed. Thus, this testing is designed to detect SNVs, small INDELS, CNVs and LGRs [39].

### Variant classification

The variants identified (sequence finding) in the patients, after they are aligned to the reference sequence were then interpretations and assigned one of the five variant classifications as per ACMG guidelines: including Pathogenic (P), Likely Pathogenic (LP), Variants of Uncertain Significance (VUS), Likely Benign (LB) and Benign (B) [40].

### Statistical analysis

Data was obtained from patients who visited the HBC clinic, from May 2017 to April 2020. Descriptive and inferential statistics have been used to analyze clinical and histopathological characteristics of breast cancer patients with positive, VUS and negative genetic test results.

Fisher’s exact and Pearson’s chi-squared tests for categorical variables and Wilcoxon rank-sum test for quantitative variables were used. For comparison between two independent groups, Mann-Whitney test was performed. Results were considered significant at a p value of < 0.05. All statistical analyses were done using STATA 16.

### Study ethical approval

Ethical Review Approval Exemption (ERC) at the Aga Khan University was received for the retrospective chart review of the patients (insert ERC approval ID 2021–1332-18,429).

### Results

Patient characteristics are shown in Table 4.Table 4Patient characteristics of the study participantsStudy Characteristics OverviewN = number of patients% of total patientsTotal Study Participants284100.00Males51.76Asymptomatic10.35Females27998.24Asymptomatic103.52Unilateral24887.32Bilateral258.80Asymptomatic113.87Breast and a secondary malignancy51.76Only breast26894.37Asymptomatic113.87Family History of Disease Negative7526.41 Positive 1st Degree Relative for Breast Cancer Related Malignancy10637.32 Positive 2nd/3rd Degree Relative for Breast Cancer Related Malignancy7225.35 Positive 1-3rd Degree Relative for Non-Breast Cancer Related Malignancy279.51 Incomplete Information41.41

Patient characteristics of the study participants

Out of 284 individuals, 273 patients were diagnosed with breast cancer and 11 asymptomatic individuals underwent testing, owing to their positive family history of breast cancer and/ovarian cancer. These 273 patients included four (1.46%) male breast cancer patients. The median age at diagnosis for all patients was 43 years (IQR: 36–50). Twenty-five of 273 (9.16%) patients had a diagnosis of bilateral breast cancer that included both synchronous and non-synchronous bilateral breast cancer. Five of 273 (1.83%) patients had a diagnosis of a second malignancy other than breast cancer, that included thyroid cancer (n = 2), ovarian cancer (n = 1), endometrial cancer (n = 1) and colon cancer (n = 1). No bilateral disease patients were recorded to have a second malignancy. Of the 11 asymptomatic individuals who underwent testing, one was male.

Histology was recorded for each tumor (n = 298), including unilateral (n = 248) and bilateral disease (n = 25). These included invasive ductal carcinoma (IDC) (n = 257, 86.24%), ductal carcinoma in situ (DCIS) (n = 22, 7.38%), invasive lobular carcinoma (ILC) (n = 9, 3.02%) and invasive papillary carcinoma (IPC) (n = 3, 1.10%). IHC was observed for each tumor recorded individually; from all unilateral and bilateral disease patients, (n = 298), out of which 71(23.83%) were triple negative breast cancer. (TNBC). Other IHC subtypes are summarized in Tables 5, 6, and 7.Table 5Immunohistochemistry subtypes summaryStudy CharacteristicsN = number of tumors% of total tumorsBreast Cancer (unilateral = 248, bilateral = 25)298100.00Grade 1103.36 212341.28 313948.94 Unknown268.72Histology IDC25786.24 DCIS227.38 ILC93.02 Papillary31.01 Unknown72.35Immunohistochemistry Triple Negative (−)7123.83 Triple Positive (+)3511.74 ER−/PR+/HER2+20.67 ER−/PR+/HER2-41.34 ER−/PR-31.01 ER−/PR−/HER2+248.05 ER+/PR−/HER2-134.36 ER+/PR+/HER2-11538.59 ER+/PR−/HER2+51.68 ER+/PR+155.03 ER+/PR-10.34 Unknown103.36Table 6Pathogenic and likely pathogenic variants identified in BRCA1 and BRCA2Sr No.Patient IDAge at diagnosisTypeGeneVariant DetailsExonConsequenceReported in multiple unrelated families in our cohort (Family #)Previously Reported in Pakistani Patients (no. of individuals)Novel VariantsPublished literatureBRCA1125635U/L Breast CaBRCA1c.68_69del (p.Glu23Valfs*17)2Frameshiftalso reported in one unrelated ovarian cancer patientYes (1,1,1)–PMID: 12181777, 31,528,241, (Risch et al., 2001) [21, 22, 41]22236U/L Breast CaBRCA1c.1399_1453dup (p.Ala485Glufs*13)10Frameshift–Yes (1)–PMID:31528241 [22]33625U/L Breast CaBRCA1c.1450G > T(p.Gly484*)10Non-sense––––45134U/L Breast CaBRCA1c.685del (p.Ser229Leufs*5)10Frameshift–Yes (7)–PMID:31528241 [22]517921U/L Breast CaBRCA1c.895_896del (p.Val299Argfs*4)10Frameshift179,191Yes (1)–PMID:31528241 [22]618126U/L Breast CaBRCA1c.3607C > T (p.Arg1203*)10Non-sense–Yes (1)–PMID:31528241 [22]719136U/L Breast CaBRCA1c.895_896del (p.Val299Argfs*4),10Frameshift179,191Yes (1)–PMID:31528241 [22]828345Asymptomatic probandBRCA1c.1583_1589del (p.Thr528Lysfs*2)10Frameshift919437U/L Breast CaBRCA1c.3228_3229del (p.Gly1077Alafs*8)10Frameshift–––1020440U/L Breast CaBRCA1c.4065_4068del (p.Asn1355Lysfs*10)10Frameshift–Yes (4, 4, 1)–PMID: 12181777, 31,528,241, 8,571,953 [21, 22, 42]1122031U/L Breast CaBRCA1c.3169_3172del, p.Ser1057Leufs*4,10Frameshift–Yes (2)–PMID:31528241 [22]1290–Asymptomatic probandBRCA1c.4485-1G > AIntron 13Splice acceptor90, 145Yes (6)–PMID:31528241 [22]1314538, 47B/L Breast CaBRCA1c.4485-1G > AIntron 13Splice acceptor90, 145Yes (6)–PMID:31528241 [22]1422332U/L Breast CaBRCA1c.4508C > A (p.Ser1503*)14Non-sense223,282Yes (3,1)–PMID: 12181777, 31,528,241 [21, 22]1528240U/L Breast Ca, Ovarian CaBRCA1c.4508C > A (p.Ser1503*)14Non-sense223,282–––161549U/L Breast CaBRCA1c.4821del, p.Ala1608Glnfs*2515Frameshift––Yes–178230,43B/L Breast CaBRCA1c.5035del (p.Leu1679*)16Non-sense82, 251Yes (2)––1825130B/L Breast CaBRCA1c.5035del (p.Leu1679*),16Non-sense82, 251–––1928436U/L Breast CaBRCA1c.5074 + 1G > AIntron 16Splice donor7284–PMID:31528241 [22]20727U/L Breast CaBRCA1c.5074 + 1G > AIntron 16Splice donor7284Yes, (2)_PMID:31528241 [22]213128U/L Breast CaBRCA1c.5176delA (p.Arg1726Glufs*4),18Frameshift––––22437U/L Breast CaBRCA1c.5278-1G > A19Splice acceptor––––2323531U/L Breast CaBRCA1,c.5135G > A (p.Trp1712*),17Non-sense––––CHEK2Gain (Exons 3–4)Gain of Ex. 3–4LGR241331U/L Breast CaBRCA1Deletion (Exons 1–11)Deletion of Ex.1–11LGR13,278–––2527837Asymptomatic probandBRCA1Deletion (Exons 1–11)Deletion of Ex.1–11LGR13,278–––262832U/L Breast CaBRCA1Deletion (Exon 1–6)Deletion of Ex.1–6LGR–––2710934U/L Breast CaBRCA1Deletion (Exon 23)Deletion of Ex.23LGR––––286333U/L Breast CaBRCA1Gain (Exon 3–11)Gain of Ex.3–11LGR––Yes–29630U/L Breast CaBRCA1Deletion of Exons 1–2Deletion of Ex.1–2LGR–Yes, (7)–PMID:31528241 [22]MSH6c.3261del (p.Phe1088Serfs*2)5FrameshiftBRCA23010562U/L Breast CaBRCA2Deletion (Exon 3)Deletion of Ex.3LGR––––ATMDeletion (Exon 43)Deletion of Ex. 43LGR314553U/L Breast CaBRCA2c.426-2A > GIntron 4Splice acceptor–Yes (1)–PMID:31528241 [22]3225735U/L Breast CaBRCA2c.859del (p.Met287Cysfs*5)10Frameshift––––3311955U/L Breast CaBRCA2c.3109C > T (p.Gln1037*)11Non-sense–Found in Asian patient (4)PMID: 19241424 [43]34152U/L Breast CaBRCA2c.4718del (p.Cys1573Leufs*6)11Frameshift––––353441U/L Breast CaBRCA2c.6444dupT (p.Ile2149Tyrfs*2)11Frameshift––––3618536U/L Breast CaBRCA2c.4471_4474del (p.Leu1491Lysfs*12)11Frameshift––––377648U/L Breast CaBRCA2c.7806-1G > CIntron 16Splice acceptor––––3820040, 55B/L Breast CaBRCA2c.9380G > A (p.Trp3127*)25Non-sense––––Table 7Pathogenic and likely pathogenic variants identified in genes other than BRCA1 and BRCA2Serial No.Patient IDAge at diagnosisTypeGeneVariant DetailsExonConsequence14429U/L Breast CaATMc.8480 T > G (p.Phe2827Cys),58Missense213558, 63B/L Breast CaBARD1Deletion (Entire coding sequence)Exon 1–11LGR316940U/L Breast CaBARD1Deletion (Entire coding sequence)Exon 1–11LGR42041U/L Breast CaCHEK2c.58C > T (p.Gln20*)2Non-sense59635U/L Breast CaCHEK2Deletion (Exon 5)Deletion of Ex. 5LGR616560B/L Breast CaCHEK2c.409C > T (p.Arg137*)3Non-sense722540U/L Breast CaCHEK2c.58C > T (p.Gln20*)2Non-sense82144U/L Breast CaCHEK2c.283C > T (p.Arg95*)2Non-senseRAD51Cc.701C > G (p.Ser234*)4Non-sense924356U/L Breast CaFANCMc.4153G > T (p.Glu1385*),14Non-sense1027943U/L Breast CaFANCMc.2199_2202del (p.Ser734Asnfs*25),13Frameshift111143U/L Breast CaMLH1Deletion (Exon 16–19)Deletion of Ex. 16–19LGR1224160U/L Breast CaMLH1c.1897-2A > G (Splice acceptor)Intron 16Splice acceptor1326939U/L Breast CaMLH1c.306G > T (p.Glu102Asp)3Missense1410230U/L Breast CaMSH6c.1222_1226del (p.Pro408Aspfs*8),4Frameshift1520251U/L Breast CaMUTYHc.312C > A (p.Tyr104*)3Non-sense1624848U/L Breast CaMUTYHc.312C > A (p.Tyr104*)3Non-sense178066U/L Breast CaNF1c.5205 + 5G > AIntron 36Splice donor1822738U/L Breast CaPALB2c.2488del (p.Glu830Serfs*21),5Frameshift1925842U/L Breast CaPALB2c.2353_2354del (p.Pro785Thrfs*16)5Frameshift205764B/L Breast CaPALB2c.2488del (p.Glu830Serfs*21)5Frameshift211834,37B/L Breast CaRAD51Dc.898C > T, p.Arg300*,9Non-sense2214455U/L Breast CaRAD51Dc.620C > T (p.Ser207Leu)7MissenseFANCMc.4318-1G > A (Splice acceptor)Intron 15Splice acceptor2318440U/L Breast CaTP53c.537 T > G (p.His179Gln)5Missense2424029U/L Breast CaTP53c.437G > A (p.Trp146*)5Non-sense

Immunohistochemistry subtypes summary

Pathogenic and likely pathogenic variants identified in BRCA1 and BRCA2

Pathogenic and likely pathogenic variants identified in genes other than BRCA1 and BRCA2

This included all patients with unilateral (n = 62) and bilateral disease (n = 8). In the bilateral disease patient group, three patients had both tumors as triple negative (n = 6), and three had either of the two tumors as triple negative (n = 3), that along with 62 unilateral disease patients made a total of 71 TNBC tumors. In the DCIS and ILC and IPC, none of the tumors were found to be triple negative. The statistical analyses involving IHC subtypes were carried out on patients with unilateral disease with IDC histology.

Out of 273 patients, 59 (22%) harbored P/LP variant (positive result), 103 (37%) had a VUS and 111 (41%) did not harbor any P/LP variant or a VUS (negative result). Of the 59 patients that harbored P/LP variants, 5 had a P/LP variant in two genes. Out of these total 64 P/LP variants, 35 (55%) were in either BRCA1 or BRCA2. The spectrum of the identified variants is summarized in Fig. 1.Fig. 1The spectrum of the identified variants is summarized, showing 45% of the positive results are attributed to non BRCA1/2 genes

The spectrum of the identified variants is summarized, showing 45% of the positive results are attributed to non BRCA1/2 genes

Patients with P/LP variants on their genetic test had a younger age at diagnosis compared to those having a VUS and negative result; median age = 37.5, IQR (31.5–48) vs median age = 45, vs, median age = 44, (p = 0.002) respectively, using Mann-Whitney test. A larger percentage of results with P/LP variants were found in the youngest age group of 25–34 years (n = 19) and 35–44 years (n = 25), accounting for 74% (n = 44/59) of the positive genetic test results. Eleven of these 44 patients (25%) had a negative family history of breast or related cancers.

One hundred and sixty-seven of the 273 patients (61%) reported a positive family history. Forty-five of these 167 patients (27%) harbored a P/LP variant, compared to 15 out of 102 (15%) patients with a negative family history. Out of all patients who harbored a P/LP variant, (n = 15/60) 25% had a negative family history for breast or related malignancies.

We also analyzed if there was an association found between family history of disease and genetic test result outcomes. Having a positive family history was associated with having a positive test result (p = 0.019). However, as expected, the association of family history was not established in case of VUS (p = 0.780) and negative (p = 0.088) result outcomes. Furthermore, the likelihood of a genetic test being positive was two times higher in the presence of a family history of cancer as compared to no family history of cancer (OR = 2.13, CI = 1.12–4.08, p = 0.021).

In patients with unilateral disease who presented without a history of another malignancy; the diagnostic yield was 20% (n = 49/243). Out of them 20% (n = 11/50) presented with negative family history of disease. Five patients with unilateral disease also presented with a secondary malignancy. For these patients, the diagnostic yield was 40% (n = 2/5) and both patients had a positive family history of disease. For patients with bilateral breast cancer, the diagnostic yield was 32% (n = 8/25), out of whom 50% (n = 4/8) presented with a negative family history.

In patients with unilateral breast cancer who had a positive result, there was a significant association between having TNBC and having a positive genetic test result (p < 0.001). Additionally, logistic regression analysis showed that patients with TNBC were almost 3 times more likely to have a positive result (OR = 2.79, CI = 1.42–5.48 p = 0.003).

Sixty-two patients had TNBC, and 24 of these 62 patients had a positive result (38.7%). Twenty of these sixty-two TNBC patients (32.3%) were positive for BRCA1/2 variants. Two of these 20 patients were also found to have pathogenic variants in a second cancer predisposition gene, CHEK2 in one patient, and MSH6 in the other (patient ID: 6 and 235 respectively). In the remaining four patients with TNBC who tested positive, two patients had di-genic pathogenic variants including, CHEK2 and RAD51C in patient 21 and RAD51D and FANCM in patient 144. The other two patients had pathogenic variants in PALB2 and MLH1. Overall, 38.7% of TNBC patients received a diagnosed of a hereditary breast cancer syndrome and out of those patients, (n = 18/24) 75% had a pathogenic variant in either BRCA1 or BRCA2 solely.

Alternatively, out of all patients who tested positive for BRCA1/2 variants, 21 of 35 (60%) had TNBC. BRCA1/2 positive patients contributed towards only 6.41% of other non-TNBC tumor IHC subtypes (n = 10/156) using Fisher’s Exact test (p value = 0.004).

On the contrary, the association of tumor immunohistochemical subtypes of patients with VUS and negative results was statistically insignificant, (p = 0.710). Furthermore, no statistically significant association was found between having a positive test result and high-grade disease presentation (OR = 1.4, Cl = 0.79–2.47, p = 0.245). This association was also not statistically significant for BRCA1 positive patients and high-grade disease (OR = 2.1, Cl = 0.70–6.40, p = 0.181) (Additional file 1).

### Patient characteristics

Patient characteristics are shown in Table 4.Table 4Patient characteristics of the study participantsStudy Characteristics OverviewN = number of patients% of total patientsTotal Study Participants284100.00Males51.76Asymptomatic10.35Females27998.24Asymptomatic103.52Unilateral24887.32Bilateral258.80Asymptomatic113.87Breast and a secondary malignancy51.76Only breast26894.37Asymptomatic113.87Family History of Disease Negative7526.41 Positive 1st Degree Relative for Breast Cancer Related Malignancy10637.32 Positive 2nd/3rd Degree Relative for Breast Cancer Related Malignancy7225.35 Positive 1-3rd Degree Relative for Non-Breast Cancer Related Malignancy279.51 Incomplete Information41.41

Patient characteristics of the study participants

Out of 284 individuals, 273 patients were diagnosed with breast cancer and 11 asymptomatic individuals underwent testing, owing to their positive family history of breast cancer and/ovarian cancer. These 273 patients included four (1.46%) male breast cancer patients. The median age at diagnosis for all patients was 43 years (IQR: 36–50). Twenty-five of 273 (9.16%) patients had a diagnosis of bilateral breast cancer that included both synchronous and non-synchronous bilateral breast cancer. Five of 273 (1.83%) patients had a diagnosis of a second malignancy other than breast cancer, that included thyroid cancer (n = 2), ovarian cancer (n = 1), endometrial cancer (n = 1) and colon cancer (n = 1). No bilateral disease patients were recorded to have a second malignancy. Of the 11 asymptomatic individuals who underwent testing, one was male.

Histology was recorded for each tumor (n = 298), including unilateral (n = 248) and bilateral disease (n = 25). These included invasive ductal carcinoma (IDC) (n = 257, 86.24%), ductal carcinoma in situ (DCIS) (n = 22, 7.38%), invasive lobular carcinoma (ILC) (n = 9, 3.02%) and invasive papillary carcinoma (IPC) (n = 3, 1.10%). IHC was observed for each tumor recorded individually; from all unilateral and bilateral disease patients, (n = 298), out of which 71(23.83%) were triple negative breast cancer. (TNBC). Other IHC subtypes are summarized in Tables 5, 6, and 7.Table 5Immunohistochemistry subtypes summaryStudy CharacteristicsN = number of tumors% of total tumorsBreast Cancer (unilateral = 248, bilateral = 25)298100.00Grade 1103.36 212341.28 313948.94 Unknown268.72Histology IDC25786.24 DCIS227.38 ILC93.02 Papillary31.01 Unknown72.35Immunohistochemistry Triple Negative (−)7123.83 Triple Positive (+)3511.74 ER−/PR+/HER2+20.67 ER−/PR+/HER2-41.34 ER−/PR-31.01 ER−/PR−/HER2+248.05 ER+/PR−/HER2-134.36 ER+/PR+/HER2-11538.59 ER+/PR−/HER2+51.68 ER+/PR+155.03 ER+/PR-10.34 Unknown103.36Table 6Pathogenic and likely pathogenic variants identified in BRCA1 and BRCA2Sr No.Patient IDAge at diagnosisTypeGeneVariant DetailsExonConsequenceReported in multiple unrelated families in our cohort (Family #)Previously Reported in Pakistani Patients (no. of individuals)Novel VariantsPublished literatureBRCA1125635U/L Breast CaBRCA1c.68_69del (p.Glu23Valfs*17)2Frameshiftalso reported in one unrelated ovarian cancer patientYes (1,1,1)–PMID: 12181777, 31,528,241, (Risch et al., 2001) [21, 22, 41]22236U/L Breast CaBRCA1c.1399_1453dup (p.Ala485Glufs*13)10Frameshift–Yes (1)–PMID:31528241 [22]33625U/L Breast CaBRCA1c.1450G > T(p.Gly484*)10Non-sense––––45134U/L Breast CaBRCA1c.685del (p.Ser229Leufs*5)10Frameshift–Yes (7)–PMID:31528241 [22]517921U/L Breast CaBRCA1c.895_896del (p.Val299Argfs*4)10Frameshift179,191Yes (1)–PMID:31528241 [22]618126U/L Breast CaBRCA1c.3607C > T (p.Arg1203*)10Non-sense–Yes (1)–PMID:31528241 [22]719136U/L Breast CaBRCA1c.895_896del (p.Val299Argfs*4),10Frameshift179,191Yes (1)–PMID:31528241 [22]828345Asymptomatic probandBRCA1c.1583_1589del (p.Thr528Lysfs*2)10Frameshift919437U/L Breast CaBRCA1c.3228_3229del (p.Gly1077Alafs*8)10Frameshift–––1020440U/L Breast CaBRCA1c.4065_4068del (p.Asn1355Lysfs*10)10Frameshift–Yes (4, 4, 1)–PMID: 12181777, 31,528,241, 8,571,953 [21, 22, 42]1122031U/L Breast CaBRCA1c.3169_3172del, p.Ser1057Leufs*4,10Frameshift–Yes (2)–PMID:31528241 [22]1290–Asymptomatic probandBRCA1c.4485-1G > AIntron 13Splice acceptor90, 145Yes (6)–PMID:31528241 [22]1314538, 47B/L Breast CaBRCA1c.4485-1G > AIntron 13Splice acceptor90, 145Yes (6)–PMID:31528241 [22]1422332U/L Breast CaBRCA1c.4508C > A (p.Ser1503*)14Non-sense223,282Yes (3,1)–PMID: 12181777, 31,528,241 [21, 22]1528240U/L Breast Ca, Ovarian CaBRCA1c.4508C > A (p.Ser1503*)14Non-sense223,282–––161549U/L Breast CaBRCA1c.4821del, p.Ala1608Glnfs*2515Frameshift––Yes–178230,43B/L Breast CaBRCA1c.5035del (p.Leu1679*)16Non-sense82, 251Yes (2)––1825130B/L Breast CaBRCA1c.5035del (p.Leu1679*),16Non-sense82, 251–––1928436U/L Breast CaBRCA1c.5074 + 1G > AIntron 16Splice donor7284–PMID:31528241 [22]20727U/L Breast CaBRCA1c.5074 + 1G > AIntron 16Splice donor7284Yes, (2)_PMID:31528241 [22]213128U/L Breast CaBRCA1c.5176delA (p.Arg1726Glufs*4),18Frameshift––––22437U/L Breast CaBRCA1c.5278-1G > A19Splice acceptor––––2323531U/L Breast CaBRCA1,c.5135G > A (p.Trp1712*),17Non-sense––––CHEK2Gain (Exons 3–4)Gain of Ex. 3–4LGR241331U/L Breast CaBRCA1Deletion (Exons 1–11)Deletion of Ex.1–11LGR13,278–––2527837Asymptomatic probandBRCA1Deletion (Exons 1–11)Deletion of Ex.1–11LGR13,278–––262832U/L Breast CaBRCA1Deletion (Exon 1–6)Deletion of Ex.1–6LGR–––2710934U/L Breast CaBRCA1Deletion (Exon 23)Deletion of Ex.23LGR––––286333U/L Breast CaBRCA1Gain (Exon 3–11)Gain of Ex.3–11LGR––Yes–29630U/L Breast CaBRCA1Deletion of Exons 1–2Deletion of Ex.1–2LGR–Yes, (7)–PMID:31528241 [22]MSH6c.3261del (p.Phe1088Serfs*2)5FrameshiftBRCA23010562U/L Breast CaBRCA2Deletion (Exon 3)Deletion of Ex.3LGR––––ATMDeletion (Exon 43)Deletion of Ex. 43LGR314553U/L Breast CaBRCA2c.426-2A > GIntron 4Splice acceptor–Yes (1)–PMID:31528241 [22]3225735U/L Breast CaBRCA2c.859del (p.Met287Cysfs*5)10Frameshift––––3311955U/L Breast CaBRCA2c.3109C > T (p.Gln1037*)11Non-sense–Found in Asian patient (4)PMID: 19241424 [43]34152U/L Breast CaBRCA2c.4718del (p.Cys1573Leufs*6)11Frameshift––––353441U/L Breast CaBRCA2c.6444dupT (p.Ile2149Tyrfs*2)11Frameshift––––3618536U/L Breast CaBRCA2c.4471_4474del (p.Leu1491Lysfs*12)11Frameshift––––377648U/L Breast CaBRCA2c.7806-1G > CIntron 16Splice acceptor––––3820040, 55B/L Breast CaBRCA2c.9380G > A (p.Trp3127*)25Non-sense––––Table 7Pathogenic and likely pathogenic variants identified in genes other than BRCA1 and BRCA2Serial No.Patient IDAge at diagnosisTypeGeneVariant DetailsExonConsequence14429U/L Breast CaATMc.8480 T > G (p.Phe2827Cys),58Missense213558, 63B/L Breast CaBARD1Deletion (Entire coding sequence)Exon 1–11LGR316940U/L Breast CaBARD1Deletion (Entire coding sequence)Exon 1–11LGR42041U/L Breast CaCHEK2c.58C > T (p.Gln20*)2Non-sense59635U/L Breast CaCHEK2Deletion (Exon 5)Deletion of Ex. 5LGR616560B/L Breast CaCHEK2c.409C > T (p.Arg137*)3Non-sense722540U/L Breast CaCHEK2c.58C > T (p.Gln20*)2Non-sense82144U/L Breast CaCHEK2c.283C > T (p.Arg95*)2Non-senseRAD51Cc.701C > G (p.Ser234*)4Non-sense924356U/L Breast CaFANCMc.4153G > T (p.Glu1385*),14Non-sense1027943U/L Breast CaFANCMc.2199_2202del (p.Ser734Asnfs*25),13Frameshift111143U/L Breast CaMLH1Deletion (Exon 16–19)Deletion of Ex. 16–19LGR1224160U/L Breast CaMLH1c.1897-2A > G (Splice acceptor)Intron 16Splice acceptor1326939U/L Breast CaMLH1c.306G > T (p.Glu102Asp)3Missense1410230U/L Breast CaMSH6c.1222_1226del (p.Pro408Aspfs*8),4Frameshift1520251U/L Breast CaMUTYHc.312C > A (p.Tyr104*)3Non-sense1624848U/L Breast CaMUTYHc.312C > A (p.Tyr104*)3Non-sense178066U/L Breast CaNF1c.5205 + 5G > AIntron 36Splice donor1822738U/L Breast CaPALB2c.2488del (p.Glu830Serfs*21),5Frameshift1925842U/L Breast CaPALB2c.2353_2354del (p.Pro785Thrfs*16)5Frameshift205764B/L Breast CaPALB2c.2488del (p.Glu830Serfs*21)5Frameshift211834,37B/L Breast CaRAD51Dc.898C > T, p.Arg300*,9Non-sense2214455U/L Breast CaRAD51Dc.620C > T (p.Ser207Leu)7MissenseFANCMc.4318-1G > A (Splice acceptor)Intron 15Splice acceptor2318440U/L Breast CaTP53c.537 T > G (p.His179Gln)5Missense2424029U/L Breast CaTP53c.437G > A (p.Trp146*)5Non-sense

Immunohistochemistry subtypes summary

Pathogenic and likely pathogenic variants identified in BRCA1 and BRCA2

Pathogenic and likely pathogenic variants identified in genes other than BRCA1 and BRCA2

This included all patients with unilateral (n = 62) and bilateral disease (n = 8). In the bilateral disease patient group, three patients had both tumors as triple negative (n = 6), and three had either of the two tumors as triple negative (n = 3), that along with 62 unilateral disease patients made a total of 71 TNBC tumors. In the DCIS and ILC and IPC, none of the tumors were found to be triple negative. The statistical analyses involving IHC subtypes were carried out on patients with unilateral disease with IDC histology.

### Multi-gene panel result distribution

Out of 273 patients, 59 (22%) harbored P/LP variant (positive result), 103 (37%) had a VUS and 111 (41%) did not harbor any P/LP variant or a VUS (negative result). Of the 59 patients that harbored P/LP variants, 5 had a P/LP variant in two genes. Out of these total 64 P/LP variants, 35 (55%) were in either BRCA1 or BRCA2. The spectrum of the identified variants is summarized in Fig. 1.Fig. 1The spectrum of the identified variants is summarized, showing 45% of the positive results are attributed to non BRCA1/2 genes

The spectrum of the identified variants is summarized, showing 45% of the positive results are attributed to non BRCA1/2 genes

Patients with P/LP variants on their genetic test had a younger age at diagnosis compared to those having a VUS and negative result; median age = 37.5, IQR (31.5–48) vs median age = 45, vs, median age = 44, (p = 0.002) respectively, using Mann-Whitney test. A larger percentage of results with P/LP variants were found in the youngest age group of 25–34 years (n = 19) and 35–44 years (n = 25), accounting for 74% (n = 44/59) of the positive genetic test results. Eleven of these 44 patients (25%) had a negative family history of breast or related cancers.

One hundred and sixty-seven of the 273 patients (61%) reported a positive family history. Forty-five of these 167 patients (27%) harbored a P/LP variant, compared to 15 out of 102 (15%) patients with a negative family history. Out of all patients who harbored a P/LP variant, (n = 15/60) 25% had a negative family history for breast or related malignancies.

We also analyzed if there was an association found between family history of disease and genetic test result outcomes. Having a positive family history was associated with having a positive test result (p = 0.019). However, as expected, the association of family history was not established in case of VUS (p = 0.780) and negative (p = 0.088) result outcomes. Furthermore, the likelihood of a genetic test being positive was two times higher in the presence of a family history of cancer as compared to no family history of cancer (OR = 2.13, CI = 1.12–4.08, p = 0.021).

In patients with unilateral disease who presented without a history of another malignancy; the diagnostic yield was 20% (n = 49/243). Out of them 20% (n = 11/50) presented with negative family history of disease. Five patients with unilateral disease also presented with a secondary malignancy. For these patients, the diagnostic yield was 40% (n = 2/5) and both patients had a positive family history of disease. For patients with bilateral breast cancer, the diagnostic yield was 32% (n = 8/25), out of whom 50% (n = 4/8) presented with a negative family history.

In patients with unilateral breast cancer who had a positive result, there was a significant association between having TNBC and having a positive genetic test result (p < 0.001). Additionally, logistic regression analysis showed that patients with TNBC were almost 3 times more likely to have a positive result (OR = 2.79, CI = 1.42–5.48 p = 0.003).

Sixty-two patients had TNBC, and 24 of these 62 patients had a positive result (38.7%). Twenty of these sixty-two TNBC patients (32.3%) were positive for BRCA1/2 variants. Two of these 20 patients were also found to have pathogenic variants in a second cancer predisposition gene, CHEK2 in one patient, and MSH6 in the other (patient ID: 6 and 235 respectively). In the remaining four patients with TNBC who tested positive, two patients had di-genic pathogenic variants including, CHEK2 and RAD51C in patient 21 and RAD51D and FANCM in patient 144. The other two patients had pathogenic variants in PALB2 and MLH1. Overall, 38.7% of TNBC patients received a diagnosed of a hereditary breast cancer syndrome and out of those patients, (n = 18/24) 75% had a pathogenic variant in either BRCA1 or BRCA2 solely.

Alternatively, out of all patients who tested positive for BRCA1/2 variants, 21 of 35 (60%) had TNBC. BRCA1/2 positive patients contributed towards only 6.41% of other non-TNBC tumor IHC subtypes (n = 10/156) using Fisher’s Exact test (p value = 0.004).

On the contrary, the association of tumor immunohistochemical subtypes of patients with VUS and negative results was statistically insignificant, (p = 0.710). Furthermore, no statistically significant association was found between having a positive test result and high-grade disease presentation (OR = 1.4, Cl = 0.79–2.47, p = 0.245). This association was also not statistically significant for BRCA1 positive patients and high-grade disease (OR = 2.1, Cl = 0.70–6.40, p = 0.181) (Additional file 1).

### Discussion

To our knowledge, this is the first study to report the spectrum of germline pathogenic variants in genes beyond BRCA1/2 identified in breast cancer patients from Pakistan, using an NGS based multi-cancer gene panel and correlating with clinical characteristics. We report a positivity rate of 22% in our patients, confirming a diagnosis for HBC. This is within but on the higher end of the range of positivity due to P/LP variants in BRCA1/2 variants in various other countries, where it is reported to be between 9.4 to 29.8% [11, 41–54]. However, most of these studies focused on BRCA1/2 testing only and used different clinical criteria for testing. In our study, we used standardized NCCN criteria and expanded multi-gene NGS panel testing.

In previous studies from Pakistan in which only BRCA1/2 variants were tested, positive results were obtained in 25% of high-risk families for breast and ovarian cancer syndrome. Rashid et al. identified nine variants that were specific to the Pakistani population, with 18 and three recurrent variants in BRCA1 and BRCA2 respectively [22, 23, 44, 45]. In our patients, we observed seven variants in BRCA1 that had been previously reported by Rashid el al, that included c.68_69del, c.1399_1453dup, c.895_896del, c.4065_4068del, c.4508C > A, c.5074 + 1G > A and Deletion of Exons 1–2. Twenty-nine patients had pathogenic variants in BRCA1, harboring 23 unique variants including 6 variants that were found in multiple unrelated patients, which included c. 895_896del, c. 4485-1G > A, c.4508C > A, c.5035del, c.5074 + 1G > A and Deletion of Exon 1–11. One patient harbored c.3109C > T in BRCA2, which is a variant previously reported in Asian patients, but not specifically in Pakistani patients [46]. We also report two novel pathogenic variants in BRCA1 that had not been previously reported in the literature, including Gain (Exon 3–11) and c.4821del.

Our work demonstrates that genes beyond BRCA1/2 genes contributed to 45% (n = 29/64) of the positive results. This validates the utility of multi-gene panel testing over BRCA1/2 only, improving the diagnostic yield by 8.8% (from n = 35/273 to n = 59/273) with multi-gene panel, comparable to what other studies have reported [29–33]. The diagnostic yield for unilateral disease was found to be 20%, while that for patients with a secondary malignancy was 40%, and for bilateral disease it was 32%, showing the predictive value of these two important clinical factors in identifying high-risk patients for testing. It also indicates that more work needs to be done to identify other genetic or non-hereditary biological factors in patients with bilateral disease, or breast cancer with an additional malignancy.

It is worth noting that out of 59 patients who had tested positive for genes present in the panel, five patients had disease-causing variants in multiple genes, (patient ID: 6, 21, 105, 144 and 235); in BRCA1 and MSH6, CHEK2 and RAD51C, ATM and BRCA2, FANCM and RAD51D and BRCA1 and CHEK2; respectively. This further signifies the utility of multi-gene panel testing looking beyond BRCA1/2 for optimal patient care. We will need to follow these patients over time to see if they follow a course that is more aggressive compared to patients with single pathogenic variants.

In our patient population, 26% (n = 15) patients who tested positive were over the age of 45. Out of the 44 patients who were below the age of 45 years at the time of diagnosis, 11 patients had a negative family history of breast or related cancers. This validates that age cut-off of 45 years as a stand-alone criterion for referral for germline testing as recommended by NCCN 2020 guidelines for our population as well. It is crucial to understand the importance of following recommended referral criteria. Otherwise, often the referring providers tend to use age cut-off of 45 plus a positive family history of disease to make the referrals to genetics clinic. If that was done in our clinic, then 26% patients would not have received their diagnosis of an HBC. In the larger group, we observed that positive family history doubles the likelihood of testing positive. Nonetheless, a quarter of patients testing positive had a negative family history of disease. This highlights the importance of offering germline testing to individuals even without a positive family history, as long as they meet other criteria for genetic testing. Making use of the phenotype and genotype information, we also attempted to evaluate the effect of updating our institutional testing criteria from NCCN guidelines 2016 to NCCN 2020 [9, 35]. We found that if the age cut-off as a stand-alone testing criterion would to be reduced from 50 years to 45 years, we would fail to establish HBC diagnosis in 1 (1.66%) of the study participants (n = 1). Additional file 2 discusses our patient stratification pipeline in detail.

We also observed that over a half of the patients with a VUS or a negative result had a positive family history. This may indicate the polygenic and multifactorial nature of breast cancer in those families, as well as the need to continue to identify novel monogenic causes of HBC that are currently not covered by commercially available multi-gene NGS panels [55, 56].

While reporting correlation between positive results and histopathological findings, we observed that the likelihood of a positive result across all genes was 3 times higher in TNBC (OR = 2.8, CI = 1.42–5.48 p = 0.003. The established association of BRCA1/2 positive results with TNBC, was supported in our study population, with BRCA1/2 positive patients contributed towards 32.3% cases with TNBC (p = < 0.001) [49, 50].

It was noted that other positive results in patients with TNBC included pathogenic variants in CHEK2, FANCM, RAD51C, RAD51D, MLH1 and PALB2. Four TNBC patients were identified to have di-genic pathogenic variants. Out of the genes that have a role in the BRCA1/2 HR pathway (Additional file 3), NCCN Guidelines (version 2.2021) indicate that the risk of triple negative breast cancer is potentially increased in patients with P/LP variants in BRIP1, RAD51C and RAD51D [9, 49, 50]. Couch et al. and Shimelis et al. also found association of TNBC with BARD1 and PALB2, in addition to BRIP1, RAD51C and RAD51D [57, 58].

In patients with TNBC, a diagnosis of HBC was not established in 61.3% of patients. The diagnostic yield of 38.7% in our patients, however, was 2-folds higher than that reported by Couch et al. (14.6%) in a multicenter study cohort, unselected for family history of disease [40].

For patients who tested positive for BRCA1/2, 60% had TNBC; emphasizing that TNBC should not be seen as an isolated testing criterion; otherwise in about 40% cases, one would miss a diagnosis of HBC.

Studies have shown that BRCA1 P/LP variant positive patients present with high-grade disease, compared to BRCA2 P/LP variant positive patients and with sporadic/non-hereditary cases of breast cancer [59–65]. In our study, no statistically significant association was found in the analyzed subsets, and at large, the likelihood of getting a positive result with a higher disease grade was not established.

Since the patients were referred to our centre from multiple institutions, we could not systemically capture the ethnicity details for the patients thereby, the ethnic diversity in the patient population cannot be objectively discussed. Given the fact that we are a referral center for the region, we believe that our patients do indeed represent the ethnic diversity present in the population. We will address this in future studies by documenting ethnicity information in our databases, as well as collaborating with multiple centers elsewhere in the country to ensure that we can objectively look at ethnic differences in the HBC population in our country.

In addition, of the 273 patients, the final histopathological statistical analysis was carried out on a total of 239 patients diagnosed with invasive ductal carcinoma (unilateral breast cancer n = 217 and bilateral breast cancer n = 22). Information on IHC subtypes for unilateral IDC cancer patients was available for (n = 215 out of 217), analysis carried out only on unilateral cancers, as bilateral cancer patients also included those with multiple possible IHC subtypes. The patient disease characteristics and triage framework is summarized in Figs. 2(A and B).Fig. 2A Patient characteristics of the study participants (flow chart). B Patient Disease characteristics and triage framework of the study

A Patient characteristics of the study participants (flow chart). B Patient Disease characteristics and triage framework of the study

### Study limitations

Since the patients were referred to our centre from multiple institutions, we could not systemically capture the ethnicity details for the patients thereby, the ethnic diversity in the patient population cannot be objectively discussed. Given the fact that we are a referral center for the region, we believe that our patients do indeed represent the ethnic diversity present in the population. We will address this in future studies by documenting ethnicity information in our databases, as well as collaborating with multiple centers elsewhere in the country to ensure that we can objectively look at ethnic differences in the HBC population in our country.

In addition, of the 273 patients, the final histopathological statistical analysis was carried out on a total of 239 patients diagnosed with invasive ductal carcinoma (unilateral breast cancer n = 217 and bilateral breast cancer n = 22). Information on IHC subtypes for unilateral IDC cancer patients was available for (n = 215 out of 217), analysis carried out only on unilateral cancers, as bilateral cancer patients also included those with multiple possible IHC subtypes. The patient disease characteristics and triage framework is summarized in Figs. 2(A and B).Fig. 2A Patient characteristics of the study participants (flow chart). B Patient Disease characteristics and triage framework of the study

A Patient characteristics of the study participants (flow chart). B Patient Disease characteristics and triage framework of the study

### Conclusion

In our HBC clinic, where NCCN guidelines were used to triage patients for genetic testing, the percentage of patients who tested positive was 22%, a rate on the higher end of what is reported in other populations.

The importance of expanded, multi-gene panel testing over just BRCA1/2 testing is highlighted, as 41% (n = 24/59) of the patients had P or PL variants in genes other than BRCA1/2. Our test positivity rate (excluding the asymptomatic individuals) would have only been 12.8% (n = 35/273) if only BRCA1/2 testing was done. We also observed that in patients with unilateral disease without a second malignancy, 20% were identified to have HBC. For unilateral patients with a secondary malignancy, and those with bilateral disease; 40 and 32% were identified to have HBC, respectively. Consistent with data from other countries, we also found that in our population, having TNBC or a young age at diagnosis increased the likelihood of a positive result. In over 60% of the patients with TNBC, the diagnosis of HBC was not established. Up to a quarter of patients with positive result had a negative family history. Over half of patients with a VUS or a negative result presented with a positive disease history.

To our knowledge, this is the first study from Pakistan to report the clinical utility and clinical correlation in patients who were tested through a multi-gene panel for HBC. As the database expands and protocol-driven referrals are made across the country, our insight about the genetic architecture of HBC in ethnically diverse Pakistani population will continue to increase. Subsequently, this will enable better elucidation of the underlying disease pathology to help devise improved preventive, diagnostic and therapeutic interventions to reduce breast cancer associated morbidity and mortality.

### Supplementary Information

Additional file 1. Presenting histopathological characteristics and Positive result association.Additional file 2. Updating the institutional referral criteria from NCCN criteria 2016 to 2020.Additional file 3. BRCA1/2 DDR and DNA repair pathway.

Additional file 1. Presenting histopathological characteristics and Positive result association.

Additional file 2. Updating the institutional referral criteria from NCCN criteria 2016 to 2020.

Additional file 3. BRCA1/2 DDR and DNA repair pathway.

Additional file 1. Presenting histopathological characteristics and Positive result association.Additional file 2. Updating the institutional referral criteria from NCCN criteria 2016 to 2020.Additional file 3. BRCA1/2 DDR and DNA repair pathway.

Additional file 1. Presenting histopathological characteristics and Positive result association.

Additional file 2. Updating the institutional referral criteria from NCCN criteria 2016 to 2020.

Additional file 3. BRCA1/2 DDR and DNA repair pathway.



# SUPPLEMENTAL FILE 1: 13053_2022_Article_232.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)