# MAIN TEXT

## Validation of AmpliSeq NGS Panel for BRCA1 and BRCA2 Variant Detection in Canine Formalin-Fixed Paraffin-Embedded Mammary Tumors

### Abstract

Mammary carcinomas are the most common neoplasms observed in women and in female dogs. Canine mammary tumors show epidemiological, clinical, genetic, and prognostic characteristics comparable to human breast cancers. The recent introduction of next generation sequencing (NGS) technologies has greatly improved research and diagnostics for humans, while these new tools still need to be implemented in animal models. In this study we developed and validated an AmpliSeq Panel assay for the identification of BRCA variants in twenty-two different dogs. The amplicon mean coverage was 5499× and uniformity was higher than 98% in all samples. The results of germline single nucleotide variants (SNVs) and insertions/deletions (INDELs) were fully concordant regardless of the types of samples considered (blood, fresh and FFPE tissues). Moreover, despite the high DNA degradation observed in older FFPE blocks (>5 years), the assay allowed full coverage of all amplicons for downstream analyses. We consider the NGS panel developed in this study as a useful tool for expanding information on BRCA genes in the veterinary field and for human health from a comparative oncology perspective.

### 1. Introduction

BRCA1 and BRCA2 are tumor suppressor genes that have been associated with increased risk, mainly for breast and ovarian human cancers, when carrying specific mutations. These genes are also involved in canine mammary tumors (CMTs), which are the most common neoplasms in female animals, representing a major health threat also for dogs [1].

Similarities in the biological, epidemiological, and genetic characteristics of canine and human mammary tumors suggested that dogs may provide a useful model for human breast cancer research. Humans and dogs share over 80% of the aminoacidic residues of BRCA genes [2]. Moreover, the clinically common traits of human and dog breast cancer include spontaneous tumor incidence, onset age, hormonal etiology, and the identical course of the disease. In addition, canine tumors also exhibit the principal pathologic features of human cancers, including a long-term oncogenic setting, intratumoral heterogeneity, acquired resistance to treatments, tumor size, clinical stage, lymph node invasion and distant metastases [3,4].

The recent introduction of next-generation sequencing (NGS) is changing the genetic analysis and research of human BRCA due to its high throughput and cost-effectiveness [5,6,7]. These new tools allow researchers to detect already known SNVs, but also make possible the discovery of new variants within these two large genes.

The most common technique for downstream diagnosis, classification, and storage of cancer tissues in many veterinary pathology laboratories is formalin-fixed, paraffin-embedded (FFPE). The use of this technique has also intensified to enlarge the biobanks. All this has led to a significant increase in FFPE samples that could be available for further analysis, such as retrospective investigations. FFPE blocks also provide useful information on histotypes, in addition to the presence of precancerous lesions. Such data, combined with other techniques such as immunohistochemistry and NGS, can complete the scenario of specific lesions.

In human medicine, many studies have shown the effectiveness of using the NGS technique applied to FFPE tissues [8,9]; however, despite this, in veterinary medicine there are few reports that use this technique, many of which are focused on etiological agent detection [10]. An AmpliSeq panel for BRCA1 and BRCA2 has recently been published for cats [11], while the great potential of NGS is not yet explored for BRCA research and diagnostics in dogs. To date, studies on dog BRCA mutations and the possible association with mammary cancer are very limited. Unfortunately, most investigations have focused on highly conserved regions only, which often represent sites of interaction with the RAD51 protein [12,13,14].

Aiming to fill in this gap, we present here the development and validation of a new AmpliSeq BRCA Panel assay on an Illumina platform for the identification of variants across BRCA1 and BRCA2 genes in dogs. Rather than the differentiation between somatic or germline variations introduced in neoplastic tissues and those inherited, this study focuses instead on the analysis of FFPE blocks. It is also worth considering that this method may open a new window for the retrospective analysis of historical samples.

Finally, we compared variants found in dogs with those already described in humans to evaluate the possible clinical effects and a new strategy for classification in canine samples.

### 2. Materials and Methods

Twenty-two FFPE tissue samples from pathological (n. 20) and normal (n. 2) canine mammary tissues were retrieved from the archives of the Diagnostic Department of Istituto Zooprofilattico Sperimentale dell’Abruzzo e del Molise (Table 1). Seven out of twenty mammary tumors were submitted to the laboratory immediately after surgery, together with a 1 mL blood sample collected in EDTA. These neoplastic specimens were sectioned in half: one part was fixed in 10% buffered formalin for 48 h, processed and embedded in paraffin wax, and the other part was frozen at −80 °C. From these seven animals (ID: 1, 2, 3, 4, 7, 8, 9), we sampled blood, FFPE blocks and fresh tissues. Normal mammary gland tissues were collected during routine necropsies from female dogs who died from causes unrelated to oncologic diseases. Written consent for specimen collection and analysis was provided by each owner. Medical history and anamnesis, including breed, age and sex, were taken when available. FFPE samples were also grouped as old (2014–2016), medium (2017–2019) and recent (2020–2021), depending on the time of storage.

Genomic DNA was extracted from blood, fresh tissues, and FFPE specimens. More precisely, DNA from blood samples (200 µL) and fresh tissues (25 mg) were extracted by means of the Maxwell 16 Blood DNA Purification Kit (Promega, Madison, CA, USA) and the Maxwell 16 Tissue DNA Purification Kit (Promega), respectively. FFPE samples were extracted using the Maxwell® RSC DNA FFPE Kit (Promega) with a slight modification to the protocol at point 7 where the samples were incubated on a heat block at 65 °C overnight instead of 80 °C for 4 h. Cartridges provided by the three different kits were loaded on the Maxwell 16 Instrument (Promega). Quality control was performed on DNA by the Agilent 4200 TapeStation System (Agilent Technologies, Santa Clara, CA, USA) using the Genomic DNA ScreenTape and Genomic DNA reagents (Agilent Technologies). DNA was also quantified by the Qubit 2.0 Fluorometer (Thermo Fisher Scientific, Waltham, MA, USA) and then diluted to 4 ng/µL using Low TE buffer (10 mM Tris-HCl (pH 8.0) + 0.1 mM EDTA) for library preparation.

A total of 197 custom amplicons (Supplementary Materials) were used for specific amplification of 19,804 bp BRCA1 and BRCA2 exonic regions of Canis familiaris (UCSC canFam3). The amplicon average length was 162 bp, with a maximum length of 175 bp. Libraries were prepared by means of the AmpliSeq Library PLUS (Illumina, San Diego, CA, USA) and AmpliSeq UD Indexes for Illumina (IDT, Coralville, IA, USA), following the protocol described in the AmpliSeq for Illumina On-demand, Custom, and Community Panels Reference Guide (Document # 1000000036408 v09). Quality control of the individual libraries (size and concentration) was fulfilled by the Agilent 4200 TapeStation System (Agilent Technologies), using the High Sensitivity D5000 ScreenTape and reagents (Agilent Technologies), and Qubit 2.0 Fluorometer (Thermo Fisher Scientific). Manually normalized libraries were then pooled, diluted to 1.2 pM and loaded in the MiniSeq Mid Output reagent cartridge (Illumina). Sequencing was performed by the Illumina MiniSeq Sequencer setting 300 cycles (150 bp paired-end reads), followed by primary analysis performed by the on-instrument software SCS/RTA (Illumina). Secondary analysis was accomplished by the DNA Amplicon workflow available on Local Run Manager Version 2.4.1 (Illumina, San Diego, CA, USA), using the manifest file (IAA19398_179_manifest.txt) (Supplementary Materials). Analyses were performed by using the software aligner BWA-MEM Whole-Genome (v0.7.9a-isis-1.0.2), Pisces Variant Caller (v5.2.9.23), Bam Metrics (v0.0.22) and SAMtools (v0.1.19-isis-1.0.3), all provided by Illumina (San Diego, CA, USA). Only variants with quality values 100 and filter “pass” were considered for the analysis. Finally, tertiary analysis was established by using both the Genome Browser, available online at https://genome.ucsc.edu/ (accessed on 27 May 2022), on CanFam3/CanFam3.1 genome, and Alamut Plus software v.1.0 (SOPHiA GENETICS SA, Lausanne, VD, CH), in order to gain biological significance for BRCA1/BRCA2 variants from dog versus human comparisons. Statistical analysis was completed by the Kruskal–Wallis non-parametric test.

### 2.1. Sample Selection

Twenty-two FFPE tissue samples from pathological (n. 20) and normal (n. 2) canine mammary tissues were retrieved from the archives of the Diagnostic Department of Istituto Zooprofilattico Sperimentale dell’Abruzzo e del Molise (Table 1). Seven out of twenty mammary tumors were submitted to the laboratory immediately after surgery, together with a 1 mL blood sample collected in EDTA. These neoplastic specimens were sectioned in half: one part was fixed in 10% buffered formalin for 48 h, processed and embedded in paraffin wax, and the other part was frozen at −80 °C. From these seven animals (ID: 1, 2, 3, 4, 7, 8, 9), we sampled blood, FFPE blocks and fresh tissues. Normal mammary gland tissues were collected during routine necropsies from female dogs who died from causes unrelated to oncologic diseases. Written consent for specimen collection and analysis was provided by each owner. Medical history and anamnesis, including breed, age and sex, were taken when available. FFPE samples were also grouped as old (2014–2016), medium (2017–2019) and recent (2020–2021), depending on the time of storage.

### 2.2. DNA Extraction

Genomic DNA was extracted from blood, fresh tissues, and FFPE specimens. More precisely, DNA from blood samples (200 µL) and fresh tissues (25 mg) were extracted by means of the Maxwell 16 Blood DNA Purification Kit (Promega, Madison, CA, USA) and the Maxwell 16 Tissue DNA Purification Kit (Promega), respectively. FFPE samples were extracted using the Maxwell® RSC DNA FFPE Kit (Promega) with a slight modification to the protocol at point 7 where the samples were incubated on a heat block at 65 °C overnight instead of 80 °C for 4 h. Cartridges provided by the three different kits were loaded on the Maxwell 16 Instrument (Promega). Quality control was performed on DNA by the Agilent 4200 TapeStation System (Agilent Technologies, Santa Clara, CA, USA) using the Genomic DNA ScreenTape and Genomic DNA reagents (Agilent Technologies). DNA was also quantified by the Qubit 2.0 Fluorometer (Thermo Fisher Scientific, Waltham, MA, USA) and then diluted to 4 ng/µL using Low TE buffer (10 mM Tris-HCl (pH 8.0) + 0.1 mM EDTA) for library preparation.

### 2.3. Sequencing and Data Analysis

A total of 197 custom amplicons (Supplementary Materials) were used for specific amplification of 19,804 bp BRCA1 and BRCA2 exonic regions of Canis familiaris (UCSC canFam3). The amplicon average length was 162 bp, with a maximum length of 175 bp. Libraries were prepared by means of the AmpliSeq Library PLUS (Illumina, San Diego, CA, USA) and AmpliSeq UD Indexes for Illumina (IDT, Coralville, IA, USA), following the protocol described in the AmpliSeq for Illumina On-demand, Custom, and Community Panels Reference Guide (Document # 1000000036408 v09). Quality control of the individual libraries (size and concentration) was fulfilled by the Agilent 4200 TapeStation System (Agilent Technologies), using the High Sensitivity D5000 ScreenTape and reagents (Agilent Technologies), and Qubit 2.0 Fluorometer (Thermo Fisher Scientific). Manually normalized libraries were then pooled, diluted to 1.2 pM and loaded in the MiniSeq Mid Output reagent cartridge (Illumina). Sequencing was performed by the Illumina MiniSeq Sequencer setting 300 cycles (150 bp paired-end reads), followed by primary analysis performed by the on-instrument software SCS/RTA (Illumina). Secondary analysis was accomplished by the DNA Amplicon workflow available on Local Run Manager Version 2.4.1 (Illumina, San Diego, CA, USA), using the manifest file (IAA19398_179_manifest.txt) (Supplementary Materials). Analyses were performed by using the software aligner BWA-MEM Whole-Genome (v0.7.9a-isis-1.0.2), Pisces Variant Caller (v5.2.9.23), Bam Metrics (v0.0.22) and SAMtools (v0.1.19-isis-1.0.3), all provided by Illumina (San Diego, CA, USA). Only variants with quality values 100 and filter “pass” were considered for the analysis. Finally, tertiary analysis was established by using both the Genome Browser, available online at https://genome.ucsc.edu/ (accessed on 27 May 2022), on CanFam3/CanFam3.1 genome, and Alamut Plus software v.1.0 (SOPHiA GENETICS SA, Lausanne, VD, CH), in order to gain biological significance for BRCA1/BRCA2 variants from dog versus human comparisons. Statistical analysis was completed by the Kruskal–Wallis non-parametric test.

### 3. Results

Results from Qubit quantification showed a DNA concentration higher than the input required by the protocol (4 ng/µL) in all the samples considered in this study. DNA integrity (DIN) evidenced different grades of fragmentation according to the time of storage of FFPE samples and to the different matrices considered (FFPE, blood, fresh tissue). As expected, the highest level of degradation was observed for FFPE samples and, within them, for those stored for a larger period (Table 2, Figure 1).

The amplicon mean (deep) coverage was 5499×, and a mean of 1.2 M passing filter reads (Q score > 30) were obtained from the samples. Uniformity of coverage was higher than 98% in all samples (Figure 2). Reproducibility of single nucleotide variants (SNVs) and insertions/deletions (INDELs) found in the three different matrices (blood, fresh and FFPE tissues) of 7 animals was always 100%. Moreover, no statistically significant differences were observed for both total reads number and amplicon mean coverage in the three matrices (p > 0.05).

Conversely, FFPE samples stored for different numbers of years showed progressively better results from the oldest to the recent ones in terms of total reads number and amplicon mean coverage (Figure 3). These results are statistically significant with p-values 0.00819 and 0.00174 for total reads and coverage, respectively.

A total of 42 different genetic variants were identified in 22 dogs. In particular, 12 exonic and 4 intronic variants were found in BRCA1, while 23 exonic and 3 intronic variants were identified in BRCA2 (Table 3). According to human classification, variations in BRCA1 were catalogued as likely benign (n. 2), and not referenced (n. 14), while mutations in BRCA2 were classified as likely benign (n. 3), variants of unknown significance (n. 5), not referenced (n. 15), and not found in humans (n. 3).

Regarding the exonic mutations, we observed 10 coding single nucleotide polymorphisms (SNPs) and 2 synonymous SNPs in BRCA1, whereas in BRCA2, 17 variants were coding SNPs, 4 synonymous SNPs, 1 deletion and 1 insertion both of 3 bp (Table 4).

### 3.1. DNA Quality

Results from Qubit quantification showed a DNA concentration higher than the input required by the protocol (4 ng/µL) in all the samples considered in this study. DNA integrity (DIN) evidenced different grades of fragmentation according to the time of storage of FFPE samples and to the different matrices considered (FFPE, blood, fresh tissue). As expected, the highest level of degradation was observed for FFPE samples and, within them, for those stored for a larger period (Table 2, Figure 1).

### 3.2. Amplicon Coverage and Variants Found

The amplicon mean (deep) coverage was 5499×, and a mean of 1.2 M passing filter reads (Q score > 30) were obtained from the samples. Uniformity of coverage was higher than 98% in all samples (Figure 2). Reproducibility of single nucleotide variants (SNVs) and insertions/deletions (INDELs) found in the three different matrices (blood, fresh and FFPE tissues) of 7 animals was always 100%. Moreover, no statistically significant differences were observed for both total reads number and amplicon mean coverage in the three matrices (p > 0.05).

Conversely, FFPE samples stored for different numbers of years showed progressively better results from the oldest to the recent ones in terms of total reads number and amplicon mean coverage (Figure 3). These results are statistically significant with p-values 0.00819 and 0.00174 for total reads and coverage, respectively.

A total of 42 different genetic variants were identified in 22 dogs. In particular, 12 exonic and 4 intronic variants were found in BRCA1, while 23 exonic and 3 intronic variants were identified in BRCA2 (Table 3). According to human classification, variations in BRCA1 were catalogued as likely benign (n. 2), and not referenced (n. 14), while mutations in BRCA2 were classified as likely benign (n. 3), variants of unknown significance (n. 5), not referenced (n. 15), and not found in humans (n. 3).

Regarding the exonic mutations, we observed 10 coding single nucleotide polymorphisms (SNPs) and 2 synonymous SNPs in BRCA1, whereas in BRCA2, 17 variants were coding SNPs, 4 synonymous SNPs, 1 deletion and 1 insertion both of 3 bp (Table 4).

### 4. Discussion

Amplicon sequencing (AmpliSeq) is a targeted approach which enables researchers to analyze genetic variation in specific genomic regions. The ultra-deep sequencing of PCR products allows efficient variant identification and characterization in a wide range of fields [17,18,19]. AmpliSeq is particularly useful for the discovery of rare somatic mutations in complex samples such as tumors mixed with germline DNA. Moreover, the information generated by targeted gene panels can be helpful for diagnostic classification, therapeutic decisions, and/or prognostic insights [20]. In this study, we described the development and validation of an AmpliSeq approach for the identification BRCA1 and BRCA2 variants in dogs (Canis familiaris). The Amplicon panel has been designed to cover the whole BRCA1/BRCA2 exome and the proximal intronic regions (~30–50 bp), in both directions. The short-length PCR panel has been properly designed for fragmented DNA (e.g., FFPE samples). Indeed, despite the high degradation of DNA observed in FFPE samples, particularly true for DNA obtained from older FFPE blocks (>5 years), the assay was able to produce good quality results for all the samples. In the first sequencing attempt, we pooled together all the samples without previous individual quantification and normalization. The pool was then quantified and diluted as recommended but the resulting coverage was unbalanced between samples and sometimes unsuitable for downstream analysis. The poorer the DNA quality the lower the total reads number and amplicon mean coverage occurred. The same libraries were thus quantified and normalized at the equal molar concentration before pooling and sequencing. Following this procedure, we obtained more standardized reads number and amplicon mean coverage among the samples. Despite the variation observed in FFPE samples, depending on the time of storage, all the samples were successfully analyzed. Individual normalization is thus a critical point to obtain reliable quality results for strong fragmented DNA samples.

Variant analysis of the seven dogs for which all three matrices were available in this study, confirmed the high reproducibility of the whole workflow, from DNA extraction to variant calling. In such samples, we could establish with certainty the germinal nature of the variants found, although this was not among our goals.

In addition to the method validation, this study also attempted the interpretation of the variants found in dogs. In humans, family information is generally integrated to evaluate the probability that sequence variants are associated with genetic predisposition [21]. In contrast, much of this information is not available for dogs. The results of dog samples were thus interpreted on the basis of software and classifications developed for humans.

In this study, the dog versus human BRCA1/BRCA2 comparison was established by using Alamut Visual Plus, while the interpretation of the dog variants found in both humans and dogs were carried out using ClinVar, a software that archives and aggregates information about relationships among variation and health in humans. Only two BRCA1 variants were referenced in humans and classified as likely benign, while eight variants of BRCA2 were also reported in humans; five were classified as of uncertain significance (UV) and three as likely benign. Among UVs, the human BRCA2 variants p.(Lys1440Arg), p.(Arg1997Lys) and p.(Trp395Cys), showed the same amino acid substitution described here in dogs, while at the position of human BRCA2 p.(His150Arg), we found p.(His143Pro).

Four out of 22 different dogs (ID: 2, 4, 14, 15) showed c.4314A>G—p.(Lys1435Arg) together with c.4283A>C—p.(Thr1425Pro), both in exon 11 of BRCA2 and previously described in the literature as implicated in carcinogenesis of dogs [13,14,22]. Both mutations were also confirmed by Sanger sequencing (data not shown). Of them, two showed an intraductal papillary carcinoma, one a solid carcinoma and one a complex carcinoma in a broad range of grade of malignancy (I–III); correlation to a specific histotype is thus excluded.

In healthy dogs (n = 2), we did not find c.4314A>G nor c.4283A>C in FFPE samples, nor in fresh tissues. These variations are located in the BRC3 repeat domain, which plays a fundamental role in the interaction with the protein RAD51, and affect well conserved positions [23,24] in available vertebrate species. Only one of these two dog BRCA2 variants, c.4314A>G—p.(Lys1435Arg), was also reported by ClinVar and classified in humans as of uncertain significance. We found this mutation alone in 18 out of 22 dogs (81%). Another dog BRCA2 variant, found in exon 11, c.4175G>A—p.(Arg2022Lys), was reported in humans and classified by ClinVar as Uncertain Significance. This variant was found on the mammary gland of a male dog, described as intraductal carcinoma, located in one of the BRCA repeats critical for binding to RAD51 [25,26]. Further analyses are needed to evaluate possible correlations.

Finally, it is worth considering the c.10204_10206insAAA variation that is positioned in the nuclear localization signal 2 (NLS2), which plays a key role in the mislocation of BRCA2 in humans [27]. Recent studies suggest that c.10204_10206insAAA may be associated with significantly higher mammary tumor morbidity rates in dogs [13,14]. We found this insertion in 18 out of 22 dogs (81%) in exon 27 of BRCA2.

### 5. Conclusions

Canine and human tumors share numerous traits: genetic and histopathological characteristics, biological behavior, molecular targets and response to standard antineoplastic treatments. For this reason, dogs offer an extraordinary opportunity to improve our knowledge of tumor biology in both species. The AmpliSeq NGS assay developed in this study for dog BRCA1 and BRCA2 will help to generate reproducible and robust results. The data obtained here are part of a database of variants that will be further increased by new findings from pathologic and healthy tissues of both historical and new samples. All these data will certainly lead to a better understanding of the disease and a more careful management of health care in both humans and dogs.



# SUPPLEMENTAL FILE 1: life-12-00851.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: PMC9225004

[File available at: automated_output/BRCA2/20251125_175029/pmc_fulltext/35743882_supplements/PMC9225004]

