# MAIN TEXT

## Dipeptidyl peptidase 9 triggers BRCA2 degradation and promotes DNA damage repair

### Abstract

AbstractN‐terminal sequences are important sites for post‐translational modifications that alter protein localization, activity, and stability. Dipeptidyl peptidase 9 (DPP9) is a serine aminopeptidase with the rare ability to cleave off N‐terminal dipeptides with imino acid proline in the second position. Here, we identify the tumor‐suppressor BRCA2 as a DPP9 substrate and show this interaction to be induced by DNA damage. We present crystallographic structures documenting intracrystalline enzymatic activity of DPP9, with the N‐terminal Met1‐Pro2 of a BRCA21‐40 peptide captured in its active site. Intriguingly, DPP9‐depleted cells are hypersensitive to genotoxic agents and are impaired in the repair of DNA double‐strand breaks by homologous recombination. Mechanistically, DPP9 targets BRCA2 for degradation and promotes the formation of RAD51 foci, the downstream function of BRCA2. N‐terminal truncation mutants of BRCA2 that mimic a DPP9 product phenocopy reduced BRCA2 stability and rescue RAD51 foci formation in DPP9‐deficient cells. Taken together, we present DPP9 as a regulator of BRCA2 stability and propose that by fine‐tuning the cellular concentrations of BRCA2, DPP9 alters the BRCA2 interactome, providing a possible explanation for DPP9's role in cancer.

### Introduction

N‐termini sequences are important sites for post‐translational modifications that alter protein localization, activity, and also stability via the N‐degron pathway that targets proteins with a destabilizing amino acid in the first N‐terminal position for proteasomal degradation (Varshavsky, 2019). Regulated degradation of cellular proteins is critical for the removal of damaged or mislocalized proteins, to establish optimal protein stoichiometry and to allow for changes in the cellular proteome in response to different stimuli. Among others, the N‐degron pathway regulates apoptosis (Ditzel et al, 2003; Piatkov et al, 2012; Eldeeb & Fahlman, 2014; Wang et al, 2017; Weaver et al, 2017), pyroptosis (Chui et al, 2019; Xu et al, 2019), glucose homeostasis (Chen et al, 2017), mitochondrial import (Finger et al, 2020), autophagy (Cha‐Molstad et al, 2017), G‐protein signaling (Lee et al, 2005; Park et al, 2015), and B‐cell signaling (Justa‐Schuch et al, 2016). The initiator methionine acts as an N‐Degron only in rare cases (Kim et al, 2014). More frequently, N‐degrons are formed following post‐translational modifications of the substrate such as N‐terminal acetylation (Hwang et al, 2010; Shemorry et al, 2013), N‐arginylation (Wang et al, 2009; Park et al, 2020), or limited proteolysis (Rao et al, 2001; Piatkov et al, 2012, 2014; Brower et al, 2013; Eldeeb & Fahlman, 2014; Weaver et al, 2017; Nguyen et al, 2019). Initial characterization of the N‐degron pathway had shown a highly stabilizing effect of the imino acid proline, both in the first or second N‐terminal position (Bachmair et al, 1986). N‐terminal prolines cannot be arginylated and are only rarely acetylated (Arnesen et al, 2009; Helbig et al, 2010). In yeast, however, gluconeogenic enzymes with N‐terminal prolines are rapidly degraded upon transition to glucose‐rich medium. Ubiquitination of these otherwise highly stable enzymes is carried out by multi‐subunit E3 ubiquitin ligases called the GID/CTLH complexes (Pro/N‐Recognins). These multi‐subunit E3 complexes associate in response to extracellular stimuli (Santt et al, 2008; Chen et al, 2017; Dong et al, 2018; Menssen et al, 2018; Melnykov et al, 2019; Qiao et al, 2020).

Dipeptidyl peptidase DPP9 and its homolog DPP8 are two intracellular serine aminopeptidases with the rare ability to cleave postproline (Xaa‐Pro/Ala↓Zaa) (Geiss‐Friedlander et al, 2009; Zhang et al, 2015). DPP9 is more abundant than DPP8 and its depletion leads to a large reduction in the capacity of cells to process proline‐containing peptides (Geiss‐Friedlander et al, 2009). Two isoforms of DPP9 are expressed in cells, DPP9‐S that localizes to the cytoplasm (Ajami et al, 2004) and DPP9‐L that contains an N‐terminal nuclear localization signal targeting the protease to the nucleus (Justa‐Schuch et al, 2014). DPP9 plays a role in neonatal survival (Gall et al, 2013) and in the immune response (Geiss‐Friedlander et al, 2009; Justa‐Schuch et al, 2016; Okondo et al, 2017, 2018; Johnson et al, 2018; de Vasconcelos et al, 2019). Deregulation of DPP9 is connected with tumorigenicity (Spagnuolo et al, 2013; Smebye et al, 2017; Tang et al, 2017; Saso et al, 2020), albeit the underlying mechanisms are poorly understood. Recently, we showed that DPP9 acts upstream to the N‐degron pathway by processing the precursor of the mitochondria protein AK2 to prevent its accumulation in the cytosol (Finger et al, 2020). Another protein that is targeted by DPP9 to the N‐degron pathway is the tyrosine kinase Syk, which is central for B‐cell receptor‐mediated signaling. The interaction between DPP9 and Syk requires FLNA that acts as a scaffold linking DPP9 to Syk (Justa‐Schuch et al, 2016). Here we asked whether additional FLNA interaction partners are targeted by DPP9 for proteasome‐degradation. Thus, we inspected the N‐terminus of known FLNA‐binding proteins for N‐terminal prolines, which are predicted to localize to disordered regions. Analysis of 90 different proteins known to interact with FLNA (Savoy & Ghosh, 2013) highlighted the human breast cancer‐associated protein 2 (BRCA2) (Yuan & Shen, 2001; Mondal et al, 2012; Yue et al, 2012). The N‐terminal sequence of BRCA2 is intrinsically disordered (Le et al, 2020; Sidhu et al, 2020) and includes a classical DPP9 cleavage site. The alignment of BRCA2 protein sequences from 108 placental mammals reveals high conservation of the Met1‐Pro2‐Ile3/Val3 sequence in the N‐terminus of BRCA2 (Fig 1A).

BRCA2 is a 384 kDa protein that is crucial for the repair of DNA double‐strand breaks (DSBs) (Moynahan et al, 2001) by the high‐fidelity Homologous Recombination (HR) pathway (Chen et al, 2018; Scully et al, 2019). Briefly, DSB repair by HR starts with the generation of 3′ single‐strand DNA (ssDNA) overhangs in a process termed 5′ end resection. Next, protein filaments of RAD51 cover the ssDNA overhangs, and search for homologous sequences preferentially in the sister chromatid, that serve as a template for further repair by DNA polymerases (Jasin & Rothstein, 2013). BRCA2 is critical for HR by promoting the loading and assembly of RAD51 into protein filaments selectively on the ssDNA (Jensen et al, 2010; Liu et al, 2010; Thorslund et al, 2010). HR repair of DSBs occurs in S and G2 phases when sister chromatids are available for repair (Hustedt & Durocher, 2017).

Intriguingly, DNA damage triggers the proteasomal degradation of BRCA2 (Schoenfeld et al, 2004; Liu et al, 2017), a process that is antagonized by the ubiquitin‐specific protease USP21 (Liu et al, 2017). Notably, elevated expression of USP21, which leads to stabilization of BRCA2, is observed in hepatocellular carcinoma and inversely correlates with patient survival suggesting that regulated degradation of BRCA2 is critical for DNA repair (Liu et al, 2017). Given the presence of an evolutionary conserved N‐terminal sequence Met1‐Pro2‐Ile3/Val3 in BRCA2 and the rate‐limiting role of DPP9 in cleaving Pro‐containing peptide bonds (Geiss‐Friedlander et al, 2009), we asked whether DPP9 and the N‐terminal Met1‐Pro2‐Ile3 sequence determine the half‐life of BRCA2.

Here we show that BRCA2 stability is reduced by the removal of the N‐terminal residues Met1Pro2 and increased in cells depleted of DPP9 activity. Crystal structures combined with mass spectrometry analysis demonstrate that DPP9 hydrolyses the peptide bond between Pro2 and Ile3 in an N‐terminal peptide of BRCA2. In cells, the interaction between DPP9 and BRCA2 is observed mostly in the cytosol and is stimulated by DNA damage. Depletion of DPP9 leads to fewer RAD51 foci formation, a hypersensitivity of cells to genotoxic agents, and defects in HR.

### Results

As a starting point, we asked whether DPP9 and BRCA2 are in close proximity in cells, using proximity ligation assays (PLAs), which are especially well‐suited for the detection of dynamic and transient interactions, such as those between an enzyme and its substrate. Initially, only few DPP9‐BRCA2 PLA signals were detected in HeLa cells. However, exposure to Mitomycin C (MMC), a chemotherapeutic agent that forms inter‐strand DNA crosslinks that are converted into secondary DSBs, resulted in a clear increase in the number of DPP9‐BRCA2 PLA events (Figs 1B and C, and EV1A; Appendix Fig S1A and B). Although DPP9 and BRCA2 localize to both the nucleus and the cytoplasm, most of the MMC‐induced BRCA2‐DPP9 PLA signals were located in the cytosol (Fig 1B and C). Given its interactions with both DPP9 (Justa‐Schuch et al, 2016) and BRCA2 (Yuan & Shen, 2001; Mondal et al, 2012; Yue et al, 2012), cells were silenced for FLNA to test whether it is relevant for the observed DPP9‐BRCA2 proximity. Cell silenced for FLNA did not show an increase in the number of BRCA2‐DPP9 interactions in response to MMC, implying that the DNA damage triggered proximity of DPP9 and BRCA2 requires FLNA (Figs 1B and C, and EV1A; Appendix Fig S1A and B).

A
Sequence alignment of BRCA2 homologs from 108 placental mammals (Logo plot) shows conservation of the cleavage site for DPP9 at the BRCA2 N‐terminus. The cartoon depicts the full‐length BRCA2 and its conserved domains with the corresponding interaction partners.B
Quantification of PLAs between BRCA2 and DPP9 in control HeLa WT cells treated with nontargeting siRNA (siNT) or silenced with the indicated oligos. 300 ng/ml MMC was added for 24 h. Each dot represents the number of PLA events in a single cell, from two to seven biological replicates (siNT‐MMC (n = 6), siNT+MMC (n = 7), siDPP9‐MMC (n = 4), siDPP9+MMC (n = 5), siBRCA2‐MMC (n = 2), siBRCA2+MMC (n = 3), siFLNA‐MMC (n = 5), siFLNA+MMC (n = 6), NgtCntrl‐MMC (n = 3), NgtCntrl+MMC (n = 6)). The number of foci are shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (*P ≤ 0.05, ****P ≤ 0.0001).C
Representative PLA images showing close proximity between endogenous DPP9 and endogenous BRCA2 in HeLa WT cells. Exposure of cells to MMC triggers more PLA events (white). Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_2889071 (these images, along with the corresponding controls can be found in Appendix Fig S1A and B).D
Quantification of PLAs between BRCA2 and DPP9 in asynchronous HeLa WT cells or in cells in S‐Phase. Each dot represents the number of PLA events in a single cell, from three biological replicates. Cells were synchronized with a double thymidine block and released for 3 h before the addition of NCS (250 ng / mL for 30 min). Cells were sampled prior to NCS treatment, after NCS treatment, and with 1.5 and 3 h of recovery after NCS treatment. The number of foci is shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (*P ≤ 0.05, **P ≤ 0.01, ****P ≤ 0.0001).E
Representative PLA images showing close proximity between endogenous DPP9 and endogenous BRCA2 in HeLa cells. Exposure of cells to NCS triggers more PLA events (white), which increase in time. Cells in S‐Phase show a greater fold increase in the number of PLA events in comparison with asynchronous cells. Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Antibodies as in (C).F
Co‐immunoprecipitation assays showing binding of BRCA2 and DPP9‐SWT. HEK293 DPP9 KO+DPP9WT cells, were treated with 1 μg/ml Dox (24 h) to induce the expression of DPP9‐FLAG. DNA damage was induced with 300 nM MMC treatment for 24 h. Control cells do not express DPP9 (− Dox). Bound proteins were eluted with a FLAG peptide and analyzed by western blotting (anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_731947, anti‐FLAG RRID:AB_262044).

Sequence alignment of BRCA2 homologs from 108 placental mammals (Logo plot) shows conservation of the cleavage site for DPP9 at the BRCA2 N‐terminus. The cartoon depicts the full‐length BRCA2 and its conserved domains with the corresponding interaction partners.

Quantification of PLAs between BRCA2 and DPP9 in control HeLa WT cells treated with nontargeting siRNA (siNT) or silenced with the indicated oligos. 300 ng/ml MMC was added for 24 h. Each dot represents the number of PLA events in a single cell, from two to seven biological replicates (siNT‐MMC (n = 6), siNT+MMC (n = 7), siDPP9‐MMC (n = 4), siDPP9+MMC (n = 5), siBRCA2‐MMC (n = 2), siBRCA2+MMC (n = 3), siFLNA‐MMC (n = 5), siFLNA+MMC (n = 6), NgtCntrl‐MMC (n = 3), NgtCntrl+MMC (n = 6)). The number of foci are shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (*P ≤ 0.05, ****P ≤ 0.0001).

Representative PLA images showing close proximity between endogenous DPP9 and endogenous BRCA2 in HeLa WT cells. Exposure of cells to MMC triggers more PLA events (white). Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_2889071 (these images, along with the corresponding controls can be found in Appendix Fig S1A and B).

Quantification of PLAs between BRCA2 and DPP9 in asynchronous HeLa WT cells or in cells in S‐Phase. Each dot represents the number of PLA events in a single cell, from three biological replicates. Cells were synchronized with a double thymidine block and released for 3 h before the addition of NCS (250 ng / mL for 30 min). Cells were sampled prior to NCS treatment, after NCS treatment, and with 1.5 and 3 h of recovery after NCS treatment. The number of foci is shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (*P ≤ 0.05, **P ≤ 0.01, ****P ≤ 0.0001).

Representative PLA images showing close proximity between endogenous DPP9 and endogenous BRCA2 in HeLa cells. Exposure of cells to NCS triggers more PLA events (white), which increase in time. Cells in S‐Phase show a greater fold increase in the number of PLA events in comparison with asynchronous cells. Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Antibodies as in (C).

Co‐immunoprecipitation assays showing binding of BRCA2 and DPP9‐SWT. HEK293 DPP9 KO+DPP9WT cells, were treated with 1 μg/ml Dox (24 h) to induce the expression of DPP9‐FLAG. DNA damage was induced with 300 nM MMC treatment for 24 h. Control cells do not express DPP9 (− Dox). Bound proteins were eluted with a FLAG peptide and analyzed by western blotting (anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_731947, anti‐FLAG RRID:AB_262044).

Source data are available online for this figure.

A
Summary of PLAs between BRCA2 and DPP9 in control HeLa WT cells treated with nontargeting siRNA (siNT) or silenced with the indicated oligos (Fig 1B). Each dot represents the mean number of PLA events in a single repetition, from two to seven biological replicates. More than 100 cells were quantified per condition in each experiment. Data from the summary of all PLA events per cell were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (***P ≤ 0.001, ****P ≤ 0.0001).B
Summary of PLAs between BRCA2 and DPP9 in asynchronous HeLa WT cells and HeLa WT cells synchronized to S‐phase. DNA damage was induced by NCS (250 ng/ml for 30 min). Each dot represents the mean number of PLA events in a single repetition. More than 100 cells were quantified per condition in each experiment, from three biological replicates. Data from the summary of all PLA events per cell were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).C
Summary of PLAs between BRCA2 and DPP9 in HeLa WT cells treated with 1G244 (Fig 2C). DNA damage was induced by MMC (300 ng/ml for 24 h). Each dot represents the mean number of PLA events in a single repetition, from three biological replicates. More than 100 cells were quantified per condition in each experiment. Data from the summary of all PLA events per cell were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

Summary of PLAs between BRCA2 and DPP9 in control HeLa WT cells treated with nontargeting siRNA (siNT) or silenced with the indicated oligos (Fig 1B). Each dot represents the mean number of PLA events in a single repetition, from two to seven biological replicates. More than 100 cells were quantified per condition in each experiment. Data from the summary of all PLA events per cell were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (***P ≤ 0.001, ****P ≤ 0.0001).

Summary of PLAs between BRCA2 and DPP9 in asynchronous HeLa WT cells and HeLa WT cells synchronized to S‐phase. DNA damage was induced by NCS (250 ng/ml for 30 min). Each dot represents the mean number of PLA events in a single repetition. More than 100 cells were quantified per condition in each experiment, from three biological replicates. Data from the summary of all PLA events per cell were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

Summary of PLAs between BRCA2 and DPP9 in HeLa WT cells treated with 1G244 (Fig 2C). DNA damage was induced by MMC (300 ng/ml for 24 h). Each dot represents the mean number of PLA events in a single repetition, from three biological replicates. More than 100 cells were quantified per condition in each experiment. Data from the summary of all PLA events per cell were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

To extend these observations, cells were treated with another DSBs causing agent, the radiomimetic drug Neocarzinostatin (NCS). Similarly, NCS‐treated cells presented more DPP9‐BRCA2 PLA signals compared with mock‐treated cells, with most of these events in the cytosol (Figs 1D and E, and EV1B). The fold increase of DPP9‐BRCA2 PLA events in response to NCS was even higher in cells that were first arrested by double thymidine blocks and treated with NCS following release from the second block, validating a close proximity between DPP9 and BRCA2 in S‐phase (Figs 1D and E, and EV1B).

Additionally, we performed co‐immunoprecipitation against DPP9 and tested for the presence of BRCA2. Knockouts of DPP9 were established in HEK Flp‐In™ T‐REx™‐293 cells (HEK293 DPP9 KO) using CRISPR single guide RNAs. These were then used to establish a stable cell line with a Doxycycline (Dox) inducible expression of the cytoplasmic DPP9 isoform (FLAG‐ DPP9‐SWT), (cells characterized in Appendix Fig S2A–E). To enhance the interaction between DPP9 and BRCA2, DNA damage was induced by the addition of MMC. BRCA2 co‐purified with FLAG‐DPP9 (Fig 1F). Collectively, these data strongly suggest that DPP9 and BRCA2 interact in cells in response to DNA damage.

To test whether DPP9 binds to the N‐terminus of BRCA2, a 40 amino acid peptide was selected. This peptide includes the binding site of PALB2, a protein that ensures the correct intra‐nuclear localization of BRCA2. A shorter BRCA221‐39 peptide was previously co‐crystallized in a complex with PALB2 (Xia et al, 2006; Oliver et al, 2009). Surface Plasmon Resonance (SPR) assays with the synthetic BRCA21‐40 peptide confirmed direct binding to immobilized recombinant DPP9 (Fig 2A). The SPR response curves obtained for DPP9 and BRCA21‐40 were not at perfect equilibrium, whereas the binding behavior of DPP9 to a shorter peptide lacking the Met‐Pro (BRCA23‐40) was less complex, suggesting that the curves for DPP9‐BRCA21‐40 reflect not only the binding of DPP9 and BRCA21‐40, but possibly also on‐chip processing of the BRCA21‐40 peptide by DPP9, and the release of the product (Fig 2A; Appendix Fig S3A–D). Additionally, we cloned and purified a recombinant protein containing the first 40 amino acids of BRCA2 followed by a C‐terminal HA tag, and examined whether DPP9 and BRCA2 show characteristics of an enzyme‐substrate interaction. Previously, we had shown that the binding of the competitive inhibitors 1G244 and SLRFLYEG (Wu et al, 2009; Pilla et al, 2013) leads to a rearrangement of the DPP8/9 active sites resulting in a closed conformation (Ross et al, 2018). Pull‐down assays with immobilized BRCA21‐40HA showed reduced binding to DPP9 in the presence of either 1G244 or SLRFLYEG (Fig 2B). Consistently, cells treated with 1G244 displayed fewer DPP9‐BRCA2 PLA events in response to MMC, compared with mock‐treated cells (Figs 2C and D, and EV1C; Appendix Fig S4A). Thus, both PLA in cells and pull‐down assays with purified recombinant components show the reduced association of BRCA2 and DPP9 in the presence of competitive inhibitors that occupy the active site of DPP9, suggesting that BRCA2 acts as a ligand that competes with these inhibitors for binding to DPP9. Direct processing of the BRCA21‐40 peptide by purified DPP9 was verified by mass spectrometry analysis, revealing the hydrolysis of the peptide bond between Pro2 and Ile3 (Met1Pro2↓Ile3) in the BRCA21‐40 peptide. The resulting BRCA23‐40 product presents a neo N‐terminus with isoleucine in the first position (Fig 2E). No cleavage was observed in control reactions that included the competitive inhibitor SLRFLYEG.

A
Surface Plasmon Resonance (SPR) data showing a direct interaction of purified DPP9 with a BRCA21‐40 peptide and a truncated BRCA23‐40 peptide, which lacks the N‐terminal dipeptide Met‐Pro. A serial dilution of BRCA2‐derived peptides was injected over a surface covered with DPP9. Equilibrium binding isotherms obtained for interactions measured between DPP9 and BRCA21‐40 (black line) and BRCA23‐40 (red line). Data were fitted to a sigmoidal dose–response curve fit. Mean ± SEM of technical triplicates of a representative experiment out of 3.B
Pull‐down assay showing direct binding of purified recombinant DPP9 to a BRCA2 N‐terminal1‐40 fragment immobilized on HA beads. The DPP9 inhibitors 1G244 or SLRFLYEG compete with BRCA21‐40HA for interaction with DPP9. Representative data of three technical replicates are shown. Anti‐HA: RRID:AB_2565334, anti‐DPP9: RRID:AB_2889071.C
Quantification of PLAs showing fewer MMC‐induced DPP9‐BRCA2 PLA events in HeLa cells treated with 10 μM 1G244. Each dot represents the number of PLA events in a single cell, from three biological replicates. The number of foci is shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (***P ≤ 0.001, ****P ≤ 0.0001).D
Representative images of DPP9‐BRCA2 PLA in the presence of 1G244—a competitive inhibitor of DPP9. Control cells were mock treated with DMSO. Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_2889071 (these images, along with the corresponding controls can be found in Appendix Fig S1C).E
In vitro processing of BRCA21‐40 synthetic peptide by purified recombinant DPP9. Samples were analyzed by high‐resolution liquid chromatography/tandem mass spectrometry, in quadruplicate. The panels show extracted MS1 ion chromatograms for both substrate BRCA21‐40 peptide (MPIGSKERPT…) (labeled S, [M+5H]5+
m/z 917.8637; retention time 17.7 min) and product BRCA23‐40 peptide (IGSKERPT…) (labeled P, [M+5H]5+
m/z 872.2451; retention time 17.6 min). The identity of the product peak was established both by accurate mass measurement to within 5 ppm and by product ion spectra.F
Pull‐down assay showing a competition between DPP9 and PALB2 for binding to the BRCA2 N‐terminus. While each protein can bind to the BRCA2 N‐terminal peptide, in the presence of DPP9, the PALB2‐BRCA2 binding is negatively affected. Representative data of three technical replicates are shown. Anti‐PALB2: RRID:AB_890607, anti‐HA: RRID:AB_2565334, anti‐DPP9: RRID:AB_2889071.G
Peptide sequences used in the DPP9 competition assays shown in (H).H
DPP9 activity assays showing different competitive effects of BRCA2 peptides (G), hydrolysis of GP‐AMC is used as read out. Shown is % of inhibition in GP‐AMC processing normalized to the control reactions (DPP9 without competing peptides). Syk1‐31 was used as a positive control. n = 3 independent experiments. Data were analyzed by a one‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (***P ≤ 0.001, ****P ≤ 0.0001).

Surface Plasmon Resonance (SPR) data showing a direct interaction of purified DPP9 with a BRCA21‐40 peptide and a truncated BRCA23‐40 peptide, which lacks the N‐terminal dipeptide Met‐Pro. A serial dilution of BRCA2‐derived peptides was injected over a surface covered with DPP9. Equilibrium binding isotherms obtained for interactions measured between DPP9 and BRCA21‐40 (black line) and BRCA23‐40 (red line). Data were fitted to a sigmoidal dose–response curve fit. Mean ± SEM of technical triplicates of a representative experiment out of 3.

Pull‐down assay showing direct binding of purified recombinant DPP9 to a BRCA2 N‐terminal1‐40 fragment immobilized on HA beads. The DPP9 inhibitors 1G244 or SLRFLYEG compete with BRCA21‐40HA for interaction with DPP9. Representative data of three technical replicates are shown. Anti‐HA: RRID:AB_2565334, anti‐DPP9: RRID:AB_2889071.

Quantification of PLAs showing fewer MMC‐induced DPP9‐BRCA2 PLA events in HeLa cells treated with 10 μM 1G244. Each dot represents the number of PLA events in a single cell, from three biological replicates. The number of foci is shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (***P ≤ 0.001, ****P ≤ 0.0001).

Representative images of DPP9‐BRCA2 PLA in the presence of 1G244—a competitive inhibitor of DPP9. Control cells were mock treated with DMSO. Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_2889071 (these images, along with the corresponding controls can be found in Appendix Fig S1C).

In vitro processing of BRCA21‐40 synthetic peptide by purified recombinant DPP9. Samples were analyzed by high‐resolution liquid chromatography/tandem mass spectrometry, in quadruplicate. The panels show extracted MS1 ion chromatograms for both substrate BRCA21‐40 peptide (MPIGSKERPT…) (labeled S, [M+5H]5+
m/z 917.8637; retention time 17.7 min) and product BRCA23‐40 peptide (IGSKERPT…) (labeled P, [M+5H]5+
m/z 872.2451; retention time 17.6 min). The identity of the product peak was established both by accurate mass measurement to within 5 ppm and by product ion spectra.

Pull‐down assay showing a competition between DPP9 and PALB2 for binding to the BRCA2 N‐terminus. While each protein can bind to the BRCA2 N‐terminal peptide, in the presence of DPP9, the PALB2‐BRCA2 binding is negatively affected. Representative data of three technical replicates are shown. Anti‐PALB2: RRID:AB_890607, anti‐HA: RRID:AB_2565334, anti‐DPP9: RRID:AB_2889071.

Peptide sequences used in the DPP9 competition assays shown in (H).

DPP9 activity assays showing different competitive effects of BRCA2 peptides (G), hydrolysis of GP‐AMC is used as read out. Shown is % of inhibition in GP‐AMC processing normalized to the control reactions (DPP9 without competing peptides). Syk1‐31 was used as a positive control. n = 3 independent experiments. Data were analyzed by a one‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (***P ≤ 0.001, ****P ≤ 0.0001).

Source data are available online for this figure.

Since amino acids 1–40 in the N‐terminus of BRCA2 include the PALB2 interaction site, we asked whether DPP9 and PALB2 compete for binding to the N‐terminus of BRCA2, or whether they bind independently of each other. Pull‐down assays show reduced PALB2 binding in the presence of DPP9 implying that these two proteins sterically hinder mutual binding to BRCA21‐40HA (Fig 2F). To test for residues in BRCA2 that affect cleavage by DPP9, competition assays were performed in which we measured the hydrolysis of the artificial substrate H‐Gly‐Pro‐7‐amino‐4‐methylcoumarin (GP‐AMC) in the presence of the BRCA21‐40 peptide, or variants of this peptide (Fig 2G and H). For control, assays were carried out with the Syk1‐31 peptide (Justa‐Schuch et al, 2016) which as expected inhibited GP‐AMC cleavage. Stronger inhibition was observed with the BRCA21‐40 peptide, while a BRCA2 peptide in which the Pro in position 2 was substituted with a Gly (BRCA21‐40P2G) was a poor inhibitor. Interestingly, a BRCA21‐20 peptide was a less efficient competitor in comparison with BRCA21‐40 suggesting that residues between 20–40 contribute to the interaction with DPP9. We looked more closely into selected residues, based on natural mutations in BRCA2 that were depicted in cBioPortal databases. These assays show that substitution of the conserved Glu in position 13 with a Lys reduces the competitive capacity of the BRCA21‐40 peptide. In line with the pull‐down assays, which suggest a competition between DPP9 and PALB2 for interaction with BRCA2, a stronger effect was observed for the substitutions of Trp31 and Phe32, which are important for interaction with PALB2 (Xia et al, 2006; Oliver et al, 2009).

To better portray the cleavage event, DPP9 crystals were soaked with the BRCA21‐40 peptide. DPP9 crystals exhibited positive electron density at the active site corresponding to the N‐terminal Met1Pro2 dipeptide with the proline residue occupying the S1 subsite (Fig 3), in a dipeptide conformation that was identical to that seen earlier in the DPP9‐SLRFLYEG complex (Ross et al, 2018). No trace of electron density of a peptide continued after the proline residue was present (Fig 3; Appendix Table S1). Since DPP8 and DPP9 share very similar substrate‐binding mechanisms in vitro, and DPP9 crystals are notorious for being less well‐ordered (Ross et al, 2018), we also tested for a possible interaction between DPP8 and BRCA2. Similarly, in DPP8, all 3 copies of the asymmetric unit exhibit clearly interpretable electron density for the dipeptide. The well‐ordered R‐segment, a hallmark of ligand binding, was well‐defined and ordered and supports the full occupation of the active sites. These structures, with the dipeptide captured in the active site, clearly document intracrystalline enzymatic activity of DPP8 and DPP9 resulting in the hydrolysis of the BRCA21‐40 N‐terminal peptide, in accord with the mass spectrometry data (Fig 2E), and release of the BRCA23‐40 peptide.

Shown is the N‐terminal dipeptide of BRCA2 bound to both DPP9 and DPP8. Omit map (Fo‐Fc; 3σ) of a BRCA21‐40 peptide soaked in DPP8 crystals (C2221) and in DPP9 crystals (P1211). The R‐Segment and SUMO‐binding arm (SUBA Pilla et al, 2012) are highlighted in orange and blue, respectively. The arrow marks the position where the BRCA2 peptide is cleaved. For simplification, shown is a monomer of DPP8. Both zoomed views are rotated 45° with respect to the monomer view to better display the ligand.

Source data are available online for this figure.

Based on our previous findings showing that DPP9 targets Syk and AK2 to the N‐degron pathway (Justa‐Schuch et al, 2016; Finger et al, 2020), we here asked whether DPP9 also influences the stability of BRCA2. Consistent with previous publications (Schoenfeld et al, 2004; Liu et al, 2017), exposure of cells to MMC induced an accelerated degradation of BRCA2, which was less pronounced in the presence of the proteasome inhibitor MG132 (Fig 4A and B). Importantly, cycloheximide (CHX) chase assays revealed a significantly lower turnover of BRCA2 in the HeLa DPP9 Knock‐Down cells (HeLa DPP9 KD; cells previously described in Justa‐Schuch et al (2016) and further characterized in Fig EV2A–C), compared with HeLa WT cells in response to MMC (Fig 4A and B). Similarly, we observed an increase in BRCA2 stability in HeLa WT cells transiently silenced for DPP9 (Fig 4A and B), and in the HEK 293 DPP9 KO cells (Fig 4C and D). In contrast to BRCA2, the stability of endogenous RAD51 was not affected by DPP9 (Fig 4A and B). To test whether the catalytic activity of DPP9 regulates BRCA2 stability, CHX chase assays were carried out in HEK 293 DPP9 KO cells in which we induced the expression of DPP9‐SWT. To test for the relevance of DPP9 activity, we generated a stable cell line HEK293 DPP9 KO+DPP9‐SS729A with an induced expression of the enzymatically inactive DPP9 (DPP9‐SS729A). Importantly, the re‐expression of DPP9‐SWT but not of DPP9‐SS729A restored the MMC‐dependent degradation of BRCA2 (Fig 4C and D; Appendix Fig S2A and B).

A, B
Representative western blots and accompanying graph from more than three biological replicates show that MMC (300 nM, 24 h) induces a rapid turnover of endogenous BRCA2 in HeLa WT cells, which is less pronounced in HeLa WT cells treated with the proteasome inhibitor MG132 (100 μM), in HeLa DPP9 KD cells and in cells transiently silenced for DPP9 (siDPP9). RAD51 stability is not altered in HeLa DPP9 KD cells. Vinculin is a loading control for BRCA2, Tubulin is a loading control for RAD51. Shown images originate from one representative cycloheximide (CHX) chase assay. The ratios of BRCA2 to Vinculin and RAD51 to Tubulin are defined as 100% at time 0 h. Mean ± SEM, data were analyzed by a paired two‐tailed t‐test. (*P ≤ 0.03). Anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_731947, anti‐Vinculin: RRID:AB_477629, anti‐Tubulin: RRID:AB_628412, anti‐RAD51: RRID:AB_1142428.C, D
Representative western blots and graph summarizing results of more than three biological replicates. CHX chase assays show that the MMC‐induced degradation of BRCA2 is less pronounced in HEK293 DPP9 KO cells and in HEK293 DPP9 KO+DPP9‐SS729A cells overexpressing an inactive DPP9‐S mutant. Expression of DPP9 was induced (+ Dox, 1 μg/ml) simultaneously with MMC (300 nM), 24 h. Cells that were overexpressing the active variant (DPP9‐SWT) show similar levels of MMC‐induced BRCA2 degradation. RAD51 stability is not altered by MMC and is similar in all cell lines. Vinculin is a loading control for BRCA2, Tubulin is a loading control for RAD51. Shown images originate from one representative cycloheximide (CHX) chase assay. The ratios of BRCA2 to Vinculin are defined as 100% at time 0 h. Mean ± SEM, data were analyzed by a paired two‐tailed t‐test (*P ≤ 0.03, **P ≤ 0.002). Antibodies as described in (A‐B).E
Graphical presentation of the plasmid used for the production of the N‐terminal‐truncated mutant BRCA2ΔMP3‐1000 that was transfected in H‐I. The BRCA21‐1000 insert was cloned in a similar manner. Both constructs were tagged with a C‐terminal FLAG. The ubiquitin moiety is removed in cells by endogenous ubiquitin isopeptidases.F, G
Representative western blots and graphs summarizing CHX assays from three biological replicates show that the BRCA2ΔMP3‐1000 truncation mutant is degraded at a higher rate compared with the untruncated BRCA21‐1000. Cells were co‐transfected with a GFP expressing plasmid, as transfection and loading control. Control cells were transfected with GFP only (U.T). Image originates from one representative experiment. BRCA2‐FLAG signals are related to the transfection and loading control GFP. BRCA2 levels in relation to GFP were defined as 100% at time 0 h. Mean ± SEM, data were analyzed by a paired two‐tailed t‐test (*P ≤ 0.03). Anti‐FLAG: RRID:AB_262044, anti‐GFP: RRID:AB_641123.

Representative western blots and accompanying graph from more than three biological replicates show that MMC (300 nM, 24 h) induces a rapid turnover of endogenous BRCA2 in HeLa WT cells, which is less pronounced in HeLa WT cells treated with the proteasome inhibitor MG132 (100 μM), in HeLa DPP9 KD cells and in cells transiently silenced for DPP9 (siDPP9). RAD51 stability is not altered in HeLa DPP9 KD cells. Vinculin is a loading control for BRCA2, Tubulin is a loading control for RAD51. Shown images originate from one representative cycloheximide (CHX) chase assay. The ratios of BRCA2 to Vinculin and RAD51 to Tubulin are defined as 100% at time 0 h. Mean ± SEM, data were analyzed by a paired two‐tailed t‐test. (*P ≤ 0.03). Anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_731947, anti‐Vinculin: RRID:AB_477629, anti‐Tubulin: RRID:AB_628412, anti‐RAD51: RRID:AB_1142428.

Representative western blots and graph summarizing results of more than three biological replicates. CHX chase assays show that the MMC‐induced degradation of BRCA2 is less pronounced in HEK293 DPP9 KO cells and in HEK293 DPP9 KO+DPP9‐SS729A cells overexpressing an inactive DPP9‐S mutant. Expression of DPP9 was induced (+ Dox, 1 μg/ml) simultaneously with MMC (300 nM), 24 h. Cells that were overexpressing the active variant (DPP9‐SWT) show similar levels of MMC‐induced BRCA2 degradation. RAD51 stability is not altered by MMC and is similar in all cell lines. Vinculin is a loading control for BRCA2, Tubulin is a loading control for RAD51. Shown images originate from one representative cycloheximide (CHX) chase assay. The ratios of BRCA2 to Vinculin are defined as 100% at time 0 h. Mean ± SEM, data were analyzed by a paired two‐tailed t‐test (*P ≤ 0.03, **P ≤ 0.002). Antibodies as described in (A‐B).

Graphical presentation of the plasmid used for the production of the N‐terminal‐truncated mutant BRCA2ΔMP3‐1000 that was transfected in H‐I. The BRCA21‐1000 insert was cloned in a similar manner. Both constructs were tagged with a C‐terminal FLAG. The ubiquitin moiety is removed in cells by endogenous ubiquitin isopeptidases.

Representative western blots and graphs summarizing CHX assays from three biological replicates show that the BRCA2ΔMP3‐1000 truncation mutant is degraded at a higher rate compared with the untruncated BRCA21‐1000. Cells were co‐transfected with a GFP expressing plasmid, as transfection and loading control. Control cells were transfected with GFP only (U.T). Image originates from one representative experiment. BRCA2‐FLAG signals are related to the transfection and loading control GFP. BRCA2 levels in relation to GFP were defined as 100% at time 0 h. Mean ± SEM, data were analyzed by a paired two‐tailed t‐test (*P ≤ 0.03). Anti‐FLAG: RRID:AB_262044, anti‐GFP: RRID:AB_641123.

Source data are available online for this figure.

A
Representative histograms of HeLa WT and HeLa DPP9 KD cells investigating their relative DNA content via PI staining. Asynchronized cells and cells that were double thymidine‐treated were investigated. Shown are representatives of the asynchronized, double thymidine release, 3 h of recovery and 24 h of recovery samples.B
Cell cycle distribution of the HeLa WT and HeLa DPP9 KD populations of asynchronized cells and synchronized cells at 0 h, 3, 8, and 24 after double thymidine release. Data from three to six biological replicates (asynchronized (n = 3), synchronized HeLa WT (n = 6), synchronized HeLa DPP9 KD (n = 4)) were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. No significant differences could be detected between the two cell lines. Shown are mean ± SEM.C
Representative western blots showing a CHX chase of HeLa WT cells transiently silenced for DPP9, and of control cells treated with nontargeting siRNA (siNT). Cells were treated with MMC (300 nM, 24 h) prior to the addition of CHX. Anti‐BRCA1: RRID:AB_626761, anti‐DPP9: RRID:AB_731947, anti‐Vinculin: RRID:AB_477629.

Representative histograms of HeLa WT and HeLa DPP9 KD cells investigating their relative DNA content via PI staining. Asynchronized cells and cells that were double thymidine‐treated were investigated. Shown are representatives of the asynchronized, double thymidine release, 3 h of recovery and 24 h of recovery samples.

Cell cycle distribution of the HeLa WT and HeLa DPP9 KD populations of asynchronized cells and synchronized cells at 0 h, 3, 8, and 24 after double thymidine release. Data from three to six biological replicates (asynchronized (n = 3), synchronized HeLa WT (n = 6), synchronized HeLa DPP9 KD (n = 4)) were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. No significant differences could be detected between the two cell lines. Shown are mean ± SEM.

Representative western blots showing a CHX chase of HeLa WT cells transiently silenced for DPP9, and of control cells treated with nontargeting siRNA (siNT). Cells were treated with MMC (300 nM, 24 h) prior to the addition of CHX. Anti‐BRCA1: RRID:AB_626761, anti‐DPP9: RRID:AB_731947, anti‐Vinculin: RRID:AB_477629.

Source data are available online for this figure.

To test whether the N‐terminal Pro determines BRCA2 stability, we determined the turnover of BRCA21‐1000 revealing a half‐life greater than 8 h, similar to the endogenous BRCA2 in the absence of MMC (Fig 4E–G). Additionally, we cloned an N‐terminal BRCA2 truncation mutant (BRCA2ΔMP3‐1000) with isoleucine in the first position that mimics the product generated in vitro upon DPP9 cleavage (MP↓IGSK → IGSK). The half‐life of BRCA2ΔMP3‐1000 was 2–3 h, similar to the turnover rate of endogenous BRCA2 in the presence of MMC, and significantly lower than BRCA2 1‐1000 (Fig 4F and G). To generate the desired N‐terminus both constructs were cloned in frame with an N‐terminal ubiquitin tag, which is removed in cells by endogenous ubiquitin isopeptidases based on the well‐established ubiquitin‐fusion technique (Varshavsky, 2005). We have also observed that the steady‐state levels of BRCA21‐1000 constructs with a P2H or P2G mutation are reduced (Appendix Fig S4B). In summary, Pro2 plays a stabilizing role for BRCA2, removal of which leads to accelerated turnover of BRCA2, phenocopying the MMC‐induced degradation of BRCA2.

Previous studies have shown that high levels of wild‐type BRCA2 or the BRC4 repeat suppress DSB repair by homologous recombination and reduce RAD51 foci formation (Chen et al, 1999; Magwood et al, 2012; Abe & Branzei, 2014). Since DPP9 targets BRCA2 for degradation, we asked whether depletion of DPP9 and stabilization of BRCA2 alter the cellular response to DNA damage. First, we monitored the phosphorylation state of histone H2AX on serine 139 (γH2AX), which is an early and decisive step in the repair of DSBs, that is used as a marker for DSBs (Kuo & Yang, 2008). This analysis revealed significantly more γH2AX in HeLa DPP9 KD cells in response to MMC compared with the corresponding control cells (Fig EV3A). Likewise, γH2AX signals were higher in HeLa DPP9 KD cells following 30 min exposure to NCS (time 0 h) and remained higher also 3 h after removal of NCS (Fig EV3B and C), pointing to a delay in recovery of DSBs in HeLa DPP9 KD cells. At 6 h we observed that the γH2AX levels in the HeLa DPP9 KD cells were comparable to those of WT cells, suggesting that the DSBs have been resolved. To assess whether the enzymatic activity of DPP9 contributes to the elevated levels of γH2AX, we analyzed the response of the DPP9 gene knock‐in mouse embryonic fibroblasts (MEFs), which express enzymatically inactive DPP9 (gki MEF DPP9S729A cells were first described in Gall et al (2013)). Consistently, gki MEF DPP9S729A cells accumulated significantly more γH2AX in response to MMC and displayed a clear delay in the resolution of γH2AX that was formed in response to NCS (Figs 5A and B, and EV3D and E).

A
Representative immunofluorescence images of γH2AX signals (white, RRID:AB_309864) in gki MEF DPP9S729A, showing more γH2AX in gki MEF DPP9S729A cells following removal of Neocarzinostatin (NCS), quantification in (B). MEF DPP9S729A and control MEF cells were treated with 250 ng/mL NCS for 30 min and allowed to recover for the indicated time points. γH2AX signals at time 0, reflect 30 min of NCS and no recovery time. Nuclei are shown in blue (DAPI). Scale bar 10 μm.B
Quantification of γH2AX in gki MEF DPP9S729A and control MEF cells as described in (A). Signals from more than 1,700 cells were quantified per condition per experiment. Mean ± SEM from six biological replicates, each in technical duplicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (*P ≤ 0.05, **P ≤ 0.01).C–E
Dose‐dependent viability assays show a higher sensitivity of HeLa DPP9 KD cells (C and D) and DPP9 silenced cells (siDPP9) (E) to MMC (C) and Olaparib (D, E). BRCA2 was silenced for control. Mean ± SEM of three biological replicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (*P ≤ 0.05, **P ≤ 0.01, ***P ≤ 0.001, ****P ≤ 0.0001).F, G
Quantification of colonies formed after γ‐radiation of HeLa WT and DPP9 KD (F), and MEF WT and gki MEF DPP9S729A cells (G), showing the mean ± SEM of the survival fraction (SF) from three biological replicates. Data were analyzed by an unpaired two‐way ANOVA.

Representative immunofluorescence images of γH2AX signals (white, RRID:AB_309864) in gki MEF DPP9S729A, showing more γH2AX in gki MEF DPP9S729A cells following removal of Neocarzinostatin (NCS), quantification in (B). MEF DPP9S729A and control MEF cells were treated with 250 ng/mL NCS for 30 min and allowed to recover for the indicated time points. γH2AX signals at time 0, reflect 30 min of NCS and no recovery time. Nuclei are shown in blue (DAPI). Scale bar 10 μm.

Quantification of γH2AX in gki MEF DPP9S729A and control MEF cells as described in (A). Signals from more than 1,700 cells were quantified per condition per experiment. Mean ± SEM from six biological replicates, each in technical duplicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (*P ≤ 0.05, **P ≤ 0.01).

Dose‐dependent viability assays show a higher sensitivity of HeLa DPP9 KD cells (C and D) and DPP9 silenced cells (siDPP9) (E) to MMC (C) and Olaparib (D, E). BRCA2 was silenced for control. Mean ± SEM of three biological replicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (*P ≤ 0.05, **P ≤ 0.01, ***P ≤ 0.001, ****P ≤ 0.0001).

Quantification of colonies formed after γ‐radiation of HeLa WT and DPP9 KD (F), and MEF WT and gki MEF DPP9S729A cells (G), showing the mean ± SEM of the survival fraction (SF) from three biological replicates. Data were analyzed by an unpaired two‐way ANOVA.

A
Increased γH2AX signals in HeLa DPP9 KD cells following exposure to 300 nM MMC for 24 h. Tubulin was a loading control. Quantification of the γH2AX / Tubulin signals in HeLa DPP9 KD cells and HeLa WT cells from three biological replicates. The γH2AX / Tubulin ratio in HeLa DPP9 KD cells at 24 h MMC was defined as 100%. Mean ± SEM. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test (****P ≤ 0.0001). Anti‐γH2AX: RRID:AB_2118009, anti‐Tubulin: RRID:AB_628412.B
Representative immunofluorescence images showing more γH2AX (white) in HeLa DPP9 KD cells following removal of Neocarzinostatin (NCS). Nuclei are shown in blue (DAPI). Scale bar 10 μm. HeLa DPP9 KD cells and HeLa WT cells were treated with 250 ng/ml Neocarzinostatin (NCS) for 30 min and allowed to recover for the indicated time points. γH2AX signals of each cell type at time 0, reflect 30 min of NCS, and no recovery time. Anti‐γH2AX: RRID:AB_309864.C
Quantification of mean γH2AX signals from HeLa WT and HeLa DPP9 KD cells. More than 1,300 cells were quantified per condition per experiment. Mean ± SEM from four biological replicates, each in technical duplicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (**P ≤ 0.01, ****P ≤ 0.0001).D
Higher γH2AX signals in gki MEF DPP9S729A cells expressing enzymatically inactive DPP9 compared with MEF WT control cells following exposure to 300 nM MMC for 24 h. Tubulin was a loading control. Quantification of the γH2AX / Tubulin ratios in gki MEF DPP9S729A cells and MEF WT cells from three biological replicates. For normalization, the γH2AX / Tubulin ratio in gki MEF DPP9S729A cells at 24 h MMC was defined as 100%. Mean ± SEM. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test (****P ≤ 0.0001).E
Same as (C) for gki MEF DPP9S729A cells and MEF WT cells. Signals from more than 1,700 cells were quantified per condition per experiment. Mean ± SEM from six biological replicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (****P ≤ 0.0001).

Increased γH2AX signals in HeLa DPP9 KD cells following exposure to 300 nM MMC for 24 h. Tubulin was a loading control. Quantification of the γH2AX / Tubulin signals in HeLa DPP9 KD cells and HeLa WT cells from three biological replicates. The γH2AX / Tubulin ratio in HeLa DPP9 KD cells at 24 h MMC was defined as 100%. Mean ± SEM. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test (****P ≤ 0.0001). Anti‐γH2AX: RRID:AB_2118009, anti‐Tubulin: RRID:AB_628412.

Representative immunofluorescence images showing more γH2AX (white) in HeLa DPP9 KD cells following removal of Neocarzinostatin (NCS). Nuclei are shown in blue (DAPI). Scale bar 10 μm. HeLa DPP9 KD cells and HeLa WT cells were treated with 250 ng/ml Neocarzinostatin (NCS) for 30 min and allowed to recover for the indicated time points. γH2AX signals of each cell type at time 0, reflect 30 min of NCS, and no recovery time. Anti‐γH2AX: RRID:AB_309864.

Quantification of mean γH2AX signals from HeLa WT and HeLa DPP9 KD cells. More than 1,300 cells were quantified per condition per experiment. Mean ± SEM from four biological replicates, each in technical duplicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (**P ≤ 0.01, ****P ≤ 0.0001).

Higher γH2AX signals in gki MEF DPP9S729A cells expressing enzymatically inactive DPP9 compared with MEF WT control cells following exposure to 300 nM MMC for 24 h. Tubulin was a loading control. Quantification of the γH2AX / Tubulin ratios in gki MEF DPP9S729A cells and MEF WT cells from three biological replicates. For normalization, the γH2AX / Tubulin ratio in gki MEF DPP9S729A cells at 24 h MMC was defined as 100%. Mean ± SEM. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test (****P ≤ 0.0001).

Same as (C) for gki MEF DPP9S729A cells and MEF WT cells. Signals from more than 1,700 cells were quantified per condition per experiment. Mean ± SEM from six biological replicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (****P ≤ 0.0001).

Source data are available online for this figure.

We also tested the sensitivity of DPP9‐deprived cells to the genotoxic agents MMC and Olaparib, an inhibitor of poly(ADP‐ribose) polymerase (PARP). Since inhibition of PARP leads to the conversion of ssDNA breaks into DSBs, defects in HR‐mediated repair are associated with increased sensitivity to PARP inhibitors, and are applied for the treatment of germline BRCA1‐ and BRCA2‐mutated metastatic breast cancers (Menear et al, 2008; Rottenberg et al, 2008; Lord & Ashworth, 2016). We detected a hypersensitivity of HeLa DPP9 KD cells to MMC and Olaparib compared with the HeLa WT cells (Fig 5C and D). Similarly, DPP9‐silenced cells were more sensitive to Olaparib compared with control cells treated with nontargeting siRNA (Fig 5E), albeit not to the same extent as cells transiently silenced for BRCA2. Furthermore, HeLa DPP9 KD cells were slightly, but significantly more sensitive to ionizing radiation (γ‐radiation) compared with the corresponding control cells (Fig 5F). This effect was more pronounced in gki MEF DPP9S729A cells (Fig 5G), linking the hypersensitivity to ionizing radiation to the absence of DPP9 enzymatic activity. Thus, cells lacking DPP9 activity accumulate more unrepaired DNA damage (Fig 5A and B) and are hypersensitive to genotoxic stress caused by MMC, Olaparib, and IR (Fig 5C–G), phenocopying cells overexpressing BRCA2 or the BRC4 repeat (Chen et al, 1999; Magwood et al, 2012; Abe & Branzei, 2014).

To directly investigate whether DPP9 plays a role in HR, we applied the well‐established DR‐GFP reporter system (Pierce et al, 1999; Fig 6A and B) in the colon cancer cell line HCT116 stably expressing DR‐GFP (Kari et al, 2019). As expected, control cells silenced for BRCA2, RAD51, or FLNA were deficient in HR‐mediated repair of the GFP reporter constructs (Fig 6B; Moynahan et al, 2001; Stark et al, 2004; Yue et al, 2009). Importantly, depletion of DPP9 also resulted in a significant reduction in HR frequency, implying that DPP9 promotes the repair of DSBs by HR. Intriguingly, despite the in vitro structural similarities between DPP8 and DPP9 (Fig 3), depletion of DPP8 did not significantly reduce HR efficiency, suggesting that DPP8 and DPP9 are not biologically redundant for this pathway.

To better understand the observed effect of DPP9 in the repair of DNA, we investigated BRCA2 in DPP9‐depleted cells, specifically its colocalization with γH2AX and with PALB2. In line with the higher stability of BRCA2 (Fig 4), and the observed partially overlapping binding site of both DPP9 and PALB2 to BRCA2 (Fig 2F and H), DPP9‐silenced cells contained significantly more BRCA2‐PALB2 PLA events both in the absence and presence of MMC (Fig 6C and D; Appendix Fig S4C). However, despite the increased colocalization with PALB2, and the higher γH2AX levels, fewer BRCA2‐γH2AX PLA events were formed in HeLa DPP9 KD cells in response to MMC (Fig 6E and F; Appendix Fig S4D and E). Similarly, fewer RAD51 foci were detected in HeLa DPP9 KD cells that were gated for EdU positive cells, HeLa DPP9‐silenced cells, and gki MEF DPP9S729A cells expressing inactive DPP9, in response to MMC compared with the number of RAD51 foci observed in the corresponding control cells (Figs 6G and EV4A–E; Appendix Fig S5A–C). Similarly, less RAD51 accumulated to chromatin fractions following 24 h of exposure to MMC in HeLa DPP9 KD cells (Appendix Fig S5D). To test whether the expression of DPP9 promotes the MMC‐induced appearance of RAD51, we monitored RAD51 in the HEK293 DPP9 KO, in HEK293 DPP9 KO+DPP9‐SWTand HEK293 DPP9 KO+DPP9‐SS729A. The re‐expression of DPP9‐SWT but not DPP9‐SS729A resulted in a significantly greater number of RAD51 foci in response to MMC (Figs 7A and B, and EV4F). Since DPP9‐S does not localize to the nucleus, we asked whether DPP9‐L which contains a nuclear localization signal (Justa‐Schuch et al, 2014) can rescue the formation of RAD51 foci. To address this question, we generated a stable cell line HEK293 DPP9 KO+DPP9‐L (Appendix Fig S2E). Comparing the RAD51 foci formed in the parental cell line and cells overexpressing the DPP9‐S or DPP9‐L isoform, we observed that the overexpression of the DPP9‐S results in more RAD51 foci formation compared with the DPP9‐L (Fig EV4G). Taken together, these results show that DPP9 activity promotes the formation of MMC‐induced RAD51 foci and a prominent role for the cytosolic form of DPP9 in the formation of RAD51 foci in response to MMC‐induced DNA damage.

A
Representative immunofluorescence images showing that re‐expression of DPP9‐SWT leads to an increase in the number of RAD51 foci formed following exposure to MMC in HEK293 DPP9 KO+DPP9‐SWT cells. Expression of DPP9 was induced (+ Dox, 1 μg/ml) simultaneously with MMC (300 nM), 24 h. RAD51 foci are shown in white, and nuclei (DAPI) are shown in blue. Scale bar 10 μm. Anti‐RAD51: RRID:AB_1142428.B
Graph showing the number of RAD51 foci following induction of DPP9‐SWT expression, compared with uninduced HEK293 DPP9 KO+DPP9WT cells (‐Dox). Induction of HEK293 DPP9 KO+DPP9S729A for expression of DPP9‐SS729A did not result in more RAD51 foci. Each dot represents the number of RAD51 foci in a single cell, from three biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).C–F
Representative images (C, E) and summarizing graph (D, F) showing the number of RAD51 foci per nucleus in HeLa WT cells (C, D) or HeLa DPP9 KD cells (E, F). Where stated, cells were treated with control siRNA (siNT) or silenced for BRCA2, and transiently transfected with the BRCA21‐3418 or BRCA23‐3418 constructs. Both BRCA2 constructs can rescue the RAD51 foci formation phenotype to the control levels in HeLa WT cells (C, D). In HeLa DPP9 KD cells significantly more RAD51 foci were present in cells transfected with BRCA2ΔMP3‐3418 compared with BRCA2 silenced cells and cells expressing the untruncated BRCA21‐3418 (E, F). Each dot represents the number of RAD51 foci in a single cell, from three (D) or four (F) biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001). RAD51 foci are shown in white, and nuclei (DAPI) are shown in blue. Scale bar 10 μm. Anti‐RAD51: RRID:AB_1142428; anti‐DPP9: RRID:AB_2889071; anti‐BRCA2: RRID:AB_2259370.

Representative immunofluorescence images showing that re‐expression of DPP9‐SWT leads to an increase in the number of RAD51 foci formed following exposure to MMC in HEK293 DPP9 KO+DPP9‐SWT cells. Expression of DPP9 was induced (+ Dox, 1 μg/ml) simultaneously with MMC (300 nM), 24 h. RAD51 foci are shown in white, and nuclei (DAPI) are shown in blue. Scale bar 10 μm. Anti‐RAD51: RRID:AB_1142428.

Graph showing the number of RAD51 foci following induction of DPP9‐SWT expression, compared with uninduced HEK293 DPP9 KO+DPP9WT cells (‐Dox). Induction of HEK293 DPP9 KO+DPP9S729A for expression of DPP9‐SS729A did not result in more RAD51 foci. Each dot represents the number of RAD51 foci in a single cell, from three biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

Representative images (C, E) and summarizing graph (D, F) showing the number of RAD51 foci per nucleus in HeLa WT cells (C, D) or HeLa DPP9 KD cells (E, F). Where stated, cells were treated with control siRNA (siNT) or silenced for BRCA2, and transiently transfected with the BRCA21‐3418 or BRCA23‐3418 constructs. Both BRCA2 constructs can rescue the RAD51 foci formation phenotype to the control levels in HeLa WT cells (C, D). In HeLa DPP9 KD cells significantly more RAD51 foci were present in cells transfected with BRCA2ΔMP3‐3418 compared with BRCA2 silenced cells and cells expressing the untruncated BRCA21‐3418 (E, F). Each dot represents the number of RAD51 foci in a single cell, from three (D) or four (F) biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001). RAD51 foci are shown in white, and nuclei (DAPI) are shown in blue. Scale bar 10 μm. Anti‐RAD51: RRID:AB_1142428; anti‐DPP9: RRID:AB_2889071; anti‐BRCA2: RRID:AB_2259370.

A
Graphical presentation of the DR‐GFP reporter assay for HR. The construct includes two mutated GFP genes (SceGFP and iGFP) oriented as direct repeats. The SceGFP gene is mutated to contain the recognition site for the rare‐cutting endonuclease I‐SceI and a STOP codon, while the iGFP is a truncated form of GFP. Cells expressing the DR‐GFP are transfected with I‐SceI, which forms a DSB within the SceGFP. For HR‐mediated repair, the iGFP serves as a template for HR‐mediated repair of the DSB in GFP (Pierce et al, 1999).B
DPP9‐silenced cells are less efficient in the repair of DSBs by HR. HCT116 cells stably expressing the DR‐GFP reporter assay for HR efficiency were transiently transfected with the indicated siRNAs, and transfected with an I‐SceI‐expression vector. 48 h after transfection, the percentage of GFP‐positive cells was measured using flow cytometry analysis as an indication of HR efficiency. The graph shows the mean ± SEM from seven biological replicates. Data were analyzed by a paired two‐tailed t‐test (**P ≤ 0.01, ***P ≤ 0.001, ****P ≤ 0.0001).C
Representative PLA images of BRCA2‐PALB2 PLA experiments showing more PLA events (white) in HeLa cells silenced for DPP9 (siDPP9) with respect to cells treated with nontargeting siRNA (siNT). Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐PALB2: RRID:AB_890607, anti‐BRCA2: RRID:AB_2259370.D
Quantification of PLAs experiments showing more BRCA2‐PALB2 PLA events in cells silenced for DPP9 (siDPP9) in comparison with nontargeting controls (siNT). Each dot represents the number of PLA events in a single cell, from four biological replicates. The technical control samples (NgtCntrl) omitted the BRCA2 antibody. The number of foci is shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).E
Representative PLA images of γH2Ax‐BRCA2 PLA experiment showing a reduction in the number of PLA events (white) between γH2Ax and BRCA2 in HeLa DPP9 KD cells. Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐γH2Ax: RRID:AB_2118009, anti‐BRCA2: RRID:AB_2259370 (these images, along with the corresponding controls can be found in Appendix Fig S1D).F
Quantification of PLAs showing fewer MMC‐induced γH2Ax‐BRCA2 PLA events in HeLa DPP9 KD cells, in comparison with HeLa WT cells. Each dot represents the number of PLA events in a single cell, from three biological replicates. The technical control samples (NgtCntrl) omitted the γH2Ax antibody. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).G
Graph showing fewer RAD51 foci in DPP9‐silenced cells compared with control cells, following exposure to MMC. Each dot represents the number of RAD51 foci in a single cell, from two to six biological replicates: siNT‐MMC (n = 6), siNT+MMC (n = 6), siDPP9‐MMC (n = 5), siDPP9+MMC (n = 5), siBRCA2‐MMC (n = 3), siBRCA2+MMC (n = 3), siRAD51‐MMC (n = 2), siRAD51+MMC (n = 2). Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

Graphical presentation of the DR‐GFP reporter assay for HR. The construct includes two mutated GFP genes (SceGFP and iGFP) oriented as direct repeats. The SceGFP gene is mutated to contain the recognition site for the rare‐cutting endonuclease I‐SceI and a STOP codon, while the iGFP is a truncated form of GFP. Cells expressing the DR‐GFP are transfected with I‐SceI, which forms a DSB within the SceGFP. For HR‐mediated repair, the iGFP serves as a template for HR‐mediated repair of the DSB in GFP (Pierce et al, 1999).

DPP9‐silenced cells are less efficient in the repair of DSBs by HR. HCT116 cells stably expressing the DR‐GFP reporter assay for HR efficiency were transiently transfected with the indicated siRNAs, and transfected with an I‐SceI‐expression vector. 48 h after transfection, the percentage of GFP‐positive cells was measured using flow cytometry analysis as an indication of HR efficiency. The graph shows the mean ± SEM from seven biological replicates. Data were analyzed by a paired two‐tailed t‐test (**P ≤ 0.01, ***P ≤ 0.001, ****P ≤ 0.0001).

Representative PLA images of BRCA2‐PALB2 PLA experiments showing more PLA events (white) in HeLa cells silenced for DPP9 (siDPP9) with respect to cells treated with nontargeting siRNA (siNT). Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐PALB2: RRID:AB_890607, anti‐BRCA2: RRID:AB_2259370.

Quantification of PLAs experiments showing more BRCA2‐PALB2 PLA events in cells silenced for DPP9 (siDPP9) in comparison with nontargeting controls (siNT). Each dot represents the number of PLA events in a single cell, from four biological replicates. The technical control samples (NgtCntrl) omitted the BRCA2 antibody. The number of foci is shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

Representative PLA images of γH2Ax‐BRCA2 PLA experiment showing a reduction in the number of PLA events (white) between γH2Ax and BRCA2 in HeLa DPP9 KD cells. Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐γH2Ax: RRID:AB_2118009, anti‐BRCA2: RRID:AB_2259370 (these images, along with the corresponding controls can be found in Appendix Fig S1D).

Quantification of PLAs showing fewer MMC‐induced γH2Ax‐BRCA2 PLA events in HeLa DPP9 KD cells, in comparison with HeLa WT cells. Each dot represents the number of PLA events in a single cell, from three biological replicates. The technical control samples (NgtCntrl) omitted the γH2Ax antibody. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

Graph showing fewer RAD51 foci in DPP9‐silenced cells compared with control cells, following exposure to MMC. Each dot represents the number of RAD51 foci in a single cell, from two to six biological replicates: siNT‐MMC (n = 6), siNT+MMC (n = 6), siDPP9‐MMC (n = 5), siDPP9+MMC (n = 5), siBRCA2‐MMC (n = 3), siBRCA2+MMC (n = 3), siRAD51‐MMC (n = 2), siRAD51+MMC (n = 2). Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

A
A graph summarizing the mean number of RAD51 foci in HeLa WT cells treated with nontargeting siRNA (siNT) or silenced with the indicated oligos. Each dot represents the mean number of RAD51 foci in a single repetition, from two to six biological replicates. Statistical analysis on the data is shown in Fig 6G. Shown are mean ± SEM.B
A graph summarizing the number of RAD51 foci in HeLa WT and DPP9 KD cells treated with EdU and MMC. Shown is the quantification of RAD51 foci in EdU positive cells. HeLa DPP9 KD cells display fewer RAD51 foci in comparison with their WT counterparts. Each dot represents the number of RAD51 foci in a single cell, from three biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM.C
A graph summarizing the mean number of RAD51 foci in EdU positive HeLa WT and DPP9 KD cells. Each dot represents the mean number of RAD51 foci in a single repetition, from three biological replicates. Statistical analysis on the data is shown in Fig EV4B. Shown are mean ± SEM.D
A graph summarizing the number of RAD51 foci in MEF WT and gki MEF DPP9S729A cells treated with MMC. Shown is the quantification of RAD51 foci. Each dot represents the number of RAD51 foci in a single cell, from three biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM.E
A graph summarizing the mean number of RAD51 foci in MEF WT and gki MEF DPP9S729A cells upon MMC treatment. Each dot represents the mean number of RAD51 foci in a single repetition, from three biological replicates. Statistical analysis on the data is shown in Fig EV4D. Shown are mean ± SEM.F
A graph summarizing the mean number of RAD51 foci following induction of DPP9‐SWT expression, compared with uninduced HEK293 DPP9 KO+DPP9WT cells (‐Dox). Induction of HEK293 DPP9 KO+DPP9S729A for expression of DPP9‐SS729A did not result in more RAD51 foci. Each dot represents the mean number of RAD51 foci in a single repetition, from three biological replicates. Statistical analysis on the data is shown in Fig 7B. Shown are mean ± SEM.G
A graph summarizing the number of RAD51 foci following induction of DPP9‐SWT or DPP9‐LWT expression, compared with HEK293 DPP9WT cells. The number of RAD51 foci formed by the HEK293 DPP9 KO+DPP9‐SWT cells was similar to the HEK293 DPP9WT cells, compared with the HEK293 DPP9 KO+DPP9‐LWT cells. Each dot represents the mean number of RAD51 foci in a single repetition, from two biological replicates. Shown are mean ± SEM.H, I
Summary of RAD51 foci in HeLa WT and HeLa DPP9 KD cells as described in (Fig 7C and F). Each dot represents the mean number of RAD51 foci in a single repetition, from three or four biological replicates (HeLa WT (n = 3), HeLa DPP9 KD (n = 4)). Data from the summary of all RAD51 foci per nucleus upon MMC treatment, between biological replicates, were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001). Shown are mean ± SEM.

A graph summarizing the mean number of RAD51 foci in HeLa WT cells treated with nontargeting siRNA (siNT) or silenced with the indicated oligos. Each dot represents the mean number of RAD51 foci in a single repetition, from two to six biological replicates. Statistical analysis on the data is shown in Fig 6G. Shown are mean ± SEM.

A graph summarizing the number of RAD51 foci in HeLa WT and DPP9 KD cells treated with EdU and MMC. Shown is the quantification of RAD51 foci in EdU positive cells. HeLa DPP9 KD cells display fewer RAD51 foci in comparison with their WT counterparts. Each dot represents the number of RAD51 foci in a single cell, from three biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM.

A graph summarizing the mean number of RAD51 foci in EdU positive HeLa WT and DPP9 KD cells. Each dot represents the mean number of RAD51 foci in a single repetition, from three biological replicates. Statistical analysis on the data is shown in Fig EV4B. Shown are mean ± SEM.

A graph summarizing the number of RAD51 foci in MEF WT and gki MEF DPP9S729A cells treated with MMC. Shown is the quantification of RAD51 foci. Each dot represents the number of RAD51 foci in a single cell, from three biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM.

A graph summarizing the mean number of RAD51 foci in MEF WT and gki MEF DPP9S729A cells upon MMC treatment. Each dot represents the mean number of RAD51 foci in a single repetition, from three biological replicates. Statistical analysis on the data is shown in Fig EV4D. Shown are mean ± SEM.

A graph summarizing the mean number of RAD51 foci following induction of DPP9‐SWT expression, compared with uninduced HEK293 DPP9 KO+DPP9WT cells (‐Dox). Induction of HEK293 DPP9 KO+DPP9S729A for expression of DPP9‐SS729A did not result in more RAD51 foci. Each dot represents the mean number of RAD51 foci in a single repetition, from three biological replicates. Statistical analysis on the data is shown in Fig 7B. Shown are mean ± SEM.

A graph summarizing the number of RAD51 foci following induction of DPP9‐SWT or DPP9‐LWT expression, compared with HEK293 DPP9WT cells. The number of RAD51 foci formed by the HEK293 DPP9 KO+DPP9‐SWT cells was similar to the HEK293 DPP9WT cells, compared with the HEK293 DPP9 KO+DPP9‐LWT cells. Each dot represents the mean number of RAD51 foci in a single repetition, from two biological replicates. Shown are mean ± SEM.

Summary of RAD51 foci in HeLa WT and HeLa DPP9 KD cells as described in (Fig 7C and F). Each dot represents the mean number of RAD51 foci in a single repetition, from three or four biological replicates (HeLa WT (n = 3), HeLa DPP9 KD (n = 4)). Data from the summary of all RAD51 foci per nucleus upon MMC treatment, between biological replicates, were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001). Shown are mean ± SEM.

To directly test whether the N‐terminus of BRCA2 affects the appearance of the RAD51 foci, we constructed a BRCA2ΔMP truncation variant (BRCA2ΔMP3‐3418) lacking the N‐terminal dipeptide Met‐Pro. Both BRCA2ΔMP3‐3418 and full‐length BRCA21‐3418 were cloned into ubiquitin‐fusion vectors to express the two constructs with the desired N‐termini. To avoid possible toxicity by the overexpression of BRCA2, cells were first silenced for endogenous BRCA2 prior to transfection with either construct (Appendix Fig S6), and a silent mutation was introduced into both plasmids to acquire resistance to the BRCA2 siRNA. As expected, RAD51 foci were formed in HeLa WT cells following MMC treatment, a process that was down‐regulated by silencing of BRCA2 (Fig 7C and D). Expression of either BRCA2ΔMP3‐3418 or BRCA21‐3418 resulted in a significant increase in the number of RAD51 foci that appeared in response to MMC, indicating that both BRCA21‐3418 and BRCA2ΔMP3‐3418 can restore the silencing of BRCA2 (Figs 7C and D, and EV4H). However, since DPP9 is active in HeLa WT cells, BRCA21‐3418 may have been processed by DPP9 to BRCA2ΔMP3‐3418. Thus, the assay was also carried out in HeLa DPP9 KD cells to compare the capacities of both BRCA21‐3418 and BRCA2ΔMP3‐3418 constructs to promote RAD51 foci formation. Similar to HeLa WT cells, expression of BRCA2ΔMP3‐3418 into the HeLa DPP9 KD cells first silenced for BRCA2, was accompanied by a significant increase in the number of RAD51 foci in response to MMC. On the other hand, expression of WT BRCA21‐3418 did not lead to a significant increase in the number of MMC‐induced RAD51 foci in the BRCA2‐silenced HeLa DPP9 KD cells (Fig 7E and F, and EV4I). Taken together, these results strongly suggest that DPP9 promotes the formation of RAD51 foci by processing the N‐terminus of BRCA2.

### DNA damaging conditions trigger an interaction between BRCA2 and DPP9

As a starting point, we asked whether DPP9 and BRCA2 are in close proximity in cells, using proximity ligation assays (PLAs), which are especially well‐suited for the detection of dynamic and transient interactions, such as those between an enzyme and its substrate. Initially, only few DPP9‐BRCA2 PLA signals were detected in HeLa cells. However, exposure to Mitomycin C (MMC), a chemotherapeutic agent that forms inter‐strand DNA crosslinks that are converted into secondary DSBs, resulted in a clear increase in the number of DPP9‐BRCA2 PLA events (Figs 1B and C, and EV1A; Appendix Fig S1A and B). Although DPP9 and BRCA2 localize to both the nucleus and the cytoplasm, most of the MMC‐induced BRCA2‐DPP9 PLA signals were located in the cytosol (Fig 1B and C). Given its interactions with both DPP9 (Justa‐Schuch et al, 2016) and BRCA2 (Yuan & Shen, 2001; Mondal et al, 2012; Yue et al, 2012), cells were silenced for FLNA to test whether it is relevant for the observed DPP9‐BRCA2 proximity. Cell silenced for FLNA did not show an increase in the number of BRCA2‐DPP9 interactions in response to MMC, implying that the DNA damage triggered proximity of DPP9 and BRCA2 requires FLNA (Figs 1B and C, and EV1A; Appendix Fig S1A and B).

A
Sequence alignment of BRCA2 homologs from 108 placental mammals (Logo plot) shows conservation of the cleavage site for DPP9 at the BRCA2 N‐terminus. The cartoon depicts the full‐length BRCA2 and its conserved domains with the corresponding interaction partners.B
Quantification of PLAs between BRCA2 and DPP9 in control HeLa WT cells treated with nontargeting siRNA (siNT) or silenced with the indicated oligos. 300 ng/ml MMC was added for 24 h. Each dot represents the number of PLA events in a single cell, from two to seven biological replicates (siNT‐MMC (n = 6), siNT+MMC (n = 7), siDPP9‐MMC (n = 4), siDPP9+MMC (n = 5), siBRCA2‐MMC (n = 2), siBRCA2+MMC (n = 3), siFLNA‐MMC (n = 5), siFLNA+MMC (n = 6), NgtCntrl‐MMC (n = 3), NgtCntrl+MMC (n = 6)). The number of foci are shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (*P ≤ 0.05, ****P ≤ 0.0001).C
Representative PLA images showing close proximity between endogenous DPP9 and endogenous BRCA2 in HeLa WT cells. Exposure of cells to MMC triggers more PLA events (white). Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_2889071 (these images, along with the corresponding controls can be found in Appendix Fig S1A and B).D
Quantification of PLAs between BRCA2 and DPP9 in asynchronous HeLa WT cells or in cells in S‐Phase. Each dot represents the number of PLA events in a single cell, from three biological replicates. Cells were synchronized with a double thymidine block and released for 3 h before the addition of NCS (250 ng / mL for 30 min). Cells were sampled prior to NCS treatment, after NCS treatment, and with 1.5 and 3 h of recovery after NCS treatment. The number of foci is shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (*P ≤ 0.05, **P ≤ 0.01, ****P ≤ 0.0001).E
Representative PLA images showing close proximity between endogenous DPP9 and endogenous BRCA2 in HeLa cells. Exposure of cells to NCS triggers more PLA events (white), which increase in time. Cells in S‐Phase show a greater fold increase in the number of PLA events in comparison with asynchronous cells. Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Antibodies as in (C).F
Co‐immunoprecipitation assays showing binding of BRCA2 and DPP9‐SWT. HEK293 DPP9 KO+DPP9WT cells, were treated with 1 μg/ml Dox (24 h) to induce the expression of DPP9‐FLAG. DNA damage was induced with 300 nM MMC treatment for 24 h. Control cells do not express DPP9 (− Dox). Bound proteins were eluted with a FLAG peptide and analyzed by western blotting (anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_731947, anti‐FLAG RRID:AB_262044).

Sequence alignment of BRCA2 homologs from 108 placental mammals (Logo plot) shows conservation of the cleavage site for DPP9 at the BRCA2 N‐terminus. The cartoon depicts the full‐length BRCA2 and its conserved domains with the corresponding interaction partners.

Quantification of PLAs between BRCA2 and DPP9 in control HeLa WT cells treated with nontargeting siRNA (siNT) or silenced with the indicated oligos. 300 ng/ml MMC was added for 24 h. Each dot represents the number of PLA events in a single cell, from two to seven biological replicates (siNT‐MMC (n = 6), siNT+MMC (n = 7), siDPP9‐MMC (n = 4), siDPP9+MMC (n = 5), siBRCA2‐MMC (n = 2), siBRCA2+MMC (n = 3), siFLNA‐MMC (n = 5), siFLNA+MMC (n = 6), NgtCntrl‐MMC (n = 3), NgtCntrl+MMC (n = 6)). The number of foci are shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (*P ≤ 0.05, ****P ≤ 0.0001).

Representative PLA images showing close proximity between endogenous DPP9 and endogenous BRCA2 in HeLa WT cells. Exposure of cells to MMC triggers more PLA events (white). Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_2889071 (these images, along with the corresponding controls can be found in Appendix Fig S1A and B).

Quantification of PLAs between BRCA2 and DPP9 in asynchronous HeLa WT cells or in cells in S‐Phase. Each dot represents the number of PLA events in a single cell, from three biological replicates. Cells were synchronized with a double thymidine block and released for 3 h before the addition of NCS (250 ng / mL for 30 min). Cells were sampled prior to NCS treatment, after NCS treatment, and with 1.5 and 3 h of recovery after NCS treatment. The number of foci is shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (*P ≤ 0.05, **P ≤ 0.01, ****P ≤ 0.0001).

Representative PLA images showing close proximity between endogenous DPP9 and endogenous BRCA2 in HeLa cells. Exposure of cells to NCS triggers more PLA events (white), which increase in time. Cells in S‐Phase show a greater fold increase in the number of PLA events in comparison with asynchronous cells. Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Antibodies as in (C).

Co‐immunoprecipitation assays showing binding of BRCA2 and DPP9‐SWT. HEK293 DPP9 KO+DPP9WT cells, were treated with 1 μg/ml Dox (24 h) to induce the expression of DPP9‐FLAG. DNA damage was induced with 300 nM MMC treatment for 24 h. Control cells do not express DPP9 (− Dox). Bound proteins were eluted with a FLAG peptide and analyzed by western blotting (anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_731947, anti‐FLAG RRID:AB_262044).

Source data are available online for this figure.

A
Summary of PLAs between BRCA2 and DPP9 in control HeLa WT cells treated with nontargeting siRNA (siNT) or silenced with the indicated oligos (Fig 1B). Each dot represents the mean number of PLA events in a single repetition, from two to seven biological replicates. More than 100 cells were quantified per condition in each experiment. Data from the summary of all PLA events per cell were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (***P ≤ 0.001, ****P ≤ 0.0001).B
Summary of PLAs between BRCA2 and DPP9 in asynchronous HeLa WT cells and HeLa WT cells synchronized to S‐phase. DNA damage was induced by NCS (250 ng/ml for 30 min). Each dot represents the mean number of PLA events in a single repetition. More than 100 cells were quantified per condition in each experiment, from three biological replicates. Data from the summary of all PLA events per cell were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).C
Summary of PLAs between BRCA2 and DPP9 in HeLa WT cells treated with 1G244 (Fig 2C). DNA damage was induced by MMC (300 ng/ml for 24 h). Each dot represents the mean number of PLA events in a single repetition, from three biological replicates. More than 100 cells were quantified per condition in each experiment. Data from the summary of all PLA events per cell were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

Summary of PLAs between BRCA2 and DPP9 in control HeLa WT cells treated with nontargeting siRNA (siNT) or silenced with the indicated oligos (Fig 1B). Each dot represents the mean number of PLA events in a single repetition, from two to seven biological replicates. More than 100 cells were quantified per condition in each experiment. Data from the summary of all PLA events per cell were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (***P ≤ 0.001, ****P ≤ 0.0001).

Summary of PLAs between BRCA2 and DPP9 in asynchronous HeLa WT cells and HeLa WT cells synchronized to S‐phase. DNA damage was induced by NCS (250 ng/ml for 30 min). Each dot represents the mean number of PLA events in a single repetition. More than 100 cells were quantified per condition in each experiment, from three biological replicates. Data from the summary of all PLA events per cell were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

Summary of PLAs between BRCA2 and DPP9 in HeLa WT cells treated with 1G244 (Fig 2C). DNA damage was induced by MMC (300 ng/ml for 24 h). Each dot represents the mean number of PLA events in a single repetition, from three biological replicates. More than 100 cells were quantified per condition in each experiment. Data from the summary of all PLA events per cell were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

To extend these observations, cells were treated with another DSBs causing agent, the radiomimetic drug Neocarzinostatin (NCS). Similarly, NCS‐treated cells presented more DPP9‐BRCA2 PLA signals compared with mock‐treated cells, with most of these events in the cytosol (Figs 1D and E, and EV1B). The fold increase of DPP9‐BRCA2 PLA events in response to NCS was even higher in cells that were first arrested by double thymidine blocks and treated with NCS following release from the second block, validating a close proximity between DPP9 and BRCA2 in S‐phase (Figs 1D and E, and EV1B).

Additionally, we performed co‐immunoprecipitation against DPP9 and tested for the presence of BRCA2. Knockouts of DPP9 were established in HEK Flp‐In™ T‐REx™‐293 cells (HEK293 DPP9 KO) using CRISPR single guide RNAs. These were then used to establish a stable cell line with a Doxycycline (Dox) inducible expression of the cytoplasmic DPP9 isoform (FLAG‐ DPP9‐SWT), (cells characterized in Appendix Fig S2A–E). To enhance the interaction between DPP9 and BRCA2, DNA damage was induced by the addition of MMC. BRCA2 co‐purified with FLAG‐DPP9 (Fig 1F). Collectively, these data strongly suggest that DPP9 and BRCA2 interact in cells in response to DNA damage.

### DPP9 removes the BRCA2 N‐terminal dipeptide Met1Pro2

To test whether DPP9 binds to the N‐terminus of BRCA2, a 40 amino acid peptide was selected. This peptide includes the binding site of PALB2, a protein that ensures the correct intra‐nuclear localization of BRCA2. A shorter BRCA221‐39 peptide was previously co‐crystallized in a complex with PALB2 (Xia et al, 2006; Oliver et al, 2009). Surface Plasmon Resonance (SPR) assays with the synthetic BRCA21‐40 peptide confirmed direct binding to immobilized recombinant DPP9 (Fig 2A). The SPR response curves obtained for DPP9 and BRCA21‐40 were not at perfect equilibrium, whereas the binding behavior of DPP9 to a shorter peptide lacking the Met‐Pro (BRCA23‐40) was less complex, suggesting that the curves for DPP9‐BRCA21‐40 reflect not only the binding of DPP9 and BRCA21‐40, but possibly also on‐chip processing of the BRCA21‐40 peptide by DPP9, and the release of the product (Fig 2A; Appendix Fig S3A–D). Additionally, we cloned and purified a recombinant protein containing the first 40 amino acids of BRCA2 followed by a C‐terminal HA tag, and examined whether DPP9 and BRCA2 show characteristics of an enzyme‐substrate interaction. Previously, we had shown that the binding of the competitive inhibitors 1G244 and SLRFLYEG (Wu et al, 2009; Pilla et al, 2013) leads to a rearrangement of the DPP8/9 active sites resulting in a closed conformation (Ross et al, 2018). Pull‐down assays with immobilized BRCA21‐40HA showed reduced binding to DPP9 in the presence of either 1G244 or SLRFLYEG (Fig 2B). Consistently, cells treated with 1G244 displayed fewer DPP9‐BRCA2 PLA events in response to MMC, compared with mock‐treated cells (Figs 2C and D, and EV1C; Appendix Fig S4A). Thus, both PLA in cells and pull‐down assays with purified recombinant components show the reduced association of BRCA2 and DPP9 in the presence of competitive inhibitors that occupy the active site of DPP9, suggesting that BRCA2 acts as a ligand that competes with these inhibitors for binding to DPP9. Direct processing of the BRCA21‐40 peptide by purified DPP9 was verified by mass spectrometry analysis, revealing the hydrolysis of the peptide bond between Pro2 and Ile3 (Met1Pro2↓Ile3) in the BRCA21‐40 peptide. The resulting BRCA23‐40 product presents a neo N‐terminus with isoleucine in the first position (Fig 2E). No cleavage was observed in control reactions that included the competitive inhibitor SLRFLYEG.

A
Surface Plasmon Resonance (SPR) data showing a direct interaction of purified DPP9 with a BRCA21‐40 peptide and a truncated BRCA23‐40 peptide, which lacks the N‐terminal dipeptide Met‐Pro. A serial dilution of BRCA2‐derived peptides was injected over a surface covered with DPP9. Equilibrium binding isotherms obtained for interactions measured between DPP9 and BRCA21‐40 (black line) and BRCA23‐40 (red line). Data were fitted to a sigmoidal dose–response curve fit. Mean ± SEM of technical triplicates of a representative experiment out of 3.B
Pull‐down assay showing direct binding of purified recombinant DPP9 to a BRCA2 N‐terminal1‐40 fragment immobilized on HA beads. The DPP9 inhibitors 1G244 or SLRFLYEG compete with BRCA21‐40HA for interaction with DPP9. Representative data of three technical replicates are shown. Anti‐HA: RRID:AB_2565334, anti‐DPP9: RRID:AB_2889071.C
Quantification of PLAs showing fewer MMC‐induced DPP9‐BRCA2 PLA events in HeLa cells treated with 10 μM 1G244. Each dot represents the number of PLA events in a single cell, from three biological replicates. The number of foci is shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (***P ≤ 0.001, ****P ≤ 0.0001).D
Representative images of DPP9‐BRCA2 PLA in the presence of 1G244—a competitive inhibitor of DPP9. Control cells were mock treated with DMSO. Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_2889071 (these images, along with the corresponding controls can be found in Appendix Fig S1C).E
In vitro processing of BRCA21‐40 synthetic peptide by purified recombinant DPP9. Samples were analyzed by high‐resolution liquid chromatography/tandem mass spectrometry, in quadruplicate. The panels show extracted MS1 ion chromatograms for both substrate BRCA21‐40 peptide (MPIGSKERPT…) (labeled S, [M+5H]5+
m/z 917.8637; retention time 17.7 min) and product BRCA23‐40 peptide (IGSKERPT…) (labeled P, [M+5H]5+
m/z 872.2451; retention time 17.6 min). The identity of the product peak was established both by accurate mass measurement to within 5 ppm and by product ion spectra.F
Pull‐down assay showing a competition between DPP9 and PALB2 for binding to the BRCA2 N‐terminus. While each protein can bind to the BRCA2 N‐terminal peptide, in the presence of DPP9, the PALB2‐BRCA2 binding is negatively affected. Representative data of three technical replicates are shown. Anti‐PALB2: RRID:AB_890607, anti‐HA: RRID:AB_2565334, anti‐DPP9: RRID:AB_2889071.G
Peptide sequences used in the DPP9 competition assays shown in (H).H
DPP9 activity assays showing different competitive effects of BRCA2 peptides (G), hydrolysis of GP‐AMC is used as read out. Shown is % of inhibition in GP‐AMC processing normalized to the control reactions (DPP9 without competing peptides). Syk1‐31 was used as a positive control. n = 3 independent experiments. Data were analyzed by a one‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (***P ≤ 0.001, ****P ≤ 0.0001).

Surface Plasmon Resonance (SPR) data showing a direct interaction of purified DPP9 with a BRCA21‐40 peptide and a truncated BRCA23‐40 peptide, which lacks the N‐terminal dipeptide Met‐Pro. A serial dilution of BRCA2‐derived peptides was injected over a surface covered with DPP9. Equilibrium binding isotherms obtained for interactions measured between DPP9 and BRCA21‐40 (black line) and BRCA23‐40 (red line). Data were fitted to a sigmoidal dose–response curve fit. Mean ± SEM of technical triplicates of a representative experiment out of 3.

Pull‐down assay showing direct binding of purified recombinant DPP9 to a BRCA2 N‐terminal1‐40 fragment immobilized on HA beads. The DPP9 inhibitors 1G244 or SLRFLYEG compete with BRCA21‐40HA for interaction with DPP9. Representative data of three technical replicates are shown. Anti‐HA: RRID:AB_2565334, anti‐DPP9: RRID:AB_2889071.

Quantification of PLAs showing fewer MMC‐induced DPP9‐BRCA2 PLA events in HeLa cells treated with 10 μM 1G244. Each dot represents the number of PLA events in a single cell, from three biological replicates. The number of foci is shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (***P ≤ 0.001, ****P ≤ 0.0001).

Representative images of DPP9‐BRCA2 PLA in the presence of 1G244—a competitive inhibitor of DPP9. Control cells were mock treated with DMSO. Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_2889071 (these images, along with the corresponding controls can be found in Appendix Fig S1C).

In vitro processing of BRCA21‐40 synthetic peptide by purified recombinant DPP9. Samples were analyzed by high‐resolution liquid chromatography/tandem mass spectrometry, in quadruplicate. The panels show extracted MS1 ion chromatograms for both substrate BRCA21‐40 peptide (MPIGSKERPT…) (labeled S, [M+5H]5+
m/z 917.8637; retention time 17.7 min) and product BRCA23‐40 peptide (IGSKERPT…) (labeled P, [M+5H]5+
m/z 872.2451; retention time 17.6 min). The identity of the product peak was established both by accurate mass measurement to within 5 ppm and by product ion spectra.

Pull‐down assay showing a competition between DPP9 and PALB2 for binding to the BRCA2 N‐terminus. While each protein can bind to the BRCA2 N‐terminal peptide, in the presence of DPP9, the PALB2‐BRCA2 binding is negatively affected. Representative data of three technical replicates are shown. Anti‐PALB2: RRID:AB_890607, anti‐HA: RRID:AB_2565334, anti‐DPP9: RRID:AB_2889071.

Peptide sequences used in the DPP9 competition assays shown in (H).

DPP9 activity assays showing different competitive effects of BRCA2 peptides (G), hydrolysis of GP‐AMC is used as read out. Shown is % of inhibition in GP‐AMC processing normalized to the control reactions (DPP9 without competing peptides). Syk1‐31 was used as a positive control. n = 3 independent experiments. Data were analyzed by a one‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (***P ≤ 0.001, ****P ≤ 0.0001).

Source data are available online for this figure.

Since amino acids 1–40 in the N‐terminus of BRCA2 include the PALB2 interaction site, we asked whether DPP9 and PALB2 compete for binding to the N‐terminus of BRCA2, or whether they bind independently of each other. Pull‐down assays show reduced PALB2 binding in the presence of DPP9 implying that these two proteins sterically hinder mutual binding to BRCA21‐40HA (Fig 2F). To test for residues in BRCA2 that affect cleavage by DPP9, competition assays were performed in which we measured the hydrolysis of the artificial substrate H‐Gly‐Pro‐7‐amino‐4‐methylcoumarin (GP‐AMC) in the presence of the BRCA21‐40 peptide, or variants of this peptide (Fig 2G and H). For control, assays were carried out with the Syk1‐31 peptide (Justa‐Schuch et al, 2016) which as expected inhibited GP‐AMC cleavage. Stronger inhibition was observed with the BRCA21‐40 peptide, while a BRCA2 peptide in which the Pro in position 2 was substituted with a Gly (BRCA21‐40P2G) was a poor inhibitor. Interestingly, a BRCA21‐20 peptide was a less efficient competitor in comparison with BRCA21‐40 suggesting that residues between 20–40 contribute to the interaction with DPP9. We looked more closely into selected residues, based on natural mutations in BRCA2 that were depicted in cBioPortal databases. These assays show that substitution of the conserved Glu in position 13 with a Lys reduces the competitive capacity of the BRCA21‐40 peptide. In line with the pull‐down assays, which suggest a competition between DPP9 and PALB2 for interaction with BRCA2, a stronger effect was observed for the substitutions of Trp31 and Phe32, which are important for interaction with PALB2 (Xia et al, 2006; Oliver et al, 2009).

To better portray the cleavage event, DPP9 crystals were soaked with the BRCA21‐40 peptide. DPP9 crystals exhibited positive electron density at the active site corresponding to the N‐terminal Met1Pro2 dipeptide with the proline residue occupying the S1 subsite (Fig 3), in a dipeptide conformation that was identical to that seen earlier in the DPP9‐SLRFLYEG complex (Ross et al, 2018). No trace of electron density of a peptide continued after the proline residue was present (Fig 3; Appendix Table S1). Since DPP8 and DPP9 share very similar substrate‐binding mechanisms in vitro, and DPP9 crystals are notorious for being less well‐ordered (Ross et al, 2018), we also tested for a possible interaction between DPP8 and BRCA2. Similarly, in DPP8, all 3 copies of the asymmetric unit exhibit clearly interpretable electron density for the dipeptide. The well‐ordered R‐segment, a hallmark of ligand binding, was well‐defined and ordered and supports the full occupation of the active sites. These structures, with the dipeptide captured in the active site, clearly document intracrystalline enzymatic activity of DPP8 and DPP9 resulting in the hydrolysis of the BRCA21‐40 N‐terminal peptide, in accord with the mass spectrometry data (Fig 2E), and release of the BRCA23‐40 peptide.

Shown is the N‐terminal dipeptide of BRCA2 bound to both DPP9 and DPP8. Omit map (Fo‐Fc; 3σ) of a BRCA21‐40 peptide soaked in DPP8 crystals (C2221) and in DPP9 crystals (P1211). The R‐Segment and SUMO‐binding arm (SUBA Pilla et al, 2012) are highlighted in orange and blue, respectively. The arrow marks the position where the BRCA2 peptide is cleaved. For simplification, shown is a monomer of DPP8. Both zoomed views are rotated 45° with respect to the monomer view to better display the ligand.

Source data are available online for this figure.

### DPP9 targets BRCA2 for degradation following DNA damage

Based on our previous findings showing that DPP9 targets Syk and AK2 to the N‐degron pathway (Justa‐Schuch et al, 2016; Finger et al, 2020), we here asked whether DPP9 also influences the stability of BRCA2. Consistent with previous publications (Schoenfeld et al, 2004; Liu et al, 2017), exposure of cells to MMC induced an accelerated degradation of BRCA2, which was less pronounced in the presence of the proteasome inhibitor MG132 (Fig 4A and B). Importantly, cycloheximide (CHX) chase assays revealed a significantly lower turnover of BRCA2 in the HeLa DPP9 Knock‐Down cells (HeLa DPP9 KD; cells previously described in Justa‐Schuch et al (2016) and further characterized in Fig EV2A–C), compared with HeLa WT cells in response to MMC (Fig 4A and B). Similarly, we observed an increase in BRCA2 stability in HeLa WT cells transiently silenced for DPP9 (Fig 4A and B), and in the HEK 293 DPP9 KO cells (Fig 4C and D). In contrast to BRCA2, the stability of endogenous RAD51 was not affected by DPP9 (Fig 4A and B). To test whether the catalytic activity of DPP9 regulates BRCA2 stability, CHX chase assays were carried out in HEK 293 DPP9 KO cells in which we induced the expression of DPP9‐SWT. To test for the relevance of DPP9 activity, we generated a stable cell line HEK293 DPP9 KO+DPP9‐SS729A with an induced expression of the enzymatically inactive DPP9 (DPP9‐SS729A). Importantly, the re‐expression of DPP9‐SWT but not of DPP9‐SS729A restored the MMC‐dependent degradation of BRCA2 (Fig 4C and D; Appendix Fig S2A and B).

A, B
Representative western blots and accompanying graph from more than three biological replicates show that MMC (300 nM, 24 h) induces a rapid turnover of endogenous BRCA2 in HeLa WT cells, which is less pronounced in HeLa WT cells treated with the proteasome inhibitor MG132 (100 μM), in HeLa DPP9 KD cells and in cells transiently silenced for DPP9 (siDPP9). RAD51 stability is not altered in HeLa DPP9 KD cells. Vinculin is a loading control for BRCA2, Tubulin is a loading control for RAD51. Shown images originate from one representative cycloheximide (CHX) chase assay. The ratios of BRCA2 to Vinculin and RAD51 to Tubulin are defined as 100% at time 0 h. Mean ± SEM, data were analyzed by a paired two‐tailed t‐test. (*P ≤ 0.03). Anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_731947, anti‐Vinculin: RRID:AB_477629, anti‐Tubulin: RRID:AB_628412, anti‐RAD51: RRID:AB_1142428.C, D
Representative western blots and graph summarizing results of more than three biological replicates. CHX chase assays show that the MMC‐induced degradation of BRCA2 is less pronounced in HEK293 DPP9 KO cells and in HEK293 DPP9 KO+DPP9‐SS729A cells overexpressing an inactive DPP9‐S mutant. Expression of DPP9 was induced (+ Dox, 1 μg/ml) simultaneously with MMC (300 nM), 24 h. Cells that were overexpressing the active variant (DPP9‐SWT) show similar levels of MMC‐induced BRCA2 degradation. RAD51 stability is not altered by MMC and is similar in all cell lines. Vinculin is a loading control for BRCA2, Tubulin is a loading control for RAD51. Shown images originate from one representative cycloheximide (CHX) chase assay. The ratios of BRCA2 to Vinculin are defined as 100% at time 0 h. Mean ± SEM, data were analyzed by a paired two‐tailed t‐test (*P ≤ 0.03, **P ≤ 0.002). Antibodies as described in (A‐B).E
Graphical presentation of the plasmid used for the production of the N‐terminal‐truncated mutant BRCA2ΔMP3‐1000 that was transfected in H‐I. The BRCA21‐1000 insert was cloned in a similar manner. Both constructs were tagged with a C‐terminal FLAG. The ubiquitin moiety is removed in cells by endogenous ubiquitin isopeptidases.F, G
Representative western blots and graphs summarizing CHX assays from three biological replicates show that the BRCA2ΔMP3‐1000 truncation mutant is degraded at a higher rate compared with the untruncated BRCA21‐1000. Cells were co‐transfected with a GFP expressing plasmid, as transfection and loading control. Control cells were transfected with GFP only (U.T). Image originates from one representative experiment. BRCA2‐FLAG signals are related to the transfection and loading control GFP. BRCA2 levels in relation to GFP were defined as 100% at time 0 h. Mean ± SEM, data were analyzed by a paired two‐tailed t‐test (*P ≤ 0.03). Anti‐FLAG: RRID:AB_262044, anti‐GFP: RRID:AB_641123.

Representative western blots and accompanying graph from more than three biological replicates show that MMC (300 nM, 24 h) induces a rapid turnover of endogenous BRCA2 in HeLa WT cells, which is less pronounced in HeLa WT cells treated with the proteasome inhibitor MG132 (100 μM), in HeLa DPP9 KD cells and in cells transiently silenced for DPP9 (siDPP9). RAD51 stability is not altered in HeLa DPP9 KD cells. Vinculin is a loading control for BRCA2, Tubulin is a loading control for RAD51. Shown images originate from one representative cycloheximide (CHX) chase assay. The ratios of BRCA2 to Vinculin and RAD51 to Tubulin are defined as 100% at time 0 h. Mean ± SEM, data were analyzed by a paired two‐tailed t‐test. (*P ≤ 0.03). Anti‐BRCA2: RRID:AB_2259370, anti‐DPP9: RRID:AB_731947, anti‐Vinculin: RRID:AB_477629, anti‐Tubulin: RRID:AB_628412, anti‐RAD51: RRID:AB_1142428.

Representative western blots and graph summarizing results of more than three biological replicates. CHX chase assays show that the MMC‐induced degradation of BRCA2 is less pronounced in HEK293 DPP9 KO cells and in HEK293 DPP9 KO+DPP9‐SS729A cells overexpressing an inactive DPP9‐S mutant. Expression of DPP9 was induced (+ Dox, 1 μg/ml) simultaneously with MMC (300 nM), 24 h. Cells that were overexpressing the active variant (DPP9‐SWT) show similar levels of MMC‐induced BRCA2 degradation. RAD51 stability is not altered by MMC and is similar in all cell lines. Vinculin is a loading control for BRCA2, Tubulin is a loading control for RAD51. Shown images originate from one representative cycloheximide (CHX) chase assay. The ratios of BRCA2 to Vinculin are defined as 100% at time 0 h. Mean ± SEM, data were analyzed by a paired two‐tailed t‐test (*P ≤ 0.03, **P ≤ 0.002). Antibodies as described in (A‐B).

Graphical presentation of the plasmid used for the production of the N‐terminal‐truncated mutant BRCA2ΔMP3‐1000 that was transfected in H‐I. The BRCA21‐1000 insert was cloned in a similar manner. Both constructs were tagged with a C‐terminal FLAG. The ubiquitin moiety is removed in cells by endogenous ubiquitin isopeptidases.

Representative western blots and graphs summarizing CHX assays from three biological replicates show that the BRCA2ΔMP3‐1000 truncation mutant is degraded at a higher rate compared with the untruncated BRCA21‐1000. Cells were co‐transfected with a GFP expressing plasmid, as transfection and loading control. Control cells were transfected with GFP only (U.T). Image originates from one representative experiment. BRCA2‐FLAG signals are related to the transfection and loading control GFP. BRCA2 levels in relation to GFP were defined as 100% at time 0 h. Mean ± SEM, data were analyzed by a paired two‐tailed t‐test (*P ≤ 0.03). Anti‐FLAG: RRID:AB_262044, anti‐GFP: RRID:AB_641123.

Source data are available online for this figure.

A
Representative histograms of HeLa WT and HeLa DPP9 KD cells investigating their relative DNA content via PI staining. Asynchronized cells and cells that were double thymidine‐treated were investigated. Shown are representatives of the asynchronized, double thymidine release, 3 h of recovery and 24 h of recovery samples.B
Cell cycle distribution of the HeLa WT and HeLa DPP9 KD populations of asynchronized cells and synchronized cells at 0 h, 3, 8, and 24 after double thymidine release. Data from three to six biological replicates (asynchronized (n = 3), synchronized HeLa WT (n = 6), synchronized HeLa DPP9 KD (n = 4)) were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. No significant differences could be detected between the two cell lines. Shown are mean ± SEM.C
Representative western blots showing a CHX chase of HeLa WT cells transiently silenced for DPP9, and of control cells treated with nontargeting siRNA (siNT). Cells were treated with MMC (300 nM, 24 h) prior to the addition of CHX. Anti‐BRCA1: RRID:AB_626761, anti‐DPP9: RRID:AB_731947, anti‐Vinculin: RRID:AB_477629.

Representative histograms of HeLa WT and HeLa DPP9 KD cells investigating their relative DNA content via PI staining. Asynchronized cells and cells that were double thymidine‐treated were investigated. Shown are representatives of the asynchronized, double thymidine release, 3 h of recovery and 24 h of recovery samples.

Cell cycle distribution of the HeLa WT and HeLa DPP9 KD populations of asynchronized cells and synchronized cells at 0 h, 3, 8, and 24 after double thymidine release. Data from three to six biological replicates (asynchronized (n = 3), synchronized HeLa WT (n = 6), synchronized HeLa DPP9 KD (n = 4)) were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. No significant differences could be detected between the two cell lines. Shown are mean ± SEM.

Representative western blots showing a CHX chase of HeLa WT cells transiently silenced for DPP9, and of control cells treated with nontargeting siRNA (siNT). Cells were treated with MMC (300 nM, 24 h) prior to the addition of CHX. Anti‐BRCA1: RRID:AB_626761, anti‐DPP9: RRID:AB_731947, anti‐Vinculin: RRID:AB_477629.

Source data are available online for this figure.

To test whether the N‐terminal Pro determines BRCA2 stability, we determined the turnover of BRCA21‐1000 revealing a half‐life greater than 8 h, similar to the endogenous BRCA2 in the absence of MMC (Fig 4E–G). Additionally, we cloned an N‐terminal BRCA2 truncation mutant (BRCA2ΔMP3‐1000) with isoleucine in the first position that mimics the product generated in vitro upon DPP9 cleavage (MP↓IGSK → IGSK). The half‐life of BRCA2ΔMP3‐1000 was 2–3 h, similar to the turnover rate of endogenous BRCA2 in the presence of MMC, and significantly lower than BRCA2 1‐1000 (Fig 4F and G). To generate the desired N‐terminus both constructs were cloned in frame with an N‐terminal ubiquitin tag, which is removed in cells by endogenous ubiquitin isopeptidases based on the well‐established ubiquitin‐fusion technique (Varshavsky, 2005). We have also observed that the steady‐state levels of BRCA21‐1000 constructs with a P2H or P2G mutation are reduced (Appendix Fig S4B). In summary, Pro2 plays a stabilizing role for BRCA2, removal of which leads to accelerated turnover of BRCA2, phenocopying the MMC‐induced degradation of BRCA2.

### DPP9‐deficient cells show defects in HR‐mediated repair and hypersensitivity to genotoxic agents

Previous studies have shown that high levels of wild‐type BRCA2 or the BRC4 repeat suppress DSB repair by homologous recombination and reduce RAD51 foci formation (Chen et al, 1999; Magwood et al, 2012; Abe & Branzei, 2014). Since DPP9 targets BRCA2 for degradation, we asked whether depletion of DPP9 and stabilization of BRCA2 alter the cellular response to DNA damage. First, we monitored the phosphorylation state of histone H2AX on serine 139 (γH2AX), which is an early and decisive step in the repair of DSBs, that is used as a marker for DSBs (Kuo & Yang, 2008). This analysis revealed significantly more γH2AX in HeLa DPP9 KD cells in response to MMC compared with the corresponding control cells (Fig EV3A). Likewise, γH2AX signals were higher in HeLa DPP9 KD cells following 30 min exposure to NCS (time 0 h) and remained higher also 3 h after removal of NCS (Fig EV3B and C), pointing to a delay in recovery of DSBs in HeLa DPP9 KD cells. At 6 h we observed that the γH2AX levels in the HeLa DPP9 KD cells were comparable to those of WT cells, suggesting that the DSBs have been resolved. To assess whether the enzymatic activity of DPP9 contributes to the elevated levels of γH2AX, we analyzed the response of the DPP9 gene knock‐in mouse embryonic fibroblasts (MEFs), which express enzymatically inactive DPP9 (gki MEF DPP9S729A cells were first described in Gall et al (2013)). Consistently, gki MEF DPP9S729A cells accumulated significantly more γH2AX in response to MMC and displayed a clear delay in the resolution of γH2AX that was formed in response to NCS (Figs 5A and B, and EV3D and E).

A
Representative immunofluorescence images of γH2AX signals (white, RRID:AB_309864) in gki MEF DPP9S729A, showing more γH2AX in gki MEF DPP9S729A cells following removal of Neocarzinostatin (NCS), quantification in (B). MEF DPP9S729A and control MEF cells were treated with 250 ng/mL NCS for 30 min and allowed to recover for the indicated time points. γH2AX signals at time 0, reflect 30 min of NCS and no recovery time. Nuclei are shown in blue (DAPI). Scale bar 10 μm.B
Quantification of γH2AX in gki MEF DPP9S729A and control MEF cells as described in (A). Signals from more than 1,700 cells were quantified per condition per experiment. Mean ± SEM from six biological replicates, each in technical duplicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (*P ≤ 0.05, **P ≤ 0.01).C–E
Dose‐dependent viability assays show a higher sensitivity of HeLa DPP9 KD cells (C and D) and DPP9 silenced cells (siDPP9) (E) to MMC (C) and Olaparib (D, E). BRCA2 was silenced for control. Mean ± SEM of three biological replicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (*P ≤ 0.05, **P ≤ 0.01, ***P ≤ 0.001, ****P ≤ 0.0001).F, G
Quantification of colonies formed after γ‐radiation of HeLa WT and DPP9 KD (F), and MEF WT and gki MEF DPP9S729A cells (G), showing the mean ± SEM of the survival fraction (SF) from three biological replicates. Data were analyzed by an unpaired two‐way ANOVA.

Representative immunofluorescence images of γH2AX signals (white, RRID:AB_309864) in gki MEF DPP9S729A, showing more γH2AX in gki MEF DPP9S729A cells following removal of Neocarzinostatin (NCS), quantification in (B). MEF DPP9S729A and control MEF cells were treated with 250 ng/mL NCS for 30 min and allowed to recover for the indicated time points. γH2AX signals at time 0, reflect 30 min of NCS and no recovery time. Nuclei are shown in blue (DAPI). Scale bar 10 μm.

Quantification of γH2AX in gki MEF DPP9S729A and control MEF cells as described in (A). Signals from more than 1,700 cells were quantified per condition per experiment. Mean ± SEM from six biological replicates, each in technical duplicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (*P ≤ 0.05, **P ≤ 0.01).

Dose‐dependent viability assays show a higher sensitivity of HeLa DPP9 KD cells (C and D) and DPP9 silenced cells (siDPP9) (E) to MMC (C) and Olaparib (D, E). BRCA2 was silenced for control. Mean ± SEM of three biological replicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (*P ≤ 0.05, **P ≤ 0.01, ***P ≤ 0.001, ****P ≤ 0.0001).

Quantification of colonies formed after γ‐radiation of HeLa WT and DPP9 KD (F), and MEF WT and gki MEF DPP9S729A cells (G), showing the mean ± SEM of the survival fraction (SF) from three biological replicates. Data were analyzed by an unpaired two‐way ANOVA.

A
Increased γH2AX signals in HeLa DPP9 KD cells following exposure to 300 nM MMC for 24 h. Tubulin was a loading control. Quantification of the γH2AX / Tubulin signals in HeLa DPP9 KD cells and HeLa WT cells from three biological replicates. The γH2AX / Tubulin ratio in HeLa DPP9 KD cells at 24 h MMC was defined as 100%. Mean ± SEM. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test (****P ≤ 0.0001). Anti‐γH2AX: RRID:AB_2118009, anti‐Tubulin: RRID:AB_628412.B
Representative immunofluorescence images showing more γH2AX (white) in HeLa DPP9 KD cells following removal of Neocarzinostatin (NCS). Nuclei are shown in blue (DAPI). Scale bar 10 μm. HeLa DPP9 KD cells and HeLa WT cells were treated with 250 ng/ml Neocarzinostatin (NCS) for 30 min and allowed to recover for the indicated time points. γH2AX signals of each cell type at time 0, reflect 30 min of NCS, and no recovery time. Anti‐γH2AX: RRID:AB_309864.C
Quantification of mean γH2AX signals from HeLa WT and HeLa DPP9 KD cells. More than 1,300 cells were quantified per condition per experiment. Mean ± SEM from four biological replicates, each in technical duplicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (**P ≤ 0.01, ****P ≤ 0.0001).D
Higher γH2AX signals in gki MEF DPP9S729A cells expressing enzymatically inactive DPP9 compared with MEF WT control cells following exposure to 300 nM MMC for 24 h. Tubulin was a loading control. Quantification of the γH2AX / Tubulin ratios in gki MEF DPP9S729A cells and MEF WT cells from three biological replicates. For normalization, the γH2AX / Tubulin ratio in gki MEF DPP9S729A cells at 24 h MMC was defined as 100%. Mean ± SEM. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test (****P ≤ 0.0001).E
Same as (C) for gki MEF DPP9S729A cells and MEF WT cells. Signals from more than 1,700 cells were quantified per condition per experiment. Mean ± SEM from six biological replicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (****P ≤ 0.0001).

Increased γH2AX signals in HeLa DPP9 KD cells following exposure to 300 nM MMC for 24 h. Tubulin was a loading control. Quantification of the γH2AX / Tubulin signals in HeLa DPP9 KD cells and HeLa WT cells from three biological replicates. The γH2AX / Tubulin ratio in HeLa DPP9 KD cells at 24 h MMC was defined as 100%. Mean ± SEM. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test (****P ≤ 0.0001). Anti‐γH2AX: RRID:AB_2118009, anti‐Tubulin: RRID:AB_628412.

Representative immunofluorescence images showing more γH2AX (white) in HeLa DPP9 KD cells following removal of Neocarzinostatin (NCS). Nuclei are shown in blue (DAPI). Scale bar 10 μm. HeLa DPP9 KD cells and HeLa WT cells were treated with 250 ng/ml Neocarzinostatin (NCS) for 30 min and allowed to recover for the indicated time points. γH2AX signals of each cell type at time 0, reflect 30 min of NCS, and no recovery time. Anti‐γH2AX: RRID:AB_309864.

Quantification of mean γH2AX signals from HeLa WT and HeLa DPP9 KD cells. More than 1,300 cells were quantified per condition per experiment. Mean ± SEM from four biological replicates, each in technical duplicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (**P ≤ 0.01, ****P ≤ 0.0001).

Higher γH2AX signals in gki MEF DPP9S729A cells expressing enzymatically inactive DPP9 compared with MEF WT control cells following exposure to 300 nM MMC for 24 h. Tubulin was a loading control. Quantification of the γH2AX / Tubulin ratios in gki MEF DPP9S729A cells and MEF WT cells from three biological replicates. For normalization, the γH2AX / Tubulin ratio in gki MEF DPP9S729A cells at 24 h MMC was defined as 100%. Mean ± SEM. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test (****P ≤ 0.0001).

Same as (C) for gki MEF DPP9S729A cells and MEF WT cells. Signals from more than 1,700 cells were quantified per condition per experiment. Mean ± SEM from six biological replicates. Data were analyzed by an unpaired two‐way ANOVA with the Sidak's multiple comparison test (****P ≤ 0.0001).

Source data are available online for this figure.

We also tested the sensitivity of DPP9‐deprived cells to the genotoxic agents MMC and Olaparib, an inhibitor of poly(ADP‐ribose) polymerase (PARP). Since inhibition of PARP leads to the conversion of ssDNA breaks into DSBs, defects in HR‐mediated repair are associated with increased sensitivity to PARP inhibitors, and are applied for the treatment of germline BRCA1‐ and BRCA2‐mutated metastatic breast cancers (Menear et al, 2008; Rottenberg et al, 2008; Lord & Ashworth, 2016). We detected a hypersensitivity of HeLa DPP9 KD cells to MMC and Olaparib compared with the HeLa WT cells (Fig 5C and D). Similarly, DPP9‐silenced cells were more sensitive to Olaparib compared with control cells treated with nontargeting siRNA (Fig 5E), albeit not to the same extent as cells transiently silenced for BRCA2. Furthermore, HeLa DPP9 KD cells were slightly, but significantly more sensitive to ionizing radiation (γ‐radiation) compared with the corresponding control cells (Fig 5F). This effect was more pronounced in gki MEF DPP9S729A cells (Fig 5G), linking the hypersensitivity to ionizing radiation to the absence of DPP9 enzymatic activity. Thus, cells lacking DPP9 activity accumulate more unrepaired DNA damage (Fig 5A and B) and are hypersensitive to genotoxic stress caused by MMC, Olaparib, and IR (Fig 5C–G), phenocopying cells overexpressing BRCA2 or the BRC4 repeat (Chen et al, 1999; Magwood et al, 2012; Abe & Branzei, 2014).

### DPP9 activity regulates the formation of RAD51 foci in response to DNA damage

To directly investigate whether DPP9 plays a role in HR, we applied the well‐established DR‐GFP reporter system (Pierce et al, 1999; Fig 6A and B) in the colon cancer cell line HCT116 stably expressing DR‐GFP (Kari et al, 2019). As expected, control cells silenced for BRCA2, RAD51, or FLNA were deficient in HR‐mediated repair of the GFP reporter constructs (Fig 6B; Moynahan et al, 2001; Stark et al, 2004; Yue et al, 2009). Importantly, depletion of DPP9 also resulted in a significant reduction in HR frequency, implying that DPP9 promotes the repair of DSBs by HR. Intriguingly, despite the in vitro structural similarities between DPP8 and DPP9 (Fig 3), depletion of DPP8 did not significantly reduce HR efficiency, suggesting that DPP8 and DPP9 are not biologically redundant for this pathway.

To better understand the observed effect of DPP9 in the repair of DNA, we investigated BRCA2 in DPP9‐depleted cells, specifically its colocalization with γH2AX and with PALB2. In line with the higher stability of BRCA2 (Fig 4), and the observed partially overlapping binding site of both DPP9 and PALB2 to BRCA2 (Fig 2F and H), DPP9‐silenced cells contained significantly more BRCA2‐PALB2 PLA events both in the absence and presence of MMC (Fig 6C and D; Appendix Fig S4C). However, despite the increased colocalization with PALB2, and the higher γH2AX levels, fewer BRCA2‐γH2AX PLA events were formed in HeLa DPP9 KD cells in response to MMC (Fig 6E and F; Appendix Fig S4D and E). Similarly, fewer RAD51 foci were detected in HeLa DPP9 KD cells that were gated for EdU positive cells, HeLa DPP9‐silenced cells, and gki MEF DPP9S729A cells expressing inactive DPP9, in response to MMC compared with the number of RAD51 foci observed in the corresponding control cells (Figs 6G and EV4A–E; Appendix Fig S5A–C). Similarly, less RAD51 accumulated to chromatin fractions following 24 h of exposure to MMC in HeLa DPP9 KD cells (Appendix Fig S5D). To test whether the expression of DPP9 promotes the MMC‐induced appearance of RAD51, we monitored RAD51 in the HEK293 DPP9 KO, in HEK293 DPP9 KO+DPP9‐SWTand HEK293 DPP9 KO+DPP9‐SS729A. The re‐expression of DPP9‐SWT but not DPP9‐SS729A resulted in a significantly greater number of RAD51 foci in response to MMC (Figs 7A and B, and EV4F). Since DPP9‐S does not localize to the nucleus, we asked whether DPP9‐L which contains a nuclear localization signal (Justa‐Schuch et al, 2014) can rescue the formation of RAD51 foci. To address this question, we generated a stable cell line HEK293 DPP9 KO+DPP9‐L (Appendix Fig S2E). Comparing the RAD51 foci formed in the parental cell line and cells overexpressing the DPP9‐S or DPP9‐L isoform, we observed that the overexpression of the DPP9‐S results in more RAD51 foci formation compared with the DPP9‐L (Fig EV4G). Taken together, these results show that DPP9 activity promotes the formation of MMC‐induced RAD51 foci and a prominent role for the cytosolic form of DPP9 in the formation of RAD51 foci in response to MMC‐induced DNA damage.

A
Representative immunofluorescence images showing that re‐expression of DPP9‐SWT leads to an increase in the number of RAD51 foci formed following exposure to MMC in HEK293 DPP9 KO+DPP9‐SWT cells. Expression of DPP9 was induced (+ Dox, 1 μg/ml) simultaneously with MMC (300 nM), 24 h. RAD51 foci are shown in white, and nuclei (DAPI) are shown in blue. Scale bar 10 μm. Anti‐RAD51: RRID:AB_1142428.B
Graph showing the number of RAD51 foci following induction of DPP9‐SWT expression, compared with uninduced HEK293 DPP9 KO+DPP9WT cells (‐Dox). Induction of HEK293 DPP9 KO+DPP9S729A for expression of DPP9‐SS729A did not result in more RAD51 foci. Each dot represents the number of RAD51 foci in a single cell, from three biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).C–F
Representative images (C, E) and summarizing graph (D, F) showing the number of RAD51 foci per nucleus in HeLa WT cells (C, D) or HeLa DPP9 KD cells (E, F). Where stated, cells were treated with control siRNA (siNT) or silenced for BRCA2, and transiently transfected with the BRCA21‐3418 or BRCA23‐3418 constructs. Both BRCA2 constructs can rescue the RAD51 foci formation phenotype to the control levels in HeLa WT cells (C, D). In HeLa DPP9 KD cells significantly more RAD51 foci were present in cells transfected with BRCA2ΔMP3‐3418 compared with BRCA2 silenced cells and cells expressing the untruncated BRCA21‐3418 (E, F). Each dot represents the number of RAD51 foci in a single cell, from three (D) or four (F) biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001). RAD51 foci are shown in white, and nuclei (DAPI) are shown in blue. Scale bar 10 μm. Anti‐RAD51: RRID:AB_1142428; anti‐DPP9: RRID:AB_2889071; anti‐BRCA2: RRID:AB_2259370.

Representative immunofluorescence images showing that re‐expression of DPP9‐SWT leads to an increase in the number of RAD51 foci formed following exposure to MMC in HEK293 DPP9 KO+DPP9‐SWT cells. Expression of DPP9 was induced (+ Dox, 1 μg/ml) simultaneously with MMC (300 nM), 24 h. RAD51 foci are shown in white, and nuclei (DAPI) are shown in blue. Scale bar 10 μm. Anti‐RAD51: RRID:AB_1142428.

Graph showing the number of RAD51 foci following induction of DPP9‐SWT expression, compared with uninduced HEK293 DPP9 KO+DPP9WT cells (‐Dox). Induction of HEK293 DPP9 KO+DPP9S729A for expression of DPP9‐SS729A did not result in more RAD51 foci. Each dot represents the number of RAD51 foci in a single cell, from three biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

Representative images (C, E) and summarizing graph (D, F) showing the number of RAD51 foci per nucleus in HeLa WT cells (C, D) or HeLa DPP9 KD cells (E, F). Where stated, cells were treated with control siRNA (siNT) or silenced for BRCA2, and transiently transfected with the BRCA21‐3418 or BRCA23‐3418 constructs. Both BRCA2 constructs can rescue the RAD51 foci formation phenotype to the control levels in HeLa WT cells (C, D). In HeLa DPP9 KD cells significantly more RAD51 foci were present in cells transfected with BRCA2ΔMP3‐3418 compared with BRCA2 silenced cells and cells expressing the untruncated BRCA21‐3418 (E, F). Each dot represents the number of RAD51 foci in a single cell, from three (D) or four (F) biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001). RAD51 foci are shown in white, and nuclei (DAPI) are shown in blue. Scale bar 10 μm. Anti‐RAD51: RRID:AB_1142428; anti‐DPP9: RRID:AB_2889071; anti‐BRCA2: RRID:AB_2259370.

A
Graphical presentation of the DR‐GFP reporter assay for HR. The construct includes two mutated GFP genes (SceGFP and iGFP) oriented as direct repeats. The SceGFP gene is mutated to contain the recognition site for the rare‐cutting endonuclease I‐SceI and a STOP codon, while the iGFP is a truncated form of GFP. Cells expressing the DR‐GFP are transfected with I‐SceI, which forms a DSB within the SceGFP. For HR‐mediated repair, the iGFP serves as a template for HR‐mediated repair of the DSB in GFP (Pierce et al, 1999).B
DPP9‐silenced cells are less efficient in the repair of DSBs by HR. HCT116 cells stably expressing the DR‐GFP reporter assay for HR efficiency were transiently transfected with the indicated siRNAs, and transfected with an I‐SceI‐expression vector. 48 h after transfection, the percentage of GFP‐positive cells was measured using flow cytometry analysis as an indication of HR efficiency. The graph shows the mean ± SEM from seven biological replicates. Data were analyzed by a paired two‐tailed t‐test (**P ≤ 0.01, ***P ≤ 0.001, ****P ≤ 0.0001).C
Representative PLA images of BRCA2‐PALB2 PLA experiments showing more PLA events (white) in HeLa cells silenced for DPP9 (siDPP9) with respect to cells treated with nontargeting siRNA (siNT). Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐PALB2: RRID:AB_890607, anti‐BRCA2: RRID:AB_2259370.D
Quantification of PLAs experiments showing more BRCA2‐PALB2 PLA events in cells silenced for DPP9 (siDPP9) in comparison with nontargeting controls (siNT). Each dot represents the number of PLA events in a single cell, from four biological replicates. The technical control samples (NgtCntrl) omitted the BRCA2 antibody. The number of foci is shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).E
Representative PLA images of γH2Ax‐BRCA2 PLA experiment showing a reduction in the number of PLA events (white) between γH2Ax and BRCA2 in HeLa DPP9 KD cells. Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐γH2Ax: RRID:AB_2118009, anti‐BRCA2: RRID:AB_2259370 (these images, along with the corresponding controls can be found in Appendix Fig S1D).F
Quantification of PLAs showing fewer MMC‐induced γH2Ax‐BRCA2 PLA events in HeLa DPP9 KD cells, in comparison with HeLa WT cells. Each dot represents the number of PLA events in a single cell, from three biological replicates. The technical control samples (NgtCntrl) omitted the γH2Ax antibody. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).G
Graph showing fewer RAD51 foci in DPP9‐silenced cells compared with control cells, following exposure to MMC. Each dot represents the number of RAD51 foci in a single cell, from two to six biological replicates: siNT‐MMC (n = 6), siNT+MMC (n = 6), siDPP9‐MMC (n = 5), siDPP9+MMC (n = 5), siBRCA2‐MMC (n = 3), siBRCA2+MMC (n = 3), siRAD51‐MMC (n = 2), siRAD51+MMC (n = 2). Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

Graphical presentation of the DR‐GFP reporter assay for HR. The construct includes two mutated GFP genes (SceGFP and iGFP) oriented as direct repeats. The SceGFP gene is mutated to contain the recognition site for the rare‐cutting endonuclease I‐SceI and a STOP codon, while the iGFP is a truncated form of GFP. Cells expressing the DR‐GFP are transfected with I‐SceI, which forms a DSB within the SceGFP. For HR‐mediated repair, the iGFP serves as a template for HR‐mediated repair of the DSB in GFP (Pierce et al, 1999).

DPP9‐silenced cells are less efficient in the repair of DSBs by HR. HCT116 cells stably expressing the DR‐GFP reporter assay for HR efficiency were transiently transfected with the indicated siRNAs, and transfected with an I‐SceI‐expression vector. 48 h after transfection, the percentage of GFP‐positive cells was measured using flow cytometry analysis as an indication of HR efficiency. The graph shows the mean ± SEM from seven biological replicates. Data were analyzed by a paired two‐tailed t‐test (**P ≤ 0.01, ***P ≤ 0.001, ****P ≤ 0.0001).

Representative PLA images of BRCA2‐PALB2 PLA experiments showing more PLA events (white) in HeLa cells silenced for DPP9 (siDPP9) with respect to cells treated with nontargeting siRNA (siNT). Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐PALB2: RRID:AB_890607, anti‐BRCA2: RRID:AB_2259370.

Quantification of PLAs experiments showing more BRCA2‐PALB2 PLA events in cells silenced for DPP9 (siDPP9) in comparison with nontargeting controls (siNT). Each dot represents the number of PLA events in a single cell, from four biological replicates. The technical control samples (NgtCntrl) omitted the BRCA2 antibody. The number of foci is shown based on their cellular localisation. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

Representative PLA images of γH2Ax‐BRCA2 PLA experiment showing a reduction in the number of PLA events (white) between γH2Ax and BRCA2 in HeLa DPP9 KD cells. Phalloidin (green) stains actin filaments and DAPI (blue) stains the nucleus. Scale bar 10 μm. Anti‐γH2Ax: RRID:AB_2118009, anti‐BRCA2: RRID:AB_2259370 (these images, along with the corresponding controls can be found in Appendix Fig S1D).

Quantification of PLAs showing fewer MMC‐induced γH2Ax‐BRCA2 PLA events in HeLa DPP9 KD cells, in comparison with HeLa WT cells. Each dot represents the number of PLA events in a single cell, from three biological replicates. The technical control samples (NgtCntrl) omitted the γH2Ax antibody. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

Graph showing fewer RAD51 foci in DPP9‐silenced cells compared with control cells, following exposure to MMC. Each dot represents the number of RAD51 foci in a single cell, from two to six biological replicates: siNT‐MMC (n = 6), siNT+MMC (n = 6), siDPP9‐MMC (n = 5), siDPP9+MMC (n = 5), siBRCA2‐MMC (n = 3), siBRCA2+MMC (n = 3), siRAD51‐MMC (n = 2), siRAD51+MMC (n = 2). Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001).

A
A graph summarizing the mean number of RAD51 foci in HeLa WT cells treated with nontargeting siRNA (siNT) or silenced with the indicated oligos. Each dot represents the mean number of RAD51 foci in a single repetition, from two to six biological replicates. Statistical analysis on the data is shown in Fig 6G. Shown are mean ± SEM.B
A graph summarizing the number of RAD51 foci in HeLa WT and DPP9 KD cells treated with EdU and MMC. Shown is the quantification of RAD51 foci in EdU positive cells. HeLa DPP9 KD cells display fewer RAD51 foci in comparison with their WT counterparts. Each dot represents the number of RAD51 foci in a single cell, from three biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM.C
A graph summarizing the mean number of RAD51 foci in EdU positive HeLa WT and DPP9 KD cells. Each dot represents the mean number of RAD51 foci in a single repetition, from three biological replicates. Statistical analysis on the data is shown in Fig EV4B. Shown are mean ± SEM.D
A graph summarizing the number of RAD51 foci in MEF WT and gki MEF DPP9S729A cells treated with MMC. Shown is the quantification of RAD51 foci. Each dot represents the number of RAD51 foci in a single cell, from three biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM.E
A graph summarizing the mean number of RAD51 foci in MEF WT and gki MEF DPP9S729A cells upon MMC treatment. Each dot represents the mean number of RAD51 foci in a single repetition, from three biological replicates. Statistical analysis on the data is shown in Fig EV4D. Shown are mean ± SEM.F
A graph summarizing the mean number of RAD51 foci following induction of DPP9‐SWT expression, compared with uninduced HEK293 DPP9 KO+DPP9WT cells (‐Dox). Induction of HEK293 DPP9 KO+DPP9S729A for expression of DPP9‐SS729A did not result in more RAD51 foci. Each dot represents the mean number of RAD51 foci in a single repetition, from three biological replicates. Statistical analysis on the data is shown in Fig 7B. Shown are mean ± SEM.G
A graph summarizing the number of RAD51 foci following induction of DPP9‐SWT or DPP9‐LWT expression, compared with HEK293 DPP9WT cells. The number of RAD51 foci formed by the HEK293 DPP9 KO+DPP9‐SWT cells was similar to the HEK293 DPP9WT cells, compared with the HEK293 DPP9 KO+DPP9‐LWT cells. Each dot represents the mean number of RAD51 foci in a single repetition, from two biological replicates. Shown are mean ± SEM.H, I
Summary of RAD51 foci in HeLa WT and HeLa DPP9 KD cells as described in (Fig 7C and F). Each dot represents the mean number of RAD51 foci in a single repetition, from three or four biological replicates (HeLa WT (n = 3), HeLa DPP9 KD (n = 4)). Data from the summary of all RAD51 foci per nucleus upon MMC treatment, between biological replicates, were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001). Shown are mean ± SEM.

A graph summarizing the mean number of RAD51 foci in HeLa WT cells treated with nontargeting siRNA (siNT) or silenced with the indicated oligos. Each dot represents the mean number of RAD51 foci in a single repetition, from two to six biological replicates. Statistical analysis on the data is shown in Fig 6G. Shown are mean ± SEM.

A graph summarizing the number of RAD51 foci in HeLa WT and DPP9 KD cells treated with EdU and MMC. Shown is the quantification of RAD51 foci in EdU positive cells. HeLa DPP9 KD cells display fewer RAD51 foci in comparison with their WT counterparts. Each dot represents the number of RAD51 foci in a single cell, from three biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM.

A graph summarizing the mean number of RAD51 foci in EdU positive HeLa WT and DPP9 KD cells. Each dot represents the mean number of RAD51 foci in a single repetition, from three biological replicates. Statistical analysis on the data is shown in Fig EV4B. Shown are mean ± SEM.

A graph summarizing the number of RAD51 foci in MEF WT and gki MEF DPP9S729A cells treated with MMC. Shown is the quantification of RAD51 foci. Each dot represents the number of RAD51 foci in a single cell, from three biological replicates. Data were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM.

A graph summarizing the mean number of RAD51 foci in MEF WT and gki MEF DPP9S729A cells upon MMC treatment. Each dot represents the mean number of RAD51 foci in a single repetition, from three biological replicates. Statistical analysis on the data is shown in Fig EV4D. Shown are mean ± SEM.

A graph summarizing the mean number of RAD51 foci following induction of DPP9‐SWT expression, compared with uninduced HEK293 DPP9 KO+DPP9WT cells (‐Dox). Induction of HEK293 DPP9 KO+DPP9S729A for expression of DPP9‐SS729A did not result in more RAD51 foci. Each dot represents the mean number of RAD51 foci in a single repetition, from three biological replicates. Statistical analysis on the data is shown in Fig 7B. Shown are mean ± SEM.

A graph summarizing the number of RAD51 foci following induction of DPP9‐SWT or DPP9‐LWT expression, compared with HEK293 DPP9WT cells. The number of RAD51 foci formed by the HEK293 DPP9 KO+DPP9‐SWT cells was similar to the HEK293 DPP9WT cells, compared with the HEK293 DPP9 KO+DPP9‐LWT cells. Each dot represents the mean number of RAD51 foci in a single repetition, from two biological replicates. Shown are mean ± SEM.

Summary of RAD51 foci in HeLa WT and HeLa DPP9 KD cells as described in (Fig 7C and F). Each dot represents the mean number of RAD51 foci in a single repetition, from three or four biological replicates (HeLa WT (n = 3), HeLa DPP9 KD (n = 4)). Data from the summary of all RAD51 foci per nucleus upon MMC treatment, between biological replicates, were analyzed by a two‐way ANOVA, with the Tukey's multiple comparison test. Shown are mean ± SEM (****P ≤ 0.0001). Shown are mean ± SEM.

To directly test whether the N‐terminus of BRCA2 affects the appearance of the RAD51 foci, we constructed a BRCA2ΔMP truncation variant (BRCA2ΔMP3‐3418) lacking the N‐terminal dipeptide Met‐Pro. Both BRCA2ΔMP3‐3418 and full‐length BRCA21‐3418 were cloned into ubiquitin‐fusion vectors to express the two constructs with the desired N‐termini. To avoid possible toxicity by the overexpression of BRCA2, cells were first silenced for endogenous BRCA2 prior to transfection with either construct (Appendix Fig S6), and a silent mutation was introduced into both plasmids to acquire resistance to the BRCA2 siRNA. As expected, RAD51 foci were formed in HeLa WT cells following MMC treatment, a process that was down‐regulated by silencing of BRCA2 (Fig 7C and D). Expression of either BRCA2ΔMP3‐3418 or BRCA21‐3418 resulted in a significant increase in the number of RAD51 foci that appeared in response to MMC, indicating that both BRCA21‐3418 and BRCA2ΔMP3‐3418 can restore the silencing of BRCA2 (Figs 7C and D, and EV4H). However, since DPP9 is active in HeLa WT cells, BRCA21‐3418 may have been processed by DPP9 to BRCA2ΔMP3‐3418. Thus, the assay was also carried out in HeLa DPP9 KD cells to compare the capacities of both BRCA21‐3418 and BRCA2ΔMP3‐3418 constructs to promote RAD51 foci formation. Similar to HeLa WT cells, expression of BRCA2ΔMP3‐3418 into the HeLa DPP9 KD cells first silenced for BRCA2, was accompanied by a significant increase in the number of RAD51 foci in response to MMC. On the other hand, expression of WT BRCA21‐3418 did not lead to a significant increase in the number of MMC‐induced RAD51 foci in the BRCA2‐silenced HeLa DPP9 KD cells (Fig 7E and F, and EV4I). Taken together, these results strongly suggest that DPP9 promotes the formation of RAD51 foci by processing the N‐terminus of BRCA2.

### Discussion

This work identifies the rate‐limiting proline‐cleaving protease DPP9, as a regulator of BRCA2 stability and highlights the stabilizing role of the evolutionarily conserved Pro2 in the BRCA2 N‐terminus.

The imino acid proline is unique with its rigid ring‐like side chain. N‐terminal prolines confer high stability to proteins when found in the first or second N‐terminal position (Bachmair et al, 1986), since these are not identified by the classical N‐Recognins of the Arg/N‐degron and Ac/N‐degron pathways (Varshavsky, 2019). Therefore, the targeting and removal of such prolines can be utilized as major sites for regulating changes in protein stability. The best‐characterized example is the Pro/N‐degron pathway, which employs GID ubiquitin ligase complexes. These super‐complexes include substrate receptors that bind to prolines, with a preference for a four amino acid motif (Pro‐Gly‐Leu‐Trp; Dong et al, 2018; Sherpa et al, 2021). Regulated targeting of proline‐containing proteins by this pathway is accomplished by stimulation of complex assembly and expression of the receptor subunit (Menssen et al, 2018; Melnykov et al, 2019; Qiao et al, 2020). DPP9 presents an alternative pathway that cells apply to convert otherwise stable proteins with N‐terminal prolines such as AK2 and BRCA2 into substrates for proteasomal degradation. In contrast to the seemingly constitutive processing of AK2 by DPP9 (Finger et al, 2020), the interaction of BRCA2 with DPP9 is induced in response to genotoxic stress, which also promotes BRCA2 degradation. Another verified DPP9 substrate is Syk, with alanine in the second position. Similar to BRCA2, Syk degradation does not appear to be constitutive but instead is induced following stimulation of the B‐cell receptor, a process that relies on DPP9 activity (Justa‐Schuch et al, 2016).

How DNA damage signals for increased interaction between DPP9 and BRCA2 remains to be shown. However, we note that the N‐terminus of BRCA2 is intrinsically disordered (Le et al, 2020; Sidhu et al, 2020; Paul et al, 2021) and is involved in the formation of BRCA2 dimers and multimers through self‐interactions (Shahid et al, 2014; Reuter et al, 2015; Sánchez et al, 2017; Le et al, 2020; Sidhu et al, 2020). Thus, it is tempting to speculate that BRCA2 is shielded from DPP9 in the multimeric form, and becomes available as a DPP9 substrate by a transition of BRCA2 multimers to monomers, a process that is favored by ssDNA, RAD51, increased temperature, and the BRCA2 chaperone DSS1 (Le et al, 2020; Sidhu et al, 2020). Since FLNA supports the MMC‐induced proximity between DPP9 and BRCA2 it is tempting to speculate that FLNA binds to both proteins thereby increasing their local concentration to support cleavage.

Finally, N‐terminally tagged constructs of BRCA2 are applied in several studies and have allowed important discoveries in the field of HR. We can speculate that these complementation assays with N‐terminally tagged BRCA2 constructs were possible due to differences in the expression levels of the tagged BRCA2 protein compared with its endogenous levels, which thus compensate for the lack in DPP9‐mediated turnover. We raise awareness to the role of the BRCA2 N‐terminus in regulating BRCA2 stability.

Here, we show that DPP9‐depleted cells are hypersensitive to genotoxic agents, are less efficient in HR repair, and accumulate more γH2AX in response to DNA damage. The involvement of proteases in DNA maintenance has been shown for example for the DNA‐binding metalloprotease SPRTN and the trypsin‐like protease FAM111A that remove DNA‐protein crosslinks (Lopez‐Mosqueda et al, 2016; Stingele et al, 2016; Vaz et al, 2016; Kojima et al, 2020). The ubiquitin protease system facilitates HR by ubiquitinating Rad51 and the replication protein A (RPA) on DSBs, leading to the removal of both proteins from these sites (Elia et al, 2015; Feeney et al, 2017; Gong & Chen, 2011; Inano et al, 2017; Liu et al, 2011). While we observed a similar fold increase in the number of BRCA2‐DPP9 PLA events in response to MMC‐ and NCS‐induced DNA damage in both, the nucleus and the cytosol, the majority of the interaction events are in the cytosol, suggesting that the processing of BRCA2 by DPP9 occurs predominantly in this compartment, and not necessarily on the chromatin. Previous work has shown that the ubiquitin‐specific protease USP21, which is also found in cytosol, counteracts the degradation of BRCA2 (Liu et al, 2017). Thus, N‐terminal processing and deubiquitination of BRCA2 appear to occur preferentially in the cytoplasm, presenting a sensitive system of opposing forces that fine‐tune the cellular concentrations of BRCA2. The importance of regulating BRCA2 steady‐state levels can be observed in patients with sporadic breast cancer where BRCA2 is overexpressed (Bièche et al, 1999; Egawa et al, 2002; Wang et al, 2018). Similarly, the expression levels of USP21, which deubiquitinates and stabilizes BRCA2 negatively correlate with the survival of patients with hepatocellular carcinoma (Liu et al, 2017). Similarly, low DPP9 expression, which should allow higher BRCA2 levels, correlates with a poorer prognosis for patients with breast cancer, an effect that is not seen for DPP8 (Fig EV5A and B). That lower BRCA2 levels impair HR is well‐established. A possible scenario is that the cellular levels of BRCA2 are tightly regulated and fine‐tuned to allow productive molecular interactions of BRCA2 with PALB2 and RAD51. Consistent with the higher levels of BRCA2 in DPP9‐deficient cells, more PALB2‐BRCA2 interactions are observed in these cells, suggesting that DPP9 limits this interaction by lowering the cellular concentration of BRCA2. DPP9 may additionally limit this interaction by competing with PALB2 for interaction with BRCA2. Nonetheless, despite the increase in the BRCA2‐PALB2 association, DPP9‐depleted cells show a decrease in the colocalization of BRCA2 with γH2AX. Consistently, DPP9‐depleted cells accumulate fewer RAD51 foci, a phenotype that can be partially restored by the overexpression of active DPP9‐SWT, less efficiently by DPP9‐LWT, but not by the enzymatically inactive DPP9‐SS720A mutant. Additionally, the expression of the BRCA2ΔMP truncation mutant results in a better recovery of RAD51 foci in DPP9 KD cells silenced for BRCA2, compared with the expression of the WT BRCA2 construct. Since both constructs compensate for BRCA2 silencing in cells expressing endogenous levels of DPP9, these results imply that DPP9 promotes the formation of RAD51 foci by modifying the N‐terminus of BRCA2. Similarly, fewer RAD51 foci were reported for cells expressing an excess of BRCA2 or the BRC4 repeat (Chen et al, 1999; Magwood et al, 2012; Abe & Branzei, 2014).

A, B
Kaplan–Meier survival curves of Breast cancer patients from The Human Protein Atlas into “high DPP9 or DPP8” (n = 282, 445) and “low DPP9 or DPP8” (n = 408, 630) mRNA expression (greater than or less than 9.678 or 4.36 reads per kilobase per million, respectively). P‐values calculated by log‐rank (Mantel–Cox) indicated greater overall survival in patients with high levels of DPP9 expression (A), while differences in the DPP8 levels were not statistically significant (B).

Kaplan–Meier survival curves of Breast cancer patients from The Human Protein Atlas into “high DPP9 or DPP8” (n = 282, 445) and “low DPP9 or DPP8” (n = 408, 630) mRNA expression (greater than or less than 9.678 or 4.36 reads per kilobase per million, respectively). P‐values calculated by log‐rank (Mantel–Cox) indicated greater overall survival in patients with high levels of DPP9 expression (A), while differences in the DPP8 levels were not statistically significant (B).

The interaction between BRCA2 and RAD51 involves eight BRC repeats in the centre of BRCA2 and an additional domain in the C‐terminus (CTD) allowing BRCA2 to bind multiple copies of RAD51 (Davies et al, 2001; Pellegrini et al, 2002; Esashi et al, 2005, 2007; Galkin et al, 2005; Davies & Pellegrini, 2007). Different studies estimate a ratio of one BRCA2 monomer binding simultaneously at least 5–7 molecules of RAD51 (Yang et al, 2005; Jensen et al, 2010; Liu et al, 2010; Shahid et al, 2014; Sidhu et al, 2020). In vitro studies suggest that by binding to multiple monomers of RAD51, BRCA2 provides a rapid mechanism for nucleating the RAD51 filaments on the ssDNA and promoting their growth (Jensen et al, 2010; Liu et al, 2010; Thorslund et al, 2010; Carreira & Kowalczykowski, 2011; Shahid et al, 2014; Sánchez et al, 2017). Thus, the DNA damage‐induced degradation of BRCA2 that we and others observe (Schoenfeld et al, 2004; Liu et al, 2017) may serve to establish an optimal stoichiometric ratio between BRCA2 and RAD51 for repair. In this scenario, DPP9 facilitates efficient repair by fine‐tuning the cellular concentration of BRCA2.

The interaction between RAD51 and BRCA2 is cell‐cycle‐dependent (Ayoub et al, 2009), through the CDK‐mediated phosphorylation of the BRCA2 C‐terminus at S3291 (Esashi et al, 2005) and ubiquitination of RAD51, which interfere with the interaction (Luo et al, 2016). Fine‐tuning the protein levels of BRCA2 by DPP9 presents an additional layer of regulation cells applied to ensure efficient repair by HR.

Finally, our results set the ground for future analysis of DPP9 activity in breast cancer and suggest that DPP9 inhibition in combination with Olaparib or radiation presents future potential therapies for patients with breast cancer.

### DPP9 regulates BRCA2 stability

This work identifies the rate‐limiting proline‐cleaving protease DPP9, as a regulator of BRCA2 stability and highlights the stabilizing role of the evolutionarily conserved Pro2 in the BRCA2 N‐terminus.

The imino acid proline is unique with its rigid ring‐like side chain. N‐terminal prolines confer high stability to proteins when found in the first or second N‐terminal position (Bachmair et al, 1986), since these are not identified by the classical N‐Recognins of the Arg/N‐degron and Ac/N‐degron pathways (Varshavsky, 2019). Therefore, the targeting and removal of such prolines can be utilized as major sites for regulating changes in protein stability. The best‐characterized example is the Pro/N‐degron pathway, which employs GID ubiquitin ligase complexes. These super‐complexes include substrate receptors that bind to prolines, with a preference for a four amino acid motif (Pro‐Gly‐Leu‐Trp; Dong et al, 2018; Sherpa et al, 2021). Regulated targeting of proline‐containing proteins by this pathway is accomplished by stimulation of complex assembly and expression of the receptor subunit (Menssen et al, 2018; Melnykov et al, 2019; Qiao et al, 2020). DPP9 presents an alternative pathway that cells apply to convert otherwise stable proteins with N‐terminal prolines such as AK2 and BRCA2 into substrates for proteasomal degradation. In contrast to the seemingly constitutive processing of AK2 by DPP9 (Finger et al, 2020), the interaction of BRCA2 with DPP9 is induced in response to genotoxic stress, which also promotes BRCA2 degradation. Another verified DPP9 substrate is Syk, with alanine in the second position. Similar to BRCA2, Syk degradation does not appear to be constitutive but instead is induced following stimulation of the B‐cell receptor, a process that relies on DPP9 activity (Justa‐Schuch et al, 2016).

How DNA damage signals for increased interaction between DPP9 and BRCA2 remains to be shown. However, we note that the N‐terminus of BRCA2 is intrinsically disordered (Le et al, 2020; Sidhu et al, 2020; Paul et al, 2021) and is involved in the formation of BRCA2 dimers and multimers through self‐interactions (Shahid et al, 2014; Reuter et al, 2015; Sánchez et al, 2017; Le et al, 2020; Sidhu et al, 2020). Thus, it is tempting to speculate that BRCA2 is shielded from DPP9 in the multimeric form, and becomes available as a DPP9 substrate by a transition of BRCA2 multimers to monomers, a process that is favored by ssDNA, RAD51, increased temperature, and the BRCA2 chaperone DSS1 (Le et al, 2020; Sidhu et al, 2020). Since FLNA supports the MMC‐induced proximity between DPP9 and BRCA2 it is tempting to speculate that FLNA binds to both proteins thereby increasing their local concentration to support cleavage.

Finally, N‐terminally tagged constructs of BRCA2 are applied in several studies and have allowed important discoveries in the field of HR. We can speculate that these complementation assays with N‐terminally tagged BRCA2 constructs were possible due to differences in the expression levels of the tagged BRCA2 protein compared with its endogenous levels, which thus compensate for the lack in DPP9‐mediated turnover. We raise awareness to the role of the BRCA2 N‐terminus in regulating BRCA2 stability.

### DPP9 promotes HR‐mediated DNA repair

Here, we show that DPP9‐depleted cells are hypersensitive to genotoxic agents, are less efficient in HR repair, and accumulate more γH2AX in response to DNA damage. The involvement of proteases in DNA maintenance has been shown for example for the DNA‐binding metalloprotease SPRTN and the trypsin‐like protease FAM111A that remove DNA‐protein crosslinks (Lopez‐Mosqueda et al, 2016; Stingele et al, 2016; Vaz et al, 2016; Kojima et al, 2020). The ubiquitin protease system facilitates HR by ubiquitinating Rad51 and the replication protein A (RPA) on DSBs, leading to the removal of both proteins from these sites (Elia et al, 2015; Feeney et al, 2017; Gong & Chen, 2011; Inano et al, 2017; Liu et al, 2011). While we observed a similar fold increase in the number of BRCA2‐DPP9 PLA events in response to MMC‐ and NCS‐induced DNA damage in both, the nucleus and the cytosol, the majority of the interaction events are in the cytosol, suggesting that the processing of BRCA2 by DPP9 occurs predominantly in this compartment, and not necessarily on the chromatin. Previous work has shown that the ubiquitin‐specific protease USP21, which is also found in cytosol, counteracts the degradation of BRCA2 (Liu et al, 2017). Thus, N‐terminal processing and deubiquitination of BRCA2 appear to occur preferentially in the cytoplasm, presenting a sensitive system of opposing forces that fine‐tune the cellular concentrations of BRCA2. The importance of regulating BRCA2 steady‐state levels can be observed in patients with sporadic breast cancer where BRCA2 is overexpressed (Bièche et al, 1999; Egawa et al, 2002; Wang et al, 2018). Similarly, the expression levels of USP21, which deubiquitinates and stabilizes BRCA2 negatively correlate with the survival of patients with hepatocellular carcinoma (Liu et al, 2017). Similarly, low DPP9 expression, which should allow higher BRCA2 levels, correlates with a poorer prognosis for patients with breast cancer, an effect that is not seen for DPP8 (Fig EV5A and B). That lower BRCA2 levels impair HR is well‐established. A possible scenario is that the cellular levels of BRCA2 are tightly regulated and fine‐tuned to allow productive molecular interactions of BRCA2 with PALB2 and RAD51. Consistent with the higher levels of BRCA2 in DPP9‐deficient cells, more PALB2‐BRCA2 interactions are observed in these cells, suggesting that DPP9 limits this interaction by lowering the cellular concentration of BRCA2. DPP9 may additionally limit this interaction by competing with PALB2 for interaction with BRCA2. Nonetheless, despite the increase in the BRCA2‐PALB2 association, DPP9‐depleted cells show a decrease in the colocalization of BRCA2 with γH2AX. Consistently, DPP9‐depleted cells accumulate fewer RAD51 foci, a phenotype that can be partially restored by the overexpression of active DPP9‐SWT, less efficiently by DPP9‐LWT, but not by the enzymatically inactive DPP9‐SS720A mutant. Additionally, the expression of the BRCA2ΔMP truncation mutant results in a better recovery of RAD51 foci in DPP9 KD cells silenced for BRCA2, compared with the expression of the WT BRCA2 construct. Since both constructs compensate for BRCA2 silencing in cells expressing endogenous levels of DPP9, these results imply that DPP9 promotes the formation of RAD51 foci by modifying the N‐terminus of BRCA2. Similarly, fewer RAD51 foci were reported for cells expressing an excess of BRCA2 or the BRC4 repeat (Chen et al, 1999; Magwood et al, 2012; Abe & Branzei, 2014).

A, B
Kaplan–Meier survival curves of Breast cancer patients from The Human Protein Atlas into “high DPP9 or DPP8” (n = 282, 445) and “low DPP9 or DPP8” (n = 408, 630) mRNA expression (greater than or less than 9.678 or 4.36 reads per kilobase per million, respectively). P‐values calculated by log‐rank (Mantel–Cox) indicated greater overall survival in patients with high levels of DPP9 expression (A), while differences in the DPP8 levels were not statistically significant (B).

Kaplan–Meier survival curves of Breast cancer patients from The Human Protein Atlas into “high DPP9 or DPP8” (n = 282, 445) and “low DPP9 or DPP8” (n = 408, 630) mRNA expression (greater than or less than 9.678 or 4.36 reads per kilobase per million, respectively). P‐values calculated by log‐rank (Mantel–Cox) indicated greater overall survival in patients with high levels of DPP9 expression (A), while differences in the DPP8 levels were not statistically significant (B).

The interaction between BRCA2 and RAD51 involves eight BRC repeats in the centre of BRCA2 and an additional domain in the C‐terminus (CTD) allowing BRCA2 to bind multiple copies of RAD51 (Davies et al, 2001; Pellegrini et al, 2002; Esashi et al, 2005, 2007; Galkin et al, 2005; Davies & Pellegrini, 2007). Different studies estimate a ratio of one BRCA2 monomer binding simultaneously at least 5–7 molecules of RAD51 (Yang et al, 2005; Jensen et al, 2010; Liu et al, 2010; Shahid et al, 2014; Sidhu et al, 2020). In vitro studies suggest that by binding to multiple monomers of RAD51, BRCA2 provides a rapid mechanism for nucleating the RAD51 filaments on the ssDNA and promoting their growth (Jensen et al, 2010; Liu et al, 2010; Thorslund et al, 2010; Carreira & Kowalczykowski, 2011; Shahid et al, 2014; Sánchez et al, 2017). Thus, the DNA damage‐induced degradation of BRCA2 that we and others observe (Schoenfeld et al, 2004; Liu et al, 2017) may serve to establish an optimal stoichiometric ratio between BRCA2 and RAD51 for repair. In this scenario, DPP9 facilitates efficient repair by fine‐tuning the cellular concentration of BRCA2.

The interaction between RAD51 and BRCA2 is cell‐cycle‐dependent (Ayoub et al, 2009), through the CDK‐mediated phosphorylation of the BRCA2 C‐terminus at S3291 (Esashi et al, 2005) and ubiquitination of RAD51, which interfere with the interaction (Luo et al, 2016). Fine‐tuning the protein levels of BRCA2 by DPP9 presents an additional layer of regulation cells applied to ensure efficient repair by HR.

Finally, our results set the ground for future analysis of DPP9 activity in breast cancer and suggest that DPP9 inhibition in combination with Olaparib or radiation presents future potential therapies for patients with breast cancer.

### Materials and Methods

Reagent/ResourceReference or SourceIdentifier or Catalog NumberHeLa wtJusta‐Schuch et al (2016)HeLa DPP9 KDJusta‐Schuch et al (2016)MEF wtGall et al (2013)gki MEF DPP9S729A
Gall et al (2013)Sf9Gibco™Cat#11496015HEK293 Flp‐In T‐REx‐293Thermo Fisher ScientificCat #R78007, RRID:CVCL_U427HEK293 Flp‐In T‐REx‐293 DPP9 KOThis studyHEK293 Flp‐In T‐REx‐293 DPP9 KO‐DPP9‐S WTThis studyHEK293 Flp‐In T‐REx‐293 DPP9 KO‐DPP9‐SS729A
This studyHEK293 Flp‐In T‐REx‐293 DPP9 KO‐DPP9‐L WTThis studyHCT116 cells, stably transfected with pDRGFPKari et al (2019)
Recombinant DNA
pcDNA3 236HSC WT (hBRCA2)Addgene, (Wang et al, 2002)Cat #162463xHA‐DHFR‐Ubiquitin‐BRCA2‐eGFPThis paper3xHA‐DHFR‐Ubiquitin‐BRCA2ΔMP‐FLAG‐eGFPThis paper3xHA‐DHFR‐Ubiquitin‐BRCA21‐1000‐FLAG‐eGFPThis paperpSpCas9(BB)‐2A‐GFPAddgene, Ran et al (2013)Cat #481383xHA‐DHFR‐Ubiquitin‐BRCA21‐1000 ΔMP 1‐1000‐FLAG‐eGFPThis paperFlag tagged DPP9‐SWT in pUC57‐BsaI‐FreeBioCatCustom madeFlag tagged DPP9‐SS729A in pUC57‐BsaI‐FreeBioCatCustom madeFlag tagged DPP9‐SWT in pcDNA5/FRT/TOThis paperFlag tagged DPP9‐SS729A in pcDNA5/FRT/TOThis paperFlag tagged DPP9‐LWT in pcDNA5/FRT/TOThis paperSUMO1‐BRCA2,1‐39_pet11aThis paperSUMO1‐BRCA2ΔMP,3‐39_pet11aThis paperBRCA21‐1000 in pcDNA3.1(+) P2A eGFPGenscriptCustom madeBRCA21‐1000 P2G in pcDNA3.1(+) P2A eGFPGenscriptCustom madeBRCA21‐1000 P2H in pcDNA3.1(+) P2A eGFPGenscriptCustom made
Antibodies
Mouse anti‐BRCA2 (monoclonal), IF/PLA (1:100), WB (1:1,000)R&DCat #MAB2476; RRID:AB_2259370Rabbit anti‐DPP9 (polyclonal), WB (1:500)AbcamCat #AB42080; RRID:AB_731947Goat anti‐DPP9 (polyclonal), IF and PLA (1:20)Justa‐Schuch et al (2016)RRID:AB_2889071Rabbit anti‐Filamin A (polyclonal), WB (1:500)NovusCat #nb‐100‐58812; RRID:AB_877728Mouse anti‐FLAG (monoclonal), WB (1:500)Sigma‐AldrichCat #F1804; RRID:AB_262044Rabbit anti‐GFP (polyclonal), WB (1:200)Santa Cruz BiotechnologyCat #sc‐8334; RRID:AB_641123Rabbit anti‐H2AX‐P (polyclonal), IF (1:500), WB (1:1,000)Cell SignalingCat #9718; RRID:AB_2118009Mouse anti‐H2AX‐P (monoclonal), IF and PLA (1:500)MilliporeCat #05‐636; RRID:AB_309864Mouse anti‐HA, WB (1:1,000)BiolegendCat #901515, RRID:AB_2565334Rabbit anti‐H3 (polyclonal), WB (1:100)Cell SignalingCat #9715; RRID:AB_331563Rabbit anti‐PALB2 (polyclonal), PLA (1:100), WB (1:500)BethylCat #A301‐246A; RRID:AB_890607Rabbit anti‐RAD51 (polyclonal), IF (1:500) WB (1:1,000)AbcamCat #ab63801; RRID:AB_1142428Mouse anti‐Tubulin (monoclonal), WB (1:5,000)Santa CruzCat #sc‐32293; RRID:AB_628412Mouse anti‐Vinculin (monoclonal), WB (1:6,000)Sigma‐AldrichCat #V9131; RRID:AB_477629Anti‐FLAG® M2 Magnetic Beads antibodySigma‐AldrichCat #M8823, RRID:AB_2637089Pierce Anti‐HA magnetic beadsThermo ScientificCat #88836; RRID:AB_2749815Ni‐NTA AgaroseQiagenCat #1018244
Chemicals, enzymes and other reagents
1G244 (in DMSO)AK Scientific, Inc.Cat #Y0432CycloheximideSigma‐AldrichCat #C7698DoxycyclineRatiopharmCat #PZN4314646Lipofectamine P3000Thermo Fisher ScientificCat #L3000015Lipofectamine RNAimaxThermo Fisher ScientificCat #13778150Mayer's HemalumMerckCat #1.09249.0500Mitomycin CSigma‐AldrichCat #M4287NeocarcinostatinSigma‐AldrichCat #N9162Propidium Iodide (PI) solutionSigma‐AldrichCat #P4864MG132Sigma‐AldrichCat #C2211DAPIThermo Fisher ScientificCat #D1306Phalloidin‐iFluor 488AbcamCat #ab176753OlaparibSelleck ChemicalsCat #AZD2281PuromycinSigma‐AldrichCat #P8833AprotininCarl ROTHCat #162.3LeupeptinCarl ROTHCat #N33.4PepstatinCarl ROTHCat #2936.3Phusion DNA polymeraseThermo Fisher ScientificCat #F530SBRCA21‐40 peptideGenscriptMPIGSKERPTFFEIFKTRCNKADLGPISLNWFEELSSEAP, > 85% pureBRCA23‐40 peptideGenscriptIGSKERPTFFEIFKTRCNKADLGPISLNWFEELSSEAP, > 85% pureBRCA21‐20 peptideGenscriptMPIGSKERPTFFEIFKTRCN, > 85% pureBRCA21‐40
P2G peptideGenscriptMGIGSKERPTFFEIFKTRCNKADLGPISLNWFEELSSEAP, > 85% pureBRCA21‐40
E13R peptideGenscriptMPIGSKERPTFFRIFKTRCNKADLGPISLNWFEELSSEAP, > 85% pureBRCA21‐40
I14N peptideGenscriptMPIGSKERPTFFENFKTRCNKADLGPISLNWFEELSSEAP, > 85% pureBRCA21‐40
W31C peptideGenscriptMPIGSKERPTFFEIFKTRCNKADLGPISLNCFEELSSEAP, > 85% pureBRCA21‐40
F32G peptideGenscriptMPIGSKERPTFFEIFKTRCNKADLGPISLNWGEELSSEAP, > 85% pureSLRFLYEG peptideGenscriptSLRFLYEG, > 85% pure
Software
DuoLink softwareSigma‐AldrichRRID:SCR_015574ImageStudio Lite v4.0.21LI‐CORRRID:SCR_013715LSM Image ExaminerZeissRRID:SCR_014344LSM 510 Release Version 4.0 SP2ZeissCeligo Software v2.0NexlecomGraphPad Prism 8GraphPadRRID:SCR_002798XDS Program PackageRRID:SCR_015652RefmacRRID:SCR_014225FlowJo v10.8.1BD BiosciencesRRID:SCR_008520Microsoft ExcelMicrosoftRRID:SCR_016137Empiria Studio v2.2.0.141LI‐CORBiorender
https://biorender.com/
RRID:SCR_018361Fiji

https://imagej.net/software/fiji/

Schindelin et al (2012) doi:10.1038/nmeth.2019
The Human Protein Atlas
https://www.proteinatlas.org/

Other
CellTiter‐Glo® Luminescent Cell Viability AssayPromegaCat # G7571DuoLink in Situ PLA probe Mouse plusSigma‐AldrichCat #DUO92001; RRID:AB_2810939DuoLink in Situ PLA probe Rabbit plusSigma‐AldrichCat #DUO92002; RRID:AB_2810940DuoLink in Situ PLA probe Mouse minusSigma‐AldrichCat #DUO92004; RRID:AB_2713942DuoLink in Situ PLA probe Goat minusSigma‐AldrichCat #DUO92006DuoLink in Situ Detection Reagents RedSigma‐AldrichCat #DUO92008CytoFLEX SBeckman CoulterRRID:SCR_019627Enspire Multimode Plate ReaderPerkinElmerCeligoNexlecomRRID:SCR_018808LI‐COR Odyssey CLXLI‐COR

https://imagej.net/software/fiji/

Schindelin et al (2012) doi:10.1038/nmeth.2019

See Reagents Tools table and Table EV1 for Primers and siRNA.

HeLa DPP9 stable Knock‐Down cells (DPP9 KD) and the corresponding HeLa WT cells (Genscript) (Justa‐Schuch et al, 2016), MEF WT and gki MEF DPP9S729A (Gall et al, 2013), HEK293 DPP9 KO+DPP9‐SWT or DPP9‐SS729A, HEK293 DPP9 KO+DPP9‐LWT (this study), and the corresponding HEK 293 DPP9 WT cell lines were cultured in Dulbecco's modified Eagle's medium supplemented with 10% fetal bovine serum, 2 mM L‐glutamine, 100 U/ml penicillin and 100 μg/ml streptomycin. To maintain the selection pressure on the DPP9 KD cells, 1.5 μg/ml puromycin (Sigma‐Aldrich, Germany) was added to the growth medium. HCT 116 cells, stably expressing pDRGFP were a kind gift from Holger Bastians (Kari et al, 2019). Cells were grown in McCoy's 5A medium supplemented with 10% fetal bovine serum. All cells were grown at 37°C and 5% CO2 and routinely tested for mycoplasma contamination.

For the generation of the HEK Flp‐In™ T‐REx™‐293 DPP9 knockout clone (HEK293 DPP9 KO), guide RNA sequences (CRISPR guide 3 on exon 9, see Table EV1) targeting human DPP9 were cloned into the pSpCas9(BB)‐2A‐GFP (PX458) vector, which was a gift from Feng Zhang (Addgene plasmid # 48138) (Ran et al, 2013). HEK Flp‐In™ T‐REx™‐293 (ThermoFisher Scientific) cells were transfected using PEI. After 24 h, FACS sorting was used to collect GFP‐positive cells. Single cells were seeded into 96‐well plates. Clones were screened using western blot.

For complementation, the Flp‐In™T‐REx™ system (Invitrogen) was used to create stable, inducible cell lines expressing DPP9‐SWT (HEK293 DPP9 KO+DPP9‐SWT), DPP9‐SS729A (HEK293 DPP9 KO+DPP9‐SS729A) or DPP9‐LWT (HEK293 DPP9 KO+DPP9‐LWT) in pcDNA5/FRT/TO. For the selection of positive clones, DMEM complete containing 100 μg/ml hygromycin and 10 μg/ml blasticidin was used. Unless otherwise stated, DPP9 expression was induced by the addition of Dulbecco's modified Eagle's medium supplemented with 10% fetal bovine serum, 2 mM L glutamine, 100 U/ml penicillin, and 100 μg/ml streptomycin containing 1 μg/ml doxycycline for 24 h.

The following plasmids were kind gifts, obtained from Addgene: the pCBASce I endonuclease expression vector (Addgene plasmid # 26477) from Maria Jasin (Pierce et al, 1999), the template pcDNA3 236HSC WT BRCA2 (Addgene plasmid # 16246) from Mien‐Chie Hung (Wang et al, 2002). BRCA21‐3418, BRCA21‐1000, and the corresponding mutants BRCA23‐3418, and BRCA23‐1000 that lack the N‐terminal dipeptide Met1Pro2, were cloned into pcDNA3.1+ vector with a Ubiquitin tag fused at the N‐terminus to provide a defined N‐terminus, as first described by Bachmair and Varshavsky (Bachmair et al, 1986). Silent mutations were introduced at the binding sites of the silencing oligos targeting BRCA2 (1721GAAGAATGCAGGTTTAATA1740) at the sites 1731A‐>T and 1734T‐>A. SUMO1‐BRCA21‐40‐HA‐His constructs were cloned by adding SUMO1 to the N‐terminus of BRCA21‐39‐3HA‐His pET11a (custom‐made from Genscript), using the Gibson Chew Back and Anneal Assembly (CBA) as described in (Torella et al, 2014) Primers are listed in extended view Table EV1. Flag‐tagged DPP9‐SWT, Flag‐tagged DPP9‐SS729A, and DPP9‐LWT were cloned into pcDNA5/FRT/TO (ThermoFisher Scientific #V601020). All plasmids have been verified by sequencing. HeLa cDNA was used as a template for PCR amplification of human PALB2. The PALB2 PCR product was cloned into pENTR 3C using the BamH1 and NotI sites. PALB2 was further cloned from pENTR3C into BaculoDirect, using the LR Clonase (ThermoFisher Scientific). BRCA21‐1000
WT, BRCA21‐1000
P2G, and BRCA21‐1000
P2H were custom synthesized by GenScript.

HEK293 DPP9 KO+DPP9‐SWT cells were treated for 24 h with 300 nM MMC. For induced expression of FLAG‐tagged DPP9‐SWT, 1 μg/ml of Doxycyline was added together with the MMC for 24 h. Control cells were not treated with Doxycycline. Cells were trypsinized and lysed in Lysis buffer (50 mM HEPES pH 7.4, 100 mM NaCl, 0.5% NP40, 5 mM EDTA, 1 mM DTT, PMSF and 10 mM MG132). Following a 30‐min incubation at 4°C, cell lysates were centrifuged at 20,000 g at 4°C for 20 min. Cleared cell lysates were diluted in 50 mM HEPES pH 7.4, 100 mM NaCl, 5 mM EDTA to a final NP‐40 concentration 0.1%. Diluted lysates were incubated with magnetic FLAG‐beads for 3 h at 4°C. Bound proteins were eluted with 0.5 mg/ml FLAG peptide and analyzed by western blotting.

HeLa WT and HeLa DPP9 KD were treated with MMC (300 nM) for 0, 8, or 24 h. Chromatin fractionation was performed essentially as described in (Kari et al, 2016). Briefly, cells were resuspended in lysis buffer (10 mM HEPES pH 7.9, 10 mM KCl, 1.5 mM MgCl2, 0.34 M sucrose, 10% glycerol, 0.1% Triton‐X‐100, 1 mM DTT, and protease inhibitors) and centrifuged at 1,500 g for 5 min. The nuclear pellet was lysed in nuclear lysis buffer (3 mM EDTA, 0.2 mM EGTA, 1 mM DTT, and protease inhibitors) for 30 min on ice. Soluble chromatin fractions were separated by centrifuging at 1,700 g for 5 min. Chromatin fractions were sonicated in a water sonicator (BioRad) for 15 min before loading on SDS–PAGE electrophoresis.

HeLa, MEF, HEK293 DPP9 WT, HEK293 DPP9 KO+DPP9‐SWT/+DPP9‐SS729A, and HEK293 DPP9 KO+DPP9‐LWT cells were grown on coverslips in 24‐well plates, fixed with 4% formaldehyde in PBS for 10 min and permeabilized with 0.2% Triton‐X‐100 in PBS for 5 min. Cells were washed with PBS and blocked with 2% BSA in PBS for 10 min. Cells were incubated with primary antibodies for 90 min at 37°C. Following a PBS wash, cells were incubated for 45 min at room temperature with the respective secondary antibodies. Cells were washed with PBS and water and mounted in fluorescent mounting medium (DAKO) with DAPI. EdU stained cells were treated with EdU (abcam) 30 min prior to DNA damage induction according to the manufacturer's specifications. Cells were analyzed and images were taken using either an LSM 510‐Meta confocal microscope, oil immersion objective 63x/1.3 (Carl Zeiss MicroImaging, Inc), or a Nikon Eclipse Ti2‐E Inverted microscope, Plan Apoλ oil immersion objective 60x NA1.40 WD = 0.13 (Nikon Instruments Inc). Images were processed using LSM Image Browser (Carl Zeiss MicroImaging, Inc) or NIS‐Elements AR 5.02.00 (Nikon Instruments Inc), based on the microscope used, and FIJI for the preparation of figures. RAD51 foci were quantified using the Duolink ImageTool (Sigma‐Aldrich).

Proximity Ligation Assay (PLA) was performed using the DUOLINK In Situ PLA Kit (Sigma‐Aldrich) according to the manufacturer's protocol. Briefly, HeLa cells were grown on coverslips and fixed as described above for immunofluorescence assays. In the case of DPP9 inhibition studies, 10 μM 1G244 was added to the cells for 30 min before fixation. Control cells were mock treated with DMSO. HeLa cells were incubated with primary antibodies for 90 min at 37°C and actin filaments were simultaneously counterstained with CytoPainter Phalloidin‐iFluor 488 Reagent (Abcam ‐ #ab176753). Coverslips were washed with PBS and treated with PLA reagents. Control coverslips (NgtCtrl) were treated with one primary antibody to estimate background staining in each experiment. Cells were mounted in DAKO with DAPI fluorescent mounting medium and analyzed using an LSM 510‐Meta confocal microscope, oil immersion objective 63x/1.3 (Carl Zeiss MicroImaging, Inc) or a Nikon Eclipse Ti2‐E Inverted microscope, Plan Apoλ oil immersion objective 60x NA1.40 WD = 0.13 (Nikon Instruments Inc). Images were processed using LSM Image Browser (Carl Zeiss MicroImaging, Inc) or NIS‐Elements AR 5.02.00 (Nikon Instruments Inc), based on the microscope used and subsequently analyzed using the Duolink ImageTool (Sigma).

Cells were seeded in 96‐well White Microplates (Perkin Elmer cat# 6005680) and treated with different concentrations of Olaparib for 96 h. For the MMC assays, cells were first incubated for 18–24 h to allow cell attachment. Next, different MMC concentrations were added and cells were analyzed 72 h later. Control cells were treated with DMSO. Cell viability was measured using the CellTiter‐Glo®Luminescence Cell Viability Assay (Promega, cat# Cat G7571). CellTiter‐Glo®Reagent was added in a 1:1 ratio to the cell culture medium. The microplate was shaken on an orbital shaker for 10 min for induction of cell lysis. Subsequently, the luciferase signal was measured on a LuminometerDLReady™ Centro LB 960 reader. Each experiment was performed three times, in triplicates or quadruplicates.

To calculate the respective surviving fractions (SF) after γ radiation (0, 1, 2, 4, 6, and 8G), a standard colony‐forming assay was performed, as previously described (Rave‐Fränk et al, 2007). Briefly, cells were exposed to γ‐ radiation and incubated for 7 days, fixed with 70% ethanol, and stained with Mayer's hemalum (Cat#1.09249.0500, Merck). Nonirradiated cultures were used for normalization. Colonies with > 50 cells were scored as survivors. Three biological replicates were performed, each containing three technical triplicates, and medians were calculated. To validate statistical differences between control and treatment groups, analysis of variance (ANOVA: two‐factor with replication) was performed using Microsoft Excel software (version 2016 MSO). P‐values < 0.05 were considered significant.

To measure HR frequency, HCT116 cells expressing stably integrated pHPRT‐pDR‐GFP reporter plasmid were transfected with the indicated siRNAs. Following 24 h of siRNA transfection, cells were either mock‐transfected or transfected with 2 μg pCBASceI (Addgene) to induce DSBs. 48 h after transfection, the expression of GFP was quantified by flow cytometry using a CytoFLEX S (Beckman Coulter). Cell debris and dead cells were identified using propidium iodide staining (Sigma‐Aldrich P4864). The data were analyzed using the FlowJo software (BS Bioscience). The HR frequency was determined as the percentage of GFP‐positive cells in the total number of alive cells.

Human recombinant DPP9 was expressed from Sf9 cells and purified essentially as described previously (Pilla et al, 2012). Human recombinant His‐tagged PALB2 was expressed from Sf9 cells. Cells were resuspended in PALB2 lysis buffer (25 mM Hepes pH7.5, 200 mM NaCl, 5 mM imidazole, 10% glycerol, 0.2% Triton, 5 mM ß‐mercaptoethanol, 2 mM PMSF, 10 μg/ml aprotinin, and 10 μg/ml leupeptin), and lysed by homogenization and sonication. 15 U/ml Benzonase and 1 mM MgCl2 were added to the cell lysate. Following a 1 h incubation at 4°C, cell lysate was centrifuged at 50,000 g for 90 min at 4°C. The supernatant was incubated was bound to Ni‐NTA Agarose (Qiagen) for 2 h. Following extensive washing, PALB2 was eluted with Elution Buffer (25 mM Hepes pH7.5, 50 mM NaCl, 300 mM imidazole, 5 mM ß‐mercaptoethanol) and further purified on a Superdex 200 (GE Healthcare). For purification of N‐terminally His‐HA‐tagged fragments corresponding to the N‐terminus of BRCA2 (BRCA2 1‐40HA‐His), SUMO1BRCA21‐40HA‐His was transformed into BL21 (DE3 codon+) cells and induced by 1 mM IPTG for 2 h at 30°C. Cells were lysed in 25 ml Lysis Buffer (50 mM NaH2PO4, 300 mM NaCl, 10 mM Imidazole pH 8.0, 1 mM ß‐Mercaptoethanol and protease inhibitors: 1 μg/ml Aprotinin, 1 μg/ml Leupeptin and 1 μg/ml Pepstatin), by sonification. The 100,000 g supernatant was bound to Ni‐NTA Agarose (Qiagen) for 2 h. Beads were thoroughly washed with Lysis buffer lacking protease inhibitors and incubated with 20 μg SENP overnight to release SUMO1. Following extensive washing, BRCA21‐40HA‐His was eluted with Elution buffer (50 mM Na‐Phosphate pH 8.0, 300 mM NaCl, 250 mM Imidazole, 1 mM ß‐Mercaptoethanol).

Purified BRCA21‐40HA‐His (2 μg) was immobilized on magnetic anti‐HA beads (Pierce Cat# 88836, Thermo Scientific) in a binding buffer containing (25 mM Tris–pH 7.5, 150 mM NaCl, 1 mM EDTA, 0.01% (v/v) IGEPAL, 1 mM DTT), and incubated overnight at 4°C with recombinant DPP9 (2 μg), and/or recombinant PALB2 (9 μg). Where stated, DPP9 was preincubated for 1 h with the following inhibitors: 100 μM 1G244 or 110 μM SLRFYEG. Bound proteins were eluted with the HA peptides and analyzed by western blot.

Direct interactions between recombinant DPP9 and BRCA2‐derived N‐terminal peptides (1–40 and 3–40 amino acids) were analyzed employing surface plasmon resonance (SPR). The surface of a NiHC1000m sensorchip (Xantec Bioanalytics, Duesseldorf, Germany) was rinsed with 0.5 M EDTA (pH 8.0) solution and afterwards conditioned in SPR running buffer (20 mM HEPES pH 7.4, 150 mM NaCl, 50 μM EDTA, 0.005% TWEEN‐20). The chip surface was loaded with Ni2+‐ions, injecting 300 mM NiSO4 solution. A 200 nM solution of His‐tagged DPP9 (ligand) was injected over the chip surface and immobilized to a surface density of 800–1,000 μRIU (Refractive index units; response observed), at a flow rate of 15 μl/min. A serial dilution of the analyte (BRCA2 peptides) diluted in SPR running buffer was injected, and the association was followed for 4.5 min. Dissociation of the analyte was monitored for 7 min. For each analyte concentration, the surface was regenerated, injecting 0.5 M EDTA pH 8.0 and fresh ligand was immobilized as described above (one immobilization/regeneration cycle for each analyte concentration). All binding experiments were performed on a Reichert SR 7500 DC biosensor at 20°C and a flow rate of 40 μl/min. Obtained data were analyzed with Scrubber2.0c. Raw data were double referenced (reference channel, specificity check, and buffer blanks/injections). Equilibrium‐binding response data were exported and plotted for further analysis with GraphPad Prism 6.0.

The assay was performed as described in (Justa‐Schuch et al, 2016) by incubating 50 μM of the BRCA21‐40 peptide with 125 nM DPP9. To test for inhibition, a 10 μM peptide inhibitor (SLRFLYEG) was added. Reactions were stopped after 6 h by dilution and acidification in aqueous 0.1% formic acid and 2% acetonitrile (1/20,000, v:v). In addition, bovine insulin was added to the dilution buffer at a concentration of 1 pmol/μl to prevent analyte loss due to adsorption. The resulting samples were analyzed on a nanoLC425 nanoflow chromatography system coupled to a TripleToF 5600+ Plus mass spectrometer of QqToF geometry (both AB SCIEX). 5 μl of the sample was preconcentrated on a self‐packed Reversed Phase‐C18 precolumn (Reprosil C18‐AQ, Pore Size 120 Å, Particle Size 5 μm, 4 cm length, 0.15 cm I.D., Dr. Maisch) and separated on a self‐packed Reverse Phase‐C18 microcolumn (Reprosil C18‐AQ, 120 Å, 3 μm, 15 cm, 0.075 cm) using a 15 min linear gradient (5 to 50% acetonitrile, 0.1% formic acid modifier, flow rate 300 nl/min, column temperature 50°C) followed by a 5 min high organic cleaning step and a 15 min column re‐equilibration. The eluent was introduced to the mass spectrometer using a Nanospray III ion source with Desolvation Chamber Interface (AB SCIEX) via a commercial Fused Silica tip (FS360‐20‐10‐N‐C15, New Objective) at a spray voltage of 2.2 kV, a sheath gas setting of 15 and an interface heater temperature of 150°C. The MS acquisition cycle consisted of a 500 ms TOF MS survey scan that was used for profiling substrate and product concentrations followed by the data‐dependent triggering of up to 5,100 ms TOF product ion spectra to confirm the identity of detected analytes. Data analysis was performed using the Analyst TF 1.7 and the PeakView 2.1 softwares (AB SCIEX). Analyses were performed in quadruplicate.

75 ng of purified DPP9 was incubated with 200 μM of a test peptide (Syk1‐31, BRCA21‐20, BRCA21‐40, BRCA2P2G
1‐40, BRCA2E13R
1‐40, BRCA2I14N
1‐40, BRCA2W31C
1‐40, BRCA2F32G
1‐40) for 5 min at 24°C in 20 mM HEPES/KOH pH 7.3, 110 mM potassium acetate, 2 mM magnesium acetate, 1 mM EGTA, 0.02% Tween‐20 supplemented with 1 mM DTT. The peptide‐enzyme mixture was added to 100 μM H‐Gly‐Pro‐7‐amino‐4‐methylcoumarin (GP‐AMC) substrate in a final reaction volume of 20 μl. Fluorescence was analyzed using an EnSpire microplate fluorimeter (Perkin Elmer) with 360‐nm (excitation) and 460‐nm (emission) filters. Percentage inhibition was normalized to the corresponding mock‐treated DPP9 controls. All peptides (> 85% purity) were purchased from GenScript.

Purification, crystallization, and structure solution of DPP8 and DPP9 were performed as published in our previous report (Ross et al, 2018). Briefly, the DPP8 isoform 1 (Uni‐ProtKB Q6V1X1) and DPP9 isoform 2 (Uni‐ProtKB Q86TI2–2) were expressed in Spodoptera frugiperda cells and purified. The crystallization was performed in both cases using the hanging drop method with 0.46 M Na‐citrate pH 6.75 as precipitant solution and 10 mg/ml of DPP8 at 4°C. DPP9 crystallization occurred at 20°C, using 20 mg/ml of protein and 10% PEG 8000, 25% glycerol, 0.16 M Calcium acetate, and 0.08 M cacodylate pH 6.25 as precipitant solution. Crystals of space group C2221 (DPP8) and P1211 (DPP9) were soaked with BRCA21‐40 peptide by performing dropping using a pico‐dropper (Ross et al, 2018). Data were collected at SLS‐X10SA and solved to 3.2 Å and 3.0 Å, respectively, by molecular replacement using a DPP8 structure (PDB: 6EOO). Structure solution and refinement were performed as described previously (Ross et al, 2018). Data processing was performed with XDS (Kabsch, 2010); Molecular Replacement with Phaser (McCoy et al, 2007); data refinement with Refmac5 (Murshudov et al, 1997). Data processing, statistics, and refinement values are summarized in Appendix Table S1.

HeLa WT, DPP9 KD, HEK293 DPP9 WT, HEK293 DPP9 KO+DPP9‐SWT / + DPP9‐SS729A cells were treated with 300 nM MMC for 24 h. The CHX chase was started by the addition of 100 μg/ml CHX. Where stated, 100 μM of MG132 was added to the cells 30 min before the addition of the CHX. Samples were harvested at the indicated time points directly in Laemmli Sample Buffer. Immunoblotting and incubation with primary antibodies were performed according to standard protocols. Secondary fluorophore‐coupled antibodies were applied. Signals were developed in the Odyssey Sa Infrared Imaging System (LI‐COR) and analyzed with the ImageStudio (version 4.0.21, LI‐COR) software.

To obtain a majority population of cells in S‐phase, cells were synchronized using a double thymidine block. Briefly, HeLa WT & HeLa DPP9 KD cells were treated with 2 mM of thymidine. After the 18 h, the cells were washed, and replenished with fresh DMEM for 9 h, prior to the second thymidine block (2 mM) for another 15 h. For detection of the cell cycle stages, cells were trypsinized and collected at 0, 2, 3, 4, 5, 6, 8, 24, and 26 h after release. The cell pellet was washed once in PBS and fixed in 70% Ethanol. Cells pellets were resuspended in PBS supplemented with 0.25% Triton‐X‐100 and incubated on ice for 15 min. The residual Triton was washed off with PBS and the cells were incubated in PBS supplemented with 100 μg/ml RNAse A and 50 μg/ml propidium iodide, in the dark, overnight at 4°C. The cells were sorted in a Cytoflex S cell sorter and analyzed using the FlowJo Software.

Cells were seeded on 96‐well, black/clear, tissue culture‐treated plates (Corning Falcon cat# 353219). Following 18 h incubation at 37°C, 5% CO2, cells were treated with 250 ng/ml NCS for 30 min. NCS was then removed and replaced with a fresh medium to allow cells to recover. Cells were fixed at the indicated time points in 4% formaldehyde in PBS for 20 min and permeabilized with 0.2% Triton‐X‐100 in PBS for 20 min. Cells were washed with PBS, blocked with 10% FBS in PBS for 30 min, and incubated with primary antibodies for 2 h. Following three PBS wash steps, cells were incubated for 60 min with secondary antibodies and 1:5,000 DAPI. γH2AX signal intensities were measured on a Celigo 4 Channel Imaging Cytometer (Nexlecom Bioscience) and quantified using the Celigo Software. Images were taken using a Nikon Eclipse Ti2‐E Inverted microscope, Plan Apoλ oil immersion objective 100x NA1.45 WD = 0.13 (Nikon Instruments Inc), and processed using NIS‐Elements AR 5.02.00 (Nikon Instruments Inc), and FIJI (ImageJ) for the preparation of figures. At least three biological replicates were analyzed, each with technical duplicates, and the means calculated.

Amino acid conservation was assessed by aligning the human BRCA2 orthologs from Ensembl. The sequences of BRCA2 orthologs from 108 placental mammals were aligned with the ClustalW algorithm (Thompson et al, 1994). The conservation was illustrated as a logo plot with web logo (Schneider & Stephens, 1990; Crooks et al, 2004). This plot illustrates the alignment as letters, where the height of each letter is in proportion to the observed frequency of the corresponding amino acid, which is measured in bits.

Graphs were generated using the PRISM8 software, statistical analysis was carried out by unpaired or paired two‐tailed t‐test, one‐way or two‐way ANOVA. ImageStudio software was used for the quantification of western blots, which were visualized with LI‐COR. Bands were quantified relative to the respective loading control. Mean and SEM values were calculated by the PRISM software and represented together with SEM error bars. For Colony Formation and Cell Viability assays, measurements from three biological replicates each with technical triplicates per data point are represented as Mean ± SEM. For microscopy images, dots from PLAs or dots from foci formation assays were quantified using DUO‐LINK software.

### Reagents and Tools table

Reagent/ResourceReference or SourceIdentifier or Catalog NumberHeLa wtJusta‐Schuch et al (2016)HeLa DPP9 KDJusta‐Schuch et al (2016)MEF wtGall et al (2013)gki MEF DPP9S729A
Gall et al (2013)Sf9Gibco™Cat#11496015HEK293 Flp‐In T‐REx‐293Thermo Fisher ScientificCat #R78007, RRID:CVCL_U427HEK293 Flp‐In T‐REx‐293 DPP9 KOThis studyHEK293 Flp‐In T‐REx‐293 DPP9 KO‐DPP9‐S WTThis studyHEK293 Flp‐In T‐REx‐293 DPP9 KO‐DPP9‐SS729A
This studyHEK293 Flp‐In T‐REx‐293 DPP9 KO‐DPP9‐L WTThis studyHCT116 cells, stably transfected with pDRGFPKari et al (2019)
Recombinant DNA
pcDNA3 236HSC WT (hBRCA2)Addgene, (Wang et al, 2002)Cat #162463xHA‐DHFR‐Ubiquitin‐BRCA2‐eGFPThis paper3xHA‐DHFR‐Ubiquitin‐BRCA2ΔMP‐FLAG‐eGFPThis paper3xHA‐DHFR‐Ubiquitin‐BRCA21‐1000‐FLAG‐eGFPThis paperpSpCas9(BB)‐2A‐GFPAddgene, Ran et al (2013)Cat #481383xHA‐DHFR‐Ubiquitin‐BRCA21‐1000 ΔMP 1‐1000‐FLAG‐eGFPThis paperFlag tagged DPP9‐SWT in pUC57‐BsaI‐FreeBioCatCustom madeFlag tagged DPP9‐SS729A in pUC57‐BsaI‐FreeBioCatCustom madeFlag tagged DPP9‐SWT in pcDNA5/FRT/TOThis paperFlag tagged DPP9‐SS729A in pcDNA5/FRT/TOThis paperFlag tagged DPP9‐LWT in pcDNA5/FRT/TOThis paperSUMO1‐BRCA2,1‐39_pet11aThis paperSUMO1‐BRCA2ΔMP,3‐39_pet11aThis paperBRCA21‐1000 in pcDNA3.1(+) P2A eGFPGenscriptCustom madeBRCA21‐1000 P2G in pcDNA3.1(+) P2A eGFPGenscriptCustom madeBRCA21‐1000 P2H in pcDNA3.1(+) P2A eGFPGenscriptCustom made
Antibodies
Mouse anti‐BRCA2 (monoclonal), IF/PLA (1:100), WB (1:1,000)R&DCat #MAB2476; RRID:AB_2259370Rabbit anti‐DPP9 (polyclonal), WB (1:500)AbcamCat #AB42080; RRID:AB_731947Goat anti‐DPP9 (polyclonal), IF and PLA (1:20)Justa‐Schuch et al (2016)RRID:AB_2889071Rabbit anti‐Filamin A (polyclonal), WB (1:500)NovusCat #nb‐100‐58812; RRID:AB_877728Mouse anti‐FLAG (monoclonal), WB (1:500)Sigma‐AldrichCat #F1804; RRID:AB_262044Rabbit anti‐GFP (polyclonal), WB (1:200)Santa Cruz BiotechnologyCat #sc‐8334; RRID:AB_641123Rabbit anti‐H2AX‐P (polyclonal), IF (1:500), WB (1:1,000)Cell SignalingCat #9718; RRID:AB_2118009Mouse anti‐H2AX‐P (monoclonal), IF and PLA (1:500)MilliporeCat #05‐636; RRID:AB_309864Mouse anti‐HA, WB (1:1,000)BiolegendCat #901515, RRID:AB_2565334Rabbit anti‐H3 (polyclonal), WB (1:100)Cell SignalingCat #9715; RRID:AB_331563Rabbit anti‐PALB2 (polyclonal), PLA (1:100), WB (1:500)BethylCat #A301‐246A; RRID:AB_890607Rabbit anti‐RAD51 (polyclonal), IF (1:500) WB (1:1,000)AbcamCat #ab63801; RRID:AB_1142428Mouse anti‐Tubulin (monoclonal), WB (1:5,000)Santa CruzCat #sc‐32293; RRID:AB_628412Mouse anti‐Vinculin (monoclonal), WB (1:6,000)Sigma‐AldrichCat #V9131; RRID:AB_477629Anti‐FLAG® M2 Magnetic Beads antibodySigma‐AldrichCat #M8823, RRID:AB_2637089Pierce Anti‐HA magnetic beadsThermo ScientificCat #88836; RRID:AB_2749815Ni‐NTA AgaroseQiagenCat #1018244
Chemicals, enzymes and other reagents
1G244 (in DMSO)AK Scientific, Inc.Cat #Y0432CycloheximideSigma‐AldrichCat #C7698DoxycyclineRatiopharmCat #PZN4314646Lipofectamine P3000Thermo Fisher ScientificCat #L3000015Lipofectamine RNAimaxThermo Fisher ScientificCat #13778150Mayer's HemalumMerckCat #1.09249.0500Mitomycin CSigma‐AldrichCat #M4287NeocarcinostatinSigma‐AldrichCat #N9162Propidium Iodide (PI) solutionSigma‐AldrichCat #P4864MG132Sigma‐AldrichCat #C2211DAPIThermo Fisher ScientificCat #D1306Phalloidin‐iFluor 488AbcamCat #ab176753OlaparibSelleck ChemicalsCat #AZD2281PuromycinSigma‐AldrichCat #P8833AprotininCarl ROTHCat #162.3LeupeptinCarl ROTHCat #N33.4PepstatinCarl ROTHCat #2936.3Phusion DNA polymeraseThermo Fisher ScientificCat #F530SBRCA21‐40 peptideGenscriptMPIGSKERPTFFEIFKTRCNKADLGPISLNWFEELSSEAP, > 85% pureBRCA23‐40 peptideGenscriptIGSKERPTFFEIFKTRCNKADLGPISLNWFEELSSEAP, > 85% pureBRCA21‐20 peptideGenscriptMPIGSKERPTFFEIFKTRCN, > 85% pureBRCA21‐40
P2G peptideGenscriptMGIGSKERPTFFEIFKTRCNKADLGPISLNWFEELSSEAP, > 85% pureBRCA21‐40
E13R peptideGenscriptMPIGSKERPTFFRIFKTRCNKADLGPISLNWFEELSSEAP, > 85% pureBRCA21‐40
I14N peptideGenscriptMPIGSKERPTFFENFKTRCNKADLGPISLNWFEELSSEAP, > 85% pureBRCA21‐40
W31C peptideGenscriptMPIGSKERPTFFEIFKTRCNKADLGPISLNCFEELSSEAP, > 85% pureBRCA21‐40
F32G peptideGenscriptMPIGSKERPTFFEIFKTRCNKADLGPISLNWGEELSSEAP, > 85% pureSLRFLYEG peptideGenscriptSLRFLYEG, > 85% pure
Software
DuoLink softwareSigma‐AldrichRRID:SCR_015574ImageStudio Lite v4.0.21LI‐CORRRID:SCR_013715LSM Image ExaminerZeissRRID:SCR_014344LSM 510 Release Version 4.0 SP2ZeissCeligo Software v2.0NexlecomGraphPad Prism 8GraphPadRRID:SCR_002798XDS Program PackageRRID:SCR_015652RefmacRRID:SCR_014225FlowJo v10.8.1BD BiosciencesRRID:SCR_008520Microsoft ExcelMicrosoftRRID:SCR_016137Empiria Studio v2.2.0.141LI‐CORBiorender
https://biorender.com/
RRID:SCR_018361Fiji

https://imagej.net/software/fiji/

Schindelin et al (2012) doi:10.1038/nmeth.2019
The Human Protein Atlas
https://www.proteinatlas.org/

Other
CellTiter‐Glo® Luminescent Cell Viability AssayPromegaCat # G7571DuoLink in Situ PLA probe Mouse plusSigma‐AldrichCat #DUO92001; RRID:AB_2810939DuoLink in Situ PLA probe Rabbit plusSigma‐AldrichCat #DUO92002; RRID:AB_2810940DuoLink in Situ PLA probe Mouse minusSigma‐AldrichCat #DUO92004; RRID:AB_2713942DuoLink in Situ PLA probe Goat minusSigma‐AldrichCat #DUO92006DuoLink in Situ Detection Reagents RedSigma‐AldrichCat #DUO92008CytoFLEX SBeckman CoulterRRID:SCR_019627Enspire Multimode Plate ReaderPerkinElmerCeligoNexlecomRRID:SCR_018808LI‐COR Odyssey CLXLI‐COR

https://imagej.net/software/fiji/

Schindelin et al (2012) doi:10.1038/nmeth.2019

### Methods and Protocols

See Reagents Tools table and Table EV1 for Primers and siRNA.

HeLa DPP9 stable Knock‐Down cells (DPP9 KD) and the corresponding HeLa WT cells (Genscript) (Justa‐Schuch et al, 2016), MEF WT and gki MEF DPP9S729A (Gall et al, 2013), HEK293 DPP9 KO+DPP9‐SWT or DPP9‐SS729A, HEK293 DPP9 KO+DPP9‐LWT (this study), and the corresponding HEK 293 DPP9 WT cell lines were cultured in Dulbecco's modified Eagle's medium supplemented with 10% fetal bovine serum, 2 mM L‐glutamine, 100 U/ml penicillin and 100 μg/ml streptomycin. To maintain the selection pressure on the DPP9 KD cells, 1.5 μg/ml puromycin (Sigma‐Aldrich, Germany) was added to the growth medium. HCT 116 cells, stably expressing pDRGFP were a kind gift from Holger Bastians (Kari et al, 2019). Cells were grown in McCoy's 5A medium supplemented with 10% fetal bovine serum. All cells were grown at 37°C and 5% CO2 and routinely tested for mycoplasma contamination.

For the generation of the HEK Flp‐In™ T‐REx™‐293 DPP9 knockout clone (HEK293 DPP9 KO), guide RNA sequences (CRISPR guide 3 on exon 9, see Table EV1) targeting human DPP9 were cloned into the pSpCas9(BB)‐2A‐GFP (PX458) vector, which was a gift from Feng Zhang (Addgene plasmid # 48138) (Ran et al, 2013). HEK Flp‐In™ T‐REx™‐293 (ThermoFisher Scientific) cells were transfected using PEI. After 24 h, FACS sorting was used to collect GFP‐positive cells. Single cells were seeded into 96‐well plates. Clones were screened using western blot.

For complementation, the Flp‐In™T‐REx™ system (Invitrogen) was used to create stable, inducible cell lines expressing DPP9‐SWT (HEK293 DPP9 KO+DPP9‐SWT), DPP9‐SS729A (HEK293 DPP9 KO+DPP9‐SS729A) or DPP9‐LWT (HEK293 DPP9 KO+DPP9‐LWT) in pcDNA5/FRT/TO. For the selection of positive clones, DMEM complete containing 100 μg/ml hygromycin and 10 μg/ml blasticidin was used. Unless otherwise stated, DPP9 expression was induced by the addition of Dulbecco's modified Eagle's medium supplemented with 10% fetal bovine serum, 2 mM L glutamine, 100 U/ml penicillin, and 100 μg/ml streptomycin containing 1 μg/ml doxycycline for 24 h.

The following plasmids were kind gifts, obtained from Addgene: the pCBASce I endonuclease expression vector (Addgene plasmid # 26477) from Maria Jasin (Pierce et al, 1999), the template pcDNA3 236HSC WT BRCA2 (Addgene plasmid # 16246) from Mien‐Chie Hung (Wang et al, 2002). BRCA21‐3418, BRCA21‐1000, and the corresponding mutants BRCA23‐3418, and BRCA23‐1000 that lack the N‐terminal dipeptide Met1Pro2, were cloned into pcDNA3.1+ vector with a Ubiquitin tag fused at the N‐terminus to provide a defined N‐terminus, as first described by Bachmair and Varshavsky (Bachmair et al, 1986). Silent mutations were introduced at the binding sites of the silencing oligos targeting BRCA2 (1721GAAGAATGCAGGTTTAATA1740) at the sites 1731A‐>T and 1734T‐>A. SUMO1‐BRCA21‐40‐HA‐His constructs were cloned by adding SUMO1 to the N‐terminus of BRCA21‐39‐3HA‐His pET11a (custom‐made from Genscript), using the Gibson Chew Back and Anneal Assembly (CBA) as described in (Torella et al, 2014) Primers are listed in extended view Table EV1. Flag‐tagged DPP9‐SWT, Flag‐tagged DPP9‐SS729A, and DPP9‐LWT were cloned into pcDNA5/FRT/TO (ThermoFisher Scientific #V601020). All plasmids have been verified by sequencing. HeLa cDNA was used as a template for PCR amplification of human PALB2. The PALB2 PCR product was cloned into pENTR 3C using the BamH1 and NotI sites. PALB2 was further cloned from pENTR3C into BaculoDirect, using the LR Clonase (ThermoFisher Scientific). BRCA21‐1000
WT, BRCA21‐1000
P2G, and BRCA21‐1000
P2H were custom synthesized by GenScript.

HEK293 DPP9 KO+DPP9‐SWT cells were treated for 24 h with 300 nM MMC. For induced expression of FLAG‐tagged DPP9‐SWT, 1 μg/ml of Doxycyline was added together with the MMC for 24 h. Control cells were not treated with Doxycycline. Cells were trypsinized and lysed in Lysis buffer (50 mM HEPES pH 7.4, 100 mM NaCl, 0.5% NP40, 5 mM EDTA, 1 mM DTT, PMSF and 10 mM MG132). Following a 30‐min incubation at 4°C, cell lysates were centrifuged at 20,000 g at 4°C for 20 min. Cleared cell lysates were diluted in 50 mM HEPES pH 7.4, 100 mM NaCl, 5 mM EDTA to a final NP‐40 concentration 0.1%. Diluted lysates were incubated with magnetic FLAG‐beads for 3 h at 4°C. Bound proteins were eluted with 0.5 mg/ml FLAG peptide and analyzed by western blotting.

HeLa WT and HeLa DPP9 KD were treated with MMC (300 nM) for 0, 8, or 24 h. Chromatin fractionation was performed essentially as described in (Kari et al, 2016). Briefly, cells were resuspended in lysis buffer (10 mM HEPES pH 7.9, 10 mM KCl, 1.5 mM MgCl2, 0.34 M sucrose, 10% glycerol, 0.1% Triton‐X‐100, 1 mM DTT, and protease inhibitors) and centrifuged at 1,500 g for 5 min. The nuclear pellet was lysed in nuclear lysis buffer (3 mM EDTA, 0.2 mM EGTA, 1 mM DTT, and protease inhibitors) for 30 min on ice. Soluble chromatin fractions were separated by centrifuging at 1,700 g for 5 min. Chromatin fractions were sonicated in a water sonicator (BioRad) for 15 min before loading on SDS–PAGE electrophoresis.

HeLa, MEF, HEK293 DPP9 WT, HEK293 DPP9 KO+DPP9‐SWT/+DPP9‐SS729A, and HEK293 DPP9 KO+DPP9‐LWT cells were grown on coverslips in 24‐well plates, fixed with 4% formaldehyde in PBS for 10 min and permeabilized with 0.2% Triton‐X‐100 in PBS for 5 min. Cells were washed with PBS and blocked with 2% BSA in PBS for 10 min. Cells were incubated with primary antibodies for 90 min at 37°C. Following a PBS wash, cells were incubated for 45 min at room temperature with the respective secondary antibodies. Cells were washed with PBS and water and mounted in fluorescent mounting medium (DAKO) with DAPI. EdU stained cells were treated with EdU (abcam) 30 min prior to DNA damage induction according to the manufacturer's specifications. Cells were analyzed and images were taken using either an LSM 510‐Meta confocal microscope, oil immersion objective 63x/1.3 (Carl Zeiss MicroImaging, Inc), or a Nikon Eclipse Ti2‐E Inverted microscope, Plan Apoλ oil immersion objective 60x NA1.40 WD = 0.13 (Nikon Instruments Inc). Images were processed using LSM Image Browser (Carl Zeiss MicroImaging, Inc) or NIS‐Elements AR 5.02.00 (Nikon Instruments Inc), based on the microscope used, and FIJI for the preparation of figures. RAD51 foci were quantified using the Duolink ImageTool (Sigma‐Aldrich).

Proximity Ligation Assay (PLA) was performed using the DUOLINK In Situ PLA Kit (Sigma‐Aldrich) according to the manufacturer's protocol. Briefly, HeLa cells were grown on coverslips and fixed as described above for immunofluorescence assays. In the case of DPP9 inhibition studies, 10 μM 1G244 was added to the cells for 30 min before fixation. Control cells were mock treated with DMSO. HeLa cells were incubated with primary antibodies for 90 min at 37°C and actin filaments were simultaneously counterstained with CytoPainter Phalloidin‐iFluor 488 Reagent (Abcam ‐ #ab176753). Coverslips were washed with PBS and treated with PLA reagents. Control coverslips (NgtCtrl) were treated with one primary antibody to estimate background staining in each experiment. Cells were mounted in DAKO with DAPI fluorescent mounting medium and analyzed using an LSM 510‐Meta confocal microscope, oil immersion objective 63x/1.3 (Carl Zeiss MicroImaging, Inc) or a Nikon Eclipse Ti2‐E Inverted microscope, Plan Apoλ oil immersion objective 60x NA1.40 WD = 0.13 (Nikon Instruments Inc). Images were processed using LSM Image Browser (Carl Zeiss MicroImaging, Inc) or NIS‐Elements AR 5.02.00 (Nikon Instruments Inc), based on the microscope used and subsequently analyzed using the Duolink ImageTool (Sigma).

Cells were seeded in 96‐well White Microplates (Perkin Elmer cat# 6005680) and treated with different concentrations of Olaparib for 96 h. For the MMC assays, cells were first incubated for 18–24 h to allow cell attachment. Next, different MMC concentrations were added and cells were analyzed 72 h later. Control cells were treated with DMSO. Cell viability was measured using the CellTiter‐Glo®Luminescence Cell Viability Assay (Promega, cat# Cat G7571). CellTiter‐Glo®Reagent was added in a 1:1 ratio to the cell culture medium. The microplate was shaken on an orbital shaker for 10 min for induction of cell lysis. Subsequently, the luciferase signal was measured on a LuminometerDLReady™ Centro LB 960 reader. Each experiment was performed three times, in triplicates or quadruplicates.

To calculate the respective surviving fractions (SF) after γ radiation (0, 1, 2, 4, 6, and 8G), a standard colony‐forming assay was performed, as previously described (Rave‐Fränk et al, 2007). Briefly, cells were exposed to γ‐ radiation and incubated for 7 days, fixed with 70% ethanol, and stained with Mayer's hemalum (Cat#1.09249.0500, Merck). Nonirradiated cultures were used for normalization. Colonies with > 50 cells were scored as survivors. Three biological replicates were performed, each containing three technical triplicates, and medians were calculated. To validate statistical differences between control and treatment groups, analysis of variance (ANOVA: two‐factor with replication) was performed using Microsoft Excel software (version 2016 MSO). P‐values < 0.05 were considered significant.

To measure HR frequency, HCT116 cells expressing stably integrated pHPRT‐pDR‐GFP reporter plasmid were transfected with the indicated siRNAs. Following 24 h of siRNA transfection, cells were either mock‐transfected or transfected with 2 μg pCBASceI (Addgene) to induce DSBs. 48 h after transfection, the expression of GFP was quantified by flow cytometry using a CytoFLEX S (Beckman Coulter). Cell debris and dead cells were identified using propidium iodide staining (Sigma‐Aldrich P4864). The data were analyzed using the FlowJo software (BS Bioscience). The HR frequency was determined as the percentage of GFP‐positive cells in the total number of alive cells.

Human recombinant DPP9 was expressed from Sf9 cells and purified essentially as described previously (Pilla et al, 2012). Human recombinant His‐tagged PALB2 was expressed from Sf9 cells. Cells were resuspended in PALB2 lysis buffer (25 mM Hepes pH7.5, 200 mM NaCl, 5 mM imidazole, 10% glycerol, 0.2% Triton, 5 mM ß‐mercaptoethanol, 2 mM PMSF, 10 μg/ml aprotinin, and 10 μg/ml leupeptin), and lysed by homogenization and sonication. 15 U/ml Benzonase and 1 mM MgCl2 were added to the cell lysate. Following a 1 h incubation at 4°C, cell lysate was centrifuged at 50,000 g for 90 min at 4°C. The supernatant was incubated was bound to Ni‐NTA Agarose (Qiagen) for 2 h. Following extensive washing, PALB2 was eluted with Elution Buffer (25 mM Hepes pH7.5, 50 mM NaCl, 300 mM imidazole, 5 mM ß‐mercaptoethanol) and further purified on a Superdex 200 (GE Healthcare). For purification of N‐terminally His‐HA‐tagged fragments corresponding to the N‐terminus of BRCA2 (BRCA2 1‐40HA‐His), SUMO1BRCA21‐40HA‐His was transformed into BL21 (DE3 codon+) cells and induced by 1 mM IPTG for 2 h at 30°C. Cells were lysed in 25 ml Lysis Buffer (50 mM NaH2PO4, 300 mM NaCl, 10 mM Imidazole pH 8.0, 1 mM ß‐Mercaptoethanol and protease inhibitors: 1 μg/ml Aprotinin, 1 μg/ml Leupeptin and 1 μg/ml Pepstatin), by sonification. The 100,000 g supernatant was bound to Ni‐NTA Agarose (Qiagen) for 2 h. Beads were thoroughly washed with Lysis buffer lacking protease inhibitors and incubated with 20 μg SENP overnight to release SUMO1. Following extensive washing, BRCA21‐40HA‐His was eluted with Elution buffer (50 mM Na‐Phosphate pH 8.0, 300 mM NaCl, 250 mM Imidazole, 1 mM ß‐Mercaptoethanol).

Purified BRCA21‐40HA‐His (2 μg) was immobilized on magnetic anti‐HA beads (Pierce Cat# 88836, Thermo Scientific) in a binding buffer containing (25 mM Tris–pH 7.5, 150 mM NaCl, 1 mM EDTA, 0.01% (v/v) IGEPAL, 1 mM DTT), and incubated overnight at 4°C with recombinant DPP9 (2 μg), and/or recombinant PALB2 (9 μg). Where stated, DPP9 was preincubated for 1 h with the following inhibitors: 100 μM 1G244 or 110 μM SLRFYEG. Bound proteins were eluted with the HA peptides and analyzed by western blot.

Direct interactions between recombinant DPP9 and BRCA2‐derived N‐terminal peptides (1–40 and 3–40 amino acids) were analyzed employing surface plasmon resonance (SPR). The surface of a NiHC1000m sensorchip (Xantec Bioanalytics, Duesseldorf, Germany) was rinsed with 0.5 M EDTA (pH 8.0) solution and afterwards conditioned in SPR running buffer (20 mM HEPES pH 7.4, 150 mM NaCl, 50 μM EDTA, 0.005% TWEEN‐20). The chip surface was loaded with Ni2+‐ions, injecting 300 mM NiSO4 solution. A 200 nM solution of His‐tagged DPP9 (ligand) was injected over the chip surface and immobilized to a surface density of 800–1,000 μRIU (Refractive index units; response observed), at a flow rate of 15 μl/min. A serial dilution of the analyte (BRCA2 peptides) diluted in SPR running buffer was injected, and the association was followed for 4.5 min. Dissociation of the analyte was monitored for 7 min. For each analyte concentration, the surface was regenerated, injecting 0.5 M EDTA pH 8.0 and fresh ligand was immobilized as described above (one immobilization/regeneration cycle for each analyte concentration). All binding experiments were performed on a Reichert SR 7500 DC biosensor at 20°C and a flow rate of 40 μl/min. Obtained data were analyzed with Scrubber2.0c. Raw data were double referenced (reference channel, specificity check, and buffer blanks/injections). Equilibrium‐binding response data were exported and plotted for further analysis with GraphPad Prism 6.0.

The assay was performed as described in (Justa‐Schuch et al, 2016) by incubating 50 μM of the BRCA21‐40 peptide with 125 nM DPP9. To test for inhibition, a 10 μM peptide inhibitor (SLRFLYEG) was added. Reactions were stopped after 6 h by dilution and acidification in aqueous 0.1% formic acid and 2% acetonitrile (1/20,000, v:v). In addition, bovine insulin was added to the dilution buffer at a concentration of 1 pmol/μl to prevent analyte loss due to adsorption. The resulting samples were analyzed on a nanoLC425 nanoflow chromatography system coupled to a TripleToF 5600+ Plus mass spectrometer of QqToF geometry (both AB SCIEX). 5 μl of the sample was preconcentrated on a self‐packed Reversed Phase‐C18 precolumn (Reprosil C18‐AQ, Pore Size 120 Å, Particle Size 5 μm, 4 cm length, 0.15 cm I.D., Dr. Maisch) and separated on a self‐packed Reverse Phase‐C18 microcolumn (Reprosil C18‐AQ, 120 Å, 3 μm, 15 cm, 0.075 cm) using a 15 min linear gradient (5 to 50% acetonitrile, 0.1% formic acid modifier, flow rate 300 nl/min, column temperature 50°C) followed by a 5 min high organic cleaning step and a 15 min column re‐equilibration. The eluent was introduced to the mass spectrometer using a Nanospray III ion source with Desolvation Chamber Interface (AB SCIEX) via a commercial Fused Silica tip (FS360‐20‐10‐N‐C15, New Objective) at a spray voltage of 2.2 kV, a sheath gas setting of 15 and an interface heater temperature of 150°C. The MS acquisition cycle consisted of a 500 ms TOF MS survey scan that was used for profiling substrate and product concentrations followed by the data‐dependent triggering of up to 5,100 ms TOF product ion spectra to confirm the identity of detected analytes. Data analysis was performed using the Analyst TF 1.7 and the PeakView 2.1 softwares (AB SCIEX). Analyses were performed in quadruplicate.

75 ng of purified DPP9 was incubated with 200 μM of a test peptide (Syk1‐31, BRCA21‐20, BRCA21‐40, BRCA2P2G
1‐40, BRCA2E13R
1‐40, BRCA2I14N
1‐40, BRCA2W31C
1‐40, BRCA2F32G
1‐40) for 5 min at 24°C in 20 mM HEPES/KOH pH 7.3, 110 mM potassium acetate, 2 mM magnesium acetate, 1 mM EGTA, 0.02% Tween‐20 supplemented with 1 mM DTT. The peptide‐enzyme mixture was added to 100 μM H‐Gly‐Pro‐7‐amino‐4‐methylcoumarin (GP‐AMC) substrate in a final reaction volume of 20 μl. Fluorescence was analyzed using an EnSpire microplate fluorimeter (Perkin Elmer) with 360‐nm (excitation) and 460‐nm (emission) filters. Percentage inhibition was normalized to the corresponding mock‐treated DPP9 controls. All peptides (> 85% purity) were purchased from GenScript.

Purification, crystallization, and structure solution of DPP8 and DPP9 were performed as published in our previous report (Ross et al, 2018). Briefly, the DPP8 isoform 1 (Uni‐ProtKB Q6V1X1) and DPP9 isoform 2 (Uni‐ProtKB Q86TI2–2) were expressed in Spodoptera frugiperda cells and purified. The crystallization was performed in both cases using the hanging drop method with 0.46 M Na‐citrate pH 6.75 as precipitant solution and 10 mg/ml of DPP8 at 4°C. DPP9 crystallization occurred at 20°C, using 20 mg/ml of protein and 10% PEG 8000, 25% glycerol, 0.16 M Calcium acetate, and 0.08 M cacodylate pH 6.25 as precipitant solution. Crystals of space group C2221 (DPP8) and P1211 (DPP9) were soaked with BRCA21‐40 peptide by performing dropping using a pico‐dropper (Ross et al, 2018). Data were collected at SLS‐X10SA and solved to 3.2 Å and 3.0 Å, respectively, by molecular replacement using a DPP8 structure (PDB: 6EOO). Structure solution and refinement were performed as described previously (Ross et al, 2018). Data processing was performed with XDS (Kabsch, 2010); Molecular Replacement with Phaser (McCoy et al, 2007); data refinement with Refmac5 (Murshudov et al, 1997). Data processing, statistics, and refinement values are summarized in Appendix Table S1.

HeLa WT, DPP9 KD, HEK293 DPP9 WT, HEK293 DPP9 KO+DPP9‐SWT / + DPP9‐SS729A cells were treated with 300 nM MMC for 24 h. The CHX chase was started by the addition of 100 μg/ml CHX. Where stated, 100 μM of MG132 was added to the cells 30 min before the addition of the CHX. Samples were harvested at the indicated time points directly in Laemmli Sample Buffer. Immunoblotting and incubation with primary antibodies were performed according to standard protocols. Secondary fluorophore‐coupled antibodies were applied. Signals were developed in the Odyssey Sa Infrared Imaging System (LI‐COR) and analyzed with the ImageStudio (version 4.0.21, LI‐COR) software.

To obtain a majority population of cells in S‐phase, cells were synchronized using a double thymidine block. Briefly, HeLa WT & HeLa DPP9 KD cells were treated with 2 mM of thymidine. After the 18 h, the cells were washed, and replenished with fresh DMEM for 9 h, prior to the second thymidine block (2 mM) for another 15 h. For detection of the cell cycle stages, cells were trypsinized and collected at 0, 2, 3, 4, 5, 6, 8, 24, and 26 h after release. The cell pellet was washed once in PBS and fixed in 70% Ethanol. Cells pellets were resuspended in PBS supplemented with 0.25% Triton‐X‐100 and incubated on ice for 15 min. The residual Triton was washed off with PBS and the cells were incubated in PBS supplemented with 100 μg/ml RNAse A and 50 μg/ml propidium iodide, in the dark, overnight at 4°C. The cells were sorted in a Cytoflex S cell sorter and analyzed using the FlowJo Software.

Cells were seeded on 96‐well, black/clear, tissue culture‐treated plates (Corning Falcon cat# 353219). Following 18 h incubation at 37°C, 5% CO2, cells were treated with 250 ng/ml NCS for 30 min. NCS was then removed and replaced with a fresh medium to allow cells to recover. Cells were fixed at the indicated time points in 4% formaldehyde in PBS for 20 min and permeabilized with 0.2% Triton‐X‐100 in PBS for 20 min. Cells were washed with PBS, blocked with 10% FBS in PBS for 30 min, and incubated with primary antibodies for 2 h. Following three PBS wash steps, cells were incubated for 60 min with secondary antibodies and 1:5,000 DAPI. γH2AX signal intensities were measured on a Celigo 4 Channel Imaging Cytometer (Nexlecom Bioscience) and quantified using the Celigo Software. Images were taken using a Nikon Eclipse Ti2‐E Inverted microscope, Plan Apoλ oil immersion objective 100x NA1.45 WD = 0.13 (Nikon Instruments Inc), and processed using NIS‐Elements AR 5.02.00 (Nikon Instruments Inc), and FIJI (ImageJ) for the preparation of figures. At least three biological replicates were analyzed, each with technical duplicates, and the means calculated.

Amino acid conservation was assessed by aligning the human BRCA2 orthologs from Ensembl. The sequences of BRCA2 orthologs from 108 placental mammals were aligned with the ClustalW algorithm (Thompson et al, 1994). The conservation was illustrated as a logo plot with web logo (Schneider & Stephens, 1990; Crooks et al, 2004). This plot illustrates the alignment as letters, where the height of each letter is in proportion to the observed frequency of the corresponding amino acid, which is measured in bits.

Graphs were generated using the PRISM8 software, statistical analysis was carried out by unpaired or paired two‐tailed t‐test, one‐way or two‐way ANOVA. ImageStudio software was used for the quantification of western blots, which were visualized with LI‐COR. Bands were quantified relative to the respective loading control. Mean and SEM values were calculated by the PRISM software and represented together with SEM error bars. For Colony Formation and Cell Viability assays, measurements from three biological replicates each with technical triplicates per data point are represented as Mean ± SEM. For microscopy images, dots from PLAs or dots from foci formation assays were quantified using DUO‐LINK software.

### Method details

See Reagents Tools table and Table EV1 for Primers and siRNA.

### Cell lines and cell culture

HeLa DPP9 stable Knock‐Down cells (DPP9 KD) and the corresponding HeLa WT cells (Genscript) (Justa‐Schuch et al, 2016), MEF WT and gki MEF DPP9S729A (Gall et al, 2013), HEK293 DPP9 KO+DPP9‐SWT or DPP9‐SS729A, HEK293 DPP9 KO+DPP9‐LWT (this study), and the corresponding HEK 293 DPP9 WT cell lines were cultured in Dulbecco's modified Eagle's medium supplemented with 10% fetal bovine serum, 2 mM L‐glutamine, 100 U/ml penicillin and 100 μg/ml streptomycin. To maintain the selection pressure on the DPP9 KD cells, 1.5 μg/ml puromycin (Sigma‐Aldrich, Germany) was added to the growth medium. HCT 116 cells, stably expressing pDRGFP were a kind gift from Holger Bastians (Kari et al, 2019). Cells were grown in McCoy's 5A medium supplemented with 10% fetal bovine serum. All cells were grown at 37°C and 5% CO2 and routinely tested for mycoplasma contamination.

### Generation of DPP9 knockout (HEK293 DPP9 KO) cells and complementation cell lines (HEK293 DPP9 KO+DPP9‐SWT
, HEK293 DPP9 KO+DPP9‐SS729A
, and HEK293 DPP9 KO+DPP9‐LWT
)

For the generation of the HEK Flp‐In™ T‐REx™‐293 DPP9 knockout clone (HEK293 DPP9 KO), guide RNA sequences (CRISPR guide 3 on exon 9, see Table EV1) targeting human DPP9 were cloned into the pSpCas9(BB)‐2A‐GFP (PX458) vector, which was a gift from Feng Zhang (Addgene plasmid # 48138) (Ran et al, 2013). HEK Flp‐In™ T‐REx™‐293 (ThermoFisher Scientific) cells were transfected using PEI. After 24 h, FACS sorting was used to collect GFP‐positive cells. Single cells were seeded into 96‐well plates. Clones were screened using western blot.

For complementation, the Flp‐In™T‐REx™ system (Invitrogen) was used to create stable, inducible cell lines expressing DPP9‐SWT (HEK293 DPP9 KO+DPP9‐SWT), DPP9‐SS729A (HEK293 DPP9 KO+DPP9‐SS729A) or DPP9‐LWT (HEK293 DPP9 KO+DPP9‐LWT) in pcDNA5/FRT/TO. For the selection of positive clones, DMEM complete containing 100 μg/ml hygromycin and 10 μg/ml blasticidin was used. Unless otherwise stated, DPP9 expression was induced by the addition of Dulbecco's modified Eagle's medium supplemented with 10% fetal bovine serum, 2 mM L glutamine, 100 U/ml penicillin, and 100 μg/ml streptomycin containing 1 μg/ml doxycycline for 24 h.

### Plasmids

The following plasmids were kind gifts, obtained from Addgene: the pCBASce I endonuclease expression vector (Addgene plasmid # 26477) from Maria Jasin (Pierce et al, 1999), the template pcDNA3 236HSC WT BRCA2 (Addgene plasmid # 16246) from Mien‐Chie Hung (Wang et al, 2002). BRCA21‐3418, BRCA21‐1000, and the corresponding mutants BRCA23‐3418, and BRCA23‐1000 that lack the N‐terminal dipeptide Met1Pro2, were cloned into pcDNA3.1+ vector with a Ubiquitin tag fused at the N‐terminus to provide a defined N‐terminus, as first described by Bachmair and Varshavsky (Bachmair et al, 1986). Silent mutations were introduced at the binding sites of the silencing oligos targeting BRCA2 (1721GAAGAATGCAGGTTTAATA1740) at the sites 1731A‐>T and 1734T‐>A. SUMO1‐BRCA21‐40‐HA‐His constructs were cloned by adding SUMO1 to the N‐terminus of BRCA21‐39‐3HA‐His pET11a (custom‐made from Genscript), using the Gibson Chew Back and Anneal Assembly (CBA) as described in (Torella et al, 2014) Primers are listed in extended view Table EV1. Flag‐tagged DPP9‐SWT, Flag‐tagged DPP9‐SS729A, and DPP9‐LWT were cloned into pcDNA5/FRT/TO (ThermoFisher Scientific #V601020). All plasmids have been verified by sequencing. HeLa cDNA was used as a template for PCR amplification of human PALB2. The PALB2 PCR product was cloned into pENTR 3C using the BamH1 and NotI sites. PALB2 was further cloned from pENTR3C into BaculoDirect, using the LR Clonase (ThermoFisher Scientific). BRCA21‐1000
WT, BRCA21‐1000
P2G, and BRCA21‐1000
P2H were custom synthesized by GenScript.

### Co‐immunoprecipitation

HEK293 DPP9 KO+DPP9‐SWT cells were treated for 24 h with 300 nM MMC. For induced expression of FLAG‐tagged DPP9‐SWT, 1 μg/ml of Doxycyline was added together with the MMC for 24 h. Control cells were not treated with Doxycycline. Cells were trypsinized and lysed in Lysis buffer (50 mM HEPES pH 7.4, 100 mM NaCl, 0.5% NP40, 5 mM EDTA, 1 mM DTT, PMSF and 10 mM MG132). Following a 30‐min incubation at 4°C, cell lysates were centrifuged at 20,000 g at 4°C for 20 min. Cleared cell lysates were diluted in 50 mM HEPES pH 7.4, 100 mM NaCl, 5 mM EDTA to a final NP‐40 concentration 0.1%. Diluted lysates were incubated with magnetic FLAG‐beads for 3 h at 4°C. Bound proteins were eluted with 0.5 mg/ml FLAG peptide and analyzed by western blotting.

### Chromatin fractionation

HeLa WT and HeLa DPP9 KD were treated with MMC (300 nM) for 0, 8, or 24 h. Chromatin fractionation was performed essentially as described in (Kari et al, 2016). Briefly, cells were resuspended in lysis buffer (10 mM HEPES pH 7.9, 10 mM KCl, 1.5 mM MgCl2, 0.34 M sucrose, 10% glycerol, 0.1% Triton‐X‐100, 1 mM DTT, and protease inhibitors) and centrifuged at 1,500 g for 5 min. The nuclear pellet was lysed in nuclear lysis buffer (3 mM EDTA, 0.2 mM EGTA, 1 mM DTT, and protease inhibitors) for 30 min on ice. Soluble chromatin fractions were separated by centrifuging at 1,700 g for 5 min. Chromatin fractions were sonicated in a water sonicator (BioRad) for 15 min before loading on SDS–PAGE electrophoresis.

### Immunofluorescence

HeLa, MEF, HEK293 DPP9 WT, HEK293 DPP9 KO+DPP9‐SWT/+DPP9‐SS729A, and HEK293 DPP9 KO+DPP9‐LWT cells were grown on coverslips in 24‐well plates, fixed with 4% formaldehyde in PBS for 10 min and permeabilized with 0.2% Triton‐X‐100 in PBS for 5 min. Cells were washed with PBS and blocked with 2% BSA in PBS for 10 min. Cells were incubated with primary antibodies for 90 min at 37°C. Following a PBS wash, cells were incubated for 45 min at room temperature with the respective secondary antibodies. Cells were washed with PBS and water and mounted in fluorescent mounting medium (DAKO) with DAPI. EdU stained cells were treated with EdU (abcam) 30 min prior to DNA damage induction according to the manufacturer's specifications. Cells were analyzed and images were taken using either an LSM 510‐Meta confocal microscope, oil immersion objective 63x/1.3 (Carl Zeiss MicroImaging, Inc), or a Nikon Eclipse Ti2‐E Inverted microscope, Plan Apoλ oil immersion objective 60x NA1.40 WD = 0.13 (Nikon Instruments Inc). Images were processed using LSM Image Browser (Carl Zeiss MicroImaging, Inc) or NIS‐Elements AR 5.02.00 (Nikon Instruments Inc), based on the microscope used, and FIJI for the preparation of figures. RAD51 foci were quantified using the Duolink ImageTool (Sigma‐Aldrich).

### Proximity ligation assay

Proximity Ligation Assay (PLA) was performed using the DUOLINK In Situ PLA Kit (Sigma‐Aldrich) according to the manufacturer's protocol. Briefly, HeLa cells were grown on coverslips and fixed as described above for immunofluorescence assays. In the case of DPP9 inhibition studies, 10 μM 1G244 was added to the cells for 30 min before fixation. Control cells were mock treated with DMSO. HeLa cells were incubated with primary antibodies for 90 min at 37°C and actin filaments were simultaneously counterstained with CytoPainter Phalloidin‐iFluor 488 Reagent (Abcam ‐ #ab176753). Coverslips were washed with PBS and treated with PLA reagents. Control coverslips (NgtCtrl) were treated with one primary antibody to estimate background staining in each experiment. Cells were mounted in DAKO with DAPI fluorescent mounting medium and analyzed using an LSM 510‐Meta confocal microscope, oil immersion objective 63x/1.3 (Carl Zeiss MicroImaging, Inc) or a Nikon Eclipse Ti2‐E Inverted microscope, Plan Apoλ oil immersion objective 60x NA1.40 WD = 0.13 (Nikon Instruments Inc). Images were processed using LSM Image Browser (Carl Zeiss MicroImaging, Inc) or NIS‐Elements AR 5.02.00 (Nikon Instruments Inc), based on the microscope used and subsequently analyzed using the Duolink ImageTool (Sigma).

### Viability assay

Cells were seeded in 96‐well White Microplates (Perkin Elmer cat# 6005680) and treated with different concentrations of Olaparib for 96 h. For the MMC assays, cells were first incubated for 18–24 h to allow cell attachment. Next, different MMC concentrations were added and cells were analyzed 72 h later. Control cells were treated with DMSO. Cell viability was measured using the CellTiter‐Glo®Luminescence Cell Viability Assay (Promega, cat# Cat G7571). CellTiter‐Glo®Reagent was added in a 1:1 ratio to the cell culture medium. The microplate was shaken on an orbital shaker for 10 min for induction of cell lysis. Subsequently, the luciferase signal was measured on a LuminometerDLReady™ Centro LB 960 reader. Each experiment was performed three times, in triplicates or quadruplicates.

### Colony formation assay

To calculate the respective surviving fractions (SF) after γ radiation (0, 1, 2, 4, 6, and 8G), a standard colony‐forming assay was performed, as previously described (Rave‐Fränk et al, 2007). Briefly, cells were exposed to γ‐ radiation and incubated for 7 days, fixed with 70% ethanol, and stained with Mayer's hemalum (Cat#1.09249.0500, Merck). Nonirradiated cultures were used for normalization. Colonies with > 50 cells were scored as survivors. Three biological replicates were performed, each containing three technical triplicates, and medians were calculated. To validate statistical differences between control and treatment groups, analysis of variance (ANOVA: two‐factor with replication) was performed using Microsoft Excel software (version 2016 MSO). P‐values < 0.05 were considered significant.

### Analysis of DSB repair by GFP‐based reporter assay

To measure HR frequency, HCT116 cells expressing stably integrated pHPRT‐pDR‐GFP reporter plasmid were transfected with the indicated siRNAs. Following 24 h of siRNA transfection, cells were either mock‐transfected or transfected with 2 μg pCBASceI (Addgene) to induce DSBs. 48 h after transfection, the expression of GFP was quantified by flow cytometry using a CytoFLEX S (Beckman Coulter). Cell debris and dead cells were identified using propidium iodide staining (Sigma‐Aldrich P4864). The data were analyzed using the FlowJo software (BS Bioscience). The HR frequency was determined as the percentage of GFP‐positive cells in the total number of alive cells.

### Protein purification

Human recombinant DPP9 was expressed from Sf9 cells and purified essentially as described previously (Pilla et al, 2012). Human recombinant His‐tagged PALB2 was expressed from Sf9 cells. Cells were resuspended in PALB2 lysis buffer (25 mM Hepes pH7.5, 200 mM NaCl, 5 mM imidazole, 10% glycerol, 0.2% Triton, 5 mM ß‐mercaptoethanol, 2 mM PMSF, 10 μg/ml aprotinin, and 10 μg/ml leupeptin), and lysed by homogenization and sonication. 15 U/ml Benzonase and 1 mM MgCl2 were added to the cell lysate. Following a 1 h incubation at 4°C, cell lysate was centrifuged at 50,000 g for 90 min at 4°C. The supernatant was incubated was bound to Ni‐NTA Agarose (Qiagen) for 2 h. Following extensive washing, PALB2 was eluted with Elution Buffer (25 mM Hepes pH7.5, 50 mM NaCl, 300 mM imidazole, 5 mM ß‐mercaptoethanol) and further purified on a Superdex 200 (GE Healthcare). For purification of N‐terminally His‐HA‐tagged fragments corresponding to the N‐terminus of BRCA2 (BRCA2 1‐40HA‐His), SUMO1BRCA21‐40HA‐His was transformed into BL21 (DE3 codon+) cells and induced by 1 mM IPTG for 2 h at 30°C. Cells were lysed in 25 ml Lysis Buffer (50 mM NaH2PO4, 300 mM NaCl, 10 mM Imidazole pH 8.0, 1 mM ß‐Mercaptoethanol and protease inhibitors: 1 μg/ml Aprotinin, 1 μg/ml Leupeptin and 1 μg/ml Pepstatin), by sonification. The 100,000 g supernatant was bound to Ni‐NTA Agarose (Qiagen) for 2 h. Beads were thoroughly washed with Lysis buffer lacking protease inhibitors and incubated with 20 μg SENP overnight to release SUMO1. Following extensive washing, BRCA21‐40HA‐His was eluted with Elution buffer (50 mM Na‐Phosphate pH 8.0, 300 mM NaCl, 250 mM Imidazole, 1 mM ß‐Mercaptoethanol).

### Pull‐down assay of purified proteins

Purified BRCA21‐40HA‐His (2 μg) was immobilized on magnetic anti‐HA beads (Pierce Cat# 88836, Thermo Scientific) in a binding buffer containing (25 mM Tris–pH 7.5, 150 mM NaCl, 1 mM EDTA, 0.01% (v/v) IGEPAL, 1 mM DTT), and incubated overnight at 4°C with recombinant DPP9 (2 μg), and/or recombinant PALB2 (9 μg). Where stated, DPP9 was preincubated for 1 h with the following inhibitors: 100 μM 1G244 or 110 μM SLRFYEG. Bound proteins were eluted with the HA peptides and analyzed by western blot.

### Surface plasmon resonance (SPR)

Direct interactions between recombinant DPP9 and BRCA2‐derived N‐terminal peptides (1–40 and 3–40 amino acids) were analyzed employing surface plasmon resonance (SPR). The surface of a NiHC1000m sensorchip (Xantec Bioanalytics, Duesseldorf, Germany) was rinsed with 0.5 M EDTA (pH 8.0) solution and afterwards conditioned in SPR running buffer (20 mM HEPES pH 7.4, 150 mM NaCl, 50 μM EDTA, 0.005% TWEEN‐20). The chip surface was loaded with Ni2+‐ions, injecting 300 mM NiSO4 solution. A 200 nM solution of His‐tagged DPP9 (ligand) was injected over the chip surface and immobilized to a surface density of 800–1,000 μRIU (Refractive index units; response observed), at a flow rate of 15 μl/min. A serial dilution of the analyte (BRCA2 peptides) diluted in SPR running buffer was injected, and the association was followed for 4.5 min. Dissociation of the analyte was monitored for 7 min. For each analyte concentration, the surface was regenerated, injecting 0.5 M EDTA pH 8.0 and fresh ligand was immobilized as described above (one immobilization/regeneration cycle for each analyte concentration). All binding experiments were performed on a Reichert SR 7500 DC biosensor at 20°C and a flow rate of 40 μl/min. Obtained data were analyzed with Scrubber2.0c. Raw data were double referenced (reference channel, specificity check, and buffer blanks/injections). Equilibrium‐binding response data were exported and plotted for further analysis with GraphPad Prism 6.0.

### Peptidase activity assay by liquid chromatography–tandem mass spectrometry (LC/MS/MS)

The assay was performed as described in (Justa‐Schuch et al, 2016) by incubating 50 μM of the BRCA21‐40 peptide with 125 nM DPP9. To test for inhibition, a 10 μM peptide inhibitor (SLRFLYEG) was added. Reactions were stopped after 6 h by dilution and acidification in aqueous 0.1% formic acid and 2% acetonitrile (1/20,000, v:v). In addition, bovine insulin was added to the dilution buffer at a concentration of 1 pmol/μl to prevent analyte loss due to adsorption. The resulting samples were analyzed on a nanoLC425 nanoflow chromatography system coupled to a TripleToF 5600+ Plus mass spectrometer of QqToF geometry (both AB SCIEX). 5 μl of the sample was preconcentrated on a self‐packed Reversed Phase‐C18 precolumn (Reprosil C18‐AQ, Pore Size 120 Å, Particle Size 5 μm, 4 cm length, 0.15 cm I.D., Dr. Maisch) and separated on a self‐packed Reverse Phase‐C18 microcolumn (Reprosil C18‐AQ, 120 Å, 3 μm, 15 cm, 0.075 cm) using a 15 min linear gradient (5 to 50% acetonitrile, 0.1% formic acid modifier, flow rate 300 nl/min, column temperature 50°C) followed by a 5 min high organic cleaning step and a 15 min column re‐equilibration. The eluent was introduced to the mass spectrometer using a Nanospray III ion source with Desolvation Chamber Interface (AB SCIEX) via a commercial Fused Silica tip (FS360‐20‐10‐N‐C15, New Objective) at a spray voltage of 2.2 kV, a sheath gas setting of 15 and an interface heater temperature of 150°C. The MS acquisition cycle consisted of a 500 ms TOF MS survey scan that was used for profiling substrate and product concentrations followed by the data‐dependent triggering of up to 5,100 ms TOF product ion spectra to confirm the identity of detected analytes. Data analysis was performed using the Analyst TF 1.7 and the PeakView 2.1 softwares (AB SCIEX). Analyses were performed in quadruplicate.

### Peptide competition assay

75 ng of purified DPP9 was incubated with 200 μM of a test peptide (Syk1‐31, BRCA21‐20, BRCA21‐40, BRCA2P2G
1‐40, BRCA2E13R
1‐40, BRCA2I14N
1‐40, BRCA2W31C
1‐40, BRCA2F32G
1‐40) for 5 min at 24°C in 20 mM HEPES/KOH pH 7.3, 110 mM potassium acetate, 2 mM magnesium acetate, 1 mM EGTA, 0.02% Tween‐20 supplemented with 1 mM DTT. The peptide‐enzyme mixture was added to 100 μM H‐Gly‐Pro‐7‐amino‐4‐methylcoumarin (GP‐AMC) substrate in a final reaction volume of 20 μl. Fluorescence was analyzed using an EnSpire microplate fluorimeter (Perkin Elmer) with 360‐nm (excitation) and 460‐nm (emission) filters. Percentage inhibition was normalized to the corresponding mock‐treated DPP9 controls. All peptides (> 85% purity) were purchased from GenScript.

### Protein crystallography

Purification, crystallization, and structure solution of DPP8 and DPP9 were performed as published in our previous report (Ross et al, 2018). Briefly, the DPP8 isoform 1 (Uni‐ProtKB Q6V1X1) and DPP9 isoform 2 (Uni‐ProtKB Q86TI2–2) were expressed in Spodoptera frugiperda cells and purified. The crystallization was performed in both cases using the hanging drop method with 0.46 M Na‐citrate pH 6.75 as precipitant solution and 10 mg/ml of DPP8 at 4°C. DPP9 crystallization occurred at 20°C, using 20 mg/ml of protein and 10% PEG 8000, 25% glycerol, 0.16 M Calcium acetate, and 0.08 M cacodylate pH 6.25 as precipitant solution. Crystals of space group C2221 (DPP8) and P1211 (DPP9) were soaked with BRCA21‐40 peptide by performing dropping using a pico‐dropper (Ross et al, 2018). Data were collected at SLS‐X10SA and solved to 3.2 Å and 3.0 Å, respectively, by molecular replacement using a DPP8 structure (PDB: 6EOO). Structure solution and refinement were performed as described previously (Ross et al, 2018). Data processing was performed with XDS (Kabsch, 2010); Molecular Replacement with Phaser (McCoy et al, 2007); data refinement with Refmac5 (Murshudov et al, 1997). Data processing, statistics, and refinement values are summarized in Appendix Table S1.

### Cycloheximide (CHX) chase assays

HeLa WT, DPP9 KD, HEK293 DPP9 WT, HEK293 DPP9 KO+DPP9‐SWT / + DPP9‐SS729A cells were treated with 300 nM MMC for 24 h. The CHX chase was started by the addition of 100 μg/ml CHX. Where stated, 100 μM of MG132 was added to the cells 30 min before the addition of the CHX. Samples were harvested at the indicated time points directly in Laemmli Sample Buffer. Immunoblotting and incubation with primary antibodies were performed according to standard protocols. Secondary fluorophore‐coupled antibodies were applied. Signals were developed in the Odyssey Sa Infrared Imaging System (LI‐COR) and analyzed with the ImageStudio (version 4.0.21, LI‐COR) software.

### Cell synchronization

To obtain a majority population of cells in S‐phase, cells were synchronized using a double thymidine block. Briefly, HeLa WT & HeLa DPP9 KD cells were treated with 2 mM of thymidine. After the 18 h, the cells were washed, and replenished with fresh DMEM for 9 h, prior to the second thymidine block (2 mM) for another 15 h. For detection of the cell cycle stages, cells were trypsinized and collected at 0, 2, 3, 4, 5, 6, 8, 24, and 26 h after release. The cell pellet was washed once in PBS and fixed in 70% Ethanol. Cells pellets were resuspended in PBS supplemented with 0.25% Triton‐X‐100 and incubated on ice for 15 min. The residual Triton was washed off with PBS and the cells were incubated in PBS supplemented with 100 μg/ml RNAse A and 50 μg/ml propidium iodide, in the dark, overnight at 4°C. The cells were sorted in a Cytoflex S cell sorter and analyzed using the FlowJo Software.

### Accumulation and recovery of γH2AX signals

Cells were seeded on 96‐well, black/clear, tissue culture‐treated plates (Corning Falcon cat# 353219). Following 18 h incubation at 37°C, 5% CO2, cells were treated with 250 ng/ml NCS for 30 min. NCS was then removed and replaced with a fresh medium to allow cells to recover. Cells were fixed at the indicated time points in 4% formaldehyde in PBS for 20 min and permeabilized with 0.2% Triton‐X‐100 in PBS for 20 min. Cells were washed with PBS, blocked with 10% FBS in PBS for 30 min, and incubated with primary antibodies for 2 h. Following three PBS wash steps, cells were incubated for 60 min with secondary antibodies and 1:5,000 DAPI. γH2AX signal intensities were measured on a Celigo 4 Channel Imaging Cytometer (Nexlecom Bioscience) and quantified using the Celigo Software. Images were taken using a Nikon Eclipse Ti2‐E Inverted microscope, Plan Apoλ oil immersion objective 100x NA1.45 WD = 0.13 (Nikon Instruments Inc), and processed using NIS‐Elements AR 5.02.00 (Nikon Instruments Inc), and FIJI (ImageJ) for the preparation of figures. At least three biological replicates were analyzed, each with technical duplicates, and the means calculated.

### Conservation and logo plot analysis

Amino acid conservation was assessed by aligning the human BRCA2 orthologs from Ensembl. The sequences of BRCA2 orthologs from 108 placental mammals were aligned with the ClustalW algorithm (Thompson et al, 1994). The conservation was illustrated as a logo plot with web logo (Schneider & Stephens, 1990; Crooks et al, 2004). This plot illustrates the alignment as letters, where the height of each letter is in proportion to the observed frequency of the corresponding amino acid, which is measured in bits.

### Statistical analysis

Graphs were generated using the PRISM8 software, statistical analysis was carried out by unpaired or paired two‐tailed t‐test, one‐way or two‐way ANOVA. ImageStudio software was used for the quantification of western blots, which were visualized with LI‐COR. Bands were quantified relative to the respective loading control. Mean and SEM values were calculated by the PRISM software and represented together with SEM error bars. For Colony Formation and Cell Viability assays, measurements from three biological replicates each with technical triplicates per data point are represented as Mean ± SEM. For microscopy images, dots from PLAs or dots from foci formation assays were quantified using DUO‐LINK software.

### Author contributions

Oguz Bolgi: Data curation; validation; investigation; visualization; formal analysis, writing – review and editing. Maria Silva‐Garcia: Data curation; investigation; visualization; writing – review and editing. Breyan Ross: Data curation; investigation; visualization; writing – original draft; writing – review and editing. Esther Pilla: Investigation; writing – review and editing. Vijayalakshmi Kari: Investigation; writing – review and editing. Markus Killisch: Investigation; formal analysis, visualization; writing – review and editing. Melanie Spitzner: Formal analysis, investigation; visualization; writing – review and editing. Nadine Stark: Investigation; writing – review and editing. Christof Lenz: Formal analysis, visualization, investigation; writing – review and editing. Konstantin Weiss: Investigation; Resources; writing – review and editing. Laura Donzelli: Investigation; Resources; writing – review and editing. Mark D Gorrell: Resources; supervision; writing – review and editing. Marian Grade: Resources; supervision; writing – review and editing. Jan Riemer: Resources; supervision; methodology; writing – original draft; writing – review and editing. Henning Urlaub: Supervision; methodology; writing – original draft; writing – review and editing. Matthias Dobbelstein: Conceptualization; supervision; investigation; methodology; writing – review and editing. Robert Huber: Conceptualization; data curation; supervision; methodology; writing – original draft; project administration; writing – review and editing. Ruth Geiss‐Friedlander: Conceptualization; resources; data curation; formal analysis; supervision; funding acquisition; visualization; methodology; writing – original draft; project administration; writing – review and editing.

In addition to the CRediT author contributions listed above, the contributions in detail are:

Conceptualization: R.G‐F; laboratory experiments: OB, MS‐G, BR, EP, VK, MK, MS, NS and CL. Writing original draft preparation: R.G‐F, R.H; writing, review, and editing: all authors. Visualization: OB, MK, BR, MS, CL. supervision: MG, JR, HU, MD, RH and RG‐F.

### Disclosure and competing interest statement

The authors declare that they have no conflict of interest.

### Supporting information

Appendix

Click here for additional data file.

Expanded View Figures PDF

Click here for additional data file.

Table EV1

Click here for additional data file.

Source Data for Expanded View and Appendix

Click here for additional data file.

PDF+

Click here for additional data file.

Source Data for Figure 1

Click here for additional data file.

Source Data for Figure 2

Click here for additional data file.

Source Data for Figure 3

Click here for additional data file.

Source Data for Figure 4

Click here for additional data file.



# SUPPLEMENTAL FILE 1: EMBR-23-e54136.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)