# MAIN TEXT

## Pure somatic pathogenic variation profiles for patients with serrated polyposis syndrome: a case series

### Abstract

ObjectiveThe serrated pathway is a distinct genetic/epigenetic mechanism of the adenoma-carcinoma sequence in colorectal carcinogenesis. Although many groups have reported the genetic-phenotypic correlation of serrated lesions (SLs), previous studies regarding the serrated pathway were conducted on patients with SLs that have different germline and environmental genetic backgrounds. We aimed to compare pure somatic genetic profiles among SLs within identical patient with SPS.ResultsWe analyzed SLs from one patient with SPS (Case #1) and compared DNA variant profiles using targeted DNA multigene panels via NGS among the patient’s hyperplastic polyp (HP), three sessile serrated lesions (SSLs), and one traditional serrated adenoma (TSA), and separately analyzed three SSLs and one tubular adenoma (TA) within another patient with SPS (Case #2). In two patients, known pathogenic variant of BRAF (c.1799 T > A, p.Val600Glu) was observed in one TSA and one SSL in Case #1, and in three SSLs within Case #2. The pure somatic pathogenic variant BRAF (c.1799 T > A, p.Val600Glu) among SLs with identical germline genetic background supports its importance as a strong contributor for SLs.Supplementary InformationThe online version contains supplementary material available at 10.1186/s13104-022-06245-3.

### Introduction

Colorectal cancer (CRC) is one of the most common cancers worldwide and ranks as the sixth leading cause of cancer-related deaths [1, 2]. Since CRC arises from premalignant polyps, the detection and removal of these lesions decreases both CRC incidence and mortality [3]. Some groups have reported that 15–30% of all CRCs are initiated from serrated lesions (SLs) rather than conventional adenomas arising through the adenoma-carcinoma sequence [4–6]. SLs are histologically heterogeneous, including benign hyperplastic polyps (HPs), precancerous sessile serrated lesions (SSLs), or traditional serrated adenomas (TSAs) [4]. Among these SLs, HPs are the most frequent subtype and SSLs are the second most common form of SLs. SSLs are recognized as important precursors of the serrated pathway showing a high CpG island methylator phenotype (CIMP) [7, 8].

The serrated pathway is a distinct genetic/epigenetic mechanism of colorectal carcinogenesis, but this has not been fully characterized. Although many groups have reported the genetic-phenotypic correlation regarding SLs, the precise profile and mechanisms of these serrated pathways for the prevention of colorectal carcinogenesis are not fully elucidated, as previous reports involved many patients with SLs with different germline and environmental backgrounds [9–21]. Therefore, genetic comparison and analysis of multiple SLs within the same patient should be conducted.

Serrated polyposis syndrome (SPS) is characterized by multiple SLs located throughout the colon and is accompanied by an increased risk of CRC. The diagnosis of SPS is based on the cumulative lifetime number of HPs, TSAs, and SSPs in a patient who meets one of the two following World Health Organization (WHO) criteria, including (1) > 5SPs proximal to the rectum, all being ≥ 5 mm in size, including ≥ 2 that are ≥ 10 mm; or (2) > 20 SPs of any size distributed throughout the colon, with ≥ 5 being proximal to the rectum [22]. Therefore, it is important to compare genetic profiles among SLs within identical patient with SPS to understand the pure somatic genetic variant associated with the serrated pathway.

In the present study, we customized a set of targeted DNA multigene panels and used it to evaluate the variant SL profiles. Herein, we show differences in the main genetic contributors to the serrated pathway among SLs within the same SPS patient.

### Main text

We analyzed nine SLs and one non-SL (tubular adenoma) from two patients with SPS who met the WHO 2019 criteria for the diagnosis of SPS. Both patients provided written informed consent, and the study was approved by the Institutional Review Board of the Hamamatsu University School of Medicine (Approval No. 17–222).

SL samples were obtained from the Department of Diagnostic Pathology at Hamamatsu University Hospital as formalin-fixed paraffin-embedded (FFPE) tissue from the two patients with SPS. The polyps were resected by EMR. We also obtained matched normal blood samples. Genomic DNA was extracted from macrodissected tumorous and non-tumorous tissue using a QIAamp DNA FFPE Tissue Kit (Qiagen, Hilden, Germany), and extracted from the blood using an EZ1 DNA Blood 350 µl Kit (Qiagen). The quality of the gDNA was analyzed using the 2200 TapeStation (Agilent Technologies, Santa Clara, CA, USA) system using the TapeStation Analysis software (Agilent), which automatically determines and displays the DNA integrity number (DIN) as a measure of DNA integrity (https://www.agilent.com/cs/library/applications/5991-5258EN.pdf).

We customized the multigene panel (72 genes) by adding the QIAseq Human Colorectal Cancer Panel (71 genes, DHS-002Z; Qiagen) to the RNF43 gene because the pathogenicity of the RNF43 gene variant has been reported in SLs [23]. The customized multigene panel was used for library construction according to the manufacturer’s instructions. The libraries were assessed using a QIAseq Library Quant Assay Kit (#QSTF-ILZ-R; Qiagen) and applied to a MiniSeq sequencer (Illumina, San Diego, CA, USA). The Qiagen web portal (https://geneglobe.qiagen.com/jp/analyze/) and VariantStudio software (Illumina) were used for data analysis and alignment. GRCH37 was used as the reference genome. All detected variants were validated using Integrative Genomics Viewer 2.9.2 (IGV; http://software.broadinstitute.org/software/igv/home).

IHC was performed as described previously [24].

Statistical analyses were performed using IBM SPSS Statistics for Windows (version 25; IBM Corp., Armonk, NY, USA), and a value of P < 0.05 was considered statistically significant.

### Methods

We analyzed nine SLs and one non-SL (tubular adenoma) from two patients with SPS who met the WHO 2019 criteria for the diagnosis of SPS. Both patients provided written informed consent, and the study was approved by the Institutional Review Board of the Hamamatsu University School of Medicine (Approval No. 17–222).

SL samples were obtained from the Department of Diagnostic Pathology at Hamamatsu University Hospital as formalin-fixed paraffin-embedded (FFPE) tissue from the two patients with SPS. The polyps were resected by EMR. We also obtained matched normal blood samples. Genomic DNA was extracted from macrodissected tumorous and non-tumorous tissue using a QIAamp DNA FFPE Tissue Kit (Qiagen, Hilden, Germany), and extracted from the blood using an EZ1 DNA Blood 350 µl Kit (Qiagen). The quality of the gDNA was analyzed using the 2200 TapeStation (Agilent Technologies, Santa Clara, CA, USA) system using the TapeStation Analysis software (Agilent), which automatically determines and displays the DNA integrity number (DIN) as a measure of DNA integrity (https://www.agilent.com/cs/library/applications/5991-5258EN.pdf).

We customized the multigene panel (72 genes) by adding the QIAseq Human Colorectal Cancer Panel (71 genes, DHS-002Z; Qiagen) to the RNF43 gene because the pathogenicity of the RNF43 gene variant has been reported in SLs [23]. The customized multigene panel was used for library construction according to the manufacturer’s instructions. The libraries were assessed using a QIAseq Library Quant Assay Kit (#QSTF-ILZ-R; Qiagen) and applied to a MiniSeq sequencer (Illumina, San Diego, CA, USA). The Qiagen web portal (https://geneglobe.qiagen.com/jp/analyze/) and VariantStudio software (Illumina) were used for data analysis and alignment. GRCH37 was used as the reference genome. All detected variants were validated using Integrative Genomics Viewer 2.9.2 (IGV; http://software.broadinstitute.org/software/igv/home).

IHC was performed as described previously [24].

Statistical analyses were performed using IBM SPSS Statistics for Windows (version 25; IBM Corp., Armonk, NY, USA), and a value of P < 0.05 was considered statistically significant.

### Patients

We analyzed nine SLs and one non-SL (tubular adenoma) from two patients with SPS who met the WHO 2019 criteria for the diagnosis of SPS. Both patients provided written informed consent, and the study was approved by the Institutional Review Board of the Hamamatsu University School of Medicine (Approval No. 17–222).

### Samples, DNA extraction, and quality assessment

SL samples were obtained from the Department of Diagnostic Pathology at Hamamatsu University Hospital as formalin-fixed paraffin-embedded (FFPE) tissue from the two patients with SPS. The polyps were resected by EMR. We also obtained matched normal blood samples. Genomic DNA was extracted from macrodissected tumorous and non-tumorous tissue using a QIAamp DNA FFPE Tissue Kit (Qiagen, Hilden, Germany), and extracted from the blood using an EZ1 DNA Blood 350 µl Kit (Qiagen). The quality of the gDNA was analyzed using the 2200 TapeStation (Agilent Technologies, Santa Clara, CA, USA) system using the TapeStation Analysis software (Agilent), which automatically determines and displays the DNA integrity number (DIN) as a measure of DNA integrity (https://www.agilent.com/cs/library/applications/5991-5258EN.pdf).

### Next-generation sequencing (NGS)

We customized the multigene panel (72 genes) by adding the QIAseq Human Colorectal Cancer Panel (71 genes, DHS-002Z; Qiagen) to the RNF43 gene because the pathogenicity of the RNF43 gene variant has been reported in SLs [23]. The customized multigene panel was used for library construction according to the manufacturer’s instructions. The libraries were assessed using a QIAseq Library Quant Assay Kit (#QSTF-ILZ-R; Qiagen) and applied to a MiniSeq sequencer (Illumina, San Diego, CA, USA). The Qiagen web portal (https://geneglobe.qiagen.com/jp/analyze/) and VariantStudio software (Illumina) were used for data analysis and alignment. GRCH37 was used as the reference genome. All detected variants were validated using Integrative Genomics Viewer 2.9.2 (IGV; http://software.broadinstitute.org/software/igv/home).

### IHC staining

IHC was performed as described previously [24].

### Statistical analyses

Statistical analyses were performed using IBM SPSS Statistics for Windows (version 25; IBM Corp., Armonk, NY, USA), and a value of P < 0.05 was considered statistically significant.

### Results

We analyzed SLs from two patients with SPS (Cases #1 and #2). A man (Case #1) underwent colonoscopy and six protruded lesions were detected throughout the colon (Fig. 1). We performed endoscopic mucosal resection (EMR) on all lesions, and SLs (one HP in the transverse colon, three SSLs in the ascending colon, and two TSAs in the transverse colon and the sigmoid colon, respectively) were diagnosed histopathologically in each resected specimen. Case #2 is a woman who underwent colonoscopy as part of a routine medical examination, and six protruded lesions were detected throughout the colon. All lesions were located proximal to the rectum (Fig. 1) and were endoscopically resected. Among the lesions, five were histologically SSLs (three lesions in the transverse colon and two in the descending colon), and one was non-SL (tubular adenoma) in the cecum. Both two patients were diagnosed with SPS because they met the WHO 2019 criteria.Fig. 1Endoscopic/histological features of analyzed lesions. A Case #1. A man who initially exhibited a positive fecal occult blood test (FOBT). He underwent colonoscopy and six protruded lesions were detected throughout the colon, and SLs (one HP, three SSLs, and two TSAs) were diagnosed histopathologically in each resected specimen. B Case #2. A woman who underwent colonoscopy as part of a routine medical examination, and six protruded lesions were detected throughout the colon. Among the lesions, five were histologically SSLs, and one was non-SL (tubular adenoma). SSL sessile serrated lesion, HP hyperplastic polyp, TSA traditional serrated adenoma, TA tubular adenoma, A Ascending Colon, T Transverse Colon, S Sigmoid Colon, C Cecum, D Descending Colon, NT not tested

Endoscopic/histological features of analyzed lesions. A Case #1. A man who initially exhibited a positive fecal occult blood test (FOBT). He underwent colonoscopy and six protruded lesions were detected throughout the colon, and SLs (one HP, three SSLs, and two TSAs) were diagnosed histopathologically in each resected specimen. B Case #2. A woman who underwent colonoscopy as part of a routine medical examination, and six protruded lesions were detected throughout the colon. Among the lesions, five were histologically SSLs, and one was non-SL (tubular adenoma). SSL sessile serrated lesion, HP hyperplastic polyp, TSA traditional serrated adenoma, TA tubular adenoma, A Ascending Colon, T Transverse Colon, S Sigmoid Colon, C Cecum, D Descending Colon, NT not tested

The patient in Case #1 exhibited all pathological SL types (HP, SSL, and TSA) and we analyzed the DNA variant profile of one HP (three SSLs, and one TSA (Table 1, Additional file 1: Table S1). When focusing on gene variants known to be associated with SLs, a known pathogenic variant of BRAF (c.1799 T > A, p.Val600Glu) was detected in one SSL located in the ascending colon (#1–4) and one TSA in the transverse colon (#1–5) among the six SLs. One SSL in the ascending colon displayed a splice site variant at RNF43 (c.687G > A) without any BRAF variant. The SSL with the BRAF c.1799 T > A pathogenic variant located in the ascending colon also displayed the MLH1 variant (c.687G > A, p.Val213Glu).Table 1DNA variant profile (Case #1)GeneDNA variantAmino acid alterationVariant allele frequency (%)(Bold: VAF > 5%)#1–0blood#1–1SSL(A)#1–2HP(T)#1–3SSL(A)#1–4SSL(A)#1–5TSA(S)DCCNM_005215.3:c.2277 T > GNP_005206.2:p.Ile759Met47.353.554.250.850.954.9MSH3NM_002439.4:c.1718G > ANP_002430.3:p.Arg573Lys51.152.981.842.358.650BAXNM_004324.3:c.32G > ANP_004315.1:p.Gly11Glu35.946.742.945.749.549.0SRCNM_005417.4:c.532C > TNP_005408.1:p.Arg178Ter06.00000RETNM_020975.4:c.296G > ANP_066124.1:p.Arg99Gln06.70000MLH3NM_001040108.1:c.3769 T > CNP_001035197.1:p.Ser1257Pro011.50000TCERG1NM_006706.3:c.1705G > CNP_006697.2:p.Asp569His025.70000BUB1BNM_001211.5:c.898A > CNP_001202.4:p.Met300Leu0018.8000PALB2NM_024675.3:c.829G > ANP_078951.2:p.Asp277Asn0030.0000CHEK2NM_001005735.1:c.1696C > TNP_001005735.1:p.Arg566Cys0016.7000CTNNA1NM_001903.2:c.2281C > TNP_001894.2:p.Arg761Cys0015.8000TCF7L2NM_001146274.1:c.1001C > TNP_001139746.1:p.Ser334Leu006.2000ATMNM_000051.3:c.5189G > ANP_000042.3:p.Arg1730Gln0007.900TP53NM_000546.5:c.818G > TNP_000537.3:p.Arg273Leu0009.100RNF43NM_017763.4:c.687G > Asplicing site0007.400BLMNM_000057.2:c.1544delANP_000048.1:p.Asn515MetfsTer1600009.10AXIN2NM_004655.3:c.1419_1421delCCANP_004646.3:p.His474del00006.10CDC27NM_001114091.1:c.1750A > GNP_001107563.1:p.Ser584Gly00005.80CDC27NM_001114091.1:c.1665 T > GNP_001107563.1:p.Asp555Glu00007.40MLH1NM_000249.3:c.638 T > ANP_000240.1:p.Val213Glu00008.30BRAFNM_004333.4:c.1799 T > ANP_004324.2:p.Val600Glu000014.713.9RETNM_020975.4:c.2842G > ANP_066124.1:p.Gly948Arg000006.3ERBB2NM_004448.2:c.1295G > ANP_004439.2:p.Arg432Gln0000010.5ERBB2NM_004448.2:c.1846 T > CNP_004439.2:p.Phe616Leu000005.7STK11NM_000455.4:c.928C > TNP_000446.1:p.Arg310Trp000006.7TCERG1NM_006706.3:c.2870delANP_006697.2:p.Lys957ArgfsTer1700003.511.3HP hyperplastic polyp, SSL sessile serrated lesion, TSA traditional serrated adenoma, A Ascending Colon, T Transverse Colon, S Sigmoid Colon

DNA variant profile (Case #1)

HP hyperplastic polyp, SSL sessile serrated lesion, TSA traditional serrated adenoma, A Ascending Colon, T Transverse Colon, S Sigmoid Colon

The patient in Case #2 had only one type of SL (three SSLs), but it is unique that we could compare the DNA profile of three SSLs with that of one non-SL lesion (tubular adenoma) (Table 2, Additional file 2: Table S2). A known pathogenic variant of BRAF (c.1799 T > A, p.Val600Glu) was detected in all SSLs analyzed (#2–1, #2–3, and #2–4), whereas we detected another two BRAF variants, not known to be pathogenic in the previous database, in tubular adenomas of patients. No KRAS or RNF43 variants were detected among the four lesions, including tubular adenomas. Interestingly, a tubular adenoma displayed two pathogenic variants that are highly associated with the adenoma-carcinoma sequence (APC; c.4249_4265delATTATAAGCCCCAGTGA, p.Ile1417SerfsTer4, TP53; c.818G > A, p.Arg273His), but not the other three SSLs. Among all the nine lesions, we detected no lesions with defective MLH1 proteins by IHC.Table 2DNA variant profile (Case #2)GeneDNA variantAmino acid alterationVariant allele frequency (%)(Bold: VAF > 5%)#2–0Blood#2–1SSL(T)#2–2TA(C)#2–3SSL(D)#2–4SSL(T)BRCA2NM_000059.3:c.2350A > GNP_000050.2:p.Met784Val49.356.546.445.551.5BRCA2NM_000059.3:c.3420 T > ANP_000050.2:p.Ser1140Arg53.752.950.051.950.3BRCA1NM_007300.3:c.670 + 1G > Tsplicing site58.957.150.055.053.5AXIN2NM_004655.3:c.2140C > TNP_004646.3:p.Arg714Trp43.844.436.744.349.8METNM_001127500.1:c.4141G > ANP_001120972.1:p.Ala1381Thr48.648.449.446.655.8BRAFNM_004333.4:c.1799 T > ANP_004324.2:p.Val600Glu015.8010.69.5ERBB2NM_004448.2:c.1846 T > CNP_004439.2:p.Phe616Leu06.6000DMDNM_004006.2:c.8851C > ANP_003997.1:p.Arg2951Ser06.9000APCNM_000038.5:c.4249_4265delATTATAAGCCCCAGTGANP_000029.2:p.Ile1417SerfsTer40055.400TP53NM_000546.5:c.818G > ANP_000537.3:p.Arg273His0027.900MSH2NM_000251.2:c.727C > TNP_000242.1:p.Arg243Trp0010.500CTNNB1NM_001904.3:c.1267A > TNP_001895.1:p.Ile423Phe0034.600FBXW7NM_033632.3:c.227A > TNP_361014.1:p.Gln76Leu0024.500BRAFNM_004333.4:c.1781A > GNP_004324.2:p.Asp594Gly0038.500BRAFNM_004333.4:c.1085G > ANP_004324.2:p.Arg362Gln0026.400GALNT12NM_024642.4:c.1250G > ANP_078918.3:p.Arg417Gln0022.400FGFR3NM_001163213.1:c.929A > CNP_001156685.1:p.Lys310Thr0010.300FGFR3NM_001163213.1:c.930 + 5G > Csplicing site0011.500CDC27NM_001114091.1:c.1750A > GNP_001107563.1:p.Ser584Gly00006.2CDC27NM_001114091.1:c.1060C > ANP_001107563.1:p.Gln354Lys00005.5CDC27NM_001114091.1:c.1039G > ANP_001107563.1:p.Glu347Lys00005.5CDC27NM_001114091.1:c.80 T > CNP_001107563.1:p.Leu27Pro00005.2CDC27NM_001114091.1:c.77 T > CNP_001107563.1:p.Phe26Ser00006.5SSL sessile serrated lesion, TA tubular adenoma, C Cecum, A Ascending Colon, T Transverse Colon, D Descending Colon

DNA variant profile (Case #2)

NM_000038.5:

c.4249_4265delATTATAAGCCCCAGTGA

SSL sessile serrated lesion, TA tubular adenoma, C Cecum, A Ascending Colon, T Transverse Colon, D Descending Colon

Mutational signatures (MS) were analyzed by examining combinations of single base substitutions and further including flanking 5’ and 3’ bases of each mutated site. As shown in Additional file 3: Table S3, the most common type of single base substitution (SBS) was C > T, followed by T > C among nine SSLs. Especially, C > T SBS tended to be commonly observed in HP (#1–2) and TSA (#1–5) of case 1 and TA (#2–2) of case 2, and T > C occurred in one SSL (#2–4) of case 2. It is possible that both non-SSL serrated legions such as HP, TSA, and non-SLs as TA, may be characterized as MS patterns seen by aging. Additionally, among nine regions, we observed nine substitutional sets of CCG > CTG, six sets of TCG > TTG and GTG > GAG, and five sets of ACC > AGC and GCT > GGT, but a typical MS pattern was not identified (Additional file 4: Table S4).

### Clinicopathological features

We analyzed SLs from two patients with SPS (Cases #1 and #2). A man (Case #1) underwent colonoscopy and six protruded lesions were detected throughout the colon (Fig. 1). We performed endoscopic mucosal resection (EMR) on all lesions, and SLs (one HP in the transverse colon, three SSLs in the ascending colon, and two TSAs in the transverse colon and the sigmoid colon, respectively) were diagnosed histopathologically in each resected specimen. Case #2 is a woman who underwent colonoscopy as part of a routine medical examination, and six protruded lesions were detected throughout the colon. All lesions were located proximal to the rectum (Fig. 1) and were endoscopically resected. Among the lesions, five were histologically SSLs (three lesions in the transverse colon and two in the descending colon), and one was non-SL (tubular adenoma) in the cecum. Both two patients were diagnosed with SPS because they met the WHO 2019 criteria.Fig. 1Endoscopic/histological features of analyzed lesions. A Case #1. A man who initially exhibited a positive fecal occult blood test (FOBT). He underwent colonoscopy and six protruded lesions were detected throughout the colon, and SLs (one HP, three SSLs, and two TSAs) were diagnosed histopathologically in each resected specimen. B Case #2. A woman who underwent colonoscopy as part of a routine medical examination, and six protruded lesions were detected throughout the colon. Among the lesions, five were histologically SSLs, and one was non-SL (tubular adenoma). SSL sessile serrated lesion, HP hyperplastic polyp, TSA traditional serrated adenoma, TA tubular adenoma, A Ascending Colon, T Transverse Colon, S Sigmoid Colon, C Cecum, D Descending Colon, NT not tested

Endoscopic/histological features of analyzed lesions. A Case #1. A man who initially exhibited a positive fecal occult blood test (FOBT). He underwent colonoscopy and six protruded lesions were detected throughout the colon, and SLs (one HP, three SSLs, and two TSAs) were diagnosed histopathologically in each resected specimen. B Case #2. A woman who underwent colonoscopy as part of a routine medical examination, and six protruded lesions were detected throughout the colon. Among the lesions, five were histologically SSLs, and one was non-SL (tubular adenoma). SSL sessile serrated lesion, HP hyperplastic polyp, TSA traditional serrated adenoma, TA tubular adenoma, A Ascending Colon, T Transverse Colon, S Sigmoid Colon, C Cecum, D Descending Colon, NT not tested

### Somatic variant profile of analyzed lesions from patients with SPS

The patient in Case #1 exhibited all pathological SL types (HP, SSL, and TSA) and we analyzed the DNA variant profile of one HP (three SSLs, and one TSA (Table 1, Additional file 1: Table S1). When focusing on gene variants known to be associated with SLs, a known pathogenic variant of BRAF (c.1799 T > A, p.Val600Glu) was detected in one SSL located in the ascending colon (#1–4) and one TSA in the transverse colon (#1–5) among the six SLs. One SSL in the ascending colon displayed a splice site variant at RNF43 (c.687G > A) without any BRAF variant. The SSL with the BRAF c.1799 T > A pathogenic variant located in the ascending colon also displayed the MLH1 variant (c.687G > A, p.Val213Glu).Table 1DNA variant profile (Case #1)GeneDNA variantAmino acid alterationVariant allele frequency (%)(Bold: VAF > 5%)#1–0blood#1–1SSL(A)#1–2HP(T)#1–3SSL(A)#1–4SSL(A)#1–5TSA(S)DCCNM_005215.3:c.2277 T > GNP_005206.2:p.Ile759Met47.353.554.250.850.954.9MSH3NM_002439.4:c.1718G > ANP_002430.3:p.Arg573Lys51.152.981.842.358.650BAXNM_004324.3:c.32G > ANP_004315.1:p.Gly11Glu35.946.742.945.749.549.0SRCNM_005417.4:c.532C > TNP_005408.1:p.Arg178Ter06.00000RETNM_020975.4:c.296G > ANP_066124.1:p.Arg99Gln06.70000MLH3NM_001040108.1:c.3769 T > CNP_001035197.1:p.Ser1257Pro011.50000TCERG1NM_006706.3:c.1705G > CNP_006697.2:p.Asp569His025.70000BUB1BNM_001211.5:c.898A > CNP_001202.4:p.Met300Leu0018.8000PALB2NM_024675.3:c.829G > ANP_078951.2:p.Asp277Asn0030.0000CHEK2NM_001005735.1:c.1696C > TNP_001005735.1:p.Arg566Cys0016.7000CTNNA1NM_001903.2:c.2281C > TNP_001894.2:p.Arg761Cys0015.8000TCF7L2NM_001146274.1:c.1001C > TNP_001139746.1:p.Ser334Leu006.2000ATMNM_000051.3:c.5189G > ANP_000042.3:p.Arg1730Gln0007.900TP53NM_000546.5:c.818G > TNP_000537.3:p.Arg273Leu0009.100RNF43NM_017763.4:c.687G > Asplicing site0007.400BLMNM_000057.2:c.1544delANP_000048.1:p.Asn515MetfsTer1600009.10AXIN2NM_004655.3:c.1419_1421delCCANP_004646.3:p.His474del00006.10CDC27NM_001114091.1:c.1750A > GNP_001107563.1:p.Ser584Gly00005.80CDC27NM_001114091.1:c.1665 T > GNP_001107563.1:p.Asp555Glu00007.40MLH1NM_000249.3:c.638 T > ANP_000240.1:p.Val213Glu00008.30BRAFNM_004333.4:c.1799 T > ANP_004324.2:p.Val600Glu000014.713.9RETNM_020975.4:c.2842G > ANP_066124.1:p.Gly948Arg000006.3ERBB2NM_004448.2:c.1295G > ANP_004439.2:p.Arg432Gln0000010.5ERBB2NM_004448.2:c.1846 T > CNP_004439.2:p.Phe616Leu000005.7STK11NM_000455.4:c.928C > TNP_000446.1:p.Arg310Trp000006.7TCERG1NM_006706.3:c.2870delANP_006697.2:p.Lys957ArgfsTer1700003.511.3HP hyperplastic polyp, SSL sessile serrated lesion, TSA traditional serrated adenoma, A Ascending Colon, T Transverse Colon, S Sigmoid Colon

DNA variant profile (Case #1)

HP hyperplastic polyp, SSL sessile serrated lesion, TSA traditional serrated adenoma, A Ascending Colon, T Transverse Colon, S Sigmoid Colon

The patient in Case #2 had only one type of SL (three SSLs), but it is unique that we could compare the DNA profile of three SSLs with that of one non-SL lesion (tubular adenoma) (Table 2, Additional file 2: Table S2). A known pathogenic variant of BRAF (c.1799 T > A, p.Val600Glu) was detected in all SSLs analyzed (#2–1, #2–3, and #2–4), whereas we detected another two BRAF variants, not known to be pathogenic in the previous database, in tubular adenomas of patients. No KRAS or RNF43 variants were detected among the four lesions, including tubular adenomas. Interestingly, a tubular adenoma displayed two pathogenic variants that are highly associated with the adenoma-carcinoma sequence (APC; c.4249_4265delATTATAAGCCCCAGTGA, p.Ile1417SerfsTer4, TP53; c.818G > A, p.Arg273His), but not the other three SSLs. Among all the nine lesions, we detected no lesions with defective MLH1 proteins by IHC.Table 2DNA variant profile (Case #2)GeneDNA variantAmino acid alterationVariant allele frequency (%)(Bold: VAF > 5%)#2–0Blood#2–1SSL(T)#2–2TA(C)#2–3SSL(D)#2–4SSL(T)BRCA2NM_000059.3:c.2350A > GNP_000050.2:p.Met784Val49.356.546.445.551.5BRCA2NM_000059.3:c.3420 T > ANP_000050.2:p.Ser1140Arg53.752.950.051.950.3BRCA1NM_007300.3:c.670 + 1G > Tsplicing site58.957.150.055.053.5AXIN2NM_004655.3:c.2140C > TNP_004646.3:p.Arg714Trp43.844.436.744.349.8METNM_001127500.1:c.4141G > ANP_001120972.1:p.Ala1381Thr48.648.449.446.655.8BRAFNM_004333.4:c.1799 T > ANP_004324.2:p.Val600Glu015.8010.69.5ERBB2NM_004448.2:c.1846 T > CNP_004439.2:p.Phe616Leu06.6000DMDNM_004006.2:c.8851C > ANP_003997.1:p.Arg2951Ser06.9000APCNM_000038.5:c.4249_4265delATTATAAGCCCCAGTGANP_000029.2:p.Ile1417SerfsTer40055.400TP53NM_000546.5:c.818G > ANP_000537.3:p.Arg273His0027.900MSH2NM_000251.2:c.727C > TNP_000242.1:p.Arg243Trp0010.500CTNNB1NM_001904.3:c.1267A > TNP_001895.1:p.Ile423Phe0034.600FBXW7NM_033632.3:c.227A > TNP_361014.1:p.Gln76Leu0024.500BRAFNM_004333.4:c.1781A > GNP_004324.2:p.Asp594Gly0038.500BRAFNM_004333.4:c.1085G > ANP_004324.2:p.Arg362Gln0026.400GALNT12NM_024642.4:c.1250G > ANP_078918.3:p.Arg417Gln0022.400FGFR3NM_001163213.1:c.929A > CNP_001156685.1:p.Lys310Thr0010.300FGFR3NM_001163213.1:c.930 + 5G > Csplicing site0011.500CDC27NM_001114091.1:c.1750A > GNP_001107563.1:p.Ser584Gly00006.2CDC27NM_001114091.1:c.1060C > ANP_001107563.1:p.Gln354Lys00005.5CDC27NM_001114091.1:c.1039G > ANP_001107563.1:p.Glu347Lys00005.5CDC27NM_001114091.1:c.80 T > CNP_001107563.1:p.Leu27Pro00005.2CDC27NM_001114091.1:c.77 T > CNP_001107563.1:p.Phe26Ser00006.5SSL sessile serrated lesion, TA tubular adenoma, C Cecum, A Ascending Colon, T Transverse Colon, D Descending Colon

DNA variant profile (Case #2)

NM_000038.5:

c.4249_4265delATTATAAGCCCCAGTGA

SSL sessile serrated lesion, TA tubular adenoma, C Cecum, A Ascending Colon, T Transverse Colon, D Descending Colon

### Mutational signature patterns

Mutational signatures (MS) were analyzed by examining combinations of single base substitutions and further including flanking 5’ and 3’ bases of each mutated site. As shown in Additional file 3: Table S3, the most common type of single base substitution (SBS) was C > T, followed by T > C among nine SSLs. Especially, C > T SBS tended to be commonly observed in HP (#1–2) and TSA (#1–5) of case 1 and TA (#2–2) of case 2, and T > C occurred in one SSL (#2–4) of case 2. It is possible that both non-SSL serrated legions such as HP, TSA, and non-SLs as TA, may be characterized as MS patterns seen by aging. Additionally, among nine regions, we observed nine substitutional sets of CCG > CTG, six sets of TCG > TTG and GTG > GAG, and five sets of ACC > AGC and GCT > GGT, but a typical MS pattern was not identified (Additional file 4: Table S4).

### Discussion

Some groups have reported on the molecular characteristics of various types of serrated lesions. However, the collected tumor samples had various genetic germline backgrounds and were obtained from patients who were subjected to different environmental factors, lifestyles and microbiomes. Therefore, molecular analysis should be performed using SLs from patients with identical genetic backgrounds. Our study demonstrated that (a) favorable DNA samples (≥ 4.0 DIN) can be obtained from FFPE tissues stored for 2 years or more to detect appropriate somatic DNA profiles using NGS, (b) pure somatic SL DNA profiles within a SPS patient were compatible with previous SL reports using patients with heterogeneous germline genetic backgrounds, and (c) pure DNA profiles of TA are quite different from that of other SLs within a patient with SPS. To our knowledge, this is the first study to demonstrate a pure somatic genetic profile compared among SLs within the same patient.

Many groups have reported the influence of pathological genetic variations of BRAF, such as c.1799 T > A and p.Val600Glu, on the progression of HPs, SSLs, TSAs, and KRAS pathogenic variants for HPs and TSAs, but these analyses were performed among patients with heterogeneous germline backgrounds [10–13]. To detect a pure somatic genetic variation profile, we compared the genetic profiles of dome-serrated lesions within the same identical patient. In Case #1, a known pathogenic variant of BRAF (c.1799 T > A, p.Val600Glu) was detected in one SSL (#1–4) and one TSA (#1–5). Previous reports have demonstrated that the BRAF variant was found in almost all SSLs. Accordingly, we detected the BRAF variant in two different SLs in patient #1 [12, 13, 20, 21]. When focusing on the two lesions, it is interesting that genetic profiles, other than that of the BRAF variant, appear quite different (BLM, AXIN2, CDC27, and MLH1 in #1–4, and RET, ERBB2, STK11, and TCERG1 in #1–5, as seen in Additional file 4: Table S4). Therefore, the two SLs must be initiated by the common BRAF pathogenic variant, followed by progression via the accumulation of different genetic profiles, but further accumulated findings should be considered. In Case #2, all SSLs displayed pathogenic variants of BRAF (c.1799 T > A, p.Val600Glu), as expected from previous reports [14]. In addition, it is interesting that we detected a somatic APC deletion (c.4249_4265delATTATAAGCCCCAGTGA) as a driver variant (VAF: 55.4%) in TA (#2–2). Notably, the somatic genetic profile of the TA was quite different from other SSLs within Case #2 (#2–1, #2–3, #2–4), which indicates that the serrated pathway and adenoma-carcinoma sequence do not have common driver variants at the initiation stage, and that the accumulated genetic variant profile is distinct between the two pathways.

RNF43 has been reported as one of the key genes when pathogenic germline or somatic variants are detected in SLs [23, 25, 26]. Giannakis et al. demonstrated that somatic mutations in RNF43 occur in 18.9–17.6% of CRC cases, and the majority of RNF43 somatic mutations were truncating events. Taken together, it is possible that the somatic RNF43 splice-site variant detected in our study in SSLs of Case #1 (#1–3) is pathogenic in the serrated polyposis-cancer sequence, although additional questions remain as limitations, such as the existence of two hits for the lesion by genetic or epigenetic alteration.

As for epigenetic features in SLs, it has been reported that silencing of MLH1 plays an important role in the progression of SLs, especially with the BRAF pathogenic variant [4, 16], but in our IHC study, no deficiency of MLH1 protein could be seen among SLs in two patients with SPS. Apparently, this result does not agree with a previous report, but it is not clear whether MLH1 was silenced to completely suppress the expression of MLH1 protein. Moreover, it must be noted that previous clinical reports have demonstrated that deficient-MMR has not been identified in HPs, TSAs, or SSLs, but has been reported in SSL with dysplasia (SSLD) only [21, 27]. Additionally, SSLD is the only pre-cancerous colorectal lesion in which MLH1 is methylated [28]. Although low sensitivity of the IHC cannot be excluded, it is possible that MLH1 may have not been methylated yet in SSLs without dysplasia. Regarding the occurrence of deficient MMR, patients with pre-cancerous lesions, especially with SSLD, require careful surveillance after resection.

In conclusion, the identification of a pure somatic pathogenic variant of BRAF (c.1799 T > A, p.Val600Glu), which was observed among SLs with an identical germline genetic and environmental background, highlights the importance of this variant as a strong contributor for SLs.

### Limitations

The present study has several limitations: i) IHC for DNA MMR was performed only for pre-cancerous lesions and not cancerous lesions, because cancer was not found in the two analyzed SPS patients; ii) the analyzed number of patients with SPS was small; and iii) the methylation profile was not evaluated. These findings require further investigation in future studies.

### Supplementary Information

Additional file 1: Table S1. DNA variant profile (Case #1).Additional file 2: Table S2. DNA variant profile (Case #2).Additional file 3: Table S3. Mutational signature of the single base substitutions (SBS).Additional file 4: Table S4. Mutational signature of the single base substitution.

Additional file 1: Table S1. DNA variant profile (Case #1).

Additional file 2: Table S2. DNA variant profile (Case #2).

Additional file 3: Table S3. Mutational signature of the single base substitutions (SBS).

Additional file 4: Table S4. Mutational signature of the single base substitution.

Additional file 1: Table S1. DNA variant profile (Case #1).Additional file 2: Table S2. DNA variant profile (Case #2).Additional file 3: Table S3. Mutational signature of the single base substitutions (SBS).Additional file 4: Table S4. Mutational signature of the single base substitution.

Additional file 1: Table S1. DNA variant profile (Case #1).

Additional file 2: Table S2. DNA variant profile (Case #2).

Additional file 3: Table S3. Mutational signature of the single base substitutions (SBS).

Additional file 4: Table S4. Mutational signature of the single base substitution.



# SUPPLEMENTAL FILE 1: 13104_2022_Article_6245.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)