# MAIN TEXT

## Sequencing-based functional assays for classification of BRCA2 variants in mouse ESCs

### Abstract

SummarySequencing of genes, such as BRCA1 and BRCA2, is recommended for individuals with a personal or family history of early onset and/or bilateral breast and/or ovarian cancer or a history of male breast cancer. Such sequencing efforts have resulted in the identification of more than 17,000 BRCA2 variants. The functional significance of most variants remains unknown; consequently, they are called variants of uncertain clinical significance (VUSs). We have previously developed mouse embryonic stem cell (mESC)-based assays for functional classification of BRCA2 variants. We now developed a next-generation sequencing (NGS)-based approach for functional evaluation of BRCA2 variants using pools of mESCs expressing 10–25 BRCA2 variants from a given exon. We use this approach for functional evaluation of 223 variants listed in ClinVar. Our functional classification of BRCA2 variants is concordant with the classification reported in ClinVar or those reported by other orthogonal assays.

### Introduction

BRCA2 is one of the frequently mutated genes in the general population (1 mutation in 1,000 unaffected individuals).1 Germline pathogenic variants of BRCA2 are associated with increased risk of breast, ovarian, prostate, and pancreatic cancer.2 Clinical management of individuals with a family history of early-onset breast or ovarian cancer or a history of male breast cancer includes sequence-based genetic testing of BRCA1 and BRCA2. Identification of individuals carrying a pathogenic BRCA variant can lead to better cancer surveillance, prevention, and therapeutic options.3 Sequencing-based genetic testing has resulted in the identification of more than 17,000 BRCA2 variants that are listed in the ClinVar database (https://www.ncbi.nlm.nih.gov/clinvar/). More than 3,000 of these variants are considered to be variants of uncertain significance (VUSs) because their association with the disease is unknown. Individuals carrying a VUS must cope with the uncertainty of clinical management. Majority (89.5%) of the VUSs reported in the ClinVar database are missense variants, and the functional consequences of these variants with a single amino acid change remains unknown. Many of those VUSs are relatively rare in the general population and in cancer patients. This makes it difficult to reliably classify them using population-based studies, including multifactorial models of pathology, co-occurrence with cancer, and co-segregation data.

The American College of Medical Genetics and Genomics (ACMG) has developed guidelines to classify variants based on criteria of evidence from population data, in silico predictions, functional data, and segregation analyses. The ACMG has recommended a five-tier classification system to classify variants: benign, likely benign, uncertain, pathogenic, and likely pathogenic.4 According to the ACMG guidelines, a well-established functional assay to determine the impact of a mutation on the function is regarded as strong evidence to classify the variants.5 The Evidence-Based Network for the Interpretation of Germline Mutant Alleles (ENIGMA) expert panel and ACMG determined that the variants should be classified as functionally normal or functionally abnormal based on their impact on function in a functional assay. The functionally abnormal variants can be further classified as complete loss of function, partial loss of function/intermediate, gain of function, or dominant negative.6

Several assays, including our mouse embryonic stem cell (mESC)-based assay, have been developed in the recent years to evaluate the functional significance of BRCA2 variants.7,8,9,10,11,12,13,14,15,16,17,18 Many of these assays have been used to analyze hundreds of BRCA2 variants by improving throughput and often classifying the variants using a computational model to assess the functional impact of the variants.10,12,19 The mESC-based assay is a comprehensive assay to evaluate BRCA2 variants.7,13 In this assay, individual variants are generated by recombineering in a bacterial artificial chromosome (BAC) clone containing full-length human BRCA2. The individual BAC clone encoding a single BRCA2 variant is then expressed in Brca2cko/ko mESCs and examined for its ability to complement the loss of Brca2 in mESCs by assessing cell viability and sensitivity to different DNA-damaging agents7,8,13,14,16,17,18 (Figure 1A). We have recently used a statistical algorithm to predict the probabilities on impact of function (PIFs) using the data from the mESC-based assay.9 However, generating experimental data using an mESC-based assay is time consuming and laborious. Consequently, very few VUSs can be analyzed at any given time.Figure 1Schematic representation of the multiplexed mESC-based assay(A) Schematic representation of the mESC-based functional assay. The PL2F7 mESCs (containing a conditional allele containing two loxP sites and a knockout allele of Brca2) is complemented for the loss of Brca2 allele after Cre expression by the BAC DNA encoding the human BRCA2 gene containing one variant. The recombinants were selected in hypoxanthine, aminopterin, thymidine (HAT)-containing medium. Viable HATr cells show no impact of BRCA2 variants on function. Viable HATr cells were further tested to distinguish the variants that have moderate loss of function by their sensitivity to different DNA-damaging agents. A star in the BAC construct represents the variant. Solid arrows denote loxP sites, and the two halves of the Hypoxanthine-guanine phosphoribosyltransferase (HPRT) mini gene are marked in solid boxes as HP and RT on the conditional allele of Brca2.(B) Selection of two independent clones of mESCs that express BRCA2 variants. After introduction of BAC into PL2F7 cells, transfected cells were selected and analyzed for protein expression. Two clones were selected from each variant to eliminate positional bias of BAC integration.(C) Multiplexing of the downstream process by mixing mESC clones expressing different variants. After mixing, two independent experiments were performed from each mix, and at the indicated steps, samples were collected for DNA isolation and deep sequencing.(D) Schematic representation of loss of BRCA2 variants after selection with HAT, cisplatin, or olaparib. Genomic DNA was isolated, PCR amplified, and deep sequenced to quantify the relative abundance of variants. The ratio of abundance was normalized to the no-drug (M15) control. The relative viability data were further analyzed using the Bayesian statistical model to determine the probability of impact on function of each variant. Oval colored shapes represent cells with a variant, and solid-colored rectangles represent sequence reads.

Schematic representation of the multiplexed mESC-based assay

(A) Schematic representation of the mESC-based functional assay. The PL2F7 mESCs (containing a conditional allele containing two loxP sites and a knockout allele of Brca2) is complemented for the loss of Brca2 allele after Cre expression by the BAC DNA encoding the human BRCA2 gene containing one variant. The recombinants were selected in hypoxanthine, aminopterin, thymidine (HAT)-containing medium. Viable HATr cells show no impact of BRCA2 variants on function. Viable HATr cells were further tested to distinguish the variants that have moderate loss of function by their sensitivity to different DNA-damaging agents. A star in the BAC construct represents the variant. Solid arrows denote loxP sites, and the two halves of the Hypoxanthine-guanine phosphoribosyltransferase (HPRT) mini gene are marked in solid boxes as HP and RT on the conditional allele of Brca2.

(B) Selection of two independent clones of mESCs that express BRCA2 variants. After introduction of BAC into PL2F7 cells, transfected cells were selected and analyzed for protein expression. Two clones were selected from each variant to eliminate positional bias of BAC integration.

(C) Multiplexing of the downstream process by mixing mESC clones expressing different variants. After mixing, two independent experiments were performed from each mix, and at the indicated steps, samples were collected for DNA isolation and deep sequencing.

(D) Schematic representation of loss of BRCA2 variants after selection with HAT, cisplatin, or olaparib. Genomic DNA was isolated, PCR amplified, and deep sequenced to quantify the relative abundance of variants. The ratio of abundance was normalized to the no-drug (M15) control. The relative viability data were further analyzed using the Bayesian statistical model to determine the probability of impact on function of each variant. Oval colored shapes represent cells with a variant, and solid-colored rectangles represent sequence reads.

Here, we describe an experimental approach that improves the throughput of the mESC-based assay using next-generation sequencing (NGS). We further used the cell viability and drug (cisplatin [a DNA interstrand crosslinker] and olaparib [a poly-ADP ribose polymerase (PARP) inhibitor, (PARPi)]) sensitivity data from NGS to generate PIF values of BRCA2 VUSs using a statistical model. We multiplexed the mESC-based assay at two steps: (1) generation of multiple variants of the same exon using recombineering20 and (2) multiplexing variants of the same exon for cell viability and drug sensitivity assays. The BAC clones with variants were individually transfected into Pl2F7 (Brca2cko/ko) cells. We then selected the clones that expressed the BRCA2 protein13 (Figure 1B). Nonsense variants, predicted to encode a truncated protein, were screened by RT-PCR. The mESC clones expressing BRCA2 variants from the same exons were then pooled and transfected with Pgk-Cre to induce loss of the conditional Brca2 allele. Cells undergoing Cre Recombinase (CRE)-mediated recombination were then selected in a medium containing hypoxanthine-aminopterin-thymidine (HAT).13 After HAT selection, the pool of cells was subjected to olaparib and cisplatin for a drug sensitivity assay (Figure 1C). Cell viability in response to loss of the Brca2 conditional allele or to the drug treatment was determined by the presence of a variant allele detected by NGS (Figure 1D). Further, the functional score calculated from the assay for 223 variants was used to build a statistical model to generate the PIF and interpret the functional impact of the variants.

### Results

We selected 223 BRCA2 variants listed in the ClinVar database that impact residues encoded by exons 11 (65 variants), 14 (52 variants), 20 (19 variants), 21 (50 variants), and 25 (37 variants) (Table 1). Among these variants, 32 are nonsense variants that are classified as pathogenic in ClinVar (Table 1). Sixteen variants classified as benign or likely benign in ClinVar were used as neutral controls (Table 1). All control variants are in different exons. Sixty-five variants are in exon 11 of BRCA2, the 3,387-bp exon that codes for the region containing 8 BRC repeats. Among the selected variants in exon 11, eight are located in the BRC1 repeat, and the remaining are in the region flanking the BRC1 repeat (Figure S1). All selected variants from exon 14 are located in the region preceding the DNA binding domain of BRCA2, whereas the variants from exons 20, 21, and 25 are located in the Oligonucleotides/oligosaccharides binding domains, OB2 or OB3 region of the DNA binding domain of BRCA2 (Figure S1). Six variants are present in introns 20 and 21 (Figure S1).Table 1Classification of variants using the mESC-based assayHGVS nucleotideHGVS proteinExonClinVarESC assayHAT FSCis FSOla FSHAT PIFCis PIFOla PIFCombined PIFClassc.7022G>Tp.Arg2341Leu14US−0.2140−0.7767−0.86360.00140.04510.02280.00238888Fc.7021C>Gp.Arg2341Gly14US−0.0298−0.4448−0.44060.00070.02190.00680.00083833Fc.7078T>Cp.Ser2360Pro14US−0.0611−0.0113−0.10990.00080.01030.00310.0008022Fc.7040C>Ap.Pro2347Gln14US0.1133−0.3929−0.35910.00040.01980.00560.00053352Fc.7082A>Gp.His2361Arg14US0.18760.07510.12130.00030.00910.00200.00035197Fc.7069C>Gp.Leu2357Val14US0.1014−0.1810−0.18740.00040.01350.00370.00049037Fc.7010C>Tp.Thr2337Ile14US0.0353−0.3679−0.38350.00050.01890.00590.00066076Fc.7033C>Gp.Gln2345Glu14US−0.1095−0.5951−0.69370.00090.02990.01370.00132712Fc.7042A>Cp.Asn2348His14US0.25230.38260.22060.00030.00640.00170.00028432Fc.7049C>Tp.Thr2350Ile14US0.2557−0.1121−0.18810.00030.01210.00370.00031581Fc.7045T>Cp.Phe2349Leu14US−4.8694−5.3479−5.51311.00001.00001.00001NFc.7022G>Ap.Arg2341His14US0.0016−0.1774−0.15320.00060.01340.00350.00066299Fc.7066T>Gp.Phe2356Val14US0.48260.66870.72900.00010.00510.00090.00014814Fc.7067T>Ap.Phe2356Tyr14US0.0292−0.4491−0.44050.00060.02210.00680.00071176Fc.7037A>Gp.Asn2346Ser14US−0.0138−0.03950.06400.00070.01080.00220.00067466Fc.7051G>Tp.Ala2351Ser14US0.10690.71910.79940.00040.00490.00080.00043611Fc.7034A>Gp.Gln2345Arg14US−0.3732−0.5543−0.88010.00260.02740.02400.00324057Fc.7030A>Gp.Ile2344Val14US−0.1915−0.2654−0.26830.00120.01560.00450.00131887Fc.7017G>Cp.Lys2339Asn14B−0.2044−0.1943−0.10840.00130.01380.00310.00135582Fc.7027G>Ap.Glu2343Lys14US−0.5738−0.1653−0.15400.00620.01320.00350.0062201Fc.7051G>Cp.Ala2351Pro14US−0.3355−0.4772−0.52800.00220.02340.00860.00241123Fc.7072T>Cp.Ser2358Pro14US0.03490.59210.72000.00060.00530.00090.00055467Fc.C7021Tp.Arg2341Cys14US−0.22950.08490.02440.00140.00900.00240.00146803Fc.7025A>Gp.Gln2342Arg14US−0.34780.26510.28190.00230.00720.00150.00233593Fc.7081C>Tp.His2361Tyr14US−0.3196−0.3121−0.12470.00210.01700.00320.00212651Fc.3170A>Gp.Lys1057Arg11US0.45490.87790.25460.00020.00460.00160.00016178Fc.3206C>Tp.Ser1069Phe11US−0.01210.5133−0.08540.00060.00570.00300.00066375Fc.3256A>Gp.Ile1086Val11CIP0.52430.48440.24580.00010.00580.00160.00013868Fc.3310A>Cp.Thr1104Pro11US0.4720−0.06270.01470.00010.01120.00240.00017503Fc.3319C>Tp.Gln1107Stop11P−3.5779−4.1744−4.14901.00001.00001.00001NFc.3395A>Gp.Lys1132Arg11US0.53091.16540.49360.00010.00430.00110.00013209Fc.3403T>Cp.Tyr1135His11US−0.4727−0.2659−0.12370.00390.01560.00320.00399517Fc.3413A>Tp.Gln1138Leu11US−0.0826−0.1557−0.17970.00080.01290.00370.00087922Fc.3419G>Cp.Ser1140Thr11US−0.06842.76840.93490.00080.02140.00070.00080588Fc.3446T>Cp.Met1149Thr11US0.51850.77780.34490.00010.00480.00140.00013781Fc.3458A>Gp.Lys1153Arg11CIP0.22781.10510.45660.00030.00430.00120.00029983Fc.3469G>Tp.Glu1157Stop11P−5.1118−6.1137−6.34121.00001.00001.00001NFc.3475T>Ap.Cys1159Ser110.46561.31190.61740.00020.00440.00100.00015452Fc.3499A>Gp.Ile1167Val11US−0.13090.16160.07750.00100.00820.00220.0010112Fc.3509C>Tp.Ala1170Val11CIP1.10740.16530.51770.00000.00810.00115.0152E-05Fc.3515C>Gp.Ser1172Trp11US0.94920.80250.10950.00010.00470.00206.2456E-05Fc.9458G>Cp.Gly3153Ala25CIP1.21321.73361.65170.00000.00530.00073.9357E-05Fc.9401G>Tp.Gly3134Val25US0.99601.96342.00110.00000.00640.00085.4015E-05Fc.9307A>Gp.Ile3103Val25US1.17451.58361.69280.00000.00480.00074.0886E-05Fc.9275A>Gp.Tyr3092Cys25CIP0.94070.92711.18630.00010.00450.00065.654E-05Fc.9263C>Tp.Ala3088Val25US1.23761.34731.59950.00000.00440.00063.7711E-05Fc.9375C>Gp.Leu3125 =25LB1.35941.39591.40450.00000.00450.00063.3459E-05Fc.9477C>Ap.Phe3159Leu25CIP1.22131.39381.57440.00000.00450.00063.8411E-05Fc.9275A>Cp.Tyr3092Ser25US0.3896−1.5691−1.10560.00020.32880.04990.01659796Fc.9449C>Tp.Pro3150Leu25US0.0503−1.7698−0.96060.00050.50130.03100.01605649Fc.9356T>Gp.Leu3119Stop25P−3.6916−3.6811−3.37571.00000.99990.99981NFc.9500A>Cp.Glu3167Ala25US2.43922.54512.44580.00000.01410.00135.1474E-05Fc.9285C>Gp.Asp3095Glu25P/LP−3.8228−5.0215−4.92081.00001.00001.00001NFc.9374T>Ap.Leu3125His25LP−4.1949−5.9723−5.91961.00001.00001.00001NFc.9294C>Gp.Tyr3098Stop25P−3.1644−3.7933−3.58011.00001.00001.00001NFc.9302T>Gp.Leu3101Arg25CIP−3.0114−5.1579−4.44681.00001.00001.00001NFc.9481A>Tp.Lys3161Stop25P−3.3027−3.2420−3.22041.00000.99890.99951NFc.9455A>Gp.Glu3152Gly25US−4.0623−4.8274−4.15321.00001.00001.00001NFc.2813C>Ap.Ala938Glu11CIP−0.6316−0.7041−0.37130.00800.03810.00570.00825901Fc.3085A>Gp.Met1029Val11US0.28740.37940.36550.00020.00640.00130.00025506Fc.3075G>Tp.Lys1025Asn11US0.52700.48290.40880.00010.00580.00130.00013577Fc.2786T>Cp.Leu929Ser11B0.39330.25430.19750.00020.00730.00170.00019514Fc.2987T>Gp.Leu996Arg11B0.86170.69700.61410.00010.00500.00106.6665E-05Fc.2965T>Gp.Tyr989Asp11US−0.5989−0.8827−0.50990.00690.05810.00820.00739321Fc.2798C>Gp.Thr933Arg11US0.36250.28280.23330.00020.00710.00160.0002103Fc.2849T>Ap.Val950Asp11CIP0.61140.96570.88700.00010.00440.00080.0001081Fc.2803G>Cp.Asp935His11B−0.0744−0.3379−0.44630.00080.01780.00690.00093119Fc.2927C>Tp.Ser976Phe11B/LB0.3687−0.2071−0.46070.00020.01410.00720.00029688Fc.3073A>Gp.Lys1025Glu11CIP−0.6727−0.7125−0.47280.00970.03890.00740.010024Fc.2944A>Cp.Ile982Leu11CIP0.73770.41800.28040.00010.00620.00158.8507E-05Fc.2979G>Ap.Trp993Stop11P−4.8245−6.1596−6.42921.00001.00001.00001NFc.3142G>Ap.Val1048Ile11US0.48540.71020.55390.00010.00490.00100.00014791Fc.3166C>Tp.Gln1056Stop11P−3.2693−2.4294−2.71271.00000.93840.98920.99999994NFc.3088T>Gp.Phe1030Val11CIP0.52980.02400.66110.00010.00980.00090.00013664Fc.2836G>Cp.Asp946His11US0.65960.94790.84670.00010.00450.00089.7257E-05Fc.2899C>Gp.Leu967Val11US0.6510−0.18070.08670.00010.01350.00210.00012434Fc.3092T>Cp.Phe1031Ser11US0.5260−0.5788−0.47470.00010.02890.00750.00034466Fc.2926T>Ap.Ser976Thr11B/LB0.75740.44050.60340.00010.00600.00108.1881E-05Fc.3122G>Ap.Ser1041Asn11US0.59440.82110.82850.00010.00470.00080.00011271Fc.3137A>Gp.Glu1046Gly11CIP0.0539−0.3310−0.14310.00050.01760.00340.000575Fc.2854G>Tp.Ala952Ser11US0.80830.22440.44040.00010.00760.00127.7641E-05Fc.3109C>Tp.Gln1037Stop11P−4.4515−5.5910−5.56691.00001.00001.00001NFc.3515C>Tp.Ser1172Leu11B0.00070.81510.65680.00060.00470.00090.00062294Fc.3503T>Ap.Met1168Lys11US0.88170.58520.69970.00010.00540.00096.4338E-05Fc.3367A>Gp.Ser1123Gly11US0.94310.87150.96870.00010.00460.00075.6687E-05Fc.3437A>Gp.Glu1146Gly11US0.94900.58470.76850.00010.00540.00085.7345E-05Fc.3362C>Ap.Ser1121Stop11P−3.1092−4.1491−4.36501.00001.00001.00001NFc.3265C>Tp.Gln1089Stop11P−3.0456−4.5726−4.61211.00001.00001.00001NFc.3539A>Gp.Lys1180Arg11CIP0.80010.26320.47440.00010.00720.00127.7971E-05Fc.3262C>Tp.Pro1088Ser11CIP0.40080.45170.24310.00020.00600.00160.00018833Fc.3526G>Ap.Val1176Ile11US0.29020.65210.18060.00020.00510.00180.00025369Fc.3103G>Tp.Glu1035Stop11P−4.6635−5.2485−5.38761.00001.00001.00001NFc.7102T>Gp.Leu2368Val14CIP0.56050.38580.39380.00010.00630.00130.00012646Fc.7142C>Tp.Pro2381Leu14US−0.0513−0.09180.11500.00070.01170.00200.00076683Fc.7025A>Cp.Gln2342Pro14US−0.0644−0.1821−0.27590.00080.01350.00460.00084049Fc.7009A>Gp.Thr2337Ala14US−0.3626−0.5921−0.71560.00250.02970.01460.00290388Fc.7100C>Tp.Thr2367Ile14US−0.25390.76000.73630.00160.00480.00090.00159616Fc.7150C>Gp.Gln2384Glu14US0.74740.60180.60460.00010.00530.00108.2728E-05Fc.7133C>Gp.Ser2378Stop14P−3.6377−4.6589−4.84371.00001.00001.00001NFc.7095T>Ap.His2365Gln14US−0.10410.43520.36000.00090.00610.00140.00090799Fc.7139A>Gp.His2380Arg14US0.57620.27250.27270.00010.00720.00150.00012491Fc.7039C>Gp.Pro2347Ala14US0.31550.22420.22790.00020.00760.00170.00023964Fc.7093C>Tp.His2365Tyr14US0.03060.07140.00700.00060.00920.00250.0005808Fc.7147T>Cp.Tyr2383His14US−2.6611−3.0326−3.17490.99980.99660.99930.99999917NFc.7138C>Tp.His2380Tyr14US0.20680.04540.04540.00030.00950.00230.00033626Fc.7107A>Cp.Glu2369Asp14US−3.1135−4.0509−4.27921.00001.00001.00001NFc.7142C>Ap.Pro2381Gln14US−1.3017−2.1833−2.55700.21840.83830.97440.8568118Ic.7073C>Gp.Ser2358Cys14US0.1058−0.0417−0.01930.00040.01080.00260.00046198Fc.7115C>Gp.Ser2372Stop14P−2.7216−3.5670−4.14430.99990.99981.00000.99999998NFc.7028A>Tp.Glu2343Val14US0.35330.36320.35950.00020.00650.00140.00021264Fc.7126G>Cp.Ala2376Pro14US−0.5944−0.9135−0.71310.00680.06260.01450.00768141Fc.7136G>Tp.Gly2379Val14US−5.4959−5.6089−6.16011.00001.00001.00001NFc.7121A>Gp.Asn2374Ser14CIP0.0312−0.3099−0.25010.00060.01690.00430.00062956Fc.7136G>Ap.Gly2379Glu14US−1.6839−3.5697−3.72990.75380.99981.00000.9999561NFc.8657C>Gp.Pro2886Arg210.0958−0.7838−0.54460.00040.04580.00900.0008604Fc.8679G>Tp.Gln2893His21US0.2385−0.23110.32080.00030.01470.00140.00030634Fc.8672C>Gp.Thr2891Arg21US−0.0015−0.2788−0.01830.00060.01600.00260.00066501Fc.8681A>Gp.Gln2894Arg21US−0.4399−1.5203−0.92340.00340.29310.02750.01146638Fc.8651A>Tp.Tyr2884Phe21US−0.0402−0.29310.02780.00070.01640.00240.00075337Fc.8710C>Tp.Leu2904Phe21US−0.3337−2.8621−1.94160.00220.99190.61950.61527078Ic.8686C>Tp.Arg2896Cys21US−0.3156−1.0178−0.52790.00200.08130.00860.00273603Fc.8651A>Gp.Tyr2884Cys21CIP−0.0501−0.8476−0.36350.00070.05330.00560.00103992Fc.8663G>Tp.Arg2888Leu21US−0.1668−0.6515−0.32790.00110.03390.00520.00131122Fc.8680C>Tp.Gln2894Stop21P−5.0250−5.9901−5.43011.00001.00001.00001NFc.8707G>Tp.Glu2903Stop21P−5.9357−7.5663−6.65601.00001.00001.00001NFc.8665G>Tp.Ala2889Ser21US0.64341.70821.12820.00010.00520.00070.00010071Fc.8633-1G>AIntronP/LP−5.3153−5.5640−5.29941.00001.00001.00001NFc.8704G>Ap.Ala2902Thr21US0.4549−1.1871−0.41890.00020.12560.00650.00096687Fc.8687G>Ap.Arg2896His21CIP−0.2058−0.8948−0.58290.00130.05980.01000.00191661Fc.8722G>Ap.Val2908Met21US−0.0797−0.7944−0.46950.00080.04700.00740.00116919Fc.8690C>Gp.Ala2897Gly21US−0.0893−0.8324−0.44010.00090.05140.00680.00120308Fc.8663G>Ap.Arg2888His21US0.49291.12851.12050.00010.00430.00070.00014291Fc.8662C>Tp.Arg2888Cys21B0.1056−0.27580.03690.00040.01590.00230.00047119Fc.8639C>Gp.Thr2880Arg21US−0.1524−0.4348−0.24050.00110.02150.00420.00116683Fc.8668C>Ap.Leu2890Ile21CIP−0.5353−2.0841−1.31810.00520.77420.10290.08441721Ic.8663G>Cp.Arg2888Pro21US−0.0962−0.0197−0.06260.00090.01050.00280.00090399Fc.8714A>Gp.Tyr2905Cys21US−0.0733−1.5721−1.14940.00080.33120.05790.01994992Fc.8702G>Tp.Gly2901Val21CIP−0.0639−1.6055−1.05530.00080.35740.04220.01586066Fc.8708A>Gp.Glu2903Gly21US0.2096−0.2141−0.19200.00030.01430.00380.00036535Fc.8732C>Tp.Ala2911Val21US−0.04230.56440.41600.00070.00540.00120.00072643Fc.8737G>Ap.Asp2913Asn21US0.0662−0.3298−0.32590.00050.01760.00510.00058478Fc.8746T>Ap.Tyr2916Asn21US0.12400.04120.06570.00040.00960.00220.0004299Fc.8732C>Ap.Ala2911Glu21P0.55650.49660.58660.00010.00570.00100.00012521Fc.8737G>Cp.Asp2913His21US0.29360.38920.39130.00020.00630.00130.00025023Fc.8744C>Tp.Ala2915Val21US0.38410.39750.46090.00020.00630.00120.00019441Fc.8644A>Tp.Lys2882Stop21P−5.0381−6.6902−6.33561.00001.00001.00001NFc.8752G>Cp.Glu2918Gln21US0.26010.21590.34730.00030.00760.00140.00027783Fc.8754 + 2T>GIntronP−4.4425−8.5100−8.58231.00001.00001.00001NFc.8633-2A>GIntronP−6.1292−7.9408−7.08951.00001.00001.00001NFc.8738A>Tp.Asp2913Val21US−0.35070.4720−0.44000.00240.00590.00680.00239288Fc.8677C>Tp.Gln2893Stop21P−6.3245−8.7391−9.43221.00001.00001.00001NFc.8695C>Tp.Gln2899Stop21P−5.9213−8.2198−7.49671.00001.00001.00001NFc.8754 + 1G>TIntronP/LP−6.1534−9.3828−7.89631.00001.00001.00001NFc.8646A>Tp.Lys2882Asn21US0.11950.39450.38090.00040.00630.00130.00042298Fc.8633-2A>TIntronP−5.0434−6.9790−7.23121.00001.00001.00001NFc.8754 + 1G>AIntronP−6.0750−7.4268−6.98581.00001.00001.00001NFc.9301C>Gp.Leu3101Val25US0.74951.02481.11940.00010.00440.00078.0084E-05Fc.9271G>Ap.Val3091Ile25CIP1.92551.94171.88300.00000.00630.00072.8814E-05Fc.9414A>Gp.Leu3138 =25LB1.90711.76831.66610.00000.00540.00072.7722E-05Fc.9367A>Gp.Ser3123Gly25US1.09171.94321.55690.00000.00630.00064.6219E-05Fc.9292T>Cp.Tyr3098His25B1.31831.52671.54870.00000.00470.00063.4899E-05Fc.9350A>Cp.His3117Pro25US1.04661.08601.35830.00000.00430.00064.7865E-05Fc.9501G>Ap.Glu3167 =251.69191.43481.45650.00000.00450.00062.7661E-05Fc.9466C>Tp.Gln3156Stop25P−4.2907−6.1321−5.22031.00001.00001.00001NFc.9376C>Tp.Gln3126Stop25P−4.1324−4.7481−4.44961.00001.00001.00001NFc.9371A>Tp.Asn3124Ile25P−3.9263−7.6510−6.48421.00001.00001.00001NFc.8539G>Ap.Glu2847Lys20CIP−0.9734−4.4171−4.58930.04251.00001.00000.99999935NFc.8593T>Gp.Leu2865Val20CIP1.05980.81850.83290.00000.00470.00084.7936E-05Fc.8575C>Tp.Gln2859Stop20P−2.7456−6.2237−6.16330.99991.00001.00001NFc.8618T>Gp.Phe2873Cys20US1.25911.33251.39070.00000.00440.00063.6782E-05Fc.8503T>Cp.Ser2835Pro20B1.68131.35911.38280.00000.00440.00062.7686E-05Fc.8573A>Gp.Gln2858Arg20CIP0.87181.14061.17640.00010.00430.00066.349E-05Fc.8548G>Ap.Glu2850Lys20US1.07690.78840.78260.00000.00470.00084.7041E-05Fc.8518A>Gp.Ile2840Val20US1.51071.05921.09040.00000.00440.00073.0081E-05Fc.8525G>Ap.Arg2842His20B1.4195−0.2924−0.42820.00000.01640.00660.00013772Fc.8572C>Ap.Gln2858Lys20CIP1.04251.20391.25580.00000.00430.00064.818E-05Fc.8525G>Tp.Arg2842Leu20CIP0.6106−1.0058−1.18000.00010.07880.06420.00516208Fc.8599A>Cp.Thr2867Pro20US1.05080.79400.78470.00000.00470.00084.8739E-05Fc.8591C>Tp.Ala2864Val20US0.68140.39220.34340.00010.00630.00149.8065E-05Fc.8629G>Tp.Glu2877Stop20P−4.2046−4.8618−4.87291.00001.00001.00001NFc.8504C>Ap.Ser2835Stop20P−3.2603−3.4495−3.63581.00000.99971.00001NFc.8567A>Cp.Glu2856Ala20CIP1.29200.89970.97750.00000.00450.00073.6047E-05Fc.7068T>Gp.Phe2356Leu14US0.0051−1.4464−1.72470.00060.24460.37810.09306036Ic.3299A>Tp.Asn1100Ile11CIP0.15781.16380.10820.00040.00430.00200.00037551Fc.3362C>Gp.Ser1121Stop11P−4.0407−4.7562−5.28771.00001.00001.00001NFc.3496G>Ap.Val1166Ile11US−0.3355−0.2721−0.34440.00220.01580.00540.00229499Fc.9425A>Gp.Asp3142Gly25US1.25291.11271.18230.00000.00430.00063.708E-05Fc.9385C>Gp.Pro3129Ala25US0.81411.15960.94350.00010.00430.00077.0883E-05Fc.9276T>Gp.Tyr3092Stop25P−3.5541−3.8671−3.90631.00001.00001.00001NFc.9382C>Tp.Arg3128Stop25P−4.1565−4.7846−3.96421.00001.00001.00001NFc.9317G>Ap.Trp3106Stop25P−2.3918−2.4653−2.31960.99810.94720.91380.99974674NFc.9309A>Gp.Ile3103Met25US1.11990.21470.10920.00000.00760.00205.6182E-05Fc.2971A>Gp.Asn991Asp11B1.64901.52172.24750.00000.00470.00102.9989E-05Fc.2881C>Tp.Gln961Stop11P−4.8245−5.9455−6.75921.00001.00001.00001NFc.2818C>Tp.Gln940Stop11P−7.1483−7.0299−8.11131.00001.00001.00001NFc.3076A>Tp.Lys1026Stop11P−5.6613−8.4729−8.19211.00001.00001.00001NFc.3032C>Gp.Thr1011Arg11CIP0.92360.36610.41840.00010.00650.00126.3314E-05Fc.3132T>Gp.Cys1044Trp11US0.79060.34350.21610.00010.00660.00178.2198E-05Fc.2872A>Gp.Ser958Gly11US0.65810.83780.80540.00010.00460.00089.7826E-05Fc.2803G>Ap.Asp935Asn11B/LB0.57350.14040.40890.00010.00840.00130.00012519Fc.3270G>Cp.Met1090Ile11US2.50231.13082.29570.00000.00430.00113.9885E-05Fc.3304A>Tp.Asn1102Tyr11B0.5268−0.3602−0.05900.00010.01860.00280.0001811Fc.7141C>Tp.Pro2381Ser14US0.19340.20640.14410.00030.00770.00190.00034255Fc.7141C>Gp.Pro2381Ala14US0.2626−0.0195−0.02830.00030.01050.00270.00029313Fc.7119C>Gp.Ser2373Arg14US0.14170.18740.08880.00040.00790.00210.00040277Fc.7115C>Ap.Ser2372Stop14P−1.9846−5.4167−5.56990.96061.00001.00001NFc.7106A>Gp.Glu2369Gly14US−0.5600−1.3960−1.11400.00580.21550.05140.0168051Fc.8702G>Ap.Gly2901Asp21CIP−0.0512−0.7515−0.98490.00070.04250.03350.00216736Fc.8692T>Gp.Leu2898Val21US−0.8861−2.5100−1.43430.02740.95670.15280.16956546Ic.8739C>Gp.Asp2913Glu21US0.76811.20350.85670.00010.00430.00087.7631E-05Fc.8699A>Tp.Asp2900Val21US−0.09180.58540.51010.00090.00540.00110.00086614Fc.8740C>Gp.Pro2914Ala21US0.59991.02491.01700.00010.00440.00070.00011065Fc.8678A>Gp.Gln2893Arg21US0.18130.17160.41550.00030.00810.00120.00035048Fc.8656C>Ap.Pro2886Thr21US0.2438−0.1361−0.26280.00030.01250.00440.00033615Fc.2837A>Gp.Asp946Gly11CIP0.44080.45130.24640.00020.00600.00160.00017005Fc.2771A>Tp.Asn924Ile11CIP−0.45800.59800.60230.00370.00530.00100.00370591Fc.8647C>Tp.Pro2883Ser21CIP−6.4547−10.133−9.10981.00001.00001.00001NFc.9353T>Cp.Met3118Thr25CIP−0.8240−0.8750−0.22290.02010.05700.00400.02037364Fc.9454G>Ap.Glu3152Lys25CIP1.68992.06131.84250.00000.00710.00072.9922E-05Fc.9380G>Ap.Trp3127Stop25P−5.8464−7.9743−7.74841.00001.00001.00001NFc.8572C>Tp.Gln2858Stop20P−4.3478−6.3224−6.25831.00001.00001.00001NFc.8594T>Gp.Leu2865Stop20P−6.6953−8.8540−8.13871.00001.00001.00001NFc.8489G>Ap.Trp2830Stop20P−4.6542−3.6163−3.72871.00000.99991.00001NFFS, functional score; PIF, probability of impact on function; Cis, cisplatin; Ola, olaparib; F, functional; NF, non-functional; I, indeterminate; B, benign; LB, likely benign; P, pathogenic; LP, likely pathogenic; US, uncertain significance; CIP, conflicting interpretation of pathogenicity.

Classification of variants using the mESC-based assay

FS, functional score; PIF, probability of impact on function; Cis, cisplatin; Ola, olaparib; F, functional; NF, non-functional; I, indeterminate; B, benign; LB, likely benign; P, pathogenic; LP, likely pathogenic; US, uncertain significance; CIP, conflicting interpretation of pathogenicity.

Each variant was generated by recombineering in a BAC clone (CTD-2342K5 with a 127-kb insert) containing full-length BRCA2.8,20 All BACs were sequenced to confirm the presence of the desired mutation and the lack of any undesired mutations in that region. For most of the variants, two independent mouse embryonic stem cells (PL2F7) clones expressing full-length BRCA2 were selected for further analysis. A single clone was used for variants where two equally expressing clones were not obtained. Subsequently, mESC clones expressing variants from the same exons were pooled together.

Two pools of mESCs were generated for each batch of exon using two independently generated BRCA2-variant expressing clones. Cell samples from each pool were collected before Pgk-Cre expression (termed “M15”), after HAT selection of recombinant clones (termed “HAT”), and after the selection of recombinant clones in the presence of olaparib or cisplatin. The genomic DNA obtained from collected cell samples was used to PCR amplify the genomic region where the variants are located and sequenced using the Illumina Mi-Seq platform.

The frequency of each variant was calculated as described in the STAR Methods. The frequency of variants in the M15 sample followed normal distribution consisting of a bell-shaped curve (Figure S2). The frequency of variants based on HAT, cisplatin, and olaparib treatment were used to calculate the functional score for each variants (STAR Methods). Functional scores of two biological replicates for each treatment (HAT, cisplatin, or olaparib) and for each pool were calculated. Functional scores of two biological replicates under all conditions showed a high correlation (r = 0.95, 0.95 for HAT; 0.98, 0.97 for cisplatin; 0.98, 0.95 for olaparib) (Figure S3A), suggesting equal representation between the replicates. Hence, we generated the average functional score from the two biological replicates for each treatment and for each pool. The average functional scores followed a bimodal distribution (neutral or pathogenic controls) for the control variants under all conditions (Figure S3B). Further, we analyzed the correlation of average functional scores between the two pools (technical replicates) for each treatment and found a high correlation between the two technical replicates under all conditions (r = 0.99 for HAT, 0.67 for cisplatin, and 0.70 for olaparib) (Figure S3C). Finally, we took the average of the functional scores of two technical replicates and generated the functional score for each assay (HAT, cisplatin, and olaparib) (Table 1). For 41 variants, we either failed to get two independent BRCA2-expressing clones or they had a frequency of sequencing reads of less than 0.001 in the M15 sample. The functional scores for these variants were calculated from two replicates, and the functional score for the remaining 182 variants were calculated from four replicates (Table S1).

We next examined the functional scores of the variants. The functional scores in all three assays were bimodally distributed (Figure 2A). All pathogenic control variants were scored below −1.98 (n = 27; median −4.13, −4.86, and −5.22 for HAT, cisplatin, and olaparib, respectively), and all neutral variants were scored at or above −0.46 (n = 16; median 0.66, 0.34, and 0.50 for HAT, cisplatin, and olaparib, respectively). The functional scores obtained in the cisplatin and olaparib sensitivity assays showed the highest correlation (r = 0.98), whereas the correlation of the HAT functional score with the cisplatin (r = 0.87) or the olaparib (r = 0.89) functional score was less than 0.90 (Figure 2B).Figure 2Functional score (FS) distribution and the correlation of FS between three assays(A) The distribution of FSs calculated from HAT, cisplatin, and olaparib sensitivity of mESCs with different BRCA2 variants. The color codes demarcate controls and variants of uncertain significance (VUSs). Variants that were classified as benign or likely benign in ClinVar were used as neutral/functional controls, and those that were classified as pathogenic or likely pathogenic were used as pathogenic/non-functional controls. A total of 223 variants were analyzed.(B) Correlation of FSs from three different assays (sensitivity to HAT, cisplatin, or olaparib) are plotted to compare the results of different assays. The Spearman correlation coefficient is displayed in each plot.

Functional score (FS) distribution and the correlation of FS between three assays

(A) The distribution of FSs calculated from HAT, cisplatin, and olaparib sensitivity of mESCs with different BRCA2 variants. The color codes demarcate controls and variants of uncertain significance (VUSs). Variants that were classified as benign or likely benign in ClinVar were used as neutral/functional controls, and those that were classified as pathogenic or likely pathogenic were used as pathogenic/non-functional controls. A total of 223 variants were analyzed.

(B) Correlation of FSs from three different assays (sensitivity to HAT, cisplatin, or olaparib) are plotted to compare the results of different assays. The Spearman correlation coefficient is displayed in each plot.

We used the functional scores in a statistical model to calculate the PIF for each BRCA2 variant in the dataset (n = 223) (Tables 1 and S2). We further used these PIF to classify the variants as non-functional (PIF > 0.99), functional (PIF ≤ 0.05), and indeterminate (0.05 < PIF ≤ 0.99) (Table 1).9 Because the functional score distributions for the BRCA2 variants with known pathogenicity (i.e., the control variants in our study: 27 pathogenic and 16 benign variants) could be regarded as normal (Figure S4; Lilliefors test, p > 0.1), mixtures of two normal distributions were fitted to the data from each of the three assays (Figure S5), and the PIFs were calculated from these distributions, as described in the STAR Methods. The main version of our PIF calculation algorithm relied on the semi-supervised learning paradigm21 and had a special structure that combined the mixture models for all three assays. The resulting PIFs are shown in Figure 3, which compares three reduced algorithm versions (Figures 3A–3C) with its full version (Figure 3D). These reduced versions of our algorithm also relied on semi-supervised learning but used the data from only one of the three assays (HAT, cisplatin, or olaparib) for PIF calculation. Remarkably, for the majority of the calculated PIFs, the width of the confidence intervals was negligible, indicating a high robustness of the PIF calculation results (Figure 3D; Table S2). Moreover, only a few BRCA2 variants were classified as indeterminate. The full algorithm version yielded a 100% variant classification accuracy when applied to the full dataset (i.e., fitting accuracy) as well as a 100% cross-validation accuracy. In contrast, for each of the reduced algorithm versions, both full-dataset accuracy and cross-validation accuracy were lower (Figures 3A–3C). This clearly indicates the benefit of using the full, three-assay dataset for PIF calculation.Figure 3Probability of impact on function (PIF) for the BRCA2 variants in the full dataset (n= 223), calculated using the main (semi-supervised learning) version of our PIF calculation algorithmThe four subplots correspond to the four types of assays that provided FS data used in PIF calculation: HAT only (A), cisplatin only (B), olaparib only (C), and HAT, cisplatin, and olaparib combined (D). The dashed lines correspond to the variant classification thresholds of 0.05 and 0.99; a PIF value is considered indeterminate when 0.05 < PIF ≤ 0.99. In each subplot, the circles represent individual BRCA2 variants. Vertical lines show the 95% confidence intervals for each PIF. Values of confidence intervals are listed in Table S2.

Probability of impact on function (PIF) for the BRCA2 variants in the full dataset (n= 223), calculated using the main (semi-supervised learning) version of our PIF calculation algorithm

The four subplots correspond to the four types of assays that provided FS data used in PIF calculation: HAT only (A), cisplatin only (B), olaparib only (C), and HAT, cisplatin, and olaparib combined (D). The dashed lines correspond to the variant classification thresholds of 0.05 and 0.99; a PIF value is considered indeterminate when 0.05 < PIF ≤ 0.99. In each subplot, the circles represent individual BRCA2 variants. Vertical lines show the 95% confidence intervals for each PIF. Values of confidence intervals are listed in Table S2.

To gain insights into the advantages of semi-supervised learning for PIF calculation, we used the analysis strategy described above in combination with a supervised learning version of our algorithm.21 While the main, semi-supervised version of our algorithm uses all of the data in the training set for model fitting, the supervised learning version is trained only using the control variants (whose functional status, benign or pathogenic, is known). The resulting PIF (Figure S6) demonstrated that, while the classification accuracy of the supervised learning algorithm applied to the full dataset was still 100%, the number of variants classified as indeterminate was considerably larger, and the number of PIFs with wide confidence intervals was also considerably larger than those for the semi-supervised learning algorithm version (Table S3). This suggests that supervised learning, compared with semi-supervised learning, was characterized by increased uncertainty and, evidently, decreased reliability of the PIF calculation results. Moreover, the cross-validation accuracy for supervised learning was less than 100%. We thus concluded that our semi-supervised learning algorithmic approach combining data from all three functional assays (HAT, cisplatin, and olaparib) demonstrated superior performance compared with the other PIF calculation approaches we considered here.

To gain further insights into our semi-supervised learning results, we generated PIF-PIF plots for different combinations of assay data used in the PIF calculation (Figure S7). We observed that the PIFs calculated using different data for the same BRCA2 variant could be notably different, suggesting that the incorporation of the information from different assays in reliable PIF calculation is necessary because of their complementary and non-redundant nature. For the PIF based on a single assay, the cisplatin-based PIF typically exceeded the olaparib-based PIF, which, in turn, almost always exceeded the HAT-based PIF (Figures S7A–S7C). The three-assay PIF always exceeded (but in many cases negligibly) the corresponding HAT-only PIF (Figure S7D), a relation that was largely reversed for the cisplatin- and olaparib-only PIF (Figures S7E and S7F). However, in all PIF-PIF plots, the largest PIF-PIF differences were detected for the variants that were classified as non-functional by at least one of the PIFs. This implies that our suggested use of all three assays (HAT, cisplatin, and olaparib) in PIF calculation will be particularly important for the identification of nonfunctional BRCA2 variants.

We next compared our calculated PIF for the variants with the results from other published functional assays, ClinVar data, and in silico predictions (Bayes-del and Priors) and ACMG codes (Figure 4; Tables S4 and S5). Comparison of our prediction of the impact on function for the variants with the results from the high-throughput functional assay based on complementation of PARPi sensitivity (mixed-all nominated-in-one-BRCA [MANO-B] method)12 showed a high concordance with the variants that were classified as fClass 1/2 (neutral) or fClass 4/5 (pathogenic) (Figures 4A and 4D; Table S4). Nine variants that were classified as fClass 1/2 showed a PIF in favor of functional variant classification in our assay, except p.Glu2847Lys, which is classified as pathogenic in our assay (PIF 0.99) (Figure 5A). This variant is classified as intermediate in the homologous recombination (HR)-based functional assay with a probability of 0.343.19 All the variants, except p.Arg2842Leu, that were classified as fClass 4/5 by MANO-B method, showed a PIF in favor of non-functionality (Figures 4A, 4D, and 5A; Table S4). The variant p.Arg2842Leu is classified as functional in our assay (PIF 0.005) (Figure 5A). This variant is intermediate in the HR-based assay with a probability of 0.06 and complements cell survival and HR in the ESC-based assay.15,19 Eight variants (p.Ser1172Leu, p.Thr2337Ala, p.Tyr2905Cys, p.Gly2901Val, p.Asp2913His, p.Asp2913Val, p.Arg2842His, and p.Asp2900Val) that were classified as fClass 2/3 or fClass 3/4 showed a PIF in favor of neutrality in our assay (Figures 4A, 4D, and 5A; Table S4). Among these eight variants, p.Ser1172Leu and p.Arg2842His are classified as benign in ClinVar (Table 1).22 The p.Arg2842His is classified as benign using the HR assay.19 We could not compare the variants that were identified as intermediate by our functional assay because they were not tested by the MANO-B or the HR assay.Figure 4Comparison of multiplexed mESC-based variant classification with other functional, multifactorial, and ClinVar classifications(A) Comparison of variants that were classified using the MANO-B functional assay.12(B) Homologous recombination (HR)-based assay11 and classification using the mESC-based assay.(C) Plot showing the PIFs and classification using the mESC-based assay and classification of the variants in ClinVar. Overlapping dots are circled (dotted), and the number of dots in each circle is represented as n (numbers of variants on each circle). The dots that are not circled represent a single variant.(D) Side-by-side comparison of mESC assay-based classification of variants that have other functional assay data published. We used the data from the MANO-B (mixed-all nominated-in-one-BRCA [MANO-B]) method of classification, the mESC-based assay reported by Mesman et al.,15 and the HR assay-based classification.12,19Figure 5Position of classified variants in BRCA2(A) BRCA2 domains are marked with colored rectangles, with the amino acid (aa) position marked in parentheses. The variants located in different regions are marked below. OB, oligonucleotide/oligosaccharides binding; CTRB, C-terminal RAD51 binding.(B) Position of the splicing variants analyzed in this study. Functional variants are marked in green, non-functionals in magenta, and indeterminants in saffron.

Comparison of multiplexed mESC-based variant classification with other functional, multifactorial, and ClinVar classifications

(A) Comparison of variants that were classified using the MANO-B functional assay.12

(B) Homologous recombination (HR)-based assay11 and classification using the mESC-based assay.

(C) Plot showing the PIFs and classification using the mESC-based assay and classification of the variants in ClinVar. Overlapping dots are circled (dotted), and the number of dots in each circle is represented as n (numbers of variants on each circle). The dots that are not circled represent a single variant.

(D) Side-by-side comparison of mESC assay-based classification of variants that have other functional assay data published. We used the data from the MANO-B (mixed-all nominated-in-one-BRCA [MANO-B]) method of classification, the mESC-based assay reported by Mesman et al.,15 and the HR assay-based classification.12,19

Position of classified variants in BRCA2

(A) BRCA2 domains are marked with colored rectangles, with the amino acid (aa) position marked in parentheses. The variants located in different regions are marked below. OB, oligonucleotide/oligosaccharides binding; CTRB, C-terminal RAD51 binding.

(B) Position of the splicing variants analyzed in this study. Functional variants are marked in green, non-functionals in magenta, and indeterminants in saffron.

Among the selected variants, 15 variants with mutations in the DNA-binding domains of BRCA2 have been evaluated previously using the HR assay for prediction of pathogenicity.19 Nine of them (p.Leu2865Val, p.Phe2873Cys, p.Arg2842His, p.Glu2856Ala, p.Glu3152Lys, p.Tyr3098His, p.Ser3123Gly, p.Ala3088Val, and p.Tyr3092Cys) were classified as benign, three (p.Asn3124Ile, p.Leu3125His, and p.Asp3095Glu) as pathogenic, and three (p.Arg2842Leu, p.Tyr3092Ser, and p.Glu2847Lys) as intermediate using the HR assay. All variants classified as benign or pathogenic by the HR assay were classified in our NGS-based mouse ESC assay as functional or non-functional, respectively (Figures 4A, 4D, and 5A; Tables 1 and S4). Two intermediate variants, p.Arg2842Leu and p.Tyr3092Ser, with probabilities of 0.06 and 0.1, respectively, in the HR assay are classified as functional in our assay, with a PIF of 0.005 for p.Arg2842Leu and 0.016 for p.Tyr3092Ser (Figure 5A). Both variants rescued ESC lethality and complemented the HR defect in mESCs.15 The p.Glu2847Lys variant is classified as non-functional in our assay (Figure 5A; Table 1). Based on cell survival, p.Glu2847Lys is functional (HAT PIF 0.04). However, when the cell viability and drug sensitivity assays were combined, it was classified as non-functional (PIF 0.99). Seven of the selected variants (p.Tyr3092Ser, p.Arg2888Cys, p.Tyr3098His, p.Asn3124Ile, p.Asp3095Glu, p.Arg2842Leu, and Glu2856Ala) were analyzed by Mesman et al.15 using an mESC-based assay, and two of them (p.Asn3124Ile and p.Asp3095Glu), which failed to complement cell viability, are classified as non-functional in our assay (Figures 4D and 5A). The remaining five variants that were able to complement cell viability and supported HR in the mESC-based assay are classified as functional in our assay (Figure 5A; Table S4).15

All variants that were classified as pathogenic in ClinVar (p.Asp3095Glu, p.Leu3125His, c.8633-1G>A, c.8633-2 A>G, c.8633-2 A>T, c.8754 + 1 G>A, c.8754 + 1 G>T, and c.8754 + 2 T>G) are classified as non-functional in our assay, except for p.Ala2911Glu, which has a PIF in favor of functional (Figures 4C, 5A, and 5B; Table 1). This variant is classified as fClass 1/2 by the MANO-B method.12 ClinVar has classified this variant with no assertion criteria, and the classification was based on a single occurrence in a Fanconi anemia patient.12,23

Computational determination of prior probabilities of pathogenicity is based on the location of the mutated amino acid in the protein as well as the impact of the mutated nucleotide on splicing (database: http://hci-priors.hci.utah.edu/PRIORS/),24 and the in silico prediction tool BayesDel score combines multiple deleteriousness predictors to calculate an overall score.25 In BayesDel, the PIF criterion for benign for BRCA2 is PIF < 0.08, and the PIF criterion for pathogenic is PIF > 0.5.26 These comparisons show high agreement with the PIF values and functionality predictions obtained from our assay (Tables 1 and S4).

### Selection of variants

We selected 223 BRCA2 variants listed in the ClinVar database that impact residues encoded by exons 11 (65 variants), 14 (52 variants), 20 (19 variants), 21 (50 variants), and 25 (37 variants) (Table 1). Among these variants, 32 are nonsense variants that are classified as pathogenic in ClinVar (Table 1). Sixteen variants classified as benign or likely benign in ClinVar were used as neutral controls (Table 1). All control variants are in different exons. Sixty-five variants are in exon 11 of BRCA2, the 3,387-bp exon that codes for the region containing 8 BRC repeats. Among the selected variants in exon 11, eight are located in the BRC1 repeat, and the remaining are in the region flanking the BRC1 repeat (Figure S1). All selected variants from exon 14 are located in the region preceding the DNA binding domain of BRCA2, whereas the variants from exons 20, 21, and 25 are located in the Oligonucleotides/oligosaccharides binding domains, OB2 or OB3 region of the DNA binding domain of BRCA2 (Figure S1). Six variants are present in introns 20 and 21 (Figure S1).Table 1Classification of variants using the mESC-based assayHGVS nucleotideHGVS proteinExonClinVarESC assayHAT FSCis FSOla FSHAT PIFCis PIFOla PIFCombined PIFClassc.7022G>Tp.Arg2341Leu14US−0.2140−0.7767−0.86360.00140.04510.02280.00238888Fc.7021C>Gp.Arg2341Gly14US−0.0298−0.4448−0.44060.00070.02190.00680.00083833Fc.7078T>Cp.Ser2360Pro14US−0.0611−0.0113−0.10990.00080.01030.00310.0008022Fc.7040C>Ap.Pro2347Gln14US0.1133−0.3929−0.35910.00040.01980.00560.00053352Fc.7082A>Gp.His2361Arg14US0.18760.07510.12130.00030.00910.00200.00035197Fc.7069C>Gp.Leu2357Val14US0.1014−0.1810−0.18740.00040.01350.00370.00049037Fc.7010C>Tp.Thr2337Ile14US0.0353−0.3679−0.38350.00050.01890.00590.00066076Fc.7033C>Gp.Gln2345Glu14US−0.1095−0.5951−0.69370.00090.02990.01370.00132712Fc.7042A>Cp.Asn2348His14US0.25230.38260.22060.00030.00640.00170.00028432Fc.7049C>Tp.Thr2350Ile14US0.2557−0.1121−0.18810.00030.01210.00370.00031581Fc.7045T>Cp.Phe2349Leu14US−4.8694−5.3479−5.51311.00001.00001.00001NFc.7022G>Ap.Arg2341His14US0.0016−0.1774−0.15320.00060.01340.00350.00066299Fc.7066T>Gp.Phe2356Val14US0.48260.66870.72900.00010.00510.00090.00014814Fc.7067T>Ap.Phe2356Tyr14US0.0292−0.4491−0.44050.00060.02210.00680.00071176Fc.7037A>Gp.Asn2346Ser14US−0.0138−0.03950.06400.00070.01080.00220.00067466Fc.7051G>Tp.Ala2351Ser14US0.10690.71910.79940.00040.00490.00080.00043611Fc.7034A>Gp.Gln2345Arg14US−0.3732−0.5543−0.88010.00260.02740.02400.00324057Fc.7030A>Gp.Ile2344Val14US−0.1915−0.2654−0.26830.00120.01560.00450.00131887Fc.7017G>Cp.Lys2339Asn14B−0.2044−0.1943−0.10840.00130.01380.00310.00135582Fc.7027G>Ap.Glu2343Lys14US−0.5738−0.1653−0.15400.00620.01320.00350.0062201Fc.7051G>Cp.Ala2351Pro14US−0.3355−0.4772−0.52800.00220.02340.00860.00241123Fc.7072T>Cp.Ser2358Pro14US0.03490.59210.72000.00060.00530.00090.00055467Fc.C7021Tp.Arg2341Cys14US−0.22950.08490.02440.00140.00900.00240.00146803Fc.7025A>Gp.Gln2342Arg14US−0.34780.26510.28190.00230.00720.00150.00233593Fc.7081C>Tp.His2361Tyr14US−0.3196−0.3121−0.12470.00210.01700.00320.00212651Fc.3170A>Gp.Lys1057Arg11US0.45490.87790.25460.00020.00460.00160.00016178Fc.3206C>Tp.Ser1069Phe11US−0.01210.5133−0.08540.00060.00570.00300.00066375Fc.3256A>Gp.Ile1086Val11CIP0.52430.48440.24580.00010.00580.00160.00013868Fc.3310A>Cp.Thr1104Pro11US0.4720−0.06270.01470.00010.01120.00240.00017503Fc.3319C>Tp.Gln1107Stop11P−3.5779−4.1744−4.14901.00001.00001.00001NFc.3395A>Gp.Lys1132Arg11US0.53091.16540.49360.00010.00430.00110.00013209Fc.3403T>Cp.Tyr1135His11US−0.4727−0.2659−0.12370.00390.01560.00320.00399517Fc.3413A>Tp.Gln1138Leu11US−0.0826−0.1557−0.17970.00080.01290.00370.00087922Fc.3419G>Cp.Ser1140Thr11US−0.06842.76840.93490.00080.02140.00070.00080588Fc.3446T>Cp.Met1149Thr11US0.51850.77780.34490.00010.00480.00140.00013781Fc.3458A>Gp.Lys1153Arg11CIP0.22781.10510.45660.00030.00430.00120.00029983Fc.3469G>Tp.Glu1157Stop11P−5.1118−6.1137−6.34121.00001.00001.00001NFc.3475T>Ap.Cys1159Ser110.46561.31190.61740.00020.00440.00100.00015452Fc.3499A>Gp.Ile1167Val11US−0.13090.16160.07750.00100.00820.00220.0010112Fc.3509C>Tp.Ala1170Val11CIP1.10740.16530.51770.00000.00810.00115.0152E-05Fc.3515C>Gp.Ser1172Trp11US0.94920.80250.10950.00010.00470.00206.2456E-05Fc.9458G>Cp.Gly3153Ala25CIP1.21321.73361.65170.00000.00530.00073.9357E-05Fc.9401G>Tp.Gly3134Val25US0.99601.96342.00110.00000.00640.00085.4015E-05Fc.9307A>Gp.Ile3103Val25US1.17451.58361.69280.00000.00480.00074.0886E-05Fc.9275A>Gp.Tyr3092Cys25CIP0.94070.92711.18630.00010.00450.00065.654E-05Fc.9263C>Tp.Ala3088Val25US1.23761.34731.59950.00000.00440.00063.7711E-05Fc.9375C>Gp.Leu3125 =25LB1.35941.39591.40450.00000.00450.00063.3459E-05Fc.9477C>Ap.Phe3159Leu25CIP1.22131.39381.57440.00000.00450.00063.8411E-05Fc.9275A>Cp.Tyr3092Ser25US0.3896−1.5691−1.10560.00020.32880.04990.01659796Fc.9449C>Tp.Pro3150Leu25US0.0503−1.7698−0.96060.00050.50130.03100.01605649Fc.9356T>Gp.Leu3119Stop25P−3.6916−3.6811−3.37571.00000.99990.99981NFc.9500A>Cp.Glu3167Ala25US2.43922.54512.44580.00000.01410.00135.1474E-05Fc.9285C>Gp.Asp3095Glu25P/LP−3.8228−5.0215−4.92081.00001.00001.00001NFc.9374T>Ap.Leu3125His25LP−4.1949−5.9723−5.91961.00001.00001.00001NFc.9294C>Gp.Tyr3098Stop25P−3.1644−3.7933−3.58011.00001.00001.00001NFc.9302T>Gp.Leu3101Arg25CIP−3.0114−5.1579−4.44681.00001.00001.00001NFc.9481A>Tp.Lys3161Stop25P−3.3027−3.2420−3.22041.00000.99890.99951NFc.9455A>Gp.Glu3152Gly25US−4.0623−4.8274−4.15321.00001.00001.00001NFc.2813C>Ap.Ala938Glu11CIP−0.6316−0.7041−0.37130.00800.03810.00570.00825901Fc.3085A>Gp.Met1029Val11US0.28740.37940.36550.00020.00640.00130.00025506Fc.3075G>Tp.Lys1025Asn11US0.52700.48290.40880.00010.00580.00130.00013577Fc.2786T>Cp.Leu929Ser11B0.39330.25430.19750.00020.00730.00170.00019514Fc.2987T>Gp.Leu996Arg11B0.86170.69700.61410.00010.00500.00106.6665E-05Fc.2965T>Gp.Tyr989Asp11US−0.5989−0.8827−0.50990.00690.05810.00820.00739321Fc.2798C>Gp.Thr933Arg11US0.36250.28280.23330.00020.00710.00160.0002103Fc.2849T>Ap.Val950Asp11CIP0.61140.96570.88700.00010.00440.00080.0001081Fc.2803G>Cp.Asp935His11B−0.0744−0.3379−0.44630.00080.01780.00690.00093119Fc.2927C>Tp.Ser976Phe11B/LB0.3687−0.2071−0.46070.00020.01410.00720.00029688Fc.3073A>Gp.Lys1025Glu11CIP−0.6727−0.7125−0.47280.00970.03890.00740.010024Fc.2944A>Cp.Ile982Leu11CIP0.73770.41800.28040.00010.00620.00158.8507E-05Fc.2979G>Ap.Trp993Stop11P−4.8245−6.1596−6.42921.00001.00001.00001NFc.3142G>Ap.Val1048Ile11US0.48540.71020.55390.00010.00490.00100.00014791Fc.3166C>Tp.Gln1056Stop11P−3.2693−2.4294−2.71271.00000.93840.98920.99999994NFc.3088T>Gp.Phe1030Val11CIP0.52980.02400.66110.00010.00980.00090.00013664Fc.2836G>Cp.Asp946His11US0.65960.94790.84670.00010.00450.00089.7257E-05Fc.2899C>Gp.Leu967Val11US0.6510−0.18070.08670.00010.01350.00210.00012434Fc.3092T>Cp.Phe1031Ser11US0.5260−0.5788−0.47470.00010.02890.00750.00034466Fc.2926T>Ap.Ser976Thr11B/LB0.75740.44050.60340.00010.00600.00108.1881E-05Fc.3122G>Ap.Ser1041Asn11US0.59440.82110.82850.00010.00470.00080.00011271Fc.3137A>Gp.Glu1046Gly11CIP0.0539−0.3310−0.14310.00050.01760.00340.000575Fc.2854G>Tp.Ala952Ser11US0.80830.22440.44040.00010.00760.00127.7641E-05Fc.3109C>Tp.Gln1037Stop11P−4.4515−5.5910−5.56691.00001.00001.00001NFc.3515C>Tp.Ser1172Leu11B0.00070.81510.65680.00060.00470.00090.00062294Fc.3503T>Ap.Met1168Lys11US0.88170.58520.69970.00010.00540.00096.4338E-05Fc.3367A>Gp.Ser1123Gly11US0.94310.87150.96870.00010.00460.00075.6687E-05Fc.3437A>Gp.Glu1146Gly11US0.94900.58470.76850.00010.00540.00085.7345E-05Fc.3362C>Ap.Ser1121Stop11P−3.1092−4.1491−4.36501.00001.00001.00001NFc.3265C>Tp.Gln1089Stop11P−3.0456−4.5726−4.61211.00001.00001.00001NFc.3539A>Gp.Lys1180Arg11CIP0.80010.26320.47440.00010.00720.00127.7971E-05Fc.3262C>Tp.Pro1088Ser11CIP0.40080.45170.24310.00020.00600.00160.00018833Fc.3526G>Ap.Val1176Ile11US0.29020.65210.18060.00020.00510.00180.00025369Fc.3103G>Tp.Glu1035Stop11P−4.6635−5.2485−5.38761.00001.00001.00001NFc.7102T>Gp.Leu2368Val14CIP0.56050.38580.39380.00010.00630.00130.00012646Fc.7142C>Tp.Pro2381Leu14US−0.0513−0.09180.11500.00070.01170.00200.00076683Fc.7025A>Cp.Gln2342Pro14US−0.0644−0.1821−0.27590.00080.01350.00460.00084049Fc.7009A>Gp.Thr2337Ala14US−0.3626−0.5921−0.71560.00250.02970.01460.00290388Fc.7100C>Tp.Thr2367Ile14US−0.25390.76000.73630.00160.00480.00090.00159616Fc.7150C>Gp.Gln2384Glu14US0.74740.60180.60460.00010.00530.00108.2728E-05Fc.7133C>Gp.Ser2378Stop14P−3.6377−4.6589−4.84371.00001.00001.00001NFc.7095T>Ap.His2365Gln14US−0.10410.43520.36000.00090.00610.00140.00090799Fc.7139A>Gp.His2380Arg14US0.57620.27250.27270.00010.00720.00150.00012491Fc.7039C>Gp.Pro2347Ala14US0.31550.22420.22790.00020.00760.00170.00023964Fc.7093C>Tp.His2365Tyr14US0.03060.07140.00700.00060.00920.00250.0005808Fc.7147T>Cp.Tyr2383His14US−2.6611−3.0326−3.17490.99980.99660.99930.99999917NFc.7138C>Tp.His2380Tyr14US0.20680.04540.04540.00030.00950.00230.00033626Fc.7107A>Cp.Glu2369Asp14US−3.1135−4.0509−4.27921.00001.00001.00001NFc.7142C>Ap.Pro2381Gln14US−1.3017−2.1833−2.55700.21840.83830.97440.8568118Ic.7073C>Gp.Ser2358Cys14US0.1058−0.0417−0.01930.00040.01080.00260.00046198Fc.7115C>Gp.Ser2372Stop14P−2.7216−3.5670−4.14430.99990.99981.00000.99999998NFc.7028A>Tp.Glu2343Val14US0.35330.36320.35950.00020.00650.00140.00021264Fc.7126G>Cp.Ala2376Pro14US−0.5944−0.9135−0.71310.00680.06260.01450.00768141Fc.7136G>Tp.Gly2379Val14US−5.4959−5.6089−6.16011.00001.00001.00001NFc.7121A>Gp.Asn2374Ser14CIP0.0312−0.3099−0.25010.00060.01690.00430.00062956Fc.7136G>Ap.Gly2379Glu14US−1.6839−3.5697−3.72990.75380.99981.00000.9999561NFc.8657C>Gp.Pro2886Arg210.0958−0.7838−0.54460.00040.04580.00900.0008604Fc.8679G>Tp.Gln2893His21US0.2385−0.23110.32080.00030.01470.00140.00030634Fc.8672C>Gp.Thr2891Arg21US−0.0015−0.2788−0.01830.00060.01600.00260.00066501Fc.8681A>Gp.Gln2894Arg21US−0.4399−1.5203−0.92340.00340.29310.02750.01146638Fc.8651A>Tp.Tyr2884Phe21US−0.0402−0.29310.02780.00070.01640.00240.00075337Fc.8710C>Tp.Leu2904Phe21US−0.3337−2.8621−1.94160.00220.99190.61950.61527078Ic.8686C>Tp.Arg2896Cys21US−0.3156−1.0178−0.52790.00200.08130.00860.00273603Fc.8651A>Gp.Tyr2884Cys21CIP−0.0501−0.8476−0.36350.00070.05330.00560.00103992Fc.8663G>Tp.Arg2888Leu21US−0.1668−0.6515−0.32790.00110.03390.00520.00131122Fc.8680C>Tp.Gln2894Stop21P−5.0250−5.9901−5.43011.00001.00001.00001NFc.8707G>Tp.Glu2903Stop21P−5.9357−7.5663−6.65601.00001.00001.00001NFc.8665G>Tp.Ala2889Ser21US0.64341.70821.12820.00010.00520.00070.00010071Fc.8633-1G>AIntronP/LP−5.3153−5.5640−5.29941.00001.00001.00001NFc.8704G>Ap.Ala2902Thr21US0.4549−1.1871−0.41890.00020.12560.00650.00096687Fc.8687G>Ap.Arg2896His21CIP−0.2058−0.8948−0.58290.00130.05980.01000.00191661Fc.8722G>Ap.Val2908Met21US−0.0797−0.7944−0.46950.00080.04700.00740.00116919Fc.8690C>Gp.Ala2897Gly21US−0.0893−0.8324−0.44010.00090.05140.00680.00120308Fc.8663G>Ap.Arg2888His21US0.49291.12851.12050.00010.00430.00070.00014291Fc.8662C>Tp.Arg2888Cys21B0.1056−0.27580.03690.00040.01590.00230.00047119Fc.8639C>Gp.Thr2880Arg21US−0.1524−0.4348−0.24050.00110.02150.00420.00116683Fc.8668C>Ap.Leu2890Ile21CIP−0.5353−2.0841−1.31810.00520.77420.10290.08441721Ic.8663G>Cp.Arg2888Pro21US−0.0962−0.0197−0.06260.00090.01050.00280.00090399Fc.8714A>Gp.Tyr2905Cys21US−0.0733−1.5721−1.14940.00080.33120.05790.01994992Fc.8702G>Tp.Gly2901Val21CIP−0.0639−1.6055−1.05530.00080.35740.04220.01586066Fc.8708A>Gp.Glu2903Gly21US0.2096−0.2141−0.19200.00030.01430.00380.00036535Fc.8732C>Tp.Ala2911Val21US−0.04230.56440.41600.00070.00540.00120.00072643Fc.8737G>Ap.Asp2913Asn21US0.0662−0.3298−0.32590.00050.01760.00510.00058478Fc.8746T>Ap.Tyr2916Asn21US0.12400.04120.06570.00040.00960.00220.0004299Fc.8732C>Ap.Ala2911Glu21P0.55650.49660.58660.00010.00570.00100.00012521Fc.8737G>Cp.Asp2913His21US0.29360.38920.39130.00020.00630.00130.00025023Fc.8744C>Tp.Ala2915Val21US0.38410.39750.46090.00020.00630.00120.00019441Fc.8644A>Tp.Lys2882Stop21P−5.0381−6.6902−6.33561.00001.00001.00001NFc.8752G>Cp.Glu2918Gln21US0.26010.21590.34730.00030.00760.00140.00027783Fc.8754 + 2T>GIntronP−4.4425−8.5100−8.58231.00001.00001.00001NFc.8633-2A>GIntronP−6.1292−7.9408−7.08951.00001.00001.00001NFc.8738A>Tp.Asp2913Val21US−0.35070.4720−0.44000.00240.00590.00680.00239288Fc.8677C>Tp.Gln2893Stop21P−6.3245−8.7391−9.43221.00001.00001.00001NFc.8695C>Tp.Gln2899Stop21P−5.9213−8.2198−7.49671.00001.00001.00001NFc.8754 + 1G>TIntronP/LP−6.1534−9.3828−7.89631.00001.00001.00001NFc.8646A>Tp.Lys2882Asn21US0.11950.39450.38090.00040.00630.00130.00042298Fc.8633-2A>TIntronP−5.0434−6.9790−7.23121.00001.00001.00001NFc.8754 + 1G>AIntronP−6.0750−7.4268−6.98581.00001.00001.00001NFc.9301C>Gp.Leu3101Val25US0.74951.02481.11940.00010.00440.00078.0084E-05Fc.9271G>Ap.Val3091Ile25CIP1.92551.94171.88300.00000.00630.00072.8814E-05Fc.9414A>Gp.Leu3138 =25LB1.90711.76831.66610.00000.00540.00072.7722E-05Fc.9367A>Gp.Ser3123Gly25US1.09171.94321.55690.00000.00630.00064.6219E-05Fc.9292T>Cp.Tyr3098His25B1.31831.52671.54870.00000.00470.00063.4899E-05Fc.9350A>Cp.His3117Pro25US1.04661.08601.35830.00000.00430.00064.7865E-05Fc.9501G>Ap.Glu3167 =251.69191.43481.45650.00000.00450.00062.7661E-05Fc.9466C>Tp.Gln3156Stop25P−4.2907−6.1321−5.22031.00001.00001.00001NFc.9376C>Tp.Gln3126Stop25P−4.1324−4.7481−4.44961.00001.00001.00001NFc.9371A>Tp.Asn3124Ile25P−3.9263−7.6510−6.48421.00001.00001.00001NFc.8539G>Ap.Glu2847Lys20CIP−0.9734−4.4171−4.58930.04251.00001.00000.99999935NFc.8593T>Gp.Leu2865Val20CIP1.05980.81850.83290.00000.00470.00084.7936E-05Fc.8575C>Tp.Gln2859Stop20P−2.7456−6.2237−6.16330.99991.00001.00001NFc.8618T>Gp.Phe2873Cys20US1.25911.33251.39070.00000.00440.00063.6782E-05Fc.8503T>Cp.Ser2835Pro20B1.68131.35911.38280.00000.00440.00062.7686E-05Fc.8573A>Gp.Gln2858Arg20CIP0.87181.14061.17640.00010.00430.00066.349E-05Fc.8548G>Ap.Glu2850Lys20US1.07690.78840.78260.00000.00470.00084.7041E-05Fc.8518A>Gp.Ile2840Val20US1.51071.05921.09040.00000.00440.00073.0081E-05Fc.8525G>Ap.Arg2842His20B1.4195−0.2924−0.42820.00000.01640.00660.00013772Fc.8572C>Ap.Gln2858Lys20CIP1.04251.20391.25580.00000.00430.00064.818E-05Fc.8525G>Tp.Arg2842Leu20CIP0.6106−1.0058−1.18000.00010.07880.06420.00516208Fc.8599A>Cp.Thr2867Pro20US1.05080.79400.78470.00000.00470.00084.8739E-05Fc.8591C>Tp.Ala2864Val20US0.68140.39220.34340.00010.00630.00149.8065E-05Fc.8629G>Tp.Glu2877Stop20P−4.2046−4.8618−4.87291.00001.00001.00001NFc.8504C>Ap.Ser2835Stop20P−3.2603−3.4495−3.63581.00000.99971.00001NFc.8567A>Cp.Glu2856Ala20CIP1.29200.89970.97750.00000.00450.00073.6047E-05Fc.7068T>Gp.Phe2356Leu14US0.0051−1.4464−1.72470.00060.24460.37810.09306036Ic.3299A>Tp.Asn1100Ile11CIP0.15781.16380.10820.00040.00430.00200.00037551Fc.3362C>Gp.Ser1121Stop11P−4.0407−4.7562−5.28771.00001.00001.00001NFc.3496G>Ap.Val1166Ile11US−0.3355−0.2721−0.34440.00220.01580.00540.00229499Fc.9425A>Gp.Asp3142Gly25US1.25291.11271.18230.00000.00430.00063.708E-05Fc.9385C>Gp.Pro3129Ala25US0.81411.15960.94350.00010.00430.00077.0883E-05Fc.9276T>Gp.Tyr3092Stop25P−3.5541−3.8671−3.90631.00001.00001.00001NFc.9382C>Tp.Arg3128Stop25P−4.1565−4.7846−3.96421.00001.00001.00001NFc.9317G>Ap.Trp3106Stop25P−2.3918−2.4653−2.31960.99810.94720.91380.99974674NFc.9309A>Gp.Ile3103Met25US1.11990.21470.10920.00000.00760.00205.6182E-05Fc.2971A>Gp.Asn991Asp11B1.64901.52172.24750.00000.00470.00102.9989E-05Fc.2881C>Tp.Gln961Stop11P−4.8245−5.9455−6.75921.00001.00001.00001NFc.2818C>Tp.Gln940Stop11P−7.1483−7.0299−8.11131.00001.00001.00001NFc.3076A>Tp.Lys1026Stop11P−5.6613−8.4729−8.19211.00001.00001.00001NFc.3032C>Gp.Thr1011Arg11CIP0.92360.36610.41840.00010.00650.00126.3314E-05Fc.3132T>Gp.Cys1044Trp11US0.79060.34350.21610.00010.00660.00178.2198E-05Fc.2872A>Gp.Ser958Gly11US0.65810.83780.80540.00010.00460.00089.7826E-05Fc.2803G>Ap.Asp935Asn11B/LB0.57350.14040.40890.00010.00840.00130.00012519Fc.3270G>Cp.Met1090Ile11US2.50231.13082.29570.00000.00430.00113.9885E-05Fc.3304A>Tp.Asn1102Tyr11B0.5268−0.3602−0.05900.00010.01860.00280.0001811Fc.7141C>Tp.Pro2381Ser14US0.19340.20640.14410.00030.00770.00190.00034255Fc.7141C>Gp.Pro2381Ala14US0.2626−0.0195−0.02830.00030.01050.00270.00029313Fc.7119C>Gp.Ser2373Arg14US0.14170.18740.08880.00040.00790.00210.00040277Fc.7115C>Ap.Ser2372Stop14P−1.9846−5.4167−5.56990.96061.00001.00001NFc.7106A>Gp.Glu2369Gly14US−0.5600−1.3960−1.11400.00580.21550.05140.0168051Fc.8702G>Ap.Gly2901Asp21CIP−0.0512−0.7515−0.98490.00070.04250.03350.00216736Fc.8692T>Gp.Leu2898Val21US−0.8861−2.5100−1.43430.02740.95670.15280.16956546Ic.8739C>Gp.Asp2913Glu21US0.76811.20350.85670.00010.00430.00087.7631E-05Fc.8699A>Tp.Asp2900Val21US−0.09180.58540.51010.00090.00540.00110.00086614Fc.8740C>Gp.Pro2914Ala21US0.59991.02491.01700.00010.00440.00070.00011065Fc.8678A>Gp.Gln2893Arg21US0.18130.17160.41550.00030.00810.00120.00035048Fc.8656C>Ap.Pro2886Thr21US0.2438−0.1361−0.26280.00030.01250.00440.00033615Fc.2837A>Gp.Asp946Gly11CIP0.44080.45130.24640.00020.00600.00160.00017005Fc.2771A>Tp.Asn924Ile11CIP−0.45800.59800.60230.00370.00530.00100.00370591Fc.8647C>Tp.Pro2883Ser21CIP−6.4547−10.133−9.10981.00001.00001.00001NFc.9353T>Cp.Met3118Thr25CIP−0.8240−0.8750−0.22290.02010.05700.00400.02037364Fc.9454G>Ap.Glu3152Lys25CIP1.68992.06131.84250.00000.00710.00072.9922E-05Fc.9380G>Ap.Trp3127Stop25P−5.8464−7.9743−7.74841.00001.00001.00001NFc.8572C>Tp.Gln2858Stop20P−4.3478−6.3224−6.25831.00001.00001.00001NFc.8594T>Gp.Leu2865Stop20P−6.6953−8.8540−8.13871.00001.00001.00001NFc.8489G>Ap.Trp2830Stop20P−4.6542−3.6163−3.72871.00000.99991.00001NFFS, functional score; PIF, probability of impact on function; Cis, cisplatin; Ola, olaparib; F, functional; NF, non-functional; I, indeterminate; B, benign; LB, likely benign; P, pathogenic; LP, likely pathogenic; US, uncertain significance; CIP, conflicting interpretation of pathogenicity.

Classification of variants using the mESC-based assay

FS, functional score; PIF, probability of impact on function; Cis, cisplatin; Ola, olaparib; F, functional; NF, non-functional; I, indeterminate; B, benign; LB, likely benign; P, pathogenic; LP, likely pathogenic; US, uncertain significance; CIP, conflicting interpretation of pathogenicity.

### Generation of variants and multiplexing the mESC-based assay

Each variant was generated by recombineering in a BAC clone (CTD-2342K5 with a 127-kb insert) containing full-length BRCA2.8,20 All BACs were sequenced to confirm the presence of the desired mutation and the lack of any undesired mutations in that region. For most of the variants, two independent mouse embryonic stem cells (PL2F7) clones expressing full-length BRCA2 were selected for further analysis. A single clone was used for variants where two equally expressing clones were not obtained. Subsequently, mESC clones expressing variants from the same exons were pooled together.

Two pools of mESCs were generated for each batch of exon using two independently generated BRCA2-variant expressing clones. Cell samples from each pool were collected before Pgk-Cre expression (termed “M15”), after HAT selection of recombinant clones (termed “HAT”), and after the selection of recombinant clones in the presence of olaparib or cisplatin. The genomic DNA obtained from collected cell samples was used to PCR amplify the genomic region where the variants are located and sequenced using the Illumina Mi-Seq platform.

### Functional scores of variants

The frequency of each variant was calculated as described in the STAR Methods. The frequency of variants in the M15 sample followed normal distribution consisting of a bell-shaped curve (Figure S2). The frequency of variants based on HAT, cisplatin, and olaparib treatment were used to calculate the functional score for each variants (STAR Methods). Functional scores of two biological replicates for each treatment (HAT, cisplatin, or olaparib) and for each pool were calculated. Functional scores of two biological replicates under all conditions showed a high correlation (r = 0.95, 0.95 for HAT; 0.98, 0.97 for cisplatin; 0.98, 0.95 for olaparib) (Figure S3A), suggesting equal representation between the replicates. Hence, we generated the average functional score from the two biological replicates for each treatment and for each pool. The average functional scores followed a bimodal distribution (neutral or pathogenic controls) for the control variants under all conditions (Figure S3B). Further, we analyzed the correlation of average functional scores between the two pools (technical replicates) for each treatment and found a high correlation between the two technical replicates under all conditions (r = 0.99 for HAT, 0.67 for cisplatin, and 0.70 for olaparib) (Figure S3C). Finally, we took the average of the functional scores of two technical replicates and generated the functional score for each assay (HAT, cisplatin, and olaparib) (Table 1). For 41 variants, we either failed to get two independent BRCA2-expressing clones or they had a frequency of sequencing reads of less than 0.001 in the M15 sample. The functional scores for these variants were calculated from two replicates, and the functional score for the remaining 182 variants were calculated from four replicates (Table S1).

We next examined the functional scores of the variants. The functional scores in all three assays were bimodally distributed (Figure 2A). All pathogenic control variants were scored below −1.98 (n = 27; median −4.13, −4.86, and −5.22 for HAT, cisplatin, and olaparib, respectively), and all neutral variants were scored at or above −0.46 (n = 16; median 0.66, 0.34, and 0.50 for HAT, cisplatin, and olaparib, respectively). The functional scores obtained in the cisplatin and olaparib sensitivity assays showed the highest correlation (r = 0.98), whereas the correlation of the HAT functional score with the cisplatin (r = 0.87) or the olaparib (r = 0.89) functional score was less than 0.90 (Figure 2B).Figure 2Functional score (FS) distribution and the correlation of FS between three assays(A) The distribution of FSs calculated from HAT, cisplatin, and olaparib sensitivity of mESCs with different BRCA2 variants. The color codes demarcate controls and variants of uncertain significance (VUSs). Variants that were classified as benign or likely benign in ClinVar were used as neutral/functional controls, and those that were classified as pathogenic or likely pathogenic were used as pathogenic/non-functional controls. A total of 223 variants were analyzed.(B) Correlation of FSs from three different assays (sensitivity to HAT, cisplatin, or olaparib) are plotted to compare the results of different assays. The Spearman correlation coefficient is displayed in each plot.

Functional score (FS) distribution and the correlation of FS between three assays

(A) The distribution of FSs calculated from HAT, cisplatin, and olaparib sensitivity of mESCs with different BRCA2 variants. The color codes demarcate controls and variants of uncertain significance (VUSs). Variants that were classified as benign or likely benign in ClinVar were used as neutral/functional controls, and those that were classified as pathogenic or likely pathogenic were used as pathogenic/non-functional controls. A total of 223 variants were analyzed.

(B) Correlation of FSs from three different assays (sensitivity to HAT, cisplatin, or olaparib) are plotted to compare the results of different assays. The Spearman correlation coefficient is displayed in each plot.

### Evaluation of BRCA2 variants by a combined-mixture statistical model and prediction of the impact on function

We used the functional scores in a statistical model to calculate the PIF for each BRCA2 variant in the dataset (n = 223) (Tables 1 and S2). We further used these PIF to classify the variants as non-functional (PIF > 0.99), functional (PIF ≤ 0.05), and indeterminate (0.05 < PIF ≤ 0.99) (Table 1).9 Because the functional score distributions for the BRCA2 variants with known pathogenicity (i.e., the control variants in our study: 27 pathogenic and 16 benign variants) could be regarded as normal (Figure S4; Lilliefors test, p > 0.1), mixtures of two normal distributions were fitted to the data from each of the three assays (Figure S5), and the PIFs were calculated from these distributions, as described in the STAR Methods. The main version of our PIF calculation algorithm relied on the semi-supervised learning paradigm21 and had a special structure that combined the mixture models for all three assays. The resulting PIFs are shown in Figure 3, which compares three reduced algorithm versions (Figures 3A–3C) with its full version (Figure 3D). These reduced versions of our algorithm also relied on semi-supervised learning but used the data from only one of the three assays (HAT, cisplatin, or olaparib) for PIF calculation. Remarkably, for the majority of the calculated PIFs, the width of the confidence intervals was negligible, indicating a high robustness of the PIF calculation results (Figure 3D; Table S2). Moreover, only a few BRCA2 variants were classified as indeterminate. The full algorithm version yielded a 100% variant classification accuracy when applied to the full dataset (i.e., fitting accuracy) as well as a 100% cross-validation accuracy. In contrast, for each of the reduced algorithm versions, both full-dataset accuracy and cross-validation accuracy were lower (Figures 3A–3C). This clearly indicates the benefit of using the full, three-assay dataset for PIF calculation.Figure 3Probability of impact on function (PIF) for the BRCA2 variants in the full dataset (n= 223), calculated using the main (semi-supervised learning) version of our PIF calculation algorithmThe four subplots correspond to the four types of assays that provided FS data used in PIF calculation: HAT only (A), cisplatin only (B), olaparib only (C), and HAT, cisplatin, and olaparib combined (D). The dashed lines correspond to the variant classification thresholds of 0.05 and 0.99; a PIF value is considered indeterminate when 0.05 < PIF ≤ 0.99. In each subplot, the circles represent individual BRCA2 variants. Vertical lines show the 95% confidence intervals for each PIF. Values of confidence intervals are listed in Table S2.

Probability of impact on function (PIF) for the BRCA2 variants in the full dataset (n= 223), calculated using the main (semi-supervised learning) version of our PIF calculation algorithm

The four subplots correspond to the four types of assays that provided FS data used in PIF calculation: HAT only (A), cisplatin only (B), olaparib only (C), and HAT, cisplatin, and olaparib combined (D). The dashed lines correspond to the variant classification thresholds of 0.05 and 0.99; a PIF value is considered indeterminate when 0.05 < PIF ≤ 0.99. In each subplot, the circles represent individual BRCA2 variants. Vertical lines show the 95% confidence intervals for each PIF. Values of confidence intervals are listed in Table S2.

To gain insights into the advantages of semi-supervised learning for PIF calculation, we used the analysis strategy described above in combination with a supervised learning version of our algorithm.21 While the main, semi-supervised version of our algorithm uses all of the data in the training set for model fitting, the supervised learning version is trained only using the control variants (whose functional status, benign or pathogenic, is known). The resulting PIF (Figure S6) demonstrated that, while the classification accuracy of the supervised learning algorithm applied to the full dataset was still 100%, the number of variants classified as indeterminate was considerably larger, and the number of PIFs with wide confidence intervals was also considerably larger than those for the semi-supervised learning algorithm version (Table S3). This suggests that supervised learning, compared with semi-supervised learning, was characterized by increased uncertainty and, evidently, decreased reliability of the PIF calculation results. Moreover, the cross-validation accuracy for supervised learning was less than 100%. We thus concluded that our semi-supervised learning algorithmic approach combining data from all three functional assays (HAT, cisplatin, and olaparib) demonstrated superior performance compared with the other PIF calculation approaches we considered here.

To gain further insights into our semi-supervised learning results, we generated PIF-PIF plots for different combinations of assay data used in the PIF calculation (Figure S7). We observed that the PIFs calculated using different data for the same BRCA2 variant could be notably different, suggesting that the incorporation of the information from different assays in reliable PIF calculation is necessary because of their complementary and non-redundant nature. For the PIF based on a single assay, the cisplatin-based PIF typically exceeded the olaparib-based PIF, which, in turn, almost always exceeded the HAT-based PIF (Figures S7A–S7C). The three-assay PIF always exceeded (but in many cases negligibly) the corresponding HAT-only PIF (Figure S7D), a relation that was largely reversed for the cisplatin- and olaparib-only PIF (Figures S7E and S7F). However, in all PIF-PIF plots, the largest PIF-PIF differences were detected for the variants that were classified as non-functional by at least one of the PIFs. This implies that our suggested use of all three assays (HAT, cisplatin, and olaparib) in PIF calculation will be particularly important for the identification of nonfunctional BRCA2 variants.

### Comparison of ESC-based variant classification with other available variant information

We next compared our calculated PIF for the variants with the results from other published functional assays, ClinVar data, and in silico predictions (Bayes-del and Priors) and ACMG codes (Figure 4; Tables S4 and S5). Comparison of our prediction of the impact on function for the variants with the results from the high-throughput functional assay based on complementation of PARPi sensitivity (mixed-all nominated-in-one-BRCA [MANO-B] method)12 showed a high concordance with the variants that were classified as fClass 1/2 (neutral) or fClass 4/5 (pathogenic) (Figures 4A and 4D; Table S4). Nine variants that were classified as fClass 1/2 showed a PIF in favor of functional variant classification in our assay, except p.Glu2847Lys, which is classified as pathogenic in our assay (PIF 0.99) (Figure 5A). This variant is classified as intermediate in the homologous recombination (HR)-based functional assay with a probability of 0.343.19 All the variants, except p.Arg2842Leu, that were classified as fClass 4/5 by MANO-B method, showed a PIF in favor of non-functionality (Figures 4A, 4D, and 5A; Table S4). The variant p.Arg2842Leu is classified as functional in our assay (PIF 0.005) (Figure 5A). This variant is intermediate in the HR-based assay with a probability of 0.06 and complements cell survival and HR in the ESC-based assay.15,19 Eight variants (p.Ser1172Leu, p.Thr2337Ala, p.Tyr2905Cys, p.Gly2901Val, p.Asp2913His, p.Asp2913Val, p.Arg2842His, and p.Asp2900Val) that were classified as fClass 2/3 or fClass 3/4 showed a PIF in favor of neutrality in our assay (Figures 4A, 4D, and 5A; Table S4). Among these eight variants, p.Ser1172Leu and p.Arg2842His are classified as benign in ClinVar (Table 1).22 The p.Arg2842His is classified as benign using the HR assay.19 We could not compare the variants that were identified as intermediate by our functional assay because they were not tested by the MANO-B or the HR assay.Figure 4Comparison of multiplexed mESC-based variant classification with other functional, multifactorial, and ClinVar classifications(A) Comparison of variants that were classified using the MANO-B functional assay.12(B) Homologous recombination (HR)-based assay11 and classification using the mESC-based assay.(C) Plot showing the PIFs and classification using the mESC-based assay and classification of the variants in ClinVar. Overlapping dots are circled (dotted), and the number of dots in each circle is represented as n (numbers of variants on each circle). The dots that are not circled represent a single variant.(D) Side-by-side comparison of mESC assay-based classification of variants that have other functional assay data published. We used the data from the MANO-B (mixed-all nominated-in-one-BRCA [MANO-B]) method of classification, the mESC-based assay reported by Mesman et al.,15 and the HR assay-based classification.12,19Figure 5Position of classified variants in BRCA2(A) BRCA2 domains are marked with colored rectangles, with the amino acid (aa) position marked in parentheses. The variants located in different regions are marked below. OB, oligonucleotide/oligosaccharides binding; CTRB, C-terminal RAD51 binding.(B) Position of the splicing variants analyzed in this study. Functional variants are marked in green, non-functionals in magenta, and indeterminants in saffron.

Comparison of multiplexed mESC-based variant classification with other functional, multifactorial, and ClinVar classifications

(A) Comparison of variants that were classified using the MANO-B functional assay.12

(B) Homologous recombination (HR)-based assay11 and classification using the mESC-based assay.

(C) Plot showing the PIFs and classification using the mESC-based assay and classification of the variants in ClinVar. Overlapping dots are circled (dotted), and the number of dots in each circle is represented as n (numbers of variants on each circle). The dots that are not circled represent a single variant.

(D) Side-by-side comparison of mESC assay-based classification of variants that have other functional assay data published. We used the data from the MANO-B (mixed-all nominated-in-one-BRCA [MANO-B]) method of classification, the mESC-based assay reported by Mesman et al.,15 and the HR assay-based classification.12,19

Position of classified variants in BRCA2

(A) BRCA2 domains are marked with colored rectangles, with the amino acid (aa) position marked in parentheses. The variants located in different regions are marked below. OB, oligonucleotide/oligosaccharides binding; CTRB, C-terminal RAD51 binding.

(B) Position of the splicing variants analyzed in this study. Functional variants are marked in green, non-functionals in magenta, and indeterminants in saffron.

Among the selected variants, 15 variants with mutations in the DNA-binding domains of BRCA2 have been evaluated previously using the HR assay for prediction of pathogenicity.19 Nine of them (p.Leu2865Val, p.Phe2873Cys, p.Arg2842His, p.Glu2856Ala, p.Glu3152Lys, p.Tyr3098His, p.Ser3123Gly, p.Ala3088Val, and p.Tyr3092Cys) were classified as benign, three (p.Asn3124Ile, p.Leu3125His, and p.Asp3095Glu) as pathogenic, and three (p.Arg2842Leu, p.Tyr3092Ser, and p.Glu2847Lys) as intermediate using the HR assay. All variants classified as benign or pathogenic by the HR assay were classified in our NGS-based mouse ESC assay as functional or non-functional, respectively (Figures 4A, 4D, and 5A; Tables 1 and S4). Two intermediate variants, p.Arg2842Leu and p.Tyr3092Ser, with probabilities of 0.06 and 0.1, respectively, in the HR assay are classified as functional in our assay, with a PIF of 0.005 for p.Arg2842Leu and 0.016 for p.Tyr3092Ser (Figure 5A). Both variants rescued ESC lethality and complemented the HR defect in mESCs.15 The p.Glu2847Lys variant is classified as non-functional in our assay (Figure 5A; Table 1). Based on cell survival, p.Glu2847Lys is functional (HAT PIF 0.04). However, when the cell viability and drug sensitivity assays were combined, it was classified as non-functional (PIF 0.99). Seven of the selected variants (p.Tyr3092Ser, p.Arg2888Cys, p.Tyr3098His, p.Asn3124Ile, p.Asp3095Glu, p.Arg2842Leu, and Glu2856Ala) were analyzed by Mesman et al.15 using an mESC-based assay, and two of them (p.Asn3124Ile and p.Asp3095Glu), which failed to complement cell viability, are classified as non-functional in our assay (Figures 4D and 5A). The remaining five variants that were able to complement cell viability and supported HR in the mESC-based assay are classified as functional in our assay (Figure 5A; Table S4).15

All variants that were classified as pathogenic in ClinVar (p.Asp3095Glu, p.Leu3125His, c.8633-1G>A, c.8633-2 A>G, c.8633-2 A>T, c.8754 + 1 G>A, c.8754 + 1 G>T, and c.8754 + 2 T>G) are classified as non-functional in our assay, except for p.Ala2911Glu, which has a PIF in favor of functional (Figures 4C, 5A, and 5B; Table 1). This variant is classified as fClass 1/2 by the MANO-B method.12 ClinVar has classified this variant with no assertion criteria, and the classification was based on a single occurrence in a Fanconi anemia patient.12,23

Computational determination of prior probabilities of pathogenicity is based on the location of the mutated amino acid in the protein as well as the impact of the mutated nucleotide on splicing (database: http://hci-priors.hci.utah.edu/PRIORS/),24 and the in silico prediction tool BayesDel score combines multiple deleteriousness predictors to calculate an overall score.25 In BayesDel, the PIF criterion for benign for BRCA2 is PIF < 0.08, and the PIF criterion for pathogenic is PIF > 0.5.26 These comparisons show high agreement with the PIF values and functionality predictions obtained from our assay (Tables 1 and S4).

### Discussion

There are more than 3,000 BRCA2 missense variants that are classified as VUS in ClinVar. It is important to classify these variants using multiple functional assays with high sensitivity and specificity to determine their functional impact. Several functional assays, including our mESC-based assays, have been developed to determine the functional impact of BRCA2 VUSs.12,13,15,19 The results of these functional assays have been used in computational models to calculate the PIF for the individual variant.9,12,19

Although we have used the mESC-based functional assays to functionally classify more than 150 BRCA2 variants, the process to generate the variants and examine their functional impact is laborious and time consuming. To expedite the process, we now multiplexed the assay using an NGS-based approach. We used the change in the variant sequence counts in the pools of variants to determine the effect of each variant on cell viability and sensitivity to cisplatin and olaparib. This is based on the change in the frequency of the sequence reads corresponding to each variant relative to their frequency prior to the HAT, cisplatin, and olaparib treatments. A significant reduction in the frequency after the treatments suggests that a loss of protein function is caused by the variant. By this approach, we were successful in simultaneously analyzing multiple variants located within the same exon of BRCA2. We also combined the results on cell viability and sensitivity to two different drugs. This is an advantage over other functional assays, where impact on cell viability, HR, or sensitivity to DNA-damaging drugs was evaluated.12,19,27 We previously reported a Bayesian hierarchical model that provided improved accuracy in predicting PIFs.9 The PIF calculation algorithm we developed in this study shares a number of important features with that model, such as the use of semi-supervised learning and mixtures of normal distributions as well as the idea of combining the mixtures that model the data obtained from different experimental assays. At the same time, our algorithm appears to be simpler, logically and computationally, than our previously used model.

It is to be noted that the mESCs expressing the variants were treated with drugs after HAT selection and that the variants that failed to rescue viability in HAT were excluded from the drug assay. The variants that survived HAT selection are the only ones that can provide information on the response to cisplatin and olaparib treatment. Calculation of PIF from combining multiple assays helps us to classify the variants with more confidence. We have observed previously that some variants surviving in HAT (0.05 < PIF ≤ 0.99) exhibited a wide range of drug sensitivities (none to extreme sensitivity), impacting their functional status obtained only from cell viability data.9 Here, we identify five variants (p.Pro2381Gln, p.Leu2904Phe, p.Leu2890Ile, p.Phe2356Leu, and p.Leu2898Val) as intermediate variants (0.05 < PIF ≤ 0.99), with more susceptibility to cisplatin or olaparib compared with HAT (Figure S1A; Table 1). All of these variants are in the DNA-binding domain of BRCA2, a region where most of the variants that affect BRCA2 function are located. However, no other functional assay data are available for these variants. Their intermediate status will be confirmed in the future based on data from different functional assays and/or epidemiological data.

In our assay, other than the non-sense variants, 16 missense variants had a PIF greater than 0.99. The defective function of p.Asp3095Glu, p.Leu3125His (PIF > 0.998), p.Asn3124Ile, c.8633-1G>A, c.8633-2 A>G, c.8633-2 A>T, c.8754+1 G>A, c.8754+1 G>T, and c.8754+2 T>G is supported by multifactorial data or other functional assays.12,19,22,28 When we previously analyzed p.Asp3095Glu and p.Asn3124Ile using our mESC-based assay, they were classified as non-functional.9 This further supports that our approach of multiplexing can classify the variants as accurately as when we examine them individually. We found seven missense variants (p.Phe2349Leu, p.Leu3101Arg, p.Glu3152Gly, p.Gly2379Val, p.Gly2379Glu, p.Glu2369Asp, p.Tyr2383His, and p.Pro2883Ser) with PIFs in favor of non-functionality. Among those missense variants, p.Leu3101Arg is classified as fClass 4/5 using a functional assay based on complementation of PARPi sensitivity, supporting our data.12 There are no functional or multifactorial data available for other variants. The missense variants that have an impact on BRCA2 function are all located in the DNA-binding domain, where many non-functional variants are located.

Among the 223 variants analyzed in this report, 65 are in the largest exon of BRCA2, exon 11. All BRC repeats involved in RAD51 binding are encoded by exon 11. We failed to identify any non-functional missense variants in this region, suggesting possible redundancy of function of BRC repeats (Figure 5A). Notably, two missense mutations located in BRC2 and BRC7 (p.Ser1221Pro and p.Thr1980Ile) have been reported recently to have a significant impact on BRCA2 function.29 These variants have not been evaluated by us or by other functional assays.

The results obtained by our sequencing-based multiplexing approach using our mESC-based functional assays are highly consistent with the International Agency for Research on Cancer (IARC) classification and other functional assays. We could not compare our data with CRISPR-based prime editing data because of the lack of overlap between the variants that were analyzed.10 There is a clear discrepancy between our results and ClinVar in the classification of the p.Ala2911Glu variant, which was found in a single Fanconi anemia (FA) patient.23 Our functional evaluation of the variant suggests that p.Ala2911Glu has no functional impact. We hypothesize that the variant contributing to the FA phenotype remains to be identified. Another BRCA2 variant, p.Lys2729Asn (c.8187G>T), was identified in an FA patient and was a pathogenic variant.23 Functional evaluations, including those by us, suggested this variant to be functional.7,11 Re-evaluation of the patient DNA and cDNA samples revealed the presence of another variant in cis in the 5′ untranslated region.30 This variant has been shown to reduce mRNA stability, thus affecting BRCA2 function.30 It will be interesting to find out whether there are other variants present in the FA patient that may be responsible for the FA phenotype. These findings demonstrate the importance of inclusion of functional data in evaluating the impact of BRCA2 variants to determine the risk of cancer for mutation carriers.

To keep pace with the rate at which new BRCA2 VUSs are being identified, high-throughput functional assays, such as the clustered regularly interspaced short palindromic repeats (CRISPR)-based saturation genome editing (SGE), are being developed. Although SGE has the potential to classify thousands of variants, the number of BRCA2 VUSs listed in ClinVar that have been analyzed by SGE so far is quite limited.10,27 Future SGE studies targeting the entire coding sequence of the gene will lead to the functional classification of more VUSs. However, VUS classification results using the data from a single functional assay are less likely to be used for patient risk assessment and clinical management. Having results from multiple well-established functional assays that have high sensitivity and specificity, as well as availability of epidemiological data and co-segregation and/or co-co-occurrence data, will together help with functional assessment of VUSs. In conclusion, we developed a multiplexing strategy for an established mESC-based functional assay that classifies BRCA2 variants with high sensitivity and specificity and can be used to expedite VUS classification in the future.

In this study, BRCA2 variants were classified based on functional scores that were calculated using the DNA sequencing reads, which does not take into consideration the transcript levels of the variants in the pool. We did not demonstrate that our approach can be used to examine the impact of variants on splicing. RNA sequencing (RNA-seq) analysis using appropriate PCR primers and total RNA from cells in the initial pool and HAT pool, when combined with the DNA sequencing reads, can identify variants that affect splicing or mRNA stability.

### Limitations of study

In this study, BRCA2 variants were classified based on functional scores that were calculated using the DNA sequencing reads, which does not take into consideration the transcript levels of the variants in the pool. We did not demonstrate that our approach can be used to examine the impact of variants on splicing. RNA sequencing (RNA-seq) analysis using appropriate PCR primers and total RNA from cells in the initial pool and HAT pool, when combined with the DNA sequencing reads, can identify variants that affect splicing or mRNA stability.

### STAR★Methods

REAGENT or RESOURCESOURCEIDENTIFIERBacterial and virus strainsOne shot Top10 competent cellsInvitrogenCat# C404003E. coli SW102National Cancer InstituteNCI recombineering website: https://redrecombineering.ncifcrf.gov/Chemicals, peptides, and recombinant proteinsGelatin (0.1%)Stemcell TechnologiesCat# 07903Knockout Dulbecco’s Modified Eagle Medium (DMEM)GibcoCat# 10829-018FBS (Hi Performance)GibcoCat# 16000-044b-mercaptoethanolSigmaCat# M3148Glutamine Penicillin StreptomycinThermo Fisher ScientificCat# 10378016Trypsin EDTA (0.5%), no phenol redThermo Fisher ScientificCat# 15400054DPBS (1X)GibcoCat# 14190-144Dimethyl sulfoxide (DMSO)Millipore SigmaCat# D2650CisplatinSigma AldrichCat# P4394OlaparibSelleckchemCat# AZD2281G418Thermo Fisher ScientificCat# 10131035AmpicillinSigma AldrichCat# A9393GalactoseSigma AldrichCat# 482602-deoxy-galactoseSigma AldrichCat# D4407BiotinSigma AldrichCat# B4501L-LeucineSigma AldrichCat# L8000KanamycinSigma AldrichCat# K1637GlycerolVWRCat# J61059.K2(NH4)2SO4VWRCat# BDH9216KH2PO4VWRCat# BDH9268FeSO4⋅7H2OVWRCat# 95033-256MgSO4·7H2OVWRCat# MK569124KOHVWRCat# BDH9262Bacto-tryptoneVWRCat# 76628Yeast extractVWRCat# AAJ60287-36NaClVWRCat# BDH9286WaterThermo Fisher ScientificCat# 10977015KH2PO4VWRCat# BDH9268Na2HPO4⋅7H2OVWRCat# 95035-872NH4ClVWRCat# BDH9208AgaroseVWRCat# 97064AntibodiesRabbit polyclonal BRCA2Bethyl LabCat # A303-434A-T-1, RRID: AB_3073617Mouse monoclonal Vinculin antibodySanta Cruz BiotechCat# sc25336, RRID: AB_628438Goat anti-rabbit HRP conjugatedThermo Fisher ScientificCat# 31460, RRID:AB_228341Goat anti-mouse HRP conjugatedThermo Fisher ScientificCat# 31430, RRID: AB_228307Recombinant DNApGalKNational Cancer InstituteNCI recombineering website: https://redrecombineering.ncifcrf.gov/Critical commercial assaysQIAprep Spin miniprep kitQiagenCat# 27104Qiagen Plasmid Maxi kitQiagenCat# 12165Mouse embryonic stem cell nucleofector kitLonzaCat# VPH-1001Zymo DNA isolation kitZymo researchCat# D3020Platinum Taq Hifi DNA PolymeraseInvitrogenCat# 11304011QIAquick Spin columnsQiagenCat# 28115QIAquick PCR purification kitQiagenCat# 28106One-tube RT-PCR kitQiagenCat# 21021096-well RNA isolation kitThermo Fisher ScientificCat# 12173011AECL plus Western blotting detection systemAmershamCat# RPN2132QIAquick gel extraction kitQiagenCat# 28704TruSeq Nano DNA Library Prep KitIlluminaCat# 20015965Experimental models: Cell linesMouse embryonic stem cells (Brca2Cko/-) [Clone: PL2F7]Sharan labKuznetsov et al., 2008SNLP mouse feedersSharan labKuznetsov et al., 2008OligonucleotidesEx11d-ForwardATACCTTGGCATTAGATAATCIntegrated DNA TechnologiesN/AEx11d-ReverseCAACTGTACCTTCAAATTGCIntegrated DNA TechnologiesN/AEx11c-ForwardCAGACTTGACTTGTGTAAACGAACCIntegrated DNA TechnologiesN/AEx11c-ReverseGTATTAATTGACTGAGGCTTGCIntegrated DNA TechnologiesN/AExon 14-forwardAGAATAGTATCACCATGTAGCIntegrated DNA TechnologiesN/AExon 14-reverseAAGACTTTGGTTGGTCTGCCIntegrated DNA TechnologiesN/AExon 20-Forward cgaactcctgacctcaggtgatccIntegrated DNA TechnologiesN/AExon 20-Reverse ggcttagacctgatatttctgtcccIntegrated DNA TechnologiesN/AExon21-ForwardCTTTGGGTGTTTTATGCTTGGIntegrated DNA TechnologiesN/AExon21-ReverseATCAAGCCTCATTATATGTCCIntegrated DNA TechnologiesN/AExon 25-ForwardCATCTAACACATCTATAATAACATTCIntegrated DNA TechnologiesN/AExon 25-ReverseGTGGTGATGCTGAAAAGTAACCIntegrated DNA TechnologiesN/AExon 11-RT-FwdTGGTTTTGTCAAATTCAAGAATTGGIntegrated DNA TechnologiesN/AExon 14 RT-RevCCAATCAAGCAGTAGCTGTAACTTTCACIntegrated DNA TechnologiesN/ADeposited dataNGS sequencing dataThis paperData: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244578Statistical analysis CodeThis paperCode: https://zenodo.org/record/8160202Software and algorithmsGenomic data analysis, statistical analysisR studioSoftware: https://posit.co/download/rstudio-desktop/Sanger sequencing analysis4PeaksSoftware: https://nucleobytes.com/4peaks/Genome editing analysisCrispresso2Tools: http://crispresso2.pinellolab.org/submissionVariant callerANNOVARDatabase: https://annovar.openbioinformatics.org/en/latest/#referenceExcelMicrosoftSoftware: https://www.microsoft.com/en-us/microsoft-365/excelAcrobatAdobeSoftware: https://www.adobe.com/acrobat.htmlIllustratorAdobeSoftware: https://www.adobe.com/products/illustrator.htmlOtherVac-Man vacuum manifoldPromegaCat# PR-A7231Micropulser electroporatorBio-RadCat# 1652100Gene Pulser Xcell Electroporation SystemsBio-RadCat# 1652660

Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Shyam K. Sharan (sharans@mail.nih.gov).

All cell lines generated in this study are available upon request.

PL2F7 mouse embryonic stem cells are derivative of AB2.2 cells (male)13 and were maintained in Knock-out DMEM supplemented with 15% FBS, 1X GPS and 0.0072μl/ml b-mercaptoethanol at 37°C and 5% CO2.

HGVS nomenclature for cDNA and proteins were followed, in which cDNA numbers with +1 correspond to A of the ATG initiation codon in BRCA2 sequence (GenBank accession number NM_000059.3). Bayes-del scores were obtained from BayesDel database (https://fengbj-laboratory.org/BayesDel). Variants with a high BayesDel score are predicted to be non-functional and those with a low score are functional. The PRIOR probability of variants were obtained from the HCI database (http://priors.hci.utah.edu/PRIORS).

All selected variants of an exon were generated using PCR by amplifying about 400 bp of exon or exons with adjacent intron sequences to make a pool of variant DNA. The pooled DNA was used to introduce the variants into the BAC clone (CTD-2342K5 with a 127 kb insert) containing full-length BRCA2 in SW102 cells by a recombineering method as described previously.7,15 After recombineering, BACs were sequenced to confirm the presence of the correct variant and absence of any undesired mutations. Oligonucleotide sequences are available upon request.

BAC DNA (20 μg) carrying various mutant alleles of BRCA2 was electroporated into 1.0 × 107 PL2F7 mESCs, selected in the presence of G418 (Invitrogen) and characterized as described previously.9 Each BAC carrying a single BRCA2 variant was electroporated and selected individually to ensure that the ES cells harbor a single mutant BAC clone. For selecting the BRCA2 expressing clones, protein isolation and Western blot was carried out as described before.9 Rabbit polyclonal BRCA2 (recognizes an epitope between residues 450–500) antibody (BETHYL lab, Cat # A303-434A-T-1, 1:2000 dilution), and mouse monoclonal Vinculin antibody (Santa Cruz biotech, Cat# sc25336, 1:200,000 dilution) were used to detect proteins. ECL plus Western blotting detection system (Amersham) was used for chemiluminescent detection. To confirm BRCA2 expression for intronic and nonsense variants, RT-PCR was performed using One tube RT-PCR kit (Qiagen) primers from exon 11 (5′-TGGTTTTGTCAAATTCAAGAATTGG-3′) and exon 14 (5′-CCAATCAAGCAGTAGCTGTAACTTTCAC-3′) following manufacturer’s protocol. DNA from BAC containing mESC clones expressing the human BRCA2 was used to further confirm the presence of the correct variant without any undesired mutation by sequencing.

Ten to twenty-five BAC containing PL2F7 ESC clones, each expressing a single BRCA2 variant, were pooled and cultured for two passages. The variants in each pool were from the same exon and were located within the span of ∼300 bp. Two such pools using two independent clones expressing each BRCA2 variant were generated. The conditional allele of Brca2 in mESCs was deleted by electroporating 20 μg of Pgk-Cre plasmid as described previously.9 Two independent electroporations were carried out for each pool and were subjected to the same treatment at subsequent steps. After electroporation, 7.5 × 106 cells were subjected to HAT selection as described previously.9 An equal number of unelectroporated cells were cultured without drug selection. This sample served as the M15 control. Cells were collected for genomic DNA isolation after HAT selection and for the M15 control. One million HAT selected cells were subjected to drug sensitivity. For drug selection, 0.4 μM cisplatin and 0.05 μM PARP inhibitor (olaparib) were used for five days with re-feeding drug in media on day 3 and 5. After drug selection, cells were collected for genomic DNA isolation.

The genomic DNA obtained from collected cell samples was used for PCR-amplification of the respective exons of BRCA2 using Invitrogen Platinum High-fidelity Taq polymerase according to the manufacturer’s protocol. Ten PCR reactions for each sample were carried out and the PCR products pooled together and purified using Qiagen PCR purification kit. For each batch of variants, sixteen samples were utilized [2 pools (biological replicates) × 2 Pgk-Cre electroporation (technical replicates) × 4 treatments (M15, HAT, Cis, Ola)] for deep sequencing of the respective exons using Illumina MiSeq paired end sequencing on the MiSeq sequencer (2 × 300 cycles) allocating ∼3 million reads for each sample to quantify the relative abundance of each variant at different conditions used. Library for Next Gen sequencing was prepared using TruSeq Nano DNA Library Prep Kit from Illumina according to the manufacturer’s protocol. The quantity and quality of the obtained libraries were evaluated using a Qubit 2.0 fluorometer (Thermo Fisher Scientific) and an Agilent 2200 TapeStation system.

The paired-end reads were aligned to the reference sequence using the Needleman-Wunsch alignment algorithm after the reads were demultiplexed using bcl2fastq (Illumina). The fastq files were merged using FLASH and CRISPResso2 was used for quantifying the total number of aligned reads.31 The individual unique alignments were annotated with a custom-made variant caller modified from ANNOVAR (release 2019-10-21) and the R/Bioconductor Biostrings package in R version 4.1.2 (Software: https://bioconductor.org/packages/Biostrings).31,32 Merged reads containing “N” bases and any insertions or deletions were removed from the analysis. The abundances of SNVs were quantified when the reads contained a single-nucleotide substitution and no additional mutations or deletions in the sequencing read. Read counts for each SNVs were then normalized to the total read coverage of the sequencing library after adding a pseudo-count of 1 to all reads for all conditions. Individual variants with a read count of more than 1 in 1,000 reads in the M15 condition were used for further analysis. Dropout or enrichment scores were calculated by taking the ratio of frequency of SNV after HAT, cisplatin, or olaparib treatment over that in M15. The scores were expressed in log2 scale, which we define as the functional scores of SNVs in HAT, cisplatin, and olaparib. The functional scores, averaged between 2 or 4 independent replicates, were used to calculate the probabilities of impact on function (PIF) for all the variants in the dataset.

Abundance of variants was calculated only using sequencing reads that had complete sequence alignment with BRCA2, except for single-nucleotide variation. The frequency of each variant was calculated as the number of variant reads divided by the total number of aligned reads. The variants that had a frequency of at least 1 in 1000 in the M15 sample were subjected to further analysis. Functional scores were calculated for the HAT, cisplatin, and olaparib treatments separately by calculating the log2 ratio of the frequency of treatment over the average frequency of the two biological replicates (two independent Pgk-Cre delivery) for the M15 samples.

The functional scores from each of the three assays (HAT, cisplatin, and olaparib) were regarded as statistical samples (with one measurement for a given assay and a given BRCA2 variant constituting one random data point) and were analyzed using statistical and machine-learning methods to calculate PIF for the variants. For each assay, the distributions of the replicate-averaged and log-transformed functional scores were tested for normality using normal quantile-quantile plots; such plots were generated separately for benign and pathogenic BRCA2 variants. Moreover, each of the six statistical samples (3 assays × 2 ClinVar-classification phenotypes) was tested for normality using the Lilliefors test. The details of the statistical models and algorithms for PIF calculation are described below.

For each of the three assays (HAT, cisplatin, and olaparib), the functional-score distribution characterizing the entire dataset (n=223, the total number of BRCA2 variants analyzed) was modeled as a mixture of two normal distributions. The two mixture components represented pathogenic and benign BRCA2 variants, respectively. The distributions were fit to the functional-score data by numerical likelihood maximization (implemented as negative log likelihood minimization in the code).33

Before the fitting, each mixture model was initialized by assigning equal weights of 0.5 to the mixture components (i.e., the prior probability of pathogenicity was set to 0.5), and the components’ means and variances were set to the sample means and variances estimated from the corresponding data subset [characterizing either benign (n=16) or pathogenic (n=27) BRCA2 variants]. For the fitting, we used the R-language optim() function with typical parameters (the Nelder-Mead optimization algorithm with a relative tolerance of 1.0 × e−8).

The semi-supervised-learning nature of our method was reflected in the structure of the likelihood function, which contained information on BRCA2 variants with a known phenotype (i.e., pathogenic or benign), as well as variants of uncertain significance (VUS). Let f(x,m,v) denote the normal probability density with mean m and variance v. Then, the two-component mixture in our case has the general form,g(x,p,mp,vp,mb,vb)=pf(x,mp,vp)+(1−p)f(x,mb,vb),where p is the prior probability of pathogenicity, mp and vp are the mean and variance of the mixture component for the pathogenic variants, and mb and vb are the mean and variance of the mixture component for the benign variants. Using this notation, the likelihood, l(…), can be written as follows:l(p,mp,vp,mb,vb|x)=∏i=1Nppf(xi(p),mp,vp)×∏i=1Nb(1−p)f(xi(b),mb,vb)×∏i=1Nug(xi(u),p,mp,vp,mb,vb),where x is the vector of the functional scores of the variants in the training set, x=(xi(p),xi(b),xi(u)); xi(p), xi(b), and xi(u) are the functional scores for pathogenic, benign, and VUS variants, respectively; Np, Nb, and Nu are the numbers of pathogenic, benign, and VUS variants, respectively, in the training set.

The parameters of the two-component mixture, including the prior probability of pathogenicity p, were obtained as a result of maximizing the likelihood the way described at the beginning of this subsection. This fitting strategy was applied independently to the data characterizing each of the three assays (HAT, cisplatin, and olaparib). The distributions obtained as the result of the fitting were used to calculate assay-specific probabilities of pathogenicity (denoted PPi) for individual BRCA2 variants using the Bayes formula:PPi=pf(xi,mp,vp)g(xi,p,mp,vp,mb,vb),where xi is the functional score of the variant i, and the distribution-parameter values on the right-hand side (i.e., p,mp,vp,mb,vb) are the ones maximizing the likelihood. Importantly, this formula was also used to calculate PPi in the case when the model trained on one dataset was applied to another dataset, here termed the target set (such as in cross-validation and bootstrapping; see below). In such a case, the variable values (i.e., the functional scores for HAT, cisplatin, and olaparib) were taken from the target set, whereas the probability-density parameters were taken from the distributions fitted to the training set.

For each of the three assays, we denote the PPi by PPi(HAT), PPi(Cis), and PPi(Ola), respectively, where i is an index marking individual BRCA2 variants in the target dataset (i=1,…,223 in the case when the target set is our full dataset; the cases of target subsets are treated in a similar way). The PPi were used to calculate the probabilities of impact on function (PIFi) as follows:(Equation 1)PIFi=PPi(HAT)+(1−PPi(HAT))PPi(Cis)PPi(Ola).

This is a heuristic formula motivated by the total probability formula, and it can be interpreted as follows. If we know that PPi(HAT) is large (i.e., close to 1), then it is likely that the variant i is pathogenic, i.e., PIFi should also be large. Alternatively, if PPi(HAT) is rather small (i.e., close to 0), then the variant i will still likely be pathogenic if both PPi(Cis) and PPi(Ola) are large. The formula in Equation 1 allows us to define PIFi in a way that reflects this logic and also makes sure that PIFi vary between 0 and 1, as properly defined probabilities should.

Equation 1 is the PIF formula used in the main version of our algorithm. Additionally, we considered algorithm versions where the PIFi were calculated using the data from only one of the three assays: HAT, cisplatin, or olaparib. In those cases, the PIFi were set to be equal to PPi(HAT), PPi(Cis), or PPi(Ola), respectively.

We compared our main, semi-supervised-learning version of the algorithm (described above) with a supervised-learning version to assess the relative advantages of semi-supervised learning in the context of PIF estimation. The difference between the semi-supervised-learning and supervised-learning algorithm versions was that, in the latter, the likelihood function incorporated the data only from the BRCA2 variants for which the phenotype was known (i.e., only the pathogenic and benign variants; no VUS). Specifically, the supervised-learning likelihood function had the form:l(p,mp,vp,mb,vb|x)=∏i=1Nppf(xi(p),mp,vp)×∏i=1Nb(1−p)f(xi(b),mb,vb),where the notation is the same as above.

From the calculated PIFi, the phenotype (pathogenic or benign) for each BRCA2 variant, i, was predicted using the thresholds accepted in the BRCA2 research community: PIFi≤ 0.05 indicates a benign BRCA2 variant, PIFi>0.99 indicates a pathogenic variant, and all other cases are considered indeterminate.9 When applied to the full dataset (or to its subsets), the classification accuracy of our algorithm was calculated as the percentage of the BRCA2 variants with a known phenotype (i.e., pathogenic or benign) whose phenotype was predicted correctly from the algorithm-generated PIFi. Predictive accuracy of the algorithm on the full dataset (n=223) was assessed using K-fold cross-validation with K=3,6,9,43. For each variant, the PIF that was used in the K-fold cross-validation accuracy assessment was calculated – and compared to the known benign/pathogenic labels – only for the partition in which this variant was in the cross-validation test set (i.e., the set not used for model fitting in that cross-validation round). For each K, the folds were defined by (randomized) partitioning of the data subset containing only the variants with a known phenotype. Because the number of known-phenotype BRCA2 variants in our full dataset was 43, 43-fold cross-validation was equivalent to leave-one-out cross-validation. The accuracy of K-fold cross-validation was calculated as the classification accuracy on each of the K folds, averaged across all the folds.

While the commonly used values for K include 5 and 10, our choice of K represented a spectrum of possible K values, intended to generate a more complete cross-validation picture.34 We used a generalized cross-validation algorithm, in which the integer K did not have to be a divisor of 43 (indeed, 43 is a prime). In that generalization, we set the fold size, F(K), to [43/K] (here, the brackets denote the integer part of a number) and, for each K, the cross-validation training set consisted of F(K)K randomly selected variants with a known phenotype.

For the semi-supervised-learning algorithm version, the datasets for each of the K−1 folds used for training included all the VUS, which were appended to the pathogenic and benign variants (that were randomly selected for that fold) and used together in the training procedure; in the remaining fold – used for testing – VUS were not included. That way, for each K, the training data were completely separated from the test data. For the supervised-learning algorithm version, the VUS were not used in the cross-validation procedure.

We calculated 95% confidence intervals for the PIFi using bootstrapping. Bootstrapping is a common resampling strategy that allows one to quantify uncertainty in the output of a statistical model. It involves random sampling with replacement from a given dataset, which yields a bootstrapped dataset.34 Ten thousand bootstrapped datasets were generated from the full experimental dataset (n=223). To preserve the structure of the dataset, the pathogenic (n=27), benign (n=16), and VUS (n=180) variants were bootstrapped independently. We then fitted our statistical model (i.e., the combined mixtures of two normal distributions described above) to each of the bootstrapped datasets using the semi-supervised-learning approach, and then applied the fitted model to our full dataset. Thus, for each i (i=1,…,223), we had a sample of 10,000 values of PIFi, from which we calculated the confidence interval for PIFi using the percentile method.35 The confidence intervals for the PIFi from our supervised-learning approach were calculated in a similar way, but VUS were not used in the bootstrap.

The PIF calculation and analysis code was written in R 4.0.2 (2020-06-22), using RStudio 1.3.1073, and was developed and run on a Dell Latitude 7400 laptop computer with an Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz processor and 16 Gb RAM under the 64-bit Windows 10 Enterprise 20H2 operating system.

### Key resources table

REAGENT or RESOURCESOURCEIDENTIFIERBacterial and virus strainsOne shot Top10 competent cellsInvitrogenCat# C404003E. coli SW102National Cancer InstituteNCI recombineering website: https://redrecombineering.ncifcrf.gov/Chemicals, peptides, and recombinant proteinsGelatin (0.1%)Stemcell TechnologiesCat# 07903Knockout Dulbecco’s Modified Eagle Medium (DMEM)GibcoCat# 10829-018FBS (Hi Performance)GibcoCat# 16000-044b-mercaptoethanolSigmaCat# M3148Glutamine Penicillin StreptomycinThermo Fisher ScientificCat# 10378016Trypsin EDTA (0.5%), no phenol redThermo Fisher ScientificCat# 15400054DPBS (1X)GibcoCat# 14190-144Dimethyl sulfoxide (DMSO)Millipore SigmaCat# D2650CisplatinSigma AldrichCat# P4394OlaparibSelleckchemCat# AZD2281G418Thermo Fisher ScientificCat# 10131035AmpicillinSigma AldrichCat# A9393GalactoseSigma AldrichCat# 482602-deoxy-galactoseSigma AldrichCat# D4407BiotinSigma AldrichCat# B4501L-LeucineSigma AldrichCat# L8000KanamycinSigma AldrichCat# K1637GlycerolVWRCat# J61059.K2(NH4)2SO4VWRCat# BDH9216KH2PO4VWRCat# BDH9268FeSO4⋅7H2OVWRCat# 95033-256MgSO4·7H2OVWRCat# MK569124KOHVWRCat# BDH9262Bacto-tryptoneVWRCat# 76628Yeast extractVWRCat# AAJ60287-36NaClVWRCat# BDH9286WaterThermo Fisher ScientificCat# 10977015KH2PO4VWRCat# BDH9268Na2HPO4⋅7H2OVWRCat# 95035-872NH4ClVWRCat# BDH9208AgaroseVWRCat# 97064AntibodiesRabbit polyclonal BRCA2Bethyl LabCat # A303-434A-T-1, RRID: AB_3073617Mouse monoclonal Vinculin antibodySanta Cruz BiotechCat# sc25336, RRID: AB_628438Goat anti-rabbit HRP conjugatedThermo Fisher ScientificCat# 31460, RRID:AB_228341Goat anti-mouse HRP conjugatedThermo Fisher ScientificCat# 31430, RRID: AB_228307Recombinant DNApGalKNational Cancer InstituteNCI recombineering website: https://redrecombineering.ncifcrf.gov/Critical commercial assaysQIAprep Spin miniprep kitQiagenCat# 27104Qiagen Plasmid Maxi kitQiagenCat# 12165Mouse embryonic stem cell nucleofector kitLonzaCat# VPH-1001Zymo DNA isolation kitZymo researchCat# D3020Platinum Taq Hifi DNA PolymeraseInvitrogenCat# 11304011QIAquick Spin columnsQiagenCat# 28115QIAquick PCR purification kitQiagenCat# 28106One-tube RT-PCR kitQiagenCat# 21021096-well RNA isolation kitThermo Fisher ScientificCat# 12173011AECL plus Western blotting detection systemAmershamCat# RPN2132QIAquick gel extraction kitQiagenCat# 28704TruSeq Nano DNA Library Prep KitIlluminaCat# 20015965Experimental models: Cell linesMouse embryonic stem cells (Brca2Cko/-) [Clone: PL2F7]Sharan labKuznetsov et al., 2008SNLP mouse feedersSharan labKuznetsov et al., 2008OligonucleotidesEx11d-ForwardATACCTTGGCATTAGATAATCIntegrated DNA TechnologiesN/AEx11d-ReverseCAACTGTACCTTCAAATTGCIntegrated DNA TechnologiesN/AEx11c-ForwardCAGACTTGACTTGTGTAAACGAACCIntegrated DNA TechnologiesN/AEx11c-ReverseGTATTAATTGACTGAGGCTTGCIntegrated DNA TechnologiesN/AExon 14-forwardAGAATAGTATCACCATGTAGCIntegrated DNA TechnologiesN/AExon 14-reverseAAGACTTTGGTTGGTCTGCCIntegrated DNA TechnologiesN/AExon 20-Forward cgaactcctgacctcaggtgatccIntegrated DNA TechnologiesN/AExon 20-Reverse ggcttagacctgatatttctgtcccIntegrated DNA TechnologiesN/AExon21-ForwardCTTTGGGTGTTTTATGCTTGGIntegrated DNA TechnologiesN/AExon21-ReverseATCAAGCCTCATTATATGTCCIntegrated DNA TechnologiesN/AExon 25-ForwardCATCTAACACATCTATAATAACATTCIntegrated DNA TechnologiesN/AExon 25-ReverseGTGGTGATGCTGAAAAGTAACCIntegrated DNA TechnologiesN/AExon 11-RT-FwdTGGTTTTGTCAAATTCAAGAATTGGIntegrated DNA TechnologiesN/AExon 14 RT-RevCCAATCAAGCAGTAGCTGTAACTTTCACIntegrated DNA TechnologiesN/ADeposited dataNGS sequencing dataThis paperData: https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE244578Statistical analysis CodeThis paperCode: https://zenodo.org/record/8160202Software and algorithmsGenomic data analysis, statistical analysisR studioSoftware: https://posit.co/download/rstudio-desktop/Sanger sequencing analysis4PeaksSoftware: https://nucleobytes.com/4peaks/Genome editing analysisCrispresso2Tools: http://crispresso2.pinellolab.org/submissionVariant callerANNOVARDatabase: https://annovar.openbioinformatics.org/en/latest/#referenceExcelMicrosoftSoftware: https://www.microsoft.com/en-us/microsoft-365/excelAcrobatAdobeSoftware: https://www.adobe.com/acrobat.htmlIllustratorAdobeSoftware: https://www.adobe.com/products/illustrator.htmlOtherVac-Man vacuum manifoldPromegaCat# PR-A7231Micropulser electroporatorBio-RadCat# 1652100Gene Pulser Xcell Electroporation SystemsBio-RadCat# 1652660

### Resource availability

Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Shyam K. Sharan (sharans@mail.nih.gov).

All cell lines generated in this study are available upon request.

### Lead contact

Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Shyam K. Sharan (sharans@mail.nih.gov).

### Materials availability

All cell lines generated in this study are available upon request.

### Experimental model and study participant details

PL2F7 mouse embryonic stem cells are derivative of AB2.2 cells (male)13 and were maintained in Knock-out DMEM supplemented with 15% FBS, 1X GPS and 0.0072μl/ml b-mercaptoethanol at 37°C and 5% CO2.

### Method details

HGVS nomenclature for cDNA and proteins were followed, in which cDNA numbers with +1 correspond to A of the ATG initiation codon in BRCA2 sequence (GenBank accession number NM_000059.3). Bayes-del scores were obtained from BayesDel database (https://fengbj-laboratory.org/BayesDel). Variants with a high BayesDel score are predicted to be non-functional and those with a low score are functional. The PRIOR probability of variants were obtained from the HCI database (http://priors.hci.utah.edu/PRIORS).

All selected variants of an exon were generated using PCR by amplifying about 400 bp of exon or exons with adjacent intron sequences to make a pool of variant DNA. The pooled DNA was used to introduce the variants into the BAC clone (CTD-2342K5 with a 127 kb insert) containing full-length BRCA2 in SW102 cells by a recombineering method as described previously.7,15 After recombineering, BACs were sequenced to confirm the presence of the correct variant and absence of any undesired mutations. Oligonucleotide sequences are available upon request.

BAC DNA (20 μg) carrying various mutant alleles of BRCA2 was electroporated into 1.0 × 107 PL2F7 mESCs, selected in the presence of G418 (Invitrogen) and characterized as described previously.9 Each BAC carrying a single BRCA2 variant was electroporated and selected individually to ensure that the ES cells harbor a single mutant BAC clone. For selecting the BRCA2 expressing clones, protein isolation and Western blot was carried out as described before.9 Rabbit polyclonal BRCA2 (recognizes an epitope between residues 450–500) antibody (BETHYL lab, Cat # A303-434A-T-1, 1:2000 dilution), and mouse monoclonal Vinculin antibody (Santa Cruz biotech, Cat# sc25336, 1:200,000 dilution) were used to detect proteins. ECL plus Western blotting detection system (Amersham) was used for chemiluminescent detection. To confirm BRCA2 expression for intronic and nonsense variants, RT-PCR was performed using One tube RT-PCR kit (Qiagen) primers from exon 11 (5′-TGGTTTTGTCAAATTCAAGAATTGG-3′) and exon 14 (5′-CCAATCAAGCAGTAGCTGTAACTTTCAC-3′) following manufacturer’s protocol. DNA from BAC containing mESC clones expressing the human BRCA2 was used to further confirm the presence of the correct variant without any undesired mutation by sequencing.

Ten to twenty-five BAC containing PL2F7 ESC clones, each expressing a single BRCA2 variant, were pooled and cultured for two passages. The variants in each pool were from the same exon and were located within the span of ∼300 bp. Two such pools using two independent clones expressing each BRCA2 variant were generated. The conditional allele of Brca2 in mESCs was deleted by electroporating 20 μg of Pgk-Cre plasmid as described previously.9 Two independent electroporations were carried out for each pool and were subjected to the same treatment at subsequent steps. After electroporation, 7.5 × 106 cells were subjected to HAT selection as described previously.9 An equal number of unelectroporated cells were cultured without drug selection. This sample served as the M15 control. Cells were collected for genomic DNA isolation after HAT selection and for the M15 control. One million HAT selected cells were subjected to drug sensitivity. For drug selection, 0.4 μM cisplatin and 0.05 μM PARP inhibitor (olaparib) were used for five days with re-feeding drug in media on day 3 and 5. After drug selection, cells were collected for genomic DNA isolation.

The genomic DNA obtained from collected cell samples was used for PCR-amplification of the respective exons of BRCA2 using Invitrogen Platinum High-fidelity Taq polymerase according to the manufacturer’s protocol. Ten PCR reactions for each sample were carried out and the PCR products pooled together and purified using Qiagen PCR purification kit. For each batch of variants, sixteen samples were utilized [2 pools (biological replicates) × 2 Pgk-Cre electroporation (technical replicates) × 4 treatments (M15, HAT, Cis, Ola)] for deep sequencing of the respective exons using Illumina MiSeq paired end sequencing on the MiSeq sequencer (2 × 300 cycles) allocating ∼3 million reads for each sample to quantify the relative abundance of each variant at different conditions used. Library for Next Gen sequencing was prepared using TruSeq Nano DNA Library Prep Kit from Illumina according to the manufacturer’s protocol. The quantity and quality of the obtained libraries were evaluated using a Qubit 2.0 fluorometer (Thermo Fisher Scientific) and an Agilent 2200 TapeStation system.

The paired-end reads were aligned to the reference sequence using the Needleman-Wunsch alignment algorithm after the reads were demultiplexed using bcl2fastq (Illumina). The fastq files were merged using FLASH and CRISPResso2 was used for quantifying the total number of aligned reads.31 The individual unique alignments were annotated with a custom-made variant caller modified from ANNOVAR (release 2019-10-21) and the R/Bioconductor Biostrings package in R version 4.1.2 (Software: https://bioconductor.org/packages/Biostrings).31,32 Merged reads containing “N” bases and any insertions or deletions were removed from the analysis. The abundances of SNVs were quantified when the reads contained a single-nucleotide substitution and no additional mutations or deletions in the sequencing read. Read counts for each SNVs were then normalized to the total read coverage of the sequencing library after adding a pseudo-count of 1 to all reads for all conditions. Individual variants with a read count of more than 1 in 1,000 reads in the M15 condition were used for further analysis. Dropout or enrichment scores were calculated by taking the ratio of frequency of SNV after HAT, cisplatin, or olaparib treatment over that in M15. The scores were expressed in log2 scale, which we define as the functional scores of SNVs in HAT, cisplatin, and olaparib. The functional scores, averaged between 2 or 4 independent replicates, were used to calculate the probabilities of impact on function (PIF) for all the variants in the dataset.

Abundance of variants was calculated only using sequencing reads that had complete sequence alignment with BRCA2, except for single-nucleotide variation. The frequency of each variant was calculated as the number of variant reads divided by the total number of aligned reads. The variants that had a frequency of at least 1 in 1000 in the M15 sample were subjected to further analysis. Functional scores were calculated for the HAT, cisplatin, and olaparib treatments separately by calculating the log2 ratio of the frequency of treatment over the average frequency of the two biological replicates (two independent Pgk-Cre delivery) for the M15 samples.

### Variant nomenclature and in silico analysis

HGVS nomenclature for cDNA and proteins were followed, in which cDNA numbers with +1 correspond to A of the ATG initiation codon in BRCA2 sequence (GenBank accession number NM_000059.3). Bayes-del scores were obtained from BayesDel database (https://fengbj-laboratory.org/BayesDel). Variants with a high BayesDel score are predicted to be non-functional and those with a low score are functional. The PRIOR probability of variants were obtained from the HCI database (http://priors.hci.utah.edu/PRIORS).

### BRCA2 variant expressing mESC generation

All selected variants of an exon were generated using PCR by amplifying about 400 bp of exon or exons with adjacent intron sequences to make a pool of variant DNA. The pooled DNA was used to introduce the variants into the BAC clone (CTD-2342K5 with a 127 kb insert) containing full-length BRCA2 in SW102 cells by a recombineering method as described previously.7,15 After recombineering, BACs were sequenced to confirm the presence of the correct variant and absence of any undesired mutations. Oligonucleotide sequences are available upon request.

BAC DNA (20 μg) carrying various mutant alleles of BRCA2 was electroporated into 1.0 × 107 PL2F7 mESCs, selected in the presence of G418 (Invitrogen) and characterized as described previously.9 Each BAC carrying a single BRCA2 variant was electroporated and selected individually to ensure that the ES cells harbor a single mutant BAC clone. For selecting the BRCA2 expressing clones, protein isolation and Western blot was carried out as described before.9 Rabbit polyclonal BRCA2 (recognizes an epitope between residues 450–500) antibody (BETHYL lab, Cat # A303-434A-T-1, 1:2000 dilution), and mouse monoclonal Vinculin antibody (Santa Cruz biotech, Cat# sc25336, 1:200,000 dilution) were used to detect proteins. ECL plus Western blotting detection system (Amersham) was used for chemiluminescent detection. To confirm BRCA2 expression for intronic and nonsense variants, RT-PCR was performed using One tube RT-PCR kit (Qiagen) primers from exon 11 (5′-TGGTTTTGTCAAATTCAAGAATTGG-3′) and exon 14 (5′-CCAATCAAGCAGTAGCTGTAACTTTCAC-3′) following manufacturer’s protocol. DNA from BAC containing mESC clones expressing the human BRCA2 was used to further confirm the presence of the correct variant without any undesired mutation by sequencing.

### Multiplexed functional analysis

Ten to twenty-five BAC containing PL2F7 ESC clones, each expressing a single BRCA2 variant, were pooled and cultured for two passages. The variants in each pool were from the same exon and were located within the span of ∼300 bp. Two such pools using two independent clones expressing each BRCA2 variant were generated. The conditional allele of Brca2 in mESCs was deleted by electroporating 20 μg of Pgk-Cre plasmid as described previously.9 Two independent electroporations were carried out for each pool and were subjected to the same treatment at subsequent steps. After electroporation, 7.5 × 106 cells were subjected to HAT selection as described previously.9 An equal number of unelectroporated cells were cultured without drug selection. This sample served as the M15 control. Cells were collected for genomic DNA isolation after HAT selection and for the M15 control. One million HAT selected cells were subjected to drug sensitivity. For drug selection, 0.4 μM cisplatin and 0.05 μM PARP inhibitor (olaparib) were used for five days with re-feeding drug in media on day 3 and 5. After drug selection, cells were collected for genomic DNA isolation.

The genomic DNA obtained from collected cell samples was used for PCR-amplification of the respective exons of BRCA2 using Invitrogen Platinum High-fidelity Taq polymerase according to the manufacturer’s protocol. Ten PCR reactions for each sample were carried out and the PCR products pooled together and purified using Qiagen PCR purification kit. For each batch of variants, sixteen samples were utilized [2 pools (biological replicates) × 2 Pgk-Cre electroporation (technical replicates) × 4 treatments (M15, HAT, Cis, Ola)] for deep sequencing of the respective exons using Illumina MiSeq paired end sequencing on the MiSeq sequencer (2 × 300 cycles) allocating ∼3 million reads for each sample to quantify the relative abundance of each variant at different conditions used. Library for Next Gen sequencing was prepared using TruSeq Nano DNA Library Prep Kit from Illumina according to the manufacturer’s protocol. The quantity and quality of the obtained libraries were evaluated using a Qubit 2.0 fluorometer (Thermo Fisher Scientific) and an Agilent 2200 TapeStation system.

### Sequencing data analysis

The paired-end reads were aligned to the reference sequence using the Needleman-Wunsch alignment algorithm after the reads were demultiplexed using bcl2fastq (Illumina). The fastq files were merged using FLASH and CRISPResso2 was used for quantifying the total number of aligned reads.31 The individual unique alignments were annotated with a custom-made variant caller modified from ANNOVAR (release 2019-10-21) and the R/Bioconductor Biostrings package in R version 4.1.2 (Software: https://bioconductor.org/packages/Biostrings).31,32 Merged reads containing “N” bases and any insertions or deletions were removed from the analysis. The abundances of SNVs were quantified when the reads contained a single-nucleotide substitution and no additional mutations or deletions in the sequencing read. Read counts for each SNVs were then normalized to the total read coverage of the sequencing library after adding a pseudo-count of 1 to all reads for all conditions. Individual variants with a read count of more than 1 in 1,000 reads in the M15 condition were used for further analysis. Dropout or enrichment scores were calculated by taking the ratio of frequency of SNV after HAT, cisplatin, or olaparib treatment over that in M15. The scores were expressed in log2 scale, which we define as the functional scores of SNVs in HAT, cisplatin, and olaparib. The functional scores, averaged between 2 or 4 independent replicates, were used to calculate the probabilities of impact on function (PIF) for all the variants in the dataset.

### Calculation of functional scores

Abundance of variants was calculated only using sequencing reads that had complete sequence alignment with BRCA2, except for single-nucleotide variation. The frequency of each variant was calculated as the number of variant reads divided by the total number of aligned reads. The variants that had a frequency of at least 1 in 1000 in the M15 sample were subjected to further analysis. Functional scores were calculated for the HAT, cisplatin, and olaparib treatments separately by calculating the log2 ratio of the frequency of treatment over the average frequency of the two biological replicates (two independent Pgk-Cre delivery) for the M15 samples.

### Quantification and statistical analysis

The functional scores from each of the three assays (HAT, cisplatin, and olaparib) were regarded as statistical samples (with one measurement for a given assay and a given BRCA2 variant constituting one random data point) and were analyzed using statistical and machine-learning methods to calculate PIF for the variants. For each assay, the distributions of the replicate-averaged and log-transformed functional scores were tested for normality using normal quantile-quantile plots; such plots were generated separately for benign and pathogenic BRCA2 variants. Moreover, each of the six statistical samples (3 assays × 2 ClinVar-classification phenotypes) was tested for normality using the Lilliefors test. The details of the statistical models and algorithms for PIF calculation are described below.

For each of the three assays (HAT, cisplatin, and olaparib), the functional-score distribution characterizing the entire dataset (n=223, the total number of BRCA2 variants analyzed) was modeled as a mixture of two normal distributions. The two mixture components represented pathogenic and benign BRCA2 variants, respectively. The distributions were fit to the functional-score data by numerical likelihood maximization (implemented as negative log likelihood minimization in the code).33

Before the fitting, each mixture model was initialized by assigning equal weights of 0.5 to the mixture components (i.e., the prior probability of pathogenicity was set to 0.5), and the components’ means and variances were set to the sample means and variances estimated from the corresponding data subset [characterizing either benign (n=16) or pathogenic (n=27) BRCA2 variants]. For the fitting, we used the R-language optim() function with typical parameters (the Nelder-Mead optimization algorithm with a relative tolerance of 1.0 × e−8).

The semi-supervised-learning nature of our method was reflected in the structure of the likelihood function, which contained information on BRCA2 variants with a known phenotype (i.e., pathogenic or benign), as well as variants of uncertain significance (VUS). Let f(x,m,v) denote the normal probability density with mean m and variance v. Then, the two-component mixture in our case has the general form,g(x,p,mp,vp,mb,vb)=pf(x,mp,vp)+(1−p)f(x,mb,vb),where p is the prior probability of pathogenicity, mp and vp are the mean and variance of the mixture component for the pathogenic variants, and mb and vb are the mean and variance of the mixture component for the benign variants. Using this notation, the likelihood, l(…), can be written as follows:l(p,mp,vp,mb,vb|x)=∏i=1Nppf(xi(p),mp,vp)×∏i=1Nb(1−p)f(xi(b),mb,vb)×∏i=1Nug(xi(u),p,mp,vp,mb,vb),where x is the vector of the functional scores of the variants in the training set, x=(xi(p),xi(b),xi(u)); xi(p), xi(b), and xi(u) are the functional scores for pathogenic, benign, and VUS variants, respectively; Np, Nb, and Nu are the numbers of pathogenic, benign, and VUS variants, respectively, in the training set.

The parameters of the two-component mixture, including the prior probability of pathogenicity p, were obtained as a result of maximizing the likelihood the way described at the beginning of this subsection. This fitting strategy was applied independently to the data characterizing each of the three assays (HAT, cisplatin, and olaparib). The distributions obtained as the result of the fitting were used to calculate assay-specific probabilities of pathogenicity (denoted PPi) for individual BRCA2 variants using the Bayes formula:PPi=pf(xi,mp,vp)g(xi,p,mp,vp,mb,vb),where xi is the functional score of the variant i, and the distribution-parameter values on the right-hand side (i.e., p,mp,vp,mb,vb) are the ones maximizing the likelihood. Importantly, this formula was also used to calculate PPi in the case when the model trained on one dataset was applied to another dataset, here termed the target set (such as in cross-validation and bootstrapping; see below). In such a case, the variable values (i.e., the functional scores for HAT, cisplatin, and olaparib) were taken from the target set, whereas the probability-density parameters were taken from the distributions fitted to the training set.

For each of the three assays, we denote the PPi by PPi(HAT), PPi(Cis), and PPi(Ola), respectively, where i is an index marking individual BRCA2 variants in the target dataset (i=1,…,223 in the case when the target set is our full dataset; the cases of target subsets are treated in a similar way). The PPi were used to calculate the probabilities of impact on function (PIFi) as follows:(Equation 1)PIFi=PPi(HAT)+(1−PPi(HAT))PPi(Cis)PPi(Ola).

This is a heuristic formula motivated by the total probability formula, and it can be interpreted as follows. If we know that PPi(HAT) is large (i.e., close to 1), then it is likely that the variant i is pathogenic, i.e., PIFi should also be large. Alternatively, if PPi(HAT) is rather small (i.e., close to 0), then the variant i will still likely be pathogenic if both PPi(Cis) and PPi(Ola) are large. The formula in Equation 1 allows us to define PIFi in a way that reflects this logic and also makes sure that PIFi vary between 0 and 1, as properly defined probabilities should.

Equation 1 is the PIF formula used in the main version of our algorithm. Additionally, we considered algorithm versions where the PIFi were calculated using the data from only one of the three assays: HAT, cisplatin, or olaparib. In those cases, the PIFi were set to be equal to PPi(HAT), PPi(Cis), or PPi(Ola), respectively.

We compared our main, semi-supervised-learning version of the algorithm (described above) with a supervised-learning version to assess the relative advantages of semi-supervised learning in the context of PIF estimation. The difference between the semi-supervised-learning and supervised-learning algorithm versions was that, in the latter, the likelihood function incorporated the data only from the BRCA2 variants for which the phenotype was known (i.e., only the pathogenic and benign variants; no VUS). Specifically, the supervised-learning likelihood function had the form:l(p,mp,vp,mb,vb|x)=∏i=1Nppf(xi(p),mp,vp)×∏i=1Nb(1−p)f(xi(b),mb,vb),where the notation is the same as above.

From the calculated PIFi, the phenotype (pathogenic or benign) for each BRCA2 variant, i, was predicted using the thresholds accepted in the BRCA2 research community: PIFi≤ 0.05 indicates a benign BRCA2 variant, PIFi>0.99 indicates a pathogenic variant, and all other cases are considered indeterminate.9 When applied to the full dataset (or to its subsets), the classification accuracy of our algorithm was calculated as the percentage of the BRCA2 variants with a known phenotype (i.e., pathogenic or benign) whose phenotype was predicted correctly from the algorithm-generated PIFi. Predictive accuracy of the algorithm on the full dataset (n=223) was assessed using K-fold cross-validation with K=3,6,9,43. For each variant, the PIF that was used in the K-fold cross-validation accuracy assessment was calculated – and compared to the known benign/pathogenic labels – only for the partition in which this variant was in the cross-validation test set (i.e., the set not used for model fitting in that cross-validation round). For each K, the folds were defined by (randomized) partitioning of the data subset containing only the variants with a known phenotype. Because the number of known-phenotype BRCA2 variants in our full dataset was 43, 43-fold cross-validation was equivalent to leave-one-out cross-validation. The accuracy of K-fold cross-validation was calculated as the classification accuracy on each of the K folds, averaged across all the folds.

While the commonly used values for K include 5 and 10, our choice of K represented a spectrum of possible K values, intended to generate a more complete cross-validation picture.34 We used a generalized cross-validation algorithm, in which the integer K did not have to be a divisor of 43 (indeed, 43 is a prime). In that generalization, we set the fold size, F(K), to [43/K] (here, the brackets denote the integer part of a number) and, for each K, the cross-validation training set consisted of F(K)K randomly selected variants with a known phenotype.

For the semi-supervised-learning algorithm version, the datasets for each of the K−1 folds used for training included all the VUS, which were appended to the pathogenic and benign variants (that were randomly selected for that fold) and used together in the training procedure; in the remaining fold – used for testing – VUS were not included. That way, for each K, the training data were completely separated from the test data. For the supervised-learning algorithm version, the VUS were not used in the cross-validation procedure.

We calculated 95% confidence intervals for the PIFi using bootstrapping. Bootstrapping is a common resampling strategy that allows one to quantify uncertainty in the output of a statistical model. It involves random sampling with replacement from a given dataset, which yields a bootstrapped dataset.34 Ten thousand bootstrapped datasets were generated from the full experimental dataset (n=223). To preserve the structure of the dataset, the pathogenic (n=27), benign (n=16), and VUS (n=180) variants were bootstrapped independently. We then fitted our statistical model (i.e., the combined mixtures of two normal distributions described above) to each of the bootstrapped datasets using the semi-supervised-learning approach, and then applied the fitted model to our full dataset. Thus, for each i (i=1,…,223), we had a sample of 10,000 values of PIFi, from which we calculated the confidence interval for PIFi using the percentile method.35 The confidence intervals for the PIFi from our supervised-learning approach were calculated in a similar way, but VUS were not used in the bootstrap.

The PIF calculation and analysis code was written in R 4.0.2 (2020-06-22), using RStudio 1.3.1073, and was developed and run on a Dell Latitude 7400 laptop computer with an Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz processor and 16 Gb RAM under the 64-bit Windows 10 Enterprise 20H2 operating system.

### Statistical methods and PIF calculation

The functional scores from each of the three assays (HAT, cisplatin, and olaparib) were regarded as statistical samples (with one measurement for a given assay and a given BRCA2 variant constituting one random data point) and were analyzed using statistical and machine-learning methods to calculate PIF for the variants. For each assay, the distributions of the replicate-averaged and log-transformed functional scores were tested for normality using normal quantile-quantile plots; such plots were generated separately for benign and pathogenic BRCA2 variants. Moreover, each of the six statistical samples (3 assays × 2 ClinVar-classification phenotypes) was tested for normality using the Lilliefors test. The details of the statistical models and algorithms for PIF calculation are described below.

### PIF-calculation algorithm: The main (semi-supervised-learning) version

For each of the three assays (HAT, cisplatin, and olaparib), the functional-score distribution characterizing the entire dataset (n=223, the total number of BRCA2 variants analyzed) was modeled as a mixture of two normal distributions. The two mixture components represented pathogenic and benign BRCA2 variants, respectively. The distributions were fit to the functional-score data by numerical likelihood maximization (implemented as negative log likelihood minimization in the code).33

Before the fitting, each mixture model was initialized by assigning equal weights of 0.5 to the mixture components (i.e., the prior probability of pathogenicity was set to 0.5), and the components’ means and variances were set to the sample means and variances estimated from the corresponding data subset [characterizing either benign (n=16) or pathogenic (n=27) BRCA2 variants]. For the fitting, we used the R-language optim() function with typical parameters (the Nelder-Mead optimization algorithm with a relative tolerance of 1.0 × e−8).

The semi-supervised-learning nature of our method was reflected in the structure of the likelihood function, which contained information on BRCA2 variants with a known phenotype (i.e., pathogenic or benign), as well as variants of uncertain significance (VUS). Let f(x,m,v) denote the normal probability density with mean m and variance v. Then, the two-component mixture in our case has the general form,g(x,p,mp,vp,mb,vb)=pf(x,mp,vp)+(1−p)f(x,mb,vb),where p is the prior probability of pathogenicity, mp and vp are the mean and variance of the mixture component for the pathogenic variants, and mb and vb are the mean and variance of the mixture component for the benign variants. Using this notation, the likelihood, l(…), can be written as follows:l(p,mp,vp,mb,vb|x)=∏i=1Nppf(xi(p),mp,vp)×∏i=1Nb(1−p)f(xi(b),mb,vb)×∏i=1Nug(xi(u),p,mp,vp,mb,vb),where x is the vector of the functional scores of the variants in the training set, x=(xi(p),xi(b),xi(u)); xi(p), xi(b), and xi(u) are the functional scores for pathogenic, benign, and VUS variants, respectively; Np, Nb, and Nu are the numbers of pathogenic, benign, and VUS variants, respectively, in the training set.

The parameters of the two-component mixture, including the prior probability of pathogenicity p, were obtained as a result of maximizing the likelihood the way described at the beginning of this subsection. This fitting strategy was applied independently to the data characterizing each of the three assays (HAT, cisplatin, and olaparib). The distributions obtained as the result of the fitting were used to calculate assay-specific probabilities of pathogenicity (denoted PPi) for individual BRCA2 variants using the Bayes formula:PPi=pf(xi,mp,vp)g(xi,p,mp,vp,mb,vb),where xi is the functional score of the variant i, and the distribution-parameter values on the right-hand side (i.e., p,mp,vp,mb,vb) are the ones maximizing the likelihood. Importantly, this formula was also used to calculate PPi in the case when the model trained on one dataset was applied to another dataset, here termed the target set (such as in cross-validation and bootstrapping; see below). In such a case, the variable values (i.e., the functional scores for HAT, cisplatin, and olaparib) were taken from the target set, whereas the probability-density parameters were taken from the distributions fitted to the training set.

For each of the three assays, we denote the PPi by PPi(HAT), PPi(Cis), and PPi(Ola), respectively, where i is an index marking individual BRCA2 variants in the target dataset (i=1,…,223 in the case when the target set is our full dataset; the cases of target subsets are treated in a similar way). The PPi were used to calculate the probabilities of impact on function (PIFi) as follows:(Equation 1)PIFi=PPi(HAT)+(1−PPi(HAT))PPi(Cis)PPi(Ola).

This is a heuristic formula motivated by the total probability formula, and it can be interpreted as follows. If we know that PPi(HAT) is large (i.e., close to 1), then it is likely that the variant i is pathogenic, i.e., PIFi should also be large. Alternatively, if PPi(HAT) is rather small (i.e., close to 0), then the variant i will still likely be pathogenic if both PPi(Cis) and PPi(Ola) are large. The formula in Equation 1 allows us to define PIFi in a way that reflects this logic and also makes sure that PIFi vary between 0 and 1, as properly defined probabilities should.

Equation 1 is the PIF formula used in the main version of our algorithm. Additionally, we considered algorithm versions where the PIFi were calculated using the data from only one of the three assays: HAT, cisplatin, or olaparib. In those cases, the PIFi were set to be equal to PPi(HAT), PPi(Cis), or PPi(Ola), respectively.

### PIF-calculation algorithm: The supervised-learning version

We compared our main, semi-supervised-learning version of the algorithm (described above) with a supervised-learning version to assess the relative advantages of semi-supervised learning in the context of PIF estimation. The difference between the semi-supervised-learning and supervised-learning algorithm versions was that, in the latter, the likelihood function incorporated the data only from the BRCA2 variants for which the phenotype was known (i.e., only the pathogenic and benign variants; no VUS). Specifically, the supervised-learning likelihood function had the form:l(p,mp,vp,mb,vb|x)=∏i=1Nppf(xi(p),mp,vp)×∏i=1Nb(1−p)f(xi(b),mb,vb),where the notation is the same as above.

### PIF-based variant-phenotype prediction and algorithm-accuracy assessment

From the calculated PIFi, the phenotype (pathogenic or benign) for each BRCA2 variant, i, was predicted using the thresholds accepted in the BRCA2 research community: PIFi≤ 0.05 indicates a benign BRCA2 variant, PIFi>0.99 indicates a pathogenic variant, and all other cases are considered indeterminate.9 When applied to the full dataset (or to its subsets), the classification accuracy of our algorithm was calculated as the percentage of the BRCA2 variants with a known phenotype (i.e., pathogenic or benign) whose phenotype was predicted correctly from the algorithm-generated PIFi. Predictive accuracy of the algorithm on the full dataset (n=223) was assessed using K-fold cross-validation with K=3,6,9,43. For each variant, the PIF that was used in the K-fold cross-validation accuracy assessment was calculated – and compared to the known benign/pathogenic labels – only for the partition in which this variant was in the cross-validation test set (i.e., the set not used for model fitting in that cross-validation round). For each K, the folds were defined by (randomized) partitioning of the data subset containing only the variants with a known phenotype. Because the number of known-phenotype BRCA2 variants in our full dataset was 43, 43-fold cross-validation was equivalent to leave-one-out cross-validation. The accuracy of K-fold cross-validation was calculated as the classification accuracy on each of the K folds, averaged across all the folds.

While the commonly used values for K include 5 and 10, our choice of K represented a spectrum of possible K values, intended to generate a more complete cross-validation picture.34 We used a generalized cross-validation algorithm, in which the integer K did not have to be a divisor of 43 (indeed, 43 is a prime). In that generalization, we set the fold size, F(K), to [43/K] (here, the brackets denote the integer part of a number) and, for each K, the cross-validation training set consisted of F(K)K randomly selected variants with a known phenotype.

For the semi-supervised-learning algorithm version, the datasets for each of the K−1 folds used for training included all the VUS, which were appended to the pathogenic and benign variants (that were randomly selected for that fold) and used together in the training procedure; in the remaining fold – used for testing – VUS were not included. That way, for each K, the training data were completely separated from the test data. For the supervised-learning algorithm version, the VUS were not used in the cross-validation procedure.

### PIF confidence intervals via bootstrapping

We calculated 95% confidence intervals for the PIFi using bootstrapping. Bootstrapping is a common resampling strategy that allows one to quantify uncertainty in the output of a statistical model. It involves random sampling with replacement from a given dataset, which yields a bootstrapped dataset.34 Ten thousand bootstrapped datasets were generated from the full experimental dataset (n=223). To preserve the structure of the dataset, the pathogenic (n=27), benign (n=16), and VUS (n=180) variants were bootstrapped independently. We then fitted our statistical model (i.e., the combined mixtures of two normal distributions described above) to each of the bootstrapped datasets using the semi-supervised-learning approach, and then applied the fitted model to our full dataset. Thus, for each i (i=1,…,223), we had a sample of 10,000 values of PIFi, from which we calculated the confidence interval for PIFi using the percentile method.35 The confidence intervals for the PIFi from our supervised-learning approach were calculated in a similar way, but VUS were not used in the bootstrap.

### Software and hardware implementation of the computational procedures

The PIF calculation and analysis code was written in R 4.0.2 (2020-06-22), using RStudio 1.3.1073, and was developed and run on a Dell Latitude 7400 laptop computer with an Intel(R) Core(TM) i7-8665U CPU @ 1.90GHz processor and 16 Gb RAM under the 64-bit Windows 10 Enterprise 20H2 operating system.



# SUPPLEMENTAL FILE 1: main.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)