# MAIN TEXT

## Germline mutations of 4567 patients with hereditary breast-ovarian cancer spectrum in Thailand

### Abstract

Multi-gene panel testing has led to the detection of pathogenic/likely pathogenic (P/LP) variants in many cancer susceptibility genes in patients with breast-ovarian cancer spectrum. However, the clinical and genomic data of Asian populations, including Thai cancer patients, was underrepresented, and the clinical significance of multi-gene panel testing in Thailand remains undetermined. In this study, we collected the clinical and genetic data from 4567 Thai patients with cancer in the hereditary breast-ovarian cancer (HBOC) spectrum who underwent multi-gene panel testing. Six hundred and ten individuals (13.4%) had germline P/LP variants. Detection rates of germline P/LP variants in breast, ovarian, pancreatic, and prostate cancer were 11.8%, 19.8%, 14.0%, and 7.1%, respectively. Non-BRCA gene mutations accounted for 35% of patients with germline P/LP variants. ATM was the most common non-BRCA gene mutation. Four hundred and thirty-two breast cancer patients with germline P/LP variants (80.4%) met the current NCCN genetic testing criteria. The most common indication was early-onset breast cancer. Ten patients harbored double pathogenic variants in this cohort. Our result showed that a significant proportion of non-BRCA P/LP variants were identified in patients with HBOC-related cancers. These findings support the benefit of multi-gene panel testing for inherited cancer susceptibility among Thai HBOC patients. Some modifications of the testing policy may be appropriate for implementation in diverse populations.

### Introduction

Breast cancer is the most common malignancy worldwide and has contributed to a significant impact on global cancer-related deaths1,2. Hereditary cancer syndromes accounted for ~5–10% of all cancer patients3,4. Breast cancer is among the most common cancers with genetic susceptibility, with BRCA1 and BRCA2 regarded as the most identified genes5. Besides BRCA1/2, many pathogenic/likely pathogenic (P/LP) variants in high and moderate penetrance genes for breast-ovarian cancer, including TP53, CDH1, PALB2, STK11, PTEN, CHEK2, ATM, BARD1, BRIP1, and RAD51D, were increasingly identified after the advent of next-generation sequencing (NGS)-based testing5–7. Many studies have shown that non-BRCA cancer susceptibility genes contribute to increased breast and other cancer risk8,9. With expanded access to comprehensive and lower-cost NGS-based multi-gene panels, more patients and families with cancer-predisposing gene mutations could be found, leading to proper screening, early detection, and cancer prevention in at-risk individuals10.

The prevalence of BRCA1 and BRCA2 pathogenic variants in breast and other related cancers in various populations and clinical management guidelines of affected individuals were well established, and similar data of non-BRCA breast cancer susceptibility genes were also increasingly published. Though most prevalence data of non-BRCA pathogenic variants in hereditary breast cancer were from Western countries, several studies in the Asian population have shown the significant detection rate and distribution of pathogenic variants in non-BRCA genes in different countries11–15. A study by Su Y et al.11 in China identified 12.2% of high-risk breast cancer patients who harbored pathogenic variants in non-BRCA genes. Another study in India showed that 15.1% of germline pathogenic variants in breast-ovarian cancer patients were from non-BRCA genes15. These findings have highlighted the importance of expanding genetic tests beyond BRCA1 and BRCA2 for breast and ovarian cancer patients.

Germline BRCA1 and BRCA2 testing for clinically indicated breast and ovarian cancer patients has proven cost-effective worldwide and in Thailand16–18. The test is covered by health insurance and integrated into many national healthcare systems worldwide. In Thailand, the universal reimbursement of germline BRCA1 and BRCA2 genetic testing for breast cancer patients has been approved since 2022. However, recommendations regarding genetic testing other than BRCA1 and BRCA2 continue to vary between countries, and the data and testing policy of non-BRCA genes among Thai patients with the hereditary breast-ovarian cancer (HBOC) spectrum remains insufficient. Therefore, we aim to identify the prevalence of hereditary cancer in Thai patients with HBOC spectrum and the contribution of non-BRCA breast cancer susceptibility genes detected by multi-gene panel testing. We also aim to demonstrate the clinical phenotypes of patients with identified pathogenic/likely pathogenic (P/LP) variants and compare those clinical phenotypes to the current testing criteria from the U.S. National Comprehensive Cancer Network (NCCN) clinical practice guidelines.

### Results

There were 4567 patients with HBOC spectrum tested with multi-gene panel testing due to HBOC-related cancers, consisting of 4041 breast, 394 ovarian (including fallopian tube and primary peritoneal cancer), 100 pancreatic, and 85 prostate cancer patients (Fig. 1). Six hundred and ten patients (13.4%) had at least one P/LP variant. Mutations in BRCA1, BRCA2, and non-BRCA cancer susceptibility genes accounted for 34.9% (n = 213), 31.6% (n = 193), and 35.1% (n = 214), respectively. The detection rate and distribution of P/LP variants in patients with each type of cancer are demonstrated in Table 1.Fig. 1Diagram of the study cohort.Table 1Detection rate of P/LP variants and distribution of variants in BRCA1/2 and non-BRCA genes classified to different types of HBOC-related cancersBreastOvaryPancreasProstatePatients with P/LP variants/total patients tested (% detection rate)491/4041 (11.8%)78/394 (19.8%)14/100 (14.0%)6/85 (7.1%)BRCA genes - BRCA1178 (36.3%)40 (51.3%)3 (21.4%)1 (16.7%) - BRCA2178 (36.3%)17 (21.8%)3 (21.4%)2 (33.3%)Non-BRCA genes associated with each cancer typePALB2TP53CDH1PTENATMCHEK2NF1BARD1RAD51CRAD51D36243237561358PALB2ATMRAD51DMLH1MSH2PMS2263353ATMCDKN2ABARD1RAD51C5112MLH1MSH6RAD50111Total139 (28.3%)22 (28.2%)8 (57.1%)3 (50.0%)Others cancer susceptibility genes identifiedRAD50BRIP1MLH1MSH2MSH6PMS2NBN3XRCC2CDKN2A1683627321ERCC21Total481

Diagram of the study cohort.

Detection rate of P/LP variants and distribution of variants in BRCA1/2 and non-BRCA genes classified to different types of HBOC-related cancers

PALB2

TP53

CDH1

PTEN

ATM

CHEK2

NF1

BARD1

RAD51C

RAD51D

36

24

3

2

37

5

6

13

5

8

PALB2

ATM

RAD51D

MLH1

MSH2

PMS2

2

6

3

3

5

3

ATM

CDKN2A

BARD1

RAD51C

5

1

1

2

MLH1

MSH6

RAD50

1

1

1

RAD50

BRIP1

MLH1

MSH2

MSH6

PMS2

NBN3

XRCC2

CDKN2A

16

8

3

6

2

7

3

2

1

From 214 cancer patients with non-BRCA P/LP variants, 215 P/LP variants were identified (Fig. 2). ATM was the most commonly identified gene with P/LP variants in 44 individuals (20.5%), followed by PALB2 (n = 38) and TP53 (n = 24). Other mutated genes are categorized in Fig. 2. A summary of patients’ and family history according to each BRCA and non-BRCA variant is demonstrated in Table 2.Fig. 2Distribution of identified non-BRCA P/LP variants in 214 HBOC patients.Table 2List of pathogenic/likely pathogenic gene(s), variants, classification, and patient’s historyGene (Ref.)Gene alterationProtein alterationVariant classificationCancer spectrum in PatientsFamily history of cancerBRCA1(NM_007294.3)Median ageat diagnosis46 years,IQR 17 yearsc.1 A > Gp.Met1ValPathogenicBreastBreast, bile ductc.53 T > Cp.Met18ThrLikely pathogenicOvary-c.68_69delAGp.Glu23Valfs*17PathogenicBreast-c.83_84delTGp.Leu28Argfs*PathogenicBreast, ovary-c.142delCp.His476Metfs*2PathogenicBreast-c.212 G > Tp.Arg71MetPathogenicBreastBreastc.241 C > Tp.Gln81*PathogenicBreast-c.440delTp.Leu147Cysfs*16PathogenicBreastBreast, ovaryc.500_503delCAAAp.Thr167Serfs*66PathogenicOvaryOvary, bone (sarcoma)c.772delGp.Asp258Thrfs*29PathogenicBreast-c.1053delAp.Glu352Asnfs*22Likely pathogenicOvary-c.1155 G > Ap.Trp385*PathogenicBreastBreast, ovaryc.1265_1266dupATp.Ser423Ilefs*8PathogenicOvary, breastLung, ovaryc.1426delCp.His476Metfs*2PathogenicBreastBreastc.1504_1508delTTAAAp.Leu502Alafs*2PathogenicBreastBreast, ovaryc.1542_1550delinsCGp.Glu515Valfs*15PathogenicBreast, ovary-c.1550delinsCGp.Glu515Valfs*15PathogenicOvaryOvaryc.1663_1664insTCp.Arg388Serfs*7PathogenicBreast-c.1889delAp.Asn630Ilefs*2PathogenicBreastOvary, breastc.2101_2102delAAp.Lys701Valfs*10PathogenicBreast, ovary, fallopian tubeEndometriumc.2130delinsAAp.Cys712Valfs*6PathogenicBreastBreastc.2269delGp.Val757Phefs*8PathogenicBreast-c.2273 T > Ap.Leu758*PathogenicBreast-c.2411_2412delAGp.Gln804Leufs*5PathogenicBreast-c.2500 G > Tp.Gly834*PathogenicOvary-c.2635 G > Tp.Glu879*PathogenicBreastBreastc.2643dupAp.Cys882Metfs*2PathogenicBreast-c.2766delAp.Val923Leufs*77PathogenicBreast-c.2896dupAp.lle966Asnfs*5Likely pathogenicOvary, fallopian tube, endometriumOvary, prostatec.3020 C > Ap.Ser1007*PathogenicBreast-c.3037 G > Tp.Glu1013*PathogenicBreast, ovary-c.3049 G > Tp.Glu1017*PathogenicProstate, breastProstate, stomach, pancreas, thyroid, breast, ovary, colonc.3139_3140dupGTp.Gly1048*PathogenicBreast, pancreasLungc.3179delAp.Ile1061*PathogenicBreastBreastc.3181delAp.Ile1061*PathogenicBreast, ovaryBladderc.3214delCp.Leu1072*PathogenicBreastPancreas, ovaryc.3403 C > Tp.Gln1135*PathogenicBreast, thyroid, ovaryOvaryc.3424delGp.Ala1142Hisfs*13PathogenicBreast, ovary-c.3598 C > Tp.Gln1200*PathogenicBreast-c.3637_3638delGAp.Glu1213Argfs*3PathogenicBreast-c.3647 T > Ap.Leu1216*PathogenicBreastBreastc.3661 G > Tp.Glu1221*PathogenicBreastPancreas, Ovaryc.3694_3695delinsCp.Gly1232Leufs*3PathogenicBreastProstatec.3748 G > Tp.Glu1250*PathogenicBreast, ovaryColon, breast, ovary, cervixc.3755delTp.Leu1252Argfs*12Likely pathogenicBreast, rectum-c.3756_3759delGTCTp.Ser1253Argfs*PathogenicBreast, ovaryBreast, colonc.3770_3771delAGp.Glu1257Glyfs*9PathogenicOvaryOvaryc.3882_3885delCTTGp.Leu1295Phefs*11PathogenicBreast, ovaryBreastc.4065_4068delTCAAp.Asn1355LysfsPathogenicBreast, endometrium, sarcomac.4327 C > Tp.Arg1443*PathogenicBreastPeritoneumc.4392delTp.Ile1465*PathogenicBreast-c.4506_4507insCp.Ser1503Leufs*4Likely pathogenicOvary-c.4523 G > Ap.Trp1508*PathogenicBreast, ovary-c.4547 G > Ap.Arg1516LysPathogenicBreastBreast, colon, lungc.4689 C > Gp.Tyr1563*PathogenicBreast, ovaryBreast, ovaryc.5030_5033delCTAAp.Thr1677Ilefs*2PathogenicBreast, pancreas-c.5053 A > Gp.Thr1685AlaPathogenicBreast-c.5072 C > Ap.Thr1691LysPathogenicOvaryBreast, ovaryc.5114 T > Cp.Leu1705ProPathogenicBreast-c.5251 C > Tp.Arg1751*PathogenicBreast-c.5362 G > Tp.Gly1788CysPathogenicBreast-c.5406delAp.Gly1803Valfs*31Likely pathogenicBreast-c.5511 G > Tp.Trp1837CysLikely pathogenicBreast, ovary-c.5512delGp.Val1838fsPathogenicBreast-c.5574 G > Tp.Trp1858CysLikely pathogenicBreastBreast, ovaryc.8494 C > Tp.Arg2832CysPathogenicBreast-c.8915delTp.Leu2972Cysfs*4PathogenicBreastBreastc.135-1 G > CSplice site variantLikely pathogenicBreast-c.212+1 G > ASplice site variantPathogenicBreast, ovaryBreast, ovaryc.213-12 A > GSplice site variantLikely pathogenicBreastBreastc.4357+1 G > ASplice site variantPathogenicBreastBreastc.4357+6 T > CSplice site variantPathogenicOvary-c.4485-1 G > ASplice site variantPathogenicOvary, breast-c.4676-2 A > GSplice site variantPathogenicOvaryc.4986+1 G > TSplice site variantPathogenicOvaryBreast, ovaryc.5074+3 A > GSplice site variantLikely pathogenicBreast, ovaryOvaryExon 2–12 deletionCopy number variationBreast-Exon 5–7 duplicationCopy number variationPancreas-Exon 16-18 deletionCopy number variationBreastBreastExon 20-21 deletionCopy number variationBreast-Exon 21-22 deletionCopy number variationBreast, ovary-Exon 23 deletionCopy number variationBreast-BRCA2(NM_000059.3)Median ageat diagnosis46 years,IQR 20 yearsc.18_19delAGp.Arg8Alafs*5PathogenicBreast-c.22_23delAGp.Arg8Alafs*5PathogenicBreast, ovary, pancreasBreastc.151 G > Tp.Glu51*PathogenicBreast-c.157 A > Tp.Lys53*PathogenicBreastBreastc.346delAp.Ser116Valfs*5PathogenicBreastBreast, ovary, prostatec.451_454dupGTAAp.Thr152fsLikely pathogenicBreast (male), breast-c.755_758delACAGp.Asp252Valfs*24Pathogenicc.1270_1286delp.Ser424Argfs*22PathogenicBreast-c.1324_1325insAGp.Ser442TerLikely pathogenicBreast-c.1399_1402delAAGAp.Lys467Glufs*17PathogenicBreastBreastc.1405_1406delGAp.Asp469*PathogenicBreast-c.1763_1766delATAAp.Asn588Serfs*PathogenicBreast-c.1773_1776delTTATp.lle591Metfs*PathogenicBreast, prostate-c.1813delAp.Ile605Tyrfs*9PathogenicBreastc.2327delAp.Lys776Argfs*7PathogenicBreast, ovary-c.2372 C > Gp.Ser791*PathogenicBreast-c.2781_2784delGGTTp.Met927llefs*32Likely pathogenicBreastLungc.2808_2811delACAAp.Ala938Profs*21PathogenicBreastBreast (male), ovaryc.2925delAp.Glu866Lysfs*8PathogenicBreast-c.2990 T > Gp.Leu997*PathogenicBreastBreastc.3109 C > Tp.Gln1037*PathogenicBreastBreast, oral cavityc.3449delCp.Thr1150llefs*Likely pathogenicBreastBreastc.3716_3717delAAp.Lys1239Thrfs*3PathogenicBreast, ovary, colonBreast, prostate, lung, leukemiac.3847_3848delGTp.Val1283Lysfs*2PathogenicBreast, ovaryPeritoneumc.3865_3868delAAATp.Lys1289Alafs*3PathogenicBreastBreast, endometriumc.4115_4116insCp.Met1373Tyrfs*9Likely pathogenicBreastBreastc.4126 G > Tp.Gly1376*PathogenicBreast-c.4245delGp.Glu1415Aspfs*4PathogenicBreast-c.4936_4939delGAAAp.Glu1546Glnfs*23PathogenicBreastBreastc.5087_5090dupGAATp.lle1697Metfs*4PathogenicBreast-c.5116_5119delAATAp.Asn1706Leufs*5PathogenicBreast, pancreasBreast, colonc.5130_5133delTGTAp.Tyr1710*PathogenicBreast-c.5645 C > Ap.Ser1882*PathogenicBreast, thyroid, colonBreast, prostatec.5771_5774delTTCAp.Ile1924Argfs*38BreastBreast, liver, colonc.5980 C > Tp.Gln1994*PathogenicBreast-c.6154delTp.Ser2052Hisfs*18PathogenicBreast-c.6266delAp.Glu2089Glyfs*30Likely pathogenicBreast-c.6298_6299insAp.Asn2101Lysfs*10PathogenicBreastBreast, pancreas, endometriumc.6405_6409delCTTAAp.Asn2135Lysfs*3PathogenicBreast-c.6486_6489delACAAp.Lys2162Asnfs*5PathogenicBreast, ovaryBreastc.6532dupCp.His2178Profs*11PathogenicBreastBreast, prostate, colonc.6541 G > Tp.Gly2181*PathogenicBreast-c.6673delAp.Thr2225Glnfs*4PathogenicBreastBreastc.6777_6778delTGp.Asn2259Lysfs*33PathogenicBreast, ovaryColon, endometriumc.6896delAp.Asn2299Ilefs*6PathogenicBreast-c.6952 C > Tp.Arg2318*PathogenicBreast, ovary-c.6997_6998delGTp.Val2333Thrfs*6PathogenicBreastOvary, lung, neuroendocrinec.7185_7188delCTTGp.His2395Glnfs*71PathogenicBreastBreastc.7185_7190delinsAGp.His2395Glnfs*71PathogenicBreastBreast, brainc.7288 G > Tp.Glu2430*PathogenicBreastBreast, leukemia, peritoneumc.7544_7545insAp.Ser2516Ilefs*23PathogenicBreast-c.7558 C > Tp.Arg2520*PathogenicPancreas, breast, lung-c.7643_7644delATp.His2548Leufs*5PathogenicBreast-c.7673_7674delAGp.Glu2558Valfs*7PathogenicBreastBladder, colonc.7767delCp.Ser2590Profs*58PathogenicBreast, endometriumBreast, thyroidc.7999delAp.Ser2667Alafs*6PathogenicBreastBreast, liverc.8023delAp.Ile2675*PathogenicBreastBreastc.8168 A > Cp.Asp2723AlaPathogenicBreast-c.8191 C > Tp.Gln2731*PathogenicBreast-c.8837_8841delTGGAAp.Leu2946Tyrfs*2PathogenicProstate, breast (male), esophagus-c.8854_8855insTp.Met2952Ilefs*5PathogenicBreastBreast, ovary, colonc.8890dupAp.Arg2964Lysfs*54PathogenicBreast, ovary-c.8915delTp.Leu2972Cysfs*4PathogenicBreastBreastc.9097delAp.Thr3033Leufs*29PathogenicBreastBreastc.9154 C > Tp.Arg3052TrpPathogenicBreastBreast, ovaryc.9382 C > Tp.Arg3128*PathogenicBreastBreast, colonc.7007+1 G > CSplice site variantPathogenicBreast-c.7008-1 G > ASplice site variantPathogenicBreast-c.7618-1 G > ASplice site variantPathogenicBreast, colon-c.8331+2 T > ASplice site variantPathogenicOvary, fallopian tube-c.8953+1 G > CSplice site variantLikely pathogenicBreast, ovaryBreastExon 5–7 deletionCopy number variationBreast-Exon 19-20 deletionCopy number variationBreastLung, BreastExon 25-27 deletionCopy number variationBreastBreastExon 26-27 deletionCopy number variationBreastBreastATM(NM_000051.3)Median ageat diagnosis47 years,IQR 14 yearsc.1 A > Gp.Met1ValPathogenicBreast-c.769 G > Tp.Glu257*PathogenicBreast-c.875 C > Tp.Pro292LeuLikely pathogenicBreast, ovary-c.1339 C > Tp.Arg447*PathogenicBreast, pancreas, endometriumPancreasc.1402_1403delAAp.Lys468Glufs*18PathogenicLung, ovary, pancreasColonc.1655delCp.Pro552Glnfs*4PathogenicBreast-c.2086 G > Tp.Gly696*Likely pathogenicSarcoma, breastBreast, ovary, lungc.2341 C > Tp.Gln781*PathogenicBreast-c.2413 C > Tp.Arg805*PathogenicBreast-c.3693_3697delATCTTp.Leu1231Phefs*13Likely pathogenicOvaryProstatec.3712_3716delTTATTp.Leu1238fsLikely pathogenicBreastBreastc.4335dupTp.Val1446Cysfs*2PathogenicPancreas-c.4664delTp.Leu1555Pro*PathogenicBreast-c.4852 C > Tp.Arg1618*PathogenicBreastBreastc.5040dupTp.Ile1681Tyrfs*11PathogenicPancreas, colon, breastPancreasc.5692 C > Tp.Arg1898*PathogenicBreastBreastc.7542 T > Gp.Tyr2514*Likely pathogenicBreastStomach, colonc.7843 C > Tp.Gln2615*Likely pathogenicBreastColonc.7886_7890delTATTAp.Ile2629Serfs*25PathogenicBreast-c.8287 C > Tp.Arg2763*PathogenicBreast-c.8395_8404delTTTCAGTGCCp.Phe2799Lysfs*4PathogenicBreast-c.8435_8436delCTp.Ser2812Phefs*2Likely pathogenicPancreas, colon-c.8494 C > Tp.Arg2832CysPathogenicBreast-c.8824_8834delCAGGAAACTCTp.Gln2942Valfs*10PathogenicBreast, lungLungc.8879 G > Ap.Trp2960*PathogenicBreast, colonCNS (brain)c.2377-2 A > GSplice site variantPathogenicBreast-c.6453-2 A > GSplice site variantLikely pathogenicBreast, colon-c.3994-1 G > TSplice site variantPathogenicBreastBreastExon 62_63 deletionCopy number variationBreast (male), colon-TP53(NM_000546.5)Median ageat diagnosis33 years,IQR 15.5 yearsc.325 T > Gp.Phe109ValLikely pathogenicBreast-c.371dupGp.Cys124Trpfs*25PathogenicBreastLung, HCCc.374 C > Tp.Thr125MetPathogenicBreast, thyroid, sarcoma, urothelial, pancreasLung, brain, colon, kidney, thyroid (papillary)c.375 G > Ap.Thr125=PathogenicBreast-c.422 G > Tp.Cys141PhePathogenicBreastBreastc.524 G > Ap.Arg175HisPathogenicBreast-c.528 C > Ap.Cys176*Pathogenic-Breast, lung, CNS (brain)c.538 G > Ap.Glu180LysLikely pathogenicBreastLungc.586 C > Tp.Arg196*PathogenicBreastEsophagus, cervixc.637 C > Tp.Arg213*PathogenicBreastBreastc.659 A > Gp.Tyr220CysPathogenicBreast, endometrium-c.733 G > Ap.Gly245SerPathogenicBreast, adrenocorticalBreast, head/neck cancerc.742 C > Tp.Arg248TrpPathogenicBreast, lymphoma, colonBreast, leukemia, CNS (brain), esophagus, bone, lungc.799 C > Tp.Arg267TrpPathogenicBreast-c.818 G > Ap.Arg273HisPathogenicBreast, leukemiaPancreas, kidneyc.832 C > Tp.Pro278SerLikely pathogenicBreast-c.839 G > Tp.Arg280IleLikely pathogenicBreast-c.1024 C > Tp.Arg342*PathogenicBreast, colon-c. 96 + 1 G > ASplice site variantPathogenicBreast, CNS (brain)-c.559+1 G > ASplice site variantPathogenicBreast-Exon 10_11 deletionCopy number variationBreastBreastPALB2(NM_024675.3)Median ageat diagnosis49.5 years,IQR 12.8 yearsc.7 G > Tp.Glu3*PathogenicBreast-c.276_279dupTGGAp.Glu94Trpfs*9Likely pathogenicBreastBreastc.626delCp.Ser209Phefs*14Likely pathogenicBreast-c.758dupTp.Ser254Ilefs*3PathogenicBreastLeukemiac.778 C > Tp.Gln260*PathogenicBreast-c.1059delAp.Lys353Asnfs*3PathogenicBreast-c.1660G>Tp.Glu554*Likely pathogenicBreast-c.2255_2267dupGACGAACTTGCTGp.Cys756Trpfs*21PathogenicBreastBreastc.2257 C > Tp.Arg753TerPathogenicBreastProstatec.2411_2412delCTp.Ser804Cysfs*10PathogenicBreast-c.2503delTp.Ser835Profs*16Likely pathogenicBreast-c.2520_2521delAAp.Glu840Aspfs*8PathogenicBreast-c.2546delGp.Arg753*PathogenicBreast-c.2704_2707dupGATGp.Ala903Glyfs*26Likely pathogenicBreast-c.2968 G > Tp.Glu990*PathogenicBreast-c.3267_3268delGTp.Phe1090Serfs*6Likely pathogenicOvary-c.3426_3429delACTTp.Leu1142Phefs*20PathogenicBreast-c.3625 T > Gp.Leu1209ValPathogenicBreast-c.109-2 A > CSplice site variantPathogenicBreast-c.1684+1 G > ASplice site variantPathogenicBreast, lung-c.2515-2 A > GSplice site variantLikely pathogenicBreast-c.3350+5 G > ASplice site variantLikely pathogenicBreast-Exon 2_4 deletionCopy number variationBreast-Exon 8 deletionCopy number variationBreastBreastExon 1_10 deletionCopy number variationBreast-RAD50(NM_005732.4)Median ageat diagnosis47 years,IQR 12.5 yearsc.1111delAp.Ile371Phefs*8PathogenicBreastBreastc.1751C>Gp.Ser584*Likely pathogenicBreast, stomach-c.2165delAp.Lys722Argfs*14PathogenicBreastCervixc.2263 C > Tp.Gln755*Likely pathogenicBreast-c.2983_2986delGAAAp.Glu995Argfs*2PathogenicBreastBreastc.3528delTp.Asp1177Ilefs*11Likely pathogenicBreastLungc.3553 C > Tp.Arg1185*PathogenicBreast-c.3598 C > Tp.Arg1200*PathogenicBreastBreast, liverc.3715 C > Tp.Arg1239*Likely pathogenicBreast-c.7768 C > Tp.Lys50Asnfs*29Likely pathogenicBreast-Exon 2 deletionCopy number variationBreastColonRAD51C(NM_058216.3)Median ageat diagnosis54.5 years,IQR 15.5 yearsc.394dupAp.Thr132Asnfs*23PathogenicBreast, endometriumBreast, lung, esophagusc.145+1 G > TSplice site variantPathogenicBreast-c.405-1 G > ASplice site variantPathogenicPancreasBreastc.905-2 A > CSplice site variantPathogenicBreast, pancreasBreastRAD51D(NM_002878.4)Median ageat diagnosis42.5 years,IQR 12 yearsc.27 C > Ap.Cys9*PathogenicBreast-c.270_271dupTAp.Lys91Ilefs*13PathogenicBreast, ovaryBreast (male, female), lung, Liver, colonc.385 C > Tp.Gln129*PathogenicOvaryBreastc.694 C > Tp.Arg232*PathogenicBreast-Exon 1_3 deletionCopy number variationBreast-Exon 5_10 deletionCopy number variationBreast-NF1(NM_001042492.3)Median ageat diagnosis45 years,IQR 23 yearsc.1658A>Gp.His553ArgPathogenicBreast-c.2735_2750delp.Gln912Leufs*7Likely pathogenicBreast-c.4375 G > Cp.Glu1459GlnPathogenicBreast-c.4600 C > Tp.Arg1534TerPathogenicBreast-c.1062+1 G > TSplice site variantPathogenicBreast-c.1846-1 G > TSplice site variantLikely pathogenicBreast-PTEN(NM_000314.8)Median ageat diagnosis43.5 yearsIQR 0.75 yearsc.697 C > Tp.Arg233*PathogenicBreast-c.842_843insCTGp.Pro281_Gly282insTerLikely pathogenicBreastBreast, liverCDH1(NM_004360.5)Median ageat diagnosis57 yearsIQR 12.5 yearsc.1137 G > Ap.Thr379=Likely pathogenicBreast, colonBreast, lungc.1792C>Tp.Arg598*PathogenicBreast-BARD1(NM_000465.4)Median ageat diagnosis45.5 yearsIQR 10.5 yearsc.69_70delins(25)p.Ala25Glyfs*41PathogenicBreastLiver, thyroid, leukemiac.76delAp.Met26Trpfs*32Likely pathogenicBreast-c.334 C > Tp.Arg112*PathogenicBreast-c.593delCp.Ala198Valfs*14Likely pathogenicBreast-c.808 G > Tp.Glu270*PathogenicBreast, colorectumEndometriumc.1348_1349delinsCATp.Asn450Hisfs*4PathogenicBreast, pancreas-c.1811-1 G > ASplice site variantLikely pathogenicBreastBreast, lungBRIP1(NM_032043.3)Median ageat diagnosis40.8 yearsIQR 20.5 yearsc.644_645delCTp.Ser215*PathogenicBreast-c.1066 C > Tp.Arg356*PathogenicBreastBreastc.1315 C > Tp.Arg439*PathogenicBreastLiver, brainc.1343 G > Ap.Trp448*PathogenicBreast-c.2431_2432dupCTp.Pro812Tyrfs*15Likely pathogenicBreast-c.3072delGp.Ser1025HisLikely pathogenicBreast-c.2097+1dupGSplice site variantLikely pathogenicBreast-CHEK2(NM_007194.4)Median ageat diagnosis43 yearsIQR 3.3 yearsCDKN2A(NM_000077.5)Median ageat diagnosis68.5 yearsIQR 3.5 yearsc.1238 T > Gp.Leu413*PathogenicBreast-c.1008+2 T > ASplice site variantLikely pathogenicBreast-c.1096-2 A > TSplice site variantLikely pathogenicBreast-Exon 9 deletionCopy number variationBreastBreastc.44 G > Ap.Trp15*PathogenicBreastBreastc.367delCp.His123Ilefs*23Likely pathogenicPancreas-MLH1(NM_000249.4)Median ageat diagnosis49 yearsIQR 16.5 yearsMSH2(NM_000251.3)Median ageat diagnosis50 yearsIQR 6.8 yearsc.2011 G > Tp.Glu671*PathogenicOvaryColonc.790+1G > ASplice site variantPathogenicOvary, endometrium, colon, prostate, bladderBreast, endometrium,c.884+4 A > GSplice site variantPathogenicBreast-Exon 7-8 deletionCopy number variationBreast, ovaryColon, cervixExon 15 deletionCopy number variationBreast-c.811_814delTCTGp.Ser271Argfs*2PathogenicBreast-c.1237 C > Tp.Gln413*PathogenicOvaryBreast, endometriumc.1786_1788delAATp.Asn596delPathogenicBreast, colorectalBreast, colorectal, cervixc.1930delGp.Val644Phefs*41PathogenicBreast, endometriumOvary, bladderc.2018delGp.Gly673Glufs*12Likely pathogenicOvary, colon-c.2432 T > Gp.Leu811*PathogenicBreast-c.2554 G > Tp.Glu852*Likely pathogenicOvary-c.942+3 A > TSplice site variantLikely pathogenicBreast, ovary, endometriumBreast, colorectalc.1661+1 G > ASplice site variantLikely pathogenicOvaryColonExon 7-8 deletionCopy number variationBreast, colonEndometrium, colonMSH6(NM_000179.3)Median ageat diagnosis63 yearsIQR 13 yearsc.3261dupCp.Phe1088fsPathogenicBreast-c.1145delAp.His382Profs*29PathogenicProstate, colonEndometrium, breast, colonc.628-1 G > ASplice site variantLikely pathogenicBreast (male)-PMS2(NM_000535.7)Median ageat diagnosis48 yearsIQR 15.3 yearsc.1 A > Gp.Met1ValPathogenicOvary-c.325dupGp.Glu109Glyfs*30PathogenicOvaryColonc.451delCp.Arg151Alafs*50PathogenicBreast-c.943 C > Tp.Arg315*PathogenicOvary-c.1864_1865delp.Met622Glufs*5PathogenicBreastLung, pancreaticc.2243-2246delAGAAp.Lys748Metfs*19PathogenicBreast-c.2444 C > Tp.Ser815LeuLikely pathogenicBreast-c.706-1 G > TSplice site variantPathogenicBreastBreastc.2174+1 G > ASplice site variantPathogenicBreastBreastNBN(NM_002485.5)Median ageat diagnosis54 yearsIQR 11 yearsc.5 G > Ap.Trp2*PathogenicBreast-c.89delAp.Asn30Thrfs*5Likely pathogenicBreast-c.1516 C > Tp.Gln506*PathogenicBreast-ERCC2(NM_000400.4)c.2164 C > Tp.Arg722TrpPathogenicOvaryOvaryXRCC2(NM_005431.2)c.1 A > Tp.Met1LeuLikely pathogenicBreast-c.323_326delTCAAp.108Asnfs*25Likely pathogenicBreast-

Distribution of identified non-BRCA P/LP variants in 214 HBOC patients.

List of pathogenic/likely pathogenic gene(s), variants, classification, and patient’s history

BRCA1

(NM_007294.3)

Median age

at diagnosis

46 years,

IQR 17 years

BRCA2

(NM_000059.3)

Median age

at diagnosis

46 years,

IQR 20 years

ATM

(NM_000051.3)

Median age

at diagnosis

47 years,

IQR 14 years

TP53

(NM_000546.5)

Median age

at diagnosis

33 years,

IQR 15.5 years

PALB2

(NM_024675.3)

Median age

at diagnosis

49.5 years,

IQR 12.8 years

c.2255_2267dupGACGA

ACTTGCTG

RAD50

(NM_005732.4)

Median age

at diagnosis

47 years,

IQR 12.5 years

RAD51C

(NM_058216.3)

Median age

at diagnosis

54.5 years,

IQR 15.5 years

RAD51D

(NM_002878.4)

Median age

at diagnosis

42.5 years,

IQR 12 years

NF1

(NM_001042492.3)

Median age

at diagnosis

45 years,

IQR 23 years

PTEN

(NM_000314.8)

Median age

at diagnosis

43.5 years

IQR 0.75 years

CDH1

(NM_004360.5)

Median age

at diagnosis

57 years

IQR 12.5 years

BARD1

(NM_000465.4)

Median age

at diagnosis

45.5 years

IQR 10.5 years

BRIP1

(NM_032043.3)

Median age

at diagnosis

40.8 years

IQR 20.5 years

CHEK2

(NM_007194.4)

Median age

at diagnosis

43 years

IQR 3.3 years

CDKN2A

(NM_000077.5)

Median age

at diagnosis

68.5 years

IQR 3.5 years

MLH1

(NM_000249.4)

Median age

at diagnosis

49 years

IQR 16.5 years

MSH2

(NM_000251.3)

Median age

at diagnosis

50 years

IQR 6.8 years

MSH6

(NM_000179.3)

Median age

at diagnosis

63 years

IQR 13 years

PMS2

(NM_000535.7)

Median age

at diagnosis

48 years

IQR 15.3 years

NBN

(NM_002485.5)

Median age

at diagnosis

54 years

IQR 11 years

ERCC2

(NM_000400.4)

XRCC2

(NM_005431.2)

Of 537 breast cancer patients with P/LP variants, P/LP variants in moderate-to-high penetrance breast cancer susceptibility genes were identified in 491 individuals. Meanwhile, there were 78 ovarian, 14 pancreatic, and six prostate cancer patients with P/LP variants. The detection rate of P/LP variants was 19.8% in ovarian cancer patients, followed by pancreatic (14%), breast (11.8%), and prostate cancer (7.1%).

Fifty-four patients had multiple primary cancers, consisting of 27 patients with HBOC-related cancers. The other 27 individuals had additional cancers not in the HBOC spectrum, including colorectal cancer (n = 13), endometrial cancer (n = 11), sarcomas (n = 4) (see Supplementary Table 1). In addition, 162 patients (26.6%) had a history of cancer in their families.

Of 491 breast cancer patients with P/LP variants in breast cancer susceptibility genes, 72.3% (n = 355) were BRCA1 or BRCA2 variants, while 139 patients had non-BRCA variants. ATM was the most commonly identified non-BRCA gene (n = 37, 26.6%), followed by PALB2 (n = 36, 25.9%). Details of P/LP variants in other non-BRCA genes are shown in Table 1.

A total of 432 breast cancer patients with P/LP variants (80.4%) fulfilled the 2023 NCCN testing criteria for high-penetrance breast cancer susceptibility genes (Table 3). The most common indication was early-onset breast cancer (n = 365), followed by family history of HBOC-related cancers (n = 102), multiple primary breast cancer (n = 57), triple-negative breast cancer (n = 50), primary breast and ovarian cancers (n = 22), and male breast cancer (n = 10). The proportion of patients with P/LP variants in BRCA and non-BRCA genes who met each criterion is shown in Table 3.Table 3Breast cancer patients with P/LP variants categorized by NCCN eligibility criteria for germline high-penetrance breast cancer susceptibility genes testing, and indications for genetic testing in breast cancer patients (number of all patients is less than a combination of each group due to the presence of double mutations)All patients with P/LP variants (n = 537)Patients with P/LP BRCA1/2 variants (n = 355)Patients with P/LP non-BRCA variants (n = 187)Compatible with 2023 NCCN genetic testing criteria432 (80.4%)296 (83.4%)140 (74.9%)Indications for genetic testing in breast cancer patients• Early onset breast cancer (age ≤50 years)365/432 (84.5%)241/296 (81.4%)127/140 (90.7%)• Strong family history of HBOC-related cancer102/432 (23.6%)84/296 (28.4%)19/140 (13.6%)• Multiple primary (synchronous or metachronous) breast cancer50/432 (11.6%)42/296 (14.2%)10/140 (7.1%)• Triple-negative breast cancer57/432 (13.2%)40/296 (13.5%)18/140 (12.9%)• Male breast cancer10/432 (2.3%)9/296 (3.0%)2/140 (1.4%)• Breast and ovarian cancer22/432 (5.1%)17/296 (5.7%)5/140 (3.6%)

Breast cancer patients with P/LP variants categorized by NCCN eligibility criteria for germline high-penetrance breast cancer susceptibility genes testing, and indications for genetic testing in breast cancer patients (number of all patients is less than a combination of each group due to the presence of double mutations)

In this cohort, 908 breast cancer patients who received genetic testing did not meet current NCCN criteria. In this group, 105 patients (11.6%) harbored P/LP variants in BRCA1/2 (n = 59) and non-BRCA genes (n = 47). Thirty-two patients with P/LP variants were found in moderate-to-high penetrance non-BRCA genes. Clinical data on patients not meeting the criteria are available in Supplementary Table 2.

Of 78 ovarian cancer patients with germline P/LP variants, 40 had BRCA1, 17 had BRCA2, and 22 had non-BRCA variants (Table 1). P/LP variants in mismatch repair (MMR) genes (MLH1, MSH2, and PMS2) accounted for 14.4% (n = 11) (see Supplementary Table 3).

There were 14 pancreatic cancer patients with germline P/LP variants, including 6 with BRCA1/2 variants and 8 with non-BRCA variants. Six out of 84 prostate cancer patients also had P/LP variants. Detailed results are shown in Table 1.

This cohort identified ten individuals with double pathogenic variants (Table 4). Nine patients had concomitant non-BRCA and BRCA1 or BRCA2 P/LP variants. One patient found both pathogenic RAD51C and CDKN2A variants.Table 4Identified double P/LP variants in HBOC-related cancer patientsVariants 1ClassificationVariants 2ClassificationCancer in patientsBRCA1: c.3847_3848 delGTPathogenicBRCA2: c.772delGPathogenicBreastBRCA1: c.213-12 A > GPathogenicRAD50: c.2980_2983 delAAAGPathogenicBreastBRCA1: c.83_84delTGPathogenicRAD51D: c.385 C > TPathogenicOvaryBRCA1: c.4986+1 G > TPathogenicERCC2: c.2164 C > TPathogenicOvaryBRCA2: CNV exon 25-27 deletionLikely pathogenicATM: c.5692 C > TPathogenicBreastBRCA2: c.1405_1406 delGAPathogenicATM: c.7886_7890 delTATTAPathogenicBreastBRCA2: c.631+3 A > GLikely pathogenicATM: c.2086 G > TPathogenicBreast (bilateral)BRCA2: c.7767delCPathogenicRAD50: c.3715 C > TPathogenicBreastBRCA2: c.451_454dupGTAALikely pathogenicMSH6: c.628-1 G > APathogenicMale breastRAD51C: c.405-1 G > APathogenicCDKN2A: c.367delCPathogenicPancreas

Identified double P/LP variants in HBOC-related cancer patients

### Distribution of cancers in patients with P/LP variants

Of 537 breast cancer patients with P/LP variants, P/LP variants in moderate-to-high penetrance breast cancer susceptibility genes were identified in 491 individuals. Meanwhile, there were 78 ovarian, 14 pancreatic, and six prostate cancer patients with P/LP variants. The detection rate of P/LP variants was 19.8% in ovarian cancer patients, followed by pancreatic (14%), breast (11.8%), and prostate cancer (7.1%).

Fifty-four patients had multiple primary cancers, consisting of 27 patients with HBOC-related cancers. The other 27 individuals had additional cancers not in the HBOC spectrum, including colorectal cancer (n = 13), endometrial cancer (n = 11), sarcomas (n = 4) (see Supplementary Table 1). In addition, 162 patients (26.6%) had a history of cancer in their families.

### Multi-gene panel testing in breast cancer and NCCN indication fulfillment

Of 491 breast cancer patients with P/LP variants in breast cancer susceptibility genes, 72.3% (n = 355) were BRCA1 or BRCA2 variants, while 139 patients had non-BRCA variants. ATM was the most commonly identified non-BRCA gene (n = 37, 26.6%), followed by PALB2 (n = 36, 25.9%). Details of P/LP variants in other non-BRCA genes are shown in Table 1.

A total of 432 breast cancer patients with P/LP variants (80.4%) fulfilled the 2023 NCCN testing criteria for high-penetrance breast cancer susceptibility genes (Table 3). The most common indication was early-onset breast cancer (n = 365), followed by family history of HBOC-related cancers (n = 102), multiple primary breast cancer (n = 57), triple-negative breast cancer (n = 50), primary breast and ovarian cancers (n = 22), and male breast cancer (n = 10). The proportion of patients with P/LP variants in BRCA and non-BRCA genes who met each criterion is shown in Table 3.Table 3Breast cancer patients with P/LP variants categorized by NCCN eligibility criteria for germline high-penetrance breast cancer susceptibility genes testing, and indications for genetic testing in breast cancer patients (number of all patients is less than a combination of each group due to the presence of double mutations)All patients with P/LP variants (n = 537)Patients with P/LP BRCA1/2 variants (n = 355)Patients with P/LP non-BRCA variants (n = 187)Compatible with 2023 NCCN genetic testing criteria432 (80.4%)296 (83.4%)140 (74.9%)Indications for genetic testing in breast cancer patients• Early onset breast cancer (age ≤50 years)365/432 (84.5%)241/296 (81.4%)127/140 (90.7%)• Strong family history of HBOC-related cancer102/432 (23.6%)84/296 (28.4%)19/140 (13.6%)• Multiple primary (synchronous or metachronous) breast cancer50/432 (11.6%)42/296 (14.2%)10/140 (7.1%)• Triple-negative breast cancer57/432 (13.2%)40/296 (13.5%)18/140 (12.9%)• Male breast cancer10/432 (2.3%)9/296 (3.0%)2/140 (1.4%)• Breast and ovarian cancer22/432 (5.1%)17/296 (5.7%)5/140 (3.6%)

Breast cancer patients with P/LP variants categorized by NCCN eligibility criteria for germline high-penetrance breast cancer susceptibility genes testing, and indications for genetic testing in breast cancer patients (number of all patients is less than a combination of each group due to the presence of double mutations)

In this cohort, 908 breast cancer patients who received genetic testing did not meet current NCCN criteria. In this group, 105 patients (11.6%) harbored P/LP variants in BRCA1/2 (n = 59) and non-BRCA genes (n = 47). Thirty-two patients with P/LP variants were found in moderate-to-high penetrance non-BRCA genes. Clinical data on patients not meeting the criteria are available in Supplementary Table 2.

### Multi-gene panel testing in other HBOC-related cancers

Of 78 ovarian cancer patients with germline P/LP variants, 40 had BRCA1, 17 had BRCA2, and 22 had non-BRCA variants (Table 1). P/LP variants in mismatch repair (MMR) genes (MLH1, MSH2, and PMS2) accounted for 14.4% (n = 11) (see Supplementary Table 3).

There were 14 pancreatic cancer patients with germline P/LP variants, including 6 with BRCA1/2 variants and 8 with non-BRCA variants. Six out of 84 prostate cancer patients also had P/LP variants. Detailed results are shown in Table 1.

### Detection of double pathogenic variants in HBOC patients

This cohort identified ten individuals with double pathogenic variants (Table 4). Nine patients had concomitant non-BRCA and BRCA1 or BRCA2 P/LP variants. One patient found both pathogenic RAD51C and CDKN2A variants.Table 4Identified double P/LP variants in HBOC-related cancer patientsVariants 1ClassificationVariants 2ClassificationCancer in patientsBRCA1: c.3847_3848 delGTPathogenicBRCA2: c.772delGPathogenicBreastBRCA1: c.213-12 A > GPathogenicRAD50: c.2980_2983 delAAAGPathogenicBreastBRCA1: c.83_84delTGPathogenicRAD51D: c.385 C > TPathogenicOvaryBRCA1: c.4986+1 G > TPathogenicERCC2: c.2164 C > TPathogenicOvaryBRCA2: CNV exon 25-27 deletionLikely pathogenicATM: c.5692 C > TPathogenicBreastBRCA2: c.1405_1406 delGAPathogenicATM: c.7886_7890 delTATTAPathogenicBreastBRCA2: c.631+3 A > GLikely pathogenicATM: c.2086 G > TPathogenicBreast (bilateral)BRCA2: c.7767delCPathogenicRAD50: c.3715 C > TPathogenicBreastBRCA2: c.451_454dupGTAALikely pathogenicMSH6: c.628-1 G > APathogenicMale breastRAD51C: c.405-1 G > APathogenicCDKN2A: c.367delCPathogenicPancreas

Identified double P/LP variants in HBOC-related cancer patients

### Discussion

This study provides insights into germline mutations and cancer phenotypes in Thai patients with HBOC spectrum and demonstrates the favorable diagnostic yield of multi-gene panel testing in Thai cancer patients. In this study, 13.4% of Thai patients with HBOC-related cancers were associated with cancer susceptibility genes. Among P/LP variants, non-BRCA genes accounted for 35% of these cases. The proportion of non-BRCA variants in breast cancer patients with germline mutation in this study is consistent with previous studies, ranging from 6.8% to 40% in high-risk breast cancer patients11–15,19,20. P/LP variants in non-BRCA genes are identified in 28.3% of breast and 28.2% of ovarian cancer patients with germline mutations. This suggests that the incorporation of additional cancer susceptibility genes in the test for Thai patients with HBOC spectrum may enhance the diagnostic yield by as much as 28% in comparison to BRCA-only testing. Our result aligned with a prior study that supports multi-gene panel testing among breast cancer patients21. Our findings may guide physicians to consider multi-gene panel testing for patients with HBOC-related cancers.

While PALB2 and TP53 were the most reported non-BRCA genes in breast cancer patients in various studies11–14, ATM was the most common non-BRCA gene mutations in our cohort. From previous studies, the prevalence of ATM P/LP variants among non-BRCA mutated breast cancer patients varied widely11,13,14. A meta-analysis also suggested a pooled prevalence of 7% in P/LP ATM variants among high-risk breast cancer cohorts22. This data may indicate a higher frequency of ATM carriers in the Thai population. ATM mutations are associated with breast and other cancer susceptibilities23,24 and may confer a risk of contralateral breast cancer in patients undergoing radiotherapy25. Despite its relevance, this gene is not included in the NCCN guidelines. Our results support the inclusion of ATM into the breast cancer gene panel for the Thai population.

Our study reveals that only 80% of breast cancer patients with germline mutations met the NCCN criteria for genetic testing. The observation that one-fifth of Thai breast cancer patients with pathogenic variants would miss out on testing opportunities aligned with a previous study in which NCCN criteria missed around 30% of patients with pathogenic variants26. It has been suggested that lowering the age threshold for universal genetic testing could improve the detection rate in breast cancer patients27, which was supported by the 2019 American Society of Breast Surgeons28. Owing to the high acceptance of genetic testing and counseling, more detection of cases would benefit treatment, screening, and prevention for patients and family members carrying pathogenic variants29,30.

The diagnostic yield of germline testing in Thai ovarian cancer patients was 20%, which is comparable with previous studies31–33. Mismatch repair (MMR) genes accounted for 14% of P/LP variants in ovarian cancer patients, consistent with the evidence that supports an association between ovarian cancer and Lynch syndrome5,34,35. These findings should raise physicians’ awareness of genetic testing beyond BRCA1/2 and encourage the inclusion of MMR genes in the panel for Thai ovarian cancer patients.

Twenty-seven patients (4.4%) who fulfilled breast-ovarian cancer testing indications had a history of other cancer types. It is well known that colorectal cancer, endometrial cancer, brain tumor, and sarcoma are associated with genetic predispositions such as MMR genes or TP53. Many high-penetrance genes also exhibit pleiotropic clinical manifestations of other common cancers. As multi-gene panel testing for all breast cancer patients was found to be cost-effective, it may be rational to expand testing to patients with cancers in the HBOC spectrum36.

Lastly, we identified double pathogenic variants in ten individuals. To date, there is limited data on double mutations in cancer patients. One study found double heterozygous variants in 1.2% of hereditary breast cancer patients37. However, the exact prevalence is still undetermined. It is unclear if patients with double mutations would have different cancer susceptibility and clinical severity compared to single mutation carriers. However, this information can be used for proper surveillance strategies for a broader spectrum of cancers in patients and at-risk family members.

With various screening methods for breast and ovarian cancer, as well as the availability of prophylactic surgeries in most regions of Thailand, this data will properly guide physicians for personalized surveillance and preventive strategies in patients or at-risk family members. Our findings will support the rationale of implementing multi-gene panel testing beyond BRCA1/2 in Thai patients with HBOC-related cancers and provide more information on the Southeast Asian population. With cost reduction and faster turnaround time, the clinical use of multi-gene panel tests for cancer will gradually increase among many low-to-middle-income countries.

This study had some limitations. Firstly, some variants were found in a small number of patients. Therefore, the clinical data associated with those variants were limited. Secondly, some clinical data was incomplete, and the follow-up duration may not have been long enough to demonstrate susceptibility to other types of cancer. Thirdly, this cohort may be biased towards breast cancer patients and away from prostate cancer. This is explained by greater awareness about genetic factors involved in breast and ovarian cancers, leading to more genetic testing in these patients. Moreover, guidelines for genetic testing in high-risk breast/ovarian cancer patients have been widely published, while genetic testing for prostate cancer has only been recommended in recent NCCN guidelines for patients with metastatic disease, strong family history, or high-risk features (e.g., very high PSA level or high Gleason score)5,38. In Thailand, genetic testing for prostate cancer is not included in current national guidelines, leading to underutilization in these patients. Finally, this was a single-center study with most patients from the central region of Thailand. However, Siriraj Hospital is a major cancer referral center that provides genetic testing services to other hospitals nationwide.

### Methods

This study was approved by the Siriraj Hospital Institutional Review Board Protocol No. 418/2562(EC2) and was conducted according to Good Clinical Practice and the Declaration of Helsinki. All participants provided written informed consent. All Thai patients with any cancers in the HBOC spectrum, including breast, ovarian, pancreatic, and prostate cancers, who had germline cancer susceptibility multi-gene panel testing at Siriraj Hospital between 2016 and 2023 were included. The data of all patients with germline P/LP variants in cancer susceptibility genes were collected. Patients with known clinical syndromes of Mendelian disorders (such as neurofibromatosis, ataxia-telangiectasia, or Peutz-Jeghers syndrome) and individuals who received targeted gene testing due to known affected family members were excluded. The data regarding the types of cancers, histopathological profiles, clinical stages of cancer, age of onset, and family history were systematically collected and reviewed with 2023 NCCN guidelines for genetic/familial high-risk assessment of breast and ovarian cancers5. We calculated the positive rate of P/LP BRCA and non-BRCA variants using descriptive statistics for each cancer type. Additionally, we reported the clinical spectrum of breast cancer patients with non-BRCA P/LP variants.

Peripheral blood specimens from each patient were collected to extract genomic DNA for sequencing (see Supplementary 4). The comprehensive cancer panel in this study included moderate-to-high penetrance genes for breast and ovarian cancers (Table 5)5–7,39. Sanger sequencing and Multiplex Ligation-dependent Probe Amplification (MLPA) were performed to validate the results in all identified single nucleotide variants (SNV) and copy number variants (CNV), respectively. The variant call format (VCF) and BAM files were transferred to VarSeq – VSClinical software (Golden Helix, USA) for analysis and classification. We interpreted and classified the variants following the 2015 ACMG-AMP standards and guidelines for the interpretation of sequence variants and 2020 ACMG-ClinGen technical standards for the interpretation and reporting of constitutional copy-number variants40,41. All reportable variants, including P/LP variants and VUS, were systematically verified.Table 5Lists of genes in comprehensive cancer panelPhenotypeGenesHigh-penetrance genes for breast cancerBRCA1, BRCA2, CDH1, STK11, PALB2, TP53, PTENModerate genes for breast cancerATM, CHEK2, NF1, BARD1, RAD51C, RAD51DPossible breast cancer genesNBN, RAD50, XRCC2Moderate-risk ovarian cancer genesBRIP1, MLH1, MSH2, MSH6, PMS2, EPCAMHigh-penetrance genes for other types of cancerAPC, AXIN2, BMPR1A, CDK4, CDKN2A, FANCC, MSH2, MUTYH, NTHL1, POLD1, POLE, SMAD4, VHLOMIM numbers for each gene: BRCA1 (OMIM 113705), BRCA2 (OMIM 600185), CDH1 (OMIM 192090), STK11 (OMIM 602216), TP53 (191170), ATM (OMIM 607585), BRIP1 (OMIM 605882), CHEK2 (OMIM 604373), NF1 (OMIM 613113), PALB2 (OMIM 610355), BARD1 (OMIM 601593), NBN (OMIM 602667), RAD50 (OMIM 604040), XRCC2 (OMIM 600375), RAD51C (OMIM 602774), RAD51D OMIM 602954), MLH1 (OMIM 120436), MSH2 (OMIM 609309), MSH6 (OMIM 600678), PMS2 (OMIM 600259), EPCAM (OMIM 185535), APC (OMIM 611731), AXIN2 (OMIM 604025), BMPR1A (OMIM 601299), CDK4 (OMIM 123829), CDKN2A (OMIM 600160), ERCC2 (OMIM 278730), FANCC (OMIM 613899), MSH3 (OMIM 600887), MUTYH (OMIM 604933), NTHL1 (OMIM 602656), POLD1 (OMIM 174761), POLE (OMIM 174762), PTEN (OMIM 601728), SMAD4 (OMIM 600993), VHL (OMIM 608537).

Lists of genes in comprehensive cancer panel

OMIM numbers for each gene: BRCA1 (OMIM 113705), BRCA2 (OMIM 600185), CDH1 (OMIM 192090), STK11 (OMIM 602216), TP53 (191170), ATM (OMIM 607585), BRIP1 (OMIM 605882), CHEK2 (OMIM 604373), NF1 (OMIM 613113), PALB2 (OMIM 610355), BARD1 (OMIM 601593), NBN (OMIM 602667), RAD50 (OMIM 604040), XRCC2 (OMIM 600375), RAD51C (OMIM 602774), RAD51D OMIM 602954), MLH1 (OMIM 120436), MSH2 (OMIM 609309), MSH6 (OMIM 600678), PMS2 (OMIM 600259), EPCAM (OMIM 185535), APC (OMIM 611731), AXIN2 (OMIM 604025), BMPR1A (OMIM 601299), CDK4 (OMIM 123829), CDKN2A (OMIM 600160), ERCC2 (OMIM 278730), FANCC (OMIM 613899), MSH3 (OMIM 600887), MUTYH (OMIM 604933), NTHL1 (OMIM 602656), POLD1 (OMIM 174761), POLE (OMIM 174762), PTEN (OMIM 601728), SMAD4 (OMIM 600993), VHL (OMIM 608537).

Further information on research design is available in the Nature Research Reporting Summary linked to this article.

### Patient recruitment and data collection

This study was approved by the Siriraj Hospital Institutional Review Board Protocol No. 418/2562(EC2) and was conducted according to Good Clinical Practice and the Declaration of Helsinki. All participants provided written informed consent. All Thai patients with any cancers in the HBOC spectrum, including breast, ovarian, pancreatic, and prostate cancers, who had germline cancer susceptibility multi-gene panel testing at Siriraj Hospital between 2016 and 2023 were included. The data of all patients with germline P/LP variants in cancer susceptibility genes were collected. Patients with known clinical syndromes of Mendelian disorders (such as neurofibromatosis, ataxia-telangiectasia, or Peutz-Jeghers syndrome) and individuals who received targeted gene testing due to known affected family members were excluded. The data regarding the types of cancers, histopathological profiles, clinical stages of cancer, age of onset, and family history were systematically collected and reviewed with 2023 NCCN guidelines for genetic/familial high-risk assessment of breast and ovarian cancers5. We calculated the positive rate of P/LP BRCA and non-BRCA variants using descriptive statistics for each cancer type. Additionally, we reported the clinical spectrum of breast cancer patients with non-BRCA P/LP variants.

### Multi-gene panel testing and variant analysis

Peripheral blood specimens from each patient were collected to extract genomic DNA for sequencing (see Supplementary 4). The comprehensive cancer panel in this study included moderate-to-high penetrance genes for breast and ovarian cancers (Table 5)5–7,39. Sanger sequencing and Multiplex Ligation-dependent Probe Amplification (MLPA) were performed to validate the results in all identified single nucleotide variants (SNV) and copy number variants (CNV), respectively. The variant call format (VCF) and BAM files were transferred to VarSeq – VSClinical software (Golden Helix, USA) for analysis and classification. We interpreted and classified the variants following the 2015 ACMG-AMP standards and guidelines for the interpretation of sequence variants and 2020 ACMG-ClinGen technical standards for the interpretation and reporting of constitutional copy-number variants40,41. All reportable variants, including P/LP variants and VUS, were systematically verified.Table 5Lists of genes in comprehensive cancer panelPhenotypeGenesHigh-penetrance genes for breast cancerBRCA1, BRCA2, CDH1, STK11, PALB2, TP53, PTENModerate genes for breast cancerATM, CHEK2, NF1, BARD1, RAD51C, RAD51DPossible breast cancer genesNBN, RAD50, XRCC2Moderate-risk ovarian cancer genesBRIP1, MLH1, MSH2, MSH6, PMS2, EPCAMHigh-penetrance genes for other types of cancerAPC, AXIN2, BMPR1A, CDK4, CDKN2A, FANCC, MSH2, MUTYH, NTHL1, POLD1, POLE, SMAD4, VHLOMIM numbers for each gene: BRCA1 (OMIM 113705), BRCA2 (OMIM 600185), CDH1 (OMIM 192090), STK11 (OMIM 602216), TP53 (191170), ATM (OMIM 607585), BRIP1 (OMIM 605882), CHEK2 (OMIM 604373), NF1 (OMIM 613113), PALB2 (OMIM 610355), BARD1 (OMIM 601593), NBN (OMIM 602667), RAD50 (OMIM 604040), XRCC2 (OMIM 600375), RAD51C (OMIM 602774), RAD51D OMIM 602954), MLH1 (OMIM 120436), MSH2 (OMIM 609309), MSH6 (OMIM 600678), PMS2 (OMIM 600259), EPCAM (OMIM 185535), APC (OMIM 611731), AXIN2 (OMIM 604025), BMPR1A (OMIM 601299), CDK4 (OMIM 123829), CDKN2A (OMIM 600160), ERCC2 (OMIM 278730), FANCC (OMIM 613899), MSH3 (OMIM 600887), MUTYH (OMIM 604933), NTHL1 (OMIM 602656), POLD1 (OMIM 174761), POLE (OMIM 174762), PTEN (OMIM 601728), SMAD4 (OMIM 600993), VHL (OMIM 608537).

Lists of genes in comprehensive cancer panel

OMIM numbers for each gene: BRCA1 (OMIM 113705), BRCA2 (OMIM 600185), CDH1 (OMIM 192090), STK11 (OMIM 602216), TP53 (191170), ATM (OMIM 607585), BRIP1 (OMIM 605882), CHEK2 (OMIM 604373), NF1 (OMIM 613113), PALB2 (OMIM 610355), BARD1 (OMIM 601593), NBN (OMIM 602667), RAD50 (OMIM 604040), XRCC2 (OMIM 600375), RAD51C (OMIM 602774), RAD51D OMIM 602954), MLH1 (OMIM 120436), MSH2 (OMIM 609309), MSH6 (OMIM 600678), PMS2 (OMIM 600259), EPCAM (OMIM 185535), APC (OMIM 611731), AXIN2 (OMIM 604025), BMPR1A (OMIM 601299), CDK4 (OMIM 123829), CDKN2A (OMIM 600160), ERCC2 (OMIM 278730), FANCC (OMIM 613899), MSH3 (OMIM 600887), MUTYH (OMIM 604933), NTHL1 (OMIM 602656), POLD1 (OMIM 174761), POLE (OMIM 174762), PTEN (OMIM 601728), SMAD4 (OMIM 600993), VHL (OMIM 608537).

### Reporting summary

Further information on research design is available in the Nature Research Reporting Summary linked to this article.

REPORTING SUMMARY
Supplementary

REPORTING SUMMARY

Supplementary

### Supplementary information

REPORTING SUMMARY
Supplementary

REPORTING SUMMARY

Supplementary



# SUPPLEMENTAL FILE 1: 41525_2024_Article_400.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 4: genetics_bop.pdf

[PDF file available at: genetics_bop.pdf]

