# MAIN TEXT

## Assessment of BRCA1 and BRCA2 Germline Variant Data From Patients With Breast Cancer in a Real-World Data Registry

### Abstract

PURPOSEThe emergence of large real-world clinical databases and tools to mine electronic medical records has allowed for an unprecedented look at large data sets with clinical and epidemiologic correlates. In clinical cancer genetics, real-world databases allow for the investigation of prevalence and effectiveness of prevention strategies and targeted treatments and for the identification of barriers to better outcomes. However, real-world data sets have inherent biases and problems (eg, selection bias, incomplete data, measurement error) that may hamper adequate analysis and affect statistical power.METHODSHere, we leverage a real-world clinical data set from a large health network for patients with breast cancer tested for variants in BRCA1 and BRCA2 (N = 12,423). We conducted data cleaning and harmonization, cross-referenced with publicly available databases, performed variant reassessment and functional assays, and used functional data to inform a variant's clinical significance applying American College of Medical Geneticists and the Association of Molecular Pathology guidelines.RESULTSIn the cohort, White and Black patients were over-represented, whereas non-White Hispanic and Asian patients were under-represented. Incorrect or missing variant designations were the most significant contributor to data loss. While manual curation corrected many incorrect designations, a sizable fraction of patient carriers remained with incorrect or missing variant designations. Despite the large number of patients with clinical significance not reported, original reported clinical significance assessments were accurate. Reassessment of variants in which clinical significance was not reported led to a marked improvement in data quality.CONCLUSIONWe identify the most common issues with BRCA1 and BRCA2 testing data entry and suggest approaches to minimize data loss and keep interpretation of clinical significance of variants up to date.

### INTRODUCTION

Since the identification and cloning of the major breast and ovarian cancer predisposition genes, BRCA1 and BRCA2, clinical genetic testing for germline pathogenic variants in these genes has become widely available.1-5 Initially, results from genetic testing were primarily used for risk stratification. More recently, they have also become an integral part of clinical management and therapy selection, as illustrated by the recent approval of several PARP inhibitors in multiple therapy settings in breast, ovarian, pancreatic, and prostate cancers.6-11 Therefore, accurate classification of variants as pathogenic or benign and systematic classification reassessment are required to achieve the goals of precision medicine and improve cancer outcomes.

Key Objective
To assess data quality and identify barriers for the use of BRCA1 and BRCA2 genetic testing data from a real-world clinical database.
Knowledge Generated
We identify the most common issues with BRCA1 and BRCA2 testing data entry and suggest approaches to minimize data loss and keep interpretation of clinical significance of variants up to date.
Relevance (F.P.-Y. Lin)
Systematic appraisals of the clinical significance of germline cancer susceptibility variants, as presented in this study, are critical for maintaining high-quality data curation in clinicogenomic registries. This ensures that quality real-world data can be generated to support risk prevention and outcome studies.**Relevance section written by JCO Clinical Cancer Informatics Associate Editor Frank P.-Y. Lin, PhD, MBChB, FRACP, FAIDH.

Key Objective

To assess data quality and identify barriers for the use of BRCA1 and BRCA2 genetic testing data from a real-world clinical database.

Knowledge Generated

We identify the most common issues with BRCA1 and BRCA2 testing data entry and suggest approaches to minimize data loss and keep interpretation of clinical significance of variants up to date.

Relevance (F.P.-Y. Lin)

Systematic appraisals of the clinical significance of germline cancer susceptibility variants, as presented in this study, are critical for maintaining high-quality data curation in clinicogenomic registries. This ensures that quality real-world data can be generated to support risk prevention and outcome studies.*

*Relevance section written by JCO Clinical Cancer Informatics Associate Editor Frank P.-Y. Lin, PhD, MBChB, FRACP, FAIDH.

Classification schemas for germline variants in BRCA1 and BRCA2 according to pathogenicity may vary by genetic testing provider but typically follow an overall five-tier framework: pathogenic, likely pathogenic, variant of uncertain significance (VUS), likely benign, and benign.12 These schemas use elements from three distinct, but not mutually exclusive, classification frameworks: first, a rule-based framework on the basis of the variant effect inferred by changes in genetic code; second, for variants with effects that cannot be inferred from the genetic code changes, a statistical multifactorial model which incorporates information from pathology, family history, cosegregation, and co-occurrence with other pathogenic variants is used13,14; and third, the American College of Medical Geneticists and the Association of Molecular Pathology (ACMG/AMP) have developed a framework which uses strength of evidence criteria to include a wide array of sources of information.15-17

At the center of most models is the concept that they can differentiate high-risk (typically odds ratio > 4) from non–high-risk variants. Thus, the difference between the terms “likely pathogenic” and “pathogenic” is not meant to reflect a difference in severity, in which a pathogenic variant would confer a risk higher than a likely pathogenic variant. Rather, the difference reflects the probability of a variant conferring high risk, in which a pathogenic variant has a higher probability of being a high-risk variant, than a likely pathogenic variant given the available data. Because the available data change with time, there is a need for constant reassessing of classification. This might have important implications not only for the treatment of the carrier but also for future patients who may carry the same variant.

Real-world databases contain routinely collected health data generated from the operations of health care providers and allow for relatively large observational studies with data collected under real-world circumstances.18 They provide a resource to analyze clinical data in a typical clinical setting as opposed to in a well-controlled clinical trial environment (efficacy of an intervention). Analysis of the clinical data is important to identify barriers to care and also to determine whether the treatment modalities have the expected efficacy derived from clinical trials. However, analyses of these data sets have challenges and bias inherent to observational studies and additional challenges specific to these data sets such as data linkage problems, misclassification bias, and under-reporting.18-20 Here, we report on an assessment of 12,423 cases of breast cancer tested for germline pathogenic variants in BRCA1 and BRCA2.

### CONTEXT

Key Objective
To assess data quality and identify barriers for the use of BRCA1 and BRCA2 genetic testing data from a real-world clinical database.
Knowledge Generated
We identify the most common issues with BRCA1 and BRCA2 testing data entry and suggest approaches to minimize data loss and keep interpretation of clinical significance of variants up to date.
Relevance (F.P.-Y. Lin)
Systematic appraisals of the clinical significance of germline cancer susceptibility variants, as presented in this study, are critical for maintaining high-quality data curation in clinicogenomic registries. This ensures that quality real-world data can be generated to support risk prevention and outcome studies.**Relevance section written by JCO Clinical Cancer Informatics Associate Editor Frank P.-Y. Lin, PhD, MBChB, FRACP, FAIDH.

Key Objective

To assess data quality and identify barriers for the use of BRCA1 and BRCA2 genetic testing data from a real-world clinical database.

Knowledge Generated

We identify the most common issues with BRCA1 and BRCA2 testing data entry and suggest approaches to minimize data loss and keep interpretation of clinical significance of variants up to date.

Relevance (F.P.-Y. Lin)

Systematic appraisals of the clinical significance of germline cancer susceptibility variants, as presented in this study, are critical for maintaining high-quality data curation in clinicogenomic registries. This ensures that quality real-world data can be generated to support risk prevention and outcome studies.*

*Relevance section written by JCO Clinical Cancer Informatics Associate Editor Frank P.-Y. Lin, PhD, MBChB, FRACP, FAIDH.

### METHODS

Our data set is composed of patients with breast cancer undergoing genetic testing for BRCA1 and BRCA2 (N = 12,423 unique patient IDs), diagnosed between 2000 and 2022 and retrieved from the Syapse Learning Health Network (LHN) in January 2023. The LHN contains patient populations from five large community health systems and nine reference laboratory integrations, with varying breadth and depth of clinical information, enabling insights into the practice of precision medicine. Testing information is captured either through direct integration or by Certified Tumor Registrars, who record results as they are found in clinical reports or physician notes. All data retrieved for this study were deidentified and covered under the institutional nonhuman patient research determination (AdventHealth Institutional Review Board 1694276-1; Moffitt MCC 21387). Chi-square goodness of fit was used to compare distribution of ethnic (df = 1) and racial (df = 5) groups between the current study population (N = 12,423) and the US population (2020 US census data) using a significance of 0.05.21

Variant designations were considered correct if they conformed to Human Genome Variation Society (HGVS) nomenclature standard22 and presented a coding DNA sequence designation (preceded by c) or a predicted protein sequence designation (preceded by p.) in one- or three-letter amino acid code. * can be used to indicate a stop codon in one- and three-letter code amino acid code descriptions. Variant HGVS designations only lacking or with misplaced parentheses (eg, p.Met1775Arg instead of p.(Met1775Arg)) were considered correct. Incorrect designations were then assessed to determine whether the available information was sufficient to unambiguously identify a variant. If this was the case, a correct designation was assigned to the variant. If not, the designation would be considered unrecoverable. Exceptions were made for large rearrangements because there is less agreement on the terminology to describe these variants23,24 and there was no breakpoint information associated with each variant. Therefore, it was not possible to determine whether their designation is correct. Large rearrangements have long been recognized as clinically significant,25-27 and therefore, for the purpose of this analysis, they were considered correct and assumed to represent a pathogenic variant. The procedure for variant designation analysis is illustrated in the Data Supplement (Fig S1) and further described in the Data Supplement (Materials).

We denoted variant classifications extracted from the data set as Significance, with possible values: pathogenic, likely pathogenic, uncertain significance, likely benign, benign, and not reported. To reassess the classification, we compared it with variant interpretations in the ClinVar database. Final assertion of clinical significance for the variant was guided by review status (ClinVar gold stars). In ClinVar, the level of review supporting each variant classification is reported in the format of Gold stars, which is a graphical representation of the aggregate review status as follows: four gold stars, practice guideline; three gold stars, reviewed by the expert panel; one gold star, criteria provided, single submitter; and none, no assertion criteria or no classification provided. Assertions reviewed by the expert panel (three gold stars) using Evidence-base Network for the Interpretation of Germline Mutant Alleles (ENIGMA) standards28 were used when available. The ENIGMA standard is based on a multifactorial statistical model and specific thresholds of likelihood of pathogenicity.13,14,28 For variants that have not been reviewed by ENIGMA (ClinVar three stars), we used information from other submitters (ClinVar one and two stars). Whenever there was a conflict between assertions, the assertion with the highest number of gold stars was used. When the conflicting information was derived from entries with the same number of stars, the final assessment was conflicting information. Sankey diagrams were prepared using SankeyMATIC.29

Variants of uncertain clinical significance for which there were functional data available (FYI HBOC30)31 were assigned functional evidence criteria following the ACMG/AMP guidelines as defined for BRCA1 and BRCA2.31,32 Selected variants, for which no previous functional data existed, were analyzed using the validated BRCA1 transcriptional assay.33,34 In this assay, variants of BRCA1 C terminus (aa 1,396-1,863) are fused to the yeast GAL4 DNA-binding domain and expressed in human cells containing a luciferase reporter gene driven by an inducible promoter with GAL4 DNA binding sequences.35 Variant cDNAs were generated by polymerase chain reaction using site-directed mutagenesis primers (Data Supplement, Table S1) and cloned as previously described.33 Benign variants drive the expression of luciferase using a GAL4 responsive promoter. Pathogenic variants are defective in driving expression of the reporter gene. To determine the likelihood of pathogenicity for each variant given the functional data, we applied VarCall, a Bayesian two-component mixture model extensively used with the transcriptional assay.33,36

### Data Sets

Our data set is composed of patients with breast cancer undergoing genetic testing for BRCA1 and BRCA2 (N = 12,423 unique patient IDs), diagnosed between 2000 and 2022 and retrieved from the Syapse Learning Health Network (LHN) in January 2023. The LHN contains patient populations from five large community health systems and nine reference laboratory integrations, with varying breadth and depth of clinical information, enabling insights into the practice of precision medicine. Testing information is captured either through direct integration or by Certified Tumor Registrars, who record results as they are found in clinical reports or physician notes. All data retrieved for this study were deidentified and covered under the institutional nonhuman patient research determination (AdventHealth Institutional Review Board 1694276-1; Moffitt MCC 21387). Chi-square goodness of fit was used to compare distribution of ethnic (df = 1) and racial (df = 5) groups between the current study population (N = 12,423) and the US population (2020 US census data) using a significance of 0.05.21

### Designation Assessment and Harmonization

Variant designations were considered correct if they conformed to Human Genome Variation Society (HGVS) nomenclature standard22 and presented a coding DNA sequence designation (preceded by c) or a predicted protein sequence designation (preceded by p.) in one- or three-letter amino acid code. * can be used to indicate a stop codon in one- and three-letter code amino acid code descriptions. Variant HGVS designations only lacking or with misplaced parentheses (eg, p.Met1775Arg instead of p.(Met1775Arg)) were considered correct. Incorrect designations were then assessed to determine whether the available information was sufficient to unambiguously identify a variant. If this was the case, a correct designation was assigned to the variant. If not, the designation would be considered unrecoverable. Exceptions were made for large rearrangements because there is less agreement on the terminology to describe these variants23,24 and there was no breakpoint information associated with each variant. Therefore, it was not possible to determine whether their designation is correct. Large rearrangements have long been recognized as clinically significant,25-27 and therefore, for the purpose of this analysis, they were considered correct and assumed to represent a pathogenic variant. The procedure for variant designation analysis is illustrated in the Data Supplement (Fig S1) and further described in the Data Supplement (Materials).

### Classification Reassessment

We denoted variant classifications extracted from the data set as Significance, with possible values: pathogenic, likely pathogenic, uncertain significance, likely benign, benign, and not reported. To reassess the classification, we compared it with variant interpretations in the ClinVar database. Final assertion of clinical significance for the variant was guided by review status (ClinVar gold stars). In ClinVar, the level of review supporting each variant classification is reported in the format of Gold stars, which is a graphical representation of the aggregate review status as follows: four gold stars, practice guideline; three gold stars, reviewed by the expert panel; one gold star, criteria provided, single submitter; and none, no assertion criteria or no classification provided. Assertions reviewed by the expert panel (three gold stars) using Evidence-base Network for the Interpretation of Germline Mutant Alleles (ENIGMA) standards28 were used when available. The ENIGMA standard is based on a multifactorial statistical model and specific thresholds of likelihood of pathogenicity.13,14,28 For variants that have not been reviewed by ENIGMA (ClinVar three stars), we used information from other submitters (ClinVar one and two stars). Whenever there was a conflict between assertions, the assertion with the highest number of gold stars was used. When the conflicting information was derived from entries with the same number of stars, the final assessment was conflicting information. Sankey diagrams were prepared using SankeyMATIC.29

### Functional Assessment

Variants of uncertain clinical significance for which there were functional data available (FYI HBOC30)31 were assigned functional evidence criteria following the ACMG/AMP guidelines as defined for BRCA1 and BRCA2.31,32 Selected variants, for which no previous functional data existed, were analyzed using the validated BRCA1 transcriptional assay.33,34 In this assay, variants of BRCA1 C terminus (aa 1,396-1,863) are fused to the yeast GAL4 DNA-binding domain and expressed in human cells containing a luciferase reporter gene driven by an inducible promoter with GAL4 DNA binding sequences.35 Variant cDNAs were generated by polymerase chain reaction using site-directed mutagenesis primers (Data Supplement, Table S1) and cloned as previously described.33 Benign variants drive the expression of luciferase using a GAL4 responsive promoter. Pathogenic variants are defective in driving expression of the reporter gene. To determine the likelihood of pathogenicity for each variant given the functional data, we applied VarCall, a Bayesian two-component mixture model extensively used with the transcriptional assay.33,36

### RESULTS

Demographic characteristics of the cohort (N = 12,423) are shown in Table 1. The cohort is racially diverse, and when compared with the 2020 US census data, White (61.6% v 77.7%) and Black (12.4% v 15.8%) individuals are over-represented and American Indian and Alaska Native (0.3% v 1.1%), Asian (2.8% v 6%), and Native Hawaiian and other Pacific Islander (0.1% v 0.2%) participants are under-represented (P < .01). In terms of ethnicity, Hispanic/Latino participants are notably under-represented (5.5% v 18.7%) when compared with the 2020 US census data21 (P < .01). Individuals receiving somatic testing results were more likely to be diagnosed with higher-stage disease (III and IV) than those receiving germline test results (Table 1).

Demographic Characteristics of the Cohort

Variant designations were found in a variety of formats: (1) Breast Cancer Information Core (BIC) legacy nucleotide, (2) BIC legacy protein one-letter code, (3) BIC legacy protein three-letter code, (4) BIC legacy intronic, (5) HGVS nucleotide, (6) HGVS protein one-letter code, (7) HGVS protein three-letter code, (8) HGVS nucleotide followed by HGVS protein one-letter code, (9) HGVS nucleotide followed by HGVS protein three-letter code, (10) BIC legacy protein one-letter code followed by BIC legacy nucleotide, and (11) rearrangements were described in a variety of formats (Data Supplement, Tables S2 and S3).

In addition to the variety in formats, we found a high rate of incorrect or unknown BRCA1 designations: 18.4% (58 of 315) of variants affecting data from 52.2% (303 of 581) of cases. While most incorrect designations could be corrected as they were simple typographical errors, incorrect or missing HGVS format, 7.0% (22 of 315) of BRCA1 variant entries affecting data from 45.3% (263 of 581) cases could not be unambiguously determined as a BRCA1 variant. For example, for 3.5% (11 of 315) of the variants from 43.4% (252 of 581) of patients, the report did not state the variant in the variant ID field (BRCA1 mutation—no report; BRCA1; variant not stated; Fig 1; Data Supplement, Table S4).

Overview of BRCA1 variant assessment. Subway chart illustrating the steps in data collection, curation, and harmonization of variant designation (green line). Variants were then separated according to origin as somatic (yellow line) or germline (blue line) and stratified by variant type or clinical significance. BC, breast cancer.

A similarly high rate of incorrect or unknown designations was found for BRCA2 (Fig 2; Data Supplement, Table S5): 20% (58 of 290) of variants affecting data from 36.7% (138 of 376) of the patients. Fifteen variant entries (5.2%; 15 of 290) affecting data from 25% (94 of 376) of the patients could not be unambiguously corrected. For example, 2.8% (8 of 290) of the variant entries from 23.1% (87 of 376) of patients did not state the variant (eg, variant not stated; no variant stated; BRCA2).

Overview of BRCA2 variant assessment. Subway chart illustrating the steps in data collection, curation, and harmonization of variant designation (green line). Variants were then separated according to origin as somatic (yellow line) or germline (blue line) and stratified by variant type or clinical significance. BC, breast cancer.

Next, after data harmonization (see the Methods section) we identified several instances in which the same variant had correct designations but different formats: 21.2% (62 of 293) for BRCA1 and 21.8% (60 of 275) for BRCA2 (Figs 1 and 2). We then consolidated these variants into a single entry (Data Supplement, Tables S6 and S7). Harmonization and consolidation are labor-intensive but can be automated to some extent (code availability). After harmonization, there were 231 unique BRCA1 variants from 318 patients with breast cancer and 215 unique BRCA2 variants from 285 patients with breast cancer.

In summary, although there was a high rate of incorrect variant designations, many could be corrected. However, data from approximately 45% of patients with a BRCA1 variant and 43% with a BRCA2 variant did not include the specific variant found.

Somatic variants make up 30.1% and 10% of the unique variants reported for BRCA1 and BRCA2, respectively (Figs 1 and 2). For BRCA1, data loss from entries that could not be unambiguously corrected was comparable between germline and somatic variant entries (8% v 7%). For BRCA2, data loss was notably higher for somatic (10%) than for germline (5%) variant entries.

After harmonization and consolidation of variant designations, we first determined the rate of clinically meaningful internal conflicts (when the same variant received two or more different calls for significance). We did not consider not reported as a conflict or when the difference was between likely pathogenic and pathogenic or between likely benign and benign. For BRCA1, there was only one conflict in which the variant received two calls: as benign (B) and VUS (Data Supplement, Table S6). For BRCA2, there were two conflicts between pathogenic and VUS (Data Supplement, Table S7).

Next, for every variant, we examined and reassessed the classification (Significance) of each variant using interpretations from ClinVar. For BRCA1 and BRCA2, the most frequent reason for reclassification was missing significance (not reported) in the original entry representing 49.3% (114 of 231) and 47.9% (103 of 215) of the BRCA1 and BRCA2 variants, respectively (Figs 3A and 3B). This was followed by the updating of variants that were VUS at the time of data entry and have since then been classified as likely benign/benign (LB/B), likely pathogenic/pathogenic (LP/P), or have conflicting ClinVar interpretations. Only one BRCA1 (p.(Ile925Leu)) and one BRCA2 p.(Gln347Arg) variant had consequential reclassification from LP/P to LB/B. The original incorrect classifications were most likely due to incorrect manual data entry.

Reassessment of clinical significance. Sankey diagram showing the results of reassessment for (A) BRCA1 and (B) BRCA2. Top level represents the number of variants in each significance class. Middle level represents the reclassification of variants according to ClinVar. Variants receiving VUS or conflicting assertions were further assessed using functional data to assign ACMG evidence criteria (bottom level). Note that totals are higher than the total number of unique variants because some variants were recorded with two or more different significance assertions. ACMG, American College of Medical Geneticists; VUS, variant of uncertain significance.

To improve the information for variants with a ClinVar conflicting classification or VUS designation, we retrieved integrated functional data for BRCA1 and BRCA2 missense variants.31,37 Functional data provided ACMG evidence criteria for 18 of 42 BRCA1 VUS and 20 of 62 variants with conflicting interpretation (Fig 3A; Data Supplement, Table S8). For BRCA2, functional data provided evidence criteria for 1 of 22 VUS and 6 of 72 variants with conflicting interpretation (Fig 3B; Data Supplement, Table S9).

Next, we tested eight variants for which there were no functional data using the validated BRCA1 transcription assay (Fig 4A).31 All eight variants tested expressed levels comparable with the wild type (Fig 4B; Data Supplement, Fig S2). Only variant p.(Gly1743Ala) displayed reduced activity when compared with the wild type or a known benign variant (p.(Ser1613Gly)) but not as low as the control pathogenic variant (p.(Met1775Arg)). All other variants had activity comparable with or higher than the wild type (Fig 4A). All tested variants, with the exception of p.(Gly1743Ala), which received an indeterminate result, were assigned an ACMG/AMP evidence criteria code, BS3 code, corresponding to strong evidence for benignity (Data Supplement, Tables S10-S12; see the Data Supplement, Fig S3, for VarCall analysis).

Functional assessment of BRCA1 VUS. (A) Transcription activity of BRCA1 missense VUS expressed as a fusion of the BRCA1 C-terminal region (aa 1,396-1,863) to the GAL4 DBD shown as a percentage of wild-type activity. The wild-type control is represented with a black bar. The known benign p.(Ser1613Gly) and pathogenic p.(Met1775Arg) variants are represented by blue and red bars, respectively. (B) GAL4 DBD-BRCA1 protein levels in HEK293FT cells. Immunoblot using anti-GAL4DBD and anti-β-actin. Lines indicate where irrelevant lanes were removed (see the unedited blot in the Data Supplement, Fig S1). DBD, DNA-binding domain; VUS, variant of uncertain significance.

Approximately 5% and 3% of tested patients were carriers of a germline or somatic variant in BRCA1 and BRCA2, respectively (Data Supplement, Table S13). However, fewer than 1% could be verified as likely pathogenic/pathogenic germline variants, since not reported or uncertain were listed as clinical significance for the majority of variants found for both BRCA1 and BRCA2 (Data Supplement, Tables S2 and S3).

After the elimination of variants that could not be unambiguously identified and reassessment of variant clinical significance classifications, approximately 2.2% of patients tested were carriers of LP/P variants in BRCA1 or BRCA2 (Data Supplement, Table S13). Eighty-eight patients (0.71%) had a germline finding of LP/P in BRCA1. One hundred patients (0.8%) had a germline finding of LP/P in BRCA2. The combined prevalence of germline LP/P variants in BRCA1 and BRCA2 was 1.5%. Large rearrangements constituted 4.8% of BRCA1 and 1.9% of BRCA2 variants.

### Cohort Characteristics

Demographic characteristics of the cohort (N = 12,423) are shown in Table 1. The cohort is racially diverse, and when compared with the 2020 US census data, White (61.6% v 77.7%) and Black (12.4% v 15.8%) individuals are over-represented and American Indian and Alaska Native (0.3% v 1.1%), Asian (2.8% v 6%), and Native Hawaiian and other Pacific Islander (0.1% v 0.2%) participants are under-represented (P < .01). In terms of ethnicity, Hispanic/Latino participants are notably under-represented (5.5% v 18.7%) when compared with the 2020 US census data21 (P < .01). Individuals receiving somatic testing results were more likely to be diagnosed with higher-stage disease (III and IV) than those receiving germline test results (Table 1).

Demographic Characteristics of the Cohort

### Designation Assessment

Variant designations were found in a variety of formats: (1) Breast Cancer Information Core (BIC) legacy nucleotide, (2) BIC legacy protein one-letter code, (3) BIC legacy protein three-letter code, (4) BIC legacy intronic, (5) HGVS nucleotide, (6) HGVS protein one-letter code, (7) HGVS protein three-letter code, (8) HGVS nucleotide followed by HGVS protein one-letter code, (9) HGVS nucleotide followed by HGVS protein three-letter code, (10) BIC legacy protein one-letter code followed by BIC legacy nucleotide, and (11) rearrangements were described in a variety of formats (Data Supplement, Tables S2 and S3).

In addition to the variety in formats, we found a high rate of incorrect or unknown BRCA1 designations: 18.4% (58 of 315) of variants affecting data from 52.2% (303 of 581) of cases. While most incorrect designations could be corrected as they were simple typographical errors, incorrect or missing HGVS format, 7.0% (22 of 315) of BRCA1 variant entries affecting data from 45.3% (263 of 581) cases could not be unambiguously determined as a BRCA1 variant. For example, for 3.5% (11 of 315) of the variants from 43.4% (252 of 581) of patients, the report did not state the variant in the variant ID field (BRCA1 mutation—no report; BRCA1; variant not stated; Fig 1; Data Supplement, Table S4).

Overview of BRCA1 variant assessment. Subway chart illustrating the steps in data collection, curation, and harmonization of variant designation (green line). Variants were then separated according to origin as somatic (yellow line) or germline (blue line) and stratified by variant type or clinical significance. BC, breast cancer.

A similarly high rate of incorrect or unknown designations was found for BRCA2 (Fig 2; Data Supplement, Table S5): 20% (58 of 290) of variants affecting data from 36.7% (138 of 376) of the patients. Fifteen variant entries (5.2%; 15 of 290) affecting data from 25% (94 of 376) of the patients could not be unambiguously corrected. For example, 2.8% (8 of 290) of the variant entries from 23.1% (87 of 376) of patients did not state the variant (eg, variant not stated; no variant stated; BRCA2).

Overview of BRCA2 variant assessment. Subway chart illustrating the steps in data collection, curation, and harmonization of variant designation (green line). Variants were then separated according to origin as somatic (yellow line) or germline (blue line) and stratified by variant type or clinical significance. BC, breast cancer.

Next, after data harmonization (see the Methods section) we identified several instances in which the same variant had correct designations but different formats: 21.2% (62 of 293) for BRCA1 and 21.8% (60 of 275) for BRCA2 (Figs 1 and 2). We then consolidated these variants into a single entry (Data Supplement, Tables S6 and S7). Harmonization and consolidation are labor-intensive but can be automated to some extent (code availability). After harmonization, there were 231 unique BRCA1 variants from 318 patients with breast cancer and 215 unique BRCA2 variants from 285 patients with breast cancer.

In summary, although there was a high rate of incorrect variant designations, many could be corrected. However, data from approximately 45% of patients with a BRCA1 variant and 43% with a BRCA2 variant did not include the specific variant found.

### Germline and Somatic Variants

Somatic variants make up 30.1% and 10% of the unique variants reported for BRCA1 and BRCA2, respectively (Figs 1 and 2). For BRCA1, data loss from entries that could not be unambiguously corrected was comparable between germline and somatic variant entries (8% v 7%). For BRCA2, data loss was notably higher for somatic (10%) than for germline (5%) variant entries.

### Classification Reassessment

After harmonization and consolidation of variant designations, we first determined the rate of clinically meaningful internal conflicts (when the same variant received two or more different calls for significance). We did not consider not reported as a conflict or when the difference was between likely pathogenic and pathogenic or between likely benign and benign. For BRCA1, there was only one conflict in which the variant received two calls: as benign (B) and VUS (Data Supplement, Table S6). For BRCA2, there were two conflicts between pathogenic and VUS (Data Supplement, Table S7).

Next, for every variant, we examined and reassessed the classification (Significance) of each variant using interpretations from ClinVar. For BRCA1 and BRCA2, the most frequent reason for reclassification was missing significance (not reported) in the original entry representing 49.3% (114 of 231) and 47.9% (103 of 215) of the BRCA1 and BRCA2 variants, respectively (Figs 3A and 3B). This was followed by the updating of variants that were VUS at the time of data entry and have since then been classified as likely benign/benign (LB/B), likely pathogenic/pathogenic (LP/P), or have conflicting ClinVar interpretations. Only one BRCA1 (p.(Ile925Leu)) and one BRCA2 p.(Gln347Arg) variant had consequential reclassification from LP/P to LB/B. The original incorrect classifications were most likely due to incorrect manual data entry.

Reassessment of clinical significance. Sankey diagram showing the results of reassessment for (A) BRCA1 and (B) BRCA2. Top level represents the number of variants in each significance class. Middle level represents the reclassification of variants according to ClinVar. Variants receiving VUS or conflicting assertions were further assessed using functional data to assign ACMG evidence criteria (bottom level). Note that totals are higher than the total number of unique variants because some variants were recorded with two or more different significance assertions. ACMG, American College of Medical Geneticists; VUS, variant of uncertain significance.

### Functional Assessment of BRCA1 and BRCA2 Missense Variants

To improve the information for variants with a ClinVar conflicting classification or VUS designation, we retrieved integrated functional data for BRCA1 and BRCA2 missense variants.31,37 Functional data provided ACMG evidence criteria for 18 of 42 BRCA1 VUS and 20 of 62 variants with conflicting interpretation (Fig 3A; Data Supplement, Table S8). For BRCA2, functional data provided evidence criteria for 1 of 22 VUS and 6 of 72 variants with conflicting interpretation (Fig 3B; Data Supplement, Table S9).

### Functional Assays for BRCA1 Missense Variants

Next, we tested eight variants for which there were no functional data using the validated BRCA1 transcription assay (Fig 4A).31 All eight variants tested expressed levels comparable with the wild type (Fig 4B; Data Supplement, Fig S2). Only variant p.(Gly1743Ala) displayed reduced activity when compared with the wild type or a known benign variant (p.(Ser1613Gly)) but not as low as the control pathogenic variant (p.(Met1775Arg)). All other variants had activity comparable with or higher than the wild type (Fig 4A). All tested variants, with the exception of p.(Gly1743Ala), which received an indeterminate result, were assigned an ACMG/AMP evidence criteria code, BS3 code, corresponding to strong evidence for benignity (Data Supplement, Tables S10-S12; see the Data Supplement, Fig S3, for VarCall analysis).

Functional assessment of BRCA1 VUS. (A) Transcription activity of BRCA1 missense VUS expressed as a fusion of the BRCA1 C-terminal region (aa 1,396-1,863) to the GAL4 DBD shown as a percentage of wild-type activity. The wild-type control is represented with a black bar. The known benign p.(Ser1613Gly) and pathogenic p.(Met1775Arg) variants are represented by blue and red bars, respectively. (B) GAL4 DBD-BRCA1 protein levels in HEK293FT cells. Immunoblot using anti-GAL4DBD and anti-β-actin. Lines indicate where irrelevant lanes were removed (see the unedited blot in the Data Supplement, Fig S1). DBD, DNA-binding domain; VUS, variant of uncertain significance.

### Prevalence Estimates Stratified by Variant Type and Clinical Significance

Approximately 5% and 3% of tested patients were carriers of a germline or somatic variant in BRCA1 and BRCA2, respectively (Data Supplement, Table S13). However, fewer than 1% could be verified as likely pathogenic/pathogenic germline variants, since not reported or uncertain were listed as clinical significance for the majority of variants found for both BRCA1 and BRCA2 (Data Supplement, Tables S2 and S3).

After the elimination of variants that could not be unambiguously identified and reassessment of variant clinical significance classifications, approximately 2.2% of patients tested were carriers of LP/P variants in BRCA1 or BRCA2 (Data Supplement, Table S13). Eighty-eight patients (0.71%) had a germline finding of LP/P in BRCA1. One hundred patients (0.8%) had a germline finding of LP/P in BRCA2. The combined prevalence of germline LP/P variants in BRCA1 and BRCA2 was 1.5%. Large rearrangements constituted 4.8% of BRCA1 and 1.9% of BRCA2 variants.

### DISCUSSION

Routinely collected data and real-world databases hold great promise as a platform for observational studies but suffer from several weaknesses including limited raw data availability.18 Here, we undertook a comprehensive assessment of routinely collected data on germline and somatic genetic testing in 12,430 individuals affected with breast cancer to identify challenges and barriers to the rigorous use of germline genetic testing data.

Our study showed that the data set of patients with breast cancer who underwent testing for germline and somatic BRCA1 and BRCA2 pathogenic variants was, in general, representative of the racial and ethnic distribution of the US population albeit with under-representation of Hispanic, American Indian and Alaska Native, Asian, and Native Hawaiian and Other Pacific Islander individuals.21 After harmonization and consolidation of variant designations and reassessment of clinical significance, prevalence estimates of germline PVs in BRCA1 and BRCA2 were 0.7% and 0.8%, respectively. These numbers are likely to be an underestimate because of data loss of variants that are likely to be pathogenic but could not be unambiguously identified and were excluded from the analysis. Nevertheless, these estimates are in line with those obtained in the population-based case-control CARRIERS study of 32,247 women with breast cancer, where the prevalence was 0.85% for BRCA1 and 1.29% for BRCA2 in non-Hispanic White patients and slightly higher (1.04% and 1.80%, respectively) in Black patients with breast cancer.38,39 The higher rate of large rearrangements in BRCA1 than in BRCA2, presumably because of the number of recombinogenic Alu repeats present in BRCA1,40 is consistent with previous studies.27,41

Incorrect variant designations represented an important cause of data loss. While some variants could not be corrected and were then excluded from analysis, many could be corrected although automated correction could only partially resolve the issue. Some of these issues are expected to be mitigated with the widespread adoption of standardized nomenclature (HGVS) and risk assessment criteria (ACMG/AMP guidelines, ClinGen). However, variant designations that could not be unambiguously corrected led to exclusion of 45.3% (263 of 581) of BRCA1 cases and 36.7% (138 of 376) of BRCA2 cases, a sizable portion of the data set. For analyses that do not stratify by variant or variant type, the impact would be smaller as one could assume that those cases would be carriers of pathogenic variants. However, the presence of clinically inconsequential variants (eg, synonymous variants or known benign) in the data set suggests that such an assumption might be incorrect, and caution should be exercised.

Missing (not reported) clinical significance was also a significant barrier for data usage with almost half of BRCA1 and BRCA2 variants having significance not reported. The underlying reason for the missing variant designations and clinical significance was data not being originally listed in the electronic medical record (EMR) and was that data were abstracted from clinical notes instead of original testing reports. The issue of missing data could be mitigated by more extensive training of personnel entering the data, better integration with other software from which data are derived, recovery of original testing reports, and additional safeguards during manual data entry (eg, pull-down menus instead of free entry).

The inability to identify the correct designation for several variants in the database has no direct clinical implications as it does not mean that the patient and the patient’s health care provider did not have access to the correct information. Rather, it is because many of these variants were obtained via manual abstraction of medical records. In many cases, BRCA results were captured in a clinical note without a specific variant or the variant was not captured in a recognizable format.

It is a limitation of our study that we were not able to determine if rates of missing/incorrect data differed across the network (eg, community setting v academic centers). Obtaining discrete variant results at scale in the community health system remains a challenge because of workflow and data integration gaps. Discrete results generated from reference laboratories are not consistently transmitted to the health system EMR in a structured, queryable format (eg, PDF results are faxed or e-mailed from the laboratory and scanned into the EMR by the clinical team). In addition, manual abstraction can be tedious and prone to human error. Technical interventions, including natural language processing and harmonization of variant designations, along with a thoughtful and disciplined approach to manual abstraction can mitigate these challenges and yield more structured result data. Gaps identified in this study can be addressed by standardizing the manual abstraction process using Natural Language Processes to extract variant data.42,43

Classification (significance) reassessment has emerged as an important aspect to keep information up to date and to change clinical management of carriers.44 Reassessment of variant clinical significance led to a significant increase in the fraction of pathogenic variants, from 21% to 44% for BRCA1 and from 33% to 49% in BRCA1 and BRCA2, respectively. Similarly, benign variants increased from 1.6% to 9.8% in BRCA1 and from 0.8% to 4.6% in BRCA2. Functional assessment, using previously published data and direct tests, resulted in ACMG evidence criteria that can be used to resolve 53 additional variants. Therefore, periodic reassessments would allow for the updating of clinical significance as more data accumulate.

In conclusion, we have identified causes of data loss and provide strategies to minimize loss and data inaccuracies and showed that real-world data sets have the potential to provide a resource for the analysis of clinical outcomes of patients with breast cancer who are carriers of BRCA pathogenic variants.



# SUPPLEMENTAL FILE 1: cci-8-e2300251.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)