# MAIN TEXT

## Single nucleotide variants in lung cancer

### Abstract

Germline genetic variants, including single-nucleotide variants (SNVs) and copy number variants (CNVs), account for interpatient heterogeneity. In the past several decades, genome-wide association studies (GWAS) have identified multiple lung cancer-associated SNVs in Caucasian and Chinese populations. These variants either reside within coding regions and change the structure and function of cancer-related proteins or reside within non-coding regions and alter the expression level of cancer-related proteins. The variants can be used not only for cancer risk assessment and prevention but also for the development of new therapies. In this review, we discuss the lung cancer-associated SNVs identified to date, their contributions to lung tumorigenesis and prognosis, and their potential use in predicting prognosis and implementing therapeutic strategies.

### Introduction

Lung cancer is the most frequent cause of cancer-related death worldwide, with 5-year survival rates varying from 4–17%.1 Small cell lung carcinoma (SCLC) and non-small cell lung carcinoma (NSCLC) are the two histopathologic types of lung cancer. NSCLC consists of adenocarcinoma (ADC), squamous cell carcinoma (SCC), adenosquamous carcinoma (ASC), and large cell carcinoma (LCC). The alteration in oncogenic driver genes is one of the important causes of lung cancer. Tumor-suppressor gene TP53 is the most frequently mutated gene (approximately 40%) in all types of lung cancer.2 Lung cancer subtypes also harbor specific mutations and variants in other genes, such as EGFR and KRAS in ADC,3
CDKN2A and RB1 in SCC,4 and RB1, PTEN, and MYC in SCLC.2

Pathogenic variants in the germline of an individual, which are heritable, are called germline variants. A growing amount of evidence has shown the important role of germline variations in lung cancer initiation and progression. Genetic variations include single nucleotide variants (SNVs), insertions, deletions, structural variants, and repeat variations.5 If the minor allele frequency (MAF) of an SNV is more than 1%, a variant is called a polymorphism. Single nucleotide polymorphisms (SNPs) are the most common genetic variation type among all germline genetic variations, accounting for 90% of all polymorphisms in the genome.

### SNVs in different regions

SNVs can reside in coding regions and non-coding regions of the genome. Approximately 10% of disease-associated SNVs are located in coding regions and 90% are located in non-coding regions.6 SNVs in different regions modulate diseases through different mechanisms.

Genetic variants in a coding region change the sequence of amino acids and influence protein function. Germline variants in lung cancer driver genes such as EGFR, KRAS, and P53 contribute greatly to tumorigenesis and progression. The epidermal growth factor receptor (EGFR) tyrosine kinase domain is encoded by exons 18–24. Over 90% of the known EGFR mutations in lung cancer are in exons 19–21. EGFR-K757R is the most common EGFR germline mutation in Chinese lung cancer patients. The mutation of K757R is associated with the response of lung cancer to chemotherapy. K757R and exon 19del + K757R show similar sensitivity to icotinib and osimertinib, whereas exon 19del + K757R is more sensitive to afatinib and gefitinib than K757R.7 The less commonly observed EGFR germline variants in coding exons have been associated with susceptibility or treatment response of lung cancer. For example, EGFR V834L and V843I are associated with susceptibility to lung adenocarcinoma (LUAD), and R776H, V843I, L858R, and P848L are associated with squamous cell lung cancer.8 Synonymous mutations also affect lung cancer in some cases. The common EGFR Q787Q polymorphism showed significant protective effects on the overall survival of patients with EGFR-mutant stage IV LUAD treated with EGFR tyrosine kinase inhibitors (TKIs).9

RAS proteins are a family of small GTPases that play critical roles in multiple cellular signaling pathways, such as the RAS–mitogen-activated protein kinase (MAPK) pathway. Approximately 83% of KRAS somatic mutations are found at G12, followed by G13 (14%) and Q61 (2%). Germline KRAS mutations have been observed at numerous locations, including K5N, V14I, Q22R, Q22E, P34R, P34L, T58I, G60R, E153V, and F156L, and are associated with lung cancer risk.10,11 Germline variants in codons 12, 13, or 61 in KRAS are rarely found.

The P53 tumor suppressor gene plays a central role in many cellular processes, such as DNA repair and apoptosis. The P53 V157D mutation was identified in a family with hereditary lung cancer syndrome. Further mechanistic study showed that the P53 V157D variant promotes lung cancer cell proliferation.12 The P53 P72A germline variant in the P53 coding region is also associated with lung cancer.13

In addition to hotpot oncogene mutation, germline variants in other candidate susceptibility genes have also been widely studied. Eleven lung cancer-associated germline variants are located in genes encoding components of the growth hormone/insulin-like growth factor (GH-IGF) pathway, including rs7214723 in CAMKK1 (E375G), rs6964587 in AKAP9 (M463I), and rs6183 in GHR (P495T), and genes in the DNA damage-response pathway, including rs11571833 in BRCA2 (K3326X) and rs28360135 in XRCC4 (I137T).14 Rare deleterious variants in DNA repair pathway genes, such as rs77187983 in EHBP1 (D590V), rs11571833 in BRCA2 (K326X), and rs752672077 in MPZL2 (I24M), are also associated with lung cancer risk.15 Germline mutations in PARK2 correlate with impaired mitophagy and increase the death of lung cancer cells. Mechanically, A46T Parkin results in inability to translocate to mitochondria and recruit downstream mitophagic regulators, such as optineurin (OPTN) and transcription factor EB (TFEB). Besides, N254S and R275W Parkin display slower mitochondrial translocation than WT Parkin.16

Whole-exome sequencing revealed that SCLC frequently harbors germline pathogenic variants in RAD51D (Q62X), CHEK1 (Q346X, R379X), BRCA2 (R1699W, Y2215X), and MUTYH (G396D, V493F, Y179C), which are significantly associated with recurrence-free survival after platinum-based chemotherapy.17

A significant number of lung cancer-associated SNVs have been identified in gene promoters, introns, and intergenic regions. Promoter SNVs frequently alter the binding affinity of transcription factors, subsequently influencing the transcriptional regulation of key genes involved in lung tumorigenesis. Intron or intergenic SNVs either modulate the activity of cis-regulatory elements (CREs), such as enhancers and silencers, usually by affecting transcription factor binding or changing the function of non-coding RNAs (ncRNAs).

A non-coding SNV can exert its functional effects through various mechanisms: regulating the transcription of neighboring genes, regulating the transcription of distant genes, or regulating genes located on other chromosomes. This suggests the complexity of eukaryotic genomic assembly. rs3769201 and rs722864, which are located in the introns of ZAK, are associated with decreased ZAK mRNA expression and reduced lung cancer risk.18 rs3117582, located in intron 1 of BAT3, is involved in the regulation of p53 acetylation in response to DNA damage; and rs3131379, located in intron 10 of MSH5, is involved in DNA mismatch repair and regulates lung cancer susceptibility.19 Whether these two SNP loci regulate BAT3 or MSH5 is not clear. Growing evidence has shown that a non-coding SNP may bypass nearby genes to regulate a gene that is located far away. For example, rs402710, which is located in the last intron of CLPTM1L, physically interacts with the TERT promoter by looping out the intervening sequences, regulating TERT gene expression and lung tumorigenesis20 (Fig. 1).Fig. 1Schematic representation of the mechanism by non-coding SNV rs402710, which is located in the last intron of CLPTM1L, and regulates TERT via physical interactions with the TERT promoter. SNV: Single nucleotide variant.Fig 1:

Schematic representation of the mechanism by non-coding SNV rs402710, which is located in the last intron of CLPTM1L, and regulates TERT via physical interactions with the TERT promoter. SNV: Single nucleotide variant.

Although intrachromosomal regulation is a frequently reported mechanism used by non-coding SNPs to regulate disease, some lung cancer-associated SNP loci exhibit trans-effects. For example, rs1663689, which is located in the intergenic region in chromosome 10p1.4, regulates lung cancer susceptibility and outcome through regulation of adhesion G protein-coupled receptor G6 (ADGRG6), which is located chromosome 6, through interchromosomal interaction21 (Fig. 2).Fig. 2Schematic representation of the mechanism of the intergenic SNV rs1663689 in regulating ADGRG6 by interchromosomal interaction. SNV: Single nucleotidevariant.Fig 2:

Schematic representation of the mechanism of the intergenic SNV rs1663689 in regulating ADGRG6 by interchromosomal interaction. SNV: Single nucleotidevariant.

A non-coding SNP usually exerts its function by changing the binding affinity of transcription factors. For example, rs2853677 is located within the Snail1 binding site in a TERT enhancer. The enhancer increases TERT transcription when juxtaposed to the TERT promoter. rs2853677-T results in the binding of Snail1 to the enhancer and disrupting enhancer-promoter colocalization, which subsequently silences TERT transcription (Fig. 3).22 SNP rs17079281-C, located in the DCBLD1 promoter, creates a YY1-binding site, resulting in decreased DCBLD1 expression and subsequent decreased cell proliferation.23 rs9399451 and rs9390123 reside within an enhancer region and influence the binding of POU2F1, which subsequently affects the promoter activity of PHACTR2-AS1 and PEX3 in lung cancer cell lines.24 rs4142441 is located in a MYC binding site in the OSER1-AS1 promoter region. The G allele of rs4142441 results a higher binding affinity of MYC. MYC binding suppresses the transcription of OSER1-AS1, and promotes tumor progression.25 A lung-specific p53-responsive enhancer of TNFRSF19 harbors three highly linked common SNPs (rs17336602, rs4770489, and rs34354770) and six p53 binding sequences either close to or located between the variations. The enhancer effectively protects normal lung cell lines against pulmonary carcinogen nicotine-derived nitrosamine ketone (NNK)-induced DNA damage and malignant transformation by upregulating TNFRSF19 through chromatin looping. These variations significantly weaken the enhancer activity by affecting the p53 response, especially when cells are exposed to NNK.26Fig. 3Schematic representation of rs2853677 changing the binding affinity of the Snail1 transcription factor.Fig 3:

Schematic representation of rs2853677 changing the binding affinity of the Snail1 transcription factor.

Most of the human genome is transcribed RNAs that do not encode proteins. These ncRNAs, including microRNAs (miRNAs) and long ncRNAs (lncRNAs), play crucial roles in regulating the initiation and progression of various cancers.27 Lung cancer-associated SNPs in ncRNAs have been identified. rs11614913 in hsa-mir-196a2 regulates its binding to the LSP1 3′UTR, which changes LSP1 expression and survival in individuals with NSCLC.28 rs10505477 in the lncRNA CASC8 is highly related to ADC risk in males and highly relevant to severe hematologic toxicity in NSCLC and gastrointestinal toxicity in SCLC after platinum-based chemotherapy.29 rs140618127, located in the lncRNA LOC146880, regulates binding between miR-539-5p and LOC146880, which modulates phosphorylation of enolase 1 (ENO1) and subsequent phosphorylation of phosphoinositide 3-kinase (PI3K) and Akt, and is associated with NSCLC susceptibility in the Chinese population.30 The risk T allele of rs12740674, located in the enhancer of miR-1262, reduces the expression level of miR-1262 in lung tissue through chromosomal looping and increases the expression levels of UNC-51-like kinase 1 (ULK1) and RAB3D, member RAS oncogene family, promoting lung cancer cell proliferation (Table 1).31Table 1Mechanisms of non-coding SNVs.Table 1:MechanismSNVsAlter transcriptional activity of promoters by cis- or trans-effects.rs40271020; rs166368921Influence the binding affinity of transcription factors.rs285367722; rs1707928123; rs939945124Influence the binding affinity of ncRNAs.rs1161491328; rs14061812730ncRNAs: Non-coding RNAs; SNVs: Single nucleotide variants.

Mechanisms of non-coding SNVs.

ncRNAs: Non-coding RNAs; SNVs: Single nucleotide variants.

### SNVs in coding regions

Genetic variants in a coding region change the sequence of amino acids and influence protein function. Germline variants in lung cancer driver genes such as EGFR, KRAS, and P53 contribute greatly to tumorigenesis and progression. The epidermal growth factor receptor (EGFR) tyrosine kinase domain is encoded by exons 18–24. Over 90% of the known EGFR mutations in lung cancer are in exons 19–21. EGFR-K757R is the most common EGFR germline mutation in Chinese lung cancer patients. The mutation of K757R is associated with the response of lung cancer to chemotherapy. K757R and exon 19del + K757R show similar sensitivity to icotinib and osimertinib, whereas exon 19del + K757R is more sensitive to afatinib and gefitinib than K757R.7 The less commonly observed EGFR germline variants in coding exons have been associated with susceptibility or treatment response of lung cancer. For example, EGFR V834L and V843I are associated with susceptibility to lung adenocarcinoma (LUAD), and R776H, V843I, L858R, and P848L are associated with squamous cell lung cancer.8 Synonymous mutations also affect lung cancer in some cases. The common EGFR Q787Q polymorphism showed significant protective effects on the overall survival of patients with EGFR-mutant stage IV LUAD treated with EGFR tyrosine kinase inhibitors (TKIs).9

RAS proteins are a family of small GTPases that play critical roles in multiple cellular signaling pathways, such as the RAS–mitogen-activated protein kinase (MAPK) pathway. Approximately 83% of KRAS somatic mutations are found at G12, followed by G13 (14%) and Q61 (2%). Germline KRAS mutations have been observed at numerous locations, including K5N, V14I, Q22R, Q22E, P34R, P34L, T58I, G60R, E153V, and F156L, and are associated with lung cancer risk.10,11 Germline variants in codons 12, 13, or 61 in KRAS are rarely found.

The P53 tumor suppressor gene plays a central role in many cellular processes, such as DNA repair and apoptosis. The P53 V157D mutation was identified in a family with hereditary lung cancer syndrome. Further mechanistic study showed that the P53 V157D variant promotes lung cancer cell proliferation.12 The P53 P72A germline variant in the P53 coding region is also associated with lung cancer.13

In addition to hotpot oncogene mutation, germline variants in other candidate susceptibility genes have also been widely studied. Eleven lung cancer-associated germline variants are located in genes encoding components of the growth hormone/insulin-like growth factor (GH-IGF) pathway, including rs7214723 in CAMKK1 (E375G), rs6964587 in AKAP9 (M463I), and rs6183 in GHR (P495T), and genes in the DNA damage-response pathway, including rs11571833 in BRCA2 (K3326X) and rs28360135 in XRCC4 (I137T).14 Rare deleterious variants in DNA repair pathway genes, such as rs77187983 in EHBP1 (D590V), rs11571833 in BRCA2 (K326X), and rs752672077 in MPZL2 (I24M), are also associated with lung cancer risk.15 Germline mutations in PARK2 correlate with impaired mitophagy and increase the death of lung cancer cells. Mechanically, A46T Parkin results in inability to translocate to mitochondria and recruit downstream mitophagic regulators, such as optineurin (OPTN) and transcription factor EB (TFEB). Besides, N254S and R275W Parkin display slower mitochondrial translocation than WT Parkin.16

Whole-exome sequencing revealed that SCLC frequently harbors germline pathogenic variants in RAD51D (Q62X), CHEK1 (Q346X, R379X), BRCA2 (R1699W, Y2215X), and MUTYH (G396D, V493F, Y179C), which are significantly associated with recurrence-free survival after platinum-based chemotherapy.17

### SNVs in non-coding regions

A significant number of lung cancer-associated SNVs have been identified in gene promoters, introns, and intergenic regions. Promoter SNVs frequently alter the binding affinity of transcription factors, subsequently influencing the transcriptional regulation of key genes involved in lung tumorigenesis. Intron or intergenic SNVs either modulate the activity of cis-regulatory elements (CREs), such as enhancers and silencers, usually by affecting transcription factor binding or changing the function of non-coding RNAs (ncRNAs).

A non-coding SNV can exert its functional effects through various mechanisms: regulating the transcription of neighboring genes, regulating the transcription of distant genes, or regulating genes located on other chromosomes. This suggests the complexity of eukaryotic genomic assembly. rs3769201 and rs722864, which are located in the introns of ZAK, are associated with decreased ZAK mRNA expression and reduced lung cancer risk.18 rs3117582, located in intron 1 of BAT3, is involved in the regulation of p53 acetylation in response to DNA damage; and rs3131379, located in intron 10 of MSH5, is involved in DNA mismatch repair and regulates lung cancer susceptibility.19 Whether these two SNP loci regulate BAT3 or MSH5 is not clear. Growing evidence has shown that a non-coding SNP may bypass nearby genes to regulate a gene that is located far away. For example, rs402710, which is located in the last intron of CLPTM1L, physically interacts with the TERT promoter by looping out the intervening sequences, regulating TERT gene expression and lung tumorigenesis20 (Fig. 1).Fig. 1Schematic representation of the mechanism by non-coding SNV rs402710, which is located in the last intron of CLPTM1L, and regulates TERT via physical interactions with the TERT promoter. SNV: Single nucleotide variant.Fig 1:

Schematic representation of the mechanism by non-coding SNV rs402710, which is located in the last intron of CLPTM1L, and regulates TERT via physical interactions with the TERT promoter. SNV: Single nucleotide variant.

Although intrachromosomal regulation is a frequently reported mechanism used by non-coding SNPs to regulate disease, some lung cancer-associated SNP loci exhibit trans-effects. For example, rs1663689, which is located in the intergenic region in chromosome 10p1.4, regulates lung cancer susceptibility and outcome through regulation of adhesion G protein-coupled receptor G6 (ADGRG6), which is located chromosome 6, through interchromosomal interaction21 (Fig. 2).Fig. 2Schematic representation of the mechanism of the intergenic SNV rs1663689 in regulating ADGRG6 by interchromosomal interaction. SNV: Single nucleotidevariant.Fig 2:

Schematic representation of the mechanism of the intergenic SNV rs1663689 in regulating ADGRG6 by interchromosomal interaction. SNV: Single nucleotidevariant.

A non-coding SNP usually exerts its function by changing the binding affinity of transcription factors. For example, rs2853677 is located within the Snail1 binding site in a TERT enhancer. The enhancer increases TERT transcription when juxtaposed to the TERT promoter. rs2853677-T results in the binding of Snail1 to the enhancer and disrupting enhancer-promoter colocalization, which subsequently silences TERT transcription (Fig. 3).22 SNP rs17079281-C, located in the DCBLD1 promoter, creates a YY1-binding site, resulting in decreased DCBLD1 expression and subsequent decreased cell proliferation.23 rs9399451 and rs9390123 reside within an enhancer region and influence the binding of POU2F1, which subsequently affects the promoter activity of PHACTR2-AS1 and PEX3 in lung cancer cell lines.24 rs4142441 is located in a MYC binding site in the OSER1-AS1 promoter region. The G allele of rs4142441 results a higher binding affinity of MYC. MYC binding suppresses the transcription of OSER1-AS1, and promotes tumor progression.25 A lung-specific p53-responsive enhancer of TNFRSF19 harbors three highly linked common SNPs (rs17336602, rs4770489, and rs34354770) and six p53 binding sequences either close to or located between the variations. The enhancer effectively protects normal lung cell lines against pulmonary carcinogen nicotine-derived nitrosamine ketone (NNK)-induced DNA damage and malignant transformation by upregulating TNFRSF19 through chromatin looping. These variations significantly weaken the enhancer activity by affecting the p53 response, especially when cells are exposed to NNK.26Fig. 3Schematic representation of rs2853677 changing the binding affinity of the Snail1 transcription factor.Fig 3:

Schematic representation of rs2853677 changing the binding affinity of the Snail1 transcription factor.

Most of the human genome is transcribed RNAs that do not encode proteins. These ncRNAs, including microRNAs (miRNAs) and long ncRNAs (lncRNAs), play crucial roles in regulating the initiation and progression of various cancers.27 Lung cancer-associated SNPs in ncRNAs have been identified. rs11614913 in hsa-mir-196a2 regulates its binding to the LSP1 3′UTR, which changes LSP1 expression and survival in individuals with NSCLC.28 rs10505477 in the lncRNA CASC8 is highly related to ADC risk in males and highly relevant to severe hematologic toxicity in NSCLC and gastrointestinal toxicity in SCLC after platinum-based chemotherapy.29 rs140618127, located in the lncRNA LOC146880, regulates binding between miR-539-5p and LOC146880, which modulates phosphorylation of enolase 1 (ENO1) and subsequent phosphorylation of phosphoinositide 3-kinase (PI3K) and Akt, and is associated with NSCLC susceptibility in the Chinese population.30 The risk T allele of rs12740674, located in the enhancer of miR-1262, reduces the expression level of miR-1262 in lung tissue through chromosomal looping and increases the expression levels of UNC-51-like kinase 1 (ULK1) and RAB3D, member RAS oncogene family, promoting lung cancer cell proliferation (Table 1).31Table 1Mechanisms of non-coding SNVs.Table 1:MechanismSNVsAlter transcriptional activity of promoters by cis- or trans-effects.rs40271020; rs166368921Influence the binding affinity of transcription factors.rs285367722; rs1707928123; rs939945124Influence the binding affinity of ncRNAs.rs1161491328; rs14061812730ncRNAs: Non-coding RNAs; SNVs: Single nucleotide variants.

Mechanisms of non-coding SNVs.

ncRNAs: Non-coding RNAs; SNVs: Single nucleotide variants.

### SNVs in cis-regulatory elements

A non-coding SNV can exert its functional effects through various mechanisms: regulating the transcription of neighboring genes, regulating the transcription of distant genes, or regulating genes located on other chromosomes. This suggests the complexity of eukaryotic genomic assembly. rs3769201 and rs722864, which are located in the introns of ZAK, are associated with decreased ZAK mRNA expression and reduced lung cancer risk.18 rs3117582, located in intron 1 of BAT3, is involved in the regulation of p53 acetylation in response to DNA damage; and rs3131379, located in intron 10 of MSH5, is involved in DNA mismatch repair and regulates lung cancer susceptibility.19 Whether these two SNP loci regulate BAT3 or MSH5 is not clear. Growing evidence has shown that a non-coding SNP may bypass nearby genes to regulate a gene that is located far away. For example, rs402710, which is located in the last intron of CLPTM1L, physically interacts with the TERT promoter by looping out the intervening sequences, regulating TERT gene expression and lung tumorigenesis20 (Fig. 1).Fig. 1Schematic representation of the mechanism by non-coding SNV rs402710, which is located in the last intron of CLPTM1L, and regulates TERT via physical interactions with the TERT promoter. SNV: Single nucleotide variant.Fig 1:

Schematic representation of the mechanism by non-coding SNV rs402710, which is located in the last intron of CLPTM1L, and regulates TERT via physical interactions with the TERT promoter. SNV: Single nucleotide variant.

Although intrachromosomal regulation is a frequently reported mechanism used by non-coding SNPs to regulate disease, some lung cancer-associated SNP loci exhibit trans-effects. For example, rs1663689, which is located in the intergenic region in chromosome 10p1.4, regulates lung cancer susceptibility and outcome through regulation of adhesion G protein-coupled receptor G6 (ADGRG6), which is located chromosome 6, through interchromosomal interaction21 (Fig. 2).Fig. 2Schematic representation of the mechanism of the intergenic SNV rs1663689 in regulating ADGRG6 by interchromosomal interaction. SNV: Single nucleotidevariant.Fig 2:

Schematic representation of the mechanism of the intergenic SNV rs1663689 in regulating ADGRG6 by interchromosomal interaction. SNV: Single nucleotidevariant.

A non-coding SNP usually exerts its function by changing the binding affinity of transcription factors. For example, rs2853677 is located within the Snail1 binding site in a TERT enhancer. The enhancer increases TERT transcription when juxtaposed to the TERT promoter. rs2853677-T results in the binding of Snail1 to the enhancer and disrupting enhancer-promoter colocalization, which subsequently silences TERT transcription (Fig. 3).22 SNP rs17079281-C, located in the DCBLD1 promoter, creates a YY1-binding site, resulting in decreased DCBLD1 expression and subsequent decreased cell proliferation.23 rs9399451 and rs9390123 reside within an enhancer region and influence the binding of POU2F1, which subsequently affects the promoter activity of PHACTR2-AS1 and PEX3 in lung cancer cell lines.24 rs4142441 is located in a MYC binding site in the OSER1-AS1 promoter region. The G allele of rs4142441 results a higher binding affinity of MYC. MYC binding suppresses the transcription of OSER1-AS1, and promotes tumor progression.25 A lung-specific p53-responsive enhancer of TNFRSF19 harbors three highly linked common SNPs (rs17336602, rs4770489, and rs34354770) and six p53 binding sequences either close to or located between the variations. The enhancer effectively protects normal lung cell lines against pulmonary carcinogen nicotine-derived nitrosamine ketone (NNK)-induced DNA damage and malignant transformation by upregulating TNFRSF19 through chromatin looping. These variations significantly weaken the enhancer activity by affecting the p53 response, especially when cells are exposed to NNK.26Fig. 3Schematic representation of rs2853677 changing the binding affinity of the Snail1 transcription factor.Fig 3:

Schematic representation of rs2853677 changing the binding affinity of the Snail1 transcription factor.

### SNVs in ncRNAs

Most of the human genome is transcribed RNAs that do not encode proteins. These ncRNAs, including microRNAs (miRNAs) and long ncRNAs (lncRNAs), play crucial roles in regulating the initiation and progression of various cancers.27 Lung cancer-associated SNPs in ncRNAs have been identified. rs11614913 in hsa-mir-196a2 regulates its binding to the LSP1 3′UTR, which changes LSP1 expression and survival in individuals with NSCLC.28 rs10505477 in the lncRNA CASC8 is highly related to ADC risk in males and highly relevant to severe hematologic toxicity in NSCLC and gastrointestinal toxicity in SCLC after platinum-based chemotherapy.29 rs140618127, located in the lncRNA LOC146880, regulates binding between miR-539-5p and LOC146880, which modulates phosphorylation of enolase 1 (ENO1) and subsequent phosphorylation of phosphoinositide 3-kinase (PI3K) and Akt, and is associated with NSCLC susceptibility in the Chinese population.30 The risk T allele of rs12740674, located in the enhancer of miR-1262, reduces the expression level of miR-1262 in lung tissue through chromosomal looping and increases the expression levels of UNC-51-like kinase 1 (ULK1) and RAB3D, member RAS oncogene family, promoting lung cancer cell proliferation (Table 1).31Table 1Mechanisms of non-coding SNVs.Table 1:MechanismSNVsAlter transcriptional activity of promoters by cis- or trans-effects.rs40271020; rs166368921Influence the binding affinity of transcription factors.rs285367722; rs1707928123; rs939945124Influence the binding affinity of ncRNAs.rs1161491328; rs14061812730ncRNAs: Non-coding RNAs; SNVs: Single nucleotide variants.

Mechanisms of non-coding SNVs.

ncRNAs: Non-coding RNAs; SNVs: Single nucleotide variants.

### SNVs and lung cancer susceptibility

Over the past decade, several genome-wide association studies (GWAS) focused on cancer susceptibility have been performed. To date, 51 lung cancer-associated SNP loci have been identified,32 and a substantial proportion of these loci are specific to different subgroups in terms of histological subtype, smoking status, and ancestry.33

Genomic heterogeneity is associated with different histopathological types of lung cancer. A study by Dai et al34 identified 19 SNP loci that were significantly associated with NSCLC risk in a Chinese population. Among these variants, rs17038564 (P = 1.87 × 10–8), rs35201538 (P = 1.99 × 10–8), and rs77468143 (P = 7.48 × 10–12) were significant in the LUAD subgroup, whereas rs4573350 was specific for SCC. SNPs can be verified through different cohorts. Amos et al35 confirmed the association of rs77468143 with LUADs in a European population. The authors also identified additional seven SNPs (rs13080835, rs7705526, rs4236709, rs885518, rs11591710, rs1056562, and rs41309931) that were associated with LUAD and three SNPs (rs116822326, rs7953330, and rs17879961) that were associated with SCC.35 rs3134615, located in the 3′UTR of MYCL1, is associated with an increased risk of SCLC.36 The identification of subtype-specific associations of genetic variants indicates that the genetic architecture of lung cancer varies markedly among LUAD, SCC, and SCLC (Table 2).Table 2SNVs and lung cancer susceptibility in different subgroups.Table 2:SNVsRegionGeneDiseaseEffect alleleReference alleleOR (95% CI)P valueReferencesrs170385642p14AFTPHLUADGA1.15 (1.10–1.21)1.87 × 10–834rs352015389p13.3AQP3LUADCT1.10 (1.06–1.13)1.99 × 10–8rs7746814315q21.1SECISBP2LLUADTG1.14 (1.10–1.18)7.48 × 10–12rs45733509q33.2DAB21PLUSCTC1.13 (1.09–1.18)3.23 × 10–9rs7746814315q21.1SECISBP2LLUADGT0.90 (0.83–0.89)1.7 × 10–1635rs130808353q28TP63LUADTG0.90 (0.87–0.92)7.5 × 10–12rs77055265p15.33TERTLUADAC1.30 (1.21–1.29)3.8 × 10–35rs42367098p12NRG1LUADGA1.10 (1.09–1.18)1.3 × 10–10rs8855189p21.3CDNK2ALUADGA1.20 (1.11–1.23)9.96 × 10–10rs1159171010q24.3OBFC1LUADCA1.20 (1.11–1.22)6.3 × 10–11rs105656211q23.3AMICA1LUADTC1.10 (1.07–1.14)2.8 × 10–10rs4130993120q13.33RTEL1LUADTG1.20 (1.11–1.23)1.3 × 10–9rs1168223266p21.33MHCLUSCGA1.30 (1.19–1.32)3.8 × 10–9rs795333012p13.33RAD52LUSCCG0.90 (0.83–0.90)7.3 × 10–13rs1787996122q12.1CHEK2LUSCGA0.40 (0.32–0.52)5.7 × 10–13rs31346151p34.2MYCL1SCLCTG2.08 (1.39–3.12)0.000436CI: Confidence interval; LUAD: Lung adenocarcinoma; LUSC: Lung squamous cell carcinoma; OR: Odds (log-additive) ratio; SCLC: Small cell lung carcinoma; SNVs: Single nucleotide variants.

SNVs and lung cancer susceptibility in different subgroups.

CI: Confidence interval; LUAD: Lung adenocarcinoma; LUSC: Lung squamous cell carcinoma; OR: Odds (log-additive) ratio; SCLC: Small cell lung carcinoma; SNVs: Single nucleotide variants.

Smoking is the main cause of lung cancer, especially LUAD and SCLC, and nicotine is the most addictive component in tobacco. An analysis by Gabriel et al37 demonstrated that genetic variants influence the smoking behavior of individuals, which in turn influences their carcinogenic exposure and, consequently, their somatic mutation burden. The 15q25 susceptibility region, which contains six coding genes, including three cholinergic nicotine receptor genes (CHRNA3, CHRNA5, and CHRNB4) that exhibit independent effects on smoking behavior, contains multiple SNVs that are strongly associated with lung cancer.38, 39, 40, 41–42 The most robust lung cancer-associated SNP in 15q25 is rs16969968, which results in an amino acid change from aspartate to asparagine at position 398 of the nicotinic receptor α5 subunit protein sequence. rs16969968 predicts delayed smoking cessation and earlier age of lung cancer diagnosis.43–45 rs9439519 and rs4809957 are associated with cigarette smoking. rs4809957 interacts with smoking dose to contribute to lung cancer risk.46 rs6441286, rs17723637, and rs4751674 stratify lung cancer risk by smoking behavior. rs6441286 and rs17723637 variants increase the risk for lung cancer in ever-smokers, whereas the rs4751674 variant has a protective effect in ever-smokers compared with never-smokers.47 rs910083, located in the intron of DNMT3B, is associated with an increased risk of nicotine dependence. In International Lung Cancer Consortium data, the C allele at rs910083 was found to increase the risk of squamous cell lung carcinoma.48 rs34211819, located in an intron region of tensin-3 (TNS3), and rs1143149, located in an intron region of septin 7 (SEPT7), are significantly associated with the survival of NSCLC patients who are long-term former smokers. Both SNPs have significant interaction effects with years of smoking cessation. As the years of smoking cessation increase in long-term former smokers, the protective effect of rs34211819 and the detrimental effect of rs1143149 on survival are both enhanced.49

While the primary cause of lung cancer is smoking, approximately 25% of lung cancers worldwide occur in never-smokers.50 These cancers are caused by environmental risk factors, such as passive smoking, hormonal factors, and, most importantly, genetic factors.51, 52, 53 A study using WES and RNA-sequencing data for never-smoker LUADs sequenced by The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) found that pathogenic germline variants in cancer predisposition genes such as BRCA1, BRCA2, FANCG, FANCM, HMBS, MSH6, NF1, POLD1, TMEM127, and WRN are exclusively associated with lung cancer in never-smokers.54 A GWAS of lung cancer in never-smoking females in Asia identified three lung cancer susceptibility loci, at 10q25.2 (rs7086803), 6q22.2 (rs9387478), and 6p21.32 (rs2395185), with no evidence of an association of 15q25 with lung cancer.55 The TERT and CLPTM1L genes are located in 5p15. TERT is an established telomere maintenance locus. rs10936599 variants, which are located in the TERT coding region and associated with telomere length, were robustly associated with increased lung cancer risk among never-smoking women in Asia.56
CLPTM1L was identified by screening for cisplatin (CDDP) resistance-related genes and was found to induce apoptosis in CDDP-sensitive cells. rs402710 is located in intron 4 of the CLPTM1L gene and is associated with lung cancer susceptibility.41,57

Estrogen receptor (ER) gene SNPs such as rs7753153 and rs985192, located in ESR1, and rs3020450, located in ESR2, are associated with LUAD risk in never-smoking women.58 Moreover, rs12233719, a sex hormone regulation-related SNP in UGT2B7, is associated with NSCLC risk among never-smoking Chinese women.59 These studies suggest the important role of sex hormones in regulating lung tumorigenesis in never-smoking populations. rs11080466 and rs11663246, located in the intron of PIEZO2, show a significant association with NSCLC susceptibility of never smokers in Korean populations.60 rs4648127, located in the NFKB1 gene, was associated with lung cancer in the screening arm of the Prostate, Lung, Colon, and Ovarian (PLCO) Cancer Screening Trial.61

The genetic susceptibility differences in histopathological types and in accordance with smoking status may implicate distinct biological mechanisms of lung cancer and therapeutic strategies (Table 3).Table 3SNVs and lung cancer susceptibility with different smoking status.Table 3:SNVsRegionGeneEffect alleleReference alleleOR (95% CI)P valueReferencesSmoking rs1696996815q25CHRNA5AG1.405.47 × 10–4044 rs94395191p36.32AJAP1-NPHP4GA1.11 (1.06–1.16)3.65 × 10–646 rs480995720q13.2CYP24A1TC1.13 (1.08–1.18)1.20 × 10–8 rs64412863q25.33IL12A-AS1CA1.24 (1.13–1.37)1.16 × 10–547 rs177236379q31.2ZNF462GA1.36 (1.19–1.56)1.06 × 10–5 rs475167410q25.3AFAP1L2CT0.58 (0.46–0.74)1.07 × 10–5 rs91008320q11DNMT3BCA1.06 (1.04–1.07)3.70 × 10–848 rs342118197p12.3TNS3CA0.73 (0.66–0.81)3.90 × 10–949 rs11431497p14.2SEPT7CG1.36 (1.22–1.51)9.75 × 10–9Never-smoking rs708680310q25.2VTI1AAG1.28 (1.21–1.35)3.54 × 10–1855 rs93874786q22.2ROS1-DCBLD1AC0.85 (0.81–0.90)4.14 × 10–10 rs23951856p21.32HLATG1.17 (1.11–1.23)9.51 × 10–9 rs109365993q26.2TERCCT0.10 (0.08–0.11)2.54 × 10–3156 rs4027105p15.33CLPTM1LTC1.14 (1.09–1.19)5.00 × 10–841 rs77531536q25ESR1AG1.51 (1.17–1.95)0.000858 rs9851926q25ESR1CA1.31 (1.00–1.71)0.0233 rs302045014q23ESR2AC2.11 (1.01–4.42)0.0098 rs122337194q13.2UGT2B7TG1.54 (1.21–1.96)0.00659 rs1108046618p11.22PIEZO2CT0.66 (0.56–0.78)1.19 × 10–660 rs1166324618p11.22PIEZO2TC0.67 (0.57–0.79)2.40 × 10–6 rs46481274q24NFKB1TC0.56 (0.37–0.86)0.0261CI: Confidence interval; OR: Odds (log-additive) ratio; SNVs: Single nucleotide variants.

SNVs and lung cancer susceptibility with different smoking status.

CI: Confidence interval; OR: Odds (log-additive) ratio; SNVs: Single nucleotide variants.

Most of the mutations detected clinically are somatic mutations. Germline mutations can co-occur or be mutually exclusive with somatic cancer gene alterations.62 For example, the susceptibility variant rs36600 is associated with somatic mutations within ARID1A; the susceptibility variants rs2395185 and rs3817963 are associated with somatic alterations in the cell cycle pathway; and rs3817963 is associated with somatic alterations in the MAPK signaling pathway.63 rs2395185 is associated with elevated APOBEC mutagenesis.64 Some lung cancer risk-related SNPs were shown to influence genetic damage in coke oven workers exposed to polycyclic aromatic hydrocarbons (PAHs). Some SNP loci (rs1333040, rs1663689, and rs3813572) are associated with decreased micronuclei frequency, which is a biomarker of chromosomal damage, genome instability, and cancer risk that associates acquired mutations with genetic susceptibility.65 Recently, Peng et al66 identified 111 pathogenic or likely pathogenic (P/LP) germline mutations in 35 cancer genes in 106 of 1794 Chinese patients (5.91%). Chinese patients with germline mutations show different prevalence rates of somatic KRAS mutation, MET exon 14 skipping, and TP53 mutations compared with those without germline mutations.

### SNVs in different lung cancer subgroups

Genomic heterogeneity is associated with different histopathological types of lung cancer. A study by Dai et al34 identified 19 SNP loci that were significantly associated with NSCLC risk in a Chinese population. Among these variants, rs17038564 (P = 1.87 × 10–8), rs35201538 (P = 1.99 × 10–8), and rs77468143 (P = 7.48 × 10–12) were significant in the LUAD subgroup, whereas rs4573350 was specific for SCC. SNPs can be verified through different cohorts. Amos et al35 confirmed the association of rs77468143 with LUADs in a European population. The authors also identified additional seven SNPs (rs13080835, rs7705526, rs4236709, rs885518, rs11591710, rs1056562, and rs41309931) that were associated with LUAD and three SNPs (rs116822326, rs7953330, and rs17879961) that were associated with SCC.35 rs3134615, located in the 3′UTR of MYCL1, is associated with an increased risk of SCLC.36 The identification of subtype-specific associations of genetic variants indicates that the genetic architecture of lung cancer varies markedly among LUAD, SCC, and SCLC (Table 2).Table 2SNVs and lung cancer susceptibility in different subgroups.Table 2:SNVsRegionGeneDiseaseEffect alleleReference alleleOR (95% CI)P valueReferencesrs170385642p14AFTPHLUADGA1.15 (1.10–1.21)1.87 × 10–834rs352015389p13.3AQP3LUADCT1.10 (1.06–1.13)1.99 × 10–8rs7746814315q21.1SECISBP2LLUADTG1.14 (1.10–1.18)7.48 × 10–12rs45733509q33.2DAB21PLUSCTC1.13 (1.09–1.18)3.23 × 10–9rs7746814315q21.1SECISBP2LLUADGT0.90 (0.83–0.89)1.7 × 10–1635rs130808353q28TP63LUADTG0.90 (0.87–0.92)7.5 × 10–12rs77055265p15.33TERTLUADAC1.30 (1.21–1.29)3.8 × 10–35rs42367098p12NRG1LUADGA1.10 (1.09–1.18)1.3 × 10–10rs8855189p21.3CDNK2ALUADGA1.20 (1.11–1.23)9.96 × 10–10rs1159171010q24.3OBFC1LUADCA1.20 (1.11–1.22)6.3 × 10–11rs105656211q23.3AMICA1LUADTC1.10 (1.07–1.14)2.8 × 10–10rs4130993120q13.33RTEL1LUADTG1.20 (1.11–1.23)1.3 × 10–9rs1168223266p21.33MHCLUSCGA1.30 (1.19–1.32)3.8 × 10–9rs795333012p13.33RAD52LUSCCG0.90 (0.83–0.90)7.3 × 10–13rs1787996122q12.1CHEK2LUSCGA0.40 (0.32–0.52)5.7 × 10–13rs31346151p34.2MYCL1SCLCTG2.08 (1.39–3.12)0.000436CI: Confidence interval; LUAD: Lung adenocarcinoma; LUSC: Lung squamous cell carcinoma; OR: Odds (log-additive) ratio; SCLC: Small cell lung carcinoma; SNVs: Single nucleotide variants.

SNVs and lung cancer susceptibility in different subgroups.

CI: Confidence interval; LUAD: Lung adenocarcinoma; LUSC: Lung squamous cell carcinoma; OR: Odds (log-additive) ratio; SCLC: Small cell lung carcinoma; SNVs: Single nucleotide variants.

### SNVs and smoking status

Smoking is the main cause of lung cancer, especially LUAD and SCLC, and nicotine is the most addictive component in tobacco. An analysis by Gabriel et al37 demonstrated that genetic variants influence the smoking behavior of individuals, which in turn influences their carcinogenic exposure and, consequently, their somatic mutation burden. The 15q25 susceptibility region, which contains six coding genes, including three cholinergic nicotine receptor genes (CHRNA3, CHRNA5, and CHRNB4) that exhibit independent effects on smoking behavior, contains multiple SNVs that are strongly associated with lung cancer.38, 39, 40, 41–42 The most robust lung cancer-associated SNP in 15q25 is rs16969968, which results in an amino acid change from aspartate to asparagine at position 398 of the nicotinic receptor α5 subunit protein sequence. rs16969968 predicts delayed smoking cessation and earlier age of lung cancer diagnosis.43–45 rs9439519 and rs4809957 are associated with cigarette smoking. rs4809957 interacts with smoking dose to contribute to lung cancer risk.46 rs6441286, rs17723637, and rs4751674 stratify lung cancer risk by smoking behavior. rs6441286 and rs17723637 variants increase the risk for lung cancer in ever-smokers, whereas the rs4751674 variant has a protective effect in ever-smokers compared with never-smokers.47 rs910083, located in the intron of DNMT3B, is associated with an increased risk of nicotine dependence. In International Lung Cancer Consortium data, the C allele at rs910083 was found to increase the risk of squamous cell lung carcinoma.48 rs34211819, located in an intron region of tensin-3 (TNS3), and rs1143149, located in an intron region of septin 7 (SEPT7), are significantly associated with the survival of NSCLC patients who are long-term former smokers. Both SNPs have significant interaction effects with years of smoking cessation. As the years of smoking cessation increase in long-term former smokers, the protective effect of rs34211819 and the detrimental effect of rs1143149 on survival are both enhanced.49

While the primary cause of lung cancer is smoking, approximately 25% of lung cancers worldwide occur in never-smokers.50 These cancers are caused by environmental risk factors, such as passive smoking, hormonal factors, and, most importantly, genetic factors.51, 52, 53 A study using WES and RNA-sequencing data for never-smoker LUADs sequenced by The Cancer Genome Atlas (TCGA) and Clinical Proteomic Tumor Analysis Consortium (CPTAC) found that pathogenic germline variants in cancer predisposition genes such as BRCA1, BRCA2, FANCG, FANCM, HMBS, MSH6, NF1, POLD1, TMEM127, and WRN are exclusively associated with lung cancer in never-smokers.54 A GWAS of lung cancer in never-smoking females in Asia identified three lung cancer susceptibility loci, at 10q25.2 (rs7086803), 6q22.2 (rs9387478), and 6p21.32 (rs2395185), with no evidence of an association of 15q25 with lung cancer.55 The TERT and CLPTM1L genes are located in 5p15. TERT is an established telomere maintenance locus. rs10936599 variants, which are located in the TERT coding region and associated with telomere length, were robustly associated with increased lung cancer risk among never-smoking women in Asia.56
CLPTM1L was identified by screening for cisplatin (CDDP) resistance-related genes and was found to induce apoptosis in CDDP-sensitive cells. rs402710 is located in intron 4 of the CLPTM1L gene and is associated with lung cancer susceptibility.41,57

Estrogen receptor (ER) gene SNPs such as rs7753153 and rs985192, located in ESR1, and rs3020450, located in ESR2, are associated with LUAD risk in never-smoking women.58 Moreover, rs12233719, a sex hormone regulation-related SNP in UGT2B7, is associated with NSCLC risk among never-smoking Chinese women.59 These studies suggest the important role of sex hormones in regulating lung tumorigenesis in never-smoking populations. rs11080466 and rs11663246, located in the intron of PIEZO2, show a significant association with NSCLC susceptibility of never smokers in Korean populations.60 rs4648127, located in the NFKB1 gene, was associated with lung cancer in the screening arm of the Prostate, Lung, Colon, and Ovarian (PLCO) Cancer Screening Trial.61

The genetic susceptibility differences in histopathological types and in accordance with smoking status may implicate distinct biological mechanisms of lung cancer and therapeutic strategies (Table 3).Table 3SNVs and lung cancer susceptibility with different smoking status.Table 3:SNVsRegionGeneEffect alleleReference alleleOR (95% CI)P valueReferencesSmoking rs1696996815q25CHRNA5AG1.405.47 × 10–4044 rs94395191p36.32AJAP1-NPHP4GA1.11 (1.06–1.16)3.65 × 10–646 rs480995720q13.2CYP24A1TC1.13 (1.08–1.18)1.20 × 10–8 rs64412863q25.33IL12A-AS1CA1.24 (1.13–1.37)1.16 × 10–547 rs177236379q31.2ZNF462GA1.36 (1.19–1.56)1.06 × 10–5 rs475167410q25.3AFAP1L2CT0.58 (0.46–0.74)1.07 × 10–5 rs91008320q11DNMT3BCA1.06 (1.04–1.07)3.70 × 10–848 rs342118197p12.3TNS3CA0.73 (0.66–0.81)3.90 × 10–949 rs11431497p14.2SEPT7CG1.36 (1.22–1.51)9.75 × 10–9Never-smoking rs708680310q25.2VTI1AAG1.28 (1.21–1.35)3.54 × 10–1855 rs93874786q22.2ROS1-DCBLD1AC0.85 (0.81–0.90)4.14 × 10–10 rs23951856p21.32HLATG1.17 (1.11–1.23)9.51 × 10–9 rs109365993q26.2TERCCT0.10 (0.08–0.11)2.54 × 10–3156 rs4027105p15.33CLPTM1LTC1.14 (1.09–1.19)5.00 × 10–841 rs77531536q25ESR1AG1.51 (1.17–1.95)0.000858 rs9851926q25ESR1CA1.31 (1.00–1.71)0.0233 rs302045014q23ESR2AC2.11 (1.01–4.42)0.0098 rs122337194q13.2UGT2B7TG1.54 (1.21–1.96)0.00659 rs1108046618p11.22PIEZO2CT0.66 (0.56–0.78)1.19 × 10–660 rs1166324618p11.22PIEZO2TC0.67 (0.57–0.79)2.40 × 10–6 rs46481274q24NFKB1TC0.56 (0.37–0.86)0.0261CI: Confidence interval; OR: Odds (log-additive) ratio; SNVs: Single nucleotide variants.

SNVs and lung cancer susceptibility with different smoking status.

CI: Confidence interval; OR: Odds (log-additive) ratio; SNVs: Single nucleotide variants.

### Interplay between SNVs and somatic mutations in regulating lung cancer susceptibility

Most of the mutations detected clinically are somatic mutations. Germline mutations can co-occur or be mutually exclusive with somatic cancer gene alterations.62 For example, the susceptibility variant rs36600 is associated with somatic mutations within ARID1A; the susceptibility variants rs2395185 and rs3817963 are associated with somatic alterations in the cell cycle pathway; and rs3817963 is associated with somatic alterations in the MAPK signaling pathway.63 rs2395185 is associated with elevated APOBEC mutagenesis.64 Some lung cancer risk-related SNPs were shown to influence genetic damage in coke oven workers exposed to polycyclic aromatic hydrocarbons (PAHs). Some SNP loci (rs1333040, rs1663689, and rs3813572) are associated with decreased micronuclei frequency, which is a biomarker of chromosomal damage, genome instability, and cancer risk that associates acquired mutations with genetic susceptibility.65 Recently, Peng et al66 identified 111 pathogenic or likely pathogenic (P/LP) germline mutations in 35 cancer genes in 106 of 1794 Chinese patients (5.91%). Chinese patients with germline mutations show different prevalence rates of somatic KRAS mutation, MET exon 14 skipping, and TP53 mutations compared with those without germline mutations.

### SNVs associated with lung cancer outcome and drug sensitivity

In recent years, significant efforts have been dedicated to investigate the biological mechanisms underlying the association of SNVs with lung cancer outcomes and the clinical implications. The main goal is to translate these discoveries into clinical application. rs3743073-G in the CHRNA3 gene is significantly associated with short survival among patients with advanced stage NSCLC.67 rs942190-G and rs2401863-A located in TDP1 are associated with relatively poor survival among SCLC patients.68 Four SNPs (rs2107561, rs6882451, rs1826692, and rs6595026) modulate overall survival of lung cancer, and rs2107561, an intron variant of PTPRG, exhibits the strongest association.69 rs5030740 in RPA1 and rs1776148 and rs1047840 in EXO1 are associated with disease-free survival and overall survival in lung cancer patients receiving platinum-based chemotherapy. Patients with the C allele of rs5030740 are regarded as protective allele of the prolonged progression-free survival. Patients with the A/A or A/G genotype of rs1776148 and the A/A genotype of rs1047840 have longer overall survival than G/G genotype of rs1776148 and A/G or G/G genotype of rs1047840.70

In addition to predicting prognosis, SNPs can be used in determining therapeutic strategies. The therapeutic efficacy suffers from large patient variability. Genetic variants often alter the sensitivity to the treatments in clinical practice. For example, rs712829 (216G/T) and rs4644 (191C/A) in EGFR are predictive of sensitivity to gefitinib. Mechanically, rs712829 is located in the binding site for the transcription factor Sp1. The T allele promotes Sp1 binding, enhances EGFR transcription, and increases sensitivity to gefitinib. rs4644 is located in the transcriptional start site of the EGFR promoter. The A allele increases promoter activity and protein expression and therefore increases sensitivity to gefitinib.71 rs2231142 (421C/A) in ABCG2 has been correlated with drug transport. The A allele reduces TKI transport and increases the accumulation of gefitinib, which results in adverse effects.71 Patients with the H19 rs2839698 A allele have a smaller chance of response to platinum-based chemotherapy.72 rs1052566 (A273V) in BRMS1v2 is associated with aggressive tumors in LUAD. The A allele of rs1052566 increases c-fos, thereby upregulating CEACAM6, which drives metastasis. T5224, a c-fos pharmacologic inhibitor, suppresses metastases in mice bearing A/A tumors.73 rs1663689 A enhances ADGRG6 expression, which elevates the downstream cyclic adenosine monophosphate (cAMP)-protein kinase A (PKA) signaling. rs1663689 A/A tumors are more sensitive to the PKA inhibitor H89 than the rs1663689 C/C tumors.21 The A allele of rs16906252 in MGMT is associated with increased MGMT methylation and lower MGMT expression. MGMT can reduce the tumor response to temozolomide. Thus, lung cancer patients with rs16906252-A may benefit from temozolomide treatment.74 Although the clinical treatment strategies based on germline variants have not been implemented at present, these studies demonstrate the utility for SNVs in predicting drug sensitivity of tumors, highlighting their important role in precision medicine.

### Outlook

SNVs regulate cellular behavior and subsequent disease phenotypes; therefore, SNVs can be used to select the appropriate therapeutic strategies. To achieve this goal, understanding the causal function of SNVs is needed. SNVs reside in different regions and function through different mechanisms. SNVs in coding regions change the sequence of amino acids, and subsequently change the structure and biological function of relative proteins. However, over 90% of disease-associated SNVs are located in non-coding regions of the genome, often at considerable genomic distances from annotated genes. These non-coding SNVs either change the sequences of noncoding RNAs or change the binding affinity of transcription factors and thereby posttranscriptionally regulate or transcriptionally regulate their downstream target genes (Fig. 4). Although some well-characterized non-coding SNVs regulate their neighboring genes, assignment based on linear proximity is error prone, as many cis-regulatory elements map large distances away from their targets, bypassing the nearest gene, which makes identifying their downstream genes problematic. Therefore, the functions of most lung cancer-associated non-coding SNVs remain unknown. Developing efficient strategies to decipher the regulatory pathways for non-coding SNVs is needed. Given that long-range regulation requires direct physical interactions in eukaryotes, genomic screening for non-coding SNV-interacting genes will serve as a strategy to identify target genes.Fig. 4Schematic representation of the mechanism of the SNVs located in different regions. ncRNAs: Non-coding RNAs; SNVs: Single nucleotide variants.Fig 4:

Schematic representation of the mechanism of the SNVs located in different regions. ncRNAs: Non-coding RNAs; SNVs: Single nucleotide variants.



# SUPPLEMENTAL FILE 1: main.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)