# MAIN TEXT

## Prevalence of germline variants in Brazilian pancreatic carcinoma patients

### Abstract

We evaluated the prevalence of pathogenic/likely pathogenic germline variants (PGV) in Brazilian pancreatic adenocarcinoma (PC) patients, that represent a multiethnic population, in a cross-sectional study. We included 192 PC patients unselected for family history of cancer. We evaluated a panel of 113 cancer genes, through genomic DNA sequencing and 46 ancestry-informative markers, through multiplex PCR. The median age was 61 years; 63.5% of the patients presented disease clinical stages III or IV; 8.3% reported personal history of cancer; 4.7% and 16.1% reported first-degree relatives with PC or breast and/or prostate cancer, respectively. Although the main ancestry was European, there was considerable genetic composition admixture. Twelve patients (6.25%) were PGV carriers in PC predisposition genes (ATM, BRCA1, BRCA2, CDKN2A, MSH2, PALB2) and another 25 (13.0%) were PGV carriers in genes with a limited association or not previously associated with PC (ACD, BLM, BRIP1, CHEK2, ERCC4, FANCA, FANCE, FANCM, GALNT12, MITF, MRE11, MUTYH, POLE, RAD51B, RAD51C, RECQL4, SDHA, TERF2IP). The most frequently affected genes were CHEK2, ATM and FANC. In tumor samples from PGV carriers in ACD, BRIP1, MRE11, POLE, SDHA, TERF2IP, which were examined through exome sequencing, the main single base substitutions (SBS) mutational signature was SBS1+5+18, probably associated with age, tobacco smoking and reactive oxygen species. SBS3 associated with homologous repair deficiency was also represented, but on a lower scale. There was no difference in the frequency of PGV carriers between: (a) patients with or without first-degree relatives with cancer; and (b) patients with admixed ancestry versus those with predominantly European ancestry. Furthermore, there was no difference in overall survival between PGV carriers and non-carriers. Therefore, genetic testing should be offered to all Brazilian pancreatic cancer patients, regardless of their ancestry. Genes with limited or previously unrecognized associations with pancreatic cancer should be further investigated to clarify their role in cancer risk.

### Introduction

Pancreatic ductal adenocarcinoma, referred to as pancreatic cancer (PC) from now on, encompasses sporadic and inherited categories, the latter including familial cancer and hereditary cancer syndromes1. Familial PC comprises patients who report two or more first-degree relatives (FDR) diagnosed with PC on the same parental side. Hereditary PC comprises cases attributed to genetic syndromes, with defined inheritance2.

Germline pathogenic or likely pathogenic variants (PGV) associated with PC predisposition were mainly characterized in non-Latino White people. These studies show some ethnic differences, including a higher prevalence of PGVs in BRCA1 and BRCA2 genes in Ashkenazi Jewish patients (AJ)3. In Brazil, there is no data regarding PGVs associated with PC. However, there is considerable information on breast cancer (BC) patients. In this scenario, different from studies carried out in other populations, PGVs in TP53 were relatively frequent, detected in 2.2% of patients, mainly due to the Brazilian founder variant TP53 R337H4. These studies support the importance of analyzing the spectrum and prevalence of PGVs in cancer patients with varied ethnic backgrounds.

The Brazilian population has unique ethnic characteristics, reflecting a five-century miscegenation of Portuguese settlers with Indigenous (Indig) South Americans and African people. Additionally, a high migratory wave from Italy, Germany, Spain, Poland, Japan, China, Korea, Lebanon, and Syria, occurred in the nineteenth–twentieth centuries and more recently, from South American countries5. To explore the ethnic background of the population, the “Instituto Brasileiro de Geografia e Estatística”, IBGE (https://www.ibge.gov.br/), the official agency responsible for the Brazilian Census, includes questions about individuals' self-declared skin color or race, that comprises: brown, black, white, yellow and Indigenous. According to the 2022 Brazilian Census, 45.3% of the population were self-declared brown, while 43.5%, 10.2%, 10.2%, 0.6 and 0.4%, were self-declared white, black, indigenous or yellow, respectively6. Besides that, previous studies showed that the mean genetic contributions of European, African and Amerindian ancestries to the Brazilian population are 68.1%, 19.6% and 11.6%, respectively7,8. These ethnic characteristics are diverse from the Latino (with predominance of European and Amerindian ancestries) and African American people, residing in the United States of America9,10.

Thus, this study aimed to evaluate the prevalence of PGVs in cancer predisposition genes in a genetically admixed cohort, consisting of unselected Brazilian pancreatic carcinoma patients.

### Results

In total, we prospectively and sequentially included 192 PC patients from April 2018 to June 2021 (with a pause from March 2020 until January 2021), except for 18 patients, included from March 2016 to February 2018, by the Institutional Biobank, following the same inclusion criteria. The median time between date of diagnosis and date of inclusion for all patients was 105 days (min: 0–max: 3956 days) and 38 patients had more than a year of follow-up before inclusion.

The median age at PC diagnosis was 61 years; 31 patients were young (≤ 50 years) at diagnosis; 59% were female; 35.5% presented metastatic disease (Table 1); and 8.3% reported a previous personal history of cancer (breast: n = 5; colorectal: n = 3; others: n = 8), including four patients who reported more than one additional primary cancer.Table 1Characteristics of participants.CharacteristicsN (%)N (%)N (%)pAll patientsPGV carriersNon-PGV carriersTotal19237155Age at diagnosisMedian (min–max)61 (27–87)60 (27–81)62 (34–87)0.610 ≤ 50 years31 (16)7 (19)24 (15) > 50 years161 (84)30 (81)131 (85)SexFemale114 (59)19 (51)95 (61)0.269Male78 (41)18 (49)60 (39)Clinical stage#01 (1)0 (0)1 (1)0.818I13 (7)2 (6)11 (7)II56 (29)9 (24)47 (30)III54 (28)10 (27)44 (28)IV68 (35)16 (43)52 (34)Smoking status (packs/year)Never87 (45)17 (46)70 (45)0.860 < 2053 (28)9 (24)44 (28) ≥ 2052 (27)11 (30)41 (26)Alcohol consumptionDeny91 (47)14 (38)77 (50)0.117Social drinker57 (30)11 (30)46 (30)Drinker14 (7)6 (16)8 (5)Heavy drinker30 (16)6 (16)24 (15)Body mass index (prior to diagnosis)Underweight3 (2)0 (0)3 (2)0.645Normal weight57 (30)14 (38)43 (28)Overweight72 (38)13 (35)59 (38)Obesity60 (31)10 (27)50 (32)DiabetesYes38 (20)5 (14)33 (21)0.286No154 (80)32 (86)122 (79)Personal history of cancerYes16 (8)2 (5)14 (9)0.741No176 (92)35 (95)141 (91)Family history of cancer (1st—degree relative)Yes106 (55)21 (57)85 (55)0.999No84 (44)16 (43)69 (44)Unknown2 (1)0 (0)2 (1)Type of cancer in 1st—degree relativePancreas9 (4.7)2 (5)7 (5)0.999Breast or Prostate31 (16.1)6 (16)25 (16)Others66 (34.4)13 (35)53 (34)Genetic ancestryAFR_pred1 (1)0 (0)1 (1)0.751EUR_pred62 (32)15 (41)47 (30)EAS_pred7 (4)0 (0)7 (5)ADM-adm14 (7)2 (5)12 (8)ADM_AFR13 (7)2 (5)11 (7)ADM_EUR95 (49)18 (49)77 (50)#AJCC Cancer Staging—8th ed. 2017; alcohol consumption (heavy drinker: alcohol intake ≥ 30 g/day for males and ≥ 15 g/day for females; drinker: less than the pre-specified values; social drinker, unspecified drinking amount in weekends and festivities); pred: predominance; ADM: admixed.

Characteristics of participants.

#AJCC Cancer Staging—8th ed. 2017; alcohol consumption (heavy drinker: alcohol intake ≥ 30 g/day for males and ≥ 15 g/day for females; drinker: less than the pre-specified values; social drinker, unspecified drinking amount in weekends and festivities); pred: predominance; ADM: admixed.

Most patients (77%) reported a positive family history of cancer (any primary site) in a close relative, including 55% who reported an affected first-degree relative (Table 1), among whom, 47 (24.5%) reported two or more first degree relatives with some type of cancer. The most frequent cancers in first degree relatives were breast and pancreatic cancers (Table 1).

Regarding risk factors, 105 (55%) patients reported smoking (27% heavy smoking); 16% reported heavy alcohol consumption; 31% reported previous obesity and 20% reported diabetes for at least one year before cancer diagnosis (Table 1). No patients reported Ashkenazi Jewish ancestry. Considering the birthplace, 64% and 29% of the patients were born in the Southeast and Northeast Brazilian regions, respectively (Supplementary Fig. 1).

In total, 37 out of 192 patients (19.27%) were PGV carriers among whom, 57% reported family history of cancer. Among 68 patients with metastatic disease, 26.5% were PGV variant carriers. There was no difference in age, sex, metastatic disease, personal and family history of cancer, smoking, alcohol consumption, BMI, diabetes and ancestry between carriers and non-carriers (Table 1).

We detected PGVs in 24 different genes (Tables 2 and 3). Twelve patients (6.25%) carried PGV in six genes associated with pancreatic cancer, i.e., ATM, BRCA1, BRCA2, CDKN2A, MSH2 and PALB2 (Table 2). Another 22 patients presented PGVs in 15 genes with limited association with pancreatic cancer, including BLM, BRIP1, CHEK2, ERCC4, FANCA, FANCE, FANCM, MITF, MRE11, MUTYH, POLE, RAD51B, RAD51C, RECQL4, and SDHA. Additionally, three patients carried PGVs in three genes not previously associated with pancreatic cancer, i.e., ACD, GALNT12, and TERF2IP (Table 3).Table 2Variants in genes associated with pancreatic cancer.GeneRefseq transcriptRefAltHGVS codingHGVS proteindbSNPACMG criteriaACMG ClassifGenes associated with pancreatic cancerATMNM_000051.4TGc.7789-3T>G–rs864622185PVS1, PS4, PM2PATMNM_000051.4GAc.7913G>Ap.(Trp2638Ter)rs377349459PVS1, PS4, PM2PATMNM_000051.4ATAAGc.8264_8268delp.(Tyr2755fs)rs730881294PVS1, PS3, PM2PATMNM_000051.4Ac.9079dupAp.(Ser3027Lysfs)rs587780645PVS1, PS4, PM2PATMNM_000051.4Deletion of exons 62–63BRCA1NM_007294.4AGc.470_471delp.(Ser157Ter)rs80357887PVS1, PM2, PP5PBRCA1NM_007294.4TCc.5432A>Gp.(Gln1811Arg)rs80357040PM1, PM2, PP3, PP5PBRCA2–TAc.1658_1659delp.(Cys554Serfs)newPVS1, PM2, PM6PCDKN2ANM_000077.5GTc.35C>Ap.(Ser12Ter)rs141798398PVS1, PM2, PP5PMSH2NM_000251.3CTc.1216C>Tp.(Arg406Ter)rs63751108PVS1, PS3, PS4, PM2PPALB2NM_024675Tc.2185_2186insAp.(Pro729fs)rs1597089845PVS1, PS4, PM2, PP5PPALB2NM_024675CTc.2711G>Ap.(Trp904Ter)rs1060502726PVS1, PM2, PP5PTable 3Variants in genes with limited association or in genes not previously associated with pancreatic cancer.GeneRefseq transcriptsRefAltHGVS codingHGVS proteindbSNPACMG CriteriaACMG ClassifReferencesGenes with limited association with pancreatic cancerBLMNM_000057.4GTc.646G>Tp.(Glu216Ter)newPVS1, PM2LPCheng27, Singh58, Smith25, Tzfoni24BRIP1NM_032043.3AACAc.128_131delp.(Leu43Trpfs)rs1064794202PVS1, PM2LPCheng27, Dudley87, Jiang23, Yin36BRIP1NM_032043.3TTGTc.290_293delp.(Asn97Metfs)rs763009188PVS1, PM2LPCheng27, Dudley87, Jiang23, Yin36CHEK2NM_007194.4GAc.1036C>Tp.(Arg346Cys)rs201206424PS3, PM2LPCheng27, Hu20, Lowery16, Wieme17CHEK2NM_007194.4Gc.1100delCp.(Thr367Metfs)rs555607708PVS1, PM2, PP5PCheng27, Hu20, Lowery16, Wieme17CHEK2NM_007194.4GAc.1427C>Tp.(Thr476Met)rs142763740PS3, PS4, PM2LPCheng27, Hu20, Lowery16, Wieme17CHEK2NM_007194.4GAc.1486C>Tp.(Gln496Ter)rs756250205PVS1, PM2, PP5PCheng27, Hu20, Lowery16, Wieme17CHEK2NM_007194.4Duplication of exons 6–15Cheng27, Hu20, Lowery16, Wieme17ERCC4NM_005236GAc.584+1G > A–rs2032080620PVS1, PM2LPGoldstein39, Osorio41, Puccini14, Velázquez40FANCA–Duplication of exons 1–3Cheng27, Yin36FANCA–Deletion of exons 18–19Cheng27, Yin36FANCENM_021922.3CTc.1111C>Tp.(Arg371Trp)rs775076977PS3, PM2LPCheng27FANCMNM_020937.4AAAAc.2586_2589delp.(Lys863fs)rs768006618PVS1, PM2LPBogliolo33, Hu88MITF–GAc.952G>Ap.(Glu318Lys)rs149617956PS3, PM1, PP2, PP5LPLowery16MRE11–TCc.350A>Gp.(Asn117Ser)rs137852760PS3, PM2LPJiang23, Stastna28, Tsoulos26MUTYHNM_001128425.2TCc.536A>Gp.(Tyr179Cys)rs34612342PS3, PM1, PM2LPCheng27, Guindalini4, Lowery16, Out43, Thibodeau89, Win42MUTYHNM_001128425.2CTc.1187G>Ap.(Gly396Asp)rs36053993PM1, PM2, PP3, PP5LPCheng27, Guindalini4, Lowery16, Out43, Thibodeau89, Win42POLENM_006231.4CAc.3721G>Tp.(Glu1241Ter)rs779261309PVS1, PM2LPCheng27, Garmezy46, Ghiorzo48, Hansen45, Yokoyama49RAD51BNM_002877Tc.88delTp.(Phe30fs)rs1246922072PVS1, PM2LPXie90RAD51CNM_058216.3CTc.709C>Tp.(Arg237Ter)rs770637624PVS1, PM2, PP3, PP5PUson13, Yurgelun61RECQL4NM_004260.4CGCCCGGCCc.2412_2420delp.(Ala805_Arg807del)rs766312203PM2, PM4, PP3LPCheng27SDHANM_004168.4Gc.666delGp.(Asp223Ilefs)rs587782077PVS1, PS4, PM2PBurnichon50, Puccini14, Uson13, van der Tuin91Genes not previously associated with pancreatic cancerACDNM_022914.3AGc.599_600delp.(Pro200Argfs*28)rs777476880PVS1, PM2LPAoude52, Pastorino51GALNT12NM_024642GCCGGGGCCGGGGCTGCCGAc.127_146delp.(Ala43fs)–PVS1, PM2LPEvans53TERF2IPNM_018975ATc.1152_1153delp.(Lys384fs)rs1010679336PVS1, PM2LPAoude52Bold = variant found in two patients. References provided are related to gene alterations in pancreatic cancer or specific variants identified in cancer studies.

Variants in genes associated with pancreatic cancer.

Variants in genes with limited association or in genes not previously associated with pancreatic cancer.

Bold = variant found in two patients. References provided are related to gene alterations in pancreatic cancer or specific variants identified in cancer studies.

The most frequently affected genes were CHEK2 and ATM, detected in six and five patients, respectively. FANCM c.2586-2589del and MITF c.952G>A were each one detected in two unrelated patients. We identified four CNVs: one in ATM, one in CHEK2 (multiexon deletion in a BRCA1 PGV carrier) and one deletion and one duplication in FANCA. Three patients carried a PGV in two genes: BRCA1 + CHEK2 (CNV); BRIP1 + MUTYH; CDKN2A + MITF (Supplementary Table 1).

Among all patients, 129 were VUS carriers only, 32 were VUS and PGV carriers (Fig. 1), 26 were neither PGV nor VUS carriers. Five out of 37 patients carrying PGVs did not present any VUS. Among the VUS carriers, 56 presented only one VUS and 104 presented more than one VUS (Supplementary Fig. 2).Fig. 1Solar explosion graph depicting spectrum of PGVs and VUSs in pancreatic cancer patients. Inner circle: Number of patients who are PGV carriers, VUS only carriers, neither PGV nor VUS carriers in the panel of genes (wild type); Medium circle: list of PGV affected genes in 37 participants; Outer circle: spectrum of VUS in the panel of genes in PGV carriers and non-carriers. *One patient who was a PGV carrier in CHEK2 and one patient who was a PGV carrier in ATM presented no VUS, but were considered together with the other patients who were PGV carriers in CHEK2 and ATM and were also VUS carriers.

Solar explosion graph depicting spectrum of PGVs and VUSs in pancreatic cancer patients. Inner circle: Number of patients who are PGV carriers, VUS only carriers, neither PGV nor VUS carriers in the panel of genes (wild type); Medium circle: list of PGV affected genes in 37 participants; Outer circle: spectrum of VUS in the panel of genes in PGV carriers and non-carriers. *One patient who was a PGV carrier in CHEK2 and one patient who was a PGV carrier in ATM presented no VUS, but were considered together with the other patients who were PGV carriers in CHEK2 and ATM and were also VUS carriers.

Nine patients presented more than one VUS in distinct positions of the following genes: ATM, BRIP1, EPCAM, FANCA, FANCF, GALNT12, PMS2, POLE and SLX4. Six VUS were characterized as splice site: FANCA c.3349-3C>T, FLCN c.249 + 2C>T, MSH3 c.3130+3A>G, LZTR1 c.1943-3C>T, RAD51B c.84+3Adel and FANCL c.541-3C>T; one VUS as frameshift: FANCL c.1099_1100insATTA; and one VUS as start loss: RAD51C c.3G>A.

To assess the contribution of certain pathogenic/likely pathogenic variants (ACD c.608_609delCT (plus RAD51C c.3G>A, a variant of uncertain significance), BRIP1 c.290_293del, MRE11 c.350A>G, POLE c.3721G>T, SDHA c.666delG, and TERF2IP c.1152_1153del) to pancreatic cancer risk, we conducted whole exome sequencing on DNA extracted from formalin-fixed paraffin-embedded (FFPE) tumor specimens of patients carrying these variants. Our analysis focused on evaluating loss of heterozygosity (LOH) and mutational signatures.

MRE11, POLE, RAD51C variants were detected in heterozygosity in tumor samples with VAFs of 0.539, 0.474 and 0.545, respectively. ACD, BRIP1 and SDHA VAFs were 0.645, 0.636, and 0.60, which is in or next to the lower limit of the cut off value considered for loss of heterozygosity, described in methods. We also used LOHGIC12 to estimate LOH occurrence, however we obtained no significant results for the variants tested (Akaike Information Criteria (AIC) weight < 0.7) (Supplementary table 3). TERF2IP variant could not be further evaluated due to low coverage in this gene region. Additionally, we checked for LOH, using the tools CNVkit13 and PureCN14. PureCN algorithm allows accurate estimates of copy number and LOH, while accounting for tumor purity and ploidy. PureCN can be integrated with common segmentation tools outputs, such as CNVkit, that combines both on– and off-target reads, to provide copy ratio estimates. This analysis could be performed for tumor samples from carriers of MRE11 and POLE variants, however, there was no definite evidence of LOH (Supplementary table 3).

To further evaluate the role of the above variants, we explored the mutational signature of tumor whole exome sequencing data. For this, we used the algorithm Signature Fit Multi-Step (FitMS) from the signatures.tools.lib R package, to detect common organ-specific signatures for pancreas in exome sequencing data15. The main signature detected in all samples was Single base substitutions (SBS) SBS1+5+18. Signatures SBS1 and SBS3+8 were also detected in all tumor samples (Fig. 2; Supplementary Fig. 3).Fig. 2Single Base Substitution (SBS) signatures across whole-exome-sequence tumor samples. Number of single nucleotide variants (SNVs) associated with each SBS in tumor samples of carriers of PGVs in MRE11 c.350A>G (PC32); ACD c.608_609delCT (plus RAD51C c.3G>A) (PC59), TERF2IP c.1152_1153del (PC67); SDHA c.666delG (PC109); POLE c.3721G>T (PC110); BRIP1 c.290_293del (PC115). PC: patient sample identification.

Single Base Substitution (SBS) signatures across whole-exome-sequence tumor samples. Number of single nucleotide variants (SNVs) associated with each SBS in tumor samples of carriers of PGVs in MRE11 c.350A>G (PC32); ACD c.608_609delCT (plus RAD51C c.3G>A) (PC59), TERF2IP c.1152_1153del (PC67); SDHA c.666delG (PC109); POLE c.3721G>T (PC110); BRIP1 c.290_293del (PC115). PC: patient sample identification.

We estimated ancestry admixture proportions using 46 ancestry-informative markers (AIMs) of small insertion-deletion (INDEL) polymorphisms11. Initially, we identified seven individuals with predominant East Asian (EAS) ancestry (using Admixture k = 4 analysis). We excluded EAS from further analysis, to avoid bias due to the close relationship between EAS and Indigenous (Indig) populations. Subsequently, a k = 3 analysis revealed a predominance of European genetic composition and the following mean ancestry proportions: EUR: 71.75% (SD 19.26), AFR: 19.79% (SD: 18.70), Indig: 8.46% (SD: 6.64) (Supplementary Fig. 4A).

In this cohort, 62 patients presented a European genetic ancestry predominance (EUR_pred) and 122, an admixed genetic composition, using criteria described in the methods. In the category of admixed ancestry composition, 95 individuals were classified as admixed with EUR predominance (ADM-EUR); 13, as admixed with AFR predominance (ADM-AFR) and 14, as admixed with no predominance (ADM-adm). Only one individual was categorized with an African genetic ancestry predominance (AFR_pred) (Supplementary Fig. 4B).

Considering self-declared skin color (white, brown, yellow and black), although there was some admixture in individual genetic ancestry composition (Fig. 3), there was a positive association with the main genetic ancestry (p < 0.001) (Supplementary Fig. 5).Fig. 3Individual genetic ancestry estimates of Brazilian pancreatic cancer patients in accordance with self-declared skin color. Patients self-declared skin color appears in the X axis. Genetic ancestry proportion estimates appear on the Y axis.

Individual genetic ancestry estimates of Brazilian pancreatic cancer patients in accordance with self-declared skin color. Patients self-declared skin color appears in the X axis. Genetic ancestry proportion estimates appear on the Y axis.

PGVs were detected in 22 out of 122 (18.00%) individuals with admixed ancestry composition and in 14 out of 62 (22.58%) patients with European genetic ancestry predominance (EUR_pred) (OR: 0.7543; 95% CI 0.3551–1.6021; p = 0.4631).

The mean number of VUS according with genetic ancestry were 1.64 and 1.86 for EUR_pred and admixed people, respectively. There was no difference in the number of VUS among EUR_pred and the three groups of admixed people (p = 0.509; Supplementary table 2).

Among all patients, one was detected with carcinoma in situ, after surgery, and was excluded from survival analysis. The median overall survival was 16.46 months (95% CI 12.24–20.68). There was no difference in OS for PGV carriers versus non-carriers, considering: all genes; genes associated with PC, homologous repair-related (HRR) genes; all genes in the admixed population (Fig. 4); BRCA1, BRCA2 and PALB2 genes, all genes in patients with early-stage disease and patients with advanced disease (Supplementary Fig. 6). There were differences in overall survival among patients based on their disease clinical stage and body mass index (BMI). (Supplementary Fig. 6).Fig. 4Overall survival estimates in PGV carriers and non-carriers. (A) All pancreatic cancer (PC) divided in PGV carriers in one of the 24 affected genes (P/LP variants present) and non-carriers (P/LP variants absent); (B) PC patients divided in homologous recombination repair (HRR) PGV carriers and non-carriers; (C) PC patients divided in PGV carriers in genes associated with pancreatic cancer (GAPC) and non-carriers; (D) Patients with non-European ancestry (admixed) PGV carriers and non-carriers. Abs: P/LP variants absent; Pres: P/LP variants present; CS: clinical stage.

Overall survival estimates in PGV carriers and non-carriers. (A) All pancreatic cancer (PC) divided in PGV carriers in one of the 24 affected genes (P/LP variants present) and non-carriers (P/LP variants absent); (B) PC patients divided in homologous recombination repair (HRR) PGV carriers and non-carriers; (C) PC patients divided in PGV carriers in genes associated with pancreatic cancer (GAPC) and non-carriers; (D) Patients with non-European ancestry (admixed) PGV carriers and non-carriers. Abs: P/LP variants absent; Pres: P/LP variants present; CS: clinical stage.

### Patient characteristics

In total, we prospectively and sequentially included 192 PC patients from April 2018 to June 2021 (with a pause from March 2020 until January 2021), except for 18 patients, included from March 2016 to February 2018, by the Institutional Biobank, following the same inclusion criteria. The median time between date of diagnosis and date of inclusion for all patients was 105 days (min: 0–max: 3956 days) and 38 patients had more than a year of follow-up before inclusion.

The median age at PC diagnosis was 61 years; 31 patients were young (≤ 50 years) at diagnosis; 59% were female; 35.5% presented metastatic disease (Table 1); and 8.3% reported a previous personal history of cancer (breast: n = 5; colorectal: n = 3; others: n = 8), including four patients who reported more than one additional primary cancer.Table 1Characteristics of participants.CharacteristicsN (%)N (%)N (%)pAll patientsPGV carriersNon-PGV carriersTotal19237155Age at diagnosisMedian (min–max)61 (27–87)60 (27–81)62 (34–87)0.610 ≤ 50 years31 (16)7 (19)24 (15) > 50 years161 (84)30 (81)131 (85)SexFemale114 (59)19 (51)95 (61)0.269Male78 (41)18 (49)60 (39)Clinical stage#01 (1)0 (0)1 (1)0.818I13 (7)2 (6)11 (7)II56 (29)9 (24)47 (30)III54 (28)10 (27)44 (28)IV68 (35)16 (43)52 (34)Smoking status (packs/year)Never87 (45)17 (46)70 (45)0.860 < 2053 (28)9 (24)44 (28) ≥ 2052 (27)11 (30)41 (26)Alcohol consumptionDeny91 (47)14 (38)77 (50)0.117Social drinker57 (30)11 (30)46 (30)Drinker14 (7)6 (16)8 (5)Heavy drinker30 (16)6 (16)24 (15)Body mass index (prior to diagnosis)Underweight3 (2)0 (0)3 (2)0.645Normal weight57 (30)14 (38)43 (28)Overweight72 (38)13 (35)59 (38)Obesity60 (31)10 (27)50 (32)DiabetesYes38 (20)5 (14)33 (21)0.286No154 (80)32 (86)122 (79)Personal history of cancerYes16 (8)2 (5)14 (9)0.741No176 (92)35 (95)141 (91)Family history of cancer (1st—degree relative)Yes106 (55)21 (57)85 (55)0.999No84 (44)16 (43)69 (44)Unknown2 (1)0 (0)2 (1)Type of cancer in 1st—degree relativePancreas9 (4.7)2 (5)7 (5)0.999Breast or Prostate31 (16.1)6 (16)25 (16)Others66 (34.4)13 (35)53 (34)Genetic ancestryAFR_pred1 (1)0 (0)1 (1)0.751EUR_pred62 (32)15 (41)47 (30)EAS_pred7 (4)0 (0)7 (5)ADM-adm14 (7)2 (5)12 (8)ADM_AFR13 (7)2 (5)11 (7)ADM_EUR95 (49)18 (49)77 (50)#AJCC Cancer Staging—8th ed. 2017; alcohol consumption (heavy drinker: alcohol intake ≥ 30 g/day for males and ≥ 15 g/day for females; drinker: less than the pre-specified values; social drinker, unspecified drinking amount in weekends and festivities); pred: predominance; ADM: admixed.

Characteristics of participants.

#AJCC Cancer Staging—8th ed. 2017; alcohol consumption (heavy drinker: alcohol intake ≥ 30 g/day for males and ≥ 15 g/day for females; drinker: less than the pre-specified values; social drinker, unspecified drinking amount in weekends and festivities); pred: predominance; ADM: admixed.

Most patients (77%) reported a positive family history of cancer (any primary site) in a close relative, including 55% who reported an affected first-degree relative (Table 1), among whom, 47 (24.5%) reported two or more first degree relatives with some type of cancer. The most frequent cancers in first degree relatives were breast and pancreatic cancers (Table 1).

Regarding risk factors, 105 (55%) patients reported smoking (27% heavy smoking); 16% reported heavy alcohol consumption; 31% reported previous obesity and 20% reported diabetes for at least one year before cancer diagnosis (Table 1). No patients reported Ashkenazi Jewish ancestry. Considering the birthplace, 64% and 29% of the patients were born in the Southeast and Northeast Brazilian regions, respectively (Supplementary Fig. 1).

### PGV carriers

In total, 37 out of 192 patients (19.27%) were PGV carriers among whom, 57% reported family history of cancer. Among 68 patients with metastatic disease, 26.5% were PGV variant carriers. There was no difference in age, sex, metastatic disease, personal and family history of cancer, smoking, alcohol consumption, BMI, diabetes and ancestry between carriers and non-carriers (Table 1).

We detected PGVs in 24 different genes (Tables 2 and 3). Twelve patients (6.25%) carried PGV in six genes associated with pancreatic cancer, i.e., ATM, BRCA1, BRCA2, CDKN2A, MSH2 and PALB2 (Table 2). Another 22 patients presented PGVs in 15 genes with limited association with pancreatic cancer, including BLM, BRIP1, CHEK2, ERCC4, FANCA, FANCE, FANCM, MITF, MRE11, MUTYH, POLE, RAD51B, RAD51C, RECQL4, and SDHA. Additionally, three patients carried PGVs in three genes not previously associated with pancreatic cancer, i.e., ACD, GALNT12, and TERF2IP (Table 3).Table 2Variants in genes associated with pancreatic cancer.GeneRefseq transcriptRefAltHGVS codingHGVS proteindbSNPACMG criteriaACMG ClassifGenes associated with pancreatic cancerATMNM_000051.4TGc.7789-3T>G–rs864622185PVS1, PS4, PM2PATMNM_000051.4GAc.7913G>Ap.(Trp2638Ter)rs377349459PVS1, PS4, PM2PATMNM_000051.4ATAAGc.8264_8268delp.(Tyr2755fs)rs730881294PVS1, PS3, PM2PATMNM_000051.4Ac.9079dupAp.(Ser3027Lysfs)rs587780645PVS1, PS4, PM2PATMNM_000051.4Deletion of exons 62–63BRCA1NM_007294.4AGc.470_471delp.(Ser157Ter)rs80357887PVS1, PM2, PP5PBRCA1NM_007294.4TCc.5432A>Gp.(Gln1811Arg)rs80357040PM1, PM2, PP3, PP5PBRCA2–TAc.1658_1659delp.(Cys554Serfs)newPVS1, PM2, PM6PCDKN2ANM_000077.5GTc.35C>Ap.(Ser12Ter)rs141798398PVS1, PM2, PP5PMSH2NM_000251.3CTc.1216C>Tp.(Arg406Ter)rs63751108PVS1, PS3, PS4, PM2PPALB2NM_024675Tc.2185_2186insAp.(Pro729fs)rs1597089845PVS1, PS4, PM2, PP5PPALB2NM_024675CTc.2711G>Ap.(Trp904Ter)rs1060502726PVS1, PM2, PP5PTable 3Variants in genes with limited association or in genes not previously associated with pancreatic cancer.GeneRefseq transcriptsRefAltHGVS codingHGVS proteindbSNPACMG CriteriaACMG ClassifReferencesGenes with limited association with pancreatic cancerBLMNM_000057.4GTc.646G>Tp.(Glu216Ter)newPVS1, PM2LPCheng27, Singh58, Smith25, Tzfoni24BRIP1NM_032043.3AACAc.128_131delp.(Leu43Trpfs)rs1064794202PVS1, PM2LPCheng27, Dudley87, Jiang23, Yin36BRIP1NM_032043.3TTGTc.290_293delp.(Asn97Metfs)rs763009188PVS1, PM2LPCheng27, Dudley87, Jiang23, Yin36CHEK2NM_007194.4GAc.1036C>Tp.(Arg346Cys)rs201206424PS3, PM2LPCheng27, Hu20, Lowery16, Wieme17CHEK2NM_007194.4Gc.1100delCp.(Thr367Metfs)rs555607708PVS1, PM2, PP5PCheng27, Hu20, Lowery16, Wieme17CHEK2NM_007194.4GAc.1427C>Tp.(Thr476Met)rs142763740PS3, PS4, PM2LPCheng27, Hu20, Lowery16, Wieme17CHEK2NM_007194.4GAc.1486C>Tp.(Gln496Ter)rs756250205PVS1, PM2, PP5PCheng27, Hu20, Lowery16, Wieme17CHEK2NM_007194.4Duplication of exons 6–15Cheng27, Hu20, Lowery16, Wieme17ERCC4NM_005236GAc.584+1G > A–rs2032080620PVS1, PM2LPGoldstein39, Osorio41, Puccini14, Velázquez40FANCA–Duplication of exons 1–3Cheng27, Yin36FANCA–Deletion of exons 18–19Cheng27, Yin36FANCENM_021922.3CTc.1111C>Tp.(Arg371Trp)rs775076977PS3, PM2LPCheng27FANCMNM_020937.4AAAAc.2586_2589delp.(Lys863fs)rs768006618PVS1, PM2LPBogliolo33, Hu88MITF–GAc.952G>Ap.(Glu318Lys)rs149617956PS3, PM1, PP2, PP5LPLowery16MRE11–TCc.350A>Gp.(Asn117Ser)rs137852760PS3, PM2LPJiang23, Stastna28, Tsoulos26MUTYHNM_001128425.2TCc.536A>Gp.(Tyr179Cys)rs34612342PS3, PM1, PM2LPCheng27, Guindalini4, Lowery16, Out43, Thibodeau89, Win42MUTYHNM_001128425.2CTc.1187G>Ap.(Gly396Asp)rs36053993PM1, PM2, PP3, PP5LPCheng27, Guindalini4, Lowery16, Out43, Thibodeau89, Win42POLENM_006231.4CAc.3721G>Tp.(Glu1241Ter)rs779261309PVS1, PM2LPCheng27, Garmezy46, Ghiorzo48, Hansen45, Yokoyama49RAD51BNM_002877Tc.88delTp.(Phe30fs)rs1246922072PVS1, PM2LPXie90RAD51CNM_058216.3CTc.709C>Tp.(Arg237Ter)rs770637624PVS1, PM2, PP3, PP5PUson13, Yurgelun61RECQL4NM_004260.4CGCCCGGCCc.2412_2420delp.(Ala805_Arg807del)rs766312203PM2, PM4, PP3LPCheng27SDHANM_004168.4Gc.666delGp.(Asp223Ilefs)rs587782077PVS1, PS4, PM2PBurnichon50, Puccini14, Uson13, van der Tuin91Genes not previously associated with pancreatic cancerACDNM_022914.3AGc.599_600delp.(Pro200Argfs*28)rs777476880PVS1, PM2LPAoude52, Pastorino51GALNT12NM_024642GCCGGGGCCGGGGCTGCCGAc.127_146delp.(Ala43fs)–PVS1, PM2LPEvans53TERF2IPNM_018975ATc.1152_1153delp.(Lys384fs)rs1010679336PVS1, PM2LPAoude52Bold = variant found in two patients. References provided are related to gene alterations in pancreatic cancer or specific variants identified in cancer studies.

Variants in genes associated with pancreatic cancer.

Variants in genes with limited association or in genes not previously associated with pancreatic cancer.

Bold = variant found in two patients. References provided are related to gene alterations in pancreatic cancer or specific variants identified in cancer studies.

The most frequently affected genes were CHEK2 and ATM, detected in six and five patients, respectively. FANCM c.2586-2589del and MITF c.952G>A were each one detected in two unrelated patients. We identified four CNVs: one in ATM, one in CHEK2 (multiexon deletion in a BRCA1 PGV carrier) and one deletion and one duplication in FANCA. Three patients carried a PGV in two genes: BRCA1 + CHEK2 (CNV); BRIP1 + MUTYH; CDKN2A + MITF (Supplementary Table 1).

### Variants of unknown significance (VUS)

Among all patients, 129 were VUS carriers only, 32 were VUS and PGV carriers (Fig. 1), 26 were neither PGV nor VUS carriers. Five out of 37 patients carrying PGVs did not present any VUS. Among the VUS carriers, 56 presented only one VUS and 104 presented more than one VUS (Supplementary Fig. 2).Fig. 1Solar explosion graph depicting spectrum of PGVs and VUSs in pancreatic cancer patients. Inner circle: Number of patients who are PGV carriers, VUS only carriers, neither PGV nor VUS carriers in the panel of genes (wild type); Medium circle: list of PGV affected genes in 37 participants; Outer circle: spectrum of VUS in the panel of genes in PGV carriers and non-carriers. *One patient who was a PGV carrier in CHEK2 and one patient who was a PGV carrier in ATM presented no VUS, but were considered together with the other patients who were PGV carriers in CHEK2 and ATM and were also VUS carriers.

Solar explosion graph depicting spectrum of PGVs and VUSs in pancreatic cancer patients. Inner circle: Number of patients who are PGV carriers, VUS only carriers, neither PGV nor VUS carriers in the panel of genes (wild type); Medium circle: list of PGV affected genes in 37 participants; Outer circle: spectrum of VUS in the panel of genes in PGV carriers and non-carriers. *One patient who was a PGV carrier in CHEK2 and one patient who was a PGV carrier in ATM presented no VUS, but were considered together with the other patients who were PGV carriers in CHEK2 and ATM and were also VUS carriers.

Nine patients presented more than one VUS in distinct positions of the following genes: ATM, BRIP1, EPCAM, FANCA, FANCF, GALNT12, PMS2, POLE and SLX4. Six VUS were characterized as splice site: FANCA c.3349-3C>T, FLCN c.249 + 2C>T, MSH3 c.3130+3A>G, LZTR1 c.1943-3C>T, RAD51B c.84+3Adel and FANCL c.541-3C>T; one VUS as frameshift: FANCL c.1099_1100insATTA; and one VUS as start loss: RAD51C c.3G>A.

### Loss of heterozygosity (LOH) and mutational signatures

To assess the contribution of certain pathogenic/likely pathogenic variants (ACD c.608_609delCT (plus RAD51C c.3G>A, a variant of uncertain significance), BRIP1 c.290_293del, MRE11 c.350A>G, POLE c.3721G>T, SDHA c.666delG, and TERF2IP c.1152_1153del) to pancreatic cancer risk, we conducted whole exome sequencing on DNA extracted from formalin-fixed paraffin-embedded (FFPE) tumor specimens of patients carrying these variants. Our analysis focused on evaluating loss of heterozygosity (LOH) and mutational signatures.

MRE11, POLE, RAD51C variants were detected in heterozygosity in tumor samples with VAFs of 0.539, 0.474 and 0.545, respectively. ACD, BRIP1 and SDHA VAFs were 0.645, 0.636, and 0.60, which is in or next to the lower limit of the cut off value considered for loss of heterozygosity, described in methods. We also used LOHGIC12 to estimate LOH occurrence, however we obtained no significant results for the variants tested (Akaike Information Criteria (AIC) weight < 0.7) (Supplementary table 3). TERF2IP variant could not be further evaluated due to low coverage in this gene region. Additionally, we checked for LOH, using the tools CNVkit13 and PureCN14. PureCN algorithm allows accurate estimates of copy number and LOH, while accounting for tumor purity and ploidy. PureCN can be integrated with common segmentation tools outputs, such as CNVkit, that combines both on– and off-target reads, to provide copy ratio estimates. This analysis could be performed for tumor samples from carriers of MRE11 and POLE variants, however, there was no definite evidence of LOH (Supplementary table 3).

To further evaluate the role of the above variants, we explored the mutational signature of tumor whole exome sequencing data. For this, we used the algorithm Signature Fit Multi-Step (FitMS) from the signatures.tools.lib R package, to detect common organ-specific signatures for pancreas in exome sequencing data15. The main signature detected in all samples was Single base substitutions (SBS) SBS1+5+18. Signatures SBS1 and SBS3+8 were also detected in all tumor samples (Fig. 2; Supplementary Fig. 3).Fig. 2Single Base Substitution (SBS) signatures across whole-exome-sequence tumor samples. Number of single nucleotide variants (SNVs) associated with each SBS in tumor samples of carriers of PGVs in MRE11 c.350A>G (PC32); ACD c.608_609delCT (plus RAD51C c.3G>A) (PC59), TERF2IP c.1152_1153del (PC67); SDHA c.666delG (PC109); POLE c.3721G>T (PC110); BRIP1 c.290_293del (PC115). PC: patient sample identification.

Single Base Substitution (SBS) signatures across whole-exome-sequence tumor samples. Number of single nucleotide variants (SNVs) associated with each SBS in tumor samples of carriers of PGVs in MRE11 c.350A>G (PC32); ACD c.608_609delCT (plus RAD51C c.3G>A) (PC59), TERF2IP c.1152_1153del (PC67); SDHA c.666delG (PC109); POLE c.3721G>T (PC110); BRIP1 c.290_293del (PC115). PC: patient sample identification.

### Genetic ancestry

We estimated ancestry admixture proportions using 46 ancestry-informative markers (AIMs) of small insertion-deletion (INDEL) polymorphisms11. Initially, we identified seven individuals with predominant East Asian (EAS) ancestry (using Admixture k = 4 analysis). We excluded EAS from further analysis, to avoid bias due to the close relationship between EAS and Indigenous (Indig) populations. Subsequently, a k = 3 analysis revealed a predominance of European genetic composition and the following mean ancestry proportions: EUR: 71.75% (SD 19.26), AFR: 19.79% (SD: 18.70), Indig: 8.46% (SD: 6.64) (Supplementary Fig. 4A).

In this cohort, 62 patients presented a European genetic ancestry predominance (EUR_pred) and 122, an admixed genetic composition, using criteria described in the methods. In the category of admixed ancestry composition, 95 individuals were classified as admixed with EUR predominance (ADM-EUR); 13, as admixed with AFR predominance (ADM-AFR) and 14, as admixed with no predominance (ADM-adm). Only one individual was categorized with an African genetic ancestry predominance (AFR_pred) (Supplementary Fig. 4B).

Considering self-declared skin color (white, brown, yellow and black), although there was some admixture in individual genetic ancestry composition (Fig. 3), there was a positive association with the main genetic ancestry (p < 0.001) (Supplementary Fig. 5).Fig. 3Individual genetic ancestry estimates of Brazilian pancreatic cancer patients in accordance with self-declared skin color. Patients self-declared skin color appears in the X axis. Genetic ancestry proportion estimates appear on the Y axis.

Individual genetic ancestry estimates of Brazilian pancreatic cancer patients in accordance with self-declared skin color. Patients self-declared skin color appears in the X axis. Genetic ancestry proportion estimates appear on the Y axis.

PGVs were detected in 22 out of 122 (18.00%) individuals with admixed ancestry composition and in 14 out of 62 (22.58%) patients with European genetic ancestry predominance (EUR_pred) (OR: 0.7543; 95% CI 0.3551–1.6021; p = 0.4631).

The mean number of VUS according with genetic ancestry were 1.64 and 1.86 for EUR_pred and admixed people, respectively. There was no difference in the number of VUS among EUR_pred and the three groups of admixed people (p = 0.509; Supplementary table 2).

### Survival

Among all patients, one was detected with carcinoma in situ, after surgery, and was excluded from survival analysis. The median overall survival was 16.46 months (95% CI 12.24–20.68). There was no difference in OS for PGV carriers versus non-carriers, considering: all genes; genes associated with PC, homologous repair-related (HRR) genes; all genes in the admixed population (Fig. 4); BRCA1, BRCA2 and PALB2 genes, all genes in patients with early-stage disease and patients with advanced disease (Supplementary Fig. 6). There were differences in overall survival among patients based on their disease clinical stage and body mass index (BMI). (Supplementary Fig. 6).Fig. 4Overall survival estimates in PGV carriers and non-carriers. (A) All pancreatic cancer (PC) divided in PGV carriers in one of the 24 affected genes (P/LP variants present) and non-carriers (P/LP variants absent); (B) PC patients divided in homologous recombination repair (HRR) PGV carriers and non-carriers; (C) PC patients divided in PGV carriers in genes associated with pancreatic cancer (GAPC) and non-carriers; (D) Patients with non-European ancestry (admixed) PGV carriers and non-carriers. Abs: P/LP variants absent; Pres: P/LP variants present; CS: clinical stage.

Overall survival estimates in PGV carriers and non-carriers. (A) All pancreatic cancer (PC) divided in PGV carriers in one of the 24 affected genes (P/LP variants present) and non-carriers (P/LP variants absent); (B) PC patients divided in homologous recombination repair (HRR) PGV carriers and non-carriers; (C) PC patients divided in PGV carriers in genes associated with pancreatic cancer (GAPC) and non-carriers; (D) Patients with non-European ancestry (admixed) PGV carriers and non-carriers. Abs: P/LP variants absent; Pres: P/LP variants present; CS: clinical stage.

### Discussion

In a predominantly miscegenated cohort of 192 unselected Brazilian pancreatic cancer patients, 37 (19.27%) were PGV carriers in a wide spectrum of genes, among whom, 12 (6.2%) in genes associated with pancreatic cancer and actionable for pancreatic cancer screening. Another 25 patients (13.0%) were PGV carriers in genes with limited association or genes not previously associated with pancreatic cancer. This observed high percentage of PGVs in various genes may be due to the large gene panel tested (113 genes). In agreement, in other cohorts of unselected PC patients, an increasing percentage of PGVs was detected by more comprehensive panels of genes12–15.

In the present study, the most frequently affected genes were ATM and CHEK2. ATM was among the top five most frequently affected genes in patients from Belgium, North America, and Japan, while CHEK2 was prominently affected in pancreatic cancer patients from Belgium and the Czech Republic15–19. Despite that, there are studies suggesting that pancreatic cancer risk is only modest or that pancreatic cancer may be less frequently detected in CHEK2 PGV carriers20,21.

Our study shows a differential feature compared to studies reporting BRCA2 as the most frequently affected gene in PC patients3. In these studies, a higher prevalence of BRCA2 and BRCA1 PGVs was mainly attributed to the contribution of Ashkenazi Jewish people, however, in the present series, no patient reported AJ ancestry. The detection of PGVs in BRCA1 and BRCA2 in 1.6%, and PALB2 in 1.0% of the patients have clinical implications, because PARP inhibitors are indicated for BRCA and for PALB2 pathogenic variant n carriers with metastatic disease or in certain conditions, respectively22.

We identified some PGVs in other genes classified as cancer causing genes in the Cancer Gene Census database, such as BLM (also named RECQL3), BRIP1 (also named FANCJ), ERCC4, FANCA, FANCE, MITF, MUTYH, POLE, RAD51B, RECQL4, and SDHA.

Twenty-eight patients were PGV carriers in 14 genes associated with homologous repair, including ATM, BRCA1, BRCA2, PALB2, BLM, BRIP1, CHEK2, FANCA, FANCE, FANCM, MRE11, RAD51B, RAD51C, and RECQL4. Corroborating our findings, although rare, PGVs in RAD51B, and RAD51C, as well as the mentioned genes, were previously detected in pancreatic cancer patients, as described ahead13–23. Biallelic loss of BLM is associated with the Bloom syndrome, which predisposes to an increased risk of cancer. Rare cases of pancreatic cancer associated with BLM PGVs were described, one in a young patient of Ashkenazi Jewish ancestry and one in a Canadian patient with pancreatic cancer family history24,25. In addition, biallelic MRE11A and RECQL4 deficiency are associated with ataxia-telangiectasia-like disorder and Rothmund-Thomson syndrome, respectively, and increased cancer risk. Although very rare, monoallelic MRE11 and RECQL4 PGVs were already detected in pancreatic cancer patients23,26, 27. Despite that, a recent meta-analysis could not find an association of MRE11 monoallelic disruption with pancreatic cancer risk28.

We detected PGVs in various genes involved in other DNA repair pathways, such as mismatch repair, DNA interstrand crosslinks (ICLs) repair, nucleotide excision repair (NER), and base excision repair (BER).

A pathogenic MSH2 nonsense variant was detected in one patient fulfilling the Amsterdam criteria for Lynch syndrome. Accordingly, MSH2 is the most frequently affected mismatch repair gene in Lynch syndrome families reporting PC cases29. In another patient, a VUS was detected in MSH3 (c.3130+3A>G). A similar VUS was described in a colorectal cancer patient, where tumor analysis revealed an absence of microsatellite instability (MSI)30. Additionally, MSH3 PGVs were not associated with increased OR of cancer and the most common mechanism for a loss-of-function phenotype is not genetic31.

Five patients presented PGVs in genes of the Fanconi Anemia pathway, which is associated with cancer predisposition and involved in ICL and in interaction with the HR32. Truncating FANCA, FANCE and FANCM variants were already detected in breast or pancreatic cancer patients and the variant FANCM c.2586_2589del, detected in one of our patients, was described in a breast cancer patient27,33–36., In addition, there are functional studies demonstrating that the variant FANCE R371W (c.1111C>T), detected in one of our patients, destabilizes the ternary structure of the protein, preventing its interaction with FANCD237. Besides that, the following VUS were detected: FANCL c.1099-1100insATTA, located in the last exon of the gene; FANCL c.541-3C>T and FANCA c.3349-3C>T, both affecting a nucleotide within the consensus splice site, with no functional studies evaluating the effect in RNA splicing. Despite that, FANCA c.3349-3C>T was previously described in a patient with myelodysplasia and acute myeloid leukemia38.

ERCC4 encodes an endonuclease involved in NER and ICL. ERCC4 variants were previously described in pancreatic cancer patients14,39 and ERCC4 c.584+1G>A, found in the present study, was described in familial breast and ovarian cancer, resulting in exon 3 skipping40,41. MUTYH encodes a DNA glycosylase involved in BER, and biallelic pathogenic variants in MUTYH are associated with polyposis and increased risk of colorectal cancer. The risk of cancer in MUTYH monoallelic variant carriers is uncertain and a positive contribution to cancer causality was observed by some authors but not by others42,43.

POLE encodes a DNA polymerase involved in DNA replication and proofreading. Characteristic PGVs in POLE are missense variants in the exonuclease domain, mainly detected in families with colorectal syndrome. Besides that, there are also descriptions of an affected POLE in Taiwanese pancreatic cancer patients and in families with a broader tumor spectrum, including pancreatic cancer44,45. POLE c.3721G>T, found in the present series, gives rise to a stop codon outside the exonuclease domain. In accordance, a frameshift variant outside the exonuclease domain was recently reported in a Taiwanese pancreatic cancer patient27. In addition, a few truncating variants were previously described outside the exonuclease domain, in tumors from patients who were responsive to immune checkpoint inhibitors. These findings indicate that these types of germline variants in POLE may also be therapy targets46.

MITF c.952G>A, detected in two patients, was previously related to a genetic predisposition to melanoma and renal cell carcinoma47. The same gene variant was also reported in at least five pancreatic cancer patients16,48, however, MITF c.952G>A is reasonably frequent in ExAc. Despite that, in vitro tests indicate that this variant enhances melanocytic and renal cell clonogenicity, migration, and invasion consistent with a gain-of-function role in tumorigenesis47,49.

SDHA encodes a protein involved in the mitochondrial respiratory chain that acts as a tumor suppressor gene50 in hereditary pheochromocytoma–paraganglioma. The variant found in one of the patients, SDHA c.666delG, was previously reported in a patient with paraganglioma and carotid body tumor51, while other SDHA variants were rarely reported in PC patients13,14.

Frameshift variants were also detected in genes that encode proteins of the shelterin complex, involved in telomere maintenance and recruitment of telomerase to the telomeres (ACD) or in the regulation of telomere length and protection (TERF2IP). Protein truncating variants in ACD or TERF2IP were mainly described in familial melanoma, but also in patients with families reporting other types of cancers, such as colorectal, breast, pancreatic, and leukemia, indicating that these genes may be involved in a broader spectrum of cancers51,52. Finally, we detected a truncating variant in GALNT12, which may be a gene with moderate penetrance for colorectal cancer, but without evidence of involvement in pancreatic cancer, to the present moment53.

Identification of loss of heterozygosity (LOH) in a tumor sample can aid in classifying variants in tumor suppressor genes, as LOH is a mechanism that can lead to biallelic inactivation of these genes. In the present series, we could not confirm a pathogenic role for the variants ACD c.608_609delT, BRIP1 c.290_293del, MRE11 c.350A>G, POLE c.3721G>T, RAD51C c.3G>A and SDHA c.666delG. However, some factors may have precluded a precise classification of these variants, such as limited variant read depth in the tumor samples and absence of paired germline exome sequencing11,54, 55.

We observed that the main signature in the six above-mentioned tumor samples was SBS1+5+18, followed by signatures SBS1 and SBS3+8. These are common signatures previously described in pancreatic cancer samples, that reflect pancreatic cancer causes56. Signature SBS1 is related with the individual´s age, SBS5 has unknown etiology, but is related to age and is increased in cancers attributed to tobacco smoking, such as lung, bladder, esophagus and pancreas, and SBS18, possibly reflects reactive oxygen damage57. Signature SBS3 indicates a homologous repair deficiency (HRD), while SBS8 is likely caused by uncorrected errors from late replication during cancer progression58. Both SBS3 and SBS8 contribute to HRDetect, a model developed to detect BRCA1/BRCA2-deficient samples. Despite this, HRD was not the predominant signature in tumor samples from carriers of PGVs in BRIP1, MRE11, and RAD51C, which are genes involved in homologous repair. Hence, we were not able to confirm a role for these genes in tumor development, through HRD. A limitation of this analysis is that single base substitution mutational signatures were trained using whole-genome data. As a result, while the analysis can be applied to exome samples, the results may exhibit lower cosine similarity and be less robust56.

There was a great genetic ancestry admixture in this Brazilian cohort, with a predominance of European genetic ancestry and a small contribution of Indigenous ancestry, coherent with other studies performed in Brazilian breast and colorectal cancer patients59,60. Comparing our patients with European predominance ancestry with admixed ancestry, there was neither a differential prevalence of PGVs nor VUS. These results are in accordance with previous reports. In seven studies reporting 131 black/African American and 92 Hispanic/Latino pancreatic cancer patients, 5.3% and 10.9% were PGV carriers, respectively15,18, 61–65. In the largest cohort of patients, 92 (3.0% of the total) were African American or Hispanic and the prevalence of PGVs was similar between African American, Hispanic and non-Hispanic white people19. In these studies, 16 VUS, including RAD51B c.25G>A, BAP1 c.358A>G66,67and TP53 c.467G>A68–72 were also detected in accordance with the present series (Supplementary table 2).

In this cohort, 35.4% of patients were diagnosed with metastatic disease, which is likely lower than the rates reported in other studies73,74. Additionally, the median overall survival was 16.5 months, higher than that reported for pancreatic carcinoma patients in some studies73–75. The higher proportion of patients with clinical stages II and III disease may be partly due to the recruitment approach: most participants were either already undergoing treatment or under surveillance at the time of their invitation to the study, rather than being newly diagnosed. Additionally, the median time between diagnosis and study inclusion was 105 days, with nearly 20% of the patients having more than a year of follow-up before inclusion. These factors may have contributed to the selection of patients with improved overall survival. In our study, there was no association between PGV status and overall survival, consistent with previous reports15,16. Furthermore, there was no significant difference in survival between PGV carriers and non-carriers, regardless of disease stage.

Our study has some limitations. We included patients with confirmed histopathological diagnosis of cancer but we missed patients with suspected PC, regarding signs, symptoms and radiological images, whose condition precluded biopsy. Another limitation was the absence of tumor tissue to evaluate mRNA splicing. In addition, although family history was prospectively collected, patient recall was poor. Furthermore, patients were included in a single Brazilian institution (ICESP). Despite that, 64% were born in the Southeast and 29% in the Northeast country regions, which are the most populated, home to 41.7% and 26.7% of the Brazilian people, respectively76. ICESP is a public cancer care unit, integrated to Hospital das Clinicas da Faculdade de Medicina da Universidade de Sao Paulo, which is the largest hospital complex in South America. In previous studies carried out at ICESP, we observed a similar distribution, with 50% and 30–40% patients born in the Southeast and Northeast Brazilian regions, respectively77,78. In addition, genetic ancestry of our patients reproduced data previously reported for Brazilian people8,59, 60. Thus, we consider we had a significant representation of the Brazilian population. Among the strengths of this study, we may mention the prospective accrual of a cohort of PC patients with admixed ancestry, unselected for family history of cancer, with broad stage distribution, evaluated through a large panel of genes.

In conclusion, in our study, genetic ancestry did not affect the detection rate of PGVs in pancreatic cancer patients. Although most PGVs affected homologous repair genes, we also report truncating variants in genes with functions not classically associated with pancreatic cancer predisposition, including ACD, TERF2IP, GALNT12 and SDHA, that deserve future attention. To our knowledge, this is the first study evaluating pancreatic cancer hereditary predisposition in a Brazilian cohort. These results further support that NCCN recommendations for indicating genetic testing for all patients with pancreatic carcinoma apply to Brazilian patients, regardless of genetic ancestry.

### Materials and Methods

This is a cross-sectional single-center study that took place at Instituto do Câncer do Estado de São Paulo (ICESP), São Paulo, Brazil. This study was approved by the Institutional Research Ethics Committee (CAAE 85573518.1.0000.0065) and followed the principles of the Helsinki Declaration. All patients signed the informed consent.

Inclusion criteria were: age ≥ 18 years and confirmed histopathological or clinical diagnosis of non-endocrine pancreatic carcinoma, irrespective of family history of cancer (FHC), clinical stage or treatment. Clinical diagnosis consisted of pancreatic tumor visualized through computerized tomography or magnetic resonance, accompanied by a histopathological report of the primary tumor or a metastatic lesion, indicating carcinoma. Clinical or pathological staging (the latter, for patients who were operated on) was based on the American Joint Committee on Cancer (AJCC) TNM staging 8th edition, 201722.

We prospectively applied a questionnaire evaluating personal and family history of cancer (relatives until 3rd degree), alcohol intake (heavy drinking defined as alcohol intake ≥ 30 g/day for males and ≥ 15 g/day for females; drinking, as consumption of less than the pre-specified values; and social drinking, as unspecified drinking amount in weekends and festivities)79,80, tobacco use (heavy smoking defined as smoking ≥ 20 packs years)81, diabetes and self-reported skin color [IBGE—White, Black, Yellow, Indigenous, Brown (also called Pardo)]6.

Libraries were prepared from 500 ng DNA fromblood mononuclear cells or 99–250 ng DNA from FFPE tumor specimens, using the Illumina DNA Prep with Enrichment and IDT for Illumina DNA/RNA UD Indexes. For germline analysis, hybridization and capture utilized TruSight Hereditary Cancer Panel (Illumina), targeting 113 cancer predisposition genes. DNA sequencing was performed on MiSeq and NextSeq platforms. For somatic analysis, the Exome 2.0 Plus Panel (Illumina) was used for on-bead tagmentation library preparation, followed by hybrid-capture exome enrichment and sequencing on the NovaSeq platform. Runs with quality metrics Q30 > 90% and cluster pass filter > 80%, obtained from BaseSpace (Illumina), were further analyzed.

Base calling was performed in BaseSpace and analyzed using Varstation Platform (VarsOmics, version 2.0; https://varsomics.com/varstation/), mapping according to GRCh37/UCSC hg19 using BWA-MEM, variants call generated by GATK HaplotypeCaller (https://gatk.broadinstitute.org/hc/en-us) and annotation by Varstation.

Variants selected had read depth ≥ 30x, variant allele frequency (VAF) ≥ 0.30, and location in exonic or canonical splice site (± 3) regions. Benign/likely benign variants according to ClinVar database (www.clinvar.com/, accessed March/2023) were excluded. Synonymous, novel (non-deposited in Clinvar) and variants of uncertain significance (VUS) were analyzed in the Ensembl Variant Effect Predictor (VEP)82. Interpretation followed the American College of Medical Genetics and Genomics83.

The presence copy number variations (CNVs) was investigated using the CNV-Atlas tool84 followed by Multiplex Ligation-Dependent Probe Amplification (MLPA) assay, to confirm results (Supplementary Material).

FASTQ files obtained from sequencing six DNA samples derived from FFPE tumor tissues and paired mononuclear cells from patients carrying the variants ACD c.608_609delCT (plus RAD51C c.3G>A, a variant of uncertain significance), BRIP1 c.290_293del, MRE11 c.350A>G, POLE c.3721G>T, SDHA c.666delG, and TERF2IP c.1152_1153del) were aligned to the GRCh38 genome using BWA-MEM v0.7.11. Duplicate reads were marked and removed with Picard, and variant recalibration was conducted using GATK 4.2.2. Variant calling was performed with Mutect2 in tumor-only mode, and variants were filtered using the default parameters of FilterMutectCalls. Variant annotation was carried out using the online Variant Effect Predictor (VEP) from Ensembl, and germline variants were identified within the somatic samples. Variants of interest were inspected using the Integrative Genomics Viewer (IGV).

Loss of heterozygosity (LOH) for the wild-type allele was considered to have occurred if the variant allele frequency (VAF) exceeded 60%, based on a model where VAF = 100/(% tumor cells + 2 × % normal cells). This model assumes that in hereditary syndromes, the deletion of the wild-type allele occurs early and is therefore detected in most, if not all, tumor cells85. We also used LOH-Germline Inference Calculator (LOHGIC, available at http://www.khiabanian-lab.org/pages/lohgic.html) to assess somatic variants to estimate LOH occurrence in cancer cells. This model considers that variant allele frequency in hybrid capture DNA sequencing depends on depth of sequencing, tumor ploidy and tumor purity, and Akaike Information Criteria (AIC, used to compare likelihoods) weight > 0.7 indicates unambiguous prediction11.

LOH analysis was also performed using the CNVkit (v.0.9.10) Python package54 and PureCN (v.2.8.1) R package55. Sequence-accessible coordinates were extracted with the access CNVkit function. In the absence of normal exome paired samples, a flat reference file was obtained with the reference function. GC content, sequence repeats and target density bias correction were performed with the CNVkit fix function. Finally, the package was used to extract a segmentation file, using the cbs (Circular Binary Segmentation) method. LOH estimation was performed with the PureCN function from the PureCN package, using the Hclust method to perform segment clustering.

The signature.tools.lib R package (v.2.4.4) was used to extract mutational signatures, following the methodology previously outlined56. Signatures were calculated using the fitMS (Multi-Step Mutational Signatures Fit) function, with 'Pancreas' specified as the tissue type. Estimates were based on data from 367 pancreatic tumors analyzed by the International Cancer Genome Consortium (ICGC), Hartwig Medical Foundation (HMF), and the Genomics England (GEL) 100,000 Genomes Project. Cosine similarity values varied from 0.77 to 0.83.

To estimate ancestry and admixture proportions we used 46 ancestry-informative markers (AIMs) of small insertion-deletion (INDEL) polymorphisms, previously shown to distinguish four continental population groups (African (AFR), European (EUR), East Asian (EAS) and Indigenous (Indig))86 (Supplementary table 4). Inference of ancestry was performed with Structure v2.3.4 software, (Supplementary Material).

We applied the cut-off value of the third tercile of the EUR ancestry proportion, that in this cohort corresponded to 85%, to categorically divide the whole cohort of patients in EUR predominance (EUR_pred: ≥ 85% European ancestry proportion) and non-EUR predominance. The non-EUR predominance group was further categorized in admixed with EUR predominance (ADM-EUR: 50% ≤ x < 85% European ancestry proportion), admixed with AFR predominance (ADM-AFR: 50% ≤ x < 85% African ancestry proportion) and admixed with no predominance (ADM-adm). To categorize AFR predominance, we used the same cut off value established for EUR predominance (AFR_pred: ≥ 85% African ancestry proportion).

Associations were analyzed through chi-square or Fisher’s exact tests. Overall survival (OS) was calculated from date of histopathology diagnosis until date of death or date last known alive. Kaplan–Meier curves and log-rank tests were used to estimate overall survival and to compare survival curves, respectively. The significance level was 5%. Statistical analyses were performed using SPSS for Windows v.25 and STATA/MP 14.0 for Windows.

### Patients

This is a cross-sectional single-center study that took place at Instituto do Câncer do Estado de São Paulo (ICESP), São Paulo, Brazil. This study was approved by the Institutional Research Ethics Committee (CAAE 85573518.1.0000.0065) and followed the principles of the Helsinki Declaration. All patients signed the informed consent.

Inclusion criteria were: age ≥ 18 years and confirmed histopathological or clinical diagnosis of non-endocrine pancreatic carcinoma, irrespective of family history of cancer (FHC), clinical stage or treatment. Clinical diagnosis consisted of pancreatic tumor visualized through computerized tomography or magnetic resonance, accompanied by a histopathological report of the primary tumor or a metastatic lesion, indicating carcinoma. Clinical or pathological staging (the latter, for patients who were operated on) was based on the American Joint Committee on Cancer (AJCC) TNM staging 8th edition, 201722.

We prospectively applied a questionnaire evaluating personal and family history of cancer (relatives until 3rd degree), alcohol intake (heavy drinking defined as alcohol intake ≥ 30 g/day for males and ≥ 15 g/day for females; drinking, as consumption of less than the pre-specified values; and social drinking, as unspecified drinking amount in weekends and festivities)79,80, tobacco use (heavy smoking defined as smoking ≥ 20 packs years)81, diabetes and self-reported skin color [IBGE—White, Black, Yellow, Indigenous, Brown (also called Pardo)]6.

### Next-generation sequencing (NGS)

Libraries were prepared from 500 ng DNA fromblood mononuclear cells or 99–250 ng DNA from FFPE tumor specimens, using the Illumina DNA Prep with Enrichment and IDT for Illumina DNA/RNA UD Indexes. For germline analysis, hybridization and capture utilized TruSight Hereditary Cancer Panel (Illumina), targeting 113 cancer predisposition genes. DNA sequencing was performed on MiSeq and NextSeq platforms. For somatic analysis, the Exome 2.0 Plus Panel (Illumina) was used for on-bead tagmentation library preparation, followed by hybrid-capture exome enrichment and sequencing on the NovaSeq platform. Runs with quality metrics Q30 > 90% and cluster pass filter > 80%, obtained from BaseSpace (Illumina), were further analyzed.

### Variant detection and classification

Base calling was performed in BaseSpace and analyzed using Varstation Platform (VarsOmics, version 2.0; https://varsomics.com/varstation/), mapping according to GRCh37/UCSC hg19 using BWA-MEM, variants call generated by GATK HaplotypeCaller (https://gatk.broadinstitute.org/hc/en-us) and annotation by Varstation.

Variants selected had read depth ≥ 30x, variant allele frequency (VAF) ≥ 0.30, and location in exonic or canonical splice site (± 3) regions. Benign/likely benign variants according to ClinVar database (www.clinvar.com/, accessed March/2023) were excluded. Synonymous, novel (non-deposited in Clinvar) and variants of uncertain significance (VUS) were analyzed in the Ensembl Variant Effect Predictor (VEP)82. Interpretation followed the American College of Medical Genetics and Genomics83.

### Germline variants

Base calling was performed in BaseSpace and analyzed using Varstation Platform (VarsOmics, version 2.0; https://varsomics.com/varstation/), mapping according to GRCh37/UCSC hg19 using BWA-MEM, variants call generated by GATK HaplotypeCaller (https://gatk.broadinstitute.org/hc/en-us) and annotation by Varstation.

Variants selected had read depth ≥ 30x, variant allele frequency (VAF) ≥ 0.30, and location in exonic or canonical splice site (± 3) regions. Benign/likely benign variants according to ClinVar database (www.clinvar.com/, accessed March/2023) were excluded. Synonymous, novel (non-deposited in Clinvar) and variants of uncertain significance (VUS) were analyzed in the Ensembl Variant Effect Predictor (VEP)82. Interpretation followed the American College of Medical Genetics and Genomics83.

### Exonic amplifications and deletions

The presence copy number variations (CNVs) was investigated using the CNV-Atlas tool84 followed by Multiplex Ligation-Dependent Probe Amplification (MLPA) assay, to confirm results (Supplementary Material).

### Somatic analysis

FASTQ files obtained from sequencing six DNA samples derived from FFPE tumor tissues and paired mononuclear cells from patients carrying the variants ACD c.608_609delCT (plus RAD51C c.3G>A, a variant of uncertain significance), BRIP1 c.290_293del, MRE11 c.350A>G, POLE c.3721G>T, SDHA c.666delG, and TERF2IP c.1152_1153del) were aligned to the GRCh38 genome using BWA-MEM v0.7.11. Duplicate reads were marked and removed with Picard, and variant recalibration was conducted using GATK 4.2.2. Variant calling was performed with Mutect2 in tumor-only mode, and variants were filtered using the default parameters of FilterMutectCalls. Variant annotation was carried out using the online Variant Effect Predictor (VEP) from Ensembl, and germline variants were identified within the somatic samples. Variants of interest were inspected using the Integrative Genomics Viewer (IGV).

### Loss of heterozygosity

Loss of heterozygosity (LOH) for the wild-type allele was considered to have occurred if the variant allele frequency (VAF) exceeded 60%, based on a model where VAF = 100/(% tumor cells + 2 × % normal cells). This model assumes that in hereditary syndromes, the deletion of the wild-type allele occurs early and is therefore detected in most, if not all, tumor cells85. We also used LOH-Germline Inference Calculator (LOHGIC, available at http://www.khiabanian-lab.org/pages/lohgic.html) to assess somatic variants to estimate LOH occurrence in cancer cells. This model considers that variant allele frequency in hybrid capture DNA sequencing depends on depth of sequencing, tumor ploidy and tumor purity, and Akaike Information Criteria (AIC, used to compare likelihoods) weight > 0.7 indicates unambiguous prediction11.

LOH analysis was also performed using the CNVkit (v.0.9.10) Python package54 and PureCN (v.2.8.1) R package55. Sequence-accessible coordinates were extracted with the access CNVkit function. In the absence of normal exome paired samples, a flat reference file was obtained with the reference function. GC content, sequence repeats and target density bias correction were performed with the CNVkit fix function. Finally, the package was used to extract a segmentation file, using the cbs (Circular Binary Segmentation) method. LOH estimation was performed with the PureCN function from the PureCN package, using the Hclust method to perform segment clustering.

### Mutational signatures

The signature.tools.lib R package (v.2.4.4) was used to extract mutational signatures, following the methodology previously outlined56. Signatures were calculated using the fitMS (Multi-Step Mutational Signatures Fit) function, with 'Pancreas' specified as the tissue type. Estimates were based on data from 367 pancreatic tumors analyzed by the International Cancer Genome Consortium (ICGC), Hartwig Medical Foundation (HMF), and the Genomics England (GEL) 100,000 Genomes Project. Cosine similarity values varied from 0.77 to 0.83.

### Genomic ancestry analysis

To estimate ancestry and admixture proportions we used 46 ancestry-informative markers (AIMs) of small insertion-deletion (INDEL) polymorphisms, previously shown to distinguish four continental population groups (African (AFR), European (EUR), East Asian (EAS) and Indigenous (Indig))86 (Supplementary table 4). Inference of ancestry was performed with Structure v2.3.4 software, (Supplementary Material).

We applied the cut-off value of the third tercile of the EUR ancestry proportion, that in this cohort corresponded to 85%, to categorically divide the whole cohort of patients in EUR predominance (EUR_pred: ≥ 85% European ancestry proportion) and non-EUR predominance. The non-EUR predominance group was further categorized in admixed with EUR predominance (ADM-EUR: 50% ≤ x < 85% European ancestry proportion), admixed with AFR predominance (ADM-AFR: 50% ≤ x < 85% African ancestry proportion) and admixed with no predominance (ADM-adm). To categorize AFR predominance, we used the same cut off value established for EUR predominance (AFR_pred: ≥ 85% African ancestry proportion).

### Statistical analysis

Associations were analyzed through chi-square or Fisher’s exact tests. Overall survival (OS) was calculated from date of histopathology diagnosis until date of death or date last known alive. Kaplan–Meier curves and log-rank tests were used to estimate overall survival and to compare survival curves, respectively. The significance level was 5%. Statistical analyses were performed using SPSS for Windows v.25 and STATA/MP 14.0 for Windows.

### Supplementary Information

Supplementary Information.Supplementary Table S1.

Supplementary Information.

Supplementary Table S1.



# SUPPLEMENTAL FILE 1: 41598_2024_Article_71884.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: PMC11387492

[File available at: automated_output/BRCA2/20251125_175029/pmc_fulltext/39256447_supplements/PMC11387492]

