# MAIN TEXT

## The discrepancy of somatic BRCA1/2 pathogenic variants from two different platforms in epithelial ovarian, fallopian tube, and peritoneal cancer

### Abstract

The somatic BRCA1 or BRCA2 Pathogenic Variant (PV)/Likely PV (LPV) from Next Generation Sequencing (NGS) is the most important biomarker for PARP inhibitor use and maintenance-targeted therapies. A discrepancy in the detection rates of BRCA1 and BRCA2 PV/LPV was identified among the NGS platforms. The objective of this study was to compare the somatic BRCA results from two distinct platforms using the same cohort and to identify the causes of these differences. Patients with epithelial ovarian cancer who concurrently underwent tumor NGS using two different platforms between January 2022 and June 2023 were included in this study. The two platforms used were in-house tumor NGS (Illumina NextSeq 550Dx, SureSelectXT library kit, and datasets from 1000 Genomes, ESP6500, ExAC, and ClinVar) and GreenPlan homologous recombination deficiency (HRD) test (Illumina NextSeq 550Dx, customized Twist Bioscience library kit, and datasets from COSMIC and OncoKB). The results of somatic mutations in BRCA1 and BRCA2 were compared between the two platforms. Of the 118 patients, 11.9% (n = 14) exhibited a discordant interpretation of BRCA1 or BRCA2 between the two platforms. Eleven patients (9.3%) exhibited negative results in the in-house platform but positive results (eight seven as PV of BRCA1, one as PV of BRCA2, one as LPV of BRCA1, and two as LPV of BRCA2) in the GreenPlan HRD test, while three patients (2.6%) had positive BRCA pathogenic variants (two as PV of BRCA1 [c.3340G > T, c.5152 + 3 A > C], one as LPV of BRCA2 [c.8174G > T], and one as LPV of BRCA1 [c.5017_5019delCAC]) in the in-house platform but a negative result in the GreenPlan HRD test. The discordance rate of somatic BRCA1 or BRCA2 mutations from different platforms was approximately 12%. In the case of the strong implication of BRCA PV/LPV with a negative result with one genetic test, different platforms could be considered in limited cases. Careful interpretation and further studies for the cross-validation of gene analysis platforms are needed. Supplementary InformationThe online version contains supplementary material available at 10.1038/s41598-024-75230-6.

### Introduction

Epithelial ovarian cancer (OC) is one of the most lethal malignancies, with an estimated 313,959 new cases and 207,252 deaths reported worldwide in 20201. The incidence of OC is steadily rising, with an annual percentage change of 1.7 age-standardized incidence rate per 100,000 in Korea2. Women with pathogenic variants in BRCA1 and BRCA2 genes have a cumulative lifetime risk of epithelial ovarian cancer of 40–50% in BRCA1 and 20–25% in BRCA2 by age 70–803,4. In particular, a pathogenic variant of BRCA1 is associated with the development of high-grade serous carcinoma compared to BRCA25.

The cornerstone treatment of epithelial OC is based on platinum-based frontline chemotherapy combined with paclitaxel after cytoreductive surgery with an effort of complete cytoreduction6,7. Currently, treatment strategies have evolved with the incorporation of poly (ADP-ribose) polymerase (PARP) inhibitors for patients with high-grade serous and endometrioid OC7,8. PARP inhibition and BRCA1 and BRCA2 mutations were found to interact with synthetic lethality, with accumulation of double-strand breaks in BRCA1 and BRCA2 deficient tumors due to persistent single-strand breaks caused by PARP inhibition9. Hence, to identify patients who benefit from using PARP inhibitors, the utilization of genetic information for breast cancer genes BRCA1 and BRCA2 mutations and homologous recombination deficiency has emerged as an essential approach7.

There is no consensus regarding genetic evaluations, but some guidelines recommend testing for germline BRCA (gBRCA), and somatic BRCA (sBRCA) testing is recommended for those who do not carry a germline BRCA1/2 pathogenic or likely pathogenic variant10,11. Somatic BRCA testing of tissues is mainly performed on formalin-fixed paraffin-embedded (FFPE) tissues, and complex procedures regarding the preparation of extracted tissues are one of the reasons for the failure of BRCA testing12. Furthermore, there are no standardized technologies or interpretations for BRCA testing of tumor tissues. In view of the existing challenges in tumor testing strategies for patients with epithelial OC, this study aimed to investigate the somatic BRCA results from two distinct platforms requested in parallel using the same cohort and identify the causes of these differences.

### Methods

This study was a retrospective analysis of the medical records of patients diagnosed with histologically confirmed epithelial ovarian cancer, fallopian cancer, or primary peritoneal cancer who underwent tumor NGS testing of the BRCA1 and BRCA2 genes between January 2022 and June 2023 in the National Cancer Center of Korea. The specific inclusion criteria were as follows:1) patients who underwent dual-testing of tumor NGS with different platforms, 2) patients who underwent germline testing from blood samples, and 3) patients whose histological subtype was previously identified. All patients had previously undergone germline testing of blood samples, and there were no discordant results between germline and somatic testing.

Germline BRCA NGS analysis for patients with epithelial OC at National Cancer Center Korea was launched at February 2018, replacing Sanger sequencing of BRCA genes. Tumor NGS testing was launched in December 2017, starting with the in-house NGS pan-cancer panel version.0. Tumor NGS was mostly performed after identification of the wild-type copy of germline BRCA1 and BRCA2 using a two-step strategy. After the GreenPlan® homologous recombination deficiency (HRD) test (GC Genome, Seoul, Korea) was approved in March 2022, dual in-house tumor NGS and HRD tests could be performed concurrently, or one of the two could be chosen.

The dual testing platforms used in the study were (1) in-house NGS Pan Cancer panel version.3 and (2) GreenPlan HRD test. A summary of the differences between the two platforms is provided in Table 1. DNA was extracted from archival FFPE tissues in both settings. Multiplex hybridization-based target enrichment preparation was performed using the SureSelectXT target enrichment protocol (Agilent Technologies, Carlsbad, CA) in in-house NGS, whereas the GreenPlan HRD test used a customized target enrichment kit from Twist Bioscience. NGS was performed using an Illumina NextSeq 550Dx sequencer (Illumina, San Diego CA, USA). The In-house NGS Pan Cancer panel version.3 covers all exons of 466 coding sequence genes, including BRCA1 and BRCA2, and the GreenPlan HRD test covers 28 HRR-related genes, including BRCA1 and BRCA2.

Table 1 Summary of discordant pathogenic variants identification.Platform nameGreenPlan HRDNGS PCP ver.3 (525)CompanyGC GenomeIn-houseTumor sample typeFormalin-fixed paraffin-embedded tissueFormalin-fixed paraffin-embedded tissueNGS PlatformIllumina® Nextseq 550DxIllumina® Nextseq 550DxLibrary KitTwist BioscienceAgilent(Custom Target Enrichment kit)(Sureselect Target Enrichment System kit)Reference databaseCOSMIC, OncoKB1000genome, clinvar, ESP6500, ExaC

Summary of discordant pathogenic variants identification.

Germline variants were analyzed based on the recently updated American College of Medical Genetics and Genomics (ACMG) and Association of Molecular Pathology (AMP) guidelines, which provide a five-tier classification system. For somatic variant interpretation, in-house NGS used four large-scale databases: 1000Genome Project, ClinVar, Exome Aggregation Consortium (ExAC), and Exome Sequencing Project v.6500 (ESP6500). GreenPlan HRD test referenced OncoKB, and Catalogue of Somatic Mutations in Cancer (COSMIC) which are one of the publicly available large-scale databases. The reference sequences were as follows: BRCA1 for NM_007294 and BRCA2 for NM_000059. This study considered pathological variants classified as ‘pathogenic’ or ‘likely pathogenic’ according to guidelines. Other reports considered wild-type BRCA. Statistical analysis was performed using Microsoft Excel 2021 and SankeyMATIC software (https://www.sankeymatic.com/).

This study complies with the Declaration of Helsinki and approved by the Institutional Review Board of the National Cancer Center, Korea (NCC2022-0314). Due to the retrospective nature of the study, Institutional Review Board of the National Cancer Center, Korea waived the need of obtaining informed consent.

The datasets used and/or analysed during the current study available from the corresponding author on reasonable request.

### Study population

This study was a retrospective analysis of the medical records of patients diagnosed with histologically confirmed epithelial ovarian cancer, fallopian cancer, or primary peritoneal cancer who underwent tumor NGS testing of the BRCA1 and BRCA2 genes between January 2022 and June 2023 in the National Cancer Center of Korea. The specific inclusion criteria were as follows:1) patients who underwent dual-testing of tumor NGS with different platforms, 2) patients who underwent germline testing from blood samples, and 3) patients whose histological subtype was previously identified. All patients had previously undergone germline testing of blood samples, and there were no discordant results between germline and somatic testing.

### Clinical evolution in BRCA testing strategy in single institution

Germline BRCA NGS analysis for patients with epithelial OC at National Cancer Center Korea was launched at February 2018, replacing Sanger sequencing of BRCA genes. Tumor NGS testing was launched in December 2017, starting with the in-house NGS pan-cancer panel version.0. Tumor NGS was mostly performed after identification of the wild-type copy of germline BRCA1 and BRCA2 using a two-step strategy. After the GreenPlan® homologous recombination deficiency (HRD) test (GC Genome, Seoul, Korea) was approved in March 2022, dual in-house tumor NGS and HRD tests could be performed concurrently, or one of the two could be chosen.

### Tumor BRCA1 and BRCA2 gene testing

The dual testing platforms used in the study were (1) in-house NGS Pan Cancer panel version.3 and (2) GreenPlan HRD test. A summary of the differences between the two platforms is provided in Table 1. DNA was extracted from archival FFPE tissues in both settings. Multiplex hybridization-based target enrichment preparation was performed using the SureSelectXT target enrichment protocol (Agilent Technologies, Carlsbad, CA) in in-house NGS, whereas the GreenPlan HRD test used a customized target enrichment kit from Twist Bioscience. NGS was performed using an Illumina NextSeq 550Dx sequencer (Illumina, San Diego CA, USA). The In-house NGS Pan Cancer panel version.3 covers all exons of 466 coding sequence genes, including BRCA1 and BRCA2, and the GreenPlan HRD test covers 28 HRR-related genes, including BRCA1 and BRCA2.

Table 1 Summary of discordant pathogenic variants identification.Platform nameGreenPlan HRDNGS PCP ver.3 (525)CompanyGC GenomeIn-houseTumor sample typeFormalin-fixed paraffin-embedded tissueFormalin-fixed paraffin-embedded tissueNGS PlatformIllumina® Nextseq 550DxIllumina® Nextseq 550DxLibrary KitTwist BioscienceAgilent(Custom Target Enrichment kit)(Sureselect Target Enrichment System kit)Reference databaseCOSMIC, OncoKB1000genome, clinvar, ESP6500, ExaC

Summary of discordant pathogenic variants identification.

### Interpretation of BRCA variants

Germline variants were analyzed based on the recently updated American College of Medical Genetics and Genomics (ACMG) and Association of Molecular Pathology (AMP) guidelines, which provide a five-tier classification system. For somatic variant interpretation, in-house NGS used four large-scale databases: 1000Genome Project, ClinVar, Exome Aggregation Consortium (ExAC), and Exome Sequencing Project v.6500 (ESP6500). GreenPlan HRD test referenced OncoKB, and Catalogue of Somatic Mutations in Cancer (COSMIC) which are one of the publicly available large-scale databases. The reference sequences were as follows: BRCA1 for NM_007294 and BRCA2 for NM_000059. This study considered pathological variants classified as ‘pathogenic’ or ‘likely pathogenic’ according to guidelines. Other reports considered wild-type BRCA. Statistical analysis was performed using Microsoft Excel 2021 and SankeyMATIC software (https://www.sankeymatic.com/).

### Ethical statement

This study complies with the Declaration of Helsinki and approved by the Institutional Review Board of the National Cancer Center, Korea (NCC2022-0314). Due to the retrospective nature of the study, Institutional Review Board of the National Cancer Center, Korea waived the need of obtaining informed consent.

### Data availability

The datasets used and/or analysed during the current study available from the corresponding author on reasonable request.

### Results

A total of 679 patients were included in the study based on the eligibility criteria (Fig. 1). Of the 679 patients, 325 underwent in-house tumor NGS testing, whereas 472 patients underwent the GreenPlan HRD test. Of the 118 patients who received both in-house platform and GreenPlan HRD tests concurrently, 20 (16.9%) were found to have pathogenic variants in BRCA1 or BRCA2. Discordant pathogenic variants were detected in 11.9% (14/118) of the patients. As presented in Fig. 2, somatic pathogenic BRCA variants were detected in 8 and 18 patients using the in-house platform and GreenPlan HRD test, respectively.Figure 1Flow diagram detailing next-generation sequencing platforms which were tested to patients.Figure 2Sankey and Venn diagram of two results analyzed by two different platforms of tumor next-generation sequencing.

Flow diagram detailing next-generation sequencing platforms which were tested to patients.

Sankey and Venn diagram of two results analyzed by two different platforms of tumor next-generation sequencing.

Table 2 shows the detailed information on the discordant pathogenic variants between the two tumor NGS tests. The highest prevalence of histological subtypes among the discordant pathogenic variants was high-grade serous carcinoma (85.7%, 12/14). The remaining two patients were diagnosed with endometrioid and clear cell carcinomas, respectively (Table 2). Of the patients who had a discrepancy in tumor NGS results, eight pathogenic variants (seven as ‘pathogenic’ and one as ‘likely pathogenic’) on BRCA1 and three on BRCA2 (one as ‘pathogenic’ and two as ‘likely pathogenic’) were detected in the GreenPlan HRD test but a negative result in the in-house platform. In contrast, three patients had a discordant pathogenic variant with positive results in the in-house platform, with two pathogenic variants (two as ‘pathogenic’) of BRCA1 and one pathogenic variant of BRCA2, and negative results in the GreenPlan HRD test.Table 2Summary of discordant pathogenic/likely pathogenic variants identification in tumor next-generation sequencing tests (n = 14).PatientHistologic typeAgeGeneDNAProtein*VAF(%)**Cause of discrepancyIn-house (No variant detected) GreenPlan (Variant detected) (n = 11)1High-grade serous50BRCA1c.2744delp.Ser915LeufsTer8522.8I2High-grade serous69BRCA1c.4185_4185 + 1insTTTTp.Gln1396PhefsTer954.4I3High-grade serous64BRCA1c.5138T > Cp.Val1713Ala84.5III4High-grade serous44BRCA1c.2960delp.Lys987SerfsTer1372.2I5Endometrioid50BRCA2c.1273G > Tp.Glu425Ter78.5I6High-grade serous43BRCA1c.1877delp.Val626GlufsTer647.8I7High-grade serous66BRCA1c.806T > Ap.Leu269Ter21.1I8High-grade serous64BRCA1c.3840_3871delp.Gln1281PhefsTer323.3I9High-grade serous62BRCA2c.2622_2629delp.Val875ArgfsTer351.4I10Clear cell47BRCA2c.8021dupp.Ile2675AspfsTer623.7II‡11High-grade serous55BRCA1c.5472_5473dupp.Gly1825ValfsTer1084.5IIn-house (Variant detected) GreenPlan (No variant detected) (n = 3)1High-grade serous58BRCA1c.3340G > Tp.E1114Ter8.75II†2High-grade serous57BRCA1c.5152 + 3 A > C-87.92I3High-grade serous64BRCA2c.8174G > Tp.W2725L1.85III*Variant allele frequency.**I: detected but not reported: this mutation does not classified as Tier 1, II: detected but filtered, III: not detected.‡Site filtered due to contraction of short tandem repeat region.†Variant was filtered due to low VAF (<5%).

Summary of discordant pathogenic/likely pathogenic variants identification in tumor next-generation sequencing tests (n = 14).

*Variant allele frequency.

**I: detected but not reported: this mutation does not classified as Tier 1, II: detected but filtered, III: not detected.

‡Site filtered due to contraction of short tandem repeat region.

†Variant was filtered due to low VAF (<5%).

Of the 11 cases that were not reported by in-house NGS, 9 (82%, 9/11) had the same mutation detected, but it was omitted in the final report as the mutation was not annotated in the Clinvar database. One case (9%, 1/11) had the same mutation in a short tandem repeat region, suggesting a PCR error, and was omitted in the final report. The remaining one case (9%, 1/11) did not have the mutation detected (Supplementary Fig. 1). Of the three cases where the mutation was not reported in the GreenPlan database, one case (33.3%, 1/3) had the same mutation detected but filtered due to low variant allele frequency (< 5%), and another case (33.3%, 1/3) had the same mutation detected but not reported in the database and classified as Tier 1. The remaining one (33.3%, 1/3) did not have the mutation detected (Supplementary Fig. 2).

### Discordant pathogenic variants

Table 2 shows the detailed information on the discordant pathogenic variants between the two tumor NGS tests. The highest prevalence of histological subtypes among the discordant pathogenic variants was high-grade serous carcinoma (85.7%, 12/14). The remaining two patients were diagnosed with endometrioid and clear cell carcinomas, respectively (Table 2). Of the patients who had a discrepancy in tumor NGS results, eight pathogenic variants (seven as ‘pathogenic’ and one as ‘likely pathogenic’) on BRCA1 and three on BRCA2 (one as ‘pathogenic’ and two as ‘likely pathogenic’) were detected in the GreenPlan HRD test but a negative result in the in-house platform. In contrast, three patients had a discordant pathogenic variant with positive results in the in-house platform, with two pathogenic variants (two as ‘pathogenic’) of BRCA1 and one pathogenic variant of BRCA2, and negative results in the GreenPlan HRD test.Table 2Summary of discordant pathogenic/likely pathogenic variants identification in tumor next-generation sequencing tests (n = 14).PatientHistologic typeAgeGeneDNAProtein*VAF(%)**Cause of discrepancyIn-house (No variant detected) GreenPlan (Variant detected) (n = 11)1High-grade serous50BRCA1c.2744delp.Ser915LeufsTer8522.8I2High-grade serous69BRCA1c.4185_4185 + 1insTTTTp.Gln1396PhefsTer954.4I3High-grade serous64BRCA1c.5138T > Cp.Val1713Ala84.5III4High-grade serous44BRCA1c.2960delp.Lys987SerfsTer1372.2I5Endometrioid50BRCA2c.1273G > Tp.Glu425Ter78.5I6High-grade serous43BRCA1c.1877delp.Val626GlufsTer647.8I7High-grade serous66BRCA1c.806T > Ap.Leu269Ter21.1I8High-grade serous64BRCA1c.3840_3871delp.Gln1281PhefsTer323.3I9High-grade serous62BRCA2c.2622_2629delp.Val875ArgfsTer351.4I10Clear cell47BRCA2c.8021dupp.Ile2675AspfsTer623.7II‡11High-grade serous55BRCA1c.5472_5473dupp.Gly1825ValfsTer1084.5IIn-house (Variant detected) GreenPlan (No variant detected) (n = 3)1High-grade serous58BRCA1c.3340G > Tp.E1114Ter8.75II†2High-grade serous57BRCA1c.5152 + 3 A > C-87.92I3High-grade serous64BRCA2c.8174G > Tp.W2725L1.85III*Variant allele frequency.**I: detected but not reported: this mutation does not classified as Tier 1, II: detected but filtered, III: not detected.‡Site filtered due to contraction of short tandem repeat region.†Variant was filtered due to low VAF (<5%).

Summary of discordant pathogenic/likely pathogenic variants identification in tumor next-generation sequencing tests (n = 14).

*Variant allele frequency.

**I: detected but not reported: this mutation does not classified as Tier 1, II: detected but filtered, III: not detected.

‡Site filtered due to contraction of short tandem repeat region.

†Variant was filtered due to low VAF (<5%).

Of the 11 cases that were not reported by in-house NGS, 9 (82%, 9/11) had the same mutation detected, but it was omitted in the final report as the mutation was not annotated in the Clinvar database. One case (9%, 1/11) had the same mutation in a short tandem repeat region, suggesting a PCR error, and was omitted in the final report. The remaining one case (9%, 1/11) did not have the mutation detected (Supplementary Fig. 1). Of the three cases where the mutation was not reported in the GreenPlan database, one case (33.3%, 1/3) had the same mutation detected but filtered due to low variant allele frequency (< 5%), and another case (33.3%, 1/3) had the same mutation detected but not reported in the database and classified as Tier 1. The remaining one (33.3%, 1/3) did not have the mutation detected (Supplementary Fig. 2).

### Discussion

In this single-institution retrospective study, we found an 11.9% (14/118) discrepancy in the identification of somatic BRCA1 and BRCA2 mutational status between the two platforms. To our knowledge, this is the first study to report the accrual discordant rate from dual tumor NGS testing, focused on BRCA1 and BRCA2. Currently, there is an increasing need for the molecular evaluation of BRCA because of the importance of identifying patients who benefit from PARP inhibitors. It is important to explore the reliability of tumor BRCA testing, which can identify pathogenic germline and somatic variants. This inter-platform discrepancy has left an important clinical issue for establishing an optimal platform for tumor BRCA testing.

Understanding how different assays make decisions, particularly in variant calling, is often complex and unclear. Each assay uses distinct methods, including different library kits, variant databases, and bioinformatics pipelines13,14. These variations introduce multiple factors that affect the results, making it challenging to interpret and compare findings across platforms. The lack of transparency regarding how these decisions are made further complicates consistency in variant classification and interpretation15,16. In light of these challenges, the discrepancies observed in this study can be attributed to several technical variables.

First, the two platforms introduced in this study adopted different library kits, and each protocol for library preparation varied in complexity, which might have a risk of error or contamination15,17. GreenPlan HRD utilized the Twist Bioscience Custom Target Enrichment kit, while In-house NGS PCP ver.3 used the Agilent SureSelect Target Enrichment System kit. The distinct differences in target enrichment methods could lead to varying sensitivities in detecting variants. Specifically, the efficiency of capturing the target regions may vary between kits, potentially affecting the detection of low-frequency variants. This variability in enrichment efficiency could explain some of the discrepancies observed in our study18.

Second, the two platforms referenced different variant databases, which likely contributed to the discordance in variant interpretation. GreenPlan HRD relied on COSMIC19 and OncoKB20, which are cancer-specific databases emphasizing somatic mutations. In contrast, In-house NGS PCP ver.3 used 1000genome, ClinVar21, ESP650022, and ExaC23. The American Society of Clinical Oncology and the College of American Pathologists (CAP) have suggested several recommendations for exercising certain databases to ascertain the quality of each genomic dataset based on the number of specific entries, the depth of the study, and the use of appropriate controls24. Therefore, database selection plays a critical role in the discordance observed between platforms, and greater transparency in the variant interpretation process is necessary to mitigate these differences.

Third, the pre-steps of fixation and extraction of a sufficient amount of DNA from FFPE tissues could contribute to the quality of library preparation and sequencing. An FFPE failure rate of approximately 5% was detected when the DNA quality was poor15. In cases of low tumor content when identifying mutations, macrodissection plays a role in enhancing the accuracy of NGS by increasing tumor purity, particularly in FFPE tissue samples. By manually removing non-tumor tissue, macrodissection enriches for tumor cells, which can improve the detection of somatic variants by increasing the VAF25.

Lastly, even though both platforms used the same NextSeq Illumina 550Dx NGS platform, the specific error profiles of the respective sequencing methods may still contribute to minor differences in results18. While this study did not focus on sequencing technology discrepancies, the use of different bioinformatics pipelines for variant calling and filtering may further contribute to the variability.

Considering the above-mentioned relevant variables that could affect the reliability of tumor BRCA testing, standardization of the NGS process optimized for BRCA identification, which targets patients with epithelial OC, is required. Most of the discrepancies in this study were due to differences in databases. Accordingly, we have modified our in-house pipeline to report mutations with a high likelihood of being pathogenic, such as frame-shift or nonsense mutations, as ‘unknown significance’ for genes of high clinical importance, even if they have not been reported in Clinvar.

Currently, PARP inhibitor (for example, olaparib, niraparib, and ribaparib) monotherapy can be appropriate as a maintenance therapy in BRCA1 or BRCA2 mutated patients after first remission or after more than two previous platinum chemotherapy regimens without prior PARP inhibitor use7,26. Olaparib combined with bevacizumab is another option for patients with pathological variants of BRCA or HRD tumors27. A proper BRCA testing strategy is required to select an appropriate candidate PARP inhibitor while minimizing false-negative BRCA results. Further studies are needed to cross-validate tumor NGS testing with broader types of platforms and samples.

This study had several limitations. First, extracting data that are within the scope of a retrospective design might have selection bias, such as the unintentional omission of patients who are eligible for the study criteria or patient genomic data. Second, revalidation of the identified discordant pathogenic variants was not performed. However, each result underwent an individual review of raw variant call format (VCF) files or Binary Alignment Map (BAM) files. Third, the small sample size might be one of the limitations, although somatic pathogenic variants resulting from the two different platforms are not easily obtained at the same time.

Despite these limitations, the results of our study provide important information on intra-platform discordance of BRCA1 and BRCA2 mutations with variant details of DNA and protein changes and highlight the value of well-strategized BRCA testing, which increases the reliability of tumor BRCA testing in epithelial OC.

### Limitations

This study had several limitations. First, extracting data that are within the scope of a retrospective design might have selection bias, such as the unintentional omission of patients who are eligible for the study criteria or patient genomic data. Second, revalidation of the identified discordant pathogenic variants was not performed. However, each result underwent an individual review of raw variant call format (VCF) files or Binary Alignment Map (BAM) files. Third, the small sample size might be one of the limitations, although somatic pathogenic variants resulting from the two different platforms are not easily obtained at the same time.

Despite these limitations, the results of our study provide important information on intra-platform discordance of BRCA1 and BRCA2 mutations with variant details of DNA and protein changes and highlight the value of well-strategized BRCA testing, which increases the reliability of tumor BRCA testing in epithelial OC.

### Conclusion

In conclusion, the discordance rate of somatic BRCA1 or BRCA2 mutations from two dual-tumor NGS tests, which were carried out for 118 patients with epithelial ovarian cancer, was approximately 12%. In the case of a strong implication of a BRCA pathologic variant with a negative result with one genetic test, different platforms could be employed in limited cases. Careful interpretation and further studies for cross-validation of gene analysis platforms are needed to ensure that patients benefit from PARP inhibitors.

### Supplementary Information

Supplementary Material 1.

Supplementary Material 1.

Supplementary Material 2.

Supplementary Material 2.



# SUPPLEMENTAL FILE 1: 41598_2024_Article_75230.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)