# MAIN TEXT

## Uncovering the Mechanism of Aggregation of Human Transthyretin*

### Abstract

Background: Transthyretin (TTR) aggregation is associated with systemic amyloidosis.Results: Residue replacements on the F and H strands hinder TTR aggregation.Conclusion: The F and H strands are aggregation-driving segments of TTR. The binding of designed peptides inhibits protein aggregation.Significance: We point the way to new therapeutic approaches against TTR aggregation by using peptides to block amyloid segments.

### Introduction

Transthyretin (TTR)3 is a 55-kDa protein that aggregates into amyloid fibrils under pathological conditions. In its native tetrameric conformation, TTR transports retinol-binding protein and thyroxine (T4) in the blood and cerebrospinal fluid (1). T4 binds in a hydrophobic pocket at the center of the tetramer. Amyloid aggregation of TTR occurs by dissociation of tetrameric TTR into monomers; these partially unfold into amyloidogenic intermediates and self-associate into soluble oligomers and amyloid aggregates (2–7). Familial point mutations are known to destabilize the tetramer, leading to faster dissociation and consequent amyloid aggregation (8). Compounds that bind to the hydrophobic T4-binding site are known to stabilize the tetramer and slow aggregation (9, 10).

The aggregation of TTR causes transthyretin amyloidosis (ATTR) associated with three conditions traditionally known as senile systemic amyloidosis, familial amyloidotic polyneuropathy, and familial amyloidotic cardiomyopathy. Senile systemic amyloidosis is a late onset disease in which wild-type (WT) TTR aggregates, weakening the heart muscle (2, 11). Senile systemic amyloidosis is usually diagnosed by post-mortem exams of patients over 80 years old. Familial amyloidotic polyneuropathy and familial amyloidotic cardiomyopathy are hereditary conditions characterized by extracellular deposition of TTR amyloid fibrils in the peripheral nerves and heart, respectively, which leads to system failure (12). Familial amyloidotic polyneuropathy is caused by a number of TTR mutations, including L55P and V30M, which we examine here (13). The most common familial amyloidotic cardiomyopathy mutation, V122I, is carried by 3.9% of the African-American population (14, 15).

Currently, there is no cure for transthyretin amyloidosis, and the treatment for familial cases of ATTR is liver transplantation. Tafamidis, a TTR tetramer stabilizer, has been recently approved in Europe; it delays progression of the disease. Several other therapeutics are currently in clinical trials, including other tetramer stabilizers such as diflunisal and RNAi therapies that cause a decrease in the production of TTR protein (16–18). Additional approaches are needed to prevent ATTR, and here we explore the use of peptide inhibitors that block aggregation of TTR.

Several models of the TTR amyloid spine have been proposed (19–22), but the aggregation-prone segments of the protein remain uncertain. Based on the studies of crystal structures of amyloid-driving segments, our group has proposed that fibrils can form through intermolecular self-association of one to several fibril-driving segments. Identical segments from several protein molecules stack into steric zipper structures, which form the spine of the amyloid fibril through tightly interdigitated β-sheets (23–25). Here we identify two segments of TTR that drive protein aggregation by self-association and formation of steric zipper spines of amyloid fibrils. Based on the amyloid structure of these two segments, we designed two peptide inhibitors that halt the progression of TTR aggregation.

### Experimental Procedures

Mutants of TTR were produced using a QuikChange site-directed mutagenesis kit (Stratagene). Wild-type and mutated TTR genes were cloned into the pET24(+) vector (Novagen), and the sequences were verified by DNA sequencing and/or later x-ray crystallography of pure proteins (Table 1). For protein expression, exponentially growing Escherichia coli RosettaTM(DE3) pLysS competent cells (Millipore) transformed with each of the plasmids were exponentially grown and treated with 1 mm isopropyl β-d-thiogalactoside for 3 h. Wild-type TTR or its variants were purified by nickel affinity chromatography using HisTrap columns (GE Healthcare) following the manufacturer's instructions. Fractions were pooled and subjected to gel filtration chromatography using a HiLoad 16/60 Superdex 75 gel filtration column (GE Healthcare) running on an ÄKTA FPLC system. TTR-rich fractions were confirmed by SDS-PAGE. After checking that the His tag did not modify TTR aggregation (data not shown), we chose to retain the tag to facilitate further analysis of TTR aggregation by His probe blotting.

Statistics of data collection and atomic refinement

Values in parentheses correspond to the highest resolution shell. The structures of the TTR variants were used to confirm the correct substitutions. The wild-type TTR structure was used for docking. Coordinates are deposited in the Protein Data Bank under the accession codes that are listed in the table. r.m.s.d., root mean square deviation; CC1/2, correlation coefficient.

aRmerge = ΣIj − I/ΣI. Notice that the high Rmerge value of the highest resolution shell of several structures is compensated by a high CC1/2 value.

bRwork = Σ|Fo − Fc|j/ΣFo.

cRfree = Σ|Fo − Fc|j/ΣFo calculated using a random set containing 5 or 10% of reflections that were not included throughout structure refinement for protein or peptide structure determination, respectively.

TTR aggregation assays are described elsewhere (26). Briefly, 1 mg/ml TTR sample in 10 mm sodium acetate (pH 4.3), 100 mm KCl, 10 mm EDTA was incubated at 37 °C for a maximum of 4 days. Various measurements of TTR aggregation were taken. (i) Protein aggregation was monitored by measuring absorbance of the sample at 400 nm. (ii) Protein concentration of the insoluble fraction was calculated as follows. 100 μl of sample was spun at 13,000 rpm for 30 min. The pellet was resuspended in the same volume of fresh buffer and spun again at 13,000 rpm for 30 min. The final pellet was resuspended in 6 m guanidine chloride, and absorbance at 295 nm was measured. Protein concentration of the insoluble fraction was calculated from absorbance data at 295 nm. (iii) TTR aggregation was visualized by transmission electron microscopy of the sample and/or immunodot blot of the insoluble fraction.

Aggregation assays were performed in both the presence and absence of aggregation inhibitors when appropriate at a ratio of 1:10 (TTR monomer:inhibitor) unless labeled otherwise. Turbidity of the sample (absorbance at 400 nm) and protein concentration of the insoluble fraction were measured at time 0, day 1, day 2, and day 4. Values for day 4 are presented in the figures unless labeled otherwise.

The aggregation of TTR was followed by dot blot analysis as described (27) using the SuperSignal® West HisProbeTM kit following the manufacturer's instructions (Thermo Scientific). The insoluble fraction of the samples after 4 days of incubation was dotted onto nitrocellulose membranes (0.2 μm; Bio-Rad). A dilution ratio of 1:10,000 was used for the HisProbe antibody.

Peptides were dissolved in PBS buffer (pH 7.4) at a concentration that depended on the solubility of the peptide: 12LMVKVL17 at 14 mm, 25AINVAV30 at 17 mm, 26NVAVHV32 at 16 mm, 28VAVHVF33 at 15 mm, 47GKTSES52 at 16 mm, 65VEGIYK70 at 14 mm, 68IYKVEI73 at 13 mm, 80KALGIS85 at 17 mm, 91AEVVFT96 at 0.5 mm, 91APVVFT96 at 15.8 mm, 91AEVPFT96 at 15.8 mm, 105YTIAAL110 at 15 mm, 106TIAALLS112 4.6 mm, and 119TAVVTN124 at 17 mm. Following dissolution, samples were filtered through a 0.2-μm filter and incubated at 37 °C with no shaking for 2 weeks.

TEM was performed to visualize the fibril formation of TTR proteins and peptides. 5 μl of the sample was spotted onto freshly glow-discharged, carbon-coated EM grids (Ted Pella, Redding, CA). After 3 min of incubation, grids were rinsed three times with 5 μl of distilled water and then stained with 2% uranyl acetate for 2 min. Grids were examined with a T12 Quick CryoEM and CryoET (FEI) transmission electron microscope at an accelerating voltage of 120 kV. Digital images were recorded using a Gatan 2,048 × 2,048 charge-coupled device camera.

Crystallization conditions, data collection, and refinement statistics of crystal structures are detailed in Table 1. X-ray diffraction data were collected at the Advanced Photon Source beamline 24-ID-C (for full-length proteins) or 24-ID-E (for peptides). Molecular replacement was performed with the program Phaser (28) using as search models an idealized polyalanine β-strand and chain A of the mutant T119M (Protein Data Bank code 1F86) for peptide and TTR variant data sets, respectively. Crystallographic refinement was performed using PHENIX (29), REFMAC (30), and BUSTER (31). Model building was performed with Coot (32) and illustrated with PyMOL (33).

To predict amyloidogenicity of TTR segments, the TTR sequence was submitted to ZipperDB, the Rosetta-based method that profiles the steric zipper spine, the main structural feature of amyloid fibrils (34, 35). Additionally, solvent-accessible surface area per residue of TTR was calculated to examine the accessibility of the different protein segments for self-association. Solvent-accessible surface area calculations were performed by Areaimol (36) using the structure of WT TTR (Protein Data Bank code 4TLT) in three different conformations: tetramer with 222 symmetry, dimer (the crystallographic asymmetric unit), and monomer (by removing one chain of the dimer). These conformations are based on the dissociation pathway described elsewhere (4).

We predicted the interaction of the peptides AEVVFT and TAVVTN with their identical segments at the surface of the monomer or at the tip of a growing steric zipper of TTR based on our finding that the two TTR segments form fibrils in solution by self-aggregation. To validate the prediction, computational docking was carried out using the Rosetta software as described previously (35) using chain A of WT TTR as a template (Protein Data Bank code 4TLT). The optimization of the aggregation inhibitors required the addition of an N-methyl group and tetra-arginine tag. The use of the non-natural N-methyl groups to protect aggregation inhibitors from proteolysis has been successful in other aggregation inhibitors (37). The addition of a polyarginine tag confers higher solubility and we presume also hinders self-aggregation of the inhibitors. The position of the N-methyl group was designed based on our docking model and experimentally checked by aggregation assays.

Data were taken with a DSC-II differential scanning calorimeter (Calorimetry Sciences Corp., Lindon, UT). The midpoint temperatures of the thermal unfolding transition (Tm) of wild type and mutants were determined. Thermal denaturation of TTR was irreversible under our conditions (1, 38), and absolute thermodynamic parameters were not determined. Previously, researchers found that WT TTR unfolds at 101.7 °C under 2 atm of nitrogen (39). The experiments were performed in 3.5 m guanidinium hydrochloride to depress melting below 100 °C; this guanidinium concentration also solubilized thyroxine. Protein samples were thawed on ice and filtered, and the concentration was measured by UV-visible absorption. Protein samples were diluted to 1 mg/ml in 3.5 m guanidinium chloride, 100 mm sodium acetate (pH 4.3), 100 mm potassium chloride, 1 mm EDTA. A volume of 700 μl of each protein sample was prepared in a 1.5-ml microcentrifuge tube and degassed for 5 min under vacuum while stirring. The reference cell contained only buffer components. Samples were scanned from 25 to 95 °C at a rate of 1 °C/min. The buffer was subtracted, and the baseline was corrected with the linear polynomial function in CPCalc Analysis software. In an additional experiment, T4 and aggregation inhibitors were added to WT TTR to determine whether they affect the thermodynamic stability. T4 and the aggregation inhibitors were added in 5-fold excess relative to protein concentration to both the reference and sample chambers.

Monomeric variant MTTR at 0.5 mg/ml (40) was incubated with increasing concentrations of both TTR inhibitors R4PAm and R4PTm (0–5-fold molar excess) and subjected to blue native electrophoresis using NativePAGETM Bis-Tris precast gels (4–16% Bis-Tris, 1 mm; Life Technologies) following the manufacturer's instructions. NativeMarkTM unstained standards were used for size estimation of proteins.

The soluble unbound fraction of inhibitors after incubation with the monomeric variant MTTR (40) was quantified by HPLC-MS. Samples were made as follows. Soluble TTR at a fixed concentration of 0.5 mg/ml was incubated at 37 °C for 18 h in 10 mm sodium acetate (pH 7.4), 100 mm KCl, 10 mm EDTA with increasing quantities of combined R4PAm and R4PTm. The aggregated fraction was removed by 0.22-μm filtration. The protein fraction containing soluble TTR and inhibitor-bound complexes was then removed by acid precipitation with 1% acetic acid and consecutive ultracentrifugation at 90,000 rpm for 30 min. Neither 0.22-μm filtration nor acid precipitation affected the concentration of the unbound soluble fraction of the peptide inhibitors (data not shown). The unbound soluble fraction of inhibitors was subjected to size exclusion HPLC followed by mass spectrometry. The instrument used was the Agilent 6460 Triple Quadrupole LC/MS System (Agilent Technologies, Santa Clara, CA) with a TSK gel column (2.0 × 150 mm, 4 μm, 125 Å; Michrom Bioresources, Inc., Auburn, CA). Flow rate was 0.2 ml/min, and the elution buffer was 75% acetonitrile in water containing 0.1% formic acid. Known concentrations of peptide inhibitors were subjected to the same treatment to create a standard curve. The analysis of the results was performed using GraphPad Prism 6.

To analyze the tetrameric and monomeric populations of TTR, the variant M13R/L17R/T119M (MLT) was generated. This variant forms tetramer and monomers in solution that were examined as follows. A 5-fold molar excess of TTR inhibitors was added to 1 mg/ml soluble MLT and analyzed before and after incubation in 10 mm sodium acetate (pH 4.3), 100 mm KCl, 10 mm EDTA at 37 °C for 18 h. Samples were analyzed by analytical size exclusion with a Superdex 75 16/300 GL column following the manufacturer's instructions (GE Healthcare) running on an ÄKTA FPLC system in 10 mm sodium acetate (pH 7.4), 100 mm KCl, 10 mm EDTA.

### 

Mutants of TTR were produced using a QuikChange site-directed mutagenesis kit (Stratagene). Wild-type and mutated TTR genes were cloned into the pET24(+) vector (Novagen), and the sequences were verified by DNA sequencing and/or later x-ray crystallography of pure proteins (Table 1). For protein expression, exponentially growing Escherichia coli RosettaTM(DE3) pLysS competent cells (Millipore) transformed with each of the plasmids were exponentially grown and treated with 1 mm isopropyl β-d-thiogalactoside for 3 h. Wild-type TTR or its variants were purified by nickel affinity chromatography using HisTrap columns (GE Healthcare) following the manufacturer's instructions. Fractions were pooled and subjected to gel filtration chromatography using a HiLoad 16/60 Superdex 75 gel filtration column (GE Healthcare) running on an ÄKTA FPLC system. TTR-rich fractions were confirmed by SDS-PAGE. After checking that the His tag did not modify TTR aggregation (data not shown), we chose to retain the tag to facilitate further analysis of TTR aggregation by His probe blotting.

Statistics of data collection and atomic refinement

Values in parentheses correspond to the highest resolution shell. The structures of the TTR variants were used to confirm the correct substitutions. The wild-type TTR structure was used for docking. Coordinates are deposited in the Protein Data Bank under the accession codes that are listed in the table. r.m.s.d., root mean square deviation; CC1/2, correlation coefficient.

aRmerge = ΣIj − I/ΣI. Notice that the high Rmerge value of the highest resolution shell of several structures is compensated by a high CC1/2 value.

bRwork = Σ|Fo − Fc|j/ΣFo.

cRfree = Σ|Fo − Fc|j/ΣFo calculated using a random set containing 5 or 10% of reflections that were not included throughout structure refinement for protein or peptide structure determination, respectively.

TTR aggregation assays are described elsewhere (26). Briefly, 1 mg/ml TTR sample in 10 mm sodium acetate (pH 4.3), 100 mm KCl, 10 mm EDTA was incubated at 37 °C for a maximum of 4 days. Various measurements of TTR aggregation were taken. (i) Protein aggregation was monitored by measuring absorbance of the sample at 400 nm. (ii) Protein concentration of the insoluble fraction was calculated as follows. 100 μl of sample was spun at 13,000 rpm for 30 min. The pellet was resuspended in the same volume of fresh buffer and spun again at 13,000 rpm for 30 min. The final pellet was resuspended in 6 m guanidine chloride, and absorbance at 295 nm was measured. Protein concentration of the insoluble fraction was calculated from absorbance data at 295 nm. (iii) TTR aggregation was visualized by transmission electron microscopy of the sample and/or immunodot blot of the insoluble fraction.

Aggregation assays were performed in both the presence and absence of aggregation inhibitors when appropriate at a ratio of 1:10 (TTR monomer:inhibitor) unless labeled otherwise. Turbidity of the sample (absorbance at 400 nm) and protein concentration of the insoluble fraction were measured at time 0, day 1, day 2, and day 4. Values for day 4 are presented in the figures unless labeled otherwise.

The aggregation of TTR was followed by dot blot analysis as described (27) using the SuperSignal® West HisProbeTM kit following the manufacturer's instructions (Thermo Scientific). The insoluble fraction of the samples after 4 days of incubation was dotted onto nitrocellulose membranes (0.2 μm; Bio-Rad). A dilution ratio of 1:10,000 was used for the HisProbe antibody.

Peptides were dissolved in PBS buffer (pH 7.4) at a concentration that depended on the solubility of the peptide: 12LMVKVL17 at 14 mm, 25AINVAV30 at 17 mm, 26NVAVHV32 at 16 mm, 28VAVHVF33 at 15 mm, 47GKTSES52 at 16 mm, 65VEGIYK70 at 14 mm, 68IYKVEI73 at 13 mm, 80KALGIS85 at 17 mm, 91AEVVFT96 at 0.5 mm, 91APVVFT96 at 15.8 mm, 91AEVPFT96 at 15.8 mm, 105YTIAAL110 at 15 mm, 106TIAALLS112 4.6 mm, and 119TAVVTN124 at 17 mm. Following dissolution, samples were filtered through a 0.2-μm filter and incubated at 37 °C with no shaking for 2 weeks.

TEM was performed to visualize the fibril formation of TTR proteins and peptides. 5 μl of the sample was spotted onto freshly glow-discharged, carbon-coated EM grids (Ted Pella, Redding, CA). After 3 min of incubation, grids were rinsed three times with 5 μl of distilled water and then stained with 2% uranyl acetate for 2 min. Grids were examined with a T12 Quick CryoEM and CryoET (FEI) transmission electron microscope at an accelerating voltage of 120 kV. Digital images were recorded using a Gatan 2,048 × 2,048 charge-coupled device camera.

Crystallization conditions, data collection, and refinement statistics of crystal structures are detailed in Table 1. X-ray diffraction data were collected at the Advanced Photon Source beamline 24-ID-C (for full-length proteins) or 24-ID-E (for peptides). Molecular replacement was performed with the program Phaser (28) using as search models an idealized polyalanine β-strand and chain A of the mutant T119M (Protein Data Bank code 1F86) for peptide and TTR variant data sets, respectively. Crystallographic refinement was performed using PHENIX (29), REFMAC (30), and BUSTER (31). Model building was performed with Coot (32) and illustrated with PyMOL (33).

To predict amyloidogenicity of TTR segments, the TTR sequence was submitted to ZipperDB, the Rosetta-based method that profiles the steric zipper spine, the main structural feature of amyloid fibrils (34, 35). Additionally, solvent-accessible surface area per residue of TTR was calculated to examine the accessibility of the different protein segments for self-association. Solvent-accessible surface area calculations were performed by Areaimol (36) using the structure of WT TTR (Protein Data Bank code 4TLT) in three different conformations: tetramer with 222 symmetry, dimer (the crystallographic asymmetric unit), and monomer (by removing one chain of the dimer). These conformations are based on the dissociation pathway described elsewhere (4).

We predicted the interaction of the peptides AEVVFT and TAVVTN with their identical segments at the surface of the monomer or at the tip of a growing steric zipper of TTR based on our finding that the two TTR segments form fibrils in solution by self-aggregation. To validate the prediction, computational docking was carried out using the Rosetta software as described previously (35) using chain A of WT TTR as a template (Protein Data Bank code 4TLT). The optimization of the aggregation inhibitors required the addition of an N-methyl group and tetra-arginine tag. The use of the non-natural N-methyl groups to protect aggregation inhibitors from proteolysis has been successful in other aggregation inhibitors (37). The addition of a polyarginine tag confers higher solubility and we presume also hinders self-aggregation of the inhibitors. The position of the N-methyl group was designed based on our docking model and experimentally checked by aggregation assays.

Data were taken with a DSC-II differential scanning calorimeter (Calorimetry Sciences Corp., Lindon, UT). The midpoint temperatures of the thermal unfolding transition (Tm) of wild type and mutants were determined. Thermal denaturation of TTR was irreversible under our conditions (1, 38), and absolute thermodynamic parameters were not determined. Previously, researchers found that WT TTR unfolds at 101.7 °C under 2 atm of nitrogen (39). The experiments were performed in 3.5 m guanidinium hydrochloride to depress melting below 100 °C; this guanidinium concentration also solubilized thyroxine. Protein samples were thawed on ice and filtered, and the concentration was measured by UV-visible absorption. Protein samples were diluted to 1 mg/ml in 3.5 m guanidinium chloride, 100 mm sodium acetate (pH 4.3), 100 mm potassium chloride, 1 mm EDTA. A volume of 700 μl of each protein sample was prepared in a 1.5-ml microcentrifuge tube and degassed for 5 min under vacuum while stirring. The reference cell contained only buffer components. Samples were scanned from 25 to 95 °C at a rate of 1 °C/min. The buffer was subtracted, and the baseline was corrected with the linear polynomial function in CPCalc Analysis software. In an additional experiment, T4 and aggregation inhibitors were added to WT TTR to determine whether they affect the thermodynamic stability. T4 and the aggregation inhibitors were added in 5-fold excess relative to protein concentration to both the reference and sample chambers.

Monomeric variant MTTR at 0.5 mg/ml (40) was incubated with increasing concentrations of both TTR inhibitors R4PAm and R4PTm (0–5-fold molar excess) and subjected to blue native electrophoresis using NativePAGETM Bis-Tris precast gels (4–16% Bis-Tris, 1 mm; Life Technologies) following the manufacturer's instructions. NativeMarkTM unstained standards were used for size estimation of proteins.

The soluble unbound fraction of inhibitors after incubation with the monomeric variant MTTR (40) was quantified by HPLC-MS. Samples were made as follows. Soluble TTR at a fixed concentration of 0.5 mg/ml was incubated at 37 °C for 18 h in 10 mm sodium acetate (pH 7.4), 100 mm KCl, 10 mm EDTA with increasing quantities of combined R4PAm and R4PTm. The aggregated fraction was removed by 0.22-μm filtration. The protein fraction containing soluble TTR and inhibitor-bound complexes was then removed by acid precipitation with 1% acetic acid and consecutive ultracentrifugation at 90,000 rpm for 30 min. Neither 0.22-μm filtration nor acid precipitation affected the concentration of the unbound soluble fraction of the peptide inhibitors (data not shown). The unbound soluble fraction of inhibitors was subjected to size exclusion HPLC followed by mass spectrometry. The instrument used was the Agilent 6460 Triple Quadrupole LC/MS System (Agilent Technologies, Santa Clara, CA) with a TSK gel column (2.0 × 150 mm, 4 μm, 125 Å; Michrom Bioresources, Inc., Auburn, CA). Flow rate was 0.2 ml/min, and the elution buffer was 75% acetonitrile in water containing 0.1% formic acid. Known concentrations of peptide inhibitors were subjected to the same treatment to create a standard curve. The analysis of the results was performed using GraphPad Prism 6.

To analyze the tetrameric and monomeric populations of TTR, the variant M13R/L17R/T119M (MLT) was generated. This variant forms tetramer and monomers in solution that were examined as follows. A 5-fold molar excess of TTR inhibitors was added to 1 mg/ml soluble MLT and analyzed before and after incubation in 10 mm sodium acetate (pH 4.3), 100 mm KCl, 10 mm EDTA at 37 °C for 18 h. Samples were analyzed by analytical size exclusion with a Superdex 75 16/300 GL column following the manufacturer's instructions (GE Healthcare) running on an ÄKTA FPLC system in 10 mm sodium acetate (pH 7.4), 100 mm KCl, 10 mm EDTA.

### 

Mutants of TTR were produced using a QuikChange site-directed mutagenesis kit (Stratagene). Wild-type and mutated TTR genes were cloned into the pET24(+) vector (Novagen), and the sequences were verified by DNA sequencing and/or later x-ray crystallography of pure proteins (Table 1). For protein expression, exponentially growing Escherichia coli RosettaTM(DE3) pLysS competent cells (Millipore) transformed with each of the plasmids were exponentially grown and treated with 1 mm isopropyl β-d-thiogalactoside for 3 h. Wild-type TTR or its variants were purified by nickel affinity chromatography using HisTrap columns (GE Healthcare) following the manufacturer's instructions. Fractions were pooled and subjected to gel filtration chromatography using a HiLoad 16/60 Superdex 75 gel filtration column (GE Healthcare) running on an ÄKTA FPLC system. TTR-rich fractions were confirmed by SDS-PAGE. After checking that the His tag did not modify TTR aggregation (data not shown), we chose to retain the tag to facilitate further analysis of TTR aggregation by His probe blotting.

Statistics of data collection and atomic refinement

Values in parentheses correspond to the highest resolution shell. The structures of the TTR variants were used to confirm the correct substitutions. The wild-type TTR structure was used for docking. Coordinates are deposited in the Protein Data Bank under the accession codes that are listed in the table. r.m.s.d., root mean square deviation; CC1/2, correlation coefficient.

aRmerge = ΣIj − I/ΣI. Notice that the high Rmerge value of the highest resolution shell of several structures is compensated by a high CC1/2 value.

bRwork = Σ|Fo − Fc|j/ΣFo.

cRfree = Σ|Fo − Fc|j/ΣFo calculated using a random set containing 5 or 10% of reflections that were not included throughout structure refinement for protein or peptide structure determination, respectively.

TTR aggregation assays are described elsewhere (26). Briefly, 1 mg/ml TTR sample in 10 mm sodium acetate (pH 4.3), 100 mm KCl, 10 mm EDTA was incubated at 37 °C for a maximum of 4 days. Various measurements of TTR aggregation were taken. (i) Protein aggregation was monitored by measuring absorbance of the sample at 400 nm. (ii) Protein concentration of the insoluble fraction was calculated as follows. 100 μl of sample was spun at 13,000 rpm for 30 min. The pellet was resuspended in the same volume of fresh buffer and spun again at 13,000 rpm for 30 min. The final pellet was resuspended in 6 m guanidine chloride, and absorbance at 295 nm was measured. Protein concentration of the insoluble fraction was calculated from absorbance data at 295 nm. (iii) TTR aggregation was visualized by transmission electron microscopy of the sample and/or immunodot blot of the insoluble fraction.

Aggregation assays were performed in both the presence and absence of aggregation inhibitors when appropriate at a ratio of 1:10 (TTR monomer:inhibitor) unless labeled otherwise. Turbidity of the sample (absorbance at 400 nm) and protein concentration of the insoluble fraction were measured at time 0, day 1, day 2, and day 4. Values for day 4 are presented in the figures unless labeled otherwise.

The aggregation of TTR was followed by dot blot analysis as described (27) using the SuperSignal® West HisProbeTM kit following the manufacturer's instructions (Thermo Scientific). The insoluble fraction of the samples after 4 days of incubation was dotted onto nitrocellulose membranes (0.2 μm; Bio-Rad). A dilution ratio of 1:10,000 was used for the HisProbe antibody.

Peptides were dissolved in PBS buffer (pH 7.4) at a concentration that depended on the solubility of the peptide: 12LMVKVL17 at 14 mm, 25AINVAV30 at 17 mm, 26NVAVHV32 at 16 mm, 28VAVHVF33 at 15 mm, 47GKTSES52 at 16 mm, 65VEGIYK70 at 14 mm, 68IYKVEI73 at 13 mm, 80KALGIS85 at 17 mm, 91AEVVFT96 at 0.5 mm, 91APVVFT96 at 15.8 mm, 91AEVPFT96 at 15.8 mm, 105YTIAAL110 at 15 mm, 106TIAALLS112 4.6 mm, and 119TAVVTN124 at 17 mm. Following dissolution, samples were filtered through a 0.2-μm filter and incubated at 37 °C with no shaking for 2 weeks.

TEM was performed to visualize the fibril formation of TTR proteins and peptides. 5 μl of the sample was spotted onto freshly glow-discharged, carbon-coated EM grids (Ted Pella, Redding, CA). After 3 min of incubation, grids were rinsed three times with 5 μl of distilled water and then stained with 2% uranyl acetate for 2 min. Grids were examined with a T12 Quick CryoEM and CryoET (FEI) transmission electron microscope at an accelerating voltage of 120 kV. Digital images were recorded using a Gatan 2,048 × 2,048 charge-coupled device camera.

Crystallization conditions, data collection, and refinement statistics of crystal structures are detailed in Table 1. X-ray diffraction data were collected at the Advanced Photon Source beamline 24-ID-C (for full-length proteins) or 24-ID-E (for peptides). Molecular replacement was performed with the program Phaser (28) using as search models an idealized polyalanine β-strand and chain A of the mutant T119M (Protein Data Bank code 1F86) for peptide and TTR variant data sets, respectively. Crystallographic refinement was performed using PHENIX (29), REFMAC (30), and BUSTER (31). Model building was performed with Coot (32) and illustrated with PyMOL (33).

To predict amyloidogenicity of TTR segments, the TTR sequence was submitted to ZipperDB, the Rosetta-based method that profiles the steric zipper spine, the main structural feature of amyloid fibrils (34, 35). Additionally, solvent-accessible surface area per residue of TTR was calculated to examine the accessibility of the different protein segments for self-association. Solvent-accessible surface area calculations were performed by Areaimol (36) using the structure of WT TTR (Protein Data Bank code 4TLT) in three different conformations: tetramer with 222 symmetry, dimer (the crystallographic asymmetric unit), and monomer (by removing one chain of the dimer). These conformations are based on the dissociation pathway described elsewhere (4).

We predicted the interaction of the peptides AEVVFT and TAVVTN with their identical segments at the surface of the monomer or at the tip of a growing steric zipper of TTR based on our finding that the two TTR segments form fibrils in solution by self-aggregation. To validate the prediction, computational docking was carried out using the Rosetta software as described previously (35) using chain A of WT TTR as a template (Protein Data Bank code 4TLT). The optimization of the aggregation inhibitors required the addition of an N-methyl group and tetra-arginine tag. The use of the non-natural N-methyl groups to protect aggregation inhibitors from proteolysis has been successful in other aggregation inhibitors (37). The addition of a polyarginine tag confers higher solubility and we presume also hinders self-aggregation of the inhibitors. The position of the N-methyl group was designed based on our docking model and experimentally checked by aggregation assays.

Data were taken with a DSC-II differential scanning calorimeter (Calorimetry Sciences Corp., Lindon, UT). The midpoint temperatures of the thermal unfolding transition (Tm) of wild type and mutants were determined. Thermal denaturation of TTR was irreversible under our conditions (1, 38), and absolute thermodynamic parameters were not determined. Previously, researchers found that WT TTR unfolds at 101.7 °C under 2 atm of nitrogen (39). The experiments were performed in 3.5 m guanidinium hydrochloride to depress melting below 100 °C; this guanidinium concentration also solubilized thyroxine. Protein samples were thawed on ice and filtered, and the concentration was measured by UV-visible absorption. Protein samples were diluted to 1 mg/ml in 3.5 m guanidinium chloride, 100 mm sodium acetate (pH 4.3), 100 mm potassium chloride, 1 mm EDTA. A volume of 700 μl of each protein sample was prepared in a 1.5-ml microcentrifuge tube and degassed for 5 min under vacuum while stirring. The reference cell contained only buffer components. Samples were scanned from 25 to 95 °C at a rate of 1 °C/min. The buffer was subtracted, and the baseline was corrected with the linear polynomial function in CPCalc Analysis software. In an additional experiment, T4 and aggregation inhibitors were added to WT TTR to determine whether they affect the thermodynamic stability. T4 and the aggregation inhibitors were added in 5-fold excess relative to protein concentration to both the reference and sample chambers.

Monomeric variant MTTR at 0.5 mg/ml (40) was incubated with increasing concentrations of both TTR inhibitors R4PAm and R4PTm (0–5-fold molar excess) and subjected to blue native electrophoresis using NativePAGETM Bis-Tris precast gels (4–16% Bis-Tris, 1 mm; Life Technologies) following the manufacturer's instructions. NativeMarkTM unstained standards were used for size estimation of proteins.

The soluble unbound fraction of inhibitors after incubation with the monomeric variant MTTR (40) was quantified by HPLC-MS. Samples were made as follows. Soluble TTR at a fixed concentration of 0.5 mg/ml was incubated at 37 °C for 18 h in 10 mm sodium acetate (pH 7.4), 100 mm KCl, 10 mm EDTA with increasing quantities of combined R4PAm and R4PTm. The aggregated fraction was removed by 0.22-μm filtration. The protein fraction containing soluble TTR and inhibitor-bound complexes was then removed by acid precipitation with 1% acetic acid and consecutive ultracentrifugation at 90,000 rpm for 30 min. Neither 0.22-μm filtration nor acid precipitation affected the concentration of the unbound soluble fraction of the peptide inhibitors (data not shown). The unbound soluble fraction of inhibitors was subjected to size exclusion HPLC followed by mass spectrometry. The instrument used was the Agilent 6460 Triple Quadrupole LC/MS System (Agilent Technologies, Santa Clara, CA) with a TSK gel column (2.0 × 150 mm, 4 μm, 125 Å; Michrom Bioresources, Inc., Auburn, CA). Flow rate was 0.2 ml/min, and the elution buffer was 75% acetonitrile in water containing 0.1% formic acid. Known concentrations of peptide inhibitors were subjected to the same treatment to create a standard curve. The analysis of the results was performed using GraphPad Prism 6.

To analyze the tetrameric and monomeric populations of TTR, the variant M13R/L17R/T119M (MLT) was generated. This variant forms tetramer and monomers in solution that were examined as follows. A 5-fold molar excess of TTR inhibitors was added to 1 mg/ml soluble MLT and analyzed before and after incubation in 10 mm sodium acetate (pH 4.3), 100 mm KCl, 10 mm EDTA at 37 °C for 18 h. Samples were analyzed by analytical size exclusion with a Superdex 75 16/300 GL column following the manufacturer's instructions (GE Healthcare) running on an ÄKTA FPLC system in 10 mm sodium acetate (pH 7.4), 100 mm KCl, 10 mm EDTA.

### Preparation of Recombinant TTR

Mutants of TTR were produced using a QuikChange site-directed mutagenesis kit (Stratagene). Wild-type and mutated TTR genes were cloned into the pET24(+) vector (Novagen), and the sequences were verified by DNA sequencing and/or later x-ray crystallography of pure proteins (Table 1). For protein expression, exponentially growing Escherichia coli RosettaTM(DE3) pLysS competent cells (Millipore) transformed with each of the plasmids were exponentially grown and treated with 1 mm isopropyl β-d-thiogalactoside for 3 h. Wild-type TTR or its variants were purified by nickel affinity chromatography using HisTrap columns (GE Healthcare) following the manufacturer's instructions. Fractions were pooled and subjected to gel filtration chromatography using a HiLoad 16/60 Superdex 75 gel filtration column (GE Healthcare) running on an ÄKTA FPLC system. TTR-rich fractions were confirmed by SDS-PAGE. After checking that the His tag did not modify TTR aggregation (data not shown), we chose to retain the tag to facilitate further analysis of TTR aggregation by His probe blotting.

Statistics of data collection and atomic refinement

Values in parentheses correspond to the highest resolution shell. The structures of the TTR variants were used to confirm the correct substitutions. The wild-type TTR structure was used for docking. Coordinates are deposited in the Protein Data Bank under the accession codes that are listed in the table. r.m.s.d., root mean square deviation; CC1/2, correlation coefficient.

aRmerge = ΣIj − I/ΣI. Notice that the high Rmerge value of the highest resolution shell of several structures is compensated by a high CC1/2 value.

bRwork = Σ|Fo − Fc|j/ΣFo.

cRfree = Σ|Fo − Fc|j/ΣFo calculated using a random set containing 5 or 10% of reflections that were not included throughout structure refinement for protein or peptide structure determination, respectively.

### TTR Aggregation Assay

TTR aggregation assays are described elsewhere (26). Briefly, 1 mg/ml TTR sample in 10 mm sodium acetate (pH 4.3), 100 mm KCl, 10 mm EDTA was incubated at 37 °C for a maximum of 4 days. Various measurements of TTR aggregation were taken. (i) Protein aggregation was monitored by measuring absorbance of the sample at 400 nm. (ii) Protein concentration of the insoluble fraction was calculated as follows. 100 μl of sample was spun at 13,000 rpm for 30 min. The pellet was resuspended in the same volume of fresh buffer and spun again at 13,000 rpm for 30 min. The final pellet was resuspended in 6 m guanidine chloride, and absorbance at 295 nm was measured. Protein concentration of the insoluble fraction was calculated from absorbance data at 295 nm. (iii) TTR aggregation was visualized by transmission electron microscopy of the sample and/or immunodot blot of the insoluble fraction.

Aggregation assays were performed in both the presence and absence of aggregation inhibitors when appropriate at a ratio of 1:10 (TTR monomer:inhibitor) unless labeled otherwise. Turbidity of the sample (absorbance at 400 nm) and protein concentration of the insoluble fraction were measured at time 0, day 1, day 2, and day 4. Values for day 4 are presented in the figures unless labeled otherwise.

### Immunodot Blot

The aggregation of TTR was followed by dot blot analysis as described (27) using the SuperSignal® West HisProbeTM kit following the manufacturer's instructions (Thermo Scientific). The insoluble fraction of the samples after 4 days of incubation was dotted onto nitrocellulose membranes (0.2 μm; Bio-Rad). A dilution ratio of 1:10,000 was used for the HisProbe antibody.

### Fibril Formation of TTR Peptides

Peptides were dissolved in PBS buffer (pH 7.4) at a concentration that depended on the solubility of the peptide: 12LMVKVL17 at 14 mm, 25AINVAV30 at 17 mm, 26NVAVHV32 at 16 mm, 28VAVHVF33 at 15 mm, 47GKTSES52 at 16 mm, 65VEGIYK70 at 14 mm, 68IYKVEI73 at 13 mm, 80KALGIS85 at 17 mm, 91AEVVFT96 at 0.5 mm, 91APVVFT96 at 15.8 mm, 91AEVPFT96 at 15.8 mm, 105YTIAAL110 at 15 mm, 106TIAALLS112 4.6 mm, and 119TAVVTN124 at 17 mm. Following dissolution, samples were filtered through a 0.2-μm filter and incubated at 37 °C with no shaking for 2 weeks.

### Transmission Electron Microscopy (TEM)

TEM was performed to visualize the fibril formation of TTR proteins and peptides. 5 μl of the sample was spotted onto freshly glow-discharged, carbon-coated EM grids (Ted Pella, Redding, CA). After 3 min of incubation, grids were rinsed three times with 5 μl of distilled water and then stained with 2% uranyl acetate for 2 min. Grids were examined with a T12 Quick CryoEM and CryoET (FEI) transmission electron microscope at an accelerating voltage of 120 kV. Digital images were recorded using a Gatan 2,048 × 2,048 charge-coupled device camera.

### Crystallization and Structure Determination

Crystallization conditions, data collection, and refinement statistics of crystal structures are detailed in Table 1. X-ray diffraction data were collected at the Advanced Photon Source beamline 24-ID-C (for full-length proteins) or 24-ID-E (for peptides). Molecular replacement was performed with the program Phaser (28) using as search models an idealized polyalanine β-strand and chain A of the mutant T119M (Protein Data Bank code 1F86) for peptide and TTR variant data sets, respectively. Crystallographic refinement was performed using PHENIX (29), REFMAC (30), and BUSTER (31). Model building was performed with Coot (32) and illustrated with PyMOL (33).

### Sequence and Structure Analysis

To predict amyloidogenicity of TTR segments, the TTR sequence was submitted to ZipperDB, the Rosetta-based method that profiles the steric zipper spine, the main structural feature of amyloid fibrils (34, 35). Additionally, solvent-accessible surface area per residue of TTR was calculated to examine the accessibility of the different protein segments for self-association. Solvent-accessible surface area calculations were performed by Areaimol (36) using the structure of WT TTR (Protein Data Bank code 4TLT) in three different conformations: tetramer with 222 symmetry, dimer (the crystallographic asymmetric unit), and monomer (by removing one chain of the dimer). These conformations are based on the dissociation pathway described elsewhere (4).

### Design of Aggregation Inhibitors

We predicted the interaction of the peptides AEVVFT and TAVVTN with their identical segments at the surface of the monomer or at the tip of a growing steric zipper of TTR based on our finding that the two TTR segments form fibrils in solution by self-aggregation. To validate the prediction, computational docking was carried out using the Rosetta software as described previously (35) using chain A of WT TTR as a template (Protein Data Bank code 4TLT). The optimization of the aggregation inhibitors required the addition of an N-methyl group and tetra-arginine tag. The use of the non-natural N-methyl groups to protect aggregation inhibitors from proteolysis has been successful in other aggregation inhibitors (37). The addition of a polyarginine tag confers higher solubility and we presume also hinders self-aggregation of the inhibitors. The position of the N-methyl group was designed based on our docking model and experimentally checked by aggregation assays.

### Differential Scanning Calorimetry

Data were taken with a DSC-II differential scanning calorimeter (Calorimetry Sciences Corp., Lindon, UT). The midpoint temperatures of the thermal unfolding transition (Tm) of wild type and mutants were determined. Thermal denaturation of TTR was irreversible under our conditions (1, 38), and absolute thermodynamic parameters were not determined. Previously, researchers found that WT TTR unfolds at 101.7 °C under 2 atm of nitrogen (39). The experiments were performed in 3.5 m guanidinium hydrochloride to depress melting below 100 °C; this guanidinium concentration also solubilized thyroxine. Protein samples were thawed on ice and filtered, and the concentration was measured by UV-visible absorption. Protein samples were diluted to 1 mg/ml in 3.5 m guanidinium chloride, 100 mm sodium acetate (pH 4.3), 100 mm potassium chloride, 1 mm EDTA. A volume of 700 μl of each protein sample was prepared in a 1.5-ml microcentrifuge tube and degassed for 5 min under vacuum while stirring. The reference cell contained only buffer components. Samples were scanned from 25 to 95 °C at a rate of 1 °C/min. The buffer was subtracted, and the baseline was corrected with the linear polynomial function in CPCalc Analysis software. In an additional experiment, T4 and aggregation inhibitors were added to WT TTR to determine whether they affect the thermodynamic stability. T4 and the aggregation inhibitors were added in 5-fold excess relative to protein concentration to both the reference and sample chambers.

### Non-denaturing Electrophoresis

Monomeric variant MTTR at 0.5 mg/ml (40) was incubated with increasing concentrations of both TTR inhibitors R4PAm and R4PTm (0–5-fold molar excess) and subjected to blue native electrophoresis using NativePAGETM Bis-Tris precast gels (4–16% Bis-Tris, 1 mm; Life Technologies) following the manufacturer's instructions. NativeMarkTM unstained standards were used for size estimation of proteins.

### Binding Constant Calculations

The soluble unbound fraction of inhibitors after incubation with the monomeric variant MTTR (40) was quantified by HPLC-MS. Samples were made as follows. Soluble TTR at a fixed concentration of 0.5 mg/ml was incubated at 37 °C for 18 h in 10 mm sodium acetate (pH 7.4), 100 mm KCl, 10 mm EDTA with increasing quantities of combined R4PAm and R4PTm. The aggregated fraction was removed by 0.22-μm filtration. The protein fraction containing soluble TTR and inhibitor-bound complexes was then removed by acid precipitation with 1% acetic acid and consecutive ultracentrifugation at 90,000 rpm for 30 min. Neither 0.22-μm filtration nor acid precipitation affected the concentration of the unbound soluble fraction of the peptide inhibitors (data not shown). The unbound soluble fraction of inhibitors was subjected to size exclusion HPLC followed by mass spectrometry. The instrument used was the Agilent 6460 Triple Quadrupole LC/MS System (Agilent Technologies, Santa Clara, CA) with a TSK gel column (2.0 × 150 mm, 4 μm, 125 Å; Michrom Bioresources, Inc., Auburn, CA). Flow rate was 0.2 ml/min, and the elution buffer was 75% acetonitrile in water containing 0.1% formic acid. Known concentrations of peptide inhibitors were subjected to the same treatment to create a standard curve. The analysis of the results was performed using GraphPad Prism 6.

### Analytical Size Exclusion Chromatography

To analyze the tetrameric and monomeric populations of TTR, the variant M13R/L17R/T119M (MLT) was generated. This variant forms tetramer and monomers in solution that were examined as follows. A 5-fold molar excess of TTR inhibitors was added to 1 mg/ml soluble MLT and analyzed before and after incubation in 10 mm sodium acetate (pH 4.3), 100 mm KCl, 10 mm EDTA at 37 °C for 18 h. Samples were analyzed by analytical size exclusion with a Superdex 75 16/300 GL column following the manufacturer's instructions (GE Healthcare) running on an ÄKTA FPLC system in 10 mm sodium acetate (pH 7.4), 100 mm KCl, 10 mm EDTA.

### Results

Our first step toward identifying the amyloid spine-forming segments of TTR was to test candidate segments identified by computational predictions for their ability to aggregate. We used the structure-based, computational method ZipperDB to predict 6-amino acid segments of TTR likely to form amyloid fibrils (34, 35). Of the 14 segments that met the threshold of ZipperDB (Fig. 1A, red bars), we experimentally tested 12 for their propensity to form fibrils in isolation. Eight of the 12 segments formed amyloid-like aggregates having fibrillar morphology (as viewed by electron microscopy; Fig. 1B) and cross-β x-ray diffraction characteristic of amyloid (Fig. 1C) (41, 42). The segments that formed fibrils reside in the α-helix and β-strands A, B, F, G, and H of the native structure of TTR. Based on these results, we hypothesized that any of these eight segments could potentially drive TTR aggregation.

Identifying amyloidogenic segments in TTR.
A, propensities of steric zipper formation of each 6-residue segment within the TTR sequence. Computationally predicted segments with steric zipper propensities are represented with red bars. The schematic of the secondary structure of native TTR is shown on top of the sequence. Black arrows mark residues where proline replacement did not hinder TTR aggregation. Blue arrows mark proline replacements that hindered TTR aggregation. The green arrow marks the mutation T119M, which protects TTR against fibril formation (58). The green and blue boxes highlight the segment sequences critical for TTR aggregation that we described in this study. B, TEM micrographs of the fibrils formed after 7 days of incubation (scale bar, 500 nm). C, amyloid x-ray cross-β diffraction pattern of the samples containing fibrils shown in B. The arrowheads point to the meridional reflection at 4.7–4.8-Å spacings (parallel to the fibril axis; black arrowheads) and equatorial reflections at ∼10-Å spacings (white arrowheads). Notice that all eight of the examined segments predicted to form amyloid fibrils do in fact form amyloid fibrils.

To pinpoint which of the eight segments drives aggregation within full-length TTR, we created individual proline substitutions within each segment. We chose the substitution to proline because it is known to inhibit fibril formation (43, 44). None of the proline variants fully halted TTR aggregation (Fig. 2, A and B), but substitutions S85P, E92P, and V94P and the S85P/E92P double substitution significantly decreased protein aggregation and altered aggregate morphology (Fig. 2A). In the isolated segment 91AEVVFT96, proline replacements in positions 92 and 94 completely prevented fibrillization (Fig. 2C). These data suggest that the 91AEVVFT96 segment and the N-terminally adjacent region of full-length TTR are important for fibril formation.

β-Strand F as an aggregation-driving segment in TTR suitable for aggregation inhibitor design. TTR aggregation of the protein variants that showed significant delay (A) and those that did not decrease protein aggregation (B) is shown. Those TTR proline variants that are not shown here were found to be insoluble. Histograms show percentage of TTR aggregation using WT TTR to normalize to 100%. The aggregation was measured by absorbance of the samples at 400 nm after 4 days of incubation at 37 °C and pH 4.3 with no shaking. The bottom panel shows a His probe dot blot of the insoluble fraction corresponding to the sample above after solubilization with guanidinium hydrochloride. On the right are TEM micrographs of protein aggregates (scale bar, 100 nm) after 7 days of incubation. Notice that the proline substitutions within or next to β-strand F hindered TTR aggregation. Several of the other proline substitutions enhanced aggregation; these, like ATTR familial mutations, may alter native structure and/or protein stability (8). Error bars represent S.D., and ** symbolizes a p value ≤0.003 (n = 3). C, TEM micrographs of peptides in isolation after 7 days of incubation in PBS with no shaking (scale bar, 500 nm). D, crystal structure of the segment 91AEVVFT96 from β-strand F forming a Class-7 steric zipper. One sheet is shown as blue; the other is shown as gray. On the left is a lateral view of the fibril with the fibril axis shown by the narrow black arrow. On the right is the view down the fibril axis showing two β-sheets in projection. Water molecules are shown as aquamarine spheres. Spheres represent the van der Waals radii of the side chain atoms of the tightly packed fibril core.

To better understand how 91AEVVFT96 self-associates to form fibrils, we determined the structure of the fibril-like crystals of 91AEVVFT96. We found that it forms a Class-7 steric zipper (Ref. 25 and Fig. 2D) in which the β-strands stack into antiparallel β-sheets with identical side chains on both faces. This structure shows how 91AEVVFT96 may self-associate in driving TTR aggregation.

Next, we considered the possibility that TTR contains more than one aggregation-driving segment. Experiments of others show that the fibril core of TTR contains most of the protein, suggesting that more than one segment participates in the fibril core (19, 45). Because the monomeric state is more amyloidogenic than the tetrameric state (4), we reasoned that fibril-driving segments are buried in the tetramer but exposed in the monomer. In this situation, the segments would be exposed to bind to identical segments from other monomers to form a fibril. We analyzed the solvent-accessible surface area of TTR and found that β-strand F is indeed more exposed in the TTR monomer than in the dimer or tetramer (Fig. 3A). A second region was also more exposed in the monomer: β-strand H. We therefore tested whether β-strand H might also be a driver of TTR aggregation.

β-Strand H as a second aggregation-driving segment suitable as a target for aggregation inhibitor design.
A, residue solvent-accessible surface area (ASA; Å2) of TTR was calculated by Areaimol using the structure of WT TTR in three different conformations: tetramer (blue), dimer (black), and monomer (green). Notice that the strands F and H are indeed more exposed when TTR is in a monomeric form. 5.7% of the surface of the amyloidogenic segment 91AEVVFT96 is solvent-exposed in the tetramer, 5.7% is exposed in the dimer, and 32.1% is exposed in the monomer. Additionally, the amyloidogenic segment 119TAVVTN124 from strand H is 27.4% solvent-exposed in the tetramer, 43.4% solvent-exposed in the dimer, and 55.3% solvent-exposed in the monomer. B, TTR aggregation of variants with substitutions on strand H. The histogram at the top shows the percentage of TTR aggregation after 4 days of incubation measured by absorbance at 400 nm with WT aggregation normalized to 100%. Below, a His probe dot blot shows the insoluble fraction of the samples after 4 days of incubation and solubilization with guanidinium hydrochloride. Bottom panels, TEM images from the samples after 7 days of incubation. These show that the substitutions of residues Thr119 and Val121 did indeed hinder protein aggregation. Error bars represent S.D., and ** symbolizes a p value ≤0.003 (n = 3). C, TEM micrographs of peptides in isolation after 7 days of incubation in PBS with no shaking (scale bar, 500 nm). D, crystal structure of the segment 119TAVVTN124 from β-strand H forming a Class-2 steric zipper. One sheet of the zipper is gray; the other is green. On the left is a lateral view of the fibril with the fibril axis indicated by the narrow black arrow. On the right is the view down the fibril axis showing two β-sheets in projection. Spheres represent the van der Waals radii of the side chain atoms of the tightly packed fibril core.

We found that strand H also contributes to TTR amyloid aggregation but not by applying the method of proline substitution that was effective with strand F. Proline substitutions in strand H did not hinder TTR aggregation (Fig. 2). Therefore, we decided to disrupt the capacity for a steric zipper by strand H by substituting residues Thr119 and Val121 with the bulky residues tyrosine and tryptophan. The size of these side chains is expected to prevent formation of tightly packed steric zippers formed from amino acids with short side chains such as Thr119 and Val121. Aggregation assays of TTR variants showed that the mutants T119W, T119Y, V121W, and V121Y did not produce aggregates after 4 days of incubation (Fig. 3B). For additional evidence, we synthesized and analyzed 6-residue peptides containing the same substitutions, and they did not form aggregates or fibrils in isolation (Fig. 3C). These experiments suggest that strand H plays a role during TTR aggregation by acting as a fibril-forming segment.

Next, we performed structural analysis of the isolated strand H segment to test whether it forms a steric zipper structure. We found that 119TAVVTN124 forms a Class-2 steric zipper (Ref. 25 and Fig. 3D) in which the β-strands stack into parallel β-sheets with one sheet face packing against another sheet back. Taken together, these experiments suggest that the 119TAVVTN124 segment is important for fibril formation.

To confirm that strands F and H drive protein aggregation, we tested aggregation inhibition by short peptides that we designed to bind to strands F and H, thereby shielding them from aggregation. We designed amyloid inhibitors against β-strands F and H using prior strategies that have been successful at blocking aggregation of other proteins (46, 47). The design was supported by protein-protein docking (Fig. 4). To create aggregation inhibitors capable of binding the target sequence without self-associating, we modified the sequences 91AEVVFT96 and 119TAVVTN124 by adding non-natural amino acids and a charged tag. We used the docking model to predict the most suitable positions for N-methylated residues. The model predicted that even positions (Glu92, Val94, and Thr96 from 91AEVVFT96 and Ala120, Val122, and Asn124 from 119TAVVTN124) were more favorable than odd positions (Fig. 4C). This prediction was later supported by experiments.

Design of sequence-specific peptide inhibitors of TTR aggregation.
A, computational docking model of the peptides AEVVFT and TAVVTN bound to strands F (blue) and H (green), respectively, of the TTR monomer. Peptides are shown with translucent spheres representing van der Waals radii; the TTR monomer is shown as ribbons with side chains of strands F and H as sticks. B, lateral, close-up view of the F and H strands in the docking model showing the designed pattern of hydrogen bonding (dashed lines) between the peptides (sticks) and the TTR β-strands (ribbon with sticks). C, docking models of N-methylated peptides predict which will bind TTR (same view as B). N-Methylations were added to increase peptide solubility, reduce aggregation, and reduce sensitivity. Colored spheres represent the van der Waals radii of the N-methyl groups (Nme) in favorable positions; yellow stars highlight clashes of the N-methyl modifications with the TTR monomer.

Fifteen different peptides were tested as potential inhibitors of aggregation (Fig. 5). The peptide sequences are listed in Fig. 5A. We began by testing the 6-residue F- and H-strand peptides, AEVVFT and TAVVTN, with sequential modifications of N-methylated amino acids (Fig. 5B). The experimental observation that the aggregation inhibitors with N-methyl residues at even positions were more effective than those with N-methylated odd positions agrees with our docking model (Figs. 4C and 5B). We next added an N-terminal tag of four arginine residues to further improve the inhibitors by increasing their solubility and preventing self-aggregation. The addition of the N-terminal residues significantly increased the inhibition (Fig. 5B). The position of the tag was also tested at both termini, and we found that the N-terminally tagged peptides were better inhibitors (Fig. 5C).

Designed peptides are sequence-specific inhibitors of TTR aggregation.
A, list of inhibiting peptides tested as TTR aggregation inhibitors. B–E, evaluation of the peptide inhibitors. The graphs show the percentage of TTR aggregation after 4 days of incubation in the presence or absence of peptide inhibitor measured by absorbance at 400 nm; WT inhibition-free aggregation was normalized to 100%. The initial concentration of soluble TTR was 1 (B–D) or 0.2 mg/ml, which corresponds to the concentration of TTR in plasma (E). The molar excess of peptide inhibitor over target (TTR monomer) is 3-fold unless labeled otherwise. In C and D, a His probe dot blot shows the insoluble fraction of the samples after 4 days of incubation and solubilization with guanidinium hydrochloride. B, initial screening of inhibitors showing that three modifications significantly improved the effectiveness: (i) increasing the length of the matched sequence of the peptide and the target, (ii) addition of a charged tag, and (iii) addition of an N-methyl group in positions predicted to be favorable by the docking model (Fig. 4C). The two best inhibitors of TTR aggregation were R4PAm and R4PTm. C, the peptide inhibitors are more effective with the charged tag at the N terminus than at the C terminus. D, dose-dependent effectiveness of the peptides R4PAm and R4PTm in combination. Maximal inhibition of TTR aggregation is reached when the target to peptide molar ratio is 1:3. E, aggregation assay of the familial mutants V30M and L55P in the presence of R4PAm and R4PTm in combination using a physiological concentration of protein (3.6 μm). Error bars represent S.D. (n = 3), and ** symbolizes a p value ≤0.003.

We assessed the specificity of the inhibitors in two ways. First, we tested a scrambled control peptide with the polyarginine tag intact (Fig. 5B). Charged tags might form salt bridges with several of the loops in TTR that are rich in acidic residues (48). The control peptide did not disrupt protein aggregation, confirming that the aggregation inhibition is sequence-specific and not reliant on the arginine tag (Fig. 5B). Then we tested the combination of F and H inhibitors. The peptides R4PAm and R4PTm showed a synergistic effect, suggesting two different binding sites (Fig. 5B), in a dose-dependent manner (Fig. 5D). The combination was also effective at blocking aggregation of the familial mutants V30M and L55P at physiological TTR concentration (Fig. 5E). Based on this initial screening, the two peptides R4PAm and R4PTm in combination were selected for further study.

To further characterize the mechanism of action of the inhibitor pair, we examined their effect on different assembly states of TTR. First, we examined tetramer stability by differential scanning calorimetry (Fig. 6A). We found that the inhibitors did not affect WT TTR stability or compete with T4 for its binding site (Fig. 6A). These results confirm that the mechanism for inhibiting aggregation is different from tetramer stabilization (9, 10, 49). Additionally, we examined the effect on oligomer formation of an engineered monomeric variant of TTR, MTTR (40). MTTR has two methionine substitutions at residues Phe87 and Leu110, which prevent dimer and tetramer formation, leading to rapid aggregation at acidic pH. Incubation of MTTR with the inhibitor pair reduced oligomer formation as monitored by non-denaturing gel electrophoresis (Fig. 6B). The analysis of the binding of the TTR inhibitors to MTTR species showed an apparent binding constant of 17.9 ± 6.1 and 19.3 ± 6.7 μm for R4PAm and R4PTm, respectively (Fig. 6C). This experiment did not exclude the binding of the inhibitors to aggregated species. The measured affinity is therefore a combined affinity for all species in solution, and the monomer-specific affinity might be weaker than calculated.

Analysis of the mechanism of action of the TTR inhibitors.
A, midpoint temperatures of the thermal unfolding transition (Tm) of wild-type TTR at different conditions were determined by differential scanning calorimetry. Relative stability is compared in the presence and absence of a 5:1 molar ratio of T4 and/or the combination of R4PAm and R4PTm to TTR monomer. The TTR concentration was 1 mg/ml. Notice that the addition of the natural ligand T4 increased the protein thermostability by 10.8 °C. The addition of the peptide inhibitors increased it only by 1.3 °C in the absence of T4 and by 0.9 °C when the ligand was present. B, inhibition of oligomer formation of MTTR after incubation with R4PTm and R4PAm. A fixed amount of MTTR (0.5 mg/ml) was incubated in the presence of increasing concentrations of TTR inhibitors (0–5-fold excess) and subjected to non-denaturing electrophoresis. The sizes of His-tagged TTR monomer (14 kDa) and tetramer (56 kDa) are shown next to the gel. R4PTm and R4PAm inhibited MTTR oligomer formation in a dose-dependent manner. C, fraction of the inhibitors R4PAm (solid line, squares) and R4PTm (dashed line, circles) bound to MTTR. The soluble fraction of the samples was collected after ultracentrifugation followed by acid precipitation of soluble MTTR and inhibitor-bound complexes. The unbound fraction of inhibitors was analyzed by HPLC-MS. One-site specific binding analysis was performed using GraphPad Prism 6 to calculate binding parameters KD as apparent binding affinity and Bmax as maximum bound fraction. D, analysis of tetrameric and monomeric populations of MLT after incubation with TTR inhibitors. Size exclusion chromatography of MLT reveals that the presence of TTR inhibitors does not drive the protein equilibrium to populate the monomeric species. In contrast, an increase of tetrameric species was found after incubation with inhibitors (Inh). TTR aggregation (TTR agg.) was measured by absorbance at 400 nm with the initial protein amount normalized to 100%. mAU, milli-absorbance units; N.A., not applicable.

We next found that the presence of inhibitors leads to an equilibrium shift toward the tetramer population. We first generated the TTR variant MLT, which exhibits monomers and tetramers in solution, and we analyzed both populations by size exclusion chromatography (Fig. 6D). The results show that the addition of inhibitors promotes an immediate increase in tetramer concentration when compared with the absence of inhibitors. Additionally, the monomeric concentration remains unchanged after incubation for 18 h in the presence of the inhibitors, whereas their absence results in monomer loss and aggregation. Taken together, these results suggest a mechanism of action that does not stabilize the monomer population and differs from the tetramer stabilization.

In summary, our results indicate that strands F and H play a causative role in TTR aggregation. We used this hypothesis to design inhibitors of their self-aggregation to stop amyloid formation of wild-type TTR and familial mutant variants.

### 

Our first step toward identifying the amyloid spine-forming segments of TTR was to test candidate segments identified by computational predictions for their ability to aggregate. We used the structure-based, computational method ZipperDB to predict 6-amino acid segments of TTR likely to form amyloid fibrils (34, 35). Of the 14 segments that met the threshold of ZipperDB (Fig. 1A, red bars), we experimentally tested 12 for their propensity to form fibrils in isolation. Eight of the 12 segments formed amyloid-like aggregates having fibrillar morphology (as viewed by electron microscopy; Fig. 1B) and cross-β x-ray diffraction characteristic of amyloid (Fig. 1C) (41, 42). The segments that formed fibrils reside in the α-helix and β-strands A, B, F, G, and H of the native structure of TTR. Based on these results, we hypothesized that any of these eight segments could potentially drive TTR aggregation.

Identifying amyloidogenic segments in TTR.
A, propensities of steric zipper formation of each 6-residue segment within the TTR sequence. Computationally predicted segments with steric zipper propensities are represented with red bars. The schematic of the secondary structure of native TTR is shown on top of the sequence. Black arrows mark residues where proline replacement did not hinder TTR aggregation. Blue arrows mark proline replacements that hindered TTR aggregation. The green arrow marks the mutation T119M, which protects TTR against fibril formation (58). The green and blue boxes highlight the segment sequences critical for TTR aggregation that we described in this study. B, TEM micrographs of the fibrils formed after 7 days of incubation (scale bar, 500 nm). C, amyloid x-ray cross-β diffraction pattern of the samples containing fibrils shown in B. The arrowheads point to the meridional reflection at 4.7–4.8-Å spacings (parallel to the fibril axis; black arrowheads) and equatorial reflections at ∼10-Å spacings (white arrowheads). Notice that all eight of the examined segments predicted to form amyloid fibrils do in fact form amyloid fibrils.

To pinpoint which of the eight segments drives aggregation within full-length TTR, we created individual proline substitutions within each segment. We chose the substitution to proline because it is known to inhibit fibril formation (43, 44). None of the proline variants fully halted TTR aggregation (Fig. 2, A and B), but substitutions S85P, E92P, and V94P and the S85P/E92P double substitution significantly decreased protein aggregation and altered aggregate morphology (Fig. 2A). In the isolated segment 91AEVVFT96, proline replacements in positions 92 and 94 completely prevented fibrillization (Fig. 2C). These data suggest that the 91AEVVFT96 segment and the N-terminally adjacent region of full-length TTR are important for fibril formation.

β-Strand F as an aggregation-driving segment in TTR suitable for aggregation inhibitor design. TTR aggregation of the protein variants that showed significant delay (A) and those that did not decrease protein aggregation (B) is shown. Those TTR proline variants that are not shown here were found to be insoluble. Histograms show percentage of TTR aggregation using WT TTR to normalize to 100%. The aggregation was measured by absorbance of the samples at 400 nm after 4 days of incubation at 37 °C and pH 4.3 with no shaking. The bottom panel shows a His probe dot blot of the insoluble fraction corresponding to the sample above after solubilization with guanidinium hydrochloride. On the right are TEM micrographs of protein aggregates (scale bar, 100 nm) after 7 days of incubation. Notice that the proline substitutions within or next to β-strand F hindered TTR aggregation. Several of the other proline substitutions enhanced aggregation; these, like ATTR familial mutations, may alter native structure and/or protein stability (8). Error bars represent S.D., and ** symbolizes a p value ≤0.003 (n = 3). C, TEM micrographs of peptides in isolation after 7 days of incubation in PBS with no shaking (scale bar, 500 nm). D, crystal structure of the segment 91AEVVFT96 from β-strand F forming a Class-7 steric zipper. One sheet is shown as blue; the other is shown as gray. On the left is a lateral view of the fibril with the fibril axis shown by the narrow black arrow. On the right is the view down the fibril axis showing two β-sheets in projection. Water molecules are shown as aquamarine spheres. Spheres represent the van der Waals radii of the side chain atoms of the tightly packed fibril core.

To better understand how 91AEVVFT96 self-associates to form fibrils, we determined the structure of the fibril-like crystals of 91AEVVFT96. We found that it forms a Class-7 steric zipper (Ref. 25 and Fig. 2D) in which the β-strands stack into antiparallel β-sheets with identical side chains on both faces. This structure shows how 91AEVVFT96 may self-associate in driving TTR aggregation.

Next, we considered the possibility that TTR contains more than one aggregation-driving segment. Experiments of others show that the fibril core of TTR contains most of the protein, suggesting that more than one segment participates in the fibril core (19, 45). Because the monomeric state is more amyloidogenic than the tetrameric state (4), we reasoned that fibril-driving segments are buried in the tetramer but exposed in the monomer. In this situation, the segments would be exposed to bind to identical segments from other monomers to form a fibril. We analyzed the solvent-accessible surface area of TTR and found that β-strand F is indeed more exposed in the TTR monomer than in the dimer or tetramer (Fig. 3A). A second region was also more exposed in the monomer: β-strand H. We therefore tested whether β-strand H might also be a driver of TTR aggregation.

β-Strand H as a second aggregation-driving segment suitable as a target for aggregation inhibitor design.
A, residue solvent-accessible surface area (ASA; Å2) of TTR was calculated by Areaimol using the structure of WT TTR in three different conformations: tetramer (blue), dimer (black), and monomer (green). Notice that the strands F and H are indeed more exposed when TTR is in a monomeric form. 5.7% of the surface of the amyloidogenic segment 91AEVVFT96 is solvent-exposed in the tetramer, 5.7% is exposed in the dimer, and 32.1% is exposed in the monomer. Additionally, the amyloidogenic segment 119TAVVTN124 from strand H is 27.4% solvent-exposed in the tetramer, 43.4% solvent-exposed in the dimer, and 55.3% solvent-exposed in the monomer. B, TTR aggregation of variants with substitutions on strand H. The histogram at the top shows the percentage of TTR aggregation after 4 days of incubation measured by absorbance at 400 nm with WT aggregation normalized to 100%. Below, a His probe dot blot shows the insoluble fraction of the samples after 4 days of incubation and solubilization with guanidinium hydrochloride. Bottom panels, TEM images from the samples after 7 days of incubation. These show that the substitutions of residues Thr119 and Val121 did indeed hinder protein aggregation. Error bars represent S.D., and ** symbolizes a p value ≤0.003 (n = 3). C, TEM micrographs of peptides in isolation after 7 days of incubation in PBS with no shaking (scale bar, 500 nm). D, crystal structure of the segment 119TAVVTN124 from β-strand H forming a Class-2 steric zipper. One sheet of the zipper is gray; the other is green. On the left is a lateral view of the fibril with the fibril axis indicated by the narrow black arrow. On the right is the view down the fibril axis showing two β-sheets in projection. Spheres represent the van der Waals radii of the side chain atoms of the tightly packed fibril core.

We found that strand H also contributes to TTR amyloid aggregation but not by applying the method of proline substitution that was effective with strand F. Proline substitutions in strand H did not hinder TTR aggregation (Fig. 2). Therefore, we decided to disrupt the capacity for a steric zipper by strand H by substituting residues Thr119 and Val121 with the bulky residues tyrosine and tryptophan. The size of these side chains is expected to prevent formation of tightly packed steric zippers formed from amino acids with short side chains such as Thr119 and Val121. Aggregation assays of TTR variants showed that the mutants T119W, T119Y, V121W, and V121Y did not produce aggregates after 4 days of incubation (Fig. 3B). For additional evidence, we synthesized and analyzed 6-residue peptides containing the same substitutions, and they did not form aggregates or fibrils in isolation (Fig. 3C). These experiments suggest that strand H plays a role during TTR aggregation by acting as a fibril-forming segment.

Next, we performed structural analysis of the isolated strand H segment to test whether it forms a steric zipper structure. We found that 119TAVVTN124 forms a Class-2 steric zipper (Ref. 25 and Fig. 3D) in which the β-strands stack into parallel β-sheets with one sheet face packing against another sheet back. Taken together, these experiments suggest that the 119TAVVTN124 segment is important for fibril formation.

To confirm that strands F and H drive protein aggregation, we tested aggregation inhibition by short peptides that we designed to bind to strands F and H, thereby shielding them from aggregation. We designed amyloid inhibitors against β-strands F and H using prior strategies that have been successful at blocking aggregation of other proteins (46, 47). The design was supported by protein-protein docking (Fig. 4). To create aggregation inhibitors capable of binding the target sequence without self-associating, we modified the sequences 91AEVVFT96 and 119TAVVTN124 by adding non-natural amino acids and a charged tag. We used the docking model to predict the most suitable positions for N-methylated residues. The model predicted that even positions (Glu92, Val94, and Thr96 from 91AEVVFT96 and Ala120, Val122, and Asn124 from 119TAVVTN124) were more favorable than odd positions (Fig. 4C). This prediction was later supported by experiments.

Design of sequence-specific peptide inhibitors of TTR aggregation.
A, computational docking model of the peptides AEVVFT and TAVVTN bound to strands F (blue) and H (green), respectively, of the TTR monomer. Peptides are shown with translucent spheres representing van der Waals radii; the TTR monomer is shown as ribbons with side chains of strands F and H as sticks. B, lateral, close-up view of the F and H strands in the docking model showing the designed pattern of hydrogen bonding (dashed lines) between the peptides (sticks) and the TTR β-strands (ribbon with sticks). C, docking models of N-methylated peptides predict which will bind TTR (same view as B). N-Methylations were added to increase peptide solubility, reduce aggregation, and reduce sensitivity. Colored spheres represent the van der Waals radii of the N-methyl groups (Nme) in favorable positions; yellow stars highlight clashes of the N-methyl modifications with the TTR monomer.

Fifteen different peptides were tested as potential inhibitors of aggregation (Fig. 5). The peptide sequences are listed in Fig. 5A. We began by testing the 6-residue F- and H-strand peptides, AEVVFT and TAVVTN, with sequential modifications of N-methylated amino acids (Fig. 5B). The experimental observation that the aggregation inhibitors with N-methyl residues at even positions were more effective than those with N-methylated odd positions agrees with our docking model (Figs. 4C and 5B). We next added an N-terminal tag of four arginine residues to further improve the inhibitors by increasing their solubility and preventing self-aggregation. The addition of the N-terminal residues significantly increased the inhibition (Fig. 5B). The position of the tag was also tested at both termini, and we found that the N-terminally tagged peptides were better inhibitors (Fig. 5C).

Designed peptides are sequence-specific inhibitors of TTR aggregation.
A, list of inhibiting peptides tested as TTR aggregation inhibitors. B–E, evaluation of the peptide inhibitors. The graphs show the percentage of TTR aggregation after 4 days of incubation in the presence or absence of peptide inhibitor measured by absorbance at 400 nm; WT inhibition-free aggregation was normalized to 100%. The initial concentration of soluble TTR was 1 (B–D) or 0.2 mg/ml, which corresponds to the concentration of TTR in plasma (E). The molar excess of peptide inhibitor over target (TTR monomer) is 3-fold unless labeled otherwise. In C and D, a His probe dot blot shows the insoluble fraction of the samples after 4 days of incubation and solubilization with guanidinium hydrochloride. B, initial screening of inhibitors showing that three modifications significantly improved the effectiveness: (i) increasing the length of the matched sequence of the peptide and the target, (ii) addition of a charged tag, and (iii) addition of an N-methyl group in positions predicted to be favorable by the docking model (Fig. 4C). The two best inhibitors of TTR aggregation were R4PAm and R4PTm. C, the peptide inhibitors are more effective with the charged tag at the N terminus than at the C terminus. D, dose-dependent effectiveness of the peptides R4PAm and R4PTm in combination. Maximal inhibition of TTR aggregation is reached when the target to peptide molar ratio is 1:3. E, aggregation assay of the familial mutants V30M and L55P in the presence of R4PAm and R4PTm in combination using a physiological concentration of protein (3.6 μm). Error bars represent S.D. (n = 3), and ** symbolizes a p value ≤0.003.

We assessed the specificity of the inhibitors in two ways. First, we tested a scrambled control peptide with the polyarginine tag intact (Fig. 5B). Charged tags might form salt bridges with several of the loops in TTR that are rich in acidic residues (48). The control peptide did not disrupt protein aggregation, confirming that the aggregation inhibition is sequence-specific and not reliant on the arginine tag (Fig. 5B). Then we tested the combination of F and H inhibitors. The peptides R4PAm and R4PTm showed a synergistic effect, suggesting two different binding sites (Fig. 5B), in a dose-dependent manner (Fig. 5D). The combination was also effective at blocking aggregation of the familial mutants V30M and L55P at physiological TTR concentration (Fig. 5E). Based on this initial screening, the two peptides R4PAm and R4PTm in combination were selected for further study.

To further characterize the mechanism of action of the inhibitor pair, we examined their effect on different assembly states of TTR. First, we examined tetramer stability by differential scanning calorimetry (Fig. 6A). We found that the inhibitors did not affect WT TTR stability or compete with T4 for its binding site (Fig. 6A). These results confirm that the mechanism for inhibiting aggregation is different from tetramer stabilization (9, 10, 49). Additionally, we examined the effect on oligomer formation of an engineered monomeric variant of TTR, MTTR (40). MTTR has two methionine substitutions at residues Phe87 and Leu110, which prevent dimer and tetramer formation, leading to rapid aggregation at acidic pH. Incubation of MTTR with the inhibitor pair reduced oligomer formation as monitored by non-denaturing gel electrophoresis (Fig. 6B). The analysis of the binding of the TTR inhibitors to MTTR species showed an apparent binding constant of 17.9 ± 6.1 and 19.3 ± 6.7 μm for R4PAm and R4PTm, respectively (Fig. 6C). This experiment did not exclude the binding of the inhibitors to aggregated species. The measured affinity is therefore a combined affinity for all species in solution, and the monomer-specific affinity might be weaker than calculated.

Analysis of the mechanism of action of the TTR inhibitors.
A, midpoint temperatures of the thermal unfolding transition (Tm) of wild-type TTR at different conditions were determined by differential scanning calorimetry. Relative stability is compared in the presence and absence of a 5:1 molar ratio of T4 and/or the combination of R4PAm and R4PTm to TTR monomer. The TTR concentration was 1 mg/ml. Notice that the addition of the natural ligand T4 increased the protein thermostability by 10.8 °C. The addition of the peptide inhibitors increased it only by 1.3 °C in the absence of T4 and by 0.9 °C when the ligand was present. B, inhibition of oligomer formation of MTTR after incubation with R4PTm and R4PAm. A fixed amount of MTTR (0.5 mg/ml) was incubated in the presence of increasing concentrations of TTR inhibitors (0–5-fold excess) and subjected to non-denaturing electrophoresis. The sizes of His-tagged TTR monomer (14 kDa) and tetramer (56 kDa) are shown next to the gel. R4PTm and R4PAm inhibited MTTR oligomer formation in a dose-dependent manner. C, fraction of the inhibitors R4PAm (solid line, squares) and R4PTm (dashed line, circles) bound to MTTR. The soluble fraction of the samples was collected after ultracentrifugation followed by acid precipitation of soluble MTTR and inhibitor-bound complexes. The unbound fraction of inhibitors was analyzed by HPLC-MS. One-site specific binding analysis was performed using GraphPad Prism 6 to calculate binding parameters KD as apparent binding affinity and Bmax as maximum bound fraction. D, analysis of tetrameric and monomeric populations of MLT after incubation with TTR inhibitors. Size exclusion chromatography of MLT reveals that the presence of TTR inhibitors does not drive the protein equilibrium to populate the monomeric species. In contrast, an increase of tetrameric species was found after incubation with inhibitors (Inh). TTR aggregation (TTR agg.) was measured by absorbance at 400 nm with the initial protein amount normalized to 100%. mAU, milli-absorbance units; N.A., not applicable.

We next found that the presence of inhibitors leads to an equilibrium shift toward the tetramer population. We first generated the TTR variant MLT, which exhibits monomers and tetramers in solution, and we analyzed both populations by size exclusion chromatography (Fig. 6D). The results show that the addition of inhibitors promotes an immediate increase in tetramer concentration when compared with the absence of inhibitors. Additionally, the monomeric concentration remains unchanged after incubation for 18 h in the presence of the inhibitors, whereas their absence results in monomer loss and aggregation. Taken together, these results suggest a mechanism of action that does not stabilize the monomer population and differs from the tetramer stabilization.

In summary, our results indicate that strands F and H play a causative role in TTR aggregation. We used this hypothesis to design inhibitors of their self-aggregation to stop amyloid formation of wild-type TTR and familial mutant variants.

### 

Our first step toward identifying the amyloid spine-forming segments of TTR was to test candidate segments identified by computational predictions for their ability to aggregate. We used the structure-based, computational method ZipperDB to predict 6-amino acid segments of TTR likely to form amyloid fibrils (34, 35). Of the 14 segments that met the threshold of ZipperDB (Fig. 1A, red bars), we experimentally tested 12 for their propensity to form fibrils in isolation. Eight of the 12 segments formed amyloid-like aggregates having fibrillar morphology (as viewed by electron microscopy; Fig. 1B) and cross-β x-ray diffraction characteristic of amyloid (Fig. 1C) (41, 42). The segments that formed fibrils reside in the α-helix and β-strands A, B, F, G, and H of the native structure of TTR. Based on these results, we hypothesized that any of these eight segments could potentially drive TTR aggregation.

Identifying amyloidogenic segments in TTR.
A, propensities of steric zipper formation of each 6-residue segment within the TTR sequence. Computationally predicted segments with steric zipper propensities are represented with red bars. The schematic of the secondary structure of native TTR is shown on top of the sequence. Black arrows mark residues where proline replacement did not hinder TTR aggregation. Blue arrows mark proline replacements that hindered TTR aggregation. The green arrow marks the mutation T119M, which protects TTR against fibril formation (58). The green and blue boxes highlight the segment sequences critical for TTR aggregation that we described in this study. B, TEM micrographs of the fibrils formed after 7 days of incubation (scale bar, 500 nm). C, amyloid x-ray cross-β diffraction pattern of the samples containing fibrils shown in B. The arrowheads point to the meridional reflection at 4.7–4.8-Å spacings (parallel to the fibril axis; black arrowheads) and equatorial reflections at ∼10-Å spacings (white arrowheads). Notice that all eight of the examined segments predicted to form amyloid fibrils do in fact form amyloid fibrils.

To pinpoint which of the eight segments drives aggregation within full-length TTR, we created individual proline substitutions within each segment. We chose the substitution to proline because it is known to inhibit fibril formation (43, 44). None of the proline variants fully halted TTR aggregation (Fig. 2, A and B), but substitutions S85P, E92P, and V94P and the S85P/E92P double substitution significantly decreased protein aggregation and altered aggregate morphology (Fig. 2A). In the isolated segment 91AEVVFT96, proline replacements in positions 92 and 94 completely prevented fibrillization (Fig. 2C). These data suggest that the 91AEVVFT96 segment and the N-terminally adjacent region of full-length TTR are important for fibril formation.

β-Strand F as an aggregation-driving segment in TTR suitable for aggregation inhibitor design. TTR aggregation of the protein variants that showed significant delay (A) and those that did not decrease protein aggregation (B) is shown. Those TTR proline variants that are not shown here were found to be insoluble. Histograms show percentage of TTR aggregation using WT TTR to normalize to 100%. The aggregation was measured by absorbance of the samples at 400 nm after 4 days of incubation at 37 °C and pH 4.3 with no shaking. The bottom panel shows a His probe dot blot of the insoluble fraction corresponding to the sample above after solubilization with guanidinium hydrochloride. On the right are TEM micrographs of protein aggregates (scale bar, 100 nm) after 7 days of incubation. Notice that the proline substitutions within or next to β-strand F hindered TTR aggregation. Several of the other proline substitutions enhanced aggregation; these, like ATTR familial mutations, may alter native structure and/or protein stability (8). Error bars represent S.D., and ** symbolizes a p value ≤0.003 (n = 3). C, TEM micrographs of peptides in isolation after 7 days of incubation in PBS with no shaking (scale bar, 500 nm). D, crystal structure of the segment 91AEVVFT96 from β-strand F forming a Class-7 steric zipper. One sheet is shown as blue; the other is shown as gray. On the left is a lateral view of the fibril with the fibril axis shown by the narrow black arrow. On the right is the view down the fibril axis showing two β-sheets in projection. Water molecules are shown as aquamarine spheres. Spheres represent the van der Waals radii of the side chain atoms of the tightly packed fibril core.

To better understand how 91AEVVFT96 self-associates to form fibrils, we determined the structure of the fibril-like crystals of 91AEVVFT96. We found that it forms a Class-7 steric zipper (Ref. 25 and Fig. 2D) in which the β-strands stack into antiparallel β-sheets with identical side chains on both faces. This structure shows how 91AEVVFT96 may self-associate in driving TTR aggregation.

Next, we considered the possibility that TTR contains more than one aggregation-driving segment. Experiments of others show that the fibril core of TTR contains most of the protein, suggesting that more than one segment participates in the fibril core (19, 45). Because the monomeric state is more amyloidogenic than the tetrameric state (4), we reasoned that fibril-driving segments are buried in the tetramer but exposed in the monomer. In this situation, the segments would be exposed to bind to identical segments from other monomers to form a fibril. We analyzed the solvent-accessible surface area of TTR and found that β-strand F is indeed more exposed in the TTR monomer than in the dimer or tetramer (Fig. 3A). A second region was also more exposed in the monomer: β-strand H. We therefore tested whether β-strand H might also be a driver of TTR aggregation.

β-Strand H as a second aggregation-driving segment suitable as a target for aggregation inhibitor design.
A, residue solvent-accessible surface area (ASA; Å2) of TTR was calculated by Areaimol using the structure of WT TTR in three different conformations: tetramer (blue), dimer (black), and monomer (green). Notice that the strands F and H are indeed more exposed when TTR is in a monomeric form. 5.7% of the surface of the amyloidogenic segment 91AEVVFT96 is solvent-exposed in the tetramer, 5.7% is exposed in the dimer, and 32.1% is exposed in the monomer. Additionally, the amyloidogenic segment 119TAVVTN124 from strand H is 27.4% solvent-exposed in the tetramer, 43.4% solvent-exposed in the dimer, and 55.3% solvent-exposed in the monomer. B, TTR aggregation of variants with substitutions on strand H. The histogram at the top shows the percentage of TTR aggregation after 4 days of incubation measured by absorbance at 400 nm with WT aggregation normalized to 100%. Below, a His probe dot blot shows the insoluble fraction of the samples after 4 days of incubation and solubilization with guanidinium hydrochloride. Bottom panels, TEM images from the samples after 7 days of incubation. These show that the substitutions of residues Thr119 and Val121 did indeed hinder protein aggregation. Error bars represent S.D., and ** symbolizes a p value ≤0.003 (n = 3). C, TEM micrographs of peptides in isolation after 7 days of incubation in PBS with no shaking (scale bar, 500 nm). D, crystal structure of the segment 119TAVVTN124 from β-strand H forming a Class-2 steric zipper. One sheet of the zipper is gray; the other is green. On the left is a lateral view of the fibril with the fibril axis indicated by the narrow black arrow. On the right is the view down the fibril axis showing two β-sheets in projection. Spheres represent the van der Waals radii of the side chain atoms of the tightly packed fibril core.

We found that strand H also contributes to TTR amyloid aggregation but not by applying the method of proline substitution that was effective with strand F. Proline substitutions in strand H did not hinder TTR aggregation (Fig. 2). Therefore, we decided to disrupt the capacity for a steric zipper by strand H by substituting residues Thr119 and Val121 with the bulky residues tyrosine and tryptophan. The size of these side chains is expected to prevent formation of tightly packed steric zippers formed from amino acids with short side chains such as Thr119 and Val121. Aggregation assays of TTR variants showed that the mutants T119W, T119Y, V121W, and V121Y did not produce aggregates after 4 days of incubation (Fig. 3B). For additional evidence, we synthesized and analyzed 6-residue peptides containing the same substitutions, and they did not form aggregates or fibrils in isolation (Fig. 3C). These experiments suggest that strand H plays a role during TTR aggregation by acting as a fibril-forming segment.

Next, we performed structural analysis of the isolated strand H segment to test whether it forms a steric zipper structure. We found that 119TAVVTN124 forms a Class-2 steric zipper (Ref. 25 and Fig. 3D) in which the β-strands stack into parallel β-sheets with one sheet face packing against another sheet back. Taken together, these experiments suggest that the 119TAVVTN124 segment is important for fibril formation.

To confirm that strands F and H drive protein aggregation, we tested aggregation inhibition by short peptides that we designed to bind to strands F and H, thereby shielding them from aggregation. We designed amyloid inhibitors against β-strands F and H using prior strategies that have been successful at blocking aggregation of other proteins (46, 47). The design was supported by protein-protein docking (Fig. 4). To create aggregation inhibitors capable of binding the target sequence without self-associating, we modified the sequences 91AEVVFT96 and 119TAVVTN124 by adding non-natural amino acids and a charged tag. We used the docking model to predict the most suitable positions for N-methylated residues. The model predicted that even positions (Glu92, Val94, and Thr96 from 91AEVVFT96 and Ala120, Val122, and Asn124 from 119TAVVTN124) were more favorable than odd positions (Fig. 4C). This prediction was later supported by experiments.

Design of sequence-specific peptide inhibitors of TTR aggregation.
A, computational docking model of the peptides AEVVFT and TAVVTN bound to strands F (blue) and H (green), respectively, of the TTR monomer. Peptides are shown with translucent spheres representing van der Waals radii; the TTR monomer is shown as ribbons with side chains of strands F and H as sticks. B, lateral, close-up view of the F and H strands in the docking model showing the designed pattern of hydrogen bonding (dashed lines) between the peptides (sticks) and the TTR β-strands (ribbon with sticks). C, docking models of N-methylated peptides predict which will bind TTR (same view as B). N-Methylations were added to increase peptide solubility, reduce aggregation, and reduce sensitivity. Colored spheres represent the van der Waals radii of the N-methyl groups (Nme) in favorable positions; yellow stars highlight clashes of the N-methyl modifications with the TTR monomer.

Fifteen different peptides were tested as potential inhibitors of aggregation (Fig. 5). The peptide sequences are listed in Fig. 5A. We began by testing the 6-residue F- and H-strand peptides, AEVVFT and TAVVTN, with sequential modifications of N-methylated amino acids (Fig. 5B). The experimental observation that the aggregation inhibitors with N-methyl residues at even positions were more effective than those with N-methylated odd positions agrees with our docking model (Figs. 4C and 5B). We next added an N-terminal tag of four arginine residues to further improve the inhibitors by increasing their solubility and preventing self-aggregation. The addition of the N-terminal residues significantly increased the inhibition (Fig. 5B). The position of the tag was also tested at both termini, and we found that the N-terminally tagged peptides were better inhibitors (Fig. 5C).

Designed peptides are sequence-specific inhibitors of TTR aggregation.
A, list of inhibiting peptides tested as TTR aggregation inhibitors. B–E, evaluation of the peptide inhibitors. The graphs show the percentage of TTR aggregation after 4 days of incubation in the presence or absence of peptide inhibitor measured by absorbance at 400 nm; WT inhibition-free aggregation was normalized to 100%. The initial concentration of soluble TTR was 1 (B–D) or 0.2 mg/ml, which corresponds to the concentration of TTR in plasma (E). The molar excess of peptide inhibitor over target (TTR monomer) is 3-fold unless labeled otherwise. In C and D, a His probe dot blot shows the insoluble fraction of the samples after 4 days of incubation and solubilization with guanidinium hydrochloride. B, initial screening of inhibitors showing that three modifications significantly improved the effectiveness: (i) increasing the length of the matched sequence of the peptide and the target, (ii) addition of a charged tag, and (iii) addition of an N-methyl group in positions predicted to be favorable by the docking model (Fig. 4C). The two best inhibitors of TTR aggregation were R4PAm and R4PTm. C, the peptide inhibitors are more effective with the charged tag at the N terminus than at the C terminus. D, dose-dependent effectiveness of the peptides R4PAm and R4PTm in combination. Maximal inhibition of TTR aggregation is reached when the target to peptide molar ratio is 1:3. E, aggregation assay of the familial mutants V30M and L55P in the presence of R4PAm and R4PTm in combination using a physiological concentration of protein (3.6 μm). Error bars represent S.D. (n = 3), and ** symbolizes a p value ≤0.003.

We assessed the specificity of the inhibitors in two ways. First, we tested a scrambled control peptide with the polyarginine tag intact (Fig. 5B). Charged tags might form salt bridges with several of the loops in TTR that are rich in acidic residues (48). The control peptide did not disrupt protein aggregation, confirming that the aggregation inhibition is sequence-specific and not reliant on the arginine tag (Fig. 5B). Then we tested the combination of F and H inhibitors. The peptides R4PAm and R4PTm showed a synergistic effect, suggesting two different binding sites (Fig. 5B), in a dose-dependent manner (Fig. 5D). The combination was also effective at blocking aggregation of the familial mutants V30M and L55P at physiological TTR concentration (Fig. 5E). Based on this initial screening, the two peptides R4PAm and R4PTm in combination were selected for further study.

To further characterize the mechanism of action of the inhibitor pair, we examined their effect on different assembly states of TTR. First, we examined tetramer stability by differential scanning calorimetry (Fig. 6A). We found that the inhibitors did not affect WT TTR stability or compete with T4 for its binding site (Fig. 6A). These results confirm that the mechanism for inhibiting aggregation is different from tetramer stabilization (9, 10, 49). Additionally, we examined the effect on oligomer formation of an engineered monomeric variant of TTR, MTTR (40). MTTR has two methionine substitutions at residues Phe87 and Leu110, which prevent dimer and tetramer formation, leading to rapid aggregation at acidic pH. Incubation of MTTR with the inhibitor pair reduced oligomer formation as monitored by non-denaturing gel electrophoresis (Fig. 6B). The analysis of the binding of the TTR inhibitors to MTTR species showed an apparent binding constant of 17.9 ± 6.1 and 19.3 ± 6.7 μm for R4PAm and R4PTm, respectively (Fig. 6C). This experiment did not exclude the binding of the inhibitors to aggregated species. The measured affinity is therefore a combined affinity for all species in solution, and the monomer-specific affinity might be weaker than calculated.

Analysis of the mechanism of action of the TTR inhibitors.
A, midpoint temperatures of the thermal unfolding transition (Tm) of wild-type TTR at different conditions were determined by differential scanning calorimetry. Relative stability is compared in the presence and absence of a 5:1 molar ratio of T4 and/or the combination of R4PAm and R4PTm to TTR monomer. The TTR concentration was 1 mg/ml. Notice that the addition of the natural ligand T4 increased the protein thermostability by 10.8 °C. The addition of the peptide inhibitors increased it only by 1.3 °C in the absence of T4 and by 0.9 °C when the ligand was present. B, inhibition of oligomer formation of MTTR after incubation with R4PTm and R4PAm. A fixed amount of MTTR (0.5 mg/ml) was incubated in the presence of increasing concentrations of TTR inhibitors (0–5-fold excess) and subjected to non-denaturing electrophoresis. The sizes of His-tagged TTR monomer (14 kDa) and tetramer (56 kDa) are shown next to the gel. R4PTm and R4PAm inhibited MTTR oligomer formation in a dose-dependent manner. C, fraction of the inhibitors R4PAm (solid line, squares) and R4PTm (dashed line, circles) bound to MTTR. The soluble fraction of the samples was collected after ultracentrifugation followed by acid precipitation of soluble MTTR and inhibitor-bound complexes. The unbound fraction of inhibitors was analyzed by HPLC-MS. One-site specific binding analysis was performed using GraphPad Prism 6 to calculate binding parameters KD as apparent binding affinity and Bmax as maximum bound fraction. D, analysis of tetrameric and monomeric populations of MLT after incubation with TTR inhibitors. Size exclusion chromatography of MLT reveals that the presence of TTR inhibitors does not drive the protein equilibrium to populate the monomeric species. In contrast, an increase of tetrameric species was found after incubation with inhibitors (Inh). TTR aggregation (TTR agg.) was measured by absorbance at 400 nm with the initial protein amount normalized to 100%. mAU, milli-absorbance units; N.A., not applicable.

We next found that the presence of inhibitors leads to an equilibrium shift toward the tetramer population. We first generated the TTR variant MLT, which exhibits monomers and tetramers in solution, and we analyzed both populations by size exclusion chromatography (Fig. 6D). The results show that the addition of inhibitors promotes an immediate increase in tetramer concentration when compared with the absence of inhibitors. Additionally, the monomeric concentration remains unchanged after incubation for 18 h in the presence of the inhibitors, whereas their absence results in monomer loss and aggregation. Taken together, these results suggest a mechanism of action that does not stabilize the monomer population and differs from the tetramer stabilization.

In summary, our results indicate that strands F and H play a causative role in TTR aggregation. We used this hypothesis to design inhibitors of their self-aggregation to stop amyloid formation of wild-type TTR and familial mutant variants.

### Identifying the Amyloid-driving Segments of TTR

Our first step toward identifying the amyloid spine-forming segments of TTR was to test candidate segments identified by computational predictions for their ability to aggregate. We used the structure-based, computational method ZipperDB to predict 6-amino acid segments of TTR likely to form amyloid fibrils (34, 35). Of the 14 segments that met the threshold of ZipperDB (Fig. 1A, red bars), we experimentally tested 12 for their propensity to form fibrils in isolation. Eight of the 12 segments formed amyloid-like aggregates having fibrillar morphology (as viewed by electron microscopy; Fig. 1B) and cross-β x-ray diffraction characteristic of amyloid (Fig. 1C) (41, 42). The segments that formed fibrils reside in the α-helix and β-strands A, B, F, G, and H of the native structure of TTR. Based on these results, we hypothesized that any of these eight segments could potentially drive TTR aggregation.

Identifying amyloidogenic segments in TTR.
A, propensities of steric zipper formation of each 6-residue segment within the TTR sequence. Computationally predicted segments with steric zipper propensities are represented with red bars. The schematic of the secondary structure of native TTR is shown on top of the sequence. Black arrows mark residues where proline replacement did not hinder TTR aggregation. Blue arrows mark proline replacements that hindered TTR aggregation. The green arrow marks the mutation T119M, which protects TTR against fibril formation (58). The green and blue boxes highlight the segment sequences critical for TTR aggregation that we described in this study. B, TEM micrographs of the fibrils formed after 7 days of incubation (scale bar, 500 nm). C, amyloid x-ray cross-β diffraction pattern of the samples containing fibrils shown in B. The arrowheads point to the meridional reflection at 4.7–4.8-Å spacings (parallel to the fibril axis; black arrowheads) and equatorial reflections at ∼10-Å spacings (white arrowheads). Notice that all eight of the examined segments predicted to form amyloid fibrils do in fact form amyloid fibrils.

To pinpoint which of the eight segments drives aggregation within full-length TTR, we created individual proline substitutions within each segment. We chose the substitution to proline because it is known to inhibit fibril formation (43, 44). None of the proline variants fully halted TTR aggregation (Fig. 2, A and B), but substitutions S85P, E92P, and V94P and the S85P/E92P double substitution significantly decreased protein aggregation and altered aggregate morphology (Fig. 2A). In the isolated segment 91AEVVFT96, proline replacements in positions 92 and 94 completely prevented fibrillization (Fig. 2C). These data suggest that the 91AEVVFT96 segment and the N-terminally adjacent region of full-length TTR are important for fibril formation.

β-Strand F as an aggregation-driving segment in TTR suitable for aggregation inhibitor design. TTR aggregation of the protein variants that showed significant delay (A) and those that did not decrease protein aggregation (B) is shown. Those TTR proline variants that are not shown here were found to be insoluble. Histograms show percentage of TTR aggregation using WT TTR to normalize to 100%. The aggregation was measured by absorbance of the samples at 400 nm after 4 days of incubation at 37 °C and pH 4.3 with no shaking. The bottom panel shows a His probe dot blot of the insoluble fraction corresponding to the sample above after solubilization with guanidinium hydrochloride. On the right are TEM micrographs of protein aggregates (scale bar, 100 nm) after 7 days of incubation. Notice that the proline substitutions within or next to β-strand F hindered TTR aggregation. Several of the other proline substitutions enhanced aggregation; these, like ATTR familial mutations, may alter native structure and/or protein stability (8). Error bars represent S.D., and ** symbolizes a p value ≤0.003 (n = 3). C, TEM micrographs of peptides in isolation after 7 days of incubation in PBS with no shaking (scale bar, 500 nm). D, crystal structure of the segment 91AEVVFT96 from β-strand F forming a Class-7 steric zipper. One sheet is shown as blue; the other is shown as gray. On the left is a lateral view of the fibril with the fibril axis shown by the narrow black arrow. On the right is the view down the fibril axis showing two β-sheets in projection. Water molecules are shown as aquamarine spheres. Spheres represent the van der Waals radii of the side chain atoms of the tightly packed fibril core.

To better understand how 91AEVVFT96 self-associates to form fibrils, we determined the structure of the fibril-like crystals of 91AEVVFT96. We found that it forms a Class-7 steric zipper (Ref. 25 and Fig. 2D) in which the β-strands stack into antiparallel β-sheets with identical side chains on both faces. This structure shows how 91AEVVFT96 may self-associate in driving TTR aggregation.

Next, we considered the possibility that TTR contains more than one aggregation-driving segment. Experiments of others show that the fibril core of TTR contains most of the protein, suggesting that more than one segment participates in the fibril core (19, 45). Because the monomeric state is more amyloidogenic than the tetrameric state (4), we reasoned that fibril-driving segments are buried in the tetramer but exposed in the monomer. In this situation, the segments would be exposed to bind to identical segments from other monomers to form a fibril. We analyzed the solvent-accessible surface area of TTR and found that β-strand F is indeed more exposed in the TTR monomer than in the dimer or tetramer (Fig. 3A). A second region was also more exposed in the monomer: β-strand H. We therefore tested whether β-strand H might also be a driver of TTR aggregation.

β-Strand H as a second aggregation-driving segment suitable as a target for aggregation inhibitor design.
A, residue solvent-accessible surface area (ASA; Å2) of TTR was calculated by Areaimol using the structure of WT TTR in three different conformations: tetramer (blue), dimer (black), and monomer (green). Notice that the strands F and H are indeed more exposed when TTR is in a monomeric form. 5.7% of the surface of the amyloidogenic segment 91AEVVFT96 is solvent-exposed in the tetramer, 5.7% is exposed in the dimer, and 32.1% is exposed in the monomer. Additionally, the amyloidogenic segment 119TAVVTN124 from strand H is 27.4% solvent-exposed in the tetramer, 43.4% solvent-exposed in the dimer, and 55.3% solvent-exposed in the monomer. B, TTR aggregation of variants with substitutions on strand H. The histogram at the top shows the percentage of TTR aggregation after 4 days of incubation measured by absorbance at 400 nm with WT aggregation normalized to 100%. Below, a His probe dot blot shows the insoluble fraction of the samples after 4 days of incubation and solubilization with guanidinium hydrochloride. Bottom panels, TEM images from the samples after 7 days of incubation. These show that the substitutions of residues Thr119 and Val121 did indeed hinder protein aggregation. Error bars represent S.D., and ** symbolizes a p value ≤0.003 (n = 3). C, TEM micrographs of peptides in isolation after 7 days of incubation in PBS with no shaking (scale bar, 500 nm). D, crystal structure of the segment 119TAVVTN124 from β-strand H forming a Class-2 steric zipper. One sheet of the zipper is gray; the other is green. On the left is a lateral view of the fibril with the fibril axis indicated by the narrow black arrow. On the right is the view down the fibril axis showing two β-sheets in projection. Spheres represent the van der Waals radii of the side chain atoms of the tightly packed fibril core.

We found that strand H also contributes to TTR amyloid aggregation but not by applying the method of proline substitution that was effective with strand F. Proline substitutions in strand H did not hinder TTR aggregation (Fig. 2). Therefore, we decided to disrupt the capacity for a steric zipper by strand H by substituting residues Thr119 and Val121 with the bulky residues tyrosine and tryptophan. The size of these side chains is expected to prevent formation of tightly packed steric zippers formed from amino acids with short side chains such as Thr119 and Val121. Aggregation assays of TTR variants showed that the mutants T119W, T119Y, V121W, and V121Y did not produce aggregates after 4 days of incubation (Fig. 3B). For additional evidence, we synthesized and analyzed 6-residue peptides containing the same substitutions, and they did not form aggregates or fibrils in isolation (Fig. 3C). These experiments suggest that strand H plays a role during TTR aggregation by acting as a fibril-forming segment.

Next, we performed structural analysis of the isolated strand H segment to test whether it forms a steric zipper structure. We found that 119TAVVTN124 forms a Class-2 steric zipper (Ref. 25 and Fig. 3D) in which the β-strands stack into parallel β-sheets with one sheet face packing against another sheet back. Taken together, these experiments suggest that the 119TAVVTN124 segment is important for fibril formation.

### Validation of the Role of Strands F and H in Aggregation by the Design of Amyloid Inhibitors

To confirm that strands F and H drive protein aggregation, we tested aggregation inhibition by short peptides that we designed to bind to strands F and H, thereby shielding them from aggregation. We designed amyloid inhibitors against β-strands F and H using prior strategies that have been successful at blocking aggregation of other proteins (46, 47). The design was supported by protein-protein docking (Fig. 4). To create aggregation inhibitors capable of binding the target sequence without self-associating, we modified the sequences 91AEVVFT96 and 119TAVVTN124 by adding non-natural amino acids and a charged tag. We used the docking model to predict the most suitable positions for N-methylated residues. The model predicted that even positions (Glu92, Val94, and Thr96 from 91AEVVFT96 and Ala120, Val122, and Asn124 from 119TAVVTN124) were more favorable than odd positions (Fig. 4C). This prediction was later supported by experiments.

Design of sequence-specific peptide inhibitors of TTR aggregation.
A, computational docking model of the peptides AEVVFT and TAVVTN bound to strands F (blue) and H (green), respectively, of the TTR monomer. Peptides are shown with translucent spheres representing van der Waals radii; the TTR monomer is shown as ribbons with side chains of strands F and H as sticks. B, lateral, close-up view of the F and H strands in the docking model showing the designed pattern of hydrogen bonding (dashed lines) between the peptides (sticks) and the TTR β-strands (ribbon with sticks). C, docking models of N-methylated peptides predict which will bind TTR (same view as B). N-Methylations were added to increase peptide solubility, reduce aggregation, and reduce sensitivity. Colored spheres represent the van der Waals radii of the N-methyl groups (Nme) in favorable positions; yellow stars highlight clashes of the N-methyl modifications with the TTR monomer.

Fifteen different peptides were tested as potential inhibitors of aggregation (Fig. 5). The peptide sequences are listed in Fig. 5A. We began by testing the 6-residue F- and H-strand peptides, AEVVFT and TAVVTN, with sequential modifications of N-methylated amino acids (Fig. 5B). The experimental observation that the aggregation inhibitors with N-methyl residues at even positions were more effective than those with N-methylated odd positions agrees with our docking model (Figs. 4C and 5B). We next added an N-terminal tag of four arginine residues to further improve the inhibitors by increasing their solubility and preventing self-aggregation. The addition of the N-terminal residues significantly increased the inhibition (Fig. 5B). The position of the tag was also tested at both termini, and we found that the N-terminally tagged peptides were better inhibitors (Fig. 5C).

Designed peptides are sequence-specific inhibitors of TTR aggregation.
A, list of inhibiting peptides tested as TTR aggregation inhibitors. B–E, evaluation of the peptide inhibitors. The graphs show the percentage of TTR aggregation after 4 days of incubation in the presence or absence of peptide inhibitor measured by absorbance at 400 nm; WT inhibition-free aggregation was normalized to 100%. The initial concentration of soluble TTR was 1 (B–D) or 0.2 mg/ml, which corresponds to the concentration of TTR in plasma (E). The molar excess of peptide inhibitor over target (TTR monomer) is 3-fold unless labeled otherwise. In C and D, a His probe dot blot shows the insoluble fraction of the samples after 4 days of incubation and solubilization with guanidinium hydrochloride. B, initial screening of inhibitors showing that three modifications significantly improved the effectiveness: (i) increasing the length of the matched sequence of the peptide and the target, (ii) addition of a charged tag, and (iii) addition of an N-methyl group in positions predicted to be favorable by the docking model (Fig. 4C). The two best inhibitors of TTR aggregation were R4PAm and R4PTm. C, the peptide inhibitors are more effective with the charged tag at the N terminus than at the C terminus. D, dose-dependent effectiveness of the peptides R4PAm and R4PTm in combination. Maximal inhibition of TTR aggregation is reached when the target to peptide molar ratio is 1:3. E, aggregation assay of the familial mutants V30M and L55P in the presence of R4PAm and R4PTm in combination using a physiological concentration of protein (3.6 μm). Error bars represent S.D. (n = 3), and ** symbolizes a p value ≤0.003.

We assessed the specificity of the inhibitors in two ways. First, we tested a scrambled control peptide with the polyarginine tag intact (Fig. 5B). Charged tags might form salt bridges with several of the loops in TTR that are rich in acidic residues (48). The control peptide did not disrupt protein aggregation, confirming that the aggregation inhibition is sequence-specific and not reliant on the arginine tag (Fig. 5B). Then we tested the combination of F and H inhibitors. The peptides R4PAm and R4PTm showed a synergistic effect, suggesting two different binding sites (Fig. 5B), in a dose-dependent manner (Fig. 5D). The combination was also effective at blocking aggregation of the familial mutants V30M and L55P at physiological TTR concentration (Fig. 5E). Based on this initial screening, the two peptides R4PAm and R4PTm in combination were selected for further study.

To further characterize the mechanism of action of the inhibitor pair, we examined their effect on different assembly states of TTR. First, we examined tetramer stability by differential scanning calorimetry (Fig. 6A). We found that the inhibitors did not affect WT TTR stability or compete with T4 for its binding site (Fig. 6A). These results confirm that the mechanism for inhibiting aggregation is different from tetramer stabilization (9, 10, 49). Additionally, we examined the effect on oligomer formation of an engineered monomeric variant of TTR, MTTR (40). MTTR has two methionine substitutions at residues Phe87 and Leu110, which prevent dimer and tetramer formation, leading to rapid aggregation at acidic pH. Incubation of MTTR with the inhibitor pair reduced oligomer formation as monitored by non-denaturing gel electrophoresis (Fig. 6B). The analysis of the binding of the TTR inhibitors to MTTR species showed an apparent binding constant of 17.9 ± 6.1 and 19.3 ± 6.7 μm for R4PAm and R4PTm, respectively (Fig. 6C). This experiment did not exclude the binding of the inhibitors to aggregated species. The measured affinity is therefore a combined affinity for all species in solution, and the monomer-specific affinity might be weaker than calculated.

Analysis of the mechanism of action of the TTR inhibitors.
A, midpoint temperatures of the thermal unfolding transition (Tm) of wild-type TTR at different conditions were determined by differential scanning calorimetry. Relative stability is compared in the presence and absence of a 5:1 molar ratio of T4 and/or the combination of R4PAm and R4PTm to TTR monomer. The TTR concentration was 1 mg/ml. Notice that the addition of the natural ligand T4 increased the protein thermostability by 10.8 °C. The addition of the peptide inhibitors increased it only by 1.3 °C in the absence of T4 and by 0.9 °C when the ligand was present. B, inhibition of oligomer formation of MTTR after incubation with R4PTm and R4PAm. A fixed amount of MTTR (0.5 mg/ml) was incubated in the presence of increasing concentrations of TTR inhibitors (0–5-fold excess) and subjected to non-denaturing electrophoresis. The sizes of His-tagged TTR monomer (14 kDa) and tetramer (56 kDa) are shown next to the gel. R4PTm and R4PAm inhibited MTTR oligomer formation in a dose-dependent manner. C, fraction of the inhibitors R4PAm (solid line, squares) and R4PTm (dashed line, circles) bound to MTTR. The soluble fraction of the samples was collected after ultracentrifugation followed by acid precipitation of soluble MTTR and inhibitor-bound complexes. The unbound fraction of inhibitors was analyzed by HPLC-MS. One-site specific binding analysis was performed using GraphPad Prism 6 to calculate binding parameters KD as apparent binding affinity and Bmax as maximum bound fraction. D, analysis of tetrameric and monomeric populations of MLT after incubation with TTR inhibitors. Size exclusion chromatography of MLT reveals that the presence of TTR inhibitors does not drive the protein equilibrium to populate the monomeric species. In contrast, an increase of tetrameric species was found after incubation with inhibitors (Inh). TTR aggregation (TTR agg.) was measured by absorbance at 400 nm with the initial protein amount normalized to 100%. mAU, milli-absorbance units; N.A., not applicable.

We next found that the presence of inhibitors leads to an equilibrium shift toward the tetramer population. We first generated the TTR variant MLT, which exhibits monomers and tetramers in solution, and we analyzed both populations by size exclusion chromatography (Fig. 6D). The results show that the addition of inhibitors promotes an immediate increase in tetramer concentration when compared with the absence of inhibitors. Additionally, the monomeric concentration remains unchanged after incubation for 18 h in the presence of the inhibitors, whereas their absence results in monomer loss and aggregation. Taken together, these results suggest a mechanism of action that does not stabilize the monomer population and differs from the tetramer stabilization.

In summary, our results indicate that strands F and H play a causative role in TTR aggregation. We used this hypothesis to design inhibitors of their self-aggregation to stop amyloid formation of wild-type TTR and familial mutant variants.

### Discussion

TTR aggregation is linked to ATTR, a condition that is not yet curable (50). Although new strategies are being evaluated (16–18), the development of combinatory systems that address various molecular processes driving TTR aggregation is desirable to maximize treatment options. Here we propose a novel mechanism of aggregation inhibition that consists of targeting segments of TTR that are exposed only when the TTR tetramer dissociates and drive protein aggregation.

Others have shown that a necessary step in the conversion of TTR to amyloid fibrils is its dissociation to a monomer (2–7), but the specific segments that cause the aggregation of monomers had not previously been identified. We performed experiments to identify adhesive segments in TTR that participate in fibril formation. Our procedure was first to identify possible amyloid-forming segments by ZipperDB three-dimensional profiling. In this procedure, the TTR sequence is computationally threaded through a “profile” derived from the structure of a known amyloid fibril (34, 35). This procedure identified 14 steric zipper-forming 6-residue-long segments, eight of which indeed formed fibrils in solution (Fig. 1). Also our computations with ZipperDB show that TTR familial mutations do not increase the propensity of fibril formation (data not shown). These results support the hypothesis that familial point mutations destabilize the quaternary structure rather than creating new amyloidogenic segments (8). With systematic mutagenesis of the eight fibril-forming segments and computational prediction of exposed strands in the monomer, we were able to trace the cause of TTR fibril formation to the sequence segments that form the F and H β-strands of the native structure.

Mutational analysis of amyloid proteins for identification of aggregation-driving segments has been reported by our group and others to be effective (43, 51, 52). These data led us to strands F and H in the case of TTR (Figs. 2 and 3). However, mutation to proline, tryptophan, or tyrosine does not necessarily imply prevention from aggregation. As additional evidence, we tested fibrillization of the segments 91AEVVFT96 and 119TAVVTN124 in isolation as well as their mutated versions to find that the substitution of residue 85, 92, 94, 119, or 121 did hinder self-association and fibril formation (Figs. 2 and 3). In addition, to further prove that β-strands F and H can participate in amyloid core formation of TTR fibrils, we determined the crystal structure of the steric zippers formed by the segments in isolation by x-ray crystallography (23–25, 53). The crystal structure of 91AEVVFT96 revealed a Class-7 steric zipper with antiparallel β-strands and antiparallel sheets (Ref. 25 and Fig. 2D). The side chain organization explains why the residue replacements E92P and V94P hinder TTR aggregation because the proline substitution would decrease the shape complementarity between sheets in the steric zipper structure. Moreover, the crystal structure of 119TAVVTN124 is a Class-2 steric zipper, face-to-back arrangement with parallel β-strands and parallel sheets (Ref. 25 and Fig. 3D). The side chain organization in this peptide also explains why the residue replacements T119W, T119Y, V121W, and V121Y impede TTR aggregation. Because residues Thr119 and Val121 are pointing to the hydrophobic T4-binding site, we cannot dismiss the possibility that the tryptophan or tyrosine could strengthen the hydrophobic contacts to increase the stability of the protein and delay aggregation. However, the involvement of these segments in the formation of TTR fibrils was supported by two pieces of evidence. First, peptides with Tyr/Trp substitutions did not form fibrils in isolation (Figs. 2C and 3C). Second, the aggregation inhibitors targeting these segments hindered TTR aggregation (Figs. 5 and 6). We do not dismiss the contribution of other aggregation-prone segments to the amyloid core. Strands F and H that we propose here might be segments most susceptible to structural changes in promoting aggregation.

Combining mutational and structural studies of TTR variants and isolated peptides, we uncovered the involvement of the strands F and H in TTR aggregation (Figs. 1–3). This finding is consistent with NMR relaxation dispersion studies that showed that, at physiological pH, monomeric TTR undergoes conformational fluctuations of the H and F strands that are propagated along the entire sheet (54). Kelly and co-workers (48, 54) have also analyzed TTR by NMR studies and x-ray crystallography in neutral and acidic environments to assess structural changes. These studies have confirmed that the EF-loop undergoes large conformational changes when the pH is changed from neutral to an acidic environment. We speculate that the S85P substitution, near strand F, constrains this flexibility to prevent aggregation. Substitutions in strand F, E92P and V94P, most likely hinder self-recognition and further aggregation. Kelly and co-workers (40) have engineered a monomeric variant of TTR that is non-amyloidogenic unless partially denatured. The monomeric variant aggregates much faster than the wild-type tetramer at pH 4.4 (minutes versus hours, respectively). They also determined the structure of this variant and showed that most of the deviations from the structure of WT TTR are observed on the interfaces between the subunits, involving strands F and H, which supports our findings. We reasoned that if strands F and H are important for the self-association of TTR then one could stop protein aggregation by blocking the two segments.

To validate our identification of the fibril-forming segments, we designed analog peptides that interact with the F and H strands. These inhibitors slowed fibril formation, supporting our identification of the fibril-forming segments (Figs. 5 and 6). Eisenberg and co-workers (46, 47) have previously used structure-based design of peptides or compounds to disrupt fibril development and/or growth. For the design of TTR aggregation inhibitors, N-methyl residues were incorporated into the peptide sequence to increase effectiveness (Figs. 4 and 5B). This is advantageous because non-natural amino acids can increase peptide stability by reducing proteolytic degradation (37, 55). This initial screening resulted in two optimized TTR peptide inhibitors, both 16 residues long, with separate binding sites that differ from the hydrophobic pocket (Figs. 5B and 6A).

The inhibition of TTR aggregation by peptide inhibitors as a prospective therapeutic strategy needs further optimization to improve affinity. Although similar to some amyloid inhibitors (56, 57), the apparent affinity of the monomeric variant MTTR (40) to the inhibitors is low compared with those found for tetramer stabilizers (for instance, see Refs. 9 and 10). However, the binding of the inhibitors to TTR after tetramer dissociation might differ from their binding to MTTR.

The inhibition of TTR aggregation by blocking strands F and H indicates their importance for protein aggregation. Our data suggest that the aggregation of TTR requires tetramer dissociation into monomeric species as suggested elsewhere (2–6). Several other studies propose that TTR aggregation occurs by dimer arrangement (19–21). However, strands F and H are buried in the dimer interface (Ref. 4 and Fig. 3A), impeding self-association. Therefore, the inhibition of TTR aggregation by interaction of peptide inhibitors with the F and H strands requires dimer dissociation and segment exposure.

Based on our data, we propose a pathway for peptide-based inhibition of TTR aggregation (Fig. 7). Upon tetramer dissociation, the binding of the peptide inhibitors to their identical segments hinders self-recognition and aggregation. The inhibitor-bound TTR complex is more stable than the monomer but less stable than the tetramer (Fig. 6D), thereby favoring tetramer reassembly. Additionally, although the inhibitors were designed to interact with the F and H strands after dissociation of the tetramer, they may act at the amyloid level by capping the emerging fibrils to hinder further growth (46). In either case, the steric zipper-like interaction of the peptide inhibitors with the F and H strands would block self-association and protein aggregation.

Model of peptide-based inhibition of TTR aggregation. The low free energy of TTR amyloid fibrils drives aggregation with TTR dissociation providing a kinetic barrier (4). The inhibitors do not affect stability of the TTR tetramer but bind to intermediate species, hindering unfolding and aggregation. Although the inhibitor-bound complex is more stable than the free monomer, it is less stable than the tetramer, thereby favoring tetramer reassembly. Note that in this scheme the hydrophobic pocket of the tetramer remains accessible for complementary treatment with a stabilizer compound such as tafamidis or diflunisal (10, 49).

In summary, we have shown that strands F and H are actively involved in TTR aggregation and uncovered a new strategy for the inhibition of TTR aggregation by small non-natural peptides. Their mechanism of action allows combinatory treatment with tetramer-stabilizing compounds, which might be an effective therapeutic approach. The two inhibiting peptides may serve as leads for the development of drugs against TTR systemic diseases.

### Author Contributions

L. S. and D. S. E. designed the project. L. S., L. M. J., and W. Y. L. performed the research. L. S., L. M. J., and D. S. E. analyzed the data. P. R., J. W., L. J., and R. R. contributed with analytic tools and discussion. D. C. and M. R. S. helped with crystallography and structural analysis. L. S. wrote the paper. Critical revision was done by L. M. J. and D. S. E. All authors reviewed the results and approved the final version of the manuscript.



# SUPPLEMENTAL FILE 1: zbc28932.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)