# MAIN TEXT

## C1QA and C1QC modify age‐at‐onset in familial amyloid polyneuropathy patients

### Abstract

AbstractObjectivesTransthyretin (TTR) familial amyloid polyneuropathy (FAP) (OMIM 176300) shows a variable age‐at‐onset (AO), including within families. We hypothesized that variants in C1QA and C1QC genes, might also act as genetic modifiers of AO in TTR‐FAP Val30Met Portuguese patients.MethodsWe analyzed DNA samples of 267 patients (117 families). To search for variants, all exons and flanking regions were genotyped by automated sequencing. We used generalized estimating equations (GEEs) to take into account the non‐independency of AO among relatives. Intensive in silico analyses were performed, using various software to assess miRNAs target sites, splicing sites, transcription factor binding sites alterations, and gene–gene interactions.ResultsTwo variants for C1QA gene, GA genotype of rs201693493 (P < 0.001) and CT genotype of rs149050968 (P < 0.001), were significantly associated with later AO. In silico analysis demonstrated, that rs201693493 may alter splicing activity. Regarding C1QC, we found three statistically significant results: GA genotype of rs2935537 (P = 0.003), GA genotype of rs201241346 (P < 0.001) and GA genotype of rs200952686 (P < 0.001). The first two were associated with earlier AO, whereas the third was associated with later‐onset.Interpretation
C1QA was associated with later onset, whereas C1QC may have a double role: variants may confer earlier or later AO. As found in a study in Cyprus, we confirmed the role of complement C1Q genes (and thus of inflammation) as modulator of AO in Portuguese patients with TTR‐FAP Val30Met.

### Introduction

Transthyretin (TTR) related familial amyloid polyneuropathy (FAP) is an autosomal dominant disease, characterized by the systemic extracellular deposition of TTR amyloid fibrils, in peripheral nerves and autonomic nervous system. It is due to a point variant that affects function in the TTR gene (chr18q12.1) (OMIM 176300). Val30Met, NM_000371.3: c.148G>A (p.Val50Met), is the major disease‐causing variant found in Portugal,1 but it was also described in other clusters such as Sweden, Japan, Brazil, and Balearic Islands, among others.2, 3, 4 Over time, it became apparent that the Val30Met substitution has shown considerable variation in penetrance and age‐at‐onset (AO) in different clusters. In the Portuguese population, AO ranged 19–82 years.5, 6 While in some families the disease is known to run for several generations, in others the proband had no affected parent at time of diagnosis.7

In some cases, the pathogenic variant is transmitted from generation to generation by aged asymptomatic carriers or by late‐onset parents, but this protective factor may at times disappear, leading to genetic anticipation.6 Our group confirmed the existence of true anticipation in TTR‐FAP families, all sharing the Val30Met variant, which led us to hypothesize that other genetic modifiers may explain AO variability.6

To clarify the role of those possible modifier genes, several studies were performed in different populations, such as Portuguese,8, 9, 10 French and Swedish.11, 12 Our group found some variants in candidate genes associated with AO variation, which were related to altered disease TTR‐regulatory and signaling pathways.9, 10

It has been shown that C1q and other complement activation products colocalize around amyloid deposits, within peripheral nerves, in TTR hereditary amyloid neuropathy.13 Recently, a double transgenic mouse model of Val30Met amyloid neuropathy in which C1q is ablated showed an increase in amyloid deposits by over 60%, compared to control mice.14

The role of inflammation in the pathogenesis of the disease, both in the process of TTR misfolding and amyloid deposition and as a contribution to the process of nerve degeneration has been intensively studied in the last decade. More recently, the positive results of the double blind, randomized clinical trial with Diflunisal,15 a nonsteroidal anti‐inflammatory drug, raised the hypothesis that the anti‐inflammatory effect could be superimposed on the TTR stabilizer effect, increasing the therapeutic potential of the drug.

Fonseca et al., in 200416 also has shown in transgenic mice models of Alzheimer's disease that C1q plays an important role in neurodegeneration through the activation of classical complement cascade, leading to inflammation, hypothesizing that future therapies could block amyloid interaction with C1q.

Furthermore, in a study with Cypriot TTR‐FAP carriers, Dardiotis et al. demonstrated that common variants in C1QA and C1QC genes were associated with an earlier and a later onset, respectively.17 In the light of this evidence, we hypothesized that C1QA and C1QC may act as genetic modifiers of AO in TTR‐FAP Val30Met Portuguese patients.

Since early and late‐onset may be found in the same family, we used a family centered approach for this study. Furthermore, we also aimed at performing in silico analyses of the variants found in the genes under study and its implication in gene expression and variability of AO in the TTR‐FAP Val30Met.

### Patients and Methods

Unidade Corino de Andrade do Centro Hospitalar Universitário do Porto (UCA/CHUP, Porto) has the largest TTR‐FAP Val30Met registry worldwide, collected over 75 years and clinically well characterized and coming from all geographical areas of Portugal. This study includes 267 patients (117 families with at least two generations of affecteds) under follow‐up at UCA‐CHUP. These families came not only from the high‐penetrance (and early‐onset) regions, but also from the low‐penetrance (and late‐onset) areas. The same group of neurologists established AO of each patient, considering the presence of a whole set of symptoms characteristic of small fibers'neuropathy and not isolated and unspecific symptoms. Presence of amyloid in tissues such as nerve, skin, or salivary glands is a requested criteria for diagnosis. In the rare cases of recurrent biopsies without evidence of amyloid deposition the diagnosis was accepted only if objective and unequivocal signs of neuropathy were present and other potential causes had been excluded.

DNA samples were collected at UCA‐CHUP and stored at Centro de Genética Preditiva e Preventiva (CGPP‐IBMC, Porto) biobank, authorized by CNPD (National Commission for Data Protection). The study followed the relevant legal and ethical procedures and was approved by the CHUP ethical committee. Written informed consent was obtained for all participants in the study.

Genomic DNA was extracted from peripheral blood leukocytes, using standard salting‐out method,18 or from saliva, using ORAGENE DNA Self Collection Kits (DNA Genotek, Ottawa, Canada), according to manufacturer's instructions.

Primers were designed using Primer3Plus software (http://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi). The existence of dimer and hairpin formation was excluded using the AutoDimer (www.cstl.nist.gov/strbase/AutoDimerHomepage/AutoDimerProgramHomepage.htm) and single nucleotide polymorphism (SNP) variants were checked with the SNPcheck software (secure.ngrl.org.uk/SNPCheck/snpcheck.htm;jsessionid=F6B95662DD8 4AEB46C6A27152704E75B). Polymerase chain reaction (PCR) was performed to amplify genomic DNA for the genes under study, in a final volume of 10 μL, using the PCR Master Mix (Qiagen, Hilden, Germany), according to manufacturer's instructions. QIAxcel® BioCalculator equipment was used to verify amplification of PCR products and to exclude possible contaminations. Amplified products were purified with ExoProStar (Amersham Biosciences). Genotyping was performed by automatic sequencing using Big Dye Terminator Cycle Sequencing v1.1, Ready Reaction (Applied Biosystems). Further analyses of sequences were carried out using Seqscape v2.6 software (Applied Biosystems) and Alamut Mutation Interpretation Software (Interactive Biosoftware, Rouen France).

Age‐at‐onset was used as a continuous variable. Since we included several members of the same family in the analysis, each patient was “nested” in his/her family. For this, we took into account non‐independency of AO between members of the same family, adjusted for gender, using generalized estimating equations (GEEs).19 As in previous papers from our group,9, 10 where we used the same approach, we estimated the effect on AO of each genotype in comparison to the most common genotype. We applied a Bonferroni correction to correct for multiple testing. All statistical analyses were performed using IBM SPSS Statistics software (v.23). To search for possible gene–gene interactions, we used Multifactor Dimensionality Reduction (MDR) software (v2.0).20, 21

In silico analyses were performed using diverse bioinformatics tools such as Polyphen‐2 and SNP Function Prediction (FuncPred) to predict putative SNP functional effects.22 To predict splicing changes, we used ESE (Exonic Splicing Enhancer) Finder v3.0 and Human Splicing Finder (HSF) v3.0. To predict if a 3′UTR SNPs would lead to a disruption or creation of novel miRNA binding sites, we used mirDIP23 and miRWalk.24 To further explore alterations in transcription factor binding sites (TFBS), we used the SNP2TFBS tool, which predicts a change in TF binding based on position weight matrices (PWM)25

### Subjects

Unidade Corino de Andrade do Centro Hospitalar Universitário do Porto (UCA/CHUP, Porto) has the largest TTR‐FAP Val30Met registry worldwide, collected over 75 years and clinically well characterized and coming from all geographical areas of Portugal. This study includes 267 patients (117 families with at least two generations of affecteds) under follow‐up at UCA‐CHUP. These families came not only from the high‐penetrance (and early‐onset) regions, but also from the low‐penetrance (and late‐onset) areas. The same group of neurologists established AO of each patient, considering the presence of a whole set of symptoms characteristic of small fibers'neuropathy and not isolated and unspecific symptoms. Presence of amyloid in tissues such as nerve, skin, or salivary glands is a requested criteria for diagnosis. In the rare cases of recurrent biopsies without evidence of amyloid deposition the diagnosis was accepted only if objective and unequivocal signs of neuropathy were present and other potential causes had been excluded.

DNA samples were collected at UCA‐CHUP and stored at Centro de Genética Preditiva e Preventiva (CGPP‐IBMC, Porto) biobank, authorized by CNPD (National Commission for Data Protection). The study followed the relevant legal and ethical procedures and was approved by the CHUP ethical committee. Written informed consent was obtained for all participants in the study.

### DNA extraction

Genomic DNA was extracted from peripheral blood leukocytes, using standard salting‐out method,18 or from saliva, using ORAGENE DNA Self Collection Kits (DNA Genotek, Ottawa, Canada), according to manufacturer's instructions.

### Genotyping

Primers were designed using Primer3Plus software (http://www.bioinformatics.nl/cgi-bin/primer3plus/primer3plus.cgi). The existence of dimer and hairpin formation was excluded using the AutoDimer (www.cstl.nist.gov/strbase/AutoDimerHomepage/AutoDimerProgramHomepage.htm) and single nucleotide polymorphism (SNP) variants were checked with the SNPcheck software (secure.ngrl.org.uk/SNPCheck/snpcheck.htm;jsessionid=F6B95662DD8 4AEB46C6A27152704E75B). Polymerase chain reaction (PCR) was performed to amplify genomic DNA for the genes under study, in a final volume of 10 μL, using the PCR Master Mix (Qiagen, Hilden, Germany), according to manufacturer's instructions. QIAxcel® BioCalculator equipment was used to verify amplification of PCR products and to exclude possible contaminations. Amplified products were purified with ExoProStar (Amersham Biosciences). Genotyping was performed by automatic sequencing using Big Dye Terminator Cycle Sequencing v1.1, Ready Reaction (Applied Biosystems). Further analyses of sequences were carried out using Seqscape v2.6 software (Applied Biosystems) and Alamut Mutation Interpretation Software (Interactive Biosoftware, Rouen France).

### Design and statistical analysis

Age‐at‐onset was used as a continuous variable. Since we included several members of the same family in the analysis, each patient was “nested” in his/her family. For this, we took into account non‐independency of AO between members of the same family, adjusted for gender, using generalized estimating equations (GEEs).19 As in previous papers from our group,9, 10 where we used the same approach, we estimated the effect on AO of each genotype in comparison to the most common genotype. We applied a Bonferroni correction to correct for multiple testing. All statistical analyses were performed using IBM SPSS Statistics software (v.23). To search for possible gene–gene interactions, we used Multifactor Dimensionality Reduction (MDR) software (v2.0).20, 21

### In silico analysis

In silico analyses were performed using diverse bioinformatics tools such as Polyphen‐2 and SNP Function Prediction (FuncPred) to predict putative SNP functional effects.22 To predict splicing changes, we used ESE (Exonic Splicing Enhancer) Finder v3.0 and Human Splicing Finder (HSF) v3.0. To predict if a 3′UTR SNPs would lead to a disruption or creation of novel miRNA binding sites, we used mirDIP23 and miRWalk.24 To further explore alterations in transcription factor binding sites (TFBS), we used the SNP2TFBS tool, which predicts a change in TF binding based on position weight matrices (PWM)25

### Results

Our sample included 267 patients with TTR‐FAP, molecularly confirmed as carriers of the Val30Met variant, with a mean AO of 39.7 years (±13.1), though mean AO in men (38.1 ± 13.8) was lower than in women (41.2 ± 12.3 years), similar to that described previously.5, 6 In this study, five variants (two in C1QA and three in C1QC gene) were found, for the first time, to be associated with AO variation in TTR‐FAP Val30Met (Table 2).

After genotyping of coding and flanking regions of the C1QA gene, we found six variants: one intronic and five in exons, all previously described (Table 1). Genotype GA (P < 0.001) for the rs201693493 and the genotype CT (P < 0.001) of rs149050968 were significantly associated with later‐onset, corresponding to an average increase in 16 and 10 years, respectively (Table 2). Polyphen‐2 software showed that all exonic variants in C1QA were benign (two were missense and the other three are synonymous substitutions). Also, we identified seven variants in C1QC: six in non‐coding regions and one exonic, all already described (Table 1). The exonic variant is a synonymous substitution, so it has a benign effect on protein. Three variants were found to be significantly associated with AO (Table 2): genotype GA of rs2935537 (P = 0.003) and genotype GA of rs201241346 (P < 0.001) associated with earlier onset (decreasing mean AO by 7 and 11 years, respectively). The GA genotype of rs200952686 (P < 0.001) associated to later onset, increasing mean AO by approximately 32 years.

Variants found in the C1QA and C1QC genes

NA, non applicable.

*Frequency in HapMap Northern Europeans from Utah.

**Variants studied by Dardiotis et al. (2009)17.

Statistically significant results for C1QA and C1QC variants and AO variation using GEE analysis, after Bonferroni corrections

Reference: Mean AO; B, unstandardized coefficient (estimated quantitative effect of each genotype on mean AO variation, according to the intercept, compared with the reference genotype); CI, confidence interval; significance level was set to 0.05.

To unravel possible regulatory effects of the variants found, we used diverse bioinformatics tools to explore possible changes at the level of TFBSs, miRNAs, and splicing sites. We found that rs201693493 of C1QA gene may modify the recognition site of splicing regulatory factors, since it leads to a mild reduction in a potential acceptor site; but, more interestingly, this SNP creates a significant increase in the occurrence of another acceptor site (consensus value, CV >80) compared with the common genotype. Since rs2935537 is located in the C1QC promoter, we evaluate the effect on TFBS; however, this variant does not appear to alter significantly the binding site of any transcription factor, with a stringent criterion fulfilling both the core match score of 1 and a matrix match score >0.9, to minimize false positives and negatives.

Furthermore, we assessed a putative C1QA‐C1QC gene interaction using MDR approach; however, we did not find any significant result (data not shown).

### The role of C1QA and C1QC genes

After genotyping of coding and flanking regions of the C1QA gene, we found six variants: one intronic and five in exons, all previously described (Table 1). Genotype GA (P < 0.001) for the rs201693493 and the genotype CT (P < 0.001) of rs149050968 were significantly associated with later‐onset, corresponding to an average increase in 16 and 10 years, respectively (Table 2). Polyphen‐2 software showed that all exonic variants in C1QA were benign (two were missense and the other three are synonymous substitutions). Also, we identified seven variants in C1QC: six in non‐coding regions and one exonic, all already described (Table 1). The exonic variant is a synonymous substitution, so it has a benign effect on protein. Three variants were found to be significantly associated with AO (Table 2): genotype GA of rs2935537 (P = 0.003) and genotype GA of rs201241346 (P < 0.001) associated with earlier onset (decreasing mean AO by 7 and 11 years, respectively). The GA genotype of rs200952686 (P < 0.001) associated to later onset, increasing mean AO by approximately 32 years.

Variants found in the C1QA and C1QC genes

NA, non applicable.

*Frequency in HapMap Northern Europeans from Utah.

**Variants studied by Dardiotis et al. (2009)17.

Statistically significant results for C1QA and C1QC variants and AO variation using GEE analysis, after Bonferroni corrections

Reference: Mean AO; B, unstandardized coefficient (estimated quantitative effect of each genotype on mean AO variation, according to the intercept, compared with the reference genotype); CI, confidence interval; significance level was set to 0.05.

### In silico analyses

To unravel possible regulatory effects of the variants found, we used diverse bioinformatics tools to explore possible changes at the level of TFBSs, miRNAs, and splicing sites. We found that rs201693493 of C1QA gene may modify the recognition site of splicing regulatory factors, since it leads to a mild reduction in a potential acceptor site; but, more interestingly, this SNP creates a significant increase in the occurrence of another acceptor site (consensus value, CV >80) compared with the common genotype. Since rs2935537 is located in the C1QC promoter, we evaluate the effect on TFBS; however, this variant does not appear to alter significantly the binding site of any transcription factor, with a stringent criterion fulfilling both the core match score of 1 and a matrix match score >0.9, to minimize false positives and negatives.

### Gene–gene interactions

Furthermore, we assessed a putative C1QA‐C1QC gene interaction using MDR approach; however, we did not find any significant result (data not shown).

### Discussion

Variation in AO is a hallmark of TTR‐FAP Val30Met, showing that this variant alone cannot explain it. To explore other possible genetic factors that may modulate AO, we used a family centered approach and focused on candidate genes, C1QA and C1QC, which can affect functional pathways in the course of the disease.

It is well recognized that inflammatory infiltrates are rarely seen in nerve biopsies,26 but several authors found evidence of activation of inflammatory markers in the tissues of patients27 and animal models28, 29 and even in the peripheral blood.30 However, the primary or secondary role of these markers remains a matter of controversy. Kurian et al., (2016), showed interesting results regarding male‐ and female‐specific inflammatory signatures TTR‐ FAP patients.30 Therefore, inflammation could also play a role in the phenotypic variability observed in our TTR‐FAP patients as we found for C1QA and C1QC in our study.

We found two statistically significant variants in C1QA associated to a later AO (rs201693493 and rs149050968), leading us to hypothesize that these variants could act as a protective factor for TTR‐FAP Val30Met patients. Two variants in C1QC were associated with earlier AO (rs2935537 and rs201241346) and one was associated to later AO (rs200952686). For some variants, the heterozygous genotype appeared only once in our sample, in accordance with their frequency described in European populations in HapMap and The 1000 Genomes databases (Table 1).

We found that rs20163493 in C1QA may result in significant splicing alterations, a strong active splice site leading to the creation of a new splice site. Most variants that imply new splice sites or intron retention lead to the premature termination codons into mRNA, resulting in degradation by nonsense‐mediated decay and loss of function of the minor allele.31 This may explain its association with a late‐onset of the disease. Although rs2935537 of C1QC apparently does not change TFBS, we cannot dismiss the hypothesis it causes any effect on the stability of mature mRNA and leads to decreased translation efficiency and, somehow, change gene expression, resulting in earlier onset. Furthermore, the C1QC variants associated with early‐onset may result in a mechanism similar to the inactivation of C1Q, leading to an increase in amyloid deposition; there are previous indications that amyloid deposits amounts are greater in early‐onset patients.32 Also, this observation is in line with the findings of Panayiotou et al., who recently demonstrated that Val30Met C1q knockout mice had more amyloid deposits (over 60%) than endogenous C1q control. Those differences increased with age.

C1Q complement genes demonstrated to play an important role in other neurodegenerative diseases, as Alzheimer's (AD)33 and Parkinson's.34 The complement system can have a dual role, either beneficial or damaging, depending on the associated pathophysiological mechanisms, as in some cases it can lead to tissue damage.35 In AD, it was demonstrated that under some circumstances C1Q plays a protective role and downregulates the inflammation process in early stages of neuronal injury, as well as amyloid‐induced neurotoxicity.36 In our TTR‐FAP sample, a similar mechanism might occur, as variants in C1QA seem to increase mean AO, as well as the rs200952686 in the C1QC. In contrast, other variants in C1QC may be a risk factor for earlier AO, possibly by increasing tissue damage.

In this study, we also included common variants found to be associated with AO in the Cypriot population (rs172378 in C1QA and, rs15940 and rs9434 in the C1QC gene), but their effect failed to replicate in our sample. This suggests that different approaches or diverse population samples may lead to different results. This has been discussed previously around the role of APCS in AO variation.9 Regarding rare variants, we found only one in common with their study. We can hypothesize that rare variants are population specific and may play a role as genetic modifiers. Previous studies indicate that highly penetrant rare variants are genetic modifiers for various Mendelian disorders.37

Our study is based on several strengths: (1) a large sample size 267 patients (117 families); (2) the clinical observation of all patients by the same team, minimizing differences in established AO; (3) a family centered approach (circumventing population stratification, as opposed to case‐control studies)38; (4) the use of GEE analysis, since it corrects for familial correlations of AO and provides a greater power to detect statistically significant effects39; and (5) the inclusion of multiple testing corrections to avoid type I‐errors.

We cannot exclude the possibility of other genes and other SNPs within these genes being associated with AO variability in TTR‐FAP Val30Met. Importantly, we conclude that C1QA was associated with later AO and C1QC may have a double role: some variants may confer earlier and other later AO, depending on the associated pathophysiological mechanisms. We confirmed the role of complement C1Q genes in AO variability in Portuguese patients with TTR‐FAP Val30Met. Furthermore, we reinforced the involvement of C1Q and the complement cascade as modifiers in TTR‐FAP Val30Met. However, this is a preliminary study on inflammation; therefore it is too premature to take conclusions for treatments or screening.

### C1QA and C1QC variants associated with AO variation

We found two statistically significant variants in C1QA associated to a later AO (rs201693493 and rs149050968), leading us to hypothesize that these variants could act as a protective factor for TTR‐FAP Val30Met patients. Two variants in C1QC were associated with earlier AO (rs2935537 and rs201241346) and one was associated to later AO (rs200952686). For some variants, the heterozygous genotype appeared only once in our sample, in accordance with their frequency described in European populations in HapMap and The 1000 Genomes databases (Table 1).

### In silico analysis

We found that rs20163493 in C1QA may result in significant splicing alterations, a strong active splice site leading to the creation of a new splice site. Most variants that imply new splice sites or intron retention lead to the premature termination codons into mRNA, resulting in degradation by nonsense‐mediated decay and loss of function of the minor allele.31 This may explain its association with a late‐onset of the disease. Although rs2935537 of C1QC apparently does not change TFBS, we cannot dismiss the hypothesis it causes any effect on the stability of mature mRNA and leads to decreased translation efficiency and, somehow, change gene expression, resulting in earlier onset. Furthermore, the C1QC variants associated with early‐onset may result in a mechanism similar to the inactivation of C1Q, leading to an increase in amyloid deposition; there are previous indications that amyloid deposits amounts are greater in early‐onset patients.32 Also, this observation is in line with the findings of Panayiotou et al., who recently demonstrated that Val30Met C1q knockout mice had more amyloid deposits (over 60%) than endogenous C1q control. Those differences increased with age.

### C1Q and other neurodegenerative disorders

C1Q complement genes demonstrated to play an important role in other neurodegenerative diseases, as Alzheimer's (AD)33 and Parkinson's.34 The complement system can have a dual role, either beneficial or damaging, depending on the associated pathophysiological mechanisms, as in some cases it can lead to tissue damage.35 In AD, it was demonstrated that under some circumstances C1Q plays a protective role and downregulates the inflammation process in early stages of neuronal injury, as well as amyloid‐induced neurotoxicity.36 In our TTR‐FAP sample, a similar mechanism might occur, as variants in C1QA seem to increase mean AO, as well as the rs200952686 in the C1QC. In contrast, other variants in C1QC may be a risk factor for earlier AO, possibly by increasing tissue damage.

### The role of common and rare variants

In this study, we also included common variants found to be associated with AO in the Cypriot population (rs172378 in C1QA and, rs15940 and rs9434 in the C1QC gene), but their effect failed to replicate in our sample. This suggests that different approaches or diverse population samples may lead to different results. This has been discussed previously around the role of APCS in AO variation.9 Regarding rare variants, we found only one in common with their study. We can hypothesize that rare variants are population specific and may play a role as genetic modifiers. Previous studies indicate that highly penetrant rare variants are genetic modifiers for various Mendelian disorders.37

### Final remarks

Our study is based on several strengths: (1) a large sample size 267 patients (117 families); (2) the clinical observation of all patients by the same team, minimizing differences in established AO; (3) a family centered approach (circumventing population stratification, as opposed to case‐control studies)38; (4) the use of GEE analysis, since it corrects for familial correlations of AO and provides a greater power to detect statistically significant effects39; and (5) the inclusion of multiple testing corrections to avoid type I‐errors.

We cannot exclude the possibility of other genes and other SNPs within these genes being associated with AO variability in TTR‐FAP Val30Met. Importantly, we conclude that C1QA was associated with later AO and C1QC may have a double role: some variants may confer earlier and other later AO, depending on the associated pathophysiological mechanisms. We confirmed the role of complement C1Q genes in AO variability in Portuguese patients with TTR‐FAP Val30Met. Furthermore, we reinforced the involvement of C1Q and the complement cascade as modifiers in TTR‐FAP Val30Met. However, this is a preliminary study on inflammation; therefore it is too premature to take conclusions for treatments or screening.

### Author Contributions

Conception and design of the study: AS, CL.

Acquisition and analysis of data: AD, DS, TC, MA‐F, AS, CL.

Drafting a significant portion of the manuscript or figures: AD, DS, MA‐F, AS, CL.

Critical revision of the manuscript for important intellectual content: DS, TC, MA‐F, JS, IA, AS.

Statistical expertise: AS, CL.

Obtaining funding: TC, JS, IA, AS, CL.

Administrative, technical or material support: DS, MA‐F, IA, CL.

Study supervision: DS, AS, CL.

### Conflict of Interest

D. Santos has received research support from a FCT fellowshi [SFRH/BD/91160/2012] and received funding from Pfizer Inc for scientific meeting expenses (travel, accommodations, and registration). T. Coelho's institution has received support from FoldRx Pharmaceuticals, which was acquired by Pfizer Inc in October 2010; T. Coelho has served on the scientific advisory board of Pfizer Inc and received funding from Pfizer Inc for scientific meeting expenses (travel, accommodations, and registration). She currently serves on the THAOS (natural history disease registry) scientific advisory board.

Jorge Sequeiros and Carolina Lemos received funding from Pfizer Inc for scientific meeting expenses (travel, accommodations, and registration) and honorary for presentations at TTR meetings.

Miguel Alves‐Ferreira has received research support from a FCT fellowship [SFRH/BD/101352/2014] and received funding from Pfizer Inc for scientific meeting expenses (travel, accommodations, and registration).

Andreia Dias, Isabel Alonso, and Alda Sousa declare that they have no conflict of interest.



# SUPPLEMENTAL FILE 1: ACN3-6-748.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)