# MAIN TEXT

## Three Newly Recognized Likely Pathogenic Gene Variants Associated with Hereditary Transthyretin Amyloidosis

### Abstract

IntroductionHereditary transthyretin amyloidosis (ATTRv [variant]) is a clinically heterogeneous, progressively debilitating, fatal disease resulting from the deposition of insoluble amyloid fibrils in various organs and tissues. Early diagnosis of ATTRv can be facilitated with genetic testing; however, such testing of the TTR gene identifies variants of uncertain significance (VUS) in a minority of cases, a small percentage of which have the potential to be pathogenic. The Akcea/Ambry VUS Initiative is dedicated to gathering molecular, clinical, and inheritance data for each TTR VUS identified by genetic testing programs to reclassify TTR variants to a clinically actionable status (e.g., variant likely pathogenic [VLP]) where appropriate.MethodsClassification criteria used here, based on recommendations from the American College of Medical Genetics and Genomics, are stringent and comprehensive, requiring distinct lines of evidence supporting pathogenesis.ResultsThree TTR variants have been reclassified from VUS to VLP, including c.194C>T (p.A65V), c.172G>C (p.D58H), and c.239C>T (p.T80I). In each case, the totality of genetic, structural, and clinical evidence provided strong support for pathogenicity.ConclusionsBased on several lines of evidence, three TTR VUS were reclassified as VLP, resulting in a high likelihood of disease diagnosis for those and subsequent patients as well as at-risk family members.Supplementary InformationThe online version contains supplementary material available at 10.1007/s40120-022-00385-1.

### Key Summary Points

Hereditary transthyretin amyloidosis (ATTRv [variant]) is a clinically heterogeneous, progressively debilitating, fatal genetic disease resulting from the deposition of insoluble amyloid fibrils in various organs and tissues.Early diagnosis of ATTRv can be facilitated with genetic testing; however, such testing of the TTR gene identifies variants of uncertain significance (VUS) in a minority of cases, a small percentage of which have the potential to be pathogenic.The objective of this study was to use molecular, clinical, and inheritance data for each TTR VUS identified by genetic testing programs to reclassify TTR variants to a clinically actionable status (e.g., variant likely pathogenic [VLP]) where appropriate.Three TTR variants were reclassified from VUS to VLP, including c.194C>T (p.A65V), c.172G>C (p.D58H), and c.239C>T (p.T80I).Based on genetic, structural, and clinical evidence, three TTR VUS were reclassified as VLP, which should aid in disease diagnosis for patients and at-risk family members who have these TTR variants.

### Digital Features

This article is published with digital features, including a video, to facilitate understanding of the article. To view digital features for this article, go to 10.6084/m9.figshare.19874797.

### Introduction

Hereditary transthyretin amyloidosis (ATTRv [variant]) is a clinically heterogeneous, progressively debilitating, and ultimately fatal disease that results from the deposition of insoluble amyloid fibrils in various organs and tissues [1–3]. This multisystem disorder has a variety of clinical manifestations, including polyneuropathy, cardiomyopathy-predominant phenotypes, or a mixture of both, depending on the underlying genetic variant [4–6]. Autonomic dysfunction, caused by polyneuropathy, develops in many patients, affecting the cardiocirculatory, gastrointestinal, and genitourinary systems [7]. In patients with ATTRv (p.V1421 variant, also referred to as V122I because of previous use of a protein nomenclature that was based on the primary structure of the mature protein after cleavage of a 20-amino acid signal sequence), median survival from diagnosis is 25.6 months [8].

The cause of ATTRv lies in the transthyretin (TTR) gene, which is located on chromosome 18q12.1 [3]; more than 130 pathogenic missense variants have been identified [9]. Missense pathogenic variants are single base-pair substitutions that lead to the translation of a different amino acid residue at that position. Such substitutions produce an abnormal TTR protein, causing instability and an increased propensity for protein misfolding, ultimately leading to the aggregation of insoluble amyloid deposits [3, 10]. ATTRv is known to have variable expressivity and reduced phenotypic penetrance. Even among relatives with an identical genetic mutation, some individuals will have no apparent disease, while other relatives may be symptomatic [11]. Genetic anticipation, the phenomenon where symptoms can occur earlier or have a more severe presentation with each successive generation, has also been described in ATTRv [12–14]. In addition, genetic heterogeneity can yield variable results based on the specific allelic combination expressed. For example, two variants, p.T139M/T119M and p.R124H/R104H, stabilize the TTR protein [15–19]. In patients who are compound heterozygous for one of these variants and a pathogenic variant, the clinical course is more benign than expected for the pathogenic variant alone [11, 15, 16]. Taken together, the combination of these attributes can make identifying the typically adult-onset clinical features of ATTRv challenging.

Despite the complexity of genetic expression, pathogenic variants are typically dominant and lead to progressive and fatal disease. As such, family screening and patient follow-up are strongly encouraged, even in asymptomatic individuals [2, 11, 20].

Early diagnosis facilitated by genetic testing and appropriate treatment of ATTRv are imperative to slow the progression of the disease [21]. Nevertheless, in a subset of patients, genetic testing of the TTR gene identifies variants of uncertain significance (VUS) that have insufficient information to adjudicate a pathogenic or benign classification [22].

The genetic code has a reasonable amount of built-in redundancy: many nucleic acid substitutions at the third position of a codon result in no amino acid changes and therefore result in no impact on protein structure. Even substitutions that do result in amino acid changes do not necessarily have a significant impact on overall protein structure or function. Most VUS likely reflect benign genetic variations present in the human genome; however, a small percentage of VUS have the potential to cause disease and ultimately can be reclassified to a variant likely pathogenic (VLP) [23]. Compiling additional patient reports, segregation data in large pedigrees, or compelling functional evidence is often needed for these variant upgrades. Given the serious consequences should a novel variant prove to be pathogenic, Akcea Therapeutics (a wholly owned subsidiary of Ionis Pharmaceuticals, Inc.) and Ambry Genetics partnered to form the Akcea/Ambry VUS Initiative, which is dedicated to gathering molecular, clinical, and inheritance data for TTR VUS identified by genetic testing programs and reclassifying TTR variants to a clinically actionable status (e.g., VLP) where appropriate. The importance of reclassification of VUS to VLP regarding its impact on informing patient management cannot be overstated [22, 24]. Here we report data supporting the reclassification of three TTR variants—p.A65V, p.D58H, and p.T80I—from VUS to VLP.

### Methods

Individuals underwent genetic testing for a clinical suspicion of ATTRv as part of the COMPASS program, which provides confidential genetic testing to patients in the United States (including Puerto Rico) and Canada who are suspected of having ATTRv or who report having a family history of ATTRv [25]. Eligible test types included an 81-gene panel associated with neuromuscular disorders (NeuropathySelect) performed using whole exome sequencing, a 92-gene panel associated with inherited cardiovascular conditions (CardioNext) using next-generation sequencing, or TTR single gene sequencing. All individuals consented to testing by their ordering providers. Testing outcomes were reviewed to identify alterations that were reclassified from a VUS to a VLP as part of the testing process. Data used to upgrade these variants were reviewed and collated. All work described herein was part of standard clinical laboratory efforts to supplement the clinical testing and resolve uncertain (VUS) results. Retrospective data analysis and reporting of deidentified data exempted the study from the requirement of receiving consent from patients. WGC IRB (formerly Western Institutional Review Board) determined the study to be exempt from the Office for Human Research Protections Regulations for the Protection of Human Subjects (45 CFR 46) under category 4.

The classification criteria used herein are based on recommendations from the American College of Medical Genetics and Genomics (ACMG) [24]. The criteria are stringent and comprehensive, requiring distinct lines of evidence to support variant classification. Variants were assessed for reclassification based on all the available evidence. For these variants, reclassification to VLP included the following (Table 1): other instances of the variant identified in unrelated probands; evidence of clinical disease based on diagnostic testing (e.g., biopsy or 99mTc pyrophosphate [PYP] imaging). Patients with grade 2 or 3 uptake on PYP scan were required to have negative monoclonal protein testing; rarity based on population cohorts in the Genome Aggregation Database (gnomAD); and structural biology (i.e., nearby pathogenic variants and hotspot). gnomAD (https://gnomad.broadinstitute.org/) is an international online resource developed to aggregate genetic variants from a wide variety of exome and genome large-scale sequencing projects. The purpose is to estimate the frequency of specific genetic variants in the general population. More commonly occurring alterations are less likely to be causative of rare diseases.Table 1Summary of criteria used for classification of VLPVariant 1Variant 2Variant 3Evidencep.D58H c.172G>Cp.A65V c.194C>Tp.T80I c.239C>TRarity based on population cohorts in gnomADXXXStructural biology (i.e., nearby pathogenic variants and hotspot)XXXEvidence of clinical disease based on diagnostic testing (e.g., biopsy or 99mTc pyrophosphate [PYP] imaging)XX–Other instances of the variant identified in unrelated probands meeting diagnostic criteria––XVLP variant likely pathogenic

Summary of criteria used for classification of VLP

VLP variant likely pathogenic

In silico structural and functional analyses were performed using computer simulation to determine a particular gene expression profile. These computational profiling techniques are relatively inexpensive, generate a large data set, and easily allow comparisons among numerous tissue types using opensource tools to integrate and compare against microarray data [26, 27].

The structure of TTR is derived from a known crystal structure (Protein Data Bank: 1TYR [28]); however, it was built using the homology modeling module of the ROSETTA de novo structure prediction method [29]. Sequence alignments were generated using TCoffee [30]. The initial build from the alignment was generated using a fixed backbone residue substitution with relaxation of the sidechains. Energy calculations for the destabilization of the internal structure and the protein–protein binding were performed with the FoldX program [31]. The lower energy bound for pathogenic variants that cause structural disruption of protein folding is usually observed to be around 2.5 kcal/mol [32, 33]. Variants that were lower in energy but more destabilizing than known, and nearby (within 1 Angstrom [Å]) pathogenic variants were also considered pathogenic. Images and protein structural alignments were generated using PYMOL (The PyMOL Molecular Graphics System, Version 1.8; Schrödinger, LLC). The structural analyses are updated over time as new pathogenic alterations are reported in the literature.

All data was obtained from requisition forms, clinic notes submitted at the time of testing, and internal laboratory case notes as part of standard clinical testing. As part of the clinical testing process, the authors also recontacted clinicians for additional phenotype data for individuals who harbored a VUS that were considered a candidate for upgrade to VLP. Reports of other patients with the same or allelic variants were identified through reviews of the literature, the Human Gene Mutation Database (HGMD®), the National Institutes of Health database, ClinVar (http://www.ncbi.nlm.nih.gov/clinvar/) [34], and other queries including variant-based Google and PubMed searches. The authors contacted other clinical laboratories reporting the variant in the ClinVar database for additional case-level data.

Western Institutional Review Board determined the study to be exempt from the Office for Human Research Protections Regulations for the Protection of Human Subjects (45 CFR 46) under category 4. All work described herein was part of standard clinical laboratory efforts to supplement the clinical testing and resolve uncertain (VUS) results. Retrospective data analysis and reporting of deidentified data exempted the study from the requirement of receiving consent from patients.

### Study Population

Individuals underwent genetic testing for a clinical suspicion of ATTRv as part of the COMPASS program, which provides confidential genetic testing to patients in the United States (including Puerto Rico) and Canada who are suspected of having ATTRv or who report having a family history of ATTRv [25]. Eligible test types included an 81-gene panel associated with neuromuscular disorders (NeuropathySelect) performed using whole exome sequencing, a 92-gene panel associated with inherited cardiovascular conditions (CardioNext) using next-generation sequencing, or TTR single gene sequencing. All individuals consented to testing by their ordering providers. Testing outcomes were reviewed to identify alterations that were reclassified from a VUS to a VLP as part of the testing process. Data used to upgrade these variants were reviewed and collated. All work described herein was part of standard clinical laboratory efforts to supplement the clinical testing and resolve uncertain (VUS) results. Retrospective data analysis and reporting of deidentified data exempted the study from the requirement of receiving consent from patients. WGC IRB (formerly Western Institutional Review Board) determined the study to be exempt from the Office for Human Research Protections Regulations for the Protection of Human Subjects (45 CFR 46) under category 4.

### Variant Classification Standards

The classification criteria used herein are based on recommendations from the American College of Medical Genetics and Genomics (ACMG) [24]. The criteria are stringent and comprehensive, requiring distinct lines of evidence to support variant classification. Variants were assessed for reclassification based on all the available evidence. For these variants, reclassification to VLP included the following (Table 1): other instances of the variant identified in unrelated probands; evidence of clinical disease based on diagnostic testing (e.g., biopsy or 99mTc pyrophosphate [PYP] imaging). Patients with grade 2 or 3 uptake on PYP scan were required to have negative monoclonal protein testing; rarity based on population cohorts in the Genome Aggregation Database (gnomAD); and structural biology (i.e., nearby pathogenic variants and hotspot). gnomAD (https://gnomad.broadinstitute.org/) is an international online resource developed to aggregate genetic variants from a wide variety of exome and genome large-scale sequencing projects. The purpose is to estimate the frequency of specific genetic variants in the general population. More commonly occurring alterations are less likely to be causative of rare diseases.Table 1Summary of criteria used for classification of VLPVariant 1Variant 2Variant 3Evidencep.D58H c.172G>Cp.A65V c.194C>Tp.T80I c.239C>TRarity based on population cohorts in gnomADXXXStructural biology (i.e., nearby pathogenic variants and hotspot)XXXEvidence of clinical disease based on diagnostic testing (e.g., biopsy or 99mTc pyrophosphate [PYP] imaging)XX–Other instances of the variant identified in unrelated probands meeting diagnostic criteria––XVLP variant likely pathogenic

Summary of criteria used for classification of VLP

VLP variant likely pathogenic

### In Silico Analysis

In silico structural and functional analyses were performed using computer simulation to determine a particular gene expression profile. These computational profiling techniques are relatively inexpensive, generate a large data set, and easily allow comparisons among numerous tissue types using opensource tools to integrate and compare against microarray data [26, 27].

The structure of TTR is derived from a known crystal structure (Protein Data Bank: 1TYR [28]); however, it was built using the homology modeling module of the ROSETTA de novo structure prediction method [29]. Sequence alignments were generated using TCoffee [30]. The initial build from the alignment was generated using a fixed backbone residue substitution with relaxation of the sidechains. Energy calculations for the destabilization of the internal structure and the protein–protein binding were performed with the FoldX program [31]. The lower energy bound for pathogenic variants that cause structural disruption of protein folding is usually observed to be around 2.5 kcal/mol [32, 33]. Variants that were lower in energy but more destabilizing than known, and nearby (within 1 Angstrom [Å]) pathogenic variants were also considered pathogenic. Images and protein structural alignments were generated using PYMOL (The PyMOL Molecular Graphics System, Version 1.8; Schrödinger, LLC). The structural analyses are updated over time as new pathogenic alterations are reported in the literature.

### Collection of Supportive Clinical Data

All data was obtained from requisition forms, clinic notes submitted at the time of testing, and internal laboratory case notes as part of standard clinical testing. As part of the clinical testing process, the authors also recontacted clinicians for additional phenotype data for individuals who harbored a VUS that were considered a candidate for upgrade to VLP. Reports of other patients with the same or allelic variants were identified through reviews of the literature, the Human Gene Mutation Database (HGMD®), the National Institutes of Health database, ClinVar (http://www.ncbi.nlm.nih.gov/clinvar/) [34], and other queries including variant-based Google and PubMed searches. The authors contacted other clinical laboratories reporting the variant in the ClinVar database for additional case-level data.

Western Institutional Review Board determined the study to be exempt from the Office for Human Research Protections Regulations for the Protection of Human Subjects (45 CFR 46) under category 4. All work described herein was part of standard clinical laboratory efforts to supplement the clinical testing and resolve uncertain (VUS) results. Retrospective data analysis and reporting of deidentified data exempted the study from the requirement of receiving consent from patients.

### Results

To date, three TTR variants within this testing program have been reclassified from VUS to VLP: c.172G>C (p.D58H), c.194C>T (p.A65V, also known as p.A45V), and c.239C>T (p.T80I) [Fig. 1 (and video 1 in the online/HTML version of the manuscript or follow the digital features link under the abstract)]. In each case, the totality of stringent genetic, structural, and clinical evidence provided strong support for pathogenicity. Additional supportive clinical evidence also was noted. No VUS were downgraded to benign.Fig. 1Crystal structure of the TTR protein dimer highlighting the loci of the three variants. The light blue and green colors indicate the individual monomers of the TTR dimer. The green TTR monomer is labeled with select variants. The red text indicates the position of the TTR variants reclassified from VUS to VLP (D58H, A65V, and T80I) and nearby pathogenic variants are shown in black. TTR, transthyretin; VLP variant likely pathogenic, VUS variant of uncertain significance

Crystal structure of the TTR protein dimer highlighting the loci of the three variants. The light blue and green colors indicate the individual monomers of the TTR dimer. The green TTR monomer is labeled with select variants. The red text indicates the position of the TTR variants reclassified from VUS to VLP (D58H, A65V, and T80I) and nearby pathogenic variants are shown in black. TTR, transthyretin; VLP variant likely pathogenic, VUS variant of uncertain significance

Video 1: A rotating crystal structure of the TTR protein dimer highlighting the loci of the three variants. The light blue and green colors indicate the individual monomers of the TTR dimer. The green TTR monomer is labeled with select variants. The red text indicates the position of the TTR variants reclassified from VUS to VLP (p.D58H, p.A65V, and p.T80I) and nearby pathogenic variants are shown in black (MP4 53315 KB)

Video 1: A rotating crystal structure of the TTR protein dimer highlighting the loci of the three variants. The light blue and green colors indicate the individual monomers of the TTR dimer. The green TTR monomer is labeled with select variants. The red text indicates the position of the TTR variants reclassified from VUS to VLP (p.D58H, p.A65V, and p.T80I) and nearby pathogenic variants are shown in black (MP4 53315 KB)

The p.D58H variant is located in coding exon 2 of the TTR gene, and results from a G to C substitution at nucleotide position 172. The aspartic acid at codon 58 is replaced by histidine, an amino acid with similar properties [35]. The final classification of VLP for this variant was based on the following evidence: the variant results in significant decrease in the structural stability of TTR; a different missense variant impacting the same amino acid position is known to be pathogenic; the variant is absent from gnomAD; and the variant was identified in a patient meeting the diagnostic criteria for ATTRv (Table 1).

The aspartic acid residue at codon 58 is well conserved in studied vertebrate species, and there is a moderate physicochemical difference between aspartic acid and histidine. The BayesDel in silico meta-prediction for this alteration was inconclusive [36]. Based on structural analysis, however, this variant is expected to result in a decrease in the structural stability of TTR. In particular, the variant p.D58H is more disruptive than pathogenic variants and VLPs located within 20 Å in the three-dimensional protein structure (e.g., c.157T>C [p.F53L] and c.250T>C [p.F84L]) [37–39].

Structural analysis also confirmed that this variant is more disruptive than a known pathogenic missense substitution in the same codon, c.173A>C (p.D58A) [40]. This p.D58A sequence change replaces the aspartic acid at codon 58 with alanine [41]. The p.D58A pathogenic alteration has been observed in numerous persons affected with ATTRv and is a predominant cause of ATTRv in Korea [42–44]. Other variants that disrupt the same residue (c.173A>T [p.D58V] and c.172G>T [p.D58Y]) have been observed in affected persons [45, 46], suggesting that it is a significant residue and that disruption is likely to be causative of disease.

Next, the p.D58H variant has not been reported in gnomAD [47]. From a clinical perspective, the patient in whom this alteration was detected was 67 years of age at diagnosis. Because this patient has been under the care of one of the authors, more detail is available. At presentation he had sensory dysfunction, spinal stenosis, heart disease, and a history of Achilles tendon rupture. Labile renal function later developed, likely because of cardiorenal syndrome. Monoclonal protein test results were abnormal; the workup for light chain (AL) amyloidosis was negative; a subsequent cardiac biopsy was positive for ATTR amyloidosis. An amino acid sequence abnormality in the TTR protein was not identified by tandem mass spectroscopy. Next, family history was notable as the patient’s brother exhibited similar manifestations at presentation. The brother had a history of heart failure at approximately the same age, attributed to agent orange exposure in the Vietnam War, and he subsequently died of a cardiac condition. In addition, the patient’s maternal first cousin has been diagnosed with ATTR-CM with the confirmed mutation of p.D58H. He was first noted to have developed left ventricular hypertrophy at the age of 42 years and diagnosed with ATTR-CM at age 64.

Based on the weight of available evidence to date, the p.D58H variant is likely to be pathogenic. Given this information, the patient is discussing cascade screening with family members.

The p.A65V variant, located in coding exon 2 of the TTR gene, results from a C to T substitution at nucleotide position 194. The alanine at codon 65 is replaced by valine, an amino acid with similar properties [35]. Variant classification of VLP was based on the following criteria: the variant results in significant decrease in the structural stability of TTR, a different missense variant that affects the same amino acid position is likely pathogenic, and the variant is absent from gnomAD; and the variant was identified in a biopsy-confirmed ATTRv case [48] (Table 1).

The impacted alanine residue is highly conserved in available vertebrate species, and there is only a small physicochemical difference between alanine and valine [35]. The BayesDel in silico meta-prediction for this alteration is inconclusive. However, based on structural analysis, this variant is expected to result in a decrease in structural stability that can result in misfolding of the native TTR structure and consequent formation of amyloidogenic intermediates. The destabilization energy is driven by pure steric interactions between the larger sidechains of valine surrounding the packed region. The region containing residues 58 to 65 in particular is a hot spot for amyloidogenic variants because it contains the C strand, the CD loop, and the D strand—located at the edge of each unfolded TTR monomer—which are the building blocks of amyloid fibrils [49].

A number of other variants impacting the same codon (p.A65T, p.A65G, p.A65S, and p.A65D) have been described in association with ATTRv [50–52], suggesting that this may be a clinically significant amino acid residue. In particular, the likely pathogenic p.A65G variant, has been reported in a Dutch family with biopsy-confirmed diagnosis of ATTRv [53], is absent from gnomAD, and exhibits a significant decrease in structural stability, albeit less severe than the decrease for p.A65V.

Finally, the p.A65V variant is absent from gnomAD. Based on the available evidence to date, this variant is likely to be pathogenic.

The p.A65V variant was detected in a patient in the seventh decade of life who presented with a history of autonomic (gastrointestinal symptoms and unintentional weight loss), motor, and sensory dysfunction. The p.A65V variant has also been described in one patient in a biopsy-confirmed ATTRv cohort [48]. To the authors’ knowledge, no cascade testing for this family was performed to date.

The p.T80I variant, located in coding exon 3 of the TTR gene, results from a C to T substitution at nucleotide position 239. The threonine at codon 80 is replaced by isoleucine, an amino acid with similar properties [35]. The variant classification of VLP was based on the following: the variant is absent from gnomAD; a different missense variant impacting the same amino acid position is a known pathogenic variant; and the variant has been identified in multiple persons with a clinical diagnosis of ATTRv (Table 1). At present, this variant is not included in gnomAD [47], indicating that it is not a common polymorphism.

Next, the threonine at amino acid residue 80 is highly conserved in available vertebrate species; there is a small physicochemical difference between threonine and isoleucine [35]. The BayesDel in silico meta-prediction for this alteration was inconclusive, as was the structural analysis. The alteration is negligibly stabilizing, and there is no clear indication about whether the p.T80I variant plays a direct role in fibril formation from known protein structures. However, an alternative amino acid substitution at this position, c.238A>G (p.T80A), is a well-described pathogenic variant that has been detected in numerous persons with ATTR, has been shown to segregate with disease, and has been demonstrated to have reduced conformational monomer stability in functional studies [37, 54–57].

This alteration has been reported in multiple patients with ATTR. Seven patients from six families with TTR p.T80I have been identified to date through testing at Ambry Genetics.

Family 1: A patient in the fourth decade of life presented with congestive heart failure and dilated cardiomyopathy. Echocardiography showed mild left ventricular hypertrophy, diffuse hypokinesis, and an ejection fraction of 25%. The patient also had a history of hypertension and chronic kidney disease.

Family 2: A patient in the sixth decade of life presented with heart disease and chronic kidney disease.

Family 3: A patient in the eighth decade of life presented with sensory dysfunction, gastrointestinal symptoms, heart disease, and bilateral carpal tunnel syndrome. In the previous 12 months, this patient had been diagnosed with congestive heart failure, with an ejection fraction of 30%.

Family 4: A patient in the fifth decade of life with restrictive cardiomyopathy, heart disease, renal issues, and positive PYP scan. This patient’s father had been diagnosed with hATTR at 68 years, although molecular testing results for the father were not available for review.

Family 5: A patient in the seventh decade of life with sensory dysfunction and bilateral carpal tunnel syndrome. A diagnosis of amyloidosis was reported confirmed by PYP, cardiac biopsy, and cardiac MRI which showed left ventricular hypertrophy. This individual also had a history of edema, shortness of breath, and numbness and tingling in his hands (possibly associated with his carpal tunnel syndrome). Family history is significant for heart disease and neuropathy. The patient’s son was also tested in the third decade of life and found to carry the p.T80I alteration, but no clinical symptoms were provided at the time of testing.

Family 6: A patient in the seventh decade with numbness and tingling in feet/hands, pain in extremities and motor dysfunction and lumbar spinal stenosis. The family history was positive for hATTR amyloidosis.

In addition to these internal probands, this variant has been detected in several patients with biopsy-confirmed ATTR tested at another clinical laboratory (Invitae, personal communication).

Based on the cumulative evidence to date, this variant is likely to be pathogenic and has been reported in the aforementioned cases. In the first case, it was reported as a VUS and later reclassified to VLP. The remaining two cases were identified after reclassification as part of the COMPASS program [25]. These cases do not appear to be linked.

### Variant 1: TTR c.172G>C (p.D58H)

The p.D58H variant is located in coding exon 2 of the TTR gene, and results from a G to C substitution at nucleotide position 172. The aspartic acid at codon 58 is replaced by histidine, an amino acid with similar properties [35]. The final classification of VLP for this variant was based on the following evidence: the variant results in significant decrease in the structural stability of TTR; a different missense variant impacting the same amino acid position is known to be pathogenic; the variant is absent from gnomAD; and the variant was identified in a patient meeting the diagnostic criteria for ATTRv (Table 1).

The aspartic acid residue at codon 58 is well conserved in studied vertebrate species, and there is a moderate physicochemical difference between aspartic acid and histidine. The BayesDel in silico meta-prediction for this alteration was inconclusive [36]. Based on structural analysis, however, this variant is expected to result in a decrease in the structural stability of TTR. In particular, the variant p.D58H is more disruptive than pathogenic variants and VLPs located within 20 Å in the three-dimensional protein structure (e.g., c.157T>C [p.F53L] and c.250T>C [p.F84L]) [37–39].

Structural analysis also confirmed that this variant is more disruptive than a known pathogenic missense substitution in the same codon, c.173A>C (p.D58A) [40]. This p.D58A sequence change replaces the aspartic acid at codon 58 with alanine [41]. The p.D58A pathogenic alteration has been observed in numerous persons affected with ATTRv and is a predominant cause of ATTRv in Korea [42–44]. Other variants that disrupt the same residue (c.173A>T [p.D58V] and c.172G>T [p.D58Y]) have been observed in affected persons [45, 46], suggesting that it is a significant residue and that disruption is likely to be causative of disease.

Next, the p.D58H variant has not been reported in gnomAD [47]. From a clinical perspective, the patient in whom this alteration was detected was 67 years of age at diagnosis. Because this patient has been under the care of one of the authors, more detail is available. At presentation he had sensory dysfunction, spinal stenosis, heart disease, and a history of Achilles tendon rupture. Labile renal function later developed, likely because of cardiorenal syndrome. Monoclonal protein test results were abnormal; the workup for light chain (AL) amyloidosis was negative; a subsequent cardiac biopsy was positive for ATTR amyloidosis. An amino acid sequence abnormality in the TTR protein was not identified by tandem mass spectroscopy. Next, family history was notable as the patient’s brother exhibited similar manifestations at presentation. The brother had a history of heart failure at approximately the same age, attributed to agent orange exposure in the Vietnam War, and he subsequently died of a cardiac condition. In addition, the patient’s maternal first cousin has been diagnosed with ATTR-CM with the confirmed mutation of p.D58H. He was first noted to have developed left ventricular hypertrophy at the age of 42 years and diagnosed with ATTR-CM at age 64.

Based on the weight of available evidence to date, the p.D58H variant is likely to be pathogenic. Given this information, the patient is discussing cascade screening with family members.

### Variant 2: TTR c.194C>T (p.A65V)

The p.A65V variant, located in coding exon 2 of the TTR gene, results from a C to T substitution at nucleotide position 194. The alanine at codon 65 is replaced by valine, an amino acid with similar properties [35]. Variant classification of VLP was based on the following criteria: the variant results in significant decrease in the structural stability of TTR, a different missense variant that affects the same amino acid position is likely pathogenic, and the variant is absent from gnomAD; and the variant was identified in a biopsy-confirmed ATTRv case [48] (Table 1).

The impacted alanine residue is highly conserved in available vertebrate species, and there is only a small physicochemical difference between alanine and valine [35]. The BayesDel in silico meta-prediction for this alteration is inconclusive. However, based on structural analysis, this variant is expected to result in a decrease in structural stability that can result in misfolding of the native TTR structure and consequent formation of amyloidogenic intermediates. The destabilization energy is driven by pure steric interactions between the larger sidechains of valine surrounding the packed region. The region containing residues 58 to 65 in particular is a hot spot for amyloidogenic variants because it contains the C strand, the CD loop, and the D strand—located at the edge of each unfolded TTR monomer—which are the building blocks of amyloid fibrils [49].

A number of other variants impacting the same codon (p.A65T, p.A65G, p.A65S, and p.A65D) have been described in association with ATTRv [50–52], suggesting that this may be a clinically significant amino acid residue. In particular, the likely pathogenic p.A65G variant, has been reported in a Dutch family with biopsy-confirmed diagnosis of ATTRv [53], is absent from gnomAD, and exhibits a significant decrease in structural stability, albeit less severe than the decrease for p.A65V.

Finally, the p.A65V variant is absent from gnomAD. Based on the available evidence to date, this variant is likely to be pathogenic.

The p.A65V variant was detected in a patient in the seventh decade of life who presented with a history of autonomic (gastrointestinal symptoms and unintentional weight loss), motor, and sensory dysfunction. The p.A65V variant has also been described in one patient in a biopsy-confirmed ATTRv cohort [48]. To the authors’ knowledge, no cascade testing for this family was performed to date.

### Variant 3: TTR c.239C>T (p.T80I)

The p.T80I variant, located in coding exon 3 of the TTR gene, results from a C to T substitution at nucleotide position 239. The threonine at codon 80 is replaced by isoleucine, an amino acid with similar properties [35]. The variant classification of VLP was based on the following: the variant is absent from gnomAD; a different missense variant impacting the same amino acid position is a known pathogenic variant; and the variant has been identified in multiple persons with a clinical diagnosis of ATTRv (Table 1). At present, this variant is not included in gnomAD [47], indicating that it is not a common polymorphism.

Next, the threonine at amino acid residue 80 is highly conserved in available vertebrate species; there is a small physicochemical difference between threonine and isoleucine [35]. The BayesDel in silico meta-prediction for this alteration was inconclusive, as was the structural analysis. The alteration is negligibly stabilizing, and there is no clear indication about whether the p.T80I variant plays a direct role in fibril formation from known protein structures. However, an alternative amino acid substitution at this position, c.238A>G (p.T80A), is a well-described pathogenic variant that has been detected in numerous persons with ATTR, has been shown to segregate with disease, and has been demonstrated to have reduced conformational monomer stability in functional studies [37, 54–57].

This alteration has been reported in multiple patients with ATTR. Seven patients from six families with TTR p.T80I have been identified to date through testing at Ambry Genetics.

Family 1: A patient in the fourth decade of life presented with congestive heart failure and dilated cardiomyopathy. Echocardiography showed mild left ventricular hypertrophy, diffuse hypokinesis, and an ejection fraction of 25%. The patient also had a history of hypertension and chronic kidney disease.

Family 2: A patient in the sixth decade of life presented with heart disease and chronic kidney disease.

Family 3: A patient in the eighth decade of life presented with sensory dysfunction, gastrointestinal symptoms, heart disease, and bilateral carpal tunnel syndrome. In the previous 12 months, this patient had been diagnosed with congestive heart failure, with an ejection fraction of 30%.

Family 4: A patient in the fifth decade of life with restrictive cardiomyopathy, heart disease, renal issues, and positive PYP scan. This patient’s father had been diagnosed with hATTR at 68 years, although molecular testing results for the father were not available for review.

Family 5: A patient in the seventh decade of life with sensory dysfunction and bilateral carpal tunnel syndrome. A diagnosis of amyloidosis was reported confirmed by PYP, cardiac biopsy, and cardiac MRI which showed left ventricular hypertrophy. This individual also had a history of edema, shortness of breath, and numbness and tingling in his hands (possibly associated with his carpal tunnel syndrome). Family history is significant for heart disease and neuropathy. The patient’s son was also tested in the third decade of life and found to carry the p.T80I alteration, but no clinical symptoms were provided at the time of testing.

Family 6: A patient in the seventh decade with numbness and tingling in feet/hands, pain in extremities and motor dysfunction and lumbar spinal stenosis. The family history was positive for hATTR amyloidosis.

In addition to these internal probands, this variant has been detected in several patients with biopsy-confirmed ATTR tested at another clinical laboratory (Invitae, personal communication).

Based on the cumulative evidence to date, this variant is likely to be pathogenic and has been reported in the aforementioned cases. In the first case, it was reported as a VUS and later reclassified to VLP. The remaining two cases were identified after reclassification as part of the COMPASS program [25]. These cases do not appear to be linked.

### Discussion

Genetic testing is becoming more commonplace across a broad range of diseases and, as such, is increasingly being requested of and managed by clinicians who are not geneticists [58]. It is therefore important that healthcare providers are competent in clinical genetics and become comfortable incorporating genetic information into routine clinical practice [58, 59]. Variant reclassification is not common but will occur as our understanding of the genetic contributions to human disease grow. Providers should discuss the possibility of variant reclassification with their patients during the pre-test consenting process and establish a plan for follow-up communication should the provider need to recontact the patient with a reclassification notice in the future. In this era of personalized medicine, it is imperative that clinicians be familiar with the implications of genetic testing on patient management and with their role in genetic counseling [58, 59].

Genetic testing is a mandatory diagnostic tool for patients with ATTRv, given the variable genotypic and phenotypic presentation that makes symptom recognition outside a specialized diagnostic environment challenging [21]. Any delay in diagnosis is a significant obstacle to the optimal management of these patients, particularly for patients with no family history of ATTRv. Several years may elapse between the emergence of clinical signs and symptoms and an accurate diagnosis [21].

However, rapid integration of genetic testing into clinical practice has outpaced the ability to interpret these large DNA data sets. The accumulation of data from additional testing, functional studies, advancements in in silico and computational techniques, and the establishment of large data sharing initiatives including population databases (e.g., gnomAD) and variants reported in patients (e.g., ClinVar) have led to the reclassification of variants [23]. This reclassification is particularly relevant to VUS. Approximately 50% of all reclassifications in ClinVar involve VUS that are later reclassified as VLP or benign/likely benign [23].

Based on distinct lines of evidence, three TTR VUS, c.194C>T (p.A65V), c.172G>C (p.D58H), and c.239C>T (p.T80I), were reclassified here as VLP, resulting in a high likelihood of disease diagnosis for current and all subsequently tested patients. The new classification of each variant will be submitted to the ClinVar database. As with other variants, a coordinated approach to family history and cascade screening will be required after these three TTR variants are reclassified.

An important limitation of this analysis was the level of clinical detail accompanying the cases. For instance, based on the current standard of care for the diagnosis of cardiac amyloidosis, patients who had a positive PYP scan probably did not have a biopsy with subsequent typing utilizing mass spectroscopy or immunohistochemistry. Thus, it is possible that one or more of these patients may have had wild type amyloidosis as the primary contributor to the presenting symptoms, along with a mutation associated with hereditary transthyretin amyloidosis.

### Conclusions

Based on several lines of evidence, three TTR VUS were reclassified as VLP, resulting in a high likelihood of disease diagnosis for those and subsequent patients as well as at-risk family members. A change in classification from VUS to VLP has important implications for the clinical management of symptomatic patients with ATTRv, asymptomatic carriers and family members [22]. With the availability of novel therapies that alter the natural disease course of ATTRv [6, 60, 61], reclassification of these variants will increase the likelihood that patients will receive early and appropriate treatment. Patients who previously received a genetic diagnosis of VUS for any of these variants will have to be contacted, and an appropriate management plan initiated in concert with genetic counseling for symptomatic patients, asymptomatic carriers, and family members [22]. Although the ACMG has published well-established criteria on the classification of sequence variants [24], there is a paucity of recommendations on practical next steps, an issue not limited to ATTRv [62, 63]. After reclassification of a VUS to a VLP, guidelines and protocols that aid clinicians to recontact and facilitate the effective management of patients and carriers are urgently needed.



# SUPPLEMENTAL FILE 1: 40120_2022_Article_385.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)