# MAIN TEXT

## Clinical Metabolomic Landscape of Cardiovascular Physiology and Disease

### METABOLOMIC ANALYTICAL PLATFORMS

The major metabolomic analytical methods are mass spectrometry (MS) and nuclear magnetic resonance (NMR) spectroscopy.
3
, 
4
 MS is the most widely used analytical method and identifies metabolites by separating ions based on charge‐to‐mass ratios.
5
, 
6
 MS requires separation of compounds before analysis, which is performed using gas chromatography or liquid chromatography.
2
, 
5
, 
6
 High‐performance liquid chromatography and recently ultra‐performance liquid chromatography are combined with MS for semiquantitative metabolite analyses.
7
 Ultra‐performance liquid chromatography has increased sensitivity and throughput compared with high‐performance liquid chromatography and also shortened run times. Advantages of MS include high sensitivity, rapid analysis, and broad metabolite coverage.

An alternative metabolomic analytical method is NMR spectrometry. This technique uses radio waves to promote nuclei to high energy states, emitting radiation during relaxation back to a low‐energy state, which is measured and used to elucidate the molecular structures of unknown compounds.
2
 Advantages of NMR spectroscopy include superior analyses for molecules that are difficult to ionize. It is also used to differentiate compounds of identical mass and to study metabolite transformations during metabolic reactions.
4
, 
6
 Compared with MS, NMR spectroscopy has a lower sensitivity and metabolite coverage.
2
, 
4
, 
5
, 
6
 However, NMR spectroscopy is particularly effective in quantifying metabolites and elucidating novel molecular structures.
4
, 
5

### METABOLOMIC STUDY DESIGN

Both MS and NMR‐based metabolomic studies use targeted or untargeted analyses. Targeted analyses identify defined metabolites from known standards and allow for more sensitivity in the testing of specific hypotheses. Because of its high specificity and sensitivity, MS is often used for targeted analyses. Meanwhile, untargeted analyses use an unbiased approach with the potential to discover novel metabolites. A challenge of untargeted studies is in the identification of new metabolites without established metabolite standards. Recently, metabolome‐ and genome‐wide association studies have been used to report metabolite quantitative trait loci to better characterize these untargeted peaks.
8

### STATISTICAL CONSIDERATIONS

Because of the complexity and quantity of data generated in metabolomic experiments, statistical analysis is complex. Raw spectral data require significant bioinformatic preprocessing, including baseline correction, normalization, scaling, peak alignment, detection, and quantification. After the preprocessing of the metabolomic data, statistical analyses of metabolomic data sets often require a layered approach because of the complexity of the data. As an initial approach, univariate analysis (eg, 1 metabolite) is often performed. Next, multivariate analysis can be performed that includes the analysis of ≥2 variables (eg, metabolites, characteristics) at a time. In univariate and multivariate analyses, correcting for multiple tests is necessary to minimize the probability of false positives. However, metabolomic data sets are high dimensional, which are computationally complex. Thus, dimensionality‐reduction tools are often used, such as principal component analysis and partial least squares discriminant analysis. Both principal component analysis and partial least squares discriminant analysis convert the original data into a lower dimensional space while capturing as much of the observed variation as possible; however, partial least squares discriminant analysis also captures information on the relation between independent and dependent variables. After results are obtained, they must be biologically interpreted, which can be challenging given the complexity of the data (FigureFigure). To address this, pathway and network analyses tools can be used to better identify and visualize relevant metabolomic pathways and relationships.
9

Tissue and/or plasma is collected and analyzed with mass spectroscopy (MS) or nuclear magnetic resonance (NMR) spectroscopy, and metabolites are identified. The metabolomics data are then interpreted in the clinical context of cardiovascular physiology or pathophysiology. Created with BioRender.com.

### DIAGNOSTIC AND PROGNOSTIC CAPABILITIES OF METABOLOMICS IN CARDIOVASCULAR RESEARCH

The rapid growth in metabolomics platforms has provided significant insights into cardiovascular physiology and pathophysiology. Metabolomics has elucidated new pathophysiological pathways for common disease states, including coronary artery disease, cardiomyopathies, stroke, and diabetes.
10
, 
11
, 
12
, 
13
 Recently, untargeted plasma metabolomics of patients with cardiomyopathy identified distinct systemic metabolomic signatures for dilated cardiomyopathy and ischemic cardiomyopathy, providing insights into underlying disease‐specific mechanisms. A panel of 6 circulating metabolites also showed excellent discrimination to distinguish patients with dilated cardiomyopathy and ischemic cardiomyopathy.
12

Metabolomics is also being used in prospective models to predict the development of disease and adverse outcomes. For example, a targeted NMR‐based analysis in the National Finnish FINRISK Study identified 33 plasma metabolites predictive of incident CVD during a 15‐year follow‐up period. After multivariable adjustment, serum phenylalanine and mono‐unsaturated fatty acids were positively associated with future CVD, whereas serum omega‐6 fatty acids and docosahexaenoic acid were associated with lower risk. A risk score using these 4 metabolites derived from FINRISK improved risk classification in individuals with moderate CVD risk in 2 separate, large validation cohorts.
10
 In a separate study, NMR spectrometry identified metabolomic profiles predictive of atherosclerotic risk in 24‐ to 39‐year‐old Finnish patients. The addition of the lipoprotein‐based metabolomic profile to the Framingham risk score was significantly more predictive of intimal‐media thickening in coronary arteries than the Framingham risk score alone.
11

### CURRENT CHALLENGES AND FUTURE DIRECTIONS

The complexity of the metabolome offers many clinical opportunities but also presents several challenges. There is a need for more universal naming standards for identifying metabolites, which will likely be driven by advancements in metabolomic technologies and improvements in open‐access spectral and chemical databases. Current public repositories to integrate, analyze, and deposit metabolomics data include the National Institutes of Health–sponsored Metabolomics Workbench. The interpretation of metabolomics data is also limited by heterogeneity in the statistical analysis, and there is a need for improved publicly available biostatistical tools.

Metabolomics is now being leveraged to personalize medicine, including patient‐specific drug therapies and dosing; pharmacometabolomic research has demonstrated that metabolomic profiles can inform how a specific patient will respond to a drug. For example, baseline levels of 2‐hydroxyvaleric acids strongly discriminated responders and nonresponders to statins.
14
 In the future, pharmacometabolomics may inform clinicians which patients should be prescribed statins as opposed to alternative therapies, such as PCSK9 (proprotein convertase subtilisin/kexin type 9) inhibitors, for the treatment of hyperlipidemia.

Metabolomics is being increasingly integrated with other ‐omics profiles (eg, genomics, proteomics) to generate more comprehensive snapshots of cardiovascular biology. Recently, whole genome sequencing was used to develop a bioinformatic tool to identify unknown metabolite peaks in MS analysis. The investigators used this tool in the Jackson Heart Study to identify a polymorphism in TTR (transthyretin) in Black individuals, previously known to be linked with heart failure, and identified a relationship with a previously unknown plasma metabolite called all‐trans‐retinol. A TTR V122I variant (found in 3%–4% of Black individuals) may promote amyloid fibril deposition, leading to heart failure by destabilizing the TTR‐RBP4 (retinol‐binding protein 4 and TTR tetramer) complex. Future studies combining genomics with metabolomics will allow researchers to link genomic loci with metabolic derangements that lead to CVD.
15
 Furthermore, transcriptomics has been used to recognize how differential tissue‐specific gene expression affects metabolite levels in specific organs in addition to circulating systemic levels, which may improve our understanding of CVD pathophysiology, such as heart failure.
5

### CONCLUSIONS

Metabolomic association studies are beginning to point toward novel pathways in CVD that can now be tested experimentally or using other techniques to assess causality. Metabolomic‐based tools are now being used in disease prediction, drug development, and personalized medicine in CVD. As technologies continue to advance, metabolomics will become an increasingly important component of cardiovascular medicine and the study of cardiometabolic diseases.

### Sources of Funding

Dr Sawicki is supported by the National Institutes of Health (T32HL069771).

### Disclosures

Dr Wilkins serves as a consultant to 3M. The remaining authors have no disclosures to report.



# SUPPLEMENTAL FILE 1: JAH3-12-e027725.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)