# MAIN TEXT

## Generation of two induced pluripotent stem cell lines from hereditary amyloidosis patients with polyneuropathy carrying heterozygous transthyretin (TTR) mutation

### Abstract

Hereditary transthyretin amyloidosis with polyneuropathy (ATTR-PN) results from specific TTR gene mutations. In this study, we generated two induced pluripotent stem cell (iPSC) lines derived from ATTR-PN patients with heterozygous TTR gene mutations (Ala97Ser and Phe64Leu). These iPSC lines exhibited normal morphology, karyotype, high pluripotency marker expression, and differentiation into cells representing all germ layers. The generation of these iPSC lines serve as a valuable tool for investigating the mechanisms of ATTR-PN across various cell types and facilitating patient-specific in vitro amyloidosis modeling.

### Resource table

### Resource utility

The use of human induced pluripotent stem cells (iPSCs) is a significant innovation in precision medicine and drug discovery. These cells enable in vitro disease modeling tailored to individual patients and serve as a rich source for various cell types, including cardiomyocytes, neurons, and hepatocytes. iPSCs offer valuable insights into the study transthyretin amyloidosis (Bonilauri et al., 2023). In this study, we successfully derived two iPSC lines from female patients with hereditary transthyretin polyneuropathy amyloidosis, both carrying specific TTR gene mutations (c.349G > T and c.250 T > C) (Table 1).

### Resource details

Transthyretin Amyloidosis (ATTR) is a severe systemic disease marked by the accumulation of amyloid fibrils formed from transthyretin protein in various tissues, posing life-threatening consequences. This condition encompasses two distinct forms: hereditary ATTR, stemming from genetic mutations in the TTR gene, and wild-type ATTR, originating from the misfolding of normal, non-mutated transthyretin protein (Ruberg and Berk, 2012). Hereditary transthyretin amyloidosis with polyneuropathy (ATTR-PN), also described as familial amyloid polyneuropathy (FAP), is of particular interest as it can affect the peripheral and autonomic nervous system. This condition is defined by the progressive accumulation of amyloid proteins in various tissues, especially nerves, giving rise to a range of neurological symptoms, including sensory, motor, and autonomic neuropathy (Adams et al., 2019). These manifestations often lead to severe disabilities, substantially affecting the patient’s quality of life, and can ultimately lead to death. The most frequently observed genotype associated with ATTR-PN includes mutations such as Val30Met, Ser77Tyr, Ala97Ser, and Phe64Leu, with Val30Met being the predominant variant globally (Waddington-Cruz et al., 2019). Each mutation is associated with a unique phenotype, including average age-of-onset, severity, and organ specificity (e.g., polyneuropathy-dominant vs. cardiomyopathy-dominant) (Adams et al., 2021). Understanding the underlying genetic mutations and molecular mechanisms contributing to ATTR-PN is crucial for developing targeted treatments aimed at alleviating symptoms and improving the prognosis for affected individuals.

In this study, we successfully derived two human iPSC lines, SCVIi091-A and SCVIi092-A, originating from peripheral blood mononuclear cells (PBMCs) of two patients with ATTR-PN carrying TTR gene variants. The first patient, a 75-year-old East Asian female (SCVIi091-A), possessed the pathogenic c.349G > T variant, resulting in the Ala97Ser mutation. The second patient, a 68-year-old Caucasian female (SCVIi092-A), carried the pathogenic c.250 T > C variant, leading to the Phe64Leu mutation. We successfully reprogrammed the donor PBMCs into iPSCs using a non-integrating Sendai virus containing the four Yamanaka factors. Both iPSC lines displayed typical morphology. RT-qPCR analysis confirmed high RNA-level expression of pluripotency markers NANOG and SOX2, similar to a non-mutated iPSC line, distinguishing them from iPSC-derived cardiomyocytes (Fig. 1B–C). Importantly, we observed the absence of the non-integrating Sendai virus in SCVIi091-A and SCVIi092-A at later passages (Fig. 1D). We also confirmed the presence of pluripotency markers OCT3/4, NANOG, and SOX2 at the protein level (Fig. 1E). Sanger sequencing validated the presence of heterozygous TTR mutations c.349G > T and c.250 T > C in the respective iPSC lines (Fig. 1F–G). Karyotyping analysis of the iPSC lines revealed chromosomal stability without insertions or deletions (Fig. 1H). Furthermore, both iPSC lines demonstrated successful differentiation into cells representing the three germ layers (i.e., ectoderm, mesoderm, and endoderm) (Fig. 1I and Supplementary Fig. 1A and 1B). Importantly, we detected no mycoplasma contamination in either line (Supplementary Fig. 1C), and STR analysis confirmed their concordance with the somatic donor cells (submitted in the journal’s archive).

### Materials and methods

To obtain PBMCs, the patient’s whole blood was subjected to isolation using Percoll® density gradient separation medium (GE Healthcare #17089109), followed by purification through multiple washes with DPBS (Thermo Fisher Scientific #14190144). Subsequently, the isolated PBMCs were cultured in StemPro®-34 SFM medium (Thermo Fisher Scientific #10639011), supplemented with 100 ng/mL SCF (Peprotech #300–07), 100 ng/mL FLT3 (Thermo Fisher Scientific #PHC9414), 20 ng/mL IL-3 (Peprotech #200–3), 20 ng/mL IL-6 (Thermo Fisher Scientific #PHC0063), and 20 ng/mL EPO (Thermo Fisher Scientific #PHC9631). The medium was refreshed every two days until the cell culture reached a stable state.

For cell reprogramming, the CytoTune™-iPSC 2.0 Sendai Reprogramming Kit (Thermo Fisher Scientific #A16517) was employed following the manufacturer’s instructions. The transduced cells were plated on Matrigel-coated plates and cultured in StemPro®-34 medium. The medium was replaced every two days until day 7, at which point it was switched to supplemented StemMACS™ iPS-Brew XF medium (Miltenyi Biotec #130–104-368) until day 10–15 post-transduction. Colonies then became visible and ready for clonal expansion. Selected colonies were expanded and cryopreserved for future experimental use.

Patient-derived iPSCs were cultured in StemMACS™ iPS-Brew XF medium (Miltenyi Biotec, #130–107-086 and #130–107-087) until they reached 90 % confluency. Upon reaching confluency, the cells were passaged using 0.5 mM EDTA (Invitrogen, #15575–038) and seeded onto Matrigel-coated plates (Corning, #356231). During the cell culture process, StemMACS™ iPS-Brew XF medium was supplemented with 10 μM of the ROCK inhibitor Y-27632 (Selleck Chemicals #S1049). The Y-27632 inhibitor was removed after 24 h. The cells were maintained in a temperature-controlled incubator at 37 °C with a 5 % CO2 atmosphere.

Trilineage differentiation was performed for 48 h using the following kits and methods. Differentiation into endoderm was achieved using the STEMdiff™ Definitive Endoderm Differentiation Kit (STEMCELL™ Technologies #05110). For ectoderm differentiation, the Human Pluripotent Stem Cell Functional Identification Kit (R&D Systems #SC027B) was used. To induce mesoderm differentiation, 6 μM CHIR (Selleck Chemicals, #S2924) in RPMI media supplemented with B27-Minus Insulin (Gibco #11875–085 and #A18956–01) was used. Throughout the differentiation process, we closely monitored the cells for morphological changes and the expression of specific markers associated with each germ layer.

For the qualitative assessment of pluripotency and trilineage differentiation, cells were fixed in 4 % paraformaldehyde (PFA) at room temperature (RT) for 10 min. Following this, we performed permeabilization using 50 μg/mL digitonin (Sigma Aldrich #D141) for 10 min. To prevent non-specific binding, we applied a 1 % Bovine Serum Albumin (BSA) solution, allowing it to incubate for 30 min at RT. Following this, the cells underwent an overnight incubation at 4 °C with the primary antibodies (Table 2). After this incubation period, the cells were thoroughly washed three times and exposed to the corresponding secondary antibodies (Table 2) for 30 min at RT. Nuclei were stained using Molecular Probes NucBlue™ (Thermo Fisher Scientific #R37606) for 10 min at RT. The cells then underwent three additional washes before being imaged using a confocal light microscope.

To quantitatively analyze pluripotency, we first extracted total RNA from iPSCs (passage number 17) using the Direct-zol™ RNA Miniprep Kit (ZYMO Research #3R2061). Subsequently, we synthesized cDNA following the manufacturer’s instructions using the iScript™ cDNA Synthesis Kit (BioRad #1708891). Gene expression analysis for SOX2, NANOG, and detection of non-integrative Sendai virus (SEV), was carried out using primers (Table 2) and the TaqMan™ Gene Expression Assay (Applied Biosystems™ #4444556). Positive and negative controls for pluripotency markers (SOX2 and NANOG) included a non-mutated iPSC line and iPSC-derived cardiomyocytes, respectively.

Genomic DNA (gDNA) was extracted from iPSCs at passage number 20 and PBMCs using DNeasy Blood & Tissue Kit (Qiagen #69504). STR analysis was conducted using the CLA Identifiler™ Direct PCR Amplification Kit (Thermo Fisher Scientific #A44661). Capillary electrophoresis was carried out on ABI3130xl by the Stanford Protein Nucleic Acid (PAN) Facility.

Approximately 2 million cells were carefully collected from the iPSC lines at passage number 20 for analysis using the precise KaryoStat™ assay from Thermo Fisher Scientific.

PCR primers were designed to identify TTR mutations (Table 2). These primers were employed to amplify the genomic region of the gDNA utilizing the KOD One PCR Master Mix (DiagnoCine, #KMM-101). The PCR reaction was conducted under the following thermal conditions: Initial denaturation at 95 °C for 1 min, followed by 35 cycles of denaturation at 95 °C for 15 sec, annealing at 60 °C for 15 sec, and extension at 72 °C for 30 sec, with a final extension step at 72 °C for 7 min. Sanger sequencing was performed by Azenta.

Mycoplasma contamination was assessed at passage 20 using the MycoAlert™ Detection Kit (Lonza #LT07–318) following manufacturer’s instructions.

### Reprogramming

To obtain PBMCs, the patient’s whole blood was subjected to isolation using Percoll® density gradient separation medium (GE Healthcare #17089109), followed by purification through multiple washes with DPBS (Thermo Fisher Scientific #14190144). Subsequently, the isolated PBMCs were cultured in StemPro®-34 SFM medium (Thermo Fisher Scientific #10639011), supplemented with 100 ng/mL SCF (Peprotech #300–07), 100 ng/mL FLT3 (Thermo Fisher Scientific #PHC9414), 20 ng/mL IL-3 (Peprotech #200–3), 20 ng/mL IL-6 (Thermo Fisher Scientific #PHC0063), and 20 ng/mL EPO (Thermo Fisher Scientific #PHC9631). The medium was refreshed every two days until the cell culture reached a stable state.

For cell reprogramming, the CytoTune™-iPSC 2.0 Sendai Reprogramming Kit (Thermo Fisher Scientific #A16517) was employed following the manufacturer’s instructions. The transduced cells were plated on Matrigel-coated plates and cultured in StemPro®-34 medium. The medium was replaced every two days until day 7, at which point it was switched to supplemented StemMACS™ iPS-Brew XF medium (Miltenyi Biotec #130–104-368) until day 10–15 post-transduction. Colonies then became visible and ready for clonal expansion. Selected colonies were expanded and cryopreserved for future experimental use.

### Cell culture

Patient-derived iPSCs were cultured in StemMACS™ iPS-Brew XF medium (Miltenyi Biotec, #130–107-086 and #130–107-087) until they reached 90 % confluency. Upon reaching confluency, the cells were passaged using 0.5 mM EDTA (Invitrogen, #15575–038) and seeded onto Matrigel-coated plates (Corning, #356231). During the cell culture process, StemMACS™ iPS-Brew XF medium was supplemented with 10 μM of the ROCK inhibitor Y-27632 (Selleck Chemicals #S1049). The Y-27632 inhibitor was removed after 24 h. The cells were maintained in a temperature-controlled incubator at 37 °C with a 5 % CO2 atmosphere.

### Trilineage differentiation

Trilineage differentiation was performed for 48 h using the following kits and methods. Differentiation into endoderm was achieved using the STEMdiff™ Definitive Endoderm Differentiation Kit (STEMCELL™ Technologies #05110). For ectoderm differentiation, the Human Pluripotent Stem Cell Functional Identification Kit (R&D Systems #SC027B) was used. To induce mesoderm differentiation, 6 μM CHIR (Selleck Chemicals, #S2924) in RPMI media supplemented with B27-Minus Insulin (Gibco #11875–085 and #A18956–01) was used. Throughout the differentiation process, we closely monitored the cells for morphological changes and the expression of specific markers associated with each germ layer.

### Immunofluorescent staining

For the qualitative assessment of pluripotency and trilineage differentiation, cells were fixed in 4 % paraformaldehyde (PFA) at room temperature (RT) for 10 min. Following this, we performed permeabilization using 50 μg/mL digitonin (Sigma Aldrich #D141) for 10 min. To prevent non-specific binding, we applied a 1 % Bovine Serum Albumin (BSA) solution, allowing it to incubate for 30 min at RT. Following this, the cells underwent an overnight incubation at 4 °C with the primary antibodies (Table 2). After this incubation period, the cells were thoroughly washed three times and exposed to the corresponding secondary antibodies (Table 2) for 30 min at RT. Nuclei were stained using Molecular Probes NucBlue™ (Thermo Fisher Scientific #R37606) for 10 min at RT. The cells then underwent three additional washes before being imaged using a confocal light microscope.

### RT-qPCR

To quantitatively analyze pluripotency, we first extracted total RNA from iPSCs (passage number 17) using the Direct-zol™ RNA Miniprep Kit (ZYMO Research #3R2061). Subsequently, we synthesized cDNA following the manufacturer’s instructions using the iScript™ cDNA Synthesis Kit (BioRad #1708891). Gene expression analysis for SOX2, NANOG, and detection of non-integrative Sendai virus (SEV), was carried out using primers (Table 2) and the TaqMan™ Gene Expression Assay (Applied Biosystems™ #4444556). Positive and negative controls for pluripotency markers (SOX2 and NANOG) included a non-mutated iPSC line and iPSC-derived cardiomyocytes, respectively.

### Short tandem repeat (STR) analysis

Genomic DNA (gDNA) was extracted from iPSCs at passage number 20 and PBMCs using DNeasy Blood & Tissue Kit (Qiagen #69504). STR analysis was conducted using the CLA Identifiler™ Direct PCR Amplification Kit (Thermo Fisher Scientific #A44661). Capillary electrophoresis was carried out on ABI3130xl by the Stanford Protein Nucleic Acid (PAN) Facility.

### Karyotyping

Approximately 2 million cells were carefully collected from the iPSC lines at passage number 20 for analysis using the precise KaryoStat™ assay from Thermo Fisher Scientific.

### Sanger sequencing

PCR primers were designed to identify TTR mutations (Table 2). These primers were employed to amplify the genomic region of the gDNA utilizing the KOD One PCR Master Mix (DiagnoCine, #KMM-101). The PCR reaction was conducted under the following thermal conditions: Initial denaturation at 95 °C for 1 min, followed by 35 cycles of denaturation at 95 °C for 15 sec, annealing at 60 °C for 15 sec, and extension at 72 °C for 30 sec, with a final extension step at 72 °C for 7 min. Sanger sequencing was performed by Azenta.

### Mycoplasma detection

Mycoplasma contamination was assessed at passage 20 using the MycoAlert™ Detection Kit (Lonza #LT07–318) following manufacturer’s instructions.

### Supplementary Material



# SUPPLEMENTAL FILE 1: nihms-1958778.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: PMC10883469

[File available at: automated_output/TTR/20251125_114028/pmc_fulltext/38100909_supplements/PMC10883469]

