# MAIN TEXT

## High prevalence of genetic variants previously associated with LQT syndrome in new exome data

### Abstract

To date, hundreds of variants in 13 genes have been associated with long QT syndrome (LQTS). The prevalence of LQTS is estimated to be between 1:2000 and 1:5000. The knowledge of genetic variation in the general population has until recently been limited, but newly published data from NHLBI GO Exome Sequencing Project (ESP) has provided important knowledge on this topic. We aimed to investigate the prevalence of previously LQTS-associated variants in ESP (5400 individuals), in order to identify possible false-positive LQTS variants. With this aim, we performed a search for previously published LQTS-associated variants in ESP. In addition, a PolyPhen-2 prediction was conducted, and the four most prevalent LQTS-associated variants with significant functional effects present in ESP were genotyped in a second control population. We identified 33 missense variants previously associated with LQTS in ESP. These 33 variants affected 173 alleles and this corresponded to a LQTS prevalence of 1:31 in the ESP population. PolyPhen-2 predicted 30% of the 33 variants present in ESP to be benign compared with 13% among LQTS-associated variants not present in ESP (P=0.019). Genotyping of the four variants KCNH2 P347S; SCN5A: S216L, V1951L; and CAV3 T78M in the control population (n=704) revealed prevalences comparable to those of ESP. Thus, we identified a much higher prevalence of previously LQTS-associated variants than expected in exome data from population studies. Great caution regarding the possible disease causation of some of these variants has to be taken, especially when used for risk stratification in family members.

### Introduction

Congenital long QT syndrome (LQTS) is an inherited cardiac disorder characterised by a prolonged QT interval on the electrocardiogram and sudden cardiac death secondary to cardiac arrhythmias.

The prevalence of LQTS in the general population has been estimated to be between 1:2000 and 1:5000.1 In most cases, LQTS is considered a monogenic disorder, and to date hundreds of variants in 13 genes have been associated with this syndrome.1, 2 However, variants previously reported to be causative in LQTS have subsequently been shown to be rare variants with only a modest disease-modifying effect or even non-pathogenic.3

Until recently, there has only been limited knowledge regarding the genetic variation in the general population, especially with regards to low-frequency variants. This was changed in June 2011, when whole exome data from the NHLBI GO Exome Sequencing Project (ESP) was published.4 In order to identify possible false-positive LQTS variants reported in the literature, we aimed to investigate the prevalence of previously LQTS-associated variants in new ESP exome data and compare the prevalence of these variants with the expected prevalence of LQTS in the same population.

### Methods

In ESP, next-generation sequencing of all protein-coding regions in 5400 persons from different population studies were carried out.4 No clinical data were available on the ESP population, nor at request. By literature search, we found inclusion- and exclusion criteria of 9 out of 12 populations used in ESP. None of these have specifically included persons with channelopathies or other heart diseases and at least two cohorts have excluded such patients.

ESP exome data were systematically searched for previously published missense and nonsense LQTS-associated variants reported in a recent comprehensive review by Hedley et al1 and in data from the The Human Gene Mutation Database (HGMD).5 In addition, we have included the recently published KCNJ5 variant reported by Yang et al,2 in order to include all 13 genes so far associated with LQTS. Due to the lack of data regarding variants positioned in introns and UTR regions in ESP, these could not be included.

The literature was searched for functional data and family co-segregation of all the previously identified LQTS-associated variants also identified in the ESP population. Co-segregation was defined as at least two family members both having the phenotype and the genotype. In addition, we conducted a PolyPhen-2 prediction on all previously reported LQTS-associated missense variants and compared protein damage predictions between LQTS-associated variants identified in ESP with those variants not identified in ESP.6 Nonsense variants were assumed probably damaging. Differences in proportions of PolyPhen-2 predictions between groups were assessed using a Fisher's exact test.

Furthermore, we genotyped (Sanger sequencing, as described previously7) variants with (1) convincing LQTS association in terms of significant functional effects on channel function and (2) a prevalence in ESP European American population high enough (≥7:5295) to have a reasonable chance to be detected in our own two independent healthy control populations (n=704) of Northern European origin with no history of arrhythmias or other cardiac diseases,7, 8 and with available ECGs.

### Results

In all, 33 out of 631 missense and nonsense variants previously associated with LQTS (5.2%) were found in the ESP population. All 33 variants were missense variants and affected 173 alleles. Thirteen of the variants were identified in three or more alleles. No nonsense variants were found. The previously LQTS-associated variants identified in the ESP population are listed in Table 1. On an average, the genes investigated in this study have been screened in 5298 individuals. If we assume that no individuals in the ESP population harbour more than one of the LQTS-associated variants and that all are heterozygote carriers, it corresponds to a LQTS genotype prevalence of 1:31 (173:5298). Thirteen variants were present in three or more alleles corresponding to a LQTS genotype prevalence of 1:37 (145:5298). Six variants associated with acquired/drug-induced LQTS were also identified in ESP; these variants were not included in the present analysis. Literature search regarding functional data and family co-segregation revealed that 19 of the 33 identified LQTS-associated variants have been investigated functionally and 16 of these did have functional effects, but there was a striking lack of data regarding family co-segregation (Table 1 and Supplementary Table).

PolyPhen-2 analysis of the 33 LQTS variants predicted 19 (58%) to be probably damaging, 4 (12%) to be possibly damaging, and 10 (30%) were predicted to be benign (Table 1). Of the remaining 558 missense and 40 nonsense LQTS-associated variants, not present in ESP, 451 (75%) were predicted probably damaging, 69 (12%) possibly damaging, and 78 (13%) benign. This difference in the distribution of the three PolyPhen-2 categories among the two groups was statistically significant both for the overall comparison (P=0.027) and when comparing the categories benign (P=0.016) and probably damaging (P=0.038) independently. As no nonsense variants were found in ESP, a more conservative approach is to compare only the missense variants in the two groups; 558 missense LQTS-associated variants were not present in ESP and of these 411 (74%) were predicted probably damaging, 69 (12%) possibly damaging, and 78 (14%) benign. In this case, only a borderline significant difference was found (P=0.051) for the overall comparison.

Four variants (KCNH2 P347S; SCN5A S216L, V1951L, and CAV3 T78M) were, based on our criteria, genotyped in our control populations (n=704), and this revealed variant prevalences comparable to those found in ESP (Table 2). One of the variant carriers harbouring CAV3 T78M had a QTc interval of 462 ms, but all other variant carriers had normal QTc intervals (QTc≤440 ms) including four other carriers of CAV3 T78M (Table 2).

### Discussion

We identified 33 missense variants out of 631 (5.2%) variants previously associated with LQTS in the ESP data.

These 33 variants correspond to a LQTS genotype prevalence of 1:31 in the ESP. If the prevalence of LQTS is set at 1:2000 (a high estimate), then we would expect only three persons in the ESP population to have LQTS. Thus, we found a very large overrepresentation of previously LQTS-associated variants in the ESP, indicating a very dubious LQTS association for these variants as a group. Of cause, we cannot exclude the possibility that there might be an overrepresentation of LQTS patients in the ESP population, although this seems unlikely, due to the size of the overrepresentation and the fact that ESP is based on population studies mainly representing the general population. Even if we had clinical data on all individuals, we could not make a significantly different conclusion due to the possibility of reduced penetrance of the variants.

To test the possible overrepresentation of LQTS-associated variants in ESP we genotyped the four most prevalent LQTS-associated variants with significant functional effects present in ESP in two cohorts of healthy individuals with available ECGs.7, 8 The four variants were present in our control population with prevalences consistent with those of ESP (Table 2). Thus, the present results indicate that overrepresentation of LQTS-associated variants does not seem to be a problem in ESP, but more importantly the results confirm that at least these four LQTS-associated variants with high prevalence in ESP and significant functional effects are not monogenic causes of LQTS.

In addition, we did a PolyPhen-2 prediction both for the variants found in ESP and the ones we did not find. Of the 33 variants found in ESP, 30% were predicted to be benign compared with 13% among the variants not found in ESP (P=0.016). This further strengthens the argument that these variants are not disease causing.

The majority of the variants have been shown to have functional effects in electrophysiological experiments performed in heterologous cell models. This suggests that they may have some pathological influence, but only a very small fraction of studies showed family co-segregation. Thirteen of the variants were present in three or more alleles in ESP, and thus it seems reasonable to assume that these variants are not monogenic causes of LQTS, as 1:37 are carriers of one of these variants.

The remaining 20 variants are present in just one or two alleles in ESP and an interpretation of the pathogenicity of these variants are therefore much less straightforward.

Taken together, the present results indicate that a large proportion of the 33 variants previously associated with LQTS but now identified in ESP are very unlikely to be monogenic causes of LQTS and are maybe just innocent bystanders. Others are likely to be variants with reduced penetrance, or in some cases disease modifiers. Thus, great caution must be taken regarding the possible disease causation of these variants when found in LQTS (Table 1).

It is noteworthy, that 94.8% of the previously LQTS-associated variants were not present in ESP and thus the pathogenicity of the majority of these variants seems to be corroborated.

Genetic screening has become an important tool in at least family examinations of LQTS. It is for this reason important that variants being associated with LQTS are truly disease causing. A patient with genuine LQTS but a false-positive variant result might have another true disease-causing variant that is not found and this could lead to misdiagnosis of family members with consequences for treatment, clinical advice, and follow-up.

### Conclusion

In recently released exome data, we identified a much higher prevalence of previously LQTS-associated variants than expected based on the prevalence of LQTS in the general population. Our findings suggest that some of the previously associated LQTS variants are false-positive findings and that great caution must be taken regarding the possible disease causation of these variants, especially when used for risk stratification in family members.



# SUPPLEMENTAL FILE 1: ejhg201223a.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: PMC3400735

[File available at: kcnH2_run/KCNH2/20251126_134514/pmc_fulltext/22378279_supplements/PMC3400735]

