# MAIN TEXT

## Genetic polymorphisms of VIP variants in the Tajik ethnic group of northwest China

### Abstract

BackgroundIndividual response to medications varies significantly among different populations, and great progress in understanding the molecular basis of drug action has been made in the past 50 years. The field of pharmacogenomics seeks to elucidate inherited differences in drug disposition and effects. While we know that different populations and ethnic groups are genetically heterogeneous, we have not found any pharmacogenomics information regarding minority groups, such as the Tajik ethnic group in northwest China.ResultsWe genotyped 85 Very Important Pharmacogene (VIP) variants selected from PharmGKB in 100 unrelated, healthy Tajiks from the Xinjiang Uygur Autonomous Region and compared our data with HapMap data from four major populations around the world: Han Chinese (CHB), Japanese in Tokyo (JPT), Utah Residents with Northern and Western European Ancestry (CEU), and Yorubia in Ibadan, Nigeria (YRI). We found that Tajiks differed from CHB, JPT and YRI in 30, 32, and 32 of the selected VIP genotypes respectively (p < 0.005), while differences between Tajiks and CEU were found in only 6 of the genotypes (p < 0.005). Haplotype analysis also demonstrated differences between the Tajiks and the other four populations.ConclusionOur results contribute to the pharmacogenomics database of the Tajik ethnic group and provide a theoretical basis for safer drug administration that may be useful for diagnosing and treating disease in this population.Electronic supplementary materialThe online version of this article (doi:10.1186/s12863-014-0102-y) contains supplementary material, which is available to authorized users.

### Background

To date, pharmacogenomic studies have focused on candidate genes involved in drug pharmacokinetics or pharmacodynamics. Many of these genes contain functional polymorphisms that are obvious pharmacological choices for investigation in appropriate clinical populations [1,2]. For some drugs, genetic information is important to avoid drug toxicity and to optimize response [2,3]. Pharmacogenomic studies are rapidly elucidating the inherited nature of differences in drug disposition and effects, thereby enhancing drug discovery and providing a stronger scientific basis for optimizing drug therapy on an individual basis [4].

Tajiks are an ethnic group with a worldwide population of 15 to 20 million; they live mostly in Tajikistan, Afghanistan, Uzbekistan, and the Xinjiang Uygur Autonomous Region [4]. According to the 2010 census, approximately 51,000 Tajiks live in China, mostly in the Tashkurgan Tajik Autonomous County, which is located in the eastern part of the Pamir Plateau.

The Pharmacogenetics and Pharmacogenomics Knowledge Base (PharmGKB: http://www.pharmgkb.org) is devoted to disseminating primary data and knowledge in pharmacogenetics and pharmacogenomics and has annotated genes that are important for drug response. This information is presented in the form of Very Important Pharmacogene (VIP) summaries, pathway diagrams, and curated literature [5]. It currently contains information for more than 3000 drugs, 3000 diseases, and 26,000 genes with genotyped variants [4].

We systematically genotyped 85 VIP variants selected from PharmGKB VIP in 100 Tajiks from Xinjiang [6]. We compared genotype frequencies and haplotype construction with those in Han Chinese (CHB), Japanese in Tokyo (JPT), Utah Residents with Northern and Western European Ancestry (CEU), and Yorubia in Ibadan, Nigeria (YRI). Our goals were to identify differences and determine their extent and provide a theoretical basis for safer drug administration and better therapeutic treatment in the Tajik population.

### Methods

All participants recruited and genotyped in the present study had at least three generations of paternal ancestry in their ethnic group, and each subject provided written informed consent. The Ethics Committees of Xinjiang University and Northwest University approved the use of human samples in this study.

We recruited a random sample of 100 healthy, unrelated Tajiks (50 males and 50 females) from Tashkurgan Tajik Autonomous County between July and October 2010 using detailed recruitment and exclusion criteria. All of the chosen subjects were Tajik Chinese living in the Xinjiang Uygur Autonomous Region.

We successfully genotyped 85 VIP variants in 37 pharmacogenomic genes in 100 participants. Genomic DNA from whole blood was isolated using the GoldMag® nanoparticles method according to the manufacturer’s protocol, and DNA concentration was measured by spectrometry (DU530 UV/VIS spectrophotometer, Beckman Instruments, Fullerton, CA, USA). We designed primers for amplification and extension reactions using Sequenom MassARRAY Assay Design 3.0 Software [6] and used a Sequenom MassARRAY RS1000 to genotype the single nucleotide polymorphisms (SNPs) using the protocol recommended by the manufacturer. Sequenom Typer 4.0 Software was used for data management and analysis [6,7].

Statistical analyses were performed using Microsoft Excel (Redmond, WA, USA) and SPSS 16.0 statistical package (SPSS, Chicago, IL, USA). All p values in this study were two-sided, and p ≤ 0.005 after Bonferroni correction was considered the statistical significance threshold [8]. We calculated and compared the genotype frequencies of Tajiks and four other populations (CHB, JPT, CEU, and YRI) using chi-squared tests [9]. We used the Haploview software package (version 4.2) for analysis of linkage disequilibrium (LD), haplotype construction, and genetic associations at polymorphic loci [10-12]. Our method excluded SNPs with minor allele frequency < 0.001 for SNPs with lower frequencies that have little power to detect LD. We also ignored SNPs with Hardy-Weinberg equilibrium (HWE) p values < 0.001 for their small probability that their deviation from HWE could be explained by chance. The D’ values on the square is a measure of the LD extent for each pair of SNPs, squares in red without D’ values indicate the two sites are in complete LD (D’ = 1). We constructed haplotypes using the common sites of the selected SNPs and sites downloaded from HapMap for the VDR gene and derived the haplotype frequencies in all five populations.

### Ethics statement

All participants recruited and genotyped in the present study had at least three generations of paternal ancestry in their ethnic group, and each subject provided written informed consent. The Ethics Committees of Xinjiang University and Northwest University approved the use of human samples in this study.

### Study participants

We recruited a random sample of 100 healthy, unrelated Tajiks (50 males and 50 females) from Tashkurgan Tajik Autonomous County between July and October 2010 using detailed recruitment and exclusion criteria. All of the chosen subjects were Tajik Chinese living in the Xinjiang Uygur Autonomous Region.

### Polymerase chain reaction (PCR) and DNA sequencing

We successfully genotyped 85 VIP variants in 37 pharmacogenomic genes in 100 participants. Genomic DNA from whole blood was isolated using the GoldMag® nanoparticles method according to the manufacturer’s protocol, and DNA concentration was measured by spectrometry (DU530 UV/VIS spectrophotometer, Beckman Instruments, Fullerton, CA, USA). We designed primers for amplification and extension reactions using Sequenom MassARRAY Assay Design 3.0 Software [6] and used a Sequenom MassARRAY RS1000 to genotype the single nucleotide polymorphisms (SNPs) using the protocol recommended by the manufacturer. Sequenom Typer 4.0 Software was used for data management and analysis [6,7].

### Data analysis

Statistical analyses were performed using Microsoft Excel (Redmond, WA, USA) and SPSS 16.0 statistical package (SPSS, Chicago, IL, USA). All p values in this study were two-sided, and p ≤ 0.005 after Bonferroni correction was considered the statistical significance threshold [8]. We calculated and compared the genotype frequencies of Tajiks and four other populations (CHB, JPT, CEU, and YRI) using chi-squared tests [9]. We used the Haploview software package (version 4.2) for analysis of linkage disequilibrium (LD), haplotype construction, and genetic associations at polymorphic loci [10-12]. Our method excluded SNPs with minor allele frequency < 0.001 for SNPs with lower frequencies that have little power to detect LD. We also ignored SNPs with Hardy-Weinberg equilibrium (HWE) p values < 0.001 for their small probability that their deviation from HWE could be explained by chance. The D’ values on the square is a measure of the LD extent for each pair of SNPs, squares in red without D’ values indicate the two sites are in complete LD (D’ = 1). We constructed haplotypes using the common sites of the selected SNPs and sites downloaded from HapMap for the VDR gene and derived the haplotype frequencies in all five populations.

### Results

We successfully sequenced 85 VIP pharmacogenomic variant genotypes from 100 Tajiks. The PCR primers used for the selected variants are listed in Additional file 1. Table 1 lists the basic characteristics of the selected variants, including gene name, chromosome number and position, and their allele frequencies in Tajiks.Table 1
Basic characteristics of the selected variants

SNP ID

Genes

Chromosome

Position

Allele

Allele frequencies

A

B

A(%)

B(%)
rs1801131MTHFR111854476CA35.065.0rs1801133MTHFR111856378TC19.280.8rs890293CYP2J2160392494GT48.551.5rs3918290DPYD197915614G/1000rs6025F51169519049CA1000rs20417PTGS21186650320GC97.03.0rs689466PTGS21186650750AG85.414.7rs4124874UGT1A102234665659CA42.857.2rs10929302UGT1A102234665782GA73.526.5rs4148323UGT1A102234669144AG3.596.5rs7626962SCN5A338620907G/1000rs1805124SCN5A338645420GA29.071.0rs6791924SCN5A338674699G/1000rs3814055NR1I23119500034CT58.042.0rs2046934P2RY123151057642TC90.010.0rs1065776P2RY13152553628TC6.193.9rs701265P2RY13152554357GA20.080.0rs975833ADH1A4100201739GC74.225.8rs2066702ADH1B4100229017CT97.52.5rs1229984ADH1B4100239319GA70.529.5rs698ADH1C4100260789AG67.033.0rs17244841HMGCR574607099A/1000rs3846662HMGCR574615328TC48.551.5rs17238540HMGCR574619742T/1000rs1042713ADRB25148206440GA60.539.5rs1042714ADRB25148206473GC34.066.0rs1800888ADRB25148206885CT98.02.0rs1142345TPMT618130918GA0100rs1800460TPMT618139228AG0100rs2066853AHR717379110GA82.517.5rs1045642ABCB1787138645TC57.142.9rs2032582ABCB1787160617GT42.757.3rs2032582ABCB1787160617GA86.413.6rs2032582ABCB1787160617TA92.77.4rs1128503ABCB1787179601TC58.141.9rs10264272CYP3A5799262835C/1000rs776746CYP3A5799270539GA89.510.5rs4986913CYP3A4799358459CT99.01.0rs4986910CYP3A4799358524T/1000rs4986909CYP3A4799359670C/1000rs12721634CYP3A4799381661T/1000rs2740574CYP3A4799382096AG98.51.5rs3815459KCNH27150644394AG40.559.5rs36210421KCNH27150644428GT99.01.0rs12720441KCNH27150647304C/1000rs3807375KCNH27150667210AG43.057.0rs4986893CYP2C191096540410G/1000rs4244285CYP2C191096541616GA92.57.5rs1799853CYP2C91096702047CT1000rs1801252ADRB110115804036GA20.279.8rs1801253ADRB110115805055CG79.820.2rs5219KCNJ111117409572CT56.143.9rs1695GSTP11167352689AG77.023.0rs1138272GSTP11167353579TC9.091.0rs1800497DRD211113270828TC17.982.1rs6277DRD211113283459GA61.538.5rs4149056SLCO1B11221331549TC90.59.5rs7975232VDR1248238837CA49.051.0rs1544410VDR1248239835GA66.034.0rs2239185VDR1248244559TC51.049.0rs1540339VDR1248257326GA67.232.8rs2239179VDR1248257766AG56.543.5rs3782905VDR1248266167CG70.030.0rs2228570VDR1248272895TC34.565.5rs10735810VDR1248272895CT66.533.5rs11568820VDR1248302545GA77.322.7rs1801030SULT1A21628617485A/1000rs3760091SULT1A11628620800CG54.145.9rs7294VKORC11631102321CT67.033.0rs9934438VKORC11631104878GA50.549.5rs28399454CYP2A61941351267G/1000rs28399444CYP2A61941354190A/1000rs1801272CYP2A61941354533T/1000rs28399433CYP2A61941356379GT10.589.5rs3745274CYP2B61941512841GT64.036.0rs28399499CYP2B61941518221T/1000rs3211371CYP2B61941522715CT50.050.0rs12659SLC19A12146951555CT56.643.4rs1051266SLC19A12146957794GA55.744.3rs1131596SLC19A12146957915TC60.639.4rs4680COMT2219951271AG53.546.5rs59421388CYP2D62242523610C/1000rs28371725CYP2D62242523805GA90.010.0rs16947CYP2D62242523943GA74.125.9rs5030656CYP2D62242524175AAGdelAAG99.50.5rs61736512CYP2D62242525134C/1000rs28371706CYP2D62242525772CT99.01.0

Basic characteristics of the selected variants

Table 2 lists the genotype frequencies in Tajiks and identifies significant variants in Tajiks compared with the other four populations (p < 0.005), all variant data are shown in Additional file 2. We also categorized the genes into different families and phases related to pharmacogenomics, the statistically significant values are shown in red (p < 0.05). We found that Tajiks differed from CHB, JPT, and YRI in 30, 32, and 32 selected VIP genotypes, respectively. These genes encode phase I drug metabolic enzymes (VCORC1, MTHFR, and CYP3A5), a phase II drug metabolic enzymes (COMT), and transporters, channel proteins, and receptors (e.g., ADRB1, KCNH2, and VDR, respectively). However, the difference between Tajiks and CEU was much smaller; just six SNP genotypes were different, and these were randomly distributed on genes such as CYP2C9, which encodes a phase I enzyme. For genes such as ADH1B and PTGS2, we observed differences between Tajiks and the other four populations.Table 2
Genotype frequencies in Tajiks compared with four other populations

SNP ID

Gene

Category

Allele

Tajik genotype frequencies

p
values against four populations (after Bonferroni correction)

Family

Phase

A

B

AA(%)

AB(%)

BB(%)

CHB

JPT

CEU

YRI
rs1045642ABCB1ABC transportersothersTC34.345.520.28.49E-033.42E-023.10E-012.17E-18rs1128503ABCB1ABC transportersothersTC32.351.516.23.09E-028.73E-011.69E-023.03E-19rs2032582ABCB1ABC transportersothersGT1849.432.69.02E-016.07E-018.24E-02-rs975833ADH1Aalcohol dehydrogenasephase IGC55.637.47.14.86E-163.45E-146.46E-015.79E-01rs1229984ADH1Balcohol dehydrogenasephase IGA484573.76E-127.23E-119.18E-111.26E-10rs2066702ADH1Balcohol dehydrogenasephase ICT955.102.97E-012.97E-011.94E-011.61E-11rs698ADH1Calcohol dehydrogenasephase IAG44.345.410.31.21E-093.22E-091.24E-027.40E-11rs1801252ADRB1adrenergic receptorsothersGA334.362.61.22E-051.52E-05-2.53E-06rs1801253ADRB1adrenergic receptorsothersCG63.632.346.12E-013.88E-015.87E-024.12E-04rs1042713ADRB2adrenergic receptorsothersGA3649154.49E-035.67E-017.34E-016.05E-02rs1042714ADRB2adrenergic receptorsothersGC1440461.26E-033.39E-052.70E-029.85E-03rs2066853AHRAHRothersGA692741.18E-057.65E-086.87E-029.06E-08rs4680COMTCOMTphase IIAG3145249.95E-051.56E-055.25E-013.15E-05rs28399454CYP2A6cytochrome P450phase IG/10000--1.00E + 007.81E-07rs3745274CYP2B6cytochrome P450phase IGT4636181.07E-032.03E-035.17E-021.50E-01rs28399499CYP2B6cytochrome P450phase IT/10000---2.04E-06rs4244285CYP2C19cytochrome P450phase IGA851501.00E-031.56E-054.07E-028.50E-02rs1799853CYP2C9cytochrome P450phase ICT10000--6.25E-05-rs776746CYP3A5cytochrome P450phase IGA811722.78E-058.74E-052.58E-021.23E-34rs10264272CYP3A5cytochrome P450phase IC/100001.00E + 001.00E + 00-8.95E-09rs6277DRD2G-protein-coupled receptorothersGA3847157.41E-081.01E-071.98E-026.51E-11rs1800497DRD2G-protein-coupled receptorothersTC3.129.667.48.59E-061.43E-057.19E-012.25E-06rs1695GSTP1glutathione S-transferasephase IIAG593655.70E-011.96E-032.32E-041.50E-03rs1138272GSTP1glutathione S-transferasephase IITC018823.00E-032.00E-039.22E-012.28E-03rs3846662HMGCRHMGCRphase ITC1861212.49E-017.26E-012.12E-022.79E-24rs3807375KCNH2eagothersAG1948336.73E-078.34E-124.02E-011.13E-11rs3815459KCNH2eagothersAG1649356.04E-062.69E-09-6.88E-01rs1801131MTHFRmethylenetetrahydrofolate reductasephase ICA756379.07E-035.46E-042.32E-017.36E-09rs1801133MTHFRmethylenetetrahydrofolate reductasephase ITC430.365.73.13E-071.21E-031.92E-029.34E-03rs3814055NR1I2nuclear receptorothersCT3056148.60E-032.15E-031.00E-011.32E-03rs701265P2RY1G-protein coupled receptorothersGA432647.61E-023.04E-019.33E-012.96E-25rs2046934P2RY12G-protein coupled receptorothersTC821625.69E-024.97E-024.49E-036.23E-03rs20417PTGS2nuclear receptorothersGC97035.43E-041.57E-033.65E-074.82E-17rs689466PTGS2nuclear receptorothersAG73.723.233.68E-114.34E-077.13E-011.12E-01rs1805124SCN5Asodium channel geneothersGA940511.23E-047.59E-047.80E-034.96E-01rs6791924SCN5Asodium channel geneothersG/10000---2.00E-03rs7626962SCN5Asodium channel geneothersG/10000---1.24E-03rs1051266SLC19A1solute carrierothersGA29.951.618.63.65E-018.28E-028.36E-012.83E-06rs4149056SLCO1B1solute carrierothersTC821712.30E-017.68E-011.47E-012.66E-04rs4124874UGT1A10UDP-glucuronosyltransferasephase IICA14.456.728.91.64E-026.88E-023.31E-012.73E-23rs4148323UGT1A10UDP-glucuronosyltransferasephase IIAG07937.23E-083.22E-039.00E-029.00E-02rs10929302UGT1A10UDP-glucuronosyltransferasephase IIGA534162.58E-033.31E-039.38E-014.91E-02rs1540339VDRnuclear receptorothersGA47.539.413.11.60E-101.42E-117.78E-011.80E-02rs1544410VDRnuclear receptorothersGA405283.05E-122.03E-061.75E-022.50E-01rs2239179VDRnuclear receptorothersAG3151184.01E-046.14E-053.63E-028.58E-03rs2239185VDRnuclear receptorothersTC2258202.86E-041.55E-01-4.37E-01rs3782905VDRnuclear receptorothersCG464861.20E-016.72E-043.92E-017.74E-02rs7975232VDRnuclear receptorothersCA2058229.30E-052.13E-031.92E-022.44E-02rs10735810VDRnuclear receptorothersCT48.436.315.48.17E-028.74E-012.72E-014.64E-03rs11568820VDRnuclear receptorothersGA60.633.36.11.12E-044.25E-056.86E-018.35E-38rs7294VKORC1VKORC1phase ICT4446108.00E-101.09E-067.69E-011.59E-04rs9934438VKORC1VKORC1phase IGA2649251.29E-174.64E-141.01E-019.39E-25

Genotype frequencies in Tajiks compared with four other populations

We counted the variants in each family, excluding those that belonged to none of the families or were not significantly different between Tajiks and the other four populations. The remaining 71 sites belonged to 26 genes in 12 families (Table 3). We found that the difference between Tajiks and CEU existed in only one site in the nuclear receptor family and 0 site in adrenergic receptors family respectively. However, in the nuclear receptor family, Tajiks differed from CHB, JPT, and YRI in 66.7%, 75%, and 33.3% of selected sites, respectively. In the adrenergic receptor family, Tajiks differed from CHB, JPT, and YRI in 60%, 40%, and 40% of selected sites, respectively. For genes in ATP-binding cassette (ABC) transporters, Tajiks differed from YRI in 66.7% of the selected sites, but there was no difference between Tajiks and CHB, JPT, CEU.Table 3
Numbers and frequencies of significant variants

Family

Variants (n)

Significant variants, n (%)

CHB

JPT

CEU

YRI
Adrenergic receptors53 (60.0)2 (40.0)0 (0)2 (40.0)Alcohol dehydrogenase43 (75.0)3 (75.0)1 (25.0)3 (75.0)ATP-binding cassette (ABC) transporters30 (0)0 (0)0 (0)2 (66.7)Cytochrome P450243 (24.9)3 (24.9)1 (8.3)4 (16.7)Eag42 (50.0)2 (50.0)0 (0)1 (25.0)Glutathione S-transferase21 (50.0)2 (100)1 (50.0)2 (100)G-protein coupled receptor52 (40.0)2 (40.0)1 (20.0)3 (60.0)Methylenetetrahydrofolate reductase21 (50.0)2 (100.0)0 (0)1 (50.0)Nuclear receptor128 (66.7)9 (75.0)1 (8.3)0)4 (33.3)Sodium channel gene31 (33.3)1 (33.3)0 (0)2 (66.7)Solute carrier40 (0)0 (0)0 (0)2 (50.0)UDP-glucuronosyltransferase32 (33.7)2 (66.7)0 (0)1 (33.3)

Numbers and frequencies of significant variants

We performed LD analysis using Haploview to define blocks and haplotypes. Using the common sites of our study and those from HapMap in the VDR gene, we identified two LD blocks in Tajiks, JPT, and CEU and one LD block in CHB and YRI (Figure 1). The block identified in all five populations spans 0.4 kb and consists of two complete LD markers (rs1540339 and rs2239179) with a D’ value equal to 1. The block identified in Tajiks, JPT, and CEU spans 0.9 kb and also consists of two complete LD markers (rs7975232 and rs1544410) with a D’ value equal to 1.Figure 1
Linkage disequilibrium (LD) analysis of
VDR
in five populations. LD is displayed by standard color schemes, with bright red for very strong LD (LOD > 2, D’ = 1), light red (LOD > 2, D’ < 1) and blue (LOD < 2, D’ = 1) for intermediate LD, and white (LOD < 2, D’ < 1) for no LD. A. Tajiks, B. CHB, C. JPT, D. CEU, E. YRI.

Linkage disequilibrium (LD) analysis of
VDR
in five populations. LD is displayed by standard color schemes, with bright red for very strong LD (LOD > 2, D’ = 1), light red (LOD > 2, D’ < 1) and blue (LOD < 2, D’ = 1) for intermediate LD, and white (LOD < 2, D’ < 1) for no LD. A. Tajiks, B. CHB, C. JPT, D. CEU, E. YRI.

Haplotype analysis results are shown in Figure 2. For the common block comprised of rs1540339 and rs2239179, three kinds of haplotypes were identified in all five populations, but they differed in frequency. Three colors of bars indicate the three kinds of haplotypes. The highest and lowest frequencies of haplotype “AA” were found in JPT (73.8%) and YRI (20.0%). The highest and lowest frequencies of haplotype “GG” were observed in CEU (47.0%) and JPT (22.1%). The highest and lowest frequencies of haplotype “GA” were found in YRI (50.4%) and JPT (4.1%). The haplotype constitutions and frequencies show that there are relatively minimal differences between Tajik and CEU, CHB, and JPT, whereas the differences between YRI and the other four populations seem obvious. These findings are in accordance with the results shown in Table 3.Figure 2
Haplotype analysis results of rs1540339 and rs2239179 in
VDR
.

Haplotype analysis results of rs1540339 and rs2239179 in
VDR
.

### Discussion

With the rapid development of pharmacogenetics, serious attention has been given to interethnic and interracial differences in drug responses [13]. Here, we genotyped 85 variants related to pharmacogenomics in the Tajik ethnic group for the first time and compared the results with other ethnic populations around the world. We found that 30, 32, 32, and 6 VIP variants differed from CHB, JPT, YRI, and CEU respectively (p < 0.005). These findings corroborate the current opinion that polymorphisms with varying frequencies occur among different populations.

Vitamin D receptor (VDR) is a gene whose function has been widely reported. Epithelial cells convert the primary circulating form of vitamin D to its active form, which binds VDR to regulate a variety of genes that keep cellular proliferation and differentiation within normal ranges to prevent malignant transformation [14]. That is to say, the active form of vitamin D can induce apoptosis and prevent angiogenesis by binding VDR, which reduces the survival potential of malignant cells. Studies have demonstrated that rs10735810 and rs1544410 SNPs in VDR might modulate the risk of breast, skin, and prostate cancers, as well as other forms [15,16]. An Italian study reported that GA and AA rs1544410 genotypes were associated with decreased cutaneous malignant melanoma (CMM) risk (odds ratio = 0.78 and 0.75, respectively) compared with the GG genotype [16]. A study in Japan found that head and neck squamous cell carcinoma patients with the TT rs10735810 genotype was associated with poor progression-free survival compared with CC or CT genotype patients (log-rank test, p = 0.0004; adjusted hazard ratio, 3.03; 95% confidence interval, 1.62 to 5.67; p = 0.001), and the A-T-G (rs11568820-rs10735810-rs7976091) haplotype showed a significant association with a higher progression rate (p = 0.02). [14] We found that the GA and AA genotype frequencies of rs1544410 in Tajiks were as much as 52% and 8% respectively, which is different from those in CHB and JPT (data not shown), suggesting that Tajiks may have decreased susceptibility to CMM.

The gene alcohol dehydrogenase 1B (ADH1B) produces a key protein for alcohol metabolism that determines blood acetaldehyde concentrations after drinking [17]. This member of the alcohol dehydrogenase family also metabolizes a wide variety of substrates besides ethanol, including retinol, other aliphatic alcohols, hydroxysteroids, and lipid peroxidation products. The minor allele “A” of rs1229984 encodes a super-active allozyme that is reportedly associated with lower rates of alcohol dependence in numerous association studies, and its frequency varies widely across different populations. It is 69% (19-91%) in normal Asian normal populations, 5.5% (1-43%) in normal European populations, and just 3% (2-7%) in normal Mexican populations [18]. Other studies have shown that rs1229984 may influence alcohol consumption behavior and is associated with upper aerodigestive (UADT) cancers [19-24]. A genome-wide association study found that the “A” allele of rs1229984 was associated with decreased UADT risk (p = 7 × 10−9) [19]. The data in our study is in accordance with previous findings; we found that the “A” allele frequency of rs1544410 in Tajiks was 29.5%, which was significantly different (p < 0.05) from 76.67%, 73.86%, 0%, and 0% in CHB, JPT, CEU, and YRI respectively, suggesting that Tajiks have an intermediate susceptibility to UADT cancer.

The catechol-o-methyltransferase gene (COMT) is responsible for eliminating dopamine from the synaptic cleft in the prefrontal cortex (PFC) [25]. Variations in the COMT gene exert complex effects on susceptibility to depression through various intermediate phenotypes, such as impulsivity and executive function [26]. The common functional COMT polymorphism rs4680 has been shown to affect enzyme activity and, consequently, intrasynaptic dopamine content. The “G” allele is associated with 40% higher enzymatic activity in the human brain compared to the “A” allele, leading to more efficient elimination of dopamine from the synaptic cleft; therefore, the GG genotype is associated with reduced synaptic dopamine in the PFC, and in turn, more active striatal dopamine neurotransmission [25,27-29]. A study in northern Italy reported an association between the GG genotype and the risks of Alzheimer’s disease (AD) and its precursor, mild cognitive impairment (MCI) [30]. The GG genotype frequency in our study was just 24% in Tajiks, compared with 51.2%, 50%, and 46% in CHB, JPT, and YRI respectively (p < 0.05). This suggests that Tajiks may be less vulnerable to diseases related to dopamine content, including AD and MCI.

Our study also found significant differences in genotype frequencies between Tajiks and other populations in genes such as DRD2 and F5. Polymorphisms in these genes have been shown to be associated with dyskinesia induced by levodopa therapy in Parkinson’s disease patients and coronary artery disease, respectively [31,32].

The Tajiks speak a western Indo-Iranian language and their presence in China dates to the 10th-century Muslim invasion, suggesting they are descendants of eastern Indo-Iranian speakers [33]. This may explain the smaller differences between Tajiks and CEU compared to other three populations we investigated.

However, intrinsic limitations still exist in our study. Our sample size is relatively not big enough, thus further investigation related to pharmacogenomics gene polymorphisms in a larger Tajik population is necessary to ascertain the results obtained in the current study.

### Conclusions

These results provide the first pharmacogenomics information in Tajiks and illustrate the difference of selected genes between Tajiks and four other populations. Present-day China is a nation with 56 distinct ethnic groups. Our study provides a theoretical basis for safer drug administration and better therapeutic treatments in this unique population, and may also be applied in the diagnosis and prognosis of specific diseases in Tajiks.



# SUPPLEMENTAL FILE 1: 12863_2014_Article_102.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)