# MAIN TEXT

## Genotype and clinical characteristics of congenital long QT syndrome in Thailand

### Abstract

BackgroundCongenital long QT syndrome (LQTS) is an inheritable arrhythmic disorder which is linked to at least 17 genes. The clinical characteristics and genetic mutations may be variable among different population groups and they have not yet been studied in Thai population.MethodsClinical characteristics were retrospectively reviewed from children and young adults with congenital long QT syndrome whose blood samples were sent for genotyping during 1998–2017. Sangers sequencing was used to sequentially identify KCNQ1 or KCNH2 genetic variants. Whole exome sequencing (WES) was used to identify variants in all other known LQTS genes.ResultsOf the 20 subjects (17 families), 45% were male, mean QTc was 550.3 ± 68.8 msec (range 470–731 msec) and total Schwartz's score was 5.6 ± 1.2 points (range 3–8 points). Fifty percent of patients had events at rest, 30% had symptoms after adrenergic mediated events, and 20% were asymptomatic. We discovered pathogenic and likely pathogenic genetic variants in KCNQ1, KCNH2, and SCN5A in 6 (35%), 4 (24%), and 2 (12%) families, respectively. One additional patient had variance of unknown significance (VUS) in KCNH2 and another one in ANK2. No pathogenic genetic variant was found in 3 patients (18%). Most patients received beta-blocker and 9 (45%) had ICD implanted. LQT1 patients were either asymptomatic or had stress-induced arrhythmia. Most of the LQT2 and LQT3 patients developed symptoms at rest or during sleep.ConclusionsOur patients with LQTS were mostly symptomatic at presentation. The genetic mutations were predominantly in LQT1, LQT2, and LQT3 genes.

### Introduction

Congenital long QT syndrome (LQTS) is an inheritable cardiac arrhythmia disorder characterized by prolonged QT interval on surface ECG, ventricular arrhythmia and sudden cardiac death. Abnormal ventricular repolarization in these patients predisposes them to malignant cardiac arrhythmia such as torsades de pointes and ventricular fibrillation. The prevalence of long QT syndrome is thought to be as high as 1 in 2500 population [1], the figure which many believed to be underestimated [2].

Currently, at least 17 genes were linked to LQTS [3,4]. Seventy-five percent of the patients possess pathogenic mutation in one of the three genes: KCNQ1, KCNH2, SCN5A [5]. Those with mutation in each of these particular gene have different clinical characteristics and are associated with different triggering events for arrhythmia as well as response to treatment [6]. There were reports on various gene mutation frequency in Asian countries [[7], [8], [9], [10], [11], [12]]. From a previous study in Thai children, the ratio of LQTS patients who had cardiac events at rest or sleep appeared to be higher than those found in other Asian countries [13]. However, genetic testing results was not previously examined in Thai population.

Our study aimed to review the clinical characteristics and genetic mutation of LQTS in Thai patients diagnosed between 1998 and 2017.

### Materials and methods

The study population included all patients diagnosed with congenital long QT syndrome in whom DNA sample were available from January 1st, 1998 to June 30th, 2017. Outpatient and inpatient medical records of the children and young adults with the diagnosis of long QT syndrome were reviewed from the database of samples that were sent to King Chulalongkorn Memorial Hospital for LQTS genotyping. The study has been approved by the Ethics Committee of the Faculty of Medicine, Chulalongkorn University. Written informed consents were mandatorily obtained from all patients and/or their guardians.

Demographic data, family history, clinical presentation, echocardiogram findings and others special cardiac investigations, treatments and outcomes were collected.

The diagnosis of congenital long QT syndrome was made in accordance to the 2013 HRS/EHRS/APHRS Consensus Guideline (HRS/EHRS/APHRS consensus) [14]. Only patients with congenital LQTS were recruited. Those with QTc interval prolongation from other identifiable causes were excluded. QT interval was measured in lead II or V5 and corrected for differences in heart rate by Bazett's formula.

Blood samples (EDTA) were collected from every patient and their family members after informed consents were obtained. Genomic DNA was isolated from peripheral white blood cells through phenol/chloroform extraction.

Sequential screenings of genetic variants in KCNQ1 and KCNH2 were first done through PCR amplification and Sangers sequencing according to patient's clinical characteristics. If pathogenic mutation was not found in these 2 genes, the other genes were then examined with whole exome sequencing (WES). Primers used in PCR amplification were located in the intronic region to cover all exons of KCNQ1 and KCNH2 genes. The amplicons were visualized on 2% agarose gel, excised and purified. Purified DNA was sequenced at 1st BASE Laboratories, Selangor Darul Ehsan, Malaysia. All sequences were aligned with their reference gene obtained from GenBank, under accession numbers NM_000218 (KCNQ1) and NM_000238 (KCNH2), through sequencer software package (Gene Codes Corporation, Ann Arbor, Michigan, USA).

Genomic DNA samples were sent to Macrogen, Inc. (Seoul, Korea) for whole exome sequencing. DNA samples were enriched by SureSelect Human All Exon V5 (Agilent Technologies, Santa Clara, Calif., USA) and were sequenced onto Hiseq 4000 (Illumina). The raw data per exome were mapped to the human reference genome hg19 using BWA. Variant calling was performed using GATK with HaplotypeCaller. Finally, SNVs and Indels were annotated by using SnpEff and annotation database that were dbpSNP 142, 1000 Genome, ClinVar and ESP.

Analysis of protein coding and flanking regions of the following genes was done: KCNQ1, KCNH2, SCN5A, ANK2, KCNE1, KCNE2, KCNJ2, CACNA1C, CAV3, SCN4B, AKAP9, SNTA1, KCNJ5, CALM1, CALM2, CALM3, TRDN. Filtering for possible pathogenic mutations (rare variants) were done by the following criteria: 1) Non-synonymous variants in either protein coding regions or splice sites, and 2) Minor allele frequency (MAF) of <0.1% on 1000 Genome project (http://www.1000genomes.org). Rare variants previously reported to be the cause of LQTS were classified as pathogenic. Rare variants with no previous report of pathogenicity were further classified as likely pathogenic, likely benign and variants of unknown significance (VUS) based on the American College of Medical Genetics and Genomics (ACMG) and Association of Molecular Pathology (AMP) 2015 guideline [15]. Only plausible pathogenic or pathogenic variants are reported as the cause of LQTS in this cohort. We also confirmed the MAF of all likely pathogenic and pathogenic variants against our in-house exome sequencing data (719 samples at the time of the study).

Continuous data are expressed as mean ± SD and nominal data are expressed as percentage.

### Data collection

Demographic data, family history, clinical presentation, echocardiogram findings and others special cardiac investigations, treatments and outcomes were collected.

### Diagnosis

The diagnosis of congenital long QT syndrome was made in accordance to the 2013 HRS/EHRS/APHRS Consensus Guideline (HRS/EHRS/APHRS consensus) [14]. Only patients with congenital LQTS were recruited. Those with QTc interval prolongation from other identifiable causes were excluded. QT interval was measured in lead II or V5 and corrected for differences in heart rate by Bazett's formula.

### Blood samples collection and DNA extraction

Blood samples (EDTA) were collected from every patient and their family members after informed consents were obtained. Genomic DNA was isolated from peripheral white blood cells through phenol/chloroform extraction.

### Mutation analysis

Sequential screenings of genetic variants in KCNQ1 and KCNH2 were first done through PCR amplification and Sangers sequencing according to patient's clinical characteristics. If pathogenic mutation was not found in these 2 genes, the other genes were then examined with whole exome sequencing (WES). Primers used in PCR amplification were located in the intronic region to cover all exons of KCNQ1 and KCNH2 genes. The amplicons were visualized on 2% agarose gel, excised and purified. Purified DNA was sequenced at 1st BASE Laboratories, Selangor Darul Ehsan, Malaysia. All sequences were aligned with their reference gene obtained from GenBank, under accession numbers NM_000218 (KCNQ1) and NM_000238 (KCNH2), through sequencer software package (Gene Codes Corporation, Ann Arbor, Michigan, USA).

### Whole exome sequencing

Genomic DNA samples were sent to Macrogen, Inc. (Seoul, Korea) for whole exome sequencing. DNA samples were enriched by SureSelect Human All Exon V5 (Agilent Technologies, Santa Clara, Calif., USA) and were sequenced onto Hiseq 4000 (Illumina). The raw data per exome were mapped to the human reference genome hg19 using BWA. Variant calling was performed using GATK with HaplotypeCaller. Finally, SNVs and Indels were annotated by using SnpEff and annotation database that were dbpSNP 142, 1000 Genome, ClinVar and ESP.

Analysis of protein coding and flanking regions of the following genes was done: KCNQ1, KCNH2, SCN5A, ANK2, KCNE1, KCNE2, KCNJ2, CACNA1C, CAV3, SCN4B, AKAP9, SNTA1, KCNJ5, CALM1, CALM2, CALM3, TRDN. Filtering for possible pathogenic mutations (rare variants) were done by the following criteria: 1) Non-synonymous variants in either protein coding regions or splice sites, and 2) Minor allele frequency (MAF) of <0.1% on 1000 Genome project (http://www.1000genomes.org). Rare variants previously reported to be the cause of LQTS were classified as pathogenic. Rare variants with no previous report of pathogenicity were further classified as likely pathogenic, likely benign and variants of unknown significance (VUS) based on the American College of Medical Genetics and Genomics (ACMG) and Association of Molecular Pathology (AMP) 2015 guideline [15]. Only plausible pathogenic or pathogenic variants are reported as the cause of LQTS in this cohort. We also confirmed the MAF of all likely pathogenic and pathogenic variants against our in-house exome sequencing data (719 samples at the time of the study).

### Statistical analysis

Continuous data are expressed as mean ± SD and nominal data are expressed as percentage.

### Results

A total of 30 patients in 27 families diagnosed with congenital long QT syndrome were identified. Due to deaths and losses to follow up, only 20 patients from 17 families were available for genetic testing at the time of this study. Among the 10 missing patients, six were lost to follow up (3 patients developed symptoms during exercise, 2 patients developed symptoms during sleep, and 1 patient was asymptomatic at the time of presentation); and 4 patients had died (3 died shortly after presenting with cardiac arrest during exercise, and 1 presented with aborted sudden cardiac death during sleep were dead 6.8 years after ICD implantation, he was lost to follow up for 4.3 years before death). Five out of the 6 patients who were lost to follow up are still alive according to the Thai citizen registration record.

Excluding the 10 patients who were lost to follow-up or died, 20 patients (17 families) were included to this study. The clinical characteristics of the study subjects are summarized in Table 1. Nine patients (45%) were male. Their mean age at diagnosis was 7.8 ± 6.2 years (range 1 day–28 years). Total Schwartz's score was 5.6 ± 1.2 points (range 3–8 points). All except one patient were categorized as high probability of LQTS according to Schwartz's score (score ≥ 4 points). Most patients had no co-morbid condition. Three patients (15%) had congenital abnormalities; 2 with congenital sensorineural hearing loss, and 1 with double aortic arch. Family history of definite LQTS was found in 5 (25%), and family history of cardiac arrest or unexplained sudden cardiac death was reported in 7 patients (35%).Table 1Clinical characteristics of all Long QT syndrome patients (N = 20).Table 1Total (N = 20)N%Male945Age at Diagnosis (years)Mean (±SD)7.8 (±6.2)Range0–28Congenital anomalies315 Double aortic arch15 SNHL210Family history Definite LQTS525 SCD735 SNHL210Total Schwartz's score (points)Mean (±SD)5.6 (±1.2)Range3–8Electrocardiographic characteristics Torsade de pointes420 T wave alternans15 Notched T wave1050 Bradycardia for age525 QTc (msec)Mean (±SD)550.3 (±68.8)Range470–731Treatments Beta blocker1995 ICD945Current Status Alive1995 Death15SNHL = sensorineural hearing loss, LQTS = Long QT syndrome, SCD = sudden cardiac death, QTc = corrected QT interval, msec = millisecond, bpm = beat per minute, HR = heart rate, ICD = implantable cardioverter-defibrillator.

Clinical characteristics of all Long QT syndrome patients (N = 20).

SNHL = sensorineural hearing loss, LQTS = Long QT syndrome, SCD = sudden cardiac death, QTc = corrected QT interval, msec = millisecond, bpm = beat per minute, HR = heart rate, ICD = implantable cardioverter-defibrillator.

The mean value of the longest QTc interval in each patient was 550.3 ± 68.8 msec (range 470–731 msec). Five patients (25%) had bradycardia for age. Four patients (20%) presented with torsades de pointes and/or ventricular fibrillation (TdP/VF). Five patients had cardiac arrest with no ECG documentation at the time of the arrest.

Patients were divided into 3 groups; 10 patients (50%) developed cardiac symptoms at rest or sleep, and 6 patients (30%) developed cardiac events during exercise/stress/startle (adrenergically mediated cardiac events). Four patients (20%) were asymptomatic at presentation; 2 had ECG done for bradycardia and 2 were screened because of family history of LQTS.

Out of the 20 patients screened from 17 families, 15 (12 families) had pathogenic mutation or likely deleterious (likely pathogenic) variants in one of the 17 LQTS genes, all in either KCNQ1, KCNH2, or SCN5A. These were broken down to 6 (35%), 4 (24%) and 2 (12%) families with KCNQ1, KCNH2 and SCN5A pathogenic or likely pathogenic mutations, respectively. All of these variants were not present in any of the 719 patients with whole exome study in our in-house database. Two patients had variants of unknown significance (VUS), 1 in KCNH2 and the other in ANK2. In 3 patients, no pathogenic mutation was found in any of the 17 known LQT genes. The details of genetic variants and their classification are shown in Table 2.Table 2Phenotypes and genotype of the patients (N = 20).Table 2No.Age at diagnosisSchwartz's scoreLongest QTc (msec)Trigger eventsRare Variants:Molecular consequenceClinical significantKCNQ1 mutation (N = 7)1a1 day4.5515Asymptomatic, bradycardia for age(exon7) c.1032 G > C (p.Ala344=)bmissense variantPathogenic24.1 years4.5573Asymptomatic, family screening(exon7) c.1032 G > C (p.Ala344=)bmissense variantPathogenic3a9.3 years5554Exercise(exon7) c.1032 G > C (p.Ala344=)bmissense variantPathogenic4a0.2 years3480Asymptomatic, bradycardia for age(exon7) c.1032 G > C (p.Ala344=)bmissense variantPathogenic5a11.5 years4470Exercise(exon7) c.940 G > T, c.941 G > T (p.Gly314Phe)missense variantLikely Pathogenic6a6.3 years6490Exercise(exon7) c.940G > A (p.Gly314Ser)missense variantPathogenic7a10 years6.5623Exercise(exon7) c.940G > A (p.Gly314Ser)missense variantPathogenicKCNH2 mutation (N = 5)8a6.8 years5.5560Rest(exon 8) c.2086C > T (p.Arg696Cys)missense variantLikely Pathogenic9a0.7 years5.5565Sleep(exon 10) c.2453C > G (p.Ser818Trp)missense variantLikely Pathogenic10a4.2 years5557Sleepc.1882–1884 del (p.Gly628del)Disruptive in-frame deletionLikely Pathogenic11a11 years7490Restc.2327 T > C (p.Leu776Pro)missense variantVUS12a8.5 years6.5528Exercisec.1838C > T (p.Thr613Met)missense variantPathogenicSCN5A mutation (N = 4)1310.1 years6.5490Sleepc.715 A > G (p.Ile239Val)missense variantLikely Pathogenic14a12.4 years5492Sleepc.715 A > G (p.Ile239Val)missense variantLikely Pathogenic157.7 years5.5520Asymptomatic, family screeningc.715 A > G (p.Ile239Val)missense variantLikely Pathogenic16a1 day5.5682Sleepc.4460 T > A (p.Met1487Lys)missense variantLikely PathogenicOther mutation (N = 4)17a8.7 years5.5565SleepANK2 c.8404 G > C (p.Asp2802His)missense variantVUS18a6.7 years8510ExerciseNegative––19a9 years7731RestNegative––20a28 years5.5610RestNegative––QTc = corrected QT interval, VUS = variants of unknown significance.aProband.bVariant is on splice site.

Phenotypes and genotype of the patients (N = 20).

QTc = corrected QT interval, VUS = variants of unknown significance.

Proband.

Variant is on splice site.

As for KCNQ1, 6 patients (30%) had pathogenic mutation; 1 patient (5%) had a likely pathogenic mutation. In this group, two patients (siblings) with phenotype of Jervell and Lange-Nielsen syndrome (prolong QT interval and congenital sensorineural hearing loss) had c.1032 G > C (Homozygous) (p.Ala344 = ). An additional 2 unrelated patients with LQTS phenotype also had c.1032 G > C (p.Ala344 = , heterozygous). This is a synonymous variant (no amino acid change), but the mutation is located in splice site which may lead to splice aberration. Two unrelated patient had pathogenic c.940 G > A mutation (p.Gly314Ser). One patient had double missense mutation in the same codon (c.940 G > T and c.941 G > T) (p.Gly314Phe), with no previous report of its pathogenicity. In silico analysis indicated a deleterious effect of this variant.

As for KCNH2, 1 patient (5%) had a pathogenic mutation which was c.1838 C > T variant (p.Thr613Met). Three patients (15%) had likely pathogenic mutations according to ACMG guideline 2015. These variants were c.1882–1884 del (p.Gly628del), c.2086 C > T (p.Arg696Cys) and c.2453 C > G (p.Ser818Trp). One additional patient had c.2327T > C VUS variant (p.Leu776Pro).

As for SCN5A, 3 siblings in the same family had a likely pathogenic mutation, c.715 A > G (p.Ile239Val). One additional patient had c.4460 T > A (p.Met1487Lys), a likely pathogenic mutation, giving the percentage of families with LQT3 of 12%.

One patient had VUS in ANK2. The variant was c.8404 G > C (p.Asp2802His) (Class 3 according to ACMG/AMP, 2015 guideline). There were no identifiable rare variants in the 17 known LQTs gene in 3 patients (18%).

Average age at presentation, mean of the longest QTc interval, mean Schwartz's score of each pathogenic or VUS are shown in Table 3.Table 3Clinical characteristic sub-grouped by rare genetic variants (N = 20, 17 families).Table 3Total (N = 20)KCNQ1KCNH2SCN5AOthersNegativeN (%)7 (35%)5 (25%)a4 (20%)1 (5%)a3 (15%)No. of family (%) (Total 17 families)6 (35%)5 (29%)a2 (12%)1 (6%)a3 (18%)Male (%)33012Age at diagnosis (years), Mean (±SD)5.9 (±4.7)6.2 (±4)7.6 (±5.4)8.714.6 (±11.7) Range0–11.5 years0.7–11 years0–12.4 years–6.7–28 yearsQTc (msec), Mean (±SD)530 (±56.3)540 (±31.5)546 (±91.7)565617 (±110.7) Range470–623490–565490–682–510–731Schwartz's score, Mean (±SD)4.8 (±1.2)5.9 (±0.8)5.6 (±1)5.56.8 (±1.3) Range3–6.55–74.5–6.5–5.5–8Trigger eventsAsymptomatic30100Adrenergically mediated cardiac events41001Cardiac events at rest04312aThe variant of 1 patient in each group was classified as VUS (variants of unknown significance). The percentage of KCNH2 variants that were pathogenic or likely pathogenic was 24% with additional 5% had VUS.

Clinical characteristic sub-grouped by rare genetic variants (N = 20, 17 families).

The variant of 1 patient in each group was classified as VUS (variants of unknown significance). The percentage of KCNH2 variants that were pathogenic or likely pathogenic was 24% with additional 5% had VUS.

In 7 patients with KCNQ1 mutation, 3 were asymptomatic (1 patient presented with fetal bradycardia, 2 with congenital sensorineural hearing loss) and 4 developed stress-induced arrhythmia or syncope.

Most patients with KCNH2 mutation developed symptoms at rest or sleep, only 1 patient developed symptom during running. Almost all of our LQTS type 2 patients had severe phenotype (cardiac arrest) and had ICD implanted. Some of these patients with severe symptoms demonstrated dynamic changing of T morphologies with time (Fig. 1). This patient has c.1882–1884 del (p. Gly628del), which would predict a severe disruption of the protein subunit of IKr channel.Fig. 1Variation of T wave in a patient with severe presentation of KCNH2 mutation.Fig. 1

Variation of T wave in a patient with severe presentation of KCNH2 mutation.

Three patients with SCN5A mutation developed symptoms at rest and the other patient was asymptomatic with siblings diagnosed with congenital LQTS.

At the time of this study, the 20 patients have been followed for a median duration of 5 years (range 0.3–11.7 years). All except one patients received beta-blocker; either atenolol or propranolol. Nine patients (45%) were treated with an implantable cardioverter-defibrillators (ICD); 8 of whom had cardiac events during sleep or at rest. One patient with SCN5A mutation died suddenly at the age of 16 years old while she was talking with her siblings. She did not have an ICD due to lack of symptom and was taking beta-blocker at the time of death. Her QTc was 492 msec. Her two sisters with sensorineural hearing loss had cardiac syncope during sitting at the age of 6.2 and 10.2 years old after non-compliance with beta-blocker. Additional five patients had appropriate ICD shock. Survival from serious cardiac event (death or ventricular arrhythmia) after diagnosis is showed in Fig. 2.Fig. 2Kaplan–Meier curve demonstrating time to first serious cardiac event (death or ventricular arrhythmia) after diagnosis categorized by genetic mutation (N = 20).Fig. 2

Kaplan–Meier curve demonstrating time to first serious cardiac event (death or ventricular arrhythmia) after diagnosis categorized by genetic mutation (N = 20).

### Clinical characteristic

A total of 30 patients in 27 families diagnosed with congenital long QT syndrome were identified. Due to deaths and losses to follow up, only 20 patients from 17 families were available for genetic testing at the time of this study. Among the 10 missing patients, six were lost to follow up (3 patients developed symptoms during exercise, 2 patients developed symptoms during sleep, and 1 patient was asymptomatic at the time of presentation); and 4 patients had died (3 died shortly after presenting with cardiac arrest during exercise, and 1 presented with aborted sudden cardiac death during sleep were dead 6.8 years after ICD implantation, he was lost to follow up for 4.3 years before death). Five out of the 6 patients who were lost to follow up are still alive according to the Thai citizen registration record.

Excluding the 10 patients who were lost to follow-up or died, 20 patients (17 families) were included to this study. The clinical characteristics of the study subjects are summarized in Table 1. Nine patients (45%) were male. Their mean age at diagnosis was 7.8 ± 6.2 years (range 1 day–28 years). Total Schwartz's score was 5.6 ± 1.2 points (range 3–8 points). All except one patient were categorized as high probability of LQTS according to Schwartz's score (score ≥ 4 points). Most patients had no co-morbid condition. Three patients (15%) had congenital abnormalities; 2 with congenital sensorineural hearing loss, and 1 with double aortic arch. Family history of definite LQTS was found in 5 (25%), and family history of cardiac arrest or unexplained sudden cardiac death was reported in 7 patients (35%).Table 1Clinical characteristics of all Long QT syndrome patients (N = 20).Table 1Total (N = 20)N%Male945Age at Diagnosis (years)Mean (±SD)7.8 (±6.2)Range0–28Congenital anomalies315 Double aortic arch15 SNHL210Family history Definite LQTS525 SCD735 SNHL210Total Schwartz's score (points)Mean (±SD)5.6 (±1.2)Range3–8Electrocardiographic characteristics Torsade de pointes420 T wave alternans15 Notched T wave1050 Bradycardia for age525 QTc (msec)Mean (±SD)550.3 (±68.8)Range470–731Treatments Beta blocker1995 ICD945Current Status Alive1995 Death15SNHL = sensorineural hearing loss, LQTS = Long QT syndrome, SCD = sudden cardiac death, QTc = corrected QT interval, msec = millisecond, bpm = beat per minute, HR = heart rate, ICD = implantable cardioverter-defibrillator.

Clinical characteristics of all Long QT syndrome patients (N = 20).

SNHL = sensorineural hearing loss, LQTS = Long QT syndrome, SCD = sudden cardiac death, QTc = corrected QT interval, msec = millisecond, bpm = beat per minute, HR = heart rate, ICD = implantable cardioverter-defibrillator.

### Electrocardiographic characteristics

The mean value of the longest QTc interval in each patient was 550.3 ± 68.8 msec (range 470–731 msec). Five patients (25%) had bradycardia for age. Four patients (20%) presented with torsades de pointes and/or ventricular fibrillation (TdP/VF). Five patients had cardiac arrest with no ECG documentation at the time of the arrest.

### Triggers for cardiac events

Patients were divided into 3 groups; 10 patients (50%) developed cardiac symptoms at rest or sleep, and 6 patients (30%) developed cardiac events during exercise/stress/startle (adrenergically mediated cardiac events). Four patients (20%) were asymptomatic at presentation; 2 had ECG done for bradycardia and 2 were screened because of family history of LQTS.

### Genetic analysis

Out of the 20 patients screened from 17 families, 15 (12 families) had pathogenic mutation or likely deleterious (likely pathogenic) variants in one of the 17 LQTS genes, all in either KCNQ1, KCNH2, or SCN5A. These were broken down to 6 (35%), 4 (24%) and 2 (12%) families with KCNQ1, KCNH2 and SCN5A pathogenic or likely pathogenic mutations, respectively. All of these variants were not present in any of the 719 patients with whole exome study in our in-house database. Two patients had variants of unknown significance (VUS), 1 in KCNH2 and the other in ANK2. In 3 patients, no pathogenic mutation was found in any of the 17 known LQT genes. The details of genetic variants and their classification are shown in Table 2.Table 2Phenotypes and genotype of the patients (N = 20).Table 2No.Age at diagnosisSchwartz's scoreLongest QTc (msec)Trigger eventsRare Variants:Molecular consequenceClinical significantKCNQ1 mutation (N = 7)1a1 day4.5515Asymptomatic, bradycardia for age(exon7) c.1032 G > C (p.Ala344=)bmissense variantPathogenic24.1 years4.5573Asymptomatic, family screening(exon7) c.1032 G > C (p.Ala344=)bmissense variantPathogenic3a9.3 years5554Exercise(exon7) c.1032 G > C (p.Ala344=)bmissense variantPathogenic4a0.2 years3480Asymptomatic, bradycardia for age(exon7) c.1032 G > C (p.Ala344=)bmissense variantPathogenic5a11.5 years4470Exercise(exon7) c.940 G > T, c.941 G > T (p.Gly314Phe)missense variantLikely Pathogenic6a6.3 years6490Exercise(exon7) c.940G > A (p.Gly314Ser)missense variantPathogenic7a10 years6.5623Exercise(exon7) c.940G > A (p.Gly314Ser)missense variantPathogenicKCNH2 mutation (N = 5)8a6.8 years5.5560Rest(exon 8) c.2086C > T (p.Arg696Cys)missense variantLikely Pathogenic9a0.7 years5.5565Sleep(exon 10) c.2453C > G (p.Ser818Trp)missense variantLikely Pathogenic10a4.2 years5557Sleepc.1882–1884 del (p.Gly628del)Disruptive in-frame deletionLikely Pathogenic11a11 years7490Restc.2327 T > C (p.Leu776Pro)missense variantVUS12a8.5 years6.5528Exercisec.1838C > T (p.Thr613Met)missense variantPathogenicSCN5A mutation (N = 4)1310.1 years6.5490Sleepc.715 A > G (p.Ile239Val)missense variantLikely Pathogenic14a12.4 years5492Sleepc.715 A > G (p.Ile239Val)missense variantLikely Pathogenic157.7 years5.5520Asymptomatic, family screeningc.715 A > G (p.Ile239Val)missense variantLikely Pathogenic16a1 day5.5682Sleepc.4460 T > A (p.Met1487Lys)missense variantLikely PathogenicOther mutation (N = 4)17a8.7 years5.5565SleepANK2 c.8404 G > C (p.Asp2802His)missense variantVUS18a6.7 years8510ExerciseNegative––19a9 years7731RestNegative––20a28 years5.5610RestNegative––QTc = corrected QT interval, VUS = variants of unknown significance.aProband.bVariant is on splice site.

Phenotypes and genotype of the patients (N = 20).

QTc = corrected QT interval, VUS = variants of unknown significance.

Proband.

Variant is on splice site.

As for KCNQ1, 6 patients (30%) had pathogenic mutation; 1 patient (5%) had a likely pathogenic mutation. In this group, two patients (siblings) with phenotype of Jervell and Lange-Nielsen syndrome (prolong QT interval and congenital sensorineural hearing loss) had c.1032 G > C (Homozygous) (p.Ala344 = ). An additional 2 unrelated patients with LQTS phenotype also had c.1032 G > C (p.Ala344 = , heterozygous). This is a synonymous variant (no amino acid change), but the mutation is located in splice site which may lead to splice aberration. Two unrelated patient had pathogenic c.940 G > A mutation (p.Gly314Ser). One patient had double missense mutation in the same codon (c.940 G > T and c.941 G > T) (p.Gly314Phe), with no previous report of its pathogenicity. In silico analysis indicated a deleterious effect of this variant.

As for KCNH2, 1 patient (5%) had a pathogenic mutation which was c.1838 C > T variant (p.Thr613Met). Three patients (15%) had likely pathogenic mutations according to ACMG guideline 2015. These variants were c.1882–1884 del (p.Gly628del), c.2086 C > T (p.Arg696Cys) and c.2453 C > G (p.Ser818Trp). One additional patient had c.2327T > C VUS variant (p.Leu776Pro).

As for SCN5A, 3 siblings in the same family had a likely pathogenic mutation, c.715 A > G (p.Ile239Val). One additional patient had c.4460 T > A (p.Met1487Lys), a likely pathogenic mutation, giving the percentage of families with LQT3 of 12%.

One patient had VUS in ANK2. The variant was c.8404 G > C (p.Asp2802His) (Class 3 according to ACMG/AMP, 2015 guideline). There were no identifiable rare variants in the 17 known LQTs gene in 3 patients (18%).

### Clinical characteristic sub-grouped by genetic mutation

Average age at presentation, mean of the longest QTc interval, mean Schwartz's score of each pathogenic or VUS are shown in Table 3.Table 3Clinical characteristic sub-grouped by rare genetic variants (N = 20, 17 families).Table 3Total (N = 20)KCNQ1KCNH2SCN5AOthersNegativeN (%)7 (35%)5 (25%)a4 (20%)1 (5%)a3 (15%)No. of family (%) (Total 17 families)6 (35%)5 (29%)a2 (12%)1 (6%)a3 (18%)Male (%)33012Age at diagnosis (years), Mean (±SD)5.9 (±4.7)6.2 (±4)7.6 (±5.4)8.714.6 (±11.7) Range0–11.5 years0.7–11 years0–12.4 years–6.7–28 yearsQTc (msec), Mean (±SD)530 (±56.3)540 (±31.5)546 (±91.7)565617 (±110.7) Range470–623490–565490–682–510–731Schwartz's score, Mean (±SD)4.8 (±1.2)5.9 (±0.8)5.6 (±1)5.56.8 (±1.3) Range3–6.55–74.5–6.5–5.5–8Trigger eventsAsymptomatic30100Adrenergically mediated cardiac events41001Cardiac events at rest04312aThe variant of 1 patient in each group was classified as VUS (variants of unknown significance). The percentage of KCNH2 variants that were pathogenic or likely pathogenic was 24% with additional 5% had VUS.

Clinical characteristic sub-grouped by rare genetic variants (N = 20, 17 families).

The variant of 1 patient in each group was classified as VUS (variants of unknown significance). The percentage of KCNH2 variants that were pathogenic or likely pathogenic was 24% with additional 5% had VUS.

In 7 patients with KCNQ1 mutation, 3 were asymptomatic (1 patient presented with fetal bradycardia, 2 with congenital sensorineural hearing loss) and 4 developed stress-induced arrhythmia or syncope.

Most patients with KCNH2 mutation developed symptoms at rest or sleep, only 1 patient developed symptom during running. Almost all of our LQTS type 2 patients had severe phenotype (cardiac arrest) and had ICD implanted. Some of these patients with severe symptoms demonstrated dynamic changing of T morphologies with time (Fig. 1). This patient has c.1882–1884 del (p. Gly628del), which would predict a severe disruption of the protein subunit of IKr channel.Fig. 1Variation of T wave in a patient with severe presentation of KCNH2 mutation.Fig. 1

Variation of T wave in a patient with severe presentation of KCNH2 mutation.

Three patients with SCN5A mutation developed symptoms at rest and the other patient was asymptomatic with siblings diagnosed with congenital LQTS.

### Treatment and follow-up

At the time of this study, the 20 patients have been followed for a median duration of 5 years (range 0.3–11.7 years). All except one patients received beta-blocker; either atenolol or propranolol. Nine patients (45%) were treated with an implantable cardioverter-defibrillators (ICD); 8 of whom had cardiac events during sleep or at rest. One patient with SCN5A mutation died suddenly at the age of 16 years old while she was talking with her siblings. She did not have an ICD due to lack of symptom and was taking beta-blocker at the time of death. Her QTc was 492 msec. Her two sisters with sensorineural hearing loss had cardiac syncope during sitting at the age of 6.2 and 10.2 years old after non-compliance with beta-blocker. Additional five patients had appropriate ICD shock. Survival from serious cardiac event (death or ventricular arrhythmia) after diagnosis is showed in Fig. 2.Fig. 2Kaplan–Meier curve demonstrating time to first serious cardiac event (death or ventricular arrhythmia) after diagnosis categorized by genetic mutation (N = 20).Fig. 2

Kaplan–Meier curve demonstrating time to first serious cardiac event (death or ventricular arrhythmia) after diagnosis categorized by genetic mutation (N = 20).

### Discussion

Most cardiac arrhythmia in congenital long QT syndrome patients results from mutation in cardiac ion channels including potassium, calcium and sodium channels. Historically, LQTS were classified into two groups by the inherited pattern: autosomal dominant (Romano-Ward syndrome), and autosomal recessive with neurosensory deafness (Jervell and Lange-Nielsen syndrome). At present, pathogenic mutation in at least one of the 17 genes (detailed in Methods section) can be found in 50–80% of all clinically diagnosed LQTS patients [5]. The yield is less in the more recent cohorts of patient in which patients with less severe phenotypes were included [16]. The high yield of genetic testing in our cohort could be partly explained by the fact that most of our cases were symptomatic with relatively long QT intervals. Genotypic classification can be used to guide individualized treatment in these patients [17,18]. The use of clinical presentation, heart rate, QTc interval, ECG phenotypes, T wave morphology and Schwartz's score cannot differentiate among the types of LQTS with complete certainty [19].

Of the 17 genes, most (75%) of the LQTS patients had pathogenic mutation in only one of the 3 genes: KCNQ1 (30–35%), KCNH2 (25–30%), and SCN5A (5–10%) [20,21]. There are few studies on the percentage of each type of LQTS in East Asian countries [8,11,12,[22], [23], [24]], as shown in Table 4. Similar to the Western countries, the majority of the East Asian patients had LQT1, and LQT2. From the present study, we found that 18% of our patients had negative genetic mutation in the known 17 of LQT genes, consistent with previous reports. As for gene-positive patients, the distributions of pathogenic or likely pathogenic genetic variants in our population (using family-based percentage) were 35%, 24%, and 12% in KCNQ1, KCNH2, and SCN5A, respectively. Two patients (12%) had VUS and 3 (18%) were negative for the known LQT genes variants. Mean age at diagnosis seemed to be lower in LQTS type 1 patients, which is not unexpected as LQT1 patients present with symptoms at a younger age. The mean QTc was slightly higher in negative LQT gene mutation in our cohort. Most of the LQT1 patients developed, as expected, symptoms during exercise and had typical broad-based prolonged T wave. Both patients with Jervell and Lange-Nielsen syndrome in our study were asymptomatic and were presented with bilateral congenital sensorineural hearing loss. Most of the LQT2 patients developed symptoms during sleep or rest and had bifid T wave, and almost all had previous history of cardiac arrest, indicating more severe phenotype than LQT1 patients in our population.Table 4Genotype-Specific Prevalence of Long QT syndrome in East Asian countries.Table 4TypeGene/LocusJapanChinaKoreaTaiwanPresent studyHorigome H. et al., 2010 [7]Yoshinaga M. et al., 2014 [20]aLiu W. et al., 2002 [10]Gao Y. et al., 2016 [18]Lee YS. et al., 2013 [19]Chang YS. et al., 2015 [11]17 Families58 Probands117 Probands42 Probands230 Probands62 Probands5 Probands(20 Probands)n (%)n (%)n (%)n (%)n (%)n (%)n (%)LQT1KCNQ111 (19%)35 (30%)4 (10%)80 (34.8%)9 (14.5%)1 (20%)6 (35%)LQT2KCNH211 (19%)22 (18.8%)3 (7%)101 (43.9%)5 (8.1%)1 (20%)4 (24%)LQT3SCN5A6 (10.5%)17 (14.5%)N/A5 (2.2%)3 (4.8%)N/A2 (12%)LQT5KCNE1N/A5 (4.3%)N/AN/AN/AN/A0LQT6KCNE2N/AN/AN/AN/AN/AN/A0LQT7KCNJ2N/A2 (1.7%)N/AN/A3 (4.8%)N/A0LQT8CACNA1C1 (1.7%)1 (0.8%)N/AN/A1 (1.6%)N/A0LQT13KCNJ5N/A1 (0.8%)N/AN/AN/AN/A0Negative12 (20.7%)44 (37.6%)35 (83%)44 (19.1%)21 (33.9%)3 (60%)3 (18%)OthersN/AN/AN/AN/AN/AN/AVUS in ANK2 (LQT4) 1 (6%)KCNH2 1 (6%)Not test17 (29.3%)NoneNoneNone20 (32.2%)NoneNoneVUS = variants of unknown significance.aMultiple mutation in some patients.

Genotype-Specific Prevalence of Long QT syndrome in East Asian countries.

VUS = variants of unknown significance.

Multiple mutation in some patients.

Our study was based on patients who presented to the hospital, which likely represent only the most severe cases of LQTS patients [16]. Moreover, we still lack an appropriate screening program for family members of genotype positive patients so the patients who were asymptomatic might not have come to medical attention and long QT syndrome might still be under recognized in Thailand. Further genetic testing in family members should be provided to identify those family members with pre-symptomatic congenital long QT syndrome.

### Study limitations

Our study was based on patients who presented to the hospital, which likely represent only the most severe cases of LQTS patients [16]. Moreover, we still lack an appropriate screening program for family members of genotype positive patients so the patients who were asymptomatic might not have come to medical attention and long QT syndrome might still be under recognized in Thailand. Further genetic testing in family members should be provided to identify those family members with pre-symptomatic congenital long QT syndrome.

### Conclusions

Our study is the first to report prevalence of genetic mutations, and demonstrate phenotype-genotype characteristics in congenital long QT syndrome patients in Thailand. Seventy-one percent of the LQTS families can be classified as LQT1, 2 or 3.

### Conflicts of interest statement

None.



# SUPPLEMENTAL FILE 1: main.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)