# MAIN TEXT

## Systematic Review of the Genetics of Sudden Unexpected Death in Epilepsy: Potential Overlap With Sudden Cardiac Death and Arrhythmia‐Related Genes

### Abstract

BackgroundSudden unexpected death in epilepsy (SUDEP) is the leading cause of epilepsy‐related death. SUDEP shares many features with sudden cardiac death and sudden unexplained death in the young and may have a similar genetic contribution. We aim to systematically review the literature on the genetics of SUDEP.Methods and ResultsPubMed, MEDLINE Epub Ahead of Print, Ovid Medline In‐Process & Other Non‐Indexed Citations, MEDLINE, EMBASE, Cochrane Database of Systematic Reviews, and Scopus were searched through April 4, 2017. English language human studies analyzing SUDEP for known sudden death, ion channel and arrhythmia‐related pathogenic variants, novel variant discovery, and copy number variant analyses were included. Aggregate descriptive statistics were generated; data were insufficient for meta‐analysis. A total of 8 studies with 161 unique individuals were included; mean was age 29.0 (±SD 14.2) years; 61% males; ECG data were reported in 7.5% of cases; 50.7% were found prone and 58% of deaths were nocturnal. Cause included all types of epilepsy. Antemortem diagnosis of Dravet syndrome and autism (with duplication of chromosome 15) was associated with 11% and 9% of cases. The most frequently detected known pathogenic variants at postmortem were in Na+ and K+ ion channel subunits, as were novel potentially pathogenic variants (11%). Overall, the majority of variants were of unknown significance. Analysis of copy number variant was insignificant.Conclusions
SUDEP case adjudication and evaluation remains limited largely because of crucial missing data such as ECGs. The most frequent pathogenic/likely pathogenic variants identified by molecular autopsy are in ion channel or arrhythmia‐related genes, with an ≈11% discovery rate. Comprehensive postmortem examination should include examination of the heart and brain by specialized pathologists and blood storage.

### Clinical Perspective

The mechanisms of sudden unexpected death in epilepsy (SUDEP) and its overlap with sudden cardiac death are not well understood.We systemically reviewed the literature regarding genetic contributions to SUDEP.The most frequent variants identified in the reviewed SUDEP cases were related to ion channel subunits and/or in genetic syndromes known to be associated with SUDEP.

The mechanisms of sudden unexpected death in epilepsy (SUDEP) and its overlap with sudden cardiac death are not well understood.

We systemically reviewed the literature regarding genetic contributions to SUDEP.

The most frequent variants identified in the reviewed SUDEP cases were related to ion channel subunits and/or in genetic syndromes known to be associated with SUDEP.

Our findings suggest a gap in knowledge of the genetic causes of SUDEP in part because of limited clinical data such as ECGs and formal cardiac and neurologic pathology evaluations to further elucidate and define causative factors in SUDEP cases.

Our findings suggest a gap in knowledge of the genetic causes of SUDEP in part because of limited clinical data such as ECGs and formal cardiac and neurologic pathology evaluations to further elucidate and define causative factors in SUDEP cases.

### What Is New?

The mechanisms of sudden unexpected death in epilepsy (SUDEP) and its overlap with sudden cardiac death are not well understood.We systemically reviewed the literature regarding genetic contributions to SUDEP.The most frequent variants identified in the reviewed SUDEP cases were related to ion channel subunits and/or in genetic syndromes known to be associated with SUDEP.

The mechanisms of sudden unexpected death in epilepsy (SUDEP) and its overlap with sudden cardiac death are not well understood.

We systemically reviewed the literature regarding genetic contributions to SUDEP.

The most frequent variants identified in the reviewed SUDEP cases were related to ion channel subunits and/or in genetic syndromes known to be associated with SUDEP.

### What Are the Clinical Implications?

Our findings suggest a gap in knowledge of the genetic causes of SUDEP in part because of limited clinical data such as ECGs and formal cardiac and neurologic pathology evaluations to further elucidate and define causative factors in SUDEP cases.

Our findings suggest a gap in knowledge of the genetic causes of SUDEP in part because of limited clinical data such as ECGs and formal cardiac and neurologic pathology evaluations to further elucidate and define causative factors in SUDEP cases.

### Introduction

Sudden unexpected death in epilepsy (SUDEP) is defined as “the sudden, unexpected, witnessed or unwitnessed, non‐traumatic and non‐drowning death in patients with epilepsy with or without evidence of a seizure, and excluding documented status epilepticus, in which postmortem examination does not reveal a structural or toxicological cause for death.” 1 The diagnosis of SUDEP can be challenging, especially when other competing causes of sudden death cannot be definitively ruled out and systematic autopsies are not performed.2, 3 SUDEP events account for up to 18% of all deaths in people with epilepsy (PWE) and by potential years of life lost ranks as the second leading cause of death in neurological diseases after stroke.4 The incidence of SUDEP is estimated to be 0.58 to 9.0 per 1000 person‐years, but in refractory epilepsy patients the lifetime cumulative risk can be as high as 35%.5, 6, 7, 8, 9, 10, 11, 12, 13 The incidence is higher in adults than it is in children and the majority of SUDEP cases occur at night during sleep with decedents found in the prone position.

Analogous to the majority of sudden cardiac deaths in the general population, SUDEP is believed to be caused by a culmination of cardiac, neurological, and respiratory factors leading to a “perfect storm” and death. However, in those under the age of 40 years, genetic cardiac conditions (channelopathies, cardiomyopathies, and aortopathies) are an important and preventable contributor to sudden cardiac deaths cases. Channelopathies can present with both an arrhythmia and seizure phenotype, and it is plausible that genotype plays an integral role in determining predisposition in SUDEP.14, 15 Interestingly, the cardiac voltage‐gated sodium channel SCN5A was discovered in brain limbic regions, and this discovery provided the first link between genetically predisposed cardiac arrhythmias and epilepsy.16 Later studies showed a phenotype of comorbid epilepsy, cardiac arrhythmias, and SUDEP in transgenic mice carrying human knock‐in mutations in the most common long QT syndrome (LQTS) gene, KCNQ1.17, 18 While the mechanisms of death in SUDEP are unknown, these discoveries suggest that cardiogenic mechanisms may be involved in the sudden death of these patients. The 2 dominant paradigms are “arrhythmogenic epilepsy” (which may be monogenic sharing a seizure and arrhythmia phenotype) and impaired neural circulatory control (which leads to respiratory arrest, cardiac arrest, or both).17, 18, 19, 20 Further evidence in support of the former hypothesis comes from studies on large cohorts of SUDEP patients in Australia who were found to have genetic variants encoding potassium, sodium, and calcium ion channel subunits expressed both in neuronal and cardiac cells.21, 22 These variants may act in isolation or require the presence of a second genetic factor or environmental influence, such as uncontrolled seizures, QT‐prolonging anti‐epileptic drugs, noncompliance with anti‐epileptic drug therapy, or autonomic dysfunction to predispose epilepsy patients to malignant arrhythmias and sudden death.17

Despite the direct clinical importance and compelling need to understand the SUDEP conundrum, there are limited studies seeking to identify genetic links to SUDEP. Researchers face logistical difficulties including incomplete (or absent) autopsies, lack of adequate postmortem DNA samples or appropriate consent, as well as the challenges inherent to studying a rare event. Most genetic studies on SUDEP have been of single cases or small series and often do not include deep phenotyping or DNA from family members (often for ethical and logistical reasons), making novel variant discovery arduous.17 The aim of this systematic review is to synthesize and critically evaluate the current knowledge about the genetics of SUDEP.

### Methods

The data that support the findings of this study are available from the corresponding author upon reasonable request. This is a systematic review that was conducted and reported in accordance with the PRISMA (Preferred Reporting Items for Systematic Reviews and Meta‐Analyses) statement.23 The protocol for this systematic review was registered on PROSPERO (CRD42017074534) and is available in full online: (https://www.crd.york.ac.uk/prospero/display_record.php?RecordID=74534).24

A comprehensive search of several databases starting from each database's inception through April 4, 2017 was conducted without restriction on language. The databases included PubMed, MEDLINE Epub Ahead of Print, Ovid Medline In‐Process & Other Non‐Indexed Citations, MEDLINE, EMBASE, Cochrane Database of Systematic Reviews, and Scopus. The search strategy was designed and conducted by a medical reference librarian with input from the lead investigator. Controlled vocabulary supplemented with keywords was used to search for genetic factors in sudden death and epilepsy. The actual strategy is available in Data S1. Duplicates were eliminated. References from the retrieved original articles were searched for missing studies.

Initial screening of the identified studies was performed by 3 independent reviewers (M.N.S., J.A.G., L.A.) working in duplicate based on the titles and abstracts, taking into consideration the inclusion criteria. After removing irrelevant and nonoriginal studies, full‐text screening was performed to assess eligibility for final inclusion. Discrepancies were resolved through discussion and consensus.

Following our a priori inclusion criteria, we selected observational English language human studies and then identified specific genes that had a relationship with SUDEP. The first selection was made by reading abstracts and the final selection by reading full articles. Two reviewers were required for each selection. Any inclusion at the first selection stage proceeded to the final selection. Disagreements among reviewers required a final review by a third reviewer. We included analyses of unique patients. If multiple studies reported results from overlapping groups of patients (articles reporting secondary or post hoc analyses), we included the most comprehensive report to avoid duplication of patient data.

Reviewers extracted data independently from the included studies in duplicate, using a standardized, piloted, web‐based form that was developed based on the protocol (Covidence, Melbourne, Australia). The following demographic data were extracted where available: age at death, sex, seizures, anti‐epileptic drug use, seizure frequency, duration of epilepsy diagnosis, and presence of autopsy or toxicology results. Genetic data extracted included genetic variants identified, variant type where available, and frequency among the study population. All disagreements or differences in extracted data were resolved by consensus.

The included studies were evaluated using a tool specific for appraising case reports and case series that included items from the Newcastle‐Ottawa scale.25, 26 The risk of bias tool used included assessment of 5 separate domains: selection of subjects, comparability of study groups where applicable, ascertainment of SUDEP status and ascertainment of genetic testing results, as well as assessment for any other sources of bias identified. Features suggesting low risk of bias include use of a consecutive or representative sample of patients, limitation of confounders or appropriate adjustments as applicable, high certainty of SUDEP classification, pathogenicity of analyzed variants, and association of variants analyzed with SUDEP. Each study was evaluated independently by 2 independent reviewers (M.N.S, J.A.G). All discrepancies were resolved by a third reviewer. The corresponding author of the study (CAC) had full access to all the data in the study and takes responsibility for its integrity and the data analysis.

As part of the a priori protocol, we planned on conducting a meta‐analysis should we have sufficient quantitative data from the included studies. Because this was not the case, the current analysis was focused on reporting and summarizing descriptive parameters of the included studies and their genetic associations.

### Data Sources and Search Strategy

A comprehensive search of several databases starting from each database's inception through April 4, 2017 was conducted without restriction on language. The databases included PubMed, MEDLINE Epub Ahead of Print, Ovid Medline In‐Process & Other Non‐Indexed Citations, MEDLINE, EMBASE, Cochrane Database of Systematic Reviews, and Scopus. The search strategy was designed and conducted by a medical reference librarian with input from the lead investigator. Controlled vocabulary supplemented with keywords was used to search for genetic factors in sudden death and epilepsy. The actual strategy is available in Data S1. Duplicates were eliminated. References from the retrieved original articles were searched for missing studies.

### Selection of Studies

Initial screening of the identified studies was performed by 3 independent reviewers (M.N.S., J.A.G., L.A.) working in duplicate based on the titles and abstracts, taking into consideration the inclusion criteria. After removing irrelevant and nonoriginal studies, full‐text screening was performed to assess eligibility for final inclusion. Discrepancies were resolved through discussion and consensus.

Following our a priori inclusion criteria, we selected observational English language human studies and then identified specific genes that had a relationship with SUDEP. The first selection was made by reading abstracts and the final selection by reading full articles. Two reviewers were required for each selection. Any inclusion at the first selection stage proceeded to the final selection. Disagreements among reviewers required a final review by a third reviewer. We included analyses of unique patients. If multiple studies reported results from overlapping groups of patients (articles reporting secondary or post hoc analyses), we included the most comprehensive report to avoid duplication of patient data.

### Data Acquisition

Reviewers extracted data independently from the included studies in duplicate, using a standardized, piloted, web‐based form that was developed based on the protocol (Covidence, Melbourne, Australia). The following demographic data were extracted where available: age at death, sex, seizures, anti‐epileptic drug use, seizure frequency, duration of epilepsy diagnosis, and presence of autopsy or toxicology results. Genetic data extracted included genetic variants identified, variant type where available, and frequency among the study population. All disagreements or differences in extracted data were resolved by consensus.

### Methodological Quality Assessment

The included studies were evaluated using a tool specific for appraising case reports and case series that included items from the Newcastle‐Ottawa scale.25, 26 The risk of bias tool used included assessment of 5 separate domains: selection of subjects, comparability of study groups where applicable, ascertainment of SUDEP status and ascertainment of genetic testing results, as well as assessment for any other sources of bias identified. Features suggesting low risk of bias include use of a consecutive or representative sample of patients, limitation of confounders or appropriate adjustments as applicable, high certainty of SUDEP classification, pathogenicity of analyzed variants, and association of variants analyzed with SUDEP. Each study was evaluated independently by 2 independent reviewers (M.N.S, J.A.G). All discrepancies were resolved by a third reviewer. The corresponding author of the study (CAC) had full access to all the data in the study and takes responsibility for its integrity and the data analysis.

### Statistical Analysis

As part of the a priori protocol, we planned on conducting a meta‐analysis should we have sufficient quantitative data from the included studies. Because this was not the case, the current analysis was focused on reporting and summarizing descriptive parameters of the included studies and their genetic associations.

### Results

The initial search yielded 175 articles after excluding duplicates. The inclusion criteria were designed to select studies investigating the genetics of patients diagnosed with SUDEP or any seizure disorder with a history of unexplained sudden death in humans. The selection process is summarized in Figure 1. A total of 8 studies were included.21, 27, 28, 29, 30, 31, 32, 33 The total number of patients from all 8 studies was 228. However, 2 of the studies included some cases from a prior study,27 leaving a total of 161 unique individuals.

PRISMA flow diagram demonstrating selection process of studies for inclusion in the systematic review. PRISMA indicates Preferred Reporting Items for Systematic Reviews and Meta‐Analyses.

The mean age of patients at diagnosis of epilepsy or presentation with first seizures was 10.3 (SD ±9.8) years, with age at SUDEP of 29.0 (SD ±14.2) years. Cases were more likely to be male (61%).These data are consistent with published data on SUDEP cases.13, 34, 35, 36, 37 In 7 studies, all patients had a history of documented seizures. Four studies were from Australia with a total of 139 patients,21, 27, 28, 29 2 from the United States with a total of 14 patients,30, 33 1 from the United Kingdom with 18 patients, and 1 from Japan with 9 patients.31 Some of the Australian studies included patients from prior series; after removal of duplicates, the final total of patients was 161. The characteristics of patients are summarized in Table 1.

Characteristics of the Included Studies

Autism‐related

Mixed

6 (33%) DS

5 (28%) focal specified

4 (22%) focal unspecified

1 (6%) generalized

2 (11%) unspecified

6 (66%) focal

3 (34%) generalized

ADHD indicates attention deficit hyperactivity disorder; DS, Dravet syndrome; dup, duplication; LGS, Lennox‐Gastaut syndrome; LQT, long QT; NA, not available or reported; SCD, sudden cardiac death; SUDEP, sudden unexpected death in epilepsy; WES, whole exome sequencing.

In years.

HUGO gene nomenclature committee gene names.

Includes 48 patients from earlier study by Tu et al (2011).

Three patients with borderline QTc prolongation.

Includes 19 patients from earlier study by Tu et al (2011).

Median (range).

The majority of studies did not report ECG findings or explain whether an ECG was available. One study reported that de‐identified samples were received and ethical requirements prohibited obtaining detailed clinical information including ECGs.21

SUDEP cases were classified into definite in 83 (51.6%), definite‐plus in 6 (3.7%), probable in 26 (16.1%), and possible 46 (28.6) as per Nashef et al (Table 2).1 In brief, SUDEP is categorized per the following criteria: (1) definite if an autopsy does not reveal a cause of death (including patients with a structurally normal heart); (2) definite‐plus if an autopsy was performed and there may be an alternate contributing factor; (3) probable if no autopsy is performed; (4) possible if there is a competing cause of death; and (5) unknown if there is no autopsy, no witness, and limited records.1, 38 Two studies focused on autism with autism‐related epilepsy and SUDEP. One study did not report the types of epilepsy33 and the other included 2 (25%) with generalized seizures related to autism, 2 (25%) with Lennox‐Gastaut syndrome and no known cause for the remainder.30 In 2 other studies the classification of epilepsy could not be determined because it was not reported for each case, but the discussion commented on including generalized, focal alcohol‐related and post‐traumatic causes, with the rationale for the latter 2 stating these events may have precipitated seizures or contributed to lowering seizure threshold in susceptible individuals.21, 33 In 1 study, this information was not reported and an earlier study using some of the same patients was cited.27 One study reported a range of epilepsy types without defined frequencies including generalized, temporal lobe epilepsy, postencephalopathic, and juvenile myoclonic epilepsy.28 In a study from the United Kingdom, 33% were Dravet syndrome (DS) patients, 28% focal specified, 22% focal unspecified, 6% generalized, and 11% unspecified.32 One study from Australia of 10 cases focused exclusively on cases of DS.29 A study from Japan included 66% with focal and 34% with generalized seizures.31 Table 3 summarizes inclusion and exclusion criteria of the cohorts.

Classification of SUDEP and Time of Death

SUDEP indicates sudden unexpected death in epilepsy.

Includes 48 patients from earlier study by Tu et al (2011).

Includes 19 patients from earlier study by Tu et al (2011).

Inclusion and Exclusion Criteria of Individual Studies

9 subjects with duplications of chromosome 15q11.2‐q13 [dup(15)]

10 subjects with autism

7 control subjects

Patients with epilepsy in whom postmortem examination failed to reveal a cause of death

SUDEP or possible SUDEP as cause of death

18 with epilepsy who died of SUDEP (probable or definite)

87 living people with epilepsy (controls)

1479 nonepilepsy disease control samples

61 SUDEP cases were recruited from 3 sources:
27 had participated in the epilepsy genetics research program in Melbourne, Australia, during life and had SUDEP on follow‐up;15 prospective coronial SUDEP cases were collected from 2010 to 2012 by the Departments of Forensic Medicine (DOFM) in New South Wales, Victoria, Queensland, and South Australia; and19 retrospective coronial SUDEP cases were collected from a review of autopsy reports over a 17‐y period from 1993 to 2010 at the DOFM in Sydney.

27 had participated in the epilepsy genetics research program in Melbourne, Australia, during life and had SUDEP on follow‐up;

15 prospective coronial SUDEP cases were collected from 2010 to 2012 by the Departments of Forensic Medicine (DOFM) in New South Wales, Victoria, Queensland, and South Australia; and

19 retrospective coronial SUDEP cases were collected from a review of autopsy reports over a 17‐y period from 1993 to 2010 at the DOFM in Sydney.

Cases classified as definite SUDEP, definite SUDEP plus, probable SUDEP, or possible SUDEP

Typical electroclinical phenotype of Dravet Syndrome

Mortality and SUDEP rates estimated in 100 cases of Dravet, 87 had SCN1A mutation

Dup indicates duplication; SUDEP, sudden unexpected death in epilepsy.

Some of the cohort was previously reported by Tu et al (2011).

Three of 8 studies did not report the position of the body.29, 30, 32 Two studies were unable to determine this because of de‐identified and limited clinical data.21, 33 In the remaining 3 studies (n=138), the majority of decedents (50.7%) were found in the prone position; in 43.4% of cases, the position was unknown. Only 2 studies reported time of death: a study from the United States on DS reported 6 (60%) nocturnal deaths and a study from Japan including multiple epilepsy syndromes reported 5 (56%) nocturnal deaths.29, 31

Five of the 8 studies were retrospective with molecular autopsies to identify pathogenic variants. Two of the studies focusing on duplications involving chromosome 15 [dup(15)] used a combination of techniques including fluorescent in situ hybridization, genotyping for copy number variant (CNV) analysis, and Southern blotting, likely reflecting historical use of technologies for case evaluation (Table 4).30, 33 These were all performed antemortem. Two studies used Sanger sequencing exclusively: 1 focused on the top 3 LQTS loci to detect frequency of known and novel variants in SUDEP cases, while the other assessed polyalanine repeat expansion alleles.21, 27 Two studies used next‐generation whole exome sequencing: 1 study compared cases to controls to identify deleterious variants and the other targeted analysis to known cardiac arrhythmia, epilepsy‐related, and respiratory control genes.28, 32 The remaining 2 studies used a combination of next‐generation and Sanger sequencing.29, 31 One study assessed the mitochondrial genome and reported no associations.28 Three studies also assessed CNVs: 2 studies focused on duplications involving chromosome 15 [dup(15)] of varying magnitude and 1 study assessed 41 (67.2%) of decedents for CNVs, in arrhythmia, respiratory, and epilepsy‐related genes, detecting no differences.

Sequencing Techniques Utilized and Genes Screened

Genotyping

FISH

Southern blot

Array CGH

1 known pathogenic in DEPDC5

1 pathogenic in PAFAH1B1

Targeted NGS

Sanger

BrS indicates Brugada syndrome; chr, chromosome; CNV, copy number variants; FISH, fluorescent in situ hybridization; Mito, mitochondrial; NGS, next‐generation sequencing; VUS, variants of unknown significance; WES, whole exome sequencing.

HUGO gene nomenclature committee gene names.

Includes 48 patients from earlier study by Tu et al (2011).

Includes 19 patients from earlier study by Tu et al (2011).

Of the reported genetic variants, the vast majority were caused by substitutions. Most of the variants were in ion channel subunits (n=21; 13%). Duplications accounted for 14 (8.7%) variants. The variants identified as well as the comparable frequency in a general epilepsy or control population where available are listed in Table S1.

Eighteen genes and 4 different duplications were reported to have a possible association with SUDEP: KCNH2, SCN5A, KCNQ1, SCN1A, LGI1, PIK3C2A, SMC4, COL6A3, TIE1, DSC2, LDB3, KCNE1, MYBPC3, MYH6, DSP, DSG2, DMD, isodicentric chromosome 15[idic(15)], derivative chromosome 15 [der(15)], tricentric chromosome 15, and triplication of chromosome 15[trp(15)] (Table 4). The vast majority of variants were classified as variants of unknown significance. The genes reported to be associated with SUDEP included cation (sodium, potassium) channel protein subunits. Only 2 studies had overlap in the reported genes. These genes were SCN5A, known to cause LQT3 (via gain of function) and Brugada syndrome (BrS; via loss of function), and SCN1A, which is associated with DS, familial hemiplegic migraine and genetic epilepsy with febrile seizures. Figure 2 illustrates known genes associated with a variety of genetic cardiac disorders, as well as whether these disorders have a reported overlap with SUDEP. While we did not identify variants in all the genes included in this figure, we included it to depict a summation of cardiac genetic disorders that possibly contribute to the development of SUDEP.

Overview of complexity of genes implicated in SUDEP and other genetic cardiovascular diseases associated with sudden death. The figure demonstrates known genes associated with each of the following conditions: BrS, 5HT receptor mutations, ERS, LQTS, ARVC, short QT, HCM, CPVT, DCM, and primary brain SUDEP. The orange syndromes represent genetic cardiac disorders as well as primary brain SUDEP. The surrounding boxes connected via a black solid line represent genes of which a variant can result in the clinical syndrome to which it is connected. Blue boxes represent gene variants that have only 1 identified genetic cardiac phenotype of the diseases included. Purple boxes represent genes with variants that have 2 identified phenotypes. For example, variants in the gene KCNH2 can result in both long QT syndrome and short QT syndrome. Similarly, variants in RYR2 are associated with CPVT, DCM, and ARVC. However, of the included cardiac conditions, variants in SCN1B are typically only associated with BrS. The centrally pointing red arrows represent the potential contribution of select genetic cardiac disorders to the central clinical entity of primary brain SUDEP. ARVC indicates arrhythmogenic right ventricular cardiomyopathy; BrS, Brugada syndrome; CPVT, catecholaminergic polymorphic ventricular tachycardia; DCM, dilated cardiomyopathy; ERS, early repolarization syndrome; HCM, hypertrophic cardiomyopathy; 5HT, serotonin; LQTS, long QT syndrome; SUDEP, sudden unexpected death in epilepsy.

One study of 68 SUDEP patients specifically targeting LQT1 (KCNQ1), LQT2 (KCNH2), and LQT3 (SCN5A) identified 2 nonsynonymous variants in KCNH2 and 4 in SCN5A, which have all been reported in cases of LQTS and functionally characterized as pathogenic.21, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48 One of the 6 cases carried 2 nonsynonymous polymorphisms: p.Arg1047Leu‐KCNH2 and p.Ala572Asp‐SCN5A. There was no reported seizure activity 12 months preceding death and the patient died at an older age of 52 years. The SCN5A variant has been reported in LQTS, sudden infant death syndrome, BrS, and multiple cases of female victims of sudden cardiac deaths but has now been classified as likely benign.

A subsequent study from the same group, utilizing 48 of the 68 SUDEP cases from the aforementioned study, was combined with an additional 20 cases from an alternate source that were specifically screened for polyalanine repeat expansions in the homeobox PHOX2B gene.27
PHOX2B has been implicated in dysfunction of respiratory control, including congenital central hypoventilation syndrome, an autonomic disorder that can lead to hypoventilation and a blunted chemoreceptor response to hypercapnia and hypoxia. Additionally, alterations in the gene have been associated with bradyarrhythmias and coexisting epilepsy.49, 50, 51 Two synonymous variants were identified in 4 (5.9%) SUDEP decedents categorized as variants of unknown significance; however, no polyalanine repeat expansion alleles or point mutations were identified, suggesting against PHOX2B as a major genetic contributor to SUDEP.

In a third study by the same group, 61 SUDEP cases (including 19 from an earlier study) underwent whole exome sequencing with targeted analysis of 109 genes (the top 3 LQTS genes, 29 other cardiac arrhythmia genes, 5 genes involved in ventilation, and 72 epilepsy‐related genes).21, 28 This identified 6 (10%) cases with pathogenic variants in 2 LQTS genes (KCNQ1 and KCNH2 as reported previously).21 No known pathogenic variants were identified in the other cardiac arrhythmia or respiratory control genes, and 2 known pathogenic variants were identified in epilepsy‐related genes (DEPDC5 and PAFAH1B1). Nine candidate variants in cardiac arrhythmia genes were identified, none of which had undergone functional characterization. Therefore, classification was based on in silico prediction. No candidate pathogenic variants were identified in the respiratory control genes. An additional 10 candidate pathogenic variants were identified in epilepsy‐related genes (Table 4 and Table S1), which were also based on in silico prediction. Of these, DEPDC5 nonsense variants were considered to confer the greatest risk because these included mutations in highly conserved areas and were identified in 6 unrelated SUDEP cases. In addition, the investigators conducted mitochondrial genome analysis and did not detect any known pathogenic variants. Similarly, CNV analysis did not detect any significant deletions or duplications.

A study from the United Kingdom performed whole exome sequencing on 18 decedent cases and compared them with both 87 living PWE and 1479 nonepilepsy disease controls to identify likely deleterious variants.32 The frequency of DS in living PWE was 34.5% (n=30) and SUDEP cases was 33% (n=6). Of the living PWE with DS, 26 (87%) had a known pathogenic variant in SCN1A compared with all 6 of the decedent SUDEP cases with DS. Of the 89 512 variants identified (in SUDEP, living PWE, and nonepilepsy controls), these linked to 13 887 genes and a high genomewide burden score per individual, suggesting a polygenic contribution to SUDEP causation. Likely deleterious variants in 373 genes were identified exclusively in the SUDEP group, 1 of which was CACNB2 associated with BrS (the exact variant is not reported and is predicted by in silico modeling only).52, 53 Other than the known SCN1A pathogenic variants in DS cases, no known sudden death or genetic cardiac disease–related pathogenic variants were identified. However, 5 additional candidate pathogenic variants were identified in other genes: LGI1, PIK3C2A, SMC4, COL6A3, and TIE1. The cumulative minor allele frequency for each of these variants was 5.56% compared with <1% in both non‐SUDEP PWE and healthy controls used as comparators in the study (see Table S1 for detailed statistics regarding frequency for all variants). All were Sanger‐confirmed except the variant in PIK3C2A. The LGI1gene is associated with autosomal‐dominant partial epilepsy with auditory features and COL6A3 is associated with muscular dystrophies (from mild to severe phenotypes).54, 55 The other genes are involved in cellular signaling but have no reported association with epilepsy, arrhythmia, or sudden death.

Another study from Australia focused exclusively on all‐cause mortality and SUDEP in 100 unrelated people with DS, of whom 87% were genotyped antemortem for pathogenic variants in SCN1A.29 Ten SUDEP events occurred with a SUDEP incidence rate of 9.32 (95% CI 4.46, 19.45) per 1000 person‐years. SUDEP cases were classified as definite in 30%, definite‐plus in 10%, and probable in 60%. SUDEP events were unwitnessed in 90% of cases and 60% occurred during sleep. The most frequent types of mutations in the SUDEP cases were truncations (40%), splice site (20%), deletions exons 1 to 24 (10%), and missense (10%). In the remaining 20%, the mutation was unclear.

A recent study from Japan reported on 9 definite SUDEP cases with full autopsies and performed targeted whole exome sequencing focusing on 73 known inherited cardiovascular disease–related genes. Six variants were identified; 3 are known to be pathogenic (LDB3, DSC2, and KCNE1) and 3 are considered potentially pathogenic variants predicted by in silico modeling (MYH6, DSP, and DSG2).31 ECGs were available in 6 of the 9 SUDEP cases and 3 had a mildly prolonged QTc interval for their age and sex (470, 469, and 475 ms). One of these cases carried the KCNE1 (p.Asp85Asn) variant, which is a polymorphism present in 1% to 24% of whites but may be considered pro‐arrhythmic in the setting of QT‐prolonging drugs.56 This study also detected variants normally associated with structural proteins such dystrophin (DMD) associated with dilated cardiomyopathy, desmoglein (DSG2) and desmoplakin (DSP) associated with arrhythmogenic ventricular cardiomyopathy (AVC), MYH6 associated with both dilated cardiomyopathy and hypertrophic cardiomyopathy, and LDB3 associated with both noncompaction cardiomyopathy and dilated cardiomyopathy. Although there are notable cases of inherited ion channelopathies associated with seizure disorders (because of the overlap of heart and brain ion channel expression), seizures associated with inherited structural heart disease is unusual.14, 15, 57, 58 One of the SUDEP cases carried 3 variants: a possibly arrhythmogenic variant of KCNE1 (p.Asp85Asn), a known likely pathogenic variant in DSC2 (p.Thr275Met), and a likely pathogenic variant in DSG2 based on in silico modeling. This case demonstrated fibrofatty replacement of both the left and right ventricles. It is plausible that this was a rare case of a patient with early “hot phase” arrhythmogenic ventricular cardiomyopathy, EEG‐confirmed seizures and undiagnosed LQT1, with the combination increasing likelihood of SUDEP. However, the variants in DSG2 have not been characterized or reported by other groups and may be benign bystanders. One SUDEP case had hypertrabeculation of the left ventricle and mild dilatation of the right ventricle on gross inspection, in addition to a known pathogenic variant of LDB3 as well as an in silico predicted pathogenic variant in MYH6. However, hypertrabeculation does not necessarily mean the presence of noncompaction cardiomyopathy because this can be seen in normal individuals and those with idiopathic dilated cardiomyopathy.59 This case also had conduction system disease with a notable decrease in the density of fibers of the sinoatrial node, idiopathic generalized seizures, a normal ECG, and no detectable anti‐epileptic drugs on postmortem testing. Similar to the preceding case, it is plausible there was dual pathology, which together increased the likelihood of sudden death. The major strength of this article was the comprehensive autopsy reporting on detailed cardiac structural changes that were not known during life. Some of these changes such as nonspecific fibrosis and mild degrees of fatty replacement can be seen in “normal” hearts and reflect old healed myocarditis. Nevertheless, they underscore the importance of comprehensive autopsy evaluation for phenotyping to guide subsequent molecular autopsies. The same group also evaluated for an overlooked and important mechanism in sudden death: cardiac conduction system disease and bradyarrhythmias directly causing sudden cardiac deaths, including bradyarrhythmia‐related tachyarrhythmias (eg, QT prolongation and torsade de pointes). One of the SUDEP cases had markedly decreased sinoatrial node tissue, which may have contributed to undiagnosed sinus node dysfunction. Although most sinus node dysfunction is generally safe provided the remaining cardiac conduction system is intact, a bradycardic episode with a prolonged repolarization period may be sufficient to increase the propensity for a malignant arrhythmia in a PWE and altered neural circulatory control. With the onset of an abrupt seizure, the “perfect storm” of SUDEP could occur. The pathology may also be the result of failure to generate an appropriate tachycardia. In the MORTEMUS (Mortality in Epilepsy Monitoring Unit Study) study of video‐recorded SUDEP events from epilepsy monitoring units, the most frequent arrhythmia was bradycardia and asystole.60

Finally, 2 studies from the United States focused on dup(15) changes, autism, autism‐related epilepsy, and SUDEP.30, 33 Both series showed a high frequency of epilepsy (up to 84.3%) with dup(15), with the leading cause of death being a SUDEP event (67% and 43%). While autism is known to have a higher frequency of epilepsy when compared with the general population including mixed types of generalized tonic–clonic, generalized absent and focal, there was also a high frequency of Lennox‐Gastaut syndrome. Notably, autism with dup(15) was associated with SUDEP in 67% of cases compared with 10% in PWE and autism without dup(15).33

The summarized risk of bias for the included studies is described in Table 5.

Risk of Bias Assessment of the Included Studies

### Characteristics of Patients

The mean age of patients at diagnosis of epilepsy or presentation with first seizures was 10.3 (SD ±9.8) years, with age at SUDEP of 29.0 (SD ±14.2) years. Cases were more likely to be male (61%).These data are consistent with published data on SUDEP cases.13, 34, 35, 36, 37 In 7 studies, all patients had a history of documented seizures. Four studies were from Australia with a total of 139 patients,21, 27, 28, 29 2 from the United States with a total of 14 patients,30, 33 1 from the United Kingdom with 18 patients, and 1 from Japan with 9 patients.31 Some of the Australian studies included patients from prior series; after removal of duplicates, the final total of patients was 161. The characteristics of patients are summarized in Table 1.

Characteristics of the Included Studies

Autism‐related

Mixed

6 (33%) DS

5 (28%) focal specified

4 (22%) focal unspecified

1 (6%) generalized

2 (11%) unspecified

6 (66%) focal

3 (34%) generalized

ADHD indicates attention deficit hyperactivity disorder; DS, Dravet syndrome; dup, duplication; LGS, Lennox‐Gastaut syndrome; LQT, long QT; NA, not available or reported; SCD, sudden cardiac death; SUDEP, sudden unexpected death in epilepsy; WES, whole exome sequencing.

In years.

HUGO gene nomenclature committee gene names.

Includes 48 patients from earlier study by Tu et al (2011).

Three patients with borderline QTc prolongation.

Includes 19 patients from earlier study by Tu et al (2011).

Median (range).

The majority of studies did not report ECG findings or explain whether an ECG was available. One study reported that de‐identified samples were received and ethical requirements prohibited obtaining detailed clinical information including ECGs.21

### SUDEP and Epilepsy Classification

SUDEP cases were classified into definite in 83 (51.6%), definite‐plus in 6 (3.7%), probable in 26 (16.1%), and possible 46 (28.6) as per Nashef et al (Table 2).1 In brief, SUDEP is categorized per the following criteria: (1) definite if an autopsy does not reveal a cause of death (including patients with a structurally normal heart); (2) definite‐plus if an autopsy was performed and there may be an alternate contributing factor; (3) probable if no autopsy is performed; (4) possible if there is a competing cause of death; and (5) unknown if there is no autopsy, no witness, and limited records.1, 38 Two studies focused on autism with autism‐related epilepsy and SUDEP. One study did not report the types of epilepsy33 and the other included 2 (25%) with generalized seizures related to autism, 2 (25%) with Lennox‐Gastaut syndrome and no known cause for the remainder.30 In 2 other studies the classification of epilepsy could not be determined because it was not reported for each case, but the discussion commented on including generalized, focal alcohol‐related and post‐traumatic causes, with the rationale for the latter 2 stating these events may have precipitated seizures or contributed to lowering seizure threshold in susceptible individuals.21, 33 In 1 study, this information was not reported and an earlier study using some of the same patients was cited.27 One study reported a range of epilepsy types without defined frequencies including generalized, temporal lobe epilepsy, postencephalopathic, and juvenile myoclonic epilepsy.28 In a study from the United Kingdom, 33% were Dravet syndrome (DS) patients, 28% focal specified, 22% focal unspecified, 6% generalized, and 11% unspecified.32 One study from Australia of 10 cases focused exclusively on cases of DS.29 A study from Japan included 66% with focal and 34% with generalized seizures.31 Table 3 summarizes inclusion and exclusion criteria of the cohorts.

Classification of SUDEP and Time of Death

SUDEP indicates sudden unexpected death in epilepsy.

Includes 48 patients from earlier study by Tu et al (2011).

Includes 19 patients from earlier study by Tu et al (2011).

Inclusion and Exclusion Criteria of Individual Studies

9 subjects with duplications of chromosome 15q11.2‐q13 [dup(15)]

10 subjects with autism

7 control subjects

Patients with epilepsy in whom postmortem examination failed to reveal a cause of death

SUDEP or possible SUDEP as cause of death

18 with epilepsy who died of SUDEP (probable or definite)

87 living people with epilepsy (controls)

1479 nonepilepsy disease control samples

61 SUDEP cases were recruited from 3 sources:
27 had participated in the epilepsy genetics research program in Melbourne, Australia, during life and had SUDEP on follow‐up;15 prospective coronial SUDEP cases were collected from 2010 to 2012 by the Departments of Forensic Medicine (DOFM) in New South Wales, Victoria, Queensland, and South Australia; and19 retrospective coronial SUDEP cases were collected from a review of autopsy reports over a 17‐y period from 1993 to 2010 at the DOFM in Sydney.

27 had participated in the epilepsy genetics research program in Melbourne, Australia, during life and had SUDEP on follow‐up;

15 prospective coronial SUDEP cases were collected from 2010 to 2012 by the Departments of Forensic Medicine (DOFM) in New South Wales, Victoria, Queensland, and South Australia; and

19 retrospective coronial SUDEP cases were collected from a review of autopsy reports over a 17‐y period from 1993 to 2010 at the DOFM in Sydney.

Cases classified as definite SUDEP, definite SUDEP plus, probable SUDEP, or possible SUDEP

Typical electroclinical phenotype of Dravet Syndrome

Mortality and SUDEP rates estimated in 100 cases of Dravet, 87 had SCN1A mutation

Dup indicates duplication; SUDEP, sudden unexpected death in epilepsy.

Some of the cohort was previously reported by Tu et al (2011).

### Body Position and Time When Found Dead

Three of 8 studies did not report the position of the body.29, 30, 32 Two studies were unable to determine this because of de‐identified and limited clinical data.21, 33 In the remaining 3 studies (n=138), the majority of decedents (50.7%) were found in the prone position; in 43.4% of cases, the position was unknown. Only 2 studies reported time of death: a study from the United States on DS reported 6 (60%) nocturnal deaths and a study from Japan including multiple epilepsy syndromes reported 5 (56%) nocturnal deaths.29, 31

### Study Design and Genetic Techniques

Five of the 8 studies were retrospective with molecular autopsies to identify pathogenic variants. Two of the studies focusing on duplications involving chromosome 15 [dup(15)] used a combination of techniques including fluorescent in situ hybridization, genotyping for copy number variant (CNV) analysis, and Southern blotting, likely reflecting historical use of technologies for case evaluation (Table 4).30, 33 These were all performed antemortem. Two studies used Sanger sequencing exclusively: 1 focused on the top 3 LQTS loci to detect frequency of known and novel variants in SUDEP cases, while the other assessed polyalanine repeat expansion alleles.21, 27 Two studies used next‐generation whole exome sequencing: 1 study compared cases to controls to identify deleterious variants and the other targeted analysis to known cardiac arrhythmia, epilepsy‐related, and respiratory control genes.28, 32 The remaining 2 studies used a combination of next‐generation and Sanger sequencing.29, 31 One study assessed the mitochondrial genome and reported no associations.28 Three studies also assessed CNVs: 2 studies focused on duplications involving chromosome 15 [dup(15)] of varying magnitude and 1 study assessed 41 (67.2%) of decedents for CNVs, in arrhythmia, respiratory, and epilepsy‐related genes, detecting no differences.

Sequencing Techniques Utilized and Genes Screened

Genotyping

FISH

Southern blot

Array CGH

1 known pathogenic in DEPDC5

1 pathogenic in PAFAH1B1

Targeted NGS

Sanger

BrS indicates Brugada syndrome; chr, chromosome; CNV, copy number variants; FISH, fluorescent in situ hybridization; Mito, mitochondrial; NGS, next‐generation sequencing; VUS, variants of unknown significance; WES, whole exome sequencing.

HUGO gene nomenclature committee gene names.

Includes 48 patients from earlier study by Tu et al (2011).

Includes 19 patients from earlier study by Tu et al (2011).

### Types of Variants Identified

Of the reported genetic variants, the vast majority were caused by substitutions. Most of the variants were in ion channel subunits (n=21; 13%). Duplications accounted for 14 (8.7%) variants. The variants identified as well as the comparable frequency in a general epilepsy or control population where available are listed in Table S1.

### Genes Identified and Characteristics

Eighteen genes and 4 different duplications were reported to have a possible association with SUDEP: KCNH2, SCN5A, KCNQ1, SCN1A, LGI1, PIK3C2A, SMC4, COL6A3, TIE1, DSC2, LDB3, KCNE1, MYBPC3, MYH6, DSP, DSG2, DMD, isodicentric chromosome 15[idic(15)], derivative chromosome 15 [der(15)], tricentric chromosome 15, and triplication of chromosome 15[trp(15)] (Table 4). The vast majority of variants were classified as variants of unknown significance. The genes reported to be associated with SUDEP included cation (sodium, potassium) channel protein subunits. Only 2 studies had overlap in the reported genes. These genes were SCN5A, known to cause LQT3 (via gain of function) and Brugada syndrome (BrS; via loss of function), and SCN1A, which is associated with DS, familial hemiplegic migraine and genetic epilepsy with febrile seizures. Figure 2 illustrates known genes associated with a variety of genetic cardiac disorders, as well as whether these disorders have a reported overlap with SUDEP. While we did not identify variants in all the genes included in this figure, we included it to depict a summation of cardiac genetic disorders that possibly contribute to the development of SUDEP.

Overview of complexity of genes implicated in SUDEP and other genetic cardiovascular diseases associated with sudden death. The figure demonstrates known genes associated with each of the following conditions: BrS, 5HT receptor mutations, ERS, LQTS, ARVC, short QT, HCM, CPVT, DCM, and primary brain SUDEP. The orange syndromes represent genetic cardiac disorders as well as primary brain SUDEP. The surrounding boxes connected via a black solid line represent genes of which a variant can result in the clinical syndrome to which it is connected. Blue boxes represent gene variants that have only 1 identified genetic cardiac phenotype of the diseases included. Purple boxes represent genes with variants that have 2 identified phenotypes. For example, variants in the gene KCNH2 can result in both long QT syndrome and short QT syndrome. Similarly, variants in RYR2 are associated with CPVT, DCM, and ARVC. However, of the included cardiac conditions, variants in SCN1B are typically only associated with BrS. The centrally pointing red arrows represent the potential contribution of select genetic cardiac disorders to the central clinical entity of primary brain SUDEP. ARVC indicates arrhythmogenic right ventricular cardiomyopathy; BrS, Brugada syndrome; CPVT, catecholaminergic polymorphic ventricular tachycardia; DCM, dilated cardiomyopathy; ERS, early repolarization syndrome; HCM, hypertrophic cardiomyopathy; 5HT, serotonin; LQTS, long QT syndrome; SUDEP, sudden unexpected death in epilepsy.

One study of 68 SUDEP patients specifically targeting LQT1 (KCNQ1), LQT2 (KCNH2), and LQT3 (SCN5A) identified 2 nonsynonymous variants in KCNH2 and 4 in SCN5A, which have all been reported in cases of LQTS and functionally characterized as pathogenic.21, 39, 40, 41, 42, 43, 44, 45, 46, 47, 48 One of the 6 cases carried 2 nonsynonymous polymorphisms: p.Arg1047Leu‐KCNH2 and p.Ala572Asp‐SCN5A. There was no reported seizure activity 12 months preceding death and the patient died at an older age of 52 years. The SCN5A variant has been reported in LQTS, sudden infant death syndrome, BrS, and multiple cases of female victims of sudden cardiac deaths but has now been classified as likely benign.

A subsequent study from the same group, utilizing 48 of the 68 SUDEP cases from the aforementioned study, was combined with an additional 20 cases from an alternate source that were specifically screened for polyalanine repeat expansions in the homeobox PHOX2B gene.27
PHOX2B has been implicated in dysfunction of respiratory control, including congenital central hypoventilation syndrome, an autonomic disorder that can lead to hypoventilation and a blunted chemoreceptor response to hypercapnia and hypoxia. Additionally, alterations in the gene have been associated with bradyarrhythmias and coexisting epilepsy.49, 50, 51 Two synonymous variants were identified in 4 (5.9%) SUDEP decedents categorized as variants of unknown significance; however, no polyalanine repeat expansion alleles or point mutations were identified, suggesting against PHOX2B as a major genetic contributor to SUDEP.

In a third study by the same group, 61 SUDEP cases (including 19 from an earlier study) underwent whole exome sequencing with targeted analysis of 109 genes (the top 3 LQTS genes, 29 other cardiac arrhythmia genes, 5 genes involved in ventilation, and 72 epilepsy‐related genes).21, 28 This identified 6 (10%) cases with pathogenic variants in 2 LQTS genes (KCNQ1 and KCNH2 as reported previously).21 No known pathogenic variants were identified in the other cardiac arrhythmia or respiratory control genes, and 2 known pathogenic variants were identified in epilepsy‐related genes (DEPDC5 and PAFAH1B1). Nine candidate variants in cardiac arrhythmia genes were identified, none of which had undergone functional characterization. Therefore, classification was based on in silico prediction. No candidate pathogenic variants were identified in the respiratory control genes. An additional 10 candidate pathogenic variants were identified in epilepsy‐related genes (Table 4 and Table S1), which were also based on in silico prediction. Of these, DEPDC5 nonsense variants were considered to confer the greatest risk because these included mutations in highly conserved areas and were identified in 6 unrelated SUDEP cases. In addition, the investigators conducted mitochondrial genome analysis and did not detect any known pathogenic variants. Similarly, CNV analysis did not detect any significant deletions or duplications.

A study from the United Kingdom performed whole exome sequencing on 18 decedent cases and compared them with both 87 living PWE and 1479 nonepilepsy disease controls to identify likely deleterious variants.32 The frequency of DS in living PWE was 34.5% (n=30) and SUDEP cases was 33% (n=6). Of the living PWE with DS, 26 (87%) had a known pathogenic variant in SCN1A compared with all 6 of the decedent SUDEP cases with DS. Of the 89 512 variants identified (in SUDEP, living PWE, and nonepilepsy controls), these linked to 13 887 genes and a high genomewide burden score per individual, suggesting a polygenic contribution to SUDEP causation. Likely deleterious variants in 373 genes were identified exclusively in the SUDEP group, 1 of which was CACNB2 associated with BrS (the exact variant is not reported and is predicted by in silico modeling only).52, 53 Other than the known SCN1A pathogenic variants in DS cases, no known sudden death or genetic cardiac disease–related pathogenic variants were identified. However, 5 additional candidate pathogenic variants were identified in other genes: LGI1, PIK3C2A, SMC4, COL6A3, and TIE1. The cumulative minor allele frequency for each of these variants was 5.56% compared with <1% in both non‐SUDEP PWE and healthy controls used as comparators in the study (see Table S1 for detailed statistics regarding frequency for all variants). All were Sanger‐confirmed except the variant in PIK3C2A. The LGI1gene is associated with autosomal‐dominant partial epilepsy with auditory features and COL6A3 is associated with muscular dystrophies (from mild to severe phenotypes).54, 55 The other genes are involved in cellular signaling but have no reported association with epilepsy, arrhythmia, or sudden death.

Another study from Australia focused exclusively on all‐cause mortality and SUDEP in 100 unrelated people with DS, of whom 87% were genotyped antemortem for pathogenic variants in SCN1A.29 Ten SUDEP events occurred with a SUDEP incidence rate of 9.32 (95% CI 4.46, 19.45) per 1000 person‐years. SUDEP cases were classified as definite in 30%, definite‐plus in 10%, and probable in 60%. SUDEP events were unwitnessed in 90% of cases and 60% occurred during sleep. The most frequent types of mutations in the SUDEP cases were truncations (40%), splice site (20%), deletions exons 1 to 24 (10%), and missense (10%). In the remaining 20%, the mutation was unclear.

A recent study from Japan reported on 9 definite SUDEP cases with full autopsies and performed targeted whole exome sequencing focusing on 73 known inherited cardiovascular disease–related genes. Six variants were identified; 3 are known to be pathogenic (LDB3, DSC2, and KCNE1) and 3 are considered potentially pathogenic variants predicted by in silico modeling (MYH6, DSP, and DSG2).31 ECGs were available in 6 of the 9 SUDEP cases and 3 had a mildly prolonged QTc interval for their age and sex (470, 469, and 475 ms). One of these cases carried the KCNE1 (p.Asp85Asn) variant, which is a polymorphism present in 1% to 24% of whites but may be considered pro‐arrhythmic in the setting of QT‐prolonging drugs.56 This study also detected variants normally associated with structural proteins such dystrophin (DMD) associated with dilated cardiomyopathy, desmoglein (DSG2) and desmoplakin (DSP) associated with arrhythmogenic ventricular cardiomyopathy (AVC), MYH6 associated with both dilated cardiomyopathy and hypertrophic cardiomyopathy, and LDB3 associated with both noncompaction cardiomyopathy and dilated cardiomyopathy. Although there are notable cases of inherited ion channelopathies associated with seizure disorders (because of the overlap of heart and brain ion channel expression), seizures associated with inherited structural heart disease is unusual.14, 15, 57, 58 One of the SUDEP cases carried 3 variants: a possibly arrhythmogenic variant of KCNE1 (p.Asp85Asn), a known likely pathogenic variant in DSC2 (p.Thr275Met), and a likely pathogenic variant in DSG2 based on in silico modeling. This case demonstrated fibrofatty replacement of both the left and right ventricles. It is plausible that this was a rare case of a patient with early “hot phase” arrhythmogenic ventricular cardiomyopathy, EEG‐confirmed seizures and undiagnosed LQT1, with the combination increasing likelihood of SUDEP. However, the variants in DSG2 have not been characterized or reported by other groups and may be benign bystanders. One SUDEP case had hypertrabeculation of the left ventricle and mild dilatation of the right ventricle on gross inspection, in addition to a known pathogenic variant of LDB3 as well as an in silico predicted pathogenic variant in MYH6. However, hypertrabeculation does not necessarily mean the presence of noncompaction cardiomyopathy because this can be seen in normal individuals and those with idiopathic dilated cardiomyopathy.59 This case also had conduction system disease with a notable decrease in the density of fibers of the sinoatrial node, idiopathic generalized seizures, a normal ECG, and no detectable anti‐epileptic drugs on postmortem testing. Similar to the preceding case, it is plausible there was dual pathology, which together increased the likelihood of sudden death. The major strength of this article was the comprehensive autopsy reporting on detailed cardiac structural changes that were not known during life. Some of these changes such as nonspecific fibrosis and mild degrees of fatty replacement can be seen in “normal” hearts and reflect old healed myocarditis. Nevertheless, they underscore the importance of comprehensive autopsy evaluation for phenotyping to guide subsequent molecular autopsies. The same group also evaluated for an overlooked and important mechanism in sudden death: cardiac conduction system disease and bradyarrhythmias directly causing sudden cardiac deaths, including bradyarrhythmia‐related tachyarrhythmias (eg, QT prolongation and torsade de pointes). One of the SUDEP cases had markedly decreased sinoatrial node tissue, which may have contributed to undiagnosed sinus node dysfunction. Although most sinus node dysfunction is generally safe provided the remaining cardiac conduction system is intact, a bradycardic episode with a prolonged repolarization period may be sufficient to increase the propensity for a malignant arrhythmia in a PWE and altered neural circulatory control. With the onset of an abrupt seizure, the “perfect storm” of SUDEP could occur. The pathology may also be the result of failure to generate an appropriate tachycardia. In the MORTEMUS (Mortality in Epilepsy Monitoring Unit Study) study of video‐recorded SUDEP events from epilepsy monitoring units, the most frequent arrhythmia was bradycardia and asystole.60

Finally, 2 studies from the United States focused on dup(15) changes, autism, autism‐related epilepsy, and SUDEP.30, 33 Both series showed a high frequency of epilepsy (up to 84.3%) with dup(15), with the leading cause of death being a SUDEP event (67% and 43%). While autism is known to have a higher frequency of epilepsy when compared with the general population including mixed types of generalized tonic–clonic, generalized absent and focal, there was also a high frequency of Lennox‐Gastaut syndrome. Notably, autism with dup(15) was associated with SUDEP in 67% of cases compared with 10% in PWE and autism without dup(15).33

The summarized risk of bias for the included studies is described in Table 5.

Risk of Bias Assessment of the Included Studies

### Discussion

This systematic review summarizes the body of evidence related to the genetics of SUDEP and has included 8 studies with a total of 161 unique individuals. This review reveals some important findings. First, genes encoding for sodium and potassium ion channel subunits are the most frequently reported variants discovered by postmortem molecular autopsy, as well as those with the highest yield for known pathogenic or likely pathogenic variants. Second, DEPDC5, which encodes for a protein in the IML1 family involved in G‐protein signaling and is associated with autosomal‐dominant familial focal epilepsy, was the second highly ranked variant (using in silico prediction) in a large SUDEP series. Third, although DS is rare, given the high SUDEP rate and the majority carrying a pathogenic variant in SCN1A, these cases are more likely to be included in SUDEP series. Indeed, at least 11% of the pooled cases in this study were caused by DS. This percentage may be higher because some studies did not report the cause. Fourth, CNVs in chromosome 15 are associated with autism and a high frequency of epilepsy and SUDEP. It is conceivable that changes to CNVs in chromosome 15 modify the risk of SUDEP and this is underexplored in SUDEP cases without associated autism. Finally, most SUDEP events are likely oligogenic or polygenic, although we estimate 10% to 20% overlap with monogenic disorders based on our experience and the largest human study of SUDEP.28 Insufficient data prevented an aggregate or individual participant data‐based meta‐analysis to be conducted.

Aside from the pre‐autopsy known cases of DS with SCN1A mutations and known dup(15) cases, which together make up at least 31 (19.3%) of the cases of SUDEP in this systematic review, the most frequent pathogenic variants identified postmortem remain in ion channel and arrhythmia‐related genes. In the 161 unique cases, 7 known pathogenic variants and 9 in silico models predicted as highly likely pathogenic variants for a total of 16 (11%). However, the in silico prediction alone would not meet American College of Medical Genetics and Genomics criteria for classifying variants. One of the variants was described as pathogenic, but is a pro‐arrhythmogenic functional polymorphism (KCNE1‐D85N). The association between LQTS, catecholaminergic polymorphic ventricular tachycardia, and BrS with either a dual co‐existing seizure phenotype or secondary arrhythmia‐related seizures is recognized.14, 15, 57, 58, 61, 62, 63, 64, 65, 66, 67 It is somewhat surprising the yield is not higher, although this could be because of a number of reasons: lack of a 12‐lead ECG, insufficient or absent postmortem blood for DNA, or misclassification as a non‐SUDEP death. These estimates represent a minimum frequency and may improve with dissemination of the need to systematically and comprehensively investigate SUDEP cases, including storing postmortem blood, which should result in larger‐scale studies.

The structural cardiac genes identified in 2 studies are unusual but should not be overlooked. It is possible some of these are benign variants. However, the Japanese study included detailed comprehensive autopsy examination, which suggested an early arrhythmogenic ventricular cardiomyopathy phenotype, and this underscores the importance of seeking out dual pathology. Arrhythmogenic ventricular cardiomyopathy is a difficult diagnosis to make, even by experienced cardiologists and pathologists. It is particularly difficult in the early phases of the disease, which is highly arrhythmogenic but presents with only subtle microscopic changes.68 The other important variable to consider is subtle damage to the heart from repeated seizures as nonspecific fibrosis has been described in SUDEP, as well as abnormal perfusion in vivo in the absence of epicardial coronary artery disease.19

There are a number of additional points gleaned from this systematic review, which if addressed could strengthen research on SUDEP. There is a dearth of cases undergoing comprehensive autopsy examination by a neuropathologist for the nervous system and a cardiac pathologist for the heart despite guidelines recommending that these organs undergo evaluation by a subspecializing pathologist.2, 66, 69, 70, 71, 72 Furthermore, postmortem blood is not always retained for molecular autopsies. This hampers progress in elucidating the genetic contribution to the associated pathology. None of the reported studies had blood and sufficient DNA from all decedents and focused analysis on a subset with sufficient DNA, thereby introducing error. Had mandatory collection of postmortem blood in epilepsy cases or antemortem biobank collection been performed, this would have facilitated genomic, epigenomic, and gene‐expression analyses. Improvement in both sample collection and clinical data collection would allow for more in‐depth retrospective phenotyping and accurate capturing of SUDEP. Continued reduction of genetic testing costs may make this process more feasible. The use of formalin‐fixed paraffin‐embedded tissue as a source of DNA for archival series of SUDEP may provide a solution to the current limitations.73

Furthermore, the majority of studies did not report or obtain ECG data, which are crucial to determining an accurate phenotype, particularly since rhythm disorders of the heart cannot be diagnosed postmortem. Although genetic testing for the top 3 LQTS genes can identify a pathogenic variant in ≈75% of cases with ECG‐confirmed LQTS, there are a significant number (10–20%) of patients who have variants in LQTS genes and who do not express any phenotypic symptoms.74 There are also those who are digenic or compound heterozygotes. The requirement of an ECG is essential in BrS where penetrance of known pathologic variants (eg, SCN5A) is low.75 Thus, we advocate that a 12‐lead ECG should be performed on PWE and be made available to the pathologist with a specific comment on the autopsy report.

Basic science research using murine models of epilepsy have shown a relationship with refractory epilepsy and sudden death. For example, KCNA1 knockout mice show seizures, cardiac arrhythmias, increased vagal tone, and premature death76 and subsequently validated in a human SUDEP case.77 These models have also shown abnormal neural‐circulatory control, which can be treated to reduce the frequency of ictal bradycardia and SUDEP.78 A murine knockout model of SCN1B expresses spontaneous seizures and QT prolongation and early mortality. The SCN1B gene is linked to generalized epilepsy with febrile seizures plus (GEFS+), temporal lobe epilepsy, and DS.79, 80 A murine model with deficiency in glutamic acid decarboxylase isoform (GAD65) displays spontaneous epilepsy and premature mortality.81 Antibodies to GAD65 have been reported in a case of immune‐mediated epilepsy and bitemporal ictal asystole, and although death did not occur, ictal asystole could be a SUDEP mechanism.82 The role of serotonin (5‐hydroxytryptamine) has been linked to both SUDEP and sudden infant death syndrome, because 5‐hydroxytryptamine plays a critical role in respiration and arousal.83 Murine models deficient in the 5‐hydroxytryptamine 2c receptor are prone to epilepsy and premature death; mice without 5‐hydroxytryptamine neurons develop apnea, hypercapnia, blunted chemoreceptor sensitivity, and premature mortality.84, 85 Mice deficient in DEPDC5 display epilepsy, and enlarged brains with malformations;86 2 human cases of definite SUDEP in a single family with DEPDC5‐related epilepsy have been reported.87

Clarity is also required regarding the interpretation of what constitutes a SUDEP event. The current definition requires nontraumatic, nondrowning, non–status epilepticus, unexpected death in an otherwise healthy PWE. Definitions provided by experts in the field and described earlier in this review provide some guidance. However, there is often co‐existing cardiac pathology such as bystander coronary artery disease and some interpret this to be non‐SUDEP. We believe the presence of bystander coronary artery disease in an epicardial coronary artery with normal origin and course, and without evidence of an acute plaque event, ischemia, or infarction, should still be considered as definite SUDEP. Some of the studies acknowledged this, including 1 study that also included a patient with co‐existent pneumonia as a definite SUDEP.21, 27, 28 However, other studies excluded these cases as non‐SUDEP.30 It is plausible that some of these misclassified cases are harboring a potentially pathogenic variant in ion channel or epilepsy‐related genes, and perhaps fever in the context of pneumonia triggered a fatal sudden death event.

There are several strengths of this study. Given that SUDEP is a rare event and underinvestigated, we have systematically searched the literature on human decedent SUDEP cases, including data from international SUDEP studies with genetic data. This has also highlighted some of the inconsistencies between studies and the lack of important phenotyping. There are a number of limitations including a reporting bias of DS and dup(15) studies. This is likely because of sampling bias from studies that focused on evaluation exclusively of these populations. While the reviewers systematically assessed risk of bias and methodological quality via standardized tools, the ability to generalize the findings of this study to the general population may be limited. Additionally, there are many ion channel subunit genes associated with epilepsy.88 The finding that the literature demonstrates a preponderance of variants related to ion channels is not surprising compared with other potential variants with fewer associated genes. Lastly, we were unable to perform a meta‐analysis because of a lack of sufficient data.

We support a team science approach with collaborations between centers and colleagues in disciplines including pathology, neurology, epileptology, cardiology, and genetics to pool cases and resources for better understanding of the SUDEP conundrum. SUDEP remains the leading cause of epilepsy‐related death with little progress in screening or prevention. Addressing this could save many potential years of life. Interested readers can visit the Partners Against Mortality in Epilepsy website for more information including current research, collaboration opportunities, and public policy efforts: https://pame.aesnet.org/.

### Strengths and Weaknesses

There are several strengths of this study. Given that SUDEP is a rare event and underinvestigated, we have systematically searched the literature on human decedent SUDEP cases, including data from international SUDEP studies with genetic data. This has also highlighted some of the inconsistencies between studies and the lack of important phenotyping. There are a number of limitations including a reporting bias of DS and dup(15) studies. This is likely because of sampling bias from studies that focused on evaluation exclusively of these populations. While the reviewers systematically assessed risk of bias and methodological quality via standardized tools, the ability to generalize the findings of this study to the general population may be limited. Additionally, there are many ion channel subunit genes associated with epilepsy.88 The finding that the literature demonstrates a preponderance of variants related to ion channels is not surprising compared with other potential variants with fewer associated genes. Lastly, we were unable to perform a meta‐analysis because of a lack of sufficient data.

### Future Directions

We support a team science approach with collaborations between centers and colleagues in disciplines including pathology, neurology, epileptology, cardiology, and genetics to pool cases and resources for better understanding of the SUDEP conundrum. SUDEP remains the leading cause of epilepsy‐related death with little progress in screening or prevention. Addressing this could save many potential years of life. Interested readers can visit the Partners Against Mortality in Epilepsy website for more information including current research, collaboration opportunities, and public policy efforts: https://pame.aesnet.org/.

### Conclusions

SUDEP case adjudication and evaluation remain limited. The most frequent known pathogenic variants and likely pathogenic novel variants identified by molecular autopsy are in ion channel or arrhythmia‐related genes with an ≈11% discovery rate. The most frequent known genetic defects antemortum are SCN1A in DS and dup(15) associated with autism in PWE. ECG use in SUDEP evaluation is poor, either through under‐reporting or lack of availability to investigators. Comprehensive postmortem examination of the decedent should include examination of the heart and brain by cardiac and neuropathologists, respectively.

### Sources of Funding

This work was supported by the National Center for Research Resources and the National Center for Advancing Translational Sciences, NIH, through grant number 1 UL1 TR002377. Chahal and Somers are supported by NIH HL65176 and NIH HL134885. Chahal is supported by the American Heart Association (award number 17POST33400211). Ackerman is supported by the Mayo Clinic Windland Smith Rice Comprehensive Sudden Death Program. Gottwald is supported by the Yale University‐Mayo Clinic Center of Excellence in Regulatory Science and Innovation Scholar Award. The contents of this article are solely the responsibility of the authors and do not necessarily represent the official view of the NIH.

### Disclosures

No off‐label medication use is reported. St. Louis reports that he receives research support from the Mayo Clinic Center for Translational Science Activities (CCaTS), supported by the National Center for Research Resources and the National Center for Advancing Translational Sciences, NIH, through Grant Number 1 UL1 RR024150‐01. Ackerman is a consultant for Audentes Therapeutics, Boston Scientific, Gilead Sciences, Invitae, Medtronic, MyoKardia, and St. Jude Medical. Ackerman and Mayo Clinic have an equity/royalty relationship with AliveCor, Blue Ox Health Corporation, and StemoniX. However, none of these entities were involved in this study in any way. Somers has served as a consultant for ResMed, Phillips, GlaxoSmithKline, Respicardia, Ronda Grey, Biosense Webster, Dane Garvin, Bayer, Itamar, and U‐Health. He works with Mayo Health Solutions and their industry partners on intellectual property related to sleep and to obesity. The remaining authors have no disclosures to report.

### Supporting information

Data S1. Supplemental methods.

Table S1. Key Genes and Loci Identified

Click here for additional data file.



# SUPPLEMENTAL FILE 1: JAH3-9-e012264.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)