# MAIN TEXT

## Analysis of Comprehensive Pharmacogenomic Profiling of VIP Variants Among the Genetically Isolated Chechen Subpopulation from Jordan

### Abstract

BackgroundProfiling rare variants in isolated populations can significantly clarify and understand the development of a clinically relevant process. Therefore, leading to a better identifying novel targeted treatment.ObjectiveThis study aimed to determine the allele frequencies of 56 single nucleotide polymorphisms (SNPs) within several important pharmacogenes.MethodsThis study consisted of 166 unrelated subjects from a genetically isolated group (Chechen) who were living in Jordan. In this study, the distribution of the variants among Chechen was compared to other ethnic groups available at two databases (Genome 1000 and (ExAC)). The frequency of genotypes and alleles was calculated and tested using the chi-square test and the Hardy–Weinberg equilibrium equation (HWE).ResultsOur results revealed that the distribution of allele frequencies within different pharmacogenes among Chechen showed different similarities with other populations. The CEU and TSI showed the highest resemblance with the Chechen population (75% similarity), in contrast to LWK which had the lowest similarity (30%).ConclusionThis study sheds light on clinically relevant SNPs to enhance medical research and apply pharmacogenomics in clinical settings.

### Introduction

Pharmacogenomics has become a promising future to enhance individuals’ response to treatment.1 Single nucleotide polymorphisms (SNPs) are one of the most common variants among populations that affect enzyme activity as well as drug metabolism.2 Consequently, clinical significant SNPs have been studied and screened extensively in different genes in different ethnic backgrounds and given the term very important pharmacogenomic (VIP) variants.3

These VIP variants help in drug dose optimization for patients relevant to their genetic makeup and lead to enhanced therapy outcomes and maintain the efficacy of drug treatment.4 Remarkably, the frequency distribution of these VIP variants varies among different ethnic groups.5 The importance of the VIP variants depends on their presence within critical genes including drug-metabolizing genes.6 However, the underlying mechanism of the correlation between the genetic variants and drug response is still beyond clinical practice.7 Rare variants that impact gene function may account for some uncovered differences in the pharmacological response and metabolism process.8 Genetic variants in drug target enzymes; metabolizing, transporters, receptors help in predicting toxicity and treatment failure among patients.9

Genetically isolated populations, such as Chechen, can help medical research to overcome the extensive allelic heterogeneity which is a major cause of complex diseases.10 Chechen, a Caucasian ethnic group of the Nakh peoples, lived in the highlands of the North Caucasus region of Eastern Europe.11 In 1860 they immigrated to Ottoman lands; however, today, tens of thousands of Chechens live in Turkey, Jordan, Syria, Iraq, Egypt, Saudi Arabia and other Persian Gulf countries.12 Not many genetic studies have been conducted on Chechens and Circassians living in Jordan. As a genetically isolated population, they can provide a baseline for future studies to discover novel markers associated with diseases.13–16 Moreover, there is an urgent need for pharmacogenetic research as not many studies conducted in Jordan.17,18 Therefore, this study aimed to determine the allele frequencies of 52 SNPs within several pharmacogenes in Chechen living in Jordan in comparison to other ethnic groups.

### Materials and Methods

This study was subjected to and in agreement with the Human ethics committee at National Center for Diabetes, Endocrinology and Genetics (NCDEG) and Jordan University of Science and Technology (JUST). Details about subjects and written informed consent were obtained from all volunteers in the study. Ten milliliters of venous blood was collected from unrelated healthy 166 Chechen subjects living in Jordan.

This study was conducted in agreement with the Human ethics committee at National Center for Diabetes, Endocrinology and Genetics (NCDEG) and Jordan University of Science and Technology (JUST), policy number (GM7601). Written informed consent was obtained from all volunteers in the study. This research was also conducted in accordance with the Declaration of Helsinki.

Blood samples were drawn from subject and then stored at 4°C for 24 hours before the purification of the genetic material. The extracted DNA was then stored at −20°C.

Genotyping was performed using Agena Bioscience MassARRAY® on a Compact Spectrometer iPLEX GOLD chemistry by Australian Genome Research Facility. Genomic DNA was extracted from each blood sample using the Wizard® Genomic DNA Purification Kit (Promega Corporation, USA) according to the manufacturer’s instructions. The quality and quantity of the purified DNA were ascertained via agarose gel electrophoresis and the Nano-Drop ND-1000 UV-Vis Spectrophotometer (BioDrop, UK), respectively.

Candidate genes polymorphisms were analyzed using Sequencing technique. Loci of candidate SNPs were amplified using Multiplex PCR followed by Mass EXTEND which is a primer extension process is resulting in allele-specific DNA products. The genotyping after the minisequencing reaction product analysis was reported by Mass spectrometry. After that, the extension PCR products were separated onto a 384 well spectroCHIP and placed into the MALDI-TOF (Matrix-Assisted Laser Desorption/Ionization Time-of-Flight) mass spectrometer. Lastly, a software system (SpectroTYPER-RT (RT for real-time)) was used to analyze the results.

The studied VIP variants were selected from the public databases including the PharmGKB database (https://www.pharmgkb.org/), the SNP database of the National Centre for Biotechnology Information (NCBI) (http://www.ncbi.nlm.nih.gov/SNP/), and the International HapMap Project (http://www.hapmap.org/). SNPs selected within significant genes that play roles in drug metabolism which make their variants clinically important variants. Finally, the primers were used to genotype these VIP variants were shown in the Supplementary Material.

The allele counts variation data for other populations were obtained from HapMap Project website: CHE: Chechens from Jordan, ASW: African ancestry in Southwest USA, CEU: Utah, USA residents with Northern and Western European ancestry from the CEPH collection, CHB: Han Chinese in Beijing, China, CDX: Chinese Dai in Xishuangbanna, China, GIH: Gujarati Indians in Houston, Texas, USA, GBR: British in England and Scotland, JPT: Japanese in Tokyo, Japan, LWK: Luhya in Webuye, Kenya, MXL: Mexican ancestry in Los Angeles, California, USA, TSI: Toscani in Italy, YRI: Yoruba in Ibadan, Nigeria, CAR: Circassian from Jordan (unpublished data), ACB; African Caribbeans in Barbados. Also, the allele count data of populations around the world were obtained from the Exome Aggregation Consortium (ExAC): African, East Asian, Latino, European (non-Finnish), South Asian and European (Finnish).

Statistical Package for Social Sciences SPSS (version 19) was used to statistically analyze the data. The genotypes and alleles frequency was calculated and tested using the chi-square test and the Hardy–Weinberg equilibrium equation (HWE) and all p-values accepted as p-value <0.05. Fisher exact test was used for the correction of small sizes.

### Study Subjects

This study was subjected to and in agreement with the Human ethics committee at National Center for Diabetes, Endocrinology and Genetics (NCDEG) and Jordan University of Science and Technology (JUST). Details about subjects and written informed consent were obtained from all volunteers in the study. Ten milliliters of venous blood was collected from unrelated healthy 166 Chechen subjects living in Jordan.

This study was conducted in agreement with the Human ethics committee at National Center for Diabetes, Endocrinology and Genetics (NCDEG) and Jordan University of Science and Technology (JUST), policy number (GM7601). Written informed consent was obtained from all volunteers in the study. This research was also conducted in accordance with the Declaration of Helsinki.

### DNA Extraction and Genotyping

Blood samples were drawn from subject and then stored at 4°C for 24 hours before the purification of the genetic material. The extracted DNA was then stored at −20°C.

Genotyping was performed using Agena Bioscience MassARRAY® on a Compact Spectrometer iPLEX GOLD chemistry by Australian Genome Research Facility. Genomic DNA was extracted from each blood sample using the Wizard® Genomic DNA Purification Kit (Promega Corporation, USA) according to the manufacturer’s instructions. The quality and quantity of the purified DNA were ascertained via agarose gel electrophoresis and the Nano-Drop ND-1000 UV-Vis Spectrophotometer (BioDrop, UK), respectively.

Candidate genes polymorphisms were analyzed using Sequencing technique. Loci of candidate SNPs were amplified using Multiplex PCR followed by Mass EXTEND which is a primer extension process is resulting in allele-specific DNA products. The genotyping after the minisequencing reaction product analysis was reported by Mass spectrometry. After that, the extension PCR products were separated onto a 384 well spectroCHIP and placed into the MALDI-TOF (Matrix-Assisted Laser Desorption/Ionization Time-of-Flight) mass spectrometer. Lastly, a software system (SpectroTYPER-RT (RT for real-time)) was used to analyze the results.

### Variants Selection

The studied VIP variants were selected from the public databases including the PharmGKB database (https://www.pharmgkb.org/), the SNP database of the National Centre for Biotechnology Information (NCBI) (http://www.ncbi.nlm.nih.gov/SNP/), and the International HapMap Project (http://www.hapmap.org/). SNPs selected within significant genes that play roles in drug metabolism which make their variants clinically important variants. Finally, the primers were used to genotype these VIP variants were shown in the Supplementary Material.

### Population’s Variation Data

The allele counts variation data for other populations were obtained from HapMap Project website: CHE: Chechens from Jordan, ASW: African ancestry in Southwest USA, CEU: Utah, USA residents with Northern and Western European ancestry from the CEPH collection, CHB: Han Chinese in Beijing, China, CDX: Chinese Dai in Xishuangbanna, China, GIH: Gujarati Indians in Houston, Texas, USA, GBR: British in England and Scotland, JPT: Japanese in Tokyo, Japan, LWK: Luhya in Webuye, Kenya, MXL: Mexican ancestry in Los Angeles, California, USA, TSI: Toscani in Italy, YRI: Yoruba in Ibadan, Nigeria, CAR: Circassian from Jordan (unpublished data), ACB; African Caribbeans in Barbados. Also, the allele count data of populations around the world were obtained from the Exome Aggregation Consortium (ExAC): African, East Asian, Latino, European (non-Finnish), South Asian and European (Finnish).

### Statistical Analysis

Statistical Package for Social Sciences SPSS (version 19) was used to statistically analyze the data. The genotypes and alleles frequency was calculated and tested using the chi-square test and the Hardy–Weinberg equilibrium equation (HWE) and all p-values accepted as p-value <0.05. Fisher exact test was used for the correction of small sizes.

### Results

Table 1 shows basic information about the selected variants within 28 studied genes, in addition to describe the quality control of samples’ genotyping. Genotype call rates ranged from 96.5% to 100%. Allele and genotype frequencies of the selected variants among 166 Chechen subjects are listed in Table 2. SNPs were tested for Hardy–Weinberg Equilibrium (HWE), all the studied polymorphisms met the HWE standards (P-value > 0.05) except for rs1801131 within MTHFR (P-value = 0.027) did not fulfil the HWE equation (Figure 1).Table 1List of Genes, Their VIP Variants, Positions, and Genotyping Data Based on the Whole Cohort (N= 166)GeneSNP_IDPositionaSNPSNP LocationAssay Pass RatebCall RatecMTHFRrs18011311:11794419A>CMAMissense variant100%100%rs18011331:11796321C>TMAMissense variant100%99.7%DPYDrs39182901:97450058C>TMASplice donor variant100%99.7%PTGS2rs6894661:186681619G>AMA2KB Upstream variant100%100%SCN5Ars76269623:38579416G>A/G>TMissense variant100%100%rs18051243:38603929T>CMAMissense variant100%100%rs67919243:38633208G>AMAMissense variant100%100%NR1I2rs38140553:119781188C>TMA5 Prime UTR variant100%100%P2RY12rs20469343:151339854GMA>AIntron variant99%99.2%P2RY1rs10657763:152835839C>TMASynonymous variant100%100%rs7012653:152836568A>GMASynonymous variant100%100%ADH1Ars9758334:99280582G>CMAIntron variant100%99.7%ADH1Brs20667024:99307860G>AMAMissense variant100%100%rs12299844:99318162TMA>CMissense variant100%100%ADH1Crs6984:99339632T>CMAMissense variant99%99.2%HMGCRrs172448415:75347030A>TMAIntron variant99%98.6%rs38466625:75355259AMA>GIntron variant100%99.7%rs172385405:75359673T>GMAIntron variant100%100%ADRB2rs10427135:148826877G>AMAMissense variant100%100%rs10427145:148826910GMA>CStop gained100%100%rs18008885:148827322C>TMAMissense variant100%99.7%AHRrs20668537:17339486G>AMAMissense variant100%99.7%KCNH2rs38154597:150947306C>TMAIntron variant100%99.7%rs38073757:150970122C>TMAIntron variant100%100%KCNJ11rs521911:17388025TMA>CStop gained100%100%SLCO1B1rs414905612:21178615T>CMAMissense variant100%100%VKORC1rs729416:31091000C>TMA3 Prime UTR variant100%100%rs993443816:31093557G>AMAIntron variant100%100%SLC19A1rs1265921:45531642AMA>GSynonymous variant100%100%rs105126621:45537880T>CMAMissense variant99%98.6%rs113159621:45538002G>AMAMissense variant100%99.7%CYP2J2rs8902931:59926822C>AMAUpstream variant100%100%CYP3A4rs49869137:99760836G>AMAMissense variant100%100%rs49869107:99760901A>GMAMissense variant100%100%rs49869097:99762047G>A MAMissense variant100%100%rs127216347:99784038A>GMissense variant100%100%rs27405747:99784473CMA>TUpstream variant100%100%CYP2C19rs498689310:94780653G>AMAStop gained100%100%rs424428510:94781859G>AMASynonymous variant100%99.7%CYP2C9rs179985310:94942290C>TMAMissense variant100%100%CYP2A6rs2839945419:40845362C>TMAMissense variant100%100%rs180127219:40848628A>TMAMissense variant100%100%rs2839943319:40850474A>CMAUpstream Variant99%99.2%CYP2B6rs374527419:41006936G>TMAMissense variant96%96.5%rs2839949919:41012316T>CMAMissense variant100%100%CYP2D6rs5942138822:42127608C>TMAMissense variant100%100%rs2837172522:42127803C>TMAIntron variant98%98.4%rs6173651222:42129132C>TMAMissense variant100%100%rs2837170622:42129770G>AMAMissense variant100%100%rs503065622:42128174–42128178delTCTInframe deletion100%100%CYP3A5rs7767467:99672916C>TMASplice Acceptor Variant100%99%UGT1A1rs41248742:233757013TMA>GIntronic variant100%100%rs109293022:23375713G>AMAIntronic variant100%100%rs41483232:233760498G>AMAMissense variant100%100%COMTrs468022:19963748G>AMAMissense variant100%99.7%GSTP1rs169511:67585218A>GMAMissense variant100%100%rs113827211:67586108C>TMAMissense variant100%100%Notes:
aChromosome positions are based on NCBI Human Genome Assembly Build. bRatio of the number of discordant genotypes to the number of duplicates. cRatio of the number of valid genotypes to the number of subjects genotyped. MA: Global minor allele <1.
Table 2The Minor Allele Frequencies and HWE p Values for the VIP Variants in Chechen (N= 166)GeneSNP_IDAlleleAllelic FrequencyGenotypeGenotypic FrequencyP-valueMTHFRrs1801131C0.64AA0.370.027A0.36CA0.54CC0.08rs1801133T0.18CC0.670.88C0.82CT0.3TT0.03DPYDrs3918290C99.7CC0.990.96T0.003CT0.01PTGS2rs689466T0.89CC0.010.7C0.11CT0.21TT0.78SCN5Ars7626962G1.0GG1.0NArs1805124T0.75CC0.040.22C0.25CT0.42TT0.54rs6791924G1.0GG1.0NANR1I2rs3814055C0.67CC0.460.39T0.33CT0.42TT0.13P2RY12rs2046934A0.85AA0.720.97G0.15AG0.26GG0.02P2RY1rs1065776C0.99CC0.980.90T0.01CT0.02rs701265A0.83AA0.690.75G0.17AG0.28GG0.02ADH1Ars975833G0.76CC0.050.78C0.24CG0.37GG0.57ADH1Brs2066702G0.99GA0.020.87A0.01GG0.98rs1229984C0.88CC0.750.14T0.12CT0.25ADH1Crs698C0.27CC0.070.91T0.73CT0.39TT0.54HMGCRrs17244841A0.97AA0.940.72T0.03AT0.06rs3846662G0.43AA0.310.53A0.57GA0.52GG0.17rs17238540G0.03TT0.930.65T0.97GT0.07ADRB2rs1042713G0.63GG0.40.62A0.37GA0.45AA0.15rs1042714C0.69CC0.490.21G0.31CG0.39GG0.12rs1800888C0.99CC0.990.97T0.003CT0.01AHRrs2066853G0.87GG0.770.73A0.13GA0.22AA0.02KCNH2rs3815459C0.73CC0.520.37T0.27CT0.43TT0.05rs3807375C0.63CC0.390.74T0.37CT0.48TT0.13KCNJ11rs5219C0.54CC0.310.35T0.46CT0.46TT0.23SLCO1B1rs4149056C0.04CT0.080.59T0.96TT0.92VKORC1rs7294C0.62CC0.360.33T0.38CT0.51TT0.13rs9934438G0.59GG0.340.63A0.41GA0.51AA0.16SLC19A1rs12659G0.51GG0.270.54A0.49GA0.48AA0.25rs1051266C0.48CC0.230.86T0.52CT0.51TT0.43rs1131596A0.47AA0.220.88G0.53AG0.49GG0.28CYP2J2rs890293C0.93CA0.130.36A0.07CC0.87CYP3A5rs10264272C1.0CC1.0N/Ars776746C0.97CC0.950.13T0.03CT0.05TT0.01CYP3A4rs4986913G1.0GG1.0N/Ars4986910A1.0AA1.0N/Ars4986909G1.0GG1.0N/Ars12721634A1.0AA1.0N/Ars2740574T0.99TC0.010.96C0.004TT0.99CYP2C19rs4986893G1.0GG1.0N/Ars4244285G0.91AA0.020.13A0.09GA0.14GG0.84CYP2C9rs1799853C0.98CC0.960.083T0.02CT0.04TT0.01CYP2A6rs28399454C1.0CC1.0N/Ars1801272A1.0AA1.0N/Ars28399433A0.91AA0.820.37C0.09AC0.18CYP2B6rs3745274G0.69GG0.50.072T0.31GT0.37TT0.13rs28399499T1.0TT1.0N/ACYP2D6rs59421388C1.0CC1.0N/Ars28371725C0.86CC0.740.33T0.14CT0.23TT0.03rs61736512C1.0CC1.0N/Ars28371706G1.0GG1.0N/Ars5030656C/T/T1.0-1.0N/AUGT1A1rs4124874G0.54GG0.330.06T0.46GT0.41TT0.26rs10929302G0.61GG0.370.90A0.39GA0.47AA0.16rs4148323G0.97GG0.950.80A0.03GA0.05COMTrs4680G0.59GG0.350.90A0.41GA0.47AA0.18GSTP1rs1695A0.67AA0.460.69G0.33AG0.43GG0.12rs1138272C0.84CC0.710.50T0.16CT0.26TT0.02Note: *p-value > 0.05 is considered normally distributed.Abbreviations: HWE, Hardy–Weinberg equilibrium; N/A, not applicable.
Figure 1The number of single nucleotide polymorphisms (SNPs) in this study and their genotyping accuracy.

List of Genes, Their VIP Variants, Positions, and Genotyping Data Based on the Whole Cohort (N= 166)

Notes:
aChromosome positions are based on NCBI Human Genome Assembly Build. bRatio of the number of discordant genotypes to the number of duplicates. cRatio of the number of valid genotypes to the number of subjects genotyped. MA: Global minor allele <1.

The Minor Allele Frequencies and HWE p Values for the VIP Variants in Chechen (N= 166)

Note: *p-value > 0.05 is considered normally distributed.

Abbreviations: HWE, Hardy–Weinberg equilibrium; N/A, not applicable.

The number of single nucleotide polymorphisms (SNPs) in this study and their genotyping accuracy.

Table 3 compares minor allele frequencies (MAF) of the significant polymorphisms among Chechens and other populations. For example, the MAF of rs776746 (T) of the CYP3A5 gene within the Chechen group was similar to both CEU and GBR but different from ASW, CHB, CDX, GIH, and others. Also, Table 3 compares the allelic distribution of selected variants among Chechens and Circassians and other populations by estimating the P-value. We found that each studied variant distribution within Chechens in Jordan is significantly different from various populations. For example, Rs1229984 and rs28371725 had a highly different distribution with around 90% of the compared populations.Table 3VIP Variants Within the Pharmacogenes in Chechen Compared to HapMap PopulationsSNP IDPopulationsCHEASWCEUCHBCDXGBRGIHJPTLWKMXLTSIYRIACBrs39182900.003a0.008a0.005a0.00a0.00a0.00a0.146a0.00a0.00a0.00a0.005a0.00a0.00a0.54b0.13b0.62b0.56b0.55b2.3b0.63b0.60b0.39b0.09b0.65b0.57b0.46c0.71c0.43c0.45c0.45c0.13c0.43c0.44c0.53c0.75c0.42c0.45crs6894660.11a0.13a0.19a0.47a0.52a0.20a0.13a0.44a0.03a0.26a0.20a0.07a0.10a0.34b6.6bN/AbN/Ab7.2b0.26bN/Ab10.9b15.3b7.5b2.1b0.1b0.56c0.01cN/AcN/Ac0.01c0.60cN/Ac1e-3c1e-4c0.01c0.14c0.79crs18051240.25a0.26a0.18a0.12a0.07a0.22a0.20a0.13a0.30a0.16a0.23a0.32a0.28a0.04b3.5b12.5b26.2b0.49b1.7b10.9b1.6b4.1b0.26b3.7b0.5b0.84c0.06c4e-4c3e-7c0.48c0.19c1e-3c0.21c0.04c0.61c0.05c0.47crs38140550.33a0.29a0.34a0.27a0.15a0.40a0.42a0.25a0.30a0.33a0.36a0.27a0.39a0.62b0.01b2.3b19.2b33.2b3.8b3.8b0.55b0.001b0.37b2.2b2.0b0.42c0.92c0.12c1e-5c1e-8c0.05c0.05c0.45c0.97c0.54c0.13c0.15crs10657760.01a0.20a0.05a0.03a0.02a0.02a0.10a0.07a0.21a0.05a0.03a0.22a0.20aN/Ab10.43b4.3b0.52b1.4b25.7b15.7bN/Ab6.9b4.1bN/AbN/AbN/Ac0.001c0.04c0.46c0.22c3.9e-7c7e-5cN/Ac0.01c0.04cN/AcN/Acrs7012650.17a0.67a0.17a0.28a0.19a0.13a0.21a0.27a0.80a0.20a0.16a0.81a0.72aN/Ab0.001b11.1b0.42b1.0b1.6b9.1bN/Ab0.89b0.004bN/AbN/AbN/Ac0.97c0.001c0.51c0.31c0.21c0.002cN/Ac0.34c0.95cN/AcN/Acrs9758330.24a0.25a0.24a0.79a0.75a0.24a0.49a0.80a0.19a0.13a0.27a0.30a0.28a0.08b0.001bN/AbN/AbN/AbN/AbN/Ab1.7b6.5b0.6b2.4b0.79b0.77c0.97cN/AcN/AcN/AcN/AcN/Ac0.19c0.01c0.43c0.12c0.37crs20667020.01a0.20a0.00a0.00a0.00a0.00a0.00a0.00a0.14a0.03a0.00a0.28a0.19aN/Ab2.4b2.5b2.3b2.2b2.5b2.5bN/Ab1.9b2.6bN/AbN/AbN/Ac0.12c0.11c0.13c0.14c0.11c0.11cN/Ac0.16c0.11cN/AcN/Acrs12299840.12a0.00a0.02a0.71a0.63a0.01a0.02a0.73a0.00a0.08a0.05a0.00a0.01a16.5b5e-5c19.1b1e-5cN/AbN/AcN/AbN/Ac21.8b3e-6c16.0b6e-5cN/AbN/Ac26.5b2.6e-7c1.3b0.25c7.8b0.005c28.8b8e-8c20.6b6e-6crs6980.27a0.14a0.47a0.05a0.11a0.44a0.28a0.07a0.14a0.28a0.31a0.07a0.11a8.1b23.7bN/Ab16.9b15.9b0.14b31.2b11.3b0.10b1.1b33.1b18.1b0.004c1e-6cN/Ac4e-5c7e-5c0.71c2e-8c0.001c0.75c0.29c1e-8c2e-5crs172448410.03a0.10a0.01a0.00a0.00a0.03a0.00a0.02a0.08a0.04a0.03a0.09a0.10a9.8b0.87b5.8b5.2bN/Ab5.2b0.39b8.7b0.38b0.11b10.7b13.3b0.002c0.35c0.020.02cN/Ac0.02c0.53c0.003c0.53c0.74c0.001c3e-4crs38466620.43a0.13a0.57a0.480.49a0.55a0.34a0.47a0.03a0.56a0.57a0.04a0.11aN/Ab0.04b3.3b2.5b0.14b26.1b7.1bN/Ab0.005b0.008bN/AbN/AbN/Ac0.83c0.06c0.12c0.17c3.2e-7c0.008cN/Ac0.94c0.93cN/AcN/Acrs172385400.03a0.10a0.01a0.00a0.00a0.02a0.00a0.00a0.09a0.04a0.03a0.10a0.09a7.9b1.5b6.9b6.3b0.5b6.9b7.0b8.0b0.09b0.001b9.7b8.5b0.005c0.21c0.008c0.01c0.47c0.008c0.008c0.004c0.76c0.97c0.002c0.003crs10427130.37a0.54a0.35a0.55a0.58a0.42a0.43a0.44a0.49a0.48a0.37a0.53a0.51a11.3a0.33b15.8b20.7b0.96b1.8b2.5b7.0b4.1b0.01b13.4b9.3b0.001c0.56c7e-5c5e-6c0.320.18c0.11c0.008c0.04c0.92c0.0002c0.002crs10427140.31a0.12a0.46a0.11a0.10a0.39a0.23a0.06a0.21a0.14a0.14a0.12a0.16a16.7b12.1b30.2b29.3b3.5b4.6bN/Ab6.3b14.1b3.6b27.0b14.6b4e-5c5e-4c4e-8c6e-8c0.06c0.03cN/Ac0.01c2e-4c0.06c2.1e-7c1e-4crs18008880.003a0.00a0.02a0.00a0.00a0.02a0.00a0.00a0.00a0.00a0.01a0.00a0.00a0.36b2.4b0.62b0.56b4.3b0.62b0.62b0.59b0.38b2.2b0.65b0.57b0.54c0.11c0.43c0.45c0.04c0.43c0.42c0.44c0.53c0.14c0.42c0.45crs20668530.13a0.34a0.09a0.37a0.26a0.09a0.12a0.46a0.48a0.13a0.09a0.45a0.47a28.0b1.5bN/Ab14.4b0.87b0.11bN/AbN/Ab0.03b1.02bN/AbN/Ab1.2e-7c0.21cN/Ac2e-4c0.35c0.73cN/AcN/Ac0.85c0.31cN/AcN/Acrs38154590.27a0.42a0.20a0.71a0.56a0.15a0.35a0.80a0.39a0.39a0.23a0.33a0.31a9.5b3.2bN/Ab6.6b8.5b3.2bN/Ab8.6b7.6b0.74b2.8b1.5b0.002c0.07cN/Ac0.01c0.003c0.07cN/Ac0.003c0.006c0.39c0.09c0.22crs38073750.63a0.27a0.65a0.25a0.26a0.70a0.61a0.20a0.20a0.43a0.66a0.23a0.26aN/Ab0.33bN/AbN/Ab3.1b1.0bN/AbN/AbN/Ab0.59bN/AbN/AbN/Ac0.56cN/AcN/Ac0.08c0.31cN/AcN/AcN/Ac0.44cN/AcN/Acrs52190.46a0.14aN/Ab0.38a3.2b0.38a3.7b0.23a28.7b0.26a19.7b0.42a1.1b0.33a9.2b0.01aN/Ab0.41a1.24b0.29a17.4b0.00aN/Ab0.06aN/AbN/Ac0.07c0.05c8e-8c1e-5c0.29c0.002cN/Ac0.26c3e-5cN/AcN/Acrs41490560.04a0.06a0.15a0.14a0.14a0.14a0.02a0.12a0.02a0.08a0.21a0.01a0.02a0.38b3.0b2.19b2.4b2.6b8.5b1.1b7.9b0.05b11.0b13.3b7.6b0.53c0.08c0.14c0.12c0.10c0.004c0.29c0.004c0.82c0.001c3e-4c0.01crs72940.38a0.48a0.31a0.04a0.17a0.42a0.67a0.09a0.43a0.35a0.34a0.51a0.46aN/Ab14.5b0.64b0.98b15.2b0.02b2.9bN/Ab23.4b10.7bN/AbN/AbN/Ac1e-4c0.42c0.32c1e-4c0.89c0.09cN/Ac1e-6c0.001cN/AcN/Acrs99344380.41a0.15a0.43a0.96a0.82a0.36a0.17a0.90a0.04a0.47a0.48a0.03a0.06a27.3b0.19bN/AbN/Ab1.4b32.2bN/AbN/Ab1.3b2.4bN/AbN/Ab1.7e-7c0.65cN/AcN/Ac0.24c1e-8cN/AcN/Ac0.25c0.12cN/AcN/Acrs126590.49a0.40a0.42a0.49a0.53a0.39a0.36a0.54a0.53a0.34a0.44a0.48a0.55a2.8b2.5b0.02b0.81b4.8b8.3b1.4b0.77b8.1b1.2b0.05b1.8b0.09c0.10c0.90c0.37c0.03c0.004c0.23c0.38c0.004c0.28c0.83c0.18crs10512660.48a0.43a0.57a0.52a0.46a0.60a0.60a0.46a0.31a0.65a0.55a0.33a0.31a0.79b3.9b0.92b0.17b4.6b7.9b0.31b15.2b10.3b2.5b12.4b14.1b0.37c0.05c0.33c0.68c0.01c0.005c0.58c1e-4c0.001c0.11c4e-4c2e-4crs11315960.47a0.38a0.57a0.52a0.46a0.60a0.61a0.46a0.24a0.65a0.55a0.28a0.28a3.1b5.1b1.5b0.03b7.8b9.6b0.09b28.3b11.8b3.5b201b19.0b0.08c0.02c0.22c0.87c0.005c0.001c0.77c1.1e-7c0.001c0.06c1e-5c1e-5crs41248740.44a0.22a0.57a0.72a0.55a0.62a0.39a0.69a0.11a0.48a0.56a0.09a0.22a47.3b7.8b42.7b5.6b15.4b1.1b31.5b63.9b0.74b7.04b74.6b0.9bN/Ac0.005cN/AC0.02c9e−5 c0.28c2e−8 cN/Ac0.38c008e−3 cN/A c0.3crs109293020.35a0.33a0.30a0.11a0.15a0.23a0.44a0.18a0.36a0.34a0.24a0.64a0.30a0.183b0.95b37.5b23.4b7.7b4.1b18.5b0.05 b0.01b7.7b0.08b0.9b0.67c0.33cN/Ac2e−6 c5e−3 c0.04c2e−5 c0.83c0.91c6e−3 c0.78c0.32crs41483230.0a0.01a0.0a0.23a0.10a0.0a0.02a0.13a0.0a0.02a0.0a0.0a0.0a2.7bN/Ab82.9b35.2bN/Ab6.4b45.3bN/Ab7.8bN/AbN/AbN/Ab0.09cN/AcN/AcN/AcN/Ac0.01cN/AcN/Ab5e−3 cN/AcN/AcN/Acrs46800.50a0.27a0.46a0.31a0.26a0.52a0.43a0.28a0.28a0.39a0.45a0.30a0.36a19.0b0.6b17.6b26.1b0.3b2.0b24.5b22.8b3.8b1.1b20.2b15.4b1e−5 c0.43c3e-5 c3.1e-7 c0.55c0.15c7.2e-7 c1e-6c0.05c0.28c6e-6 c8e-5 crs16950.30a0.45a10.2b0.39a5.1b0.18a8.6b0.22a3.6b0.31a0.2b0.31a0.01b0.10a28.7b0.51a23.7b0.56a27.6b0.29a23.7b0.39a5.8b0.43a10.3b0.001c0.02c0.003c0.05c0.62c0.76c8e-8 c1e-6 c1.5e-7c1e-6 c0.01c0.001crs11382720.10a0.00a0.09a0.00a0.00a0.05a0.08a0.00a0.01a0.05a0.05a0.00a0.001a6.4bN/A21.1b19.1b2.6b0.2bN/Ab13.2b2.07b3.6b22.1b17.1b0.01cN/A4e-6c1e-5c0.1c0.58cN/Ac3e-4c0.15c0.05c2e-6c3e-5crs8902930.07a0.16a0.05a0.04a0.02a0.05a0.05a0.02a0.14a0.03a0.05a0.15a0.10a10.1b0.5b1.2b6.5b0.6b0.4b4.8b7.3b2.1b0.5b10.8b2.3b0.001c0.46c0.27C0.01c0.44 c0.54c0.02 c0.007c0.14c0.47 c0.001 c0.12crs102642720.0a0.05a0.00a0.00a0.00a0.00a0.00a0.0a0.24a0.02a0.01a0.17a0.11a16.5bN/AbN/AbN/AbN/AbN/AbN/AbN/Ab7.8b1.6bN/AbN/Ab5e−5 cN/AcN/AcN/AcN/AcN/AcN/AcN/Ac0.005c0.21cN/AcN/Acrs7767460.03a0.68a0.04a0.31a0.31a0.05a0.28a0.25a0.88a0.23a0.05a0.83a0.75aN/Ab0.4bN/AbN/Ab1.9bN/AbN/AbN/AbN/Ab1.5bN/AbN/AbN/Ac0.52cN/AcN/Ac0.16cN/AcN/AcN/AcN/Ac0.21cN/AcN/Acrs49869130.0a0.0a0.0a0.0a0.0a0.0a0.005a0.0a0.0a0.0a0.0a0.0a0.0aN/AbN/AbN/AbN/AbN/Ab1.6bN/AbN/AbN/AbN/AbN/AbN/AbN/AcN/AcN/AcN/AcN/Ac0.20cN/AcN/AcN/AcN/AcN/AcN/Acrs49869100.0a0.0a0.01a0.0a0.0a0.01a0.0a0.0a0.0a0.0a0.0a0.0a0.0aN/Ab3.6bN/AbN/Ab3.6bN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/Ac0.06cN/AcN/Ac0.06cN/AcN/AcN/AcN/AcN/AcN/AcN/Acrs27405740.004a0.67a0.02a0.0a0.0a0.03a0.08a0.0a0.83a0.07a0.03a0.76a0.66aN/Ab2.4b0.6b0.6b7.8b23.2b0.6bN/Ab19.7b6.4bN/AbN/AbN/Ac0.12c0.43c0.45c0.005c1.4e-6b0.43cN/Ac1e-5c0.01cN/AcN/Acrs49868930.0a0.0a0.0a0.04a0.07a0.0a0.0a0.07a0.01a0.0a0.0a0.0a0.005aN/AbN/Ab14.7b25.6bN/AbN/Ab30.8b3.4bN/AbN/AbN/Ab1.7bN/AcN/Ac1.4e-4c4e-7cN/AcN/Ac3e-8c0.07cN/AcN/AcN/Ac0.19crs42442850.09a0.14a0.13a0.33a0.26a0.14a0.33a0.32a0.21a0.12a0.09a0.17a0.15a2.3b2.2bN/Ab27.6b3.3bN/AbN/Ab15.7b1.2b0.02b7.2b4.5b0.12c0.13cN/Ac1.5e-7c0.07cN/AcN/Ac8e-5c0.27c0.9c0.007c0.03crs17998530.02a0.04a0.15a0.0a0.0a0.08a0.05a0.0a0.0a0.10a0.15a0.0a0.03a0.9b30.3b5.0b4.5b10.7b2.3b5.1b4.8b12.7b31.7b5.2b0.02b0.34c4e-8c0.02c0.03c0.001c0.12c0.02c0.02c4e-4c2e-8c0.02c0.89crs283994540.0a0.07a24.9b0.0aN/Ab0.0aN/Ab0.0aN/Ab0.0aN/Ab0.0aN/Ab0.0aN/Ab0.06a18.8b0.0aN/Ab0.0aN/Ab0.13aN/Ab0.15aN/Ab5.8e-7cN/AcN/AcN/AcN/AcN/AcN/Ac1e-5cN/AcN/AcN/AcN/Acrs18012720.0a0.01a0.04a0.0a0.0a0.03a0.01a0.0a0.0a0.0a0.05a0.0a0.0a2.7b11.8bN/AbN/Ab9.2b3.2bN/AbN/AbN/Ab15.8bN/AbN/Ab0.09c6e-4cN/AcN/Ac0.002c0.07cN/AcN/AcN/Ac7e-5cN/AcN/Acrs283994330.09a0.06a0.05a0.27a0.18a0.04a0.19a0.28a0.09a0.10a0.06a0.10a0.06a0.2b3.0b28.7b8.0b3.9b10.5b32.2b0.002b0.1b1.2b0.1b1.4b0.64c0.08c8e-8c0.01c0.04c0.001c1e-8c0.96c0.75c0.27c0.70c0.23crs37452740.31a0.35a0.28a0.27a0.33a0.23a0.23a0.22a0.36a0.31a0.30a0.40a0.38a0.05b0.6b15.5b0.1b3.8b4.6b5.3b1.2bN/Ab0.04b4.7b2.5b0.82c0.40c8.4e-5c0.71c0.05c0.03c0.02c0.27cN/Ac0.83c0.03c0.11crs283994990.0a0.1a0.0a0.0a0.0a0.0a0.0a0.0a0.06a0.08a0.0a0.12a0.05a33.5bN/AbN/AbN/AbN/AbN/AbN/Ab20.6b2.5bN/AbN/Ab32.2b1e-8cN/AcN/AcN/AcN/AcN/AcN/Ac6e-6c0.11cN/AcN/Ac1e-8crs594213880.0a0.04a0.0a0.0a0.0a0.0a0.0a0.0a0.09a0.0a0.0a0.11a0.08a13.7bN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/Ab2e-4cN/AcN/AcN/AcN/AcN/AcN/AcN/AcN/AcN/AcN/AcN/Acrs283717250.14a0.02a0.12a0.03a0.08a0.07a0.15a0.01a0.03a0.02a0.14a0.01a0.05a14.8b0.5b16.8b4.4b5.9b0.002b30.1b17.6b15.8bN/Ab28.7b11.8b1e-4c0.46c4e-5c0.03c0.01c0.96c4e-8c3e-5c7e-5cN/Ac8e-8c6e-4crs617365120.0a0.04a0.0a0.0a0.0a0.0a0.0a0.0a0.17a0.0a0.0a0.11a0.09a13.7bN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/Ab32.2b2e-4cN/AcN/AcN/AcN/AcN/AcN/AcN/AcN/AcN/AcN/Ac1e-8crs283717060.0a0.15a0.0a0.0a0.0a0.0a0.0a0.0a0.19a0.0a0.0a0.25a0.20aN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/AbN/AcN/AcN/AcN/AcN/AcN/AcN/AcN/AcN/AcN/AcN/AcN/Acrs50306560.0a0.01a0.02a0.0a0.0a0.04a0.0a0.0a0.0a0.01a0.01a0.0a0.0a2.7b6.8bN/AbN/Ab13.0bN/AbN/AbN/Ab2.6b11.0bN/AbN/Ab0.99c0.01cN/AcN/Ac3e-4cN/AcN/AcN/Ac0.11c0.001cN/AcN/AcNotes:
aMinor allele frequency, bChi-square value, cp-value, when p-value < 0.05 is considered a significant difference. CHE: Chechens from Jordan, ASW: African ancestry in Southwest USA, CEU: Utah, USA residents with Northern and Western European ancestry from the CEPH collection, CHB: Han Chinese in Beijing, China, CDX: Chinese Dai in Xishuangbanna, China, GIH: Gujarati Indians in Houston, Texas, USA, GBR: British in England and Scotland, JPT: Japanese in Tokyo, Japan, LWK: Luhya in Webuye, Kenya, MXL: Mexican ancestry in Los Angeles, California, USA, TSI: Toscani in Italy, YRI: Yoruba in Ibadan, Nigeria, ACB; African Caribbeans in Barbados.Abbreviation: N/A, not applicable.

VIP Variants Within the Pharmacogenes in Chechen Compared to HapMap Populations

Notes:
aMinor allele frequency, bChi-square value, cp-value, when p-value < 0.05 is considered a significant difference. CHE: Chechens from Jordan, ASW: African ancestry in Southwest USA, CEU: Utah, USA residents with Northern and Western European ancestry from the CEPH collection, CHB: Han Chinese in Beijing, China, CDX: Chinese Dai in Xishuangbanna, China, GIH: Gujarati Indians in Houston, Texas, USA, GBR: British in England and Scotland, JPT: Japanese in Tokyo, Japan, LWK: Luhya in Webuye, Kenya, MXL: Mexican ancestry in Los Angeles, California, USA, TSI: Toscani in Italy, YRI: Yoruba in Ibadan, Nigeria, ACB; African Caribbeans in Barbados.

Abbreviation: N/A, not applicable.

It is also clear from Table 3 that the similarity of the studied polymorphisms within the Chechens population with other compared population varied from one to another. The CEU Utah (USA residents with Northern and Western European ancestry from the CEPH collection) and TSI (Toscani in Italy) had the highest similarity among the studied population with around 25% differences; ie, around 75% of the studied variants had the same frequency and distribution in Chechens and other compared populations. This similarity in the distribution of the compared SNPs decreased to reach around 30% when compared to LWK (Luhya in Webeye).

Furthermore, the significant variants within the selected genes within Chechen were compared to the available variants within six populations listed in the Exome Aggregation Consortium (ExAC) database as shown in Table 4.Table 4VIP Variants Within the Pharmacogenes in Chechen Compared to Six ExAC Populations WorldwideSNP\PopulationChechenAfricanEast AsianLatinoEuropean (Non-Finnish)South AsianEuropean (Finnish)rs39182900.003a0.001a0.00a0.001a0.006a0.005a0.022a2.4b26.1b2.2b0.45b0.20b5.6b0.12c3.3e-7c0.13c0.50c0.65c0.02crs10657760.01a0.18a0.04a0.05a0.04a0.11a0.07aN/Ab9.4b12.5b8.6bN/Ab17.7bN/Ac0.002c4e-4c0.003cN/Ac3e-5crs7012650.17a0.69a0.29a0.22a0.14a0.19a0.15aN/Ab22.8b6.4b1.3b1.4b0.71bN/Ac2e-6c0.01c0.26c0.24c0.40crs20667020.01a0.19a0.0001a0.01a0.002a0.002a0.00aN/AbN/Ab0.46b17.9b21bN/AbN/AcN/Ac0.50c2e-5c5e-6cN/Acrs12299840.12a0.99a0.27a0.94a0.95a0.95a0.99aN/Ab34.0bN/AbN/AbN/AbN/AbN/Ac1e-8cN/AcN/AcN/AcN/Acrs6980.27a0.15a0.08a0.33a0.40a0.32a0.51a33.8bN/Ab6.2b23.4b5.1bN/Ab1e-8cN/Ac0.01c1e-6c0.02cN/Acrs38466620.43a0.88a0.53a0.47a0.44a0.59a0.47aN/Ab1.4b12.2b19.7b0.50b11.5bN/Ac0.24c5e-4c1e-5c0.47c0.001crs10427130.37a0.49a0.55a0.42a0.38a0.46a0.45a18.4bN/Ab2.9b0.1b8.9b6.7b2e-5cN/Ac0.09c0.78c0.003c0.01crs10427140.31a0.82a0.91a0.83a0.58a0.79a0.63aN/AbN/AbN/AbN/AbN/AbN/AbN/AcN/AcN/AcN/AcN/AcN/Acrs18008880.003a0.002a0.00a0.006a0.01a0.004a0.004a0.05b26.1b0.45b3.0b0.05b0.1b0.82c3.3e-7c0.49c0.08c0.82c0.76crs20668530.13a0.45a0.37a0.12a0.10a0.14a0.11aN/AbN/Ab0.1b2.8b0.3b0.49bN/AcN/Ac0.77c0.09c0.59c0.482crs38154590.27a0.37a0.72a0.38a0.33a0.39a0.44a1.6b33.4b0.59b6.9b0.50b0.03b0.21c1e-8c0.44c0.01c0.48c0.86crs52190.46a0.94a0.64a0.61a0.63a0.63a0.53aN/AbN/Ab29.3bN/AbN/Ab4.9bN/AcN/Ac6e-8cN/AcN/Ac0.03crs41490560.04a0.03a0.13a0.11a0.16a0.05a0.21a1.3b22.4b15.6bN/Ab0.84bN/Ab0.24c2e-6c1e-4cN/Ac0.35cN/Acrs126590.49a0.55a0.49a0.56a0.58a0.64a0.56a4.0b0.007b7.3b11.2b31.7b5.6b0.05c0.93c0.006c0.001c2e-8c0.02crs10512660.48a0.42a0.52a0.59a0.60a0.61a0.60a13.5b0.01b6.6b8.1b10.3b8.8b2e-4c0.93c0.01c0.004c0.001c0.003crs11315960.47a0.36a0.51a0.56a0.59a0.58a0.61a16.6b1.4b8.1b20.3bN/Ab12.8b4e-5c0.23c0.004c1e-5cN/Ac3e-4crs41483230.0a0.0009a0.15a0.03a0.004a0.02a0.05a0.2b5.9b8.9b1.1b6.4b15.9b0.59c0.01c0.003c0.27c0.01c7.7e-5crs46800.50a0.32a0.28a0.41a0.53a0.45a0.57aN/AbN/Ab11.3b1.1b3.5b6.6bN/AcN/Ac7.6e-4c0.30c0.06c0.009crs16950.30a0.44a0.18a0.53a0.31a0.29a0.27a27.7b29.3bN/Ab0.7b0.1b0.8b1.4e-7c6e-8cN/Ac0.41c0.71c0.35crs11382720.10a0.02a0.0003a0.03a0.08a0.07a0.08aN/AbN/AbN/Ab1.5b3.6b0.7bN/AcN/AcN/Ac0.22c0.05c0.41crs102642720.0a0.12a0.0a0.01a0.001a0.0002a0.0aN/AbN/Ab2.67b0.27b0.08bN/AbN/AcN/Ac0.10c0.60c0.78cN/Acrs49869130.0a0.0a0.0a0.0a0.0a0.001a0.0aN/AbN/AbN/AbN/Ab0.26bN/AbN/AcN/AcN/AcN/Ac0.61cN/Acrs49869100.0a0.001a0.0a0.002a0.007a0.0a0.02a0.41bN/Ab0.60b2.5bN/Ab7.0b0.52cN/Ac0.43c0.11cN/Ac0.008crs49869090.0a0.0a0.0a0.002a5.994e-050.0a0.0aN/AbN/Ab0.05b0.02bN/AbN/AbN/AcN/Ac0.81c0.88cN/AcN/Acrs49868930.0a0.0005a0.07a0.0004a0.0002a0.004a0.0002a0.16b23.9b0.08b0.06b1.2b0.05b0.68c1e-6c0.76c0.80c0.27c0.82crs42442850.09a0.18a0.31a0.10a0.15a0.34a0.18a18.0bN/Ab0.39b8.6bN/Ab18.3b2.2e-5cN/Ac0.53c0.003cN/Ac1.8e-5crs17998530.02a0.02a0.0003a0.07a0.13a0.05a0.12a0.004bN/Ab9.3b31.5b3.5b27.3b0.94cN/Ac0.002c2e-8c0.05c1.7e-7crs283994540.0a0.11a0.0a0.006a0.0004a0.0a0.0aN/AbN/Ab1.9b0.1bN/AbN/AbN/AcN/Ac0.15c0.73cN/AcN/Acrs18012720.0a0.005a0.0a0.012a0.025a0.011a0.023a1.5bN/Ab4.0b8.7b3.8b7.9b0.20cN/Ac0.04c0.003c0.04c0.004crs283994330.09a0.083a0.23a0.138a0.068a0.144a0.11a0.3b0.30b5.6b2.8b7.1b1.1b0.57c0.57c0.01c0.09c0.007c0.28crs37452740.31a0.37a0.19a0.32a0.24a0.19a0.19a4.1b31.6b0.08b8.9b8.2b29.4b0.04c2e-8c0.76c0.003c0.004c6e-8crs283994990.0a0.07a0.0a0.003a0.0001a0.0001a0.0a25.2bN/Ab1.1b0.04b0.04bN/Ab5.1e-7cN/Ac0.30c0.83c0.83cN/Acrs594213880.0a0.092a33.6b1e-8c0.0001a0.04b0.84c0.004a1.2b0.26c0.0003a0.09b0.76c0.0001a0.04b0.84c0.0aN/AbN/Acrs283717250.14a0.03a0.03a0.03a0.09a0.15a0.03aN/AbN/Ab9.3b9.3b0.2bN/AbN/AcN/Ac0.002c0.002c0.64cN/Acrs617365120.0a0.10a0.0005a0.004a0.0003a0.0002a0.0aN/Ab0.15a1.3a0.08a0.08aN/AbN/Ac0.69c0.25c0.76c0.77cN/Acrs283717060.0a0.197a0.0a0.007a0.002a0.001a0.0002aN/AbN/Ab2.4b0.71b0.1b0.05bN/AcN/Ac0.12c0.39c0.75c0.81crs50306560.0a0.004a0.0a0.012a0.03a0.002a0.015a1.4bN/Ab4.3b10.4b0.6b5.2b0.22cN/Ac0.04c0.001c0.41c0.02cNotes:
aMinor allele frequency, bChi-square value, cp-value, when p-value < 0.05 is considered a significant difference.Abbreviation: N/A, not applicable.

VIP Variants Within the Pharmacogenes in Chechen Compared to Six ExAC Populations Worldwide

Notes:
aMinor allele frequency, bChi-square value, cp-value, when p-value < 0.05 is considered a significant difference.

Abbreviation: N/A, not applicable.

### Discussion

This study was conducted on Chechens group living in Jordan. This subpopulation has a distinct genetic makeup as a genetically isolated population.19 Our findings demonstrate that the Chechen living in Jordan may have widely varying genetic allele distribution for clinically relevant SNPs compared to other populations. Assessment of the VIP genetic polymorphism frequencies within such a minority group can attribute to studies of the theoretical basis of drug toxicity and efficacy.

Several single nucleotide variants within the CYP450 genes have been reported as clinically significant. For example; within the CYP2J2 gene, the variant rs890293 genotypes AA + AC have been shown to be correlated to an increased risk of nausea and vomiting when treated with tacrolimus in individuals with Kidney Transplantation as compared to genotype CC.20

rs4244285 (GG) genotype of CYP2C19 is associated with increased risk of toxicity when treated with cyclophosphamide in women with Lupus Erythematosus, as compared to genotypes AA + AG.21 Besides, patients with the genotype GG of rs4986893 and treated with phenytoin may have a decreased likelihood of drug reaction with eosinophilia and systemic symptoms (DRESS) as compared to patients with the AG and AA genotypes.22 It also was found that allele A is associated with an increased risk of cardiovascular events when treated with clopidogrel in patients with coronary disease as compared to genotype GG.23 Moreover, allele A of rs4986893 is associated with decreased clopidogrel inhibition of ADP-induced platelet aggregation when treated with clopidogrel in healthy individuals as compared to genotype GG.24,25

Furthermore, our findings reveal that the alternative allele (T) frequency of rs1229984 in ADH1B gene among Chechen was much less than in Latino (94%), African (99%), European non-Finnish (95%), South Asian (95%) and European Finnish (99%) according to ExAC database. rs1229984 was reported as a clinically significant variant, whereas TT genotype was associated with increased Vmax of ethanol in healthy individuals as compared to genotypes CC and CT among mixed population.26

Furthermore, within ADRB2 gene, the frequency of the minor allele (G) of rs1042714 among Chechen was significantly lower than it among others Latino (83%), African 82%, European non-Finnish (58%), South Asian 79 and European Finnish (63%) and East Asian (91%). It was suggested that allele G of rs1042714 was associated with an increased reduction in resting blood pressure when treated with carvedilol in healthy individuals as compared to allele G in a study conducted in a mixed population.27 While Genotype CC was reported to be associated with decreased improvement in heart failure patients when treated with carvedilol compared to genotypes GG and CG.28

In this study, we found that rs5219 of KCNJ11 gene among Chechen was distributed differently from other populations. It was proposed that the Genotype (TT) is associated with an increased likelihood of Diabetes II Mellitus.29

One significant limitation for this study is the sample size, large study cohort is recommended to compare the variants frequencies with other population. In addition, better designed statistical analyses should be provided for accurate results. In conclusion, rare variants detected in isolated populations can significantly guide to understanding a potential biological process and identifying genetic loci involved in the development of clinically relevant treatment for different diseases.



# SUPPLEMENTAL FILE 1: pgpm-13-199.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: get_supplementary_file.php

[File available at: kcnH2_run/KCNH2/20251126_134514/pmc_fulltext/32765042_supplements/get_supplementary_file.php]

