# MAIN TEXT

## Generation of three heterozygous KCNH2 mutation-carrying human induced pluripotent stem cell lines for modeling LQT2 syndrome

### Abstract

Congenital long QT syndrome type 2 (LQT2) results from KCNH2 mutations that cause loss of Kv11.1 channel function which can lead to arrhythmias, syncope, and sudden death. Here, we generated three human-induced pluripotent stem cell (iPSC) lines from peripheral blood mononuclear cells (PBMCs) of two LQT2 patients carrying pathogenic variants (c.1714G > A and c.2960del) and one LQT2 patient carrying a variant of uncertain significance (c.1870A > T) in KCNH2. All lines show typical iPSC morphology, high expression of pluripotent markers, normal karyotype, and differentiate into three germ layers in vitro. These lines are valuable resources for studying the pathological mechanisms of LQTS caused by caused by KCNH2 mutations.

### Resource utility

These iPSC lines generated from two individuals carrying pathogenic variants and from an individual carrying a variant of uncertain significance in KCNH2 gene provide an unlimited source for differentiating iPSC-derived cardiomyocytes (iPSC-CMs) in vitro. They are an excellent tool for modelling LQT2 to elucidate underlying pathological mechanisms and for drug screenings (Table 1).

### Resource details

The long QT syndrome (LQTS) is a hereditary arrhythmogenic disorder characterized by a prolongation of the QT interval on electrocardiogram (ECG) that predisposes patients to life-threatening ventricular arrhythmias such as Torsade de Pointes (TdP) and sudden cardiac death (Anderson et al., 2006). The KCNH2 or human ether-a-go-go-related gene 1 (hERG1) encodes the Kv11.1 channel α subunit, which underlies the rapidly activating delayed rectifier K+ current (IKr) in the heart during phases 2 and 3 of the cardiac action potential, thus playing an important role in cardiac repolarization (De Zio et al., 2019). LQT2 results from KCNH2 mutations that cause loss of Kv11.1 channel function (Anderson et al., 2006). Genetic analyses have identified ~200 LQT2-associated KCNH2 mutations, with autosomal dominant missense (single amino acid substitution) mutations resulting most commonly in protein functional abnormalities (Anderson et al., 2006). Treatment begins with β-blockers unless there are valid contraindications (De Zio et al., 2019).

Here, we report three iPSC lines from patients carrying pathogenic variants (c.1714G > A; c.2960del) or a variant of uncertain significance (c.1870A > T) in the KCNH2 gene, providing a resource for investigating inherited arrhythmia syndromes and testing antiarrhythmic strategies (Garg et al., 2018). The iPSC lines were reprogrammed from peripheral blood mononuclear cells (PBMCs) isolated from an 18-year-old woman with LQT2 (c.1714G > A), a 27-year-old man with LQT2 (c.1870A > T), and a 20-year-old woman with LQT2 (c.2960del) (Table 3). Reprogramming from PBMCs to iPSCs was performed using Sendai virus containing the four Yamanaka factors. The three iPSC clones had normal morphology and karyotype, as evaluated by the KaryoStat assay (passages 10 and 13, respectively). High expression of the pluripotency markers OCT3/4, NANOG, and SOX2 was shown by immunostaining. Reverse transcription-quantitative polymerase chain reaction (RT-qPCR) confirmed the comparable expression of Sox2 and Nanog mRNA in these three iPSC lines with that of the widely used positive control iPSC line (Sun et al., 2012) (Fig. 1C). Additionally, the iPSCs were able to differentiate into derivatives of all three germ layers (Fig. 1D). The three heterozygous (c.1714G > A, c.1870A > T, and c.2960del) mutations were confirmed by Sanger sequencing (Fig. 1D and Table 2). All lines were mycoplasma-negative. The origin of these three lines was confirmed by short tandem repeat (STR) analysis which demonstrated that the line profiles completely overlapped with those of the donor somatic cells.

### Materials and methods

PBMCs were isolated from blood by PercollR separation, purified using DPBS and plated in a 24-well plate as previously described (Liu et al., 2021). PBMCs (~1–2 × 106) were cultured in 1 mL of complete Stem-Pro™-34 medium combined with its specific supplements (in ng/mL): 100 FLT3, 20 IL-6 and 20 EPO, 20 IL-3 and 100 SCF as previously described (Liu et al., 2021). Stem-Pro™-34 medium was changed every two days until cell culture stabilization. About 2 × 105 PBMCs resuspended in 300 μl of Stem-Pro™-34 medium were infected with a Sendai virus reprogramming cocktail instructions of the CytoTune®-iPSC Sendai Reprogramming Kit from Thermo Fisher following manufacturer’s (Liu et al., 2021). After 24 h, cells were replated in one well of a Matrigel-coated 12-well plate. Stem-Pro™-34 medium was replaced every two days until day 7, when 1 mL of supplemented StemMACS™ iPS-Brew XF medium (Miltenyi Biotec) was added on top of the Stem-Pro™-34 medium. On day 8, the medium was replaced with fresh StemMACS™ medium. Fresh StemMACS™ iPS-Brew XF medium was changed after Sendai virus infection every two days until days 10–15, when colonies appeared and were prepared for picking and expanded as previously described (Liu et al., 2021).

### Reprogramming

PBMCs were isolated from blood by PercollR separation, purified using DPBS and plated in a 24-well plate as previously described (Liu et al., 2021). PBMCs (~1–2 × 106) were cultured in 1 mL of complete Stem-Pro™-34 medium combined with its specific supplements (in ng/mL): 100 FLT3, 20 IL-6 and 20 EPO, 20 IL-3 and 100 SCF as previously described (Liu et al., 2021). Stem-Pro™-34 medium was changed every two days until cell culture stabilization. About 2 × 105 PBMCs resuspended in 300 μl of Stem-Pro™-34 medium were infected with a Sendai virus reprogramming cocktail instructions of the CytoTune®-iPSC Sendai Reprogramming Kit from Thermo Fisher following manufacturer’s (Liu et al., 2021). After 24 h, cells were replated in one well of a Matrigel-coated 12-well plate. Stem-Pro™-34 medium was replaced every two days until day 7, when 1 mL of supplemented StemMACS™ iPS-Brew XF medium (Miltenyi Biotec) was added on top of the Stem-Pro™-34 medium. On day 8, the medium was replaced with fresh StemMACS™ medium. Fresh StemMACS™ iPS-Brew XF medium was changed after Sendai virus infection every two days until days 10–15, when colonies appeared and were prepared for picking and expanded as previously described (Liu et al., 2021).

### Cell culture

iPSCs were cultured in sterile conditions using a supplemented StemMACS™ iPS-Brew XF medium and changed every other day. At 90% confluency, iPSCs were passaged using 10 μM Y-27632, a potent inhibitor of ROCK1 (Selleck Chemicals), diluted in StemMACS™ medium. Y27632 was withdrawn after 24 h. iPSCs were maintained at 37 °C, 85% RH and 5% CO2.

### Karyotyping

The whole-genome array for detection of chromosomal abnormalities was performed with KaryoStat™ assay from Thermo Fisher in iPSCs at passages 10–13.

### Trilineage differentiation

The STEMdiff™ Trilineage Differentiation Kit from STEMCELL Technologies was used to functionally validate the ability of new iPSCs lines to differentiate into the three germ layers following manufacturer’s instructions.

### Immunofluorescent staining

For qualitative analysis of pluripotency and positive three germ layer markers, iPSCs were fixed for 10 min at room temperature using 4% paraformaldehyde. Hereinafter iPSCs were permeabilized 10 min at room temperature using 50 μg/mL digitonin (Sigma Aldrich), and blocked for 30 min at room temperature with 1% of BSA plus 5% of FBS in PBS. iPSCs were incubated overnight at 4 °C with primary antibodies diluted in 1% BSA-PBS (Table 2). At second day, iPSCs were incubated with secondary antibody in 1% BSA-PBS (Table 2) for 30–60 min at room temperature. Nuclei were counterstained using NucBlue™ from Invitrogen™.

### RT-qPCR

Total RNA was extracted and isolated using the Direct-zol™ RNA Miniprep Kit from ZYMO RESEARCH according to manufacturer’s protocol. RT-PCR was performed using iScript™cDNA Synthesis Kit from BIORAD using following protocol: 5 min at 25 °C, 20 min at 46 °C, and 1 min at 95 °C. Sox2 and Nanog were amplified using commercial primers (Table 2) and TaqMan™ Gene Expression Assay from Applied Biosystems™.

### Short tandem repeat analysis (STR)

To verify the origin of these newly generated iPSC lines, gDNA from PBMCs and iPSCs were isolated and purified using DNeasy Blood & Tissue Kit from Qiagen. STR analysis were performed using CLA Identifiler™ Plus and Identifiler™ Direct PCR Amplification Kits from Thermo Fisher by Stanford PAN Facility.

### Sanger sequencing

PCR primers were designed to amplify a 300- to 500-bp products flanking each KCNH2 mutations (Table 2). gDNA was isolated from iPSCs using DNeasy Blood & Tissue Kit (Qiagen) following manufacturer’s protocol. KCNH2 mutation analysis was performed on a purified PCR product obtained from gDNA amplification using NEBNext High-Fidelity (New England Biolabs). PCR product was purified using QIA-quick Purification Kit from Qiagen and sent for Sanger sequencing.

### Mycoplasma detection

Mycoplasma contamination was evaluated using the MycoAlert Detection Kit (Lonza) following manufacturer’s instructions.

### Supplementary Material



# SUPPLEMENTAL FILE 1: nihms-1955648.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: PMC10875632

[File available at: kcnH2_run/KCNH2/20251126_134514/pmc_fulltext/34051449_supplements/PMC10875632]

