# MAIN TEXT

## Population Genetic Difference of Pharmacogenomic VIP Variants in the Tibetan Population

### Abstract

BackgroundGenetic variation influences drug reaction or adverse prognosis. The purpose of this research was to genotype very important pharmacogenetic (VIP) variants in the Tibetan population.Methods and MaterialsBlood samples from 200 Tibetans were randomly collected and 59 VIP variants were genotyped, and then compared our data to 26 other populations in the 1000 project to further analyze and identify significant difference.ResultsThe results showed that on comparing with most of the 26 populations from the 1000 project, rs4291 (ACE), rs1051296 (SLC19A1) and rs1065852 (CYP2D6) significantly differed in the Tibetan population. Furthermore, three significant loci were related to drug response. In addition, the allele frequency of Tibetans least differed from that of East Asian populations, and most differed from that of Americans.ConclusionThree significant loci of variation ACE rs4291, SLC19A1 rs1051296 and CYP2D6 rs1065852 were associated with drug response. This result will contribute to improving the information of the Tibetan in the pharmacogenomics database, and providing a theoretical basis for clinical individualised drug use in Tibetans.

### Introduction

A major challenge in current drug clinical practice, drug development and drug regulation is the huge difference in drug treatment1 which is reasoned to the individual differences for the same drug.2 Single nucleotide polymorphism (SNP) is an important determinant of individual differences in drug therapy. Pharmacogenomics mainly involves knowledge of pharmacokinetic, pharmacodynamic and focuses on the inheritance of individual variation in the course of drug treatment, clarifies the influence of genomic variation on drug distribution and function, and provides guidance for individualised precise medical treatment3,4 so on. The Pharmacogenomics Knowledge Base (PharmGKB: http://www.pharmgkb.org) is a useful resource that aims to explain the gene-drug-disease relationship. In recent years, pharmacogenomics research has largely focused on genetic variations that may be related to drug response or metabolism.5 These variations–very important genetic (VIP) variants are mainly concentrated in the pharmacogenomics database. They assessed the relationship between the VIP variants and specific drugs, and provide individuals with appropriate drugs at proper doses.6

There are many differences in genetic heterogeneity and genetic polymorphism among different ethnic groups.7 An important research hotspot in pharmacogenomics is to compare the efficacy of drugs among different races. Tibetan is one of the largest of the 56 ethnic Chinese nation, with a long history. Most Tibetans reside in the Tibet Autonomous Region on the Qinghai-Tibet Plateau, and some people also live in Qinghai, Gansu, Sichuan, Yunnan and other regions of China. The Tibetans have gradually formed unique ethnic custom, lifestyles and eating habits based on the climate and altitude of their residing areas. A study of the genomic variants in the Tibetan population showed that most of the Tibetan gene pool may be differ from the gene pool of the Han people.8

This study aimed to assess the genotype frequencies of 59 VIP variants between Tibetans and the other 26 ethnicities from the 1000 genome project. The results will contribute to improving the information of the Tibetan pharmacogenomics database, and providing a theoretical basis for drug management in Tibetans.

### Materials and Methods

200 unrelated Tibetans were recruited from the Tibet Autonomous Region. According to the paternal lineage of at least past three generations, all candidates were confirmed to be Tibetan. Based on the results of physical examination, people with chronic diseases were excluded. The protocol of this study was known to all participants and they have signed informed consent. According to the ethics committee of Xizang Minzu University (2019-12), all participants provided blood samples. The procedures met the agency’s ethical standards and the 1964 Helsinki Declaration and its subsequent amendments or similar ethical standards.

The 59 VIP variants were selected from the PharmGKB database (http://www.pharmgkb.org). The genomic DNA of participants was extracted according to the extraction method of the blood DNA extraction kit (GoldMag Ltd. Xi’an, China), and then its DNA purity and concentration were inspected. Multiplexed SNP MassEXTEND assays were designed by the Agena MassARRAY Assay Design 3.0 software, and the Agena MassARRAY RS1000 was used to genotype the 59 VIP variants.9 Data arrangement and analysis was performed by Agena Typer 4.0 software.10,11

To determine whether the genotype frequencies of variants were in Hardy-Weinberg equilibrium, we used Microsoft Excel and SPSS 22.0 (SPSS, Chicago, IL) for statistical analysis. The genotype frequencies of 59 variants in the Tibetan population were separately compared with those of the other super-populations downloaded from the 1000 genomes project (http://www.internationalgenome.org/), including ①African: the African Caribbeans in Barbados (ACB); the African Ancestry in Southwest USA (ASW); Esan in Nigeria (ESN); Gambian in Western Divisions, The Gambia (GWD); the Luhya in Webuye, Kenya (LWK); the Mende in Sierra Leone (MSL); the Yoruba in Ibadan, Nigeria (YRI); ②American: the Colombian in Medellin, Colombia (CLM); the Mexican Ancestry in Los Angeles, Colombia (MXL); the Peruvian in Lima, Peru (Peruvian in Lima, Peru); the Puerto Rican in Puerto Rico (PUR); ③East Asian: the Chinese Dai in Xishuangbanna, China (CDX); the Han Chinese in Beijing, China (CHB0; the Southern Han Chinese, China (CHS); the Japanese in Tokyo, Japan (JPT); the Kinh in Ho Chi Minh City, Vietnam (KHV); ④European: the Utah residents with Northern and Western European ancestry (CEU); the Finnish in Finland (FIN); the British in England and Scotland (GBR); the Iberian populations in Spain (IBS); the Toscani in Italy (TSI); ⑤South Asian: the Bengali in Bangladesh (BEB); the Gujarati Indian in Houston, Texas (GIH); the Indian Telugu in the UK (ITU); the Punjabi in Lahore, Pakistan (PJL) and the Sri Lankan Tamil in the UK (STU).12 Also, Bonferroni’s multiple adjustments were used for significance assessment, and all p values were double-sided when p < 0.05 and p < 0.05/(59 × 27), the p value was considered to be statistically significant.13

### Study Participants

200 unrelated Tibetans were recruited from the Tibet Autonomous Region. According to the paternal lineage of at least past three generations, all candidates were confirmed to be Tibetan. Based on the results of physical examination, people with chronic diseases were excluded. The protocol of this study was known to all participants and they have signed informed consent. According to the ethics committee of Xizang Minzu University (2019-12), all participants provided blood samples. The procedures met the agency’s ethical standards and the 1964 Helsinki Declaration and its subsequent amendments or similar ethical standards.

### Variant Selection, DNA Extraction and Genotyping

The 59 VIP variants were selected from the PharmGKB database (http://www.pharmgkb.org). The genomic DNA of participants was extracted according to the extraction method of the blood DNA extraction kit (GoldMag Ltd. Xi’an, China), and then its DNA purity and concentration were inspected. Multiplexed SNP MassEXTEND assays were designed by the Agena MassARRAY Assay Design 3.0 software, and the Agena MassARRAY RS1000 was used to genotype the 59 VIP variants.9 Data arrangement and analysis was performed by Agena Typer 4.0 software.10,11

### Statistical Analyses

To determine whether the genotype frequencies of variants were in Hardy-Weinberg equilibrium, we used Microsoft Excel and SPSS 22.0 (SPSS, Chicago, IL) for statistical analysis. The genotype frequencies of 59 variants in the Tibetan population were separately compared with those of the other super-populations downloaded from the 1000 genomes project (http://www.internationalgenome.org/), including ①African: the African Caribbeans in Barbados (ACB); the African Ancestry in Southwest USA (ASW); Esan in Nigeria (ESN); Gambian in Western Divisions, The Gambia (GWD); the Luhya in Webuye, Kenya (LWK); the Mende in Sierra Leone (MSL); the Yoruba in Ibadan, Nigeria (YRI); ②American: the Colombian in Medellin, Colombia (CLM); the Mexican Ancestry in Los Angeles, Colombia (MXL); the Peruvian in Lima, Peru (Peruvian in Lima, Peru); the Puerto Rican in Puerto Rico (PUR); ③East Asian: the Chinese Dai in Xishuangbanna, China (CDX); the Han Chinese in Beijing, China (CHB0; the Southern Han Chinese, China (CHS); the Japanese in Tokyo, Japan (JPT); the Kinh in Ho Chi Minh City, Vietnam (KHV); ④European: the Utah residents with Northern and Western European ancestry (CEU); the Finnish in Finland (FIN); the British in England and Scotland (GBR); the Iberian populations in Spain (IBS); the Toscani in Italy (TSI); ⑤South Asian: the Bengali in Bangladesh (BEB); the Gujarati Indian in Houston, Texas (GIH); the Indian Telugu in the UK (ITU); the Punjabi in Lahore, Pakistan (PJL) and the Sri Lankan Tamil in the UK (STU).12 Also, Bonferroni’s multiple adjustments were used for significance assessment, and all p values were double-sided when p < 0.05 and p < 0.05/(59 × 27), the p value was considered to be statistically significant.13

### Results

In the Table 1, we listed the basic characteristics of 59 VIP variants in 200 Tibetans and because of the nonspecific primers, 8 SNPs (rs1801252, rs1801160, rs2892949, rs12208357, rs1801279, rs10509681, rs1058930 and rs11572080) have been deleted. The selected SNP PCR primers were shown in Supplementary Table 1. The selected VIP gene information contained gene name, position, genotype frequency, the minor allele frequency and the functional consequence.Table 1Basic Characteristics of the Selected VIP Variants from the PharmGKB Database and Genotype Frequencies in Tibetan PopulationSNPGenesChrPositionGenotypeMAFFunctional ConsequenceAAABBBrs11572325CYP2J2159,896,030323173T=0.073Intron_variantrs10889160CYP2J2159,896,449647146C=0.148Intron_variantrs890293CYP2J2159,926,82209894A=0.255Upstream_transcript_variantrs1760217DPYD197,137,438269579G=0.368Genic_downstream_transcript_variantrs1801159DPYD197,515,8391165124C=0.218Coding_sequence_variantrs1801158DPYD197,515,86501199T=0.003Coding_sequence_variantrs1801265DPYD197,883,329641152G=0.133Non_coding_transcript_variantrs5275PTGS21186,673,9261668115G=0.2513_prime_UTR_variantrs20417PTGS21186,681,18907193G=0.018Upstream_transcript_variantrs12139527CACNA1S1201,040,054127172G=0.073Missense_variantrs13374149CACNA1S1201,043,35603197T=0.008Missense_variantrs3850625CACNA1S1201,047,168012188A=0.030Coding_sequence_variantrs12742169CACNA1S1201,083,182018182T=0.045Missense_variantrs12406479CACNA1S1201,110,21601199C=0.003Missense_variantrs2306238RYR21237,550,8031273113A=0.245Intron_variantrs2231142ABCG2488,131,171029171T=0.073Coding_sequence_variantrs2231137ABCG2488,139,962368874T=0.404Coding_sequence_variantrs698ADH1C499,339,632538156C=0.121Coding_sequence_variantrs776746CYP3A5799,672,9161248139T=0.181Intron_variantrs2242480CYP3A4799,763,8431166116T=0.228Intron_variantrs1805123KCNH27150,948,44602000G=0.500Missense_variantrs4646244NAT2818,390,208670123A=0.206Upstream_transcript_variantrs4271002NAT2818,390,758475118C=0.211Upstream_transcript_variantrs1041983NAT2818,400,285398474T=0.411Coding_sequence_variantrs1801280NAT2818,400,344012188C=0.030Missense_variantrs1799929NAT2818,400,484011189T=0.028Coding_sequence_variantrs1799930NAT2818,400,5931312066A=0.367Missense_variantrs1208NAT2818,400,806012188G=0.030Missense_variantrs1799931NAT2818,400,860765128A=0.198Missense_variantrs1495741NAT2818,415,371418865G=0.438Noners2115819ALOX51045,405,641657137A=0.173Intron_variantrs12248560CYP2C191094,761,90006189T=0.015Upstream_transcript_variantrs4244285CYP2C191094,781,859878107A=0.244Coding_sequence_variantrs1057910CYP2C91094,981,296123176C=0.063Missense_variantrs11572103CYP2C81095,058,349012188A=0.030Missense_variantrs7909236CYP2C81095,069,673558136T=0.171Upstream_transcript_variantrs17110453CYP2C81095,069,7721066123C=0.216Upstream_transcript_variantrs3813867CYP2E110133,526,101026174C=0.065Non_coding_transcript_variantrs2031920CYP2E110133,526,341252142T=0.143Non_coding_transcript_variantrs6413432CYP2E110133,535,04001199A=0.003Intron_variantrs2070676CYP2E110133,537,6331058132G=0.195Intron_variantrs5219KCNJ111117,388,025611076T=0.318Missense_variantrs1801028DRD211113,412,762027173C=0.068Missense_variantrs2306283SLCO1B11221,176,8041087100A=0.272Missense_variantrs4516035VDR1247,906,043038162C=0.095Upstream_transcript_variantrs2472297CYP1A21574,735,53901199T=0.003Noners762551CYP1A21574,749,576338084C=0.371Intron_variantrs2472304CYP1A21574,751,897121178A=0.058Intron_variantrs750155SULT1A11628,609,2511516022T=0.4825_prime_UTR_variantrs1800764ACE1763,473,168379073C=0.410Noners4291ACE1763,476,83301991T=0.498Upstream_transcript_variantrs4267385ACE1763,506,395555138T=0.164Noners2108622CYP4F21915,879,6211770113T=0.260Missense_variantrs3093105CYP4F21915,897,578339158C=0.113Missense_variantrs8192726CYP2A61940,848,591939149A=0.145Intron_variantrs1051298SLC19A12145,514,912812564G=0.358Intron_variantrs1051296SLC19A12145,514,947614841A=0.410Intron_variantrs1131596SLC19A12145,538,002914540A=0.420Missense_variantrs1065852CYP2D62242,130,6921313547A=0.413Intron_variant

Basic Characteristics of the Selected VIP Variants from the PharmGKB Database and Genotype Frequencies in Tibetan Population

Based on the chi-squared test, as for the genotype frequencies of 59 variations, we made a comparison between Tibetans and other 26 nationalities (Supplementary Table 2). Before adjustments, ACE rs4291, SLC19A1 rs1051296 and CYP2D6 rs1065852 significantly differed in the Tibetan population when compared with the other 26 nationalities. The SULT1A1 rs750155, SLC19A1 rs1051298 and SLC19A1 rs1131596 in the Tibetan population were obviously different from those of the other nationalities. Bonferroni adjustments have been made (p < 0.05/(59 × 27)). The results showed that ACE rs4291 and CYP2D6 rs1065852 were significantly different in the Tibetan population when compared to the other 26 nationalities. Except for CHS, SLC19A1 rs1051296 significantly differed in Tibetans when compared with the other 25 ethnic groups. Moreover, SULT1A1 rs750155 and SLC19A1 rs1131596 in the Tibetan population differed from those of the other 22 nationalities.

In addition, on the basis of Pharmgkb database (https://www.pharmgkb.org/), Table 2 showed the drug-related information of significant difference. The rs4291 carriers with different genotypes have different responses to drugs (sertraline, captopril, aspirin and amlodipinechlorthalidonelisinoprilj). Rs1065852 was found to be related to alpha-hydroxymetoprolol, citalopram escitalopram and iloperidone. As for rs1051298, compared to the allele A individuals, individuals with allele G were associated with increased progression free survival after bevacizumab and pemetrexed treatment. Compared with allele C, rs750155 T allele was not associated with ABT-751 pharmacokinetic parameters in cancer patients receiving ABT-751 treatment. And when comparing to the allele A, rs1131596 G allele was not correlated with the response to methotrexate in children with precursor cell lymphoblastic leukaemia lymphoma and patients with rheumatoid arthritis. The correlation between rs1051296 and clinical drugs has not been reported to date.Table 2Significant Difference SNP and Drug Related InformationVariantPMIDMoleculesAssociationSingnificantP-value#Of Case#Of ControlStudy SizeBipgeographical GroupPaper DiscussesGeneMore Detailsrs429127,262,302SertralineGenotype TT is not associated with response to sertraline in people with Depressive Disorder.No0.8035571126Near EasternEfficacyACEAs compared to fluoxetine. Patients with major depressive disorder were randomized to receive sertraline or fluoxetine. Depression severity determined w/ the 21-item Hamilton Rating Scale for Depression (HAMD-21) before and after the treatment. Response calculated based on 50% reduction in the reported scores. Given allele frequencies were compared between treatment groups in the responsive patients only.rs429127,546,928CaptoprilGenotype AA is associated with decreased severity of Kidney Failure when treated with captopril in people with Alzheimer Disease as compared to genotypes AT + TT.Yes0.029190190UnknownEfficacyACEThe A allele conferred a protective effect of creatinine increases over 1 year. Association with the effects on creatinine also found in conjunction with the rs1800764CT genotype.rs429118,727,619AspirinGenotypes AT + TT are associated with increased risk of aspirin intolerance when exposed to aspirin in people with Asthma as compared to genotype AA.Yes0.01581231312East AsianToxicityACErs429120,577,119AmlodipinechlorthalidonelisinoprilGenotypes AA + AT are associated with decreased fasting glucose when treated with amlodipine, chlorthalidone or lisinopril in people with Hypertension as compared to genotype TT.Yes0.001493099309Mixed PopulationOtherACErs106585210,223,777Alpha-hydroxymetoprololAllele A is associated with decreased clearance of alpha-hydroxymetoprolol in healthy individuals as compared to allele G.Yes< 0.054040East AsianMetabolism/PKCYP2D6Alleles complemented to plus strand, relationship described for T188 (P34S). As measured by urinary metabolite concentrations.rs106585224,528,284CitalopramescitalopramAllele A is associated with plasma concentration of S-didesmethyl-citalopram when treated with citalopram or escitalopram in people with Depressive Disorder, Major as compared to allele G.Yes2E-16435435EuropeanOtherCYP2D6Direction of the association was not given, p = 2.0E-16rs106585223,277,250IloperidoneGenotype GG is associated with increased QTc interval when treated with iloperidone in people with Schizophrenia as compared to genotypes AA + AG.Yes0.028128128UnknownOtherCYP2D6The baseline QTc interval was calculated by averaging the results of 3 electrocardiogram (ECG) measurements per day over 3 consecutive days leading up to iloperidone treatment initiation. The QTc interval at maximum iloperidone blood concentration (Tmax) was calculated by averaging 3 ECG measurements per day (taken 2, 3 and 4 hours after iloperidone was administered) over the last 3 consecutive days of the iloperidone treatment period. From these two averaged measurements, the least squares mean (LSM) change in QTc interval was calculated for each genotype. Patients with the GG genotype had a significantly higher LSM change from baseline, and therefore a greater increase in QTc interval after iloperidone administration, compared to those with the AA or AG genotypes. Please note alleles have been complemented to the positive chromosomal strand.rs105129819,841,321BevacizumabpemetrexedAllele G is associated with increased progression-free survival when treated with bevacizumab and pemetrexed in people with Lung Neoplasms as compared to allele A.Yes0.014848EuropeanEfficacySLC19A1This clinical trial evaluated the efficacy and toxicity of pemetrexed combined with bevacizumab as second-line therapy for patients with advanced non-small-cell lung cancer (NSCLC) and to correlate allelic variants in pemetrexed-metabolizing genes with clinical outcome. Patients with previously treated NSCLC received pemetrexed (500 mg/m2 intravenous) combined with bevacizumab (15 mg/kg intravenous) every 3 weeks. The primary end point, evaluated using a one-stage Fleming design for detecting a true success rate of at least 70%, was the proportion of patients who were progression free and on treatment at 3 months. Polymorphisms in genes responsible for pemetrexed transport (reduced folate carrier [SLC19A1]) and metabolism (folylpolyglutamate synthase [FPGS] and gamma-glutamyl hydrolase [GGH]) evaluated in germline DNA (blood) were correlated with treatment outcome. The primary end point for this study was 3-month PFS, and only patients with the SLC19A1 exon 6 (2522)C3T polymorphism showed a significant association with that end point (GG vs AG vs AA: 57%v29%v 78%, respectively; Fisher’s exact test, P = 0.01). Please note that alleles have been complemented to the plus chromosomal strand.rs105129824,732,178PemetrexedGenotypes AA + AG is associated with decreased overall survival when treated with pemetrexed in people with Carcinoma, Non-Small-Cell Lung and Mesothelioma as compared to genotype GG.Yes0.016136136Mixed PopulationEfficacySLC19A1Advanced stage non-small-cell lung cancer (n=94), malignant mesothelioma (n=42). No association with progression-free survival was seen. Please note alleles have been complemented to the plus chromosomal strand.rs75015523,670,235ABT-751Allele T is not associated with ABT-751 pharmacokinetic parameters when treated with ABT-751 in people with Neoplasms as compared to allele C.No4747Mixed PopulationMetabolism/PKSULT1A1p-value>0.05rs75015515,970,794Allele T is associated with increased activity of SULT1A1 platelets.Not statedSULT1A1In Caucasians but decreased activity in African Americans.rs113159628,525,903MethotrexateAllele G is not associated with response to methotrexate in children with Precursor Cell Lymphoblastic Leukemia-Lymphoma as compared to allele A.No> 0.05317317East AsianEfficacySLC19A1Please note: alleles have been complemented to the + chromosomal strand.rs113159617,404,734MethotrexateAllele G is not associated with response to methotrexate in people with Arthritis, Rheumatoid.Not stated106106UnknownSLC19A1This variant alone is insufficient to predict patient response to MTX therapy.rs113159628,525,903MethotrexateGenotypes AG + GG are not associated with concentrations of methotrexate in children with Precursor Cell Lymphoblastic Leukemia-Lymphoma as compared to genotype AA.No0.219317317East AsianMetabolism/PKSLC19A1There is no association between selected SNPs and methotrexate plasma level at 48 h between the first dose of methotrexate infusion. med. MTX concentration: AG+GG 0.41 (0.09–34.05)) vs AA (0.6 (0.14–41.63)). Please note: alleles have been complemented to the + chromosomal strand.rs113159617,404,734Allele G is associated with decreased expression of SLC19A1.Not statedSLC19A1rs113159628,525,903MethotrexateGenotypes AG + GG are not associated with risk of mucositis when treated with methotrexate in children with Precursor Cell Lymphoblastic Leukemia-Lymphoma as compared to genotype AA.No0.671317317East AsianToxicitySLC19A1Mucositis refers to grade 3–4 oral mucositis. Please note: alleles have been complemented to the + chromosomal strand.

Significant Difference SNP and Drug Related Information

The MAF distribution map of SNPs ACE (rs4291 T allele), SLC19A1 (rs1051296 A allele), CYP2D6 (rs1065852 A allele), SULT1A1 (rs750155 T allele), SLC19A1 (rs1051298 G allele) and SLC19A1 (rs1131596 A allele) were compared with significant differences between the Tibetans and the other super-populations. As shown in Supplementary Figure 1, the allele frequency of the Tibetans least differed from the East Asian populations, and most differed from Americans. Among them, the frequencies of rs4291-T and rs1065852-A were the highest in the East Asian population, whereas the frequency of rs1051296-A was the highest in the African population. The frequency of rs1051298-G, rs750155-T and rs1131596-A were the highest in the American population.

### Discussion

In recent years, many studies have focused on the efficacy comparison of drug reactions among different races, which lays a foundation for clinical individualised drug use. This study identified 59 VIP loci in Tibetans and compared them with the other 26 different populations. Three significant loci of variation were inferred from the genotyping results, ACE rs4291, SLC19A1 rs1051296 and CYP2D6 rs1065852, which were associated with drug response.

Angiotensin-converting enzyme (ACE) encodes an enzyme involved in blood pressure regulation and electrolyte balance. It catalyses the conversion of angiotensin I into the physiologically active peptide angiotensin II. Angiotensin II promotes hypertension through vasoconstriction and salt and water retention, and is helpful for heart remodelling, inflammation, thrombosis and plaque rupture.14 At present, ACE inhibitors are the first-line treatment for hypertension, which can favourably affect the remodelling of patients with myocardial infarction and heart failure, and reduce the incidence rate and mortality.15 Therefore, ACE-encoded enzyme is the drug target.16,17 Polymorphic loci have been reported to play a role.18,19 Martínez-Rodríguez et al20 reported that rs4291 was associated with an increased risk of hypertension after adjusting for age, gender, BMI, triglyceride, drinking and smoking and with the elevation of ACE enzyme levels. In addition, the function of ACE polymorphisms in Alzheimer’s Disease cannot be ignored. The study by de Oliveira et al21 showed that ACE inhibitors, not related to blood pressure, can delay cognitive decline in ACE haplotype carriers with rs1800764-T and rs4291-A. In 2016, they also observed that each A allele of rs4291 resulted in an increase of 3.074 mg/dL urea and 0.044 mg/dl creatinine per year. The use of ACE inhibitors had a protective effect on the change of creatinine, and had no effect on the change of blood pressure.22 The above research showed that rs4291 carriers with different genotypes had different drug sensitivity.

The CYP2D6 gene encoded Cytochrome P450 Family 2 Subfamily D Member 6 that catalyses drug metabolism and the synthesis of cholesterol, steroids and other lipids. One report has shown that the gene is highly polymorphic in the population. Lee et al23 evaluated the association between CYP polymorphisms and the blood concentrations of hydroxychloroquine (HCQ) and its metabolite N-desethyl HCQ (DHCQ) in Korean patients with lupus. They observed that patients with CYP2D6*10 (rs1065852) GG genotype had the highest [DHCQ]/[HCQ] ratio, while patients with AA genotype had the lowest [DHCQ]/[HCQ] ratio. This study suggested that [DHCQ]/[HCQ] ratio was associated with CYP2D6 rs1065852 polymorphism in lupus patients receiving oral HCQ. The CYP polymorphism may explain why the HCQ concentration in blood varies greatly. The effect of individual CYP polymorphisms should be considered in oral HCQ. Recently, López-García et al24 found single nucleotide polymorphisms associated with pharmacogenomics include CYP2D6*4 (rs1065852), which may affect the efficacy of antiepileptic drugs. Dlouhá et al reported no significant difference in the genotype/allelic frequencies of CYP2D6 (rs1065852) in Roma/Gypsy and Czech (non-Roma) populations.25 However, Zhang et al observed the frequency of CYP2D6 rs1065852 in the Lisu population different from the other populations.26 In our study, rs1065852-A frequency was the highest in the South East Asian populations. Therefore, to evaluate the effect of CYP2D6 (rs1065852) on drug metabolism, we should not ignore the ethnic factors, especially in the Asian populations.

The membrane protein Folate carrier protein 1 is involved in the regulation of intracellular folate concentration, and is encoded by SLC19A1 (Solute Carrier Family 19 Member 1) gene.27 The mutant allele of SLC19A1 −43T>C was reported to be associated with low folate levels.28 At present, the treatment response to pemetrexed has been proven to be individual specific. Zhang et al investigated the genetic characteristics of pemetrexed response in 203 Han patients with advanced non-small cell lung cancer (NSCLC).29 The participants who received pemetrexed alone, the SNP rs1051298 of SLC19A1 gene increased the risk of all adverse reactions in different cycles of the treatment. Therefore, rs1051298 may be a marker associated with adverse reactions and efficacy of pemetrexed related therapy in Chinese Han patients. In addition, some researchers have found that rs1051296 was also related to drug reactions. Wang et al explored the influence of the miRNA binding site polymorphism (rs1051296) in SLC19A1 on the serum methotrexate (MTX) concentration in children with acute lymphoblastic leukaemia (ALL) after receiving MTX treatment.30 In comparison with the GT and TT carriers, the MTX concentration in GG carriers was higher than the treatment threshold. Compared with GT and TT carriers, the delayed elimination of MTX was observed in the GG carriers. Rs1051296 G>T correlated with the MTX plasma concentration. In the research, the frequency of rs1051296 in Tibetans differed from that in the super-populations, leading to different genotypes of carriers such as MTX, for some drug efficacy.

The results will improve the pharmacogenomic information of the Tibetan population, and deepen the study on the differences in some genetic polymorphisms between the Tibetan population and the other 26 populations in the world. Some limitations of this study are: a small sample size and lack of experiment verification. Further studies are required on the drug efficacy of drug-related gene polymorphism loci in the Tibetan patients, especially for significant loci.

### Conclusions

In conclusion, ACE rs4291, SLC19A1 rs1051296 and CYP2D6 rs1065852 were significant in Tibetans compared to the other 26 nationalities. This study supplemented the knowledge of Tibetan pharmacology, and also showed the relationship between the SNPs with significant differences and drugs more perfectly, thus making the clinical medication safer and personalized for the Tibetan population.



# SUPPLEMENTAL FILE 1: pgpm-14-1027.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: get_supplementary_file.php

[File available at: kcnH2_run/KCNH2/20251126_134514/pmc_fulltext/34429635_supplements/get_supplementary_file.php]

