# MAIN TEXT

## Analysis of Very Important Pharmacogenomics Variants in the Chinese Lahu Population

### Abstract

BackgroundGenetic polymorphism, obviously, has a potential clinical role in determining differences in drug efficacy; however, there are no reports about the pharmacogenomic information of the Lahu population. Therefore, our research aimed to screen the genotypic frequencies of the very important pharmacogenomics (VIP) mutations and determined the differences between Lahu and the other 11 populations.MethodsAgena MassARRAY (AgenaMassARRAY) single nucleotide polymorphism (SNP) genotyping technique was used to detect 81 VIP mutations of pharmacogenomics genes in Lahu, and their genotypic frequencies were compared with the other major 11 populations. Chi-square tests were used to identify different loci among these populations. Finally, the genetic structure and pairwise Fst values of Lahu and the other 11 populations were analyzed.ResultsWe found that the distribution of allele frequencies within different pharmacogenes in Lahu showed significantly different with other populations. Additionally, the pairwise F-statistics (Fst) values and genetic structure revealed the variants in the Lahu population as well were mostly related to the Han Chinese in Beijing, China (CHB) and the Japanese population in Tokyo, Japan (JPT) genetically.ConclusionThis study will provide a theoretical basis for safe drug use and help to establish the appropriate individualized treatment strategies in the Lahu population.

### Introduction

Pharmacogenomics is an organic combination of molecular pharmacology and gene function. Researchers use information from the entire genome to identify and describe the genetic basis and genetic influence of patients on drug therapy. As the most common type of genetic variation among people, single nucleotide polymorphisms (SNPs) constitute the basis of pharmacogenetics, which means the monogenic variants, which alter the drug response. Most importantly, the SNPs of drug metabolic enzymes and drug transporter genes are important determinants of variation among individual drug metabolites and of human therapeutic responses and disease susceptibility.1,2 What is more, individual differences in drug reactions and side effects are a major challenge in clinical pharmacology. Therefore, identifying these polymorphisms and understanding how they affect drug response and genetic disease trends are the key to drug genetics research.3 Pharmacogenomics can enhance the outcome of treatment by adopting pharmacogenomic testing to maximize drug efficacy and minimize the risk of serious adverse events.4 The most well-known pharmacogenes are cytochrome P450 genes, encoding Phase 1 cytochrome P450 (CYP) or Phase 2 drug-metabolizing enzymes, transporters, drug targets, or human leukocyte antigen (HLA) alleles and predicting drug efficacy or toxicity.5 CYP2C19*2 (rs4244285), CYP2C19*3 (rs4986893), and CYP2C19*17 (rs12248560) have been studied commonly. One study showed that CYP2C19*2 is the most common variant of the reduced function allele, accounting for more than 95% of the African whites and blacks, and more than 75% of the Asian population.6 Yi et al found that there was at least one allele with impaired CYP2C19 function, and the main prognostic risk was three times higher in clopidogrel carriers than in non-carriers,7 suggesting that the failure of clopidogrel antiplatelet drug therapy may be related to CYP2C19 gene mutation. These very important pharmacogenomics (VIP) genes have been summarized in the Pharmacogenomics Knowledge Base (PharmGKB; http://www.pharmgkb.org). A South Korean survey showed that pre-emptive genotyping can help many people avoid adverse drug reactions, suggesting that pharmacogenomics is promising.8 Exploring the VIP variants among different races is an acceptable way to find suitable drugs for patients or specific populations.

Lahu, distributed in 31 provinces, autonomous regions and municipalities directly under the Central Government in China, is an ancient ethnic group evolved from the Ancient Qiang people in Gansu and Qinghai provinces, whose population in total is 485,966. In the process of ethnic development, Lahu moved to the current Yunnan province and Southeast Asia, such as Myanmar and Thailand. In Yunnan, they mainly lived in Lancang and Shuangjiang counties near the border, with nearly 447,600 people, accounting for 98.66% of the total population of the Lahu ethnic group.9 The Lahu have not only their own unique genetic characteristics but also their own lifestyle patterns, particularly in terms of traditional practices related to the use of alcohol.10 However, the pharmacogenomic VIP variants in Lahu people are seldom reported. The study of drug genome in rare and population-specific mutation groups, such as the Lahu, is of great significance to the realization of individualized drug therapy and the development of new drugs. We hope our findings could conduce to the supplement of pharmacogenomic data and support the clinical application of personalized medication in the Lahu population.

In this study, the genotype frequencies of 81 VIP variants in the Lahu population and 11 major HapMap populations were compared and analyzed statistically. Finally, Fst pairwise comparisons and Bayesian clustering analysis were applied to analyze the Lahu population genetics.

### Materials and Methods

We randomly collected a sample of 100 unrelated Lahu healthy adults aged 25–55 years from the Department of Physical examination, Yunnan First People’s Hospital and drew blood samples. The participants, who must reach several detailed inclusion criteria, were considered to be eligible. What is more, all individuals were at least three generations of Lahu paternal ancestor without any known ancestry from other ethnicities. The exclusion criteria were as follows: with the presence of chronic cancer, contagious disease, drugs or alcohol addiction, with severe heart, liver, and kidney dysfunction, immune disorders, pregnancy, or lactation. We have obtained informed consent and blood samples from the volunteers according to the study protocol approved by the Ethics Committee of the Yunnan First People’s Hospital. The sample size and the proportion were determined through G*Power 3.1.9.2 software.11

The study was approved by the Ethics Committee of the Yunnan First People’s Hospital (YYLH054) and was performed in accordance with the Declaration of Helsinki. Written informed consent was obtained from all volunteers in the study.

In the current study, we screened the genetic variants related to VIP variants from the PharmGKB database (https://www.pharmgkb.org/) with the minor allele frequency (MAF) > 0.05, Hardy–Weinberg equilibrium (HWE) > 0.05, and call rate > 0.95. The loci that could not be designed were excluded. Ultimately, a total of 81 genetic variants located in 45 genes were selected. The fresh blood samples were stored at −80°C. According to the manufacturer’s standard procedure, genomic DNA was isolated using GoldMag-Mini Genomic DNA Purification Kit (GoldMag Ltd. Xi’an, China). The DNA concentration and purity were performed using Nanodrop 2000.

(Thermo Scientific, Waltham, MA, USA) and agarose electrophoresis. The MassARRAY Assay Design 3.0 software (San Diego, California, USA) was applied to design amplification primers for the selected variants.12 The Agena MassARRAY RS1000 (SanDiego, California, USA) was utilized to perform genotype following the manufacturer's protocol (San Diego, California, USA). Finally, we used Agena Typer 4.0 software for data management and analysis.3,13

Microsoft Excel and SPSS 19.0 statistical packages (SPSS, Chicago, IL) were applied to perform HWE and chi-square tests. HWE was assessed using chi-square test and p < 0.05 indicated the disequilibrium of HWE. All genotype frequencies of variants in the Lahu population and the other 11 populations from HapMap (http://hapmap.ncbi.nlm.nih.gov) were calculated and compared using the chi-square test. The other 11 people included the Han Chinese in Beijing, China (CHB); Gambian in Western Divisions, The Gambia (GWD); the Japanese population in Tokyo, Japan (JPT); British in England and Scotland (GBR); a northwestern European population (CEU); the Tuscan people of Italy (TSI); the Luhya people in Webuye, Kenya (LWK); African ancestry in the southwestern USA (ASW); Mexican Ancestry in Los Angeles, California (MXL); the Gujarati Indians in Houston, Texas, USA (GIH); Indian Telugu in the UK (ITU). All p values in this study were two-sided. Then, we reduced the false discovery rate of multiple testing by Bonferroni’s multiple comparison adjustment. When p values were less than 0.05/(81*11), it was considered to be statistically significant.

F-statistics (Fst) and structure analyses were usually adopted in population genetic studies. In this study, the Arlequin v3.5 program was used to calculate global Fst along with the pairwise Fst among all the populations using the loci, which were polymorphic at the 5% level.14 Therefore, we could estimate the pairwise distances between the populations. The diversity of population genetic structures was analyzed through Structure (version 2.3.4) software in 12 populations.15,16

### Subjects

We randomly collected a sample of 100 unrelated Lahu healthy adults aged 25–55 years from the Department of Physical examination, Yunnan First People’s Hospital and drew blood samples. The participants, who must reach several detailed inclusion criteria, were considered to be eligible. What is more, all individuals were at least three generations of Lahu paternal ancestor without any known ancestry from other ethnicities. The exclusion criteria were as follows: with the presence of chronic cancer, contagious disease, drugs or alcohol addiction, with severe heart, liver, and kidney dysfunction, immune disorders, pregnancy, or lactation. We have obtained informed consent and blood samples from the volunteers according to the study protocol approved by the Ethics Committee of the Yunnan First People’s Hospital. The sample size and the proportion were determined through G*Power 3.1.9.2 software.11

The study was approved by the Ethics Committee of the Yunnan First People’s Hospital (YYLH054) and was performed in accordance with the Declaration of Helsinki. Written informed consent was obtained from all volunteers in the study.

### Selection and Genotyping of VIP Variants

In the current study, we screened the genetic variants related to VIP variants from the PharmGKB database (https://www.pharmgkb.org/) with the minor allele frequency (MAF) > 0.05, Hardy–Weinberg equilibrium (HWE) > 0.05, and call rate > 0.95. The loci that could not be designed were excluded. Ultimately, a total of 81 genetic variants located in 45 genes were selected. The fresh blood samples were stored at −80°C. According to the manufacturer’s standard procedure, genomic DNA was isolated using GoldMag-Mini Genomic DNA Purification Kit (GoldMag Ltd. Xi’an, China). The DNA concentration and purity were performed using Nanodrop 2000.

(Thermo Scientific, Waltham, MA, USA) and agarose electrophoresis. The MassARRAY Assay Design 3.0 software (San Diego, California, USA) was applied to design amplification primers for the selected variants.12 The Agena MassARRAY RS1000 (SanDiego, California, USA) was utilized to perform genotype following the manufacturer's protocol (San Diego, California, USA). Finally, we used Agena Typer 4.0 software for data management and analysis.3,13

### Statistical Analyses

Microsoft Excel and SPSS 19.0 statistical packages (SPSS, Chicago, IL) were applied to perform HWE and chi-square tests. HWE was assessed using chi-square test and p < 0.05 indicated the disequilibrium of HWE. All genotype frequencies of variants in the Lahu population and the other 11 populations from HapMap (http://hapmap.ncbi.nlm.nih.gov) were calculated and compared using the chi-square test. The other 11 people included the Han Chinese in Beijing, China (CHB); Gambian in Western Divisions, The Gambia (GWD); the Japanese population in Tokyo, Japan (JPT); British in England and Scotland (GBR); a northwestern European population (CEU); the Tuscan people of Italy (TSI); the Luhya people in Webuye, Kenya (LWK); African ancestry in the southwestern USA (ASW); Mexican Ancestry in Los Angeles, California (MXL); the Gujarati Indians in Houston, Texas, USA (GIH); Indian Telugu in the UK (ITU). All p values in this study were two-sided. Then, we reduced the false discovery rate of multiple testing by Bonferroni’s multiple comparison adjustment. When p values were less than 0.05/(81*11), it was considered to be statistically significant.

F-statistics (Fst) and structure analyses were usually adopted in population genetic studies. In this study, the Arlequin v3.5 program was used to calculate global Fst along with the pairwise Fst among all the populations using the loci, which were polymorphic at the 5% level.14 Therefore, we could estimate the pairwise distances between the populations. The diversity of population genetic structures was analyzed through Structure (version 2.3.4) software in 12 populations.15,16

### Results

In this study, 81 genetic variants were selected for investigation in the Lahu population, which was based on previously published VIP variants from the PharmGKB database. The VIP variants were distributed in 45 genes. Basic characteristics of these selected variants in the Lahu population are listed in Table 1.Table 1Basic Characteristics Selected Variants in the LahuChromosomeGenePositionSNPFunctional ConsequenceAlleles (A/B)Freq.AFreq.B1MTHFR11794419rs1801131MissenseG/T0.200.801MTHFR11796321rs1801133MissenseA/G0.320.681CYP2J259926822rs890293Upstream variant 2KBA/C0.020.981DPYD97450058rs3918290Splice donor variantT/C0.001.001DPYD97515839rs1801159Intron variant, missenseC/T0.270.741DPYD97883329rs1801265Intron variant, missense, nc transcript variant, utr variant 5 primeG/A0.170.831F5169549811rs6025MissenseT/C0.001.001PTGS2186673926rs5275utr variant 3 primeG/A0.240.761PACERR186681189rs20417nc transcript variant, upstream variant 2KBG/C0.010.991PACERR186681619rs689466Downstream variant 500B, upstream variant 2KBC/T0.420.582LOC100286922233757013rs4124874Intron variant, upstream variant 2KBG/T0.390.622LOC100286922233757136rs10929302Intron variant, upstream variant 2KBA/G0.070.932UGT1A1233760498rs4148323Intron variant, missenseA/G0.200.803SCN5A38603929rs1805124MissenseC/T0.070.933SCN5A38633208rs6791924MissenseA/G0.001.003NR1I2119781188rs3814055Upstream variant 2KB, utr variant 5 primeT/C0.220.793MED12L151339854rs2046934Intron variantG/A0.230.773P2RY1152835839rs1065776Synonymous codonT/C0.030.973P2RY1152836568rs701265Synonymous codonG/A0.410.594ADH1A99280582rs975833Intron variantG/C0.280.724ADH1B99307860rs2066702MissenseA/G0.001.004ADH1C99339632rs698Missense, nc transcript variantC/T0.210.795HMGCR75347030rs17244841Intron variantA/T1.000.005HMGCR75355259rs3846662Intron variantG/A0.440.565ADRB2148826877rs1042713MissenseG/A0.360.645ADRB2148826910rs1042714MissenseG/C0.010.996TPMT18130687rs1142345MissenseT/C0.990.017AHR17339486rs2066853MissenseA/G0.260.747ABCB187509329rs1045642Synonymous codonA/G0.460.557ABCB187550285rs1128503Synonymous codonG/A0.360.647CYP3A499784473rs2740574Upstream variant 2KBC/T0.001.007KCNH2150970122rs3807375Intron variantC/T0.440.568NAT218390208rs4646244Intron variant, upstream variant 2KBA/T0.340.668NAT218390758rs4271002Intron variant, upstream variant 2KBC/G0.260.758NAT218400344rs1801280MissenseC/T0.090.928NAT218400484rs1799929Synonymous codonT/C0.090.928NAT218400806rs1208MissenseG/A0.090.928NAT218400860rs1799931MissenseA/G0.250.7510CYP2C1994761900rs12248560Upstream variant 2KBT/C0.001.0010CYP2C1994780653rs4986893Stop gainedA/G0.130.8710CYP2C1994781859rs4244285Synonymous codonA/G0.350.6510CYP2C994981296rs1057910MissenseC/A0.050.9610CYP2C895069673rs7909236Upstream variant 2KBT/G0.070.9310CYP2C895069772rs17110453Upstream variant 2KBC/A0.390.6210CYP2E1133537633rs2070676Intron variantG/C0.080.9211GSTP167585218rs1695MissenseG/A0.370.6311GSTP167586108rs1138272MissenseT/C0.001.0011ANKK1113400106rs1800497MissenseA/G0.440.5611DRD2113412737rs6277Synonymous codonA/G0.030.9711DRD2113412762rs1801028MissenseC/G0.020.9812SLCO1B121130388rs4149015Upstream variant 2KBA/G0.060.9412SLCO1B121176804rs2306283MissenseA/G0.220.7812SLCO1B121178615rs4149056MissenseC/T0.050.9512VDR47844974rs731236Synonymous codonG/A0.030.9712VDR47845054rs7975232Intron variantA/C0.290.7112VDR47846052rs1544410Intron variantT/C0.070.9312VDR47850776rs2239185Intron variantA/G0.290.7112VDR47863543rs1540339Intron variantC/T0.230.7712VDR47863983rs2239179Intron variantC/T0.210.7912VDR47872384rs3782905Intron variantC/G0.130.8712VDR47906043rs4516035Upstream variant 2KBC/T0.020.9812None47908762rs11568820NoneT/C0.400.6015CYP1A274749576rs762551Intron variantC/A0.280.7216SULT1A128609479rs3760091Intron variant, upstream variant 2KBG/C0.370.6316PRSS5331091000rs7294Upstream variant 2KB, utr variant 3 primeT/C0.170.8316VKORC131093557rs9934438Intron variantG/A0.180.8216NQO169711242rs1800566MissenseA/G0.380.6219CYP4F215879621rs2108622MissenseT/C0.130.8819CYP2A640848591rs8192726Intron variantA/C0.110.8919CYP2A640848628rs1801272MissenseT/A0.020.9819CYP2A640850474rs28399433Upstream variant 2KBC/A0.150.8619CYP2B641016810rs3211371Downstream variant 500B, missense, utr variant 3 primeT/C0.500.5020PTGIS49513169rs5629Stop gained, synonymous codonT/G0.220.7821SLC19A145514912rs1051298Intron variant, utr variant 3 primeA/G0.460.5421SLC19A145514947rs1051296Intron variant, utr variant 3 primeC/A0.460.5421SLC19A145537880rs1051266Missense, utr variant 5 primeT/C0.360.6421SLC19A145538002rs1131596Synonymous codon, utr variant 5 primeA/G0.620.3922COMT19963748rs4680Missense, upstream variant 2KBA/G0.300.7022CYP2D642127608rs59421388Missense, synonymous codon, upstream variant 2KBT/C0.001.0022CYP2D642127803rs28371725Intron variant, upstream variant 2KBT/C0.120.8822CYP2D642129132rs61736512Intron variant, missense, upstream variant 2KBT/C0.001.00

Basic Characteristics Selected Variants in the Lahu

Chi-square test was performed for significant difference assessment on genotype frequency distribution of 81 loci among Lahu people and the other 11 populations from HapMap project, which are demonstrated in Table 2. On the one hand, compared to the 11 groups (CHB, GWD, JPT, GBR, CEU, TSI, ASW, LWK, GIH, ITU, and MXL) without adjustment (p < 0.05), the number of significantly different variants in the Lahu was 32, 59, 32, 49, 52, 54, 51, 55, 51, 52, and 49, respectively. In these 81 different SNPs, the genotype frequencies distribution in thiopurine S-methyltransferase (TPMT) rs1142345 and vitamin K epoxide reductase complex subunit 1 (VKORC1) rs9934438 were found to be different in the Lahu population when compared to the other 11 ethnic groups. After adjustment using the Bonferroni correction (p < 0.05/(81*11)), there were 5, 49, 6, 38, 39, 40, 40, 46, 39, 34, and 22 loci of significant differences between Lahu and the other 11 populations, respectively. The significance of rs1142345 and rs9934438 still existed between Lahu and the other 11 populations. After adjustment, the results also exhibited that GWD was the most different population compared with Lahu, with the number of 49 distinct SNPs loci, followed by LWK with the number of 46 distinct SNPs loci. It was also noteworthy that the different loci between CHB, JPT and the Lahu were the least. However, according to the statistics, the frequencies of alcohol dehydrogenase 1C (ADH1C) rs698, glutathione S-transferase pi 1 (GSTP1) rs1695, cytochrome P450 family 2 subfamily A member 6 (CYP2A6) rs28399433 were distinct from that of CHB groups, respectively. On the other hand, in addition to the above loci, we also found that the genotype distribution of potassium voltage-gated channel subfamily H member 2 (KCNH2) rs3807375 was significantly different between Lahu and JPT.Table 2Different VIP Variants Loci in the Lahu Compared with the 11 Populations After Bonferroni’s Multiple AdjustmentGeneSNP-IDp < 0.05/(81*11)CHBGWDJPTGBRCEUTSIASWLWKGIHITUMXLMTHFRrs18011310.68490.00700.68490.00070.00160.00440.94600.60159.33E-081.87E-100.6546MTHFRrs18011330.00243.20E-120.00240.76840.80320.00164.61E-052.63E-105.49E-053.65E-080.0094CYP2J2rs8902930.25962.80E-080.25960.1756––2.12E-072.21E-07–0.0495–DPYDrs3918290–––––––––––DPYDrs18011590.63044.42E-070.63040.05190.00990.55080.03340.43943.16E-059.00E-090.4038DPYDrs18012650.00281.61E-130.00280.83940.82110.17652.81E-111.01E-142.15E-052.15E-040.0499F5rs6025––––0.0167––––––PTGS2rs52750.01683.43E-150.01680.29430.00030.18962.57E-103.47E-160.00060.00200.0308PACERRrs204170.00011.31E-256.07E-059.23E-125.75E-142.82E-158.95E-259.32E-235.91E-151.64E-131.83E-18PACERRrs6894660.14988.66E-180.14987.59E-061.40E-066.91E-073.80E-071.83E-191.32E-106.55E-100.0074LOC100286922rs41248740.00811.01E-360.00810.89230.51390.33971.13E-121.40E-271.45E-066.94E-060.0310LOC100286922rs10929302–1.17E-16–2.15E-071.76E-132.69E-081.34E-148.50E-184.06E-212.36E-185.16E-14UGT1A10rs41483230.21944.64E-120.21944.61E-108.53E-111.61E-111.35E-068.53E-119.94E-091.81E-099.78E-06SCN5Ars18051240.02253.64E-160.02257.78E-070.00017.13E-081.73E-073.48E-123.04E-061.10E-100.0069SCN5Ars6791924–2.00E-14–––––8.16E-15–––NR1I2rs38140550.31760.00180.31765.31E-050.00670.00080.08570.07872.93E-060.00060.0062MED12Lrs20469340.67250.00070.67250.77390.44790.00520.09340.00080.00073.62E-060.0123P2RY1rs1065776–1.32E-19––0.17840.37522.75E-102.73E-12–1.25E-08–P2RY1rs7012650.00795.36E-220.00792.13E-101.14E-081.79E-091.65E-066.28E-183.46E-062.76E-065.97959E-05ADH1Ars9758330.09822.93E-260.09821.81E-232.27E-247.13E-231.37E-188.90E-291.37E-070.00075.10E-29ADH1Brs2066702–1.52E-12––––7.85E-18––––ADH1Crs6987.26E-067.45E-047.26`E-061.44E-065.63E-100.02010.25700.05410.14550.21450.0509HMGCRrs17244841-8.63E-65-1.06E-631.95E-653.18E-663.48E-571.84E-64--7.77E-58HMGCRrs38466620.19627.93E-330.19620.43580.93550.93053.02E-163.60E-332.86E-060.01690.9571ADRB2rs10427130.02140.03850.02146.57E-061.11E-091.27E-080.22850.00382.20E-058.01E-050.0046ADRB2rs1042714-2.68E-08-6.90E-341.55E-364.70E-349.95E-083.31E-184.16E-196.14E-138.72E-09TPMTrs11423451.56E-657.13E-661.56E-652.80E-614.36E-631.55E-651.48E-532.25E-593.70E-648.09E-655.26E-55AHRrs20668530.00874.66E-060.00873.80E-057.27E-061.07E-050.13381.76E-077.17E-050.00510.0094ABCB1rs10456420.03506.93E-110.03500.19750.02610.43279.52E-081.40E-130.01470.00150.9044ABCB1rs11285030.17544.90E-270.17543.67E-063.55E-065.21E-074.82E-195.52E-310.20130.44340.0019CYP3A4rs2740574-4.21E-64----1.73E-516.09E-60---KCNH2rs3807375-1.73E-078.22E-064.58E-089.80E-061.73E-060.00223.08E-090.00042.07E-050.6611NAT2rs46462440.00090.00140.00090.01400.31810.33320.21720.05440.65790.32301.73E-05NAT2rs42710020.07231.41E-100.07230.05207.27E-070.04300.00448.58E-060.00347.72E-050.2957NAT2rs18012800.03001.01E-120.03001.45E-201.60E-181.71E-197.06E-098.92E-151.00E-124.60E-134.70E-13NAT2rs17999290.03001.47E-090.03004.22E-193.32E-181.71E-192.47E-062.78E-122.96E-106.34E-111.58E-12NAT2rs12080.03001.69E-200.03002.70E-195.00E-175.32E-201.22E-119.64E-211.00E-121.15E-136.61E-17NAT2rs17999310.01626.78E-130.01622.03E-103.15E-133.14E-128.10E-061.44E-123.55E-085.94E-080.0279CYP2C19rs12248560-1.18E-21-2.79E-211.15E-201.33E-191.48E-171.46E-157.28E-123.94E-125.71E-09CYP2C19rs49868930.00955.23E-070.00957.56E-062.85E-061.08E-060.00033.76E-056.81E-065.37E-060.0002CYP2C19rs42442850.41052.05E-080.41051.90E-061.03E-073.73E-113.73E-050.00220.20010.56524.58E-06CYP2C9rs1057910-----0.1068--0.00050.0248-CYP2C8rs79092360.2181-0.21811.33E-064.93E-100.0001--3.11E-081.40E-061.92E-10CYP2C8rs171104530.52926.02E-270.52921.26E-111.53E-134.64E-132.85E-161.86E-240.08060.37229.57E-09CYP2E1rs20706760.00012.37E-400.00010.79210.01150.00034.38E-251.27E-430.05150.00150.0306GSTP1rs16951.26E-050.00031.26E-050.32470.48960.12960.24290.00360.29950.44420.0006GSTP1rs1138272----------6.82029E-05ANKK1rs18004970.93330.05130.93333.39E-067.02E-072.79E-060.80650.08820.00220.31970.1703DRD2rs6277-0.1737-1.60E-359.18E-378.88E-452.99E-05-1.64E-202.85E-223.16E-15DRD2rs18010280.0983-0.0983-0.2920---2.79E-10--SLCO1B1rs41490150.03850.00060.03850.11900.50770.43030.05500.24970.77210.51530.2353SLCO1B1rs23062830.96040.45820.96044.64E-191.67E-161.74E-190.69670.09131.37E-072.33E-053.91E-15SLCO1B1rs41490560.00070.00450.00070.00027.39E-051.37E-090.63180.28000.24690.75850.2271VDRrs731236-5.09E-16-1.57E-192.31E-301.56E-282.07E-131.29E-166.98E-213.17E-337.10E-10VDRrs79752320.92522.15E-150.92520.00069.00E-111.24E-103.82E-109.57E-191.33E-072.64E-130.0359VDRrs1544410-3.12E-07-4.45E-146.08E-231.28E-207.10E-093.65E-094.12E-231.50E-300.0002VDRrs22391850.91226.08E-110.91220.00041.86E-116.13E-112.07E-065.33E-141.15E-076.02E-150.0270VDRrs15403390.48231.53E-300.48233.83E-141.09E-206.51E-198.80E-214.25E-361.05E-183.70E-244.60E-13VDRrs22391790.52870.28340.52870.00543.96E-111.41E-060.00080.00102.76E-115.76E-180.1391VDRrs37829050.74740.11330.74740.00371.81E-091.18E-080.03330.00620.00041.17E-080.0465VDRrs4516035---1.22E-304.01E-275.64E-332.11E-05-9.81E-135.38E-131.56E-16Noners115688200.00423.79E-400.00422.29E-065.98E-075.81E-062.94E-121.81E-290.02389.27E-051.24E-09CYP1A2rs7625510.11400.00350.11400.97170.90080.04420.13931.93E-072.43E-050.00020.4248SULT1A1rs37600910.23850.98780.23850.76670.49100.00160.72260.76520.00020.28690.0625PRSS53rs72940.00014.67E-120.00019.47E-100.00031.39E-053.89E-101.13E-104.36E-291.25E-370.0001VKORC1rs99344383.29463E-052.30E-513.29E-058.52E-245.84E-189.24E-161.86E-343.66E-538.27E-391.52E-455.57E-13NQO1rs18005660.00333.89E-070.00330.00018.48E-060.00400.00012.42E-060.17760.86360.8644CYP4F2rs21086220.00400.06500.00400.00010.00161.11E-070.37790.55581.83E-142.40E-120.0003CYP2A6rs81927260.03960.20280.03960.04810.08280.17090.91640.66840.34130.91890.1496CYP2A6rs1801272---0.00160.00032.46E-050.1054-0.0512-0.0231CYP2A6rs283994331.52E-05-1.52E-054.25E-05--0.05080.02160.00260.0014-CYP2B6rs3211371---4.95E-466.39E-507.46E-49---1.06E-50-PTGISrs56290.18232.46E-050.18230.39700.93280.01320.28000.00050.40160.96870.1124SLC19A1rs10512980.52804.10E-050.52800.18000.34960.85070.19780.11800.00790.19410.0341SLC19A1rs10512960.30140.65670.30140.17310.36260.77410.83440.10130.27980.99250.0330SLC19A1rs10512660.02275.65E-180.02270.06470.13700.08690.00031.06E-120.75230.57570.1027SLC19A1rs11315960.10403.87E-170.10400.45460.05510.32514.00E-058.55E-150.55420.63450.3417COMTrs46800.32710.12180.32712.22E-078.28E-050.00030.11280.94260.00040.00020.0805CYP2D6rs59421388------0.00138.16E-15---CYP2D6rs283717250.00860.00000.00860.28530.42830.02140.00900.00630.62760.45180.0066CYP2D6rs61736512------0.00138.16E-15---Different SNPs54963839404046393422Note: Bold italics indicates that after adjustment p < 0.05/(81*11) the locus has statistically significant.Abbreviations: ASW, African ancestry in southwestern USA; CEU, Utah residents with Northern and Western European ancestry; CHB, Han Chinese in Beijing, China; GIH, Gujarati Indians in Houston, Texas, USA; JPT, Japanese in Tokyo, Japan; LWK, Luhya people in Webuye, Kenya; TSI, Toscans in Italy; GWD, Gambian in Western Divisions, The Gambia; GBR, British in England and Scotland; ITU, Indian Telugu in the UK; MXL, Mexican Ancestry in Los Angeles, Colombia.

Different VIP Variants Loci in the Lahu Compared with the 11 Populations After Bonferroni’s Multiple Adjustment

Note: Bold italics indicates that after adjustment p < 0.05/(81*11) the locus has statistically significant.

Abbreviations: ASW, African ancestry in southwestern USA; CEU, Utah residents with Northern and Western European ancestry; CHB, Han Chinese in Beijing, China; GIH, Gujarati Indians in Houston, Texas, USA; JPT, Japanese in Tokyo, Japan; LWK, Luhya people in Webuye, Kenya; TSI, Toscans in Italy; GWD, Gambian in Western Divisions, The Gambia; GBR, British in England and Scotland; ITU, Indian Telugu in the UK; MXL, Mexican Ancestry in Los Angeles, Colombia.

Then, we performed linkage disequilibrium (LD) analysis using Haploview to define blocks and haplotypes. In the vitamin D receptor (VDR) gene, we found LD blocks in Lahu, CHB, JPT, GBR, CEU, TSI, GIH, ITU, and MXL, and however, there was no strong linkage between GWD, ASW, and LWK (Figure 1). Haplotype constitutions and frequencies showed that Lahu was differed from the other 11 populations. These findings, which are in accordance with the results, are shown in Table 2.Figure 1LD analysis of the VDR in each of the twelve populations. LD is displayed by standard color schemes with bright red for very strong LD (LOD>2, D′ =1), pink red (LOD> 2, D′ <1), blue (LOD< 2, D′ =1) for intermediate LD, and white (LOD<2, D′ <1) for no LD.

LD analysis of the VDR in each of the twelve populations. LD is displayed by standard color schemes with bright red for very strong LD (LOD>2, D′ =1), pink red (LOD> 2, D′ <1), blue (LOD< 2, D′ =1) for intermediate LD, and white (LOD<2, D′ <1) for no LD.

The Fst values were calculated with the help of Arlequin 3.5 to demonstrate the pairwise difference. With a detailed and comprehensive estimate and assessment for different population pairs, we figured out the magnitude of the differentiation among all the 12 geographic populations (0 means no divergence, and 1 indicates complete separation). As shown in Table 3, pairwise Fst values between Lahu and the other 11 HapMap groups measured the genetic divergence based on the genetic polymorphism data, which were variously ranged from 0.02782 to 0.23350. When Fst value is less than 0.15, there is no genetic differentiation between the two populations. Compared to other populations, the results showed that the lowest level (Fst = 0.02782) existed between the Lahu and CHB populations, followed by the JPT (Fst = 0.03449) and GIH (Fst = 0.114). The LWK population showed the greatest divergence (Fst = 0.2335).Table 3Estimates of Pairwise Fst Among the 12 PopulationLahuCHBJPTGIHITUCEUGBRTSIASWGWDLWKMXLLahu0CHB0.02780JPT0.03450.00450GIH0.11400.12770.11550ITU0.13500.15110.13830.00450CEU0.13540.14910.13960.03840.04430GBR0.12390.14040.13300.03710.04840.00400TSI0.12700.13330.12540.04050.04770.00400.00470ASW0.16680.17840.16390.08790.10010.11500.11910.11180GWD0.22630.23880.22130.14690.16020.18250.18820.17900.01180LWK0.23350.24300.22640.14400.15650.17720.18600.17410.01340.00550MXL0.09390.09160.10790.10720.09970.05150.04900.06360.03990.06720.03040

Estimates of Pairwise Fst Among the 12 Population

The Bayesian-based structure analysis showed us complementary methods for patterns of genetic similarity and differentiation of the total 12 populations, which works well for 81 loci in the current study. The most suitable K value was observed at 3. The proportion of each ancestor in a single individual was represented with a vertical bar, which was divided into three colors. In Figure 2, the BAR diagram showed that individuals sampled in Lahu were close to the clustering of people with CHB people.Figure 2Results of STRUCTURE analyses (K=3) among 12 populations. Most suitable K value is 3.

Results of STRUCTURE analyses (K=3) among 12 populations. Most suitable K value is 3.

### Identification of VIP Variants

In this study, 81 genetic variants were selected for investigation in the Lahu population, which was based on previously published VIP variants from the PharmGKB database. The VIP variants were distributed in 45 genes. Basic characteristics of these selected variants in the Lahu population are listed in Table 1.Table 1Basic Characteristics Selected Variants in the LahuChromosomeGenePositionSNPFunctional ConsequenceAlleles (A/B)Freq.AFreq.B1MTHFR11794419rs1801131MissenseG/T0.200.801MTHFR11796321rs1801133MissenseA/G0.320.681CYP2J259926822rs890293Upstream variant 2KBA/C0.020.981DPYD97450058rs3918290Splice donor variantT/C0.001.001DPYD97515839rs1801159Intron variant, missenseC/T0.270.741DPYD97883329rs1801265Intron variant, missense, nc transcript variant, utr variant 5 primeG/A0.170.831F5169549811rs6025MissenseT/C0.001.001PTGS2186673926rs5275utr variant 3 primeG/A0.240.761PACERR186681189rs20417nc transcript variant, upstream variant 2KBG/C0.010.991PACERR186681619rs689466Downstream variant 500B, upstream variant 2KBC/T0.420.582LOC100286922233757013rs4124874Intron variant, upstream variant 2KBG/T0.390.622LOC100286922233757136rs10929302Intron variant, upstream variant 2KBA/G0.070.932UGT1A1233760498rs4148323Intron variant, missenseA/G0.200.803SCN5A38603929rs1805124MissenseC/T0.070.933SCN5A38633208rs6791924MissenseA/G0.001.003NR1I2119781188rs3814055Upstream variant 2KB, utr variant 5 primeT/C0.220.793MED12L151339854rs2046934Intron variantG/A0.230.773P2RY1152835839rs1065776Synonymous codonT/C0.030.973P2RY1152836568rs701265Synonymous codonG/A0.410.594ADH1A99280582rs975833Intron variantG/C0.280.724ADH1B99307860rs2066702MissenseA/G0.001.004ADH1C99339632rs698Missense, nc transcript variantC/T0.210.795HMGCR75347030rs17244841Intron variantA/T1.000.005HMGCR75355259rs3846662Intron variantG/A0.440.565ADRB2148826877rs1042713MissenseG/A0.360.645ADRB2148826910rs1042714MissenseG/C0.010.996TPMT18130687rs1142345MissenseT/C0.990.017AHR17339486rs2066853MissenseA/G0.260.747ABCB187509329rs1045642Synonymous codonA/G0.460.557ABCB187550285rs1128503Synonymous codonG/A0.360.647CYP3A499784473rs2740574Upstream variant 2KBC/T0.001.007KCNH2150970122rs3807375Intron variantC/T0.440.568NAT218390208rs4646244Intron variant, upstream variant 2KBA/T0.340.668NAT218390758rs4271002Intron variant, upstream variant 2KBC/G0.260.758NAT218400344rs1801280MissenseC/T0.090.928NAT218400484rs1799929Synonymous codonT/C0.090.928NAT218400806rs1208MissenseG/A0.090.928NAT218400860rs1799931MissenseA/G0.250.7510CYP2C1994761900rs12248560Upstream variant 2KBT/C0.001.0010CYP2C1994780653rs4986893Stop gainedA/G0.130.8710CYP2C1994781859rs4244285Synonymous codonA/G0.350.6510CYP2C994981296rs1057910MissenseC/A0.050.9610CYP2C895069673rs7909236Upstream variant 2KBT/G0.070.9310CYP2C895069772rs17110453Upstream variant 2KBC/A0.390.6210CYP2E1133537633rs2070676Intron variantG/C0.080.9211GSTP167585218rs1695MissenseG/A0.370.6311GSTP167586108rs1138272MissenseT/C0.001.0011ANKK1113400106rs1800497MissenseA/G0.440.5611DRD2113412737rs6277Synonymous codonA/G0.030.9711DRD2113412762rs1801028MissenseC/G0.020.9812SLCO1B121130388rs4149015Upstream variant 2KBA/G0.060.9412SLCO1B121176804rs2306283MissenseA/G0.220.7812SLCO1B121178615rs4149056MissenseC/T0.050.9512VDR47844974rs731236Synonymous codonG/A0.030.9712VDR47845054rs7975232Intron variantA/C0.290.7112VDR47846052rs1544410Intron variantT/C0.070.9312VDR47850776rs2239185Intron variantA/G0.290.7112VDR47863543rs1540339Intron variantC/T0.230.7712VDR47863983rs2239179Intron variantC/T0.210.7912VDR47872384rs3782905Intron variantC/G0.130.8712VDR47906043rs4516035Upstream variant 2KBC/T0.020.9812None47908762rs11568820NoneT/C0.400.6015CYP1A274749576rs762551Intron variantC/A0.280.7216SULT1A128609479rs3760091Intron variant, upstream variant 2KBG/C0.370.6316PRSS5331091000rs7294Upstream variant 2KB, utr variant 3 primeT/C0.170.8316VKORC131093557rs9934438Intron variantG/A0.180.8216NQO169711242rs1800566MissenseA/G0.380.6219CYP4F215879621rs2108622MissenseT/C0.130.8819CYP2A640848591rs8192726Intron variantA/C0.110.8919CYP2A640848628rs1801272MissenseT/A0.020.9819CYP2A640850474rs28399433Upstream variant 2KBC/A0.150.8619CYP2B641016810rs3211371Downstream variant 500B, missense, utr variant 3 primeT/C0.500.5020PTGIS49513169rs5629Stop gained, synonymous codonT/G0.220.7821SLC19A145514912rs1051298Intron variant, utr variant 3 primeA/G0.460.5421SLC19A145514947rs1051296Intron variant, utr variant 3 primeC/A0.460.5421SLC19A145537880rs1051266Missense, utr variant 5 primeT/C0.360.6421SLC19A145538002rs1131596Synonymous codon, utr variant 5 primeA/G0.620.3922COMT19963748rs4680Missense, upstream variant 2KBA/G0.300.7022CYP2D642127608rs59421388Missense, synonymous codon, upstream variant 2KBT/C0.001.0022CYP2D642127803rs28371725Intron variant, upstream variant 2KBT/C0.120.8822CYP2D642129132rs61736512Intron variant, missense, upstream variant 2KBT/C0.001.00

Basic Characteristics Selected Variants in the Lahu

### Statistical Analyses

Chi-square test was performed for significant difference assessment on genotype frequency distribution of 81 loci among Lahu people and the other 11 populations from HapMap project, which are demonstrated in Table 2. On the one hand, compared to the 11 groups (CHB, GWD, JPT, GBR, CEU, TSI, ASW, LWK, GIH, ITU, and MXL) without adjustment (p < 0.05), the number of significantly different variants in the Lahu was 32, 59, 32, 49, 52, 54, 51, 55, 51, 52, and 49, respectively. In these 81 different SNPs, the genotype frequencies distribution in thiopurine S-methyltransferase (TPMT) rs1142345 and vitamin K epoxide reductase complex subunit 1 (VKORC1) rs9934438 were found to be different in the Lahu population when compared to the other 11 ethnic groups. After adjustment using the Bonferroni correction (p < 0.05/(81*11)), there were 5, 49, 6, 38, 39, 40, 40, 46, 39, 34, and 22 loci of significant differences between Lahu and the other 11 populations, respectively. The significance of rs1142345 and rs9934438 still existed between Lahu and the other 11 populations. After adjustment, the results also exhibited that GWD was the most different population compared with Lahu, with the number of 49 distinct SNPs loci, followed by LWK with the number of 46 distinct SNPs loci. It was also noteworthy that the different loci between CHB, JPT and the Lahu were the least. However, according to the statistics, the frequencies of alcohol dehydrogenase 1C (ADH1C) rs698, glutathione S-transferase pi 1 (GSTP1) rs1695, cytochrome P450 family 2 subfamily A member 6 (CYP2A6) rs28399433 were distinct from that of CHB groups, respectively. On the other hand, in addition to the above loci, we also found that the genotype distribution of potassium voltage-gated channel subfamily H member 2 (KCNH2) rs3807375 was significantly different between Lahu and JPT.Table 2Different VIP Variants Loci in the Lahu Compared with the 11 Populations After Bonferroni’s Multiple AdjustmentGeneSNP-IDp < 0.05/(81*11)CHBGWDJPTGBRCEUTSIASWLWKGIHITUMXLMTHFRrs18011310.68490.00700.68490.00070.00160.00440.94600.60159.33E-081.87E-100.6546MTHFRrs18011330.00243.20E-120.00240.76840.80320.00164.61E-052.63E-105.49E-053.65E-080.0094CYP2J2rs8902930.25962.80E-080.25960.1756––2.12E-072.21E-07–0.0495–DPYDrs3918290–––––––––––DPYDrs18011590.63044.42E-070.63040.05190.00990.55080.03340.43943.16E-059.00E-090.4038DPYDrs18012650.00281.61E-130.00280.83940.82110.17652.81E-111.01E-142.15E-052.15E-040.0499F5rs6025––––0.0167––––––PTGS2rs52750.01683.43E-150.01680.29430.00030.18962.57E-103.47E-160.00060.00200.0308PACERRrs204170.00011.31E-256.07E-059.23E-125.75E-142.82E-158.95E-259.32E-235.91E-151.64E-131.83E-18PACERRrs6894660.14988.66E-180.14987.59E-061.40E-066.91E-073.80E-071.83E-191.32E-106.55E-100.0074LOC100286922rs41248740.00811.01E-360.00810.89230.51390.33971.13E-121.40E-271.45E-066.94E-060.0310LOC100286922rs10929302–1.17E-16–2.15E-071.76E-132.69E-081.34E-148.50E-184.06E-212.36E-185.16E-14UGT1A10rs41483230.21944.64E-120.21944.61E-108.53E-111.61E-111.35E-068.53E-119.94E-091.81E-099.78E-06SCN5Ars18051240.02253.64E-160.02257.78E-070.00017.13E-081.73E-073.48E-123.04E-061.10E-100.0069SCN5Ars6791924–2.00E-14–––––8.16E-15–––NR1I2rs38140550.31760.00180.31765.31E-050.00670.00080.08570.07872.93E-060.00060.0062MED12Lrs20469340.67250.00070.67250.77390.44790.00520.09340.00080.00073.62E-060.0123P2RY1rs1065776–1.32E-19––0.17840.37522.75E-102.73E-12–1.25E-08–P2RY1rs7012650.00795.36E-220.00792.13E-101.14E-081.79E-091.65E-066.28E-183.46E-062.76E-065.97959E-05ADH1Ars9758330.09822.93E-260.09821.81E-232.27E-247.13E-231.37E-188.90E-291.37E-070.00075.10E-29ADH1Brs2066702–1.52E-12––––7.85E-18––––ADH1Crs6987.26E-067.45E-047.26`E-061.44E-065.63E-100.02010.25700.05410.14550.21450.0509HMGCRrs17244841-8.63E-65-1.06E-631.95E-653.18E-663.48E-571.84E-64--7.77E-58HMGCRrs38466620.19627.93E-330.19620.43580.93550.93053.02E-163.60E-332.86E-060.01690.9571ADRB2rs10427130.02140.03850.02146.57E-061.11E-091.27E-080.22850.00382.20E-058.01E-050.0046ADRB2rs1042714-2.68E-08-6.90E-341.55E-364.70E-349.95E-083.31E-184.16E-196.14E-138.72E-09TPMTrs11423451.56E-657.13E-661.56E-652.80E-614.36E-631.55E-651.48E-532.25E-593.70E-648.09E-655.26E-55AHRrs20668530.00874.66E-060.00873.80E-057.27E-061.07E-050.13381.76E-077.17E-050.00510.0094ABCB1rs10456420.03506.93E-110.03500.19750.02610.43279.52E-081.40E-130.01470.00150.9044ABCB1rs11285030.17544.90E-270.17543.67E-063.55E-065.21E-074.82E-195.52E-310.20130.44340.0019CYP3A4rs2740574-4.21E-64----1.73E-516.09E-60---KCNH2rs3807375-1.73E-078.22E-064.58E-089.80E-061.73E-060.00223.08E-090.00042.07E-050.6611NAT2rs46462440.00090.00140.00090.01400.31810.33320.21720.05440.65790.32301.73E-05NAT2rs42710020.07231.41E-100.07230.05207.27E-070.04300.00448.58E-060.00347.72E-050.2957NAT2rs18012800.03001.01E-120.03001.45E-201.60E-181.71E-197.06E-098.92E-151.00E-124.60E-134.70E-13NAT2rs17999290.03001.47E-090.03004.22E-193.32E-181.71E-192.47E-062.78E-122.96E-106.34E-111.58E-12NAT2rs12080.03001.69E-200.03002.70E-195.00E-175.32E-201.22E-119.64E-211.00E-121.15E-136.61E-17NAT2rs17999310.01626.78E-130.01622.03E-103.15E-133.14E-128.10E-061.44E-123.55E-085.94E-080.0279CYP2C19rs12248560-1.18E-21-2.79E-211.15E-201.33E-191.48E-171.46E-157.28E-123.94E-125.71E-09CYP2C19rs49868930.00955.23E-070.00957.56E-062.85E-061.08E-060.00033.76E-056.81E-065.37E-060.0002CYP2C19rs42442850.41052.05E-080.41051.90E-061.03E-073.73E-113.73E-050.00220.20010.56524.58E-06CYP2C9rs1057910-----0.1068--0.00050.0248-CYP2C8rs79092360.2181-0.21811.33E-064.93E-100.0001--3.11E-081.40E-061.92E-10CYP2C8rs171104530.52926.02E-270.52921.26E-111.53E-134.64E-132.85E-161.86E-240.08060.37229.57E-09CYP2E1rs20706760.00012.37E-400.00010.79210.01150.00034.38E-251.27E-430.05150.00150.0306GSTP1rs16951.26E-050.00031.26E-050.32470.48960.12960.24290.00360.29950.44420.0006GSTP1rs1138272----------6.82029E-05ANKK1rs18004970.93330.05130.93333.39E-067.02E-072.79E-060.80650.08820.00220.31970.1703DRD2rs6277-0.1737-1.60E-359.18E-378.88E-452.99E-05-1.64E-202.85E-223.16E-15DRD2rs18010280.0983-0.0983-0.2920---2.79E-10--SLCO1B1rs41490150.03850.00060.03850.11900.50770.43030.05500.24970.77210.51530.2353SLCO1B1rs23062830.96040.45820.96044.64E-191.67E-161.74E-190.69670.09131.37E-072.33E-053.91E-15SLCO1B1rs41490560.00070.00450.00070.00027.39E-051.37E-090.63180.28000.24690.75850.2271VDRrs731236-5.09E-16-1.57E-192.31E-301.56E-282.07E-131.29E-166.98E-213.17E-337.10E-10VDRrs79752320.92522.15E-150.92520.00069.00E-111.24E-103.82E-109.57E-191.33E-072.64E-130.0359VDRrs1544410-3.12E-07-4.45E-146.08E-231.28E-207.10E-093.65E-094.12E-231.50E-300.0002VDRrs22391850.91226.08E-110.91220.00041.86E-116.13E-112.07E-065.33E-141.15E-076.02E-150.0270VDRrs15403390.48231.53E-300.48233.83E-141.09E-206.51E-198.80E-214.25E-361.05E-183.70E-244.60E-13VDRrs22391790.52870.28340.52870.00543.96E-111.41E-060.00080.00102.76E-115.76E-180.1391VDRrs37829050.74740.11330.74740.00371.81E-091.18E-080.03330.00620.00041.17E-080.0465VDRrs4516035---1.22E-304.01E-275.64E-332.11E-05-9.81E-135.38E-131.56E-16Noners115688200.00423.79E-400.00422.29E-065.98E-075.81E-062.94E-121.81E-290.02389.27E-051.24E-09CYP1A2rs7625510.11400.00350.11400.97170.90080.04420.13931.93E-072.43E-050.00020.4248SULT1A1rs37600910.23850.98780.23850.76670.49100.00160.72260.76520.00020.28690.0625PRSS53rs72940.00014.67E-120.00019.47E-100.00031.39E-053.89E-101.13E-104.36E-291.25E-370.0001VKORC1rs99344383.29463E-052.30E-513.29E-058.52E-245.84E-189.24E-161.86E-343.66E-538.27E-391.52E-455.57E-13NQO1rs18005660.00333.89E-070.00330.00018.48E-060.00400.00012.42E-060.17760.86360.8644CYP4F2rs21086220.00400.06500.00400.00010.00161.11E-070.37790.55581.83E-142.40E-120.0003CYP2A6rs81927260.03960.20280.03960.04810.08280.17090.91640.66840.34130.91890.1496CYP2A6rs1801272---0.00160.00032.46E-050.1054-0.0512-0.0231CYP2A6rs283994331.52E-05-1.52E-054.25E-05--0.05080.02160.00260.0014-CYP2B6rs3211371---4.95E-466.39E-507.46E-49---1.06E-50-PTGISrs56290.18232.46E-050.18230.39700.93280.01320.28000.00050.40160.96870.1124SLC19A1rs10512980.52804.10E-050.52800.18000.34960.85070.19780.11800.00790.19410.0341SLC19A1rs10512960.30140.65670.30140.17310.36260.77410.83440.10130.27980.99250.0330SLC19A1rs10512660.02275.65E-180.02270.06470.13700.08690.00031.06E-120.75230.57570.1027SLC19A1rs11315960.10403.87E-170.10400.45460.05510.32514.00E-058.55E-150.55420.63450.3417COMTrs46800.32710.12180.32712.22E-078.28E-050.00030.11280.94260.00040.00020.0805CYP2D6rs59421388------0.00138.16E-15---CYP2D6rs283717250.00860.00000.00860.28530.42830.02140.00900.00630.62760.45180.0066CYP2D6rs61736512------0.00138.16E-15---Different SNPs54963839404046393422Note: Bold italics indicates that after adjustment p < 0.05/(81*11) the locus has statistically significant.Abbreviations: ASW, African ancestry in southwestern USA; CEU, Utah residents with Northern and Western European ancestry; CHB, Han Chinese in Beijing, China; GIH, Gujarati Indians in Houston, Texas, USA; JPT, Japanese in Tokyo, Japan; LWK, Luhya people in Webuye, Kenya; TSI, Toscans in Italy; GWD, Gambian in Western Divisions, The Gambia; GBR, British in England and Scotland; ITU, Indian Telugu in the UK; MXL, Mexican Ancestry in Los Angeles, Colombia.

Different VIP Variants Loci in the Lahu Compared with the 11 Populations After Bonferroni’s Multiple Adjustment

Note: Bold italics indicates that after adjustment p < 0.05/(81*11) the locus has statistically significant.

Abbreviations: ASW, African ancestry in southwestern USA; CEU, Utah residents with Northern and Western European ancestry; CHB, Han Chinese in Beijing, China; GIH, Gujarati Indians in Houston, Texas, USA; JPT, Japanese in Tokyo, Japan; LWK, Luhya people in Webuye, Kenya; TSI, Toscans in Italy; GWD, Gambian in Western Divisions, The Gambia; GBR, British in England and Scotland; ITU, Indian Telugu in the UK; MXL, Mexican Ancestry in Los Angeles, Colombia.

Then, we performed linkage disequilibrium (LD) analysis using Haploview to define blocks and haplotypes. In the vitamin D receptor (VDR) gene, we found LD blocks in Lahu, CHB, JPT, GBR, CEU, TSI, GIH, ITU, and MXL, and however, there was no strong linkage between GWD, ASW, and LWK (Figure 1). Haplotype constitutions and frequencies showed that Lahu was differed from the other 11 populations. These findings, which are in accordance with the results, are shown in Table 2.Figure 1LD analysis of the VDR in each of the twelve populations. LD is displayed by standard color schemes with bright red for very strong LD (LOD>2, D′ =1), pink red (LOD> 2, D′ <1), blue (LOD< 2, D′ =1) for intermediate LD, and white (LOD<2, D′ <1) for no LD.

LD analysis of the VDR in each of the twelve populations. LD is displayed by standard color schemes with bright red for very strong LD (LOD>2, D′ =1), pink red (LOD> 2, D′ <1), blue (LOD< 2, D′ =1) for intermediate LD, and white (LOD<2, D′ <1) for no LD.

### Analyses of Genetic Background

The Fst values were calculated with the help of Arlequin 3.5 to demonstrate the pairwise difference. With a detailed and comprehensive estimate and assessment for different population pairs, we figured out the magnitude of the differentiation among all the 12 geographic populations (0 means no divergence, and 1 indicates complete separation). As shown in Table 3, pairwise Fst values between Lahu and the other 11 HapMap groups measured the genetic divergence based on the genetic polymorphism data, which were variously ranged from 0.02782 to 0.23350. When Fst value is less than 0.15, there is no genetic differentiation between the two populations. Compared to other populations, the results showed that the lowest level (Fst = 0.02782) existed between the Lahu and CHB populations, followed by the JPT (Fst = 0.03449) and GIH (Fst = 0.114). The LWK population showed the greatest divergence (Fst = 0.2335).Table 3Estimates of Pairwise Fst Among the 12 PopulationLahuCHBJPTGIHITUCEUGBRTSIASWGWDLWKMXLLahu0CHB0.02780JPT0.03450.00450GIH0.11400.12770.11550ITU0.13500.15110.13830.00450CEU0.13540.14910.13960.03840.04430GBR0.12390.14040.13300.03710.04840.00400TSI0.12700.13330.12540.04050.04770.00400.00470ASW0.16680.17840.16390.08790.10010.11500.11910.11180GWD0.22630.23880.22130.14690.16020.18250.18820.17900.01180LWK0.23350.24300.22640.14400.15650.17720.18600.17410.01340.00550MXL0.09390.09160.10790.10720.09970.05150.04900.06360.03990.06720.03040

Estimates of Pairwise Fst Among the 12 Population

The Bayesian-based structure analysis showed us complementary methods for patterns of genetic similarity and differentiation of the total 12 populations, which works well for 81 loci in the current study. The most suitable K value was observed at 3. The proportion of each ancestor in a single individual was represented with a vertical bar, which was divided into three colors. In Figure 2, the BAR diagram showed that individuals sampled in Lahu were close to the clustering of people with CHB people.Figure 2Results of STRUCTURE analyses (K=3) among 12 populations. Most suitable K value is 3.

Results of STRUCTURE analyses (K=3) among 12 populations. Most suitable K value is 3.

### Discussion

There is increasing interest in pharmacogenomics because of genetic variations leading to each person’s different metabolism of and reactions to some drugs. As is known to all, race is an important factor leading to large differences in drug metabolism, treatment response, and toxicity among individuals.17 In our results, we genotyped the pharmacogenomic VIP variants in the Lahu population and determined the differences between Lahu and the other 11 populations. We found that 32, 59, 32, 49, 52, 54, 51, 55, 51, 52, and 49 of the selected variants in the Lahu population significantly differed from those of CHB, GWD, JPT, GBR, CEU, TSI, ASW, LWK, GIH, ITU, and MXL, respectively. These results suggest that the Lahu ethnic group has genetic heterogeneity that distinguishes it from other ethnic groups. Interestingly, the difference of loci genotype frequencies between CHB, JPT and Lahu was the least. Additionally, the pairwise Fst values and genetic structure also revealed that the variants in the Lahu were mostly similar to the JPT and CHB populations genetically.

Nonetheless, we found that compared to the other 11 populations, TPMT rs1142345 was significantly different in Lahu people. Pharmacogenomics studies have shown that genetic polymorphisms in TPMT are variable and that TPMT activity is regulated by genetic polymorphisms, which is also the cause of adverse drug reactions.18 The TPMT genotype has been considered as an indicator of the initial dose of thiopurine drugs,19 and race-specific differences in TPMT activity and mercaptopurine metabolism have been observed.20 African ancestry is associated with the lower TPMT activity, and some studies have reported a higher prevalence of TPMT variants in blacks.21,22 The CC genotype carrying the TPMT*3C (c.719 T>C, rs1142345) variant is susceptible to the toxicity of the standard dose of 6-mercaptopurine. A high-frequency CC genotype of the TPMT*3C variant was found in traditional indigenous people in the Amazon region.23 Compared with the other 11 populations, TPMT rs1142345 variants in the Lahu population are statistically different. The C allele is associated with mercaptopurine exposure in children with leukemia when compared with the T allele. The relationship between the polymorphism of rs1142345 and the risk of acute lymphoblastic leukemia has been widely reported.23,24 The personalized medication (mercaptopurine) for acute lymphoblastic leukemia of the Lahu ethnic group is worthy of attention.

Furthermore, we found that VKORC1 rs9934438 (A>G) was significantly different in Lahu compared to the other 11 populations as well. A common variant of the vitamin K epoxide reductase complex subunit 1 (VKORC1) gene has also been strongly associated with inter-individual warfarin dosing variability25,26 The warfarin dose of patients from Southern Italy GG genotype carriers at rs9934438 was significantly higher than that of AA genotype carriers or GA genotype patients.26 In different populations, such as the whites and Asians, the VKORC1 polymorphism has showed a sustained and significant effect on the warfarin response, accounting for 11% to 32% of the dose variation27,28 More attention should be paid to warfarin and related agents in the Lahu population.

Our research further found that differences in gene frequency of ADH1C rs698, CYP2A6 rs28399433, and GSTP1 rs1695 between the Lahu and the CHB. Their polymorphism has been reported to be closely related to alcohol metabolism,29 tobacco metabolism,30 and carcinogen metabolism.31 What is more, there are population differences, especially in Asia with the other states. Lahu and CHB were found to be two close populations. However, our study implied that individual medications in clinical practice should also be considered separately in the Lahu population.

In conclusion, the VIP variation detected in Lahu group is different from those of the other 11 populations. Determination of the allele distribution and frequencies of VIP variants in such a minority group would provide a theoretical basis for the safer drug administration and much better therapeutic effects. Our results first provide a basic overview of VIP in Lahu groups, and it is hoped that these data will help to develop the population-specific pharmacogenetics studies. However, this study still has limitations. Presently, the sample size is small. A large number of samples were needed to provide strong evidence for the results, to provide a broad overview of better efficacy and safer drug strategies for the Lahu people, and to influence the rational drug selection and the dosage of the Lahu people. Finally, we hope to help optimize personalized treatment strategies.



# SUPPLEMENTAL FILE 1: pgpm-14-1275.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)