# MAIN TEXT

## Digenic heterozygous mutations of KCNH2 and SCN5A induced young and early‐onset long QT syndrome and sinoatrial node dysfunction

### Abstract

AbstractIntroductionLong QT syndrome (LQTS) is a life‐threatening inherited channelopathy, and prolonged QT intervals easily trigger malignant arrhythmias, especially torsades de pointes and ventricular fibrillation.Materials and methodsThe proband with overlapped phenotypes of LQTS and sinoatrial node dysfunction underwent some necessary examinations, including echocardiography, electrocardiogram (ECG), and Holter monitoring. Next, whole‐exome sequencing was performed, and candidate genes were validated by Sanger sequencing. RNA secondary structure and protein physical‐chemical parameter analyses were used to predict the possible structural change of the proteins induced by the mutations.ResultsWe identified the digenic heterozygous mutations of KCNH2 p.307_308del (NM_001204798, c.921_923del) and SCN5A p.R1865H (NM_001160160, c.G5594A) in the female and young proband (II: 1) of LQTS and ventricular fibrillation with repeat syncope at rest. Subsequently, she occurred with obvious sinus arrest with persistent ventricular pacing of implantable cardioverter‐defibrillator. The heterozygous SCN5Ap.R1865H was carried by her father and sister but not carried by I:2. II:1 carried with KCNH2 p.307_308del as a de novo mutation, but not existed in other family members. RNA secondary structure of KCNH2 p.307_308del showed a false regional double helix, and its amino acids' hydrophobicity was significantly weakened. For the Nav1.5 protein property, SCN5A p.R1865H slightly increased the molecular weight and aliphatic index but reduced the instability index.ConclusionsThe digenic heterozygous KCNH2 and SCN5A mutations were associated with young early‐onset long QT syndrome and sinoatrial node dysfunction.

### INTRODUCTION

Long QT syndrome (LQTS) is an inherited cardiac electrophysiologic disorder characterized by QT interval prolongation in the electrocardiogram and family history of sudden cardiac death (SCD) (Neira et al., 2019; Wallace et al., 2019). A prolonged QT interval is vulnerable to malignant arrhythmias, including torsade de pointes and ventricular fibrillation, which result in unexplained syncope, seizures, and even sudden death in patients, especially young individuals (Shah et al., 2019). LQTS is caused mainly by functional abnormalities of changed repolarizing ionic channels, including mutant potassium channels, sodium channels, or calcium channels. Sixteen genes and more than 600 mutations are linked to LQTS. Mutations of KCNQ1, KCNH2, SCN5A, and ankyrin‐B are more common, and different types of LQTS were associated with different mutations of the genes (Nakano & Shimizu, 2016). In this study, we examined a young proband characterized as LQTS and sinoatrial node dysfunction. The predisposing genes of potential risk were investigated by whole‐exome sequencing (WES) and validated by Sanger sequencing.

### MATERIAL AND METHODS

Here, we studied a Chinese family and the proband (II:1), an 18‐year‐old female who suffered from LQTS since five years old. We obtained detailed clinical information including initial ventricular tachycardia manifestation, presentation age, therapeutic process, and family history. The proband underwent echocardiography, electrocardiogram (ECG), Holter monitoring, implantable cardioverter‐defibrillator (ICD) programming, and other necessary examinations. Her family members were asked to accept routine ECG and echocardiography. Their peripheral blood samples were collected to perform the following detection.

This study was approved by the Guangdong Medical Institutional Review Board and Medical Ethics Committees [no. GDREC2016001H (R1)]. Informed written consent was obtained from the family members.

The clinical diagnostic criteria for LQTS mainly depended on the clinical standard score (Goldenberg & Moss, 2008; Skinner & CSANZ, 2007). (1) Corrected QT interval: when the QT interval ≥0.48 s, the point is 3; 0.46–0.47, 2; 0.45 (in males), 1. (2) when showing torsades de pointes, the points is 2; T‐wave alternans, 1; notched T wave in three leads, 1; and low heart rate for ages, 0.5. (3) Clinical history: when suffering from syncope with stress, the point is 2; syncope without stress, 1; and congenital deafness, 0.5. (4) Family history: family members with definite LQTS, the point is 1; unexplained sudden cardiac death at/under 30‐year old  among immediate family member(s), 0.5.

The probability of LQTS is high when scoring ≥4 points; intermediate probability of LQTS when scoring 2 to 3 points; low probability of LQTS when scoring ≤1 point (Goldenberg & Moss, 2008).

The proband (II: 1) was diagnosed with LQTS and thus selected to conduct WES in this family. Genomic DNA samples were extracted from the peripheral blood of the family members using a standard DNA extraction protocol. The isolated genomic DNA was fragmented into 150–200 bp and then subjected to DNA library preparation Illumina paired‐end protocols. A portion of each library was used to create an equimolar pool. Each pool was amplified to enrich for targets to be sequenced by the Agilent SureSelectXT Target Enrichment System (Agilent Technologies Inc., Santa Clara, CA, USA). Accordingto the manufacturer's protocol, whole‐exome capture was performed with the Agilent SureSelectXT Human All Exon 50 Mb Kit (Agilent Technologies Inc.)

Low‐quality reads were filtered, and 3′/5′ adapters were trimmed using the Trim Galore program. Clean reads were mapped to the hg19 version of the human genome using the Burrows‐Wheeler Aligner (BWA) program. Following the exclusion of duplicate reads, insertion–deletions (InDels) and single nucleotide polymorphisms (SNPs) calling, GATK or Sequence Alignment/Map tools (SAM tools) were used. Functional annotation was used to annotate each SNP and InDel, and nonsynonymous SNPs were predicted by SIFT (Kumar et al., 2009) and PolyPhen‐2 (polymorphism phenotyping v2; Adzhubei et al., 2010). For II: 1 patient, there were ~106 variants of candidate genes occurring in coding regions. The criteria of filtering variants were as follows: (1) missense, nonsense and InDel variants; (2) same variants in the WES data; and (3) SNPs with a minor allele frequency not more than 0.01 according to the NCBI‐SNP database (Smigielski et al., 2000; Via et al., 2010).

The candidate gene sequences were obtained from nucleotide sequence database GenBank (Kuznetsov & Bollin, 2021). The primers (Table 1) were designed using PrimerBank (Wang et al., 2012). These genes from genomic DNA were amplified, and products of amplification were purified using the PCR Purification Kit (Qiagen). They were sequenced directly with the BigDye terminator method (Applied Biosystems, Foster City, CA, USA) on a capillary autosequencer (ABI Prism 3100).

Predisposing gene analysis of whole‐exome sequencing

We searched for the candidate protein from the universal protein knowledgebase (UniProt; UniProt Consortium T, 2018). Multiple cross‐species amino acid sequences were selected in the list and extracted into FASTA format. For Multiple Sequence Alignment, T‐Coffee (https://www.ebi.ac.uk/Tools/msa/tcoffee/) was used to compare the conservation of target mutation sites.

We used the RNAfold WebSever (http://rna.tbi.univie.ac.at/cgi‐bin/RNAWebSuite/RNAfold.cgi) to predict the RNA secondary structure. The difference of the minimum free energy (MFE) in the centroid secondary structure was evaluated between the mutant and wild‐type mRNA.

To compare the potential functional influence on the protein physical‐chemical properties, we focused on analyzing the changes of hydrophobicity, transmembrane domain and protein phosphorylation of amino acids induced by mutant and wild‐type SCN5A and KCNH2. The ProtScale (https://web.expasy.org/protscale/) and NetPhos (http://www.cbs.dtu.dk/services/NetPhos/) were used for hydrophobicity, transmembrane domain and phosphorylation analysis. Next, we used ProtParam tool (https://web.expasy.org/protparam/) to predict the theoretical pI, instability index, aliphatic index, grand average of hydropathicity (GRAVY) and so on.

### Study objective

Here, we studied a Chinese family and the proband (II:1), an 18‐year‐old female who suffered from LQTS since five years old. We obtained detailed clinical information including initial ventricular tachycardia manifestation, presentation age, therapeutic process, and family history. The proband underwent echocardiography, electrocardiogram (ECG), Holter monitoring, implantable cardioverter‐defibrillator (ICD) programming, and other necessary examinations. Her family members were asked to accept routine ECG and echocardiography. Their peripheral blood samples were collected to perform the following detection.

This study was approved by the Guangdong Medical Institutional Review Board and Medical Ethics Committees [no. GDREC2016001H (R1)]. Informed written consent was obtained from the family members.

### Diagnosis of long QT syndrome

The clinical diagnostic criteria for LQTS mainly depended on the clinical standard score (Goldenberg & Moss, 2008; Skinner & CSANZ, 2007). (1) Corrected QT interval: when the QT interval ≥0.48 s, the point is 3; 0.46–0.47, 2; 0.45 (in males), 1. (2) when showing torsades de pointes, the points is 2; T‐wave alternans, 1; notched T wave in three leads, 1; and low heart rate for ages, 0.5. (3) Clinical history: when suffering from syncope with stress, the point is 2; syncope without stress, 1; and congenital deafness, 0.5. (4) Family history: family members with definite LQTS, the point is 1; unexplained sudden cardiac death at/under 30‐year old  among immediate family member(s), 0.5.

The probability of LQTS is high when scoring ≥4 points; intermediate probability of LQTS when scoring 2 to 3 points; low probability of LQTS when scoring ≤1 point (Goldenberg & Moss, 2008).

### Whole‐exome sequencing, sequence alignment, variation calling, and data annotation

The proband (II: 1) was diagnosed with LQTS and thus selected to conduct WES in this family. Genomic DNA samples were extracted from the peripheral blood of the family members using a standard DNA extraction protocol. The isolated genomic DNA was fragmented into 150–200 bp and then subjected to DNA library preparation Illumina paired‐end protocols. A portion of each library was used to create an equimolar pool. Each pool was amplified to enrich for targets to be sequenced by the Agilent SureSelectXT Target Enrichment System (Agilent Technologies Inc., Santa Clara, CA, USA). Accordingto the manufacturer's protocol, whole‐exome capture was performed with the Agilent SureSelectXT Human All Exon 50 Mb Kit (Agilent Technologies Inc.)

Low‐quality reads were filtered, and 3′/5′ adapters were trimmed using the Trim Galore program. Clean reads were mapped to the hg19 version of the human genome using the Burrows‐Wheeler Aligner (BWA) program. Following the exclusion of duplicate reads, insertion–deletions (InDels) and single nucleotide polymorphisms (SNPs) calling, GATK or Sequence Alignment/Map tools (SAM tools) were used. Functional annotation was used to annotate each SNP and InDel, and nonsynonymous SNPs were predicted by SIFT (Kumar et al., 2009) and PolyPhen‐2 (polymorphism phenotyping v2; Adzhubei et al., 2010). For II: 1 patient, there were ~106 variants of candidate genes occurring in coding regions. The criteria of filtering variants were as follows: (1) missense, nonsense and InDel variants; (2) same variants in the WES data; and (3) SNPs with a minor allele frequency not more than 0.01 according to the NCBI‐SNP database (Smigielski et al., 2000; Via et al., 2010).

### Sanger sequencing for candidate genes

The candidate gene sequences were obtained from nucleotide sequence database GenBank (Kuznetsov & Bollin, 2021). The primers (Table 1) were designed using PrimerBank (Wang et al., 2012). These genes from genomic DNA were amplified, and products of amplification were purified using the PCR Purification Kit (Qiagen). They were sequenced directly with the BigDye terminator method (Applied Biosystems, Foster City, CA, USA) on a capillary autosequencer (ABI Prism 3100).

Predisposing gene analysis of whole‐exome sequencing

### Conservation analyses

We searched for the candidate protein from the universal protein knowledgebase (UniProt; UniProt Consortium T, 2018). Multiple cross‐species amino acid sequences were selected in the list and extracted into FASTA format. For Multiple Sequence Alignment, T‐Coffee (https://www.ebi.ac.uk/Tools/msa/tcoffee/) was used to compare the conservation of target mutation sites.

### RNA secondary structure prediction

We used the RNAfold WebSever (http://rna.tbi.univie.ac.at/cgi‐bin/RNAWebSuite/RNAfold.cgi) to predict the RNA secondary structure. The difference of the minimum free energy (MFE) in the centroid secondary structure was evaluated between the mutant and wild‐type mRNA.

### Protein physical and chemical parameter prediction

To compare the potential functional influence on the protein physical‐chemical properties, we focused on analyzing the changes of hydrophobicity, transmembrane domain and protein phosphorylation of amino acids induced by mutant and wild‐type SCN5A and KCNH2. The ProtScale (https://web.expasy.org/protscale/) and NetPhos (http://www.cbs.dtu.dk/services/NetPhos/) were used for hydrophobicity, transmembrane domain and phosphorylation analysis. Next, we used ProtParam tool (https://web.expasy.org/protparam/) to predict the theoretical pI, instability index, aliphatic index, grand average of hydropathicity (GRAVY) and so on.

### RESULT

The family pedigree and genotype–phenotype correlations are shown in Figure 1a. The female proband II: 1 suffered from repeated convulsions and occasional syncope in April 2008 since five years old. Her ECG monitoring showed torsades de pointes, but the monitoring information did not be saved. The Holter and ECG demonstrated LQTS. The QTc interval was 550 milliseconds (ms) when the heart rate was 79 bpm. It was 560 ms when the heart rate was 131 bpm (Figure 1b, c). The ECGs of her families showed normal QTc intervals (Figure 1d, f). There is no history of syncope, seizures, or unexplained sudden death among her father, mother, and younger sister. The clinical standard score of the proband was 6 points according to the guidelines. Echocardiography, chest radiographs, and thyroid examination presented no significant abnormalities. II: 1 had experienced repeated amaurosis and syncope in May 2012 and been admitted to hospital. Next, she had been treated with ICD. One month later, ICD monitoring showed that the patient experienced multiple ventricular tachycardias (235 bmp) and even ventricular fibrillation. The episodes had triggered ICD to diagnose and release cardioversion and defibrillation treatment (Figure 1g). The charging energy gradually increased from 15J to 36J and achieve successful treatment. Three years later (in Feb 2018), the Holter monitoring presented sinus arrest with persistent ventricular pacing of ICD (Figure 1h). The ICD pacing rhythm accounted for 81.2% (no abnormalities in pacing and sense). Ventricular fusion beat, occasional ventricular premature beats, and long QT intervals were also found in the Holter recording.

Family pedigree and representative ECGs. (a) The proband diagnosed with LQTS is shown using arrows. II: 1 carried the digenic heterozygous mutations of KCNH2 p.307_308del and SCN5A p.R1865H. I: 1 and II: 2 were heterozygous for SCN5A p.R1865H. Except II: 1, other family members did not carry KCNH2 mutation. (b, c), The ECG images of II: 1 demonstrated LQTS. QTc interval was 550 milliseconds (ms) when the heart rate was 79 bpm. It was 560 ms when the heart rate was 131 bpm. (d–f), Representative ECGs of family members of the proband (the chest lead V6 of II: 2 fell off). (g) The proband experienced multiple ventricular tachycardias and even ventricular fibrillation by ICD monitoring. ICD had released cardioversion and defibrillation treatment. (h) The Holter presented sinus arrest with persistent ventricular pacing of ICD

There were 552 variantss in exon and splicing regions by WES. Four potential pathogenic variants, including SCN5A p.R1865H (NM_001160160, c.G5594A), LAMA2 p.A961T (NM_000426, c.G2881A), KCNH2 p.307_308del (NM_001204798, c.921_923del), and DMD p.E1028V (NM_004011, c.A3083T) were involved in the occurrence of arrhythmia and cardiomyopathy (Table 2). In these known and candidate genes, KCNH2 gene encodes voltage‐gated potassium channel activity of cardiomyocytes, which participated in the action potential repolarization. SCN5A gene encodes for voltage‐gated sodium channel subunit as an integral membrane protein, responsible for the initial upstroke of the action potential (obtained from GenBank database). Mutations of KCNH2 and SCN5A genes are closely related to LQTS. The mutations of KCNH2 p.307_308del and SCN5A p.R1865H were found in the proband by WES and validated as positive by Sanger sequencing.

Additionally, the heterozygous SCN5A p.R1865H was carried by I: 1 and II: 2, but not carried by I: 2 (Figure 1a). Except II: 1, other family members without cardiac event or cardiac disease did not carry KCNH2 mutation. Moreover, the conservation analyses demonstrated that the mutant sites of amino acid sequences of SCN5A and KCNH2 protein were highly conserved (Figure 2). Therefore, KCNH2 p.307_308del was considered as de novo mutation in II: 1 (Figure 1a and Figure 3).

Predisposing gene analysis of whole‐exome sequencing

Chr, chromosome; 1000G, 1000 genomes (2015 version); SNP, single nucleotide polymorphism; B, benign; D, damaging; T, tolerated; –, no report; OMIM, Online Mendelian Inheritance in Man.

Conservation analyses at the mutant sites of SCN5A and KCNH2 protein. SCN5A p.R1865 and KCNH2 p.307_308 of amino acid sequences were highly conserved across the common species

Sanger sequencing for SCN5A and KCNH2 mutations. KCNH2 p.307_308del and SCN5A p.R1865H of the proband were validated as positive by Sanger sequencing. Additionally, I: 1 and II: 2 carried with the heterozygous for SCN5A p.R1865H. Except II: 1, other family members did not carry with the KCNH2 mutation

The RNA secondary structure differences were presented by the RNAfold WebSever (Figure 4). Compared with wild‐type KCNH2 (Figure 4a), the structure of KCNH2 p.307_308del affected the single‐stranded RNA folding, resulting in a false regional double helix (Figure 4b). The minimum free energy (MFE) of KCNH2 p.307_308del increased, which thus lead to a reduction of structural stability. However, SCN5A p.R1865H showed no significant influence on the RNA structure (Figure 4c,d). The MFE of SCN5A p.R1865H mutation (−178.70 kcal/mol) was approximately similar to that of the wild type (−178.30 kcal/mol), which probably induced no obvious change in the centroid secondary structure.

RNA secondary structural prediction. (a, b) Compared with wild‐type KCNH2, the structure of KCNH2 p.307_308del affected the single‐stranded RNA folding, resulting in a false regional double helix. The minimum free energy (MFE) of KCNH2 p.307_308del increased, which thus led to a reduction of structural stability. (c, d) SCN5A p.R1865H showed no significant influence on the RNA structure, and the MFE value of SCN5A p.R1865H mutation was approximately similar to that of the wild type

Compared with the amino acids of wild‐type KCNH2 (Table 3), KCNH2 p.307_308del showed a decreasing trend in molecular weight and increasing instability. However, the prediction of theoretical pI, aliphatic index and GRAVY presented no significant differences. Compared to the Nav1.5 protein properties of wild‐type SCN5A, SCN5A p.R1865H slightly increased its molecular weight and aliphatic index but reduced its instability index. Theoretical pI, aliphatic index, and GRAVY were not affected by SCN5A p.R1865H.

Amino acids physical and chemical parameter prediction

Abbreviation: GRAVY, Grand average of hydropathicity.

Next, hydrophobicity analyses for wild‐type and mutant proteins were performed (Table 4, Figure 5). The changed site (position 307) of KCNH2 p.307_308del was located close to the largest hydrophobic region of the protein (Figure 5a). The hydrophobicity of predicted amino acid residues and adjacent sequences was significantly weakened, which probably made the largest hydrophobic domain (position 303) disorganized. The changed site of the SCN5A gene (position 1864) increased the corresponding amino acid residues and nearby sequences hydrophobicity, but the influence was not significant (Figure 5b). Transmembrane structure analysis (Table 4) showed that the mutant site of KCNH2 p.307_308del was located in the protein's transmembrane domain, which suggested that the site may be associated with potential functions, including protein localization, substances transportation, ion channels, and others. Phosphorylation analysis of KCNH2 p.307_308del showed that this mutation caused amino acid residues near position 309 dephosphorylation. The result indicated that KCNH2 p.307_308del might impact the catalytic efficiency of protein. However, there was no significant change in protein phosphorylation for SCN5A p.R1865H (Table 4).

Hydrophobicity, transmembrane domain, and phosphorylation analyses

The hydrophobicity prediction of protein. (a) The changed site (position 307) of KCNH2 p.307_308del was located close to the largest hydrophobic region of the protein. The hydrophobicity of predicted amino acid residues and adjacent sequences was significantly weaken. (b) The changed site of SCN5A gene (position 1864) increased the corresponding amino acid residues and nearby sequences hydrophobicity, but the influence was not significant

### Baseline characteristics

The family pedigree and genotype–phenotype correlations are shown in Figure 1a. The female proband II: 1 suffered from repeated convulsions and occasional syncope in April 2008 since five years old. Her ECG monitoring showed torsades de pointes, but the monitoring information did not be saved. The Holter and ECG demonstrated LQTS. The QTc interval was 550 milliseconds (ms) when the heart rate was 79 bpm. It was 560 ms when the heart rate was 131 bpm (Figure 1b, c). The ECGs of her families showed normal QTc intervals (Figure 1d, f). There is no history of syncope, seizures, or unexplained sudden death among her father, mother, and younger sister. The clinical standard score of the proband was 6 points according to the guidelines. Echocardiography, chest radiographs, and thyroid examination presented no significant abnormalities. II: 1 had experienced repeated amaurosis and syncope in May 2012 and been admitted to hospital. Next, she had been treated with ICD. One month later, ICD monitoring showed that the patient experienced multiple ventricular tachycardias (235 bmp) and even ventricular fibrillation. The episodes had triggered ICD to diagnose and release cardioversion and defibrillation treatment (Figure 1g). The charging energy gradually increased from 15J to 36J and achieve successful treatment. Three years later (in Feb 2018), the Holter monitoring presented sinus arrest with persistent ventricular pacing of ICD (Figure 1h). The ICD pacing rhythm accounted for 81.2% (no abnormalities in pacing and sense). Ventricular fusion beat, occasional ventricular premature beats, and long QT intervals were also found in the Holter recording.

Family pedigree and representative ECGs. (a) The proband diagnosed with LQTS is shown using arrows. II: 1 carried the digenic heterozygous mutations of KCNH2 p.307_308del and SCN5A p.R1865H. I: 1 and II: 2 were heterozygous for SCN5A p.R1865H. Except II: 1, other family members did not carry KCNH2 mutation. (b, c), The ECG images of II: 1 demonstrated LQTS. QTc interval was 550 milliseconds (ms) when the heart rate was 79 bpm. It was 560 ms when the heart rate was 131 bpm. (d–f), Representative ECGs of family members of the proband (the chest lead V6 of II: 2 fell off). (g) The proband experienced multiple ventricular tachycardias and even ventricular fibrillation by ICD monitoring. ICD had released cardioversion and defibrillation treatment. (h) The Holter presented sinus arrest with persistent ventricular pacing of ICD

### WES and predisposing gene analysis

There were 552 variantss in exon and splicing regions by WES. Four potential pathogenic variants, including SCN5A p.R1865H (NM_001160160, c.G5594A), LAMA2 p.A961T (NM_000426, c.G2881A), KCNH2 p.307_308del (NM_001204798, c.921_923del), and DMD p.E1028V (NM_004011, c.A3083T) were involved in the occurrence of arrhythmia and cardiomyopathy (Table 2). In these known and candidate genes, KCNH2 gene encodes voltage‐gated potassium channel activity of cardiomyocytes, which participated in the action potential repolarization. SCN5A gene encodes for voltage‐gated sodium channel subunit as an integral membrane protein, responsible for the initial upstroke of the action potential (obtained from GenBank database). Mutations of KCNH2 and SCN5A genes are closely related to LQTS. The mutations of KCNH2 p.307_308del and SCN5A p.R1865H were found in the proband by WES and validated as positive by Sanger sequencing.

Additionally, the heterozygous SCN5A p.R1865H was carried by I: 1 and II: 2, but not carried by I: 2 (Figure 1a). Except II: 1, other family members without cardiac event or cardiac disease did not carry KCNH2 mutation. Moreover, the conservation analyses demonstrated that the mutant sites of amino acid sequences of SCN5A and KCNH2 protein were highly conserved (Figure 2). Therefore, KCNH2 p.307_308del was considered as de novo mutation in II: 1 (Figure 1a and Figure 3).

Predisposing gene analysis of whole‐exome sequencing

Chr, chromosome; 1000G, 1000 genomes (2015 version); SNP, single nucleotide polymorphism; B, benign; D, damaging; T, tolerated; –, no report; OMIM, Online Mendelian Inheritance in Man.

Conservation analyses at the mutant sites of SCN5A and KCNH2 protein. SCN5A p.R1865 and KCNH2 p.307_308 of amino acid sequences were highly conserved across the common species

Sanger sequencing for SCN5A and KCNH2 mutations. KCNH2 p.307_308del and SCN5A p.R1865H of the proband were validated as positive by Sanger sequencing. Additionally, I: 1 and II: 2 carried with the heterozygous for SCN5A p.R1865H. Except II: 1, other family members did not carry with the KCNH2 mutation

### RNA secondary structure prediction

The RNA secondary structure differences were presented by the RNAfold WebSever (Figure 4). Compared with wild‐type KCNH2 (Figure 4a), the structure of KCNH2 p.307_308del affected the single‐stranded RNA folding, resulting in a false regional double helix (Figure 4b). The minimum free energy (MFE) of KCNH2 p.307_308del increased, which thus lead to a reduction of structural stability. However, SCN5A p.R1865H showed no significant influence on the RNA structure (Figure 4c,d). The MFE of SCN5A p.R1865H mutation (−178.70 kcal/mol) was approximately similar to that of the wild type (−178.30 kcal/mol), which probably induced no obvious change in the centroid secondary structure.

RNA secondary structural prediction. (a, b) Compared with wild‐type KCNH2, the structure of KCNH2 p.307_308del affected the single‐stranded RNA folding, resulting in a false regional double helix. The minimum free energy (MFE) of KCNH2 p.307_308del increased, which thus led to a reduction of structural stability. (c, d) SCN5A p.R1865H showed no significant influence on the RNA structure, and the MFE value of SCN5A p.R1865H mutation was approximately similar to that of the wild type

### Physical and chemical parameter prediction of protein

Compared with the amino acids of wild‐type KCNH2 (Table 3), KCNH2 p.307_308del showed a decreasing trend in molecular weight and increasing instability. However, the prediction of theoretical pI, aliphatic index and GRAVY presented no significant differences. Compared to the Nav1.5 protein properties of wild‐type SCN5A, SCN5A p.R1865H slightly increased its molecular weight and aliphatic index but reduced its instability index. Theoretical pI, aliphatic index, and GRAVY were not affected by SCN5A p.R1865H.

Amino acids physical and chemical parameter prediction

Abbreviation: GRAVY, Grand average of hydropathicity.

Next, hydrophobicity analyses for wild‐type and mutant proteins were performed (Table 4, Figure 5). The changed site (position 307) of KCNH2 p.307_308del was located close to the largest hydrophobic region of the protein (Figure 5a). The hydrophobicity of predicted amino acid residues and adjacent sequences was significantly weakened, which probably made the largest hydrophobic domain (position 303) disorganized. The changed site of the SCN5A gene (position 1864) increased the corresponding amino acid residues and nearby sequences hydrophobicity, but the influence was not significant (Figure 5b). Transmembrane structure analysis (Table 4) showed that the mutant site of KCNH2 p.307_308del was located in the protein's transmembrane domain, which suggested that the site may be associated with potential functions, including protein localization, substances transportation, ion channels, and others. Phosphorylation analysis of KCNH2 p.307_308del showed that this mutation caused amino acid residues near position 309 dephosphorylation. The result indicated that KCNH2 p.307_308del might impact the catalytic efficiency of protein. However, there was no significant change in protein phosphorylation for SCN5A p.R1865H (Table 4).

Hydrophobicity, transmembrane domain, and phosphorylation analyses

The hydrophobicity prediction of protein. (a) The changed site (position 307) of KCNH2 p.307_308del was located close to the largest hydrophobic region of the protein. The hydrophobicity of predicted amino acid residues and adjacent sequences was significantly weaken. (b) The changed site of SCN5A gene (position 1864) increased the corresponding amino acid residues and nearby sequences hydrophobicity, but the influence was not significant

### DISCUSSION

In our study, the proband with overlapped phenotypes of young early‐onset LQTS and sinoatrial node dysfunction was first demonstrated to carry with the digenic heterozygous and pathogenic mutations of KCNH2 p.307_308del and SCN5A p.R1865H. KCNH2 p.307_308del induced a regional double helix of the amino acids misfolded and largest hydrophobic domain disorganized, which thus mainly caused LQTS. The SCN5A p.R1865H slightly increased the molecular weight and aliphatic index, but reduced the instability index of Nav1.5 protein property, which potentially induced subsequent sinoatrial node dysfunction.

Genetic screening for LQTS is of great clinical value. The genotype is closely related to gene‐specific epidemiology, risk stratification, and treatment of LQTS. Different channel mutations present their own electrocardiogram signature, typical presentations, and prognosis (Skinner et al., 2019; Wilde & Amin, 2017). KCNH2 gene (HERG) encodes a voltage‐sensitive channel protein mediating the rapid component of delayed rectifier K+ current (Kv11.1) (Butler et al., 2019). The channel protein is expressed mainly in the myocardium and acts as a molecular target for most antiarrhythmic drugs. KCNH2 mutations are associated with inherited arrhythmias involving LQTS, ventricular tachycardia, and SCD (Vandenberg et al., 2012). Especially, loss‐of‐function mutations in KCNH2 are the leading causes of LQTS (type 2) by delaying the ventricular action potential repolarization (Delisle et al., 2004; Ono et al., 2020). In this family, the patient (II: 1) with digenic heterozygous mutations of KCNH2 p.307_308del and SCN5A p.R1865H presented the earliest phenotype of LQTS, and she suffered from syncope, torsades de pointes, and ventricular fibrillation more frequently at rest, whereas the members (I:1 and II:2) without KCNH2 p.307_308del showed normal QT intervals and cardiac function. The changed site of KCNH2 p.307_308 was highly conserved across most species, suggesting p.307_308 of KCNH2 protein playing a significant role in function. The result indicated that the novel genetic background, KCNH2 p.307_308del, may affect and even induce the phenotype of LQTS. The predictions of the RNA secondary structure and physical‐chemical parameters showed KCNH2 p.307_308del affected the single‐stranded RNA folding, and subsequently and significantly weaken the hydrophobicity of mutant amino acid residues, which indicated that KCNH2 mutation probably played a dominant hereditary role in the occurrence of LQTS in this proband.

SCN5A gene is located in chromosome 3p22.2, encoding the alpha subunit of the voltage‐gated sodium channel (Nav1.5) in human cardiomyocytes (Li et al., 2018). Nav1.5 channel was described first in 1955  to be responsible for inward sodium current, which participated in the rapid flow of Na+ during the depolarization phase in cardiomyocytes (George et al., 1995). The protein contains four homologous domains (DI–DIV), and each includes six transmembrane spanning segments (S1–S6). SCN5A mutations may influence the expression and function of the Nav1.5 channel. Therefore, the pathogenic mutations of SCN5A probably caused distinct clinical diseases, including LQTS, Brugada syndrome, sick sinus syndrome, and even fatal arrhythmias (Benson et al., 2003; Remme et al., 2006; Wilde & Amin, 2018). Over 300 gain‐of‐function mutations were associated with type 3 of LQTS (Flaim et al., 2007; Jenewein et al., 2017; Remme, 2013; Wilde et al., 2016). Our study firstly identified SCN5A p.R1865H in the patient with overlapped phenotypes of LQTS and sinoatrial node dysfunction. The RNA secondary structure predictions of SCN5A p.R1865H slightly affected its RNA folding. In our study, the proband was diagnosed with the earliest LQTS and subsequently experienced sinoatrial node dysfunction, such as sinus arrest with persistent ventricular pacing of ICD. It was worth mentioning that SCN5A p.R1865H was found in one patient with arrhythmogenic right ventricular cardiomyopathy/dysplasia (ARVC/D) (Te Riele et al., 2017) and demonstrated a close association with reduced sodium current. Loss‐of‐function mutations of SCN5A were commonly related to sinoatrial node dysfunction. Therefore, we speculated that SCN5A p.R1865H, a loss‐of‐function mutation, led to subsequent sinus arrest in the proband by reducing sodium channel current. As mentioned above, some family members carrying heterozygous SCN5A p.R1865H mutation showed no evidence of cardiac events or cardiac diseases. The reason may be as follows: First, I:1 and II:2 who carried with the heterozygous SCN5A p.R1865H presented no clinical syndromes because of incomplete penetrance or delayed onset. Moreover, gain‐of‐function mutation of SCN5A commonly induced LQTS, while loss‐of‐function mutation of SCN5A ordinary led to sinoatrial node dysfunction, atrioventricular block, atrial fibrillation and cardiomyopathy (e.g., ARVC/D; Blana et al., 2010; Han et al., 2018). Therefore, in this study, SCN5A p.R1865H may be the main cause of sinoatrial node dysfunction, whereas KCNH2 p.307_308del only carried by II: 1 may potentially induce the phenotype of LQTS. However, it was hard to determine whether the coexisting interactions of KCNH2 p.307_308del and SCN5A p.R1865H increased the risk of young and early‐onset LQTS, or whether KCNH2 mutation was only associated with LQTS, while SCN5A mutation was only associated with sinoatrial node dysfunction.

### CONCLUSIONS

We firstly identified the novel digenic heterozygous mutations by WES, KCNH2 p.307_308del and SCN5A p.R1865H, which resulted in LQTS with repeat syncope, torsades de pointes, ventricular fibrillation, and sinoatrial node dysfunction. KCNH2 p.307_308del may affect the function of Kv11.1 channel in cardiomyocytes by inducing a regional double helix of the amino acids misfolded and largest hydrophobic domain disorganized. SCN5A p.R1865H reduced the instability index of Nav1.5 protein and sodium current. All of these were closely related to young early‐onset LQTS and sinoatrial node dysfunction.

### LIMITATIONS

Our study was performed only in the statistical field on KCNH2 p.307_308del and SCN5A p.R1865H by WES and predisposing genes analyses. More cellular and animal research is needed to further investigate whether the coexisting interaction of KCNH2 p.307_308del and SCN5A p.R1865H increases the risk of the early‐onset LQTS and sinoatrial node dysfunction.

### CONSENT FOR PUBLICATION

Not applicable.

### CONFLICT OF INTEREST

The authors declare that they have no competing interest.

### AUTHOR CONTRIBUTIONS

ZY and YBL contributed to pedigree analysis and writing; YTM contributed to bioinformatics analysis; JNH, WLZ, and LC contributed to Sanger sequencing; JZX, YH, LBW, and FW made case collection and follow‐up; XFL and YBL made quality control of clinical data and clinical design.



# SUPPLEMENTAL FILE 1: ANEC-27-e12889.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)