# MAIN TEXT

## Genetic Polymorphisms of Very Important Pharmacogene Variants in the Blang Population from Yunnan Province in China

### Abstract

BackgroundWe aimed to enrich the pharmacogenomic information of a Blang population (BP) from Yunnan Province in China.MethodsWe genotyped 55 very important pharmacogene (VIP) variants from the PharmGKB database and compared their genotype distribution (GD) in a BP with that of 26 populations by the χ2 test. The minor allele frequency (MAF) distribution of seven significantly different single-nucleotide polymorphisms (SNPs) was conducted to compare the difference between the BP and 26 other populations.ResultsCompared with the GD of 55 loci in the BP, among 26 studied populations, GWD, YRI, GIH, ESN, MSL, TSI, PJL, ACB, FIN and IBS were the top-10 populations, which showed a significantly different GD >35 loci. CHB, JPT, CDX, CHS, and KHV populations had a significantly different GD <20 loci. A GD difference of 27–34 loci was found between the BP and 11 populations (LWK, CEU, ITU, STU, PUR, CLM, GBR, ASW, BEB, MXL and PEL). The GD of five loci (rs750155 (SULT1A1), rs4291 (ACE), rs1051298 (SLC19A1), rs1131596 (SLC19A1) and rs1051296 (SLC19A1)) were the most significantly different in the BP as compared with that of the other 26 populations. The genotype frequency of rs1800764 (ACE) and rs1065852 (CYP2D6) was different in all populations except for PEL and LWK, respectively. MAFs of rs1065852 (CYP2D6) and rs750155 (SULT1A1) showed the largest fluctuation between the BP and SAS, EUR, AFR and AMR populations.ConclusionOur data can provide theoretical guidance for safe and efficacious personalized drug use in the Blang population.

### Introduction

The use of drugs should be different among diverse ethnic groups because of differences in ethnicity, age, sex, environmental factors and genetic factors. If these differences are ignored, then drug sensitivity, metabolic rate, and adverse reactions are affected, which influences the curative effect of drugs and aggravates the illness of patients.

Genetic factors can explain up to 20–95% of the variability in drug response.1 Variations in genes can affect the pharmacokinetics/pharmacodynamics of drugs, as well as their absorption and metabolism. “Pharmacogenes” are genes that decide the fate of drug pharmacology in a biological system. In general, pharmacogenes correspond to specific gene “superfamilies”. Among numerous gene superfamilies, the cytochrome P450 superfamily is the most widely researched in pharmacogenomics studies. It has been reported that polymorphisms of cytochrome P450 account for the most frequent variations in the phase-I metabolism of drugs.2 Variations in most gene superfamilies can affect the metabolism of drugs and disease risk.

The single-nucleotide polymorphism (SNP) is the most common variation of very important pharmacogenes (VIPs). Usually, SNPs are employed to analyze the pharmacogenomic information in different populations.3,4 “Pharmacogenomics” is an emerging approach to “precision medicine”. Pharmacogenomics plays a major part in precision medicine by “tailoring” the selection and dosing to the patient’s genetic features.5 The Pharmacogenomics Knowledge Base (PharmGKB; www.pharmgkb.org/) is one of the most commonly used databases on primary pharmacogenomics. PharmGKB contains information on gene-variant annotations, drug-centered pathways, VIPs and diverse diseases. PharmGKB aims to share genotype, phenotype, or other data on genetic variations among researchers.6

It has been demonstrated that pharmacogenomic analysis of a specific population can aid the efficacious, accurate use of drugs in a population.7,8 For example, Bader et al found that variants of the vitamin K epoxide reductase complex gene (VKORC) and cytochrome P450 family 2 subfamily C member 9 gene (CYP2C9), which encode enzymes for warfarin metabolism, were the strongest predictors of variability in the warfarin dose among different populations in Middle East and North Africa.7 In addition, Kim et al demonstrated that adverse drug reactions could be avoided if preemptive genotyping was employed in a South Korean population.8

The US Food and Drug Administration (www.fda.gov/) have recognized >250 biomarkers with known pharmacogenomic value, and provided recommendations for therapeutic management.9 Recently, pharmacogenomics information on increasing numbers of ethnic minorities in China has been explored. For example, Liu et al found that, compared with 11 populations in a dataset from the International HapMap Project (www.genome.gov/), differences in expression between the rs2070676 of the cytochrome P450 family 2 subfamily E member 1 gene (CYP2E1) and rs1065852 of cytochrome P450 family 2 subfamily D member 6 gene (CYP2D6) in people of Zhuang nationality were the greatest according to genotyping of samples of 105 people of Zhuang nationality.3 Besides, He et al concluded that expression of rs4291 of the angiotensin I-converting enzyme gene (ACE), rs1051296 of the solute carrier family 19 member 1 gene (SLC19A1) and rs1065852 of CYP2D6 differed significantly in a Tibetan population compared with that of 26 other populations after genotyping of 200 samples from a Tibetan population. They also found that the allele frequency in this Tibetan population differed least from that of an East Asian population, and differed most from that of a North American population.4

China has 56 ethnic groups. The Blang ethnic group is found in Yunnan Province in China. According to the Sixth National Census in 2010, the total number of people of Blang ethnicity was 119,639. Among them, >30,000 people live in Mount Blang, Xiding, Bada, Daluo, Mengman, Menggang and other towns in Menghai County in Xishuangbanna Dai Autonomous Prefecture.10 People of Blang ethnicity live in mountainous areas with a mild climate and abundant rainfall, which is very conducive to plant growth. The area in which Blang populations live is one of the main raw material-producing areas of “Pu’er tea” and “Mengku tea”. Even though genetic studies on Blang populations have been conducted,10–12 pharmacogenomics information of the Blang population is lacking. Cheng et al explored the pharmacogenomics information of a Blang population.13

Here, we shed light on the pharmacogenomic information of a Blang population by genotyping 55 different loci of 27 VIPs using 200 samples from Yunnan Province. These samples are different from those investigated by Cheng and collaborators. We also compared the distribution of genotype frequency and minor allele frequency (MAF) differences (55 loci of 27 VIPs) with a Blang population and 26 other populations. The genetic variations of the 15 gene superfamilies involved in the present study were related mainly to changes in drug metabolism and disease risk.2,14–27 We wished to enrich the pharmacogenomics information of a Blang population and provide a theoretical foundation for promoting the development of personalized precise medication for Blang populations in the future.

### Materials and Methods

The study protocol was approved by the Clinical Research Ethics Committee of Xizang Minzu University (Xianyang, China). Written informed consent was obtained from each study participant before a blood sample was given.

Two-hundred randomly selected healthy, unrelated individuals of Blang ethnicity from Yunnan Province were recruited. Whole-blood samples were collected according to the study protocol. Candidate participants were healthy individuals and had exclusive Blang ancestry for ≥3 previous generations. People suffering from cancer, infectious diseases, drug/alcohol addiction, severe dysfunction of the heart, liver, or kidney or immune disorders were excluded, as were women who were pregnant or lactating. Thus, the recruited individuals were representative of a Blang population.

PharmGKB was used for selection of genetic variants from published polymorphisms associated with VIP variants. Assays for the loci of 55 genetic variants in 27 VIPs were designed. Loci that could not be designed for an assay were excluded.

We extracted the genomic DNA from the peripheral blood of participants using the GoldMag®-Mini Whole Blood Genomic DNA Purification Kit (GoldMag. Xi’an, China) according to manufacturer protocols. The DNA concentration was measured using the NanoDrop 2000C spectrophotometer (Thermo Scientific, Waltham, MA, USA). MassARRAY Assay Design 3.0 (Sequenom, San Diego, CA, USA) was employed to design multiplexed SNP MassEXTEND assays.28 SNP genotyping was done using MassARRAY RS1000 (Sequenom) according to manufacturer protocols. Sequenom Typer 4.0 was employed to manage and analyze the data on SNP genotyping.29 The basic information on the selected 55 loci related to 27 VIPs of the Blang population are listed in Table 1. The polymerase chain reaction (PCR) primers designed for the selected SNPs are shown in Supplemental Table 1. The basic information comprised the gene name, SNP ID, positions, functional consequence, genotype frequencies and MAF in the Blang population. All samples from the Blang population were genotyped with respect to these variants. PharmGKB was also used for the clinical and variant annotations for seven significantly different SNPs in the Blang population compared with 26 other populations.Table 1The Basic Information of 55 SNPsSNP IDChromosomePositionFunctional ConsequenceGenesGenotypeMAFAAABBBrs11572325159896030Intron_variantCYP2J20191810.048rs10889160159896449Intron_variantCYP2J22331650.093rs890293159926822Upstream_transcript_variantCYP2J2041960.010rs1760217197137438Genic_downstream_transcript_variant,intron_variantDPYD1583990.287rs1801159197515839Coding_sequence_variant,genic_downstream_transcript_variant,intron_variant,missense_variantDPYD14771090.263rs1801265197883329Non_coding_transcript_variant,intron_variant,coding_sequence_variant,5_prime_UTR_variant,missense_variantDPYD2361620.100rs527511866739263_prime_UTR_variantPTGS29761150.235rs204171186681189Upstream_transcript_variant,non_coding_transcript_variantPTGS2011990.003rs121395271201040054Missense_variant,coding_sequence_variant,intron_variantCACNA1S0441560.110rs133741491201043356Missense_variant,coding_sequence_variant,downstream_transcript_variantCACNA1S0181820.045rs38506251201047168Coding_sequence_variant,missense_variantCACNA1S061940.015rs23062381237550803Intron_variantRYR29731170.229rs2231142488131171Coding_sequence_variant,missense_variantABCG26571360.173rs2231137488139962Coding_sequence_variant,missense_variantABCG23898610.442rs698499339632Coding_sequence_variant,non_coding_transcript_variant,missense_variantADH1C6371570.123rs776746799672916Intron_variant,splice_acceptor_variant,genic_downstream_transcript_variant,downstream_transcript_variantCYP3A527131600.168rs2242480799763843Intron_variantCYP3A436104580.444rs18051237150948446Missense_variant,coding_sequence_variant,genic_downstream_transcript_variantKCNH2020000.500rs4646244818390208Upstream_transcript_variant,genic_upstream_transcript_variant,intron_variantNAT212106820.325rs4271002818390758Upstream_transcript_variant,genic_upstream_transcript_variant,intron_variantNAT28901020.265rs1041983818400285Coding_sequence_variant,synonymous_variantNAT240103570.458rs1801280818400344Missense_variant,coding_sequence_variantNAT20211790.053rs1799929818400484Coding_sequence_variant,synonymous_variantNAT21221770.060rs1799930818400593Missense_variant,coding_sequence_variantNAT21885970.303rs1208818400806Missense_variant,coding_sequence_variantNAT21221770.060rs1799931818400860Missense_variant,coding_sequence_variantNAT25531420.158rs1495741818415371NoneNAT24395590.459rs21158191045405641Intron_variantALOX511371520.148rs122485601094761900Upstream_transcript_variantCYP2C190121880.030rs42442851094781859Coding_sequence_variant,synonymous_variantCYP2C198651270.203rs10579101094981296Missense_variant,coding_sequence_variantCYP2C91121870.035rs115721031095058349Missense_variant,coding_sequence_variantCYP2C8041950.010rs79092361095069673Upstream_transcript_variantCYP2C81201790.055rs171104531095069772Upstream_transcript_variantCYP2C89701190.222rs381386710133526101Non_coding_transcript_variant,upstream_transcript_variantCYP2E12411570.113rs203192010133526341Non_coding_transcript_variant,upstream_transcript_variantCYP2E16301640.105rs641343210133535040Intron_variantCYP2E1011990.003rs207067610133537633Intron_variantCYP2E111421470.160rs52191117388025Missense_variant,stop_gained,5_prime_UTR_variant,intron_variant,coding_sequence_variantKCNJ110321680.080rs180102811113412762Missense_variant,coding_sequence_variantDRD2031970.008rs23062831221176804Missense_variant,coding_sequence_variantSLCO1B15471480.143rs45160351247906043Upstream_transcript_variantVDR071930.018rs7625511574749576Intron_variantCYP1A26741190.216rs24723041574751897Intron_variantCYP1A25531420.158rs75015516286092515_prime_UTR_variant,intron_variant,genic_upstream_transcript_variant,upstream_transcript_variantSULT1A10931070.233rs18007641763473168NoneACE0731270.183rs42911763476833Upstream_transcript_variantACE119900.503rs42673851763506395NoneACE7871060.253rs21086221915879621Missense_variant,coding_sequence_variantCYP4F27421510.140rs30931051915897578Missense_variant,coding_sequence_variantCYP4F2020000.500rs81927261940848591Intron_variantCYP2A65631320.183rs10512982145514912Intron_variant,3_prime_UTR_variantSLC19A10183170.458rs10512962145514947Intron_variant,3_prime_UTR_variantSLC19A19156350.435rs11315962145538002Missense_variant,5_prime_UTR_variant,synonymous_variant,genic_upstream_transcript_variant,coding_sequence_variantSLC19A114167190.488rs10658522242130692Intron_variant,missense_variant,coding_sequence_variantCYP2D6020000.500Note: A, reference allele; B, other allele.Abbreviations: SNP, single-nucleotide polymorphism; MAF, minor allele frequency; CYP2J2, cytochrome P450 family 2 subfamily J member 2; DPYD, dihydropyrimidine dehydrogenase; PTGS2, prostaglandin-endoperoxide synthase 2; CACNA1S, calcium voltage-gated channel subunit alpha1 S; RYR2, ryanodine receptor 2; ABCG2, ATP-binding cassette sub-family G member 2; ADH1C, alcohol dehydrogenase 1C (Class I), gamma polypeptide; CYP3A5, cytochrome P450 family 3 subfamily A member 5; CYP3A4, cytochrome P450 family 3 subfamily A member 4; KCNH2, potassium voltage-gated channel subfamily H member 2; NAT2, N-acetyltransferase 2; ALOX5, arachidonate 5-lipoxygenase; CYP2C19, cytochrome P450 family 2 subfamily C member 19; CYP2C9, cytochrome P450 family 2 subfamily C member 9; CYP2C8, cytochrome P450 family 2 subfamily C member 8; CYP2E1, cytochrome P450 family 2 subfamily E member 1; KCNJ11, potassium inwardly rectifying channel subfamily J member 11; DRD2, dopamine receptor D2; SLCO1B1, solute carrier organic anion transporter family member 1B1; VDR, vitamin D3 receptor; CYP1A2, cytochrome P450 family 1 subfamily A member 2; SULT1A1, sulfotransferase family 1A member 1; ACE, angiotensin-converting enzyme; CYP4F2, cytochrome P450 family 4 subfamily F member 2; CYP2A6, cytochrome P450 family 2 subfamily A member 6; SLC19A1, solute carrier family 19 member 1; CYP2D6, cytochrome P450 family 2 subfamily D member 6.

The Basic Information of 55 SNPs

Note: A, reference allele; B, other allele.

Abbreviations: SNP, single-nucleotide polymorphism; MAF, minor allele frequency; CYP2J2, cytochrome P450 family 2 subfamily J member 2; DPYD, dihydropyrimidine dehydrogenase; PTGS2, prostaglandin-endoperoxide synthase 2; CACNA1S, calcium voltage-gated channel subunit alpha1 S; RYR2, ryanodine receptor 2; ABCG2, ATP-binding cassette sub-family G member 2; ADH1C, alcohol dehydrogenase 1C (Class I), gamma polypeptide; CYP3A5, cytochrome P450 family 3 subfamily A member 5; CYP3A4, cytochrome P450 family 3 subfamily A member 4; KCNH2, potassium voltage-gated channel subfamily H member 2; NAT2, N-acetyltransferase 2; ALOX5, arachidonate 5-lipoxygenase; CYP2C19, cytochrome P450 family 2 subfamily C member 19; CYP2C9, cytochrome P450 family 2 subfamily C member 9; CYP2C8, cytochrome P450 family 2 subfamily C member 8; CYP2E1, cytochrome P450 family 2 subfamily E member 1; KCNJ11, potassium inwardly rectifying channel subfamily J member 11; DRD2, dopamine receptor D2; SLCO1B1, solute carrier organic anion transporter family member 1B1; VDR, vitamin D3 receptor; CYP1A2, cytochrome P450 family 1 subfamily A member 2; SULT1A1, sulfotransferase family 1A member 1; ACE, angiotensin-converting enzyme; CYP4F2, cytochrome P450 family 4 subfamily F member 2; CYP2A6, cytochrome P450 family 2 subfamily A member 6; SLC19A1, solute carrier family 19 member 1; CYP2D6, cytochrome P450 family 2 subfamily D member 6.

The genotype data of individuals from 26 populations was obtained from the International HapMap Project Internet website (www.genome.gov/10001688/international-hapmap-project/). The 26 populations were as follows: 1) Chinese Dai in Xishuangbanna, China (CDX); 2) Han Chinese in Beijing, China (CHB); 3) Southern Han Chinese, China (CHS); 4) Japanese in Tokyo, Japan (JPT); 5) Kinh in Ho Chi Minh City, Vietnam (KHV); 6) African Caribbeans in Barbados (ACB); 7) African Ancestry in Southwest USA (ASW); 8) Esan in Nigeria (ESN); 9) Gambian in Western Divisions, The Gambia (GWD); 10) Luhya in Webuye, Kenya (LWK); 11) Mende in Sierra Leone (MSL); 12) Yoruba in Ibadan, Nigeria (YRI); 13) Colombian in Medellin, Colombia (CLM); 14) Mexican Ancestry in Los Angeles, Colombia (MXL); 15) Peruvian in Lima, Peru (PEL); 16) Puerto Rican in Puerto Rico (PUR); 17) Utah residents with Northern and Western European ancestry (CEU); 18) Finnish in Finland (FIN); 19) British in England and Scotland (GBR); 20) Iberian populations in Spain (IBS); 21) Toscani in Italy (TSI); 22) Bengali in Bangladesh (BEB); 23) Gujarati Indian in Houston, Texas (GIH); 24) Indian Telugu in the UK (ITU); 25) Punjabi in Lahore, Pakistan (PJL); 26) Sri Lankan Tamil in the UK (STU).

An exact test was used to test the frequency validity of each VIP variant by assessing the departure from the Hardy–Weinberg equilibrium. The comparison of genotype frequencies between the Blang population and 26 other populations was conducted using the χ2 test. SPSS 17.0 (Armonk, NY, USA) and Excel™ (Microsoft, Redmond, WA, USA) were used to analyze the distribution of genotypes and MAFs. The Bonferroni correction was applied to p < 0.05 (two-sided).

### Ethical Approval of the Study Protocol

The study protocol was approved by the Clinical Research Ethics Committee of Xizang Minzu University (Xianyang, China). Written informed consent was obtained from each study participant before a blood sample was given.

### Participants

Two-hundred randomly selected healthy, unrelated individuals of Blang ethnicity from Yunnan Province were recruited. Whole-blood samples were collected according to the study protocol. Candidate participants were healthy individuals and had exclusive Blang ancestry for ≥3 previous generations. People suffering from cancer, infectious diseases, drug/alcohol addiction, severe dysfunction of the heart, liver, or kidney or immune disorders were excluded, as were women who were pregnant or lactating. Thus, the recruited individuals were representative of a Blang population.

### Variant Selection and Genotyping

PharmGKB was used for selection of genetic variants from published polymorphisms associated with VIP variants. Assays for the loci of 55 genetic variants in 27 VIPs were designed. Loci that could not be designed for an assay were excluded.

We extracted the genomic DNA from the peripheral blood of participants using the GoldMag®-Mini Whole Blood Genomic DNA Purification Kit (GoldMag. Xi’an, China) according to manufacturer protocols. The DNA concentration was measured using the NanoDrop 2000C spectrophotometer (Thermo Scientific, Waltham, MA, USA). MassARRAY Assay Design 3.0 (Sequenom, San Diego, CA, USA) was employed to design multiplexed SNP MassEXTEND assays.28 SNP genotyping was done using MassARRAY RS1000 (Sequenom) according to manufacturer protocols. Sequenom Typer 4.0 was employed to manage and analyze the data on SNP genotyping.29 The basic information on the selected 55 loci related to 27 VIPs of the Blang population are listed in Table 1. The polymerase chain reaction (PCR) primers designed for the selected SNPs are shown in Supplemental Table 1. The basic information comprised the gene name, SNP ID, positions, functional consequence, genotype frequencies and MAF in the Blang population. All samples from the Blang population were genotyped with respect to these variants. PharmGKB was also used for the clinical and variant annotations for seven significantly different SNPs in the Blang population compared with 26 other populations.Table 1The Basic Information of 55 SNPsSNP IDChromosomePositionFunctional ConsequenceGenesGenotypeMAFAAABBBrs11572325159896030Intron_variantCYP2J20191810.048rs10889160159896449Intron_variantCYP2J22331650.093rs890293159926822Upstream_transcript_variantCYP2J2041960.010rs1760217197137438Genic_downstream_transcript_variant,intron_variantDPYD1583990.287rs1801159197515839Coding_sequence_variant,genic_downstream_transcript_variant,intron_variant,missense_variantDPYD14771090.263rs1801265197883329Non_coding_transcript_variant,intron_variant,coding_sequence_variant,5_prime_UTR_variant,missense_variantDPYD2361620.100rs527511866739263_prime_UTR_variantPTGS29761150.235rs204171186681189Upstream_transcript_variant,non_coding_transcript_variantPTGS2011990.003rs121395271201040054Missense_variant,coding_sequence_variant,intron_variantCACNA1S0441560.110rs133741491201043356Missense_variant,coding_sequence_variant,downstream_transcript_variantCACNA1S0181820.045rs38506251201047168Coding_sequence_variant,missense_variantCACNA1S061940.015rs23062381237550803Intron_variantRYR29731170.229rs2231142488131171Coding_sequence_variant,missense_variantABCG26571360.173rs2231137488139962Coding_sequence_variant,missense_variantABCG23898610.442rs698499339632Coding_sequence_variant,non_coding_transcript_variant,missense_variantADH1C6371570.123rs776746799672916Intron_variant,splice_acceptor_variant,genic_downstream_transcript_variant,downstream_transcript_variantCYP3A527131600.168rs2242480799763843Intron_variantCYP3A436104580.444rs18051237150948446Missense_variant,coding_sequence_variant,genic_downstream_transcript_variantKCNH2020000.500rs4646244818390208Upstream_transcript_variant,genic_upstream_transcript_variant,intron_variantNAT212106820.325rs4271002818390758Upstream_transcript_variant,genic_upstream_transcript_variant,intron_variantNAT28901020.265rs1041983818400285Coding_sequence_variant,synonymous_variantNAT240103570.458rs1801280818400344Missense_variant,coding_sequence_variantNAT20211790.053rs1799929818400484Coding_sequence_variant,synonymous_variantNAT21221770.060rs1799930818400593Missense_variant,coding_sequence_variantNAT21885970.303rs1208818400806Missense_variant,coding_sequence_variantNAT21221770.060rs1799931818400860Missense_variant,coding_sequence_variantNAT25531420.158rs1495741818415371NoneNAT24395590.459rs21158191045405641Intron_variantALOX511371520.148rs122485601094761900Upstream_transcript_variantCYP2C190121880.030rs42442851094781859Coding_sequence_variant,synonymous_variantCYP2C198651270.203rs10579101094981296Missense_variant,coding_sequence_variantCYP2C91121870.035rs115721031095058349Missense_variant,coding_sequence_variantCYP2C8041950.010rs79092361095069673Upstream_transcript_variantCYP2C81201790.055rs171104531095069772Upstream_transcript_variantCYP2C89701190.222rs381386710133526101Non_coding_transcript_variant,upstream_transcript_variantCYP2E12411570.113rs203192010133526341Non_coding_transcript_variant,upstream_transcript_variantCYP2E16301640.105rs641343210133535040Intron_variantCYP2E1011990.003rs207067610133537633Intron_variantCYP2E111421470.160rs52191117388025Missense_variant,stop_gained,5_prime_UTR_variant,intron_variant,coding_sequence_variantKCNJ110321680.080rs180102811113412762Missense_variant,coding_sequence_variantDRD2031970.008rs23062831221176804Missense_variant,coding_sequence_variantSLCO1B15471480.143rs45160351247906043Upstream_transcript_variantVDR071930.018rs7625511574749576Intron_variantCYP1A26741190.216rs24723041574751897Intron_variantCYP1A25531420.158rs75015516286092515_prime_UTR_variant,intron_variant,genic_upstream_transcript_variant,upstream_transcript_variantSULT1A10931070.233rs18007641763473168NoneACE0731270.183rs42911763476833Upstream_transcript_variantACE119900.503rs42673851763506395NoneACE7871060.253rs21086221915879621Missense_variant,coding_sequence_variantCYP4F27421510.140rs30931051915897578Missense_variant,coding_sequence_variantCYP4F2020000.500rs81927261940848591Intron_variantCYP2A65631320.183rs10512982145514912Intron_variant,3_prime_UTR_variantSLC19A10183170.458rs10512962145514947Intron_variant,3_prime_UTR_variantSLC19A19156350.435rs11315962145538002Missense_variant,5_prime_UTR_variant,synonymous_variant,genic_upstream_transcript_variant,coding_sequence_variantSLC19A114167190.488rs10658522242130692Intron_variant,missense_variant,coding_sequence_variantCYP2D6020000.500Note: A, reference allele; B, other allele.Abbreviations: SNP, single-nucleotide polymorphism; MAF, minor allele frequency; CYP2J2, cytochrome P450 family 2 subfamily J member 2; DPYD, dihydropyrimidine dehydrogenase; PTGS2, prostaglandin-endoperoxide synthase 2; CACNA1S, calcium voltage-gated channel subunit alpha1 S; RYR2, ryanodine receptor 2; ABCG2, ATP-binding cassette sub-family G member 2; ADH1C, alcohol dehydrogenase 1C (Class I), gamma polypeptide; CYP3A5, cytochrome P450 family 3 subfamily A member 5; CYP3A4, cytochrome P450 family 3 subfamily A member 4; KCNH2, potassium voltage-gated channel subfamily H member 2; NAT2, N-acetyltransferase 2; ALOX5, arachidonate 5-lipoxygenase; CYP2C19, cytochrome P450 family 2 subfamily C member 19; CYP2C9, cytochrome P450 family 2 subfamily C member 9; CYP2C8, cytochrome P450 family 2 subfamily C member 8; CYP2E1, cytochrome P450 family 2 subfamily E member 1; KCNJ11, potassium inwardly rectifying channel subfamily J member 11; DRD2, dopamine receptor D2; SLCO1B1, solute carrier organic anion transporter family member 1B1; VDR, vitamin D3 receptor; CYP1A2, cytochrome P450 family 1 subfamily A member 2; SULT1A1, sulfotransferase family 1A member 1; ACE, angiotensin-converting enzyme; CYP4F2, cytochrome P450 family 4 subfamily F member 2; CYP2A6, cytochrome P450 family 2 subfamily A member 6; SLC19A1, solute carrier family 19 member 1; CYP2D6, cytochrome P450 family 2 subfamily D member 6.

The Basic Information of 55 SNPs

Note: A, reference allele; B, other allele.

Abbreviations: SNP, single-nucleotide polymorphism; MAF, minor allele frequency; CYP2J2, cytochrome P450 family 2 subfamily J member 2; DPYD, dihydropyrimidine dehydrogenase; PTGS2, prostaglandin-endoperoxide synthase 2; CACNA1S, calcium voltage-gated channel subunit alpha1 S; RYR2, ryanodine receptor 2; ABCG2, ATP-binding cassette sub-family G member 2; ADH1C, alcohol dehydrogenase 1C (Class I), gamma polypeptide; CYP3A5, cytochrome P450 family 3 subfamily A member 5; CYP3A4, cytochrome P450 family 3 subfamily A member 4; KCNH2, potassium voltage-gated channel subfamily H member 2; NAT2, N-acetyltransferase 2; ALOX5, arachidonate 5-lipoxygenase; CYP2C19, cytochrome P450 family 2 subfamily C member 19; CYP2C9, cytochrome P450 family 2 subfamily C member 9; CYP2C8, cytochrome P450 family 2 subfamily C member 8; CYP2E1, cytochrome P450 family 2 subfamily E member 1; KCNJ11, potassium inwardly rectifying channel subfamily J member 11; DRD2, dopamine receptor D2; SLCO1B1, solute carrier organic anion transporter family member 1B1; VDR, vitamin D3 receptor; CYP1A2, cytochrome P450 family 1 subfamily A member 2; SULT1A1, sulfotransferase family 1A member 1; ACE, angiotensin-converting enzyme; CYP4F2, cytochrome P450 family 4 subfamily F member 2; CYP2A6, cytochrome P450 family 2 subfamily A member 6; SLC19A1, solute carrier family 19 member 1; CYP2D6, cytochrome P450 family 2 subfamily D member 6.

### HapMap Genotype Data

The genotype data of individuals from 26 populations was obtained from the International HapMap Project Internet website (www.genome.gov/10001688/international-hapmap-project/). The 26 populations were as follows: 1) Chinese Dai in Xishuangbanna, China (CDX); 2) Han Chinese in Beijing, China (CHB); 3) Southern Han Chinese, China (CHS); 4) Japanese in Tokyo, Japan (JPT); 5) Kinh in Ho Chi Minh City, Vietnam (KHV); 6) African Caribbeans in Barbados (ACB); 7) African Ancestry in Southwest USA (ASW); 8) Esan in Nigeria (ESN); 9) Gambian in Western Divisions, The Gambia (GWD); 10) Luhya in Webuye, Kenya (LWK); 11) Mende in Sierra Leone (MSL); 12) Yoruba in Ibadan, Nigeria (YRI); 13) Colombian in Medellin, Colombia (CLM); 14) Mexican Ancestry in Los Angeles, Colombia (MXL); 15) Peruvian in Lima, Peru (PEL); 16) Puerto Rican in Puerto Rico (PUR); 17) Utah residents with Northern and Western European ancestry (CEU); 18) Finnish in Finland (FIN); 19) British in England and Scotland (GBR); 20) Iberian populations in Spain (IBS); 21) Toscani in Italy (TSI); 22) Bengali in Bangladesh (BEB); 23) Gujarati Indian in Houston, Texas (GIH); 24) Indian Telugu in the UK (ITU); 25) Punjabi in Lahore, Pakistan (PJL); 26) Sri Lankan Tamil in the UK (STU).

### Statistical Analyses

An exact test was used to test the frequency validity of each VIP variant by assessing the departure from the Hardy–Weinberg equilibrium. The comparison of genotype frequencies between the Blang population and 26 other populations was conducted using the χ2 test. SPSS 17.0 (Armonk, NY, USA) and Excel™ (Microsoft, Redmond, WA, USA) were used to analyze the distribution of genotypes and MAFs. The Bonferroni correction was applied to p < 0.05 (two-sided).

### Results

The VIPs corresponding to 55 loci could be classified into 15 gene superfamilies (Table 1): cytochrome P450 superfamily; dihydropyrimidine dehydrogenase; prostaglandin-endoperoxide synthase; calcium voltage-gated channel; ryanodine receptor; alcohol dehydrogenase; potassium voltage-gated ion channel; N-acetyltransferase; angiotensin I-converting enzyme; potassium inwardly rectifying channel; G-protein coupled receptor family; solute carrier organic anion transporter family; nuclear receptor family; sulfotransferase family; solute carrier family. The sequence function of these 55 loci was classified mainly into eight types: intron variant; upstream transcript variant; downstream transcript variant; coding sequence variant; missense; 3′ untranslated region (UTR) variant; non-coding transcript variant; 5′ UTR variant.

All selected loci met the Hardy–Weinberg equilibrium (p>0.05) with a call rate >99.9%. Among the 26 populations studied, GWD, YRI, GIH, ESN, MSL, TSI, PJL, ACB, FIN and IBS were the top-10 populations which showed significant differences compared with the Blang population (>35 loci) (Table 2). Conversely, CHB, JPT, CDX, CHS and KHV populations showed the most similarities with the Blang population (genotype distribution <20 loci). The genotype distribution of 27–34 loci in the Blang population showed a significant difference from that of 11 other populations, (LWK, CEU, ITU, STU, PUR, CLM, GBR, ASW, BEB, MXL and PEL). On the one hand, among 26 populations, the GWD population had the greatest number of significantly different loci after Bonferroni correction compared with that in the Blang population, indicating that GWD was the most different population from the Blang population. This significant difference may have resulted from a difference in the genetic background between them. On the other hand, the KHV population showed the least number of different loci after Bonferroni correction. The relatively greater number of similar loci was probably caused by a similar geographic location (East Asian) between them. The distribution of genotypes and allele frequencies of the seven significantly different SNPs are shown in Supplemental Table 2 and Supplemental Figures 1–7.Table 2The Genotype Distribution Difference Between Blang and 26 Other Populations After Bonferroni’s Multiple AdjustmentsSNP IDGenesEASAFRAMREURSASCDXCHBCHSJPTKHVACBASWESNGWDLWKMSLYRICLMMXLPELPURCEUFINGBRIBSTSIBEBGIHITUPJLSTUrs11572325CYP2J2–3.55E-01---3.09E-061.55E-056.55E-061.61E-056.00E-052.38E-041.15E-051.21E-02--2.65E-052.04E-021.90E-041.08E-013.40E-022.56E-02--3.48E-01--rs10889160CYP2J24.55E-013.81E-024.33E-021.75E-043.96E-017.04E-209.00E-111.21E-221.60E-166.80E-176.87E-203.92E-223.38E-012.39E-011.87E-011.21E-044.49E-011.68E-052.10E-012.35E-021.40E-016.40E-012.24E-014.11E-019.66E-017.87E-02rs890293CYP2J2-3.91E-02---5.59E-074.63E-108.77E-145.59E-114.02E-101.15E-151.27E-10---1.68E-03-4.17E-041.96E-02----3.37E-03--rs1760217DPYD2.92E-017.58E-011.74E-013.49E-025.72E-011.43E-023.45E-012.16E-014.90E-041.62E-021.27E-011.92E-034.94E-039.26E-014.35E-021.27E-044.24E-017.04E-041.34E-033.79E-039.42E-033.69E-022.27E-014.62E-073.45E-042.02E-06rs1801159DPYD7.64E-018.28E-013.94E-027.46E-016.49E-012.42E-033.35E-029.31E-023.52E-075.19E-011.76E-065.36E-031.05E-015.22E-011.17E-052.04E-011.17E-023.07E-035.15E-023.91E-013.71E-016.95E-052.89E-053.85E-091.46E-051.55E-07rs1801265DPYD1.47E-013.55E-014.06E-011.53E-015.94E-019.83E-148.87E-184.50E-184.94E-211.58E-211.19E-133.18E-194.82E-052.07E-048.95E-022.63E-061.31E-018.44E-091.93E-024.50E-044.15E-048.23E-032.20E-111.36E-092.54E-083.94E-03rs5275PTGS26.18E-016.55E-022.65E-019.03E-017.74E-017.36E-181.59E-114.77E-222.46E-174.06E-181.26E-203.28E-222.43E-041.10E-023.31E-045.83E-027.08E-042.97E-011.72E-012.95E-021.07E-012.93E-047.46E-044.78E-032.85E-081.27E-04rs20417PTGS2-----5.38E-291.81E-261.16E-373.26E-272.82E-243.93E-365.95E-332.31E-193.44E-196.57E-171.57E-185.14E-152.82E-092.64E-122.26E-132.04E-161.85E-151.81E-151.45E-141.54E-198.50E-18rs12139527CACNA1S2.69E-012.51E-023.60E-011.39E-01-1.30E-235.47E-209.80E-321.53E-345.79E-277.64E-311.14E-30---1.04E-021.17E-013.46E-015.18E-023.79E-011.81E-01-2.70E-04-3.54E-023.90E-02rs13374149CACNA1S-----1.24E-111.10E-083.05E-165.43E-175.43E-101.51E-178.94E-202.59E-01----2.52E-01--1.40E-01----3.74E-01rs3850625CACNA1S------------1.35E-06-7.60E-06-1.86E-041.93E-127.75E-098.41E-055.43E-062.13E-093.72E-211.14E-132.03E-124.44E-10rs2306238RYR22.03E-014.71E-013.91E-013.80E-013.24E-011.25E-067.62E-042.71E-093.87E-082.38E-072.35E-081.17E-073.00E-015.77E-031.01E-037.37E-024.91E-018.12E-019.21E-018.69E-019.45E-015.86E-016.13E-019.64E-035.24E-022.10E-02rs2231142ABCG23.83E-017.28E-044.98E-021.01E-042.00E-056.82E-082.69E-022.34E-091.24E-082.34E-091.78E-064.56E-102.06E-017.26E-017.37E-018.76E-021.87E-012.69E-024.91E-011.97E-033.84E-043.01E-011.82E-037.00E-026.51E-021.89E-02rs2231137ABCG24.42E-012.58E-034.35E-023.22E-104.68E-011.08E-201.73E-177.65E-212.84E-245.61E-146.23E-152.16E-213.07E-111.47E-059.00E-011.48E-117.58E-232.46E-166.64E-228.18E-221.68E-181.70E-059.43E-127.47E-167.52E-153.18E-08rs698ADH1C1.89E-012.21E-021.83E-011.94E-011.82E-018.83E-018.77E-011.89E-011.70E-011.92E-012.72E-019.91E-022.15E-036.10E-057.92E-022.88E-111.86E-181.08E-181.12E-131.17E-064.24E-075.59E-022.48E-058.45E-043.94E-084.51E-09rs776746CYP3A53.65E-131.47E-111.27E-113.71E-141.23E-134.31E-312.64E-254.08E-382.94E-341.06E-357.18E-346.79E-393.35E-072.33E-091.47E-044.23E-086.36E-041.68E-037.13E-043.69E-042.60E-041.58E-145.94E-131.05E-131.75E-122.07E-13rs2242480CYP3A41.10E-044.63E-062.16E-061.40E-046.99E-032.26E-141.87E-087.16E-269.51E-192.91E-265.51E-231.17E-209.71E-043.29E-019.02E-037.93E-032.45E-212.74E-197.98E-193.97E-167.62E-191.13E-012.75E-027.51E-025.52E-011.51E-01rs1805123KCNH2-8.42E-62----------9.01E-381.58E-38-2.64E-372.71E-328.05E-418.46E-373.53E-361.52E-337.44E-304.15E-392.84E-333.31E-362.65E-35rs4646244NAT24.76E-043.80E-049.25E-025.50E-033.62E-024.31E-026.84E-022.54E-025.48E-043.84E-021.40E-026.42E-041.59E-028.10E-068.44E-104.94E-031.43E-021.21E-013.97E-047.38E-031.40E-011.91E-021.39E-019.23E-031.17E-012.23E-02rs4271002NAT21.60E-027.26E-024.71E-022.33E-024.71E-023.71E-066.54E-041.92E-121.61E-124.58E-071.19E-056.13E-081.59E-051.23E-017.97E-035.57E-022.57E-084.56E-057.76E-034.21E-044.40E-031.75E-013.57E-041.00E-054.48E-044.01E-04rs1041983NAT21.48E-018.16E-028.87E-011.40E-029.33E-024.96E-018.62E-019.51E-021.59E-016.87E-018.27E-024.87E-012.81E-039.50E-043.42E-064.03E-042.61E-046.17E-043.32E-044.74E-035.44E-041.08E-017.79E-012.18E-014.49E-015.39E-01rs1801280NAT2----3.41E-011.58E-121.14E-121.21E-123.42E-177.64E-201.03E-095.34E-101.20E-193.04E-173.71E-132.07E-201.67E-231.31E-252.01E-251.96E-281.39E-241.80E-174.68E-175.09E-185.22E-231.73E-13rs1799929NAT27.43E-012.56E-014.87E-017.99E-026.50E-011.20E-081.79E-082.31E-061.02E-121.01E-151.33E-051.85E-043.82E-173.51E-152.39E-111.51E-165.68E-229.29E-231.43E-227.70E-272.95E-234.47E-152.36E-136.28E-151.19E-182.19E-11rs1799930NAT21.52E-012.12E-024.15E-012.89E-012.79E-015.62E-015.36E-015.15E-015.15E-036.59E-019.95E-022.16E-022.06E-016.83E-041.31E-073.90E-022.42E-013.02E-011.55E-017.65E-018.14E-016.98E-019.10E-025.47E-013.10E-011.36E-02rs1208NAT27.43E-012.56E-016.09E-017.99E-028.57E-011.13E-172.25E-144.00E-192.38E-242.31E-244.87E-163.76E-196.68E-195.81E-205.69E-124.96E-191.05E-202.87E-221.08E-223.74E-277.83E-241.28E-188.11E-166.73E-172.31E-223.00E-14rs1799931NAT23.96E-031.54E-015.44E-015.65E-026.41E-011.16E-048.47E-031.04E-063.81E-072.94E-071.33E-035.10E-059.61E-034.38E-014.63E-023.08E-027.69E-087.48E-051.18E-052.32E-059.66E-071.38E-011.12E-032.51E-032.75E-032.27E-02rs1495741NAT24.00E-013.54E-037.00E-011.13E-034.51E-013.38E-013.32E-028.69E-016.81E-012.19E-026.57E-017.58E-013.15E-043.73E-026.55E-013.32E-039.05E-054.08E-061.81E-063.86E-095.79E-051.41E-041.40E-077.46E-063.73E-108.61E-09rs2115819ALOX51.34E-036.16E-045.95E-021.20E-029.48E-022.81E-361.34E-237.95E-368.90E-401.18E-311.67E-301.21E-381.93E-154.49E-101.08E-065.41E-143.73E-232.13E-182.35E-173.96E-194.46E-204.25E-153.21E-226.81E-204.10E-153.43E-14rs12248560CYP2C19-----3.08E-171.17E-091.83E-141.92E-146.70E-094.56E-155.78E-142.31E-057.80E-04-3.52E-092.98E-132.69E-128.44E-141.78E-121.36E-12-6.49E-064.16E-063.07E-067.37E-06rs4244285CYP2C192.12E-014.41E-046.33E-055.24E-038.39E-022.47E-012.93E-019.06E-017.14E-029.15E-017.45E-013.02E-011.28E-021.41E-011.07E-048.15E-029.76E-023.79E-011.98E-017.60E-022.40E-036.66E-031.52E-035.34E-053.91E-046.82E-07rs1057910CYP2C97.73E-016.54E-017.70E-015.56E-017.35E-011.16E-016.06E-013.46E-022.17E-023.46E-025.53E-022.56E-021.15E-017.85E-013.42E-015.35E-018.88E-022.35E-015.32E-027.77E-033.01E-021.04E-042.08E-052.44E-035.39E-031.10E-03rs11572103CYP2C8-----1.89E-166.65E-108.36E-151.07E-183.96E-095.73E-121.88E-14--------------rs7909236CYP2C82.33E-029.92E-025.29E-014.06E-017.66E-015.84E-013.41E-013.73E-031.73E-037.89E-028.09E-032.28E-032.01E-134.96E-121.43E-172.60E-054.80E-124.84E-117.46E-081.66E-068.49E-062.97E-065.61E-102.88E-083.10E-061.45E-03rs17110453CYP2C87.88E-017.43E-039.72E-041.22E-044.29E-018.63E-111.13E-074.23E-117.52E-142.06E-126.08E-111.15E-125.08E-041.62E-021.88E-063.27E-032.05E-047.93E-011.58E-036.95E-017.96E-042.62E-022.34E-021.80E-022.56E-011.44E-03rs3813867CYP2E12.10E-011.21E-049.66E-032.47E-022.85E-035.71E-024.31E-013.91E-013.31E-011.23E-031.32E-014.68E-018.49E-014.02E-012.24E-011.23E-011.15E-015.41E-032.83E-035.36E-044.01E-022.37E-043.39E-053.80E-057.53E-051.09E-05rs2031920CYP2E18.14E-025.81E-046.33E-034.16E-032.16E-045.35E-051.61E-023.99E-051.02E-053.99E-051.58E-041.66E-058.39E-025.48E-025.67E-021.32E-011.63E-014.00E-021.38E-023.77E-038.79E-021.57E-033.24E-043.55E-046.19E-041.10E-04rs6413432CYP2E16.35E-261.06E-221.12E-211.81E-223.82E-26--2.62E-071.40E-08-3.64E-051.64E-087.89E-162.51E-142.09E-148.49E-157.98E-122.72E-102.28E-068.94E-071.62E-095.68E-187.09E-248.52E-182.80E-132.38E-18rs2070676CYP2E12.05E-041.66E-021.76E-012.81E-024.85E-035.15E-271.93E-152.26E-272.46E-295.40E-331.17E-272.93E-274.86E-029.13E-015.97E-011.77E-043.06E-026.61E-021.49E-012.85E-014.58E-031.50E-012.79E-011.94E-011.53E-014.12E-01rs5219KCNJ111.47E-064.47E-168.46E-201.57E-131.13E-14-2.08E-02-----1.04E-041.72E-162.61E-123.68E-101.21E-182.33E-237.54E-091.51E-173.22E-101.18E-173.00E-194.27E-192.69E-191.72E-11rs1801028DRD2-7.96E-039.04E-03-------------7.13E-02-----7.46E-13--4.77E-08rs2306283SLCO1B13.04E-015.03E-021.29E-012.28E-073.97E-025.66E-021.85E-022.80E-013.26E-014.87E-012.84E-013.02E-011.43E-171.12E-211.91E-181.43E-141.32E-232.11E-214.11E-268.88E-257.30E-281.32E-111.75E-131.96E-101.62E-192.33E-13rs4516035VDR------1.96E-05-----2.59E-201.29E-161.67E-106.90E-273.02E-275.07E-418.95E-315.06E-284.06E-333.49E-128.41E-135.15E-122.47E-171.47E-14rs762551CYP1A23.86E-011.32E-044.37E-044.15E-064.03E-014.84E-052.59E-032.04E-103.06E-076.14E-135.59E-071.95E-091.18E-019.46E-034.26E-027.11E-021.30E-011.15E-021.01E-011.87E-043.26E-056.78E-072.63E-103.58E-095.35E-094.15E-12rs2472304CYP1A26.74E-013.04E-019.05E-012.39E-015.01E-013.93E-024.55E-017.69E-083.81E-071.04E-067.77E-076.90E-081.20E-089.89E-034.64E-017.02E-153.98E-277.17E-204.64E-261.52E-232.03E-177.03E-014.30E-015.99E-012.99E-022.52E-01rs750155SULT1A11.53E-131.10E-216.16E-131.52E-242.79E-162.51E-191.42E-212.46E-281.38E-335.49E-245.54E-254.44E-271.31E-138.50E-133.16E-051.52E-201.43E-184.47E-145.33E-186.25E-201.09E-201.75E-325.10E-256.27E-311.42E-271.49E-38rs1800764ACE2.58E-082.67E-061.89E-064.61E-131.81E-082.40E-451.17E-358.71E-532.63E-589.85E-487.85E-568.85E-581.72E-131.78E-062.44E-035.15E-141.09E-144.56E-133.25E-129.62E-124.06E-172.53E-092.56E-136.97E-111.27E-099.22E-09rs4291ACE2.66E-291.08E-344.49E-302.01E-254.19E-341.14E-272.40E-282.30E-296.11E-254.21E-346.39E-252.97E-342.72E-281.77E-334.86E-417.35E-282.28E-291.16E-231.08E-304.98E-274.26E-303.76E-286.58E-272.23E-345.37E-298.21E-28rs4267385ACE3.22E-012.05E-019.11E-018.29E-023.75E-012.75E-291.85E-202.95E-316.81E-382.22E-391.33E-361.96E-353.38E-092.86E-053.72E-015.61E-111.90E-128.53E-121.67E-143.12E-151.67E-244.82E-015.48E-055.94E-022.46E-051.62E-03rs2108622CYP4F27.51E-021.74E-024.11E-023.01E-025.72E-026.59E-012.74E-013.06E-032.39E-024.39E-014.86E-019.56E-032.88E-041.54E-032.06E-017.88E-058.67E-032.43E-024.99E-042.80E-091.41E-066.00E-116.61E-132.18E-106.11E-109.78E-12rs3093105CYP4F2---2.56E-59-1.15E-371.86E-305.69E-332.44E-417.01E-382.46E-363.69E-355.66E-441.39E-421.35E-551.54E-441.21E-441.84E-49-7.51E-351.34E-396.52E-451.47E-461.85E-423.34E-422.80E-46rs8192726CYP2A65.87E-011.07E-011.88E-024.70E-022.40E-022.17E-045.86E-021.71E-027.30E-059.81E-031.06E-035.91E-036.33E-066.22E-048.59E-057.64E-064.51E-051.60E-011.13E-052.79E-052.94E-041.23E-012.95E-012.20E-022.34E-012.20E-02rs1051298SLC19A18.53E-185.07E-191.64E-121.63E-181.95E-169.09E-193.59E-165.15E-152.32E-231.78E-192.68E-174.92E-177.46E-184.53E-251.55E-241.90E-172.01E-181.54E-172.72E-235.20E-221.00E-184.21E-161.89E-184.75E-173.15E-163.93E-20rs1051296SLC19A15.68E-071.09E-095.76E-044.79E-082.02E-069.10E-062.37E-083.68E-092.95E-091.55E-101.18E-068.84E-089.92E-091.92E-152.65E-153.14E-091.12E-108.19E-098.43E-141.17E-101.40E-108.78E-065.30E-074.94E-097.81E-098.11E-09rs1131596SLC19A13.38E-091.29E-092.58E-059.40E-114.20E-073.13E-202.89E-083.62E-107.00E-274.69E-211.13E-192.40E-181.43E-091.98E-141.99E-121.82E-095.01E-061.35E-085.87E-151.13E-121.30E-107.51E-081.48E-094.33E-105.09E-123.51E-11rs1065852CYP2D64.67E-334.64E-293.15E-342.68E-312.43E-282.31E-451.33E-427.97E-531.41E-51-9.48E-444.64E-515.66E-441.58E-399.87E-537.74E-472.41E-341.99E-452.51E-383.61E-431.34E-394.53E-391.68E-431.02E-411.23E-482.80E-46Note: “-” means no data. Bold values means after adjustment p < 0.05/(55*26) the locus has statistical significance.Abbreviations: EAS, East Asian; SAS, South Asian; EUR, European; AFR, African; AMR, American; CDX, Chinese Dai in Xishuangbanna, China; CHB, Han Chinese in Beijing, China; CHS, Southern Han Chinese, China; JPT, Japanese in Tokyo, Japan; KHV, Kinh in Ho Chi Minh City; Vietnam; BEB, Bengali in Bangladesh; GIH, Gujarati Indian in Houston, Texas; ITU, Indian Telugu in the UK; PJL, Punjabi in Lahore, Pakistan; STU, Sri Lankan Tamil in the UK; CEU, Western European ancestry; FIN, Finnish in Finland; GBR, British in England and Scotland; IBS, Iberian populations in Spain; TSI, Toscani in Italy; ACB, African Caribbeans in Barbados; ASW, African Ancestry in Southwest USA; ESN, Esan in Nigeria; GWD, Gambian in Western Divisions, The Gambia; LWK, Luhya in Webuye, Kenya; MSL, Mende in Sierra Leone; YRI, Yoruba in Ibadan, Nigeria; CLM, Colombian in Medellin, Colombia; MXL, Mexican Ancestry in Los Angeles, Colombia; PEL, Peruvian in Lima, Peru; PUR, Puerto Rican in Puerto Rico.

The Genotype Distribution Difference Between Blang and 26 Other Populations After Bonferroni’s Multiple Adjustments

Note: “-” means no data. Bold values means after adjustment p < 0.05/(55*26) the locus has statistical significance.

Abbreviations: EAS, East Asian; SAS, South Asian; EUR, European; AFR, African; AMR, American; CDX, Chinese Dai in Xishuangbanna, China; CHB, Han Chinese in Beijing, China; CHS, Southern Han Chinese, China; JPT, Japanese in Tokyo, Japan; KHV, Kinh in Ho Chi Minh City; Vietnam; BEB, Bengali in Bangladesh; GIH, Gujarati Indian in Houston, Texas; ITU, Indian Telugu in the UK; PJL, Punjabi in Lahore, Pakistan; STU, Sri Lankan Tamil in the UK; CEU, Western European ancestry; FIN, Finnish in Finland; GBR, British in England and Scotland; IBS, Iberian populations in Spain; TSI, Toscani in Italy; ACB, African Caribbeans in Barbados; ASW, African Ancestry in Southwest USA; ESN, Esan in Nigeria; GWD, Gambian in Western Divisions, The Gambia; LWK, Luhya in Webuye, Kenya; MSL, Mende in Sierra Leone; YRI, Yoruba in Ibadan, Nigeria; CLM, Colombian in Medellin, Colombia; MXL, Mexican Ancestry in Los Angeles, Colombia; PEL, Peruvian in Lima, Peru; PUR, Puerto Rican in Puerto Rico.

Among 55 loci, after Bonferroni correction between the Blang population and 26 other populations, the distribution of genotype frequencies was significantly different in five loci: rs750155 of sulfotransferase family 1A member gene (SULT1A1), rs4291 of ACE, rs1051298, rs1131596 and rs1051296 of SLC19A1. Besides, the genotype distribution of rs1800764 (ACE) and rs1065852 (CYP2D6) was different in all populations except for PEL and LWK, respectively. Conversely, the genotype distribution of rs1801028 of the dopamine receptor D2 gene (DRD2) was significantly different only in the GIH population compared with that in the Blang population. In addition to the eight loci mentioned above, the genotype distribution of the remaining loci in the Blang population also showed a significant difference compared with that in the other 26 populations, but to different degrees.

The MAF distribution of seven significantly different SNPs is shown in Table 3 and Figure 1. The MAFs of rs1065852 (CYP2D6) and rs750155 (SULT1A1) showed the greatest similarities among SAS, EUR, AFR and AMR populations, but also showed the largest fluctuation between the Blang population and SAS, EUR, AFR and AMR populations. The MAFs of rs1800764 (ACE) and rs1131596 (SLC19A1) among the seven subpopulations of AFR showed distinct differences when compared with those of the Blang population. However, the MAFs of rs4291 (ACE), rs1051298 (SLC19A1), and rs1051296 (SLC19A1) showed relatively less fluctuation between the Blang population and the other 26 populations. Besides, the MAFs of rs1800764 (ACE) and rs750155 (SULT1A1) in the Blang population were close to those of the PEL population, even though most of other populations showed distinct differences on it. To better observe the phenotypes of these seven significantly different SNPs in the Blang population, their clinical and variant annotations were retrieved from PharmGKB (Supplemental Table 3 and Supplemental Table 4, respectively).Table 3The Minor Allele Frequency Distribution of Seven SNPs Among 27 PopulationsGeographic RegionPopulationrs750155-Crs1800764-Crs4291-Ars1051298-Grs1051296-Ars1131596-Ars1065852-AEASBlang0.2330.1830.5030.4580.4350.4880.500CDX0.5000.3660.6400.4300.4250.5380.629CHB0.6020.3110.6990.5000.4900.4760.602CHS0.4860.3330.6760.4240.4240.5330.614JPT0.6390.4470.5630.4380.4380.5430.361KHV0.5350.3480.6520.3890.3990.5400.662SASBEB0.7500.3900.6450.4360.4830.3900.256GIH0.6360.4510.5920.4080.4710.3930.150ITU0.7160.4070.6370.4560.5390.3970.172PJL0.6820.3910.6510.4900.5160.4220.104STU0.7940.3770.6670.4360.4950.4310.152EURCEU0.5660.4700.6260.5960.5960.4290.242FIN0.5050.4490.6410.5400.5400.4550.146GBR0.5600.4340.6590.5930.5930.4010.247IBS0.5790.4250.6360.4720.4770.5230.173TSI0.5890.5050.5840.5610.5650.4490.206AFRACB0.5780.8540.6410.3700.4270.7240.146ASW0.6310.7870.7050.4510.5330.6230.156ESN0.6820.9190.6770.4850.5510.6460.091GWD0.7350.9420.5800.3630.5090.7740.115LWK0.6360.8740.7020.4600.4750.7630.035MSL0.6590.9530.6290.4120.4820.7410.165YRI0.6670.9440.6160.4350.5140.7220.106AMRCLM0.4890.4570.6540.5270.5270.4950.186MXL0.4920.3130.7500.6410.6410.3520.148PEL0.1650.2240.8240.5940.6000.3710.071PUR0.5820.4620.6880.5380.5430.4380.178Note: Loci with minor allele frequency (MAF) > 0.05 could be considered into this study.Abbreviations: EAS, East Asian; SAS, South Asian; EUR, European; AFR, African; AMR, American; CDX, Chinese Dai in Xishuangbanna, China; CHB, Han Chinese in Beijing, China; CHS, Southern Han Chinese, China; JPT, Japanese in Tokyo, Japan; KHV, Kinh in Ho Chi Minh City, Vietnam; BEB, Bengali in Bangladesh; GIH, Gujarati Indian in Houston, Texas; ITU, Indian Telugu in the UK; PJL, Punjabi in Lahore, Pakistan; STU, Sri Lankan Tamil in the UK; CEU, Western European ancestry; FIN, Finnish in Finland; GBR, British in England and Scotland; IBS, Iberian populations in Spain; TSI, Toscani in Italy; ACB, African Caribbeans in Barbados; ASW, African Ancestry in Southwest USA; ESN, Esan in Nigeria; GWD, Gambian in Western Divisions, The Gambia; LWK, Luhya in Webuye, Kenya; MSL, Mende in Sierra Leone; YRI, Yoruba in Ibadan, Nigeria; CLM, Colombian in Medellin, Colombia; MXL, Mexican Ancestry in Los Angeles, Colombia; PEL, Peruvian in Lima, Peru; PUR, Puerto Rican in Puerto Rico.
Figure 1The minor allele frequency (MAF) distribution of seven significantly different SNPs between Blang population and other 26 populations. The value of the Y axis represents the MAF.

The Minor Allele Frequency Distribution of Seven SNPs Among 27 Populations

Note: Loci with minor allele frequency (MAF) > 0.05 could be considered into this study.

Abbreviations: EAS, East Asian; SAS, South Asian; EUR, European; AFR, African; AMR, American; CDX, Chinese Dai in Xishuangbanna, China; CHB, Han Chinese in Beijing, China; CHS, Southern Han Chinese, China; JPT, Japanese in Tokyo, Japan; KHV, Kinh in Ho Chi Minh City, Vietnam; BEB, Bengali in Bangladesh; GIH, Gujarati Indian in Houston, Texas; ITU, Indian Telugu in the UK; PJL, Punjabi in Lahore, Pakistan; STU, Sri Lankan Tamil in the UK; CEU, Western European ancestry; FIN, Finnish in Finland; GBR, British in England and Scotland; IBS, Iberian populations in Spain; TSI, Toscani in Italy; ACB, African Caribbeans in Barbados; ASW, African Ancestry in Southwest USA; ESN, Esan in Nigeria; GWD, Gambian in Western Divisions, The Gambia; LWK, Luhya in Webuye, Kenya; MSL, Mende in Sierra Leone; YRI, Yoruba in Ibadan, Nigeria; CLM, Colombian in Medellin, Colombia; MXL, Mexican Ancestry in Los Angeles, Colombia; PEL, Peruvian in Lima, Peru; PUR, Puerto Rican in Puerto Rico.

The minor allele frequency (MAF) distribution of seven significantly different SNPs between Blang population and other 26 populations. The value of the Y axis represents the MAF.

### Basic Information of VIP Loci

The VIPs corresponding to 55 loci could be classified into 15 gene superfamilies (Table 1): cytochrome P450 superfamily; dihydropyrimidine dehydrogenase; prostaglandin-endoperoxide synthase; calcium voltage-gated channel; ryanodine receptor; alcohol dehydrogenase; potassium voltage-gated ion channel; N-acetyltransferase; angiotensin I-converting enzyme; potassium inwardly rectifying channel; G-protein coupled receptor family; solute carrier organic anion transporter family; nuclear receptor family; sulfotransferase family; solute carrier family. The sequence function of these 55 loci was classified mainly into eight types: intron variant; upstream transcript variant; downstream transcript variant; coding sequence variant; missense; 3′ untranslated region (UTR) variant; non-coding transcript variant; 5′ UTR variant.

### Relationship Between the Blang Population and 26 Other Populations

All selected loci met the Hardy–Weinberg equilibrium (p>0.05) with a call rate >99.9%. Among the 26 populations studied, GWD, YRI, GIH, ESN, MSL, TSI, PJL, ACB, FIN and IBS were the top-10 populations which showed significant differences compared with the Blang population (>35 loci) (Table 2). Conversely, CHB, JPT, CDX, CHS and KHV populations showed the most similarities with the Blang population (genotype distribution <20 loci). The genotype distribution of 27–34 loci in the Blang population showed a significant difference from that of 11 other populations, (LWK, CEU, ITU, STU, PUR, CLM, GBR, ASW, BEB, MXL and PEL). On the one hand, among 26 populations, the GWD population had the greatest number of significantly different loci after Bonferroni correction compared with that in the Blang population, indicating that GWD was the most different population from the Blang population. This significant difference may have resulted from a difference in the genetic background between them. On the other hand, the KHV population showed the least number of different loci after Bonferroni correction. The relatively greater number of similar loci was probably caused by a similar geographic location (East Asian) between them. The distribution of genotypes and allele frequencies of the seven significantly different SNPs are shown in Supplemental Table 2 and Supplemental Figures 1–7.Table 2The Genotype Distribution Difference Between Blang and 26 Other Populations After Bonferroni’s Multiple AdjustmentsSNP IDGenesEASAFRAMREURSASCDXCHBCHSJPTKHVACBASWESNGWDLWKMSLYRICLMMXLPELPURCEUFINGBRIBSTSIBEBGIHITUPJLSTUrs11572325CYP2J2–3.55E-01---3.09E-061.55E-056.55E-061.61E-056.00E-052.38E-041.15E-051.21E-02--2.65E-052.04E-021.90E-041.08E-013.40E-022.56E-02--3.48E-01--rs10889160CYP2J24.55E-013.81E-024.33E-021.75E-043.96E-017.04E-209.00E-111.21E-221.60E-166.80E-176.87E-203.92E-223.38E-012.39E-011.87E-011.21E-044.49E-011.68E-052.10E-012.35E-021.40E-016.40E-012.24E-014.11E-019.66E-017.87E-02rs890293CYP2J2-3.91E-02---5.59E-074.63E-108.77E-145.59E-114.02E-101.15E-151.27E-10---1.68E-03-4.17E-041.96E-02----3.37E-03--rs1760217DPYD2.92E-017.58E-011.74E-013.49E-025.72E-011.43E-023.45E-012.16E-014.90E-041.62E-021.27E-011.92E-034.94E-039.26E-014.35E-021.27E-044.24E-017.04E-041.34E-033.79E-039.42E-033.69E-022.27E-014.62E-073.45E-042.02E-06rs1801159DPYD7.64E-018.28E-013.94E-027.46E-016.49E-012.42E-033.35E-029.31E-023.52E-075.19E-011.76E-065.36E-031.05E-015.22E-011.17E-052.04E-011.17E-023.07E-035.15E-023.91E-013.71E-016.95E-052.89E-053.85E-091.46E-051.55E-07rs1801265DPYD1.47E-013.55E-014.06E-011.53E-015.94E-019.83E-148.87E-184.50E-184.94E-211.58E-211.19E-133.18E-194.82E-052.07E-048.95E-022.63E-061.31E-018.44E-091.93E-024.50E-044.15E-048.23E-032.20E-111.36E-092.54E-083.94E-03rs5275PTGS26.18E-016.55E-022.65E-019.03E-017.74E-017.36E-181.59E-114.77E-222.46E-174.06E-181.26E-203.28E-222.43E-041.10E-023.31E-045.83E-027.08E-042.97E-011.72E-012.95E-021.07E-012.93E-047.46E-044.78E-032.85E-081.27E-04rs20417PTGS2-----5.38E-291.81E-261.16E-373.26E-272.82E-243.93E-365.95E-332.31E-193.44E-196.57E-171.57E-185.14E-152.82E-092.64E-122.26E-132.04E-161.85E-151.81E-151.45E-141.54E-198.50E-18rs12139527CACNA1S2.69E-012.51E-023.60E-011.39E-01-1.30E-235.47E-209.80E-321.53E-345.79E-277.64E-311.14E-30---1.04E-021.17E-013.46E-015.18E-023.79E-011.81E-01-2.70E-04-3.54E-023.90E-02rs13374149CACNA1S-----1.24E-111.10E-083.05E-165.43E-175.43E-101.51E-178.94E-202.59E-01----2.52E-01--1.40E-01----3.74E-01rs3850625CACNA1S------------1.35E-06-7.60E-06-1.86E-041.93E-127.75E-098.41E-055.43E-062.13E-093.72E-211.14E-132.03E-124.44E-10rs2306238RYR22.03E-014.71E-013.91E-013.80E-013.24E-011.25E-067.62E-042.71E-093.87E-082.38E-072.35E-081.17E-073.00E-015.77E-031.01E-037.37E-024.91E-018.12E-019.21E-018.69E-019.45E-015.86E-016.13E-019.64E-035.24E-022.10E-02rs2231142ABCG23.83E-017.28E-044.98E-021.01E-042.00E-056.82E-082.69E-022.34E-091.24E-082.34E-091.78E-064.56E-102.06E-017.26E-017.37E-018.76E-021.87E-012.69E-024.91E-011.97E-033.84E-043.01E-011.82E-037.00E-026.51E-021.89E-02rs2231137ABCG24.42E-012.58E-034.35E-023.22E-104.68E-011.08E-201.73E-177.65E-212.84E-245.61E-146.23E-152.16E-213.07E-111.47E-059.00E-011.48E-117.58E-232.46E-166.64E-228.18E-221.68E-181.70E-059.43E-127.47E-167.52E-153.18E-08rs698ADH1C1.89E-012.21E-021.83E-011.94E-011.82E-018.83E-018.77E-011.89E-011.70E-011.92E-012.72E-019.91E-022.15E-036.10E-057.92E-022.88E-111.86E-181.08E-181.12E-131.17E-064.24E-075.59E-022.48E-058.45E-043.94E-084.51E-09rs776746CYP3A53.65E-131.47E-111.27E-113.71E-141.23E-134.31E-312.64E-254.08E-382.94E-341.06E-357.18E-346.79E-393.35E-072.33E-091.47E-044.23E-086.36E-041.68E-037.13E-043.69E-042.60E-041.58E-145.94E-131.05E-131.75E-122.07E-13rs2242480CYP3A41.10E-044.63E-062.16E-061.40E-046.99E-032.26E-141.87E-087.16E-269.51E-192.91E-265.51E-231.17E-209.71E-043.29E-019.02E-037.93E-032.45E-212.74E-197.98E-193.97E-167.62E-191.13E-012.75E-027.51E-025.52E-011.51E-01rs1805123KCNH2-8.42E-62----------9.01E-381.58E-38-2.64E-372.71E-328.05E-418.46E-373.53E-361.52E-337.44E-304.15E-392.84E-333.31E-362.65E-35rs4646244NAT24.76E-043.80E-049.25E-025.50E-033.62E-024.31E-026.84E-022.54E-025.48E-043.84E-021.40E-026.42E-041.59E-028.10E-068.44E-104.94E-031.43E-021.21E-013.97E-047.38E-031.40E-011.91E-021.39E-019.23E-031.17E-012.23E-02rs4271002NAT21.60E-027.26E-024.71E-022.33E-024.71E-023.71E-066.54E-041.92E-121.61E-124.58E-071.19E-056.13E-081.59E-051.23E-017.97E-035.57E-022.57E-084.56E-057.76E-034.21E-044.40E-031.75E-013.57E-041.00E-054.48E-044.01E-04rs1041983NAT21.48E-018.16E-028.87E-011.40E-029.33E-024.96E-018.62E-019.51E-021.59E-016.87E-018.27E-024.87E-012.81E-039.50E-043.42E-064.03E-042.61E-046.17E-043.32E-044.74E-035.44E-041.08E-017.79E-012.18E-014.49E-015.39E-01rs1801280NAT2----3.41E-011.58E-121.14E-121.21E-123.42E-177.64E-201.03E-095.34E-101.20E-193.04E-173.71E-132.07E-201.67E-231.31E-252.01E-251.96E-281.39E-241.80E-174.68E-175.09E-185.22E-231.73E-13rs1799929NAT27.43E-012.56E-014.87E-017.99E-026.50E-011.20E-081.79E-082.31E-061.02E-121.01E-151.33E-051.85E-043.82E-173.51E-152.39E-111.51E-165.68E-229.29E-231.43E-227.70E-272.95E-234.47E-152.36E-136.28E-151.19E-182.19E-11rs1799930NAT21.52E-012.12E-024.15E-012.89E-012.79E-015.62E-015.36E-015.15E-015.15E-036.59E-019.95E-022.16E-022.06E-016.83E-041.31E-073.90E-022.42E-013.02E-011.55E-017.65E-018.14E-016.98E-019.10E-025.47E-013.10E-011.36E-02rs1208NAT27.43E-012.56E-016.09E-017.99E-028.57E-011.13E-172.25E-144.00E-192.38E-242.31E-244.87E-163.76E-196.68E-195.81E-205.69E-124.96E-191.05E-202.87E-221.08E-223.74E-277.83E-241.28E-188.11E-166.73E-172.31E-223.00E-14rs1799931NAT23.96E-031.54E-015.44E-015.65E-026.41E-011.16E-048.47E-031.04E-063.81E-072.94E-071.33E-035.10E-059.61E-034.38E-014.63E-023.08E-027.69E-087.48E-051.18E-052.32E-059.66E-071.38E-011.12E-032.51E-032.75E-032.27E-02rs1495741NAT24.00E-013.54E-037.00E-011.13E-034.51E-013.38E-013.32E-028.69E-016.81E-012.19E-026.57E-017.58E-013.15E-043.73E-026.55E-013.32E-039.05E-054.08E-061.81E-063.86E-095.79E-051.41E-041.40E-077.46E-063.73E-108.61E-09rs2115819ALOX51.34E-036.16E-045.95E-021.20E-029.48E-022.81E-361.34E-237.95E-368.90E-401.18E-311.67E-301.21E-381.93E-154.49E-101.08E-065.41E-143.73E-232.13E-182.35E-173.96E-194.46E-204.25E-153.21E-226.81E-204.10E-153.43E-14rs12248560CYP2C19-----3.08E-171.17E-091.83E-141.92E-146.70E-094.56E-155.78E-142.31E-057.80E-04-3.52E-092.98E-132.69E-128.44E-141.78E-121.36E-12-6.49E-064.16E-063.07E-067.37E-06rs4244285CYP2C192.12E-014.41E-046.33E-055.24E-038.39E-022.47E-012.93E-019.06E-017.14E-029.15E-017.45E-013.02E-011.28E-021.41E-011.07E-048.15E-029.76E-023.79E-011.98E-017.60E-022.40E-036.66E-031.52E-035.34E-053.91E-046.82E-07rs1057910CYP2C97.73E-016.54E-017.70E-015.56E-017.35E-011.16E-016.06E-013.46E-022.17E-023.46E-025.53E-022.56E-021.15E-017.85E-013.42E-015.35E-018.88E-022.35E-015.32E-027.77E-033.01E-021.04E-042.08E-052.44E-035.39E-031.10E-03rs11572103CYP2C8-----1.89E-166.65E-108.36E-151.07E-183.96E-095.73E-121.88E-14--------------rs7909236CYP2C82.33E-029.92E-025.29E-014.06E-017.66E-015.84E-013.41E-013.73E-031.73E-037.89E-028.09E-032.28E-032.01E-134.96E-121.43E-172.60E-054.80E-124.84E-117.46E-081.66E-068.49E-062.97E-065.61E-102.88E-083.10E-061.45E-03rs17110453CYP2C87.88E-017.43E-039.72E-041.22E-044.29E-018.63E-111.13E-074.23E-117.52E-142.06E-126.08E-111.15E-125.08E-041.62E-021.88E-063.27E-032.05E-047.93E-011.58E-036.95E-017.96E-042.62E-022.34E-021.80E-022.56E-011.44E-03rs3813867CYP2E12.10E-011.21E-049.66E-032.47E-022.85E-035.71E-024.31E-013.91E-013.31E-011.23E-031.32E-014.68E-018.49E-014.02E-012.24E-011.23E-011.15E-015.41E-032.83E-035.36E-044.01E-022.37E-043.39E-053.80E-057.53E-051.09E-05rs2031920CYP2E18.14E-025.81E-046.33E-034.16E-032.16E-045.35E-051.61E-023.99E-051.02E-053.99E-051.58E-041.66E-058.39E-025.48E-025.67E-021.32E-011.63E-014.00E-021.38E-023.77E-038.79E-021.57E-033.24E-043.55E-046.19E-041.10E-04rs6413432CYP2E16.35E-261.06E-221.12E-211.81E-223.82E-26--2.62E-071.40E-08-3.64E-051.64E-087.89E-162.51E-142.09E-148.49E-157.98E-122.72E-102.28E-068.94E-071.62E-095.68E-187.09E-248.52E-182.80E-132.38E-18rs2070676CYP2E12.05E-041.66E-021.76E-012.81E-024.85E-035.15E-271.93E-152.26E-272.46E-295.40E-331.17E-272.93E-274.86E-029.13E-015.97E-011.77E-043.06E-026.61E-021.49E-012.85E-014.58E-031.50E-012.79E-011.94E-011.53E-014.12E-01rs5219KCNJ111.47E-064.47E-168.46E-201.57E-131.13E-14-2.08E-02-----1.04E-041.72E-162.61E-123.68E-101.21E-182.33E-237.54E-091.51E-173.22E-101.18E-173.00E-194.27E-192.69E-191.72E-11rs1801028DRD2-7.96E-039.04E-03-------------7.13E-02-----7.46E-13--4.77E-08rs2306283SLCO1B13.04E-015.03E-021.29E-012.28E-073.97E-025.66E-021.85E-022.80E-013.26E-014.87E-012.84E-013.02E-011.43E-171.12E-211.91E-181.43E-141.32E-232.11E-214.11E-268.88E-257.30E-281.32E-111.75E-131.96E-101.62E-192.33E-13rs4516035VDR------1.96E-05-----2.59E-201.29E-161.67E-106.90E-273.02E-275.07E-418.95E-315.06E-284.06E-333.49E-128.41E-135.15E-122.47E-171.47E-14rs762551CYP1A23.86E-011.32E-044.37E-044.15E-064.03E-014.84E-052.59E-032.04E-103.06E-076.14E-135.59E-071.95E-091.18E-019.46E-034.26E-027.11E-021.30E-011.15E-021.01E-011.87E-043.26E-056.78E-072.63E-103.58E-095.35E-094.15E-12rs2472304CYP1A26.74E-013.04E-019.05E-012.39E-015.01E-013.93E-024.55E-017.69E-083.81E-071.04E-067.77E-076.90E-081.20E-089.89E-034.64E-017.02E-153.98E-277.17E-204.64E-261.52E-232.03E-177.03E-014.30E-015.99E-012.99E-022.52E-01rs750155SULT1A11.53E-131.10E-216.16E-131.52E-242.79E-162.51E-191.42E-212.46E-281.38E-335.49E-245.54E-254.44E-271.31E-138.50E-133.16E-051.52E-201.43E-184.47E-145.33E-186.25E-201.09E-201.75E-325.10E-256.27E-311.42E-271.49E-38rs1800764ACE2.58E-082.67E-061.89E-064.61E-131.81E-082.40E-451.17E-358.71E-532.63E-589.85E-487.85E-568.85E-581.72E-131.78E-062.44E-035.15E-141.09E-144.56E-133.25E-129.62E-124.06E-172.53E-092.56E-136.97E-111.27E-099.22E-09rs4291ACE2.66E-291.08E-344.49E-302.01E-254.19E-341.14E-272.40E-282.30E-296.11E-254.21E-346.39E-252.97E-342.72E-281.77E-334.86E-417.35E-282.28E-291.16E-231.08E-304.98E-274.26E-303.76E-286.58E-272.23E-345.37E-298.21E-28rs4267385ACE3.22E-012.05E-019.11E-018.29E-023.75E-012.75E-291.85E-202.95E-316.81E-382.22E-391.33E-361.96E-353.38E-092.86E-053.72E-015.61E-111.90E-128.53E-121.67E-143.12E-151.67E-244.82E-015.48E-055.94E-022.46E-051.62E-03rs2108622CYP4F27.51E-021.74E-024.11E-023.01E-025.72E-026.59E-012.74E-013.06E-032.39E-024.39E-014.86E-019.56E-032.88E-041.54E-032.06E-017.88E-058.67E-032.43E-024.99E-042.80E-091.41E-066.00E-116.61E-132.18E-106.11E-109.78E-12rs3093105CYP4F2---2.56E-59-1.15E-371.86E-305.69E-332.44E-417.01E-382.46E-363.69E-355.66E-441.39E-421.35E-551.54E-441.21E-441.84E-49-7.51E-351.34E-396.52E-451.47E-461.85E-423.34E-422.80E-46rs8192726CYP2A65.87E-011.07E-011.88E-024.70E-022.40E-022.17E-045.86E-021.71E-027.30E-059.81E-031.06E-035.91E-036.33E-066.22E-048.59E-057.64E-064.51E-051.60E-011.13E-052.79E-052.94E-041.23E-012.95E-012.20E-022.34E-012.20E-02rs1051298SLC19A18.53E-185.07E-191.64E-121.63E-181.95E-169.09E-193.59E-165.15E-152.32E-231.78E-192.68E-174.92E-177.46E-184.53E-251.55E-241.90E-172.01E-181.54E-172.72E-235.20E-221.00E-184.21E-161.89E-184.75E-173.15E-163.93E-20rs1051296SLC19A15.68E-071.09E-095.76E-044.79E-082.02E-069.10E-062.37E-083.68E-092.95E-091.55E-101.18E-068.84E-089.92E-091.92E-152.65E-153.14E-091.12E-108.19E-098.43E-141.17E-101.40E-108.78E-065.30E-074.94E-097.81E-098.11E-09rs1131596SLC19A13.38E-091.29E-092.58E-059.40E-114.20E-073.13E-202.89E-083.62E-107.00E-274.69E-211.13E-192.40E-181.43E-091.98E-141.99E-121.82E-095.01E-061.35E-085.87E-151.13E-121.30E-107.51E-081.48E-094.33E-105.09E-123.51E-11rs1065852CYP2D64.67E-334.64E-293.15E-342.68E-312.43E-282.31E-451.33E-427.97E-531.41E-51-9.48E-444.64E-515.66E-441.58E-399.87E-537.74E-472.41E-341.99E-452.51E-383.61E-431.34E-394.53E-391.68E-431.02E-411.23E-482.80E-46Note: “-” means no data. Bold values means after adjustment p < 0.05/(55*26) the locus has statistical significance.Abbreviations: EAS, East Asian; SAS, South Asian; EUR, European; AFR, African; AMR, American; CDX, Chinese Dai in Xishuangbanna, China; CHB, Han Chinese in Beijing, China; CHS, Southern Han Chinese, China; JPT, Japanese in Tokyo, Japan; KHV, Kinh in Ho Chi Minh City; Vietnam; BEB, Bengali in Bangladesh; GIH, Gujarati Indian in Houston, Texas; ITU, Indian Telugu in the UK; PJL, Punjabi in Lahore, Pakistan; STU, Sri Lankan Tamil in the UK; CEU, Western European ancestry; FIN, Finnish in Finland; GBR, British in England and Scotland; IBS, Iberian populations in Spain; TSI, Toscani in Italy; ACB, African Caribbeans in Barbados; ASW, African Ancestry in Southwest USA; ESN, Esan in Nigeria; GWD, Gambian in Western Divisions, The Gambia; LWK, Luhya in Webuye, Kenya; MSL, Mende in Sierra Leone; YRI, Yoruba in Ibadan, Nigeria; CLM, Colombian in Medellin, Colombia; MXL, Mexican Ancestry in Los Angeles, Colombia; PEL, Peruvian in Lima, Peru; PUR, Puerto Rican in Puerto Rico.

The Genotype Distribution Difference Between Blang and 26 Other Populations After Bonferroni’s Multiple Adjustments

Note: “-” means no data. Bold values means after adjustment p < 0.05/(55*26) the locus has statistical significance.

Abbreviations: EAS, East Asian; SAS, South Asian; EUR, European; AFR, African; AMR, American; CDX, Chinese Dai in Xishuangbanna, China; CHB, Han Chinese in Beijing, China; CHS, Southern Han Chinese, China; JPT, Japanese in Tokyo, Japan; KHV, Kinh in Ho Chi Minh City; Vietnam; BEB, Bengali in Bangladesh; GIH, Gujarati Indian in Houston, Texas; ITU, Indian Telugu in the UK; PJL, Punjabi in Lahore, Pakistan; STU, Sri Lankan Tamil in the UK; CEU, Western European ancestry; FIN, Finnish in Finland; GBR, British in England and Scotland; IBS, Iberian populations in Spain; TSI, Toscani in Italy; ACB, African Caribbeans in Barbados; ASW, African Ancestry in Southwest USA; ESN, Esan in Nigeria; GWD, Gambian in Western Divisions, The Gambia; LWK, Luhya in Webuye, Kenya; MSL, Mende in Sierra Leone; YRI, Yoruba in Ibadan, Nigeria; CLM, Colombian in Medellin, Colombia; MXL, Mexican Ancestry in Los Angeles, Colombia; PEL, Peruvian in Lima, Peru; PUR, Puerto Rican in Puerto Rico.

### Genotype Distribution of 55 Loci

Among 55 loci, after Bonferroni correction between the Blang population and 26 other populations, the distribution of genotype frequencies was significantly different in five loci: rs750155 of sulfotransferase family 1A member gene (SULT1A1), rs4291 of ACE, rs1051298, rs1131596 and rs1051296 of SLC19A1. Besides, the genotype distribution of rs1800764 (ACE) and rs1065852 (CYP2D6) was different in all populations except for PEL and LWK, respectively. Conversely, the genotype distribution of rs1801028 of the dopamine receptor D2 gene (DRD2) was significantly different only in the GIH population compared with that in the Blang population. In addition to the eight loci mentioned above, the genotype distribution of the remaining loci in the Blang population also showed a significant difference compared with that in the other 26 populations, but to different degrees.

### MAF Distribution of Seven Significantly Different SNPs

The MAF distribution of seven significantly different SNPs is shown in Table 3 and Figure 1. The MAFs of rs1065852 (CYP2D6) and rs750155 (SULT1A1) showed the greatest similarities among SAS, EUR, AFR and AMR populations, but also showed the largest fluctuation between the Blang population and SAS, EUR, AFR and AMR populations. The MAFs of rs1800764 (ACE) and rs1131596 (SLC19A1) among the seven subpopulations of AFR showed distinct differences when compared with those of the Blang population. However, the MAFs of rs4291 (ACE), rs1051298 (SLC19A1), and rs1051296 (SLC19A1) showed relatively less fluctuation between the Blang population and the other 26 populations. Besides, the MAFs of rs1800764 (ACE) and rs750155 (SULT1A1) in the Blang population were close to those of the PEL population, even though most of other populations showed distinct differences on it. To better observe the phenotypes of these seven significantly different SNPs in the Blang population, their clinical and variant annotations were retrieved from PharmGKB (Supplemental Table 3 and Supplemental Table 4, respectively).Table 3The Minor Allele Frequency Distribution of Seven SNPs Among 27 PopulationsGeographic RegionPopulationrs750155-Crs1800764-Crs4291-Ars1051298-Grs1051296-Ars1131596-Ars1065852-AEASBlang0.2330.1830.5030.4580.4350.4880.500CDX0.5000.3660.6400.4300.4250.5380.629CHB0.6020.3110.6990.5000.4900.4760.602CHS0.4860.3330.6760.4240.4240.5330.614JPT0.6390.4470.5630.4380.4380.5430.361KHV0.5350.3480.6520.3890.3990.5400.662SASBEB0.7500.3900.6450.4360.4830.3900.256GIH0.6360.4510.5920.4080.4710.3930.150ITU0.7160.4070.6370.4560.5390.3970.172PJL0.6820.3910.6510.4900.5160.4220.104STU0.7940.3770.6670.4360.4950.4310.152EURCEU0.5660.4700.6260.5960.5960.4290.242FIN0.5050.4490.6410.5400.5400.4550.146GBR0.5600.4340.6590.5930.5930.4010.247IBS0.5790.4250.6360.4720.4770.5230.173TSI0.5890.5050.5840.5610.5650.4490.206AFRACB0.5780.8540.6410.3700.4270.7240.146ASW0.6310.7870.7050.4510.5330.6230.156ESN0.6820.9190.6770.4850.5510.6460.091GWD0.7350.9420.5800.3630.5090.7740.115LWK0.6360.8740.7020.4600.4750.7630.035MSL0.6590.9530.6290.4120.4820.7410.165YRI0.6670.9440.6160.4350.5140.7220.106AMRCLM0.4890.4570.6540.5270.5270.4950.186MXL0.4920.3130.7500.6410.6410.3520.148PEL0.1650.2240.8240.5940.6000.3710.071PUR0.5820.4620.6880.5380.5430.4380.178Note: Loci with minor allele frequency (MAF) > 0.05 could be considered into this study.Abbreviations: EAS, East Asian; SAS, South Asian; EUR, European; AFR, African; AMR, American; CDX, Chinese Dai in Xishuangbanna, China; CHB, Han Chinese in Beijing, China; CHS, Southern Han Chinese, China; JPT, Japanese in Tokyo, Japan; KHV, Kinh in Ho Chi Minh City, Vietnam; BEB, Bengali in Bangladesh; GIH, Gujarati Indian in Houston, Texas; ITU, Indian Telugu in the UK; PJL, Punjabi in Lahore, Pakistan; STU, Sri Lankan Tamil in the UK; CEU, Western European ancestry; FIN, Finnish in Finland; GBR, British in England and Scotland; IBS, Iberian populations in Spain; TSI, Toscani in Italy; ACB, African Caribbeans in Barbados; ASW, African Ancestry in Southwest USA; ESN, Esan in Nigeria; GWD, Gambian in Western Divisions, The Gambia; LWK, Luhya in Webuye, Kenya; MSL, Mende in Sierra Leone; YRI, Yoruba in Ibadan, Nigeria; CLM, Colombian in Medellin, Colombia; MXL, Mexican Ancestry in Los Angeles, Colombia; PEL, Peruvian in Lima, Peru; PUR, Puerto Rican in Puerto Rico.
Figure 1The minor allele frequency (MAF) distribution of seven significantly different SNPs between Blang population and other 26 populations. The value of the Y axis represents the MAF.

The Minor Allele Frequency Distribution of Seven SNPs Among 27 Populations

Note: Loci with minor allele frequency (MAF) > 0.05 could be considered into this study.

Abbreviations: EAS, East Asian; SAS, South Asian; EUR, European; AFR, African; AMR, American; CDX, Chinese Dai in Xishuangbanna, China; CHB, Han Chinese in Beijing, China; CHS, Southern Han Chinese, China; JPT, Japanese in Tokyo, Japan; KHV, Kinh in Ho Chi Minh City, Vietnam; BEB, Bengali in Bangladesh; GIH, Gujarati Indian in Houston, Texas; ITU, Indian Telugu in the UK; PJL, Punjabi in Lahore, Pakistan; STU, Sri Lankan Tamil in the UK; CEU, Western European ancestry; FIN, Finnish in Finland; GBR, British in England and Scotland; IBS, Iberian populations in Spain; TSI, Toscani in Italy; ACB, African Caribbeans in Barbados; ASW, African Ancestry in Southwest USA; ESN, Esan in Nigeria; GWD, Gambian in Western Divisions, The Gambia; LWK, Luhya in Webuye, Kenya; MSL, Mende in Sierra Leone; YRI, Yoruba in Ibadan, Nigeria; CLM, Colombian in Medellin, Colombia; MXL, Mexican Ancestry in Los Angeles, Colombia; PEL, Peruvian in Lima, Peru; PUR, Puerto Rican in Puerto Rico.

The minor allele frequency (MAF) distribution of seven significantly different SNPs between Blang population and other 26 populations. The value of the Y axis represents the MAF.

### Discussion

We genotyped 55 VIP variants from PharmGKB and compared the genotype distribution and MAF of variants in a Blang population with those of 26 other populations. Among 55 loci, the genotype distribution of five SNPs (rs750155 (SULT1A1), rs4291 (ACE), rs1051298 (SLC19A1), rs1051296 (SLC19A1) and rs1131596 (SLC19A1)) was significantly different in the Blang population compared with that in the other 26 populations. Two SNPs (rs1800764 (ACE) and rs1065852 (CYP2D6)) showed a significantly different genotype distribution in the Blang population compared with that in the other 25 populations but, compared with PEL and LWK populations, respectively, a significant difference was not observed. In addition, the MAFs of rs1065852 (CYP2D6) and rs750155 (SULT1A1) showed the greatest fluctuation between the Blang population and SAS, EUR, AFR and AMR populations.

SULT1A1, encoded by SULT1A1, is an isoform of sulfotransferases. The latter are phase-II detoxification enzymes and have a crucial role in the metabolism of several xenobiotics and endogenous compounds (eg, tamoxifen).30,31 High polymorphism of SULT1A1 has been reported among Caucasian, Chinese, African–American and Korean populations.32,33 Moyer et al reported that the genetic variation in SULT1A1, including rs750155, which is located in the promoter region (the short arm of chromosome 16) of SULT1A1, could explain (at least in part) the interindividual variability in the onset of menopause and symptoms before initiation of hormone therapy, and may represent a step towards individualizing decisions for hormone therapy.34 Besides, Innocenti et al demonstrated that allele T of rs750155 is not associated with the pharmacokinetic parameters of ABT-751 (novel anticancer agent) in people with neoplasms as compared with allele C.35 In our study, the genotype frequency distribution of rs750155 (SULT1A1) in the Blang population was significantly different from that of the other 26 populations. Also, the MAF distribution of rs750155 (SULT1A1) showed the greatest difference between the Blang population and SAS, EUR, AFR and AMR populations. Besides, the allele T frequency of rs750155 was far higher than that of allele C [T (76.7%) vs C (23.3%)], which indicated that the T allele of rs750155 in members of the Blang population with neoplasms could metabolize ABT-751 more readily.

ACE, encoded by ACE, is an enzyme that can affect the renin–angiotensin system and regulation of blood pressure.36,37 ACE inhibitors are first-line treatment for hypertension. They can favorably affect the vascular remodeling of patients with myocardial infarction and heart failure, and reduce its risk and mortality.38 The functional SNPs rs1800764 (ACE) and rs4291 (ACE) are located in the promoter region (chromosome 17) of ACE.39 Linkage disequilibrium has been identified between these two SNPs in ACE in multiple populations.40,41 These two SNPs possess the same pharmacokinetic characteristics and are associated with the risk of breast cancer, end-stage renal disease and Alzheimer’s disease.42–44 The SNPs rs1800764 (ACE) and rs4291 (ACE) show different drug responses in different populations.45–47 In the present study, the genotype frequency distribution of SNPs rs1800764 (ACE) and rs4291 (ACE) in the Blang population was different from that of the other populations studied, even though rs1800764 (ACE) was not significantly different in the Blang population compared with that in the PEL population. Besides, the MAF of rs1800764 (ACE) in the AFR population showed a distinct difference compared with that in the Blang population. However, rs4291 (ACE) showed relatively less fluctuation of MAF between the Blang population and the other 26 populations. Although the association between SNPs rs1800764 (ACE) and rs4291 (ACE) and the risk of breast cancer, end-stage renal disease and Alzheimer’s disease have not been elucidated in the Blang population, our pharmacogenomics study of the SNPs rs1800764 (ACE) and rs4291 (ACE) in the Blang population is important for disease prevention and safe use of drugs.

Reduced folate carrier protein 1 (RFC1), encoded by SLC19A1, is a high-capacity, bidirectional transporter of 5-methyl-tetrahydrofolate and thiamine monophosphate. RFC1 is involved in the uptake, homeostasis, folate deficiency as well as the transportation and sensitivity of antifolate chemotherapeutic agents, such as methotrexate.48–50 The SNPs rs1051298 and rs1051296 are intron variants, and rs1131596 is the missense variant of SLC19A1. Scholars have postulated genotype (AA + AG) of rs1051298 to be associated with reduced overall survival upon treatment with pemetrexed in people with non-small-cell lung cancer or mesothelioma compared with that with genotype GG.51 In addition, allele G of rs1051298 has been reported to be associated with longer progression-free survival after treatment with bevacizumab and pemetrexed in patients with lung neoplasms compared with that with allele A of rs1051298.52 Besides, the SNP rs1051296 is associated with higher plasma concentrations of methotrexate in pediatric patients with acute lymphoblastic leukemia.53 Evidence suggests that rs1131596 variants have a positive effect on methotrexate toxicity.54 Research has shown that the SNP rs1131596-G is not associated with alteration of the concentration or side-effects of methotrexate treatment compared with that of the SNP rs1131596-A in Chinese children with precursor cell lymphoblastic leukemia/lymphoma and people with rheumatoid arthritis.55 In our study, the genotype distribution of rs1051298, rs1051296, and rs1131596 in the Blang population was significantly different from that of the other 26 populations. MAF analyses showed that rs1051298 (SLC19A1), and rs1051296 (SLC19A1) showed relatively less fluctuation between the Blang population and the other 26 populations, even though the MAF of rs1131596 (SLC19A1) in the AFR population showed a distinct difference when compared with that of the Blang population. These observations suggested that pharmacogenomic research of variants of rs1051298, rs1051296 and rs1131596 may help to provide guidance for individualized drug use for the Blang population.

CYP2D6, encoded by CYP2D6, is an enzyme of the cytochrome P450 superfamily. It is involved in the metabolism of ≤25% of drugs in common use in the clinic.56 Debrisoquine and sparteine are CYP2D6 variation-related drugs.57 The genetic variation of CYP2D6 has been reported to be closely related to the metabolism of antipsychotic, antiarrhythmic and antiepileptic drugs.58–60 The SNP rs1065852 is an intron variant of CYP2D6. It is related to alteration of the encoded amino acids of CYP2D6 protein, reduction of CYP2D6 activity and to have a “poor metabolizer” phenotype.61 In addition, the genotype GG of rs1065852 (CYP2D6) is a factor of increased corrected QT (QTc) interval after treatment with iloperidone in people suffering from schizophrenia.61 The distribution of rs1065852 (CYP2D6) has been shown to be significantly different in a Zhuang population as compared with that in 11 other ethnic groups by Liu et al.3 In the present study, the genotype distribution of rs1065852 (CYP2D6) was different in the Blang population when compared with that in all other ethnic groups except for the LWK population, and the MAF distribution showed the largest fluctuation between the Blang population and SAS, EUR, AFR and AMR populations. Hence, the different corrected QTc interval may occur in schizophrenia patients of Blang ethnicity upon treatment with iloperidone. All the above evidence indicated the non-negligible roles of CYP2D6 (rs1065852) in effective drug usage and normal drug metabolism in Blang individuals.

We provided information on the genetic polymorphisms of VIP variants in the Blang population from Yunnan Province. Nevertheless, the sample size was small: a much larger sample size is needed to verify our results.

### Conclusions

The genotype distribution of five SNPs (rs750155 (SULT1A1), rs4291 (ACE), rs1051298 (SLC19A1), rs1051296 (SLC19A1) and rs1131596 (SLC19A1)) was significantly different in the Blang population compared with that in the other 26 populations tested. Two SNPs (rs1800764 (ACE) and rs1065852 (CYP2D6)) showed a significantly different genotype distribution in the Blang population as compared with all other populations tested except for PEL and LWK populations, respectively. The MAF of rs1065852 (CYP2D6) and rs750155 (SULT1A1) showed the largest fluctuation between the Blang population and SAS, EUR, AFR and AMR populations. Our data can provide theoretical guidance for safe and efficacious personalized drug use in the Blang population.



# SUPPLEMENTAL FILE 1: pgpm-14-1647.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)

# SUPPLEMENTAL FILE 2: get_supplementary_file.php

[File available at: kcnH2_run/KCNH2/20251126_134514/pmc_fulltext/34949935_supplements/get_supplementary_file.php]

