# MAIN TEXT

## Whole‐exome sequencing: Clinical characterization of pediatric and adult Italian patients affected by different forms of hereditary cardiovascular diseases

### Abstract

AbstractBackgroundHereditary cardiovascular diseases comprise several different entities. In this study, we focused on cardiomyopathies (i.e., hypertrophic, dilated, arrhythmogenic, and left ventricular non‐compaction), channelopathies (i.e., Brugada syndrome and long QT syndrome), and aortopathies and pulmonary arterial hypertension (i.e., thoracic/abdominal aortic aneurysm and pulmonary arterial hypertension), and genetically characterized 200 Italian patients affected by these diseases.MethodsWe employed whole‐exome sequencing (WES), focused on four in silico gene panels, and the MLPA method for hypertrophic and arrhythmogenic right ventricular cardiomyopathy cases.ResultsCardiomyopathies affected 87.5% of analyzed patients, channelopathies 7%, and aortopathies and pulmonary arterial hypertension 5.5%. The molecular diagnosis was confirmed for 21.5% of cases with a higher detection rate in familial forms (34%) than sporadic ones (14%). We highlighted the importance of family segregation to better understand the pathogenic role of the identified variants and their involvement in the clinical phenotype. Negative results could be ascribed to the high genetic and clinical heterogeneity of hereditary cardiovascular diseases; clinical follow‐up and revaluation of WES data will be essential.ConclusionThis study highlights the importance of a multi‐step approach (WES and MLPA) to characterize hereditary cardiovascular diseases, provides crucial information for clinical management and recurrence risk estimation, and lays the foundation for future personalized therapies.

### INTRODUCTION

Cardiovascular diseases (CVDs) are a major cause of morbidity and mortality worldwide affecting almost 471 million people (Benjamin et al., 2019). Hereditary CVDs (hCVDs) can be distinguished into different disorders. In this study, we focused on cardiomyopathies (CMs), arrhythmic disorders (i.e., channelopathies), and vascular disorders (i.e., thoracic aortic aneurysms) (Musunuru et al., 2020; Wilde et al., 2022).

Cardiomyopathies represent a heterogeneous group of structural and functional abnormalities of the heart muscle and are, globally, a major cause of morbidity and mortality (Cecchi et al., 2012). According to the 2008 position statement from the European Society of Cardiology (ESC) Working Group (Elliott et al., 2008), CMs can be classified as primary when predominantly confined to the heart muscle or secondary when they result from different conditions. This distinction can be challenging since many diseases classified as primary can have extra‐cardiac components and many systemic diseases can affect the heart (McCartan et al., 2012). These two broad categories are further classified considering the etiology of the pathology. In particular, primary CMs are classified into (a) genetic, including hypertrophic cardiomyopathy (HCM), arrhythmogenic right or left ventricular cardiomyopathy (ARVC or ALVC), and left ventricular non‐compaction cardiomyopathy (LVNC), (b) mixed (genetic and non‐genetic, including dilated cardiomyopathy (DCM) and restrictive cardiomyopathy (RCM)), and (c) acquired. Secondary CMs include infiltrative conditions, cardiomyopathy from storage disorders, toxic agents, endomyocardial causes, systemic inflammatory, autoimmune, endocrine, neuromuscular, nutritional conditions, electrolyte imbalances, or cancer treatment (Maron et al., 2006).

Channelopathies are a heterogenous group of cardiac diseases possibly responsible for the appearance of life‐threatening arrhythmias, leading sometimes to sudden death (Ferreira et al., 2022). Several channelopathies, such as Brugada syndrome (BrS), long QT syndrome (LQTS), short QT syndrome (SQTS), and J‐wave syndrome (JWS), catecholaminergic polymorphic ventricular tachycardia (CPVT), cardiac conduction disorder (CCD), and unexplained cardiac arrest (UCA) can be classified as acquired or inherited and the latter are mainly due to mutations in genes encoding cardiac ion channels (Chahine et al., 2022).

Aortopathies and pulmonary arterial hypertension, including thoracic aortic aneurysm (TAA), abdominal aortic aneurysm (AAA), pulmonary arterial hypertension (PAH), Rendu–Osler–Weber syndrome, Marfan syndrome, and type A aortic dissection, are mainly responsible for aneurysm formation and dissection and pulmonary hypertension (Bhandari et al., 2020; Monda et al., 2022). Aortopathies and pulmonary arterial hypertension may occur as a sporadic phenomenon or as a familial disorder following a classical Mendelian or a non‐classical inheritance pattern (Goyal et al., 2017).

To date, the global burden of genetically driven hCVDs is difficult to estimate, given the limited epidemiological studies. Thanks to the recent introduction of next‐generation sequencing (NGS) technologies, such as targeted re‐sequencing (TRS) and whole‐exome sequencing (WES), the knowledge of the genetic bases of hCVDs has largely increased, providing clinicians with essential information for the diagnosis, prognosis, treatment, and recurrence risk estimation of patients. In this light, many genes and loci have been identified, especially for primary CMs (Martinez et al., 2021), for which variants in more than 70 genes have been described so far (Hershberger et al., 2013).

Here, we describe the results obtained from the WES analysis of a cohort of 200 pediatric and adult patients. Each of them has been clinically and instrumentally carefully analyzed and classified. Four in silico panels have been applied to analyze WES data: (1) 37 genes for CMs, (2) 16 genes for channelopathies, (3) 19 genes for aortopathies and pulmonary arterial hypertension, and (4) 6 HCM minor genes (Tables S1 and S2). In addition, negative HCM and ARVC cases were analyzed with the multiplex ligation probe amplification (MLPA) method to search for copy number variations (CNVs). The results led to a whole picture of the molecular bases of hCVDs in the Italian population.

### MATERIALS AND METHODS

In this study, 200 consecutive patients affected by hCVDs were recruited during the last year at the Medical Cardiological Unit of the Cattinara Hospital and transferred to the Medical Genetics Unit of the IRCCS Burlo Garofolo in Trieste. Enrolled cases underwent an in‐depth phenotypical evaluation that comprised a detailed familial anamnesis, with particular attention to possible sudden death cases, and a personal anamnesis mainly aimed at identifying possible comorbidities and previous myocarditis episodes. Additionally, every patient underwent a cardiological evaluation through ECG, echocardiography, and heart magnetic resonance imaging (MRI). In specific cases, a myocardial biopsy was also performed (Figure S1). Enrolled subjects were classified according to their phenotype and clinical condition as affected by CMs, channelopathies, and aortopathies and pulmonary arterial hypertension as per international guidelines (Corrado et al., 2020; Humbert et al., 2022; Ommen et al., 2020; Pinto et al., 2016; Priori et al., 2013). Seventy‐six female (38%) and 124 male (62%) participants were enrolled in the study with an average age of 50 years. Overall, there are seven pediatric patients (3.5%) with ages between 0 and 17. Written informed consent was obtained from all participants or their legal guardians.

The research was conducted according to the ESC Clinical Practice Guidelines on ethics and the Helsinki declaration and approved by the Ethics Committee of the Institute for Maternal and Child Health—I.R.C.C.S. “Burlo Garofolo” of Trieste (Italy) (2007 242/07).

Genomic DNA was extracted from whole peripheral blood using the QIAsymphony DSP DNA midi kit v1 and QIAsymphony Robotic Device (Qiagen, Venlo, The Netherlands) following the manufacturer's instructions. DNA samples were stored at ‐20°C until use, and their integrity was evaluated with 1% agarose gel electrophoresis. DNA concentration was measured with the QIAxpert Spectrophotometer System (Qiagen, Venlo, The Netherlands).

According to the manufacturer's instructions, WES was performed using the Illumina NextSeq550 instrument (Illumina Inc., San Diego, CA; USA). Genomic libraries were prepared using the Twist Human Core Exome + Human RefSeq Panel kit (Twist Bioscience, South San Francisco, CA, USA) to cover 99% of the protein‐coding genes.

FastQ files were processed using a custom pipeline developed by enGenome srl (https://www.engenome.com/), including FastQ Quality Check, FastQ Mapping, FastQ trimming, Mark of Duplicates, Base Quality Score Recalibration, and Variant Calling.

This workflow, designed for Illumina paired‐end sequencing data, enables the generation of a final VCF file containing information regarding germline variants, such as single nucleotide variants (SNVs), short insertion/deletions (INDELs), and exon‐level CNVs. Finally, VCF files are analyzed on EnGenome Expert Variant Interpreter (eVai) software (https://evai.engenome.com) that allows variant annotation and interpretation. eVai combines artificial intelligence with the American College of Medical Genetics (ACMG) guidelines (Richards et al., 2015) to classify and prioritize genomic variants.

Three in silico gene panels for clinical exome targeting 37 genes responsible for CMs, 16 genes for channelopathies, and 19 genes responsible for aortopathies and pulmonary arterial hypertension (Table S1) were designed. Furthermore, in the specific case of HCM, whenever the molecular analysis was negative, we searched for variants within six HCM minor genes included in an additional in silico panel (Table S2). Genes were identified through the ClinGen resource (https://clinicalgenome.org/) and supporting literature data.

SNVs and INDELs were excluded if they led to synonymous amino acid substitutions that were not predicted as damaging or did not affect splicing or highly conserved residues. Furthermore, variants with a quality score (QUAL) < 20 or called in off‐target regions were excluded as well. Variants previously reported as polymorphism were removed, comparing the identified genetic variants and data reported in NCBI dbSNP build153 (http://www.ncbi.nlm.nih.gov/SNP/) as well as in gnomAD (http://gnomad.broadinstitute.org/). A minor allele frequency (MAF) cut‐off of 0.1% was used.

The pathogenicity of known genetic variants was evaluated using ClinVar (http://www.ncbi.nlm.nih.gov/clinvar/), Cardiodb (https://www.cardiodb.org/) and The Human Gene Mutation Database (http://www.hgmd.cf.ac.uk/ac/index.php). All the databases were last accessed on the 10th of October 2022.

Several in silico tools, such as PolyPhen‐2 (Adzhubei et al., 2013), SIFT (Ng & Henikoff, 2003), pseudo amino acid protein intolerance variant predictor (for coding variants SNVs/INDELs) (PaPI score) (Limongelli et al., 2015), and deep neural network variant predictor (for coding/non‐coding variants, SNVs) (DANN score) (Quang et al., 2015) were used to evaluate the effect of all variants. PolyPhen‐2 scores ranging from 0.85 to 1.0, SIFT scores between 0.0 and 0.05, PaPI scores ≥ 0.5, and DANN score ≥ 0.9 define deleterious variants. The BDGP in silico tool was employed for the splicing variants to define splicing sites' loss or addition (Reese et al., 1997). All the identified variants were confirmed by Sanger sequencing.

All variants included in the article have been submitted to the Leiden Open Variation Database (https://www.lovd.nl/).

For patients affected by HCM negative at WES, MLPA analysis was carried out to evaluate deletions or duplications within the genes MYBPC3 and MYH7. Following the manufacturer's instructions, the kits used were SALSA® MLPA® probe mix P100 MYBPC3 and P418 MYH7 (MRC‐Holland, Amsterdam, the Netherlands). Furthermore, for patients affected by ARVC and negative at WES, MLPA analysis was performed using the SALSA® MLPA® probe mix P168 ARVC‐PKP2 (MRC‐Holland, Amsterdam, the Netherlands) and following the manufacturer's instructions to evaluate deletions or duplications within the genes PKP2, DSG2, DSC2, JUP, DSP, TGFB3, and RYR2.

The software Coffalyser.Net was used combined with the lot‐specific MLPA Coffalyser sheet to perform the data analysis. The probes' dosage quotient (DQ) was used for MLPA results interpretation. In particular, the following cut‐offs have been applied: 0.80 < DQ < 1.20 (no deletion/duplication), DQ = 0 (deletion), and 1.75 < DQ < 2.15 (duplication).

### Samples collection

In this study, 200 consecutive patients affected by hCVDs were recruited during the last year at the Medical Cardiological Unit of the Cattinara Hospital and transferred to the Medical Genetics Unit of the IRCCS Burlo Garofolo in Trieste. Enrolled cases underwent an in‐depth phenotypical evaluation that comprised a detailed familial anamnesis, with particular attention to possible sudden death cases, and a personal anamnesis mainly aimed at identifying possible comorbidities and previous myocarditis episodes. Additionally, every patient underwent a cardiological evaluation through ECG, echocardiography, and heart magnetic resonance imaging (MRI). In specific cases, a myocardial biopsy was also performed (Figure S1). Enrolled subjects were classified according to their phenotype and clinical condition as affected by CMs, channelopathies, and aortopathies and pulmonary arterial hypertension as per international guidelines (Corrado et al., 2020; Humbert et al., 2022; Ommen et al., 2020; Pinto et al., 2016; Priori et al., 2013). Seventy‐six female (38%) and 124 male (62%) participants were enrolled in the study with an average age of 50 years. Overall, there are seven pediatric patients (3.5%) with ages between 0 and 17. Written informed consent was obtained from all participants or their legal guardians.

### Ethical considerations

The research was conducted according to the ESC Clinical Practice Guidelines on ethics and the Helsinki declaration and approved by the Ethics Committee of the Institute for Maternal and Child Health—I.R.C.C.S. “Burlo Garofolo” of Trieste (Italy) (2007 242/07).

### DNA extraction and quality control

Genomic DNA was extracted from whole peripheral blood using the QIAsymphony DSP DNA midi kit v1 and QIAsymphony Robotic Device (Qiagen, Venlo, The Netherlands) following the manufacturer's instructions. DNA samples were stored at ‐20°C until use, and their integrity was evaluated with 1% agarose gel electrophoresis. DNA concentration was measured with the QIAxpert Spectrophotometer System (Qiagen, Venlo, The Netherlands).

### WES

According to the manufacturer's instructions, WES was performed using the Illumina NextSeq550 instrument (Illumina Inc., San Diego, CA; USA). Genomic libraries were prepared using the Twist Human Core Exome + Human RefSeq Panel kit (Twist Bioscience, South San Francisco, CA, USA) to cover 99% of the protein‐coding genes.

FastQ files were processed using a custom pipeline developed by enGenome srl (https://www.engenome.com/), including FastQ Quality Check, FastQ Mapping, FastQ trimming, Mark of Duplicates, Base Quality Score Recalibration, and Variant Calling.

This workflow, designed for Illumina paired‐end sequencing data, enables the generation of a final VCF file containing information regarding germline variants, such as single nucleotide variants (SNVs), short insertion/deletions (INDELs), and exon‐level CNVs. Finally, VCF files are analyzed on EnGenome Expert Variant Interpreter (eVai) software (https://evai.engenome.com) that allows variant annotation and interpretation. eVai combines artificial intelligence with the American College of Medical Genetics (ACMG) guidelines (Richards et al., 2015) to classify and prioritize genomic variants.

Three in silico gene panels for clinical exome targeting 37 genes responsible for CMs, 16 genes for channelopathies, and 19 genes responsible for aortopathies and pulmonary arterial hypertension (Table S1) were designed. Furthermore, in the specific case of HCM, whenever the molecular analysis was negative, we searched for variants within six HCM minor genes included in an additional in silico panel (Table S2). Genes were identified through the ClinGen resource (https://clinicalgenome.org/) and supporting literature data.

SNVs and INDELs were excluded if they led to synonymous amino acid substitutions that were not predicted as damaging or did not affect splicing or highly conserved residues. Furthermore, variants with a quality score (QUAL) < 20 or called in off‐target regions were excluded as well. Variants previously reported as polymorphism were removed, comparing the identified genetic variants and data reported in NCBI dbSNP build153 (http://www.ncbi.nlm.nih.gov/SNP/) as well as in gnomAD (http://gnomad.broadinstitute.org/). A minor allele frequency (MAF) cut‐off of 0.1% was used.

The pathogenicity of known genetic variants was evaluated using ClinVar (http://www.ncbi.nlm.nih.gov/clinvar/), Cardiodb (https://www.cardiodb.org/) and The Human Gene Mutation Database (http://www.hgmd.cf.ac.uk/ac/index.php). All the databases were last accessed on the 10th of October 2022.

Several in silico tools, such as PolyPhen‐2 (Adzhubei et al., 2013), SIFT (Ng & Henikoff, 2003), pseudo amino acid protein intolerance variant predictor (for coding variants SNVs/INDELs) (PaPI score) (Limongelli et al., 2015), and deep neural network variant predictor (for coding/non‐coding variants, SNVs) (DANN score) (Quang et al., 2015) were used to evaluate the effect of all variants. PolyPhen‐2 scores ranging from 0.85 to 1.0, SIFT scores between 0.0 and 0.05, PaPI scores ≥ 0.5, and DANN score ≥ 0.9 define deleterious variants. The BDGP in silico tool was employed for the splicing variants to define splicing sites' loss or addition (Reese et al., 1997). All the identified variants were confirmed by Sanger sequencing.

All variants included in the article have been submitted to the Leiden Open Variation Database (https://www.lovd.nl/).

### MLPA

For patients affected by HCM negative at WES, MLPA analysis was carried out to evaluate deletions or duplications within the genes MYBPC3 and MYH7. Following the manufacturer's instructions, the kits used were SALSA® MLPA® probe mix P100 MYBPC3 and P418 MYH7 (MRC‐Holland, Amsterdam, the Netherlands). Furthermore, for patients affected by ARVC and negative at WES, MLPA analysis was performed using the SALSA® MLPA® probe mix P168 ARVC‐PKP2 (MRC‐Holland, Amsterdam, the Netherlands) and following the manufacturer's instructions to evaluate deletions or duplications within the genes PKP2, DSG2, DSC2, JUP, DSP, TGFB3, and RYR2.

The software Coffalyser.Net was used combined with the lot‐specific MLPA Coffalyser sheet to perform the data analysis. The probes' dosage quotient (DQ) was used for MLPA results interpretation. In particular, the following cut‐offs have been applied: 0.80 < DQ < 1.20 (no deletion/duplication), DQ = 0 (deletion), and 1.75 < DQ < 2.15 (duplication).

### RESULTS

A detailed medical history and accurate deep phenotyping of the 200 unrelated subjects enrolled in the study were carried out, leading to the following classification: 87.5% (175/200) of patients were affected by CMs, 7% (14/200) by channelopathies, and 5.5% (11/200) by aortopathies and pulmonary arterial hypertension (Figure 1).

Among the 200 patients, 87.5% were affected by cardiomyopathies, 7% suffer from Channelopathies, and 5.5% have aortopathies and pulmonary arterial hypertension as shown on the left pie chart. On the right, 27% of patients with hypertrophic cardiomyopathy, 55% dilated cardiomyopathy, 17% arrhythmogenic right ventricular cardiomyopathy, 1% left ventricular non‐compaction are shown. ARVC, Arrhythmogenic Right Ventricular Cardiomyopathy; CMs, Cardiomyopathies; DCM, Dilated Cardiomyopathy; HCM, Hypertrophic Cardiomyopathy; LVNC, Left Ventricular Non‐Compaction

Specifically, among the CMs' patients, 55% (97/175) show a DCM phenotype, 27% (47/175) an HCM phenotype, 17% (29/175) an ARVC phenotype, and 1% (2/175) an LVNC phenotype (Figure 1).

The cases can be classified either as familial (33.5%—67/200) or sporadic (66.5%—133/200).

Data analysis revealed that 21.5% (43/200) of patients had been solved at the molecular level, 26.5% (53/200) carried a variant of uncertain significance (VUS) (Table S3), and 52% (104/200) resulted negative after the molecular analysis (Table S4).

The molecular analysis of the 97 patients with DCM allowed the identification of 19% (18/97) of positive cases (Table 1).

List of likely causative variants identified by WES in patients affected by dilated cardiomyopathy

Abbreviations: DCM, Dilated Cardiomyopathy; NA, not available; Polyphen‐2 (D: Probably damaging; P: Possibly damaging; B: Benign); SIFT (D: Deleterious; T: Tolerated).

The major player involved in DCM is the TTN gene (MIM:*188840; NM_003319.4), responsible for 61% (11/18) of our positive cases. Titin controls the relaxation and contraction of the sarcomere, force transmission, and transduction, and it is the largest protein that has ever been described. It spans half of the sarcomere, with the N‐terminus in the Z‐line and the C‐terminus in the M‐line (Trinick & Tskhovrebova, 2010). However, the genetic studies described in the literature are more focused on variants placed in the A‐band, which are constitutively expressed in the heart and are associated with the disease, and on the most distal ones (located at the end of band A), which cause a more severe phenotype than the proximal ones (close to the band I) (J. S. Ware & Cook, 2018). We identified nine frameshift and two nonsense variants never described before in the literature. The phenotypes have an autosomal dominant inheritance pattern, and the variants lead to the formation of truncated forms of the protein (Tharp et al., 2019).

The remaining 39% (7/18) of positive cases are characterized by the presence of pathogenic variants within MYH7 (MIM:*160760; NM_000257.3), TNNT2 (MIM:*191045; NM_001001430.2), TNNC1 (MIM:*191040; NM_003280.2), LMNA (MIM:*150330; NM_170707.3), and FLNC (MIM:*102565; NM_001458.4) genes.

Among sporadic cases, patient CM560, a 14‐year‐old male patient, displayed an early‐onset DCM symptomatic of fatigue, with severe dilation and dysfunction of the left ventricle (LV), mild arrhythmic burden, and diffuse fibrosis at endomyocardial biopsy. Data analysis revealed the presence of the nonsense de novo unknown variant c.59569G>T; p.(Glu19857*) in the TTN gene.

Another sporadic patient is CM591, a 67‐year‐old man displaying acute heart failure in newly diagnosed DCM, with non‐sustained ventricular tachycardia (NSVT), subepicardial fibrosis in the posterolateral LV walls at MRI, and carrying the unknown frameshift variant c.6763_6764dupAC; p.(Ser2256Profs*169) in the FLNC gene (MIM:*102565; NM_001458.4) responsible for the premature termination of the 169 codons downstream, resulting in premature translation termination.

Sixteen patients of 47 affected by HCM were positive (34%) in the molecular analysis (Table 2). Major players are the MYBPC3 gene (MIM:*600958; NM_000256.3), responsible for 44% (7/16) of the cases, and the MYH7 gene (MIM:*160760; NM_000257.3), responsible for 19% (3/16). Other genes involved are GLA (MIM:*300644; NM_000169.2), associated with Fabry disease, PLN (MIM:*172405; NM_002667.3), TPM1 (MIM:*191010; NM_001018005.1), ALPK3 (MIM:*617608; NM_020778.5), and TRIM63 (MIM:*606131; NM_032588.4). The last two genes are included in the HCM minor genes panel (Table S2).

List of likely causative variants identified by WES in patients affected by hypertrophic cardiomyopathy

Abbreviations: HCM, Hypertrophic Cardiomyopathy; NA: not available; Polyphen‐2 (D: Probably damaging; P: Possibly damaging; B: Benign); SIFT (D: Deleterious; T: Tolerated).

One interesting sporadic case is CM741, a 55‐year‐old woman carrying the unknown truncating variant c.2043_2044delinsCT; p.(Gln681_Glu682delinsHis*) in the ALPK3 gene, leading to the formation of a premature stop codon and the consequent termination of the transcription. The diagnosis of HCM was confirmed at 16 years of age, and at 52 years, the proband underwent orthotopic heart transplantation.

Moreover, the MLPA analysis performed only on negative HCM cases did not detect any CNV within the MYBPC3 and MYH7 genes.

Seventeen percent (29/175) of our patients display ARVC. Twenty‐one percent of them (6/29) carry a pathogenic variant (Table 3), the PKP2 gene (MIM:*602861; NM_004572.3) responsible for 67% (4/6) of cases. Other mutated genes are FLNC (MIM:*102565; NM_001458.4) and DSP (MIM:*125647; NM_004415.2).

List of likely causative variants identified by WES in patients affected by arrhythmogenic right ventricular cardiomyopathy. CM745, previously classified as negative after the molecular analysis and underwent the multiplex ligation‐dependent probe amplification (MLPA) analysis

Abbreviations: ARVC, Arrhythmogenic Right Ventricular Cardiomyopathy; NA, not available; PolyPhen‐2 (D: Probably damaging; P: Possibly damaging; B: Benign); SIFT (D: Deleterious; T: Tolerated).

Two different familial cases, CM596 and CM752, a 69‐year‐old man and a 59‐year‐old woman, displayed typical predominant right ventricular (RV) involvement, characterized by chamber dilation with segmental aneurysms, RV dysfunction, and fibro‐fatty replacement. WES data suggested the presence of two novel large deletions, respectively, of 72.4 and 75 kb in the PKP2 gene. Both deletions spanned from exon 6 to exon 14 and were confirmed by the MLPA method (Figure 2). The lost domains of the encoded protein are involved in intracellular signaling, cytoskeletal regulation, and linking cadherins to the intermediate filaments in the cytoskeleton (Hatzfeld, 1999). In the case of CM596, the proband's brother had a suspect of ARVC but died suddenly at a young age. CM752's mother is reported to be affected by ARVC.

Multiplex ligation‐dependent probe amplification results for CM596, CM745, and CM752. The analysis revealed the presence of three large deletions within the PKP2 gene (NM_004572.3) responsible for arrhythmogenic right ventricular cardiomyopathy. ARVC, Arrhythmogenic Right Ventricular Cardiomyopathy; MLPA, Multiplex Ligation‐dependent Probe Amplification

Moreover, the MLPA analysis on negative cases allowed us to solve patient CM745. In this subject, a 38‐year‐old woman affected by arrhythmias and atrial fibrillation with a family history of sudden death, deletion of exon 12 within the PKP2 gene was identified.

In our cohort, channelopathies affect 7% (14/200) of patients and the clinical phenotypes identified correspond to BrS (21%—3/14), LQTS (29%—4/14), CCD (29%—4/14), and UCA (21%—3/14). Overall, 14% of patients (2/14) tested positive for the molecular analysis (Table 4A), 7% (1/14) carried a VUS, and 79% (11/14) were classified as negative after the molecular analysis. The major player genes involved in channelopathies were SCN5A (MIM:*600163; NM_001099404.1) and KCNH2 (MIM:*152427; NM_000238.3).

(A) List of likely causative variants identified by WES in patients affected by channelopathies. (B) List of likely causative variants identified by WES in patients affected by aortopathies and pulmonary arterial hypertension

Abbreviations: BrS, Brugada Syndrome; LQTS, long QT Syndrome; NA, not available; PolyPhen‐2 (D: Probably damaging; P: Possibly damaging; B: Benign); SIFT (D: Deleterious; T: Tolerated).

One interesting case is CAP52, a 45‐year‐old man found to be affected by LQTs during hospital admission for pulmonary embolism. He displayed QT interval variability, ranging from near normal to mildly prolonged. WES analysis revealed the presence of the unknown missense variant c.1889T>G; p.(Val630Gly) in the KCNH2 gene, responsible for the LQTS phenotype. The variant is in a transmembrane helix region of a voltage‐activated potassium channel that repolarizes the ventricular action potential (Gianulis & Trudeau, 2011). The variant has also been identified in the proband's son and daughter. The 6‐year‐old son was hospitalized after syncopal events, and he displayed a markedly prolonged QT interval, which led to defibrillator implantation. The 4‐year‐old daughter is affected by LQTS as the father and brother but apparently shows a less severe phenotype, with normal or mildly prolonged QT interval at rest.

The 5.5% (11/200) of our patients display aortopathies and pulmonary arterial hypertension, and the clinical conditions identified were Rendu–Osler–Weber syndrome (9%—1/11), aortic aneurysm (27%—3/11), Marfan syndrome (18%—2/11), PAH (27%—3/11), and type A aortic dissection (18%—2/11). Overall, 9% of patients (1/11) were positive for the molecular analysis (Table 4B), 18% (2/11) carried a VUS, and 73% (8/11) were classified as negative after the molecular analysis.

The familial case CM668, a 68‐year‐old woman, affected by Rendu–Osler–Weber Syndrome manifested multiple pulmonary and hepatic arteriovenous malformations determining a high cardiac output state with pulmonary hypertension and severe right heart failure. Data analysis revealed the presence of the non‐canonical splicing variant c.625+5G>C in the ACVRL1 gene (MIM:*601284; NM_000020.2) responsible for the loss of a splice donor site according to the BDGP prediction tool (Reese et al., 1997). In addition, the proband's father, who died suddenly at 56‐year‐old, had a suspect of Rendu–Osler–Weber Syndrome.

### DCM

The molecular analysis of the 97 patients with DCM allowed the identification of 19% (18/97) of positive cases (Table 1).

List of likely causative variants identified by WES in patients affected by dilated cardiomyopathy

Abbreviations: DCM, Dilated Cardiomyopathy; NA, not available; Polyphen‐2 (D: Probably damaging; P: Possibly damaging; B: Benign); SIFT (D: Deleterious; T: Tolerated).

The major player involved in DCM is the TTN gene (MIM:*188840; NM_003319.4), responsible for 61% (11/18) of our positive cases. Titin controls the relaxation and contraction of the sarcomere, force transmission, and transduction, and it is the largest protein that has ever been described. It spans half of the sarcomere, with the N‐terminus in the Z‐line and the C‐terminus in the M‐line (Trinick & Tskhovrebova, 2010). However, the genetic studies described in the literature are more focused on variants placed in the A‐band, which are constitutively expressed in the heart and are associated with the disease, and on the most distal ones (located at the end of band A), which cause a more severe phenotype than the proximal ones (close to the band I) (J. S. Ware & Cook, 2018). We identified nine frameshift and two nonsense variants never described before in the literature. The phenotypes have an autosomal dominant inheritance pattern, and the variants lead to the formation of truncated forms of the protein (Tharp et al., 2019).

The remaining 39% (7/18) of positive cases are characterized by the presence of pathogenic variants within MYH7 (MIM:*160760; NM_000257.3), TNNT2 (MIM:*191045; NM_001001430.2), TNNC1 (MIM:*191040; NM_003280.2), LMNA (MIM:*150330; NM_170707.3), and FLNC (MIM:*102565; NM_001458.4) genes.

Among sporadic cases, patient CM560, a 14‐year‐old male patient, displayed an early‐onset DCM symptomatic of fatigue, with severe dilation and dysfunction of the left ventricle (LV), mild arrhythmic burden, and diffuse fibrosis at endomyocardial biopsy. Data analysis revealed the presence of the nonsense de novo unknown variant c.59569G>T; p.(Glu19857*) in the TTN gene.

Another sporadic patient is CM591, a 67‐year‐old man displaying acute heart failure in newly diagnosed DCM, with non‐sustained ventricular tachycardia (NSVT), subepicardial fibrosis in the posterolateral LV walls at MRI, and carrying the unknown frameshift variant c.6763_6764dupAC; p.(Ser2256Profs*169) in the FLNC gene (MIM:*102565; NM_001458.4) responsible for the premature termination of the 169 codons downstream, resulting in premature translation termination.

### HCM

Sixteen patients of 47 affected by HCM were positive (34%) in the molecular analysis (Table 2). Major players are the MYBPC3 gene (MIM:*600958; NM_000256.3), responsible for 44% (7/16) of the cases, and the MYH7 gene (MIM:*160760; NM_000257.3), responsible for 19% (3/16). Other genes involved are GLA (MIM:*300644; NM_000169.2), associated with Fabry disease, PLN (MIM:*172405; NM_002667.3), TPM1 (MIM:*191010; NM_001018005.1), ALPK3 (MIM:*617608; NM_020778.5), and TRIM63 (MIM:*606131; NM_032588.4). The last two genes are included in the HCM minor genes panel (Table S2).

List of likely causative variants identified by WES in patients affected by hypertrophic cardiomyopathy

Abbreviations: HCM, Hypertrophic Cardiomyopathy; NA: not available; Polyphen‐2 (D: Probably damaging; P: Possibly damaging; B: Benign); SIFT (D: Deleterious; T: Tolerated).

One interesting sporadic case is CM741, a 55‐year‐old woman carrying the unknown truncating variant c.2043_2044delinsCT; p.(Gln681_Glu682delinsHis*) in the ALPK3 gene, leading to the formation of a premature stop codon and the consequent termination of the transcription. The diagnosis of HCM was confirmed at 16 years of age, and at 52 years, the proband underwent orthotopic heart transplantation.

Moreover, the MLPA analysis performed only on negative HCM cases did not detect any CNV within the MYBPC3 and MYH7 genes.

### ARVC

Seventeen percent (29/175) of our patients display ARVC. Twenty‐one percent of them (6/29) carry a pathogenic variant (Table 3), the PKP2 gene (MIM:*602861; NM_004572.3) responsible for 67% (4/6) of cases. Other mutated genes are FLNC (MIM:*102565; NM_001458.4) and DSP (MIM:*125647; NM_004415.2).

List of likely causative variants identified by WES in patients affected by arrhythmogenic right ventricular cardiomyopathy. CM745, previously classified as negative after the molecular analysis and underwent the multiplex ligation‐dependent probe amplification (MLPA) analysis

Abbreviations: ARVC, Arrhythmogenic Right Ventricular Cardiomyopathy; NA, not available; PolyPhen‐2 (D: Probably damaging; P: Possibly damaging; B: Benign); SIFT (D: Deleterious; T: Tolerated).

Two different familial cases, CM596 and CM752, a 69‐year‐old man and a 59‐year‐old woman, displayed typical predominant right ventricular (RV) involvement, characterized by chamber dilation with segmental aneurysms, RV dysfunction, and fibro‐fatty replacement. WES data suggested the presence of two novel large deletions, respectively, of 72.4 and 75 kb in the PKP2 gene. Both deletions spanned from exon 6 to exon 14 and were confirmed by the MLPA method (Figure 2). The lost domains of the encoded protein are involved in intracellular signaling, cytoskeletal regulation, and linking cadherins to the intermediate filaments in the cytoskeleton (Hatzfeld, 1999). In the case of CM596, the proband's brother had a suspect of ARVC but died suddenly at a young age. CM752's mother is reported to be affected by ARVC.

Multiplex ligation‐dependent probe amplification results for CM596, CM745, and CM752. The analysis revealed the presence of three large deletions within the PKP2 gene (NM_004572.3) responsible for arrhythmogenic right ventricular cardiomyopathy. ARVC, Arrhythmogenic Right Ventricular Cardiomyopathy; MLPA, Multiplex Ligation‐dependent Probe Amplification

Moreover, the MLPA analysis on negative cases allowed us to solve patient CM745. In this subject, a 38‐year‐old woman affected by arrhythmias and atrial fibrillation with a family history of sudden death, deletion of exon 12 within the PKP2 gene was identified.

### Channelopathies

In our cohort, channelopathies affect 7% (14/200) of patients and the clinical phenotypes identified correspond to BrS (21%—3/14), LQTS (29%—4/14), CCD (29%—4/14), and UCA (21%—3/14). Overall, 14% of patients (2/14) tested positive for the molecular analysis (Table 4A), 7% (1/14) carried a VUS, and 79% (11/14) were classified as negative after the molecular analysis. The major player genes involved in channelopathies were SCN5A (MIM:*600163; NM_001099404.1) and KCNH2 (MIM:*152427; NM_000238.3).

(A) List of likely causative variants identified by WES in patients affected by channelopathies. (B) List of likely causative variants identified by WES in patients affected by aortopathies and pulmonary arterial hypertension

Abbreviations: BrS, Brugada Syndrome; LQTS, long QT Syndrome; NA, not available; PolyPhen‐2 (D: Probably damaging; P: Possibly damaging; B: Benign); SIFT (D: Deleterious; T: Tolerated).

One interesting case is CAP52, a 45‐year‐old man found to be affected by LQTs during hospital admission for pulmonary embolism. He displayed QT interval variability, ranging from near normal to mildly prolonged. WES analysis revealed the presence of the unknown missense variant c.1889T>G; p.(Val630Gly) in the KCNH2 gene, responsible for the LQTS phenotype. The variant is in a transmembrane helix region of a voltage‐activated potassium channel that repolarizes the ventricular action potential (Gianulis & Trudeau, 2011). The variant has also been identified in the proband's son and daughter. The 6‐year‐old son was hospitalized after syncopal events, and he displayed a markedly prolonged QT interval, which led to defibrillator implantation. The 4‐year‐old daughter is affected by LQTS as the father and brother but apparently shows a less severe phenotype, with normal or mildly prolonged QT interval at rest.

### Aortopathies and pulmonary arterial hypertension

The 5.5% (11/200) of our patients display aortopathies and pulmonary arterial hypertension, and the clinical conditions identified were Rendu–Osler–Weber syndrome (9%—1/11), aortic aneurysm (27%—3/11), Marfan syndrome (18%—2/11), PAH (27%—3/11), and type A aortic dissection (18%—2/11). Overall, 9% of patients (1/11) were positive for the molecular analysis (Table 4B), 18% (2/11) carried a VUS, and 73% (8/11) were classified as negative after the molecular analysis.

The familial case CM668, a 68‐year‐old woman, affected by Rendu–Osler–Weber Syndrome manifested multiple pulmonary and hepatic arteriovenous malformations determining a high cardiac output state with pulmonary hypertension and severe right heart failure. Data analysis revealed the presence of the non‐canonical splicing variant c.625+5G>C in the ACVRL1 gene (MIM:*601284; NM_000020.2) responsible for the loss of a splice donor site according to the BDGP prediction tool (Reese et al., 1997). In addition, the proband's father, who died suddenly at 56‐year‐old, had a suspect of Rendu–Osler–Weber Syndrome.

### DISCUSSION

The development of NGS technologies has largely improved the molecular diagnosis of hCVDs, helping the dissection of these diseases characterized by remarkable clinical and genetic heterogeneity. Moreover, the molecular characterization of hCVDs is essential for a better medical management, allowing, in some cases, an early diagnosis of the disease before the appearance of any clinical phenotype (Wilcox & Hershberger, 2018).

A fundamental step in diagnosing hCVDs is genetic counseling, which is recommended by ACMG guidelines (Hershberger et al., 2018). It is necessary to improve patients' medical management by analyzing the psychosocial impact of a heritable disease, to provide recurrence risk estimation by considering the proband's family history, and describing the inheritance pattern of hCVDs (Hershberger et al., 2009). Recurrence risk estimation is of the utmost importance both to promote an informed family planning and to identify at‐risk children. Indeed, the early identification of pediatric patients who are carrier of known pathogenic variants in hCVDs‐associated genes is fundamental to fostering the implementation of preventive strategies and paving the way for future personalized therapeutic approaches (Illikova et al., 2015; S. M. Ware, 2017). Specifically, predictive genetic testing in related children of subjects affected by inherited arrhythmia syndromes (e.g., LQTS, CPVT, BrS, CCD) is recommended from birth onward. Predictive testing in related children of subjects affected by cardiomyopathies is recommended in those aged >10‐12 years and earlier testing may be considered if there is a family history of early‐onset disease (Wilde et al., 2022).

Here, for the first time to our knowledge, we describe a complete overview of hCVD Italian patients, molecularly characterizing 21.5% of cases (93% of them display a CM), with an increase in the detection rate in familial forms (34%) compared to sporadic cases (14%). Family segregation of the variants has been carried out when possible, identifying, in some cases, examples of genotype–phenotype correlation. For instance, one familial genotype–phenotype correlation can be observed in the case of patient CM658's family. The proband, a 27‐year‐old man affected by ARVC, displayed an RV dilation with diffuse hypokinesia of the walls. The MRI revealed also a localized area of adipose replacement at the level of the RV papillary. Data analysis revealed the presence of the known variant c.962delT; p.(V al321Alafs*31) in the PKP2 gene identified also in the proband's paternal grandmother, father, brother, and sister. They all display a clinical phenotype overlapping the one identified in the proband.

Another example of genotype–phenotype correlation can be observed in the case of CM626's family. The proband and both his sons presented LV dilation, severe systolic dysfunction, and hypokinetic RV. WES data analysis and familial segregation revealed in all of them the presence of the unknown variant c.41193_41196dupTCCG; p.(Ile13733Serfs*8) in the TTN gene.

Our data agree with the literature studies (Burns et al., 2017; de Asmundis et al., 2017; Ye et al., 2019), although it is important to highlight that our cohort is enriched of DCM individuals (48.5%), being the Medical Cardiological Unit of the Cattinara Hospital the Italian DCM Reference Centre.

Regarding DCM, apart from the TTN gene, which explains 61% of positive cases, another interesting gene is FLNC. Indeed, one sporadic case, CM591, carries the unknown frameshift mutation c.6763_6764dupAC; p.(Ser2256Profs*169) within FLNC. The literature reports three disease‐causing variants associated with DCM (Begay et al., 2018; Janin et al., 2017; Xiao et al., 2020) and located within the same protein domain, corresponding to a filamin type 1. In the past, these variants were related to distal and myofibrillar skeletal myopathies (Begay et al., 2018), whereas recent studies (Janin et al., 2017) revealed that truncating mutations in the FLNC gene can cause a cardiac phenotype corresponding to ventricular tachycardia and atrial fibrillation, as seen in our patient.

Regarding the HCM patients analyzed in our cohort, 44% of the positive cases carried a mutation within the MYBPC3 gene. Interestingly, patient CM741, who resulted negative at the in silico CMs gene panel, was further analyzed through an HCM minor genes panel (Walsh et al., 2022). The analysis highlighted the presence of an unknown truncating variant, c.2043_2044delinsCT; p.(Gln681_Glu682delinsHis*), in the ALPK3 gene. Recent studies revealed that truncating variants in the ALPK3 gene are associated with autosomal dominant HCM (Lopes et al., 2021), as observed in the patient. On the same note, the employment of the secondary minor gene panel allowed the identification in patient CM621 of a known homozygous pathogenic nonsense variant c.739C>T; p.(Gln247*) within TRIM63, a gene associated with autosomal recessive HCM (Chen et al., 2012; Ploski et al., 2014).

As regards ARVC, this study highlighted that PKP2 mutations have a high prevalence in our cohort (67%). In particular, three cases, CM596, CM745, and CM752, carried CNVs within PKP2, a gene already associated with large genomic rearrangements (Cox et al., 2011; Pilichou et al., 2017). For CM596 and CM752 patients, the clinical phenotype corresponds to a severe LV dilation which, along with RV abnormalities, is common in other ARVC cases carrying large deletions within the PKP2 gene, as described in the literature (Roberts et al., 2013; Sonoda et al., 2017). In addition, according to literature data (Alhassani et al., 2018), other phenotypes associated with CNVs within PKP2 are atrial fibrillations and arrhythmia, as displayed by patient CM745. Indeed, molecular analyses (WES and MLPA) highlighted the presence of PKP2 deletions.

Regarding channelopathies, an interesting finding regards an LQTS familial case. The proband CAP52 and his daughter have a mild phenotype, while his son displays severe LQTS symptoms leading to the implantation of a defibrillator. Interestingly, channelopathies usually appear in people around 40–50 years of age, and they are quite uncommon in pediatric patients (Sieira et al., 2016). All three subjects carry the variant c.1889T>G; p.(Val630Gly) in the KCNH2 gene, located in a transmembrane region. Interestingly, several other pathogenic variants (Lahrouchi et al., 2017; Splawski et al., 1998; Tanaka et al., 1997) associated with LQTS and sudden cardiac death were detected in the same domain. These alleles are responsible for aberrant cardiac repolarization leading to arrhythmias (Splawski et al., 1998). These considerations highlight that channelopathies are characterized by high clinical variability, but the underlying mechanisms are still poorly understood. For example, hormonal influence and sex differences might have a role. Indeed, females are often asymptomatic, probably due to lower testosterone concentrations (Brugada et al., 2018). It is possible to hypothesize that the proband's daughter will always display a milder phenotype than her male relatives.

Finally, as regards aortopathies and pulmonary arterial hypertension, the solved case corresponds to CM668, a 68‐year‐old woman affected by the Rendu–Osler–Weber Syndrome and showing heart failure, impairments of the tricuspid valve, and atrial fibrillation. The WES analysis revealed the non‐canonical splicing mutation c.625+5G>C within the ACVRL1 gene. According to the literature, loss‐of‐function alleles and non‐canonical splicing mutations, like the one described here, are responsible for epistasis or nose bleedings, symptomatic liver diseases, and anemia (Sánchez‐Martínez et al., 2020) and a few cases for abnormalities in the endothelial cells leading to pulmonary hypertension (Trembath et al., 2001). Interestingly, the patient displays all the symptoms described above.

Among the 200 patients enrolled, 21.5% were solved while 26.5% carried a VUS, whose interpretation remains an open question. In those cases, segregation within families is essential to confirm their role in the etiology of hCVDs. Finally, for 52% of patients, a molecular diagnosis was not defined, probably due to the high genetic and clinical heterogeneity of these diseases and all the limitations of the applied technologies. Thus, for these patients, it would be essential to perform a clinical follow‐up combined with a re‐evaluation of WES data and, eventually, a whole‐genome sequencing approach.

### CONCLUSION

In conclusion, the use of a multistep approach (e.g., WES and MLPA) allowed the molecular diagnosis of patients affected by different forms of hCVDs, shedding light on the true complexity of this group of diseases. This work highlights the importance of a deep clinical characterization combined with the use of high‐throughput technologies and suggests the importance of evaluating also minor genes such as ALPK3 and TRIM63 which should be included in CMs panel. All these findings have relevant practical outcomes, influencing the clinical management of both pediatric and adult patients, providing recurrence risk estimation, and laying the foundation for developing future personalized therapeutic strategies.

### AUTHOR CONTRIBUTION

Conceptualization: Matteo Dal Ferro and Giorgia Girotto; Data curation: Stefania Lenarduzzi, Beatrice Alessandrini, Paola Tesolin, and Giorgia Girotto; Formal analysis: Stefania Lenarduzzi, Beatrice Spedicati, Beatrice Alessandrini, and Paola Tesolin; Funding acquisition: Giorgia Girotto; Investigation: Stefania Lenarduzzi, Beatrice Spedicati, and Beatrice Alessandrini; Methodology: NA.; Project administration: Giorgia Girotto; Resources: Stefania Lenarduzzi, Beatrice Alessandrini, Paola Tesolin, Beatrice Spedicati, Alessia Paldino, Marta Gigli, Gianfranco Sinagra, Paolo Gasparini, Matteo Dal Ferro, and Giorgia Girotto; Software: NA; Supervision: Paolo Gasparini and Giorgia Girotto; Validation: Beatrice Alessandrini, Stefania Lenarduzzi, and Paola Tesolin; Visualization: Stefania Lenarduzzi, Beatrice Spedicati, Beatrice Alessandrini, and Paola Tesolin; Writing—original draft preparation: Stefania Lenarduzzi, Beatrice Spedicati, Beatrice Alessandrini, and Paola Tesolin; Writing—review and editing: Stefania Lenarduzzi, Beatrice Spedicati, Beatrice Alessandrini, Paola Tesolin, Alessia Paldino, Marta Gigli, Gianfranco Sinagra, Paolo Gasparini, Matteo Dal Ferro, and Giorgia Girotto. All authors have read and agreed to the published version of the manuscript.

### FUNDING INFORMATION

This research was supported by D70‐RESRICGIROTTO to GG. The funders had no role in study design, data collection, analysis, decision to publish, or preparation of the manuscript.

### CONFLICT OF INTEREST

The authors report no conflict of interest.

### ETHICS STATEMENT

The research was conducted according to the ESC Clinical Practice Guidelines on ethics and the Helsinki declaration and approved by the Ethics Committee of the Institute for Maternal and Child Health—I.R.C.C.S. “Burlo Garofolo” of Trieste (Italy) (2007 242/07).

### Supporting information

Figure S1

Click here for additional data file.

Table S1

Click here for additional data file.

Table S2.

Click here for additional data file.

Table S3

Click here for additional data file.

Table S4

Click here for additional data file.



# SUPPLEMENTAL FILE 1: MGG3-11-e2143.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)