# MAIN TEXT

## Identification of a novel pathogenic variant in 
KCNH2
 in an Iranian family with long QT syndrome 2 by whole‐exome sequencing

### Abstract

AbstractBackgroundLong QT syndrome (LQTS) is a lethal cardiac condition. However, the clinical implementation of genetic testing has now made LQTS eminently treatable. Next‐generation sequencing has remarkable potential for both clinical diagnostics and research of LQTS. Here, we investigated the genetic etiology in an LQTS‐suspected Iranian pedigree by whole‐exome sequencing and collected all KCNH2 variants with consensus based on publications.MethodsWES was performed on the proband of this pedigree to reveal the underlying cause of sudden cardiac death (SCD). The variant found was validated and segregated by polymerase chain reaction and Sanger sequencing. Based on the literature review, KCNH2 variants were retrospectively analyzed to identify pathogenic variants, likely pathogenic variants, and variants of uncertain significance by using different prediction tools.ResultsWES identified an autosomal dominant nonsense variant, c.1425C>A: p.Tyr475Ter, in the KCNH2 gene, which appeared to be the most likely cause of LQTS in this pedigree. Moreover, our comprehensive literature review yielded 511 KCNH2 variants in association with the LQTS phenotype, with c.3002G>A (CADD Phred=49) being the most pathogenic variant.ConclusionsVariants in the KCNH2 gene are considered a major cause of LQTS worldwide. The detected c.1425C>A is a novel variant to be reported from Iran for the first time. This result indicates the importance of KCNH2 screening in a pedigree with SCD cases.

### INTRODUCTION

Long QT syndrome (LQTS), as an arrhythmic inherited disorder, is determined by the QT interval prolongation in the electrocardiogram (ECG), syncope, and sudden cardiac death (SCD) owing to ventricular tachyarrhythmia.
1
, 
2
 LQTS is caused by a late potassium ion input or an increase in the sodium/calcium membrane channel, followed by a repolarizing power reduction. The most common cause of LQTS is a genetic mutation. KCNQ1 (potassium channel), HERG/KCNH2 (potassium channel), and SCN5A (sodium channel) are the main genes leading to LQT1, LQT2, and LQT3, respectively.
3
 Approximately, 45% of LQTS cases result from potassium channel changes (i.e., KCNQ1 and KCNH2). KCNE1, KCNE2, KCNJ2, and CACNA1c, which encode cardiac ion channel subunits, are also correlated with LQTS.
4

KCNH2 maps to chromosome 7q36.1 and has 19 exons. This gene encodes a voltage‐gated potassium ion channel protein, Kv11.1, which leads to cardiomyocyte repolarization.
1
 Thus far, KCNH2 variants have been reported in LQTS, sudden death, and arrhythmia phenotypes.
5
, 
6
, 
7
 The loss‐of‐function variant of KCNH2, as the second cause of LQT2 (MIM: #613688), makes the Kv11.1 protein dysfunction, which is followed by the loss of plasma membrane trafficking and, then, a reduction in cell‐surface functional channels.
8

The emergence of new genetic diagnostic tools such as next‐generation sequencing (NGS) has augmented the detection of disease etiology and the understanding of genotype/phenotype relationships. In the present study, using whole‐exome sequencing, we identified a novel pathogenic nonsense c.1425C>A: p.Tyr475Ter variant in KCNH2, which was the most likely cause of LQT2 in an Iranian family. In addition, we conducted a comprehensive review on all reported KCNH2 variants in patients with LQTS. To our knowledge, the current study is the first report of this heterozygous variant in the KCNH2 gene the world over.

### METHODS

The current investigation recruited a 3‐generation Iranian pedigree (Figure 1A) with 2 SCDs and 2 suspected individuals with a history of several fainting episodes. The suspected proband, a 52‐year‐old woman (II‐5; Figure 1A), was referred to our center, Cardiogenetics Research Center, Rajaie Cardiovascular Medical and Research Center, Iran University of Medical Sciences, Tehran, Iran, for genetic testing. According to the proband, her mother (I‐2; Figure 1A) and sister (II‐3; Figure 1A) had SCD aged 25 and 39 years, respectively. The proband had experienced fainting at 28 years of age during pregnancy, after which time she suffered another 3 episodes of fainting. The proband's daughter (III‐4; Figure 1A) had a similar experience to her mother (II‐5; Figure 1A) in that she became pregnant at age 30. The echocardiographic examinations of the mother and the daughter were normal (Figure 1C); nonetheless, the Holter monitoring test showed nonsustained ventricular tachycardias.

The image presents the pedigree, the chromatogram, and the electrocardiogram. (A) The LQTS family pedigree is presented herein. The black symbols indicate the affected members, and white symbols denote the normal members. The arrow represents the proband. (B) The image illustrates the chromatograms of a novel pathogenic variant, c.C1425A, in the KCNH2 gene detected in the affected members of the pedigree. (C) The image demonstrates the electrocardiogram of the proband (II‐5), as well as the suspected (III‐4) and normal (II‐1) individuals. The QTs in II‐5 and III‐4 were 480 ms and 476 ms, respectively, and 425 ms in II‐1. A bifid T wave was also observed in II‐5.

The study was conducted based on the ethical standards of the Declaration of Helsinki, and the study protocol was approved by the Ethics Committee of Rajaie Cardiovascular Medical and Research Center, Iran University of Medical Sciences, Tehran, Iran (IR.RHC.REC.1400.113). Written informed consent was obtained from all the available members of the pedigree.

Electrophysiological examination was performed by a 2‐electrode voltage‐clamp amplifier (TEC10CD, NPI Electronics) featuring KCl‐filled electrodes of about 0.8 MU resistance.
9
 The currents were measured at room temperature by applying bath solutions, including 105 mM of NaCl, 10 mM of KCl, 1.8 mM of CaCl2, and 10 mM of HEPES (pH 7.2). The QT interval was measured manually via the Bazett formula.

The genomic DNAs of the proband (II‐5; Figure 1A) and all the available family members were extracted from peripheral blood by using the DNSol Midi kit (Roche: Product No. 50072012). WES was done on the affected proband (II‐5; Figure 1A). Exome was enriched with the aid of an Agilent SureSelect All Exon V6 kit, and the library was sequenced on an Illumina HiSeq 4000 with a 100 average coverage depth. An in‐house setup pipeline, comprising read alignments to the human reference genome (GRCh37/hg19) (BWA: http://bio‐bwa.sourceforge.net/), the variant calling of single‐nucleotide variants and insertions/deletions (GATK: http://www.broadinstitute.org/gatk/), annotation (ANNOVAR: http://annovar.openbioinformatics.org/), propitiation, and filtering, was applied. All variants with minor allele frequencies of less than 1% in the 1000 Genomes Project (http://www.1000genomes.org/), the Exome Aggregation Consortium (ExAC: http://exac.broadinstitute.org/), the Genome Aggregation Database (gnomAD: http://gnomad.broadinstitute.org/), and the Exome Sequencing Project (ESP: http://evs.gs.washington.edu/EVS/) were considered.

Our results yielded only one novel pathogenic variant, c.C1425A, in the KCNH2 gene (NM_000238.4).

The consequence prediction of c.1425C>A: p.Tyr475Ter was obtained from online tools, namely CADD (http://cadd.gs.washington.edu/home) and MutationTaster (http://www.mutationtaster.org/). Additionally, ClustalW (http://www.genome.jp/tools‐bin/clustalw) was used to check the conservation in the variant region, and the American College of Medical Genetics standards were applied to interpret the sequence variants.
10
 Segregation analysis was conducted on the available members of the pedigree. To that end, primers were designed surrounding the identified variant by using the Primer3 version 0.4.0 server (http://bioinfo.ut.ee/primer3‐0.4.0/). Polymerase chain reaction (PCR) was performed with the aid of a SimpliAmp Thermal Cycler (Thermo Fisher Scientific) with 10 pmol/L primers (forward primer: TGCTGCTGGTCATCTACACG and reverse primer: CAGAGCGAGCATCAGAGGT), 100 ng of DNA, 200 mmol/L of dNTP, 1.5 mmol/L of MgCl2, and 1 U of Taq DNA polymerase (Amplicon). The PCR schedule was incubation at 95°C for 5 min, and then 35 cycles (40 s at 95°C, 30 s at 59°C, and 30 s at 72°C). Subsequently, the PCR products were sequenced on an ABI Sequencer 3500XL PE (Applied Biosystems) and Codon Code Aligner version 7.1.2 (CodonCode Corp).

For the evaluation of variant effects, structural modeling (3D) of the KCNH2 protein was applied by using the SWISS‐MODEL server (http://swissmodel.expasy.org/). The FASTA (canonical) sequence of the KCNH2 protein was obtained from the UniProt online database (UniProt ID: Q12809) (http://www.uniprot.org/) and entered as an input. The SWISS‐MODEL makes structures of target proteins and includes conserved ligands by drawing upon HHblits and BLAST. It displays an estimated model quality per residue based on a Qualitative Model Energy ANalysis (QMEAN) score, and CAMEO checks the SWISS‐MODEL accuracy. Additionally, the Berkeley Drosophila Genome Project (BDGP) (http://www.fruitfly.org/seq_tools/splice.html) and the Human Splicing Finder online server (HSF) V.3.1 (http://umd.be/Redirect.html) were applied to analyze variant consequences on splicing.

Variants reported in ClinVar (http://www.ncbi.nlm.nih.gov/clinvar/) and the Human Gene Mutation Database (HGMD) (http://www.hgmd.cf.ac.uk/ac/index.php) were surveyed up to September 2022. In addition, a combination of the keywords “KCNH2” and “long QT syndrome” and “KCNH2 mutations” and “KCNH2” [title/abstract] was applied by searching PubMed and Google Scholar. The variants consisted of pathogenic variants, likely pathogenic variants, and variants of unknown significance. All the collected variants were analyzed with different bioinformatics tools such as CADD, SIFT, PolyPhen‐2, PROVEAN, and MutationTaster (Table 1). The inclusion criterion was the existence of LQTS‐related variants considered damaging by at least one prediction tool.

Bioinformatics analysis of pathogenic, likely pathogenic, unknown significance reported variants in KCNH2 gene related to long QT.

c.1243‐1256delCTGC

TGCTGGTCAT

p.R176Afs*14

P.A244Sfs*112

P.R1032Afs*21

Note: NM_000238.4.

Abbreviations: B, Benign; CADD, Phred≤20, Neutral; CP, Conflicting interpretations of pathogenicity; D, Deleterious; D, Deleterious; DC, Disease Causing; LB, Likely benign; LC, Low complexity; LP, Likely pathogenetic; N, Neutral; NA, Not Available; P, Pathogenic; PD, Possibly Damaging; Phred>20, Damaging; PO, Polymorphism; POD, Possibly Damaging; T, Tolerated; TR, Transmembrane; VUS, Uncertain signicivance.

### Clinical presentation and ethics

The current investigation recruited a 3‐generation Iranian pedigree (Figure 1A) with 2 SCDs and 2 suspected individuals with a history of several fainting episodes. The suspected proband, a 52‐year‐old woman (II‐5; Figure 1A), was referred to our center, Cardiogenetics Research Center, Rajaie Cardiovascular Medical and Research Center, Iran University of Medical Sciences, Tehran, Iran, for genetic testing. According to the proband, her mother (I‐2; Figure 1A) and sister (II‐3; Figure 1A) had SCD aged 25 and 39 years, respectively. The proband had experienced fainting at 28 years of age during pregnancy, after which time she suffered another 3 episodes of fainting. The proband's daughter (III‐4; Figure 1A) had a similar experience to her mother (II‐5; Figure 1A) in that she became pregnant at age 30. The echocardiographic examinations of the mother and the daughter were normal (Figure 1C); nonetheless, the Holter monitoring test showed nonsustained ventricular tachycardias.

The image presents the pedigree, the chromatogram, and the electrocardiogram. (A) The LQTS family pedigree is presented herein. The black symbols indicate the affected members, and white symbols denote the normal members. The arrow represents the proband. (B) The image illustrates the chromatograms of a novel pathogenic variant, c.C1425A, in the KCNH2 gene detected in the affected members of the pedigree. (C) The image demonstrates the electrocardiogram of the proband (II‐5), as well as the suspected (III‐4) and normal (II‐1) individuals. The QTs in II‐5 and III‐4 were 480 ms and 476 ms, respectively, and 425 ms in II‐1. A bifid T wave was also observed in II‐5.

The study was conducted based on the ethical standards of the Declaration of Helsinki, and the study protocol was approved by the Ethics Committee of Rajaie Cardiovascular Medical and Research Center, Iran University of Medical Sciences, Tehran, Iran (IR.RHC.REC.1400.113). Written informed consent was obtained from all the available members of the pedigree.

### Electrocardiography

Electrophysiological examination was performed by a 2‐electrode voltage‐clamp amplifier (TEC10CD, NPI Electronics) featuring KCl‐filled electrodes of about 0.8 MU resistance.
9
 The currents were measured at room temperature by applying bath solutions, including 105 mM of NaCl, 10 mM of KCl, 1.8 mM of CaCl2, and 10 mM of HEPES (pH 7.2). The QT interval was measured manually via the Bazett formula.

### DNA extraction and WES

The genomic DNAs of the proband (II‐5; Figure 1A) and all the available family members were extracted from peripheral blood by using the DNSol Midi kit (Roche: Product No. 50072012). WES was done on the affected proband (II‐5; Figure 1A). Exome was enriched with the aid of an Agilent SureSelect All Exon V6 kit, and the library was sequenced on an Illumina HiSeq 4000 with a 100 average coverage depth. An in‐house setup pipeline, comprising read alignments to the human reference genome (GRCh37/hg19) (BWA: http://bio‐bwa.sourceforge.net/), the variant calling of single‐nucleotide variants and insertions/deletions (GATK: http://www.broadinstitute.org/gatk/), annotation (ANNOVAR: http://annovar.openbioinformatics.org/), propitiation, and filtering, was applied. All variants with minor allele frequencies of less than 1% in the 1000 Genomes Project (http://www.1000genomes.org/), the Exome Aggregation Consortium (ExAC: http://exac.broadinstitute.org/), the Genome Aggregation Database (gnomAD: http://gnomad.broadinstitute.org/), and the Exome Sequencing Project (ESP: http://evs.gs.washington.edu/EVS/) were considered.

Our results yielded only one novel pathogenic variant, c.C1425A, in the KCNH2 gene (NM_000238.4).

### Bioinformatics and segregation analysis

The consequence prediction of c.1425C>A: p.Tyr475Ter was obtained from online tools, namely CADD (http://cadd.gs.washington.edu/home) and MutationTaster (http://www.mutationtaster.org/). Additionally, ClustalW (http://www.genome.jp/tools‐bin/clustalw) was used to check the conservation in the variant region, and the American College of Medical Genetics standards were applied to interpret the sequence variants.
10
 Segregation analysis was conducted on the available members of the pedigree. To that end, primers were designed surrounding the identified variant by using the Primer3 version 0.4.0 server (http://bioinfo.ut.ee/primer3‐0.4.0/). Polymerase chain reaction (PCR) was performed with the aid of a SimpliAmp Thermal Cycler (Thermo Fisher Scientific) with 10 pmol/L primers (forward primer: TGCTGCTGGTCATCTACACG and reverse primer: CAGAGCGAGCATCAGAGGT), 100 ng of DNA, 200 mmol/L of dNTP, 1.5 mmol/L of MgCl2, and 1 U of Taq DNA polymerase (Amplicon). The PCR schedule was incubation at 95°C for 5 min, and then 35 cycles (40 s at 95°C, 30 s at 59°C, and 30 s at 72°C). Subsequently, the PCR products were sequenced on an ABI Sequencer 3500XL PE (Applied Biosystems) and Codon Code Aligner version 7.1.2 (CodonCode Corp).

### Structural modeling and variant effects on splicing

For the evaluation of variant effects, structural modeling (3D) of the KCNH2 protein was applied by using the SWISS‐MODEL server (http://swissmodel.expasy.org/). The FASTA (canonical) sequence of the KCNH2 protein was obtained from the UniProt online database (UniProt ID: Q12809) (http://www.uniprot.org/) and entered as an input. The SWISS‐MODEL makes structures of target proteins and includes conserved ligands by drawing upon HHblits and BLAST. It displays an estimated model quality per residue based on a Qualitative Model Energy ANalysis (QMEAN) score, and CAMEO checks the SWISS‐MODEL accuracy. Additionally, the Berkeley Drosophila Genome Project (BDGP) (http://www.fruitfly.org/seq_tools/splice.html) and the Human Splicing Finder online server (HSF) V.3.1 (http://umd.be/Redirect.html) were applied to analyze variant consequences on splicing.

### Search strategy and data extraction

Variants reported in ClinVar (http://www.ncbi.nlm.nih.gov/clinvar/) and the Human Gene Mutation Database (HGMD) (http://www.hgmd.cf.ac.uk/ac/index.php) were surveyed up to September 2022. In addition, a combination of the keywords “KCNH2” and “long QT syndrome” and “KCNH2 mutations” and “KCNH2” [title/abstract] was applied by searching PubMed and Google Scholar. The variants consisted of pathogenic variants, likely pathogenic variants, and variants of unknown significance. All the collected variants were analyzed with different bioinformatics tools such as CADD, SIFT, PolyPhen‐2, PROVEAN, and MutationTaster (Table 1). The inclusion criterion was the existence of LQTS‐related variants considered damaging by at least one prediction tool.

Bioinformatics analysis of pathogenic, likely pathogenic, unknown significance reported variants in KCNH2 gene related to long QT.

c.1243‐1256delCTGC

TGCTGGTCAT

p.R176Afs*14

P.A244Sfs*112

P.R1032Afs*21

Note: NM_000238.4.

Abbreviations: B, Benign; CADD, Phred≤20, Neutral; CP, Conflicting interpretations of pathogenicity; D, Deleterious; D, Deleterious; DC, Disease Causing; LB, Likely benign; LC, Low complexity; LP, Likely pathogenetic; N, Neutral; NA, Not Available; P, Pathogenic; PD, Possibly Damaging; Phred>20, Damaging; PO, Polymorphism; POD, Possibly Damaging; T, Tolerated; TR, Transmembrane; VUS, Uncertain signicivance.

### RESULTS

The ECG of the normal (II‐1) and affected (II‐5 and III‐4) individuals (Figure 1C) illustrated borderline QTs in II‐5 and III‐4 (480 ms and 476 ms, respectively) and 425 ms QT in II‐1. A bifid T wave was also observed in II‐5.

The WES analysis detected a novel pathogenic variant in the heterozygous state, c.1425C>A: p.Tyr475Ter in exon 6 of KCNH2 in an Iranian pedigree suspected of LQT2 (Figure 1A). The heterozygote/homozygote pathogenic/normal states for this position in the available members of the pedigree were confirmed by PCR and Sanger sequencing (Figure 1B). According to the segregation analysis results, LQT2 was inherited in this pedigree with an autosomal dominant pattern (Figure 1A). The CADD Phred of the c.1425C>A variant was 32. Based on the American College of Medical Genetics guideline, PVS1 was the null variant in the KCNH2 gene as a loss‐of‐function variant and was associated with LQT2, and PM2 was a variant not found in gnomAD exomes/genomes. Furthermore, PP3 was a pathogenic computational verdict based on 3 pathogenic predictions from BayesDel addAF, FATHMM‐MKL, and MutationTaster. The investigation by BDGP and HSF indicated that this valiant did not affect the splicing process. Moreover, the alignment of the KCNH2 variant region displayed its highly conservative position among the various species.

A 3D model of wild and truncated KCNH2 proteins was obtained through SWISS‐MODEL. The best model of KCNH2 was provided with 5va3 by SWISS‐MODEL experimental structures with a QMEAN score of 0.75 ± 0.05, which suggested that the model was more definitive with a low local error score. Given the destruction of KCNH2 in the early steps of protein synthesis, resulting from the p.Tyr475Ter variant, no mutant structure model was achieved (Figure 3A).

Our search strategy and data extraction led to the collection of 511 variants causing LQTS. The characteristics of these variants are presented in Table 1, according to which most variants (353 variants) were in the low complexity region. The highest CADD score was for c.3002G>A (Phred = 49), followed by c.3157G>T and c.3208C>T (Phred = 47), as well as c.3040C>T (Phred = 46). These variants were in the low complexity region.

### Clinical and genetic findings

The ECG of the normal (II‐1) and affected (II‐5 and III‐4) individuals (Figure 1C) illustrated borderline QTs in II‐5 and III‐4 (480 ms and 476 ms, respectively) and 425 ms QT in II‐1. A bifid T wave was also observed in II‐5.

The WES analysis detected a novel pathogenic variant in the heterozygous state, c.1425C>A: p.Tyr475Ter in exon 6 of KCNH2 in an Iranian pedigree suspected of LQT2 (Figure 1A). The heterozygote/homozygote pathogenic/normal states for this position in the available members of the pedigree were confirmed by PCR and Sanger sequencing (Figure 1B). According to the segregation analysis results, LQT2 was inherited in this pedigree with an autosomal dominant pattern (Figure 1A). The CADD Phred of the c.1425C>A variant was 32. Based on the American College of Medical Genetics guideline, PVS1 was the null variant in the KCNH2 gene as a loss‐of‐function variant and was associated with LQT2, and PM2 was a variant not found in gnomAD exomes/genomes. Furthermore, PP3 was a pathogenic computational verdict based on 3 pathogenic predictions from BayesDel addAF, FATHMM‐MKL, and MutationTaster. The investigation by BDGP and HSF indicated that this valiant did not affect the splicing process. Moreover, the alignment of the KCNH2 variant region displayed its highly conservative position among the various species.

### Structural analysis

A 3D model of wild and truncated KCNH2 proteins was obtained through SWISS‐MODEL. The best model of KCNH2 was provided with 5va3 by SWISS‐MODEL experimental structures with a QMEAN score of 0.75 ± 0.05, which suggested that the model was more definitive with a low local error score. Given the destruction of KCNH2 in the early steps of protein synthesis, resulting from the p.Tyr475Ter variant, no mutant structure model was achieved (Figure 3A).

### Literature review findings

Our search strategy and data extraction led to the collection of 511 variants causing LQTS. The characteristics of these variants are presented in Table 1, according to which most variants (353 variants) were in the low complexity region. The highest CADD score was for c.3002G>A (Phred = 49), followed by c.3157G>T and c.3208C>T (Phred = 47), as well as c.3040C>T (Phred = 46). These variants were in the low complexity region.

### DISCUSSION

In the current study, we detected that the KCNH2 c.1425C>A (p.Tyr475Ter) novel variant was related to the LQT2 phenotype in an Iranian pedigree. The most common LQTS‐causing genes are KCNH2, KCNQ1, KCNE1, KCNE2, and SCN5A.
1
 Napolitano et al.
11
 screened 430 patients with LQTS for 5 common genes and reported that 322 individuals had positive genetic tests. Tester et al.
6
 reported a 75% yield among patients with LQTS for genetic testing. These studies underscore the necessity of genetic testing for patients suspected of LQTS and their family members. Research have also demonstrated that KCNQ1 causes LQT1, KCNH2 LQT2, SCN5A LQT3, KCNE1 LQT5, KCNE2 LQT6, KCNJ2 LQT7, and CACNA1c LQT8.
3
, 
12
, 
13
 Strong genotype–phenotype correlations have also been reported in LQTS. In other words, there are auditory triggers and/or postpartum accompanying LQT2 and swimming with LQT1, somewhat elucidating the LQTS genetic etiology and improving efficient treatment.
14

The secondary structure of the KCNH2 protein, Kv11.1, is a tetrameric protein with low complexity regions, PAS, PAC, and 1 transmembrane domain per subunit (Figure 2A). As is shown in Table 1, regarding KCNH2, the most pathogenic variant, likely pathogenic variant, and variants of unknown significance were located in low complexity regions. The identified p.Tyr475Ter variant was also located in the low complexity region and seemed to affect the Kv11.1 channel or the plasma membrane traffic. Structural analysis indicated the altered amino acid, Tyrosine 475, was highly conserved in the protein sequence among the species (Figure 2B). Hence, this variant was predicted to be damaging. The 3D structure of the Kv11.1 protein has 6 transmembrane α helices (S1–S6), N and C terminals situated in the cytoplasm, and a pore located between S5 and S6 (Figure 3). A total of 511 variants were found in KCNH2 that could cause LQTS by different mechanisms such as defects in ion permeation, gating, trafficking from the endoplasmic reticulum to the plasma membrane, and HERG synthesis reduction, all of which impact Ikr, the main cardiac repolarization current.
15
, 
16
 Nonsense variants comprised a small percentage of these variants (39/511, 7.6%); still, they exerted a more destructive effect on protein, especially if this cessation of protein synthesis occurred in the early stages of synthesis. The p.Tyr475Ter variant was in the early low complexity region and made truncated Kv11.1 protein (Figure 1A).

A schematic view of the KCNH2 gene and the KCNH2 protein. (A) The image depicts the location of KCNH2 on the chromosome and the variant location in the exon. The protein's secondary structure was obtained from SMART web server (https://smart.embl.de/smart/show_motifs.pl?ID=Q12809. The blue boxes are the transmembrane domains, and the pink boxes are the low complexity regions. (B) The image demonstrates the conservation of the p.Tyr475Ter amino acid in the KCNH2 protein among the species.

The 3D structure of the KCNH2 protein. (A) The image shows the SWISS‐MODEL homology modeling of wild (right) and truncated (left) KCNH2. (B) The schematic view of the domains of KCNH2, as a transmembrane protein, is illustrated herein.

The present study indicates the potential effects of KCNH2 changes on the LQT2 phenotype; nevertheless, it suffers from some limitations. First, the DNA samples of the individuals who had SCD (I‐2 and II‐3) were not available for a further survey of genotype–phenotype correlations. Second, although our bioinformatics analysis demonstrated the pathogenic effect of c.1425C>A: p.Tyr475Ter, functional evaluation could have helped to confirm this pathogenicity.

### CONCLUSIONS

We have presented the first genetic and clinical assessment of the p.Tyr475Ter KCNH2 variant.

This finding indicates the importance of potassium support and monitoring in patients with LQTS, especially those developing Kv11.1 dysfunction resulting from loss‐of‐function variants. NGS has significant potential for clinical and research use. We suggest the application of the WES approach for individuals suspected of LQTS with a family history of sudden death whose genetic etiology has remained unidentified.

### AUTHOR CONTRIBUTIONS

Samira Kalayinia, Amir Farjam Fazelifar, and Majid Maleki wrote the article. Maryam Pourirahim and Tannaz Masoumi carried out the experiments. Amir Farjam Fazelifar, Alireza Biglari, and Majid Maleki contributed to the patient's diagnosis. Samira Kalayinia performed WES analysis data interpretation. All the authors read and approved the final manuscript.

### FUNDING INFORMATION

The authors received no specific funding for this research.

### CONFLICT OF INTEREST STATEMENT

The authors declare that they have no competing interests.

### ETHICS APPROVAL AND CONSENT TO PARTICIPATE

The study protocol was approved by the Ethics Committee of Rajaie Cardiovascular Medical and Research Center, Iran University of Medical Sciences, Tehran, Iran (IR.RHC.REC.1400.113). Written informed consent was obtained from all the available members of the pedigree.

### CONSENT FOR PUBLICATION

Not applicable.



# SUPPLEMENTAL FILE 1: JOA3-39-430.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)