# MAIN TEXT

## Cardiac Genetic Investigation of Sudden Infant and Early Childhood Death: A Study From Victims to Families

### Abstract

BackgroundSudden infant death syndrome (SIDS) is the leading cause of death up to age 1. Sudden unexplained death in childhood (SUDC) is similar but affects mostly toddlers aged 1 to 4. SUDC is rarer than SIDS, and although cardiogenetic testing (molecular autopsy) identifies an underlying cause in a fraction of SIDS, less is known about SUDC.Methods and ResultsSeventy‐seven SIDS and 16 SUDC cases underwent molecular autopsy with 25 definitive‐evidence arrhythmia‐associated genes. In 18 cases, another 76 genes with varying degrees of evidence were analyzed. Parents were offered cascade screening. Double‐blind review of clinical‐genetic data established genotype–phenotype correlations. The yield of likely pathogenic variants in the 25 genes was higher in SUDC than in SIDS (18.8% [3/16] versus 2.6% [2/77], respectively; P=0.03), whereas novel/ultra‐rare variants of uncertain significance were comparably represented. Rare variants of uncertain significance and likely benign variants were found only in SIDS. In cases with expanded analyses, likely pathogenic/likely benign variants stemmed only from definitive‐evidence genes, whereas all other genes contributed only variants of uncertain significance. Among 24 parents screened, variant status and phenotype largely agreed, and 3 cases positively correlated for cardiac channelopathies. Genotype–phenotype correlations significantly aided variant adjudication.ConclusionsGenetic yield is higher in SUDC than in SIDS although, in both, it is contributed only by definitive‐evidence genes. SIDS/SUDC cascade family screening facilitates establishment or dismissal of a diagnosis through definitive variant adjudication indicating that anonymity is no longer justifiable. Channelopathies may underlie a relevant fraction of SUDC. Binary classifications of genetic causality (pathogenic versus benign) could not always be adequate.

### What Is New?

Genetic arrhythmias underlie an important fraction of sudden unexplained death in childhood (SUDC).The cardiogenetic yield of molecular autopsy both in SUDC and sudden infant death syndrome stems mostly from definitive‐evidence genes associated with inherited arrhythmia syndromes.Cascade screening of sudden infant death syndrome/SUDC parents facilitates establishment or dismissal of a diagnosis through definitive variant adjudication and, when appropriate, allows the implementation of preventive measures.

Genetic arrhythmias underlie an important fraction of sudden unexplained death in childhood (SUDC).

The cardiogenetic yield of molecular autopsy both in SUDC and sudden infant death syndrome stems mostly from definitive‐evidence genes associated with inherited arrhythmia syndromes.

Cascade screening of sudden infant death syndrome/SUDC parents facilitates establishment or dismissal of a diagnosis through definitive variant adjudication and, when appropriate, allows the implementation of preventive measures.

### What Are the Clinical Implications?

Cardiogenetic investigations focused specifically on SUDC may be particularly fruitful and may eventually aid in the identification of time‐sensitive factors that could operate differently after the first year of life, thus shaping different outcomes (sudden infant death syndrome or SUDC).The molecular autopsy benefits most from the interrogation of definitive‐evidence genes associated with inherited cardiac arrhythmias, such as long QT syndrome and catecholaminergic polymorphic ventricular tachycardia.Case anonymity in sudden infant death syndrome research is no longer justifiable, given the potential implications, both medical and psychological, for the families left behind.

Cardiogenetic investigations focused specifically on SUDC may be particularly fruitful and may eventually aid in the identification of time‐sensitive factors that could operate differently after the first year of life, thus shaping different outcomes (sudden infant death syndrome or SUDC).

The molecular autopsy benefits most from the interrogation of definitive‐evidence genes associated with inherited cardiac arrhythmias, such as long QT syndrome and catecholaminergic polymorphic ventricular tachycardia.

Case anonymity in sudden infant death syndrome research is no longer justifiable, given the potential implications, both medical and psychological, for the families left behind.

### Methods

The data that support the findings of this study are available from the corresponding author upon reasonable request.

The study was supported and organized by the Lullaby Trust (London, UK), a registered charity operating nationwide across England and Wales. The study's main partners were the Sheffield Teaching Hospitals and Sheffield Children's National Health Service Foundation Trust (SCH) in Sheffield, UK, and the Istituto Auxologico Italiano (IAI), Istituto di Ricovero e Cura a Carattere Scientifico, in Milan, Italy. Case samples were obtained through consecutive referrals from the Lullaby Trust, originating either from coroners at SCH or from other collaborating hospitals in the United Kingdom. SCH initially provided to the IAI linked anonymized tissue samples obtained during autopsy of victims with SIDS/SUDC, whose parents had consented for the tissue to be used in the joint research study conducted by the 2 partners.

The study population comprised 104 unrelated cases of infants and young children, collected between 2011 and 2018, whose death was classified as SIDS or SUDC according to the coroners' reports and for whom comprehensive autopsies, including histological, toxicological, and microbiological examinations, were performed, in addition to evaluations of medical and family history, and to investigations as to the circumstances of death. Self‐reported race was White (specifically, White‐British, according to the ethnic groups adopted by the 2021 Census of England and Wales) in all but 5 cases for which race or ethnicity information was not available.

The study was approved by the respective institutional review boards. Bilateral material transfer agreements were established for the transfer of samples, written informed consent was obtained from all participants, and the study protocol complied with the Declaration of Helsinki.

Genomic DNA was extracted from frozen tissue samples obtained during autopsy with the QIAamp DNA Mini kit according to the manufacturer's recommendations (Qiagen). DNA concentrations were measured fluorometrically with the Quant‐iT PicoGreen dsDNA assay kit (Invitrogen). Genetic testing was performed through next‐generation sequencing (NGS) on a MiSeq platform (Illumina) mainly with a custom
18
 amplicon NGS panel (TruSeq Custom Amplicon, TSCA, Illumina) targeting 21 channelopathies and cardiomyopathy genes with mostly definitive or strong evidence of association to inherited cardiac diseases,
19
, 
20
, 
21
, 
22
, 
23
, 
24
 such as LQTS, CPVT, BrS, arrhythmogenic cardiomyopathy etc, as previously described
18
, 
25
 (Table S1). Moreover, the 3 calmodulin genes CALM1, CALM2, and CALM3, as well as the gap junction protein alpha 1 GJA1 gene, were screened with NGS in all samples through a Nextera XT assay (Illumina; Table S1).

The last 18 samples enrolled in the study were sequenced with a commercially available expanded NGS panel (TruSight Cardio, Illumina) and, in addition to the 25 genes in common with the custom NGS panel/assay, another 76 genes with varying degrees of disease association
20
, 
21
, 
22
, 
23
, 
24
 were also analyzed (Table S2). The only definitive‐evidence nonsyndromic genes not targeted by this panel were FLNC and TECRL. In‐house pipelines were created according to Broad Institute's Genome Analysis Toolkit Best Practices recommendations
26
 with custom scripts in Bash (Data S1).

Several databases and web‐based resources were used for the prioritization and classification of genetic variants (Data S1). Allelic frequencies were obtained through the 1000 Genomes browser and the gnomAD browser (v.2.1.1). As a first step, the genetic data set was interrogated for rare nonsynonymous variants with a minor allele frequency (MAF) that fell below the SIDS incidence (ie, 0.3:1000 live births), as reported by the United Kingdom's Office of National Statistics for the year that enrollment in the study was concluded.
27
 This resulted in a first‐step filter of MAF<0.00015 for rare variants, and ultra‐rare genetic variants with a MAF<0.00005 were prioritized at a second step. In the additional 76 genes of the expanded NGS analysis, only ultra‐rare variants were assessed. Variant status and potential clinical significance were evaluated with the use of several publicly available and licensed databases, in silico pathogenicity meta scores, as well as published literature (Data S1). Classification of genetic variants was performed in accordance with the American College of Medical Genetics guidelines.
28
 All rare genetic variants identified by NGS were validated with Sanger DNA sequencing at the IAI in Milan, Italy, and in some cases were further validated by the Sheffield Diagnostic Genetics Service in Sheffield, UK, before being communicated to the victims' families.

As per the study's protocol, the parents of victims with SIDS/SUDC consented to be informed of the results of the study in case a genotype‐positive genetic test result were obtained and opted for the possibility to be called back to undergo clinical and genetic cascade family screening. Most parents opted to be informed of the test result even if the results were considered inconclusive. For this purpose, SCH maintained a joint follow‐up SIDS/SUDC clinic toward the conclusion of the study where parents were informed of the genetic tests results of the deceased infants, were subjected to clinical examinations, or were referred to perform these at their place of residence, while also providing DNA samples for cascade genetic screening. Clinical examinations included an ECG and, depending on the case, 24‐hour Holter ECG, signal‐averaged ECG, echocardiogram, and exercise stress test (EST). Parental DNA samples were sent for genetic screening at the IAI in Milan, Italy, and targeted variant analysis was performed through direct Sanger sequencing. Parental genetic results were then again validated by the Sheffield Diagnostic Genetics Service in Sheffield, UK.

At the conclusion of all genetic analyses, carrier status of the parents for the variants found in the SIDS/SUDC cases were communicated to the SCH partners while the latter shared with the IAI the clinical data that had been obtained from the clinical family screening. At that stage, a double‐blind review of the clinical and genetic data was performed. Genetic and clinical data were first reviewed separately for each genotype‐positive case and the respective parents. The reviewers of the clinical data (P.J.S. and L.C.) were blinded as to the gene or the classification status of the genetic variant involved, as well as to its parental origin, and the reviewers of the genetic data (M.C.K. and M.T.) were blinded as to the clinical status of either parent. This resulted, for each case, in the simultaneous and reciprocal disclosure of the potential diagnosis, suspicion, or dismissal of a cardiac disease in a parent, with the disclosure of the genetic variant identified in the subject, its classification, and its parental origin.

Statistical analyses were performed using IBM SPSS Statistics version 28.0. Continuous variables are summarized as mean±SD, median, and range. Categorical variables are presented as absolute (n) and relative frequencies (%) and compared among groups with Fisher's exact test. A 2‐sided P value <0.05 was considered statistically significant. Binomial exact 95% CI are provided for the estimated genetic yield in SUDC versus SIDS cases.

### Study Population and Partners

The study was supported and organized by the Lullaby Trust (London, UK), a registered charity operating nationwide across England and Wales. The study's main partners were the Sheffield Teaching Hospitals and Sheffield Children's National Health Service Foundation Trust (SCH) in Sheffield, UK, and the Istituto Auxologico Italiano (IAI), Istituto di Ricovero e Cura a Carattere Scientifico, in Milan, Italy. Case samples were obtained through consecutive referrals from the Lullaby Trust, originating either from coroners at SCH or from other collaborating hospitals in the United Kingdom. SCH initially provided to the IAI linked anonymized tissue samples obtained during autopsy of victims with SIDS/SUDC, whose parents had consented for the tissue to be used in the joint research study conducted by the 2 partners.

The study population comprised 104 unrelated cases of infants and young children, collected between 2011 and 2018, whose death was classified as SIDS or SUDC according to the coroners' reports and for whom comprehensive autopsies, including histological, toxicological, and microbiological examinations, were performed, in addition to evaluations of medical and family history, and to investigations as to the circumstances of death. Self‐reported race was White (specifically, White‐British, according to the ethnic groups adopted by the 2021 Census of England and Wales) in all but 5 cases for which race or ethnicity information was not available.

The study was approved by the respective institutional review boards. Bilateral material transfer agreements were established for the transfer of samples, written informed consent was obtained from all participants, and the study protocol complied with the Declaration of Helsinki.

### Next‐Generation Sequencing

Genomic DNA was extracted from frozen tissue samples obtained during autopsy with the QIAamp DNA Mini kit according to the manufacturer's recommendations (Qiagen). DNA concentrations were measured fluorometrically with the Quant‐iT PicoGreen dsDNA assay kit (Invitrogen). Genetic testing was performed through next‐generation sequencing (NGS) on a MiSeq platform (Illumina) mainly with a custom
18
 amplicon NGS panel (TruSeq Custom Amplicon, TSCA, Illumina) targeting 21 channelopathies and cardiomyopathy genes with mostly definitive or strong evidence of association to inherited cardiac diseases,
19
, 
20
, 
21
, 
22
, 
23
, 
24
 such as LQTS, CPVT, BrS, arrhythmogenic cardiomyopathy etc, as previously described
18
, 
25
 (Table S1). Moreover, the 3 calmodulin genes CALM1, CALM2, and CALM3, as well as the gap junction protein alpha 1 GJA1 gene, were screened with NGS in all samples through a Nextera XT assay (Illumina; Table S1).

The last 18 samples enrolled in the study were sequenced with a commercially available expanded NGS panel (TruSight Cardio, Illumina) and, in addition to the 25 genes in common with the custom NGS panel/assay, another 76 genes with varying degrees of disease association
20
, 
21
, 
22
, 
23
, 
24
 were also analyzed (Table S2). The only definitive‐evidence nonsyndromic genes not targeted by this panel were FLNC and TECRL. In‐house pipelines were created according to Broad Institute's Genome Analysis Toolkit Best Practices recommendations
26
 with custom scripts in Bash (Data S1).

### Genetic Variant Analysis

Several databases and web‐based resources were used for the prioritization and classification of genetic variants (Data S1). Allelic frequencies were obtained through the 1000 Genomes browser and the gnomAD browser (v.2.1.1). As a first step, the genetic data set was interrogated for rare nonsynonymous variants with a minor allele frequency (MAF) that fell below the SIDS incidence (ie, 0.3:1000 live births), as reported by the United Kingdom's Office of National Statistics for the year that enrollment in the study was concluded.
27
 This resulted in a first‐step filter of MAF<0.00015 for rare variants, and ultra‐rare genetic variants with a MAF<0.00005 were prioritized at a second step. In the additional 76 genes of the expanded NGS analysis, only ultra‐rare variants were assessed. Variant status and potential clinical significance were evaluated with the use of several publicly available and licensed databases, in silico pathogenicity meta scores, as well as published literature (Data S1). Classification of genetic variants was performed in accordance with the American College of Medical Genetics guidelines.
28
 All rare genetic variants identified by NGS were validated with Sanger DNA sequencing at the IAI in Milan, Italy, and in some cases were further validated by the Sheffield Diagnostic Genetics Service in Sheffield, UK, before being communicated to the victims' families.

### Cascade Family Screening

As per the study's protocol, the parents of victims with SIDS/SUDC consented to be informed of the results of the study in case a genotype‐positive genetic test result were obtained and opted for the possibility to be called back to undergo clinical and genetic cascade family screening. Most parents opted to be informed of the test result even if the results were considered inconclusive. For this purpose, SCH maintained a joint follow‐up SIDS/SUDC clinic toward the conclusion of the study where parents were informed of the genetic tests results of the deceased infants, were subjected to clinical examinations, or were referred to perform these at their place of residence, while also providing DNA samples for cascade genetic screening. Clinical examinations included an ECG and, depending on the case, 24‐hour Holter ECG, signal‐averaged ECG, echocardiogram, and exercise stress test (EST). Parental DNA samples were sent for genetic screening at the IAI in Milan, Italy, and targeted variant analysis was performed through direct Sanger sequencing. Parental genetic results were then again validated by the Sheffield Diagnostic Genetics Service in Sheffield, UK.

### Double‐Blind Genotype–Phenotype Correlation

At the conclusion of all genetic analyses, carrier status of the parents for the variants found in the SIDS/SUDC cases were communicated to the SCH partners while the latter shared with the IAI the clinical data that had been obtained from the clinical family screening. At that stage, a double‐blind review of the clinical and genetic data was performed. Genetic and clinical data were first reviewed separately for each genotype‐positive case and the respective parents. The reviewers of the clinical data (P.J.S. and L.C.) were blinded as to the gene or the classification status of the genetic variant involved, as well as to its parental origin, and the reviewers of the genetic data (M.C.K. and M.T.) were blinded as to the clinical status of either parent. This resulted, for each case, in the simultaneous and reciprocal disclosure of the potential diagnosis, suspicion, or dismissal of a cardiac disease in a parent, with the disclosure of the genetic variant identified in the subject, its classification, and its parental origin.

### Statistical Analysis

Statistical analyses were performed using IBM SPSS Statistics version 28.0. Continuous variables are summarized as mean±SD, median, and range. Categorical variables are presented as absolute (n) and relative frequencies (%) and compared among groups with Fisher's exact test. A 2‐sided P value <0.05 was considered statistically significant. Binomial exact 95% CI are provided for the estimated genetic yield in SUDC versus SIDS cases.

### Results

The initial study population included 104 unrelated SIDS/SUDC cases. We applied stringently the age cutoff of 1 year as a criterion for a SIDS definition,
2
 while reserving the SUDC definition
15
 for all cases exceeding 1 year, even by just 1 day. In the whole cohort, there were 88 SIDS cases (56 male, 64%) with an age range at the time of death of 2 to 365 days (mean 105±88, median 76 days). The remaining 16 were SUDC cases (10 male, 62.5%), with an age range at the time of death of 12.0 to 54.1 months (mean 23.6±11.1, median 20.5 months). In 11 SIDS cases, DNA integrity was low, and this resulted in partial genetic analyses (Data S1).

Complete genetic analyses were possible in 93 SIDS/SUDC cases. These underwent NGS of 25 major arrhythmia‐associated genes (Table S1) with mostly definitive evidence of disease associations
19
, 
20
, 
21
, 
22
, 
23
, 
24
 or specifically implicated in SIDS.
29

In the SIDS cohort (n=77), NGS of the 25 arrhythmia‐associated genes yielded 25 rare nonsynonymous variants with a MAF<0.00015 in 24 SIDS cases (Table 1). Among these, there were 2 novel and 13 ultra‐rare (MAF<0.00005) variants in 15 cases. Most genetic variants were missense, with only 3 being non‐missense. In accordance with the American College of Medical Genetics guidelines,
28
 2 variants were classified as likely pathogenic (LP), 21 variants were classified as variants of uncertain significance (VUS), and 2 as likely benign (LB). Three VUS were further classified as favor nonpathogenic (FNP) according to expert judgment. Altogether, the yield of LP variants (n=2, 2.6%) was equal to that of novel VUS and of LB variants, with a higher yield of ultra‐rare and rare VUS (n=11, 14.3% and n=8, 10.4%, respectively; Figure 1A).

Novel, Ultra‐Rare, and Rare Genetic Variants in the 25 Major Genes Across the SIDS Cohort

Genetic variants are ordered with increasing allelic frequency, and nomenclature refers to the main transcript. In silico mostly refers to the pathogenicity meta‐scores MetaSVM and REVEL (Damaging‐D/Tolerated‐T and Pathogenic‐P/Benign‐B, respectively; see Methods). The most representative variant citations are given under PMID.

Denotes a secondary classification, according to expert judgment.

KCNH2‐p.E289K initially, under gnomAD v2.0.2, fell below the rare variant threshold set in the study but exceeded the cutoff under gnomAD v2.1.1 at the study's conclusion. Since cascade family screening had already been initiated, the variant is presented, without being considered in the overall yield (Data S1).

1000G indicates 1000 Genomes Project Browser; AC, allele count; ACMG, American College of Medical Genetics and Genomics; ClinVar, Clinical Significance of Variants public archive; dbSNP, Single Nucleotide Polymorphism database; FNP, favor nonpathogenic; GRCh37, Genome Reference Consortium Human Build 37; LB, likely benign; LP, likely pathogenic; MAF, minor allele frequency; PMID, PubMed IDentifier; SIDS, sudden infant death syndrome; and VUS, variant of uncertain significance.

A, Yield of genetic testing in the 77 SIDS and 18 SUDC cases for different variant classes across the 25 major arrhythmia‐associated genes. Statistically significant (P<0.05) comparisons are shown above the bars. B, Distribution of the 29 novel, ultra‐rare, and rare genetic variants across the 25 major genes that yielded >1 genetic variant in the whole cohort. Absolute numbers (n) and relative frequencies (%) are shown within and above the bars, respectively. LB indicates likely benign; LP, likely pathogenic; SIDS, sudden infant death syndrome; SUDC, sudden unexplained death in childhood; and VUS, variant of uncertain significance.

Genetic analyses of the 25 major genes in the SUDC cohort (n=16) yielded in total 3 novel and 5 ultra‐rare nonsynonymous variants in 7 SUDC cases (Table 2). Novel variants were contributed by the SCN5A, RYR2, and DSC2 genes in 3 cases. Most genetic variants were missense, with 2 being non‐missense. American College of Medical Genetics variant classification
28
 adjudicated 3 variants as LP and 5 (2 novel, 3 ultra‐rare) as VUS. Thus, the yield of LP variants in the SUDC cohort was 18.8% (3/16). Altogether, the yield of LP variants was equal to that of ultra‐rare VUS (n=3, 18.8%), with a lower yield of novel VUS (n=2, 12.5%), whereas there was no yield of rare VUS or LB variants (Figure 1A).

Novel, Ultra‐Rare, and Rare Genetic Variants in the 25 Major Genes Across the SUDC Cohort

Genetic variants are ordered with increasing allelic frequency, and nomenclature refers to the main transcript. In silico mostly refers to the pathogenicity meta‐scores MetaSVM and REVEL (Damaging‐D/Tolerated‐T and Pathogenic‐P/Benign‐B, respectively; see Methods). The most representative variant citations are given under PMID.

Denotes a secondary classification, according to expert judgment.

1000G indicates 1000 Genomes Project Browser; AC, allele count; ACMG, American College of Medical Genetics and Genomics; ClinVar, Clinical Significance of Variants public archive; dbSNP, Single Nucleotide Polymorphism database; GRCh37, Genome Reference Consortium Human Build 37; LP, likely pathogenic; MAF, minor allele frequency; PMID, PubMed IDentifier; SUDC, sudden unexplained death in childhood; and VUS, variant of uncertain significance.

Thus, among the 93 SIDS/SUDC cases, there was a significantly higher percentage of LP variants in SUDC cases with respect to SIDS (18.8% [95% CI, 4.0–45.6] versus 2.6% [95% CI, 0.32–9.1], respectively, P=0.03), whereas ultra‐rare VUS (18.8% [95% CI, 4.0–45.6] versus 14.3% [95% CI, 7.4–24.1] respectively, P=0.70) and novel VUS (12.5% [95% CI, 1.6–38.3] versus 2.6% [95% CI, 0.32–9.1], respectively, P=0.14) were comparably represented in the 2 groups. Of note, all the 10 rare VUS and LB variants across the 25 major genes (10.4% and 2.6%, respectively) were contributed exclusively by SIDS cases and none were found in the 16 SUDC cases (Figure 1A).

In the whole cohort, across the 25 major genes, there were 8 genes (RYR2, LMNA, MYBPC3, DSC2, DSP, MYH7, KCNQ1, KCNH2) that yielded >1 rare variant, for a total of 29 variants, mostly represented by ultra‐rare (n=16, 55.1%) and less frequently by rare (n=9, 31%) and novel variants (n=4, 13.8%). The RYR2 gene contributed most variants (n=7, 24.1%), of which 1 was novel, 5 were ultra‐rare, and 1 was rare (Figure 1B). In fact, ultra‐rare nonsynonymous RYR2 variants appeared to be overrepresented in the SIDS/SUDC cases with respect to the gnomAD population (RYR2 nonsynonymous ultra‐rare alleles, 6/186 [3.22%] in cases versus 3890/282 912 [1.37%] in controls; P=0.04). No genetic variants were identified in the CALM1‐3 and GJA1 genes.

For the last 18 cases enrolled in the study (15 SIDS, 3 SUDC), genetic analysis was extended to an additional 76 genes (Table S2), for a total of 101 genes, with varying degrees of association to inherited arrhythmias.
20
, 
21
, 
22
, 
23
, 
24
 However, the contribution of novel and ultra‐rare genetic variants stemmed principally from genes with definitive evidence of disease associations
19
, 
20
, 
21
, 
22
, 
23
, 
24
 (n=12, 63%), and secondarily from disputed genes (n=5, 26%; Figure 2A). The only gene that contributed >1 novel/ultra‐rare variant, absent from the smaller 25‐gene panel, was TTN (Table S3). Most important, LP and LB variants stemmed only from definitive‐evidence genes (6/18, 33.3%), whereas limited, disputed, and noncurated genes contributed only VUS (5/18, 27.8%) (Figure 2B).

A, Distribution of the 19 novel and ultra‐rare genetic variants across 101 genes with different levels of evidence of disease association in 18 SIDS/SUDC cases. Absolute numbers (n) and relative frequencies (%) are shown within and above the bars, respectively. B, Yield of expanded genetic testing according to different variant classes across 101 genes with different levels of evidence of disease association in 18 SIDS/SUDC cases. LB indicates likely benign; LP, likely pathogenic; NA, noncurated genes or genes with anecdotal evidence; SIDS, sudden infant death syndrome; SUDC, sudden unexplained death in childhood; and VUS, variant of uncertain significance.

All genotype‐positive SIDS/SUDC cases were communicated to SCH, along with a prioritization scheme for cascade family screening, according to the classification status of the variants identified in the infants (LP>VUS>LB). SCH then attempted to inform the respective parents, offering them the possibility to attend the follow‐up SIDS/SUDC clinic. At the study's initiation, and after proper genetic counseling, many parents had opted to be informed of the genetic test result even if it were inconclusive or with unlikely causative genetic findings (ie, LB variants) and to aid in the variant adjudication process.

As such, after genetic analyses in the victims had been concluded, 24 parents of 13 victims with SIDS/SUDC were subjected to genetic and clinical cascade family screening, thus enabling genotype–phenotype correlations to be performed. Parental DNA screening of 22 parents for the variants found in the respective 11 infants revealed that these were of parental origin. In the remaining 2 cases, where only 1 parent attended follow‐up and did not carry the respective variants, parental origin or de novo inheritance status could not be ascertained. Unfortunately, several families (some of whom with potentially clinically significant variants, such as novel) opted to be followed up locally at their place of residence and some could not be recontacted (see Limitations).

All genotype–phenotype correlations resulting from the double‐blind review of the clinical and genetic data in the victims' families are shown in Table 3. Among families who underwent a complete evaluation, there was, in most cases, good agreement between initial variant classification status and clinical phenotype, that is, cases where the genetic variants had been originally classified as LP, LB, or VUS‐FNP (cases S.011, S.051, S.059, S.103, S.117). In 3 cases (S.003, S.059, and S.113), the correlation was, to some degree, positive for LQTS and CPVT. In the other 3 cases, with KCNH2 and RYR2 VUS, the correlation was negative (S.024, S.068, S.090). Finally, in 3 cases (S.010, S.057, S.121), genotype–phenotype correlation could not be completed due to lack of parental data.

Cascade Family Screening

AV indicates atrioventricular; bpm, beats per minute; EST, exercise stress test; FNP, favor nonpathogenic; LB, likely benign; LP, likely pathogenic; LVEF, left ventricular ejection fraction; MRI, magnetic resonance imaging; NA, not available; SIDS, sudden infant death syndrome; SUDC, sudden unexplained death in childhood; and VUS, variant of uncertain significance.

MRI not available for review.

In 3 cases (S.003, S.059 and S.113), the correlation of genotype to phenotype was, at least in some degree, positive.

In the SIDS case S.003 (male, aged 6 days), a KCNH2 rare (MAF=0.0001205) variant (p.G903R) was identified that was initially classified as VUS. This variant has an equivocal VUS classification from 7 laboratories in ClinVar (VCV000067428.23, last accessed March 2023). It was originally identified in 3 patients with LQTS
30
 then in 1 young child with sudden unexplained death and normal heart at autopsy, and with positive cosegregation for QT prolongation in 1 family member,
31
 as well as in 1 patient with unspecified arrhythmia.
32
 Because its MAF exceeds what would be expected for a monogenic disease‐causing variant (ie, MAF<0.0001), it was recently dismissed as a potential monogenic cause of disease.
33
 More recently, an in vitro functional study showed a mild functional effect, with modestly accelerated deactivation of the mutant channel.
34
 Cascade family screening evidenced that the variant was inherited from the child's father. His ECG presented with several abnormalities but was not typical of LQTS. In our center in Milan, we have been recently following up a family (mother and adult son) with the KCNH2‐p.G903R variant. They are both asymptomatic and have ECGs with borderline QT intervals (QTc=450 ms), which have, however, been found prolonged in Holter ECG recordings (mother, QTc=467 ms; son, QTc=453 ms in nocturnal hours). In addition, the mother presents with ventricular extrasystoles in Holter ECG and EST and has constant notched T waves. As such, our S.003 SIDS case was adjudicated as having a partially positive genotype–phenotype correlation (see Discussion).

In the SUDC case S.059 (male, aged 1.3 years), a KCNQ1 ultra‐rare (MAF=0.00001205) variant (p.G460D) was identified that was classified as likely pathogenic. This variant has been previously described in a SADS case during molecular autopsy,
35
 and it affects an amino acid residue previously implicated in SIDS.
1
 Cascade family screening revealed that the variant was of maternal origin. The clinical workup of the father was unremarkable. The mother's ECG showed a borderline QTc in basal conditions but with a clearly pathological prolongation (QTc>550 ms) at the pre‐EST ECG performed in the upright position, which often unmasks the impaired QT interval response to brisk standing.
36
 During the double‐blind review process of this case, genotype and phenotype correlated positively for LQTS and the variant was adjudicated as LP/P. The SCH partners were informed with an indication to extend the screening to other family members/siblings, if any, to follow up the mother, and to initiate treatment with beta blockers.

In the SUDC case S.113 (male, aged 1 year and 19 days), 2 ultra‐rare genetic variants were identified, in the RYR2 and DSP genes. The RYR2‐p.T153I ultra‐rare (MAF=0.00001809) genetic variant was initially classified as LP. This variant has been previously identified postmortem in at least 2 unrelated cases of SADS.
12
, 
35
 The S.113 subject also carried the ultra‐rare DSP‐p.A2019S variant (MAF=0.00003898) that was classified as VUS. This variant was originally described in a patient with arrhythmogenic cardiomyopathy, who fulfilled the task force diagnostic criteria, and with positive family history of SCD,
37
 but it has recently been questioned as a potential monogenic cause of disease.
38
 Cascade genetic screening revealed that both variants were inherited from the child's father. At follow‐up, he was subjected to an ECG that showed QRS fragmentation in some leads. An echocardiogram showed mild dilatation of the left atrium and normal biventricular function; however, magnetic resonance imaging was not performed. At the EST, isolated polymorphic ventricular extrasystoles, but not repetitive forms, were observed at heart rates >125 bpm and disappeared during recovery.

Upon the double‐blind review process, the clinical reviewers disclosed that the father had an exercise‐induced arrhythmic phenotype, which, despite not fully reaching the criteria, was highly suggestive of CPVT. The disclosure that the father was a carrier of RYR2‐LP and DSP‐VUS variants adjudicated the case as having a positive genotype–phenotype correlation, although not of typical CPVT. A contribution to the arrhythmic phenotype by the DSP variant could not be excluded. The SCH partners were informed of these results with an indication to follow up, perform magnetic resonance imaging, and further extend the family screening, if applicable.

Among cases with a negative genotype–phenotype correlation there were cases S.051, S.068, and S.103.

In the SIDS case S.051 (female, aged 2 months), a KCNH2 rare (MAF=0.0001999) variant (p.E289K) was identified that was formally classified as VUS, but FNP according to expert judgment. The variant initially, under gnomAD v2.0.2, fell below the rare variant threshold set in the study but was found to exceed the cutoff when MAFs were interrogated again under gnomAD v2.1.1 at the study's conclusion. Because, in the meantime, cascade family screening had already been initiated, the case is presented here, without being considered in the genetic yield of the SIDS subgroup.

The variant was first reported in association to LQTS in a mutation update compendium.
39
 An in vitro functional study of the mutant channel showed normal cell surface channel expression, however, the biophysical properties of the mutant channel were not assessed.
40
 It was later identified in a SIDS case and classified as a VUS.
5
 The ClinVar database contains 9 entries of this variant with an equivocal VUS classification from different laboratories (VCV000067535.18, last accessed March 2023). In our S.051 SIDS case, the variant was of maternal origin. Her clinical workup was completely within normal limits and medical and family history were unremarkable. Genotype–phenotype correlation was thus negative and, post family screening, the variant was adjudicated as LB.

In the SUDC case S.103 (female, aged 1.5 years), we downgraded the RYR2‐p.K2053R variant from VUS‐FNP to LB due to lack of EST‐induced arrhythmias in the S.103 carrier mother, taking also into account its MAF and lack of in silico support of pathogenicity. However, due to a QT interval prolongation observed during the recovery phase of EST, more data are needed to definitively adjudicate this variant as benign.

In the SIDS case S.068 (male, aged 1.3 months), carrying the RYR2‐p.R4257Q variant that was initially classified as a VUS, a cardiomyopathy phenotype was present in the carrier mother that could not be fully evaluated clinically. Although the variant was downgraded to LB status post family screening in association with CPVT, there are some considerations to be made. First, there is anecdotal evidence of a RYR2‐hypertrophic cardiomyopathy disease association (limited evidence hypertrophic cardiomyopathy gene).
21
, 
41
 Second, the variant has been previously identified in a parent–child trio whole‐exome sequencing study in a child with benign epilepsy of childhood with centrotemporal spikes and nocturnal seizures who also had an arrhythmic phenotype (sinus arrhythmia and atrial premature beats).
42
 Therefore, although “acquitted” for CPVT, this variant as well needs more data for a definitive adjudication of a benign role in disease.

Among the cases with incomplete clinical evaluation was the SUDC case S.010 (female, aged 1.6 years) carrying the novel SCN5A‐p.S1964Y variant that was classified as VUS. The disclosure, however, of a history of febrile convulsions in the deceased infant upgraded the variant to a VUS‐favor pathogenic status according to expert judgment. Residue 1964 has been previously implicated in BrS,43
 which may masquerade as febrile seizures in young children.
44
 In addition, inherited febrile seizures have been described in a significant proportion of SUDC.
45
 Unfortunately, the carrier father opted to perform the pharmacological provocation test at his place of residence and the result is not available.

Altogether, cascade family screening and the subsequent genotype–phenotype correlations, aided in downgrading 7 VUS to a LB/B classification status (S.011, S.024, S.051, S.068, S.090, S.103, and S.117), while upgrading 1 VUS and 2 LP variants to a LP/P (S.003, S.059, S.113) or VUS‐favor pathogenic status (S.010). One variant in RYR2 was downgraded from VUS to LB/B specifically with respect to a CPVT phenotype (S.068); however, a potential role in arrhythmia susceptibility could not be completely excluded.

### Study Cohort

The initial study population included 104 unrelated SIDS/SUDC cases. We applied stringently the age cutoff of 1 year as a criterion for a SIDS definition,
2
 while reserving the SUDC definition
15
 for all cases exceeding 1 year, even by just 1 day. In the whole cohort, there were 88 SIDS cases (56 male, 64%) with an age range at the time of death of 2 to 365 days (mean 105±88, median 76 days). The remaining 16 were SUDC cases (10 male, 62.5%), with an age range at the time of death of 12.0 to 54.1 months (mean 23.6±11.1, median 20.5 months). In 11 SIDS cases, DNA integrity was low, and this resulted in partial genetic analyses (Data S1).

### Genetic Variants and Yield of Genetic Testing

Complete genetic analyses were possible in 93 SIDS/SUDC cases. These underwent NGS of 25 major arrhythmia‐associated genes (Table S1) with mostly definitive evidence of disease associations
19
, 
20
, 
21
, 
22
, 
23
, 
24
 or specifically implicated in SIDS.
29

In the SIDS cohort (n=77), NGS of the 25 arrhythmia‐associated genes yielded 25 rare nonsynonymous variants with a MAF<0.00015 in 24 SIDS cases (Table 1). Among these, there were 2 novel and 13 ultra‐rare (MAF<0.00005) variants in 15 cases. Most genetic variants were missense, with only 3 being non‐missense. In accordance with the American College of Medical Genetics guidelines,
28
 2 variants were classified as likely pathogenic (LP), 21 variants were classified as variants of uncertain significance (VUS), and 2 as likely benign (LB). Three VUS were further classified as favor nonpathogenic (FNP) according to expert judgment. Altogether, the yield of LP variants (n=2, 2.6%) was equal to that of novel VUS and of LB variants, with a higher yield of ultra‐rare and rare VUS (n=11, 14.3% and n=8, 10.4%, respectively; Figure 1A).

Novel, Ultra‐Rare, and Rare Genetic Variants in the 25 Major Genes Across the SIDS Cohort

Genetic variants are ordered with increasing allelic frequency, and nomenclature refers to the main transcript. In silico mostly refers to the pathogenicity meta‐scores MetaSVM and REVEL (Damaging‐D/Tolerated‐T and Pathogenic‐P/Benign‐B, respectively; see Methods). The most representative variant citations are given under PMID.

Denotes a secondary classification, according to expert judgment.

KCNH2‐p.E289K initially, under gnomAD v2.0.2, fell below the rare variant threshold set in the study but exceeded the cutoff under gnomAD v2.1.1 at the study's conclusion. Since cascade family screening had already been initiated, the variant is presented, without being considered in the overall yield (Data S1).

1000G indicates 1000 Genomes Project Browser; AC, allele count; ACMG, American College of Medical Genetics and Genomics; ClinVar, Clinical Significance of Variants public archive; dbSNP, Single Nucleotide Polymorphism database; FNP, favor nonpathogenic; GRCh37, Genome Reference Consortium Human Build 37; LB, likely benign; LP, likely pathogenic; MAF, minor allele frequency; PMID, PubMed IDentifier; SIDS, sudden infant death syndrome; and VUS, variant of uncertain significance.

A, Yield of genetic testing in the 77 SIDS and 18 SUDC cases for different variant classes across the 25 major arrhythmia‐associated genes. Statistically significant (P<0.05) comparisons are shown above the bars. B, Distribution of the 29 novel, ultra‐rare, and rare genetic variants across the 25 major genes that yielded >1 genetic variant in the whole cohort. Absolute numbers (n) and relative frequencies (%) are shown within and above the bars, respectively. LB indicates likely benign; LP, likely pathogenic; SIDS, sudden infant death syndrome; SUDC, sudden unexplained death in childhood; and VUS, variant of uncertain significance.

Genetic analyses of the 25 major genes in the SUDC cohort (n=16) yielded in total 3 novel and 5 ultra‐rare nonsynonymous variants in 7 SUDC cases (Table 2). Novel variants were contributed by the SCN5A, RYR2, and DSC2 genes in 3 cases. Most genetic variants were missense, with 2 being non‐missense. American College of Medical Genetics variant classification
28
 adjudicated 3 variants as LP and 5 (2 novel, 3 ultra‐rare) as VUS. Thus, the yield of LP variants in the SUDC cohort was 18.8% (3/16). Altogether, the yield of LP variants was equal to that of ultra‐rare VUS (n=3, 18.8%), with a lower yield of novel VUS (n=2, 12.5%), whereas there was no yield of rare VUS or LB variants (Figure 1A).

Novel, Ultra‐Rare, and Rare Genetic Variants in the 25 Major Genes Across the SUDC Cohort

Genetic variants are ordered with increasing allelic frequency, and nomenclature refers to the main transcript. In silico mostly refers to the pathogenicity meta‐scores MetaSVM and REVEL (Damaging‐D/Tolerated‐T and Pathogenic‐P/Benign‐B, respectively; see Methods). The most representative variant citations are given under PMID.

Denotes a secondary classification, according to expert judgment.

1000G indicates 1000 Genomes Project Browser; AC, allele count; ACMG, American College of Medical Genetics and Genomics; ClinVar, Clinical Significance of Variants public archive; dbSNP, Single Nucleotide Polymorphism database; GRCh37, Genome Reference Consortium Human Build 37; LP, likely pathogenic; MAF, minor allele frequency; PMID, PubMed IDentifier; SUDC, sudden unexplained death in childhood; and VUS, variant of uncertain significance.

Thus, among the 93 SIDS/SUDC cases, there was a significantly higher percentage of LP variants in SUDC cases with respect to SIDS (18.8% [95% CI, 4.0–45.6] versus 2.6% [95% CI, 0.32–9.1], respectively, P=0.03), whereas ultra‐rare VUS (18.8% [95% CI, 4.0–45.6] versus 14.3% [95% CI, 7.4–24.1] respectively, P=0.70) and novel VUS (12.5% [95% CI, 1.6–38.3] versus 2.6% [95% CI, 0.32–9.1], respectively, P=0.14) were comparably represented in the 2 groups. Of note, all the 10 rare VUS and LB variants across the 25 major genes (10.4% and 2.6%, respectively) were contributed exclusively by SIDS cases and none were found in the 16 SUDC cases (Figure 1A).

In the whole cohort, across the 25 major genes, there were 8 genes (RYR2, LMNA, MYBPC3, DSC2, DSP, MYH7, KCNQ1, KCNH2) that yielded >1 rare variant, for a total of 29 variants, mostly represented by ultra‐rare (n=16, 55.1%) and less frequently by rare (n=9, 31%) and novel variants (n=4, 13.8%). The RYR2 gene contributed most variants (n=7, 24.1%), of which 1 was novel, 5 were ultra‐rare, and 1 was rare (Figure 1B). In fact, ultra‐rare nonsynonymous RYR2 variants appeared to be overrepresented in the SIDS/SUDC cases with respect to the gnomAD population (RYR2 nonsynonymous ultra‐rare alleles, 6/186 [3.22%] in cases versus 3890/282 912 [1.37%] in controls; P=0.04). No genetic variants were identified in the CALM1‐3 and GJA1 genes.

For the last 18 cases enrolled in the study (15 SIDS, 3 SUDC), genetic analysis was extended to an additional 76 genes (Table S2), for a total of 101 genes, with varying degrees of association to inherited arrhythmias.
20
, 
21
, 
22
, 
23
, 
24
 However, the contribution of novel and ultra‐rare genetic variants stemmed principally from genes with definitive evidence of disease associations
19
, 
20
, 
21
, 
22
, 
23
, 
24
 (n=12, 63%), and secondarily from disputed genes (n=5, 26%; Figure 2A). The only gene that contributed >1 novel/ultra‐rare variant, absent from the smaller 25‐gene panel, was TTN (Table S3). Most important, LP and LB variants stemmed only from definitive‐evidence genes (6/18, 33.3%), whereas limited, disputed, and noncurated genes contributed only VUS (5/18, 27.8%) (Figure 2B).

A, Distribution of the 19 novel and ultra‐rare genetic variants across 101 genes with different levels of evidence of disease association in 18 SIDS/SUDC cases. Absolute numbers (n) and relative frequencies (%) are shown within and above the bars, respectively. B, Yield of expanded genetic testing according to different variant classes across 101 genes with different levels of evidence of disease association in 18 SIDS/SUDC cases. LB indicates likely benign; LP, likely pathogenic; NA, noncurated genes or genes with anecdotal evidence; SIDS, sudden infant death syndrome; SUDC, sudden unexplained death in childhood; and VUS, variant of uncertain significance.

### Cascade Family Screening

All genotype‐positive SIDS/SUDC cases were communicated to SCH, along with a prioritization scheme for cascade family screening, according to the classification status of the variants identified in the infants (LP>VUS>LB). SCH then attempted to inform the respective parents, offering them the possibility to attend the follow‐up SIDS/SUDC clinic. At the study's initiation, and after proper genetic counseling, many parents had opted to be informed of the genetic test result even if it were inconclusive or with unlikely causative genetic findings (ie, LB variants) and to aid in the variant adjudication process.

As such, after genetic analyses in the victims had been concluded, 24 parents of 13 victims with SIDS/SUDC were subjected to genetic and clinical cascade family screening, thus enabling genotype–phenotype correlations to be performed. Parental DNA screening of 22 parents for the variants found in the respective 11 infants revealed that these were of parental origin. In the remaining 2 cases, where only 1 parent attended follow‐up and did not carry the respective variants, parental origin or de novo inheritance status could not be ascertained. Unfortunately, several families (some of whom with potentially clinically significant variants, such as novel) opted to be followed up locally at their place of residence and some could not be recontacted (see Limitations).

### Genotype–Phenotype Correlations

All genotype–phenotype correlations resulting from the double‐blind review of the clinical and genetic data in the victims' families are shown in Table 3. Among families who underwent a complete evaluation, there was, in most cases, good agreement between initial variant classification status and clinical phenotype, that is, cases where the genetic variants had been originally classified as LP, LB, or VUS‐FNP (cases S.011, S.051, S.059, S.103, S.117). In 3 cases (S.003, S.059, and S.113), the correlation was, to some degree, positive for LQTS and CPVT. In the other 3 cases, with KCNH2 and RYR2 VUS, the correlation was negative (S.024, S.068, S.090). Finally, in 3 cases (S.010, S.057, S.121), genotype–phenotype correlation could not be completed due to lack of parental data.

Cascade Family Screening

AV indicates atrioventricular; bpm, beats per minute; EST, exercise stress test; FNP, favor nonpathogenic; LB, likely benign; LP, likely pathogenic; LVEF, left ventricular ejection fraction; MRI, magnetic resonance imaging; NA, not available; SIDS, sudden infant death syndrome; SUDC, sudden unexplained death in childhood; and VUS, variant of uncertain significance.

MRI not available for review.

In 3 cases (S.003, S.059 and S.113), the correlation of genotype to phenotype was, at least in some degree, positive.

In the SIDS case S.003 (male, aged 6 days), a KCNH2 rare (MAF=0.0001205) variant (p.G903R) was identified that was initially classified as VUS. This variant has an equivocal VUS classification from 7 laboratories in ClinVar (VCV000067428.23, last accessed March 2023). It was originally identified in 3 patients with LQTS
30
 then in 1 young child with sudden unexplained death and normal heart at autopsy, and with positive cosegregation for QT prolongation in 1 family member,
31
 as well as in 1 patient with unspecified arrhythmia.
32
 Because its MAF exceeds what would be expected for a monogenic disease‐causing variant (ie, MAF<0.0001), it was recently dismissed as a potential monogenic cause of disease.
33
 More recently, an in vitro functional study showed a mild functional effect, with modestly accelerated deactivation of the mutant channel.
34
 Cascade family screening evidenced that the variant was inherited from the child's father. His ECG presented with several abnormalities but was not typical of LQTS. In our center in Milan, we have been recently following up a family (mother and adult son) with the KCNH2‐p.G903R variant. They are both asymptomatic and have ECGs with borderline QT intervals (QTc=450 ms), which have, however, been found prolonged in Holter ECG recordings (mother, QTc=467 ms; son, QTc=453 ms in nocturnal hours). In addition, the mother presents with ventricular extrasystoles in Holter ECG and EST and has constant notched T waves. As such, our S.003 SIDS case was adjudicated as having a partially positive genotype–phenotype correlation (see Discussion).

In the SUDC case S.059 (male, aged 1.3 years), a KCNQ1 ultra‐rare (MAF=0.00001205) variant (p.G460D) was identified that was classified as likely pathogenic. This variant has been previously described in a SADS case during molecular autopsy,
35
 and it affects an amino acid residue previously implicated in SIDS.
1
 Cascade family screening revealed that the variant was of maternal origin. The clinical workup of the father was unremarkable. The mother's ECG showed a borderline QTc in basal conditions but with a clearly pathological prolongation (QTc>550 ms) at the pre‐EST ECG performed in the upright position, which often unmasks the impaired QT interval response to brisk standing.
36
 During the double‐blind review process of this case, genotype and phenotype correlated positively for LQTS and the variant was adjudicated as LP/P. The SCH partners were informed with an indication to extend the screening to other family members/siblings, if any, to follow up the mother, and to initiate treatment with beta blockers.

In the SUDC case S.113 (male, aged 1 year and 19 days), 2 ultra‐rare genetic variants were identified, in the RYR2 and DSP genes. The RYR2‐p.T153I ultra‐rare (MAF=0.00001809) genetic variant was initially classified as LP. This variant has been previously identified postmortem in at least 2 unrelated cases of SADS.
12
, 
35
 The S.113 subject also carried the ultra‐rare DSP‐p.A2019S variant (MAF=0.00003898) that was classified as VUS. This variant was originally described in a patient with arrhythmogenic cardiomyopathy, who fulfilled the task force diagnostic criteria, and with positive family history of SCD,
37
 but it has recently been questioned as a potential monogenic cause of disease.
38
 Cascade genetic screening revealed that both variants were inherited from the child's father. At follow‐up, he was subjected to an ECG that showed QRS fragmentation in some leads. An echocardiogram showed mild dilatation of the left atrium and normal biventricular function; however, magnetic resonance imaging was not performed. At the EST, isolated polymorphic ventricular extrasystoles, but not repetitive forms, were observed at heart rates >125 bpm and disappeared during recovery.

Upon the double‐blind review process, the clinical reviewers disclosed that the father had an exercise‐induced arrhythmic phenotype, which, despite not fully reaching the criteria, was highly suggestive of CPVT. The disclosure that the father was a carrier of RYR2‐LP and DSP‐VUS variants adjudicated the case as having a positive genotype–phenotype correlation, although not of typical CPVT. A contribution to the arrhythmic phenotype by the DSP variant could not be excluded. The SCH partners were informed of these results with an indication to follow up, perform magnetic resonance imaging, and further extend the family screening, if applicable.

Among cases with a negative genotype–phenotype correlation there were cases S.051, S.068, and S.103.

In the SIDS case S.051 (female, aged 2 months), a KCNH2 rare (MAF=0.0001999) variant (p.E289K) was identified that was formally classified as VUS, but FNP according to expert judgment. The variant initially, under gnomAD v2.0.2, fell below the rare variant threshold set in the study but was found to exceed the cutoff when MAFs were interrogated again under gnomAD v2.1.1 at the study's conclusion. Because, in the meantime, cascade family screening had already been initiated, the case is presented here, without being considered in the genetic yield of the SIDS subgroup.

The variant was first reported in association to LQTS in a mutation update compendium.
39
 An in vitro functional study of the mutant channel showed normal cell surface channel expression, however, the biophysical properties of the mutant channel were not assessed.
40
 It was later identified in a SIDS case and classified as a VUS.
5
 The ClinVar database contains 9 entries of this variant with an equivocal VUS classification from different laboratories (VCV000067535.18, last accessed March 2023). In our S.051 SIDS case, the variant was of maternal origin. Her clinical workup was completely within normal limits and medical and family history were unremarkable. Genotype–phenotype correlation was thus negative and, post family screening, the variant was adjudicated as LB.

In the SUDC case S.103 (female, aged 1.5 years), we downgraded the RYR2‐p.K2053R variant from VUS‐FNP to LB due to lack of EST‐induced arrhythmias in the S.103 carrier mother, taking also into account its MAF and lack of in silico support of pathogenicity. However, due to a QT interval prolongation observed during the recovery phase of EST, more data are needed to definitively adjudicate this variant as benign.

In the SIDS case S.068 (male, aged 1.3 months), carrying the RYR2‐p.R4257Q variant that was initially classified as a VUS, a cardiomyopathy phenotype was present in the carrier mother that could not be fully evaluated clinically. Although the variant was downgraded to LB status post family screening in association with CPVT, there are some considerations to be made. First, there is anecdotal evidence of a RYR2‐hypertrophic cardiomyopathy disease association (limited evidence hypertrophic cardiomyopathy gene).
21
, 
41
 Second, the variant has been previously identified in a parent–child trio whole‐exome sequencing study in a child with benign epilepsy of childhood with centrotemporal spikes and nocturnal seizures who also had an arrhythmic phenotype (sinus arrhythmia and atrial premature beats).
42
 Therefore, although “acquitted” for CPVT, this variant as well needs more data for a definitive adjudication of a benign role in disease.

Among the cases with incomplete clinical evaluation was the SUDC case S.010 (female, aged 1.6 years) carrying the novel SCN5A‐p.S1964Y variant that was classified as VUS. The disclosure, however, of a history of febrile convulsions in the deceased infant upgraded the variant to a VUS‐favor pathogenic status according to expert judgment. Residue 1964 has been previously implicated in BrS,43
 which may masquerade as febrile seizures in young children.
44
 In addition, inherited febrile seizures have been described in a significant proportion of SUDC.
45
 Unfortunately, the carrier father opted to perform the pharmacological provocation test at his place of residence and the result is not available.

Altogether, cascade family screening and the subsequent genotype–phenotype correlations, aided in downgrading 7 VUS to a LB/B classification status (S.011, S.024, S.051, S.068, S.090, S.103, and S.117), while upgrading 1 VUS and 2 LP variants to a LP/P (S.003, S.059, S.113) or VUS‐favor pathogenic status (S.010). One variant in RYR2 was downgraded from VUS to LB/B specifically with respect to a CPVT phenotype (S.068); however, a potential role in arrhythmia susceptibility could not be completely excluded.

### Discussion

The present study, specifically designed to overcome the major stumbling block of anonymity thus far dominating sudden infant death research, has provided multiple novel findings. SUDC should be brought to the foreground of SADS investigations. The genetic yield in SIDS and SUDC stems principally from genes with definitive evidence of disease associations to inherited arrhythmia syndromes, such as LQTS, and CPVT, thus suggesting that molecular autopsy could be more rewarding if performed with definitive‐gene panels. Rare variants in established genes should also be explored because this facilitates the definitive adjudication of such variants still populating the literature as VUS and it relays information on intermediate effect variants, possibly contributing to an oligogenic risk model. Finally, our study shows that anonymity hinders SIDS research without any benefit and whenever genetic results remain inconclusive, family screening may prove invaluable in determining variant pathogenicity, with implications that extend, beyond the child's family, to other patients carrying the same VUS.

In 2017, under the National Institutes of Health Clinical Genome Resource, an evidence‐based gene curation framework for the assessment of the clinical relevance of genes was developed that assigned different levels of evidence of a gene‐disease relationship.
21
 Interestingly, the recommended genes in the 2011 consensus document
19
 largely survived this reappraisal, while novel genes were added or disputed.
20
, 
21
, 
22
, 
23
, 
24
, 
42
 We sequenced most cases with a custom NGS panel that had been designed endorsing the 2011 consensus document.
18
, 
19
 The yield hereby reported (2.6% in SIDS and 18.8% in SUDC) stems only from definitive‐evidence genes,
20
, 
21
, 
22
, 
23
, 
24
 although not all such genes were analyzed across the cohort. This implies that the yield could be higher in both SIDS and SUDC if all definitive‐evidence genes are systematically screened. Comparisons of yield with other studies,
5
, 
12
 targeting genes with varying degrees of evidence of pathogenicity, are not readily feasible.

The expanded panel used in our last 18 cases was commercialized around 2015 and, despite being highly performing,
46
 includes many genes that have now limited or disputed evidence of pathogenicity (Table S2). However, the yield of novel or ultra‐rare variants from such genes was minimal. The fact that LP or LB variants were contributed exclusively by definitive‐evidence genes, whereas disputed and limited‐evidence genes yielded only VUS, likely reflects both the concept of weaker disease associations and our inability to attribute causality to variants in such genes. This altogether indicates that molecular autopsy should preferably be performed either through panels targeting definitive‐evidence genes, for which variant interpretation is more feasible and clinically actionable, either exome‐ or genome‐wide, but in parent–child trios or with family screening available,
47
 to uncover novel disease associations.

The gene in our cohort that contributed most novel, ultra‐rare, and rare variants was RYR2, with an overrepresentation of ultra‐rare variants in cases with respect to gnomAD, whereas cascade family screening was possible in 4/6 cases harboring such variants. Among these, we adjudicated only 1 case as having a positive genotype–phenotype correlation (S.113) due to an exercise‐induced arrhythmic phenotype suggestive for CPVT. Given the presence of polymorphic ventricular premature beats in the EST of the carrier father, we confirmed the RYR2‐p.T153I variant as LP
12
, 
35
 post family screening.

The role of RYR2 in SIDS has been previously demonstrated both in human
48
 and animal studies,
49
 although overrepresentation of ultra‐rare RYR2 variants in cases has been previously reported in SADS.
12
 In rare cases of RYR2‐associated CPVT, genotype‐positive family members may not present inducible arrhythmias in EST, but may nevertheless later experience cardiac events.
50
 Moreover, CPVT may include also atypical forms, associated with loss‐of‐function RYR2 variants, where ventricular fibrillation occurs in the absence of complex arrhythmias upon adrenergic stimulation.
51
 These are important considerations for RYR2 variant adjudication given that the tolerability of RYR2 to genetic variation could, probably in a few cases, incorrectly lead to dismissal of pathogenicity of novel/ultra‐rare genetic variants.
52

No pathogenic or rare genetic variants were identified in GJA1‐encoded connexin43, a gap junction previously implicated in SIDS with support from molecular and functional data,
29
 demonstrating that it is a minor player in SIDS pathogenesis. We, as others,
5
, 
53
, 
54
 did not identify any clinically relevant variants in the CALM1‐3 and TRDN genes in our SIDS or SUDC (≤4.5 years) cases. Although CALM1‐3 pathogenic variants seem thus far not to contribute to SIDS,
5
, 
53
 they may, however, be identified in a small fraction of sudden unexplained death in the young (<10 years).
53
 In the study that focused also on a 1‐ to 5‐year‐old SUDC subgroup,
11
 in the few cases that the CALM genes were screened, no clinically relevant variants were identified either.
11
, 
55
 This could be partly explained by the lack of significant numbers in SUDC cohorts, impeding the possibility of identifying such a rare disease as calmodulinopathy,
56
 and the fact that infants with calmodulinopathy manifest with extreme phenotypes that, in many cases, are likely to come to medical attention before a sentinel event occurs. Another possible explanation could be related to the differential expression of the CALM genes between fetal, infant, and adult stages of development.
57
 However, in the International Calmodulinopathy Registry,
56
 SUDC represents a small subgroup (9/140) of an already rare clinical entity, and among these 9 cases, all CALM genes are represented.

In the 18 cases in which the TTN gene was screened with the expanded panel, we identified 1 novel and 5 previously described ultra‐rare TTN missense variants in 6 cases that were classified as either VUS or LB. In one such case (S.117, TTN‐p.R4160S), cascade family screening aided in adjudicating the variant as probably benign. Because TTN seems not to be an important contributor to the rare dilated cardiomyopathy infantile forms,
58

TTN variants, especially missense, are expected to result in extremely low clinical actionability in the setting of SIDS or SUDC.

The upgrade of the novel SCN5A‐p.S1964Y VUS to VUS‐favor pathogenic in the S.010 SUDC case deserves a comment. The fact that inherited febrile seizures have been described in a significant proportion of SUDC,
45
 combined with BrS underlying some cases of febrile seizures in young children,
44
 implies that the traditional concept that BrS is unlikely to contribute to SIDS/SUDC should probably be reconsidered and the combination of SCN5A variants with febrile convulsions should be looked for.

To the best of our knowledge, this is the first study designed to overcome the obstacle of SIDS case anonymity
10
 and to include cascade family screening. Albeit with lower numbers than what we had envisioned, this has resulted in (1) a cardiogenetic diagnosis, or dismissal of a diagnosis, in some families and (2) providing evidence in favor or against pathogenicity of several variants, mostly VUS. These results strongly argue in favor of implementing procedures and regulations which, although safeguarding the victims' families, will allow to connect them with the molecular autopsy findings, because this may result in prevention of avoidable deaths.
10

To this end, we complied with all best‐practice recommendations for molecular autopsy and family screening,
59
, 
60
 that is, consent after expert genetic counseling, possibility to opt out, results confidentiality, and validation. Our study highlights that family screening postmolecular autopsy, in the setting of an expert multidisciplinary team, may have important implications for the clarification of inconclusive test results, such as VUS. Although a VUS, let alone a LB variant, cannot be used to initiate family screening in the setting of a purely genetic testing laboratory, it may be used in the setting of a specialized arrhythmia clinic.
60
 In the latter case, family genotype–phenotype correlations may prove invaluable in determining variant pathogenicity, thus avoiding further population of the literature with VUS, which would leave not only variants but also families in a genetic purgatory.
61

Besides infants with SIDS, we also included infants beyond 1 year of age, defined as SUDC,
2
, 
15
, 
16
 a definition more appropriate than SADS, because SUDC mostly affects children aged 1 to 4 years, while also sharing some epidemiological features with SIDS.
16
, 
17
 This clear separation goes beyond semantics as the incidence of unexplained sudden death in the young follows the order of SIDS>SUDC>SADS,
1
, 
2
, 
11
, 
13
, 
16
 dropping dramatically after age 5, while rising again only after age 10 to 11.
11
, 
14

The reported yield of expanded cardiogenetic testing in recent studies is 4.3% in SIDS5 and 13% in SADS.
12
 However, when SUDC is lumped under SADS, valuable information on yield, and thus causality, may be lost (such in a SADS cohort of 302 cases, with a median age of 24 years, where 5/40 cases carrying LP/P variants are children <4 years).
12
 Preliminary reports of cardiogenetic yield specifically in the SUDC age subgroup
55
 are comparable to ours: 16.2% (6/37) versus 18.8% (3/16) in our study (P=0.35). This warrants future studies to focus separately on this age subgroup, since SIDS and younger SUDC victims may share common causes and risk factors of SCD,
11
 which may easily be diluted when accounted for as unexplained SCD in the young.

Genetic yield in our SUDC subgroup was mostly contributed by male infants in the 1.0 to 1.3 years age range. One of these (S.059), carried the KCNQ1‐p.G460D LP variant and there was positive family correlation for LQTS. It is tempting to speculate whether it was by chance or some other factor that this infant apparently escaped SIDS, just to undergo SUDC. We postulated long ago
3
 that asymmetry and immaturity of cardiac sympathetic innervation in infants, with REM sleep‐related changes in sympathetic activity and heart rate, may influence arrhythmic risk. More recently, specific changes in differential expression of alternative ion channel spliceoforms (KCNQ1 included) were described to underline different developmental stages, thus potentially creating uniquely vulnerable, time‐limited arrhythmogenic substrates.
62
 This fits with the triple‐risk etiological model,
63
 in which 3 factors (a stress‐related trigger, a biologically vulnerable infant, and a critical developmental period) converge for SIDS to occur. This altogether suggests that, depending on the nature and effects of underlying genetic and contributory factors, some infants may remain at high risk after the first year of life, whereas others, if they survive the vulnerable time window, may escape SCD.
3
, 
6

Although the former group may represent those with highly penetrant pathogenic variants, the latter may represent those in whom SCD risk is shaped by oligogenic or polygenic genetic signatures made of variants with intermediate or lower effect sizes.
64
 Genome‐wide molecular autopsies support this concept as younger victims with SCD harbor both more VUS as well as rare variants in cardiac disease genes than matched controls,
47
 whereas a burden of ultra‐rare variants in channelopathies genes exists both in SIDS
5
 and SADS.
12
 Of note, in our study, despite the small sample size, all the 10 rare VUS and LB variants across the 25 major genes were contributed exclusively by SIDS cases and none were found in the 16 SUDC cases.

These considerations altogether readily imply that binary classifications of genetic causality (pathogenic or benign) may not be able to capture the whole spectrum of effect sizes variants may have on phenotype and, while we strive for being stringent in the search for monogenic causes, information on intermediate effect variants, possibly contributing to an oligogenic risk model, may be easily lost. For this reason, we have previously argued in favor of exploring not only ultra‐rare, but also rare genetic variants in established arrhythmia‐associated genes.
1
, 
6
 As we show here, this may have 2 important implications: first, it may aid in the definitive adjudication of variants still populating the literature as VUS, such as the higher MAF KCNH2‐p.E289K variant, previously identified in SIDS,
5
 which we initially classified as VUS‐FNP. The higher MAF of this variant, particularly its even higher frequency among Black people in whom LQTS is exceedingly rare,
65
 in addition to a completely negative genotype–phenotype correlation here, altogether seem to “acquit” it as a monogenic cause of LQTS.

The second implication of also exploring rare variants is demonstrated in another SIDS case (S.003, KCNH2‐p.G903R). The mild ECG abnormalities in the carrier father, as well as the mild QT prolongation and notched T waves observed in 2 of our own patients with this variant, appear to be in line with its recently demonstrated mild in vitro functional effect.
34
 Variants such as this challenge our ability to dichotomize causality in binary terms of pathogenic or benign, especially when, upon American College of Medical Genetics classification, neither the in vitro effect (PS3) nor the no effect (BS3) criterion
28
 can be safely applied. Although we do not demonstrate here that this variant was indeed SIDS causative, we postulate that such variants may contribute to an increased inherent arrhythmic risk, a risk that may be modulated differently in infant and adult life and shape different outcomes under the influence of extrinsic factors (eg, a QT prolonging drug).
66
 The definition of appropriate penetrance‐based MAF‐disease thresholds has been invaluable in avoiding overattribution of genetic causality, a long‐term evil.
6
 However, with the risk of SCD being always probabilistic, the underlying genetic causes cannot be viewed as deterministic.

Our study has limitations. First, 2 definitive‐evidence nonsyndromic genes,
20
, 
21
, 
22
, 
23
, 
24

FLNC and TECRL, were not targeted by the expanded analysis, whereas the 25‐gene panel contained only, but not all, definitive‐evidence genes. Second, partly due to the COVID‐19 pandemic and to changes in living circumstances, such as relocation, several families opted to be followed up locally at their place of residence. This led to several cases with potentially clinically significant variants (eg, novel) missing cascade screening, thus diminishing the family screening capacity that we had envisioned for the study. In addition, family screening was restricted to the victims' parents, thus impeding the possibility of performing wider and more comprehensive genotype–phenotype correlations. Furthermore, another limitation is the small sample size of the SUDC cohort, which, however, also reflects the much lower incidence of SUDC with respect to SIDS. Moreover, epidemiological data were not systematically recorded across cases, thus not allowing us to present combined epidemiological data on our cohort.

### Molecular Autopsy Genes

In 2017, under the National Institutes of Health Clinical Genome Resource, an evidence‐based gene curation framework for the assessment of the clinical relevance of genes was developed that assigned different levels of evidence of a gene‐disease relationship.
21
 Interestingly, the recommended genes in the 2011 consensus document
19
 largely survived this reappraisal, while novel genes were added or disputed.
20
, 
21
, 
22
, 
23
, 
24
, 
42
 We sequenced most cases with a custom NGS panel that had been designed endorsing the 2011 consensus document.
18
, 
19
 The yield hereby reported (2.6% in SIDS and 18.8% in SUDC) stems only from definitive‐evidence genes,
20
, 
21
, 
22
, 
23
, 
24
 although not all such genes were analyzed across the cohort. This implies that the yield could be higher in both SIDS and SUDC if all definitive‐evidence genes are systematically screened. Comparisons of yield with other studies,
5
, 
12
 targeting genes with varying degrees of evidence of pathogenicity, are not readily feasible.

The expanded panel used in our last 18 cases was commercialized around 2015 and, despite being highly performing,
46
 includes many genes that have now limited or disputed evidence of pathogenicity (Table S2). However, the yield of novel or ultra‐rare variants from such genes was minimal. The fact that LP or LB variants were contributed exclusively by definitive‐evidence genes, whereas disputed and limited‐evidence genes yielded only VUS, likely reflects both the concept of weaker disease associations and our inability to attribute causality to variants in such genes. This altogether indicates that molecular autopsy should preferably be performed either through panels targeting definitive‐evidence genes, for which variant interpretation is more feasible and clinically actionable, either exome‐ or genome‐wide, but in parent–child trios or with family screening available,
47
 to uncover novel disease associations.

### Gene‐Specific Yield

The gene in our cohort that contributed most novel, ultra‐rare, and rare variants was RYR2, with an overrepresentation of ultra‐rare variants in cases with respect to gnomAD, whereas cascade family screening was possible in 4/6 cases harboring such variants. Among these, we adjudicated only 1 case as having a positive genotype–phenotype correlation (S.113) due to an exercise‐induced arrhythmic phenotype suggestive for CPVT. Given the presence of polymorphic ventricular premature beats in the EST of the carrier father, we confirmed the RYR2‐p.T153I variant as LP
12
, 
35
 post family screening.

The role of RYR2 in SIDS has been previously demonstrated both in human
48
 and animal studies,
49
 although overrepresentation of ultra‐rare RYR2 variants in cases has been previously reported in SADS.
12
 In rare cases of RYR2‐associated CPVT, genotype‐positive family members may not present inducible arrhythmias in EST, but may nevertheless later experience cardiac events.
50
 Moreover, CPVT may include also atypical forms, associated with loss‐of‐function RYR2 variants, where ventricular fibrillation occurs in the absence of complex arrhythmias upon adrenergic stimulation.
51
 These are important considerations for RYR2 variant adjudication given that the tolerability of RYR2 to genetic variation could, probably in a few cases, incorrectly lead to dismissal of pathogenicity of novel/ultra‐rare genetic variants.
52

No pathogenic or rare genetic variants were identified in GJA1‐encoded connexin43, a gap junction previously implicated in SIDS with support from molecular and functional data,
29
 demonstrating that it is a minor player in SIDS pathogenesis. We, as others,
5
, 
53
, 
54
 did not identify any clinically relevant variants in the CALM1‐3 and TRDN genes in our SIDS or SUDC (≤4.5 years) cases. Although CALM1‐3 pathogenic variants seem thus far not to contribute to SIDS,
5
, 
53
 they may, however, be identified in a small fraction of sudden unexplained death in the young (<10 years).
53
 In the study that focused also on a 1‐ to 5‐year‐old SUDC subgroup,
11
 in the few cases that the CALM genes were screened, no clinically relevant variants were identified either.
11
, 
55
 This could be partly explained by the lack of significant numbers in SUDC cohorts, impeding the possibility of identifying such a rare disease as calmodulinopathy,
56
 and the fact that infants with calmodulinopathy manifest with extreme phenotypes that, in many cases, are likely to come to medical attention before a sentinel event occurs. Another possible explanation could be related to the differential expression of the CALM genes between fetal, infant, and adult stages of development.
57
 However, in the International Calmodulinopathy Registry,
56
 SUDC represents a small subgroup (9/140) of an already rare clinical entity, and among these 9 cases, all CALM genes are represented.

In the 18 cases in which the TTN gene was screened with the expanded panel, we identified 1 novel and 5 previously described ultra‐rare TTN missense variants in 6 cases that were classified as either VUS or LB. In one such case (S.117, TTN‐p.R4160S), cascade family screening aided in adjudicating the variant as probably benign. Because TTN seems not to be an important contributor to the rare dilated cardiomyopathy infantile forms,
58

TTN variants, especially missense, are expected to result in extremely low clinical actionability in the setting of SIDS or SUDC.

The upgrade of the novel SCN5A‐p.S1964Y VUS to VUS‐favor pathogenic in the S.010 SUDC case deserves a comment. The fact that inherited febrile seizures have been described in a significant proportion of SUDC,
45
 combined with BrS underlying some cases of febrile seizures in young children,
44
 implies that the traditional concept that BrS is unlikely to contribute to SIDS/SUDC should probably be reconsidered and the combination of SCN5A variants with febrile convulsions should be looked for.

### Cascade Family Screening

To the best of our knowledge, this is the first study designed to overcome the obstacle of SIDS case anonymity
10
 and to include cascade family screening. Albeit with lower numbers than what we had envisioned, this has resulted in (1) a cardiogenetic diagnosis, or dismissal of a diagnosis, in some families and (2) providing evidence in favor or against pathogenicity of several variants, mostly VUS. These results strongly argue in favor of implementing procedures and regulations which, although safeguarding the victims' families, will allow to connect them with the molecular autopsy findings, because this may result in prevention of avoidable deaths.
10

To this end, we complied with all best‐practice recommendations for molecular autopsy and family screening,
59
, 
60
 that is, consent after expert genetic counseling, possibility to opt out, results confidentiality, and validation. Our study highlights that family screening postmolecular autopsy, in the setting of an expert multidisciplinary team, may have important implications for the clarification of inconclusive test results, such as VUS. Although a VUS, let alone a LB variant, cannot be used to initiate family screening in the setting of a purely genetic testing laboratory, it may be used in the setting of a specialized arrhythmia clinic.
60
 In the latter case, family genotype–phenotype correlations may prove invaluable in determining variant pathogenicity, thus avoiding further population of the literature with VUS, which would leave not only variants but also families in a genetic purgatory.
61

### Definitions

Besides infants with SIDS, we also included infants beyond 1 year of age, defined as SUDC,
2
, 
15
, 
16
 a definition more appropriate than SADS, because SUDC mostly affects children aged 1 to 4 years, while also sharing some epidemiological features with SIDS.
16
, 
17
 This clear separation goes beyond semantics as the incidence of unexplained sudden death in the young follows the order of SIDS>SUDC>SADS,
1
, 
2
, 
11
, 
13
, 
16
 dropping dramatically after age 5, while rising again only after age 10 to 11.
11
, 
14

The reported yield of expanded cardiogenetic testing in recent studies is 4.3% in SIDS5 and 13% in SADS.
12
 However, when SUDC is lumped under SADS, valuable information on yield, and thus causality, may be lost (such in a SADS cohort of 302 cases, with a median age of 24 years, where 5/40 cases carrying LP/P variants are children <4 years).
12
 Preliminary reports of cardiogenetic yield specifically in the SUDC age subgroup
55
 are comparable to ours: 16.2% (6/37) versus 18.8% (3/16) in our study (P=0.35). This warrants future studies to focus separately on this age subgroup, since SIDS and younger SUDC victims may share common causes and risk factors of SCD,
11
 which may easily be diluted when accounted for as unexplained SCD in the young.

### Rethinking Genetic Causality

Genetic yield in our SUDC subgroup was mostly contributed by male infants in the 1.0 to 1.3 years age range. One of these (S.059), carried the KCNQ1‐p.G460D LP variant and there was positive family correlation for LQTS. It is tempting to speculate whether it was by chance or some other factor that this infant apparently escaped SIDS, just to undergo SUDC. We postulated long ago
3
 that asymmetry and immaturity of cardiac sympathetic innervation in infants, with REM sleep‐related changes in sympathetic activity and heart rate, may influence arrhythmic risk. More recently, specific changes in differential expression of alternative ion channel spliceoforms (KCNQ1 included) were described to underline different developmental stages, thus potentially creating uniquely vulnerable, time‐limited arrhythmogenic substrates.
62
 This fits with the triple‐risk etiological model,
63
 in which 3 factors (a stress‐related trigger, a biologically vulnerable infant, and a critical developmental period) converge for SIDS to occur. This altogether suggests that, depending on the nature and effects of underlying genetic and contributory factors, some infants may remain at high risk after the first year of life, whereas others, if they survive the vulnerable time window, may escape SCD.
3
, 
6

Although the former group may represent those with highly penetrant pathogenic variants, the latter may represent those in whom SCD risk is shaped by oligogenic or polygenic genetic signatures made of variants with intermediate or lower effect sizes.
64
 Genome‐wide molecular autopsies support this concept as younger victims with SCD harbor both more VUS as well as rare variants in cardiac disease genes than matched controls,
47
 whereas a burden of ultra‐rare variants in channelopathies genes exists both in SIDS
5
 and SADS.
12
 Of note, in our study, despite the small sample size, all the 10 rare VUS and LB variants across the 25 major genes were contributed exclusively by SIDS cases and none were found in the 16 SUDC cases.

These considerations altogether readily imply that binary classifications of genetic causality (pathogenic or benign) may not be able to capture the whole spectrum of effect sizes variants may have on phenotype and, while we strive for being stringent in the search for monogenic causes, information on intermediate effect variants, possibly contributing to an oligogenic risk model, may be easily lost. For this reason, we have previously argued in favor of exploring not only ultra‐rare, but also rare genetic variants in established arrhythmia‐associated genes.
1
, 
6
 As we show here, this may have 2 important implications: first, it may aid in the definitive adjudication of variants still populating the literature as VUS, such as the higher MAF KCNH2‐p.E289K variant, previously identified in SIDS,
5
 which we initially classified as VUS‐FNP. The higher MAF of this variant, particularly its even higher frequency among Black people in whom LQTS is exceedingly rare,
65
 in addition to a completely negative genotype–phenotype correlation here, altogether seem to “acquit” it as a monogenic cause of LQTS.

The second implication of also exploring rare variants is demonstrated in another SIDS case (S.003, KCNH2‐p.G903R). The mild ECG abnormalities in the carrier father, as well as the mild QT prolongation and notched T waves observed in 2 of our own patients with this variant, appear to be in line with its recently demonstrated mild in vitro functional effect.
34
 Variants such as this challenge our ability to dichotomize causality in binary terms of pathogenic or benign, especially when, upon American College of Medical Genetics classification, neither the in vitro effect (PS3) nor the no effect (BS3) criterion
28
 can be safely applied. Although we do not demonstrate here that this variant was indeed SIDS causative, we postulate that such variants may contribute to an increased inherent arrhythmic risk, a risk that may be modulated differently in infant and adult life and shape different outcomes under the influence of extrinsic factors (eg, a QT prolonging drug).
66
 The definition of appropriate penetrance‐based MAF‐disease thresholds has been invaluable in avoiding overattribution of genetic causality, a long‐term evil.
6
 However, with the risk of SCD being always probabilistic, the underlying genetic causes cannot be viewed as deterministic.

### Limitations

Our study has limitations. First, 2 definitive‐evidence nonsyndromic genes,
20
, 
21
, 
22
, 
23
, 
24

FLNC and TECRL, were not targeted by the expanded analysis, whereas the 25‐gene panel contained only, but not all, definitive‐evidence genes. Second, partly due to the COVID‐19 pandemic and to changes in living circumstances, such as relocation, several families opted to be followed up locally at their place of residence. This led to several cases with potentially clinically significant variants (eg, novel) missing cascade screening, thus diminishing the family screening capacity that we had envisioned for the study. In addition, family screening was restricted to the victims' parents, thus impeding the possibility of performing wider and more comprehensive genotype–phenotype correlations. Furthermore, another limitation is the small sample size of the SUDC cohort, which, however, also reflects the much lower incidence of SUDC with respect to SIDS. Moreover, epidemiological data were not systematically recorded across cases, thus not allowing us to present combined epidemiological data on our cohort.

### Conclusions

Our findings should help steering research on sudden infant and early childhood death toward directions more likely to clarify the underlying causes. Most important is the evidence that, by overcoming case anonymity, which has dominated and bogged down SIDS research, it is possible to provide answers much needed by the grieving parents
67
 and essential to prevent life‐threatening arrhythmias and sudden death in the family members with the same disease‐causing variant. When SIDS or SUDC are investigated in the setting of specialized and multidisciplinary research clinics, with appropriate procedures of counseling and consent in place, most parents appear willing to exercise their right to know and to also aid in the process of clarifying genetic ambiguities. In addition, our study brings SUDC to the foreground of SADS, not only because cardiogenetic investigations focused on the former may be particularly fruitful, but also because these may eventually aid in the identification of factors that, by operating in a time‐limited manner, may shape different outcomes, such as SIDS or SUDC.

### Sources of Funding

The study was supported by the Lullaby Trust (https://www.lullabytrust.org.uk), a registered charity in England and Wales (No. 262191) and a member of the Association of Medical Research Charities. The study was also partially supported by the Leducq Foundation Cardiovascular Research grant 18CVD05 “Towards Precision Medicine with Human iPSCs for Cardiac Channelopathies,” by the Sheffield Teaching Hospitals NHS Foundation project “The prevalence of long QT syndrome gene variants in Sudden Infant Death Syndrome in England. A molecular clinical study from victims to families,” and partially by the Italian Ministry of Health.

### Disclosures

None.

### Supporting information

Data S1

Tables S1–S3

References 68–82

Click here for additional data file.



# SUPPLEMENTAL FILE 1: JAH3-12-e029100.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)