# MAIN TEXT

## Very important pharmacogenetic variants landscape and potential clinical relevance in the Zhuang population from Yunnan province

### Abstract

The gradual evolution of pharmacogenomics has shed light on the genetic basis for inter-individual drug response variations across diverse populations. This study aimed to identify pharmacogenomic variants that differ in Zhuang population compared with other populations and investigate their potential clinical relevance in gene-drug and genotypic-phenotypic associations. A total of 48 variants from 24 genes were genotyped in 200 Zhuang subjects using the Agena MassARRAY platform. The allele frequencies and genotype distribution data of 26 populations were obtained from the 1000 Genomes Project, followed by a comparison and statistical analysis. After Bonferroni correction, significant differences in genotype frequencies were observed of CYP3A5 (rs776746), ACE (rs4291), KCNH2 (rs1805123), and CYP2D6 (rs1065852) between the Zhuang population and the other 26 populations. It was also found that the Chinese Dai in Xishuangbanna, China, Han Chinese in Beijing, China, and Southern Han Chinese, China showed least deviation from the Zhuang population. The Esan in Nigeria, Gambian in Western Division, The Gambia, and Yoruba in Ibadan, Nigeria exhibited the largest differences. This was also proved by structural analysis, Fst analysis and phylogenetic tree. Furthermore, these differential variants may be associated with the pharmacological efficacy and toxicity of Captopril, Amlodipine, Lisinopril, metoclopramide, and alpha-hydroxymetoprolol in the Zhuang population. Our study has filled the gap of pharmacogenomic information in the Zhuang population and has provided a theoretical framework for the secure administration of drugs in the Zhuang population.

### Introduction

Adverse drug reactions (ADRs) constitute a significant contributor to morbidity and mortality, ranking among the top 10 leading causes of death and disease in developed nations1,2. The characteristics of ADRs exhibit variability contingent upon factors such as genotype, age, gender, population, pathology, drug type, route of administration, and drug interaction3,4. According to Ingelman-Sundberg, genetic factors may account for approximately 10% to 20% of the occurrence of ADRs5. Genetic factors have been found to significantly influence pharmacokinetics, pharmacodynamics, and susceptibility to allergic reactions, resulting in changes in both local and systemic drug exposure and/or drug target functionality, ultimately impeding drug responses6. Recent investigations have elucidated the genetic underpinnings of ADRs7, thereby highlighting the close association between genetic factors and drug response.

Pharmacogenomics, as a key area of precision medicine, is the use of genomic and other “omic” information to personalize drugs selection and administration to avoid ADRs and maximize drugs therapeutic efficacy8,9. Pharmacogenomics accounted for 80% of the variations in drug treatment and safety. More than 400 genes were found to be involved in drug metabolism, and around 200 drug genes were linked to ADRs. It has been shown that substantial differences in distribution and frequencies of single nucleotide variants (SNVs) worldwide affect the key genes involved in drug absorption, distribution, metabolism, and elimination of abnormalities10. SNVs were a vast resource of genetic variation in humans, resulting in phenotypic differences among individuals11,12. The Pharmacogenetics and Pharmacogenomics Knowledge Base (PharmGKB; http://www.pharmgkb.org) that collects, organizes, and disseminates information on the impact of genetic variations in humans on drug responses. It provides free clinical-related information, including dosing guidelines, annotated drug labels, potentially viable gene-drug associations, and genotype–phenotype relationships13.

In recent years, numerous researchers have investigated very important pharmacogene (VIP) variants in ethnic minorities in China, such as the Tibetans14 and Lahu15. According to the 7th National Census, the Zhuang population totaled 15,721,956, ranking second only to the Han Chinese among the 56 populations. They are widespread in China’s Yunnan and Guizhou provinces, mainly in the Guangxi Zhuang Autonomous Region. Over a long period of time, they have developed customs and cultures with their own ethnic characteristics. However, we still have limited information on pharmacogenetic variants in the Zhuang population.

In this study, the VIP variants selected were derived from the PharmGKB, the SNP database of National Center for Biotechnology Information (NCBI, http://www.ncbi.nlm.nih.gov/SNP/), and the International HapMap Project (http://www.hapmap.org/), in addition to relevant pharmacogenomics literature. Then, the allele and genotype frequencies of the Zhuang population were compared with those of 26 other populations to obtain significant differences in SNVs after genotyping 200 unrelated Zhuang subjects from Yunnan province. The results of this study may complement current pharmacogenomics data of the Zhuang population, providing a theoretical basis for the safe use of drugs and predicting certain diseases in the Zhuang population.

### Materials and methods

In total, 200 unrelated Zhuang subjects (110 females and 90 males) were recruited from Wen Shan in the Yunnan Province of China. The sample size and the proportion were determined using G*Power 3.1.9.2 software16. The participants were healthy based on their medical history and physical examination. Additionally, they had at least three generations of Zhuang ancestry, while none of the other populations had any known ancestral background. Subjects with chronic diseases, infectious diseases, drug or alcohol abuse, severe heart, liver or kidney dysfunction, immune disorders, pregnancy, and lactation were excluded. The informed consent forms have been signed by all subjects. According to the study protocol approved by the Clinical Research Ethics Committee of Northwest University, 5 mL of peripheral blood was collected from each subject and stored at 4 °C for 24 h.

Through an extensive literature review on drug metabolism and toxicity, we identified 24 genes associated with these phenomena. By utilizing resources such as the PharmGKB database, the SNP database of NCBI, and the International HapMap Project, in addition to relevant pharmacogenomics literature, we selected variants linked to drug therapy responsiveness. A preliminary screening identified 59 variants. However, only homozygous genotypes were observed for 11 of these variants, making it impossible to compare the distribution and differences in genotype frequency. Consequently, these 11 variants were excluded from our analysis, leaving 48 variants for further investigation.

Genomic DNA was extracted from participants’ peripheral blood using GoldMag-Mini Whole Blood Genomic DNA Purification Kit (GoldMag Ltd., Xi’an, China). The concentration of genomic DNA was measured using NanoDrop 2000C spectrophotometer (Thermo Scientific, Waltham, MA, USA). Subsequently, multiplexed SNV MassEXTEND assays were designed using Agena MassARRAY Assay Design 4.0 software (San Diego, California, USA), which allowed for the design of PCR primers for the selected VIP variants. Agena MassARRAY RS1000 (San Diego, California, USA) was able to genotype the 48 VIP variants according to the manufacturer's instructions17. Finally, the data of SNV genotypes were collected and managed using Agena Typer 4.0 software18, as mentioned in previous studies.

We downloaded the genotype data from the 1000 Genomes website (https://www.internationalgenome.org/). The 26 populations included: (1) Chinese Dai in Xishuangbanna, China (CDX); (2) Han Chinese in Beijing, China (CHB); (3) Southern Han Chinese, China (CHS); (4) Japanese in Tokyo, Japan (JPT); (5) Kinh in Ho Chi Minh City, Vietnam (KHV); (6) African Caribbeans in Barbados (ACB); (7) African Ancestry in Southwest USA (ASW); (8) Esan in Nigeria (ESN); (9) Gambian in Western Division, The Gambia (GWD); (10) Luhya in Webuye, Kenya (LWK); (11) Mende in Sierra Leone (MSL); (12) Yoruba in Ibadan, Nigeria (YRI); (13) Colombian in Medellin, Colombia (CLM); (14) Mexican Ancestry in Los Angeles, California (MXL); (15) Peruvian in Lima, Peru (PEL); (16) Puerto Rican in Puerto Rico (PUR); (17) Utah residents with Northern and Western European ancestry (CEU); (18) Finnish in Finland (FIN); (19) British in England and Scotland (GBR); (20) Iberian populations in Spain (IBS); (21) Toscani in Italy (TSI); (22) Bengali in Bangladesh (BEB); (23) Gujarati Indian in Houston, Texas (GIH); (24) Indian Telugu in the UK (ITU); (25) Punjabi in Lahore, Pakistan (PJL) and (26) Sri Lankan Tamil in the UK (STU).

The Structure 2.3.4 software was used to analyze the structure of 27 populations, and Arlequin3.1 software was used to evaluate pairwise Fst values for assessing the relationship between 27 population groups. In addition, MEGA11 software was utilized to plot phylogenetic tree.

We performed a functional analysis of missense variants using online tools such as Polyphen2 (http://genetics.bwh.harvard.edu/pph2/), SNAP2 (https://rostlab.org/services/snap/), Mutationassessor (http://mutationassessor.org/r3/), FATHMM (http://fathmm.biocompute.org.uk/index.html), and Mutationtaster (https://www.mutationtaster.org/) to assess the impact of SNVs mutations to predict protein function.

A single amino acid change has the potential to significantly affect protein activity and function. We downloaded the protein structures of CYP2D6 and KCNH2 from the PDB database (https://www.rcsb.org/) and utilized the Chimera v1.16 software to predict and visualize the mutant protein structures.

The data were compiled, ordered, and analyzed using Microsoft Excel 2019 (Microsoft, Redmond, WA, USA) and SPSS 26.0 (SPSS, Chicago, IL, USA). The χ2 test was utilized to estimate the Hardy–Weinberg equilibrium (HWE) and compare the divergences in genotype frequencies of 48 VIP variants between the Zhuang population and the other 26 populations. All statistical tests were two-tailed (p < 0.05). Bonferroni corrections were performed to determine the significance level. After the Bonferroni’s multiple tests, p < 4.01 × 10−5 was recognized as statistically significant.

This study was conducted by the World Medical Association Declaration of Helsinki and was approved by the Northwestern University Clinical Research Ethics Committee (Approval number of Ethics Committee: 230,413,002). All subjects signed an informed consent form.

### Study subjects

In total, 200 unrelated Zhuang subjects (110 females and 90 males) were recruited from Wen Shan in the Yunnan Province of China. The sample size and the proportion were determined using G*Power 3.1.9.2 software16. The participants were healthy based on their medical history and physical examination. Additionally, they had at least three generations of Zhuang ancestry, while none of the other populations had any known ancestral background. Subjects with chronic diseases, infectious diseases, drug or alcohol abuse, severe heart, liver or kidney dysfunction, immune disorders, pregnancy, and lactation were excluded. The informed consent forms have been signed by all subjects. According to the study protocol approved by the Clinical Research Ethics Committee of Northwest University, 5 mL of peripheral blood was collected from each subject and stored at 4 °C for 24 h.

### Variants selection and genotyping

Through an extensive literature review on drug metabolism and toxicity, we identified 24 genes associated with these phenomena. By utilizing resources such as the PharmGKB database, the SNP database of NCBI, and the International HapMap Project, in addition to relevant pharmacogenomics literature, we selected variants linked to drug therapy responsiveness. A preliminary screening identified 59 variants. However, only homozygous genotypes were observed for 11 of these variants, making it impossible to compare the distribution and differences in genotype frequency. Consequently, these 11 variants were excluded from our analysis, leaving 48 variants for further investigation.

Genomic DNA was extracted from participants’ peripheral blood using GoldMag-Mini Whole Blood Genomic DNA Purification Kit (GoldMag Ltd., Xi’an, China). The concentration of genomic DNA was measured using NanoDrop 2000C spectrophotometer (Thermo Scientific, Waltham, MA, USA). Subsequently, multiplexed SNV MassEXTEND assays were designed using Agena MassARRAY Assay Design 4.0 software (San Diego, California, USA), which allowed for the design of PCR primers for the selected VIP variants. Agena MassARRAY RS1000 (San Diego, California, USA) was able to genotype the 48 VIP variants according to the manufacturer's instructions17. Finally, the data of SNV genotypes were collected and managed using Agena Typer 4.0 software18, as mentioned in previous studies.

### Populations variation data

We downloaded the genotype data from the 1000 Genomes website (https://www.internationalgenome.org/). The 26 populations included: (1) Chinese Dai in Xishuangbanna, China (CDX); (2) Han Chinese in Beijing, China (CHB); (3) Southern Han Chinese, China (CHS); (4) Japanese in Tokyo, Japan (JPT); (5) Kinh in Ho Chi Minh City, Vietnam (KHV); (6) African Caribbeans in Barbados (ACB); (7) African Ancestry in Southwest USA (ASW); (8) Esan in Nigeria (ESN); (9) Gambian in Western Division, The Gambia (GWD); (10) Luhya in Webuye, Kenya (LWK); (11) Mende in Sierra Leone (MSL); (12) Yoruba in Ibadan, Nigeria (YRI); (13) Colombian in Medellin, Colombia (CLM); (14) Mexican Ancestry in Los Angeles, California (MXL); (15) Peruvian in Lima, Peru (PEL); (16) Puerto Rican in Puerto Rico (PUR); (17) Utah residents with Northern and Western European ancestry (CEU); (18) Finnish in Finland (FIN); (19) British in England and Scotland (GBR); (20) Iberian populations in Spain (IBS); (21) Toscani in Italy (TSI); (22) Bengali in Bangladesh (BEB); (23) Gujarati Indian in Houston, Texas (GIH); (24) Indian Telugu in the UK (ITU); (25) Punjabi in Lahore, Pakistan (PJL) and (26) Sri Lankan Tamil in the UK (STU).

### Structure analysis and Fst analysis

The Structure 2.3.4 software was used to analyze the structure of 27 populations, and Arlequin3.1 software was used to evaluate pairwise Fst values for assessing the relationship between 27 population groups. In addition, MEGA11 software was utilized to plot phylogenetic tree.

### Protein hazard prediction

We performed a functional analysis of missense variants using online tools such as Polyphen2 (http://genetics.bwh.harvard.edu/pph2/), SNAP2 (https://rostlab.org/services/snap/), Mutationassessor (http://mutationassessor.org/r3/), FATHMM (http://fathmm.biocompute.org.uk/index.html), and Mutationtaster (https://www.mutationtaster.org/) to assess the impact of SNVs mutations to predict protein function.

### Mutant protein structure prediction

A single amino acid change has the potential to significantly affect protein activity and function. We downloaded the protein structures of CYP2D6 and KCNH2 from the PDB database (https://www.rcsb.org/) and utilized the Chimera v1.16 software to predict and visualize the mutant protein structures.

### Statistical analysis

The data were compiled, ordered, and analyzed using Microsoft Excel 2019 (Microsoft, Redmond, WA, USA) and SPSS 26.0 (SPSS, Chicago, IL, USA). The χ2 test was utilized to estimate the Hardy–Weinberg equilibrium (HWE) and compare the divergences in genotype frequencies of 48 VIP variants between the Zhuang population and the other 26 populations. All statistical tests were two-tailed (p < 0.05). Bonferroni corrections were performed to determine the significance level. After the Bonferroni’s multiple tests, p < 4.01 × 10−5 was recognized as statistically significant.

### Ethics approval

This study was conducted by the World Medical Association Declaration of Helsinki and was approved by the Northwestern University Clinical Research Ethics Committee (Approval number of Ethics Committee: 230,413,002). All subjects signed an informed consent form.

### Results

The 48 VIP variants on 24 genes that satisfied the HWE equation (p > 0.05) were collected in this study. Table 1 summarizes the fundamental characteristics of these variants, including gene name, SNVs ID, position, functional consequence, genotype frequency, and minor allele frequency (MAF) in the Zhuang population. Additionally, Table S1 shows the PCR primers for the gathered VIP variants.Table 1Basic information of 48 selected VIP variants in the Zhuang population.GenesSNVs IDChrBPFunctional consequenceZhuang AlleleGenotype frequenciesMAFABAAABBBCYP2J2rs11572325159,896,030Intron VariantTA0 (0.000)21 (0.105)179 (0.895)0.053rs10889160159,896,449Intron VariantCT1 (0.005)37 (0.185)162 (0.810)0.098rs890293159,926,822Upstream Transcript VariantAC0 (0.000)11 (0.055)189 (0.945)0.028DPYDrs1760217197,137,438Genic Downstream Transcript Variant, Intron VariantGA11 (0.055)80 (0.400)109 (0.545)0.255rs1801159197,515,839Coding Sequence Variant, Genic Downstream Transcript Variant, Intron Variant, Missense VariantCT26 (0.131)90 (0.455)82 (0.414)0.359rs1801265197,883,329Non-Coding Transcript Variant, Intron Variant, Coding Sequence Variant, 5 Prime UTR Variant, Missense VariantGA0 (0.000)40 (0.200)160 (0.800)0.100PTGS2rs52751186,673,9263 Prime UTR VariantGA9 (0.046)67 (0.340 121 (0.614)0.216CACNA1Srs121395271201,040,054Missense Variant, Coding Sequence Variant, Intron VariantGA2 (0.010)36 (0.182)160 (0.808)0.101rs38506251201,047,168Coding Sequence Variant, Missense VariantAG1 (0.005)7 (0.035)192 (0.960)0.023RYR2rs23062381237,550,803Intron VariantAG12 (0.060)62 (0.312)125 (0.628)0.216ABCG2rs2231142488,131,171Coding Sequence Variant, Missense VariantTG8 (0.040)65 (0.327)126 (0.633)0.204rs2231137488,139,962Coding Sequence Variant, Missense VariantTC27 (0.136)99 (0.497)73 (0.367)0.384ADH1Crs698499,339,632Coding Sequence Variant, Non-Coding Transcript Variant, Missense VariantCT2 (0.010)49 (0.245)149 (0.745)0.133CYP3A5rs776746799,672,916Intron Variant, splice acceptor variant, genic Downstream Transcript Variant, Downstream Transcript VariantTC19 (0.095)1 (0.005)180 (0.900)0.098CYP3A4rs2242480799,763,843Intron VariantTC16 (0.08)83 (0.415)101 (0.505)0.288NAT2rs4646244818,390,208Upstream Transcript Variant, Genic Upstream Transcript Variant, Intron VariantAT7 (0.035)66 (0.330)127 (0.635)0.200rs4271002818,390,758Upstream Transcript Variant, Genic Upstream Transcript Variant, Intron VariantCG4 (0.020)50 (0.253)144 (0.727)0.146rs1041983818,400,285Coding Sequence Variant, Synonymous VariantTC24 (0.120)96 (0.480)80 (0.400)0.360rs1801280818,400,344Missense Variant, Coding Sequence VariantCT1 (0.005)6 (0.030)193 (0.965)0.020rs1799929818,400,484Coding Sequence Variant, Synonymous VariantTC1 (0.005)7 (0.035)192 (0.960)0.023rs1799930818,400,593Missense Variant, Coding Sequence VariantAG7 (0.035)69 (0.347)123 (0.618)0.209rs1208818,400,806Missense Variant, Coding Sequence VariantGA1 (0.005)7 (0.035)192 (0.960)0.023rs1799931818,400,860Missense Variant, Coding Sequence VariantAG4 (0.020)50 (0.250)146 (0.730)0.145rs1495741818,415,371NoneAG28 (0.146)90 (0.469)74 (0.385)0.380ALOX5rs21158191045,405,641Intron VariantAG8 (0.040)36 (0.181)155 (0.779)0.131CYP2C19rs122485601094,761,900Upstream Transcript VariantTC0 (0.000)1 (0.005)199 (0.995)0.003rs42442851094,781,859Coding Sequence Variant, Synonymous VariantAG17 (0.085)85 (0.425)98 (0.490)0.298CYP2C8rs79092361095,069,673Upstream Transcript VariantTG2 (0.010)44 (0.220)154 (0.770)0.120rs171104531095,069,772Upstream Transcript VariantCA11 (0.055)82 (0.410)107 (0.535)0.260CYP2E1rs381386710133,526,101Non-Coding Transcript Variant, Upstream Transcript VariantCG4 (0.020)49 (0.245)147 (0.735)0.143rs641343210133,535,040Intron VariantAT0 (0.000)44 (0.229)148 (0.771)0.115rs207067610133,537,633Intron VariantGC10 (0.050)57 (0.285)133 (0.665)0.193KCNJ11rs52191117,388,025Missense Variant, Stop Gained, 5 Prime UTR Variant, Intron Variant, Coding Sequence VariantTC12 (0.061)123 (0.628)61 (0.311)0.375SLCO1B1rs23062831221,176,804Missense Variant, Coding Sequence VariantAG21 (0.106)71 (0.357)107 (0.538)0.284CYP1A2rs7625511574,749,576Intron VariantCA12 (0.060)90 (0.450)98 (0.490)0.285rs24723041574,751,897Intron VariantAG2 (0.010)43 (0.216)154 (0.774)0.118SULT1A1rs7501551628,609,2515 Prime UTR Variant, Intron Variant, Genic Upstream Transcript Variant, Upstream Transcript VariantCT28 (0.144)118 (0.608)48 (0.247)0.448ACErs18007641763,473,168NoneCT23 (0.116)102 (0.515)73 (0.369)0.374rs42911763,476,833Upstream Transcript VariantTA0 (0.000)177 (0.898)20 (0.102)0.449rs42673851763,506,395NoneTC14 (0.070)71 (0.357)114 (0.573)0.249CYP4F2rs21086221915,879,621Missense Variant, Coding Sequence VariantTC3 (0.015)66 (0.332)130 (0.653)0.181rs30931051915,897,578Missense Variant, Coding Sequence VariantCA0 (0.000)200 (1.000)0 (0.000)0.500CYP2A6rs81927261940,848,591Intron VariantAC7 (0.035)63 (0.315)130 (0.650)0.193SLC19A1rs10512982145,514,912Intron Variant, 3 Prime UTR VariantGA34 (0.172)120 (0.606)44 (0.222)0.475rs10512962145,514,947Intron Variant, 3 Prime UTR VariantAC24 (0.122)131 (0.668)41 (0.209)0.457rs11315962145,538,002Missense Variant, 5 Prime UTR Variant, Synonymous Variant, Genic Upstream Transcript Variant, Coding Sequence VariantAG32 (0.162)127 (0.644)38 (0.193)0.485CYP2D6rs10658522242,130,692Intron Variant, Missense Variant, Coding Sequence VariantAG45 (0.238)117 (0.619)27 (0.143)0.452KCNH2rs18051237150,948,446Missense Variant, Coding Sequence Variant, Genic Downstream Transcript VariantGT151 (0.774)44 (0.226)0 (0.000)0.113SNVs: single nucleotide variants, Chr: chromosome, BP: base pairs, ID: identity documents, MAF: minor allele frequency.

Basic information of 48 selected VIP variants in the Zhuang population.

SNVs: single nucleotide variants, Chr: chromosome, BP: base pairs, ID: identity documents, MAF: minor allele frequency.

We compared the discrepancies in the genotype frequency distribution of the selected VIP variants between the Zhuang population and 26 other populations based on Chi-square tests. After the Bonferroni correction, the results were considered significant when p < 4.01 × 10–5. The number of SNVs with significant differences in genotype frequencies between the Zhuang population and 26 populations is shown in Fig. 1. The investigation demonstrated that the Zhuang population exhibited significant differences in four SNVs when compared to CDX, CHS, and KHV, and 31 SNVs when compared to ESN, GWD, and YRI. The Zhuang population exhibited differences in a number of SNVs when compared to other populations, including JPT (8), KHV (5), ACB (27), ASN (24), LWK (29), MSL (28), CLM (22), MXL (17), PEL (18), PUR (18), CEU (21), FIN (22), GBR (22), IBS (20), TSI (23), BEB (15), GIH (23), ITH (20), PJL (22), and STU (22). Furthermore, the Zhuang population showed significant differences in rs776746 (CYP3A5), rs4291 (ACE), and rs1805123 (KCNH2) compared to 26 other populations. Moreover, the Zhuang population exhibited significant differences in rs1065852 (CYP2D6) compared to 21 other populations (refer to Table 2 and Table S2).Figure 1The amount of difference variants between the Zhuang population and 26 populations. The size of the rectangle indicates the number of different variants between the Zhuang population and the other 26 populations from five regions.Table 2Genotype frequency distribution differences of 26 populations compared with the Zhuang population after Bonferroni’s multiple adjustment.SNVs IDGenesEASAFRAMRCDXCHBCHSJPTKHVACBASWESNGWDLWKMSLYRICLMrs11572325CYP2J2–0.376–––8.15E-063.82E-051.96E-055.08E-051.51E-040.0014.14E-050.024rs10889160CYP2J20.6020.0140.0482.54E-040.2011.47E-199.71E-112.94E-228.02E-168.45E-171.82E-199.11E-220.304rs890293CYP2J2–0.338–––0.0011.17E-068.44E-101.99E-071.56E-062.57E-113.43E-07-rs1760217DPYD0.0540.3910.3060.0020.9780.0910.6970.6120.0070.0930.4360.0200.037rs1801159DPYD0.1470.0660.0010.0380.1912.47E-076.90E-058.52E-057.27E-130.3602.49E-114.31E-071.49E-04rs1801265DPYD0.012––0.028–1.64E-147.54E-191.56E-181.82E-213.06E-221.85E-141.34E-192.30E-05rs5275PTGS20.9470.0880.4520.5460.7809.28E-192.50E-126.19E-235.67E-186.59E-191.92E-213.04E-235.31E-05rs12139527CACNA1S0.9980.4160.9280.7230.5091.10E-231.42E-191.26E-313.10E-341.04E-261.08E-307.57E-310.451rs3850625CACNA1S0.4170.0040.3530.7620.1030.7740.5030.1310.0980.1310.1740.1091.52E-05rs2306238RYR20.1600.2530.2120.1280.6341.70E-050.0046.93E-081.02E-064.01E-064.26E-072.66E-060.133rs2231142ABCG20.7840.0140.2840.0030.0011.30E-090.0043.59E-111.42E-103.59E-115.40E-085.17E-120.063rs2231137ABCG20.0930.0900.3872.13E-070.9073.27E-171.40E-142.72E-172.11E-207.43E-116.58E-121.04E-172.06E-08rs698ADH1C0.5770.0050.0430.0480.1460.3360.4090.0440.2010.9460.3610.0480.001rs776746CYP3A58.15E-225.32E-201.62E-195.81E-226.45E-222.28E-435.81E-389.74E-501.68E-467.50E-472.72E-452.35E-511.03E-13rs2242480CYP3A40.7150.4150.4200.5400.6609.64E-261.65E-174.74E-387.33E-312.10E-381.13E-349.72E-330.978rs4646244NAT20.4560.9360.1620.0782.04E-040.2140.2430.1590.9090.4800.2150.9910.279rs4271002NAT22.87E-040.1340.1160.1530.3510.2530.6499.08E-052.06E-040.0910.3160.0720.185rs1041983NAT20.0020.9880.0680.2487.78E-050.0050.1481.35E-040.6130.2060.0010.0030.220rs1801280NAT20.0440.3820.1460.7150.0654.85E-194.86E-191.62E-192.96E-241.05E-275.47E-165.09E-169.78E-27rs1799929NAT20.0770.4980.2240.7620.2016.55E-156.25E-156.18E-133.38E-201.00E-239.37E-123.67E-091.12E-24rs1799930NAT20.7560.6550.4410.1633.97E-040.3390.1410.2000.4990.1970.3810.9230.338rs1208NAT20.0770.4980.1320.7620.0582.31E-257.37E-224.83E-279.05E-331.32E-324.64E-242.36E-271.05E-26rs1799931NAT20.0010.1370.3260.0980.4113.98E-040.0184.30E-061.84E-061.26E-060.0041.98E-040.024rs1495741NAT22.03E-040.9080.0140.5097.39E-061.42E-051.05E-072.29E-045.63E-053.17E-090.0020.0073.46E-12rs2115819ALOX50.0021.09E-040.0400.0030.0511.49E-388.51E-264.34E-384.74E-427.49E-349.70E-336.76E-414.00E-17rs12248560CYP2C19–––––4.50E-251.48E-177.51E-221.18E-211.46E-157.69E-234.81E-212.28E-11rs4244285CYP2C190.6060.1970.1000.8000.9050.0010.0020.0541.47E-050.0770.0100.0012.61E-06rs7909236CYP2C80.6200.7690.1020.1580.0110.0040.4541.43E-062.42E-076.42E-058.66E-064.56E-071.26E-06rs17110453CYP2C80.9270.1000.0260.0050.5751.75E-131.36E-097.43E-145.84E-173.10E-151.82E-131.21E-158.33E-06rs3813867CYP2E10.7790.0030.1400.2820.0670.0070.1310.0810.0565.46E-050.0230.0970.820rs6413432CYP2E11.81E-063.90E-077.62E-067.17E-062.88E-07––0.1190.031–0.0200.2370.039rs2070676CYP2E10.0210.2660.6770.4230.0581.78E-251.45E-131.04E-243.05E-268.00E-312.96E-263.50E-240.073rs5219KCNJ111.13E-040.0010.2300.0130.0342.98E-196.49E-095.01E-283.94E-293.34E-273.40E-258.78E-308.52E-07rs2306283SLCO1B10.0820.2870.0860.2070.1530.0860.6506.16E-050.0360.0040.0820.0231.48E-06rs762551CYP1A20.3020.0410.1360.0060.8320.0130.2193.48E-060.0014.53E-080.0022.86E-050.321rs2472304CYP1A20.3800.9690.1700.0080.1870.2050.5687.35E-064.86E-058.28E-054.17E-058.77E-063.48E-12rs750155SULT1A10.3855.03E-050.5963.47E-060.0750.0021.01E-048.71E-109.53E-138.18E-061.42E-066.52E-080.072rs1800764ACE0.5870.0850.5730.1750.0681.07E-264.95E-174.17E-346.98E-404.78E-295.98E-372.54E-390.080rs4291ACE2.44E-179.38E-227.87E-184.20E-151.50E-215.93E-164.76E-154.06E-171.65E-143.91E-211.36E-134.08E-222.53E-16rs4267385ACE0.5480.9870.4130.5800.4125.38E-261.54E-161.35E-271.57E-337.19E-351.15E-311.41E-319.42E-07rs2108622CYP4F20.6230.3520.6960.0030.0420.0100.0414.11E-061.84E-040.0580.0471.65E-050.004rs3093105CYP4F2–––2.56E-59–1.15E-371.86E-305.69E-332.44E-417.01E-382.46E-363.69E-355.66E-44rs8192726CYP2A60.4640.2250.0440.1150.0611.18E-040.0430.0103.55E-050.0060.0010.0053.04E-06rs1051298SLC19A10.0750.0730.3980.0650.0350.0040.1870.5731.26E-040.0460.0540.1490.097rs1051296SLC19A10.0110.0010.2750.0040.0110.0500.0030.0010.0021.82E-040.0340.0110.003rs1131596SLC19A10.0490.0170.7560.0130.2893.32E-090.0240.0023.17E-141.37E-103.85E-092.32E-080.022rs1065852CYP2D60.0010.0300.0015.47E-070.0057.31E-234.59E-188.94E-301.34E-288.17E-373.59E-209.51E-291.66E-19rs1805123KCNH21.30E-513.59E-604.17E-603.57E-593.69E-534.89E-626.05E-511.44E-641.08E-661.29E-631.58E-611.60E-661.41E-42SNVs IDGenesAMREURSASMXLPELPURCEUFINGBRIBSTSIBEBGIHITUPJLSTUrs11572325CYP2J2––7.89E-050.0394.74E-040.1580.0440.049––0.372––rs10889160CYP2J20.4580.4071.48E-040.3821.34E-050.3410.0160.1790.7850.1570.2030.8590.221rs890293CYP2J2––0.099–0.0450.252––––0.138––rs1760217DPYD0.8190.2060.0020.8620.0080.0130.0260.0660.1840.6711.07E-050.0044.81E-05rs1801159DPYD0.0200.0442.76E-043.12E-062.63E-073.21E-050.0010.0092.43E-094.13E-104.57E-151.80E-104.52E-13rs1801265DPYD9.52E-060.0131.00E-060.0242.85E-090.0081.50E-046.20E-050.0031.16E-114.66E-106.11E-090.001rs5275PTGS20.0053.84E-050.0178.01E-050.4570.0990.0070.0537.24E-051.07E-040.0015.02E-091.52E-05rs12139527CACNA1S0.0210.1700.0170.7520.9160.6260.8790.8770.0300.0060.5750.3370.355rs3850625CACNA1S1.91E-046.07E-050.1270.0013.04E-111.07E-070.0013.77E-052.24E-081.17E-192.68E-124.27E-117.10E-09rs2306238RYR20.0040.0010.0210.5310.3010.7940.8270.4990.1540.3580.0120.0060.003rs2231142ABCG20.9580.2800.0100.0300.0020.1559.27E-051.28E-050.0668.87E-050.0100.0090.001rs2231137ABCG20.0010.2371.38E-083.39E-194.61E-132.17E-184.02E-185.28E-150.0028.91E-091.42E-121.07E-111.12E-05rs698ADH1C2.91E-040.3542.54E-101.08E-175.94E-196.67E-149.12E-079.77E-070.3022.55E-050.0021.37E-072.15E-08rs776746CYP3A51.45E-171.19E-091.15E-151.61E-057.01E-061.47E-062.30E-071.23E-065.72E-245.31E-212.07E-221.10E-212.70E-22rs2242480CYP3A40.0681.40E-090.1375.05E-101.86E-082.29E-084.78E-065.34E-080.0880.4790.0600.0100.130rs4646244NAT20.1220.0040.8010.0030.0670.0180.0420.0720.2102.12E-054.70E-044.15E-049.95E-07rs4271002NAT20.5350.0150.2000.0270.5840.7790.7750.6470.2410.9300.4150.9090.788rs1041983NAT20.1850.0050.1840.0720.2240.0980.4800.2690.8700.1120.2890.2010.003rs1801280NAT24.50E-241.34E-201.78E-273.56E-312.53E-335.89E-331.73E-362.44E-321.69E-245.68E-241.80E-257.16E-311.41E-20rs1799929NAT21.84E-223.15E-191.08E-232.04E-303.87E-311.07E-301.03E-351.30E-312.11E-227.73E-217.06E-231.13E-269.82E-19rs1799930NAT20.1480.0010.7820.0070.2330.0190.0310.0970.2631.11E-050.0012.15E-042.54E-07rs1208NAT26.69E-287.16E-209.13E-274.93E-291.85E-301.02E-309.31E-362.80E-321.17E-262.75E-233.12E-247.53E-312.92E-22rs1799931NAT20.4790.0300.0733.37E-072.68E-044.28E-059.27E-054.32E-060.2420.0040.0070.0080.055rs1495741NAT21.12E-070.0047.38E-111.93E-131.66E-158.63E-166.88E-206.96E-141.18E-121.13E-173.28E-156.09E-212.66E-19rs2115819ALOX51.53E-112.59E-071.26E-154.32E-253.56E-203.24E-196.08E-217.28E-229.82E-174.99E-241.42E-211.00E-162.89E-15rs12248560CYP2C195.71E-09-7.65E-161.15E-201.68E-192.79E-211.33E-191.33E-19-7.28E-125.54E-123.36E-121.76E-11rs4244285CYP2C194.23E-042.99E-092.39E-054.15E-050.0583.32E-049.91E-056.12E-080.7020.2350.1570.0950.018rs7909236CYP2C81.27E-058.08E-100.0704.53E-053.35E-050.0060.0710.1320.0100.0010.0100.0160.227rs17110453CYP2C80.0011.36E-086.18E-052.48E-060.2733.68E-050.1899.12E-060.1050.0940.1580.3690.017rs3813867CYP2E10.8320.7800.0130.0133.79E-041.63E-041.83E-050.0031.20E-051.00E-061.16E-062.78E-063.11E-07rs6413432CYP2E10.1140.0080.0150.3610.0150.0600.0780.3350.0023.46E-060.0030.1060.002rs2070676CYP2E10.2990.1130.0200.0740.0050.0240.2530.0510.5780.3220.7050.4920.886rs5219KCNJ110.1030.1063.07E-040.3840.0370.0040.0180.0010.0020.0010.0132.17E-060.005rs2306283SLCO1B17.78E-103.53E-071.09E-048.99E-115.18E-095.74E-132.70E-111.20E-130.0040.0010.0229.27E-080.001rs762551CYP1A20.0292.17E-040.8420.5030.2550.3210.0540.0220.0015.13E-062.96E-055.70E-051.26E-07rs2472304CYP1A21.01E-040.9892.22E-191.27E-313.26E-243.18E-302.90E-287.29E-220.0840.2340.7900.0010.921rs750155SULT1A10.2181.17E-126.26E-050.0040.3300.0050.0023.76E-043.83E-126.89E-086.20E-112.10E-081.25E-17rs1800764ACE0.1290.0010.0790.0110.1720.1140.3840.0010.6850.1470.2100.5800.949rs4291ACE3.27E-192.23E-269.16E-161.28E-171.51E-122.50E-181.41E-158.52E-193.55E-166.16E-165.81E-225.34E-176.88E-16rs4267385ACE3.89E-040.8754.81E-103.57E-112.23E-101.49E-121.72E-133.89E-220.0744.20E-040.1312.91E-040.004rs2108622CYP4F20.1880.1230.0020.0160.3450.0013.00E-063.05E-068.50E-097.07E-116.54E-095.88E-085.84E-10rs3093105CYP4F21.39E-421.35E-551.54E-441.21E-441.84E-49–7.51E-351.34E-396.52E-451.47E-461.85E-423.34E-422.80E-46rs8192726CYP2A63.72E-044.46E-053.48E-062.15E-050.0906.09E-061.27E-051.38E-040.0960.2130.0190.3560.019rs1051298SLC19A12.48E-051.15E-040.0920.0050.0833.25E-040.0080.0230.1960.0290.2050.3420.021rs1051296SLC19A16.24E-089.47E-080.0017.75E-050.0021.09E-061.71E-041.69E-040.0920.0250.0020.0030.003rs1131596SLC19A14.37E-071.21E-050.0060.0450.0222.49E-060.0010.0030.0020.0014.88E-042.09E-040.001rs1065852CYP2D66.20E-188.28E-301.30E-218.70E-144.41E-235.54E-144.52E-218.40E-181.55E-131.24E-221.76E-204.59E-273.85E-23rs1805123KCNH26.73E-413.21E-513.55E-413.82E-401.03E-444.00E-368.87E-389.24E-374.67E-343.55E-391.83E-373.46E-427.87E-42Bolded font indicates significant results.EAS, East Asian; SAS, South Asian; EUR, European; AFR, African; AMR, American; CDX, Chinese Dai in Xishuangbanna, China; CHB, Han Chinese in Beijing, China; CHS, Southern Han Chinese, China; JPT, Japanese in Tokyo, Japan; KHV, Kinh in Ho Chi Minh City; Vietnam; BEB, Bengali in Bangladesh; GIH, Gujarati Indian in Houston, Texas; ITU, Indian Telugu in the UK; PJL, Punjabi in Lahore, Pakistan; STU, Sri Lankan Tamil in the UK; CEU, Western European ancestry; FIN, Finnish in Finland; GBR, British in England and Scotland; IBS, Iberian populations in Spain; TSI, Toscani in Italy; ACB, African Caribbeans in Barbados; ASW, African Ancestry in Southwest USA; ESN, Esan in Nigeria; GWD, Gambian in Western Divisions, The Gambia; LWK, Luhya in Webuye, Kenya; MSL, Mende in Sierra Leone; YRI, Yoruba in Ibadan, Nigeria; CLM, Colombian in Medellin, Colombia; MXL, Mexican Ancestry in Los Angeles, Colombia; PEL, Peruvian in Lima, Peru; PUR, Puerto Rican in Puerto Rico.

The amount of difference variants between the Zhuang population and 26 populations. The size of the rectangle indicates the number of different variants between the Zhuang population and the other 26 populations from five regions.

Genotype frequency distribution differences of 26 populations compared with the Zhuang population after Bonferroni’s multiple adjustment.

Bolded font indicates significant results.

EAS, East Asian; SAS, South Asian; EUR, European; AFR, African; AMR, American; CDX, Chinese Dai in Xishuangbanna, China; CHB, Han Chinese in Beijing, China; CHS, Southern Han Chinese, China; JPT, Japanese in Tokyo, Japan; KHV, Kinh in Ho Chi Minh City; Vietnam; BEB, Bengali in Bangladesh; GIH, Gujarati Indian in Houston, Texas; ITU, Indian Telugu in the UK; PJL, Punjabi in Lahore, Pakistan; STU, Sri Lankan Tamil in the UK; CEU, Western European ancestry; FIN, Finnish in Finland; GBR, British in England and Scotland; IBS, Iberian populations in Spain; TSI, Toscani in Italy; ACB, African Caribbeans in Barbados; ASW, African Ancestry in Southwest USA; ESN, Esan in Nigeria; GWD, Gambian in Western Divisions, The Gambia; LWK, Luhya in Webuye, Kenya; MSL, Mende in Sierra Leone; YRI, Yoruba in Ibadan, Nigeria; CLM, Colombian in Medellin, Colombia; MXL, Mexican Ancestry in Los Angeles, Colombia; PEL, Peruvian in Lima, Peru; PUR, Puerto Rican in Puerto Rico.

A model-based clustering approach was used to analyze the genetic structure of the 27 populations distributed in Africa, America, East Asia, Europe and South Asia to further analyze their relationship. Based on the Structure 2.3.1 Software, different K values ranging from 5 to 8 were hypothetically considered in structure analysis. When K = 5, the groups were divided into 5 subgroups based on the relative majority probability of assigning individuals to subgroups (subgroup 1: GWD and LWK; Subgroup 2: BEB, CEU, FIN, GBR, IBS, TSI, CLM, MXL and PUR; Subgroup 3: Zhuang, CDX, CHB, CHS, JPT, KHV and PUR; Subgroup 4: GIH, ITU, PJL and STU; Subgroup 5: ACB, ASW, ESN, MSL and YRI). It can be observed from Fig. 2 that Zhuang population have a stronger affinity with CDX, CHB, CHS, JPT, KHV and PUR. This is consistent with the results in Table 2.Figure 2Structure analysis of the genetic relationship between the Zhuang population and the other 26 populations. K denotes the possible numbers of parental population clusters. Each vertical bar represents a sample, dividing into color sections. K = 5 were utilized to evaluate the relationship between Zhuang and 26 populations.

Structure analysis of the genetic relationship between the Zhuang population and the other 26 populations. K denotes the possible numbers of parental population clusters. Each vertical bar represents a sample, dividing into color sections. K = 5 were utilized to evaluate the relationship between Zhuang and 26 populations.

The pairwise Fst values were used to assess relationships among 27 populations, as shown in Table 3 and Fig. 3A. The Fst values between the Zhuang population and the East Asian population (CDX, CHB, CHS, JPT and KHV) were small, which were 0.065, 0.068, 0.066, 0.073 and 0.067, respectively (Table 3). Smaller Fst values indicate closer relationships between the two groups and suggest that they share similar genetic backgrounds. The result is confirmed by the phylogenetic trees of 27 populations shown in Fig. 3B.Table 3Pairwise Fst values among the Zhuang and 26 populations.ZhuangCDXCHBCHSJPTKHVBEBGIHITUPJLSTUCEUFINGBRIBSTSIACBASWESNGWDLWKZhuang0.000CDX0.0650.000CHB0.0680.0130.000CHS0.0660.0090.0050.000JPT0.0730.0260.0120.0150.000KHV0.0690.0070.0130.0060.0240.000BEB0.0920.0670.0640.0640.0600.0660.000GIH0.1080.0820.0810.0790.0720.0780.0100.000ITU0.1030.0820.0760.0750.0690.0780.0060.0050.000PJL0.1220.0950.0940.0910.0820.0910.0100.0070.0060.000STU0.1080.0770.0750.0700.0650.0710.0120.0110.0070.0110.000CEU0.1400.1230.1240.1220.1110.1220.0570.0540.0580.0470.0690.000FIN0.1380.1150.1150.1100.1040.1150.0490.0420.0470.0370.0570.0100.000GBR0.1460.1280.1310.1280.1190.1280.0570.0560.0590.0470.0690.0050.0100.000IBS0.1290.1160.1190.1140.1050.1150.0420.0400.0440.0330.0530.0110.0120.0090.000TSI0.1250.1140.1170.1160.1050.1150.0480.0450.0500.0410.0590.0110.0150.0080.0060.000ACB0.2370.1820.1950.1960.1800.1880.1460.1450.1410.1360.1430.1790.1750.1860.1620.1580.000ASW0.1980.1470.1580.1610.1460.1540.1070.1070.1020.0970.1080.1380.1340.1440.1250.1210.0090.000ESN0.2830.2280.2360.2400.2210.2320.1880.1870.1790.1760.1800.2300.2250.2370.2130.2060.0130.0210.000GWD0.2800.2260.2350.2390.2190.2310.1790.1780.1700.1680.1760.2130.2080.2190.1940.1870.0110.0190.0130.000LWK0.2830.2270.2410.2420.2260.2330.1810.1790.1740.1650.1740.2220.2130.2250.1990.1940.0150.0230.0120.0160.000MSL0.2770.2220.2330.2360.2180.2270.1880.1880.1810.1770.1810.2290.2250.2350.2100.2040.0090.0220.0060.0100.016YRI0.2720.2200.2280.2330.2130.2260.1840.1830.1760.1740.1780.2220.2180.2290.2040.1970.0080.0200.0040.0080.015CLM0.1070.0770.0840.0820.0750.0800.0290.0310.0330.0280.0420.0210.0210.0200.0180.0170.1280.0930.1720.1570.164MXL0.1120.0840.0820.0870.0800.0900.0320.0420.0400.0320.0520.0400.0360.0390.0380.0370.1590.1160.2000.1900.190PEL0.1250.0910.0970.0980.1010.1020.0880.1000.0980.0960.1080.1100.1000.1100.1070.1040.2010.1650.2420.2370.231PUR0.1030.0780.0840.0820.0740.0840.0280.0350.0320.0260.0390.0230.0210.0210.0180.0170.1120.0790.1530.1390.143LWKMSLYRICLMMXLPELPURZhuangCDXCHBCHSJPTKHVBEBGIHITUPJLSTUCEUFINGBRIBSTSIACBASWESNGWDLWK0.000MSL0.0160.000YRI0.0150.0040.000CLM0.1640.1710.1650.000MXL0.1900.2010.1940.0170.000PEL0.2310.2440.2350.0580.0350.000PUR0.1430.1510.1460.0090.0190.0670.000Figure 3Fst value heamap and phylogenetic tree among 27 populations. (A) Heatmap based on the pairwise Fst values between 27 populations. (B) The phylogenetic tree was constructed by the neighboring-joining method among 27 populations.

Pairwise Fst values among the Zhuang and 26 populations.

Fst value heamap and phylogenetic tree among 27 populations. (A) Heatmap based on the pairwise Fst values between 27 populations. (B) The phylogenetic tree was constructed by the neighboring-joining method among 27 populations.

Moreover, the genotype frequency distribution of rs776746 (CYP3A5), rs4291 (ACE), rs1805123 (KCNH2), and rs1065852 (CYP2D6) in 26 populations are shown in Fig. 4. The genotype frequency of rs4291-AT in the Zhuang population is remarkably higher than that of the other 26 populations. The CC genotype frequency of rs776746 is similar to that of EUR and significantly higher than that of AFR. In the Zhuang population, the frequency of the rs1805123-GG genotype is notably higher compared to that observed in the other 26 populations. The frequency of rs1065852-GG is similar to that of KHV, CHS, CHB, and CDX, and lower than that of other populations.Figure 4The distribution of genotype frequencies for significantly different SNVs in 27 populations at the rs776746, rs4291, rs1805123 and rs1065852.

The distribution of genotype frequencies for significantly different SNVs in 27 populations at the rs776746, rs4291, rs1805123 and rs1065852.

Based on the allele frequencies calculated in this study, we plotted a map of the MAF distribution of VIP variants that substantially differed from the other 26 populations. According to Fig. 5, the allele frequency of rs776746 at CYP3A5 in the Zhuang population was similar to that in the European population, despite their close genetic affinity with East Asians. The G allele of rs1805123 at KCNH2 was nearly fixed in the Zhuang population and had a low frequency in other global populations. The MAF of rs1065852 (CYP2D6) was similar to that of East Asians and higher than other populations. However, there were no significant differences in T allele frequency for rs4291 at ACE among different populations.Figure 5The map of the allele frequency distribution for significantly different SNVs in 27 populations at the rs1065852, rs4291, rs776746 and rs1805123.

The map of the allele frequency distribution for significantly different SNVs in 27 populations at the rs1065852, rs4291, rs776746 and rs1805123.

The Table 4 presents the clinical annotation information of the VIP variants in PharmGKB. The genotype frequency of rs776746 (CYP3A5) has been shown to have an impact on the dose, toxicity and metabolism of tacrolimus19–21. Specific mutations in rs4291 (ACE) have been implicated in the metabolism of anti-hypertensive drugs such as amlodipine, sodium chlorthalidone and lisinopril22. Furthermore, they also influenced the risk of aspirin intolerance in asthmatics exposed to aspirin23. The efficacy of metoclopramide in patients with gastric disease was found to be associated with rs1805123 (KCNH2)24. Rs1065852 played an indispensable role in the regulation of the α-hydroxymetoprolol metabolism in patients with non-small cell lung cancer25.Table 4Clinical annotation of very important pharmacogenomic variants with significant differences.GeneVariantPMIDMoleculesAssociationP-valueTypePhenotypeCYP3A5rs77674623,073,468TacrolimusGenotype CC is associated with decreased dose of tacrolimus in people with Kidney Transplantation as compared to genotypes CT + TT0.016DosageKidney TransplantCYP3A5rs77674621,677,300TacrolimusAllele T is associated with increased risk of tacrolimus nephrotoxicity when treated with tacrolimus in people with Kidney Transplantation as compared to allele C0.025ToxicityKidney TransplantCYP3A5rs77674624,120,259TacrolimusGenotype CT is associated with increased dose of tacrolimus in people with Kidney Transplantation as compared to genotype CC < 0.001Metabolism/PKKidney TransplantACErs429127,546,928CaptoprilGenotype AA is associated with decreased severity of Kidney Failure when treated with captopril in people with Alzheimer Disease as compared to genotypes AT + TT0.029EfficacyAlzheimer diseaseACErs429118,727,619AspirinGenotypes AT + TT are associated with increased risk of aspirin intolerance when exposed to aspirin in people with Asthma as compared to genotype AA0.015Toxicity/ADRAsthmaACErs429120,577,119Amlodipine/lisinopril/chlorthalidoneGenotypes AA + AT are associated with decreased fasting glucose when treated with amlodipine, chlorthalidone or lisinopril in people with Hypertension as compared to genotype TT0.001EfficacyAnti-HypertensionCYP2D6rs106585210,223,777Alpha-hydroxymetoprololAllele A is associated with decreased clearance of alpha-hydroxymetoprolol in healthy individuals as compared to allele G < 0.050Metabolism/PKCarcinoma, Non-Small-Cell LungMesotheliomaCYP2D6rs106585224,528,284CitalopramescitalopramAllele A is associated with plasma concentration of S-didesmethyl-citalopram when treated with citalopram or escitalopram in people with Depressive Disorder, Major as compared to allele G2E-16OtherDepressive DisorderCYP2D6rs106585223,277,250IloperidoneGenotype GG is associated with increased QTc interval when treated with iloperidone in people with Schizophrenia as compared to genotypes AA + AG0.028OtherSchizophreniaKCNH2rs180512322,688,145MetoclopramideThe efficacy of metoclopramide in patients with gastric disease was correlated with the polymorphism of KCNH2 (rs1805123, P = 0.020) gene0.020Dose effectGastric diseaseADR: Adverse drug reactions.

Clinical annotation of very important pharmacogenomic variants with significant differences.

ADR: Adverse drug reactions.

Subsequently, we used the PolyPhen-2, SNAP2, FATHMM, Mutationtaster, and Mutationassessor online databases to predicte whether the four SNVs would affect protein structure and function (Table 5). The results indicated that rs1805123 would cause a mutation from K to T at position 897 of KCNH2, however, this mutation was considered benign and less harmful to the protein in most databases. In contrast, rs1065852 caused a mutation from P to A at the 34th position of CYP2D6. The database predicted that this change would severely impair the protein's function and potentially contribute to certain diseases. Additionally, Chimera v1.16 was utilized for predicting the structure of point mutations of CYP2D6 and KCNH2, as shown in Fig. 6.Table 5The functional analysis of missense variants using PolyPhen-2, SNAP2, Mutationassessor, FATHMM, and Mutationtaster.SNVs IDGeneAA changePolyPhen-2SNAP2FATHMMMutation tasterMutationassessorScorePredicted effectScorePredicted effectCoding ScorePredicted effectProbPredictedFunc.ImpactFI scorers1805123KCNH2K897T0Benign-53Neutral0.760pathogenic0.205polymorphismlow1.735rs1065852CYP2D6P34A0.953Deleterious60Effect0.820pathogenic0.999disease causinghigh4.080Figure 6Structural prediction of point mutated proteins. (A) 3D structure of the CYP2D6 protein, with the yellow part being a SNV. (B) rs1065852 mutated local structure. (C) 3D structure of the KCNH2 protein, with the yellow part being a SNV mutation. (D) rs1805123 mutated local structure.

The functional analysis of missense variants using PolyPhen-2, SNAP2, Mutationassessor, FATHMM, and Mutationtaster.

Structural prediction of point mutated proteins. (A) 3D structure of the CYP2D6 protein, with the yellow part being a SNV. (B) rs1065852 mutated local structure. (C) 3D structure of the KCNH2 protein, with the yellow part being a SNV mutation. (D) rs1805123 mutated local structure.

### Basic characteristics of candidate VIP variants

The 48 VIP variants on 24 genes that satisfied the HWE equation (p > 0.05) were collected in this study. Table 1 summarizes the fundamental characteristics of these variants, including gene name, SNVs ID, position, functional consequence, genotype frequency, and minor allele frequency (MAF) in the Zhuang population. Additionally, Table S1 shows the PCR primers for the gathered VIP variants.Table 1Basic information of 48 selected VIP variants in the Zhuang population.GenesSNVs IDChrBPFunctional consequenceZhuang AlleleGenotype frequenciesMAFABAAABBBCYP2J2rs11572325159,896,030Intron VariantTA0 (0.000)21 (0.105)179 (0.895)0.053rs10889160159,896,449Intron VariantCT1 (0.005)37 (0.185)162 (0.810)0.098rs890293159,926,822Upstream Transcript VariantAC0 (0.000)11 (0.055)189 (0.945)0.028DPYDrs1760217197,137,438Genic Downstream Transcript Variant, Intron VariantGA11 (0.055)80 (0.400)109 (0.545)0.255rs1801159197,515,839Coding Sequence Variant, Genic Downstream Transcript Variant, Intron Variant, Missense VariantCT26 (0.131)90 (0.455)82 (0.414)0.359rs1801265197,883,329Non-Coding Transcript Variant, Intron Variant, Coding Sequence Variant, 5 Prime UTR Variant, Missense VariantGA0 (0.000)40 (0.200)160 (0.800)0.100PTGS2rs52751186,673,9263 Prime UTR VariantGA9 (0.046)67 (0.340 121 (0.614)0.216CACNA1Srs121395271201,040,054Missense Variant, Coding Sequence Variant, Intron VariantGA2 (0.010)36 (0.182)160 (0.808)0.101rs38506251201,047,168Coding Sequence Variant, Missense VariantAG1 (0.005)7 (0.035)192 (0.960)0.023RYR2rs23062381237,550,803Intron VariantAG12 (0.060)62 (0.312)125 (0.628)0.216ABCG2rs2231142488,131,171Coding Sequence Variant, Missense VariantTG8 (0.040)65 (0.327)126 (0.633)0.204rs2231137488,139,962Coding Sequence Variant, Missense VariantTC27 (0.136)99 (0.497)73 (0.367)0.384ADH1Crs698499,339,632Coding Sequence Variant, Non-Coding Transcript Variant, Missense VariantCT2 (0.010)49 (0.245)149 (0.745)0.133CYP3A5rs776746799,672,916Intron Variant, splice acceptor variant, genic Downstream Transcript Variant, Downstream Transcript VariantTC19 (0.095)1 (0.005)180 (0.900)0.098CYP3A4rs2242480799,763,843Intron VariantTC16 (0.08)83 (0.415)101 (0.505)0.288NAT2rs4646244818,390,208Upstream Transcript Variant, Genic Upstream Transcript Variant, Intron VariantAT7 (0.035)66 (0.330)127 (0.635)0.200rs4271002818,390,758Upstream Transcript Variant, Genic Upstream Transcript Variant, Intron VariantCG4 (0.020)50 (0.253)144 (0.727)0.146rs1041983818,400,285Coding Sequence Variant, Synonymous VariantTC24 (0.120)96 (0.480)80 (0.400)0.360rs1801280818,400,344Missense Variant, Coding Sequence VariantCT1 (0.005)6 (0.030)193 (0.965)0.020rs1799929818,400,484Coding Sequence Variant, Synonymous VariantTC1 (0.005)7 (0.035)192 (0.960)0.023rs1799930818,400,593Missense Variant, Coding Sequence VariantAG7 (0.035)69 (0.347)123 (0.618)0.209rs1208818,400,806Missense Variant, Coding Sequence VariantGA1 (0.005)7 (0.035)192 (0.960)0.023rs1799931818,400,860Missense Variant, Coding Sequence VariantAG4 (0.020)50 (0.250)146 (0.730)0.145rs1495741818,415,371NoneAG28 (0.146)90 (0.469)74 (0.385)0.380ALOX5rs21158191045,405,641Intron VariantAG8 (0.040)36 (0.181)155 (0.779)0.131CYP2C19rs122485601094,761,900Upstream Transcript VariantTC0 (0.000)1 (0.005)199 (0.995)0.003rs42442851094,781,859Coding Sequence Variant, Synonymous VariantAG17 (0.085)85 (0.425)98 (0.490)0.298CYP2C8rs79092361095,069,673Upstream Transcript VariantTG2 (0.010)44 (0.220)154 (0.770)0.120rs171104531095,069,772Upstream Transcript VariantCA11 (0.055)82 (0.410)107 (0.535)0.260CYP2E1rs381386710133,526,101Non-Coding Transcript Variant, Upstream Transcript VariantCG4 (0.020)49 (0.245)147 (0.735)0.143rs641343210133,535,040Intron VariantAT0 (0.000)44 (0.229)148 (0.771)0.115rs207067610133,537,633Intron VariantGC10 (0.050)57 (0.285)133 (0.665)0.193KCNJ11rs52191117,388,025Missense Variant, Stop Gained, 5 Prime UTR Variant, Intron Variant, Coding Sequence VariantTC12 (0.061)123 (0.628)61 (0.311)0.375SLCO1B1rs23062831221,176,804Missense Variant, Coding Sequence VariantAG21 (0.106)71 (0.357)107 (0.538)0.284CYP1A2rs7625511574,749,576Intron VariantCA12 (0.060)90 (0.450)98 (0.490)0.285rs24723041574,751,897Intron VariantAG2 (0.010)43 (0.216)154 (0.774)0.118SULT1A1rs7501551628,609,2515 Prime UTR Variant, Intron Variant, Genic Upstream Transcript Variant, Upstream Transcript VariantCT28 (0.144)118 (0.608)48 (0.247)0.448ACErs18007641763,473,168NoneCT23 (0.116)102 (0.515)73 (0.369)0.374rs42911763,476,833Upstream Transcript VariantTA0 (0.000)177 (0.898)20 (0.102)0.449rs42673851763,506,395NoneTC14 (0.070)71 (0.357)114 (0.573)0.249CYP4F2rs21086221915,879,621Missense Variant, Coding Sequence VariantTC3 (0.015)66 (0.332)130 (0.653)0.181rs30931051915,897,578Missense Variant, Coding Sequence VariantCA0 (0.000)200 (1.000)0 (0.000)0.500CYP2A6rs81927261940,848,591Intron VariantAC7 (0.035)63 (0.315)130 (0.650)0.193SLC19A1rs10512982145,514,912Intron Variant, 3 Prime UTR VariantGA34 (0.172)120 (0.606)44 (0.222)0.475rs10512962145,514,947Intron Variant, 3 Prime UTR VariantAC24 (0.122)131 (0.668)41 (0.209)0.457rs11315962145,538,002Missense Variant, 5 Prime UTR Variant, Synonymous Variant, Genic Upstream Transcript Variant, Coding Sequence VariantAG32 (0.162)127 (0.644)38 (0.193)0.485CYP2D6rs10658522242,130,692Intron Variant, Missense Variant, Coding Sequence VariantAG45 (0.238)117 (0.619)27 (0.143)0.452KCNH2rs18051237150,948,446Missense Variant, Coding Sequence Variant, Genic Downstream Transcript VariantGT151 (0.774)44 (0.226)0 (0.000)0.113SNVs: single nucleotide variants, Chr: chromosome, BP: base pairs, ID: identity documents, MAF: minor allele frequency.

Basic information of 48 selected VIP variants in the Zhuang population.

SNVs: single nucleotide variants, Chr: chromosome, BP: base pairs, ID: identity documents, MAF: minor allele frequency.

### SNVs with significant differences in genotype frequencies between the Zhuang population and the other 26 populations

We compared the discrepancies in the genotype frequency distribution of the selected VIP variants between the Zhuang population and 26 other populations based on Chi-square tests. After the Bonferroni correction, the results were considered significant when p < 4.01 × 10–5. The number of SNVs with significant differences in genotype frequencies between the Zhuang population and 26 populations is shown in Fig. 1. The investigation demonstrated that the Zhuang population exhibited significant differences in four SNVs when compared to CDX, CHS, and KHV, and 31 SNVs when compared to ESN, GWD, and YRI. The Zhuang population exhibited differences in a number of SNVs when compared to other populations, including JPT (8), KHV (5), ACB (27), ASN (24), LWK (29), MSL (28), CLM (22), MXL (17), PEL (18), PUR (18), CEU (21), FIN (22), GBR (22), IBS (20), TSI (23), BEB (15), GIH (23), ITH (20), PJL (22), and STU (22). Furthermore, the Zhuang population showed significant differences in rs776746 (CYP3A5), rs4291 (ACE), and rs1805123 (KCNH2) compared to 26 other populations. Moreover, the Zhuang population exhibited significant differences in rs1065852 (CYP2D6) compared to 21 other populations (refer to Table 2 and Table S2).Figure 1The amount of difference variants between the Zhuang population and 26 populations. The size of the rectangle indicates the number of different variants between the Zhuang population and the other 26 populations from five regions.Table 2Genotype frequency distribution differences of 26 populations compared with the Zhuang population after Bonferroni’s multiple adjustment.SNVs IDGenesEASAFRAMRCDXCHBCHSJPTKHVACBASWESNGWDLWKMSLYRICLMrs11572325CYP2J2–0.376–––8.15E-063.82E-051.96E-055.08E-051.51E-040.0014.14E-050.024rs10889160CYP2J20.6020.0140.0482.54E-040.2011.47E-199.71E-112.94E-228.02E-168.45E-171.82E-199.11E-220.304rs890293CYP2J2–0.338–––0.0011.17E-068.44E-101.99E-071.56E-062.57E-113.43E-07-rs1760217DPYD0.0540.3910.3060.0020.9780.0910.6970.6120.0070.0930.4360.0200.037rs1801159DPYD0.1470.0660.0010.0380.1912.47E-076.90E-058.52E-057.27E-130.3602.49E-114.31E-071.49E-04rs1801265DPYD0.012––0.028–1.64E-147.54E-191.56E-181.82E-213.06E-221.85E-141.34E-192.30E-05rs5275PTGS20.9470.0880.4520.5460.7809.28E-192.50E-126.19E-235.67E-186.59E-191.92E-213.04E-235.31E-05rs12139527CACNA1S0.9980.4160.9280.7230.5091.10E-231.42E-191.26E-313.10E-341.04E-261.08E-307.57E-310.451rs3850625CACNA1S0.4170.0040.3530.7620.1030.7740.5030.1310.0980.1310.1740.1091.52E-05rs2306238RYR20.1600.2530.2120.1280.6341.70E-050.0046.93E-081.02E-064.01E-064.26E-072.66E-060.133rs2231142ABCG20.7840.0140.2840.0030.0011.30E-090.0043.59E-111.42E-103.59E-115.40E-085.17E-120.063rs2231137ABCG20.0930.0900.3872.13E-070.9073.27E-171.40E-142.72E-172.11E-207.43E-116.58E-121.04E-172.06E-08rs698ADH1C0.5770.0050.0430.0480.1460.3360.4090.0440.2010.9460.3610.0480.001rs776746CYP3A58.15E-225.32E-201.62E-195.81E-226.45E-222.28E-435.81E-389.74E-501.68E-467.50E-472.72E-452.35E-511.03E-13rs2242480CYP3A40.7150.4150.4200.5400.6609.64E-261.65E-174.74E-387.33E-312.10E-381.13E-349.72E-330.978rs4646244NAT20.4560.9360.1620.0782.04E-040.2140.2430.1590.9090.4800.2150.9910.279rs4271002NAT22.87E-040.1340.1160.1530.3510.2530.6499.08E-052.06E-040.0910.3160.0720.185rs1041983NAT20.0020.9880.0680.2487.78E-050.0050.1481.35E-040.6130.2060.0010.0030.220rs1801280NAT20.0440.3820.1460.7150.0654.85E-194.86E-191.62E-192.96E-241.05E-275.47E-165.09E-169.78E-27rs1799929NAT20.0770.4980.2240.7620.2016.55E-156.25E-156.18E-133.38E-201.00E-239.37E-123.67E-091.12E-24rs1799930NAT20.7560.6550.4410.1633.97E-040.3390.1410.2000.4990.1970.3810.9230.338rs1208NAT20.0770.4980.1320.7620.0582.31E-257.37E-224.83E-279.05E-331.32E-324.64E-242.36E-271.05E-26rs1799931NAT20.0010.1370.3260.0980.4113.98E-040.0184.30E-061.84E-061.26E-060.0041.98E-040.024rs1495741NAT22.03E-040.9080.0140.5097.39E-061.42E-051.05E-072.29E-045.63E-053.17E-090.0020.0073.46E-12rs2115819ALOX50.0021.09E-040.0400.0030.0511.49E-388.51E-264.34E-384.74E-427.49E-349.70E-336.76E-414.00E-17rs12248560CYP2C19–––––4.50E-251.48E-177.51E-221.18E-211.46E-157.69E-234.81E-212.28E-11rs4244285CYP2C190.6060.1970.1000.8000.9050.0010.0020.0541.47E-050.0770.0100.0012.61E-06rs7909236CYP2C80.6200.7690.1020.1580.0110.0040.4541.43E-062.42E-076.42E-058.66E-064.56E-071.26E-06rs17110453CYP2C80.9270.1000.0260.0050.5751.75E-131.36E-097.43E-145.84E-173.10E-151.82E-131.21E-158.33E-06rs3813867CYP2E10.7790.0030.1400.2820.0670.0070.1310.0810.0565.46E-050.0230.0970.820rs6413432CYP2E11.81E-063.90E-077.62E-067.17E-062.88E-07––0.1190.031–0.0200.2370.039rs2070676CYP2E10.0210.2660.6770.4230.0581.78E-251.45E-131.04E-243.05E-268.00E-312.96E-263.50E-240.073rs5219KCNJ111.13E-040.0010.2300.0130.0342.98E-196.49E-095.01E-283.94E-293.34E-273.40E-258.78E-308.52E-07rs2306283SLCO1B10.0820.2870.0860.2070.1530.0860.6506.16E-050.0360.0040.0820.0231.48E-06rs762551CYP1A20.3020.0410.1360.0060.8320.0130.2193.48E-060.0014.53E-080.0022.86E-050.321rs2472304CYP1A20.3800.9690.1700.0080.1870.2050.5687.35E-064.86E-058.28E-054.17E-058.77E-063.48E-12rs750155SULT1A10.3855.03E-050.5963.47E-060.0750.0021.01E-048.71E-109.53E-138.18E-061.42E-066.52E-080.072rs1800764ACE0.5870.0850.5730.1750.0681.07E-264.95E-174.17E-346.98E-404.78E-295.98E-372.54E-390.080rs4291ACE2.44E-179.38E-227.87E-184.20E-151.50E-215.93E-164.76E-154.06E-171.65E-143.91E-211.36E-134.08E-222.53E-16rs4267385ACE0.5480.9870.4130.5800.4125.38E-261.54E-161.35E-271.57E-337.19E-351.15E-311.41E-319.42E-07rs2108622CYP4F20.6230.3520.6960.0030.0420.0100.0414.11E-061.84E-040.0580.0471.65E-050.004rs3093105CYP4F2–––2.56E-59–1.15E-371.86E-305.69E-332.44E-417.01E-382.46E-363.69E-355.66E-44rs8192726CYP2A60.4640.2250.0440.1150.0611.18E-040.0430.0103.55E-050.0060.0010.0053.04E-06rs1051298SLC19A10.0750.0730.3980.0650.0350.0040.1870.5731.26E-040.0460.0540.1490.097rs1051296SLC19A10.0110.0010.2750.0040.0110.0500.0030.0010.0021.82E-040.0340.0110.003rs1131596SLC19A10.0490.0170.7560.0130.2893.32E-090.0240.0023.17E-141.37E-103.85E-092.32E-080.022rs1065852CYP2D60.0010.0300.0015.47E-070.0057.31E-234.59E-188.94E-301.34E-288.17E-373.59E-209.51E-291.66E-19rs1805123KCNH21.30E-513.59E-604.17E-603.57E-593.69E-534.89E-626.05E-511.44E-641.08E-661.29E-631.58E-611.60E-661.41E-42SNVs IDGenesAMREURSASMXLPELPURCEUFINGBRIBSTSIBEBGIHITUPJLSTUrs11572325CYP2J2––7.89E-050.0394.74E-040.1580.0440.049––0.372––rs10889160CYP2J20.4580.4071.48E-040.3821.34E-050.3410.0160.1790.7850.1570.2030.8590.221rs890293CYP2J2––0.099–0.0450.252––––0.138––rs1760217DPYD0.8190.2060.0020.8620.0080.0130.0260.0660.1840.6711.07E-050.0044.81E-05rs1801159DPYD0.0200.0442.76E-043.12E-062.63E-073.21E-050.0010.0092.43E-094.13E-104.57E-151.80E-104.52E-13rs1801265DPYD9.52E-060.0131.00E-060.0242.85E-090.0081.50E-046.20E-050.0031.16E-114.66E-106.11E-090.001rs5275PTGS20.0053.84E-050.0178.01E-050.4570.0990.0070.0537.24E-051.07E-040.0015.02E-091.52E-05rs12139527CACNA1S0.0210.1700.0170.7520.9160.6260.8790.8770.0300.0060.5750.3370.355rs3850625CACNA1S1.91E-046.07E-050.1270.0013.04E-111.07E-070.0013.77E-052.24E-081.17E-192.68E-124.27E-117.10E-09rs2306238RYR20.0040.0010.0210.5310.3010.7940.8270.4990.1540.3580.0120.0060.003rs2231142ABCG20.9580.2800.0100.0300.0020.1559.27E-051.28E-050.0668.87E-050.0100.0090.001rs2231137ABCG20.0010.2371.38E-083.39E-194.61E-132.17E-184.02E-185.28E-150.0028.91E-091.42E-121.07E-111.12E-05rs698ADH1C2.91E-040.3542.54E-101.08E-175.94E-196.67E-149.12E-079.77E-070.3022.55E-050.0021.37E-072.15E-08rs776746CYP3A51.45E-171.19E-091.15E-151.61E-057.01E-061.47E-062.30E-071.23E-065.72E-245.31E-212.07E-221.10E-212.70E-22rs2242480CYP3A40.0681.40E-090.1375.05E-101.86E-082.29E-084.78E-065.34E-080.0880.4790.0600.0100.130rs4646244NAT20.1220.0040.8010.0030.0670.0180.0420.0720.2102.12E-054.70E-044.15E-049.95E-07rs4271002NAT20.5350.0150.2000.0270.5840.7790.7750.6470.2410.9300.4150.9090.788rs1041983NAT20.1850.0050.1840.0720.2240.0980.4800.2690.8700.1120.2890.2010.003rs1801280NAT24.50E-241.34E-201.78E-273.56E-312.53E-335.89E-331.73E-362.44E-321.69E-245.68E-241.80E-257.16E-311.41E-20rs1799929NAT21.84E-223.15E-191.08E-232.04E-303.87E-311.07E-301.03E-351.30E-312.11E-227.73E-217.06E-231.13E-269.82E-19rs1799930NAT20.1480.0010.7820.0070.2330.0190.0310.0970.2631.11E-050.0012.15E-042.54E-07rs1208NAT26.69E-287.16E-209.13E-274.93E-291.85E-301.02E-309.31E-362.80E-321.17E-262.75E-233.12E-247.53E-312.92E-22rs1799931NAT20.4790.0300.0733.37E-072.68E-044.28E-059.27E-054.32E-060.2420.0040.0070.0080.055rs1495741NAT21.12E-070.0047.38E-111.93E-131.66E-158.63E-166.88E-206.96E-141.18E-121.13E-173.28E-156.09E-212.66E-19rs2115819ALOX51.53E-112.59E-071.26E-154.32E-253.56E-203.24E-196.08E-217.28E-229.82E-174.99E-241.42E-211.00E-162.89E-15rs12248560CYP2C195.71E-09-7.65E-161.15E-201.68E-192.79E-211.33E-191.33E-19-7.28E-125.54E-123.36E-121.76E-11rs4244285CYP2C194.23E-042.99E-092.39E-054.15E-050.0583.32E-049.91E-056.12E-080.7020.2350.1570.0950.018rs7909236CYP2C81.27E-058.08E-100.0704.53E-053.35E-050.0060.0710.1320.0100.0010.0100.0160.227rs17110453CYP2C80.0011.36E-086.18E-052.48E-060.2733.68E-050.1899.12E-060.1050.0940.1580.3690.017rs3813867CYP2E10.8320.7800.0130.0133.79E-041.63E-041.83E-050.0031.20E-051.00E-061.16E-062.78E-063.11E-07rs6413432CYP2E10.1140.0080.0150.3610.0150.0600.0780.3350.0023.46E-060.0030.1060.002rs2070676CYP2E10.2990.1130.0200.0740.0050.0240.2530.0510.5780.3220.7050.4920.886rs5219KCNJ110.1030.1063.07E-040.3840.0370.0040.0180.0010.0020.0010.0132.17E-060.005rs2306283SLCO1B17.78E-103.53E-071.09E-048.99E-115.18E-095.74E-132.70E-111.20E-130.0040.0010.0229.27E-080.001rs762551CYP1A20.0292.17E-040.8420.5030.2550.3210.0540.0220.0015.13E-062.96E-055.70E-051.26E-07rs2472304CYP1A21.01E-040.9892.22E-191.27E-313.26E-243.18E-302.90E-287.29E-220.0840.2340.7900.0010.921rs750155SULT1A10.2181.17E-126.26E-050.0040.3300.0050.0023.76E-043.83E-126.89E-086.20E-112.10E-081.25E-17rs1800764ACE0.1290.0010.0790.0110.1720.1140.3840.0010.6850.1470.2100.5800.949rs4291ACE3.27E-192.23E-269.16E-161.28E-171.51E-122.50E-181.41E-158.52E-193.55E-166.16E-165.81E-225.34E-176.88E-16rs4267385ACE3.89E-040.8754.81E-103.57E-112.23E-101.49E-121.72E-133.89E-220.0744.20E-040.1312.91E-040.004rs2108622CYP4F20.1880.1230.0020.0160.3450.0013.00E-063.05E-068.50E-097.07E-116.54E-095.88E-085.84E-10rs3093105CYP4F21.39E-421.35E-551.54E-441.21E-441.84E-49–7.51E-351.34E-396.52E-451.47E-461.85E-423.34E-422.80E-46rs8192726CYP2A63.72E-044.46E-053.48E-062.15E-050.0906.09E-061.27E-051.38E-040.0960.2130.0190.3560.019rs1051298SLC19A12.48E-051.15E-040.0920.0050.0833.25E-040.0080.0230.1960.0290.2050.3420.021rs1051296SLC19A16.24E-089.47E-080.0017.75E-050.0021.09E-061.71E-041.69E-040.0920.0250.0020.0030.003rs1131596SLC19A14.37E-071.21E-050.0060.0450.0222.49E-060.0010.0030.0020.0014.88E-042.09E-040.001rs1065852CYP2D66.20E-188.28E-301.30E-218.70E-144.41E-235.54E-144.52E-218.40E-181.55E-131.24E-221.76E-204.59E-273.85E-23rs1805123KCNH26.73E-413.21E-513.55E-413.82E-401.03E-444.00E-368.87E-389.24E-374.67E-343.55E-391.83E-373.46E-427.87E-42Bolded font indicates significant results.EAS, East Asian; SAS, South Asian; EUR, European; AFR, African; AMR, American; CDX, Chinese Dai in Xishuangbanna, China; CHB, Han Chinese in Beijing, China; CHS, Southern Han Chinese, China; JPT, Japanese in Tokyo, Japan; KHV, Kinh in Ho Chi Minh City; Vietnam; BEB, Bengali in Bangladesh; GIH, Gujarati Indian in Houston, Texas; ITU, Indian Telugu in the UK; PJL, Punjabi in Lahore, Pakistan; STU, Sri Lankan Tamil in the UK; CEU, Western European ancestry; FIN, Finnish in Finland; GBR, British in England and Scotland; IBS, Iberian populations in Spain; TSI, Toscani in Italy; ACB, African Caribbeans in Barbados; ASW, African Ancestry in Southwest USA; ESN, Esan in Nigeria; GWD, Gambian in Western Divisions, The Gambia; LWK, Luhya in Webuye, Kenya; MSL, Mende in Sierra Leone; YRI, Yoruba in Ibadan, Nigeria; CLM, Colombian in Medellin, Colombia; MXL, Mexican Ancestry in Los Angeles, Colombia; PEL, Peruvian in Lima, Peru; PUR, Puerto Rican in Puerto Rico.

The amount of difference variants between the Zhuang population and 26 populations. The size of the rectangle indicates the number of different variants between the Zhuang population and the other 26 populations from five regions.

Genotype frequency distribution differences of 26 populations compared with the Zhuang population after Bonferroni’s multiple adjustment.

Bolded font indicates significant results.

EAS, East Asian; SAS, South Asian; EUR, European; AFR, African; AMR, American; CDX, Chinese Dai in Xishuangbanna, China; CHB, Han Chinese in Beijing, China; CHS, Southern Han Chinese, China; JPT, Japanese in Tokyo, Japan; KHV, Kinh in Ho Chi Minh City; Vietnam; BEB, Bengali in Bangladesh; GIH, Gujarati Indian in Houston, Texas; ITU, Indian Telugu in the UK; PJL, Punjabi in Lahore, Pakistan; STU, Sri Lankan Tamil in the UK; CEU, Western European ancestry; FIN, Finnish in Finland; GBR, British in England and Scotland; IBS, Iberian populations in Spain; TSI, Toscani in Italy; ACB, African Caribbeans in Barbados; ASW, African Ancestry in Southwest USA; ESN, Esan in Nigeria; GWD, Gambian in Western Divisions, The Gambia; LWK, Luhya in Webuye, Kenya; MSL, Mende in Sierra Leone; YRI, Yoruba in Ibadan, Nigeria; CLM, Colombian in Medellin, Colombia; MXL, Mexican Ancestry in Los Angeles, Colombia; PEL, Peruvian in Lima, Peru; PUR, Puerto Rican in Puerto Rico.

### Genetic structure analysis of 27 populations

A model-based clustering approach was used to analyze the genetic structure of the 27 populations distributed in Africa, America, East Asia, Europe and South Asia to further analyze their relationship. Based on the Structure 2.3.1 Software, different K values ranging from 5 to 8 were hypothetically considered in structure analysis. When K = 5, the groups were divided into 5 subgroups based on the relative majority probability of assigning individuals to subgroups (subgroup 1: GWD and LWK; Subgroup 2: BEB, CEU, FIN, GBR, IBS, TSI, CLM, MXL and PUR; Subgroup 3: Zhuang, CDX, CHB, CHS, JPT, KHV and PUR; Subgroup 4: GIH, ITU, PJL and STU; Subgroup 5: ACB, ASW, ESN, MSL and YRI). It can be observed from Fig. 2 that Zhuang population have a stronger affinity with CDX, CHB, CHS, JPT, KHV and PUR. This is consistent with the results in Table 2.Figure 2Structure analysis of the genetic relationship between the Zhuang population and the other 26 populations. K denotes the possible numbers of parental population clusters. Each vertical bar represents a sample, dividing into color sections. K = 5 were utilized to evaluate the relationship between Zhuang and 26 populations.

Structure analysis of the genetic relationship between the Zhuang population and the other 26 populations. K denotes the possible numbers of parental population clusters. Each vertical bar represents a sample, dividing into color sections. K = 5 were utilized to evaluate the relationship between Zhuang and 26 populations.

The pairwise Fst values were used to assess relationships among 27 populations, as shown in Table 3 and Fig. 3A. The Fst values between the Zhuang population and the East Asian population (CDX, CHB, CHS, JPT and KHV) were small, which were 0.065, 0.068, 0.066, 0.073 and 0.067, respectively (Table 3). Smaller Fst values indicate closer relationships between the two groups and suggest that they share similar genetic backgrounds. The result is confirmed by the phylogenetic trees of 27 populations shown in Fig. 3B.Table 3Pairwise Fst values among the Zhuang and 26 populations.ZhuangCDXCHBCHSJPTKHVBEBGIHITUPJLSTUCEUFINGBRIBSTSIACBASWESNGWDLWKZhuang0.000CDX0.0650.000CHB0.0680.0130.000CHS0.0660.0090.0050.000JPT0.0730.0260.0120.0150.000KHV0.0690.0070.0130.0060.0240.000BEB0.0920.0670.0640.0640.0600.0660.000GIH0.1080.0820.0810.0790.0720.0780.0100.000ITU0.1030.0820.0760.0750.0690.0780.0060.0050.000PJL0.1220.0950.0940.0910.0820.0910.0100.0070.0060.000STU0.1080.0770.0750.0700.0650.0710.0120.0110.0070.0110.000CEU0.1400.1230.1240.1220.1110.1220.0570.0540.0580.0470.0690.000FIN0.1380.1150.1150.1100.1040.1150.0490.0420.0470.0370.0570.0100.000GBR0.1460.1280.1310.1280.1190.1280.0570.0560.0590.0470.0690.0050.0100.000IBS0.1290.1160.1190.1140.1050.1150.0420.0400.0440.0330.0530.0110.0120.0090.000TSI0.1250.1140.1170.1160.1050.1150.0480.0450.0500.0410.0590.0110.0150.0080.0060.000ACB0.2370.1820.1950.1960.1800.1880.1460.1450.1410.1360.1430.1790.1750.1860.1620.1580.000ASW0.1980.1470.1580.1610.1460.1540.1070.1070.1020.0970.1080.1380.1340.1440.1250.1210.0090.000ESN0.2830.2280.2360.2400.2210.2320.1880.1870.1790.1760.1800.2300.2250.2370.2130.2060.0130.0210.000GWD0.2800.2260.2350.2390.2190.2310.1790.1780.1700.1680.1760.2130.2080.2190.1940.1870.0110.0190.0130.000LWK0.2830.2270.2410.2420.2260.2330.1810.1790.1740.1650.1740.2220.2130.2250.1990.1940.0150.0230.0120.0160.000MSL0.2770.2220.2330.2360.2180.2270.1880.1880.1810.1770.1810.2290.2250.2350.2100.2040.0090.0220.0060.0100.016YRI0.2720.2200.2280.2330.2130.2260.1840.1830.1760.1740.1780.2220.2180.2290.2040.1970.0080.0200.0040.0080.015CLM0.1070.0770.0840.0820.0750.0800.0290.0310.0330.0280.0420.0210.0210.0200.0180.0170.1280.0930.1720.1570.164MXL0.1120.0840.0820.0870.0800.0900.0320.0420.0400.0320.0520.0400.0360.0390.0380.0370.1590.1160.2000.1900.190PEL0.1250.0910.0970.0980.1010.1020.0880.1000.0980.0960.1080.1100.1000.1100.1070.1040.2010.1650.2420.2370.231PUR0.1030.0780.0840.0820.0740.0840.0280.0350.0320.0260.0390.0230.0210.0210.0180.0170.1120.0790.1530.1390.143LWKMSLYRICLMMXLPELPURZhuangCDXCHBCHSJPTKHVBEBGIHITUPJLSTUCEUFINGBRIBSTSIACBASWESNGWDLWK0.000MSL0.0160.000YRI0.0150.0040.000CLM0.1640.1710.1650.000MXL0.1900.2010.1940.0170.000PEL0.2310.2440.2350.0580.0350.000PUR0.1430.1510.1460.0090.0190.0670.000Figure 3Fst value heamap and phylogenetic tree among 27 populations. (A) Heatmap based on the pairwise Fst values between 27 populations. (B) The phylogenetic tree was constructed by the neighboring-joining method among 27 populations.

Pairwise Fst values among the Zhuang and 26 populations.

Fst value heamap and phylogenetic tree among 27 populations. (A) Heatmap based on the pairwise Fst values between 27 populations. (B) The phylogenetic tree was constructed by the neighboring-joining method among 27 populations.

### Genotype frequencies of four significantly different SNVs

Moreover, the genotype frequency distribution of rs776746 (CYP3A5), rs4291 (ACE), rs1805123 (KCNH2), and rs1065852 (CYP2D6) in 26 populations are shown in Fig. 4. The genotype frequency of rs4291-AT in the Zhuang population is remarkably higher than that of the other 26 populations. The CC genotype frequency of rs776746 is similar to that of EUR and significantly higher than that of AFR. In the Zhuang population, the frequency of the rs1805123-GG genotype is notably higher compared to that observed in the other 26 populations. The frequency of rs1065852-GG is similar to that of KHV, CHS, CHB, and CDX, and lower than that of other populations.Figure 4The distribution of genotype frequencies for significantly different SNVs in 27 populations at the rs776746, rs4291, rs1805123 and rs1065852.

The distribution of genotype frequencies for significantly different SNVs in 27 populations at the rs776746, rs4291, rs1805123 and rs1065852.

### MAF distribution of four significantly different SNVs

Based on the allele frequencies calculated in this study, we plotted a map of the MAF distribution of VIP variants that substantially differed from the other 26 populations. According to Fig. 5, the allele frequency of rs776746 at CYP3A5 in the Zhuang population was similar to that in the European population, despite their close genetic affinity with East Asians. The G allele of rs1805123 at KCNH2 was nearly fixed in the Zhuang population and had a low frequency in other global populations. The MAF of rs1065852 (CYP2D6) was similar to that of East Asians and higher than other populations. However, there were no significant differences in T allele frequency for rs4291 at ACE among different populations.Figure 5The map of the allele frequency distribution for significantly different SNVs in 27 populations at the rs1065852, rs4291, rs776746 and rs1805123.

The map of the allele frequency distribution for significantly different SNVs in 27 populations at the rs1065852, rs4291, rs776746 and rs1805123.

### Clinical relevance of significant variants

The Table 4 presents the clinical annotation information of the VIP variants in PharmGKB. The genotype frequency of rs776746 (CYP3A5) has been shown to have an impact on the dose, toxicity and metabolism of tacrolimus19–21. Specific mutations in rs4291 (ACE) have been implicated in the metabolism of anti-hypertensive drugs such as amlodipine, sodium chlorthalidone and lisinopril22. Furthermore, they also influenced the risk of aspirin intolerance in asthmatics exposed to aspirin23. The efficacy of metoclopramide in patients with gastric disease was found to be associated with rs1805123 (KCNH2)24. Rs1065852 played an indispensable role in the regulation of the α-hydroxymetoprolol metabolism in patients with non-small cell lung cancer25.Table 4Clinical annotation of very important pharmacogenomic variants with significant differences.GeneVariantPMIDMoleculesAssociationP-valueTypePhenotypeCYP3A5rs77674623,073,468TacrolimusGenotype CC is associated with decreased dose of tacrolimus in people with Kidney Transplantation as compared to genotypes CT + TT0.016DosageKidney TransplantCYP3A5rs77674621,677,300TacrolimusAllele T is associated with increased risk of tacrolimus nephrotoxicity when treated with tacrolimus in people with Kidney Transplantation as compared to allele C0.025ToxicityKidney TransplantCYP3A5rs77674624,120,259TacrolimusGenotype CT is associated with increased dose of tacrolimus in people with Kidney Transplantation as compared to genotype CC < 0.001Metabolism/PKKidney TransplantACErs429127,546,928CaptoprilGenotype AA is associated with decreased severity of Kidney Failure when treated with captopril in people with Alzheimer Disease as compared to genotypes AT + TT0.029EfficacyAlzheimer diseaseACErs429118,727,619AspirinGenotypes AT + TT are associated with increased risk of aspirin intolerance when exposed to aspirin in people with Asthma as compared to genotype AA0.015Toxicity/ADRAsthmaACErs429120,577,119Amlodipine/lisinopril/chlorthalidoneGenotypes AA + AT are associated with decreased fasting glucose when treated with amlodipine, chlorthalidone or lisinopril in people with Hypertension as compared to genotype TT0.001EfficacyAnti-HypertensionCYP2D6rs106585210,223,777Alpha-hydroxymetoprololAllele A is associated with decreased clearance of alpha-hydroxymetoprolol in healthy individuals as compared to allele G < 0.050Metabolism/PKCarcinoma, Non-Small-Cell LungMesotheliomaCYP2D6rs106585224,528,284CitalopramescitalopramAllele A is associated with plasma concentration of S-didesmethyl-citalopram when treated with citalopram or escitalopram in people with Depressive Disorder, Major as compared to allele G2E-16OtherDepressive DisorderCYP2D6rs106585223,277,250IloperidoneGenotype GG is associated with increased QTc interval when treated with iloperidone in people with Schizophrenia as compared to genotypes AA + AG0.028OtherSchizophreniaKCNH2rs180512322,688,145MetoclopramideThe efficacy of metoclopramide in patients with gastric disease was correlated with the polymorphism of KCNH2 (rs1805123, P = 0.020) gene0.020Dose effectGastric diseaseADR: Adverse drug reactions.

Clinical annotation of very important pharmacogenomic variants with significant differences.

ADR: Adverse drug reactions.

### Prediction of functional damage in proteins

Subsequently, we used the PolyPhen-2, SNAP2, FATHMM, Mutationtaster, and Mutationassessor online databases to predicte whether the four SNVs would affect protein structure and function (Table 5). The results indicated that rs1805123 would cause a mutation from K to T at position 897 of KCNH2, however, this mutation was considered benign and less harmful to the protein in most databases. In contrast, rs1065852 caused a mutation from P to A at the 34th position of CYP2D6. The database predicted that this change would severely impair the protein's function and potentially contribute to certain diseases. Additionally, Chimera v1.16 was utilized for predicting the structure of point mutations of CYP2D6 and KCNH2, as shown in Fig. 6.Table 5The functional analysis of missense variants using PolyPhen-2, SNAP2, Mutationassessor, FATHMM, and Mutationtaster.SNVs IDGeneAA changePolyPhen-2SNAP2FATHMMMutation tasterMutationassessorScorePredicted effectScorePredicted effectCoding ScorePredicted effectProbPredictedFunc.ImpactFI scorers1805123KCNH2K897T0Benign-53Neutral0.760pathogenic0.205polymorphismlow1.735rs1065852CYP2D6P34A0.953Deleterious60Effect0.820pathogenic0.999disease causinghigh4.080Figure 6Structural prediction of point mutated proteins. (A) 3D structure of the CYP2D6 protein, with the yellow part being a SNV. (B) rs1065852 mutated local structure. (C) 3D structure of the KCNH2 protein, with the yellow part being a SNV mutation. (D) rs1805123 mutated local structure.

The functional analysis of missense variants using PolyPhen-2, SNAP2, Mutationassessor, FATHMM, and Mutationtaster.

Structural prediction of point mutated proteins. (A) 3D structure of the CYP2D6 protein, with the yellow part being a SNV. (B) rs1065852 mutated local structure. (C) 3D structure of the KCNH2 protein, with the yellow part being a SNV mutation. (D) rs1805123 mutated local structure.

### Discussion

During the development of biological sciences, it has gradually been realized that genetic differences between populations have an essential influence on drug metabolism, dosages and ADRs. This can potentially affect the efficacy of certain medications in specific populations. Pharmacogenomics research is gradually illuminating the genetic factors responsible for variations in drug utilization among diverse populations. For instance, an important study conducted by Wen et al. demonstrated that there were significant differences in allele frequencies of key genetic variants affecting drug selection and dosing between Hmong and East Asian populations26. Furthermore, pharmacogenomics studies have been reported on Mongolian27, Tibetan28, and Blang29, among others. However, there have been few pharmacogenomics studies conducted on the Zhuang population.

In this study, 200 Zhuang subjects in Yunnan Province were recruited and genotyped for 48 VIP variants on 24 candidate genes. The genotypic distribution was compared to that of 26 populations from the 1000G dataset. The results revealed significant differences in CYP3A5 (rs776746), ACE (rs4291), KCNH2 (rs1805123), and CYP2D6 (rs1065852) between Zhuang population and the other 26 populations. We also used the PharmGKB database to annotate significantly different SNVs. Our study on VIP polymorphism in the Zhuang population may provide tailored therapy for the Zhuang population.

The cytochrome P450 (CYP) superfamily is an ancient enzyme family found in hundreds of eukaryotic and prokaryotic organisms30. The human genome encodes 57 putative functional CYP genes, as well as 58 pseudogenes. Among these 57 functional human CYPs, 12 are involved in the metabolism of 70–80% commonly used drugs, including CYP2D6 and CYP3A531. The human CYP2D6 gene is relatively short, spanning only about 4.3 Kbps on the long arm of chromosome 22 (22q13.2). The CYP2D6 gene is composed of 9 exons and encodes the CYP2D6 protein, which is localized in the endoplasmic reticulum. This protein exhibited highly expressed levels in the liver, brain, intestinal tissues, and lymphocytes32. There were large population differences in the distribution of CYP2D6 alleles, which could lead to variations in drug utilization among different populations33. rs1065852 has been reported to result in reduced protein stability and a poor response to drugs such as iloperidone, atorvastatin, antidepressants, and antipsychotics34–37. Moreover, allele A was associated with decreased clearance of alpha-hydroxymetoprolol in healthy individuals and a higher plasma concentration of S-didesmethyl-citalopram when treated with citalopram or escitalopram in people with Depressive Disorder compared to allele G25,38. In this study, the MAF of rs1065852-A (54.80%) was higher than that in SAS (16.70%), EUR (20.30%), AFR (11.60%), and AMR (14.60%). In addition, the frequency of rs1065852-GG was lower than that in other populations except for EAS. Therefore, the differences in drug efficacy and safety caused by CYP2D6 rs1065852 should be taken into consideration in the Zhuang population.

CYP3A5, which is located in chromosome 7q21.1, is involved in the metabolism of many drugs. Tacrolimus, one of the substrates of CYP3A5, is widely used as an immunosuppressive agent for organ transplantation39. The expression of CYP3A5 varied among different populations, which may have an impact on drug metabolism in those populations21. One study identified that genotype CT was associated with a higher tacrolimus dose in renal transplant patients compared to genotype CC21. The results of Flores-Pérez et al. revealed that critically ill Mexican pediatric patients with the CYP3A5*3 allele variant (rs776746) had increased plasma levels of midazolam and higher drug clearance 3 h after the end of the infusion compared to carriers with the normal allele40. The study by Liang et al. pointed out that individuals with the rs776746-CC had an increased risk of amlodipine-induced peripheral edema in a dominant model among Chinese Han hypertensive patients41. In our study, the frequencies of CT, TT and CC of rs776746 were 0.5%, 9.5% and 90.0%, separately. The frequency of CT genotype in the Zhuang was lower than that in the other 26 populations, highlighting the importance of considering metabolism and absorption of specific drugs in the Zhuang population.

KCNH2 is a gene that encodes a component of voltage-activated potassium channel found in cardiac muscle, neuronal cells, and microglia. Four copies of this protein interact with a copy of the KCNE2 protein to form a functional potassium channel. Mutations in this gene can lead to long QT syndrome type 2 (LQT2)42. A recent study has identified KCNH2 p.Gly262AlafsTer98 as a novel pathogenic variant associated with long QT syndrome in a Spanish population43. In a separate study, it was found that KCNH2 mutations cause fetal biventricular densified cardiomyopathy with pulmonary stenosis and bradycardia44. The efficacy of metoclopramide in patients with gastric disease was found to be correlated with the polymorphism of KCNH2 gene (rs1805123, p = 0.020)24. A Marjamaa et al. found that allele G of rs1805123 was associated with a shorter QT interval in a Finnish population compared to the TT genotype45. In our study, the frequency of rs1805123-G was significantly higher in the Zhuang population than in the other 26 populations (88.70%). The rs1805123 causes a K-T mutation at site 897 of KCNH2. Although most databases predict this mutation to be benign, attention should be paid to the shorter/long QT interval and the dose of metoclopramide in the Zhuang population.

ACE, which encodes an enzyme, is known to participate in the regulation of blood pressure and electrolyte balance. Numerous studies have shown that ACE is closely associated with nervous system diseases46,47, cardiovascular diseases48, and hypertension49,50. In a previously published study, we found that the rs4291 genotype influenced drug dosing in the treatment of the disease. De Oliveira et al. found a correlation between the use of brain-penetrating angiotensin converting enzyme inhibitors (ACEIs) (such as captopril or perindopril) in antihypertensive therapy and rs429146. Another study has confirmed that the AA genotype of rs4291, compared to the genotype AT + TT, is associated with a reduced severity of renal failure in patients with Alzheimer's disease treated with captopril51. Furthermore, rs1800764 and rs4291 also formed haplotypes. A study discovered that ACEIs decelerated cognitive decline in individuals carrying the ACE haplotype with rs1800764-T and rs4291-A, as well as those carrying the APOE4 haplotype with either rs1800764-T or rs4291-T, regardless of changes in blood pressure52. Our study demonstrated that the allele frequency of rs4291 (ACE) differed significantly between the Zhuang population and the other 26 populations, which has been found to be associated with drug metabolism for various diseases, such as captopril, aspirin, and amlodipine. It may provide guidance for precision drug administration in the Zhuang population.

Our results are likely to complement the pharmacogenomic information of the Zhuang population and refine the study on the differences between the Zhuang population and the other 26 populations. More importantly, this study may provide certain theoretical support for drug use in the Zhuang population. Nonetheless, there are some limitations to our study. Our sample size was relatively small in this study. To design a comprehensive, systematic, disease-specific treatment protocol for the Zhuang population, we need to further expand the sample size for more in-depth studies. In addition, only Agena MassARRAY was used for genotyping in this study, and no other orthogonal method was employed to validate the sequencing, which will be utilized in subsequent studies.

### Conclusion

In short, the genotype frequencies of CYP3A5 (rs776746), ACE (rs4291), CYP2D6 (rs1065852), and KCNH2 (rs1805123) showed significant disparities between the Zhuang population and 26 other populations. Our study can not only enrich the pharmacogenomics database of the Zhuang population but also provide a theoretical basis for tailored therapy in this population and ensure safe drug use for patients.

Supplementary Tables.

Supplementary Tables.

### Supplementary Information

Supplementary Tables.

Supplementary Tables.



# SUPPLEMENTAL FILE 1: 41598_2024_Article_58092.pdf

# Preparing to download ...

[HHS Vulnerability Disclosure](https://www.hhs.gov/vulnerability-disclosure-policy/index.html)